const fs = require('fs'); //need to isntall

const k0 = "dog;virus;shit;all"
const k1 = "qwer;sda;all;calit"
const k7 = "qwer;sda;all;calit"
const k8 = "qwer;sda;all;calic"
const k2 = "ssdfs;cat;all;kolokol"
const k3 = "ssdfs;shit;money;fuck"
const k4 = "ssdfs;virus;money;dog"
const k5 = "dog;virus;kolok;palma"
const k6 = "durak;roketa;virus;ritm"
const key = [k0, k1, k2, k3, k4, k5, k6, k7, k8];

// ALl Logic at the bottom 

const example = [
    {
        "keywords": "coronavirus;genome;bioinformatics",
        "abstract": "The drastic increase in the number of coronaviruses discovered and coronavirus genomes being sequenced have given us an unprecedented opportunity to perform genomics and bioinformatics analysis on this family of viruses. Coronaviruses possess the largest genomes (26.4 to 31.7 kb) among all known RNA viruses, with G + C contents varying from 32% to 43%. Variable numbers of small ORFs are present between the various conserved genes (ORF1ab, spike, envelope, membrane and nucleocapsid) and downstream to nucleocapsid gene in different coronavirus lineages. Phylogenetically, three genera, Alphacoronavirus, Betacoronavirus and Gammacoronavirus, with Betacoronavirus consisting of subgroups A, B, C and D, exist. A fourth genus, Deltacoronavirus, which includes bulbul coronavirus HKU11, thrush coronavirus HKU12 and munia coronavirus HKU13, is emerging. Molecular clock analysis using various gene loci revealed that the time of most recent common ancestor of human/civet SARS related coronavirus to be 1999&#8211;2002, with estimated substitution rate of 4&#215;10&#8722;4 to 2&#215;10&#8722;2 substitutions per site per year. Recombination in coronaviruses was most notable between different strains of murine hepatitis virus (MHV), between different strains of infectious bronchitis virus, between MHV and bovine coronavirus, between feline coronavirus (FCoV) type I and canine coronavirus generating FCoV type II, and between the three genotypes of human coronavirus HKU1 (HCoV-HKU1). Codon usage bias in coronaviruses were observed, with HCoV-HKU1 showing the most extreme bias, and cytosine deamination and selection of CpG suppressed clones are the two major independent biological forces that shape such codon usage bias in coronaviruses.",
        "body": "Introduction\nTraditionally, viruses were characterized and classified by culture, electron microscopy and serological studies. Using these phenotypic methods, coronaviruses were defined as enveloped viruses of 120&#8211;160 nm in diameter with a crown-like appearance. The name &#8220;coronavirus&#8221; is derived from the Greek &#954;&#959;&#961;&#974;&#957;&#945;, meaning crown. Based on their antigenic relationships, coronaviruses were classified into three groups. Group 1 and 2 are composed of mammalian coronaviruses and group 3 avian coronaviruses. The invention of and advances in nucleic acid amplification technologies, automated DNA sequencing and bioinformatics tools in the recent two decades have revolutionized the characterization and classification of all kinds of infectious disease agents. Using molecular methods, coronaviruses are classified as positive-sense, single-stranded RNA viruses. Furthermore, the results of using phylogenetic methods for classification also supported the group boundaries of the traditional antigenic classification. Phylogenetic methods have also enabled the classification of SARS-related coronavirus (SARSr-CoV) as a subgroup of group 2, group 2b, coronavirus; as well as the discovery of group 2c, 2d, 3b and 3c coronaviruses [1&#8211;3]. Recently, the Coronavirus Study Group of the International Committee for Taxonomy of Viruses has proposed three genera, Alphacoronavirus, Betacoronavirus and Gammacoronavirus, to replace these three traditional groups of coronaviruses [4].The first complete genome of coronavirus, mouse hepatitis virus (MHV), was sequenced more than 50 years after it was isolated. Before the SARS epidemic in 2003, there were less than 10 coronaviruses with complete genome sequences available. These include two human coronaviruses (HCoV-229E and HCoV-OC43), four other mammalian coronaviruses [MHV, bovine coronavirus (BCoV), transmissible gastroenteritis virus (TGEV), porcine epidemic diarrhea virus (PEDV)], and one avian coronavirus (IBV). The SARS epidemic that originated from southern China in 2003 has boosted interest in all areas of coronavirus research, most notably, coronavirus biodiversity and genomics [5&#8211;7]. After the SARS epidemic, up to April 2010, 15 novel coronaviruses were discovered with their complete genomes sequenced. Among these 15 previously unrecognized coronaviruses were two globally distributed human coronaviruses, human coronavirus NL63 (HCoV-NL63) and human coronavirus HKU1 (HCoV-HKU1) [8&#8211;10]; 10 other mammalian coronaviruses, SARS-related Rhinolophus bat coronavirus (SARSr-Rh-BatCoV), Rhinolophus bat coronavirus HKU2 (Rh-BatCoV HKU2), Tylonycteris bat coronavirus HKU4 (Ty-BatCoV HKU4), Pipistrellus bat coronavirus HKU5 (Pi-BatCoV HKU5), Miniopterus bat coronavirus HKU8 (Mi-BatCoV HKU8), Rousettus bat coronavirus HKU9 (Ro-BatCoV HKU9), Scotophilus bat coronavirus 512 (Sc-BatCoV 512), Miniopterus bat coronavirus 1A/B (Mi-BatCoV 1A/B), equine coronavirus (ECoV) and beluga whale coronavirus SW1 [3,6,11&#8211;15]; and three avian coronaviruses, bulbul coronavirus HKU11 (BuCoV HKU11), thrush coronavirus HKU12 (ThCoV HKU12) and munia coronavirus HKU13 (MunCoV HKU13) [2]. Most of these genomes were sequenced using the RNA extracted directly from the clinical specimens, such as nasopharyngeal aspirate or stool, as the template, while the viruses themselves were still non-cultivable [2,3,6,11&#8211;15]. This provided more accurate analysis of the in situ viral genomes avoiding mutational bias during in vitro viral replication. These sequence efforts have resulted in a marked increase in the number of coronavirus genomes and have given us an unprecedented opportunity to understand this family of virus at the genomic and in silico levels. These understandings have also led to generation of further hypotheses and experiments in the laboratory. In this article, we reviewed our current understanding on the genomics and bioinformatics analysis of coronaviruses. Details of the bioinformatics tools will not be discussed.\nGenomics\nCoronaviruses possess the largest genomes [26.4 kb (ThCoV HKU12) to 31.7 kb (SW1)] among all known RNA viruses (Figure 1) [2,13,16]. The large genome has given this family of virus extra plasticity in accommodating and modifying genes. The G + C contents of coronavirus genomes vary from 32% (HCoV-HKU1) to 43% (Pi-BatCoV HKU5 and MunCoV HKU13) (Table 1) [2,3,10]. Both the 5&#8242; and 3&#8242; ends of coronavirus genomes contain short untranslated regions. For the coding regions, the genome organizations of all coronaviruses are similar, with the characteristic gene order 5&#8242;-replicase ORF1ab, spike (S), envelope (E), membrane (M), nucleocapsid (N)-3&#8242;, although variable numbers of additional ORFs are present in each subgroup of coronavirus (Table 1, Figure 1). A transcription regulatory sequence (TRS) motif is present at the 3&#8242; end of the leader sequence preceding most ORFs (Table 1). The TRS motifs are thought to be important for a &#8220;copy-choice&#8221; mechanism that mediates the unique random template switching during RNA replication, resulting in a high frequency of homologous RNA recombination in coronaviruses [17].\nORF1ab\nORF1ab of coronaviruses occupy about two thirds of their genomes. It encodes the replicase polyprotein and is translated from ORF1a (11826 to 13425 nt) and ORF1b (7983 to 8157 nt). In all coronaviruses, a slippery sequence (UUUAAAC), followed by sequences that form a putative pseudoknot structure, are present at the junction between ORF1a and ORF1b. Translation occurs by a -1 RNA-mediated ribosomal frameshift at the end of the slippery sequence. Instead of reading the transcript as UUUAAACGGG, it will be read as UUUAAACCGGG. The replicase polyprotein is cleaved by papain-like protease(s) (PLpro) and 3C-like protease (3CLpro), proteins encoded by ORF1ab of the coronavirus genome, at consensus cleavage sites, into 15 to 16 non-structural proteins (nsps) named nsp1, nsp2, nsp3, etc (Table 1). As the number of coronavirus genomes is expanding, novel cleavage sites have been discovered [3,18]. Some of these non-structural proteins encode proteins of essential functions, such as PLpro (nsp3), 3CLpro (nsp5), RNA-dependent RNA polymerase (Pol) (nsp12) and helicase (nsp13) (Figure 1). The genomes of all known members of Alphacoronavirus and Betacoronavirus subgroup A possess two PLpro (PL1pro and PL2pro), while those of all known members of Betacoronavirus subgroup B, C and D and Gammacoronavirus possess only one PLpro (Table 1, Figure 1). The gene sequences that encode these conserved proteins are frequently used for phylogenetic analysis.In addition to the nsps with essential functions, bioinformatics analysis of some other nsps revealed their putative functions. Downstream to PLpro or PL1pro in nsp3 is the X domain which contains putative ADP-ribose 1&#8243;-phosphatase (ADRP) activity [1]. In other microorganisms, such as Saccharomyces cerevisiae and other eukaryotes, ADRP and its functionally related enzyme cyclic nucleotide phosphodiesterase (CPDase), were important for tRNA processing [19]. ADP-ribose 1&#8243;,2&#8243;-cyclic phosphate (Appr>p) is produced as a result of tRNA splicing. Appr>p is in turn converted to ADP-ribose 1&#8243;-phosphate (Appr-1&#8243;p) by CPDase and Appr-&#8243;p is then further processed by ADRP. As for nsp13, nsp14 and nsp15, they possess a putative 3&#8242;-to-5&#8242; exonuclease (ExoN) domain of the DEDD superfamily [1], a putative poly(U)-specific endoribonuclease (XendoU) domain, and a putative S-adenosylmethionine-dependent ribose 2&#8242;-O-methyltransferase (2&#8242;-O-MT) domain of the RrmJ family respectively [1]. ADRP, CPDase, ExoN, XendoU and 2&#8242;-O-MT are enzymes in RNA processing pathways. Contrary to the pre-tRNA splicing pathway that ADRP and CPDase belong to, ExoN, XendoU and 2&#8242;-O-MT are enzymes in a small nucleolar RNA processing and utilization pathway.\nHaemagglutinin esterase\nIn all members of Betacoronavirus subgroup A, a haemagglutinin esterase (HE) gene, which encodes a glycoprotein with neuraminate O-acetyl-esterase activity and the active site FGDS, is present downstream to ORF1ab and upstream to S gene (Figure 1). The HE gene of coronavirus is believed to be acquired from influenza C virus, and is the most notable example of acquisition of new genes from non-coronavirus RNA donors by heterologous recombination [20]. The presence of HE genes exclusively in members of Betacoronavirus subgroup A, but not members of Betacoronavirus subgroup B, C and D suggested that the recombination had probably occurred in the ancestor of members of Betacoronavirus subgroup A, after diverging from the ancestor of other subgroups of Betacoronavirus.\nSpike\nThe S proteins are responsible for the &#8220;spikes&#8221; present on the surface of coronaviruses and give this family of virus the characteristic crown-like appearance under electron microscopy. The S proteins are type I membrane glycoproteins with signal peptides. The S proteins are used for receptor binding and viral entry, and are the proteins with the most variable sequences in the coronavirus genomes. In some coronaviruses, the S proteins are cleaved into the S1 and S2 domains at consensus cleavage site (RRSRR of BCoV, RRSR of HCoV-OC43, RRKRR of HCoV-HKU1, RSRR of PHEV, RRADR of MHV, RRFRR of SDAV and RRFRR of IBV) (Table 1), with the sequences of the S1 domains much more variable than the S2 domains. In all coronaviruses, most of the S protein is exposed on the outside of the virus, with a short transmembrane domain at the C terminus, followed by a short cytoplasmic tail rich in cysteine residues. Two heptad repeats are present at the C termini of the extracellular parts of the S proteins. At the moment, no bioinformatics tool is available for accurate prediction of the receptor by analyzing the amino acid sequences of the S proteins of the corresponding coronaviruses.\nEnvelope and membrane\nThe E and M proteins are small transmembrane proteins associated with the envelope of all coronaviruses. In some coronaviruses, such as MHV and SDAV and possibly HCoV-HKU1, the translation of the E protein is cap-independent, via an internal ribosomal entry site. Although these two genes are conserved among all coronaviruses, they are not good targets for phylogenetic studies because of their short sequences.\nNucleocapsid\nSimilar to the conserved proteins encoded by ORF1ab, the N gene is also another common target for phylogenetic analysis. Due to its immunogenicity, it is also a common target for cloning and generation of recombinant proteins for serological assays.\nOther small ORFS\nVariable numbers of small ORFs are present between the various conserved genes in different lineages in the Coronaviridae family (Table 1, Figure 1). In some coronaviruses, small ORFs are present downstream to the N gene (Table 1, Figure 1). Most of these small ORFs are of unknown function. One exception is the small ORFs downstream to N in feline infectious peritonitis virus (FIPV) and TGEV, which are important for virulence and viral replication/assembly respectively [21&#8211;23]. Another notable exception is the 3a protein of SARSr-CoV, which forms a transmembrane homotetramer complex with ion channel function and modulates virus release [24]. For some of these small ORFs, such as ORF3a and ORF8 of SARSr-CoV genomes, their sequences are as highly variable as those of the S proteins. In particular, the most significant difference between human SARSr-CoV and civet SARSr-CoV genomes was a 29-bp deletion in the ORF8 of human SARSr-CoV [25].\nPhylogeny\nThe first impression of the phylogenetic position of a strain or species of coronavirus is usually acquired by constructing a phylogenetic tree using a short fragment of a conserved gene, such as Pol or N. However, this can sometimes be misleading because the results of phylogenetic analysis using different genes or characters can be different. When SARSr-CoV was first discovered, it was proposed that it constituted a fourth group of coronavirus [26,27]. However, analyses of the amino-terminal domain of S of SARSr-CoV revealed that 19 out of the 20 cysteine residues were spatially conserved with those of the consensus sequence for Betacoronavirus [28]. On the other hand, only five of the cysteine residues were spatially conserved with those of the consensus sequences in Alphacoronavirus and Gammacoronavirus [28]. Furthermore, subsequent phylogenetic analysis using both complete genome sequence and proteomic approaches, it was concluded that SARSr-CoV is probably an early split-off from the Betacoronavirus lineage [1], and SARSr-CoV was subsequently classified as Betacoronavirus subgroup B and the historical Betacoronavirus as Betacoronavirus subgroup A. Therefore, the phylogenetic position of a coronavirus is best appreciated and confirmed by constructing phylogenetic trees using different genes in the coronavirus genome. The most commonly used genes along the coronavirus genome for phylogenetic studies include chymotrypsin-like protease, Pol (Figure 2), helicase, S and N, because these genes are present in all coronavirus genomes and are of significant length. The envelope and membrane genes, although present in all coronavirus genomes, are too short for phylogenetic studies. It is noteworthy that the cluster formed by the three novel avian coronaviruses BuCoV HKU11, ThCoV HKU12 and MunCoV HKU13, which was originated proposed as group 3c [2], might represent a new coronavirus genus provisionally designated Deltacoronavirus (Figure 2).Using this approach of multiple gene phylogenetic studies, unique phylogeny of individual gene that may have biological significance may be discovered. During our phylogenetic study on Rh-BatCoV HKU2, another coronavirus that has was also found in the stool samples of Chinese horseshoe bats, its unique S protein phylogenetically distinct from the rest of the genome was discovered [15]. The S protein of Rh-BatCoV HKU2 is the shortest among S proteins of all coronaviruses and had less than 30% amino acid identities to those of all known coronaviruses, in contrast to other genes that showed higher amino acid identities to the corresponding genes in other members of Alphacoronavirus. When the S protein of Rh-BatCoV HKU2 is aligned with those of other members of Alphacoronavirus, many of the amino acid residues conserved among and specific to Alphacoronavirus were not found. Rather, the S protein of Rh-BatCoV HKU2 shares the two conserved regions of deletions both of 14 amino acids among members of Betacoronavirus in its C-terminus, suggesting that this segment of the S protein of Rh-BatCoV HKU2 may have co-evolved with the corresponding regions in Betacoronavirus. Most interestingly, a short peptide of 15 amino acids in the S protein of Rh-BatCoV HKU2 was found to be homologous to a corresponding peptide within the RBM in the S1 domain of SARSr-CoV. A similar peptide was also observed in SARSr-Rh-BatCoV, but not in any other known coronaviruses. These suggested that there is a common evolutionary origin in the S protein of SARSr-CoV, SARSr-Rh-BatCoV and Rh-BatCoV HKU2, and Rh-BatCoV HKU2 might have acquired its unique S protein from a yet unidentified coronavirus through recombination.\nEvolutionary rate and divergence\nIn 1992, Sanchez et al. analyzed 13 enteric and respiratory TGEV related isolates and estimated the mutation rate of TGEV to be 7&#215;10&#8722;4 nucleotide substitutions per site per year [29]. 1n 2005, using linear regression, maximum likelihood and Bayesian inference methods, Vijgen et al. estimated the rate of evolution in BCoV to be 4.3 (95% confidence internal 2.7 to 6.0) &#215;10&#8722;4 nucleotide substitutions per site per year [30]. The estimation of time of divergence was first extensively used in coronaviruses after the SARS epidemic for estimating the date of interspecies jumping of SARSr-CoV from civets to humans and that from BCoV to HCoV-OC43 [31,32]. Subsequently, when various novel human and animal coronaviruses were discovered, evolutionary rates and divergence time in the Coronaviridae family were estimated by various groups using different approaches [31,33&#8211;35]. Although Bayesian inference in BEAST is probably the most widely accepted approach and was used by most researchers, the use of different genes (ORF1ab, helicase, S and N genes) and datasets by different groups have resulted in considerable difference in the estimated history of coronaviruses. It was found that the S and N genes of PHEV, BCoV and HCoV-OC43 evolved at different rates, and the divergence time of the PHEV lineage and the HCoV-OC43 and BCoV lineage based on these two rates were 100 years different [31]. One group, using the helicase gene for analysis, has estimated the life history of coronaviruses to be as short as about 420 years [35].Recently, we used the uncorrelated exponentially distributed relaxed clock model (UCED) in BEAST version 1.4 [36] to estimate the time of divergence of SARSr-CoV based on an alignment of a large set of SARSr-Rh-BatCoV ORF1 sequences collected over a period of five years. Under this model, the rates were allowed to vary at each branch drawn independently from an exponential distribution. Using this model and large dataset, the time of emergence of SARSr-CoV was at 1972, about 31 years before the SARS epidemic; that of SARSr-CoV in civet was at 1995, about eight years before the SARS epidemic; and the most recent common ancestor date of human and civet SARSr-CoV was estimated to be 2001.36, which was comparable to the dates estimated by other groups (Table 2) [37].\nRecombination analysis\nAs a result of their unique random template switching during RNA replication, thought to be mediated by a &#8220;copy-choice&#8221; mechanism, coronaviruses have a high frequency of homologous RNA recombination [41,42]. Recombination in coronaviruses was first recognized between different strains of MHV and subsequently in other coronaviruses such as IBV, between MHV and BCoV, and between feline coronavirus (FCoV) type I and canine coronavirus (CCoV) [43&#8211;46]. As shown below, such recombination can result in the generation of coronavirus species or different genotypes within a coronavirus species. In our experience, the possibility of homologous RNA recombination and the possible part of the genome that recombination has taken place are usually first appreciated using bootscan analysis or phylogenetic analysis using different parts of the coronavirus genome. Other methods for recombination analysis, such as those in the RDP3 package, are also available. Then, the exact site of homologous RNA recombination would be best revealed by multiple sequence alignment.The best documented example of generation of coronavirus species through homologous RNA recombination is the generation of FCoV type II by double recombination between FCoV type I and CCoV. It was first observed that the sequence of the S protein in FCoV type II was closely related to that of CCoV [47,48] but the sequence downstream of the E gene in FCoV type II was more closely related to that of FCoV type I strain than to CCoV [49,50]. This observation suggested that there might have been a homologous RNA recombination event between the genomes of CCoV and FCoV type I, resulting in the generation of FCoV type II. Further analysis by multiple sequence alignments pinpointed the site of recombination to a region in the E gene. A few years later, an additional recombination region in the Pol gene was also discovered, and it was concluded that FCoV type II originated from two recombination events between the genomes of CCoV and FCoV type I [43].As for the generation of different genotypes in a coronavirus species through homologous RNA recombination, the best documented example is HCoV-HKU1. The possibility of homologous RNA recombination was first suspected when a few strains of HCoV-HKU1 showed differential clustering when the Pol, S and N genes were used for phylogenetic tree construction [51]. This observation has led to our subsequent study on complete genome sequencing of 22 strains of HCoV-HKU1. Recombination analysis by bootscan analysis and phylogenetic analysis using different parts of the 22 complete genomes revealed extensive recombination in different parts of the genomes, resulting in the generation of three genotypes, A, B and C, of HCoV-HKU1 [52]. Using multiple sequence alignment, two sites of recombination were pinpointed. The first one was observed in a stretch of 143 nucleotides near the 3&#8242; end of nsp6, where recombination between HCoV-HKU1 genotype B and genotype C has generated genotype A; and the second one in another stretch of 29 nucleotides near the 3&#8242; end of nsp16, where recombination between HCoV-HKU1 genotype A and genotype B has generated genotype C [52].\nCodon usage bias\nRecently, using the complete genome sequences of the 19 coronavirus genomes, we analyzed the codon usage bias in coronaviruses as well as selection of CpG suppressed clones by the immune system and cytosine deamination being the two major independent biochemical and biological selective forces that has shaped such codon usage bias [53]. In the study, we showed that the mean CpG relative abundance in the coronavirus genomes is markedly suppressed [53]. However, we observed that only CpG containing codons in the context of purine-CpG (ACG and GCG), pyrimidine-CpG (UCG and CCG) and CpG-purine (CGA and CGG); but not CpG-pyrimidine (CGU and CGC); are suppressed. However, when trinucleotide frequencies were analyzed in the 19 coronavirus genomes, all the eight trinucleotides with CpG were suppressed [53]. These indicate that another force that has led to an increase use of CGU and CGC as codons for arginine, but does not act on trinucleotides over the whole genome in general, is probably present. Furthermore, this force is probably unrelated to the relative abundance of the corresponding tRNA molecules in the hosts of the coronaviruses, as the pattern of bias in the hosts is not the same as that in the coronaviruses.In addition to CpG suppression, marked cytosine deamination is also observed in all the 19 coronavirus genomes [53]. Using the six amino acids that are only encoded by NNC or NNU (asparagine, histidine, aspartic acid, tyrosine, cysteine and phenylalanine), hence excluding most other pressures that may affect the relative abundance of cytosine and uracil, it was observed that all NNU are markedly over represented with usage fractions of more than 0.700, whereas the usage fractions of all NNC are less than 0.300 [53]. For all codons that encode the same amino acid and with either U or C in any position, the usage fraction of the codon that uses U is always higher than the one that uses C in all coronaviruses. Furthermore, the percentage of U showed strong inverse relationships with the percentage of C in the coronavirus genomes. These suggest that cytosine deamination is another important biochemical force that shaped coronavirus evolution.Interestingly, among all the 19 coronaviruses, HCoV-HKU1 showed the most extreme codon usage bias. HCoV-HKU1 is the only coronavirus that had effective number of codons outside the mean &#177; 2 standard deviations range. In addition, HCoV-HKU1 also possessed the lowest G + C content, highest GC skew, lowest percentages of G and C and highest percentage of U among all coronavirus genomes. Furthermore, HCoV-HKU1 showed extremely high NNU/NNC ratio of 8.835. The underlying mechanism for the extreme codon usage bias, cytosine deamination and G + C content in HCoV-HKU1 is intriguing.\nDatabase\nRapid and accurate batch sequence retrieval is always the cornerstone and bottleneck for all kinds of comparative genomics and bioinformatics analysis. During the process of batch sequence retrieval for comparative genomics and other bioinformatics analysis of the coronavirus genomes that we have sequenced, we encountered a number of major problems about the coronavirus sequences in GenBank and other coronavirus databases. First, in GenBank, the non-structural proteins encoded by ORF1ab are not annotated. Second, in all databases, the annotations for the non-structural proteins encoded by ORFs downstream to ORF1ab are often confusing because they are not annotated using a standardized system. Third, multiple accession numbers are often present for reference sequences. These problems will often lead to confusion during sequence retrieval. Fourth, coronaviruses, especially SARSr-CoV, amplified from different specimens may contain the same gene or genome sequences, which will lead to redundant work when they are analyzed. In view of these problems, we have developed a comprehensive database, CoVDB, of annotated coronavirus genes and genomes, which offers rapid, efficient and user-friendly batch sequence retrieval and analysis [54]. In CoVDB, first, annotations on all non-structural proteins in the polyprotein encoded by ORF1ab of every single sequence were performed. Second, annotation was performed for the non-structural proteins encoded by ORFs downstream to ORF1ab using a standardized system. Third, all sequences with identical nucleotide sequences were labeled and one can choose to show or not to show strains with identical sequences. Fourth, this database contains not only complete coronavirus genome sequences, but also incomplete genomes and their genes. This is useful because some genes of coronaviruses, such as Pol, S and N, are sequenced much more frequently than others because they are either most conserved or least conserved, and therefore are particularly important for primers design for RT-PCR assays and evolutionary studies.\nConcluding remarks\nAfter the SARS epidemic, there has been a marked increase in the number of coronaviruses discovered and coronavirus genomes being sequenced. This increase in the number of coronavirus species and genomes, a comprehensive and user-friendly database for efficient sequence retrieval, and the ever improving bioinformatics tools have enabled us to perform meaningful genomic, phylogenetic, evolutionary rate and divergence, recombination, and other bioinformatics analyses on the Coronaviridae family. Three genera, Alphacoronavirus, Betacoronavirus and Gammacoronavirus, have been used to replace the traditional group 1, 2 and 3 coronaviruses. A fourth genus, Deltacoronavirus, which includes BuCoV HKU11, ThCoV HKU12 and MunCoV HKU13, is likely to emerge. Under this new classification system, bat coronaviruses dominate the Alphacoronavirus and Betacoronavirus genera and bird coronaviruses dominate the Gammacoronavirus and Deltacoronavirus genera. This huge diversity of coronaviruses in bats and birds has made them excellent gene pools for coronaviruses in these four genera [55].",
        "pmid": "21994708",
        "title": "Coronavirus Genomics and Bioinformatics Analysis",
        "journal_title": "Viruses",
        "authors": "Patrick C. Y. Woo;Yi Huang;Susanna K. P. Lau;Kwok-Yung Yuen",
        "doi": "10.3390/v2081803"
    },
    {
        "keywords": "",
        "abstract": "The recent SARS epidemic has boosted interest in the discovery of novel human and animal coronaviruses. By July 2007, more than 3000 coronavirus sequence records, including 264 complete genomes, are available in GenBank. The number of coronavirus species with complete genomes available has increased from 9 in 2003 to 25 in 2007, of which six, including coronavirus HKU1, bat SARS coronavirus, group 1 bat coronavirus HKU2, groups 2c and 2d coronaviruses, were sequenced by our laboratory. To overcome the problems we encountered in the existing databases during comparative sequence analysis, we built a comprehensive database, CoVDB (http://covdb.microbiology.hku.hk), of annotated coronavirus genes and genomes. CoVDB provides a convenient platform for rapid and accurate batch sequence retrieval, the cornerstone and bottleneck for comparative gene or genome analysis. Sequences can be directly downloaded from the website in FASTA format. CoVDB also provides detailed annotation of all coronavirus sequences using a standardized nomenclature system, and overcomes the problems of duplicated and identical sequences in other databases. For complete genomes, a single representative sequence for each species is available for comparative analysis such as phylogenetic studies. With the annotated sequences in CoVDB, more specific blast search results can be generated for efficient downstream analysis.",
        "body": "INTRODUCTION\nCoronaviruses are found in a wide variety of animals and are associated with respiratory, enteric, hepatic and neurological diseases of varying severity. Based on genotypic and serological characterization, coronaviruses were divided into three distinct groups (1&#8211;3). As a result of the unique mechanism of viral replication, coronaviruses have a high frequency of recombination (2,4).The recent severe acute respiratory syndrome (SARS) epidemic, the discovery of SARS coronavirus (SARS-CoV) and identification of SARS-CoV-like viruses from Himalayan palm civets and a raccoon dog from wild live markets in China have led to a boost in interest on discovery of novel coronaviruses in both humans and animals (5&#8211;9) (Figure 1). For human coronaviruses, a novel group 1 human coronavirus, human coronavirus NL63 (HCoV-NL63) was reported in 2004 (10,11), while we described the discovery, complete genome sequence and genetic diversity of a novel group 2 human coronavirus, coronavirus HKU1 (CoV-HKU1) in 2005 (4,12&#8211;14). As for animal coronaviruses, six group 1 (15&#8211;17), four group 2, including bat SARS-CoV and two new subgroups of group 2 coronaviruses (6,8,18,19), and 11 group 3 (20&#8211;23) coronaviruses have recently been described.\nFigure 1.Number of coronavirus sequences in GenBank from 1984 to 2006.By July 2007, more than 3000 coronavirus sequence records, including a total of 264 complete genomes, are available in GenBank (24). Among the 25 coronavirus species with complete genome sequence available, six were sequenced by our group, including CoV-HKU1 and bat SARS-CoV (13,16,18,19). Furthermore, we defined two novel subgroups of group 2 coronavirus (18). During the process of batch sequence retrieval for comparative genome analysis of the coronavirus genomes that we sequenced, we encountered several major problems about the coronavirus sequences in GenBank as well as other coronavirus databases (Coronaviridae Bioinformatics Resource, http://athena.bioc.uvic.ca/database.php?db=coronaviridae; PATRIC http://patric.vbi.vt.edu) (25). First, in GenBank, the non-structural proteins in the polyprotein encoded by orf1ab were not annotated. Second, in all databases, for the non-structural proteins encoded by ORFs downstream to orf1ab, the annotations are often confusing because they are not annotated using a standardized system. Third, multiple accession numbers are often present for reference sequences (26). These problems often lead to confusion when sequence retrieval is performed. Fourth, coronaviruses, especially SARS-CoV, amplified from different specimens may contain the same genome or gene sequences. These sequences usually lead to redundant work when they are analyzed.In view of these problems, we started to develop our own database for coronavirus gene and genome sequences in 2005. In this database, CoVDB, we sought to create a user-friendly platform for efficient batch sequence retrieval, which is crucial for comparative genome analysis. In this article, we describe this comprehensive database of annotated coronavirus genes and genomes, which provides a central source of information about coronaviruses. To further increase the usefulness of CoVDB, commonly used bioinformatics tools were also included for analysis of the sequence data.\nMATERIALS AND METHODS\n\nDatabase description\n\nSequence data\nCoVDB is a web-based coronavirus database. Data of CoVDB is stored and managed by MySQL database management system. By July 2007, CoVDB contains 3982 coronavirus sequences and one torovirus genome sequence. Two hundred and sixty-four of them are complete genomes and the rest are partial genomes or genes. All data were retrieved from GenBank using modules of bioperl. We annotated sequences without gene information or non-structural protein boundary and labeled the 5&#8242; and 3&#8242; untranslated regions (UTRs) of the genomes. By July 2007, CoVDB contains 12 344 genes and UTRs.\nInformation on coronavirus genome characteristics\nIn addition to the two sequence retrieval pages, CoVDB collects information on coronavirus sequence characteristics, including genome organization, a brief description on each complete coronavirus genome, GC content, polyprotein cleavage sites, transcription regulatory sequences, acidic tandem repeat sequences and known RNA structures. These pieces of information can be accessed by clicking &#8216;Genome&#8217; in the top menu bar of CoVDB. In the &#8216;Tools&#8217; page, blast similarity search (27) against annotated coronavirus sequences in CoVDB can be performed and other commonly used tools are also provided.\nFunctionality of the database\n\nBatch sequence retrieval\nThe main goal for setting up CoVDB is to provide a convenient and efficient platform for retrieving batches of coronavirus gene sequences. The interfaces of the database are simple and user friendly. All genes and genomes contain links to GenBank and/or pubmed. CoVDB contains two main pages for sequence retrieval. From the homepage, one can enter the first main page for retrieval of complete genomes and their genes by clicking &#8216;CoVDB&#8217; (Figure 2a). From this page, users can obtain genes from specific coronavirus species by selecting the corresponding check boxes. We defined one representative genome from each species as the &#8216;Type strain&#8217;. Most of the time, this &#8216;Type strain&#8217; is the one assigned as the reference sequence in GenBank. By choosing the &#8216;Type strain only&#8217; option, users can obtain one gene sequence per species and construct phylogenetic tree or perform other comparisons. An example of retrieving complete genome or a specific gene of complete genome of selected species is shown in Figure 2b and c.\nFigure 2.Screenshots of CoVDB complete genome retrieval pages. (a) Specific gene can be retrieved using the pull-down list at the left lower corner. The number in brackets indicates the number of complete genomes for that coronavirus. (b) Example of showing genomes of selected species (some group 2a coronaviruses and SARS-CoV-related coronaviruses). Default is to show the &#8216;Type strain&#8217; for each species only. The columns NCBIacc and PMID link to GenBank and pubmed, respectively. (c) Example of showing S gene of selected species by choosing S in the pull-down list. For genes downstream to orf1ab, sequences upstream to the initiation codons can also be retrieved from this result page. This function is particularly useful for the detection of transcription regulatory sequences.From the page for retrieval of complete genomes and their genes, one can enter the second main page for retrieval of all complete and/or incomplete genes of a coronavirus (Figure 3a) by clicking &#8216;From all groups of genes&#8217;. In this page, all the gene sequences are grouped vertically according to which coronavirus group and subgroup they belong to, and horizontally by the names of the genes. The option &#8216;Exclude partial CDS&#8217; can be used if only complete genes are required. An example of retrieving all the sequence of a particular gene for a group of coronavirus is shown in Figure 3b. If the translated sequence of a selected gene has more than one stop codon which is probably due to sequencing error, the number in the &#8216;Length&#8217; column of this gene will be marked in red.\nFigure 3.Screenshots of all gene retrieval pages. (a) Gene sequences are grouped vertically according to which coronavirus group and subgroup they belong to, and horizontally by the name of the genes. The numbers next to each checkbox indicates the number of that gene in CoVDB. The option &#8216;Exclude partial CDS&#8217; can be used if only complete genes are required. (b) Example of showing the 15 sequences of nsp13 in group 3 coronaviruses. The first column is CoVDB gene id. In the Uniq column, &#8216;Uniq&#8217; will be shown if there is no other identical sequence in CoVDB. Otherwise, gene id of the sequences identical to it will be shown.\nPolyprotein annotation\nIn all coronavirus genomes, orf1ab occupies two-thirds of the genome and it is translated as a polyprotein. This polyprotein is post-translationally cleaved by 3C-like protease (3CLpro) and papain-like protease (PLpro) into 15&#8211;16 non-structural proteins. Some of the non-structural proteins, such as RNA-dependent RNA polymerase, helicase, 3CLpro and PLpro are essential for replication or virulence of the coronavirus, although the functions of others are still unclear. Due to the essentiality of the non-structural proteins, these sequences are often used for evolutionary analysis, primer design, etc. However, except for the reference sequences, detailed cleavage site information is not provided for the non-structural proteins in other sequences in GenBank. Since it has been shown that 3CLpro and PLpro of coronavirus cleave at conserved specific amino acids, the putative cleavage sites of the 15&#8211;16 non-structural proteins can be predicted by multiple sequence alignment. Using these pieces of information, we have annotated these non-structural proteins in all the coronavirus sequences for easy retrieval in CoVDB.\nProtein/gene name unification\nBy convention, all non-structural proteins in the polyprotein encoded by orf1ab are named as &#8216;nsp&#8217;, with each protein numbered consecutively starting from the 5&#8242; end (nsp1&#8211;nsp16). The structural proteins after the polyprotein are hemagglutinin esterase (HE, in group 2a coronaviruses), spike glycoprotein (S), envelope protein (E), membrane protein (M) and nucleocapsid protein (N). However, there is no unified naming system for the non-structural proteins encoded by ORFs downstream to orf1ab. This lack of a unified system greatly reduces the stability and accuracy of ortholog retrieval.In CoVDB, with the aim of facilitating gene retrieval, we tried to unify the naming of these non-structural proteins from different groups of coronaviruses. On the other hand, we have also tried to avoid radical changes in the names that may lead to confusion. In CoVDB, these non-structural proteins are named as NS2a, NS3x, NS4x, NS5x and NS7x (x = a, b, c,&#8230;). NS2a denotes the ORF between orf1ab and HE of group 2a coronaviruses. NS3x denotes the ORFs between S and E of groups 1, 2c, 2d and 3 coronaviruses. In most of these coronaviruses, there are two NS3x, named NS3a and NS3b. However, in group 1 coronaviruses, the genomes of some members (e.g. HCoV-NL63, PEDV) contain only one ORF between S and E. When we compared their putative amino acid sequences to the corresponding ones in other group 1 coronavirus genomes using BLAST, as well as searching for conserved domains using motifscan, results showed that the putative proteins encoded by these ORFs belonged to a protein family in Pfam originally assigned as &#8216;Corona_NS3b&#8217; (accession number PF03053). Therefore, we named these ORFs as NS3b. NS4x denotes the ORFs between S and E of group 2a coronaviruses. NS5x denotes the ORFs between M and N of group 3 coronaviruses. One exception is NS5a of group 2a coronaviruses. Traditionally, this name denotes an ORF upstream of E in group 2a coronaviruses. Therefore, we have kept this name for that ORF in CoVDB. NS7x denotes the ORFs downstream of N gene. It is important to note that due to variations in genome organizations among different groups of coronaviruses (Table 1), NS genes with the same name in different coronavirus groups may not be orthologs of each other. The complete genome gene search page of CoVDB contains a link to a Gene synonyms page, which includes a list of synonymous names of the various genes in the coronavirus genomes.\nTable 1.Genome organization of different groups of coronavirusGroupOrganizations15&#8242;UTR-nsp1-16-S-NS3x-E-M-N-(NS7x)-3&#8242;UTR2a5&#8242;UTR-nsp1-16-(NS2a)-HE-S-(NS4x)-NS5a-E-M-N-3&#8242;UTR2b5&#8242;UTR-nsp1-16-S-sars3x-E-M-sars6-sars7x-sars8x-N-3&#8242;UTR2c5&#8242;UTR-nsp1-16-S-NS3x-E-M-N-3&#8242;UTR2d5&#8242;UTR-nsp1-16-S-NS3x-E-M-N-(NS7x)-3&#8242;UTR35&#8242;UTR-nsp1-16-S-NS3x-E-M-NS5x-N-(NS7x)-3&#8242;UTR\nIdentical sequence labeling\nSequence redundancy is another problem of coronavirus sequences in public nucleotide databases. Different strains of the same species from samples collected in different locations or at different times may possess completely or partially identical sequences. These sequences, though containing important epidemiological information, increase the workload during sequence analysis. In CoVDB, we compared all nucleotide sequences and labeled the identical ones to mitigate this problem. Users can choose to show or not to show strains with identical sequences by clicking on the check boxes to the left of the page (Figure 3b).\nBlast similarity search\nDuring the process of coronavirus gene sequences analysis, we encountered a major problem when coronavirus gene sequences, especially those of orf1ab, were used for blast search against GenBank or any other coronavirus databases. When part of the orf1ab gene (e.g. nsp5) is used as the query sequence, instead of getting the gene for the specific non-structural protein that the query sequence is homologous to, the results will only show that the hits are within orf1ab, or in some cases, shown to be within the entire coronavirus genome. Much time will be needed for further analyzing the results manually in order to locate the positions of the cleavage sites of the corresponding genes for the non-structural proteins, making it very inefficient for further downstream work.This problem has been overcome by the annotated sequences in CoVDB. The blast search page of CoVDB is an interface for facilitating coronavirus similarity search. The background support program, blastall, is from the NCBI Blast package. The blast search page can be entered by clicking &#8216;Tools&#8217; in the top menu bar in any page of CoVDB. Since all sequences in CoVDB are annotated, they can be grouped into different datasets for blast search. Users can choose one of the three nucleotide and two protein sequence datasets as the database for comparison (Figure 4). The three nucleotide sequence datasets are: CoV genes (nsp + genes after 1ab), CoV genes (1ab + genes after 1ab) and CoV GenBank strains, which are the original sequences retrieved from GenBank. The two protein sequence datasets are the translated sequences of the first two nucleotide datasets: CoV proteins (nsp + aa after 1ab) and CoV proteins (1ab + aa after 1ab).\nFigure 4.Screenshot of blast similarity search page. Five datasets can be chosen as the database for comparison.\nMyBlast\n&#8216;MyBlast&#8217; employs the same blast program as the Blast page mentioned above. However, instead of selecting a predefined nucleotide or amino acid sequence database, multiple sequences can be pasted into the second sequence input box to generate a temporary sequence database. One or more query sequences can be pasted into the first sequence input box for blastn or blastp search against the temporary sequence database.\nORF finder for coronavirus\nThis ORF finder is specifically designed for coronavirus genome analysis. The result page shows the positions and lengths of each putative ORF and the position of the putative ribosomal frameshift site for translation of orf1ab. The nucleotide or amino acid sequences of the ORFs can be shown by selecting the corresponding check boxes. To facilitate genome comparison and annotation, the most closely related coronavirus, which had been annotated in CoVDB, can be chosen from a pull-down list for comparison using blast search. This function is particularly useful for determining the range of nsp in orf1ab.\nDISCUSSION\nRapid and accurate batch sequence retrieval is both the cornerstone and bottleneck for comparative gene or genome analysis. During the process of complete genome sequencing and comparative analysis of the various novel human and animal coronavirus genomes in the past 2 years, we have developed a comprehensive database, CoVDB, of annotated coronavirus genes and genomes, which offers efficient batch sequence retrieval and analysis. As shown by our experience in using CoVDB for comparative genome analysis of novel coronaviruses we have discovered (4,13,16,18,19), we find that CoVDB is more rapid and efficient than other existing coronavirus databases for batch sequence retrieval for the following reasons. First, we have performed annotation on all non-structural proteins in the polyprotein encoded by orf1ab of every single sequence. Second, annotation was performed for the non-structural proteins encoded by ORFs downstream to orf1ab using a standardized system, with some exceptions given to some names that have been used for a long time so as to minimize confusion. Third, all sequences with identical nucleotide sequences were labeled where one can choose to show or not to show strains with identical sequences. Fourth, CoVDB contains not only complete coronavirus genome sequences, but also incomplete genomes and their genes. Some genes of coronaviruses, such as pol, spike and nucleocapsid are sequenced much more frequently than others because they are either most conserved or least conserved. These gene sequences are particularly important for evolutionary analysis, single nucleotide polymorphism studies and design of primers for RT-PCR or quantitative RT-PCR amplification.\nAvailability\nCoVDB is constructed by the Department of Microbiology, the University of Hong Kong. It is available at no charge at http://covdb.microbiology.hku.hk.",
        "pmid": "17913743",
        "title": "CoVDB: a comprehensive database for comparative analysis of coronavirus genes and genomes",
        "journal_title": "Nucleic Acids Research",
        "authors": "Yi Huang;Susanna K. P. Lau;Patrick C. Y. Woo;Kwok-yung Yuen",
        "doi": "10.1093/nar/gkm754"
    },
    {
        "keywords": "coronavirus;infection;pig;respiratory tract",
        "abstract": "Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5&#8242;-untranslated region of coronavirus genomes.",
        "body": "INTRODUCTION\nCoronaviruses (CoVs) are found in a wide variety of animals, in which they can lead to enteric, hepatic, neurological and respiratory illnesses of differing severity. On the basis of genotypic and serological characterization, CoVs were traditionally divided into three distinct groups. In 2009, the Coronavirus Study Group of the International Committee for Taxonomy of Viruses replaced the traditional CoV groups 1, 2 and 3 with three genera, Alphacoronavirus, Betacoronavirus and Gammacoronavirus, respectively.1 In the same year, we discovered three novel CoVs in avian cloacal swabs.2 These CoVs formed a distinct novel CoV genus, named Deltacoronavirus.1 Subsequently, in a large epidemiological study, we discovered seven additional deltacoronaviruses.3 Interestingly, one of these deltacoronaviruses, which was originally named porcine CoV HKU15, was found in fecal samples of pigs in Hong Kong, and it is the only mammalian deltacoronavirus.3 In 2016, the Coronavirus Study Group of the International Committee for Taxonomy of Viruses rectified the species name for this virus to Coronavirus HKU15.4Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and was found to be associated with fatal outbreaks.5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23 In these epidemiological studies, Coronavirus HKU15 was generally only detected in fecal/intestinal samples. However, some CoVs, such as transmissible gastroenteritis CoV (TGEV)/porcine respiratory CoV (PRCV) of Alphacoronavirus 1 and bovine CoV of Betacoronavirus 1, could be detected consistently in both fecal and respiratory samples.24, 25 Therefore, we hypothesized that Coronavirus HKU15 may also be present in respiratory samples of pigs, which has implications for its transmission and potential role in respiratory diseases and for the use of nasopharyngeal sampling as an alternative method of identifying infected pigs. To test this hypothesis, we performed a molecular epidemiology study on nasopharyngeal samples collected from pigs in Hong Kong. Two complete genomes of the &#8216;respiratory&#8217; Coronavirus HKU15 were sequenced, and comparative genomic and phylogenetic studies were performed. The implications of the presence of Coronavirus HKU15 in respiratory samples are also discussed.\nMATERIALS AND METHODS\n\nSample collection\nNasopharyngeal samples from pigs were collected in Hong Kong over a 26-month period (January 2012&#8211;February 2014). These samples were obtained from slaughterhouses and pig farms in Hong Kong with the assistance of the Veterinary Public Health Section, Food and Environmental Hygiene Department; and the Agriculture, Fisheries and Conservation Department of the Government of Hong Kong. Immediately after sample collection, each nasopharyngeal swab was submerged in viral transport medium for viral transport and maintenance.\nRNA extraction and reverse transcription\nViral RNAs were extracted from the nasopharyngeal samples of pigs by using 200&#8201;&#956;L of inoculated viral transport medium for each sample and by utilizing the EZ1 Advanced XL system (Qiagen, Hilden, Germany) and EZ1 Virus Mini Kit v2.0 (Qiagen) according to the manufacturer&#8217;s protocol with RNase-free water as the eluent. Reverse transcription (RT) was performed using the SuperScript III Reverse Transcriptase (Invitrogen, Carlsbad, CA, USA) according to the manufacturer&#8217;s protocol by random priming.\nNone\nDetection of Coronavirus HKU15 was performed by polymerase chain reaction (PCR) amplifying a 289-bp fragment of the RNA-dependent RNA polymerase (RdRp) gene, using the specific primer pair LPW14077 (5&#8242;-ACA CAC TTG CTG TAA CCA AA-3&#8242;) and LPW14080 (5&#8242;-ATC ATT AGA GTC ACC ACG AT-3&#8242;). PCR and DNA sequencing were carried out following our previous publications with slight modification.26, 27 Briefly, each PCR mixture contained PCR buffer (50&#8201;mM of KCl, 10&#8201;mM of Tris-HCl at pH 8.3 and 3&#8201;mM of MgCl2; Applied Biosystems, Foster City, CA, USA), 200&#8201;&#956;M of each deoxynucleoside triphosphate (Roche Diagnostics, Basel, Switzerland), 1&#8201;&#956;M of each primer (Invitrogen), 1.0 U of AmpliTag Gold DNA polymerase (Applied Biosystems) and cDNA. The mixtures were subjected to 60 thermocycles of 94&#8201;&#176;C for 1&#8201;min, 50&#8201;&#176;C for 1&#8201;min and 72&#8201;&#176;C for 1&#8201;min, with an initial denaturation at 95&#8201;&#176;C for 10&#8201;min and a final extension at 72&#8201;&#176;C for 10&#8201;min for DNA amplification using the GeneAmp PCR System 9700 automated thermal cycler (Applied Biosystems). Standard precautions were taken to avoid contamination, and no false-positive result was observed for the negative controls. PCR products were agarose gel-purified using the QIAquick Gel Extraction kit (Qiagen). Both strands of the PCR products were sequenced twice by the ABI Prism 3130xl Genetic Analyzer (Applied Biosystems) using the two PCR primers.\nComplete genome sequencing\nThe genomes of two Coronavirus HKU15 strains detected in the nasopharyngeal samples of two different pigs were sequenced following our previous publications26, 27 with modifications. Briefly, viral RNAs were converted to cDNAs using SuperScript III reverse transcriptase with a combined random priming and oligo(dT) priming strategy. The cDNAs were PCR-amplified by primers (Supplementary Tables S1 and S2) that were designed by multiple alignment of the genome sequences of other Coronavirus HKU15 strains with complete genomes available or from the results of the first and subsequent rounds of PCR&#8211;sequencing. PCRs were performed using the iProof High-Fidelity PCR kit (Bio-Rad Laboratories, Hercules, CA, USA) according to the manufacturer&#8217;s protocol, and DNA sequencing was performed as mentioned above. When ambiguous peaks were observed consistently in the electropherograms after several attempts of PCR-sequencing for certain genomic regions, cloning followed by plasmid sequencing was performed according to our previous publication,28 except the Zero Blunt TOPO PCR Cloning kit (Invitrogen) was used to resolve the sequence ambiguities. PCRs using recombinant plasmids as templates were also performed to confirm that indels at mononucleotide polymeric regions were not the result of polymerase slippage. The 5&#8242; ends of the viral genomes were amplified and sequenced by the rapid amplification of cDNA ends (RACE) using the SMARTer RACE 5&#8242;/3&#8242; kit (Clontech Laboratories, Mountain View, CA, USA) according to the manufacturer&#8217;s protocol with the reverse primers LPW15379 (5&#8242;-TGG GTA ATG TGT CCG CTG ACG GGC GGT G-3&#8242;), LPW15380 (5&#8242;-AGA AGT GGT GGA TGG TCA GAG GAA CGG T-3&#8242;) and LPW16265 (5&#8242;-GTG GCT GGT TTC CAG GTA ATC TA-3&#8242;). The sequences of the PCR products were then assembled manually to obtain the genomes of the two nasopharyngeal strains.\nPhylogenetic characterization\nPairwise alignment was performed using BioEdit 7.2.5 (optimal GLOBAL alignment)29 or EMBOSS Stretcher (Nucleotide Alignment);30 whereas multiple sequence alignment was performed using MUSCLE 3.8,31 where the aligned sequences were further manually inspected and edited. Tests for substitution models and phylogenetic analysis by the maximum likelihood method were performed using MEGA 6.0.6.32\nEstimation of divergence times\nDivergence times for the Coronavirus HKU15 strains were calculated based on the complete genome sequence data, utilizing the Bayesian Markov chain Monte Carlo method using BEAST 1.8.033 with the substitution model GTR (general time-reversible model)+G (gamma-distributed rate variation)+I (estimated proportion of invariable sites), a strict molecular clock, and a constant coalescent. Fifty million generations were run with trees sampled every 1000th generation to yield 50 000 trees. Convergence was assessed based on the effective sampling size after a 10% burn-in using Tracer 1.6.0. The mean time to the most recent common ancestor (tMRCA) and the highest posterior density (HPD) regions at 95% were calculated. The trees, after a 10% burn-in, were summarized as a single tree using TreeAnnotator 1.8.0 by choosing the tree with the maximum sum of posterior probabilities (maximum clade credibility) and viewed using FigTree 1.4.0.\nNucleotide sequence accession number\nThe complete genome sequences of the two nasopharyngeal Coronavirus HKU15 strains were deposited into the International Nucleotide Sequence Databases with accession numbers LC216914 and LC216915.\nRESULTS\n\nAnimal surveillance\nA total of 249 nasopharyngeal samples from 249 pigs were tested. RT-PCR for a 289-bp fragment of the RdRp gene of Coronavirus HKU15 was positive in 24 (9.6%) of the 249 nasopharyngeal samples. The samples that tested positive were mostly collected during winter (December&#8211;March) (Figure 1). DNA sequencing showed that seven sequence variants were detected among the 24 positive samples, and pairwise alignment showed that these seven sequence variants possessed 98.7%&#8211;100% sequence identity to the corresponding region in the RdRp gene of Coronavirus HKU15 strain HKU15-155 that we previously found in fecal samples of pigs in Hong Kong3 (Supplementary Figure S1).\nComplete genome sequencing and genome analysis\nComplete genome sequencing was performed for the Coronavirus HKU15 found in two of the positive nasopharyngeal samples (S579N and S582N). Excluding the 3&#8242; poly(A) tail, the genomes of S579N and S582N were 25 411&#8211;25 413 and 25 397 nucleotides long, respectively. The genome organization of the two strains was the same as that of other Coronavirus HKU15 strains. The lengths of the seven open reading frames (ORFs) of the two strains S579N/S582N were 18 803/18 788, 3480, 252, 654, 285, 1029 and 603&#8201;bp, respectively. The genomes of the two strains possessed 98.9%&#8211;99.2% sequence identity to that of the representative isolate HKU15-155. Quasispecies were detected in one of the samples (S579N) at the 5&#8242; genomic region via two independent nested PCRs targeting the 122nd&#8211;505th bases of the genome using two different primer pairs for the first round and the same primer pair for the second round of reaction. For S579N, direct sequencing of the PCR products yielded ambiguous peaks in the sequencing electropherograms, which could only be resolved after cloning (Figure 2). Post-cloning DNA sequencing revealed that there were six sequence variants, with four polymorphic sites, for this genomic region (Figure 2). Two of the polymorphic sites, located at the 189th and 376th nucleotide positions, involved indels (&#916;T and &#916;A/C, respectively), whereas the other two polymorphic sites, located at the 292nd and 296th nucleotide positions, involved transition substitutions (T&#8594;C and G&#8594;A, respectively). Additionally, PCR&#8211;DNA sequencing using the recombinant plasmids as amplification templates did not generate the same sequence ambiguities observed in the pre-cloning experiment.\nPhylogenetic characterization\nPhylogenetic analysis of the complete genomes of the two nasopharyngeal strains and other Coronavirus HKU15 strains showed that the outbreak strains in the United States possessed highly similar genome sequences and that they were all clustered together monophyletically, whereas the Asian strains were more diverse and paraphyletic, with the Lao and Thai strains occupying the basal lineage; however, the South Korean strain KNU14-04 was more similar to the US strains than to the other Asian strains (Figure 3 and Supplementary Figure S2). The two nasopharyngeal strains S579N and S582N in the present study were most closely related to strains PDCoV/CHJXNI2/2015 from Jiangxi, China and CH/Sichuan/S27/2012 from Sichuan, China (Figure 3 and Supplementary Figure S2). Similar to other Chinese and Hong Kong strains (except CHN-AH-2004 and HKU15-44) and a Thai strain (P23_15_TT_1115), there was a deletion of an asparagine residue of the asparagine trimer located at the 50th&#8211;52nd amino-acid positions in the S protein compared with the other strains from South Korea, Laos and United States and to two other strains from Thailand. The deletion of this codon has been proposed as a genetic marker for the differentiation of China and Hong Kong strains from US and South Korean strains.12\nEstimation of divergence dates\nThe estimated mean evolutionary rate of the complete genome sequence data set was 6.549 &#215; 10&#8722;4 (95% HPD: 5.632&#8211;7.476 &#215; 10&#8722;4) substitutions per site per year, which is approximately 1.7-fold higher than that estimated in a previous study.19 The root of the tree was September 1638 (95% HPD: June 1570&#8211;March 1698). The tMRCA of the diversity of Coronavirus HKU15 was dated to June 1991 (95% HPD: November 1987&#8211;June 1994); and the tMRCA of the Thai/Laos strains was traced back to September 2014 (95% HPD: May 2014&#8211;January 2015). The tMRCAs for the clade containing US/Korean strains was estimated to be October 2012 (95% HPD: June 2012&#8211;January 2013), which is slightly delayed compared with that estimated in a previous study.19 For the two nasopharyngeal strains characterized in this study (S579N and S582N), they were estimated to have diverged from their respective MRCAs in December 2011 (95% HPD: May 2011&#8211;July 2012) and August 2009 (95% HPD: November 2008&#8211;April 2010), respectively (Figure 4).\nDISCUSSION\nCoronavirus HKU15 was detected in nasopharyngeal samples of pigs. Although Coronavirus HKU15 has been widely detected in various locations around the Pacific Ocean, including Canada,7 China,12, 13, 16, 17 Hong Kong,3 Laos,22, 23 Mexico,19 South Korea,5, 21 Thailand,20, 23 Vietnam23 and the United States,6, 7, 8, 9, 10, 11, 14, 15, 18, 19 the virus has principally been found in fecal or intestinal specimens. There have been a few exceptional circumstances; in one study, the presence of Coronavirus HKU15 was reported in the blood, liver, lung and kidney of one pig,15 and in a few other studies, Coronavirus HKU15 was found to exist in the blood (n=10), mesenteric lymph node (n=2) and saliva (n=10)/oral fluid (n=73) of pigs,7, 19, 20 implying that Coronavirus HKU15 can cause systemic infections in occasional cases. In this study, Coronavirus HKU15 was found in 9.6% of the nasopharyngeal samples of pigs, which is similar to the 10.1% positive rate of Coronavirus HKU15 in fecal samples of pigs that we reported previously.3 Seasonal variation in the detection rate of Coronavirus HKU15 from pigs was noted, where most of the positive samples were collected in winter. This is similar to the pattern of seasonal variation in a surveillance study carried out in the United States, where the detection rate for Coronavirus HKU15 was much lower during summer.19 It has recently been confirmed that Coronavirus HKU15 is able to cause swine enteric infections by infecting gnotobiotic and conventional pigs with Coronavirus HKU15.15 The detection of Coronavirus HKU15 in respiratory tracts of pigs has the following implications. First, in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs. Second, in addition to fecal-oral transmission, the virus may be able to spread through the respiratory route. Third, the presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Further studies will determine the full spectrum of clinical diseases and pathologies associated with Coronavirus HKU15.From the data of the present study, both the &#8216;enteric&#8217; and &#8216;respiratory&#8217; Coronavirus HKU15 may possess similar properties. A number of animal CoVs possess dual or multiple tissue tropisms. For example, TGEV, which is another enteropathogenic CoV that infects pigs, could also be found in the nasopharynx of pigs as PRCV, which is a deletion mutant of TGEV.25 Moreover, bovine CoV is both an enteric and a respiratory pathogen in cattle.24 Similar to TGEV/PRCV, Coronavirus HKU15 is recovered from both respiratory and gastrointestinal samples. However, unlike TGEV/PRCV, in which there is a 621&#8211;681 nucleotide deletion at the 5&#8242; end of the spike (S) gene leading to a loss of 1&#8211;2 antigenic sites in PRCV,34 comparative genome analysis of Coronavirus HKU15 from respiratory and fecal samples did not show any obvious difference in their S proteins or other parts of their genomes. Phylogenetic analysis also did not reveal a separate clustering of fecal/intestinal and nasopharyngeal isolates (Figure 2). Further cell culture experiments are required to confirm whether all strains of this species possess intrinsic tropism to both enteric and respiratory tissues.This is also the first report of CoV quasispecies in the 5&#8242;-untranslated region (UTR). In one (S579N) of the two Coronavirus HKU15 genomes that we sequenced in this study, variant sites were observed at four positions; two of them were due to nucleotide substitutions, and the other two were results of indels at mononucleotide polymeric regions (189th and 376th bases). These two indels were genuine variant sites instead of being due to polymerase slippage during the amplification process because recombinant plasmid-dependent PCR&#8211;sequencing no longer resulted in sequence ambiguities in the electropherograms. Although the existence of quasispecies has been reported in CoVs, the variant sites were found in coding regions or 3&#8242;-UTR.35, 36, 37, 38 In the case of severe acute respiratory syndrome-related coronavirus, all of the variant sites observed in the quasispecies were located at the S gene.35 For bovine CoV, one of the two strains with naturally occurring intra-isolate quasispecies had all seven variant sites located at ORF1a, whereas for the other strain with naturally occurring intra-isolate quasispecies, there were 85 polymorphic sites scattered across ORF1a (n=28), ORF1b (n=19), 32&#8201;kDa-non-structural protein (NSP) gene (n=6), hemagglutinin esterase (HE) gene (n=2), S gene (n=18), 4.9&#8201;kDa-NSP gene (n=2), 4.8&#8201;kDa-NSP gene (n=2), membrane (M) gene (n=2), nucleocapsid (N) gene (n=5) and 3&#8242;-UTR (n=1).36 Similar to bovine CoV, Middle East respiratory syndrome-related coronavirus also possessed all the intra-host single nucleotide variations throughout its genome except the 5&#8242;-UTR.37, 38 In this study, all four variant sites (189 &#916;T, 292 T&#8594;C, 296 G&#8594;A and 376 &#916;A/C) were present in the 5&#8242;-UTR and were not located in the leader sequence or the transcription regulatory sequence. We speculate that the existence of quasispecies in CoVs may play a role in CoV evolution, in addition to the more well-known high-recombination and mutation rates in CoV genomes.39",
        "pmid": "28634353",
        "title": "Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5&#8242;-untranslated region",
        "journal_title": "Emerging Microbes & Infections",
        "authors": "Patrick CY Woo;Susanna KP Lau;Chi-Ching Tsang;Candy CY Lau;Po-Chun Wong;Franklin WN Chow;Jordan YH Fong;Kwok-Yung Yuen",
        "doi": "10.1038/emi.2017.37"
    },
    {
        "keywords": "animal models;coronaviruses;reservoirs;the Middle East respiratory syndrome (canonical form);the severe acute respiratory syndrome",
        "abstract": "Abstract\nIn humans, infection with the coronavirus, especially the severe acute respiratory syndrome coronavirus (SARS&#8208;CoV) and the emerging Middle East respiratory syndrome coronavirus (MERS&#8208;CoV), induces acute respiratory failure, resulting in high mortality. Irregular coronavirus related epidemics indicate that the evolutionary origins of these two pathogens need to be identified urgently and there are still questions related to suitable laboratory animal models. Thus, in this review we aim to highlight key discoveries concerning the animal origin of the virus and summarize and compare current animal models.",
        "body": "INTRODUCTION\nSevere acute respiratory syndrome, an infectious disease, arose in November 2002 in the Guangdong Province of China and spread to 28 regions, causing 778 deaths among 8076 infected individuals within just 1&#160;year. Soon after, the causative agent, named severe acute respiratory syndrome&#8208;associated coronavirus (SARS&#8208;CoV), was identified as a novel coronavirus.In the aftermath of the SARS outbreak, its high morbidity and mortality made the identification of natural reservoirs and an appropriate animal model necessary in order to ascertain the interspecies transmission chain, to develop procedures for protecting public health, to promote research on the SARS&#8208;CoV mechanism, and to establish animal models for use in developing antivirals and vaccines.Although the SARS outbreak occurred over 10&#160;years ago, another member of the Coronaviridae family of viruses has since caused illness in the Middle East. This illness has been named Middle East respiratory syndrome and the pathogen (MERS&#8208;CoV) has been shown to be a type of coronavirus that is highly related to SARS&#8208;CoV. Infection with MERS&#8208;CoV results in higher mortality and new symptoms such as renal failure. Thus, failure to fully resolve the initial SARS pathogen has been followed by a more virulent infection with a related virus, MERS&#8208;CoV. Therefore, we need urgently to identify the origins of the viruses and find ways to deal with them. To make a useful comparison, this review will investigate the reservoirs of infection and provide an overview of the animal models for both viruses.\nWILD ANIMAL RESERVOIRS\n\nBat\nYuan KY et&#160;al1 assayed 13 different species of bats, 5 different species of rodents and 20 rhesus macaques. The results proved that SARS&#8208;CoV in bats was the most closely related to that in humans. In this assay, the SARS&#8208;like CoV in Chinese horseshoe bats has 88%&#8208;92% sequence homology with human SARS&#8208;CoV. Moreover, the S2&#160;m motif in its 3&#697;UTRs and the phylogenetic analyses of four fully characterized genomes of SARS&#8208;like CoV indicated that the horseshoe bats have great potential to be one of the natural reservoirs.2, 3\nGiven that, the same research suggested that, even though there are no available migration patterns, it is known that bats can migrate approximately 30 miles for hibernation, and the distance between their wild habitats and markets in Shenzheng and Hong Kong is only 17 miles, which means that geographically widespread infections of SARS&#8208;like CoV can be explained by transmission via bats.Recently, a five year study provided futher evidence that bats in the Yunnan province of China are more likely to be the prime reservoir than those detected elsewhere. The study compared the non&#8208;structural protein genes ORF1a and 1b of the virus and concluded that SARSr&#8208;CoV strains from a cave in Yunnan village were more closely related to human SARS&#8208;CoV and cell entry studies demonstrated that three newly identified SARSr&#8208;CoVs with different S protein sequences are all able to use human ACE2 as the receptor.4\nJust as horseshoe bats were postulated to be the primary SARS host, van Boheeman et&#160;al.5 indicated that two kinds of bats carry similar MERS coronaviruses. Then, Susanna K. P. Lau et&#160;al.6 used sequences of RNA polymerase (RdRp), spike (S), and nucleocapsid (N) genes to determine that human MERS&#8208;CoV RdRp is more closely related to the pipistrelle bat CoV HKU5 (92.1%&#8208;92.3% amino acid identity) and the S and N genes are more closely related to the Tylonycteris bat CoV HKU4 (respectively, 66.8%&#8208;67.4% and 71.9%&#8208;72.3% amino acid identity), indicating that these three viruses may share the same ancestor. However, these results did not definitively prove bat CoV to be the ancestor of the human CoV.Subsequently, Victor Max Corman et&#160;al7 isolated a virus named &#8220;Neo CoV&#8221; from South African Neoromicia capensis bats, and this virus has 85% similarity with MERS&#8208;CoV, with even higher rates for some specific viral RNA segments. Additionally, MERS&#8208;CoV has been found to grow better in bat cells than in human, bovine, cat, and swine cells.8\nInterestingly, within this article, it is noted that the Great Horn of Africa, where Neo CoV was discovered, is also a place where camels are transported and traded. Therefore, the original transmission from bats to camels may have occurred in sub&#8208;Saharan Africa.Based on the above evidence, there is a strong possibility that bats are the initial MERS&#8208;CoV host. A paper by Hana Fakhoury et&#160;al analyzed the possible viral distribution route that MERS&#8208;CoV traveled in the spring of 2014, leading the authors to speculate that hibernation might be the reason for seasonal outbreaks.9 After hibernation, bats wake up in the warm, food&#8208;abundant spring weather and eat palm or other seeds. Those seeds may carry bat feces and then drop to ground, where they might be eaten by camels. If MERS&#8208;CoV is found in cereal plantations near where bats congregate, this view will be confirmed.\nCivet cat\nDuring the SARS outbreak, masked palm civet cats (Paguma larvata) and raccoon dogs (Nyctereutes procyonoides) were found to carry SARS&#8208;like viruses, even before the virus was discovered in lesser bamboo bats (Tylonycteris pachypusa and Pipistrellus).10 At the same time, wild animal sellers were found to possess higher than average relevant neutralizing antibodies.11 Initial phylogenetic analyses reveal that the SARS&#8208;CoVs in civets and humans actually come from two distant branches, but the SARS&#8208;CoV from civets during the incipient phase of the epidemic had 99.8% sequence similarity to the human SARS&#8208;CoV.12 Until recently, it was thought that civets were the immediate zoonotic source of SARS&#8208;CoV in the Guangdong SARS outbreak.Regarding how the civets gain the CoV, Janies et&#160;al used dynamic homology phylogenetic analyses to investigate other species besides civets and the results further supported the idea of civets as the immediate reservoir. It is worth mentioning that the &#8220;other species&#8221; even included humans. However, some phylogenetic analyses including humans have shown a limited numbers of infectious civets in the relevant areas.13, 14\nBased on these data, it is likely that civets may be only a &#8220;bypass&#8221; reservoir that adapted transiently before the epidemic. Whatever the truth is, we can at least be sure that the civet cat is one of the intermediate hosts.\nDromedary camel\nIn ruminants, almost all evidence indicates that camels are the most important MERS&#8208;CoV intermediate host. Hamzah A. Mohd et&#160;al concluded this from two key pieces of evidence.15, 16 One piece of evidence is that among dromedary camels, cattle, sheep, goats, and any other camelid, only camels carry specific antibodies in their serum. Another piece of evidence is the infection of two humans in approximately 2013 in a Qatar pasture where there was a cluster of 11 MERS&#8208;CoV&#8208;infected dromedary camels. At the same time, M&#252;ller MA et&#160;al17 provided serum evidence that shepherds and butchers have higher infection rates than individuals in other occupations. Additionally, an investigation in Saudi Arabia revealed that younger camels are prone to infection by MERS&#8208;CoV.18\nWith regard to the transmission route, there are three possible ways for the virus to be conveyed to humans from camels: organs, flesh, and discharges such as feces and camel milk.MERS&#8208;CoV RNA was found in a camel lymph node in Qatar.19 This finding indicates that MERS&#8208;CoV may be maintained in camel organs or muscles. If that speculation proves true, the convention in the Middle East of cooking and trading camel meat and organs might be the interspecies transmission route. MERS&#8208;CoV can also survive longer in camel milk than in other ruminant milk. In addition to the above two routes, MERS&#8208;CoV RNA can be detected in 59% of nasal discharge and 15% of feces in camels. Thus, there is supportive evidence for the three postulated transmission routes, but further verification is needed to confirm them. It should also be noted that MERS is a type of infectious respiratory disease, and therefore, in addition to the above three possible transmission routes, infection via aerosols produced by camels should be considered.\nAlpaca\nResearch in Qatar in 2015 showed that MERS&#8208;CoV was found in alpacas centrally housed with camels.20 Meanwhile, Eckerle et&#160;al15 demonstrated that 14 residues in the alignment of mammalian DPP4 (dipeptidyl peptidase&#8208;4) interact with the MERS&#8208;CoV receptor binding domain of alpaca DPP4, which is identical to that of DPP4 in dromedary camels, providing proof of the transmission potential of alpacas.Although MERS&#8208;CoV has so far not been found in camelids other than dromedaries outside of the Arabian Peninsula, the increasing export of alpacas increases the risk of an outbreak, since they have the potential to be the interspecies host.While it is not clear that alpacas have the same function as the camels in the 2012 outbreak, their ability to spread the virus exists. Currently it is very popular to trade and keep ornamental alpacas, so strict quarantine is necessary.\nLivestock\nReusken et&#160;al19 infected swine and chickens with SARS&#8208;CoV and determined that they are not susceptible to becoming hosts. In contrast, dromedary camels have been found to be capable of SARS&#8208;CoV infection. A similar capability to transmit MERS&#8208;CoV makes camels a focal link in the coronavirus transmission chain.As for other MERS&#8208;CoV domestic reservoirs, between 2010 and 2013, some investigators tested serum samples from sheep, goat, cattle and chickens, which pervade Saudi Arabia and Europe.16, 21 All the samples were negative for MERS&#8208;CoV antibodies. Meyer B et&#160;al tested 192 horses from the UAE and 697 horses from Spain, as well as mules and monkeys, all of which turned out to be negative. Later, an investigation in Saudi Arabia and another in Sudan both reached the same conclusion.17, 22\nHowever, these results are of limited value since they come from nationwide general surveys rather than investigations focusing on places with prevalent MERS infections. The livestock tested in these studies may not have been in close contact with primary hosts, and thus the results do not provide sufficient evidence to disprove their potential for being hosts.On the other hand, another study22 that found MERS&#8208;CoV RNA, but no neutralizing antibodies, in 6 lambs. To some extent, in the context of methodical investigations, sheep could be another virus carrier in addition to camels. Thus, it would be prudent to conduct field research and take necessary precautionary measures to preclude the possibility of transmission of coronaviruses from sheep.\nBirds\nCoronaviruses can be found in many kinds of birds.23 Within Hong Kong alone, CoV&#8208;HKU11 has been found in nightingales, CoV&#8208;HKU12 in thrushes, CoV&#8208;HKU13 in munias, CoV&#8208;HKU16 in white&#8208;eyes, CoV&#8208;HKU17 in sparrows, and CoV&#8208;HKU18 in magpies. Fortunately, avian coronaviruses are not that closely related to SARS&#8208;CoV. Additionally, these are not migratory birds and therefore do not expand the range of the pathogen. Moreover, unlike bats, these birds were accessible enough to sterilize. However, in light of the findings above, randomly hunting them for food or for pets is unwise.\nLABORATORY ANIMAL MODELS\n\nPrimate models\nCompared to other symptom&#8208;limited models, non&#8208;human primary models are better&#8208;established models that have close psychological and physical similarities with humans24 (Table&#160;1).\nRhesus macaque\nCurrently, the most widely accepted MERS laboratory animal model is the rhesus macaque (Macaca mulatta). MERS&#8208;CoV infection can lead to a pneumonia&#8208;like syndrome within 24&#160;hours of challenge in the rhesus macaque, but it is not as severe as in humans.24 In some studies on MERS infection in rhesus macaques,24, 25 following intraoral, intranasal and intravascular inoculation with 7&#160;&#215;&#160;106 TCID50, acute, transient and mild to moderate respiratory symptoms such as tachypnea, deep abdominal breathing, coughing, fever and anorexia were presented. However, gross lesions were only visible in the lungs. Microscopically, these were typical bronchointerstitial pneumonia or interstitial pneumonia. Another experiment26 used an intravascularly inoculated infection dose of 6.5&#160;&#215;&#160;107 TCID50, which resulted in pulmonary congestion and the microscopic lesions of interstitial pneumonia. After infection, MERS&#8208;CoV RNA was identified in nasal swabs and bronchoalveolar lavage samples and partially in oropharyngeal swabs. Inside the body, it was present only in the lungs, and not in blood or any visceral organs, even the kidneys.3, 24, 27 Additionally, all blood count abnormalities in the rhesus macaques were like those reported in human cases.In contrast to the results found for MERS&#8208;CoV, rhesus macaques developed different symptoms after being challenged with different SARS&#8208;CoV lineages. Rhesus macaques infected with the Tor 2 lineage28 by intravascular inoculation exhibited clinical signs ranging from symptom&#8208;free to agitated and aggressive. Focal pulmonary consolidation was revealed microscopically. The Urbani lineage, on the other hand, could not successfully infect rhesus macaques. In addition, rhesus macaques showed obvious clinical signs and histopathology after inoculation with the PUMC01 lineage.29 The animal's age is the key factor affecting these results, but this is hard to identify in wild&#8208;caught monkeys.\nCommon marmoset\nAnother suitable and well&#8208;established model is the common marmoset (Saguinus mystax), which can show more severe clinical signs than rhesus macaques when infected with MERS&#8208;CoV.30, 31, 32 When administered through a combination of intraoral, intranasal and intravascular inoculation, with doses ranging from 5&#160;&#215;&#160;106 TCID50 to 5&#160;&#215;&#160;107 PFU, mild to moderate respiratory disease was observed, and interstitial pneumonia was observed clinically and microscopically.When infected with SARS&#8208;CoV, common marmosets exhibit fever, diarrhea, multifocal pneumonitis and hepatis.33 Research using this model is progressing. The common marmoset is a potential non&#8208;human primate model for SARS&#8208;CoV infection and deserves more attention.\nOther prime models for SARS\u2010CoV\nTo date, MERS&#8208;CoV only has two mature models. This section will deal with additional non&#8208;human prime models for SARS&#8208;CoV. Rhesus, cynomolgus (Macaca fasicularis), and African green (Chlorocebus aethiops sabaeus or Cercopithecu aethiops sabaeus) monkeys have been used to investigate vaccine immunogenicity or efficacy against SARS&#8208;CoV. Squirrel monkeys (Saimiri sciureus) and mustached tamarins (Saguinus mystax) have been shown to be incapable of being infected.In the case of cynomolgus monkeys, clinical evidence, such as lethargy, temporary skin rash or respiratory distress, has not been reported. However, 4 to 6&#160;days post&#8208;inoculation (dpi), there was diffuse alveolar damage and extensive loss of epithelium from alveolar and bronchiolar walls.28, 34\nRegarding African green monkeys, clearance of the virus takes approximately 4 dpi, and the infection niduses are patchy. Respiratory secretions cannot accurately reflect the viral titers. However, there is a report that showed that the titer is higher and the residence time is longer in African green monkeys than in two other kinds of old world monkeys (cynomolgus and rhesus).34\n\nBALB/c mice\nK. Subbarao et&#160;al35 nasally inoculated 6&#8208; to 8&#8208;week&#8208;old BALB/c mice with SARS&#8208;CoV and reached the replication peak in the respiratory tract at 2 dpi. Viral shedding was observed at 5 dpi. There was no weight loss in BALB/c or C57BL/6 mice. However, during viral replication, the infected group gained less weight than the placebo group. Mild pneumonia was occasionally observed.In contrast, young BALB/c mice aged 4&#8208;6&#160;months and 12&#8208; to 14&#8208;month&#8208;old BALB/c mice displayed clinical syndromes such as weight loss, dehydration, and ruffled fur at 3&#8208;6 dpi, along with interstitial pneumonia,36 demonstrating the vital importance of age in rodent infection.Another way to conduct similar experiments is to encapsulate SARS&#8208;CoV protein in a shell of mouse hepatitis virus (MHV). The encapsulation of SARS&#8208;CoV means there is no need for BSL&#8208;3 precautions, but the chimera virus is not identical with SARS&#8208;CoV, so some application limitations still exist.SARS&#8208;CoV can infect many small animals such as rodents.37 However, MERS&#8208;CoV cannot achieve infection because the receptor binding domain of the viral spike protein (DPP4) has glycosylation sites that block viral recognition. Thus, except for some mice such as transgenic and immunodeficient mice, there is a lack of natural small animal models for MERS&#8208;CoV that are easy to operate and house.\nGolden Syrian hamster\nA. Roberts et&#160;al38 nasally inoculated golden hamsters with SARS&#8208;CoV, reaching the viral replication peak at 2 or 3 dpi and viral shedding by 10 dpi. There was histochemical evidence of pneumonia but no clear clinical symptoms. In addition, recovered hamsters produced high levels of neutralizing antibodies to resist subsequent infection.39 Unlike mice, the hamsters developed temporary viremia and viral replication in the liver and spleen. However, no inflammation was observed in those organs. This experiment proved that hamsters develop a more severe syndrome than mice.\nFerret and domestic cats\nMartine BE et&#160;al40 determined that ferrets and domestic cats can be infected with SARS&#8208;CoV by housing them together after one of them was inoculated. Both species exhibited viral replication and specific antibodies. Ferrets died at 16 to 21 dpi after demonstrating drowsiness and epiphysitis. Histochemical assays revealed dyslipidemia in ferret livers.It is important to note that ferrets are outbred and can be infected by many pathogens. Therefore, it is vital to make sure there are no other pathogens in the ferrets before use.41\nBased on current research, ferrets cannot be infected by MERS&#8208;CoV42 and thus are unsuitable for establishing an aerosol transmission viral model.\nAlpaca\nThree alpacas were experimentally intranasally inoculated with MERS&#8208;CoV.43 All of them shed viruses and antibodies were found in their serum. Additionally, infected alpacas spread the virus to two other healthy alpacas housed in the same space, indicating the alpacas' own natural transmission capability and their potential as natural hosts. The authors also noted that MERS&#8208;CoV may be able to infect alpaca kidney cells.44\nAs with camels, alpacas never showed symptoms such as fever. However, unlike camels, alpacas did not demonstrate observable nasal secretion.\nNew Zealand white rabbit\nHagamans BL et&#160;al44, 45 described the New Zealand white rabbit as a novel MERS&#8208;CoV animal model. However, they do not display any visible clinical signs or lesions after nasal or tracheal inoculation. Microscopically, lesions in rabbits were exhibited in the respiratory tract at 3&#8208;4 dpi. In the nasal cavity, mild to moderate heterophil leukocyte infiltration was observed. Nevertheless, the rabbits shed viruses from the upper respiratory tract and might be a limited MERS animal model for asymptomatic situations.\nLivestock\n\nOvine\nDanielle R. Adney et&#160;al46 infected sheep, goats and horses with MERS&#8208;CoV. Two young goats developed mild but consistent viral replication. However, none of the goats showed any visible syndrome such as fever or rhinorrhea, and all collected organs were negative in immunohistochemistry assay. Moreover, even though they were in close contact with the young goats, the adult goats did not become infected. Thus, the infection of goats is unstable and related to age.In contrast to the experiment with goats, sheep did not show immunohistochemically detectable infection. Three sheep were infected, and one of them displayed viral replication on days 1, 2, 3 and 5 in their nasal secretions, while another one displayed viral replication on day 6. One of the two that revealed viral replication was found to have low quantities of MERS&#8208;CoV neutralizing antibody at 14 dpi.Regarding susceptibility to infection, goats are more susceptible than sheep.\nHorse\nThough the MERS&#8208;CoV neutralizing antibody has not been found in horses, three of the four experimentally infected horses had detectable viral replication in their nasal secretions starting at 3 dpi.\nCaptive civet cat\nWu et&#160;al12 infected captive civets with two extremely different lineages of SARS&#8208;CoV. In the experiment, the captive civets become apathetic and less aggressive, which was different from the results found in wild civets. The histochemistry results were similar to those of ferrets and macaques post infection and support the epidemiological findings in animals from wild markets.\nCONCLUSIONS AND FUTURE PERSPECTIVES\n\nWild animal reservoirs\nConsidering all the evidence, bats have great potential to be the common prime host for both SARS&#8208;CoV and MERS&#8208;CoV. Bat CoVs go through a long adaptation in civet cats and/or camels until the exact mutated lineage develops that allows a coronavirus to achieve interspecies transmission and start an outbreak. However, based on research to date, even if further evidence were provided, we still do not know why bats in the Yunnan province are the most likely reservoir, but the epidemics broke out over 1000 miles away. How exactly does the long adaptation process of the SARS&#8208;CoV finish and what is the epidemic singularity? A transmission experiment between bats and civets should be done to fill this gap in our knowledge.As the most important agent in the MERS epidemic, camels were acknowledged be linked to MERS&#8208;CoV transmission through all morbidity and serum assays. Though the mechanisms of interspecies transmission are not yet determined, there are still some clues. Besides, in areas where camels graze or where camel shows and races occur the risk of transmission can increase.The investigations into camels suggested that civets may spread SARS&#8208;CoV via meat, feces and even aerosols, as well as via mechanisms that are common to SARS&#8208;CoV and MERS&#8208;CoV. Key factors regarding transmission, such as contact with bats in their environment and large numbers of animals being traded over long distances, may apply to both civets and camels. At the same time, humans desire contact with wild animals and want to tame them. Thus, despite natural mechanisms keeping humans and wild animals apart, there is a constant link between them.Before coronavirus outbreaks occur in humans, part of a known coronavirus may go through a long evolutionary pathway in unknown hosts. Regarding their morphological structure, MERS&#8208;CoV and SARS&#8208;CoV are both novel coronaviruses with striking similarities. Determining the evolutionary pathway leading to the ability to transfer between species may be helpful in predicting the next epidemic outbreak.\nLaboratory animal models\nRegarding establishing a laboratory animal model of SARS&#8208;CoV, monkeys, cats, ferrets, mice, swine, chickens, hamsters, guinea pigs and rats have been experimentally infected. Most of them did not develop acute pneumonia&#8208;like symptoms and only showed SARS&#8208;CoV replication in their serum. Some of them displayed mild signs while clearing the virus. Currently, ferrets and non&#8208;human primate models exhibit the strongest clinical symptoms and immunohistochemistry.Regarding models for MERS&#8208;CoV, camels do not develop the same clinical signs as humans, although they are natural hosts. In addition, they require too much space to house and thus are not the first choice for a laboratory model. Rabbits can be infected by MERS&#8208;CoV, but the histology is unstable, which would limit routine observation of disease. Compared to those animal models, other than DPP4 transgenic mice, rhesus macaques and marmosets are currently the best MERS animal models. Since their immune system is like that of humans in terms of physiology and anatomy, they can be used to study the pathogenesis mechanism and the efficacy of vaccines and antivirals. However, research using rhesus macaques requires BSL&#8208;3 laboratories and high investments. Moreover, they are smart, fast, and strong, which means precautions must be taken against them escaping. Therefore, the need for more user&#8208;friendly models still exists, but to date non&#8208;human primate models are still the best option.However, there is still a possibility of establishing new models. Chi Wai Yip et&#160;al47 phylogenetically analyzed the DPP4 receptor in various species and determined that humans, rhesus macaques, horses and rabbits belong to one family. Cattle and swine are not in the same group, but their receptor is close to the human DPP4 receptor. Small animals (including ferrets and mice) have DPP4 receptors far more distantly related to that of humans. This analysis could provide a reference point for potential animal models.In summary, non&#8208;human primate models are still the best choice of model. Comparatively speaking, there is a greater variety of SARS&#8208;CoV animal models than MERS&#8208;CoV animal models. Regarding priorities for research on vaccines and antivirals for both coronaviruses, suitable MERS&#8208;CoV models should be considered first.\nCONFLICT OF INTEREST\nNone.",
        "pmid": "30891557",
        "title": "The battle against SARS and MERS coronaviruses: Reservoirs and Animal Models",
        "journal_title": "Animal Models and Experimental Medicine",
        "authors": "Shu\u2010ran Gong;Lin\u2010lin Bao",
        "doi": "10.1002/ame2.12017"
    },
    {
        "keywords": "",
        "abstract": "The emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002 and Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012 has generated enormous interest in the biodiversity, genomics and cross-species transmission potential of coronaviruses, especially those from bats, the second most speciose order of mammals. Herein, we identified a novel coronavirus, provisionally designated Rousettus bat coronavirus GCCDC1 (Ro-BatCoV GCCDC1), in the rectal swab samples of Rousettus leschenaulti bats by using pan-coronavirus RT-PCR and next-generation sequencing. Although the virus is similar to Rousettus bat coronavirus HKU9 (Ro-BatCoV HKU9) in genome characteristics, it is sufficiently distinct to be classified as a new species according to the criteria defined by the International Committee of Taxonomy of Viruses (ICTV). More striking was that Ro-BatCoV GCCDC1 contained a unique gene integrated into the 3&#8217;-end of the genome that has no homologs in any known coronavirus, but which sequence and phylogeny analyses indicated most likely originated from the p10 gene of a bat orthoreovirus. Subgenomic mRNA and cellular-level observations demonstrated that the p10 gene is functional and induces the formation of cell syncytia. Therefore, here we report a putative heterologous inter-family recombination event between a single-stranded, positive-sense RNA virus and a double-stranded segmented RNA virus, providing insights into the fundamental mechanisms of viral evolution.",
        "body": "Introduction\nCoronaviruses are large, enveloped viruses with single-stranded, positive-sense, non-segmented RNA genomes [1]. Based on the current nomenclature of the International Committee of Taxonomy of Viruses (ICTV), coronaviruses of the family Coronaviridae are now classified into four genera: alpha-, beta-, gamma- and deltacoronavirus [2, 3]. Betacoronaviruses can be further subdivided into four phylogenetic groups [2].Coronaviruses employ a unique mechanism of viral genome replication and RNA synthesis, resulting in high frequencies of both mutation and recombination [4]. Recombination appears to be particularly important in coronavirus evolution [5], with a number of hotspots interspersed throughout the viral genome [6]. Recombination events at 3&#8217;-end of the genome might impact the replication ability of coronaviruses since there are a number of regulatory sequences and accessory genes in this region [5].As coronaviruses were previously known to cause only mild respiratory illnesses in humans they were not a major concern of the public health community. However, the emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) [7&#8211;9] and its high infectivity and fatality generated considerable interest in the biodiversity, genomics, evolution, natural hosts and potential inter-species transmission of coronaviruses [10]. To date, at least 90 types of coronavirus have been isolated or genome-identified from humans and a wide variety of animals, including domestic animals, wild birds and bats. Bats are particularly notable in this respect because they are known to harbor a diverse range of pathogens, and are known to be the reservoir hosts of both human coronavirus 229E [11] and SARS-CoV [12], and are closely related to MERS-CoV [13, 14]. As a consequence bats have been prioritized for the surveillance of emerging zoonotic diseases [15&#8211;17].In the present study we report a novel coronavirus discovered from bat samples in China that has been tentatively named Rousettus bat coronavirus GCCDC1 (Ro-BatCoV GCCDC1). Multiple lines of evidence indicate that Ro-BatCoV GCCDC1 may have arisen from a recombination event between an ancestral coronavirus and a fusogenic orthoreovirus.\nResults\n\nA novel coronavirus with a putative recombinant reovirus gene\nA total of 118 rectal swab samples from Rousettus leschenaulti bats sampled in Yunnan province China were screened for the presence of coronavirus RNA. Of these, 47 (40%) samples were found to be coronavirus positive. The PCR products were sequenced and BLAST searches revealed the sequences to be authentic coronavirus genes, with the strongest similarity to Rousettus bat coronavirus HKU9 (Ro-BatCoV HKU9) [18], a member of the genus betacoronavirus (group D). However, our attempts to isolate the virus from samples using a number of cell lines, including Vero E6, BHK-21, MDCK, A549, HEp-2, CaCo-2 and a bat cell line from Myotis kidney, were unsuccessful. The cell lines were inoculated with positive samples and three blind passages were performed for each sample. No cytopathic effect was observed in any passage, and there was an absence of viral replication from the culture supernatant and cell pellet of each passage.The viral genomic sequences present in two coronavirus positive samples (numbers 346 and 356) were determined with next-generation sequencing (NGS). Analysis using a partial (816-bp fragment) sequence of the RNA-dependent RNA polymerase (RdRp) gene indicated that the newly identified virus was likely to be a novel coronavirus according to previously proposed criteria [19]. Therefore, this virus was tentatively designated as Rousettus bat coronavirus GCCDC1 (Ro-BatCoV GCCDC1). Gaps within the genome of Ro-BatCoV GCCDC1 were closed, and the complete genome sequence confirmed, using Sanger sequencing. Finally, the 5&#8217;- and 3&#8217;-ends of Ro-BatCoV GCCDC1 genome were obtained using 5&#8217; and 3&#8217; RACE (Fig 1).Excluding the polyadenylated tail at the 3&#8217;-terminus, the genome of Ro-BatCoV GCCDC1 was 30,129 nt in length with a G/C content of 45.4%. Comparative genomic sequence analysis indicated that Ro-BatCoV GCCDC1 was most closely related to Ro-BatCoV HKU9 strains [18] with 66.6% - 67.4% nucleotide identities. Similarly, Ro-BatCoV GCCDC1 displayed equivalent genomic characteristics to Ro-BatCoV HKU9 except for an inserted gene at 3&#8217; end (discussed in detail in the next section). The major open reading frames (ORFs) had the identical order, namely 5&#8217;- replicase ORF1ab&#8212;spike (S)&#8212;NS3&#8212;envelope (E)&#8212;membrane (M)&#8212;nucleocapsid (N) followed by the accessory genes encoding nonstructural proteins (NSPs) (Fig 1 and Table 1), although the N gene was truncated. Amino acid sequence analyses showed that the ORF1ab, S, NS3, E, M and N proteins of Ro-BatCoV GCCDC1 shared higher identities with Ro-BatCoV HKU9 strains than those of other betacoronaviruses (Table 1). Also of note was that the 3&#8217;-end of Ro-BatCoV GCCDC1 genome, just downstream of N gene, possessed a much more complicated structure than those of other members in the genus. Clearly, there were four NSP-encoding ORFs. According to the convention, the second-to-fourth ORFs were temporarily named NS7a, NS7b and NS7c, respectively, since they shared 29% - 53% amino acid identities with the accessory genes of Ro-BatCoV HKU9 strains and other related bat coronaviruses (S1 Table). Perhaps the most striking feature of Ro-BatCoV GCCDC1 genome was the presence of a small intact ORF with 276 bases embedded between the N and NS7a genes. Although this ORF that had no homology to any known coronavirus, the encoded protein exhibited 30% - 54.9% amino acid identity with the p10 protein encoded by the first ORF of segment S1 of avian and bat fusogenic orthoreoviruses [20], which are double-stranded segmented RNA viruses belonging to the family Reoviridae. Therefore, this ORF was provisionally marked as p10 according to the molar weight of protein that it encodes (Fig 1).The putative leader and body transcription regulatory sequences (TRSs) of Ro-BatCoV GCCDC1, and their genomic localizations, were predicted in accordance with consensus core sequences of the TRSs of betacoronaviruses (Table 1). The TRS core sequence, 5&#8217;-ACGAAC-3&#8217;, was consistent with those of SARS-CoV, Ro-BatCoV HKU9 and other betacoronaviruses. From the location of leader TRS, the leader sequence of the genome was then identified, which spanned genome positions 1 (G) to 78 (C) (Fig 1 and Table 1). Notably, in the putative TRS of the p10 gene, there was one nucleobase difference with the consensus core sequence (Table 1).The putative mature nonstructural proteins (NSPs) in the ORF1ab encoding the replicase were calculated based on the cleavage and recognition pattern of the 3C-like proteinase (3CLpro) and papain-like proteinase (PLpro). Comprehensive information on the size and genomic locations of nsp1 to nsp16 and the putative cleavage sites of proteinases is presented in Table 2. Previous studies indicated that the P1 position of 3CLpro specific cleavage site is exclusively occupied by a glutamine (Q) residue [22, 23]. However, nucleobase 12642 in the Ro-BatCoV GCCDC1 genome was a T nucleotide, thereby changing glutamine (Q) to histidine (H). More interestingly, there were no glutamine codons in the sequence (from -273 to +192) around this site, as also observed in the corresponding site in the genomes of Ro-BatCoV HKU9. Therefore, the LH|AG region may represent a potential alternative cleavage site of 3CLpro to cleave between NSP9 and NSP10. A similar phenomenon may occur at the cleavage site between NSP10 and NSP12 of Ro-BatCoV GCCDC1, where the CAG codon has mutated to CAC causing the conversion of Q to H in amino acid sequence (Table 2).Following the criteria for coronavirus species demarcation defined by the ICTV [1, 13], seven conserved replicase domains of Ro-BatCoV GCCDC1 were selected for analysis (Table 3). The amino acid identities of seven concatenated domains in Ro-BatCoV GCCDC1 revealed that they shared 84.4% - 84.8% identity with those of Ro-BatCoV HKU9, which was below the 90% threshold used for species demarcation (Table 3). Hence, these data suggest that the newly identified Ro-BatCoV GCCDC1 represents a novel coronavirus species in the genus betacoronavirus.To determine the evolutionary position of Ro-BatCoV GCCDC1, the RdRp, S, N and p10 proteins were subjected to phylogenetic analyses. Phylogenetic trees of the RdRp, S and N proteins illustrated that Ro-BatCoV GCCDC1, Eidolon bat coronavirus/Kenya/KY24/2006 (Ei-BatCoV Kenya), Rousettus bat coronavirus/Kenya/KY06/2006 (Ro-BatCoV Kenya) and Ro-BatCoV HKU9 strains all belong to group D of the genus betacoronavirus (Fig 2). Within this cluster the two Ro-BatCoV GCCDC1 strains (346 and 356) formed a distinct lineage that was a sister-group to Ro-BatCoV Kenya and the Ro-BatCoV HKU9 strains (maximum bootstrap value of 100%). However, a strikingly different phylogenetic pattern was observed in the distinctive p10 protein (Fig 3), in which the Ro-BatCoV GCCDC1 sequences were clearly related to bat (Pteropine) originated orthoreoviruses. Although the branch leading to the Ro-BatCoV GCCDC1 sequences is long, these viruses are clearly more closely related to the bat-origin than avian-origin orthoreoviruses (Fig 3), matching the host species from which Ro-BatCoV GCCDC1 was isolated.\nEvidence for heterologous recombination of a reovirus p10 gene recombination into Ro-BatCoV GCCDC1\nTo exclude the false amplification of DNA polymerase or the inaccurate assembly of NGS data, the NGS data was analyzed further. Read mapping determined that there were a set of reads that covered the upstream junction site (i.e. the recombination break-point) between N and p10 genes, and a downstream junction site between the p10 and NS7a genes (S1 Fig with data in S1 File). In addition, the integrity and continuity of context sequence surrounding the p10 gene were confirmed with specific primers. Agarose gel electrophoresis showed that the PCR products were intact fragments of the expected length. The amplicons were cloned for sequencing. As shown in Fig 4A, the sequence obtained covers, without interruption, the partial N gene, the whole p10 gene and partial NS7a gene (data in the S2 File). Hence, there is clear evidence that the recombination event that placed the reovirus p10 gene in the Ro-BatCoV GCCDC1 genome was genuine.Sequencing information confirmed that the TRS of p10 gene is located within the encoding sequence of N gene with the core sequence of 5&#8217;-ACAAAC-3&#8217;, which exhibited a single nucleobase difference to the consensus core sequence (5&#8217;-ACGAAC-3&#8217;) (Fig 4B). We also observed a 97 nucleobase sequence between the TRS and the p10 initiation codon (Fig 4B), which was much longer than the intervening sequences of other genes with the exception for that between the leader TRS and ORF1ab (Table 1). As shown in Fig 4B and S2 Fig, sequence comparisons also revealed that the location of TRS in the p10 gene could be discriminated from those of other genes, which are adjacent and downstream to the N gene of Group D Betacoronavirus. Notably, the ORF of the Ro-BatCoV GCCDC1 N gene was disrupted by the insertion of the &#8220;exotic&#8221; p10 gene, causing the truncation of eight amino acids at the 3&#8217;-terminus and a two amino acid deletion (Fig 5).\nSubgenomic structures of Ro-BatCoV GCCDC1\nAccording to the information provided above, the relative locations of the putative leader and body TRS(s) were identified in the genome of Ro-batCoV GCCDC1 (Fig 6A). Based on the TRSs and transcription mechanism of coronavirus, nine potential subgenomic mRNAs of Ro-BatCoV GCCDC1, including S, NS3, E, M, N, p10, NS7a, NS7b and NS7c, were depicted (Fig 6B). In addition to an identical 5&#8217; leader sequence, each lower subgenomic mRNA shared the same 3&#8217;-end structure with the upper one to comprise a 3' co-terminal nested set with the genome.The presence of subgenomic mRNAs is strong evidence of coronavirus replication in the infected cells. To determine if the bat, which sample was collected from, was likely the natural host of Ro-BatCoV GCCDC1, subgenomic mRNAs in the sample were probed with a comprehensive set of primers. The PCR products were confirmed on an agarose gel. As displayed in Fig 6C, the lowest band marked with a red arrow on each lane was the specific amplicon from each subgenomic mRNA as demonstrated in Fig 6B. However, additional amplified bands were also compatible with this inference. As shown as an example on the lane of the E gene, the upper band indicates that subgenomic mRNA NS3 was simultaneously amplified in this reaction. On each lane the lowest band was cloned for sequencing, while other bands were purified and sequenced directly. Since the specific amplicon of the subgenomic mRNA NS7c failed to be cloned into the vectors, the PCR product was used as template for a second round of nested PCR. The product was then confirmed as shown in the lane of NS7c-2 in Fig 6C, and the band was cloned for sequencing. The results (Fig 6D and S3 Fig) indicated that the core sequence of the leader and body TRS of each gene, the leader-body fusion sites, and the mode of generation of subgenomic mRNAs were consistent with the prediction and demonstration in Fig 6B, especially the p10 gene and its subgenomic mRNA. Therefore, the existence of subgenomic mRNA in the samples further proved that the p10 gene was an intact authentic gene in the genome of Ro-BatCoV GCCDC1.\nThe p10 gene of Ro-BatCoV GCCDC1 is functional\nDespite the orthoreovirus origin of p10, this protein exhibited 8 amino acid differences (including a 2 amino acid deletion) among the 28 &#8220;absolutely conserved&#8221; amino acids described previously (Fig 7). Hence, it is necessary to investigate whether the p10 gene of Ro-BatCoV GCCDC1 could play the same role as its reovirus homologs. For this purpose, the p10 gene of Ro-BatCoV GCCDC1 was transiently expressed in BHK-21 cells as well as the p10 gene of Pulau virus, which was used as a positive control. Wright-Giemsa and immunofluorescence staining showed that both genes had the same function to induce the formation of cell syncytia (Fig 8A and 8B and S4A Fig). Thus, the alteration of certain conserved amino acids did not impair the syncytiogenesis of p10 gene of Ro-BatCoV GCCDC1.The p10 subgenomic mRNA identified in the samples confirmed that the p10 gene could be transcribed from the genome of Ro-BatCoV GCCDC1. However, due to the failure of virus isolation (despite a great effort), there is no effective way to judge whether the p10 gene could be expressed during the virus replication cycle. Therefore, an artificial plasmid was constructed containing the transcribed p10 subgenomic mRNA, which confirmed the functional expression of p10 (Fig 8C). When the plasmid was transfected into BHK-21 cells, once again, cell syncytia were observed with Wright-Giemsa and immunofluorescence staining (Fig 8D and S4B Fig). Thus, this indirect evidence suggests that the p10 gene functions during the replication cycle of Ro-BatCoV GCCDC1.Immunofluorescence staining also showed that polyclonal antibodies of Ro-BatCoV GCCDC1 p10 protein reacted with the p10 protein of Pulau virus (Fig 8B and Fig 8D). In addition, the cross-reactivity further proved that the p10 gene of Ro-BatCoV GCCDC1 might have the same origin as those of fusogenic orthoreoviruses.As the p10 protein of Ro-BatCoV GCCDC1 is the first report of FAST protein in an enveloped virus, the conserved amino acids of the p10 protein of Ro-BatCoV GCCDC1 were mutated to determine whether they play a vital role in cell-to-cell fusion and syncytium formation as those sites in the p10 protein of reoviruses (S5A Fig) [24, 25]. Notably, no cell syncytia were observed for all mutant constructs of p10 which had substitutions in the previously defined key sites in the p10 protein of reoviruses (S5B Fig). This indicates that the functionality of p10 in Ro-BatCoV GCCDC1 also depends on traditional conserved domains relevant for the function of the FAST protein [24, 25].To confirm the expression of p10 by the Ro-BatCoV GCCDC1 virus, we performed Western blotting (WB) to detect the presence of p10 protein in the bat feces and concentrated rectal swab specimens. The results revealed the expression of p10 by Ro-BatCoV GCCDC1 itself (S6 Fig).\nDiscussion\nWe have identified a novel coronavirus, Ro-BatCoV GCCDC1, from Rousettus leschenaulti, that belongs to group D of the genus betacoronavirus and which is related to Ro-BatCoV HKU9 [18]. According to the criteria defined by ICTV [1], Ro-BatCoV GCCDC1 is sufficiently divergent to represent a novel bat coronavirus. More striking was that Ro-BatCoV GCCDC1 contains a p10 protein located at the 3&#8217;-end of the genome that appears to have captured from a bat-origin orthoreovirus by heterologous recombination.Homologous recombination events frequently occur during the viral RNA replication of coronaviruses, and are important for their evolution [10, 27&#8211;29]. However, it is also possible that coronaviruses are one of the few virus families that can experience heterologous recombination. For example, members of betacoronavirus group A possess an HE gene [30, 31] which was seemingly derived from ancestral influenza C virus, a negative-stranded RNA virus with a segmented genome [30, 31], and which would represent another case of inter-family recombination, although it has also been proposed that the HE gene might be captured from host RNA [32]. Uncommon inter-family recombination events have also been reported in chicken infectious anemia virus [33], bandicoot papillomatosis carcinomatosis virus type 1 [34], and recombinant viruses between Marek&#8217;s disease virus, fowlpox virus, and various avian retroviruses [35, 36]. In the current study, sequence, phylogenetic and functional analyses demonstrated that the p10 gene of Ro-BatCoV GCCDC1 was likely derived from an ancestral orthoreovirus, although that it occupies a divergent position in the phylogeny suggests that the direct ancestor of the recombination event has yet to be sampled. Hence, these data provide clear evidence for a putative inter-family recombination between a single-stranded, positive-sense RNA virus and a double-stranded segmented RNA virus. The mechanisms that underpin such inter-family heterologous recombination clear merit further investigation.The biggest difference between fusogenic and nonfusogenic orthoreoviruses is the presence/absence of a small protein encoded by the segment S1 of the genome, termed the fusion-associated small transmembrane (FAST) protein. The FAST proteins are the only known nonenveloped reovirus fusogens that can mediate cell-to-cell, but not virus-cell, membrane fusion to induce the formation of syncytia [20], and which might promote the dissemination of virus among cells [37]. Thus, the FAST proteins are the pathogenic determinants of fusogenic orthoreoviruses. To date, the FAST family comprises six members including p10 proteins encoded by avian- and bat-origin orthoreoviruses, p13, p14 and p15 encoded by broome virus, reptilian reovirus and bush viper reovirus and baboon orthoreovirus respectively, and p16 and p22 encoded by aquareoviruses. Intriguingly, a specific p10 gene was identified in Ro-BatCoV GCCDC1, so that this is the first report of a FAST protein in an enveloped virus and hence could represent the seventh member of FAST family.Unfortunately, the isolation of Ro-BatCoV GCCDC1 failed on cell culture in the present study, so it is difficult to determine the role of p10 gene during the life cycle of Ro-BatCoV GCCDC1. However, functional analysis showed that the coronavirus p10 gene could induce syncytium formation in the transfected cells, in the same manner as orthoreoviruses, which might be beneficial for cell-to-cell virus spread. It is therefore possible that the p10 protein enhances the transmission potential of Ro-BatCoV GCCDC1. Previous studies of the potential recombination between coronavirus and influenza C virus revealed the pivotal role of the shared HE gene for the pathogenesis of betacoronavirus group A [38]. Interestingly, human coronavirus HKU1, OC43 and bovine coronaviruses employ the HE protein to mediate receptor-destroying enzyme activity late in the infection cycle to facilitate viral progeny release and achieve efficient virus dissemination [39].Compared to nonfusogenic orthoreoviurses, fusogenic orthoreoviruses can cause severe pneumonia when infecting humans [40, 41], further implying that p10 is an important pathogenic determinant. Thus, the recombination of the reovirus-originated p10 into the Ro-BatCoV GCCDC1 may enable the novel virus to disseminate and replicate rapidly in the host, in turn leading to severe infections. In recent years, several coronaviruses, notably SARS-CoV and MERS-CoV, have caused severe pneumonia among humans [42, 43]. Because of the presence of human infected fusogenic orthoreoviruses such as Melaka virus (MelV) [44], there is obviously some risk that cross-family recombination events such as that described here may generate a novel coronavirus with altered pathogenicity. Our study therefore highlights the importance of investigating the mechanisms that might enable possible recombination between human coronaviruses and orthoreoviruses.In the protein sequence of ORF1ab encoded replicase of Ro-BatCoV GCCDC1, the regular P1 position at two 3CLpro cleavage sites, NSP9/NSP10 and NSP10/NSP12, contains a Q to H mutation which may impair the proteolytic efficacy and the release of NSP9, NSP10 and NSP12. As NSP12 is a typical RNA polymerase and the core of replication-transcription complexes (RTC) and NSP10 usually serves as a molecular switch that can interact with multiple NSPs to form complexes, the replication ability of Ro-BatCoV GCCDC1 might be suppressed by the decrease of release of these vital elements. It is also interesting to note that a similar situation may be observed at the NSP13/NSP14 cleavage sites of replicase polyprotein of human coronavirus HKU1 and human coronavirus NL63. Clearly, further investigation will need to focus on the isolation of the virus, construction of infectious clones, and the virulence and replication ability of Ro-BatCoV GCCDC1 influenced by knockout of p10 gene and/or reverse mutation of cleavage sites.Phylogenetically distinct virus species or lineages have been reported co-circulating in certain bat populations [45, 46]. Under this situation, co-infections of single host cells&#8212;the necessary requisite for recombination&#8212;are possible. By careful sequence analysis we show that the heterologous recombination event placed the p10 gene in Ro-BatCoV GCCDC1 is genuine. Previous studies showed the existence of p10-harboring orthoreoviruses in bat populations [40, 47&#8211;49], such that co-infections with bat coronaviruses and hence recombination events are clearly possible. In addition, a previous study reported that mammalian orthoreovirus, a type of nonfusogenic orthoreovirus, was isolated from a SARS-CoV patient along during the in 2003 outbreak [50]. We believe that future studies should investigate co-infections in specific bat cell lines using a coronavirus similar to Ro-BatCoV GCCDC1 or Ro-BatCoV HKU9 and the relevant orthoreoviruses, from which it will be possible to reveal more of the underlying basis of heterologous coronavirus recombination.\nMaterials and Methods\n\nEthics statement\nThe protocol in this study was approved by the Committee on the Ethics of Animal Care and Use of the Chinese Center for Disease Control and Prevention (Permit 20140509015). The study was conducted in accordance with the Guide for the Care and Use of Wild Mammals in Research of the People's Republic of China.\nCell cultures\nAfrican green monkey kidney cells (Vero E6), human epithelial colorectal adenocarcinoma cells (CaCo-2), human epithelial type 2 HeLa derivative cells (HEp-2) and human lung carcinoma cells (A549) were purchased from the China Center for Type Culture Collection, while baby hamster kidney (BHK-21) and Madin-Darby canine kidney (MDCK) cells were obtained from the Cell Resource Center of the Shanghai Institute for Biological Sciences, Chinese Academy of Sciences. The immortalized kidney cell line of Myotis Davidii (IKMD) was a generous gift of Dr. Zhengli Shi, Wuhan Institute of Virology, Chinese Academy of Sciences. Cells were grown in Eagle&#8217;s minimum essential medium (EMEM) (A549, BHK-21) or in Dulbecco&#8217;s modified EMEM (DMEM) (Vero E6, CaCo-2, HEp-2, MDCK, IBIE and IKMD) supplemented with 10% or 20% (CaCo-2) fetal bovine serum in a humidified chamber containing 5% CO2 at 37&#176;C.\nSample collection\nAll the bats analyzed here were captured at a roosting site with the assistance of villagers and staff of local the CDC office in Xishuangbanna, Yunnan Province, China. The rectal swab samples were collected and placed in the cryotube with viral transport medium (VTM) containing Earle's balanced salt solution (Invitrogen, United States), 5% bovine albumin, 50,000 &#956;g/ml vancomycin, 50,000 &#956;g/ml amikacin, 10,000 units/ml nystatin [51]. All samples were immediately stored in liquid nitrogen and then transported with dry ice to our laboratory in Beijing and stored in the ultra-low temperature freezer until used for RNA extraction.\nRNA extraction\nTotal RNA was extracted from 100 &#956;L of VTM suspension of each swab with the RNeasy Mini Kit (Qiagen, Germany) according to the manufacturer's protocol. The RNA was eluted in 60 &#956;L AVE buffer, of which 8 &#956;L RNA was used as the template for RT-PCR immediately, or stored at &#8722;80&#176;C until use.\nPan-coronavirus RT-PCR\nTotal RNA extracted from the rectal swab suspension was screened for the presence of coronavirus RNA using pan-coronavirus RT-PCR with universal degenerate primers. The primers were designed from a highly conserved region of the RdRp (primer sequences are presented in S4 Table). After the reverse transcription and synthesization of cDNA with SuperScript III Reverse Transcriptase (Invitrogen, United States), a semi-nested PCR was performed. The expected amplicons of two rounds were 299 bp (using primers panCoVs-OF and panCoVs-OR) and 228 bp (using primers panCoVs-IF and panCoVs-OR) in length, respectively. All positive results were repeated and confirmed with fresh RNA extracts from the original bat rectal swab suspensions. Purified DNA amplicons (both rounds) were sequenced bi-directionally with pan-coronavirus sequencing primers (S4 Table) on an ABI Prism 3730 automated capillary sequencer (Applied Biosystems, United States).\nComplete genome sequencing\nFresh RNA was extracted from sample numbers 346 and 356 which were confirmed as coronavirus positive. The RNA were subjected to Next Generation Sequencing (NGS) using the Ion Proton platform. The original NGS data were filtered, refined and mapped to the reference sequence of Ro-BatCoV HKU9 (GenBank accession number NC_009021) using SOAP (Short Oligonucleotide Alignment Program) [52]. Any remaining gaps in the genome were closed by PCR amplification of these regions with specific primers and then sequenced. Complete genome sequences were confirmed with Sanger sequencing on the fragments amplified with a set of primers that covered the whole genome. The 5&#8217;- and 3&#8217;-RACE analyses were performed with 5&#8217;- and 3&#8217;- Full RACE Kit (Takara, Japan) according to the manufacturer&#8217;s instructions.\nGenome analyses\nAs the amplification of 5&#8217;-end of the genome of Ro-BatCoV GCCDC1 strain 346 was unsuccessful, we focused our genome analyses on the complete genome of Ro-BatCoV GCCDC1 strain 356. This genome was compared to those of eight complete genomes of Ro-BatCoV HKU9 (GenBank accession numbers NC_009021, EF065514, EF065515, EF065516, HM211098, HM211099, HM211100 and HM211101, respectively) to annotate the 1ab, S, NS3, E, M and N ORFs, respectively. As the origin of the ORFs at the 3&#8217;-end of the genome were uncertain they were also blasted (tblastx) against the GenBank database. The amino acid sequence of ORF1ab was aligned with the reference sequences of SARS-CoV, human coronavirus HKU1, infectious bronchitis virus, turkey coronavirus, bovine coronavirus, mouse hepatitis virus and porcine epidemic diarrhea virus (GenBank accession numbers NC_004718, NC_006577, NC_001451, NC_010800, NC_003045, NC_001846 and NC_003436, respectively) to determine the cleavage and recognition patterns of the C-like proteinase and papain-like proteinase of the 16 nonstructural proteins. In addition, the sequences of the 5&#8217; untranslated region (5&#8217;-UTR) and 3&#8217; untranslated region (3&#8217;-UTR) were defined, and the leader sequence, the leader and body TRSs were illustrated, based on comparison with SARS-CoV.\nConfirmation of the p10 gene\nTo eliminate the possibility of false amplification of DNA polymerase or inaccurate assembly of NGS data, the raw NGS data were further scrutinized and reads extracted for mapping to check the continuity of the p10 sequence, especially the upstream junction site between N and p10 genes and the downstream junction site between the p10 and NS7a genes. In addition, two sets of specific primers were designed to confirm the integrity and continuity of sequence surrounding the p10 gene (primer sequences shown in S5 Table). The amplicons were subsequently cloned into the pMD18-T vector and recombinant plasmids were subjected to Sanger sequencing.\nPhylogenetic analyses\nTo determine the phylogenetic position of the newly identified coronavirus among the known diversity of coronaviruses, the amino acid sequences of the RdRp, S, and N proteins were used for phylogenetic analyses (GenBank accession numbers shown in S2 Table). In the case of the imported p10 gene, homologous sequences of orthoreoviruses were utilized as the background data set in the phylogenetic analysis (GenBank accession numbers listed in S3 Table). All amino acid sequences were aligned using MUSCLE [53], and all poorly or ambiguously aligned regions were removed using GBlocks [54]. Because of the short length of the p10 and N amino acid sequence alignments, more relaxed GBlocks parameters were used in these cases. In all cases phylogenetic trees of amino acid sequence alignments were inferred using the maximum likelihood method available in the PhyML package [55], with bootstrap values estimated from 1,000 replicate trees. Each tree was inferred using the LG model of amino acid substitution with values of the gamma shape parameter inferred using ProtTest [56]. Finally, all phylogenetic trees were displayed and annotated with FigTree.\nVirus culture and attempted virus isolations\nSamples positive for coronavirus were cultured in Vero E6, BHK-21, MDCK, A549, HEp-2, CaCo-2 cells, as well as in an immortalized kidney cell line of Myotis Davidii. The cell lines were inoculated with positive samples and three blind passages were performed for each sample. The culture supernatant and cell pellet of each passage were harvested. The detection of viral replication was conducted using specific primers targeting the conserved region of RdRp.\nSubgenome identification and sequencing\nNested subgenomic mRNAs are generated during the replication cycle of coronaviruses. Hence, the identification of subgenomic mRNAs in the samples provides strong evidence for the replication of coronavirus. To analyze the possibility of replication in the newly identified bat coronavirus, primers were designed to determine the presence of viral subgenomic mRNAs in the coronavirus-positive bat rectal swab samples. Forward primers were designed targeting the leader sequence at the 5&#8217;-end of the complete genome and the putative subgenomic mRNAs, while reverse primers were designed within the ORFs or downstream of the corresponding gene (primer sequences are shown in S6 Table). Specific amplicons, that matched the expected length, were purified and then cloned into the pMD-18T vector for sequencing, while the additional suspected bands on the agarose gels were excised, purified, and then subjected to direct sequencing. Since the specific amplicon of subgenomic mRNA NS7c failed to be cloned into the vectors, the PCR product was used as a template for a second round of nested PCR. The product was then confirmed with agarose gel electrophoresis and the band was cloned for sequencing.\np10 antiserum production\nThe protein family of the putative p10 protein was analyzed using PFAM [57] and InterProScan [58]. Prediction of transmembrane domains was performed using TMHMM [59], TMpred and PredictProtein [60]. Peptides corresponding to the ectodomain (from amino acid positions 2&#8211;37) and the cytoplasmic domain (the last 33 amino acids) (peptide sequences are described in S7 Table) of the putative p10 protein were synthesized (Xuheyuan Biological Technology Co., LTD, Beijing, China). After conjugation with keyhole limpet hemocyanin (KLM), the synthesized peptides were used to immunize mice for antibody production. The mice (five mice per peptide) were injected intramuscularly at their hind legs with 20 &#956;g of the conjugated peptide mixed with adjuvant, followed by boosts until 14 days with the same conjugated peptides. Seven days after the boosts, the mice were killed and their blood collected to isolate sera. Antibody titers were determined using enzyme-linked immunosorbent assay (ELISA).\nSyncytial analysis and cell staining\nIn the cells infected with avian- or bat-origin fusogenic orthoreoviruses, the formation of cell syncytia depends on a p10 protein, which is encoded by the first ORF in segment S1 of the reovirus genome. It was previously demonstrated that amino acid residues of p10 proteins could be sorted into absolutely, highly, moderately and non-conserved amino acids [26].Sequence and phylogenetic analyses indicated that the p10 gene of Ro-BatCoV GCCDC1 most likely originated in an orthoreovirus. Comparative sequence analysis revealed that although the majority of key amino acids and motifs of the Ro-BatCoV GCCDC1 p10 protein were conserved, there were 8 amino acids differences (including 2 deletions) among the 28 so-called &#8216;absolutely conserved&#8217; amino acids that characterize members of the FAST family (Fig 7). Hence, it is necessary to explore the potential role of the p10 gene during the life cycle of Ro-BatCoV GCCDC1.To determine whether the putative p10 protein could play the same role as homologous proteins of avian and bat orthoreoviruses, the p10 gene was cloned into the pCAGGS vector and the recombinant plasmid (Fig 8A) was then transfected into BHK-21 cells using Polyethylimine (PEI, Polysciences Inc.) according to the manufacturer&#8217;s protocol. At the appropriate time post-transfection, cell-to-cell fusion was observed for the syncytium formation under the light microscope using Wright-Giemsa staining and an indirect immunofluorescence assay employing the polyclonal antibodies prepared above. The p10 gene of Pulau virus, a bat orthoreovirus [49], was also cloned into the pCAGGS vector to serve as a positive control. Cells transfected with an empty pCAGGS vector were used as a mock control.The next step is to confirm that p10 can be transcribed or translated during the replication cycle of Ro-BatCoV GCCDC1. We confirmed that the p10 gene could be transcribed from the genome during the replication cycle of Ro-BatCoV GCCDC1, with the p10 subgenomic mRNA representing a distinct signal. Further, we cloned the deduced p10 subgenome into a pcDNA3.0-derived vector to construct an artificial plasmid (Fig 8C), which could be transcribed out of an mRNA that is consistent with the p10 subgenomic mRNA in the infected cells of the host. The recombinant plasmid was transfected into BHK-21 cells and cell syncytia were observed as described above. The recombinant plasmid of Pulau virus p10 gene was still served as positive control. Cells transfected with empty pcDNA3.0 vector were used as mock control.\nMutational analysis of the p10 protein of Ro-BatCoV GCCDC1\nAs the p10 protein of Ro-BatCoV GCCDC1 is the first reported in an enveloped virus, we first tried to define the key amino acids for the p10 protein in the Ro-BatCoV GCCDC1 as described previously for p10 protein of reoviruses [24, 25]. For syncytial indexing of six mutant constructs, each well of BHK-21 monolayer cells in a 6-well plate were transfected with 2 &#956;g of plasmid DNA using Polyethylimine (PEI, Polysciences Inc.) and incubated for 5 h before replacing the transfection mixture with DMEM growth media (Invitrogen) supplemented with 10% fetal bovine serum (GIBCO). Transfected cells were paraformaldehyde-fixed and stained with Wright-Giemsa at the indicated times, and syncytia were observed and pictures were taken at &#215;100 magnification on an Olympus IX51FL+DP70 microscope.\nWestern blotting (WB)\nThe original specimens of bat rectal swabs and feces were used to test the expression of p10. BHK-21 cells with transient expression plasmid of p10 gene (pCAGGS-p10) were used as a positive control. Briefly, bat specimens and BHK-21 cell lysates were subjected to SDS-PAGE and transferred to a PVDF membrane. The membranes were blocked with a 5% non-fat dry milk solution and incubated with p10 antibody overnight at 4&#176;C followed by peroxidase-conjugated affinipure goat anti-mouse IgG (H+L) (Zhongshan Goldenbridge, Beijing). After washing with TBS-T buffer, the membrane was treated with ImmobilonTM Western Chemiluminescent HRP Substrate (Millipore, Billerica) and pictures were taken with Chemiluminescence System MicroChemi 4.2 (DNR Bio-Imaging Systems Ltd, USA).\nAccession numbers\nThe complete genome sequences of Ro-BatCoV GCCDC1 strains 346 and 356 have been deposited in the GenBank database and assigned accession numbers KU762337 and KU762338, respectively. We also deposited the sequences of the p10 genes from the rectal swabs of 24 bats in GenBank. All these accession numbers are listed in S8 Table.\nSupporting Information\n",
        "pmid": "27676249",
        "title": "A Bat-Derived Putative Cross-Family Recombinant Coronavirus with a Reovirus Gene",
        "journal_title": "PLoS Pathogens",
        "authors": "Canping Huang;William J. Liu;Wen Xu;Tao Jin;Yingze Zhao;Jingdong Song;Yi Shi;Wei Ji;Hao Jia;Yongming Zhou;Honghua Wen;Honglan Zhao;Huaxing Liu;Hong Li;Qihui Wang;Ying Wu;Liang Wang;Di Liu;Guang Liu;Hongjie Yu;Edward C. Holmes;Lin Lu;George F. Gao",
        "doi": "10.1371/journal.ppat.1005883"
    },
    {
        "keywords": "",
        "abstract": "Objectives\nTo assess the prevalence of canine parainfluenza virus, canine adenovirus type 2, canine distemper virus, canine respiratory coronavirus and influenza virus A infections in: (1) privately&#8208;owned or, (2) kennelled dogs showing signs consistent with canine infectious respiratory disease and, (3) clinically healthy dogs in Vienna, Austria.Materials and Methods\nProspectively, nasal and tonsillar swabs from 214 dogs affected with infectious respiratory disease, and 50 healthy control dogs were tested for nucleic acids specific to the various viral infections. Concurrent bronchoalveolar lavage fluid from 31 dogs with chronic respiratory disease was investigated for the same viral pathogens. Additionally, anti&#8208;canine respiratory coronavirus antibody concentrations were measured in paired blood samples from 30 acutely diseased dogs.Results\nCanine respiratory coronavirus (7.5%) and canine parainfluenza virus (6.5%) were the most commonly detected viruses in samples from the upper airways of dogs with respiratory infections. Serological results showed a significant seroconversion in response to coronavirus in 50% of the examined cases. None of the samples was positive for influenza virus A&#8208;specific nucleic acid. Canine coronavirus&#8208;specific nucleic acid was detected in 4.0% of healthy dogs.Clinical Significance\nCanine coronavirus should be considered as a clinically relevant cause of infectious respiratory disease in crowded dog populations. For sample collection, the nasal mucosa can be recommended as the favoured site. Analysis of paired serum samples aids verification of canine coronavirus infection in respiratory disease.",
        "body": "INTRODUCTION\nCanine infectious respiratory disease (CIRD), synonymous for infectious tracheobronchitis or &#8220;kennel cough,&#8221; is a disease caused by single or multiple infectious agents with a high worldwide prevalence. Apart from several viral and bacterial agents, the individual health and constitution, vaccination status and environmental influences including husbandry conditions (e.g. crowding of animals) may have an impact on the manifestation of clinical signs. Non&#8208;complicated forms of typically self&#8208;limiting character may be distinguished from complicated forms associated with, possibly fatal, pneumonia. A severe course of disease typically develops as a consequence of coinfections (Chalker et&#8239;al. 2003, Chvala et&#8239;al. 2007, Schulz et&#8239;al. 2014a). However, even isolated viral infections [e.g. canine influenza virus (CIV)] may lead to clinically relevant and sometimes lethal respiratory disease (Crawford et&#8239;al. 2005). Commonly recognised viral causes of CIRD are canine parainfluenza virus (CPiV), canine adenovirus type 2 (CAV&#8208;2) and canine distemper virus (CDV) (Ford 2012).However, according to more recent studies, the understanding of this disease complex has changed. New viral pathogens have been detected within the past two decades. In 2003, canine respiratory coronavirus (CRCoV) emerged as a cause of CIRD in a rehoming centre in the UK (Erles et&#8239;al. 2003). Further studies from several countries detected CRCoV&#8208;specific nucleic acid in dogs suffering from respiratory disease (Yachi & Mochizuki 2006, Decaro et&#8239;al. 2007, Spiss et&#8239;al. 2012, Schulz et&#8239;al. 2014a, Viitanen et&#8239;al. 2015).In association with an outbreak of respiratory disease in racing greyhounds in Florida, CIV types closely related to influenza subtype H3N8, originally detected in horses were isolated (Crawford et&#8239;al. 2005). Subsequently, several studies from different countries detected CIV isolates in respiratory samples and concurrent anti&#8208;CIV antibodies in dogs with mild respiratory signs as well as cases of fatal respiratory disease (Yoon et&#8239;al. 2005, Daly et&#8239;al. 2008, Payungporn et&#8239;al. 2008, Song et&#8239;al. 2008, Kirkland et&#8239;al. 2010, Li et&#8239;al. 2010, Song et&#8239;al. 2013). Furthermore, isolation of human&#8208;related influenza strains from dogs was successful (Lin et&#8239;al. 2012). To date, at least seven influenza virus subtypes showing different ability of interspecies and intraspecies transmission have been isolated from dogs. These subtypes are mainly prevalent in the USA (H3N8), Eastern China and South Korea (e.g. H3N2), but some of them have also been reported from European countries (Sun et&#8239;al. 2013, Xie et&#8239;al. 2016) supporting the hypothesis that dogs may play a role in transmission and spread of influenza virus among animal species and even humans.Detection of further viral pathogens (e.g. canine herpesvirus, canine reovirus, canine pneumovirus (CnPnV), pantropic canine coronavirus, canine hepacivirus and canine bocavirus) has been associated with respiratory disease in dogs (Buonavoglia & Martella 2007, Decaro & Buonavoglia 2008, Kawakami et&#8239;al. 2010, Renshaw et&#8239;al. 2010, Decaro & Buonavoglia 2011, Kapoor et&#8239;al. 2011, Kapoor et&#8239;al. 2012, Mitchell et&#8239;al. 2013b, Priestnall et&#8239;al. 2014). However, these viruses are uncommonly detected in dogs with CIRD or their possible role as causative agents is not yet completely determined.The aim of this study was to assess the prevalence of common CIRD&#8208;associated viruses (CPiV, CAV&#8208;2, CDV) in dogs in and around Vienna, Austria. Although there may be environmental factors specific to this location, our findings are likely to generalise to other locations within Western Europe and possibly further afield. It was further investigated whether emerging viruses (CRCoV and CIV) have a significantly higher prevalence in dogs with CIRD compared to dogs without respiratory disease.\nMATERIALS AND METHODS\n\nAnimals\nSamples were collected from 214 dogs with signs of CIRD between April 2013 and October 2015. The main criterion for inclusion was coughing. A complete history was obtained, and a thorough physical examination was followed by further diagnostic evaluation (e.g. complete blood count, blood chemistry, thoracic radiographs) as indicated to exclude non&#8208;infectious causes of respiratory disorders (e.g. congestive heart failure, airway foreign bodies, airway&#8208;compressive neoplasia). All dogs included in this prospective study were either presented at an academic or a private veterinary clinic, or recruited from animal shelters located in the following Austrian regions: Vienna, Lower Austria, Burgenland and Styria, Austria.Dogs meeting the inclusion criteria were separated into two groups according to different forms of husbandry: group A consisted of 173 privately owned dogs and group B included 41 dogs from animal shelters. Fifty clinically healthy privately owned dogs sampled during routine examinations and procedures (annual vaccination, castration) served as controls (group C). In order to compare the results within the population, the dogs were additionally divided into three subgroups of age: puppies (younger than 6&#160;months), adolescent dogs (from 6 to 18&#8201;months) and adults (older than 18&#8201;months). According to the period of time from onset of clinical signs until presentation and sample generation the dogs were also classified into two subgroups: acute (within 14&#8201;days of onset), and chronic (after 14&#8201;days of onset).\nSampling and sample preparation\nNasal and tonsillar swabs (sterile cotton swabs) were collected from all 264 dogs (groups A, B, C) in this study. In 31 animals with progressive respiratory disorders that had been resistant to previous treatment, bronchoalveolar lavage fluid (BALF) samples were additionally collected during tracheobronchoscopy. From 30 acutely diseased dogs, paired blood samples were retrieved for serological examinations.Swab samples were transferred into tubes containing 1&#160;mL of diethyl pyrocarbonate&#8208;treated water and vortexed for 10&#160;seconds. BALF obtained from right and left lung was pooled in equivalent amounts. Swab and BALF samples were frozen at &#8722;80&#176;C until further analysis.Serum was prepared by centrifugation (10&#160;minutes at 3000g), was subsequently lifted from the blood samples, and frozen at &#8722;20&#176;C.\nPCR\nFor nucleic acid extraction from swabs and BALF samples, QIAamp Viral RNA mini kit (QIAGEN) was used according to the manufacturer's instructions. Despite its name, this kit allows extraction of RNA as well as DNA from different kinds of samples. If not immediately processed, nucleic acid extracts were stored at &#8722;80&#176;C.For detection of virus&#8208;specific nucleic acids of CAV&#8208;1/CAV&#8208;2, CDV, CPiV, CRCoV and influenza virus A, different PCR&#8208;techniques and protocols were followed. Published protocols were adapted to commercially available PCR kits quoted below (see also Table&#160;1).The PCR protocol for CAV&#8208;1/CAV&#8208;2 published by Nell et&#8239;al. (2000) was carried out using the Fast Cycling PCR Kit (QIAGEN). After an initial step of 95&#176;C for 5&#160;minutes, the PCR was run for 45&#8201;cycles of denaturation at 96&#176;C for 5&#160;seconds, primer annealing at 60&#176;C for 5&#160;seconds and primer extension at 68&#176;C for 21&#8201;seconds. PCR was finalised by an ultimate extension of 1&#160;minute at 72&#176;C.CDV&#8208;reverse transcription (RT)&#8208;PCR described by Frisk et&#8239;al. (1999) was carried out using QIAGEN's OneStep RT&#8208;PCR Kit. After two pre&#8208;steps (50&#176;C for 30&#8201;minutes and 95&#176;C for 15&#8201;minutes), the PCR was run for 45&#8201;cycles of denaturation at 94&#176;C for 30&#8201;seconds, primer annealing at 60&#176;C for 30&#8201;seconds and primer extension at 72&#176;C for 1&#160;minute. Final extension was at 72&#176;C for 10&#160;minutes. Owing to practical reasons, the protocol was changed using a comparable real&#8208;time RT&#8208;PCR assay published by Elia et&#8239;al. (2006) during the study. Overall, 18.9% of the samples from diseased dogs, and 34.0% of the samples from control dogs were examined using the Real time RT&#8208;PCR assay. Samples showing positive results after conventional PCR were reassessed via Real time RT&#8208;PCR assay.A nested RT&#8208;PCR protocol for detection of CPiV&#8208;specific nucleic acids (Erles et&#8239;al. 2004) was optimised by using OneStep RT&#8208;PCR Kit (QIAGEN) and was found to give better results if only the outer primer pair was used.Real time RT&#8208;PCRs for CRCoV (Spiss et&#8239;al. 2012), Influenza A viruses (WHO 2009) and CDV (Elia et&#8239;al. 2006) were carried out on an Applied Biosystems 7300 Real&#8208;Time PCR System in a reaction volume of 25&#8201;&#956;L using the SuperScript III platinum OneStep q&#8208;RT&#8208;PCR Kit (Invitrogen) following the manufacturer's guidelines.Positive controls and blanks were run with all PCRs. For the CRCoV q&#8208;RT&#8208;PCR a betacoronavirus (BCoV L9 strain), for the Influenza A q&#8208;RT&#8208;PCR an EIV 639/69 strain, and for the other PCRs a modified live vaccine (Virbagen canis SHPPi, Virbac) served as positive controls. Blanks consisted of sample&#8208;free extracts produced simultaneously to each extraction process.PCR products of conventional PCRs/RT&#8208;PCRs were visualised by performing gel electrophoresis on an 1.2% agarose gel at 100 Volt for 80&#8201;minutes. In case of positivity of CAV&#8208;1/CAV&#8208;2 PCR sequence analysis (Microsynth) was performed to distinguish between CAV type&#8208;1 and CAV type&#8208;2.\nSerology\nFor detection of antibodies against betacoronaviruses, 30 acute serum samples as well as the corresponding sera (obtained 2 to 3&#160;weeks later) of the convalescent dogs were examined by an indirect immunofluorescence test. Madin&#8208;Darby bovine kidney cells were disseminated on 96&#8208;well microtitre plates (100&#8201;&#956;L/well) and then incubated at 37&#176;C in a humid 5% CO2&#8208;atmosphere overnight. After washing the plates with phosphate&#8208;buffered saline (PBS) solution, the adherent cells were infected with BCoV strain 15317/82 and incubated at 37&#176;C for 48&#8201;hours. Subsequently cells were washed with PBS again and fixed with 100&#8201;mL of 96% ethanol.The sera underwent twofold serial dilutions from 1:20 to 1:5120 with PBS and immunofluorescence test was performed as follows:Ethanol was discharged, and the 96&#8208;well microtitre plates were washed three times with PBS; 50&#8201;&#956;L of the previously diluted sera per well were added and incubated at 37&#176;C for 30&#8201;minutes. Thereafter, the plates were washed three times with PBS and 50&#8201;&#956;L of 1:40 diluted fluorescein isothiocyanate (FITC)&#8208;conjugate (anti&#8208;dog IgG, Jackson) was added to each well. After incubation at 37&#176;C for another 30&#8201;minutes and three washing cycles with PBS, 50&#8201;&#956;L/well Eriochrome black T indicator (diluted 1:200 with PBS) was filled in each well of the 96&#8208;well microtitre plate to reduce background fluorescence. Plates incubated for 5&#160;minutes at room temperature before cells were washed three times with PBS once more. Finally, wells were filled with 50&#8201;&#956;L/well of glycerine buffer solution to prevent the cells from drying.For evaluation of the microtitre plates an inverse ultraviolet microscope was used. The highest dilution with a clear cytoplasmatic fluorescence was equivalent to the specific antibody titre of each serum sample. Samples that showed no fluorescence in dilution 1:20 were regarded as negative (no antibodies present). Each assay included a positive and a negative control serum.\nStatistical analysis\nThe obtained data are initially presented in a descriptive way and a 95% confidence interval was calculated. Analysis was performed using IBM SPSS.\nRESULTS\n\nStudy population\nThe dog population (n=264) predominantly comprised purebred dogs (66.7%) including 66 different breeds. Among these, the most common were Rottweilers (6.1%), Chihuahuas (5.7%), Labrador retrievers and Australian shepherds (both 3.0%). By age, the population consisted of 31 puppies (11.7%), 59 adolescent dogs (22.3%) and 174 adult dogs (65.9%).Of 214 dogs with respiratory signs (group A and B), 94 dogs were male and 120 were female. Their median age was 3.53&#8201;years (min 0.08; max 15.0). Of these, 140 dogs were presented with acute onset of signs (65.4%), and 56 were chronically diseased dogs (26.2%). For the rest of the study population these data were not available. Of the investigated diseased dogs, more than two&#8208;thirds (72.0%) were adequately core vaccinated (against CAV&#8208;2, CDV) and almost half of them (45.8%) additionally vaccinated against CPiV.The control group C consisted of 18 males and 32 females. Their median age was 1.33&#8201;years (min 0.17; max 13.3). In this group, 66.0% of the dogs were vaccinated against CAV&#8208;2 and CDV and, apart from two exceptions, also against CPiV.\nPCR results\nFocussing on upper respiratory tract samples (nasal and tonsillar swabs), viral nucleic acids were detected in 31 of 214 diseased dogs (14.5%). Sixteen dogs tested positive for CRCoV (7.5%), 14 dogs for CPiV (6.5%) and one of these dogs additionally for CAV&#8208;2&#8208;specific nucleic acid (0.5%). One single dog tested positive for CDV&#8208;specific nucleic acid (0.5%). In none of the obtained samples from the upper respiratory tract was CIV&#8208;specific nucleic acid detected. Of those 31 positive dogs, 21 were privately owned (group A), and 10 kept in shelters (group B). They consisted of five puppies, 12 adolescent dogs and 14 adult dogs. Twenty&#8208;seven of the 31 positive dogs (87.1%) showed acute onset of signs, three suffered from chronic disease (9.7%) and for one diseased dog this information was not available (Table&#160;2).Furthermore, upper respiratory tract samples from two dogs (4.0%) of the clinically healthy control group C tested positive for CRCoV&#8208;specific nucleic acid (Table&#160;2).Nine dogs from group A (5.2%) and seven dogs out of group B (17.0%) tested positive for CRCoV in either nasal, tonsillar or both samples at one time. One of these dogs belonged to the subgroup of puppies; nine dogs were from the adolescent subgroup and six animals from the subgroup of adult dogs. With one exception, all these animals showed acute onset of CIRD (93.7%).Fourteen diseased dogs (6.5%) tested positive for CPiV. From those, 11 belonged to group A and three to group B. They all harboured CPiV&#8208;specific nucleic acid in sample material from the nose and one dog concurrently from the tonsils. Four of these dogs were classified as puppies; three dogs were from the adolescent subgroup and seven dogs were adults. Twelve out of these 14 animals showed acute onset of clinical signs (85.7%), one dog was chronically ill, and for another dog this information was not available. Seven dogs (50.0%) were regularly vaccinated&#8208;including against CPiV.In one of these 14 CPiV&#8208;positive dogs, CAV&#8208;specific nucleic acid was detected concurrently. This dog was privately owned (group A) and tested positive for CAV in both nasal and tonsillar swabs and CAV&#8208;2 strain (Toronto) was confirmed by DNA sequencing. Belonging to the subgroup of adults this dog had been irregularly vaccinated and received its latest vaccine 45&#8201;days before sample collection. It presented with a several week history of clinical signs including severe coughing, nasal and ocular discharge, dyspnoea and fever. Apart from that case, in no other dog was viral nucleic acid of two or more different viruses detected. In addition, no other proband of the study tested positive for CAV.One dog from group A tested positive for CDV&#8208;specific nucleic acid in a sample retrieved from the tonsils. RNA sequencing enabled the identification of a CDV vaccine strain (Onderstepoort). The dog was an adult and presented with chronic respiratory disease but no other signs consistent with CDV infection. The vaccination status of this dog was unknown.Additional information regarding all PCR&#8208;positive dogs is summarised in Table&#160;3.All BALF samples collected from 31 chronically ill dogs revealed negative PCR results.\nSerologic examination\nFrom 30 available paired serum samples 17 (56.7%) were obtained from privately&#8208;owned dogs (group A). Five of these 17 samples (29.4%) showed a significant increase in anti&#8208;CRCoV antibody titres. The antibody titres of two dogs increased more than 128&#8208;fold. CRCoV&#8208;specific nucleic acid from nasal or tonsillar swabs was detected in these two dogs.Another 13 paired serum samples (43.3%) were collected from a population of 15 kennelled working dogs with an acute episode of CIRD (group B). As two of the 15 dogs were non&#8208;compliant with blood sampling, in these cases no serum samples were obtained. In 10 of the 13 paired serum samples (76.9%), a significant increase in anti&#8208;CRCoV antibodies was found. Six dogs revealed a 16&#8208; to 128&#8208;fold antibody titre increase and concurrent evidence of CRCoV RNA in sample material from the nose or tonsils. No further causative viral agent was detected in these cases (Table&#160;4).\nDISCUSSION\nVarious studies from Europe (Erles et&#8239;al. 2004, Schulz et&#8239;al. 2014b, Viitanen et&#8239;al. 2015, Decaro et&#8239;al. 2016), Japan (Mochizuki et&#8239;al. 2008) and the USA (Lavan & Knesl 2015) have reported detection rates for different viral pathogens causing CIRD in dogs. Influenced by their geographic origin, these studies show quite divergent results indicating a dynamic process of virus spread. Frequent surveillance is key for assessing the emergence and spread of novel, potentially zoonotic or vaccine resistant viral diseases. The present study was initiated to investigate the status quo of viral CIRD&#8208;associated pathogens in and around Vienna, Austria, with a population of almost 1.9 million citizens and 55,705 registered dogs (Magistrat Wien 2017).Despite the seasonal influenza epidemics in humans, in none of the obtained canine samples were influenza A virus RNA detected. This finding is in line with the results of other studies from Europe that also found no evidence of influenza A viruses within investigated dog populations (Schulz et&#8239;al. 2014b, Viitanen et&#8239;al. 2015, Decaro et&#8239;al. 2016, Mitchell et&#8239;al. 2017).In our study, one dog each tested positive for CDV&#8208; and CAV&#8208;2&#8208;specific nucleic acid in samples from the upper respiratory tract. In both cases, sequence analysis confirmed the detection of a vaccine&#8208;strain. This finding might be explained by the shedding of viral nucleic acid derived from recently administered live vaccines in these dogs (European Medicines Agency 2014, Wilkes et&#8239;al. 2014). CDV and CAV&#8208;2 field strains were not detected in this study. This is consistent with other studies reporting low detection rates of these viruses. In contrast, a study from Japan detected CDV in 1.5% and CAV&#8208;2 in 2.9% of the examined household dogs (Mochizuki et&#8239;al. 2008), and studies from Germany and Italy (Schulz et&#8239;al. 2014a, Decaro et&#8239;al. 2016) detected no CDV&#8208; or CAV&#8208;2&#8208;specific nucleic acid at all. However, CDV&#8208; and CAV&#8208;2&#8208;specific nucleic acid has been detected in 7.4 and 12.5% of asymptomatic shelter dogs in the USA (Lavan & Knesl 2015). This discrepancy most likely reflects consequent vaccination regimens for privately owned and kennelled dogs in Central Europe and indicates that vaccines against CDV and CAV are providing good protection.Interestingly, only solitary viral infections were detected within the examined population. This finding is in contrast to a study from Germany that reported 12.1% of viral coinfections in clinically ill and healthy dogs (Erles et&#8239;al. 2004). However, it is in accordance with the results of three other studies in which the detection rate of viral coinfections was 2.9, 3.3 and 1.4%, respectively (Mochizuki et&#8239;al. 2008, Schulz et&#8239;al. 2014a, Decaro et&#8239;al. 2016).Overall, viral nucleic acid from samples of the upper respiratory tract was detected in 31 of 214 diseased dogs (14.5%). A much higher detection rate of viruses was found in the examined shelter animals (24.4%, group B) than in the group of privately owned dogs (12.1%, group A). This finding meets the general opinion that CIRD predominates in crowded dog populations because of higher infection pressure in shelters (Erles & Brownlie 2008, Ford 2012). The most commonly detected pathogens in and around Vienna were CRCoV (7.5%) and CPiV (6.5%). Recent studies from other European countries revealed higher prevalence rates for CPiV, namely 37.7% (Schulz et&#8239;al. 2014a), 35.0% (Viitanen et&#8239;al. 2015) and 13.0% (Decaro et&#8239;al. 2016), respectively. In the present study, the comparably low detection rate of CPiV (6.5%) may be related to a generally high vaccination rate in the population under investigation, especially dogs in group B that were almost all vaccinated against CPiV (92.9%). Interestingly, 50% of the dogs that tested positive for CPiV were regularly vaccinated against it. As the time between vaccination and onset of signs was documented in most of the positive cases, positive results due to recent vaccination were unlikely. Detecting CPiV&#8208;specific nucleic acid in dogs despite a current vaccination status might be explained by the fact that the vaccine is not preventing CPiV infection but is rather diminishing the severity of disease caused by CPiV (Day et&#8239;al. 2016).The different detection rates may also be explained by different study designs. Whereas Schulz et&#8239;al. (2014a) solely included acutely diseased dogs presenting with multiple signs of infectious respiratory disease, Viitanen et&#8239;al. (2015) focused on the detection of viral coinfections in bacterial pneumonia or tracheobronchitis caused by Bordetella bronchiseptica. Both studies investigated severely diseased dogs. In contrast, the current study included all dogs presenting for clinical signs compatible with CIRD without regard to onset, extent or duration (excluding non&#8208;infectious causes). Leading to a broader spectrum of individuals, this should generalise our findings better into general veterinary practice.The relevance of CRCoV as a primary cause of CIRD remains unclear as the virus can be found in clinically healthy dogs although a moderate to high prevalence of CRCoV infection has been found in dogs with respiratory disease (Yachi & Mochizuki 2006, Schulz et&#8239;al. 2014a, Viitanen et&#8239;al. 2015). It has further been demonstrated that dogs being experimentally infected with five geographically&#8208;unrelated CRCoV isolates showed seroconversion and mild respiratory signs (Mitchell et&#8239;al. 2013a). In the present study, 16 diseased dogs (7.5%) tested positive for CRCoV&#8208;specific nucleic acid. These dogs were lacking concurrent proof of other viral pathogens. A recent European multi&#8208;centre study that focused on emerging pathogens in CIRD found a similar detection rate for CRCoV (7.7%) in the examined dog population but also indicated that the odds of CnPnV infection is doubled in CRCoV positive dogs (Mitchell et&#8239;al. 2017). As not all potential CIRD&#8208;associated viral agents have been evaluated in this study (e.g. CnPnV) viral coinfection cannot truly be excluded.The detection rate of CRCoV in group B (17.0%) was much higher than in group A (5.2%). Conversely, there was little difference in the detection rate of CRCoV between group A (5.2%) and the healthy control group C (4.0%). As seroconversion of asymptomatic home&#8208;raised working dogs after transferal to training centres at the age of 1&#160;year has been demonstrated (Erles & Brownlie 2005) the existence of subclinical CRCoV infections in certain dog populations has already been proven and could be an explanation for positive PCR results in some dogs. Nevertheless, the complementary serological examinations of the obtained paired serum samples in this study basically reveal good concordance to the PCR results for CRCoV, mostly confirming acute infection. This is especially true for 13 kennelled working dogs that showed seroconversion (fourfold increase in antibody titre or higher) in 76.9% of the cases during an outbreak of CIRD. Concurrent CRCoV&#8208;specific nucleic acid was detected in six of these dogs. As viral shedding has been shown to occur in the first days after infection, starting at day 2 and ceasing at day 6 in most cases (Mitchell et&#8239;al. 2013a), collection of swab samples too early or too late in the course of disease might be an explanation for negative PCR results in the remaining dogs. However, the results of the present study further contribute to the assumption that CRCoV can be a primary cause of CIRD especially in kennelled dog populations and emphasise the supportive value of obtaining paired serum samples to verify viruses as an underlying cause of disease.In the present study, only dogs with progressive, therapy&#8208;resistant signs attributed to CIRD underwent tracheobronchoscopy. As detection of viral pathogens in samples from the lower respiratory tract of chronically diseased dogs is thought unlikely (Ford 2012), it is not surprising that none of the obtained BALF samples revealed a positive viral PCR result.Other viral agents (e.g. canine herpesvirus, CnPnV, canine bocavirus) have been detected in dogs with respiratory disease (Buonavoglia & Martella 2007, Kapoor et&#8239;al. 2012, Mitchell et&#8239;al. 2013b). They are discussed as secondary contributing factors probably causing more severe CIRD. However, it cannot be excluded that some are actually opportunistic agents that benefit from the weakened immune system in affected dogs. In this study, these viruses were not investigated and so we cannot exclude them as causes or contributors to the clinical presentation in the evaluated dog population.In conclusion, CRCoV should be considered as a clinically&#8208;relevant cause of CIRD in crowded dog populations. Acutely diseased dogs of younger age are predisposed. For sample collection, the nasal mucosa can be recommended as the favoured site. Especially in larger dog shelters, supplementary use of paired serum samples is indicated to confirm the causal relevance of a certain virus in dogs with CIRD in order to implement adequate hygiene measures. Failure to detect CAV&#8208;2 and CDV infection in this study indicates good immunisation protection in the examined dog population and emphasises the relevance of regular core vaccinations. WSAVA guidelines for the vaccination of dogs consider vaccines against CPiV as non&#8208;core vaccines (Day et&#8239;al. 2016). This recommendation may be generally applicable for dogs in Austria unless they are exposed to possibly non&#8208;vaccinated dog populations (e.g. dog exhibitions, animal shelters). More studies are needed to discover the true relevance of CRCoV but also other emerging viral pathogens in the context of CIRD and to assess the need for new vaccines.\nConflict of interest\nNo conflicts of interest have been declared.",
        "pmid": "31301071",
        "title": "Detection of selected viral pathogens in dogs with canine infectious respiratory disease in Austria",
        "journal_title": "The Journal of Small Animal Practice",
        "authors": "A. Hiebl;A. Auer;G. Bagrinovschi;M. Stejskal;R. Hirt;H. T. R\u00fcmenapf;A. Tichy;F. K\u00fcnzel",
        "doi": "10.1111/jsap.13051"
    },
    {
        "keywords": "coronavirus;papain-like protease;autophagy;antiviral immunity;Beclin1;STING",
        "abstract": "Autophagy plays important roles in modulating viral replication and antiviral immune response. Coronavirus infection is associated with the autophagic process, however, little is known about the mechanisms of autophagy induction and its contribution to coronavirus regulation of host innate responses. Here, we show that the membrane-associated papain-like protease PLP2 (PLP2-TM) of coronaviruses acts as a novel autophagy-inducing protein. Intriguingly, PLP2-TM induces incomplete autophagy process by increasing the accumulation of autophagosomes but blocking the fusion of autophagosomes with lysosomes. Furthermore, PLP2-TM interacts with the key autophagy regulators, LC3 and Beclin1, and promotes Beclin1 interaction with STING, the key regulator for antiviral IFN signaling. Finally, knockdown of Beclin1 partially reverses PLP2-TM&#8217;s inhibitory effect on innate immunity which resulting in decreased coronavirus replication. These results suggested that coronavirus papain-like protease induces incomplete autophagy by interacting with Beclin1, which in turn modulates coronavirus replication and antiviral innate immunity.",
        "body": "Introduction\nAutophagy is an evolutionarily conserved intracellular process which plays an important role in eliminating damaged organelles and long-live proteins for maintenance of cellular homestasis (Mizushima, 2009). During autophagy, cytoplasmic constituents are engulfed by double-membrane vesicles termed autophagosomes that are destined for fusion with lysosomes for content degradation. Under stress conditions such as starvation, autophagic acitivity is greatly augmented from its basal levels, providing recyclable resources that help maintain metabolism and ATP levels for cell survival (Rabinowitz and White, 2010). Recent studies have demonstrated that autophagy is involved in various physiologic and pathologic processes including cancer, cardiovascular disease, metabolism, viral infections and immune response, etc. (Deretic, 2012; Levine and Deretic, 2007; Mihaylova and Shaw, 2011; Shintani and Klionsky, 2004; Yang et al., 2011).Autophagy is a dynamic process which includes initiation, nucleation, elongation and maturation. It is tightly regulated by a number of autophagy-related genes (ATG) and other multiple protein complexes (Kroemer and Levine, 2008). The ULK1-Atg13-FIP 2000 and class III PI3K complex, which contain Beclin1 and vps34, among other key components, regulate the initiation and nucleation of phagophore, the precursor of autophagic membranes. In the stage of elongation and complete closure of autophagosome membranes, two ubiquitin-like conjugation systems come into play. The first one is the Atg12-Atg5-Atg16L complex conjugation system and the second involves Atg4B, Atg3 and Atg7. The net outcome of the ubiquitin-like reactions catalyzed by these two systems is the conversion of microtubule associated protein 1B light chain 3 (LC3) from its cytosolic form (LC3-I) to the membrane bound, phosphatidylethanolamine conjugated form (LC3-II), a process known as LC3 lipidation. The conversion of LC3-I to LC3-II is a hallmark of autophagy and is indicative of autophagic activity, which can be monitored by LC3 immunoblotting or immunofluorescence (to monitor LC3 puncta formation) (Barth et al., 2010; Choi et al., 2013; Kabeya et al., 2000; Mizushima et al., 2010). The terminal step of autophagy, in which autophagosome is fused with lysosomes for content degradation, is assisted by UVRAG, VPS34, VPS15 and Beclin1 complex (Glick et al., 2010; Kuballa et al., 2012; Weidberg et al., 2011).During their co-evolution with hosts, viruses have acquired elaborate strategies to evade, counteract and sometimes co-opt protective mechanisms of host cells. Unfortunately, the autophagy pathway is one of these. While this lysosomal degradation pathway can lead to degradation of viral components, acting as an intrinsic antiviral defense mechanism, many viruses either block autophagy or exploit it for their own replication (Dong and Levine, 2013; Dreux and Chisari, 2010; Richards and Jackson, 2013; Sumpter and Levine, 2011). As a critical regulator in both the early and late steps of autophagy, Beclin1 has emerged as a prime target for different viruses, such as influenza virus, HIV, and herpes viruses (Munz, 2011). However, targeting Beclin1 by a positive-strand RNA virus has not been reported. Coronaviruses (CoVs) are a large group of positive-strand RNA viruses that replicate in cytoplasm of host cells but don&#8217;t induce high level of IFN products (Clementz et al., 2010; Devaraj et al., 2007; Zhou and Perlman, 2007). These viruses induce the formation of cytoplasmic double-membrane vesicles (DMVs) which are believed to be the site for assembly of viral transcription-replication complexes (Gosert et al., 2002; Hagemeijer et al., 2010; Knoops et al., 2008; Snijder et al., 2006). The origin of DMVs remains obscure, but these structures are often decorated with an autophage marker, LC3 (Bernasconi et al., 2012; Maier and Britton, 2012; Maier et al., 2013). Although coronavirus infections are known to be accompanied with activation of autophagy (Cottam et al., 2011; Knoops et al., 2008; Prentice et al., 2004; Reggiori et al., 2010), relatively little is known about the role autophagy may play in the coronavirus life cycle, as are the mechanisms of autophagy induction and its contribution(s) to viral regulation of host responses, especially those concerning innate immunity. Overall, these aspects have been under intense investigation, and new knowledge stemmed from such studies hold the key(s) to a better understanding of coronvirus pathogenesis.Previously, it has been shown that the papain-like protease (PLP) domains contained in the nonstructural protein (nsp) 3 of coronaviruses of SARS-CoV (Devaraj et al., 2007), HCoV-NL63 (Clementz et al., 2010) and porcine epidemic diarrhea Virus (PEDV) (Xing et al., 2013) act as IFN antagonists. Several mechanisms have been proposed. First, the SARS-CoV PLpro forms a complex with IRF3 and blocks IRF3 phosphorylation and nuclear translocation (Devaraj et al., 2007). Second, SARS-CoV PLP and PLP2 of HCoV-NL63 and PEDV also interact with and impede the dimerization of stimulator of IFN genes (STING, a.k.a, MITA/MPYS/ERIS) (Sun et al., 2012a). Third, the aforementioned coronavirus PLPs possess de-ubiquitinating activity, which may also contribute to the ability of these PLPs to disrupt host innate immunity by removing ubiquitin moieties from signaling molecules in the innate antiviral pathways (Bibeau-Poirier and Servant, 2008; Chen, 2012; Loo and Gale, 2011). Given that autophagy is increasingly implicated in virus-host interactions and activation of this pathway is dependent on ubiquitin-like reactions, we set out to determine whether PLP impacts on autophagy. Our data show that the membrane-associated PLP2 (PLP2-TM) of HCoV-NL63 and PEDV and PLpro-TM of SARS-CoV and the new-emerging Middle East respiratory syndrome coronavirus (MERS-CoV) (Yang et al., 2014) represent a novel class of viral proteins encoded by coronaviruses that induce autophagy. Interestingly, we found that PLP2-TM induces incomplete autophagy, leading to accumulation of autophagosomes while impairing the fusion of autophagosomes with lysosomes. Intriguingly, PLP2-TM was found to associate with LC3 and Beclin1, two critical components of the autophagy pathway, and knockdown of Beclin1 partially reversed the blockade of PLP2-TM on innate immune signaling to the IFN-&#946; promoter. The PLP-TM targeting of Beclin1 to control autophagic activity may represent a novel mechanism of coronaviral regulation of antiviral innate immunity.\nResults\n\nPLP2-TM is a novel autophagy-inducing protein encoded by coronavirus\nTo determine whether coronavirus PLPs regulate autophagy, we first studied the impact of ectopic expression of the transmembrane (TM)-containing form of PLP2 (PLP2-TM) of HCoV-NL63 on formation of autophagosomes (Chen et al., 2007; Clementz et al., 2010; Sun et al., 2012a) (Fig.&#160;1A). HEK293T cells were co-transfected with an eGFP-LC3B-expressing plasmid and the V5 eiptope-tagged PLP2-TM. At 48 h post-transfection, cells were fixed, immunostained using an anti-V5 antibody, and examined for PLP expression and evidence of autophagosome formation by confocal fluorescence microscopy. As a positive control, cells transfected with eGFP-LC3B for 48 h were treated with Rapamycin, a well-known autophagy inducer, for 6 h. eGFP-LC3 fluorescence microscopy is a well-accepted method of monitoring autophagosome formation, as measured by a change in the eGFP-LC3 fluorescence pattern from diffuse to punctate distribution (Klionsky et al., 2012). As shown in Fig.&#160;1B and Fig.&#160;1C, HEK293T cells had very low basal autophagic activity, with less than 5% of cells exhibiting punctate eGFP-LC3. Short-term treatment with Rapamycin efficiently induced autophagy, resulting in eGFP-LC3 puncta formation in ~40% of cells. Ectopic expression of PLP2-TM caused a clear redistribution of eGFP-LC3B to punctate structures, indicative of formation of autophagosomes (Fig.&#160;1B). Quantitative analysis revealed that while ~60% of cells expressing PLP2-TM exhibited eGFP-LC3 punctation (Fig.&#160;1C). These results suggest that besides the deubiquitinase (DUB) activity and IFN antagonist (Barretto et al., 2005; Chen et al., 2007; Clementz et al., 2010; Devaraj et al., 2007; Sun et al., 2012a), the PLP2-TM encoded by human coronavirus NL63 is a novel autophagy-inducer.Figure&#160;1PLP2-TM is a novel autophagy-inducing protein encoded by coronavirus. (A) Schematic diagram illustrating coronavirus NL63 genome, polyprotein (pp) 1a/b, the predicted processing of replicase polyproteins (pp) to nsp&#8217;s. The domains, including the predicted transmembrane (TM), in nsp3, and the membrane anchored-PLP2 construct (PLP2-TM) that were used in this study are indicated. (B) PLP2-TM is a novel autophagy-inducing protein encoded by coronavirus. The plasmid of HCoV-NL63 PLP2-TM-V5 was co-transfected with pcDNA3.1-eGFP-LC3B into HEK2923T cells. As the positive control for induction of autophagy, HEK293T cells were transfected with eGFP-LC3B for 48&#160;h and then treated with complete medium supplemented with 400&#160;nmol/L Rapamycin for 6&#160;h. The immunofluoresence of the cells were detected using a confocal microscope after stained with anti-V5-tagged primary antibody, followed by being stained with Alexa Fluor 594-conjugated goat anti-rabbit secondary antibody. The localization of eGFP-LC3B positive autophagosome accumulation (green) and the V5-tagged PLPs products (anti-V5, red) was visualized using a confocal microscope. (C) Quantification of cells displaying eGFP-LC3B puncta in PLP2-TM transfected cells. HEK 293T cells were transiently co-transfected with eGFP-LC3B and PLP2-TM-expression constructs. Forty-eight hours later, cells with eGFP-LC3B puncta formation were quantificated under a fluorescence confocal microscope. Three random fields, each containing at least 80 cells, were counted. Results from one representative experiment are shown in Fig.&#160;1B. (D) PLP2-TM induces autophagosome-like structures detected using a transmission electron microscope. HEK293T cells were transfected with pcDNA3.1 or PLP2-TM-V5 for 48&#160;h and then cells were analyzed for autophagosome formation using a transmission electron microscope. The cells treated by 400&#160;nmol/L Rapamycin were analyzed to serve as a positive control for induction of autophagy. Red arrows indicate representative autophagosome-like structures. N indicates the cellular nuclear. Scale bar indicates 500&#160;nmTo corroborate this finding, we performed transmission electron microscopy (TEM) to directly visualize autophagosomes in cells expressing PLP2-TM, and as a negative control, in cells expressing the empty vector. Double membrane vacuoles containing cytoplasmic organelles, which are characteristic of autophagosomes, were readily visible in the cytoplasm of PLP2-TM-transfected cells (Fig.&#160;1D, middle panel), as were in cytoplasm of Rapamycin-treated cells (Fig.&#160;1D, right panel). In contrast, no autophagosome-like structures were observed in cells transfected with the empty vector (Fig.&#160;1D, left panel). The number of autophagic vacuoles per cell was significantly higher in the PLP2-TM-transfected cells as compared with that in the empty vector-transfected cells (Fig.&#160;1D, middle panel vs. left panel). These data provide morphological evidence that PLP2-TM promotes autophagosome accumulation. In addition, they confirm that the autophagy-inducing effect was attributed to expression of PLP2-TM but not to the transfection reagent/process per se.\nExpression of PLP2-TM induces autophagy in multiple cell lines\nNext, we investigated whether PLP2-TM induces autophagic vesicle formation in cell lines other than HEK293T cells. To this end, HeLa and MCF-7 cells were co-transfected with eGFP-LC3B and either PLP2-TM or the empty vector, followed by examination of eGFP-LC3 puncta formation. Both cell lines had higher basal levels of autophagic activity than HEK293T cells, as demonstrated by the presence of a small number of eGFP-LC3 puncta in 30%&#8211;40% of cells expressing the empty vector (Fig.&#160;2A and 2B). The high basal levels of autophagic activity in control HeLa cells were not unexpected, as similar finding has been reported by others (Mizushima et al., 2010). Importantly, ectopic expression of PLP2-TM doubled the percentage of cells exhibiting eGFP-LC3 puncta in both HeLa and MCF-7 cells (Fig.&#160;2B).Figure&#160;2Autophagosome induced by CoV NL63 PLP2-TM in various cell lines. (A) PLP2-TM-V5 and pcDNA3.1-eGFP-LC3B were co-transfected respectively into HEK2923T, HeLa and MCF-7 cells. The cells were fixed at 48&#160;h post-transfection and were analyzed for eGFP-LC3B positive autophagosome accumulation (green) using a confocal microscope as described in Fig.&#160;1B. (B) Quantification of cells displaying eGFP-LC3B puncta from one representative experiment that shown in Fig.&#160;2A as described in Fig.&#160;1C. (C&#8211;E) HEK293T, HeLa and MCF-7 cells were transfectd with PLP2-TM-V5 or empty vector as a negative control for 48&#160;h. These cells were also treated by 400&#160;nmol/L Rapamycin to serve as a positive control for induction of autophagy. The cells were then lysed for Western blotting analysis using a rabbit anti-LC3 antibody to detect the endogenous LC3 expression (top panel in each Fig.&#160;2C&#8211;E). The whole cell lysate (WCL) was blotted using anti-V5 antibodies to evaluate expression of PLP2-TM (middle panel in each Fig.&#160;2C&#8211;E), and &#946;-Actin was detected from whole cell lysate (WCL) as a loading control (bottom panel in each Fig.&#160;2C to 2E)Activation of autophagy involves the association of LC3 with autophagic phagophore membranes that depends on the conversion from the cytosolic, soluble form, LC3-I, to the membrane-bound form, LC3-II. This ubiquitin-like modification, termed lipidation, can be illustrated by a change in migration on SDS-PAGE gel and is a widely used marker for monitoring autophagy (Klionsky et al., 2012; Mizushima et al., 2010). To further validate the finding that PLP2-TM promotes autophagy, we measured the expression levels of endogenous LC3-I and LC3-II in cells following expression of PLP2-TM, in comparison with cells expressing the control vector, by Western blotting. Consistent with the eGFP-LC3 fluorescence and TEM data, ectopic expression of PLP2-TM substantially increased the abundance of LC3-II in HEK293T (Fig.&#160;2C), HeLa (Fig.&#160;2D) and MCF-7 (Fig.&#160;2E) cells. Collectively, these data confirm that the induction of autophagy by PLP2-TM is not a cell-type specific phenomenon. It is of interest to point out that the basal levels of endogenous LC3-II were higher in HeLa cells than in HEK293T cells, an observation which was in agreement with the data on eGFP-LC3 punctation (Fig.&#160;2B).\nInduction of autophagy by various coronaviral PLPs in a protease-independent manner\nAs with HCoV-NL63 PLP2-TM, the PLpro-TM of SARS-CoV, MERS-CoV PLpro-TM and PLP2-TM of PEDV also possess de-ubiquitinating and interferon antagonism activities (Clementz et al., 2010; Devaraj et al., 2007; Xing et al., 2013; Yang et al., 2014). The finding that PLP2-TM of HCoV-NL63 induces autophagy prompted us to investigate whether the PLP2-TM homologues encoded by other coronaviruses also have a similar function. Western blotting was carried out to determine the expression of LC3-I and LC3-II in HEK293T cells ectopically expressing various coronaviral PLPs (Fig.&#160;3A). While LC3-II was undetectable in cells transfected with control vector (lane 1, Fig.&#160;3), its expression was strongly induced in cells expressing SARS-CoV PLpro-TM (lane 3, Fig.&#160;3A) or PEDV PLP2-TM (lane 5, Fig.&#160;3A), as was in cells expressing HCoV-NL63 PLP2-TM (lane 2, Fig.&#160;3). Although its effect was less pronounced, expression of the MERS-CoV PLpro-TM also caused substantial increase in LC3-II expression (lane 4, Fig.&#160;3A). Next, to determine if the protease activity of PLP2 is required for its induction of autophagy, NL63 PLP2-TM and its catalytic mutants (C1678A, H1836A and D1849A) (Fig.&#160;1A) were transfected into HEK293T cells and the conversion from the cytosolic, soluble form, LC3-I, to the membrane-bound form was detected. We found that, like as the PLP2-TM, ectopic expression of all the catalytic mutants substantially increased the abundance of LC3-II, indicating that the catalytic activity is not required for induction of autophagy by PLP2-TM. Collectively, these results imply that the ability to promote autophagy is a shared attribute of coronavirus PLP2-TM and its homologs in a protease-independent manner. In subsequent investigations, we focused our efforts on using HCoV-NL63 PLP2-TM to delineate the molecular details underlying interactions of coronaviral PLP with the autophagy pathway. Figure&#160;3Induction of autophagy by various coronaviral PLPs that is independent on the protease activity. (A) The plasmids of HCoV-NL63 PLP2-TM, SARS-CoV PLpro-TM, MERS-CoV PLpro-TM and PEDV PLP2-TM were transfected into HEK293T cells. As the positive control for induction of autophagy, HEK293T cells were treated with complete medium supplemented with 400&#160;nmol/L Rapamycin for 6&#160;h. The cells were then lysed for Western blotting analysis using a rabbit anti-LC3 antibody to detect endogenous LC3 expression (top panel in Fig.&#160;3A). The whole cell lysate (WCL) was blotted using anti-V5 antibodies to evaluate expression of PLP2 (PLpro)-TM (middle panel in Fig.&#160;3A), and &#946;-Actin was detected in whole cell lysate (WCL) as a loading control (bottom panel in Fig.&#160;3A). (B) PLP2-TM-V5 and the catalytic mutants (C1678A, H1836A and D1849A) as showed in Fig.&#160;1A were transfected respectively into HEK2923T, and induction of autophagy was detected as described in Fig.&#160;3A\nTime-dependent induction of autophagy by PLP2-TM\nTo understand the temporal regulation of autophagy by PLP2-TM, we monitored time-course eGFP-LC3B puncta formation in HEK293T cells co-transfected with HCoV-NL63 PLP2-TM and eGFP-LC3B. Significant eGFP-LC3B puncta formation was observed in cytoplasm of PLP2-TM-transfected cells as early as 24 h post transfection (Fig.&#160;4A), when PLP2-TM protein was first detected (Fig.&#160;4C). In addition, eGFP-LC3B puncta steadily increased at 48 h post transfection (Fig.&#160;4A and 4B), concomitant with an increase in PLP2-TM expression levels (Fig.&#160;4C). Approximately 40% of cells displayed cytoplasmic eGFP-LC3B puncta at 24 h postransfection, and that number climbed to >70% at 48 h (Fig.&#160;4B). In contrast, cells co-transfected with control vector exhibited largely diffuse eGFP-LC3B fluorescence, with less than 7% of cells having background levels of eGFP-LC3B puncta, regardless of the time points monitored (at either 24 or 48 h post transfection, Fig.&#160;4A and 4B). In line with these data, the abundance of endogenous LC3-II protein was reproducibly upregulated in a time-dependent manner in cells transfected with PLP2-TM (Fig.&#160;4C and Fig.&#160;4D), a phenomenon which was not observed with cells transfected with the control vector (data not shown). The increase in LC3-II protein levels by PLP2-TM, however, was substantially reduced in cells treated with 3-MA (Fig.&#160;4E and 4F), an autophagy inhibitor (Mizushima et al., 2010), providing additional support for the notion that PLP2-TM increases cellular autophagic activity. In aggregate, the time-dependent increase in eGFP-LC3B puncta formation and upregulation of LC3-II abundance in PLP2-TM-transfected cells but not in control vector-expressing cells confirms that the heightened autophagic activity results specifically from PLP2-TM expression.Figure&#160;4PLP2-TM induces autophagy formation in a time-dependent manner. (A) Immunofluorescence microscopy was used to detect the eGFP-LC3B-autophagosome in PLP2-TM transfected cells at different time point post-transfection. The plasmid of PLP2-TM-V5 was co-transfected with pcDNA3.1-eGFP-LC3B into HEK2923T cells. The cells were fixed at 0&#160;h, 24&#160;h and 48&#160;h post-transfection, respectively and then analyzed for eGFP-LC3B positive autophagosome accumulation using a confocal microscope as described in Fig.&#160;1B. (B) Quantification of cells displaying eGFP-LC3B puncta from one representative experiment that shown in Fig.&#160;4A as described in Fig.&#160;1B. (C and D) HEK293T was transfected with PLP2-TM-V5 or empty vector as a negative control. At 0&#160;h, 24&#160;h and 48&#160;h post-transfection, the cells were then lysed for Western blotting analysis using a rabbit anti-LC3 antibody to detect the endogenous LC3 expression (top panel in Fig.&#160;4C). The whole cell lysate (WCL) was blotted using anti-V5 antibodies to evaluate expression of PLP2-TM (middle panel in Fig.&#160;4C), and &#946;-Actin was detected in whole cell lysate (WCL) as a loading control (bottom panel in Fig.&#160;4C). The band intensity was semi-quantitated by densitometric analysis using ImageJ software. (E and F) HEK293T cells were pretreated by 250 &#181;mol/L 3-MA, an autophagy inhibitor, for 1.5 h and then transfected with PLP2-TM or empty vector for 48 h. The LC3, PLP2-TM and &#946;-Actin were detected and semi-quantitated as described above\nPLP2-TM impairs the fusion of autophagosomes with lysosomes\nThe terminal step in autophagy is the fusion of autophagosomes with lysosomes to form autolysosomes and subsequent degradation of the content. We wondered whether the PLP2-TM-induced accumulation of autophagosomes resulted in enhanced autophagic degradation. To answer this question, we measured the protein levels of a well-characterized autophagic substrate, p62/SQSTM1, which binds to LC3 and is specifically degraded as a result of complete autophagic flux (Bjorkoy et al., 2005; Pankiv et al., 2007), in PLP2-TM-expressing HEK293T cells by immunoblotting. Although the abundance of LC3-II was upregulated by PLP2-TM, which is indicative of increased autophagic activity (Fig.&#160;5A, compare lane 2 vs. lane 1 in LC3-I/II panel), the level of the p62 was not reduced but rather slightly increased (compare lane 2 vs. lane 1 in top panel). In contrast, Rapamycin, which induced complete autophagy, not only increased LC3-II levels but also caused a significant decline in p62 expression (compare lane 3 vs. lane 1). Furthermore, the decreased p62 expression that induced by Rapamycin was recovered by PLP2-TM (compare lane 4 vs. lane 3, Fig.&#160;5A). These data reveal that PLP2-TM induces incomplete autophagy, with a disconnection between autophagosome accumulation and autophagic cargo hydrolysis, implying that PLP2-TM may interfere with the fusion of autophagosomes with lysosomes or impair the autolysosomal degradation.Figure&#160;5PLP2-TM activates autophagosome formation but prevents its fusion with lysosomes. (A) HEK293T cells were transfected with pcDNA3.1 or PLP2-TM. As the positive control for induction of autophagy, HEK293T cells were treated with complete medium supplemented with 400 nmol/L Rapamycin for 6 h. At 48 h post-transfection, the levels of endogenous autophagic substrate p62, LC3 protein and PLP2-TM were determined using Western blot analysis. Beta-actin expression was examined as a protein loading control. (B) Diagram of mRFP-GFP-LC3 structure and the principle for probing autophagy flux with mRFP-GFP-LC3 construct. The GFP signal was easily quenched in autolysosome as the acidic pH lysosomal background because of lysosomal hydrolysis, while mRFP fluorescence existed in the acidic pH background. The merged yellow signal GFP+ mRFP+ was visualized using a confocal microscope in the autophagosomes and GFP- mRFP+ signal was visualized using a confocal microscope in the autolysosomes as described previously (Tang et al., 2013). (C) PLP2-TM activates autophagosome formation but blocks its fusion with lysosomes. HEK293T cells were co-transfected with mRFP-GFP-LC3 and PLP2-TM. As the positive control for induction of autophagy, HEK293T cells were transfected with mRFP-GFP-LC3 and then treated with complete medium supplemented with 400 nmol/L Rapamycin for 6 h. For the inhibition of autolysosome maturation, HEK293T cells were transfected with the plasmids of mRFP-GFP-LC3 and then treated with complete medium supplemented with 50 &#181;mol/L CQ for 6 h. At 48 h post-transfection, the cells were fixed and assessed for GFP and mRFP fluorescence. Based on differential pH sensitivity of mRFP and GFP, the mRFP-GFP-LC3 probe differentiates between early, nonacidified autophagosomes (red+green+, yellow in merged images) from acidified, degradative autolysosomes (red+green&#8722;, red in merged images)To further clarify the mechanism by which PLP2-TM induces incomplete autophagy, we took advantage of a tandem-tagged fluorescent reporter, mRFP-GFP-LC3 (Ke and Chen, 2011; Kimura et al., 2007; Klionsky et al., 2012; Mizushima et al., 2010) (Fig.&#160;5B). This reporter construct encodes a fusion protein composed of three open reading frames (ORFs), i.e., mRFP, GFP and LC3 (Amino- to Carboxyl-terminal order). In transfected cells, when autophagosomes fuse with lysosomes, the GFP-green fluorescence of the mRFP-GFP-LC3 fusion protein is quenched in the acidic autolysosomal environment, while red fluorescence emitted from mRFP (which is relatively acid-resistant) remains. By contrast, when fusion of mRFP-GFP-LC3-autophagosomes with lysosomes is interrupted, the mRFP-GFP-LC3-autophagosomes emit both green GFP- and red mRFP-fluorescences which colocalize as yellow puncta in merged fluorescent images (Tang et al., 2013) (Fig.&#160;5B). Validating this reporter system in our hands, we first showed that Rapamycin treated, mRFP-GFP-LC3-expressing cells which underwent complete autophagic flux indeed exhibited predominant red fluorescence with only dim green fluorescence (Fig.&#160;5C). In contrast, cells co-expressing PLP2-TM and mRFP-GFP-LC3 displayed strong accumulation of mRFP-GFP-LC3-puncta which emitted colocalized red and green fluorescence (shown as yellow puncta in merged images, Fig.&#160;5C). The fluorescence pattern of mRFP-GFP-LC3 in PLP2-TM-expressing cells closely resembled that of cells treated with cholorquine (Fig.&#160;5C), a potent inhibitor of autophagosome-lysosome fusion. These findings indicate that PLP2-TM may impair the autophagosome maturation process by interfering with its fusion with lysosomes.\nPLP2-TM colocalizes and interacts with LC3\nThe lipidation of LC3 plays a crucial role in the formation of autophagosomes (Glick et al., 2010; Kabeya et al., 2000; Weidberg et al., 2011). In our efforts to further delineate the mechanisms underlying the induction of autophagy by PLP2-TM, we explored the possibility that PLP2-TM may physically associate with LC3, thereby regulating LC3-mediated phagophore elongation/closure. Confocal fluorescence microscopy revealed PLP2-TM substantially co-localized with eGFP-LC3 as cytoplasmic puncta in cotransfected HEK293T cells (Fig.&#160;6A, left panels), indicating that PLP2-TM is in close proximity with LC3 on autophagosome membranes. This was not a cell-type specific phenomenon, as it was also observed in MCF-7 cells (Fig.&#160;6A, right panels). Co-immunoprecipation analyses showed that V5-tagged PLP2-TM formed a complex with endogenously expressed LC3 (Fig.&#160;6B). Interestingly, significantly more LC3-II than LC3-I was present in PLP2-TM immunoprecipitates, suggesting a higher affinity of LC3-II for PLP2-TM than that of LC3-I.Figure&#160;6PLP2-TM colocalizes and coimmunoprecipitates with autophagy marker protein LC3. (A) HEK293T and MCF-7 cells were transfectd with pcDNA3.1 empty vector, PLP2-TM-V5. After 48 h post-transfection, the cells were fixed and incubated with anti-V5-tagged primary antibody, followed by being stained with an Alexa Fluor 594-conjugated goat anti-rabbit secondary antibody. Colocalization of PLP2-TM-V5 (red) and eGFP-LC3B (green) were observed using a confocal microscope as described in Fig.&#160;1B. (B) HEK293T cells were transfected with pcDNA3.1 empty vector or PLP2-TM-V5 for 48 h, and the cell lysate was immunoprecipitated (IP) with an anti-V5 antibody and immunoblotted (IB) with an anti-LC3 and anti-V5-tagged antibody to detect expression of LC3 (top panel) and V5-tagged PLP2-TM (second panel), respectively. The whole cell lysate (WCL) was blotted with indicated antibodies to evaluate expression of endogenous LC3 and V5-tagged PLP2-TM. Beta-actin was analyzed to serve as a protein loading control\nPLP2-TM interacts with Beclin1 and promotes Beclin1-STING association\nAs a critical regulator of the autophagic process, Beclin1 functions as a scaffold for assembly of the PI3K complexes catalyzing early autophagy. In addition, it orchestrates the late stages of autophagy through interacting with UVRAG, Rubicon to modulate the maturation of autophagosomes to autolysosomes (Jung et al., 2009; Matsunaga et al., 2009; Pattingre et al., 2008; Zhong et al., 2009a; Zhong et al., 2009b). Our data that PLP2-TM not only promoted autophagosome formation but also impacted on fusion of autophagosomes with lysosomes prompted us to determine whether Beclin1 is a target for PLP2-TM. In co-IP experiments, V5-tagged PLP2-TM was found to associate with Myc-tagged Beclin1 reciprocally in co-transfected HEK293T cells (Fig.&#160;7A). Furthermore, PLP2-TM physically interacted with endogenous Beclin1 (Fig.&#160;7B, lane 2 in top panel), suggesting that Beclin1 expressed at physiologic levels also forms a complex with PLP2-TM.Figure&#160;7PLP2-TM interacts with Beclin1 to promote Beclin1-STING interaction. (A) PLP2-TM interacts with exogenous Beclin1. HEK293T cells were transfected with Myc-Beclin1 and pcDNA3.1 or PLP2-TM-V5 for 48 h. The cell lysate was subjected to immunoprecipitate (IP) with an anti-Myc antibody and immunoblot (IB) with an anti-V5 and anti-Myc antibodies (upper and second panels). The cell lysate was also subjected to immunoprecipitate (IP) with an anti-V5 antibody and immunoblot (IB) with an anti-Myc and anti-V5 antibodies (the third and fourth panels). The whole cell lysate (WCL) was blotted with indicated antibodies to evaluate expression of Myc-Beclin1 and V5-tagged PLP2-TM (the fifth and bottom panels). (B) PLP2-TM is associated with endogenous Beclin1. HEK293T cells were transfected with pcDNA3.1 or PLP2-TM-V5 for 48 h. The cell lysate was prepared to immunoprecipitate (IP) with an anti-V5 antibody and immunoblot (IB) with an anti-Beclin1 and an anti-V5 antibody (upper and second panels). The whole cell lysate (WCL) was blotted with indicated antibody to evaluate expression of V5-tagged product (bottom panel). HC indicates heavy chain of immunoglobulin. (C) PLP2-TM promotes the interaction between Beclin1 and STING. HEK293T cells were co-transfected with Myc-Beclin1, HA-STING and PLP2-TM-V5 for 48 h. The cell lysate was immunoprecipitated (IP) with an anti-HA antibody and immunoblotted (IB) with an anti-Myc (upper panel), anti-V5 (the second panel) and an anti-HA antibody (the third panel). The whole cell lysate (WCL) was analyzed to evaluate expression of each epitope-tagged product with the indicated antibodies (bottom panel)Previously, we have shown that PLP2-TM associated with STING, a key regulator of antiviral IFN signaling (Sun et al., 2012a). In light of the emerging roles for autophagy and its components in regulating innate immunity (Levine et al., 2011), we determined whether Beclin1 is involved in the regulation of STING by PLP2-TM. While HA-tagged STING did not co-immunoprecipiate with Myc-tagged Beclin1, these two proteins formed a complex when PLP2-TM was co-expressed (Fig.&#160;7C). These data support the notion that the ability to interact with Beclin1 and regulate autophagy may be linked to the PLP2-TM control of innate immune signaling.\nKnockdown of Beclin1 partially reverses PLP2-TM\u2019s inhibitory effect on innate immunity and is detrimental to PEDV replication\nTo understand the biological significance of the PLP2-TM regulation of autophagy, we studied the effects of Beclin1 knockdown on innate immune signaling and on coronavirus replication. In control siRNA transfected cells, activation of the IFN-&#946; promoter by the constitutively active RIG-I CARD (RIG-IN) was ablated by PLP2-TM. This was in line with our previous publications (Clementz et al., 2010; Devaraj et al., 2007; Sun et al., 2012a). We found that knock-down of Beclin1 slightly and not significantly increased the IFNbeta-Luc reporter activity when cotransfected with vector. Remarkably, knockdown of Beclin1 partially, but significantly reversed PLP2-TM blocking RIG-IN-activated IFN expression (Fig.&#160;8A). Immunoblotting demonstrated that Beclin1 siRNA efficiently depleted the target without affecting PLP2-TM expression (Fig.&#160;8B). Similarly, activation of the IFN-&#946; promoter by STING was inhibited by PLP2-TM in control siRNA transfected cells. However, knockdown of Beclin1 partially, but significantly reversed the inhibition that induced by PLP2-TM (Fig.&#160;8C). The data was also showed that Beclin1 siRNA efficiently knocked down the target of Beclin1 without affecting PLP2-TM expression (Fig.&#160;8D). Next, we tested how silencing of Beclin1 affected coronavirus replication. To achieve this goal, we took advantage of a Vero cell culture model for PEDV, the PLP2-TM of which also promotes autophagy (Fig.&#160;3A). While transfection of a control siRNA had no impact on M protein expression in PEDV infected cells, siRNA-mediated knockdown of Beclin1 significantly diminished M protein production (Fig.&#160;8E and 8F). Taken together, these data suggest that coronaviral PLP2-TM exploits Beclin1 and the autophagy pathway to attenuate innate antiviral responses and concomitantly facilitate viral replication.Figure&#160;8Beclin1 is required for the PLP2-TM negative regulation of IFN expression and contributes to coronavirus replication. (A, C) HEK 293T cells were mock transfected or transfected with control-siRNA, and Beclin1-siRNA at a concentration of 100 nmol/L for 24 h, and then the cells were transfected with IFN&#946;-luc reporter, pRL-TK, with expressing construct of RIG-IN (A) or STING (C) or empty vector and PLP2-TM for another 24 h. Cells were incubated for 48 h and firefly luciferase and Renilla luciferase activities were assayed. The results were expressed as mean relative luciferase (firefly luciferase activity divided by Renilla luciferase activity) with a standard deviation from repeated experiments carried out in triplicate. For statistical analysis, the data of Vector or PLP2-TM between Beclin1 siRNA and control siRNA were subjected to unpaired, two-tailed Student&#8217;s t test using the Microsoft SPSS 12.0 software, and a P value < 0.05 or less was considered statistically significant difference. (B and D) Proteins were extracted from the cells in Fig.&#160;8A (B) or Fig.&#160;8C (D) and analyzed using Western blotting with an anti-Beclin1 antibody to visualize Beclin1 proteins (top panel) and anti-V5 antibody (second panel) to visualize the PLP2-TM construct expression. Anti-Flag (B) and anti-HA (D) antibodies were used to visualize RIG-IN and STING proteins (third panel). Beta-actin was detected using Western blotting as protein loading control (bottom panel). (E) Beclin1-siRNA reduces PEDV replication in Vero cells. Vero cells were transfected with either Beclin1 siRNA or control siRNA at a concentration of 100 nmol/L for 24 h, and then the cells were treated by PEDV at a multiplicity of infection (MOI) of 0.1 for another 24 h. Cells were incubated for 48 h and the M protein expressions were assayed using Western blotting assay. (F) The optical density of M protein band in Fig.&#160;8E was measured by densitometric analysis using ImageJ software and then the ratio of M protein/&#946;-actin was calculated. For statistical analysis, the data between Beclin1 siRNA and control siRNA or mock control were subjected to unpaired, two-tailed Student&#8217;s t test using the Microsoft SPSS 12.0 software, and a P value < 0.05 or less was considered statistically significant difference\nDiscussion\nMany viruses have evolved to exploit the autophagic machinery to their own benefit (Kudchodkar and Levine, 2009; Orvedahl and Levine, 2008; Shoji-Kawata and Levine, 2009), and coronaviruses are no exception. A number of studies have shown that autophagy is induced during infections by various coronaviruses, although controversial results have been reported concerning whether autophagy is required for coronavirus replication (de Haan and Reggiori, 2008; Prentice et al., 2004; Reggiori et al., 2010; Zhao et al., 2007). At present, the underlying mechanisms by which coronaviruses promote autophagy are poorly understood. The nsp6 encoded by infectious bronchitis virus, an avian coronavirus, was recently reported to induce autophagosome formation, as were the nsp6 homologues encoded by MHV, SARS-CoV and the closely related arterivirus PRRSV (Cottam et al., 2011). We present evidence in this study that expression of the membrane-anchored coronavirus papain-like protease PLP2 domain (and its homologues) alone is capable of activating autophagy in nutrient-rich conditions, assigning a novel function to this multifunctional viral protein which is known to act as a viral protease, a DUB enzyme, and an IFN antagonist (Barretto et al., 2005; Chen et al., 2007; Clementz et al., 2010; Devaraj et al., 2007; Sun et al., 2012a). Importantly, we have demonstrated this in multiple cell types (HEK293T, HeLa and MCF-7), and shown it to be an attribute shared by PLP2-TM/PLpro-TM of different coronaviruses, including HCoV-NL63, SARS-CoV, MERS-CoV and PEDV. This finding uncovers a previously unappreciated role for PLP2-TM/PLpro-TM in regulation of autophagy by coronaviruses and may provide novel insights into the mechanisms of coronavirus pathogenesis.Our data show that the PLP2 domain and the downstream hydrophobic TM motif are both needed to promote autophagy. Neither PLP2 nor TM alone is sufficient, as evidenced by the inabilities of soluble PLP2 and PLP1-TM to induce autophagosome (LC3 puncta) formation (Data not shown). Mechanistically, we found that PLP2-TM physically interacted with Beclin1 and LC3, both of which are involved in the early steps of autophagosome formation (Kraft and Martens, 2012; Mehrpour et al., 2010). Interestingly, our data also reveal that PLP2-TM induces incomplete autophagy that does not culminate in autophagosome maturation to autolysosomes. Evidence supporting this notion came from the experiments showing that degradation of the autophagic substrate p62/SQSTM1 was retarded and that the autolysosome-liable GFP fluorescence of the mRFP-GFP-LC3 reporter protein was not lost in spite of enhanced LC3 lipidation. Beclin1, again, is likely the target responsible for the deficient autophagosome maturation in PLP2-TM expressing cells, given its involvement in the UVRAG-containing PI3K complex that controls fusion between autophagosmes and lysosomes (Kang et al., 2011; Liang et al., 2008). Of note, accumulating evidence suggests that Beclin1 is a prime target for viruses that manipulate the autophagy pathway (Munz, 2011). For example, Influenza A virus M2 and HIV Nef bind to Beclin1 to hamper the fusion of autophagosomes with lysosomes (Gannage et al., 2009; Kyei et al., 2009). We propose that the coronavirus PLP2-TM adopts a similar strategy to impede the maturation of autophagic vacuoles. However, the precise mechanism will need to be further studied. Regardless, the induction of incomplete as opposed to complete autophagy by PLP2-TM may represent an evolutionary advantage of the virus, in that it prevents autophagic degradation of viral products generated in infected cells, promoting maximal viral survival.The findings that nsp6 (Cottam et al., 2011) and PLP2-TM/PLpro-TM of coronaviruses promote autophagy argue that the autophagy pathway and/or autophagy-related protein(s) actively participate in coronavirus-host interactions. A tempting hypothesis is that autophagy may facilitate viral propagation, as demonstrated for other positive-strand RNA viruses such as dengue virus, and Poliovirus (Richards and Jackson, 2013). However, experimental evidence concerning whether autophagy is required for replication of coronaviruses has been contradictory. While earlier studies suggested autophagy to play a role (Cottam et al., 2011; Prentice et al., 2004), several latest reports argued against it (Maier and Britton, 2012; Maier et al., 2013; Reggiori et al., 2010). These recent studies support a model in which MHV and arterivirus EAV hijack the LC3-I-positive EDEMsomes, rather than autophagosome membranes, to form the DMVs in which viral replication takes place (Monastyrska et al., 2013; Reggiori et al., 2010). However, the possibility that different coronaviruses may have a disparate requirement for autophagy for optimal viral replication cannot be ruled out. Supporting this notion, several studies have shown that autophagy is induced by infection with a closely related arterivirus, PRRSV, whose replication is crippled by inhibition of autophagy (Chen et al., 2012; Liu et al., 2012; Sun et al., 2012b). We show here that knockdown of the master regulator of autophagy, Beclin1, impairs PEDV M protein production. Clearly, more studies with a greater number of different coronaviruses are warranted (Maier and Britton, 2012).Another intriguing finding of this study is that autophagy is subverted by coronaviral PLP for immune evasion. We have previously shown that coronavirus PLpro-TM/PLP2-TM negatively regulates antiviral defenses by inhibiting the activation of IRF3 (Clementz et al., 2010; Devaraj et al., 2007), and that the PLPs disrupt STING-mediated IFN induction (Sun et al., 2012a), but the mechanism(s) by which coronavirus PLPs target STING (and possibly other signaling molecules) to inhibit IRF3-dependent antiviral response is unknown. In this work, we have demonstrated that PLP2-TM promotes the interaction of STING with Beclin1, but not with LC3 (data not shown), suggesting that PLP2-TM may sequester STING to autophagosomes through Beclin1, thereby inhibiting downstream innate immune signaling. Although not tested in this study, PLP2-TM-induced autophagy may also segregate other innate immune signaling components and perhaps even viral dsRNAs from ligation to cytoplasmic viral sensors. Supporting the exploit of Beclin1 and autophagy for viral immune escape, we found that knockdown of Beclin1 (and thus autophagosome formation) partially but significantly reversed the blockade of PLP2-TM on activation of the IFN response. The reason the relief was only partial was likely due to incomplete depletion of Beclin1 and existence of autophagy-independent immune evasion mechanisms, as exemplified by the deubiquitination of RIG-I, TRAF3, etc. by PLPs (Sun et al., 2012a).In summary, results of this study demonstrate that the coronavirus papain-like proteases along with their transmembrane anchors activate incomplete autophagy by interacting with Beclin1. Autophagosomes induced by CoV PLP2-TM/PLpro-TMs may provide a platform for viral targeting Beclin1 to sequester STING and possibly other critical innate signaling components to impede downstream antiviral responses, thereby promoting viral replication. Further studies will be needed to elucidate the precise mechanisms of autophagosome induction by CoV PLPs and the exact roles that autophagy plays in coronavirus replication, antiviral innate immune responses and disease pathogenesis.\nMaterials and methods\n\nCells, plasmids, siRNAs, antibodies and other reagents\nHEK293T, HeLa and MCF-7 cells were cultured in Dulbecco&#8217;s modified Eagle&#8217;s medium (DMEM) (Gibco, Cat. No.12800-017) supplemented with 10% (v/v) fetal calf serum (Gibco, Cat. No. 1036489), 100 U/mL of penicillin and 100 &#181;g/mL of streptomycin. DNA constructs encoding HCoV-NL63 PLP2-TM, SARS-CoV PLpro-TM, PEDV PLP2-TM, MERS-CoV PLpro-TM and plasmid of HA-STING have been described (Clementz et al., 2010; Devaraj et al., 2007; Sun et al., 2012a; Yang et al., 2014). To construct a Myc-tagged Beclin1 expression vector, cDNA encoding the full-length (1353-bp long) human Beclin1 (GenBank ID: NM_003766.3) was amplified by reverse transcription-PCR (RT-PCR) from human mesenchymal stem cells (MSCs) and cloned into the pCMV-Myc vector (Clontech). The following PCR primers were used: 5&#8242;-CCCGAATTCGGATGGAAGGGTCTAAGACGTCC-3&#8242; (forward primer) and, 5&#8242;-CGCGGTACCTCATTTGTTATAAAATTGTGAGGA-3&#8242; (reverse primer). The plasmid encoding eGFP-LC3B was kindly provided by Dr. Songshu Meng (Yangzhou University, Jiangsu, China) (Kimura et al., 2007; Mizushima et al., 2010). A vector encoding tandem mRFP-GFP-LC3 was kindly provided by Dr. Shaobo Xiao (Huazhong Agricultural University, Wuhan, China) (Kimura et al., 2007; Mizushima et al., 2010). Beclin1 siRNA sequence and control siRNA sequence were described as previously (Hoyer-Hansen et al., 2005). The following primary antibodies were used for various protein analyses: anti-LC3 (Sigma-Aldrich, Cat. No. L7543); anti-p62 (MBL, Cat. No. PM045), anti-Beclin1(MBL, Cat. No. PD017), anti-V5 (MBL, Cat. No. PM003), anti-HA (MBL, Cat. No. 561) and anti-Myc (MBL, Cat. No. M047-3); anti-actin (Beyotime, Cat. No. AA128); chicken anti-V5 (Abcam, Cat. No. ab9113). Donkey anti-chicken Cy3 (Millipore, Cat. No. AP194C) and Alexa Fluor 594-conjugated goat anti-rabbit secondary antibodies (ZSGB-BIO, Cat. No. ZF-0516) were obtained from Millipore and ZSGB-BIO, respectively. Rapamycin (Sigma-Aldrich, Cat. No. R8781), Chloroquine (CQ) (Sigma-Aldrich, Cat. No. C6628) and 3-Methyladenine (3-MA) (Sigma-Aldrich, Cat. No. M9281) were obtained from Sigma-Aldrich.\nConfocal microscopy\nHEK293T, HeLa or MCF-7 cells were grown on glass coverslips in 6-well plates. For the detection of autophagosomes, plasmid DNA expressing eGFP-LC3B or mRFP-GFP-LC3 (1 &#956;g per well) was transfected in the presence or absence of 1 &#956;g of various PLP-encoding vectors using Lipofectamine 2000 (Invitrogen, Cat. No. 11668-027) according to the manufacturer&#8217;s protocol. As a positive control to visualize the induction of autophagy, HEK293T cells were transfected with eGFP-LC3B for 48 h and then treated with 400 nmol/L of Rapamycin in complete culture medium for 6 h. The fluorescence of GFP-LC3 was observed under a Zeiss LSM-510 confocal fluorescence microscope. Cells containing three or more GFP-LC3 dots were defined as autophagy-positive cells. The percentage of cells with characteristic punctate GFP-LC3 fluorescence relative to all GFP-positive cells was calculated as described previously (Li et al., 2011; Wong et al., 2008). Three random fields, each containing at least 80 GFP-positive cells, were counted, and three independent experiments were performed.\nImmunofluorescence\nTo evaluate the subcellular localization of eGFP-LC3B, mRFP-GFP-LC3 and NL63 PLPs, plasmid DNA expressing eGFP-LC3B or mRFP-GFP-LC3 (1 &#956;g per well) was transfected in the presence or absence of 1 &#956;g of various PLP-encoding vectors using Lipofectamine 2000 (Invitrogen, Cat. No. 11668-027) according to the manufacturer&#8217;s protocol. As a positive control to visualize the induction of autophagy, HEK293T cells were transfected with eGFP-LC3B for 48 h and then treated with 400 nmol/L of Rapamycin in complete culture medium for 6 h. In some experiments, HEK293T cells co-transfected with the plasmids of mRFP-GFP-LC3 and PLP2-TM for 48 h were treated with 50 &#956;mol/L of CQ in complete medium for 6 h to inhibit autolysosome maturation. At indicated time points post transfection, cells were fixed with 4% formaldehyde in PBS for 10 min at room temperature. Cells were then incubated with 1:200 dilution of rabbit anti-V5 (MBL, Cat. No. PM003) or chicken anti-V5 (abcam, Cat. No. ab9113) in ADPS (PBS + 0.1% Triton-X100 + 5% fetal calf serum) for 1 h at room temperature. Following three PBS washes, cells were incubated with 1:200 dilution of Alexa Fluor 594-conjugated goat anti-rabbit (ZSGB-BIO, Cat. No. ZF0136) or donkey anti-chicken Cy3 (Millipore, Cat. No. AP194C) secondary antibody in ADPS for 1 h in dark. Following the incubation, cells were washed three times with PBS, mounted, and examined under a Zeiss LSM-510 confocal microscope.\nWestern blotting analysis\nHEK293T, HeLa and MCF-7 cells were seeded into 24-well plates and incubated at 37&#176;C for 18 h. Cells were subsequently transfected with PLP2-TM construct or empty vector using Lipofectamine 2000 reagent according to manufacturer&#8217;s instructions. At 48 h post-transfection, cells were lysed in a buffer containing 0.5% Triton X-100, 150 mmol/L NaCl, 12.5 mmol/L &#946;-glycerolphosphate, 1.5 mmol/L MgCl2, 2 mmol/L EDTA, 10 mmol/L NaF, 1 mmol/L Na3VO4, 2 mmol/L DTT and protease inhibitor cocktail (Sigma,Cat. No. P8340). Cell extracts were clarified by centrifugation at 5000&#160;&#215;g at 4&#176;C for 10 min, and protein concentration of lysate determined using BCA Protein Assay kit (Bio-med, Cat. No. pp0101). Protein samples were mixed with 30 &#181;L of 2&#215; SDS-PAGE sample buffer, boiled for 10 min, separated on SDS-PAGE gel, and transferred onto a PVDF membrane. Blots were incubated with indicated primary antibodies, washed three times in 1&#215; TBS-T buffer, and subsequently incubated with HRP-conjugated secondary antibodies (ZSGB-BIO, Cat. No. ZF0136, Cat. No. ZF0312). Antibody-antigen reactions were detected using Western Lighting Plus-ECL chemiluminescence reagents (Applygen, Cat. No. P1010).\nCo-immunoprecipitation (Co-IP) analysis\nHEK293T cells were seeded in 100-mm dishes at a density of 1 &#215; 106 cells/dish. Twelve hours later, cells were transiently transfected with a total of 10 &#181;g of empty vector or indicated expression plasmids using Lipofectamine 2000 (Invitrogen, Cat. No. 11668-027). At 48 h post transfection, whole cell lysates were prepared and their protein concentrations determined using the procedures described above (for Western blotting analysis). The protein concentrations in cell lysates were adjusted to 1 &#181;g/&#181;L, and 500 &#181;L of each lysate was used for co-IP. Lysates were pre-cleared by adding 20 &#181;L of protein A + G Agarose (Beyotime, Cat. No. P2021) and 1 &#181;g of normal IgG and incubating for 2 h at 4&#176;C, followed by spinning down the agarose beads. The pre-cleared supernatant was then incubated with the indicated primary antibody [anti-V5 (MBL, Cat. No. PM003) or anti-HA (MBL, Cat. No. 561)/anti-Myc (MBL, Cat. No. M047-3)] with rocking overnight at 4&#176;C. Thereafter, the beads-antibody-antigen complex was pelleted and washed 3 times with 1 mL of lysis buffer. The protein complexes were then eluted from the beads in 30 &#181;L of 2&#215; SDS-PAGE sample buffer by boiling for 10 min. Samples were separated on SDS-PAGE and transferred to PVDF membranes for Western blotting.\nIFN-\u03b2 reporter assay\n12&#8211;18 h prior to transfection, HEK 293T were seeded in 24 well plates. At a confluence of 80%, the cells were transfected with the Beclin1 siRNA or control siRNA at the concentration of 100 nmol/L using JetPRIME (PolyPlus, Cat. No. 114-15). After 24 h, the cells were transfected using JetPRIME with 200 ng of IFN&#946;-Luc reporter plasmid encoding firefly luciferase and 20 ng of pRL-TK plasmid encoding Renilla luciferase for normalization along with 300 ng of empty DNA vector or RIG-I/STING-expressing construct and 300 ng of vector or PLP2-TM constructs. 24 h after DNA transfection, the cell extracts were prepared and Luciferase activity and Renilla luciferase activity were assayed using the Dual Luciferase Reporter System (Promega, Cat. No. E1910) in a Luminometer according to the supplier&#8217;s instructions. Data were shown as mean relative luciferase (firefly luciferase activity divided by Renilla luciferase activity) with standard deviation from repeated experiments that were carried out in triplicate. For statistical analysis, the data between Vector and PLP2-TM were subjected to unpaired, two-tailed Student&#8217;s t test using Microsoft SPSS 12.0 software, and P-values of <0.05 were considered to indicate statistical significance.\nTransmission electron microscopy (TEM)\nHEK293T cells were seeded in 100-mm dishes at a density of 1 &#215; 106 cells/dish. Twelve hours later, cells were transiently transfected with a total of 10 &#181;g of empty vector or the indicated expression plasmid using Lipofectamine 2000 (Invitrogen, Cat. No. 11668-027). As a positive control for induction of autophagy, HEK293T cells were treated with 400 nmol/L of Rapamycin in complete medium for 6 h. Cells were washed three times with PBS, trypsinized, and collected by centrifugation at 1000&#160;&#215;g for 10 min. The cell pellets were fixed with 3% glutaraldehyde in 0.075 mol/L phosphate buffer (pH 7.4) for 2 h at 4&#176;C. The cells were washed in the solution containing 0.075 mol/L phosphate and 0.19 mol/L sucrose three times for 10 min each and post-fixed in 1% OsO4 in 0.24 mol/L phosphate buffer (pH 7.4) for 2 h. After being washed for 15 min in 0.075 mol/L phosphate buffer and 0.19 mol/L sucrose buffer at 4&#176;C, the cells were dehydrated with a graded series of ethanol and gradually infiltrated with epoxy resin. Samples were sequentially polymerized at 35&#176;C for 12 h, 45&#176;C for 12 h, and 60&#176;C for 24 h. Ultrathin sections (about 70 nm) were cut using an LEICA microtome and mounted on copper slot grids. Sections were doubly stained with uranyl acetate for 10 min and lead citrate for another 10 min and observed under a Hitachi H-7650 transmission electron microscope.",
        "pmid": "25311841",
        "title": "Coronavirus membrane-associated papain-like proteases induce autophagy through interacting with Beclin1 to negatively regulate antiviral innate immunity",
        "journal_title": "Protein & Cell",
        "authors": "Xiaojuan Chen;Kai Wang;Yaling Xing;Jian Tu;Xingxing Yang;Qian Zhao;Kui Li;Zhongbin Chen",
        "doi": "10.1007/s13238-014-0104-6"
    },
    {
        "keywords": "lipidomics;UHPLC\u2013MS;HCoV-229E;MERS-CoV",
        "abstract": "Lipids play numerous indispensable cellular functions and are involved in multiple steps in the replication cycle of viruses. Infections by human-pathogenic coronaviruses result in diverse clinical outcomes, ranging from self-limiting flu-like symptoms to severe pneumonia with extrapulmonary manifestations. Understanding how cellular lipids may modulate the pathogenicity of human-pathogenic coronaviruses remains poor. To this end, we utilized the human coronavirus 229E (HCoV-229E) as a model coronavirus to comprehensively characterize the host cell lipid response upon coronavirus infection with an ultra-high performance liquid chromatography-mass spectrometry (UPLC&#8211;MS)-based lipidomics approach. Our results revealed that glycerophospholipids and fatty acids (FAs) were significantly elevated in the HCoV-229E-infected cells and the linoleic acid (LA) to arachidonic acid (AA) metabolism axis was markedly perturbed upon HCoV-229E infection. Interestingly, exogenous supplement of LA or AA in HCoV-229E-infected cells significantly suppressed HCoV-229E virus replication. Importantly, the inhibitory effect of LA and AA on virus replication was also conserved for the highly pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV). Taken together, our study demonstrated that host lipid metabolic remodeling was significantly associated with human-pathogenic coronavirus propagation. Our data further suggested that lipid metabolism regulation would be a common and druggable target for coronavirus infections.",
        "body": "1. Introduction\nCoronaviruses are enveloped viruses with a large single-strand, positive-sense RNA genome [1,2]. As of today, there are a total of six coronaviruses that are known to infect humans, including human coronavirus OC43 (HCoV-OC43), human coronavirus 229E (HCoV-229E), severe acute respiratory syndrome coronavirus (SARS-CoV), human coronavirus HKU1 (HCoV-HKU1), human coronavirus NL63 (HCoV-NL63), and the Middle East respiratory syndrome coronavirus (MERS-CoV) [3]. These human-pathogenic coronaviruses cause a broad range of clinical manifestations. HCoV-OC43, HCoV-229E, HCoV-HKU1, and HCoV-NL63 cause mild, self-limiting upper respiratory tract infections. In contrast, SARS-CoV and the recently emerged MERS-CoV may cause severe pneumonia with acute respiratory distress syndrome, multi-organ failure, and death in both immunocompetent and immunocompromised hosts [4,5,6,7].Lipids play crucial roles at various stages in the virus life cycle. First, lipids can serve as the direct receptors or entry co-factors for enveloped and non-enveloped viruses at the cell surface or the endosomes [8,9]. Second, lipids and lipid synthesis play important roles in the formation and function of the viral replication complex [10,11]. Third, lipid metabolism can generate the required energy for efficient viral replication [12]. Moreover, lipids can dictate the proper cellular distribution of viral proteins, as well as the trafficking, assembly, and release of virus particles [13,14]. In this regard, the host lipid biogenesis pathways play indispensable roles in modulating virus propagation.As in other viruses, lipids play key roles in the life cycle of coronaviruses. Coronaviruses confiscate intracellular membranes of the host cells to generate new compartments known as double membrane vesicles (DMVs) for the amplification of the viral genome. DMVs are membranous structures that not only harbor viral proteins but also contain a specific array of hijacked host factors, which collectively orchestrate a unique lipid micro-environment optimal for coronavirus replication [15]. A recent study indicated that a key lipid processing enzyme, cytosolic phospholipase A2&#945; enzyme (cPLA2&#945;) that belongs to the phospholipase A2 (PLA2) superfamily, was closely associated with DMVs&#8217; formation and coronaviruses&#8217; replication [16]. The viral protein and RNA accumulation, as well as the production of infectious virus progeny, were significantly diminished in the presence of cPLA2&#945; inhibitor [16]. At the same time, phospholipase A2 group IID (PLA2G2D), an enzyme that predominantly contributes to anti-inflammatory/pro-resolving lipid mediator expression, contributed to worsened outcomes in mice infected with SARS-CoV by modulating the immune response [17]. However, to date, the change and modulating effects of the specific lipids involved in lipid rearrangement upon coronavirus infection remains largely unexplored.To obtain a comprehensive and unbiased profile of perturbed lipids upon coronavirus infection, we performed mass spectrometry (MS)-based lipidomics profiling on coronavirus-infected cells using HCoV-229E as a model virus. Specific lipids including glycerophospholipids and fatty acids (FAs) upon virus infection were identified, which represented the lipid species that were rearranged by HCoV-229E infection. Further pathway analysis revealed that the linoleic acid (LA) and arachidonic acid (AA) metabolism axis was the most perturbed pathway upon HCoV-229E infection. Importantly, supplement of additional LA and AA to coronavirus-infected cells significantly inhibited virus replication of both HCoV-229E and the highly virulent MERS-CoV, suggesting that the LA&#8211;AA metabolism axis is a common and essential pathway that could modulate coronavirus replication. In this regard, temporal modulation of the host lipid profile is a potential novel strategy to combat emerging human coronaviruses.\n2. Materials and Methods\n\n2.1. Materials\nHigh performance liquid chromatography (HPLC)-grade methanol, acetonitrile, chloroform and 2-propanol were purchased from Merck (Darmstadt, Germany). HPLC-grade water was prepared using a Milli-Q water purification system (Millipore, Burlington, MA, USA). Analytical grade acetic acid and commercial standards used for biomarker identification were purchased from Sigma-Aldrich (St. Louis, MO, USA). Internal standards (IS) including Arachidonic acid-d8, 15(S)-HETE-d8, Leukotriene-B4-d4 and Platelet-activating factor C-16-d4 (PAF C-16-d4) were purchased from Cayman Chemical (Ann Arbor, MI, USA) [18].\n2.2. Viruses and Cells\nHuh-7 and VeroE6 cells were maintained in Dulbecco&#8217;s modified Eagle medium (DMEM) supplemented with 10% heat-inactivated fetal bovine serum (FBS), 100 U/mL penicillin, and 100 g/mL streptomycin (5% CO2 at 37 &#176;C). MERS-CoV (EMC/2012 strain) was kindly provided by Professor Ron Fouchier (Erasmus Medical Center, Rotterdam, The Netherlands). MERS-CoV and HCoV-229E were cultured in VeroE6 cells in serum-free DMEM supplemented with 100 U/mL penicillin and 100 g/mL streptomycin as we described previously [19,20,21]. The supernatants were harvested when cytopathic effects (CPE) were observed and centrifuged to generate the viral stocks. The viral stocks were titrated by plaque assay on VeroE6 cells and stored at &#8722;80 &#176;C as previously described [22,23]. Briefly, confluent VeroE6 cells were infected with 10-fold serial viral dilutions. The cells were incubated with diluted viruses at 37 &#176;C for 1 h and subsequently overlaid with 1% low-melting-point agarose (Promega, Madison, WI, USA). The cells were fixed with 4% formaldehyde as the plaques were observed and then stained with 0.2% crystal violet. All experiments involving live MERS-CoV followed the approved standard operating procedures of the biosafety level 3 facility as previously described [24,25,26,27].\n2.3. Lipid Treatment of Middle East Respiratory Syndrome Coronavirus (MERS-CoV)-Infected and Human Coronavirus (HCoV-229E)-Infected Huh-7 Cells\nHuh-7 cells were seeded into 24-well plate to reach 90% confluency and infected with MERS-CoV or HCoV-229E at multiplicity of infection (MOI) of 0.005 or 1, respectively. After 1 h of inoculation, the cells were washed with phosphate-buffered saline (PBS) and maintained in lipids-supplemented medium at the indicated concentrations for 24 h. AA, LA, oleic acid (OA), and palmitic acid (PA) were dissolved in ethanol and ethanol was used as a negative control. The lipids were purchased from Cayman Chemical (Ann Arbor, MI, USA). The supernatants and cell lysates were collected at 24 h post-infection. The viral genome copy numbers were determined by reverse-transcription quantitative polymerase chain reaction (RT-qPCR) as previously described [28,29,30].\n2.4. Lipid Extraction for Lipidomics Profiling\nConfluent Huh-7 cells were mock infected or infected with HCoV-229E at MOI of 1 and incubated in DMEM medium. At 24 hpi, cells were collected for cellular lipid extraction. The lipid extraction was performed for liquid chromatography-mass spectrometry (LC-MS) analysis according to a previously described protocol with slight modifications [31,32]. Inactivation of virus infectivity was confirmed before further processing as we previously described with some modifications [33]. Briefly, 500 &#181;L of ice-cold 150 mM ammonium bicarbonate solution was added to dissociate cells. Two millilitres of chloroform/methanol (v/v 2:1) containing IS were added, followed by vortexing and centrifugation at 4500 rpm for 10 min at 4 &#176;C. The bottom phase was transferred to glass vials and dried using a vacuum concentrator for storage at &#8722;80 &#176;C. The dried samples were reconstituted in 250 &#181;L solvent mixture containing methanol/2-propanol/water (v/v/v 5:4:1) for LC-MS analysis. After centrifugation at 14,000 rpm for 10 min at 4 &#176;C, supernatants were transferred to LC vials for LC-MS analysis.\n2.5. Ultra-High Performance Liquid Chromatography-Electrospray Ionization-Quadrupole-Time of Flight-Mass Spectrometry (UPLC-ESI-Q-TOF-MS) Analysis\nThe lipid extract was analyzed using an Acquity UPLC system coupled to a Synapt G2-Si High Definition Mass Spectrometry (HDMS) system (Waters Corp., Milford, MA, USA). The chromatography was performed on a Waters ACQUITY BEH C18 column (1.7 &#956;m, 2.1 &#215; 100 mm, I.D., 1.7 mm, Waters Corp., Milford, MA, USA). The mobile phase consisted of (A) 0.1% acetic acid in water and (B) acetonitrile. Gradient elution applied for ultra-high performance liquid chromatography-mass spectrometry (UPLC-MS) analysis was described in Table S1. The column and autosampler temperature were maintained at 45 &#176;C and 4 &#176;C, respectively. The injection volume was 5 &#181;L [34]. The mass spectral data were acquired in both positive and negative modes. The capillary voltage, sampling cone voltage and source offset were maintained at 2.5 kV, 60 V, and 60 V, respectively. Nitrogen was used as desolvation gas at a flow rate of 800 L/h. The source and desolvation temperatures were maintained at 120 &#176;C and 400 &#176;C, respectively. Mass spectra were acquired over the m/z range of 50 to 1200. The SYNAPT G2-Si HDMS system was calibrated using sodium formate clusters and operated in sensitivity mode. Leucine enkephalin was used as a lock mass for all experiments. MS/MS acquisition was operated in the same parameters as MS acquisition. Collision energy was applied at the range from 20 to 40 eV for fragmentation to allow putative identification and structural elucidation of the significant lipids.\n2.6. Data Processing and Statistical Data Analysis\nAcquisition of the raw data was performed using MassLynx software version 4.1 (Waters Corp., Milford, MA, USA) and raw data were converted to the common data format (NetCDF) files using conversion software Databridge (Waters Corp., Milford, MA, USA). The NetCDF data were subsequently deconvolved into a usable data matrix using the XCMS software (http://metlin.scripps.edu/download/) [35] and the grouping of features was performed using the CAMERA R package [36]. Preprocessed data were then exported as a .csv file for further data statistical analysis. MetaboAnalyst 3.0 (http://www.metaboanalyst.ca) and SIMCA-P V12.0 (Umetrics, Ume&#229;, Sweden) were used for univariate and multivariate statistical analysis, respectively [37]. For univariate analysis, statistical significance of features was determined between the mock and HCoV-229E infected group using the Student&#8217;s t-test and fold change. The p-value  2 were used as criteria for significant features selection. For multivariate analysis, the features were subjected to Pareto scaling firstly then orthogonal partial least squares discriminant analysis (OPLS-DA) was performed as a supervised method to find important variables with discriminative power. The OPLS-DA model was evaluated with the relevant R2 and Q2. The variable importance in projection (VIP), which reflects both the loading weights for each component and the variability of the response explained by this component, was used for feature selection [38].\n2.7. Lipids Identification\nMS/MS fragmentation was performed on the significant features with high abundances. The significant features identification were carried out by searching accurate MS and MS/MS fragmentation pattern data in the METLIN database (Metabolomics Database, http://metlin.scripps.edu/), Human Metabolome Database (http://www.hmdb.ca/), and LIPD MAPS (Lipidomics Gateway, http://www.lipidmaps.org/). For confirmation of lipid identity using authentic chemical standard, MS/MS fragmentation pattern of the chemical standard was compared with that of candidate lipid under the same LC-MS condition to reveal any matching [18,39].\n3. Results\n\n3.1. Omics-Based Statistical Analysis for Significant Features\nTo investigate how coronavirus perturbs host lipid metabolism, we performed lipidomics analysis on HCoV-229E-infected Huh7 cells and compared the results with those of the mock-infected cells. The preliminary features list included precursor ions, adducts and isotope ions, which were imported into the MetaboAnalyst and SIMCA-P software for further analysis. The R2X/ R2Y, represented the X/Y variables explanation rate of the OPLS-DA model, were 83.0% and 98.8%, respectively. The predicted component, as estimated by cross-validation, was 0.97 (Q2). These cross-validated parameters were satisfactory for OPLS-DA mode (Supplementary Figure S1a). At the same time, the permutation test (100 times) also indicated that the validated model was satisfied (Supplementary Figure S1b). Overall, our results demonstrated that these significant lipid features could be selected by the validated statistical model for subsequent identification.\n3.2. Identification of Lipids Specific to HCoV-229E\nA total of 206 (positive mode) and 100 (negative mode) ion features were selected according to the omics-based statistical analysis method. These ion features were significantly discriminative between HCoV-229E-infected and mock-infected cells. To observe the discrimination trend in more detail, a hierarchical clustering analysis was performed based on the degree of similarity of lipid abundance profiles to show the overview trend of all significant ion features. As indicated in Figure 1, most of the significant features from both negative mode (Figure 1A) and positive mode (Figure 1B) expressed an up-regulation trend after HCoV-229E infection compared with the mock infection controls. Furthermore, to identify lipids specific to HCoV-229E infection, these significant features were grouped and annotated using the CAMERA software, and the potential precursor ions were used to perform further MS/MS experiments for obtaining their fragmentation patterns. Finally, a total of 24 lipids were identified, which could be classified into three lipid classes, including lysophosphatidylcholine (lysoPC), lysophosphatidylethanolamine (lysoPE) and fatty acid (FA). The chromatogram peak heights of these identified lipids were generated by LC-MS raw data and the ratio between infected and mock-infected cells was determined. As demonstrated in Figure 2, we found a consistent up-regulation trend of the identified lipids in HCoV-229E-infected cells. In particular, lysoPC was the predominant lipid class of all identified, accounting for approximately 60% of all identified lipids with significant elevation (Figure 2A). At the same time, arachidonic acid (AA), which belongs to the FA class, showed the highest increase in fold-change among all identified lipids with a maximum of 7.1-fold increase (Figure 2B). In addition, the level of lysoPEs (Figure 2C) was also up-regulated with a maximum fold change of 2.93, which was comparatively less than that of the lysoPCs and FAs. The identities of lysoPC (16:0/0:0), platelet-activating factor C-16 (PAF C-16), lysoPE (16:0/0:0), AA, LA, PA and OA were confirmed by matching the retention time (RT) and MS/MS fragmentation patterns of the authentic chemical standards that distinguish between HCoV-229E-infected cases and non-infected controls (Figure 3). The detailed information of the 24 identified lipids was listed in Table 1. MS/MS fragmentation patterns of five representative lipids and corresponding standards are also demonstrated in Supplementary Figure S2.\n3.3. Pathway Analysis of HCoV-229E-Infected Huh7 Cells\nBased on the list of significantly up-regulated lipids after HCoV-229E-infection, MetaboAnalyst (http://www.metaboanalyst.ca) was applied to investigate which pathway might be markedly perturbed. The result of the pathway analysis was graphically presented in Figure 4. From the enrichment analysis results, the LA metabolism pathway and FA biosynthesis pathway had a statistically significant raw p-value (raw p < 0.05, as shown in the Y-axis). Pathway impact results indicated that the LA metabolism and AA metabolism pathways presented higher impact than the other pathways, as indicated in the X-axis value. Combining the above two analysis results, we postulated that the LA metabolism pathway to be a markedly perturbed pathway that correlated with the lipid rearrangement process induced by HCoV-229E infection.To better understand the current pathway analysis results and the cellular lipid signaling response upon HCoV-229E infection, we constructed a global LA pathway map based on the pathway information in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (https://www.genome.jp/kegg/) and literature mining (Figure 5). Upon HCoV-229E infection, the glycerophospholipids, as main components of the cell membrane, were metabolized to lysophospholipids and FAs after cPLA2 enzyme activation. Lysophospholipids such as lysoPCs and lysoPEs were correspondingly increased after HCoV-229E infection. Moreover, lysoPCs could be further metabolized to platelet-activating factor. FAs were also released from glycerophospholipids but only LA and AA could initiate downstream pathways to generate corresponding metabolites. The up-regulation of both lysophospholipids and FAs were partially confirmed by authentic standards. Furthermore, to investigate the downstream pathways trend of FAs, the authentic standards were also applied in LC-MS method to confirm whether these downstream lipids were changed correspondingly. As illustrated in Figure 5, AA is a downstream lipid of LA and the origin lipid of AA metabolism pathway. The identity of AA was confirmed by authentic standard (Supplementary Figure S2D), which was found to be significantly up-regulated. Therefore, combining pathway analysis and the authentic standards verification results, our data suggested that the LA&#8211;AA metabolism axis was the most significantly perturbed pathway and might be associated with lipids rearrangement or other processes in HCoV-229E infection.\n3.4. Lipids Treatment of Virus-Infected Cells\nTo investigate the potential implication of the perturbed LA-AA metabolism axis in HCoV-229E infection, we treated HCoV-229E-infected Huh7 cells with LA and AA and included PA and OA for comparison. The LA and AA were mapped and played a vital role in the perturbed LA-AA metabolism axis (Figure 5). In contrast, PA and OA were not mapped in the perturbed pathway and may only be produced from glycerophospholipids due to cPLA2 enzyme activation. Huh-7 cells were infected with HCoV-229E and treated with AA, LA, PA, or OA. The cell lysates and culture supernatants were harvested at 24 h post-infection to determine the viral genome copy number by RT-qPCR. As shown in Figure 6, LA and AA consistently inhibited the replication of HCoV-229E as evidenced by the decrease in virus genome copies in both cell lysate (Figure 6A,C) and supernatant samples (Figure 6B&#8211;D). In contrast, PA inhibited HCoV-229E replication only when supplied at high concentration while HCoV-229E replication was largely independent of OA (Figure 6A&#8211;D).To further investigate if the modulatory effects of LA and AA were conserved among other human-pathogenic coronaviruses, we evaluated the effects of these lipids on the replication of the recently emerged and highly virulent MERS-CoV. Our data demonstrated that LA and AA potently suppressed MERS-CoV replication in a similar manner as HCoV-229E (Figure 6E,F). Overall, our results demonstrated that exogenously supplied LA and AA could interfere with the optimal replication of human-pathogenic coronaviruses, which suggested that the LA&#8211;AA metabolism axis was significantly involved in the propagation of these viruses.\n4. Discussion\nIn this study, a MS-based lipidomics approach was established to characterize the host cell lipid changes upon coronavirus infection. Univariate and multivariate statistical analyses were applied in data processing for the selection of significant lipid features. A total of 24 lipids including lysophospholipids and FAs were identified and were consistently up-regulated in HCoV-229E-infected cells. Seven representative lipids were confirmed by authentic standards, including lysoPC (16:0/0:0), PAF C-16, lysoPE (16:0/0:0), AA, LA, PA and OA. Subsequent pathway analysis indicated that the LA&#8211;AA metabolism axis, consisting of LA and AA as important precursor lipids, was substantially perturbed after HCoV-229E infection. Moreover, we demonstrated that exogenously supplied LA and AA were capable of inhibiting the replication of HCoV-229E and the highly pathogenic MERS-CoV, which suggested the LA&#8211;AA metabolism axis to be a conserved and essential pathway in the propagation of human coronaviruses.A total of 24 lipids including lysoPCs, lysoPEs and unsaturated/saturated FAs were identified to be significantly upregulated after HCoV-229E infection. Twenty of these 24 (83.3%) lipids were lysoPC and lysoPE. LysoPC is the most abundant lysophospholipid in humans, with a high plasma concentration of several hundred micromoles. In addition, lysoPC was a potent inhibitor and could reversely arrest pore expansion during syncytium formation mediated by diverse viral fusogens [40]. Another lysophospholipid, lysoPE, is present at low concentrations in vivo but they induce various cellular responses such as activation of mitogen-activated protein kinase (MAPK) and neuronal differentiation when applied to cells in vitro [41]. Among the identified FAs, the LA and AA both belong to polyunsaturated omega-6 fatty acid and are essential fatty acids. In addition, LA is the metabolic precursor of AA, both of which are key components of the cell membrane. LA and AA also play fundamental roles in the biological function of many tissues by modulating enzymes, ion channels, receptors, as well as inflammation [42].Coronavirus replication is associated with intracellular membrane rearrangement and depends on the formation of double membrane vesicles (DMVs) and other membranous structures as replicative organelles [16]. The cell membrane components consist mainly of glycerophospholipid components such as phosphatidylcholine (PC), phosphatidylethanolamine (PE), lysophosphatidylcholine (lysoPC), and lysophosphatidylethanolamine (lysoPE). A specific phospholipids composition is required by different viruses to form the optimal replicative organelles best suited for their replication [43]. Moreover, the lysoPC/PE was produced from PC/PE by cPLA2 activation, which simultaneously generated corresponding fatty acid moiety. In this regard, cPLA2 activation is commonly believed to be beneficial for virus replication [16,17].In our study, we found that a number of lysophospholipids and FAs downstream of cPLA2 activation, were upregulated upon HCoV-229E infection. The upregulation of these lipid species including LA and AA were believed to promote efficient coronavirus replication. However, when we evaluate this hypothesis by exogenously supplementing additional LA and AA to HCoV-229E- or MERS-CoV-infected cells, we noticed a significant reduction in virus replication. Taken together, our data suggested that coronavirus infection did not randomly perturb the cellular lipid compositions. Instead, we speculate that coronaviruses precisely modulate and rearrange the host lipid profile to reach an intricate homeostasis optimized for its replication. Any exogenous manipulation that disrupts the equilibrium may interfere with the optimal replication of the viruses. Alternatively, supplementing LA and AA might disturb the LA&#8211;AA metabolism axis and result in feedback reversion of lysophospholipids into phospholipids through Land&#8217;s cycle [44], thus limiting virus replication. In addition, LA and AA are polyunsaturated fatty acids that are biological signaling precursors. They can be metabolized to important eicosanoids and metabolites, which play multiple roles in the host immune response and the pathogenesis of viral infections [45,46,47]. However, previous study had suggested that arachidonic acid (AA) downstream metabolites show no evidence of anti-coronaviral activity as observed through special inhibitors of cyclooxygenases (COX) 1/2 and 5-lipoxygenase (LOX), which are two key enzymes requiring AA as a precursor. The results indicated AA downstream products may not have a significant effect on coronaviruses replication, at least in vitro [16,48]. In this regard, the function of the downstream metabolites of LA and AA may play key roles in the pathogenesis of coronaviruses in vivo.\n5. Conclusions\nIn the present study, we revealed that the cellular lipid profile was rearranged upon HCoV-229E infection. A total of 24 lipids including lysoPCs, lysoPEs and FAs were upregulated. Among them, LA and AA, which were mapped into the LA-AA metabolism axis, demonstrated strong modulatory effects on the replication of HCoV-229E and the highly pathogenic MERS-CoV. In this regard, our data suggested that optimal coronavirus replication required a specific composition of cellular lipids and any disruption could decrease the efficiency of coronavirus replication. Thus, the MS-based lipidomics strategy could be used to monitor virus-specific lipid requirement, to discover the perturbed pathways and identify novel lipids to interfere with virus replication. In further studies, combining lipidomics data with biological and immunological data may help to elucidate specific pathogenic mechanisms and identify novel treatment strategies for virus infections.",
        "pmid": "30654597",
        "title": "Characterization of the Lipidomic Profile of Human Coronavirus-Infected Cells: Implications for Lipid Metabolism Remodeling upon Coronavirus Replication",
        "journal_title": "Viruses",
        "authors": "Bingpeng Yan;Hin Chu;Dong Yang;Kong-Hung Sze;Pok-Man Lai;Shuofeng Yuan;Huiping Shuai;Yixin Wang;Richard Yi-Tsun Kao;Jasper Fuk-Woo Chan;Kwok-Yung Yuen",
        "doi": "10.3390/v11010073"
    },
    {
        "keywords": "coronavirus;ER stress;apoptosis;signal transduction pathways;proinflammatory cytokines;unfolded protein response",
        "abstract": "The replication of coronavirus, a family of important animal and human pathogens, is closely associated with the cellular membrane compartments, especially the endoplasmic reticulum (ER). Coronavirus infection of cultured cells was previously shown to cause ER stress and induce the unfolded protein response (UPR), a process that aims to restore the ER homeostasis by global translation shutdown and increasing the ER folding capacity. However, under prolonged ER stress, UPR can also induce apoptotic cell death. Accumulating evidence from recent studies has shown that induction of ER stress and UPR may constitute a major aspect of coronavirus&#8211;host interaction. Activation of the three branches of UPR modulates a wide variety of signaling pathways, such as mitogen-activated protein (MAP) kinase activation, autophagy, apoptosis, and innate immune response. ER stress and UPR activation may therefore contribute significantly to the viral replication and pathogenesis during coronavirus infection. In this review, we summarize the current knowledge on coronavirus-induced ER stress and UPR activation, with emphasis on their cross-talking to apoptotic signaling.",
        "body": "INTRODUCTION\nCoronaviruses are a family of enveloped viruses with positive sense, non-segmented, single-stranded RNA genomes. Many coronaviruses are important veterinary pathogens. For example, avian infectious bronchitis virus (IBV) reduces the performance of both meat-type and egg-laying chickens and causes severe economic loss to the poultry industry worldwide (Cavanagh, 2007). Certain coronaviruses, such as HCoV-229E and HCoV-OC43, infect humans and account for a significant percentage of adult common colds (Hamre and Procknow, 1966; Kaye et al., 1972). Moreover, in 2003, a highly pathogenic human coronavirus (SARS-CoV) was identified as the causative agent of severe acute respiratory syndrome (SARS) with high mortality rate and led to global panic (Ksiazek et al., 2003). Later, it was found that the SARS-CoV was originated from bat and likely jumped to humans via some intermediate host (palm civets; Li et al., 2005; Wang and Eaton, 2007). Recently, a live SARS-like coronavirus was isolated from fecal samples of Chinese horseshoe bats, which could use the SARS-CoV cellular receptor &#8211; human angiotensin converting enzyme II (ACE2) for cell entry (Ge et al., 2013). This indicates that an intermediate host may not be necessary and direct human infection by some bat coronaviruses is possible. Moreover, a novel human coronavirus &#8211; the Middle East respiratory syndrome coronavirus (MERS-CoV), emerged in Saudi Arabia in September 2012 (de Groot et al., 2013). Although the risk of sustained human-to-human transmission is considered low, infection of MERS-CoV causes ~50% mortality in patients with comorbidities (Graham et al., 2013). Initial studies had pointed to bats as the source of MERS-CoV (Annan et al., 2013); however, accumulating evidence strongly suggested the dromedary camels to be the natural reservoirs and animal source of MERS-CoV (Hemida et al., 2013; Alagaili et al., 2014). Thus, coronaviruses can cross the species barrier to become lethal human pathogens, and studies on coronaviruses are both economically and medically important.Taxonomically, the family Coronaviridae is classified into two subfamilies, the coronavirinae and the torovirinae. The coronavirinae is further classified into three genera, namely the Alphacoronavirus, Betacoronavirus, and Gammacoronavirus (Masters, 2006). The classification was originally based on antigenic relationships and later confirmed by sequence comparisons of entire viral genomes (Gorbalenya et al., 2004). Almost all Alphacoronaviruses and Betacoronaviruses have mammalian hosts, including humans. In contrast, Gammacoronaviruses have mainly been isolated from avian hosts.Morphologically, coronaviruses are spherical or pleomorphic in shape with a mean diameter of 80&#8211;120 nm. They are characterized by the large (20 nm) &#8220;club-like&#8221; projections on the surface, which are the heavily glycosylated trimeric spike (S) proteins (Masters, 2006). Two additional structural proteins are found on the envelope. The abundant membrane (M) proteins give the virion its shape, whereas the small envelope (E) proteins play an essential role during assembly (Sturman et al., 1980; Liu and Inglis, 1991). Inside the envelope, the helical nucleocapsid is formed by binding of the nucleocapsid (N) proteins on the genomic RNA in a beads-on-a-string fashion. The genome, ranging from 27,000 to 32,000 nucleotides in size, is the largest RNA genomes known to date.Coronavirus infection starts with receptor binding via the S protein (Figure 1). The S proteins of most coronaviruses are cleaved by host protease into two functional subunits: an N-terminal receptor binding domain (S1) and a C-terminal domain (S2) responsible for membrane fusion (Huang et al., 2006; Qiu et al., 2006; Yamada et al., 2009). The interaction between the cell surface receptor and the S1 subunit is the major determinant of the tropism of coronaviruses (Kuo et al., 2000). Upon receptor binding of S1, a conformational change is triggered in the S2 subunit, exposing its hidden fusion peptide for insertion into the cellular membrane. This is followed by the packing of the two heptad repeats in the three monomers into a six-helix bundle fusion core. This close juxtaposition of the viral and cellular membrane enables fusion of the lipid bilayers, and the viral nucleocapsid is thus delivered into the cytoplasm (Masters, 2006).After uncoating, the genomic RNA first acts as an mRNA for translation of the replicase polyprotein. The replicase gene consists of two open reading frames (ORF1a and ORF1b). Translation of ORF1a produces the polyprotein 1a (pp1a). Meanwhile, a ribosomal frameshifting occurs at the junction of ORF1a and ORF1b, allowing translation to continue onto ORF1b, producing a larger polyprotein 1ab (pp1ab; Brierley et al., 1987). Autoproteolytic cleavage of pp1a produces 11 non-structural proteins (nsp1&#8211;nsp11), while cleavage of pp1ab produces 15 non-structural proteins (nsp1&#8211;nsp10 and nsp12&#8211;nsp16). The functions of these nsps are partially understood. Particularly, the autoproteolytic cleavage relies on nsp3 (a papain-like proteinase) and nsp5 (the main proteinase), whereas the RNA-dependent RNA polymerase (RdRp) is contained within nsp12 (Baker et al., 1993; Lu et al., 1995a).Using the genomic RNA as a template, the replicase then synthesizes the negative sense genomic RNAs, which are used as templates for synthesizing progeny positive sense RNA genomes. On the other hand, through discontinuous transcription of the genome, the replicase synthesizes a nested set of subgenomic RNAs (sgRNAs; Sawicki et al., 2007). Replication and transcription of the coronavirus genome involve the formation of the replication/transcription complexes (RTCs), which are anchored to the intracellular membranes via the multi-spanning transmembrane proteins nsp3, nsp4, and nsp6 (Oostra et al., 2007). Also, inside the infected cells, coronaviruses induce modification of the intracellular membrane network and formation of the double membrane vesicles (DMVs; Knoops et al., 2008). Several studies have shown that the DMVs are closely associated with the coronavirus RTCs and the de novo synthesized viral RNAs (Gosert et al., 2002; Snijder et al., 2006).The sgRNAs are translated into structural proteins and accessory proteins. Transmembrane structural proteins (S, M, and E) are synthesized, inserted, and folded in the endoplasmic reticulum (ER) and transported to the ER&#8211;Golgi intermediate compartment (ERGIC). The N proteins are translated in the cytoplasm and encapsidate the nascent progeny genomic RNA to form the nucleocapsids. Virion assembly occurs in the ERGIC and is likely to be orchestrated by the M protein through protein&#8211;protein interactions (Masters, 2006).The virions budded into the ERGIC are exported through secretory pathway in smooth-wall vesicles, which ultimately fuse with the plasma membrane and release the mature virus particles (Krijnse-Locker et al., 1994). For some coronaviruses, a portion of the S protein escapes from viral assembly and is secreted to the plasma membrane. These S proteins cause fusion of the infected cell with neighboring uninfected cells, resulting in the formation of a large, multinucleated cell known as a syncytium, which enables the virus to spread without being released into the extracellular space (Masters, 2006).In eukaryotic cells, ER is the major site for synthesis and folding of secreted and transmembrane proteins. The amount of protein entering the ER can vary substantially under different physiological states and environmental conditions. When protein synthesis surpasses the folding capacity, unfolded proteins accumulate in the ER and lead to ER stress. ER stress can also be activated by excessive lipids or pro-inflammatory cytokines (Kharroubi et al., 2004; Pineau et al., 2009). To maintain homeostasis, cells have evolved signaling pathways that are collectively known as the unfolded protein response (UPR; Ron and Walter, 2007). The UPR signaling starts with the unfolded proteins activating the three ER stress transducers: PKR-like ER protein kinase (PERK), activating transcriptional factor-6 (ATF6), or inositol-requiring protein-1 (IRE1; Figure 2). Once activated, these sensors transmit the signal across the ER membrane to the cytosol and the nucleus, and the cell responds by lowering the protein synthesis and increasing the ER folding capacity. If homeostasis cannot be re-established, apoptosis is induced for the benefit of the entire organism (Tabas and Ron, 2011).In this review, current studies on the involvement of the UPR in coronavirus infection and pathogenesis will be summarized. The role of UPR activation in host response, in particular the induction of apoptosis, will also be reviewed.\nCORONAVIRUS INFECTION AND ER STRESS\nGlobal proteomic and microarray analyses have shown that the expression of several genes related to the ER stress, such as glucose-regulated protein 94 (GRP94) and glucose-regulated protein 78 (GRP78, also known as immunoglobulin heavy chain-binding protein, or BiP), is up-regulated in cells infected with SARS-CoV or in cells overexpressing the SARS-CoV S2 subunit (Jiang et al., 2005; Yeung et al., 2008). Using a luciferase reporter system, Chan et al. (2006) found that both GRP94 and GRP78 were induced in SARS-CoV-infected FRhK4 cells. Consistently, the mRNA level of homocysteine-inducible, ER stress-inducible, ubiquitin-like domain member 1 (HERPUD1), an ER stress marker, was up-regulated in L cells infected with mouse hepatitis virus (MHV) or SARS-CoV (Versteeg et al., 2007). Data from this group have shown a similar induction of ER stress in IBV-infected Vero, H1299, and Huh-7 cells (unpublished observations). Although no parallel studies have been performed on Alphacoronaviruses, it is likely that all three genera of coronaviruses may induce ER stress in the infected cells. Current evidence suggests the following three main mechanisms.\nFORMATION OF DOUBLE MEMBRANE VESICLES\nIt is well-known that the replication of many plus-stranded RNA viruses induces modification of cellular membranes (Miller and Krijnse-Locker, 2008). Among them, coronaviruses have been shown to induce the formation of DMVs in infected cells (David-Ferreira and Manaker, 1965). Based on immunocytochemistry electron microscopy data, the DMVs co-localize with coronavirus major replicase proteins and are presumably the sites where coronavirus RTCs are located (Gosert et al., 2002; Snijder et al., 2006). Indeed, DMVs are induced in HEK293T cells co-expressing the SARS-CoV nsp3, nsp4, and nsp6, which are all multispanning transmembrane non-structural proteins (Angelini et al., 2013). There have been different perspectives regarding the origin of the coronavirus-induced DMVs. The late endosomes, autophagosomes, and the early secretary pathway have all been implicated as the membrane source of DMVs (van der Meer et al., 1999; Prentice et al., 2004; Verheije et al., 2008). Also, co-localization has been observed between SARS-CoV non-structural proteins and protein disulfide isomerase (PDI), an ER marker (Snijder et al., 2006). Using high-resolution electron tomography, Knoops et al. (2008) have shown that infection of SARS-CoV reorganizes the ER into a reticulovesicular network, which consists of convoluted membranes and interconnected DMVs. Recently, Reggiori et al. (2010) have proposed a model in which coronaviruses hijack the EDEMosomes to derive ER membrane for DMV formation. The EDEMosomes are COPII-independent vesicles that export from the ER, which are normally used to fine-tune the level of ER degradation enhancer, mannosidase alpha-like 1 (EDEM1), a regulator of ER-associated degradation (ERAD; Cal&#236; et al., 2008). It has been demonstrated that MHV infection causes accumulation of EDEM1 and osteosarcoma amplified 9 (OS-9, another EDEMosome cargo), and that both EDEM1 and OS-9 co-localize with the RTCs of MHV (Reggiori et al., 2010). These results thus add mechanical evidence to support the ER-origin of the coronavirus-induced DMVs.\nGLYCOSYLATION OF CORONAVIRAL STRUCTURAL PROTEINS\nExcept for the N protein, all coronavirus structural proteins are transmembrane proteins synthesized in the ER. The M protein, which is the most abundant component of the virus particle, is known to undergo either O-linked (for most betacoronaviruses) or N-linked (for all alpha- and gammacoronaviruses) glycosylation in the ER (Jacobs et al., 1986; Cavanagh and Davis, 1988; Nal et al., 2005). The glycosylation of M protein is proposed to play a certain function in alpha interferon (IFN) induction and in vivo tissue tropism (Charley and Laude, 1988; Laude et al., 1992; de Haan et al., 2003). The pre-glycosylated S monomers are around 128&#8211;160 kDa, whereas sizes can reach 150&#8211;200 kDa post-glycosylation (exclusively N-linked), indicating that the S protein is highly glycosylated (Masters, 2006). At least for transmissible gastroenteritis coronavirus (TGEV), glycosylation is presumed to facilitate monomer folding and trimerization (Delmas and Laude, 1990). Moreover, the glycans on SARS-CoV S proteins have been shown to bind C-type lectins DC-SIGN (dendritic cell-specific intercellular adhesion molecule-3-grabbing non-integrin) and L-SIGN (liver lymph node-specific intercellular adhesion molecule-3-grabbing non-integrin), which can serve as alternative receptors for SARS-CoV independent of the major receptor ACE2 (Han et al., 2007). The folding, maturation, and assembly of the gigantic S trimeric glycoprotein rely heavily on the protein chaperons inside the ER, such as calnexin. In fact, the N-terminal part of the S2 domain of SARS-CoV S protein has been found to interact with calnexin, and knock-down of calnexin decreases the infectivity of pseudotyped lentivirus carrying the SARS-CoV S protein (Fukushi et al., 2012). Also, treatment of &#945;-glucosidase inhibitors, which inhibit the interactions of calnexin with its substrates, dose dependently inhibits the incorporation of S into pseudovirus and suppresses SARS-CoV replication in cell cultures (Fukushi et al., 2012). During coronavirus replication, massive amount of structural proteins is synthesized to assembly progeny virions. The production, folding, and modification of these proteins undoubtedly increase the workload of the ER.\nDEPLETION OF ER LIPID DURING THE BUDDING OF VIRIONS\nBudding of coronaviruses occurs in the ERGIC, which is a structural and functional continuance of the ER. Thus, the release of mature virions by exocytosis in effect depletes the lipid component of the ER. Taken together, coronavirus infection results in: (1) massive morphological rearrangement of the ER; (2) significant increase ER burden for protein synthesis, folding and modification; and (3) extensive depletion of ER lipid component. These factors together may contribute to the coronavirus-induced ER stress.In the following sections, the activation of the three individual branches of the UPR by coronavirus infection will be discussed in detail.\nTHE PERK BRANCH OF UPR\n\nPERK-EIF2\u03b1-ATF4 SIGNALING PATHWAY\nThe PERK branch of the UPR is believed to be activated first in response to ER stress (Szegezdi et al., 2006). Activation of PERK begins with the dissociation from ER chaperon BiP, followed by oligomerization and auto-phosphorylation. Activated PERK then phosphorylates the &#945;-subunit of eukaryotic initiation factor 2 (eIF2&#945;). Phosphorylated eIF2&#945; forms a stable complex with and inhibits the turnover of eIF2B, a guanine nucleotide exchange factor that recycles inactive eIF2-GDP to active eIF2-GTP. This results in a general shutdown of cellular protein synthesis and reduces the protein flux into the ER (Ron and Walter, 2007). Besides PERK, three other kinases are known to phosphorylate eIF2&#945;, namely the protein kinase RNA-activated (PKR), heme-regulated inhibitor kinase (HRI), and general control non-derepressible-2 (GCN2; Ron and Walter, 2007). PKR is induced by IFN and activated by the binding of double-stranded RNA (dsRNA) after virus infection (Clemens and Elia, 1997). HRI is activated in red blood cells and hepatocytes by low levels of heme (McEwen et al., 2005). GCN2 senses amino acid deficiency and is activated via binding to uncharged transfer RNAs (Sood et al., 2000). Due to common outcome (eIF2&#945; phosphorylation and translation suppression), activation of these kinases is collectively known as the integrated stress response (ISR; Ron and Walter, 2007).Interestingly, the mRNAs of certain genes contain small ORFs in their 5&#8242; UTR and bypass the eIF2&#945;-dependent translation block. One of these is the activating transcription factor 4 (ATF4), which is preferentially translated under ISR. ATF4 in turn transactivates genes involved in amino acid metabolism, redox reactions, and stress response. One of ATF4&#8217;s target genes is the growth arrest and DNA damage-inducible protein 153 (GADD153, also known as C/EBP homologous protein, or CHOP). GADD153 induces the growth arrest and DNA damage-inducible protein 34 (GADD34), which recruits protein phosphatase 1 (PP1) to dephosphorylate eIF2&#945; and release the translation block. To this end, if ER stress is resolved, normal protein synthesis can be resumed. However, if ER stress persists, GADD153 can induce apoptosis by suppressing the anti-apoptotic protein B-cell lymphoma 2 (Bcl-2) and inducing the pro-apoptotic proteins such as Bcl-2-interacting mediator of cell death (Bim; Puthalakath et al., 2007). GADD153 also activates ER oxidoreductin-1&#945; (ERO1&#945;), which encodes an ER oxidase. The increase protein influx to a hyper-oxidizing ER aggravates ER stress and induces apoptosis (Marciniak et al., 2004; Figure 3).\nINVOLVEMENT OF THE PERK PATHWAY DURING VIRAL INFECTIONS\nTranslation attenuation has been widely observed as a defensive mechanism of the host cells against viral infection. By reducing the translation of viral proteins, virus replication is hampered and the spread of infection is limited, giving enough time for the immune system to initiate effective antiviral responses. Among the four eIF2&#945; kinases, PKR, due to its IFN-inducible nature and specific recognition of viral dsRNAs, plays an especially important role in inducing translation attenuation in virus-infected cells (He, 2006). It is therefore not surprising that viruses have evolved various mechanisms to counteract PKR. For example, the non-structural 5A (NS5A) protein of hepatitis C virus directly interacts with the catalytic site of PKR, whereas the NS1 protein in the influenza A virus binds to dsRNAs and thus blocks PKR activation (Lu et al., 1995b; Gale et al., 1997).During virus infection, massive production of viral proteins can overload the folding capacities of ER and lead to activation of another eIF2&#945; kinase &#8211; PERK. Activation of PERK has been observed in cells infected with various DNA and RNA viruses, such as vesicular stomatitis virus, bovine viral diarrhea virus and herpes simplex virus 1 (HSV1), to name just a few (Jordan et al., 2002; Baltzis et al., 2004; Cheng et al., 2005). However, similar to PKR, viruses have adopted counter measures to inhibit PERK-mediated translation attenuation. For example, the E2 protein of hepatitis C virus (HCV) and the glycoprotein gB of HSV1 bind to PERK and inhibit its kinase activity to rescue translation (Pavio et al., 2003; Mulvey et al., 2007).\nACTIVATION OF PERK PATHWAY DURING CORONAVIRUSES INFECTION AND ITS INVOLVEMENT IN CORONAVIRUS-INDUCED APOPTOSIS\nThere have been diverging results on the activation of PKR and/or PERK during coronavirus infection. In an early study, it has been found that there is minimal transcriptional activation of PKR and another IFN-stimulated gene, 2&#8242;5&#8242;-oligoadenylate synthetase (OAS) in cells infected with MHV-1 (Zorzitto et al., 2006). In a separate study, phosphorylation of PKR and eIF2&#945; was also not observed in MHV A59-infected cells (Ye et al., 2007). However, Bechill et al. (2008) have detected significant eIF2&#945; phosphorylation and up-regulation of ATF4 in cells infected with MHV A59, although no induction of GADD153 and GADD34 was observed. It has been suggested that due to the lack of GADD34-mediated eIF2&#945; dephosphorylation, MHV infection induces sustained translation repression of most cellular proteins (Bechill et al., 2008). However, the translation of MHV mRNAs seems to be resistant to eIF2&#945; phosphorylation, and the detailed mechanisms for such evasion are yet to be investigated. As for SARS-CoV, PKR, PERK, and eIF2&#945; phosphorylation are readily detectable in virus-infected cells (Kr&#228;hling et al., 2009). However, knock-down of PKR using specific morpholino oligomers did not affect SARS-CoV-induced eIF2&#945; phosphorylation but significantly inhibited SARS-CoV-induced apoptosis (Kr&#228;hling et al., 2009). It is possible that eIF2&#945; is phosphorylated by PERK in SARS-CoV-infected cells, but similar loss-of-function experiments have not been performed, although overexpression of SARS-CoV accessory protein 3a has been shown to activate the PERK pathway (Minakshi et al., 2009).The discrepancy regarding the activation of PKR/PERK during coronavirus infection may be a result from the different cell culture systems and virus strains used. The interpretation is further complicated by the IFN-inducible nature of PKR. It is generally believed that coronaviruses are poor type I IFN inducers in vitro (Garlinghouse et al., 1984; Spiegel et al., 2005; Roth-Cross et al., 2007), although the IFN response may be essential for antiviral activities in vivo (Ireland et al., 2008). Moreover, it is known that coronaviruses employ multiple mechanisms to antagonize the IFN response. For example, the nsp16 has been shown to utilize the 2&#8242;-O-methyltransferase activity to modify coronavirus mRNAs, so as to evade from the cytosolic RNA sensor melanoma differentiation-associated protein 5 (MDA5) and type I IFN induction (Roth-Cross et al., 2008; Z&#252;st et al., 2011). Furthermore, the activities of several IFN-induced genes (ISGs) have also been shown to be modulated by coronaviruses during infection. For instance, Zhao et al. (2012) have demonstrated that the MHV accessory protein ns2 cleaves 2&#8242;,5&#8242;-oligoadenylate, the product of an ISG called OAS. This results in the suppression of the cellular endoribonuclease RNase L activity and facilitates virus replication in vitro and in vivo (Zhao et al., 2011, 2012). Thus, similar uncharacterized mechanisms may be used by MHV and other coronaviruses to block the activation and/or downstream signaling of PKR. In this regard, the activation of PERK via ER stress seems to be an alternative pathway to activate eIF2&#945;, although coronaviruses may counteract by directly targeting eIF2&#945;, as described below.Studies done by this group have shown that, phosphorylation of PKR, PERK, and eIF2&#945; was detectable at early stage of IBV infection (0&#8211;8 hpi) but diminished quickly later (Wang et al., 2009; Liao et al., 2013). The rapid de-phosphorylation of eIF2&#945; is likely due to the accumulation of GADD34, which is a component of the PP1 complex and a downstream target gene induced by GADD153 (Wang et al., 2009). Despite of the rapid de-phosphorylation of eIF2&#945;, significant induction of GADD153 was observed at late stage of infection (16&#8211;24 h) at both mRNA and protein levels (Liao et al., 2013). The up-regulation of GADD153 was likely mediated by both PKR and PERK, since knock-down of either PKR or PERK by siRNA reduces IBV-induced GADD153 (Liao et al., 2013). The up-regulation of GADD153 promotes apoptosis in IBV-infected cells, possibly via inducing the pro-apoptotic protein tribbles-related protein 3 (TRIB3) and suppressing the pro-survival kinase extracellular signal-related kinase (ERK; Liao et al., 2013). Based on the findings so far obtained, it is safe to conclude that the PERK/PKR-eIF2&#945;-ATF4-GADD153 pathway is activated by some, but not all, coronaviruses. In the infected cells, this pathway is activated at an early stage but quickly modulated by feedback de-phosphorylation. The PERK/PKR-eIF2&#945;-ATF4-GADD153 most likely plays a pro-apoptotic function during coronavirus infection.\nINTEGRATED STRESS RESPONSE PATHWAYS AND INNATE IMMUNITY\nSeveral recent studies have demonstrated the critical roles of cellular stress response pathways in modulating the innate immune activation (Cl&#225;udio et al., 2013). One of the key regulators that bridge stress and innate immunity is GADD34, a negative regulator of eIF2&#945; activation. It has been shown that when stimulated with polyriboinosinic:polyribocytidylic acid (polyI:C), the integrated stress response pathways were activated in dendritic cells (DCs), leading to up-regulation of ATF4 and GADD34 (Clavarino et al., 2012). Interestingly, GADD34 expression did not significantly affect protein synthesis in DCs, but was shown to be crucial for the production of interferon &#946; (IFN-&#946;) and pro-inflammatory cytokines interleukin-6 (IL-6; Clavarino et al., 2012). In contrast, GADD34 has also been shown to specify PP1 to dephosphorylate the TGF-&#946;-activated kinase 1 (TAK1), thus negatively regulating the toll-like receptor (TLR) signaling and pro-inflammatory cytokines [IL-6 and TNF-&#945; (Tumor necrosis factor alpha)] production in macrophages (Gu et al., 2014). The functional disparities of GADD34 in DCs and macrophages indicate that the integrated stress response may be regulated by some other signaling pathways, resulting in cell-type specific outcomes in the innate immune activation. Since GADD34 induction was readily observed in cells infected with IBV (Wang et al., 2009), it will be intriguing to ask whether GADD34 also contributes to IBV-induced pro-inflammatory cytokine production, and to determine potential cross-talks between the PERK pathway and innate immune activation during IBV infection.The massive production of pro-inflammatory cytokines (cytokine storm) has been associated with the immunopathogenesis and high mortality rate of SARS-CoV (Perlman and Dandekar, 2005). The transcription factor nuclear factor kappa-light-chain-enhancer of activated B cells (NF-&#954;B) is a master regulator of pro-inflammatory response and innate immunity (Hayden and Ghosh, 2012). It has been well established that NF-&#954;B is required for the induction of pro-inflammatory cytokines (such as IL-6 and IL-8) and the early expression of IFN-&#946; during RNA virus infection (Libermann and Baltimore, 1990; Kunsch and Rosen, 1993; Wang et al., 2010; Balachandran and Beg, 2011; Basagoudanavar et al., 2011). Interestingly, induction of TNF-&#945;, IL-6, and IL-8 has been detected in cells overexpressing the spike protein of SARS-CoV via the NF-&#954;B pathway (Wang et al., 2007; Dosch et al., 2009). Thus, it is intriguing to consider the involvement of ER stress in activating the NF-&#954;B pathway during coronavirus infection. In its inactive form, NF-&#954;B is sequestered in the cytoplasm by inhibitor of NF-&#954;B alpha (I&#954;B&#945;), which masks the nuclear localization signal of NF-&#954;B (Karin and Ben-Neriah, 2000). The basal level of I&#954;B&#945; is maintained by constitutive synthesis and degradation of the protein (Kanarek et al., 2010). Under various stress conditions, phosphorylation of eIF2&#945; leads to global translation repression and a net decrease in I&#954;B&#945; protein level (Jiang et al., 2003). This then results in the activation of NF-&#954;B and induction of pro-inflammatory response (Figure 3). Nonetheless, further studies are needed to characterize the actual contributions of ER stress in NF-&#954;B-mediated cytokine induction during coronavirus infection.Previous study done by this group has shown that infection of IBV induced the production of IL-6 and IL-8, which was dependent on the phosphorylation of MAP kinase p38 (Liao et al., 2011). Interestingly, a protein phosphatase called dual-specificity phosphatase 1 (DUSP1) was also up-regulated in IBV-infected cells and dephosphorylated p38 to modulate pro-inflammatory cytokine production (Liao et al., 2011). Previous studies have shown that the mRNA and protein levels of DUSP1 are modulated by ER stress (Boutros et al., 2008; Li et al., 2011). ER stress-induced DUSP1 up-regulation is likely to be mediated by ATF3 in the PERK pathway, since knock-down of ATF3 significantly reduced DUSP1 induction in cells under ER stress (Gora et al., 2010). Thus, it is possible that IBV infection activates the PERK branch of UPR to induce DUSP1 expression, which in turn dephosphorylates p38 to modulate IBV-induced pro-inflammatory cytokine production (Figure 3).Besides p38, DUSP1 has also been shown to dephosphorylate c-Jun N-terminal kinase (JNK) and ERK (Sun et al., 1993; Franklin and Kraft, 1997). It has been long proposed that ERK phosphorylation promotes cell survival, whereas prolonged JNK and p38 phosphorylation is linked to the induction of apoptosis (Xia et al., 1995). Thus, the induction of DUSP1 by ER stress in coronavirus-infected cells may also contribute to virus-induced apoptosis via modulation of the MAP kinase pathways.\nTHE IRE1 BRANCH OF UPR\n\nIRE1-XBP1 SIGNALING PATHWAY\nThe IRE1-XBP1 branch of the UPR is evolutionarily conserved from yeast to humans. In response to unfolded proteins, IRE1 undergoes oligomerization (Bertolotti et al., 2000). This results in trans-autophosphorylation of the kinase domain and the activation of IRE1&#8217;s RNase domain. So far, the only known substrate for IRE1 RNase activity is the mRNA of the X box binding protein 1 (XBP1) gene (Yoshida et al., 2001a; Calfon et al., 2002). IRE1 cuts the XBP1 mRNA twice, removing a 26-nucleotide intron to form a frameshifted transcript, the spliced XBP1 (XBP1s). Whereas the unspliced XBP1 mRNA (XBP1u) encodes an inhibitor of the UPR, XBP1s encode a potent transcriptional activator, which translocates to the nucleus and enhances the expression of many UPR genes, including those encoding molecular chaperones and proteins contributing to ER-associated degradation (Ng et al., 2000; Lee et al., 2003; Figure 4).Apart from the XBP1 pathway, activated IRE1 has been shown to recruit TNF receptor-associated factor 2 (TRAF2) and induce apoptosis by activating the JNK (Urano et al., 2000). This IRE1-JNK pathway is independent of IRE1&#8217;s RNase activity, but it requires IRE1&#8217;s kinase domain and involves TRAF2-dependent activation of caspase-12 (Yoneda et al., 2001). Moreover, one recent study has demonstrated that the IRE1-JNK pathway is required for autophagy activation after pharmacological induction of ER stress. It was found that the kinase domain but not the RNase activity of IRE1 was required, and treatment of a JNK inhibitor (SP600125) abolished autophagosome formation after ER stress (Ogata et al., 2006). Therefore, the IRE1 branch of UPR is closely associated with the JNK pathway and involved in JNK-mediated apoptosis and autophagy signaling.\nACTIVATION OF THE IRE1 PATHWAY DURING CORONAVIRUSES INFECTION\nThe involvement of IRE1-XBP1 pathway during coronavirus infection has been investigated by several studies, using MHV as a model. Either MHV infection or overexpression of the MHV S protein (but not other structural proteins) induces XBP1 mRNA splicing (Versteeg et al., 2007; Bechill et al., 2008). However, although XBP1 mRNA is efficiently spliced, the protein product of spliced XBP1 cannot be detected in either the whole cell lysate or the nuclear fraction. Moreover, UPR downstream genes known to be activated by XBP1s, such as ER DNA J domain-containing protein 4 (ERdj4), EDEM1, and protein kinase inhibitor of 58 kDa (p58IPK), are not significantly induced after infection (Bechill et al., 2008). Using a luciferase reporter system, it is shown that MHV infection does not inhibit transactivation of unfolded protein response element (UPRE) and ER stress response element (ERSE) promoter by XBP1s. Because MHV infection is associated with persistent eIF2&#945; phosphorylation and host translational repression, it is likely that failure to translate the XBP1s protein may be the main reason why activation of the IRE1 branch does not occur even though XBP1 mRNA splicing is observed. On the other hand, although SARS-CoV belongs to the same genera of Betacoronavirus as MHV, neither infection with SARS-CoV nor overexpression of SARS-CoV S protein induces XBP1 mRNA splicing (Versteeg et al., 2007; DeDiego et al., 2011). It is possible that other viral proteins of SARS-CoV (such as the E protein mentioned below), function as an antagonist of IRE1-XBP1 activation.Result from this group has also shown that the IRE1-XBP1 pathway is activated in cells infected with IBV. In IBV-infected Vero cells, significant splicing of XBP1 mRNA was detected starting from 12 to 16 h post-infection till the late stage of infection. The mRNA levels of XBP1 effector genes (EDEM1, ERdj4, and p58IPK) were up-regulated in IBV-infected Vero cells. The activation of IRE1-XBP1 pathway was also detectable, though at a lower level, in other cell lines such as H1299 and Huh-7 cells. Treatment of IRE1 inhibitor effectively blocked IBV-induced XBP1 mRNA splicing and effector genes up-regulation in a dosage-dependent manner. Consistently, knockdown of IRE1 inhibited IBV-induced XBP1 mRNA splicing, whereas overexpression of wild-type IRE1 (but not its kinase dead or RNase domain deleted mutants) enhanced IBV-induced XBP1 mRNA splicing. These results suggest that the IRE1-XBP1 pathway is indeed activated in cells infected with IBV. Interestingly, an earlier onset and more significant apoptosis induction in IRE1-knockdown IBV-infected cells was observed, which is associated with hyper-phosphorylation of pro-apoptotic kinase JNK and hypo-phosphorylation of pro-survival kinase RAC-alpha serine/threonine-protein kinase (Akt). Taken together, IRE1 may modulate IBV-induced apoptosis and serve as a survival factor during coronavirus infection.Interestingly, a recent report by DeDiego et al. (2011) demonstrates that the coronavirus E protein may modulate the IRE1-XBP1 pathway. Using a recombinant SARS-CoV that lacks the E protein (rSARS-CoV-&#916;E), it is found that both XBP1 splicing and induction of UPR genes significantly increase in the absence of E protein. Moreover, E protein also suppresses ER stress induced by RSV and drugs (thapsigargin and tunicamycin; DeDiego et al., 2011). Whether the UPR modulating activity is related to the viroporin property of E protein remains to be investigated, but this study explains, at least in part, why SARS-CoV lacking the E protein is attenuated in animal models (Liao et al., 2004; DeDiego et al., 2007).\nIRE1-DEPENDENT DECAY DURING VIRUS INFECTION\nNotably, one recent study has demonstrated an alternative function of IRE1. It was found that at the late stage of ER stress, IRE1 mediates non-specific cleavage of membrane-associated mRNA species. This was dubbed IRE1-dependent decay (RIDD) and was proposed to resolve ER stress by reducing the amount of transcripts influx (Hollien et al., 2009). It is intriguing to think of RIDD as a host anti-viral mechanism. During prolonged ER stress induced by infection, non-specific RNase activity of IRE1 may decay the membrane associated viral mRNA. In fact, it has been recently suggested that RIDD is activated during Japanese encephalitis virus (JEV) infection in Neuro2a cells (Bhattacharyya et al., 2014). Interestingly, RIDD specifically degraded known target mRNA transcripts but not JEV RNAs. Also, treatment with IRE1 RNase activity inhibitor suppressed viral replication, indicating that JEV benefits from RIDD activation (Bhattacharyya et al., 2014).\nIRE1 PATHWAY AND INNATE IMMUNITY\nSimilarly to the integrated stress response, the IRE1 pathway has also been implicated in the innate immune response (Cl&#225;udio et al., 2013). Martinon et al. (2010) have shown that in murine macrophages, the IRE1-XBP1 pathway is specifically activated by TLR4 and TLR2. Interestingly, the ER stress and TLR activation synergistically activate IRE1 and induce the production of pro-inflammatory cytokines such as IL-1&#946; and IL-6 (Martinon et al., 2010). Consistently, Hu et al. (2011) have demonstrated that the IRE1-XBP1 pathway is also involved in IFN-&#946; and pro-inflammatory cytokines production in murine DCs induced by polyI:C. Significantly, it has been shown that overexpression of the spliced form of XBP1 enhanced IFN-&#946; production in DCs and significantly suppressed vesicular stomatitis virus infection (Hu et al., 2011). Preliminary results from this group have also found that the activation of IRE1-XBP1 pathway is required for IL-8 induction in cells infected with IBV (unpublished data). On the other hand, the kinase but not the RNAse activity of IRE1 has been associated with ER-stress-induced NF-kB activation (Tam et al., 2012). Under ER stress, IRE1 has been shown to phosphorylate TRAF2, which activates the I&#954;B kinase (IKK) and contributes to its basal activity (Figure 4). IKK in turn phosphorylates I&#954;B&#945; and promotes its proteasomal degradation, releasing NF-&#954;B to activate downstream genes (Tam et al., 2012). Taken together, these findings suggest that IRE1 may act synergistically with players in innate immunity and serve as a supplementary sensor and/or signaling factors during coronavirus infection.\nTHE ATF6 BRANCH OF UPR\nThe ER stress sensor ATF6 has an N-terminal cytoplasmic domain, a single transmembrane segment and an ER luminal domain that sense the presence of unfolded/misfolded proteins. Under ER stress, ATF6 is translocated from the ER to the Golgi apparatus and cleaved by protease S1P and S2P (Haze et al., 1999). The cleavage releases the cytosolic basic leucine zipper (bZIP) domain, which translocates into the nucleus and activates genes harboring the ERSE or ERSE II (Yoshida et al., 2001b). The identified target genes of ATF6 include ER chaperones (such as GRP78, GRP94), PDI, and the UPR transcription factors GADD153 and XBP1 (Szegezdi et al., 2006). Previously, it was proposed the ATF6 pathway is mainly pro-survival, as it enhances the ER protein folding capacity to counteract ER stress (Szegezdi et al., 2006). However, recent studies have demonstrated that, under certain circumstances, ATF6-mediated signals may also contribute to ER-stress-induced apoptosis, possibly via activation of CHOP and/or suppression of myeloid cell leukemia sequence 1 (Mcl-1; Gotoh et al., 2002; Nakanishi et al., 2005; Morishima et al., 2011).The infection of cells by several viruses has been shown to activate the ATF6 pathway, including the Tick-borne encephalitic virus, African swine fever virus (ASFV), West Nile virus (WNV), and HCV (Ambrose and Mackenzie, 2011; Merquiol et al., 2011; Galindo et al., 2012; Yu et al., 2013). In the case of ASFV, ATF6 activation has been shown to modulate ASFV-induced apoptosis and facilitate viral replication (Galindo et al., 2012). For WNV, it has been shown that ATF6 activation promotes efficient WNV replication by suppressing signal transducer and activator of transcription 1 (STAT1) phosphorylation and late-phase IFN signaling (Ambrose and Mackenzie, 2013). The NS4B protein of HCV has been shown to activate ATF6 signaling in cultured cells (Li et al., 2009). Induction of chronic ER stress and adaptation of infected hepatocyte to UPR have been considered important for HCV persistent infection and pathogenesis in vivo (Asselah et al., 2010; Merquiol et al., 2011).Compared with the PERK and IRE1 pathway, the induction of ATF6 pathway during coronaviruses infection has not been deeply investigated. In MHV-infected cells, significant cleavage of ATF6 could be detected starting from 7 h post-infection (Bechill et al., 2008). However, the levels of both full length and cleaved ATF6 protein diminished at later time points during infection. Moreover, activation of ATF6 target genes was not observed at the mRNA level, as determined by luciferase reporter constructs under the control of ERSE promoters (Bechill et al., 2008). It is also unlikely that MHV infection suppresses downstream signaling of the ATF6 pathway, because the reporter induction by overexpressed ATF6 was not inhibited by MHV infection. The authors thus conclude that global translation shutdown via eIF2&#945; phosphorylation prevents accumulation of ATF6 and activation of ATF6 target genes (Bechill et al., 2008). The involvement of ATF6 pathway during infection of other coronaviruses has not been well characterized.Although the spike proteins of coronaviruses have been considered as the major contributor in ER stress induction, overexpression of SARS-CoV spike protein fails to activate ATF6 reporter constructs (Chan et al., 2006). On the other hand, the accessory protein 8ab of SARS-CoV has been identified to induce ATF6 activation (Sung et al., 2009). The 8ab protein was found in SARS-CoV isolated from animals and early human isolates. In SARS-CoV isolated from humans during the peak of the epidemic, there is a 29-nt deletion in the middle of ORF8, resulting in the splitting of ORF8 into two smaller ORFs, namely ORF8a and ORF8b, which encode two truncated polypeptides 8a and 8b (Guan et al., 2003). ATF6 cleavage and nuclear translocation was observed in cells transfected with SARS-CoV 8ab (Sung et al., 2009). Physical interaction between 8ab and the luminal domain of ATF6 was also demonstrated by co-immunoprecipitation. However, similar experiments have not been performed for the 8a and 8b proteins. Also, further studies using recombinant SARS-CoV lacking 8a, 8b, or 8ab would be required.\nCONCLUSION\nCoronaviruses constitute human and animal pathogens that are medically and economically important. Much remains unknown regarding the host&#8211;virus interactions during infection. Recent studies have demonstrated that coronaviruse infection induces ER stress in infected cells and activates the UPR. Activation of the PERK pathway (possibly in synergy with PKR and/or other integrated stress response kinases) leads to phosphorylation of eIF2&#945; and a global translation shutdown. At late stage of infection, up-regulation of transcription factor GADD153 likely contributes to coronaviruses induced apoptosis. Activation of the IRE1 pathway induces XBP1 mRNA splicing and expression of downstream UPR genes. Interestingly, IRE1 but not XBP1 is also shown to modulate the JNK and Akt kinase activities, thus protecting infected cells from virus induced apoptosis. The ATF6 pathway is also activated in coronavirus-infected cells, resulting in the up-regulation of chaperon proteins to counteract ER stress.However, many questions remain to be addressed. First, although the coronaviruses spike proteins are demonstrated to induce ER stress and UPR, detailed mechanisms regarding molecular interactions between the spike proteins and PERK/IRE1/ATF6 have not been determined. Second, it should be noted that the phenotypes observed in cells overexpressing viral proteins may not essentially reflect their physiological functions in the setting of a real infection. Further experiments using recombinant viruses with deletion of or modification in the target viral proteins should be performed to validate these findings (DeDiego et al., 2011). Last but not the least, the three branches of UPR should not be considered functionally independent, but rather as an integrated regulatory network (Ron and Walter, 2007). For example, besides being spliced by IRE1, XBP1 is also transcriptionally activated by PERK and ATF6 (Yoshida et al., 2001a; Calfon et al., 2002). Also, it is difficult to separate the translation shutdown effect mediated by PERK and the induction of UPR genes by PERK and the other two ER stress sensors, as in the studies with MHV (Bechill et al., 2008).Nonetheless, there are scientific and clinical significance for studies on ER stress and UPR induction during infection with coronaviruses and other viruses. As an evolutionarily conserved and well-characterized stress response pathway, it serves as a perfect model to study host&#8211;virus interactions and pathogenesis. Moreover, besides apoptosis, UPR has been recently demonstrated to crosstalk with other major cellular signaling pathways, including MAP kinases pathways, autophagy, and innate immune responses (Yoneda et al., 2001; Ogata et al., 2006; Martinon et al., 2010; Hu et al., 2011; Clavarino et al., 2012). Thus, further investigations on coronavirus-induced UPR may also help identifying new targets for antiviral agents and developing more effective vaccines against coronaviruses.\nConflict of Interest Statement\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.",
        "pmid": "24987391",
        "title": "Coronavirus infection, ER stress, apoptosis and innate immunity",
        "journal_title": "Frontiers in Microbiology",
        "authors": "To S. Fung;Ding X. Liu",
        "doi": "10.3389/fmicb.2014.00296"
    },
    {
        "keywords": "coronavirus;virus entry;viral pathogenesis;spike protein;carcinoembryonic antigen;angiotensin converting enzyme 2;endocytosis;cathepsin;transmembrane protease; membrane fusion",
        "abstract": "Coronavirus-cell entry programs involve virus-cell membrane fusions mediated by viral spike (S) proteins. Coronavirus S proteins acquire membrane fusion competence by receptor interactions, proteolysis, and acidification in endosomes. This review describes our current understanding of the S proteins, their interactions with and their responses to these entry triggers. We focus on receptors and proteases in prompting entry and highlight the type II transmembrane serine proteases (TTSPs) known to activate several virus fusion proteins. These and other proteases are essential cofactors permitting coronavirus infection, conceivably being in proximity to cell-surface receptors and thus poised to split entering spike proteins into the fragments that refold to mediate membrane fusion. The review concludes by noting how understanding of coronavirus entry informs antiviral therapies.",
        "body": "1. Introduction\nCoronaviruses (CoVs) are enveloped RNA viruses causing respiratory and enteric diseases. The CoVs are widely distributed in nature and their zoonotic transmissions into human populations can cause epidemic disease. After entering into respiratory or gastrointestinal tracts, these viruses establish themselves by entering and infecting lumenal macrophages and epithelial cells. The cell entry programs for these viruses are orchestrated by the viral spike (S) proteins that bind cellular receptors and also mediate virus-cell membrane fusions. CoV diversity is reflected in the variable S proteins, which have evolved into forms differing in their receptor interactions and their response to various environmental triggers of virus-cell membrane fusion. As such, seemingly minor differences in CoV S protein structure and function often correlate with striking changes in CoV tropism and virulence. This review focuses on the S: receptor interactions and events leading to membrane fusion and successful cell entry. The S proteins are large; an average CoV S protein contains 1,300 amino acids and 20 asparagine-linked glycans, making the S proteins substantially larger than the glycoproteins mediating receptor binding and membrane fusion of other enveloped viruses. These S monomers are assembled into trimers within the ER of virus-producing cells [1] and incorporated into virus particles at the CoV budding sites, at or near the ER-Golgi Intermediate Compartment [2]. For the human Severe Acute Respiratory Syndrome (SARS) CoV and the Mouse Hepatitis Virus (MHV) CoV, the virion-associated S proteins have been imaged at ~20 angstrom resolution by cryo-electron microscopy [3]. There are about 80 S trimers on each SARS-CoV or MHV particle [4,5]. Each S trimer is held onto viruses by transmembrane anchors located near C-termini, with the vast majority of the S protein mass in extra-virion locations, protruding about 20 nm from the limiting virus membrane of each 80 nm-diameter particle [4] (Figure 1A). Atomic&#8211;resolution structures are known for domains comprising ~20% of the SARS and MHV S proteins. These include the receptor binding domains (RBDs), which are part of the N-terminal &#8220;S1&#8221; half of the primary S sequence (Figure 1B). RBD structures in complex with cognate cell receptors (Figure 1C and 1D) have provided several insights into the initial virus-cell interaction [6,7,8,9]. The other atomically-resolved structures amount to a small portion of the C-terminal half of the primary S sequence (Figure 1E) and reveal a coiled-coil made of three alpha helices termed heptad repeat (HR) 1 along with three chains termed HR2 that are bound onto the outer surfaces of the coiled coil (Figure 1E; note the single HR1-HR2 heteromer for clarification). This complex of three HR1s and three HR2s is termed the six-helix bundle (6-HB) and is a very stable, protease-resistant structure that is part of S proteins after they have denatured, or after they have catalyzed membrane fusion (&#8220;post-fusion&#8221; form) [10,11,12,13]. The structures of HR1 and HR2 in the native or &#8220;pre-fusion&#8221; S conformations are not known. Furthermore, atomic structures of the remaining ~80% of S proteins are not known for any of the S protein conformations. Models depicting the S-mediated membrane fusion event have extended from knowledge of S protein structures and functions. In part, these models are deemed reasonable because the postfusion 6-HB conformations in SARS and MHV S proteins are so strikingly similar to postfusion forms of influenza HA2, paramyxovirus F2, Ebolavirus GP2 and HIV gp41 [16]. In analogy to these more widely-studied and well-understood viral fusion proteins, the CoV S-mediated membrane fusion process is generally viewed as schematized in Figure 2. Following receptor binding, the CoV S proteins undergo conformational change that exposes hydrophobic domains that likely include a so-called fusion peptide (FP), which embeds into the target cell membrane. Primary sequence data, along with measured fusion activities of site-directed S mutants, suggest that the FPs are near or immediately N-terminal to the HR1 ([17,18]; see Figure 1 for proposed FP location). At this stage in which the FP is in the target cell membrane and the transmembrane (TM) span is in the virion membrane, the S proteins are considered to be in &#8220;fusion intermediate&#8221; conformations. What follows is the refolding of the regions between the FPs and the TM spans into the postfusion 6-HB configuration, a likely multi-step process that brings opposing membranes into sufficient proximity for lipid mixing and ultimate coalescence. In addition to FP, HR1, HR2 and TM spans, structural elements participating in these later steps include a hydrophobic tryptophan-rich region at the boundary between TM spans and ectodomains [19,20] and a post-translationally palmitoylated region at the boundary between TM spans and cytoplasmic domains [21,22,23]. At the completion of membrane fusion, FPs and TMs are hypothesized to be adjacent to each other, extending from the same end of the 6-HBs and into the membrane bilayer [13,24,25]. In clear analogy to the majority of viruses promoting membrane fusion in this way &#8220;HR2&#8221; peptides that bind to the HR1 regions of fusion intermediate conformations will potently block CoV cell entry at the membrane fusion stage, most likely by preventing the completion of late-stage refolding into 6-HBs [26,27,28].This understanding of CoV entry has provided a sophisticated basis for continued exploration of the CoVs. Incentives to continue research on CoV entry are clear&#8211;with proven CoV zoonotic potential and extraordinary pathogenicity in humans, a knowledge base of CoV entry offers potential to save lives. There are significant gaps in this knowledge base, and current incentives are to further understand virus-receptor interactions and S protein triggering to catalyze membrane fusion. In adding to the knowledge base, the CoV model system will also bring out new insights of general relevance to virus entry and the membrane fusion. This review covers recent progress in adding to the fund of knowledge on CoV entry.The sections of this review are organized according to the events of the CoV infection cycle, beginning with the biogenesis and secretion of virus particles. This starting point is chosen to highlight the S protein maturation events in virus-producing cells and to emphasize the continuous preparation for membrane fusion that takes place both in virus-producing and virus-target cells. The subsequent sections proceed to virus-receptor interactions and their roles in inducing S protein conformational changes. Next, the importance of endocytosis is discussed, and S proteolytic scissions at the cell surface and within endosomes are then underscored as critical features of CoV cell entry.\n2. S protein Morphogenesis and Initial S Protein Proteolysis\nThe events that ready viruses for cell entry begin in the virus-producing cell. For the CoVs, this beginning stage is in the endoplasmic reticulum of virus-producing cells, where S proteins are synthesized and assembled into trimers. This ER-localized assembly into trimers is a slow process, taking approximately 15 min to generate the greater than 500-kilodalton glycoprotein complexes [1]. Nascent S trimers then transit through the exocytic pathway, embedded by their C-terminal transmembrane domains to vesicle membranes, or to virion membranes if incorporated into viruses at the virus budding sites [2], ultimately residing on the plasma membrane or on secreted virions. It is conceivable that the CoV infection process establishes novel exocytic routes, perhaps with distended organelles able to accommodate secretion of very large S and CoV virion cargo [31]. Exocytic vesicles may even be altered so that the virions and S proteins they house are preserved and properly routed out of the cell. Of note here, the CoV E proteins, small oligomeric transmembrane peptides that accumulate on ER-Golgi intermediate compartment membranes [32] and possess ion channel activities [33], may have a general role in corrupting secretory organelles so that virions are expelled in infectious forms [34]. The virus-producing cell is where the S proteins of some CoVs undergo an early proteolysis (event &#8220;1&#8221; in Figure 3). The trans-Golgi network houses acid pH-activated serine proteases, notably furin [35], that cleave some S proteins at positions immediately C-terminal to multibasic cleavage sites, at a place that we labeled as cleavage site 1 (CS-1) in Figure 1. Scission at CS-1, also designated as S1-S2 cleavage, separates the RBDs and fusion machineries into noncovalently-associated peripheral S1and transmembrane-anchored S2 fragments. That furin proprotein convertases are the responsible proteases has been established by using peptide furin inhibitors, which arrest the S cleavages [36]. Notably, only a subset of CoVs have S proteins with the multibasic cleavage sites, and of these, only a select few have the consensus K/R-X-K/R-K/R motif at the P4-P1 position. Thus, in most CoV infections, S protein populations are left intact or are only partially cleaved in virus-producing cells. For those S proteins that do harbor the cleavage site motifs, site-directed mutation of the multibasic residues to render &#8220;uncleavable&#8221; states does not destroy virus infectivity [37,38]. Experiments involving pharmacologic inhibition of furin complement these evaluations of mutant viruses, and reveal that reduced furin activities greatly reduce S cleavages without diminishing virus infectivities [36].S1-S2 cleavage does, however, increase the potency of CoV S proteins to mediate cell-cell fusions [39,40]. This influence on syncytial developments may explain some of the evolutionary diversity of S1-S2 cleavages. Syncytia may benefit CoVs in certain niches, i.e., during acute infections, by promoting rapid virus dissemination, and this proviral effect may select for the multibasic cleavage motifs. In other niches, i.e., during chronic or smoldering infections, weakened syncytial cells may have greatly compromised virus-producing capacities, and this antiviral effect may select against the same multibasic peptide sequences. Additional, entirely distinct selective forces may target the basic peptide motifs, as the K/R-X-K/R-K/R stretch can mediate virus binding to heparan sulfate, a known receptor for murine CoVs [41]. Again, adsorption to heparan sulfate can be proviral in tissue culture and antiviral in vivo, making for complex selective forces impinging on multibasic S peptide motifs. Exactly how S1-S2 cleavages promote syncytia (cell-cell fusion), often without significant effects on virus entry (virus-cell fusion), are not yet entirely clear. Notably, the S1-S2 cleavage site is distant from any hydrophobic peptides, hundreds of residues from the presumed fusion peptide (Figure 1). This positioning is clearly unlike the analogous cleavage sites on influenza hemagglutinin and HIV gp160, which are adjacent to fusion peptides and thus create hydrophobic N-termini for target membrane insertion. With simple models of S1-S2 cleavage liberating an N-terminal fusion peptide appearing unsatisfactory, there have been hypotheses for additional S cleavages nearer to the fusion peptides. Support for this hypothesis that S is a pre-pro-protein is now in place. As these cleavages to the final fusion-active S likely take place at later times in the continuum of virus assembly, secretion and entry into new cells, they are described in detail in the following sections.\n3. Engaging the CoV Receptors\nAll CoVs, regardless of their pre-primed S protein cleavage status, initiate new infections by binding receptors on virus-target cells. These receptor-binding events, their variabilities amongst the CoVs, and their potential for priming subsequent virus-target cell membrane fusions, are discussed here. A fascinating feature of the CoVs is in the diversity of receptor-binding events. This diversity shows up readily upon inspection of the CoV receptors, which include integral-membrane proteins and sugars that, as a group, have no obvious structural similarities (Table 1). This variation presents interesting questions regarding which receptors are simply mediating CoV adsorption and which might be doing more; i.e., generating fusion-promoting S protein conformational changes and/or directing viruses through particular endocytic routes. Several studies have demonstrated that CoV receptors may not require special endocytic functions beyond S binding: Integral-membrane protein receptors lacking cytoplasmic tails and thus unlikely to regulate virus endocytosis or transduce virus-promoting signals will still provide for virus entry [42,43]. In the case of MHV-CoV and its CEACAM receptors (Table 1), cell entry can be catalyzed by soluble, exogenously-added CEACAM ectodomains [44,45,46], indicating that this receptor can advance entry through S binding, and that a role for the receptor in guiding bound viruses into particular endocytic organelles is not absolutely essential. Diversity of receptor interaction is also readily evident by examining the CoV S proteins. The S proteins are striking in that, as a group, they possess at least two RBDs. One RBD encompasses the S1 N-terminal domain (N-terminal RBD in Figure 1 and Table 1). Recent structural resolution of this NTD from MHV-CoV has revealed a galectin-like beta-sandwich fold, and indeed, the NTDs from related CoV strains (human CoV-OC43 and bovine CoV) are functional lectins and do bind various acetylated sialic acids [8]. Remarkably, the resolved MHV NTD structure lacks a peptide loop necessary for sugar binding, and thus does not operate as a lectin, instead possessing high affinity for the CEACAM proteins. The intriguing indications here are that the N-terminal RBD structures can evolve disparate sugar or protein receptor-binding activities. The findings are valuable in considering CoV receptor switching as it relates to host range and zoonotic infection potential. The other CoV RBD encompasses a region near the middle of S1 and has been termed the C-terminal domain (CTD). The C-terminal RBD structures of human SARS (shown in Figure 1D) and NL63 (not shown) CoVs are completely distinct, a beta sheet architecture in SARS and a beta sandwich in NL63, yet these two CTD RBDs bind to the same region within the ACE2 receptor, termed the &#8220;virus binding hotspot&#8221; [6,8,14,47]. In the SARS S and CTD, there are well-described mutations that change affinities for human, civet cat and bat ACE2 receptors [9,47,48], and as expected, virus affinity for the human ACE2 correlates with human infection and epidemic human to human transmission [47]. The CTD structures on those CoVs that bind APN (or other as yet undetermined receptors) are not yet available, but once determined, may foster analogous identification of S mutations that change affinities for far more diverse receptors. Such studies will promote understanding of how CoV C-terminal RBDs, like the N-terminal RBDs, possess flexible architectures that can evolve disparate receptor-binding activities, again contributing to acquisition of new host ranges and zoonoses. Notably, the existence of two CoV RBDs can make one dispensable. The MHVs, having CEACAM-binding sites on NTDs, readily delete CTD segments during passage in tissue culture [49,50]. These deletion-mutations have profound influences on CoV virulence, greatly decreasing the murine virus neurovirulence [51]. Another interesting example is with the porcine CoVs. Transmissible gastroenteritis virus (TGEV), with two functional RBDs (Table 1), can infect both respiratory and enteric tracts [52]. However, naturally-occurring mutants with substitutions or deletions in N-terminal RBDs lack lectin activities [53] and are strictly limited to the respiratory tract and therefore are attenuated [54,55]. Thus a dual RBD architecture may correlate with in vivo tropism and virulence, likely due to a complexity of multiple receptor interactions that the CoVs employ in the natural animal environment. Diverse receptor interactions precede the CoV S-mediated fusion reactions, and while the binding of relatively low-affinity carbohydrate receptors may not generate fusion-promoting S protein conformations protein receptors that bind S proteins at high affinity clearly do, as evidenced most extensively by studies with MHV. Early seminal findings using MHV demonstrated that alkaline pH increased S fusion activities and S1 release, a readily observed conformational change [29]. Soluble CEACAM receptors were then found to catalyze S1 release [30,56], and biological relevance was subsequently established when soluble receptors were found to support infectious MHV entry into CEACAM-negative cells [44]. More recently, using an MHV2 strain, soluble CEACAM receptors generated SDS-resistant S protein trimers with unique lipophilicities and protease susceptibilities [57]. Thus the MHV model system divulges relatively stable CEACAM receptor-induced S conformations that are quite likely the intermediate structures on the way to membrane fusion (see Figure 2 for hypothetical illustration of receptor-induced generation of fusion intermediate S structures). What is not disclosed by the MHV model system, however, is how CEACAM binding to the NTD RBDs can uncover the fusion machinery in S2. In the primary S sequence, the NTD RBDs are distant from the fusion-inducing peptides. Structural biologists will undoubtedly address this issue most effectively, but at present, intriguing molecular genetic data strongly suggest connections between RBDs and fusion apparati in the context of the native S trimers. One of the first findings in support of such connections was with the identification of a mutation in the fusion domain that destroyed an antibody epitope in the RBD [58]. There have been numerous comparable observations since then. Indeed, MHV evolution, both in vitro and in vivo, frequently fixes substitution mutations in or near the RBDs, and into the fusion regions [59,60,61]. In several instances, these mutations do not change RBD affinities for CEACAMs, but do increase or decrease the tendency for S proteins to undergo CEACAM-induced transitions into fusion-intermediate forms [57,62], suggesting that one selective pressure relates to MHV responsiveness to the CEACAM receptor &#8220;trigger&#8221;. Finally, the fact that these mutations are frequently associated with reduced virus virulence creates the additional suggestion that any complete understanding of CoV disease requires appreciation of how receptors operate as fusion triggers. Relative to CEACAM and its effect on MHVs, the roles of ACE2 and APN receptors in triggering structural transitions of human CoVs is less clear. It is, for example, uncertain whether ACE2 binding to SARS or NL63 S proteins dissociates RBDs from fusion modules and exposes hydrophobic &#8220;fusion peptides&#8221;. Compelling data do, however, suggest that ACE2-bound S proteins are set up for proteolytic scissions that activate membrane fusion, as described in the following sections. The human CoV receptors, therefore, may be similar to the murine CEACAM CoV receptors in that their binding energies drive S proteins into intermediate forms with exposed proteolytic substrate sites. Host proteases may then ultimately liberate the fusion machineries and allow them to operate in CoV entry.Finally, there is the question of whether virus entry and the receptor-induced conformational changes leading to membrane fusion take place at the plasma membrane or at a later time, after viruses have entered into endosomes. This question is explored in the following section.\n4. Endocytosis Following Coronavirus: Cell Binding\nEndocytosis is the principal cell entry route for most viruses [63]. Receptor-mediated endocytosis is an advantageous entry route as it can allow the virus (in endosomes) to pass through a barrier of cortical actin and traffic deeply into the cell before fusing and expelling its genome to cytosol [64]. The endosome also bathes the virus in an acidic, proteolytic environment that is frequently necessary to activate viral fusion. It appears that the majority of CoVs enter cells via endocytosis. However, a subset of CoVs have been thought to enter in a more straightforward direct fusion with the plasma membrane, making the CoVs useful for dissecting various cell entry routes.The idea that some CoVs do not endocytose but rather enter by fusion directly at the plasma membrane comes from several findings, including the fact that several CoVs readily induce syncytia, i.e., plasma membrane-localized cell-cell fusions. In those studies where pH effects were investigated, syncytial developments increased in slightly alkaline media [56], suggesting no need for endosomal acid exposures to activate fusions. Similarly, some CoVs, notably the JHM strain of MHV, are impervious to inhibitors of endosomal acidification [65]. Finally, when incubated with receptor-bearing cells at extraordinarily high input multiplicities, some CoVs can induce immediate cell-cell fusions, often called &#8220;fusion-from-without&#8221; [66]. But none of these observations necessarily confirm that CoVs naturally enter cells via fusion at the plasma membrane. First, it may be inappropriate to equate syncytial formation (cell-cell fusion) with virus entry (virus-cell fusion), because these two processes have distinct requirements, as revealed by S mutations that abolish syncytial formation without affecting virus infectivity, as discussed later in this review. Second, increased syncytial activity at basic pH may reflect the pH optima of the cell-surface trypsin-like proteases that create fusion-active S fragments, and may primarily apply to fusions mediated by S proteins that are not associated with virions. Third, the resistance of some CoVs to endosome neutralization can indicate fusion in early non-acidic endosomes, but cannot be used to conclude that virus-cell fusion takes place at the plasma membrane. Indeed there are several reports, particularly in HIV research, that viruses acquiring membrane fusion competence at neutral pH may still utilize endosomal networks during entry [67]. This leaves the &#8220;fusion from without&#8221; phenomena, clearly plasma membrane-based virus-cell fusion events, but perhaps occurring with very low frequencies and thus only observed in artificial experimental conditions in which viruses are applied to cells at extremely high doses. That the CoVs generally utilize cellular endocytic machinery during entry is also supported by studies aimed at dissecting CoV entry pathways [68]. Many of these investigations assess the efficiencies of CoV infections after specifically arresting clathrin, or caveolar, or cholesterol-dependent endocytosis with pharmacologic agents and overexpressed dominant-negative proteins. Valuable results have indicated various CoV entry pathways; caveolar endocytosis for human 229E-CoV [69], clathrin-mediated endocytosis for SARS-CoV [42], cholesterol-raft entry for MHV [70], clathrin and caveolar&#8211;independent endocytosis for feline CoV [71]. However, complete blockade of CoV entry is rarely achieved by suppressing any single endocytic pathway, suggesting that multiple cell entry routes can be utilized by the CoVs. That CoVs can resourcefully invade cells in alternate routes is exemplified by successful SARS-CoV entry when both clathrin and caveolar endocytosis is blocked [72]. Similarly, SARS-CoV entry, an ACE2 receptor-mediated process that typically traffics viruses into acidic organelles [69], is entirely distinct when mediated by antibodies and Fc-gamma-receptor II. In this so-called antibody-dependent enhanced entry, infection is entirely unaffected by the endosome -neutralizing agents and protease inhibitors that arrest the ACE2-mediated entry process [73]. In summary, different CoV endocytic entry routes can culminate in successful infections. In the endosomes, CoVs are bathed in an acidic and often proteolytic environment. The S proteins from the majority of CoV strains clearly require acid exposures to acquire membrane fusion competence. However, it is unclear whether protons are the endosomal factor(s) triggering fusion per se, as acidification itself does not enhance S-mediated fusions [29]. Acid pH-activated endosomal proteases may be the more likely triggers [74,75,76]. Hence an interesting question arises as to whether entry depends on acidic endosomal conditions solely because the fusion-activating protease(s) are themselves acid pH-dependent. This is a question considered in studies of filovirus entry [77] and the CoVs present an additional study model here because the effects of protonation on CoV S fusion structures can be dynamically evaluated by magnetic resonance [78]. The CoV model system appears poised to contribute to a general understanding of proton-triggers of virus entry. \n5. Virus Entry, S Proteolysis and Membrane Fusion\nThe initial and most obvious S proteolysis takes place in virus-producing cells, at the &#8220;CS-1&#8221; position (Figure 1), to generate secreted viruses with peripheral (S1) and transmembrane (S2) fragments (event &#8220;1&#8221; in Figure 3). As stated in the previous section on S morphogenesis, these S1-S2 scissions are not observed in all CoVs. For those CoVs with uncleaved S proteins, the relevant scissions take place during virus entry (events &#8220;3&#8221; and &#8220;4&#8221; in Figure 3). Simmons et al. [76] and Matsuyama et al. [79] made the initial seminal discovery that cathepsin L inhibitors block SARS S mediated virus entry and that trypsin proteolysis bypasses this inhibition, making it clear that cell entry via SARS S proteins depends on a proteolytic cleavage by cathepsin L. These were enlightening results that arose nearly concurrently with the discovery that Ebolavirus entry requires cathepsin cleavage of its fusion glycoproteins [80], thus establishing that endosomal proteases can promote the entry of different enveloped viruses. Subsequent questions regarding the S cleavages were addressed biochemically, and it was found that SARS S ectodomains were cleaved by cathepsin L at the S1-S2 position (CS1 in Figure 1) [81]. This made it appear that, depending on the CoV strain, S cleavage at CS1 either takes place early in the infection cycle, by furin-like serine proteases in the trans-Golgi network of virus-producing cells, or later in the cycle, by cathepsin L cysteine proteases in the endosomes of virus-target cells, with similar ultimate outcomes. Supporting this view, precleaving the SARS S by inserting a consensus furin cleavage site 1 freed the virus from its dependence on endosomal acidification [82]. More recent findings, however, have added complexity to these S proteolytic cleavages. An additional mutibasic peptide motif (at a position we designated in Figure 1 as &#8220;CS2&#8221;) was recognized in infectious bronchitis CoV (IBV) S, and shown via mutagenesis to be essential for IBV cell entry [83]. The homologous position in SARS S was also found to be highly relevant, as its mutation into a consensus furin cleavage motif allowed for SARS S&#8211;mediated entry without the need for exogenous trypsin proteases [84]. This second cleavage site (CS2) has since been biochemically confirmed via identification of the shorter C-terminal S fragment, both with SARS S proteins [85] and with MHV S proteins [57]. Although research on S proteolytic cascades is ongoing, the present findings are consistent with a model in which S proteins are cleaved multiply, likely first at the CS1 (S1-S2) position, and then at CS2 positions adjacent to the FP, to generate fusion intermediate structures of the type depicted in Figure 2. The CS2 proteolytic removal of sequences that are N-terminal to the FP may permit the rapid completion of the S refolding events that generate membrane fusion and the 6-HB configuration.There is also the question of precisely when the S proteins are available for proteolysis. The weight of current evidence argues that the target cell, not the free extracellular space, is the site of S proteolysis. For example, cell-free SARS-CoV virions were rendered non-infectious by trypsin pre-treatment of free virions [72]. However, the same trypsin exposures enhanced infectivities of cell-bound viruses [76,79] (event &#8220;3&#8221; in Figure 3). These findings suggest that it is the cell receptor-associated S proteins that are in unique protease-responsive structural forms. While these experiments used a protease that is not encountered by incoming respiratory CoVs such as SARS-CoV, the findings still do imply that S conformations change significantly upon receptor interaction or endocytosis, and in turn illuminate the importance of S conformation and subcellular location to the effects of S proteolysis. That exogenously added trypsin could increase the infectivities of cell receptor-associated SARS-CoV suggests that cell-bound ectoprotease(s) might be the natural cofactors for CoV entry. Indeed, CoV S can be primed for fusion by recently described cell-surface proteases known as Type II Transmembrane Serine Proteases (TTSPs) [85,86,87,88,89]. The TTSPs may reside in plasma membrane microdomains, although this possibility is not yet validated, and they may be available to incoming viruses at the receptor-interaction stage, to cleave at the CS1 and/or CS2 positions. The prototype TTSP, enteropeptidase, was discovered nearly a century ago, and the most recent decades of research have revealed at least 20 additional family members [90,91]. Family member variations are in the domains between C-terminal protease and N-terminal transmembrane anchors. The roles of these domains in viral glycoprotein cleavage, if any, are unknown, but they may dictate substrate specificities by influencing protease subcellular localization or association with other membrane proteins [90]. Only a subset of TTSPs have been evaluated as mediators of virus entry. Principal amongst these evaluated TTSPs is transmembrane protease, serine 2 (TMPRSS2). First described in 1997 [92], a role for TMPRSS2 in normal physiology remains elusive, largely because TMPRSS2 knockout mice develop normally [93]. But TMPRSS2 may have a fundamental organism-wide role, as it is expressed in tissues such as the heart, liver, prostate, intestines, and most notably, the lung [92]. Abundance in the lung has prompted screening of respiratory viruses for TMPRSS2 activation of their glycoproteins. TMPRSS2 can cleave the hemagglutinins (HAs) of influenza viruses, leading to their increased replication, including the highly pathogenic 1918 strain [94,95]. TMPRSS2 also cleaves the membrane glycoprotein of SARS-CoV. Matsuyama et al. discovered that increased levels of TMPRSS2 resulted in more cell death and increased SARS-CoV viral replication in vitro [86]. Shulla et al. confirmed and expanded this finding by noting that TMPRSS2 specifically increases SARS-CoV entry using pseudotyped reporter assays [88]. TMPRSS2 cleavage may also contribute to immune evasion. By cleaving S from infected-cell surfaces, TMPRSS2 releases S fragments that bind and incapacitate neutralizing antibodies [87]. Hence the in vivo infection process may be heavily influenced by TMPRSS2 and related family members, both at virus entry and release, influencing pathogenesis and immune response. Another TTSP, Human Airway Trypsin-like Protease (HAT or TMPRSS11d), has brought out enlightening details concerning member-specific proteolytic properties. In the context of influenza HA cleavage, HAT has a broader cleavage capacity than TMPRSS2, proteolyzing HA both in virus-producing cells and in progeny viruses bound to target cell receptors [96]. Thus HAT, not TMPRSS2, is the more relevant protease operating on influenza at the virus entry stage. In the context of SARS-CoV and S cleavage, HAT again exhibits a broader cleavage capacity than TMPRSS2, making it so that HAT can cleave and enhance S-mediated virus entry either in virus-producing cells or on the surface of virus-target cells [89]. However, overexpressed TMPRSS2 bypasses the requirement for endosomal acidification and therefore cathepsin activation [86,88], but HAT does not similarly replace cathepsins in SARS-CoV entry [89]. Thus a further dissection of the various TTSP substrate specificities will be necessary to precisely identify those most relevant to virus infection, and efforts in this regard are continuing. For example, the first paper to examine TTSPs in the context of SARS entry found that TMPRSS11a was capable of slightly enhancing SARS S bearing pseudoparticles [85]. Subsequent findings indicated that, while TMPRSS11a was capable of modestly increasing SARS entry at low levels of the protease, TMPRSS2 was a more potent activator of entry [88]. Most recently, various TTSPs including TMPRSS3, TMPRSS4, TMPRSS6, and Hepsin, have been evaluated, yet none have exceeded TMPRSS2 in augmenting SARS-CoV entry [87,89]. Other candidate TTSPs worth testing in SARS-CoV entry assays are MSPL and TMPRSS13, as they have been found to cleave certain influenza HAs [97]. While the TTSPs may be the most relevant proteases in natural CoV infections, they are clearly dispensable in several tissue culture settings. This is because cathepsins, specifically cathepsin L, will proteolytically activate SARS CoV S proteins following virus endocytosis (event &#8220;4&#8221; in Figure 3) Multiple proteases with possibly redundant virus entry functions make it difficult to discern which proteases are necessary for viral entry. This difficulty is perhaps most recognized by the fact that the presumed proteolytic activation of another human CoV, NL63, is entirely unclear. NL63 S-mediated entry was not affected by preventing endosomal acidification or by cathepsin inhibitors [98]. While NL63 is similar to SARS in that it binds to the same receptor, ACE2, the protease responsible for NL63 S cleavage remains a mystery.\n6. Therapeutics and Inhibition of CoV Entry\nCurrently, vaccination is the best clinical approach to eradicating viruses. Experimental SARS-CoV vaccine immunogens include inactivated whole viruses, entire S trimers, and RBDs (reviewed in [99]). All formulations elicit protective antibodies that prevent virus entry, some protecting against lethal SARS-CoV challenge, with most antibodies operating sterically to block virus-receptor engagement [100,101]. However, protective antibodies can operate variably and in novel ways, as shown by a human monoclonal antibody directed against the trypsin cleavage site in S that reduced viral titers and pathology associated with SARS-CoV infection [102]. Interestingly, despite binding to a proteolytic substrate site, this antibody did not prevent S cleavage but rather prevented membrane fusion by an unknown mechanism [102]. The antibody was created in transgenic mice that produce human immunoglobulins, representing an interesting new production method for passive human immunization. These new approaches signal subunit vaccination and therapeutic antibodies as attractive future clinical options, but the methods must be evaluated cautiously, as anti-SARS antibodies can permit virus entry via Fc&#947; receptors, bypassing the ACE2-dependent entry route and actually enhancing disease [73]. Attractive natural-product anti-CoV agents include lectins that bind avidly to the unprocessed, high-mannose carbohydrates on CoV S proteins. One such lectin is griffithsin, an oligomer with six high-affinity mannose binding sites. Griffithsin is known to block HIV infection via envelope protein binding [103]. Griffithsin was found to have analogous anti-SARS activity, preventing SARS-CoV infection and pathology both in vitro and in a mouse disease model, without incidental cytotoxity [104]. Other potential anti-human CoV lectins include the mannose-binding lectin, which arrests SARS-CoV entry [105] and the plant lectins Galanthus nivalis agglutinin, Hippeastrum hybrid agglutinin, and Urtica dioica agglutinin, which, to date, have only been evaluated for anti-MHV activity [106]. Notably, most of these lectins do not affect binding of S to the receptor, but rather seem to inhibit entry at a later stage [105,106]. Several of the proteases described in this paper are potential antiviral therapeutic targets Indeed, furin-inhibiting peptides arrest cell-cell fusions mediated by coronaviruses MHV and IBV (infectious bronchitis virus) [36,83]. A novel druglike cathepsin L inhibitor, identified by high-throughput screening, prevents Ebola and SARS-CoV cell entry [107]. TTSPs also promise to be antiviral drug targets. With their proteolytically active sites exposed to the extracellular milieu, hydrophilic inhibitors that remain excluded from cells may be suitable, thus reducing medicinal chemistry and toxicity problems. Transient, pharmacologic TMPRSS2 suppression may have no health consequences, given that the TMPRSS2-null mouse is phenotypically normal. Targeting other cellular proteases, such as furins or cathepsins, may have unintended or toxic side effects due to their importance in normal physiology. Finally, targeting host proteins is a sensible adjunct to the design of inhibitors targeted toward highly-mutable virus protein targets. In addition to preventing CoV entry, inhibiting TTSPs and other proteases would also prevent the maturation of other viral glycoproteins, blocking entry of a variety of viruses. An example of a potentially multi or pan-viral protease inhibitor is the human cathepsin L inihibitor tetrahydroquinoline oxocarbazate that blocks both SARS and Ebola glycoprotein mediated entry in vitro [107]. While animal studies have yet to be published on that cathepsin inhibitor, Bahgat et al. recently reported that a cocktail of serine protease inhibitors reduced influenza infection and subsequent disease in mice [108]. Additionally, more historic reports used single agent serine protease inhibitors such as camostat, leupeptin and aprotinin to decrease influenza infection both in mice and in humans [109,110]. Zihnov et al. reported that aerosol delivery of aprotinin shortened influenza symptomatology in humans [110]. It is possible that the antiviral aprotinin target was TMPRSS2, given that TMPRSS2 is abundant in lungs and that aprotinin cannot penetrate the plasma membrane. In support of this possibility, TMPRSS2 knockdown with peptide-conjugated morpholino oligomers, which are stable antisense oligonucleotides, rendered TMPRSS2 positive human airway cells resistant to influenza virus replication [111].In the end, inhibiting human CoVs may require a multi-target approach. Blocking multiple proteases along the entry pathway might be combined with lectins, and also with peptide-based fusion inhibitors. Fusion of the viral membrane with the target cell membrane can be arrested by peptides mimicking the S heptad repeat region 2 (HR2), which bind and inactivate fusion intermediate S forms [26], as mentioned earlier in this review. Different HR2 peptides have been investigated with IC50s in the nanomolar to micromolar range [26,112]; these high affinities and specificities are similar to the HIV HR2 peptide, enfuvirtide, currently used in humans [113]. Further refinement of the HR2 peptides, and development of peptidomimetics, might enhance antiviral activities and make them even more suitable for anti-CoV treatment [27]. HR2 peptides cover a broad viral target and thus impose barriers against virus mutational escape routes. Mutations in HR1 that both confer HR2 resistance and preserve virus viability are rare [114]. While HR2 peptides could be extremely potent entry inhibitors, our understanding of SARS entry makes multiple therapeutics targeting viral binding and fusion feasible and most likely of great value. Certainly, a multifaceted approach to SARS entry inhibition would be the most likely to succeed. \n7. Future Directions\nThe CoV S peplomers directing cell entry are the principal viral components communicating with diverse extracellular environments. These arrays of extracellular selective pressures fix diversity into the genetically-plastic CoV S proteins. Novel S structures allow the CoVs to expand into different ecological niches; zoonotic emergence of the SARS-CoV being the prime example of this emergence and its consequences. Understanding S variation, structure, and entry mechanisms are therefore central to combating potential epidemic CoV transmissions.The natural diversity of the CoV S proteins provides insights into evolution and mechanism of cell entry. For example, amongst the CoVs, there are central distinctions in RBDs and fusion&#8211;activating triggers. These differences can dramatically influence pathogenicity by redirecting cell tropism, viral entry routes and rates. The CoVs are facile models to correlate variations in cell entry with disease potential. The CoVs are also excellent tools to reveal new host cell components facilitating entry. Of relevance to disease are the possible species and organ-specific variations in CoV receptors and S-activating proteases. One of the next key steps in CoV research will be to move toward in vivo dissection of entry mediators and relate disease manifestations to their presence and abundance in distinct tissues and species.Increasingly sophisticated in vivo CoV models will also be useful in evaluating new antiviral strategies. Notably, the infection models and the prototype antiviral agents promise to deliver therapies that will both guard against potentially serious zoonotic CoVs and reveal more broad&#8211;spectrum antivirals. For example, TTSP inhibitors may be suitable therapeutics for a variety of respiratory viruses that rely on proteolysis for cell entry. CoV research will continue to reveal features of virus operating mechanisms and clinically applicable antiviral strategies for the foreseeable future.",
        "pmid": "22590686",
        "title": "Ready, Set, Fuse! The Coronavirus Spike Protein and Acquisition of Fusion Competence",
        "journal_title": "Viruses",
        "authors": "Taylor Heald-Sargent;Tom Gallagher",
        "doi": "10.3390/v4040557"
    },
    {
        "keywords": "coronavirus;ER stress;unfolded protein response;p38;JNK;eIF2\u03b1;PKR;PERK;GADD34/PP1;nsp1;translational control",
        "abstract": "Similar to other viruses, coronavirus infection triggers cellular stress responses in infected host cells. The close association of coronavirus replication with the endoplasmic reticulum (ER) results in the ER stress responses, which impose a challenge to the viruses. Viruses, in turn, have come up with various mechanisms to block or subvert these responses. One of the ER stress responses is inhibition of the global protein synthesis to reduce the amount of unfolded proteins inside the ER lumen. Viruses have evolved the capacity to overcome the protein translation shutoff to ensure viral protein production. Here, we review the strategies exploited by coronavirus to modulate cellular stress response pathways. The involvement of coronavirus-induced stress responses and translational control in viral pathogenesis will also be briefly discussed.",
        "body": "1. ER Stress Responses Regulated by Coronavirus and Its Implication in Pathogenesis\n\n1.1. The Integrated Signaling Network of the Unfolded Protein Response (UPR)\nInside a eukaryotic cell, most of the transmembrane and secreted proteins are translated, modified, and folded in the ER. The amount of proteins in the ER can fluctuate substantially when a cell is undergoing physiological changes or when it is affected by various environmental stimulations. If the protein influx overloads the protein processing machinery, unfolded/misfolded proteins will accumulate inside the ER and result in ER stress. In order to return to homeostatis, cells have evolved UPR [1], which is composed of three pathways. These pathways are initiated by three ER sensor proteins located in the ER: PKR-like ER protein kinase [1], activating transcriptional factor-6 (ATF6) and inositol-requiring protein-1 (IRE1) (Figure 1). All of them are single-pass transmembrane proteins, consisting of a luminal domain that recognizes unfolded/misfolded protein inside the ER, and a cytosolic domain that ultimately relays the signal to the nucleus and switches on a specific set of downstream genes.Under ER stress, oligomerization and activation PERK mediates the phosphorylation of eIF2&#945;, resulting in the shutdown of global translation [2,3]. The PERK-dependent inhibition of protein synthesis limits nascent protein transport to ER lumen, thereby attenuating the protein accumulation in ER. Interestingly, some proteins are preferetially translated when eIF2&#945; is phosphorylated. One example is ATF4 [4], a transcription factor that control the expression of genes involved in amino acid metabolism and transport and redox chemistry. GADD34 is one of downstream genes triggered by ATF4. As a regulator subunit, GADD34 helps PP1 to dephosphorylate eIF2&#945;, thereby limiting PERK signaling as a negative feedback loop [5]. PERK signaling will be considered as part of translational control and will be discussed in Section 3.As for the IRE1 branch of UPR, activation of IRE1 by auto-phosphorylation activates its cytosolic RNase domain, which mediates a unique splicing event that removes an intron from the transcript of X-box protein 1 (XBP1) [6]. The spliced form of XBP1 protein (XBP1s) is then translated and imported to the cell nucleus, thereby activating the expression of UPR genes, which encode various ER protein chaperones as well as components of the ER-associated degradation (ERAD) pathway [6]. Moreover, IRE1 is also known to catalyze non-specific degradation of mRNAs associated with the ER, a phenomenon dubbed as IRE1-dependent RNA decay (RIDD) that effectively reduces the translational burden of the ER [7]. In spite of these pro-survival activities, prolonged activation of IRE1 can also activate c-Jun N-terminal kinase (JNK) and promote caspase-12 dependent apoptosis [8,9].In terms of ATF6, increasing amounts of unfolded proteins activate the protein and lead to its translocation from the ER to the Golgi, in which the protein is sequentially cleaved by proteases. The cytosolic domain of ATF6 is then released and transported into the nucleus[10], where it induces the expression of UPR genes, such as some ER protein chaperones (calreticulin, glucose regulated protein 78 kDa (GRP78) and GRP94), some ERAD proteins, as well as ER-resident enzymes (protein disulfide isomerase) [11].With the help of several feedback mechanisms, the three UPR pathways mentioned above actually constitute an inter-related signaling network [1]. For example, XBP1 mRNA from the IRE1 branch has been shown to be induced by PERK and ATF6 when cells are under ER stress [6,12]. Moreover, both PERK and PKR could be inhibited by P58IPK, which is a downstream gene transcriptionally induced by XBP1s [13,14]. Finally, the expression and activation of ATF6 could be enhanced by PERK, while ATF6 may induce protein disulfide isomerase A6, which promotes the degradation of IRE1 [15,16,17]. Hence, the three branches of UPR should be recognized as a closely interrelated and intricately regulated signaling network.\n1.2. The Possible Mechanisms of Coronavirus-Induced ER Stress Responses\nIncreased expression of GRP78, GRP94 and other ER stress related genes has been determined in cells infected with various coronaviruses. These include mouse hepatitis virus (MHV), severe acute respiratory syndrome coronavirus (SARS-coronavirus) and infectious bronchitis virus (IBV) [18,19,20]. Activation of ER stress response can be also detected in cell overexpressing the SARS-coronavirus spike protein [21,22], protein 3a [23], protein 6 [24], protein 8a, and protein 8b [25].Although not completely illustrated, it is proposed that coronavirus induces ER stress via three potential processes [26]. First, translation, folding and modification of large amounts of coronavirus structural proteins, in particular the heavily N-link glycosylated spike protein, significantly increases the burden of the ER in infected cells. Indeed, overexpression of the spike protein from SARS-coronavirus [21], MHV [20] or IBV can all activate UPR. Maturation of the spike protein could further exhaust chaperones inside the ER lumen such as calnexin, which is known to physically bind to the SARS-coronavirus spike protein to facilitate its proper folding [27]. Second, in order to assemble the replication/transcription complex, coronavirus induces the rearrangement of ER membrane into double membrane vesicles (DMVs), zippered ER or ER spherules [28,29]. DMVs formation could be observed in cells overexpressing the SARS-coronavirus non-structural protein nsp3, nsp4 and nsp6, suggesting their involvement in coronavirus-induced membrane remodeling [30]. Moreover, DMVs formed in cells infected with SARS-coronavirus are derived from a modified reticulovesicular network connecting to the ER, as determined by high resolution electron tomography [31]. Also, MHV may hijack vesicles originated from the ER and use the membrane for DMV formation [32]. Finally, morphogenesis and budding of mature virus particles deplete the membrane component in the ER. Assembly and budding of coronavirus virions occur in the ER-Golgi intermediate compartment (ERGIC), a membrane system that is structurally and functionally extended from the ER [33,34]. It has been well established that depletion of ER lipid components (such as phosphatidylcholine) alters the morphology of ER and pertubs trafficking of protein cargo in the Golgi [35].Taken together, current evidence demonstrates that coronavirus infection induces ER stress and triggers UPR in general. It is apparent that coronavirus might subvert or utilize certain aspects of the UPR to benefit its own replication and pathogenesis.\n1.3. Coronavirus-Induced UPR and Its Implication in Pathogenesis\nThe IRE1-XBP1 pathway is activated by MHV [36] and IBV infection [37], which may be caused by the accumulation of the spike protein in ER lumen [20]. However, either SARS-coronavirus infection or SARS-coronavirus spike protein overexpression does not lead to XBP1 splicing [20], suggesting the modulation of UPR branches differs for different coronaviruses. Interestingly, when the E gene is deleted from the SARS-coronavirus genome, the resulting mutant virus (rSARS-coronavirus-&#916;E) is significantly attenuated, but significantly activates XBP1 splicingand up-regulates other cellular stress genes, leading to increased apoptosis [38]. The study suggests that SARS-coronavirus E protein suppresses the IRE1-XBP1 pathway activation and inhibits apoptosis induction, although the mechanism remains unknown.As for MHV, in spite of the observed XBP1 mRNA splicing, neither the spliced XBP1 protein (XBP1s) nor the upregulation of downstream UPR genes could be detected, presumably resulted from the sustained translational suppression due to eIF2&#945; phosphorylation [36]. Conversely, significant upregulation of UPR genes (such as ERdj4 and p58IPK) can be detected in cells infected with IBV, indicating full activation of the IRE1-XBP1 pathway [37]. Moreover, RNA interference of IRE1 and inhibition of XBP1s drastically potentiate IBV-induced apoptosis, whereas overexpression of IRE1 and XBP1s promotes cell survival, pointing to an anti-apoptotic nature of the IRE1-XBP1 pathway during IBV infection [37]. Activation of a pro-survival response at the late stage of infection could benefit the virus, by giving more time for virus particle assembly and release before the infected cells are disintegrated. However, knockdown of IRE1 does not significantly affect the production of infectious IBV in the supernatant [37]. Therefore, the physiological significance of IRE1 during coronavirus infection needs to be determined with further experiments and with appropriate in vivo models.Apart from apoptosis, activation of IRE1 may affect other cellular events that modulate the pathogenesis of coronavirus infection. For example, IRE1 can also modulate pro-inflammatory response by activation of nuclear factor kappa-light-chain-enhancer of activated B cells (NF-&#954;B) and induction of cytokines such as interleukin 6 (IL-6) and IL-8 [39,40]. On the other hand, XBP1 has been demonstrated to directly or indirectly modulate the expression of various cytokines and type-I interferons (IFNs) [41,42,43]. Unpublished data from this group have also implicated the involvement of IRE1 and XBP1 in the transcription of IL-8 and IFN-&#946;during IBV infection. Further investigation needs to be carried out for other coronaviruses to understand the detailed mechanisms and potential counter measures implemented by coronaviruses.Whether or not coronavirus activates the ATF6 branch of UPR during coronavirus infection has not been fully investigated. Proteolytic cleavage of ATF6 into its active form cannot be detected in cells infected with SARS-coronavirus [38]. Similarly, ATF6 reporter constructs co-transfected with the SARS-coronavirus spike protein showed minimal reporter production [21]. In contrast, ATF6 cleavage is observed by MHV infection, although both the full-length and the cleaved form of ATF6 decreased significantly at late stage of infection due to sustained translational attenuation [36]. Subsequently, neither the induction of downstream UPR genes nor the activation of the ERSE reporter construct could be detected. Interestingly, the SARS-coronavirus 8ab protein binds to ATF6 in vitro and promotes its cleavage in the co-transfected cells [25]. However, further experiments using recombinant viruses are required.Coronavirus also manipulate PERK activity and the level of phosphor-eIF2&#945; to control protein synthesis, such as IBV [18,44], MHV [45], SARS-coronavirus [23,46,47]. The regulation of PERK-eIF2&#945; pathway by various coronaviruses will be discussed in detail in Section 3. The regulation of ER stress response pathways by coronavirus is summarized in Figure 1.\n2. Activation and subversion of p38 and JNK signaling pathways by coronavirus infection\n\n2.1. The Signaling Pathways of Stress-Activated Protein Kinase p38 and JNK\nThe mitogen activated (MAP) kinases are evolutionarily conserved protein kinases that regulate a diversity of critical cellular signaling pathways, such as cell division, differentiation, autophagy, apoptosis, innate immunity and pro-inflammatory response [48]. So far, four subgroups of MAP kinases have been identified in metazoans, namely the extracellular regulated kinase 1/2 (ERK1/2), ERK5, the p38 MAP kinase and the c-Jun N-terminal kinases (JNK) [49,50]. Among them, the ERK1/2 signaling is triggered by growth factors and mitogens, whereas the p38 and JNK are mainly responsive to cellular stress, such as DNA damage, ionizing radiation and protein synthesis inhibition [49].MAP kinases are phosphorylated by the upstream MAP kinase kinases (MAPKKs), which are in turn phosphorylated by upstream kinases in response to different cellular or environmental stimulations [51]. In particular, p38 is activated by MKK3, MKK4 [52] or MKK6 [53], whereas JNK can be phosphorylated by MKK4 or MKK7 [54]. Dual phosphorylation of Thr and Tyr residues in the conserved TxY motif (Thr-Gly-Tyr for p38 and Thr-Pro-Tyr for JNK) is essential for the complete activation of MAP kinases [49].Upon activation, p38 translocates into the nucleus and activates multiple effector proteins, such as MAPK-Activated Protein Kinase-2 (MAPKAPK2), which in turn activates critical transcription factors such as cAMP Response Element-Binding protein (CREB) and Activating Transcription Factor 1 (ATF1) [55]. Moreover, p38 can also directly phosphorylate other important transcription factors such as p53 and CHOP [56,57].As for JNK, activated JNK phosphorylates c-Jun and other downstream substrates, enhancing their transcription activity [58]. Activated c-Jun dimerizes with other proteins such as cellular FBJ murine osteosarcoma (c-Fos) to form the activator protein 1 complex, thereby inducing genes harboring the 12-O-tetradecanoylphorbol-13-acetate (TPA) response element [59].\n2.2. Induction and Subversion of the p38 Pathway by Coronavirus Infection\nPhosphorylation of p38 is detected in cells infected with several coronaviruses including MHV [60], SARS-coronavirus [61], feline coronavirus (FCoV) [62], IBV [63] and transmissible gastroenteritis coronavirus (TGEV) [64]. Moreover, the p38 pathway is also activated in cells overexpressing the SARS-coronavirus accessory protein 3a [65] and 7a [66], although the physiological significance during an actual infection has not been fully examined. Coronavirus-induced p38 activation is likely mediated by the upstream kinase MKK3/6, which is phosphorylated in cells infected with MHV [67] and IBV [63].Multiple reports have suggested the critical role of p38 in the inflammatory process during coronavirus infection. For example, inhibition of p38 significantly reduces the transcription and secretion of IL-6 in MHV-infected cells [67]. Similarly, the induction of IL-6 and IL-8 is dependent on p38 activation in IBV-infected cells [63]. Moreover, production of tumor necrosis factor alpha (TNF-&#945;) and IL-1&#946; in FIPV-infected cells is significantly inhibited by p38 inhibitor [62]. Finally, p38 has been demonstrated to induce the expression of macrophage prothrombinase fibrinogen-like protein 2 (Fgl2), a protein crucial for the pathogenesis of fulminant hepatitis in MHV-3-infected mice [60,68].Notably, coronaviruses have developed various mechanisms to subvert the activation of the p38 pathway. For instance, IBV has been shown to induce the transcription of dual-specificity phosphatase 1 (DUSP1), which can dephosphorylate the active p38 and suppress the production of excessive pro-inflammatory cytokines [63]. In contrast, the SARS-coronavirus E protein seems to activate p38 and induces inflammation [69]. Specifically, the PDZ-binding motif (PBM) in the SARS-coronavirus E protein can interact with the cellular protein syntenin, resulting in its redistribution from the nucleus to the cytoplasm and activation of the p38 pathway [69]. In fact, production of inflammatory cytokines is significantly reduced in syntenin-knockdown cells, or in mice infected with PBM-deleted E recombinant virus compared with mice infected with the wild type virus [69]. Therefore, different coronaviruses may use distinct mechanisms to modulate the p38 pathway, so as to establish the cellular environment suitable for viral replication and spreading.\n2.3. Activation of JNK during Coronavirus Infection\nCompared with p38, coronavirus-induced activation of the JNK pathway has been less well characterized. JNK phosphorylation has been determined in cells infected with MHV [67], SARS-coronavirus [67] and IBV [37]. Activation of the JNK pathway has also been demonstrated in cells overexpressing the N protein, accessory protein 3a, 3b or 7a of SARS-coronavirus [70,71,72].In a study using specific inhibitors, Mizutani et al have shown that persistent SARS-coronavirus infection requires the intact signaling of JNK and Akt in Vero E6 cells, suggesting a pro-survival function for these kinases [73]. Phosphorylation of JNK and Akt is likely induced by the SARS-coronavirus N protein, and the anti-apoptotic Bcl2 and Bcl-xL proteins may also contribute to the persistent infection [74]. In contrast, JNK signaling has been found to facilitate apoptotic cell death in IBV-infected cells [37]. Therefore, it is possible that JNK serves different functions in cells infected with different coronaviruses, and presumably at different stages of infection.In terms of its involvement in proinflammatory response, induction of TNF-&#945; and IL-6 by MHV-A59 infection in primary mouse astrocytes depends on JNK, but not NF-&#954;B or other MAP kinases [75]. The SARS-coronavirus spike protein also activates the protein kinase C epsilon, which induces JNK phosphorylation in a calcium-independent manner. Moreover, induction of IL-8, activation of CREB and the transcription of cyclooxygenase-2 (COX-2) gene in cells transfected with SARS-coronavirus spike also requires JNK [76,77].The modulation of MAPK pathways by coronavirus is summarized in Figure 2.\n3. Translational Control by Coronavirus and Its Implication in Pathogenesis\n\n3.1. Viral Protein Translation of Coronavirus\nCoronavirus replicate entirely within the cytoplasm of their host cells, where they produced five to nine genomic mRNAs [41]. All mRNAs contain a common 5&#8242;-leader sequence (65&#8211;90 nucleotides long) and a co-terminal 3&#8242;-end [78]. The 5&#8242;-leader sequence binds to N protein and form a complex, may act as a strong translation initiation signal and promote viral mRNA translation [79,80]. For each mRNA, only the 5&#8242;-open reading frames (ORFs) are translated via cap-dependent manner. Most of the mRNAs are monocistronic, while some mRNAs are bicistronic or tricistronic. mRNA1 encodes two large precursor polyproteins pp1a and pp1ab, pp1ab is translated via &#8722;1 programmed ribosomal shifting manner. Both polyproteins pp1a and pp1ab are proteolytically cleaved by virus encoded proteases, papain-like proteinase (PLpro) and 3C-like proteinase (3CLpro), into 13&#8211;16 mature non-structural proteins (nsps). Many of the nsps participate in viral RNA replication and transcription. The subgenomic mRNAs are translated into structural proteins haemagglutinin-esterase (HE), spike protein (S), envelope protein (E), membrane protein (M), nucleocapsid protein (N), and several nsps, respectively. HE is only encoded by &#946; coronavirus. The expression ratio of these genes is regulated at the level of transcription, during which time the shorter mRNAs are produced more abundantly than the longer one [81,82]. The 5&#8242;-UTR (untranslated region), which is unique for each of mRNA, also regulate the rate of each mRNA translation [83].Viruses utilize host translation machinery to finish viral protein translation. In response to acute viral infection, host cell would shut down protein translation system to cope with the infection stress, which is regarded as integrated stress response. Integrated stress response is marked by phosphorylation of eIF2&#945;, down regulation of the general cap-dependent protein synthesis, and up-regulation of the expression of certain transcription factors, such as ATF4. To acquire successful production of viral proteins, viruses must overcome this obstacle to ensure viral protein synthesis. The translation regulation by virus infection occurs by a number of ways, such as degradation of cellular mRNAs, alteration the activity of ribosome and associated factors, competitive displacement of cellular mRNAs by viral mRNAs for translation, etc. Coronavirus mRNAs are 5&#8242;-capped and 3&#8242;-polyadenylated, structurally equivalent to host mRNAs. Due to the translation competition between cellular and viral mRNAs for limiting number of ribosomes and associated factors, coronavirus must hijack the host translational machinery to produce their own proteins. More and more studies focus the mechanisms of translational control by coronavirus. SARS-coronavirus [84,85,86,87], MERS-coronavirus [88], MHV [36,89,90], transmissible gastroenteritis virus (TEGV) [91,92], porcine epidemic diarrhea virus (PEDV) [93], bat coronaviruses [94], have been shown to induce host translation shutoff in susceptible cells. In the other way, infectious bronchitis virus (IBV) maintains protein translation in infected cells [44]. The mechanisms of coronavirus modulated translation will be summarized in following sections.\n3.2. Regulation of Host Protein Synthesis via Targeting to eIF2\u03b1\nHost protein translation shutoff is not only due to the host mRNA degradation and specific viral protein induced ribosome disfunction, but also induced by virus infection stress stimuli [95]. The infection stress stimili induces translational shut off via the phosphorylation of eukaryotic initiation factor &#945; (eIF2&#945;). eIF2&#945;, together with eIF2&#946;, and eIF2&#947;, forms eIF2, mediates the binding of initiator Met-tRNAi to the ribosome in a GTP-dependent manner. Once the initiation is completed, eIF2-GDP is released from the ribosome, and GDP is exchanged for GTP to form active eIF2-GTP, participating in another round of translation initiation. eIF2 is inactivated by phosphorylation of eIF2&#945; on Ser51, which has increased affinity to eIF2&#946; and blocks the exchange of GDP to GTP, thus depleting the active eIF2&#946;-GTP pool. Since the cellular concentration of eIF2&#946; is much lower than eIF2&#945;, a small proportion of phosphor-eIF2&#945; can exhaust eIF2&#946; by sequestration [96]. Therefore, phosphorylation of eIF2&#945; is a checkpoint of protein synthesis initiation and relieves stress through ceasing translation under infection stress. Given the importance of eIF2&#945; in translation initiation, the level of phosphor-eIF2&#945; is no wonder regulated under infection stress. There are four eIF2&#945; kinases, GCN2, HRI, PERK, and PKR. These kinases are activated as a result of discrete stress, such as amino acid starvation or ultraviolet light (GCN2), heme deficiency (HRI), excess unfolded proteins accumulated in the ER [97], and double-stranded RNA (dsRNA) produced in virus-infected cells (PKR) [98].Under virus infection stress, PKR is induced as an IFN stimulated gene (ISG) and activated by dsRNA produced during the course of viral infection. dsRNA binds to PKR and causes a conformational change, leading to the dimerization and autophosphorylation of this kinase [99]. PKR phosphorylates eIF2&#945; and inhibits protein synthesis, thereby rendering an antiviral effect [96]. PERK, another eIF2&#945; kinase, usually activated by excess viral proteins loaded in the ER [2,100]. PKR and PERK activation inhibits protein synthesis globally and curtails viral spread through inactivating eIF2&#945; making these two kinases key players in integrated stress response to virus infection. High level of phosphor-eIF2&#945; leads to preferential translation of transcription factor ATF4, which promotes a number of genes expression. One of ATF4 target genes is growth arrest and DNA damage-inducible protein 34 (GADD34), a regulatory subunit of protein phosphatase 1 (PP1). GADD34 helps PP1 to dephosphorylate eIF2&#945;, thereby recovering global protein synthesis. The induction of GADD34 is a canonical cellular response to counteract PERK/PKR via preserving eIF2&#945; activity [5,101]. A number of viruses encode viral protein mimicing the function of GADD34, helps PP1 to counterbalance the PKR/PERK-mediated phosphorylation of eIF2&#945;.Regarding to coronavirus, it was reported that SARS-coronavirus infection activates PERK by phosphorylation, leading to sustained phosphorylation of eIF2&#945; in 293T/ACE2 cells. The SARS-coronavirus infection also activates PKR. Unexpectedly, the activation of PKR is not involved in eIF2&#945; phosphorylation and virus replication, but plays a role in viral induced apoptosis [47]. Overexpresson of SARS-coronavirus spike glycoprotein resulted in activation of PERK and upregulation of GRP78/GRP94 [46]. Another ER localized transmembrane glycoprotein, 3a, also activates PERK and thus increases the level of phosphor-eIF2&#945;, leading to activation of GRP78, GRP94, ATF4, and CHOP promoter, which is demonstrated by co-transfection of luciferase reporter plasmid with plasmid encoding 3a [23]. It is unclear whether spike protein or 3a protein plays a major role in SARS-coronavirus induced translation shutoff, as the translational rate is not examined upon overexpression of these proteins.MHV strain A59 infection increases the level of phosphor-eIF2&#945; and attenuates the host protein synthesis, resulting in preferential translation of ATF4. Unexpectedly, the ATF4 target genes, GADD34 and CHOP, are not detected during MHV infection, which may correlates with the persistent phosphorylation of eIF2&#945; and sustained suppression of host protein synthesis [36,45]. The MHV induced translation inhibition coincides with degradation of host mRNA. The formation of stress granules and P bodies, the sites of mRNA stalling and degradation respectively, are detected in MHV-infected LR7 cells [102]. MHV infection also leads to RNase L-independent specific 28S rRNA cleavage in all susceptible cell lines, which may contribute to MHV induced translational inhibition [103]. Although MHV infection induces protein synthesis inhibition, the viral proteins are still efficiently made, which can be attributed to the 5&#8242;-leader sequence and N protein in viral mRNA or the abundance of viral mRNA. MHV replication is not negatively affected either in the host translational shutoff-deficient (eIF2&#945;S51A) MEF cells or stress granule impaired (TIA-1&#8722;/&#8722; and TIAR&#8722;/&#8722;) MEFs, suggesting the translational inhibition is not beneficial for virus replication in vitro [102].To allow adequate proteins synthesis, some coronaviruses harness strategies to prevent eIF2&#945; phosphorylation. For example, IBV activates PKR and PERK at early infection times, leading to phosphorylation of eIF2&#945;, however, the activation of PKR and PERK is suppressed at late infection times by unknown mechanisms Furthermore, IBV infection up-regulates GADD34 expression, which is a regulatory subunit of PP1. By suppression of kinase PERK and PKR activity, and enhancement of phosphotase GADD34-PP1 activity, phosphor-eIF2&#945; was reduced to low level at late infection times, resulting in robust protein synthesis throughout the infection course. The sustained protein translation is beneficial for virus replication, which is verified by overexpression of GADD34 [18,44]. Unlike IBV, which maintains protein synthesis via blockage of PKR/PERK activity and up-regulation of GADD34, TEGV infection leads to PKR activation with a maximum at 12 h.p.i.. However, the level of phosphor-eIF2&#945; in TEGV infected cell is comparable to mock-infected cells, and moderate protein synthesis is observed throughout the infection times. A genus-specific protein 7 mimicking the function of GADD34 is identified in TEGV, with a conserved sequence motif (RVIFLVL) binding to PP1c. The interaction of protein 7 and pp1c is required for dephosphorylation of eIF2&#945;, inhibition of cellular RNA degradation, and maintenance of protein synthesis. Infect cells with TGEV&#916;7 (with deletion of protein 7) results in impaired protein synthesis, faster and stronger 28S rRNA degradation, and increased apoptosis, compared to that in wild type TEGV infected cells. Moreover, TGEV&#916;7 infected piglets exhibits accelerated pathology and more lesions at 4 dpi when compared with the wild type TGEV infected one. Piglets infected with virulent enteric strain TGEV-SC11-&#916;7 develop faster and more pronounced clinical disease. Thus, TGEV protein 7 functions to relieve the integrated stress response and to recover protein synthesis, thereby attenuating virus virulence [104].\n3.3.Suppression of Host Protein Synthesis and IFN Response by Coronavirus nsp1\nCoronavirus nsp1, the most N-terminal product of the pp1a polyprotein, is only encoded by two genera, &#945; and &#946; coronavirus. This protein is highly divergent among coronaviruses and functions to regulate host and viral gene expression. Recently, a number of reports demonstratethat nsp1 is involved in the regulation of host protein translation. Among &#946; coronavirus, SARS-coronavirus nsp1 was first identified to block the expression of reporter gene under the control constitutive promoters [84]. This protein associates and inactivates the 40S ribosomal subunit, thus preventing viral and cellular mRNA translation [105]. Moreover, the Nsp1-40S ribosome complex also promotes cellular mRNA degradation via inducing an endonucleolytic mRNA cleavage in the 5&#8242; region of the capped mRNA [84,105,106]. The presence of 5&#8242;-end leader sequence in viral mRNA, and the binding of stem loop 1 in the 5&#8242;-UTR with nsp1, protect viral mRNAs from nsp1-induced RNA cleavage, and facilitate efficient viral gene expression in infected cells [87,106]. In vivo studies suggested that SARS-coronavirus nsp1 counteracts the host innate immune response via suppression of host gene expression, including type I IFN, thereby playing a crucial role in virus virulence [85,105,107]. The inhibition of host protein synthesis and IFN inhibition by nsp1 seems to be a general phenomenon in coronavirus, which is also observed in other coronaviruses. For example, the nsp1 of bat coronavirus strains, Rm1, 133 and HKU9-1, exhibit functional similarities with SARS-coronavirus nsp1, with the ability to prevent host protein synthesis and suppress innate immune functions [94,108]. MERS-coronavirus nsp1 also negatively regulate host gene expression by inducing the degradation and inhibiting the translation of host mRNA. This protein selectively targets nuclear host mRNA and transports them to the cytoplasm for degradation and translation inhibition [88]. MHV nsp1 displays similar function to suppress the expression of reporter genes and host genes [108]. Deletion or mutation of potential functional domain of nsp1 severely attenuates MHV, and influences hepatotropism and liver pathogenesis in vivo, and elicits potent immune response. Thus, nsp1 is a major virulence factor [108,109,110].Among &#945; coronavirus, TEGV nsp1 suppresses host mRNAs translation without affecting their stability. This protein also inhibits the translation of different reporter mRNAs in cells or cell extract, but not in the in vitro translation system such as rabbit reticulocyte lysate, indicating that TEGV nsp1 relies on host factors to exert the inhibitory effect [91]. HCoV-229E and HCoV-NL63 nsp1 inhibit the expression of reporter genes, probably via binding to ribosomal protein S6 and blocking the mRNA binding to the 40S ribosomal subunit [108,111]. The essential role of covonavirus nsp1 in regulation of protein synthesis and IFN response, makes it a virulence factor and was targeted to developed live attenuated coronavirus vaccine [85,107,110].\n4. Conclusions\nReplication of coronavirus induces UPR and other cellular stress responses in the infected cells, triggering innate immunity and antiviral signaling pathways. The translational control constitutes one aspect of cellular antiviral response, and it is targeted and subverted by various coronaviruses at different levels via different mechanisms. It is hoped that further studies on these subjects will promote a more complete understanding of corovirus-host interactions and shed light on the development of antiviral therapeutics.",
        "pmid": "27384577",
        "title": "Regulation of Stress Responses and Translational Control by Coronavirus",
        "journal_title": "Viruses",
        "authors": "To Sing Fung;Ying Liao;Ding Xiang Liu",
        "doi": "10.3390/v8070184"
    },
    {
        "keywords": "PCR;respiratory pathogens;syndromic testing",
        "abstract": "ABSTRACT\nThe FilmArray Respiratory Panel 2 (RP2) is a multiplex in vitro diagnostic test for the simultaneous and rapid (&#8764;45-min) detection of 22 pathogens directly from nasopharyngeal swab (NPS) samples. It contains updated (and in some instances redesigned) assays that improve upon the FilmArray Respiratory Panel (RP; version 1.7), with a faster run time. The organisms identified are adenovirus, coronavirus 229E, coronavirus HKU1, coronavirus NL63, coronavirus OC43, human metapneumovirus, human rhinovirus/enterovirus, influenza virus A, influenza virus A H1, influenza virus A H1-2009, influenza virus A H3, influenza virus B, parainfluenza virus 1, parainfluenza virus 2, parainfluenza virus 3, parainfluenza virus 4, respiratory syncytial virus, Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae. Two new targets are included in the FilmArray RP2: Middle East respiratory syndrome coronavirus and Bordetella parapertussis. This study provides data from a multicenter evaluation of 1,612 prospectively collected NPS samples, with performance compared to that of the FilmArray RP or PCR and sequencing. The overall percent agreement between the FilmArray RP2 and the comparator testing was 99.2%. The RP2 demonstrated a positive percent agreement of 91.7% or greater for detection of all but three analytes: coronavirus OC43, B. parapertussis, and B. pertussis. The FilmArray RP2 also demonstrated a negative percent agreement of &#8805;93.8% for all analytes. Of note, the adenovirus assay detects all genotypes, with a demonstrated increase in sensitivity. The FilmArray RP2 represents a significant improvement over the FilmArray RP, with a substantially shorter run time that could aid in the diagnosis of respiratory infections in a variety of clinical scenarios.",
        "body": "INTRODUCTION\nUpper respiratory infections are common and contribute significantly to morbidity and mortality. They are also one of the leading reasons for health care visits, thus resulting in significant health care costs (1, 2). Because the symptoms related to infections with many of the causative agents are very similar, definitive diagnosis requires laboratory testing. Toward that end, the concept of syndromic testing has been widely adopted, with testing for multiple agents of respiratory infection at the same time with a single test. By using these syndromic diagnostics, proper antimicrobial stewardship may be better achieved by allowing antimicrobial or antiviral therapy to be given in a timely and appropriate manner (3, 4). Most importantly, it may prevent the unnecessary use of antibiotics in the face of a viral diagnosis. Additionally, studies have demonstrated that rapid diagnosis of respiratory infections can lead to decreased length of stay, better antimicrobial stewardship, and better patient cohorting to prevent nosocomial infections (3, 5&#8211;8).The FilmArray Respiratory Panel (RP) was first introduced as a syndromic multiplex molecular test in 2011 for the detection of 15 viruses; additional viral analytes and bacteria were made available with a software update in 2012 after FDA clearance for these new indications. Adenovirus inclusivity was improved with the addition of new primers after an additional FDA clearance in 2013 (version 1.7 [v1.7]). All FilmArray RP references henceforth in this article are to the current commercially available version of the device as of the publication of this paper, i.e., the FilmArray Respiratory Panel v1.7.In order to ensure that a molecular diagnostic assay remains clinically relevant, and particularly for syndromic assays, it is important to periodically update the test to incorporate new sequence information and to accommodate emerging or previously unrecognized strains or pathogens. To this end, BioFire Diagnostics has updated the FilmArray RP product again by adding new assays to broaden the test's detection capabilities (particularly for adenoviruses), modifying a subset of assays to reflect newly available genetic sequences of currently included analytes, improving chemistry to enhance sensitivity overall, and for the inclusion of new analytes. The new test also has a decreased run time (&#8764;45 min versus &#8764;65 min). The organisms detected by the FilmArray RP2 include all of those identified by the FilmArray RP: adenovirus, coronavirus 229E (CoV-229E), coronavirus HKU1 (CoV-HKU1), coronavirus NL63 (CoV-NL63), coronavirus OC43 (CoV-OC43), human metapneumovirus (hMPV), human rhinovirus/enterovirus (HRV/EV), influenza virus A (FluA), influenza virus A H1 (FluA H1), influenza virus A H1-2009 (FluA H1-2009), influenza virus A H3 (FluA H3), influenza virus B (FluB), parainfluenza virus 1 (PIV1), parainfluenza virus 2 (PIV2), parainfluenza virus 3 (PIV3), parainfluenza virus 4 (PIV4), respiratory syncytial virus (RSV), Bordetella pertussis (detection of ptxP), Chlamydia pneumoniae (previously named Chlamydophila pneumoniae), and Mycoplasma pneumoniae. Two new targets are included: Middle East respiratory syndrome coronavirus (MERS-CoV) and Bordetella parapertussis (detection of IS1001). Note that results for MERS-CoV are masked in the FilmArray RP2 product that is FDA cleared for the U.S. market. This analyte is reported in the FilmArray Respiratory Panel 2plus (RP2plus) product, which is sold outside the U.S. for testing individuals demonstrating signs/symptoms of respiratory infection and has been cleared by the U.S. FDA, with a modified intended use to aid in the differential diagnosis of MERS-CoV infections only in cases meeting MERS-CoV clinical and/or epidemiological criteria. The FilmArray RP2 is identical to the current FilmArray RP with respect to specimen type, handling, testing workflow, pouch controls, and analysis software.In this study, data are presented for a prospective multicenter clinical evaluation of the performance of the FilmArray RP2 in residual nasopharyngeal swab (NPS) specimens collected in viral transport medium (VTM). Performance is compared to that of the FilmArray RP for 20 of 22 analytes (all those in common between the two tests) as well as that of PCR followed by bidirectional sequencing for B. parapertussis. MERS-CoV was not circulating in the United States during the time of the study; therefore, all specimens were assumed to be negative, and no comparator testing was performed for this analyte.\nMATERIALS AND METHODS\n\nClinical specimens.\nThe study was conducted at three geographically distinct U.S. sites (Nationwide Children's Hospital [NCH], Columbus, OH; Loyola University Medical Center, Maywood, IL; Primary Children's Hospital, Salt Lake City, UT) over a period of approximately 6 months (January to March and September to November 2016). Between January and March 2016, specimens were collected and immediately frozen for later testing. Between September and November 2016, specimens were collected and tested fresh. Specimens meeting the following inclusion criteria were selected: the specimen was an NPS collected in VTM with adequate residual volume (&#8805;1.5 ml), the specimen was tested with the FilmArray RP as the standard of care (SOC), and the specimen was held at room temperature for less than or equal to 4 h or at 4&#176;C for less than or equal to 3 days before enrollment. A waiver of the informed consent requirement was obtained from the institutional review boards at each study site for the use of residual NPS specimens. Clinical and demographic data were collected, including hospitalization status at the time of specimen collection, the results of the clinician-ordered SOC FilmArray RP test, the date of specimen collection, the subject sex, and the subject age at time of collection.\nFilmArray RP2 testing.\nApproximately 300 &#956;l of specimen was subjected to FilmArray RP2 testing according to the manufacturer's instructions (9). All sample processing occurred in a biosafety cabinet with operators wearing gloves and other appropriate personal protective equipment. One sample was processed at a time, and the cleaning of work areas was done in accordance with the manufacturer's instructions (9). The FilmArray RP2 test consists of automated nucleic acid extraction, reverse transcription, nucleic acid amplification, and result analysis in approximately 45 min per run (i.e., per specimen). The FilmArray software performs automated result analysis, with each target in a valid run reported as &#8220;detected&#8221; or &#8220;not detected.&#8221; If either internal control fails, the software automatically provides a result of &#8220;invalid&#8221; for all panel analytes. There are 22 targets, as shown in Table 1, two of which are new to the FilmArray RP2. This study was conducted with an investigative-use-only (IUO) version of the FilmArray RP2 that is identical to the final FDA-cleared and CE-marked version. It is important to note that results for MERS-CoV are reported in this paper but are available only for the FilmArray RP2plus version of the product.\nComparator testing.\nComparator testing consisted of SOC FilmArray RP testing performed at the source laboratory for all analytes in common between the FilmArray RP and the FilmArray RP2 (all analytes except MERS-CoV and B. parapertussis). All specimens were assumed negative for MERS-CoV, as it was not circulating in the United States during the time of enrollment for the study.For B. parapertussis, two PCR assays targeting IS1001 (the same target identified by the FilmArray RP2), followed by bidirectional sequencing, were used as the comparator method. Nucleic acid was extracted from specimens using MagNA Pure LC Total Nucleic Acid Kit&#8211;High Performance (Roche Diagnostics, Indianapolis, IN). Both real-time PCR comparator assays were validated and found to have a limit of detection (LoD) that was equivalent to that of the FilmArray RP2 assay. Testing was performed at BioFire in a blind manner. Comparator assay results were considered positive only when a bidirectional sequencing result of adequate quality was found to match a sequence for the expected analyte with an E value of 1.0E&#8722;30 or lower in the GenBank nucleotide database (Basic Local Alignment Search Tool [BLASTn] with default settings). A specimen was considered to be &#8220;positive&#8221; with a sequence-confirmed result from either assay.\nResults and discrepant analysis.\nA FilmArray RP2 result was considered a true positive (TP) or true negative (TN) only when it agreed with the result from the comparator method. Discrepant analysis ensued when results were discordant, i.e., false-positive (FP) or false-negative (FN) results. When sufficient specimen volume was available, discordant specimens were investigated using a combination of retesting with the FilmArray RP2 or comparator methods, as well as testing with additional, independent molecular assays. For additional analysis of adenovirus targets, specimens were also tested with a combination of PCR assays targeting the DBP, penton, and pol genes (combined with bidirectional sequence analysis) (9) and the results of standard-of-care testing at one of the study sites (NCH) using an adenovirus laboratory-developed test (LDT) PCR targeting the hexon gene as described previously (10&#8211;12). Note that the performance data for positive percent agreement (PPA) and negative percent agreement (NPA) presented in this paper consist of unresolved data as presented in the package insert for the FDA-cleared test; discrepancy investigation is provided but was not used to recalculate performance data.\nStatistical analysis.\nThe exact binomial two-sided 95% confidence intervals (95% CI) were calculated for performance measures according to the Wilson score method.\nRESULTS\n\nDemographics.\nA total of 1,612 prospective study specimens collected from geographically/demographically diverse subject populations were analyzed in this study. Overall, the study included specimens from more male than female subjects (54% [867/1,612] and 46% [745/1,612], respectively). Most specimens were from pediatric subjects: 55% of the specimens were from children aged 5 years and under, 21% were from those aged 6 to 21 years, 17% were from adults over the age of 50 years, and 8% were from adults aged 22 to 49 years. The majority of the specimens were obtained from hospitalized subjects and those visiting the emergency department (40% [640/1,612] and 40% [643/1,612], respectively), and 20% were obtained from subjects seen in an outpatient setting (329/1,612).\nFilmArray RP2 test performance.\nA total of 1,623 specimens met the inclusion criteria and were initially tested in the clinical evaluation. The overall success rate for the initial test of these specimens was 99.3% (1,611/1,623); 12 tests were unsuccessful (1 due to an incomplete test, 1 due to an instrument error, and 10 due to control failures). Eleven of these specimens were successfully retested. In addition, another 10 specimens were later excluded for protocol reasons, resulting in a total of 1,612 specimens included in the data analysis.\nSummary of FilmArray RP2 findings.\nThe FilmArray RP2 detected at least one analyte in 1,020 of the 1,612 specimens tested, yielding an overall positivity rate of 63.3%, as shown in Table 2. The highest detection rate was seen in young children (&#8804;5 years of age). The relative prevalence of each analyte among the positive specimens detected by the FilmArray RP2 is presented in Table 3. The most prevalent organisms detected during this study were HRV/EV, RSV, adenovirus, and FluA, which were found in 502 (31.1%), 199 (12.3%), 118 (7.3%), and 81 (5.0%) specimens, respectively. If taken together, coronaviruses (CoV-229E, -HKU1, -NL63, and -OC43) were the third most prevalent target, with 159 (9.9%) detections. For FluA H1 and the MERS-CoV targets, no positive analyte detections occurred in this prospective sample set. All other analytes were detected in fewer than 79 (<4.9%) specimens.A summary of performance characteristics for individual FilmArray RP2 targets is presented in Table 4. PPA and NPA were calculated with respect to the comparator methods along with 95% CI. The FilmArray RP2 demonstrated a PPA of 91.7% or greater for all but three analytes. Nine of 22 analytes demonstrated a PPA of 100%: CoV-HKU1, CoV-NL63, FluA, FluA H1-2009, FluA H3, FluB, PIV1, PIV4, and C. pneumoniae. Eight other targets demonstrated a PPA of <100% but &#8805;90.0%: adenovirus, CoV-229E, hMPV, HRV/EV, PIV2, PIV3, RSV, and M. pneumoniae. For FluA H1 and MERS-CoV, no PPA could be calculated. The three analytes demonstrating a PPA of <90.0% were CoV-OC43 (80.5%), B. parapertussis (85.7%), and B. pertussis (66.7%). Additionally, nine analytes demonstrated a lower bound of the two-sided 95% CI of <80.0% due to few or no observations in the study. Overall, the FilmArray RP2 demonstrated an NPA of &#8805;93.5% for all analytes, with lower bounds of the two-sided 95% CI of &#8805;91.9%.\nComparator analysis and discrepancy investigation.\nThere were a total of 33,843 analyzable FilmArray RP2 organism results for the 1,612 specimens. The overall percent agreement between the FilmArray RP2 and the comparator testing was 99.2% (33,586/33,843). There were 1,329 detected organism results with the FilmArray RP2; the comparator methods were positive for 1,138 analytes. The overall PPA with respect to the comparator method was 97.1% (1,105/1,138). There were 32,481 results not detected with the FilmArray RP2; the comparator methods were negative for 32,705 analytes. The overall NPA with respect to the comparator method was 99.3% (32,481/32,705).Using comparator testing results as the truth, there were 224 FP detections and 33 FN detections overall; additional discrepancy analysis was performed for these 257 samples. For the 114 FP cases (51%) and the 14 FN cases (42%), there was supportive evidence for the FilmArray RP2 result, bringing the adjudicated overall concordance for the positive and negative results to 98.5% and 99.7%, respectively. A summary of the discrepancy investigation is presented in Table 5.For the viral analytes, the FilmArray RP2 detected a total of 1,286 viral analytes. Using the comparator results as the truth, the overall PPA and NPA are 97.3% (1,069/1,099) and 99.2% (26,079/26,296), respectively. The results for several analytes of significant interest are further detailed below.For adenovirus, a significant increase in detections was observed in comparison to those by the FilmArray RP, with a total of 118 detections, of which 48 (40.7%) were FP. FP specimens with sufficient volume were retested with the FilmArray RP to see if the original result had been an anomaly. When possible, specimens were also tested with a combination of PCR/sequencing assays targeting the DBP (n = 38), penton (n = 25), and pol (n = 16) genes and the results of the NCH LDT assay (n = 11). Combined, these investigations found additional evidence of adenovirus presence in 40 of the 48 FP specimens (Table 6). All 40 of these specimens had late amplification on the FilmArray RP2 test, suggestive of low levels of analyte in these specimens. The four FP specimens for which the FilmArray RP retest was positive also had late amplification, suggestive of a low level of analyte.There were also 4 FN results for adenovirus. Additional discrepant analysis for these specimens included retesting with the FilmArray RP2, a combination of PCR assays as described above, and any available NCH LDT results for adenovirus. Combined, these investigations found additional evidence of adenovirus presence in three of the four FN specimens. Analysis of the FN specimen for which the FilmArray RP2 retest was positive indicated late amplification, suggesting low analyte levels. All FN were adenovirus species C based on sequence analysis. A comprehensive summary of the adenovirus discordant analysis is provided in Table 6.Among the coronaviruses, all but one of the four targets demonstrated good performance, with a PPA of &#8805;91% and an NPA of &#8805;99.1%. The exception was CoV-OC43, which demonstrated a PPA of 80.5%. The majority of FN specimens observed were due to a known cross-reactivity in the comparator method (see BioFire FilmArray RP package insert at https://www.online-ifu.com/ITI0040): a FilmArray RP-detected result for coronavirus OC43 due to cross-reactivity with CoV-HKU1 is suspected whenever the FilmArray RP reports detections for both CoV-HKU1 and CoV-OC43. This cross-reactivity has been corrected by the redesign of the CoV-OC43 assay for the FilmArray RP2. Six of eight FilmArray RP2 FN specimens were TP for CoV-HKU1, i.e., codetections reported by the FilmArray RP and suggestive of this known cross-reactivity. As stated previously, no MERS-CoV was detected in the cohort. Of note is an NPA of 100%, indicating a lack of cross-reactivity with other coronaviruses. Data for some archived MERS-CoV specimens and contrived MERS-CoV samples are provided in the manufacturer's package insert for the FilmArray RP2plus (9).The FluA targets showed no FP or FN detections; however, there were no positive detections for FluA H1 during the study period, which was predominated by FluA H1-2009. For FluB, there were two FP detections, which were confirmed on further investigation as TP. FluA, FluA H1, FluA H1-2009, and FluA H3 results were excluded from analyses for three specimens due to initial results of &#8220;influenza A equivocal&#8221; or &#8220;influenza A no subtype detected&#8221; by either FilmArray RP2 or FilmArray RP testing and insufficient specimen volume for retesting.Detections for HRV/EV were numerous, with a total of 502, the highest of all detections in the trial. There were 77 FP results and 11 FN results. Specimens with sufficient volume were retested with the FilmArray RP or FilmArray RP2. When possible, specimens were also tested with a combination of five PCR assays targeting the 5&#8242; UTR gene. For the FP samples, 29 were positive with a FilmArray RP retest; late amplification for 28 of the 29 specimens was suggestive of low levels of analyte. Four more were positive with PCR assays. For the FN samples, four specimens were positive with the FilmArray RP2 on repeat testing and one was positive with PCR assays. Three of the four FN specimens for which the FilmArray RP2 retest was positive had late amplification, suggestive of low levels of analyte.RSV detections totaled 199, making it the second most common analyte. Eight of 24 FP specimens were observed to contain RSV by independent molecular methods or retesting with the FilmArray RP. These may have been missed by the SOC FilmArray RP test due to an estimated hundredfold difference in LoD between the FilmArray RP and FilmArray RP2 (9).Using the comparator results as the truth, the overall PPA and NPA are 92.3% (36/39) and 99.9% (6,402/6,409), respectively, for all bacterial targets. The number of detections for each bacterial target was low (&#8804;6), with the exception of that for M. pneumoniae (n = 28) (Table 4). The two bacterial analytes demonstrating a PPA of <90.0% were both low prevalence: B. parapertussis (n = 6), and B. pertussis (n = 3).The bacterial targets tended to be single analyte detections (B. pertussis, 3/3; C. pneumoniae, 5/6; and M. pneumoniae, 21/28) with no copathogen present. For B. parapertussis, all six detections were in the context of a codetection with one or more viruses. No sample had two bacterial targets detected. Discordant analysis for the bacterial targets in shown in Table 5.\nDISCUSSION\nThis study of the FilmArray RP2 demonstrated the performance of the test in a large prospective study of 1,612 residual NPS samples, with 33,843 results generated. These data are significant, as this is a substantial change from the results with RP, and the test will be adopted for use in a large number of clinical laboratories. The number of positive detections was relatively high for most organisms; notable exceptions were the results for MERS-CoV and FluA H1, which were not circulating in the study populations during the study period. The FilmArray RP2 testing system was shown to be reliable, with very few failures (99.3% success on the initial test attempt), and rapid, with results available in approximately 45 min, which is shorter than that of the FilmArray RP (approximately 65 min run time). The data presented here along with testing of archived positive NPS in VTM specimens and contrived specimens (not shown) (9) were used as part of the regulatory submissions for the FilmArray RP2 and RP2plus, which received 510(k) clearance in the United States (RP2) and CE/IVD marking in the European Union (RP2plus) in June 2017. The FilmArray RP2plus received de novo clearance in the United States in November 2017.Periodically updating testing that has been implemented is an important concept. The College of American Pathologists covers this for laboratory-developed testing in its Microbiology Checklist, stating that laboratories should have written policies and procedures to evaluate nucleic acid tests for compatibility with currently circulating microbial strains (13). For testing cleared by the FDA, The FilmArray RP2 represents the fourth iteration of the multiplex panel since its introduction in 2011, providing an update of the primer probes based on a reexamination of known sequences for the majority of the pathogens and adjustment of the assay conditions to maximize performance. As noted, there were a significant number of detections by RP2 that were not found by RP (n = 224). The overall design goal for RP2 was to increase the sensitivity for all analytes relative to that of RP, and this may account for a significant number of the observed FP detections. This is supported by LoD studies reported in the product inserts (9, 14) and the discordant analysis performed in this study. The increased inclusivity/sensitivity and decreased time to result to 45 min for the FilmArray RP2 may lead to improvements in outcomes, such as length of stay or proper stewardship, and warrant further study.Viruses are a common cause of upper respiratory infections in both adult and pediatric populations, and this was also seen in our study cohort. Viral detections were notably higher than those of the bacterial targets (1,286 viral detections versus 43 bacterial detections). The FilmArray RP2 showed an increased positive detection rate for all viral targets in comparison to that of the FilmArray RP (217 more detections, with 107 supported by additional discrepancy investigation), with the exception of coronavirus OC43 (Table 5), reflecting the increased sensitivity and inclusivity of the FilmArray RP2.The most common viral analyte was HRV/EV, with a total of 502 detections versus 436 detections with the FilmArray RP. The increased number of HRV/EV detections may or may not be associated with true disease causation, as the majority (61.2% [see Table S1 in the supplemental material]) were in the context of codetection with other viral targets. Rhinovirus has been reported as a commonly detected target among asymptomatic individuals, with rates ranging from 8 to 50% depending on the study (15&#8211;17). While the FilmArray RP2 was updated to broaden inclusivity, there was no change in specificity for HRV/EV, so there are still cross-reactions with enteroviruses and hence the rhinovirus/enterovirus designation.One of the more extensive modifications occurred in the detection of adenovirus. Previous studies by Leber et al. demonstrated a lack of sensitivity with the FilmArray RP for adenovirus types A, D, and F (12) despite an earlier redesign of the FilmArray RP in 2013 (18). The redesign of the FilmArray RP2 specifically targeted all genotypes to include genotypes A to F and not only those typically associated with respiratory infections (types B, C, and E). In our cohort, genotypes A, B, C, D, and F were demonstrated to be detected by the FilmArray RP2. Detection of all genotypes is important, particularly in the immunocompromised, where the finding of adenovirus of any genotype in the NPS in VTM may precede systemic infections (11, 12). In addition, the identification of species F in respiratory specimens has been reported in patients with respiratory illness (19) as well as in 2.3% of pediatric patient samples obtained after routine adenoidectomy/tonsillectomy (20).Overall, there were relatively low numbers of bacterial detections with the FilmArray RP2 (n = 43). The reasons for this are likely due to true disease prevalence differences during the study period. Also, as seen in our data (Table S1), the codetection of bacteria and viruses is not common, particularly with B. pertussis, as has been previously reported (21, 22). The target gene for B. pertussis in both the FilmArray RP and the FilmArray RP2 is the toxin promoter region. This single-copy gene is known to be more specific than the more commonly used insertion sequence 481 (IS481) gene that is a multicopy target present in several Bordetella species. While having greater specificity, the toxin gene target may be less sensitive, as has been reported previously (23). The diagnosis of pertussis-like illness is improved with the inclusion of the insertion sequence element 1001 target for B. parapertussis in the FilmArray RP2. B. parapertussis is known to cause a pertussis-like illness and can cocirculate with other Bordetella species (24, 25). The prevalence of B. parapertussis is uncertain, as it is not a reportable disease like B. pertussis and is not tested for as commonly (26, 27). M. pneumoniae was the most common of the bacterial analytes, with 28 detections, more than with the FilmArray RP. However, it should be noted that use of an NPS specimen for the detection of M. pneumoniae may be suboptimal, particularly when diagnosing lower respiratory tract infection (28, 29).Overall, the percentage of discrepant results was low (0.76%, n = 257) (Table 5), suggesting that the previous version of the FilmArray RP had relatively robust performance. Discrepancy analysis using FilmArray RP retests and PCR and bidirectional sequencing confirmed 114 of 224 FP (51%), which is strong evidence that the FilmArray RP2 has increased sensitivity compared to the FilmArray RP. There are some limitations of this study. This was a prospective study; however, some samples were frozen at &#8722;70&#176;C or less prior to testing. However, data indicated that the frozen storage did not significantly affect performance (9). The study period bridges one calendar year (2016) and includes only two partial respiratory seasons, so variations in circulating strains, particularly FluA, are limited. The comparator method for 20 of the targets was the FilmArray RP. Data concerning FilmArray RP2 performance compared to that of other amplified platforms or culture are not provided and will await other studies. Finally, the lack of detection of MERS-CoV and FluA H1 in the prospective study limited the data on the performance for these targets.A significant redesign of the FilmArray RP2 has demonstrated excellent sensitivity and specificity in this multicenter clinical trial. This is an important step both for individual improvements in pathogen detection and as recognition by the manufacturer that continuous improvements in monitoring and inclusion of new or emerging strains or species are important. Both have been incorporated into the design of the FilmArray RP2, improving its performance for the detection of infectious agents that are involved in respiratory infections. These changes include new targets (MERS-CoV and B. parapertussis), improvements to existing targets, and a decreased time to result. These improvements, combined with the simplicity of the testing process and a shorter time to result, make the FilmArray RP2 a significant improvement in diagnostic testing.\nSupplementary Material\n",
        "pmid": "29593057",
        "title": "Multicenter Evaluation of BioFire FilmArray Respiratory Panel 2 for Detection of Viruses and Bacteria in Nasopharyngeal Swab Samples",
        "journal_title": "Journal of Clinical Microbiology",
        "authors": "Amy L. Leber;Kathy Everhart;Judy A. Daly;Aubrey Hopper;Amanda Harrington;Paul Schreckenberger;Kathleen McKinley;Matthew Jones;Kristen Holmberg;Bart Kensinger",
        "doi": "10.1128/JCM.01945-17"
    },
    {
        "keywords": "",
        "abstract": "The N-terminal domain of the coronavirus nucleocapsid (N) protein adopts a fold resembling a right hand with a flexible, positively charged &#946;-hairpin and a hydrophobic palm. This domain was shown to interact with the genomic RNA for coronavirus infectious bronchitis virus (IBV) and severe acute respiratory syndrome coronavirus (SARS-CoV). Based on its 3D structure, we used site-directed mutagenesis to identify residues essential for the RNA-binding activity of the IBV N protein and viral infectivity. Alanine substitution of either Arg-76 or Tyr-94 in the N-terminal domain of IBV N protein led to a significant decrease in its RNA-binding activity and a total loss of the infectivity of the viral RNA to Vero cells. In contrast, mutation of amino acid Gln-74 to an alanine, which does not affect the binding activity of the N-terminal domain, showed minimal, if any, detrimental effect on the infectivity of IBV. This study thus identifies residues critical for RNA binding on the nucleocapsid surface, and presents biochemical and genetic evidence that directly links the RNA binding capacity of the coronavirus N protein to the viral infectivity in cultured cells. This information would be useful in development of preventive and treatment approaches against coronavirus infection.",
        "body": "INTRODUCTION\nRNA viruses, including coronaviruses, encode a group of structural phosphoproteins with basic patches on their surface. As a pivotal structural component of the virion, this group of proteins plays essential roles in packaging the RNA genome to form a ribonucleoprotein (RNP) complex resulting from assembly of the viral RNA and multiple copies of the (nucleo)capsid protein. A detailed dissection and characterization of the intrinsic RNA-binding properties of these proteins are thus essential for understanding several important processes in the life cycle of RNA viruses, including assembly of the nucleocapsid, the specific encapsidation of viral RNA and morphogenesis of virions. In this study, amino acid residues in the N-terminal domain of the nucleocapsid (N) protein of a coronavirus that are critical for its RNA-binding activity are identified and their roles in viral infectivity are analyzed, using site-directed mutagenesis based on our previous structural studies.Coronaviruses are enveloped viruses containing the largest known single-stranded, positive-sense RNA genome of &#8764;30 kb (1). During the coronavirus life cycle, the N protein is synthesized in large amounts and is thought to play an important role by specifically packaging the viral genome into a filamentous nucleocapsid of &#8764;10 to 15 nm in diameter and several 100 nm in length, a macromolecular structure that is visible by using electron microscopy (2). In addition to its structural role, the N protein also participates in viral RNA transcription, replication and in modulating the metabolism of host cells (3&#8211;13). Using X-ray crystallography techniques, we recently determined a 3D structure of the N protein from coronavirus infectious bronchitis virus (IBV), a prototype coronavirus, and showed that the basic building block for nucleocapsid assembly was a dimer of the N protein (14). The 409 amino acids long IBV N protein is composed of two globular domains that are resistant to proteolysis. Amino acids 29 to 160 form the N-terminal RNA binding domain and residues 218 to 329 form its C-terminal dimerization domain (5,14). Even though the structure of the N-terminal domain was determined in the absence of a nucleic acid ligand, its overall shape and the distribution of electrostatic charges suggest a plausible model for RNA binding. The outer surface of the N-terminal domain is enriched in aromatic and basic residues, a common feature for RNA-binding proteins (15). The shape of this domain is reminiscent of a hand having basic fingers, a hydrophobic palm and an acidic &#8216;wrist&#8217;. The positively charged &#8216;fingers-like&#8217; &#946;-hairpin extension could neutralize the phosphate groups emanating from RNA, whilst the base moieties could make contact with exposed aromatic residues from the hydrophobic palm (14).Based on this structural hypothesis, we chose to mutate a number of evolutionarily conserved residues in the N-terminal domain of the IBV N protein in order to assess their roles in viral genomic RNA binding and in viral replication. We used purified recombinant wild type and mutant proteins containing the N-terminal domain expressed in Escherichia coli for RNA binding assays. These studies led to the identification of a number of amino acid residues essential for the RNA-binding activity of the domain. Subsequently, we introduced mutations that either severely or mildly impair the RNA binding activities of the N-terminal domain into an infectious cDNA clone system derived from the genomic RNA of IBV to assess their effects on viral replication and infectivity. We identified several residues exposed to the solvent whose substitutions to alanine yield a decrease of RNA binding and a concomitant reduction in virus replication. Interestingly, residues Arg-76 and Tyr-94, located at the base of the positively charged flexible hairpin loop and on the hydrophobic platform, respectively, were critical for RNA binding and viral infectivity.\nMATERIALS AND METHODS\n\nExpression and purification of IBV N protein\nHis-tagged wild type and mutant N-terminal domains of IBV N protein were expressed in E.coli BL-21 by induction with 1 mM isopropyl-&#946;-d-thiogalactopyranoside (IPTG). Cells were lysed by sonication and purified by metal affinity chromatography with Protino-Ni 150 kit (Macherey Nagel).\nSynthesis of RNA probes\nPCR fragments covering the IBV genome from 27&#8201;100 to 27&#8201;608 nt were cloned into a plasmid in either forward or reverse orientation under the control of a T7 promoter. The Dig-labeled sense (+) and anti-sense (&#8722;) RNA probes were made in vitro using the DIG RNA labeling kit according to the instructions of the manufacturer (Roche).\nNorthwestern blot\nFive micrograms of purified proteins were resolved on an SDS-15% polyacrylamide gel and transferred onto nitrocellulose membranes (Hybond C-Extra, Amersham Biosciences) using a semi-dry transfer apparatus. Membranes were washed for 10 min with the probe buffer [1&#215; Denhardt's Reagent, 1 mM EDTA, 10 mM Tris&#8211;HCl (pH 7.5) and 50 mM NaCl], blocked for 1 h with 25 &#181;g/ml yeast tRNA (Ambion) and subsequently incubated with 10 &#181;g of DIG-labeled RNA probe in the same probe buffer for 1 h. Membranes were washed three times for 15 min each with the probe buffer, before proceeding to detection with CDP-Star (Roche) according to the manufacturer's instructions.\nConstruction of an infectious IBV clone, introduction of mutations into the clones and rescue of recombinant viruses\nConstruction of an infectious IBV clone was carried out essentially as described (16,17). Briefly, five fragments spanning the entire IBV genome were obtained by RT&#8211;PCR from Vero cells infected with the Vero cell-adapted IBV p65. The PCR products were purified from agarose gels and cloned into pCR-XL-TOPO (Invitrogen) or pGEM-T Easy (Promega) vectors. Subsequently, fragment A was removed from pCR-XL-TOPO by digestion with NheI and EcoRI, and subcloned into pKT0 vector. Plasmids were digested with either BsmBI (fragment A) or BsaI (fragments B, C, D and E). The digested plasmids were separated on 0.8% agarose gels containing crystal violet. Bands corresponding to each of the fragments were cut from the gels and purified with QIAquick gel extraction kit (QIAGEN Inc.). Fragments A and B, and fragments C, D and E were first ligated with T4 DNA ligase at 4&#176;C overnight. The two reaction mixtures were then mixed and further ligated at 4&#176;C overnight. The final ligation products were extracted with phenol/chloroform/isoamyl alcohol (25:24:1), precipitated with ethanol and detected by electrophoresis on 0.4% agarose gels.Full-length transcripts were generated in vitro using the mMessage mMachine T7 kit (Ambion, Austin, TX). The N transcripts were generated by using a linearized pKT0-IBVN containing IBV N gene and the 3&#8242;-untranslated region (3&#8242;-UTR) as templates. The in vitro synthesized full-length and N transcripts were treated with DNase I and purified with phenol/chloroform. Vero cells were grown to 90% confluence, trypsinized, washed twice with cold phosphate-buffered saline (PBS), and resuspended in PBS. RNA transcripts were added to 400 &#181;l of Vero cell suspension in an electroporation cuvette, and electroporated with one pulse at 450 V, 50 &#181;F with a Bio-Rad Gene Pulser II electroporator. The transfected Vero cells were cultured overnight in 1% FBS-containing MEM in a 60 mm dish or a 6-well plate and further cultured in MEM without FBS.Mutations were introduced into the corresponding fragments by using QuickChange site-directed mutagenesis kit (Stratagene), and confirmed by sequencing of the whole fragments.\nGrowth curve and plaque sizes of the recombinant viruses on Vero cells\nConfluent monolayers of Vero cells on 6-well plates were infected with wild-type and mutant viruses at a multiplicity of &#8764;1 PFU/cell. After 1 h of incubation at 37&#176;C, cells were washed twice with PBS and cultured in 3 ml of MEM containing 0.5% carboxymethy cellulose for 3 days. The cells were fixed and stained with 0.1% toluidine.Vero cells were infected with wild-type and recombinant IBV, and harvested at different times post-infection. Viral stocks were prepared by freezing/thawing of the cells three times. The plaque-forming units per ml of each sample were determined by infecting Vero cells on 6-well plates in duplicate with 10-fold serial dilution of each viral stock.\nNorthern blot analysis\nVero cells were infected with wild type and Q74A mutant virus at a multiplicity of &#8764;1 PFU/cell. Total RNA was extracted from cells infected with wild type or mutant viruses. Ten micrograms of RNA were added to a mixture of 1&#215; MOPS, 37% formaldehyde and formamide and incubated at 65&#176;C for 20 min before subjected to gel electrophoresis. The segregated RNA bands were transferred onto a Hybond N+ membrane (Amersham Biosciences) via capillary action overnight and fixed by ultraviolet (UV) crosslinking (Stratalinker). Hybridization of Dig-labeled DNA probes was carried out at 50&#176;C in hybridization oven overnight. Membranes were washed three times for 15 min each with the probe buffer, before proceeding to detection with CDP-Star (Roche) according to the manufacturer's instructions.\nWestern blot analysis\nVero cells were infected with wild type and Q74A mutant virus at a multiplicity of &#8764;1 PFU/cell. Total proteins extracted from Vero cells were lysed with 2&#215; SDS loading buffer in the presence of 200 mM DTT plus 10 mM of iodoacetamide and subjected to SDS&#8211;PAGE. Proteins were transferred to PVDF membrane (Stratagene) and blocked overnight at 4&#176;C in blocking buffer (5% fat free milk powder in PBST buffer). The membrane was incubated with 1:2000 diluted primary antibodies in blocking buffer for 2 h at room temperature. After washing three times with PBST, the membrane was incubated with 1:2000 diluted anti-mouse or anti-rabbit IgG antibodies conjugated with horseradish peroxidase (HRP) (DAKO) in blocking buffer for 1 h at room temperature. After washing three times with PBST, the polypeptides were detected with a chemiluminescence detection kit (ECL, Amersham Biosciences) according to the manufacturer's instructions.\nConstruction of plasmids\nPCR products covering the IBV sequence from 25&#8201;957&#8211;26&#8201;352 nt were amplified by using the forward primer 5&#8242;-CGGGCATATGTCTTCTGGAAATGCATCTTGG-3&#8242; and the reverse primer 5&#8242;-CGGGATCCTTACAGGGGAATGAAGTCCCAAC-3&#8242;. The PCR fragments were digested with NdeI and BamHI, and ligated into NdeI- and BamHI-digested pET-16b (Novagen). Each mutation was introduced by two-round PCR and the mutation introduced was confirmed by automated nucleotide sequencing.\nRESULTS\n\nOverview of the structure and RNA-binding properties of the N-terminal domain of coronavirus N protein\nThe N-terminal RNA-binding domain of N protein from both IBV and SARS-CoV contains a flexible and positively charged hairpin loop that extends much beyond the protein core and that could grasp a nucleic acid substrate by neutralizing its phosphate groups (Figure 1) (14,18). Positively charged lysine and arginine residues in the N-terminal domain of SARS-CoV N protein were proposed to bind a 32 nt stem&#8211;loop structure located at the 3&#8242; end of the SARS-CoV RNA genome (18). Indeed, the structure of this domain possesses some features reminiscent of RNA-binding proteins sharing the RNP fold (19&#8211;22). These RNA-binding proteins, including the U1A spliceosomal protein (23) and the coat protein from MS2 bacteriophage (24), bind the RNA ligand with residues emanating from the surface of a four-stranded anti-parallel &#946; sheet. Presumably, strands &#946;2 and &#946;3 as well as the flexible &#946;-hairpin from the IBV nucleocapsid protein could fulfill a similar role by interacting with phosphate groups from an RNA ligand (Figure 1). The &#946;3 strand is the longest in the N-terminal domain of IBV N protein and several evolutionarily conserved residues project from it (Figure 1). Nucleotide bases in single-stranded RNA have a tendency to stack either on adjacent bases or with aromatic side chains of the protein (20).\nMutational analysis of amino acid residues essential for RNA-binding in the N-terminal domain of IBV N protein\nThe above structural model was partially supported by the demonstration that the N-terminal domain of IBV N protein could interact with RNA fragments corresponding to the 3&#8242; end of the viral genome (14). To more vigorously test this model, systematic mutagenesis of residues essential for the RNA-binding activity was carried out. We targeted the exposed hydrophobic residues Tyr-92 and Tyr-94 (strand &#946;3) that could form stacking interactions with the nucleotide bases as well as several basic or polar residues emanating from the &#946;-hairpin (Figure 2a). As summarized in Figure 2a, 10 amino acids were mutated to alanine. These residues are either exposed to the solvent (Gln-74, Arg-76, Lys-78, Lys-81, Tyr-92 and Tyr-94) or have been conserved during evolution (Tyr-70 and Arg-73) and are thus likely to be involved in RNA binding or to play an undefined functional role. In addition, mutation of Asp-111 to an alanine was included as an additional control for the RNA-binding assay (Figure 2a).The wild type and mutant constructs, spanning 29 to 160 amino acids with a hexa-histidine tag at their N-termini, were cloned into a bacterial expression vector, expressed in E.coli and purified to near homogeneity. Approximately equal amounts of the purified proteins were separated on SDS&#8211;PAGE (Figure 2b, upper panels). After transfer to a Hybond C extra membrane, the RNA-binding activities of wild type and mutant proteins were analyzed by northwestern blot with two probes corresponding to the positive- and negative-sense 3&#8242;-UTR of IBV, respectively. A moderate decrease in binding to both the positive and negative sense RNA probes was consistently observed with most mutant proteins, and representative gels are shown in Figure 2b (Figure 2b, middle and lower panels). Among all mutants, mutation of Tyr-94 to an alanine residue showed the most significant reduction in binding of the domain to both positive and negative probes. The RNA binding activities of this mutant to positive and negative probes were reduced to 27 and 25%, respectively, of the wild type were observed (Figure 2b, lane 9). Relatively less reduction in the RNA-binding activity was observed for the R76A, K78A, K81A and Y92A mutant proteins which retain between 50&#8211;74% of the wild-type binding activity (Figure 2b, lanes 5&#8211;8). With the exception of the Q74A and D111A mutants, which displayed a slight increase in the RNA binding activity to both the positive- and negative-sense RNAs, (Figure 2b, lanes 4 and 10), all other mutants showed a certain degree of reduction in their binding activity to either RNA probes (Figure 2b). As a control, a circular dichroism analysis of the two corresponding R76A and Y94A single mutants was performed to check whether the mutant proteins were properly folded. The spectra obtained for these two mutants were similar to the wild-type protein, ruling out any drastic conformational change introduced by the mutations (data not shown).As none of the single mutations introduced could totally abrogate the RNA-binding activity of the IBV N-terminal domain, two mutant constructs carrying double mutations were made. As can be seen in Figure 2b (lanes 13 and 14), mutations of both Arg-73 and Lys-78 to alanine residues (R73/K78A) reduced the RNA-binding activities to the positive- and negative-sense RNA probes between 44 and 10% of wild-type, respectively. Mutations of both Arg-76 and Tyr-94 (R76/Y94A) to alanine reduced the RNA-binding to the positive- and negative-sense RNA probes to only 7 and 19% of wild-type, respectively (Figure 2b, lanes 15 and 16). Taken together, these results are consistent with the presence of an extended RNA binding site covering a large part of the accessible surface of the N-terminal domain of the coronavirus N protein. They are also in agreement with the assumption that removal of a single positive charge at the protein surface by mutation of a lysine or arginine residue to an alanine is insufficient to totally disrupt the formation of the protein&#8211;RNA complex.\nIntroduction of single amino acid substitutions into the IBV genome and analysis of their effects on the recovery of infectious viruses\nThe R76A and Y94A mutations were first introduced into an infectious clone of IBV, and in vitro transcribed full-length RNA molecules derived from wild type and mutant constructs were generated by in vitro transcription using the T7 RNA polymerase in the presence of a cap analog. Electroporation of wild-type RNA transcripts together with an RNA fragment covering the N protein region into Vero cells showed the formation of massive CPE at 2 days post-electroporation (Figure 3, panel rIBV+N). In contrast, no CPE formation was observed in cells transfected with transcripts derived from either mutant up to 5 days post-electroporation, suggesting that no infectious virus could be rescued from the two mutant transcripts (Figure 3, panels R76A+N and Y94A+N). These results indicate that R76A and Y94A mutations, which severely reduce the RNA-binding activity of the N-terminal domain of the N protein, may abolish the infectivity of the in vitro synthesized full-length IBV transcripts.Introduction of Q74A and Y92A mutations into the full-length infectious clone of IBV was then carried out. Typical CPEs were observed in cells transfected with Q74A mutant transcripts at 3 days post-electroporation and recombinant viruses were recovered (Figure 3, panel Q74A). In cells electroporated with Y92A transcripts, a typical CPE was observed at 2 days post-electroporation [Figure 3, panel Y92A(1)]. Incubation of the transfected cells for one more day saw the appearance of more CPEs [Figure 3, panel Y92A(2)]. However, prolonged incubation of the transfected cells up to 6 days saw no spread of the infection from the two initiated infected loci to neighboring cells. Re-infection of fresh Vero cells with either medium collected from these cells or total cell lysates prepared by freezing and thawing of the transfected cells showed no detection of the CPE formation, indicating that the minute amount of viruses rescued from the initially transfected cells could not maintain infectivity in subsequent passage.The R76A and Y94A mutant N constructs were then used to test if they could facilitate the rescue of the in vitro synthesized full-length IBV transcripts in trans. As mentioned, efficient recovery of infectious virus from the in vitro transcribed full-length IBV RNA requires co-transfection of these in vitro transcripts together with the in vitro synthesized RNA covering the N gene. Electroporation of wild-type full-length IBV transcripts together with RNAs synthesized in vitro from wild-type and the two mutant N constructs, respectively, showed efficient recovery of the infectious virus with CPE observed on almost the whole monolayer at 3 days post-electroporation (Figure 3, panels rIBV+N-R76A and rIBV+N-Y94A). In contrast, cells transfected with the full-length IBV transcripts only (without co-electroporation with the N transcripts) showed the formation of a single CPE on the whole monolayer at 3 days post-electroporation (Figure 3, panel rIBV-N).\nAnalysis of negative-strand RNA replication and subgenomic RNA transcription in cells transfected with wild-type and mutant full-length transcripts\nAs no infectious virus was recovered from cells transfected with R76A and Y94A mutant transcripts, total RNA was extracted from cells electroporated with wild-type and mutant full-length transcripts and RT&#8211;PCR amplification of subgenomic mRNAs was carried out to check whether a low level of RNA replication and subgenomic mRNA transcription occurred in cells transfected with the mutant transcripts. The forward primer used in this reaction corresponds to the leader sequence from 26&#8211;46 nt in the genomic RNA and the downstream primers covers IBV sequences from 24&#8201;784 to 24&#8201;803 nt. If transcription of subgenomic mRNAs did occur, a 415 bp PCR product corresponding to the 5&#8242;-terminal region of the subgenomic mRNA4 and a 1010 bp fragment corresponding to the 5&#8242;-terminal region of the subgenomic mRNA3 would be expected. As shown in Figure 4a, a dominant 415 bp band and a weak 1010 bp band were observed in cells electroporated with wild-type full-length transcripts at 2 days post-electroporation (lane 2). Sequencing of the PCR fragments confirmed that they represent the correct sequences of the corresponding regions of the subgenomic mRNAs 3 and 4, respectively. The same PCR products were not detected in cells electroporated with the R76A and Y94A mutant transcripts, respectively, at 24, 48 and 72 h post-electroporation (Figure 4a, lanes 3&#8211;8). In cells electroporated with Q74A mutants at 3 days post-electroporation, RT&#8211;PCR amplification of the subgenomic mRNA 3 and 4 showed the detection of the 415 and 1010 bp bands (Figure 4a, lane 9). The same PCR products were not evident in cells electroporated with the Y92A mutant transcripts (Figure 4a, lane 10).\nRT&#8211;PCR amplification of the negative strand RNA was performed to check if RNA replication occurred in these transfected cells. The primer pair was chosen so that the IBV sequence from 14&#8201;931 to 15&#8201;600 nt would be amplified by the RT&#8211;PCR. If replication of viral RNA occurred, a 670 bp PCR fragment would be expected. As shown in Figure 4b, RT&#8211;PCR fragments amplified from both positive (lanes 4, 7 and 9) and negative (lanes 3, 6 and 10) strand RNA templates were obtained from cells transfected with the mutant transcripts. The amount of the negative strand RNA was approximately half of that of the positive strand RNA (Figure 4b). As positive control, the same RT&#8211;PCR fragment for the negative strand RNA was observed in cells transfected with wild-type transcripts (Figure 4b, lanes 8 and 11). In the negative controls, total RNA extracted from cells transfected with Y94A (Figure 4b, lane 2) and R76A (Figure 4b, lane 5) was analyzed by PCR directly using the primer set for the negative strand RNA. No corresponding PCR fragment was detected from these samples, demonstrating that the detection of the negative strand RNA in cells transfected with wild-type and mutant constructs by RT&#8211;PCR is due to the replication of viral RNA. These results confirm that transcription of the negative strand RNA has taken place in cells transfected with the mutant transcripts.The transcription of negative strand RNA in cells electroporated with wild-type, R76A, Y92A and Y94A transcripts was then quantitatively determined and compared by real time RT&#8211;PCR at 24 and 48 h post-electroporation, respectively. Compared to cells transfected with wild-type transcripts, the relative amounts of negative strand RNA in cells transfected with R76A, Y92A and Y94A were 48, 46 and 50%, respectively, at 24 h post-electroporation (Figure 4c, lanes 1&#8211;4). These figures were dropped to 41, 45 and 23%, respectively, at 48 h post-electroporation (Figure 4c, lanes 5&#8211;6). Taken together, these results demonstrate that transcription of the negative strand RNA has taken place in cells transfected with the mutant transcripts. As the level of negative strand RNA was not increased in cells transfected with the mutant transcripts over a time-course experiment, it suggests that transcription of these RNA species may occur only in the initially transfected cells using the in vitro transcripts as templates.\nGenetic stability and growth properties of the Q74A mutant virus\nThe growth properties of the Q74A mutant virus on Vero cells were tested by analysis of plaque sizes and growth curves of passage 3 mutant virus. Compared to cells infected with wild-type recombinant virus (rIBV), slightly smaller sized plaques were observed in cells infected with the Q74A mutant virus (Figure 5a). Analysis of the growth curves of wild-type and the mutant virus demonstrated that the mutant virus grew slightly more slowly than the wild type recombinant virus. When 10-fold more mutant virus was used, 2- to 10-fold more mutant virus was produced at 8&#8211;24 h post-infection (Figure 5a). The titers of the mutant virus dropped more rapidly than wild-type virus at 36 and 48 h post-infection (Figure 5a).\nFurther characterization of the mutant virus was subsequently carried out by analysis of the genomic and subgenomic RNA, and the expression of viral structural proteins S and N. Northern blot analysis of cells infected with the 5th passage of rIBV and Q74A mutant viruses showed that similar amounts of genomic and subgenomic RNAs were detected (Figure 5b). Quantitative comparison of the subgenomic RNA synthesis by densitometry analysis of the bands shown in Figure 5b demonstrated that in cells infected with Q74A mutant virus (p5), the relative amounts of mRNA 2, 3, 4, 5 and 6 are &#8764;98, 101, 121, 104, 106 and 101%, respectively, of those in cells infected with rIBV. However, Western blot analysis of cells infected with rIBV and Q74A mutant virus showed that more S and N proteins were detected in cells infected with wild-type virus (Figure 5c, lanes 1&#8211;14).To confirm if the Q74A mutation is genetically stable during passage of the mutant virus in cells, viral RNA was isolated from cells infected with passage 2 and 5 virus and amplified by RT&#8211;PCR. Sequencing analysis showed that the two mutated nucleotides (CA-GC) remained stable (Figure 5d). Furthermore, no compensatory mutation was found in other region of the N protein, confirming that Q74A mutant virus is stable in cell culture system.\nDISCUSSION\nThe intrinsic RNA-binding activity of the nucleocapsid protein of RNA viruses is essential for packaging the RNA genome into a RNP complex. In previous structural studies of the N protein from two coronaviruses, the N-terminal domain was shown to be resistant to proteolysis and to fold independently (14,15). It serves as a functional unit critical for the RNA-binding activity of the protein (14). Based on these structural results, the importance of individual residues in RNA-binding was studied. Here we showed that Tyr-94 in strand &#946;3 binds to the RNA ligand. The positively charged Arg-76 is located in the immediate vicinity of this aromatic residue at the base of the extended flexible hairpin loop, within a cluster of positively charged residues. Thus, the interaction between Arg-76 and Tyr-94 with RNA appears to play a crucial role during virus assembly. Since no single mutation could totally disrupt nucleic acid binding, other aromatic or basic residues located close to these two residues at the surface of the N-terminal domain are likely to contribute to RNA binding by creating a broad surface that makes contact with the genomic RNA. Using a simple docking experiment, the RNA binding site seems capable of accommodating up to 8&#8211;9 nt bases per N-terminal domain monomer. The Arg-76 and Tyr-94 residues identified in this paper thus provide more precise mapping of the location of the RNA-binding site on the IBV N protein.It is interesting to note that the main effect of these mutations on viral life cycles was on subgenomic RNA transcription. However, the replication of negative strand RNA is nearly normal in cells transfected with these mutant constructs. Is it possible that these mutations will directly impair an undefined function of the N protein that is essential for subgenomic RNA transcription? Two possibilities were considered. First, the coronavirus N protein is directly involved in subgenomic RNA replication. The MHV N protein was shown to be involved in coronaviral RNA transcription (4), but the underlying mechanism was unrevealed. Alternatively, the observed phenotypic changes in subgenomic RNA replication caused by these mutations may be due to a generally reduced viral replication rate and infectivity. In fact, several other mutations that affect viral replication cycles were found to have defects in subgenomic RNA transcription (25&#8211;27). In this study, we have repeatedly observed that detection of subgenomic RNA replication in cells infected with low multiplicity of infectivity of IBV by RT&#8211;PCR is technically challenging. Positive results can be obtained only if transcription of subgenomic RNA reaches a certainly high level. When in vitro transcripts containing Y92A mutation were introduced into cells, one or two CPEs were always observable in a monolayer, suggesting the presence of low level viral RNA replication. However, detection of subgenomic RNA was consistently unsuccessful in cells electroporated with this mutant transcript.Coronavirus N protein is a multi-functional protein. The protein was shown to be able to facilitate rescue of infectious virus from several coronaviruses by co-electroporation of the in vitro synthesized N transcripts with the in vitro synthesized full-length transcripts (17,28&#8211;30). The mechanism for this trans-effect is currently unknown. In this study, the R76A and Y94A mutant N constructs could promote rescue of the infectivity of the in vitro synthesized IBV RNA as efficiently as did the wild type N transcripts, suggesting that the reduced RNA-binding activity in the N-terminal domain does not affect this function of the N protein. It also suggests that RNA-binding per se is not a major factor that contributes to this important function of the N protein.Are the reduced replication rate and loss of the infectivity of the mutant transcripts caused by the effects of these mutations on the functionality of the N protein or on the integrity of the genomic RNA itself? The nice correlation between the reduced RNA-binding activity of the N-terminal domain from the mutant proteins and the loss of infectivity of the corresponding mutant transcripts would support that the failure to rescue the viral infectivity is due to the specific mutation that impairs the functionality of the N protein. In addition, in the cases of Q74A and R76A mutants, the two mutations are physically adjacent in the genome, but yet totally different phenotypes were observed from the two mutants. It would support that the failure to recover infectious viruses from the mutant transcripts is due to the fact that these mutations reduce/impair the RNA-binding activity of the N protein. If this were the case, however, supplement of wild type N protein in trans would rescue the infectivity of the mutant transcripts. Two experiments were conducted to test this possibility, but none of them were successful. First, co-transfection of the mutant full-length transcripts with wild-type N transcripts into cells failed to recover infectious virus from the mutant transcripts, suggesting that wild-type N RNA cannot rescue the infectivity of the mutant transcripts in trans. The second experiment was to introduce the mutant full-length transcripts into cells stably expressing a six-His-tagged IBV N protein. Once again, it failed to rescue the infectivity of the mutant transcripts. At present, it is unclear if this was due to the low-level expression of N protein in these cells or because the N protein expressed in these cells contains the six-His tag at its C-terminus.Residues Arg-76 and Tyr-94 in the IBV N protein are well conserved across the whole family of coronaviruses. Structurally, these two residues may correspond to the Arg-94 and Tyr-112 residues in the SARS-CoV N protein (see Figure 1a). It is possible that introduction of similar alanine mutations into the SARS-CoV genome may yield a similar phenotype observed here for IBV. The fact that substitutions of Gln-74, Arg-76, Tyr-72 and Tyr-94 in the IBV genome by using an infectious clone of IBV showed nice correlation between the RNA-binding activity and viral infectivity would suggest potential strategies for intervention of viral replication. A compound which could specifically bind to this region of the N protein might interfere with RNA binding and thus inhibit virus replication. In addition, this study points to a number of residues on the surface of the N protein that contribute less to the RNA-binding activity and viral infectivity for the design of attenuated coronavirus strains as potential vaccine candidates.",
        "pmid": "16971454",
        "title": "Amino acid residues critical for RNA-binding in the N-terminal domain of the nucleocapsid protein are essential determinants for the infectivity of coronavirus in cultured cells",
        "authors": "Yong Wah Tan;Shouguo Fang;Hui Fan;Julien Lescar;D.X. Liu",
        "doi": "10.1093/nar/gkl650"
    },
    {
        "keywords": "Avian coronavirus;Eclectus parrot;Infectious bronchitis",
        "abstract": "Background and Aim:\nAvian coronavirus has a wide range of hosts, from chickens and turkeys to wild birds. This virus causes an economically and, possibly, environmentally, important loss in the poultry industry. Therefore, research into the avian coronavirus in various species of birds is required. The Eclectus parrot (Eclectus roratus) is an endemic bird to Indonesia and Northern Australia and often kept as pets. At present, there has been limited information about avian coronavirus infection among birds. This study aimed to determine the presence of and to characterize avian coronavirus isolated from Eclectus parrots in Indonesia.Materials and Methods:\nCloacal swab samples were taken from 10 healthy Eclectus parrots (E. roratus). Each isolate was propagated into specific pathogen-free embryonated chicken eggs. The presence of avian coronavirus was determined using three sets of primers targeting the 3&#8217; untranslated region (3&#8217;-UTR) of avian coronavirus (UTR41+/11&#8722;), the N gene of the infectious bronchitis virus (IBVN+/&#8722;), and the S1 gene of the IBV (XCE2+/XCE2&#8722;). The infectious bronchitis vaccine strain H120 was used as a positive control. Resulting positive bands were sequenced for the S1 gene.Results:\nNone of the isolates was positive for the 3&#8217;-UTR, four isolates were positive for the N gene of infectious bronchitis, and two isolates were positive for the S1 gene of the IBV. However, only one isolate (parrot/Indonesia/BX9/16) was sequenced for the partial S1 gene with primers XCE2+/XCE2&#8722;. The partial nucleotide sequence of this isolate showed 100% homology with the IBV GI-13 lineage, specifically with a field isolate of the 4/91 variant 1 Israel and the 4/91 vaccine on the hypervariable region 3 site of the S1 gene.Conclusion:\nAn IB-like avian coronavirus was isolated from healthy Eclectus parrots. Our results indicate that IBV has a wide range of hosts, which prompt the need to understand the interspecies connection of this virus better.",
        "body": "Introduction\nAvian coronavirus is the main representative of genus Gammacoronavirus, family Coronaviridae, and order Nidovirales [1]. Within the avian coronavirus group, the infectious bronchitis virus (IBV) is among the most researched. This virus is known to cause an important disease that incurs a high economic loss in the poultry industry [2,3] despite an ongoing vaccination program. It causes respiratory disease while also affecting the kidneys and reproductive tract [4] through viremia with a severity that differs depending on serotypes [3]. Mutations and recombination have produced a high genetic diversity of the virus. In addition, vaccinations performed in the farm setting can influence the evolution of the virus [5]. Many serotypes of IBV are often not cross-protective [6,7]. Mismatching between the circulating strain and the administered vaccine may contribute to vaccination failure [7]. Ubiquitous IBV and IBV-like viruses have also been found in species other than chicken, such as in peafowl, guinea fowl, partridge, waterfowl, and teal [2,6,8-10]. This finding strengthens the possibility that IBV may have a wider range of hosts than previously thought [8,6]. Despite this, data relating to IBV in Indonesia is still limited to poultry. Studies of diseases on endemic species are valuable for the conservation effort, yet are rarely conducted.The Eclectus parrot (Eclectus roratus) is a sexually dichromatic parrot native to a part of Eastern Indonesia and Northern Australia [11]. It is classified as protected in Indonesia according to Government Decree Number 7, Year 1999 and Constitution Number 5, Year 1990. Visually captivating, with both male and female showing radically different plumage, the Eclectus parrot is naturally talkative and popular as a pet [11]. However, there has been limited information about viral diseases among Eclectus parrots. The latest finding on coronavirus in parrots was in 2006 when a virus distinct from IBV was found in the green-cheeked Amazon parrot [12]. Understanding viral diseases in Eclectus parrots may be beneficial for the conservation effort and may provide additional information about viral diseases in birds.There is limited information as to whether avian coronaviruses cause diseases in Psittacine birds; therefore, information about the presence of this virus among parrots might be valuable for the conservation effort of endemic birds and the poultry industry, which is robust in Indonesia. This study aimed to determine the presence of and to characterize avian coronavirus isolated from Eclectus parrots reared by an Indonesian local bird breeder.\nMaterials and Methods\n\nEthical approval\nThe methods performed in this research have been approved by the Ethical Committee of the Faculty of Veterinary Medicine, Bogor Agricultural University, Indonesia, which were validated with the certificate number 058/KEH/SKE/IV/2017.\nSamples\nCloacal swab samples were obtained from 10 healthy and clinically sound Eclectus parrots (E. roratus), which belong to a bird breeder in Bogor, West Java, Indonesia. Swabs were stored in viral transport media containing phosphate-buffered saline, penicillin (1000 IU/ml), and streptomycin (10 mg/ml). Samples were carried in a cooler box during transportation and stored at &#8722;80&#176;C until use. All birds had never been vaccinated. The IBV vaccine strain H120 was used as a positive control.\nVirus isolation\nAll isolates were propagated into three specific pathogen-free (SPF) embryonated chicken eggs. The swab suspension was mixed with antibiotics (penicillin 1000 IU/ml and streptomycin 10 mg/ml) in a 4:1 v/v ratio. A 0.2 ml swab suspension was inoculated through the allantoic route into a 9-day-old SPF embryonated chicken eggs and then incubated at 36&#176;C for 4 days or until the embryo died. Allantoic fluid was harvested on day 4 post-inoculation and stored at &#8722;20&#176;C until use.\nViral RNA extraction\nTotal viral RNA was extracted using the RNeasy Mini Kit (Qiagen, Germany) according to the manufacturer&#8217;s instructions. The RNA was dissolved in 30 &#181;l RNase-free water and used for amplification by reverse transcriptase-polymerase chain reaction (RT-PCR). The RNA was stored at &#8722;20&#176;C until use.\nAmplification\nRT-PCR was performed with the following primer pairs, as described in Table-1: UTR41+/11&#8722; targeting the 3&#8217; untranslated region (3&#8217;-UTR) of the avian coronavirus genome [13], IBVN+/IBVN&#8722; targeting the N gene of IB [14], and XCE2+/XCE2&#8722; targeting the S1 gene [15]. Polymerase chain reaction amplification was carried out using the Bioline One-Step RT-PCR kit (Bioline, UK) with the following thermal cycling profile: Reverse transcription at 45&#176;C for 20 min, polymerase activation at 95&#176;C for 1 min, then 40 cycles of denaturation at 95&#176;C for 15 s, annealing temperatures and times specific for each primer pair and described below, and extension at 72&#176;C for 30 s. This was followed by a final extension at 72&#176;C for 7 min. Annealing occurred at 48&#176;C for 1 min, 58&#176;C for 30 s, and 50&#176;C for 30 s for UTR41+/11&#8722; [13], the partial N gene [14], and the partial S1 gene [15], respectively.\nNucleotide sequencing\nFollowing amplification using the partial S1 gene primer XCE2&#8722;/XCE2+ pairs, 30 &#181;l of the nucleotide sample was sent to First Base (Malaysia) to be sequenced from both directions. The resulting nucleotide sequences were aligned and compared with the S1 gene available from GenBank. Nucleotides taken from GenBank were used for alignment and tree construction, including genes that represent each IBV lineage according to Valastro et al. [16], Indonesian isolates from Wibowo et al. [17], and LX4 (AY189157) and Israel variant 1 (AF0937940), to illustrate well-used QX and 4/91 variant sequences. The genes used in the analysis are listed in Table-2. A phylogenetic tree was constructed using the maximum likelihood method using MEGA 7 software (Pennsylvania, USA) with 1000 bootstrap replications. Nucleotide and amino acid distances were calculated and provided for the following genes: H120 vaccine as a positive control (FJ888351), peafowl/GD/KQ6/2003 (AY641576) as a non-chicken IBV-like virus, MHW-QX-MGX-2012 (MH671339) as an Indonesian Qx-type sample, MHW-O-NSTR-2-2018 (MH671342) as an Indonesian 4/91 sample, Moroccan G/83 (EU914938) as a GI-13 lineage representative, 4/91 Israel variant 1 (AF093794), 4/91 vaccine (KF377577), and the isolate, parrot/Indonesia/BX9/2016.\nResults\n\nIsolation and identification of avian coronavirus\nAmplification was performed on original swab samples and allantoic fluids. None of these samples was positive for the 3&#8217;-UTR of the avian coronavirus (UTR11&#8722;/41+). Three out of 10 swab samples were positive for the N gene coronavirus, but none of them was positive for the S1 IBV gene. However, four out of 10 samples of allantoic fluid tested were positive for the coronavirus N gene. Of the two that tested positive for the S1 gene of IBV, only one of them, which was designated as parrot/Indonesia/BX9/16, was able to be further sequenced. These results are presented in Table-3.\nNucleotide sequencing of the partial S1 gene\nOnly one isolate, defined as parrot/Indonesia/BX9/16, was sequenced for the partial S1 gene of IBV using XCE2+/XCE2&#8722; primers (Table-1). Nucleotide sequencing of 323 nucleotides from the partial S1 gene showed that there was no difference in the nucleotide sequence of the parrot/Indonesia/BX9/16 gene when compared with IBV 4/91 Israel variant 1 (AF093794.1) and the 4/91 vaccine strain (KF377577.1) (Figure-1). The nucleotide and amino acid pairwise distance also showed 100% homology with the IBV 4/91 Israel variant 1 (AF093794.1) and the 4/91 vaccine strain (KF377577.1). However, differences were observed between the sequenced gene, the H120 (FJ888351) positive control, and the non-chicken IBV-like peafowl/GD/KQ6/2003 virus (AY641576) (Table-4). A phylogenetic tree (Figure-2) of the aligned nucleotide sequence of the partial S1 gene was constructed using the maximum likelihood method with Mega 7 software with 1000 bootstrap value. The tree showed a close relatedness of viral isolate, parrot/Indonesia/BX9/16, to the IBV strain 4/91 variant 1 Israel (AF093794.1), the 4/91 vaccine strain (KF377577.1), CK/CH/YN/SL 1301-1 (KX107779.1), chicken/Attock/NARC-786/2013 (KU145467.1), and gammaCoV/Ck/Poland/G193/2015 (MK576138.1), whereas there were differences observed when compared with the H120 vaccine (FJ888351.1) positive control.\nDiscussion\nIBV and IB-like viruses, which are avian coronaviruses, have been detected among avian species other than chickens, such as pheasants, peafowl, turkeys, teal, pigeons, geese, penguins, quail, ducks, waterfowl, and Amazon parrots [2,9,10]. Advances in genetic identification by biomolecular technologies allow rapid identification of viruses without the need for prior isolation. However, the question remains as to whether these novel viruses cause disease in the species that they were found [8,3]. There have been few instances where isolates were obtained, such as a virus that was found in peafowl that was found to share >99% identity with H120 IBV vaccines [18] and Mass-type IBV [19]. A coronavirus detected in pigeons was found to be clustered as a Mass-type IBV [20], whereas a coronavirus found in teal has been shown to exhibit >80% genetic similarity with several IBV strains including infectious strains [21]. Interestingly, upon exposure to chickens, the peafowl IB-like virus, which is close to IBV vaccine strain H120, did not cause disease among these birds [18], whereas the IB-like Mass-type peafowl isolate, the IB-like Mass-type pigeon isolate, and the IB-like teal isolate caused infection in chickens [19,20,21]. These previous studies indicate that the IB-like virus may be found in hosts other than chickens and that they might originate from the chicken IBV or might be a novel virus. However, unless this is investigated further, it is uncertain whether the viruses found in other species of birds can cause disease in chickens.The S1 gene of IBV is often used for genotyping. It is important in determining attachment and houses most of the virus epitopes, which can affect the virus serotypes [2,8,7]. The nucleotide sequence of the partial S1 gene used in this study (XCE2+/XCE2&#8722;) only covers the hypervariable region (HVR)-3 of the S1 gene [22] but does not cover HVR 1 and HVR 2. This present study indicated that within the section of the parrot/Indonesia/BX9/16 gene that was sequenced, there was 100% similarity with several IBV 4/91 strains taken from GenBank, which included both infectious and vaccine strains, namely: IBV strain 4/91 variant 1 Israel (AF093794.1), 4/91 vaccine strain (KF377577.1), CK/CH/YN/SL 1301-1 (KX107779.1), chicken/Attock/NARC-786/2013 (KU145467.1), and gammaCoV/Ck/Poland/G193/2015 (MK576138.1). Recent publications regarding IBV in Indonesia by Wibowo et al. [17] and Setiawaty et al. [22] showed the presence of a circulating infectious 4/91 strain in Indonesia. Sequences obtained from Wibowo et al. [17] were 4/91-like IBV originating from Indonesia in 2017 to 2018. These sequences were included in our phylogenetic analysis but appeared to be slightly different from our isolated virus in each of the nucleotides numbered: 55, 80, 98, 175, 211, 223, and 225. However, two sequences from Setiawaty et al. [22] showed 100% similarity (Indonesia/K233A31/18 and Indonesia/K4A9/17) to our isolated gene and only one slight variation compared with their Indonesia/P3/17 gene. The study by Setiawaty et al. [22] showed that the 4/91 gene recovered from an outbreak may show sequence homology to the live vaccine. The study by Wibowo et al. [17] was conducted in East Java, while the study by Setiawaty et al. [22] was conducted in West Java, which is in close proximity to the place that this research was conducted. This might explain why, in our study, we were able to isolate an IB-like virus that was close in homology to the IBV 4/91 strain. This also indicates that the 4/91 strain is widespread in Java Island, Indonesia.According to Valastro et al. [16], the IBV 4/91 strain and 793.B belong to the IBV GI-13 lineage. This lineage is among the most widely distributed lineage in the world. This has mostly contributed to the widespread usage of vaccinations [16]. However, in Indonesia, vaccines commonly used in poultry are from the GI-1 lineage, which consists of Mass-type IBV such as M41, H52, or H120 strains [17,22]. Information dating back to 1977 and 1985 states that at that time, circulating IBV in Indonesia was close to the IBV Massachusetts, IBV Connecticut, and Australian strains [23]. The IBV variant strain was first detected in Indonesia in 2003 [24]. A recent study of circulating IBV in outbreaks among chickens in Indonesia showed that the virus isolated in 2012 in East Java was Qx-like and was similar to China and Taiwan strains in 2017 [25]; whereas, IBV isolated in 2018 was found to be close in similarity to the IB 4/91 strain [17,22]. In general, IBV strains within the same serotype share >95% amino acid similarity and those with <85% similarity usually belong to different strains [2]. A slight change in the S1 gene may affect the serotype of the virus and cross-protection [2,6]. This could probably explain why our virus was similar to vaccine and field viruses at the same time. Despite the viruses having 100% homology on the site we sequenced, differences in other parts of the genome may cause differences in virulence. Considering how 100% homology was obtained with both the vaccine virus and field strains, it is difficult to determine the origin or infectivity of the virus found. The birds sampled in this research were from a breeding institution without any vaccination program; thus, the virus found did not develop from any attempted vaccination program to said captive birds. The birds sampled appeared to be healthy, but it is unknown whether the isolated virus can cause infection in chickens. The birds were also born and bred in the same institution, so the path of infection is unknown. Further research on biosecurity, including factors such as water quality or the possibility of contact with wild birds, is needed to identify the origin of infection.Evidence of infections by the IBV vaccine virus or the infectious IBV strain to birds of other species has been reported before. Reverse spillover of the vaccine virus from domesticated birds to wild birds has been proposed by Devlin et al. [5] and Rohaim et al. [26]. Our results were similar to the previous study by Ito et al. [18], which discovered that IB-like coronavirus from peafowl has close genetic relatedness with the IBV H120 vaccine strain. This previous study also showed that the vaccine strain may infect non-chicken birds and may have few mutations that, upon infection to chickens, do not inflict disease. Liu et al. [21] have also suggested that not only the vaccine strains but also a possible infectious IBV strain may be transmitted from poultry to birds of other species. For example, cross-transmission of nephropathogenic IBV virus occurred in teal housed near domesticated poultry [21]. This means that IBV can infect a variety of bird species. In a possible case where infected birds are wild birds, these birds may act as a reservoir of the virus and spread it to other susceptible species [27]. Transmission of the virus to wild birds has likely happened due to poor biosecurity in poultry farms and contamination from improper disposal of litter and waste [5]. Most of the Indonesian poultry farmers still rely on an Open House Coop. A high density of birds without effective biosecurity and disease prevention increases the possibility of viral mutation [8]. Contamination from improper disposal of litter and waste may ease the transmission from domesticated poultry to wild birds. Spillover to wild birds is often overlooked, especially in terms of its ecological impact. This is especially important for endangered species. However, the epidemiology and effect of spillover from domesticated birds to wild birds are not yet fully understood, despite evidence of its occurrence.Not only impacting wild birds but also this spillover effect is important for the poultry industry. Other RNA avian viruses with similarly high mutation rates such as avian influenza (AI) and Newcastle disease virus (NDV) illustrate the importance of disease transmission between poultry and wild birds. The emergence and spread of highly pathogenic AI often contribute to its presence in wild birds [28,29]. In the case of AI, mutation and recombination possibly happened in wild birds and emerged as a highly pathogenic virus in poultry to a devastating effect. Its spread to various continents might also be attributed to migrating waterfowl carrying the virus [29]. Persistent outbreaks, despite a rigorous vaccination effort, such as what has happened in IBV, can be seen in NDV [30]. Studies on both free-living birds and poultry show how NDV has circulated and continued to evolve in the environment [30,31]. These cases, although they describe other viruses, illustrate the importance of biosecurity and surveillance to prevent possible emerging virulent viruses in the future through the transmission of disease between free-living birds and domestic birds. This is especially important for developing countries where many still practice backyard farming and small-scale farming with low biosecurity [28,31], such as those in Indonesia. Unfortunately, interspecies studies of IBV are rarely conducted in the country. This study is the first to have studied IBV in a species of bird other than chicken in Indonesia.The similarity between viruses isolated from captive birds and domesticated poultry, as found in this research, is alarming, as there is an indication of a reverse spillover effect that has already happened from poultry farms to the environment. Our findings also indicated that IBV has a broad range of hosts, opening more possibility of a spillover effect to happen on environmental contamination. This finding was in accordance with the previous study that isolated avian coronavirus in waterfowls and teal that did not show any symptoms of the disease [19,21]. There is a possibility that wild birds and other non-galliform birds may harbor IBV or IB-like coronavirus without showing any symptoms and illustrate the wide host range of coronavirus.\nConclusion\nIB-like coronavirus isolated from the Eclectus parrot from an Indonesian bird breeder has 100% homology with IBV 4/91 variant 1 Israel, a 4/91 vaccine strain, CK/CH/YN/SL 1301-1, chicken/Attock/NARC-786/2013, and gammaCoV/Ck/Poland/G193/2015. IB-like coronavirus was isolated from healthy parrots, indicating that this bird might act as a reservoir for coronavirus.\nAuthors\u2019 Contributions\nGKS performed the work. EH, RDS, AS, and ONP contributed to designing the study and method. All authors contributed to the writing and revision of the manuscript. All authors read and approved the final manuscript.",
        "pmid": "32009759",
        "title": "Isolation and characterization of avian coronavirus from healthy Eclectus parrots (Eclectus roratus) from Indonesia",
        "journal_title": "Veterinary World",
        "authors": "G. K. Suryaman;R. D. Soejoedono;A. Setiyono;O. N. Poetri;E. Handharyani",
        "doi": "10.14202/vetworld.2019.1797-1805"
    },
    {
        "keywords": "severe acute respiratory syndrome;SARS virus;coronavirus;electron microscopy;immunogold techniques;in situ hybridization;ultrastructure;replication complex;emerging infectious diseases",
        "abstract": "Severe acute respiratory syndrome (SARS) was first described during a 2002&#8211;2003 global outbreak of severe pneumonia associated with human deaths and person-to-person disease transmission. The etiologic agent was initially identified as a coronavirus by thin-section electron microscopic examination of a virus isolate. Virions were spherical, 78 nm in mean diameter, and composed of a helical nucleocapsid within an envelope with surface projections. Herein, we show that infection with the SARS-associated coronavirus resulted in distinct ultrastructural features: double-membrane vesicles, nucleocapsid inclusions, and large granular areas of cytoplasm. These three structures and the coronavirus particles were shown to be positive for viral proteins and RNA by using ultrastructural immunogold and in situ hybridization assays. In addition, ultrastructural examination of a bronchiolar lavage specimen from a SARS patient showed numerous coronavirus-infected cells with features similar to those in infected culture cells. Electron microscopic studies were critical in identifying the etiologic agent of the SARS outbreak and in guiding subsequent laboratory and epidemiologic investigations.",
        "body": "Methods\nInfected and uninfected Vero E6 cells were harvested 3&#8211;5 days after inoculation, inactivated by fixation and gamma irradiation (2 &#215; 106 rad), and processed for standard, immunolabeling electron microscopy (IEM) or ISH EM as previously described (6,14). For standard EM, glutaraldehyde- and osmium tetroxide&#8211;fixed specimens were embedded in Epon-substitute and Araldite (Ted Pella, Inc., Redding, CA) and sections were stained with uranyl acetate and lead citrate. Some infected and uninfected cultures were treated with 5% tannic acid solution before being embedded for standard EM (15). Specimens prepared for IEM and ISH assays were fixed in paraformaldehyde and glutaraldehyde and embedded in LR White resin (Ted Pella, Inc.), and sections were collected on nickel mesh grids.A BAL specimen was obtained from a 47-year-old man within the first week of the onset of symptoms. A portion of the specimen was centrifuged at 2,000 rpm for 10 min, and the pellet was processed for standard EM.IEM and ISH assays were performed essentially as described for Nipah virus (14). In brief, for IEM assays, sections were reacted with hyperimmune mouse ascitic fluid raised against SARS-CoV and then with a goat anti-mouse antibody conjugated to 12-nm colloidal gold particles (Jackson ImmunoResearch Laboratories, Inc., West Grove, PA). Negative-sense riboprobes for the ultrastructural ISH assays were prepared as previously described (16,17). Riboprobes were directed against the nucleocapsid or polymerase protein portions of the SARS-CoV genome (Table) and incorporated digoxigenin-11-dUTP. Because of the nested set structure of the coronavirus genomic RNA (genRNA) and messenger RNAs (mRNAs), the nucleocapsid riboprobe would detect all viral RNAs (18). Sections were reacted with a pool of nucleocapsid and polymerase probes and then with a sheep anti-digoxigenin antibody conjugated to 6-nm colloidal gold particles (Electron Microscopy Sciences, Hatfield, PA). To obtain negative controls, we performed both assays with uninfected Vero E6 cells, and infected cells were reacted with an unrelated antibody and probe for IEM and ISH procedures, respectively.\nResults\n\nUltrastructural Characteristics of SARS-CoV\u2013Infected Culture Cells\nThe morphologic features of SARS-CoV isolates were similar to those of other members of the family Coronaviridae. Multinucleated syncytial cells were occasionally seen. Nascent particles were formed by the juxtaposition of viral nucleocapsids along cytoplasmic membranes of the budding compartment (the membrane region between the rough endoplasmic reticulum and the Golgi complex) or occasionally on the membranes of the rough endoplasmic reticulum that form the outer layer of the nuclear membrane. Virions acquired an envelope by budding into the cisternae and formed mostly spherical, sometimes pleomorphic, particles that averaged 78 nm in diameter (Figure 1A). Cross-sections through the helical nucleocapsid were seen apposed to the viral envelope, and the interior of the particles was usually electron-lucent. Surface projections were faint in standard thin-section preparations and could be better visualized by using a tannic acid treatment (Figure 1A, inset).Virus particles were seen in membrane-bound vesicles, either as single particles or as groups in enlarged vesicles. In some of these vesicles, dense, granular material was seen interspersed between the virions (Figure 1B,C). Tubular structures, averaging 20 nm in diameter, were seen within some virion-containing vesicles (Figure 1D). The vesicles appeared to migrate toward the cell surface and fuse with the plasma membrane, releasing the viral particles (Figure 1E). Many of the particles adhered to the plasma membrane, creating a knob-like appearance on the surface of the cells.Viral proteins and RNA were detected in virions by IEM and ISH (Figure 2A,B), and in association with double-membrane vesicles (Figure 3A,B), nucleocapsid inclusions, and large granular areas of cytoplasm (Figure 4C,D). Double-membrane vesicles have been noted in other coronavirus-infected cells (19,20) and consist of cytoplasmic vesicles with two tightly apposed membranes (Figure 1B). In contrast, double-membrane vesicles in SARS-CoV&#8211;infected Vero E6 cells typically were composed of accumulations of multiple single-membrane vesicles enclosed within an outer membrane (Figure 3C), and virus particles were sometimes located between the two membranes (Figure 3D). Many double-membrane vesicles contained diffuse, granular material. Cytoplasmic inclusions of darkly staining viral nucleocapsids were mostly found in association with virus-containing vesicles or double-membrane vesicles (Figures 1B and 3D). Large, ill-defined areas of cytoplasm, containing ribosomelike and filamentous structures and devoid of other organelles, were noted in some SARS-CoV&#8211;infected cells (Figure 4A,B). These areas strongly labeled for viral proteins and RNA (Figure 4C,D), with IEM and ultrastructural ISH assays. No antigens or RNA were detected by reacting hyperimmune mouse ascitic fluid or riboprobes with uninfected Vero E6 cells or by reacting an unrelated hyperimmune mouse ascitic fluid or riboprobe with SARS-CoV&#8211;infected cells.Finally, as has been reported previously for other coronaviruses, SARS-CoV&#8211;infected cells also contained tubuloreticular structures, with virions sometimes forming along the membranes (Figure 3C). The tubuloreticular structures were often found in close association with double-membrane vesicles.\nUltrastructural Characteristics of SARS-CoV\u2013Infected BAL Specimen\nA number of coronavirus-infected cells were seen within a BAL specimen from a SARS patient (Figure 5A,B). Virus particles budded into, and were associated with, vesicles, and extracellular virions covered the exterior surface of the cells. Areas of double-membrane vesicles containing a diffuse granular material were also seen.\nDiscussion\nDuring the global SARS outbreak of 2002 to 2003, a virus was isolated from human patients and identified by EM as belonging to the family Coronaviridae (6,7). Detailed studies described here on the morphogenesis of the SARS-CoV by thin-section EM found many characteristics previously described for coronaviruses (19,21,22). Virus particles formed upon membranes of the &#8220;budding compartment,&#8221; a term used to describe the continuous membrane system from the rough endoplasmic reticulum to the Golgi complex (23,24). Virions accumulated in dilated vesicles that appeared to migrate to the cell surface where the virus particles were released or remained adherent to the plasma membrane. Additional cytoplasmic structures associated with coronavirus infections included nucleocapsid inclusions and double-membrane vesicles, which have been proposed as the replication complex for coronaviruses (20), and arteriviruses (25), a closely related virus family that, in addition to coronaviruses, is a member of the order Nidovirales. IEM and ultrastructural ISH assays detected viral proteins and mRNA or genRNA associated with virions, double-membrane vesicles, and nucleocapsid inclusions. Coronaviruses are known to synthesize a nested set of subgenomic mRNAs (26), such that the nucleocapsid riboprobe used here allowed detection of all viral mRNAs in addition to genRNA. Indeed, considerable amounts of RNAs were detected in the ultrastructural ISH assays performed on SARS-CoV&#8211;infected cells.As has been reported for other coronaviruses, additional cytoplasmic features were associated with SARS-CoV&#8211;infected cells. Tubular structures were occasionally seen within virus-containing vesicles (27,28); and cytoplasmic tubuloreticular structures, known to occur with numerous other infections (see 29), were also found. Large granular areas of cytoplasm, relatively devoid of organelles and containing viral proteins and RNA, were noted in SARS-CoV&#8211;infected cells; such features have not been described previously for coronaviruses. While the role of these cytoplasmic areas is unclear, the close proximity of cellular ribosomes with viral proteins and RNA suggests that they may be viral translation centers. Future ultrastructural ISH and IEM studies to characterize these areas, using riboprobes and monoclonal antibodies to specific SARS-CoV genes and gene products, should help clarify this issue.Many of these ultrastructural findings were also observed in a BAL specimen from a SARS patient (Figure 5B) (6). Characteristic virions in vesicles and lining the cell surface and the presence of double-membrane vesicles provided clear evidence of a coronavirus infection and suggested that viral replication was occurring in the lower airways early in the course of infection. EM examination of BAL specimens may prove to be a useful tool in the diagnosis of SARS-CoV, analogous to the use of BAL specimens to diagnose influenza infections. Recent studies have reported finding coronavirus particles in lung and gastroenteric tissues of SARS patients and experimentally infected macaques (7,30&#8211;33), although the viral nature of these structures has not been confirmed by IEM or ultrastructural ISH assays. Coronavirus particles may be confused morphologically with other nonviral structures routinely found in cells, including coated vesicles, multivesicular bodies, perichromatin granules, glycocalyceal bodies, and cellular projections (29). Therefore, a cautious approach is advisable when examining clinical specimens.The SARS outbreak is a prime example of an emerging infectious disease that can rapidly and easily spread, reaching global proportions. With SARS, as with previous investigations of outbreaks involving such viruses as Ebola (34&#8211;36), Hendra (37), Nipah (38), and more recently, monkeypox (39), EM played an essential role in determining the specific virus family of the pathogen involved. In all of these cases, tissue culture amplification of a virus isolate facilitated the ultrastructural examination. Thus, traditional microbiologic and EM approaches proved pivotal in determining the etiologic agents, thereby guiding subsequent laboratory and epidemiologic investigations.",
        "pmid": "15030705",
        "title": "Ultrastructural Characterization of SARS Coronavirus",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Cynthia S. Goldsmith;Kathleen M. Tatti;Thomas G. Ksiazek;Pierre E. Rollin;James A. Comer;William W. Lee;Paul A. Rota;Bettina Bankamp;William J. Bellini;Sherif R. Zaki",
        "doi": "10.3201/eid1002.030913 "
    },
    {
        "keywords": "Human coronavirus HCOV-NL63;clinical features;pathogenesis;diagnosis.",
        "abstract": "Even though coronavirus infection of humans is not normally associated with severe diseases, the identification of the coronavirus responsible for the outbreak of severe acute respiratory syndrome showed that highly pathogenic coronaviruses can enter the human population. Shortly thereafter, in Holland in 2004, another novel human coronavirus (HCoV-NL63) was isolated from a seven-month old infant suffering from respiratory symptoms. This virus has subsequently been identified in various countries, indicating a worldwide distribution. HCoV-NL63 has been shown to infect mainly children and the immunocommpromised, who presented with either mild upper respiratory symptoms (cough, fever and rhinorrhoea) or more serious lower respiratory tract involvement such as bronchiolitis and croup, which was observed mainly in younger children. In fact, HCoV-NL63 is the aetiological agent for up to 10% of all respiratory diseases. This review summarizes recent findings of human coronavirus HCoV-NL63 infections, including isolation and identification, phylogeny and taxonomy, genome structure and transcriptional regulation, transmission and pathogenesis, and detection and diagnosis.",
        "body": " INTRODUCTION\nRegardless of geographic location, respiratory tract infections rank among the top three killers of children under five years of age [1]. A significant proportion of these respiratory tract infections have no known cause. Recently, however, a number of novel coronaviruses have been identified as the causative agents for some of these infections [2,  3].Coronaviruses (CoVs) belong to the family  Coronaviridae in the order nidovirales. Members of the Coronavirus family are positive-strand RNA viruses with large genomes ranging in size from 27&#8211;33 kb. The coronavirus genome encodes for a 5&#8242; replicase polyprotein (ORF1a and ORF1b) that, in turn, encodes for all the enzymes required for viral RNA replication. The genome also encodes for the 3&#8242; structural proteins, including spike (S), envelope (E), membrane (M) and nucleocapsid (N), which are common to all coronaviruses. The structural proteins are involved in various viral processes, including virus particle formation [4]. Additional subgroup-specific accessory genes are found interspersed among the structural genes, which vary in number and location. Recent studies have shown that the proteins encoded by these genes could be modulators of pathogenicity in the natural host [5-7].Five human coronaviruses have been identified to date, four of which are known to continuously circulate in the human population, especially in young children [8, 9]. HCoV-OC43 and HCoV-229E, first identified in the mid-1960s [10,  11], were shown to cause the common cold [12], but rarely infections of the lower respiratory tract [3]. A third human coronavirus, which causes severe acute respiratory syndrome, SARS-CoV, was identified in 2003 [13, 14]. This virus had a worldwide spread, causing acute respiratory illness with a mortality rate of ~10% [15]. The last reported SARS-CoV infections were laboratory acquired in 2004, and the virus has not been detected in the human population since [16, 17]. More recently, two additional human coronaviruses were identified; HCoV-HKU1 was isolated from a 71-year-old man who presented with fever and cough [3], and HCoV-NL63 isolated from a seven-month-old baby [2]. The latter is the topic of this review.Several groups have studied different aspects of HCoV-NL63 infections, including its worldwide distribution, its association with human disease, and the replication characteristics of the causative virus. In this review we summarize recent findings of human coronavirus HCoV-NL63 infections, including virus isolation and identification, phylogeny and taxonomy, genome structure and transcriptional regulation, transmission and pathogenesis, and detection and diagnosis.\nISOLATION AND IDENTIFICATION OF HCoV-NL63\nHCoV-NL63 was first isolated in Amsterdam in 2004 from the nasopharyngeal aspirate of a seven-month old child; the patient presented with symptoms suggesting respiratory tract infection (coryza, conjunctivitis, and fever), while his chest X-ray showed typical features of bronchiolitis. The aspirate tested negative for all known respiratory viruses. A group of Dutch scientists found that the virus initiated a cytopathic effect when inoculated onto tertiary monkey kidney cells. The group used a new technique, VIDISCA, to clone and amplify the viral genome. VIDISCA is a novel approach that provides a fast and effective tool for amplification of unknown genomes based on cDNA-amplified fragment length polymorphism [2,  18]. The virus was identified as a member of the  Coronaviridae  family. It was shown to be a novel member of Group I coronaviruses because of the similarity of its genome sequence to HCoV-229E. Another group in the Netherlands [19] reported the independent isolation and identification of, essentially, the same virus at about the same time.\nPHYLOGENY AND TAXONOMY OF HCoV-NL63\nBased on antigenicity, genome organization and sequence homology, coronaviruses are divided into three distinct groups [20]. Group 1 contains transmissible gastroenteritis virus (TGEV), porcine epidemic diarrhea virus (PDEV), feline infectious peritonitis virus (FIPV), canine coronavirus and HCoV-229E, among others. Group 2 contains mouse hepatitis virus (MHV), bovine coronavirus, haemagglutinating encephalomyelitis virus, HCoV-HKU1 and HCoV-OC43, to name a few, with Bat SARS-CoV and SARS-CoV considered distantly related Group 2b coronaviruses. Group 3 contains the avian coronaviruses [21-23]. Based on phylogenetic analysis, HCoV-NL63 belongs to the Group I coronaviruses [2,  24]. Interestingly, evidence of recombination during the evolution of HCoV-NL63 has been reported, and viral isolates have, in fact, a mosaic genome structure. The authors speculate that HCoV-NL63 diverged from a HCoV-229E ancestor in the past, followed by a separation into two distinct HCoV-NL63 lineages. These two lineages recombined during co-infection, giving rise to the two currently observed genotypic subgroups [24-27]. In fact, recombination between different HCoV-NL63 isolates has been suggested, resulting in a mixture of clinical virus variants circulating in the human population [8, 27-30].\n Genome Structure and Transcriptional Regulation\nHCoV-NL63 has a single-stranded RNA genome that is capped and polyadenylated [27]. The genome is 27553 bases in size, with the genome order 5&#8242;-ORF1a-ORF1b-S-ORF3-E-M-N-polyT-3&#8242; (Fig. 1). Seven distinct ORFs are produced from six distinct mRNAs, which include the full-length genomic RNA and a nested set of five subgenomic (sg) mRNAs [4]. Coronavirus mRNAs are generated in the membrane-associated replication centers [31]. The five sg mRNAs encode for the viral structural and accessory proteins S, ORF3, E, M and N. With the exception of ORF E, a common transcription regulatory sequence (TRS), with core sequence AACUAAA, is located upstream of all the ORFs; this TRS is crucial for sg mRNA formation [27,  32]. HCoV-NL63 uses a discontinuous replication strategy to generate sg mRNAs during the minus strand synthesis [4,  27,  32], which are then copied into plus strand mRNAs. All plus strand mRNAs share a common ~70 nucleotide leader sequence at their 5&#8217; ends that is identical to the sequence at the 5&#8217; end of the genomic RNA [27].\nProtein 1a/1b\nHCoV-NL63 ORF1a/1b contains a putative elaborated pseudoknot structure that triggers a -1 ribosomal frameshift to translate the complete 1ab polyprotein; for a mini-review of the putative ORF1a/1b products and functions see Pryc et al. (2007) [27] and Van der Hoek et al. (2006) [8]. Chen and colleagues identified processed products nsp3 and nsp4 of the HCoV-NL63 replicase polyprotein, which could be detected at 24 hours post-infection. These products localize in the peri-nuclear sites of virus infected cells. Also, the group identified and characterized two viral papain-like proteases, PLP1 and PLP2, which process the viral replicase polyprotein. Interestingly, the PLP2 protease has deubiquitinating activity [9], although the function of this is not clear in viral replication.\nSpike Protein\nThe species tropism and virulence of a particular coronavirus are largely determined by the spike (S) glycoprotein. Coronavirus S proteins mediate attachment to the cellular receptors and subsequent fusion of the virus and cell membrane [20]. The N-terminal portion of HCoV-NL63 S contains a unique 179 amino acid domain not present in other coronaviruses. This region represents the most variable region of the HCoV-NL63 genome and a role in immune evasion for this region has been proposed [8, 27]. HCoV-NL63 S is a single-chain glycoprotein and consists of an N-terminal receptor-binding domain (S1) and a C-terminal transmembrane fusion domain (S2). S2 consists of two highly conserved heptad-repeat (HR) sequences that are larger than the corresponding regions for the group II and group III coronaviruses [20, 27, 33]. Initial proteolytic studies of the S2 fusion core identified an &#945;-helical domain consisting of a trimer of the HR segments N57 and C42. The resolved crystal structure of this trimeric complex shows distinctive high-affinity conformations of interacting cross-sectional layers of six helices. It has been suggested that the larger HR regions of the group I coronaviruses may be required to prime the S proteins for their fusion-activating conformational changes during entry of the virus [20].\n Accessory Protein ORF3\nCoronavirus genomes contain accessory genes found interspersed among the structural genes, which vary in number and location between the different coronavirus groups (Fig. 1). Both HCoV-NL63 and HCoV-229E are group I coronaviruses and encode for only one accessory gene product between the S and E genes [34]. Accessory genes are poorly characterized and the functions of the gene products are not well understood. Initial research into the functions of these genes has shown that they are non-essential and dispensable for virus growth in cell culture [5, 6, 35]. More recent studies have shown that the accessory genes are required for in vivo infection and pathogenecity in the natural host [6, 7, 36-38].Unlike the SARS-CoV genome, the HCoV-NL63 genome encodes for only one accessory protein, ORF3 (Fig. 1). ORF3 is expressed from distinct subgenomic (sg) mRNA 3, which is one of at least six distinct mRNAs [4]. The ORF3 gene encodes for a putative 225 amino acid protein, about 25.6 kDa in size. Pyrc et al. (2004) reports that the HCoV-NL63 ORF3 gene has a unique nucleotide composition and appears as a U-rich and A-poor region within the genome, indicating a recent gene transfer event from another viral or cellular origin [4]. The ORF3 protein of HCoV-NL63 is homologous to proteins of the other Group 1 coronaviruses, with an amino acid sequence most similar (43% identity and 62% similarity) to HCoV-229E ORF4. Based on comparative in silico analysis of HCoV-NL63 ORF3 and other human coronavirus ORF3-like homologues, Fielding and Suliman speculate that the protein could contribute to pathogenesis in the natural host [39]. More recently, studies have shown that HCoV-NL63 localizes along the secretory pathway (ERGIC, Golgi, plasma membrane) and co-localizes with the structural proteins M and E in the ERGIC. Also, studies have shown that this N-glycosylated protein is incorporated into virions during assembly. This further suggests an important function for HCoV-NL63 ORF3, particularly in virus assembly and/or budding from the infected cell [40].\nTRANSMISSION AND PATHOGENESIS\n\nMode of Entry into the Cell\nIt is well documented that SARS-CoV and HCoV-NL63 use the same receptor, angiotensin converting enzyme (ACE)-2, for entry into the host cell [41, 42]. However, the consequences following the entry are very different; SARS-CoV causes severe respiratory distress, while HCoV-NL63 might lead to a mild respiratory infection. This was attributed to the ability of each virus receptor-binding protein, spike or S protein, to bind to ACE-2 on the cell surface. It was found that the HCoV-NL63 S protein has a weaker interaction with ACE-2 than the SARS-CoV S protein. This lower-affinity interaction with ACE-2 might partly explain the different pathological consequences of infection by SARS-CoV and HCoV-NL63 [43].Nevertheless, high pathogenicity is expected to evolve with coronaviruses. As for HCoV-NL63, the possibility of the development of a recombinant virus variant is high because of its high prevalence and the possibility of a recombination event through co-infection [24]. Moreover, the virus is able to survive for up to seven days in an aqueous solution and respiratory secretions and remains infective at room temperature. In heavily populated regions, direct person-to-person transmission is considered the major route of HCoV-NL63 spread [42, 44].\nSeasonal Incidence\nThe virus has been shown to have a worldwide distribution and was observed primarily in the winter season in temperate climates. On the other hand, countries with extreme weather, like Canada, have also shown virus activity around January to March, although milder symptoms were reported [45]. Interestingly, seasonal variations have been reported in China where infection with HCoV-NL63 appeared mainly in spring and summer [26]. Also, a recent study of coronaviruses in Thailand did not show any seasonal predilection [46], while Wu et al. (2007) reported that the virus is detected during the autumn season in Taiwan [47]. It is evident that the virus has no predilection to a particular season and is not affected by temperature variations as infections can occur throughout the year (Table 1).\nPrevalence\nThe human coronaviruses account for a significant number of hospitalization for children under 18 years of age, the elderly and immunocompromised individuals. In fact, a one-year study of children hospitalized in Hong Kong (China) has shown that respiratory tract infection with coronaviruses accounts for 4.4% of all admissions for acute respiratory infections. Of these, HCoV-NL63 was the most common coronavirus identified with an incidence of 2.6% [48]. Moreover, a study in Japan has shown that out of 419 specimens that tested negative for common respiratory viruses, five (1.2%) were positive for human coronavirus HCoV-NL63 [49]. Another Japanese report has indicated that out of 118 nasopharyngeal swab samples obtained from hospitalized children younger than two years of age, three (2.5%) were positive for HCoV-NL63 [50].In Europe, high prevalence was also noted. In Italy, a study conducted on 322 infants suffering from acute respiratory disease has shown that 8.7% of the cases examined were caused by coronaviruses, with HCoV-NL63 accounting for 21.4% of the latter [51]. In France, 300 respiratory specimens were checked for HCoV-NL63 presence; of the 300 samples, 28 (9.3%) were positive [29]. Likewise, seven cases were reported in a one-year study in Belgium [28]. Interestingly, the Dutch group that first described the virus has obtained 949 samples from a German group who conducted a population-based study on lower respiratory tract infection in children under three years of age [52]; the group re-analyzed the samples and detected a 5.2% incidence of HCoV-NL63 [53]. In Australia, the virus was detected in Melbourne, where a study conducted on 543 patients with respiratory symptoms has shown 18 cases (3.3%) of human coronavirus HCoV-NL63 infection [54]. Moreover, in the USA and Canada, the virus was reported in 79 (8.8%) out of 895 and 19 (3.6%) out of 525 patients, respectively.Currently, the accuracy of the percentage of detection is hampered by two main problems: firstly, the suitability of the samples examined: a recent study has shown that there are differences between respiratory samples collected by nose/throat swabs and by nasopharyngeal aspirates, specifically regarding their potential to detect and identify respiratory pathogens [55]. The second problem is that diagnostic tests for HCoVs are not frequently used in the routine testing for viruses, which probably results in the percentage of HCoVs infections being greatly underestimated [56]. Moreover, throughout the years several methods of variable sensitivity were used to determine the incidence of the virus [57-59].\nCo-Infection\nMany groups have reported that the occurrence of co-infections with HCoV-NL63 and other respiratory viruses, including other human coronaviruses, influenza A virus, respiratory syncytial virus (RSV), parainfluenza virus and human metapneumovirus (hMPV), are common [26, 30, 46, 47, 51, 53, 54, 60, 61]. Also, co-infected patients are more likely to be hospitalized, indicating the severity of this kind of superinfection. In a study from Germany, RSV-A and HCoV-NL63 was the most common co-infection indentified in children less than three years of age. This is probably due to the high incidence of RSV-A in winter and the overlap in seasonality of the viruses [53]. Also, in Italy, HCoV-NL63 circulates as a mixture of variant strains and is often associated with other viral infections [30]. In South Africa, co-infection of patients with HCoV-NL63 and bocavirus in hospitalized children is reported. Nasopharyngeal and bronchoalveolar lavage samples from 341 patients were screened for common respiratory viruses, and the co-presence of HCoV-NL63 and bocavirus in at least one sample was reported [62].Interestingly, the viral load of HCoV-NL63 is lower in co-infected patients than in patients infected with HCOV-NL63 only. There are various possible explanations for this phenomenon [53]:The high prevalence of co-infections of HCoV-NL63 and other respiratory viruses increases the chances of genetic recombination with these human or zoonotically transmitted viruses. In fact, Pryc et al. (2006) states HCoV-NL63 resulted from a recombination event between PEDV and an ancestral HCoV-NL63 strain. Theoretically, these types of recombination events could enable highly pathogenic virus variants to arise [24].\nClinical Features\nRecent scientific and clinical evidence has indicated that the virus is found during upper and lower respiratory tract infections, causing symptoms and signs that do not differ greatly from the symptoms described for the 'old' viruses HCoV-229E and HCoV-OC43. Other systems involvement is still controversial [63].Table 1 shows that patients diagnosed with the virus have presented with mild symptoms, indicating upper respiratory tract infection such as fever, cough and rhinorrhoea. On the other hand, the disease is also known to cause significant more alarming lower respiratory tract infection. One of the most alarming symptoms is bronchiolitis, an inflammation of the membranes lining the bronchioles. This symptom was reported by several research groups [25, 50, 64], and although a population-based study in China did not report an association of HCoV-NL63 with bronchiolitis [26], it is still believed to be one of the presenting symptoms. Several research groups have linked HCoV-NL63 to croup [26, 53]. Croup children present with pharangitis, sore throat and hoarseness of voice, and are considered for hospitalization. Of the rare findings, a group has reported the association between HCoV-NL63 and Kawasaki disease, a form of childhood vasculitis that is presented as fever, polymorphic exanthema, oropharyngeal erythema and bilateral conjuctivitis [65]. However, others fail to report on this association [66, 67].It is noteworthy to say that the report of symptoms in young children, who represent the majority of patients, is based mainly on parental observations, where other possible subjective signs and symptoms fail to be recognized by the parents. Moreover, most of the studies were conducted on patients reporting to hospitals suffering from acute respiratory tract infection. To date, there are only few population-based studies and the question arises whether larger numbers of such studies might reveal the involvement of other body systems.\nLABORATORY DIAGNOSIS AND DETECTION\nIt is difficult to distinguish clinical symptoms caused by HCoV-NL63 from those caused by many of the other human viruses, which could make diagnosis and detection of the virus complex. Treatment of patients without a clear diagnosis could result in implementation of expensive and disruptive public health measures, as well as to an increased spread of the disease [68]. Therefore, improved surveillance and diagnosis of respiratory illness could reduce health-care costs drastically [68-70].\nDetection of Viral RNA\nDue to its high specificity and sensitivity, reverse transcription polymerase chain reaction (RT-PCR) and other nucleic acid tests are the preferred methods for diagnosis of coronavirus infections, such as SARS-CoV [15]. The majority of nucleic acid amplification tests is designed with the ORF1a/1b, which is genetically stable in coronaviruses, and the nucleocapsid (N) or spike (S) genes (Table 1) [25, 26, 28, 30, 45, 46, 64, 67, 71-73]. Based on in vitro  coronavirus expression studies, the N gene has the theoretical advantage of being more abundant in infected cells and therefore of higher sensitivity for PCR assays, but this has not been clearly proven in clinical studies [74]. RT-PCR of nasopharyngeal samples, both frozen and fresh, is the most popular choice for detection of HCoV-NL63 (Table 1) and viral culture is frequently used for confirmation of infection [67, 73].\nAntigen and Antibody Detection Assays\nSerum samples have been screened for antibodies against HCoV-NL63 using enzyme-linked immunosorbent assays (ELISA) [75, 76]. An ELISA assay based on the N protein showed high seropositive results for serum samples from children younger than 20 years old [76]. Interestingly, the presence of maternally-acquired N-directed antibodies was detected in serum by ELISA [75, 76], usually decreasing within the first 4-5 months of life [76]. Potent neutralizing activity directed against HCoV-NL63 S protein was detected in virtually all serum samples from patients eight years of age or older, suggesting that HCoV-NL63 infection of humans is common and usually acquired during childhood [42]. Alarmingly, SARS-CoV infections appear to stimulate cross-reactive antibody responses to other human coronaviruses, including HCoV-NL63 [77]. Such cross reaction between human coronaviruses has been reported previously for immunoflourescence and complement fixation tests [78, 79]. In fact, false positive results were also reported for SARS-CoV detection using ELISA tests based on the recombinant N antigen [80]. This cross-reactivity is most likely due to the presence of cross-reactive antigenic epitopes of the coronaviruses [77]. An awareness of this cross-reactivity is important when developing antigen- and/or antibody-diagnostic assays and selecting the most suitable antigen or antigen target is important.\nCONCLUSION\nThe discovery of the HCoV-NL63 and other novel human coronaviruses does not necessarily represent a sudden increase in emerging infections by &#8216;new&#8217; coronaviruses. In fact, in the early 1960&#8217;s, viral agents causing upper respiratory tract infections were isolated from patients and were subsequently shown to be coronaviruses by electron microscopy. At least four of these were shown to be serologically distinct from the known coronaviruses at the time [10, 11, 81]. However, with the clinical samples of these no longer available for study, it will never be known whether these &#8216;old&#8217; viruses and the &#8216;new&#8217; viruses represent the same strains [82].Molecular clock analysis is the average rate at which a species' genome accumulates mutations and is used to measure that species&#8217; evolutionary divergence. The reliability of molecular dating is dependent on the validity of the molecular clock hypothesis, which assumes that the substitution rate is roughly constant [83], with an average substitution rate for coronaviruses estimated to be 10&#8722;&#160;4 substitutions per year per site [84, 85]. With the lack of sufficient sequence data available for HCoV-NL63, the substitution rate for HCoV-229E, using partial sequences of the S gene from different known dates, was calculated. Then, assuming a constant evolutionary rate in time and a constant evolutionary rate between the branches for HCoV-NL63 and HCoV229E, the time to their most recent common ancestor was dated to the 11th century [24]. This shows that HCoV-NL63 has been present in the human population for centuries. In support of this finding, the virus described by Fouchier et al. was originally isolated from an eight-month old child in 1988 [19]. More recent studies in the Netherlands also supported the notion that only SARS-CoV has recently been introduced to the human population from an animal source, while HCoV-NL63 has been circulating in humans for a while [86]. With the exception of SARS-CoV, the coronaviruses infecting humans are not well studied. This is partly due to the prevailing view that they are only involved in mild respiratory tract infections. The introduction of sensitive molecular and cell biology techniques have aided in identifying three &#8216;new&#8217; human coronaviruses, which has improved our understanding of the classification of the coronaviruses. These modern tools now need to be developed further to make the sensitive and accurate detection of coronaviruses in clinical samples possible. This would increase our understanding of the extent to which coronaviruses affect human health. Importantly, Donaldson and colleagues has recently reported the assembly of a full-length infectious HCoV-NL63 clone, which will make the study of this virus easier, improving our understanding of the role of each of the proteins encoded by the HCoV-NL63 genome [87]. In the long run, molecular and cell biology tools could help elucidate the link, if any, between human coronaviruses and human diseases of the respiratory tract, the vascular system, the central nervous system and the gastrointestinal tract [82].",
        "pmid": "20700397",
        "title": "Understanding Human Coronavirus HCoV-NL63",
        "journal_title": "The Open Virology Journal",
        "authors": "Sahar Abdul-Rasool;Burtram C Fielding",
        "doi": "10.2174/1874357901004010076"
    },
    {
        "keywords": "",
        "abstract": "ABSTRACT\nA novel human coronavirus (HCoV-EMC/2012) was isolated from a man with acute pneumonia and renal failure in June 2012. This report describes the complete genome sequence, genome organization, and expression strategy of HCoV-EMC/2012 and its relation with known coronaviruses. The genome contains 30,119 nucleotides and contains at least 10 predicted open reading frames, 9 of which are predicted to be expressed from a nested set of seven subgenomic mRNAs. Phylogenetic analysis of the replicase gene of coronaviruses with completely sequenced genomes showed that HCoV-EMC/2012 is most closely related to Tylonycteris bat coronavirus HKU4 (BtCoV-HKU4) and Pipistrellus bat coronavirus HKU5 (BtCoV-HKU5), which prototype two species in lineage C of the genus Betacoronavirus. In accordance with the guidelines of the International Committee on Taxonomy of Viruses, and in view of the 75% and 77% amino acid sequence identity in 7 conserved replicase domains with BtCoV-HKU4 and BtCoV-HKU5, respectively, we propose that HCoV-EMC/2012 prototypes a novel species in the genus Betacoronavirus. HCoV-EMC/2012 may be most closely related to a coronavirus detected in Pipistrellus pipistrellus in The Netherlands, but because only a short sequence from the most conserved part of the RNA-dependent RNA polymerase-encoding region of the genome was reported for this bat virus, its genetic distance from HCoV-EMC remains uncertain. HCoV-EMC/2012 is the sixth coronavirus known to infect humans and the first human virus within betacoronavirus lineage C.",
        "body": "Introduction\nCoronaviruses (CoVs) infect and cause disease in a wide variety of species, including bats, birds, cats, dogs, pigs, mice, horses, whales, and humans (1, 2). Recent studies suggest that bats act as a natural reservoir for coronaviruses (3&#8211;8). Coronaviruses may cause respiratory, enteric, hepatic, or neurological diseases with highly variable severity in their hosts. Until 2003, only two coronaviruses were known to infect humans. Human coronaviruses (HCoVs) HCoV-229E and HCoV-OC43 were identified in the 1960s as the causative agents of&#8212;generally mild&#8212;respiratory illnesses (9, 10). In 2002 to 2003, a previously unknown coronavirus&#8212;severe acute respiratory syndrome coronavirus (SARS-CoV)&#8212;caused a widespread outbreak of respiratory disease in humans, resulting in approximately 800 deaths and affecting around 30 countries (11&#8211;14). As a consequence of the renewed interest in coronaviruses after the SARS outbreak, two additional human coronaviruses were discovered after 2003: HCoV-NL63 in 2004 (15, 16) and HCoV-HKU1 in 2005 (17). A recent analysis of a large collection of human nasopharyngeal specimens using a Coronaviridae-wide primer set suggested that HCoV-229E, -OC43, -NL63, and -HKU1 are the only coronaviruses circulating in the human population (18).Coronaviruses are enveloped single-stranded positive-sense RNA viruses with genomes of 25 to 32&#160;kb, and the group includes the largest known genomes among the RNA viruses (1, 19). The coronaviruses form a subfamily (Coronavirinae) within the family Coronaviridae of the order Nidovirales. The International Committee on Taxonomy of Viruses (ICTV) has recognized four genera within the Coronavirinae subfamily: Alphacoronavirus, Betacoronavirus, and Gammacoronavirus, which were previously referred to as coronavirus groups 1, 2, and 3, and Deltacoronavirus (20). Coronaviruses are assigned to a genus on the basis of rooted phylogeny and calculation of pairwise evolutionary distances for seven highly conserved domains in the replicase polyprotein (1, 21) (C. Lauber and A. E. Gorbalenya, unpublished data). HCoV-229E and HCoV-NL63 are viruses belonging to the genus Alphacoronavirus (1). Four monophyletic lineages (A through D) with no formal taxonomic standing, some of them encompassing multiple virus species, are commonly recognized within the genus Betacoronavirus. Lineage A includes HCoV-OC43 and HCoV-HKU1 and lineage B SARS-CoV, all of which belong to different species. Lineages C and D include viruses detected only in bats, such as Rousettus bat coronavirus HKU9 (BtCoV-HKU9) (lineage D), Tylonycteris bat coronavirus HKU4 (BtCoV-HKU4), and Pipistrellus bat coronavirus HKU5 (BtCoV-HKU5) (both lineage C) (1). The genetic diversity of coronaviruses is likely facilitated by a high frequency of RNA recombination and the ability of their unusually large RNA genomes to both gain and lose domains (1, 22, 23). These factors are believed to have promoted the emergence of viruses with novel traits that are able to adapt to new hosts and ecological niches, sometimes causing zoonotic events.For the present study, we report and analyze the complete genome sequence of the recently identified HCoV-EMC/2012, which was isolated from the sputum of a 60-year-old man who died in a hospital in Jeddah, Saudi Arabia, after developing acute respiratory distress syndrome (ARDS) and multiple organ dysfunction syndrome (MODS) in June 2012 (24). This virus appears to be closely related to the HCoV detected in a second patient who was transported from a hospital in Qatar to a hospital in London, 3&#160;months after hospitalization of the first patient (25). These two cases of human infection with very similar or identical coronaviruses alarmed health authorities globally, as it was a reminder of the potential threat of coronaviruses to human health that was first highlighted by the SARS outbreak of 2003 (25). The sequence analysis of a small reverse transcription-PCR (RT-PCR) fragment that was first amplified from the HCoV-EMC/2012 genome revealed the highest similarity to two betacoronaviruses circulating in bats, BtCoV-HKU4 and -HKU5 (24). Here we present the complete genome sequence of the newly isolated HCoV-EMC/2012, accompanied by a detailed annotation of its genome organization and expression strategy. Furthermore, comparative genomic analysis and state-of-the-art classification and phylogenetic analyses were applied to determine the position of the novel agent with respect to previously characterized coronaviruses. We conclude that the HCoV-EMC/2012 genome organization and expression indeed most closely resemble those of BtCoV-HKU4 and -HKU5. However, based on our analysis and in line with the ICTV guidelines for the demarcation of coronavirus species, HCoV-EMC/2012 clearly qualifies to be recognized as the prototype of a novel species, which would thus constitute the first human coronavirus in lineage C of the genus Betacoronavirus.\nRESULTS\n\nSequencing of the HCoV-EMC/2012 genome.\nUsing a combination of approaches, including deep sequencing, cycle sequencing on a more traditional capillary sequencer, and determination of the genomic termini by rapid amplification of cDNA ends (RACE), the complete genome sequence of HCoV-EMC/2012 was determined from material that had been subjected to passage in cell culture 6 times. The data from the Roche 454&#160;GS Junior deep-sequencing run yielded a total of 90,808 sequence reads, of which 87,256 were specific for HCoV-EMC/2012. Genome coverage ranged from 1 to 5,697 reads at single nucleotide positions, with an average of 1,006 reads in the deep-sequencing run. Based on the contigs assembled from these initial data, primers approximately 800 nucleotides (nt) apart were designed to amplify PCR fragments with 100-nt overlaps covering the entire virus genome (see Table&#160;S1 in the supplemental material for primer sequences). These amplicons were sequenced using Sanger sequencing, and a total of 104 sequence runs were assembled&#8212;along with the original 90,808 deep-sequencing reads&#8212;into a single contiguous sequence of 30,119 nt, including the first 12 nt of the 3&#8242; poly(A) tail. Although 454 sequencing resulted in a higher single-read error rate than Sanger sequencing, the high coverage in the first data set largely corrected for these errors. Occasionally, the correct number of bases in homopolymer stretches was difficult to determine, which is a typical problem in 454 sequencing. Nevertheless, there was excellent agreement between the deep-sequencing data and the confirmatory Sanger sequencing. The final consensus sequence was submitted to GenBank (see below). This sequence contains only two ambiguous positions, nt 11623 and 27162. The variation at position 11623 translates into a Val or Gly uncertainty at amino acid (aa) 3782 of pp1a/pp1ab. Position 27162 was either a G or an A, with the A creating a premature stop codon for translation of open reading frame 5 (ORF5) (see Discussion). The verification of our consensus sequence awaits the availability of a second HCoV-EMC/2012 virus isolate or original specimen. The overall content of G and C residues in the HCoV-EMC/2012 genome was 41%, which is similar to values reported for other coronaviruses (37% to 42%) (14).\nGenome organization and expression strategy.\nCoronavirus genomes are polycistronic positive-stranded RNAs (Fig.&#160;1A), of which the 5&#8242;-proximal three-fourths are occupied by the large replicase open reading frames ORF1a and ORF1b. These are translated from the genomic mRNA to produce polyproteins pp1a and, following &#8722;1 ribosomal frameshifting, pp1ab, which are subsequently cleaved into 15 or 16 nonstructural proteins (nsps) (19, 23, 26). The region downstream of ORF1b is characterized by containing a variable number of smaller genes, always including those encoding the spike (S), envelope (E), membrane (M), and nucleocapsid (N) structural proteins. These genes are translated from subgenomic (sg) mRNAs that form a 5&#8242;- and 3&#8242;-coterminal nested set with the viral genome. Subgenomic mRNAs are composed of a common 5&#8242; leader sequence that is identical to the genomic 5&#8242; region and a variable part of the 3&#8242; quarter of the genome, with different sg mRNAs making different ORFs available for translation. The complement of the leader and &#8220;body&#8221; segments of the sg mRNAs are assumed to be joined during discontinuous negative-strand RNA synthesis. This step produces the templates for sg mRNA synthesis and is directed by a base-pairing interaction between conserved transcription-regulatory sequences (TRSs) (27&#8211;29). Such TRSs are found at the 3&#8242; end of the leader sequence (leader TRS) and at different positions upstream of genes in the genomic 3&#8242;-proximal domain (body TRSs). The synthesis of subgenome-length negative-stranded RNAs is directed by the complement of a body TRS at the 3&#8242; end of a nascent minus-strand base pairing with the leader TRS, with the extent of sequence complementarity being an important determinant of the level at which a given sg mRNA is produced.Inspection of the genome sequence of HCoV-EMC/2012 revealed the two large, partially overlapping replicase open reading frames ORF1a and ORF1b, as well as (at least) nine downstream ORFs (Fig.&#160;1A). The ORF1a sequence encodes the two protease domains conserved in all other coronaviruses, a papain-like protease (PL2pro) in nsp3 and a 3C-like protease (3CLpro; also known as the &#8220;main protease&#8221;) in nsp5. Sequence comparison with other coronaviruses allowed us to predict the putative pp1a/pp1ab cleavage sites and annotate the resulting nsp1 through -16 (Table&#160;1). According to sequence conservation analyses performed with other coronaviruses, open reading frames ORF2, -6, -7, and -8a are predicted to encode the four canonical structural proteins of coronaviruses, the envelope proteins S, E, and M and the N protein, respectively (Fig.&#160;1A; see also Fig.&#160;S1 in the supplemental material). A leader TRS and seven putative body TRSs could be readily identified, with the sequence 5&#8242;&#160;AACGAA&#160;3&#8242; forming the conserved TRS core and potential TRS duplexes during leader-body joining ranging from 14 to 19 matches over a 22-nt window that includes the core of the leader TRS (Fig.&#160;1B). From this analysis, it can be predicted that seven subgenomic mRNAs carrying a 67-nt common leader sequence would be produced in HCoV-EMC/2012-infected cells, with sizes ranging from ~4.7&#160;kb for mRNA2 to ~1.7&#160;kb for mRNA8. Experimental studies are needed to confirm the correct identification of the TRSs in the genomes of HCoV-EMC/2012 and related lineage C betacoronaviruses.Furthermore, mRNA4 and -8 are predicted to be functionally bicistronic, with ribosomal leaky scanning being the likely translation initiation mechanism for both ORF4b and ORF8b. The ORF4b AUG codon is not preceded by a separate body TRS, and the 241-nt sequence separating the 5&#8242; ends of ORF4a and ORF4b is entirely devoid of AUG codons. The AUG codon of the current ORF8b, an internal ORF that is overlapped by the N protein gene (ORF8a) and is present in all betacoronaviruses, is the third AUG codon on mRNA8, but sequence analysis and comparison with the BtCov-HKU4 and -HKU5 sequences (8) suggests that the 5&#8242; end of ORF8b may have become truncated relatively recently (see Discussion).Twenty-two additional putative ORFs of 150 to 432 nt in length were detected throughout the genome of HCoV-EMC/2012, overlapping the major ORFs. In contrast to the ORFs shown in Fig.&#160;1A, these 22 additional ORFs are not positioned (immediately) downstream of a body TRS, and hence it is unlikely that they are expressed. The synthesis of the replicase pp1ab polyprotein of HCoV-EMC/2012 involves &#8722;1 programmed ribosomal frameshifting, with nt 13427 to 13433 predicted to form the conserved &#8220;slippery sequence&#8221; (5&#8242;&#160;UUUAAAC&#160;3&#8242;) in the ORF1a/ORF1b overlap region that is typical for coronaviruses (30). The frameshift region is followed by a predicted RNA hairpin, formed by nucleotides at positions 13439 to 13450 base pairing with those at 13462 to 13473, with potential RNA pseudoknot formation occurring by base pairing of the loop of the hairpin (nt 13452 to 13460) with a downstream complementary sequence (nt 13506 to 13514). As is common in coronavirus genomes, nontranslated sequences are found only at the genomic termini, with the 5&#8242; and 3&#8242; untranslated regions (278 and 300&#160;nt, respectively) having sizes similar to those found in other family members. The only other apparently untranslated region in the genome that is larger than 50 nucleotides concerns the intergenic region between ORF5 and -6 (nt 27515 to 27589). This region appears to be conserved between HCoV-EMC/2012, BtCoV-HKU4, and BtCoV-HKU5, with sequence identities ranging from 63% to 84%, but we have no explanation for this observation thus far.\nPhylogenetic relations and taxonomic position of HCoV-EMC/2012.\nPhylogenetic trees were inferred using nucleotide sequences for ORF1ab (Fig.&#160;2A) and a 332-nt fragment from ORF1b (Fig.&#160;2B) encoding the most conserved part of the RNA-dependent RNA polymerase (RdRp) domain, which is commonly targeted in virus discovery studies. The first tree was produced for a representative set of coronaviruses for which complete genome sequences are available. In the second tree, we also included coronaviruses for which only partial genome sequences are known, particularly that of P.pipi/VM314/2008/NLD (31) which produced the best match with HCoV-EMC/2012. In both trees, HCoV-EMC/2012 clearly groups within lineage C of the genus Betacoronavirus, relatively close to BtCoV-HKU4 and BtCoV-HKU5. However, based on the 332-nt fragment from ORF1b, HCoV-EMC/2012 is more closely related to bat-derived isolate VM314/2008 (GenBank accession number GQ259977), which was isolated from Pipistrellus bats in The Netherlands 4&#160;years ago. Phylogenetic trees were also constructed based on amino acid sequences, using coronavirus-wide conserved domains of replicative proteins in pp1ab (Fig.&#160;3A) as well as using conserved parts of structural proteins (Fig.&#160;3B). In both trees, HCoV-EMC/2012 clusters with betacoronaviruses, supporting its classification as a member of the genus Betacoronavirus.ICTV assigns newly identified members of the family Coronaviridae to a subfamily and genus on the basis of rooted phylogeny and calculation of pairwise evolutionary distances for seven replicase domains (1, 21). To establish whether HCoV-EMC/2012 indeed prototypes a new species, amino acid sequence alignments were generated for each of these domains and concatenated, after which the sequence identity of HCoV-EMC/2012 with closely related strains was calculated. For this purpose, the full genomes of 9 strains, derived from 3 species, belonging to Betacoronavirus lineage C were available (Table&#160;2). Amino acid sequence identity between conserved replicase domains of HCoV-EMC/2012 and those of other lineage C viruses ranged from 57% (ADP-ribose 1&#8243;-phosphatase [ADRP]) to 94% (helicase [Hel]). Overall amino acid sequence identities to BtCoV-HKU4 and BtCoV-HKU5 strains across the conserved domains were around 75% and 76.7%, respectively. These percentages are well below the threshold of 90% amino acid sequence identity that is used for coronavirus species identification by the ICTV. The distance between HCoV-EMC/2012 and members of these two species is as large as that observed upon interspecies comparison of other species pairs, for example, Murine coronavirus versus Human coronavirus HKU1 or Porcine epidemic diarrhea virus versus Scotophilus bat coronavirus 512 (Fig.&#160;3). Consequently, we propose that HCoV-EMC/2012 prototypes a novel species of lineage C of the genus Betacoronavirus.\nGenome similarities between coronavirus HCoV-EMC/2012 and coronaviruses BtCoV-HKU4 and BtCoV-HKU5.\nBtCoV-HKU4 and BtCoV-HKU5 (8) are the closest relatives of HCoV-EMC/2012 for which full-length genome sequences are available (see above). Accordingly, comparison of the genomes of these three viruses revealed important similarities, including the organization of the &#8220;accessory protein genes,&#8221; ORF3a through ORF5, residing between the S protein gene and those encoding the E, M, and N proteins. Upon annotating this region of the BtCoV-HKU4 and BtCoV-HKU5 genomes, Woo et al. (8) identified the body TRSs for sg mRNA3, mRNA4, and mRNA5 but unfortunately did not follow standard coronavirus nomenclature, naming the downstream open reading frames ORF3a through ORF3d (encoding ns3a through ns3d) rather than ORF3, ORF4a, ORF4b, and ORF5 (Fig.&#160;1A). The similarities of all ORFs and proteins of HCoV-EMC/2012, BtCoV-HKU4, and BtCoV-HKU5 were calculated, and percentages of sequence identity are summarized in Table&#160;3. The lowest percentages of sequence identity to BtCoV-HKU4 and BtCoV-HKU5 were observed for ORF3 at the nucleotide level (46.4% and 46.0%, respectively) and for ORF4b at the amino acid level (23.5% and 25.9%, respectively). The highest percentages of sequence identity to BtCoV-HKU4 and BtCoV-HKU5 were observed for the E ORF at the nucleotide level (74.6% and 75.1%, respectively) and for the M ORF at the amino acid level (82.6% and 82.2%, respectively). These data further supported the characterization of HCoV-EMC/2012 as a close relative of BtCoV-HKU4 and BtCoV-HKU5.\nDISCUSSION\nCoronaviruses have been known for quite some time as viruses that cause a variety of diseases in humans and animals (32, 33). The discovery of a coronavirus as the causative agent of SARS revived the interest in coronaviruses and resulted in a rapid increase of the number of identified coronaviruses, as well as of the number of full coronavirus genome sequences. Until this study, lineage C of the genus Betacoronavirus (formerly known as subgroup 2c) included virus isolates from bats. Here, we determined and analyzed the complete genome sequence of a previously unknown lineage C betacoronavirus that was isolated from the sputum of a 60-year-old male suffering from acute pneumonia and renal failure in the Kingdom of Saudi Arabia whose death was probably a consequence of this infection (24).The sequencing of the full HCoV-EMC/2012 genome was greatly facilitated by the advent of high-throughput techniques. Using an optimized random amplification deep-sequencing approach, approximately 90% of the virus genome was covered with high accuracy in a single run. Using the data from this first run, primers could be designed to perform conventional Sanger sequencing for confirmation. This combination of techniques allowed the determination of the complete virus genome within a few days, without a requirement for prior knowledge of the virus genome under investigation. The error rate in 454 deep sequencing was generally higher than in Sanger sequencing, but the high coverage across the HCoV-EMC/2012 virus genome (up to 5,697 reads per nucleotide position) corrected for most of the incorrect base callings. The sequence obtained using the 454 platform aligned almost perfectly with that obtained by Sanger sequencing, with the exception of two nucleotide positions. The deep-sequencing data revealed variation at position 11623 (U or G), with G occurring in 44% of the reads, suggesting that ORF1a-encoded residue 3782 can be either valine (codon GUC) or glycine (codon GGC). The valine codon was the more abundant codon at this position in HCoV-EMC/2012, and valine is also present in most other betacoronaviruses. At position 27162, both G and A were detected in different runs, with an A in 45% of the reads. This G-to-A substitution introduces a premature stop codon (UGG to UAG) in ORF5. The virus stock that we sequenced was derived from passage of the virus from a sputum specimen six times in Vero cell culture. Hence, the observed sequence variants may reflect either natural heterogeneity or emerging genetic changes that occurred during virus passage in cell culture. Additional HCoV-EMC/2012 virus isolates or patient materials are currently not available to verify these genome sequence ambiguities at positions 11623 and 27162.Adaptation to cell culture leading to a loss of functionality of genes, and in particular in relation to the so-called &#8220;accessory protein genes,&#8221; has previously been described for a variety of coronaviruses, including SARS-CoV (2, 34). These genes, like ORF3 through ORF5 of HCoV-EMC/2012, are dispersed between the structural protein genes (35) and in some cases may even overlap such a gene, as in the case of the ORF overlapping the N protein gene in betacoronaviruses (Fig.&#160;1A) (23, 36). The origin of most accessory protein genes remains unclear, although for some, acquisition by recombination with cellular or heterologous viral sequences seems plausible (37, 38). Accessory gene functions have been probed by reverse genetics (knockout mutants) for a variety of coronaviruses, including SARS coronavirus (39), which established that they are not essential for replication in cell culture systems. In animal models, on the other hand, profound effects on pathogenesis after the inactivation (or transfer to a heterologous coronavirus) of accessory protein genes have been previously described (40&#8211;42). In some cases, accessory gene products have been implicated in immune evasion, e.g., by interfering with cellular innate immune signaling (43).The apparent absence of selection pressure on coronavirus accessory protein genes during cell culture passage may explain the relatively high frequency with which loss of functionality appears to occur. The detection of an internal termination codon in part of the HCoV-EMC/2012 ORF5 sequences (45% of the reads) may constitute another example of such an event, which would lead to the truncation of the ORF5 protein after 107&#160;amino acids. This would resemble a 29-nt deletion that occurred in the SARS-CoV genome, which resulted in the truncation of ORF8 (34, 44), and a 45-nt in-frame deletion in ORF7b of the same virus that emerged upon cell culture passage (23).Our analysis identified a potential ORF underlying the N protein gene (ORF8a), which is a common feature in betacoronaviruses. This ORF was not previously described for BtCoV-HKU4 and BtCoV-HKU5 (8) but is conserved in the genome sequences of both viruses (see Fig.&#160;S1J in the supplemental material). Remarkably, in HCoV-EMC/2012, both the 5&#8242; and 3&#8242; parts of the ORF appear to have been truncated. In BtCoV-HKU4 and BtCoV-HKU5, the ORF8b AUG codon would be the second AUG on sg mRNA8, making leaky ribosomal scanning a likely mechanism for translation initiation. In HCoV-EMC/2012, however, this AUG codon (positions 28606 to 28608) seems to have been mutated to AUA. Conservation of the sequence immediately downstream of this position, which is now formally upstream of ORF8b in HCoV-EMC/2012, was observed with BtCoV-HKU4 and BtCoV-HKU5, suggesting that the putative loss of this AUG codon may also have been a relatively recent event. In the 3&#8242; part of ORF8b, sequence alignment of HCoV-EMC/2012 with BtCoV-HKU4 and BtCoV-HKU5 suggests that the former acquired a premature termination codon at positions 29099 to 29101 (UAA). Although we cannot at present assess the timing of these events in HCoV-EMC/2012 evolution, due to the lack of alternative samples for this species, the presumed loss of ORF8b functionality may also be a consequence of virus passage in cell culture.To classify newly identified coronaviruses as the prototype of a novel virus species, it is required that the amino acid sequence identity in the conserved replicase domains in all intervirus pairwise comparisons is below the 90% threshold (1). Here, we propose HCoV-EMC/2012 to represent a novel species of the betacoronavirus genus, since the amino acid sequence identities between HCoV-EMC/2012 and its closest relatives BtCoV-HKU4 and BtCoV-HKU5 in the seven conserved domains of ORF1ab were 75% and 77%, respectively. These viruses were originally detected in Asia in lesser bamboo bats (Tylonycteris pachypus) and Japanese house bats (Pipistrellus abramus), respectively (8). This proposed classification will remain provisional until approved by ICTV.The ICTV guidelines for coronavirus species demarcation require the availability of a (nearly) complete genome sequence prior to virus classification. However, there is considerable correlation between the results based on full-genome sequence analysis and those determined using the most conserved part of the ORF1b-encoded RdRp domain, which is commonly used in screening for new coronaviruses. In 2010, this partial sequence was reported for a betacoronavirus (VM314/2008) that was isolated 2&#160;years earlier from a Pipistrellus pipistrellus bat in The Netherlands. This virus was provisionally classified a betacoronavirus based on a 332-nt fragment from the RdRp-encoding domain of ORF1b (31), which shares 88% nucleotide sequence identity with HCoV-EMC/2012, the highest identity with any coronavirus sequence available in the public domain. Although this high similarity is not sufficient to resolve the taxonomic relation between HCoV-EMC/2012 and isolate VM314/2008, it suggests that they may both belong to the same coronavirus species. Establishing the genome sequence of VM314/2008, or closely related viruses, is urgently required to verify this hypothesis. Based on the genetic relation between HCoV-EMC/2012 and bat coronaviruses, it is tempting to speculate that HCoV-EMC/2012 emerged from bats&#8212;either directly or via an intermediate animal host, possibly Pipistrellus bats. This bat species is known to be present in the Kingdom of Saudi Arabia and neighboring countries.Although most infections of human coronaviruses are relatively mild, the infection by HCoV-EMC/2012 with fatal outcome, and a similar severe case of an infection with a closely related coronavirus in London (25), is a reminder that certain coronaviruses may cause severe and sometimes fatal infections in humans. It is important to develop an animal model that can be used to fulfill Koch&#8217;s postulates for the novel virus, by demonstrating that the isolated virus can indeed cause the observed disease. The availability of the HCoV-EMC/2012 genome sequence will facilitate the development of a variety of diagnostic assays that can be used to study the prevalence and clinical impact of HCoV-EMC/2012 infections in humans. The first generation of assays for this purpose has recently been described (45). We anticipate that the availability of this full-length virus genome sequence will be valuable for the development of additional applied and fundamental research.\nMATERIALS AND METHODS\n\nVirus propagation.\nPatient material had been subjected to passage in Vero cells four times in the Dr. Soliman Fakeeh Hospital, Jeddah, Saudi Arabia. Subsequently, in the Erasmus Medical Center, Rotterdam, The Netherlands, LLC-MK2 cells were inoculated with HCoV-EMC/2012 in minimal essential medium (MEM-Eagle) with Earle&#8217;s salts (BioWhittaker, Verviers, Belgium), supplemented with 2% serum, 100&#160;U/ml penicillin, 100&#160;mg/ml streptomycin, and 2&#160;mM glutamine. Vero cells were inoculated with virus in Dulbecco&#8217;s modified Eagle medium (BioWhittaker) supplemented with 1% serum, 100&#160;U/ml penicillin, 100&#160;mg/ml streptomycin, and 2&#160;mM glutamine. After inoculation, the cultures were incubated at 37&#176;C in a CO2 incubator and checked daily for cytopathic changes. Three&#160;days after inoculation, supernatant from Vero cells was collected and used for virus genome characterization.\nArbitrarily primed PCR and virus genome sequencing.\nTo characterize the viral genome, we used a random amplification deep-sequencing approach as the first step. Virus-containing supernatant was centrifuged for 10&#160;min at 3,000&#160;rpm to remove cellular debris. This supernatant was then filtered through a 0.45-&#181;m-pore-size centrifugal filter unit (Millipore, Amsterdam, The Netherlands) to minimize bacterial contamination. Omnicleave endonuclease (Epicenter Biotechnologies, Madison, WI) was used to remove any free DNA and RNA, according to the manufacturer&#8217;s protocol. Subsequently, viral RNA was extracted from the purified, infected cell culture supernatant using a High Pure RNA isolation kit (Roche Diagnostics, Almere, The Netherlands). To remove contaminating mammalian rRNA, a Ribo-Zero RZH110424 rRNA removal kit (Epicenter Biotechnologies, Madison, WI) was used according to the manufacturer&#8217;s protocol. RNA was reverse transcribed using circular permuted primers (46) that were extended with random hexamer sequences, namely, CCCACCACCAGAGAGAAAN(6), ACCAGAGAGAAACCCACCN(6), GAGAAACCCACCACCAGAN(6), GGAGGCAAGCGAACGCAAN(6), AAGCGAACGCAAGGAGGCN(6), and ACGCAAGGAGGCAAGCGAN(6). Per reaction, reverse transcription mixtures contained 6&#160;&#181;l RNA, 1 &#181;l primer (20&#160;pmol), 0.5&#160;&#181;l (20&#160;U) RNase inhibitor (Promega, Leiden, The Netherlands), 1&#160;&#181;l (10&#160;mM each) deoxynucleoside triphosphates (Roche), and 5 &#181;l water. After a 5 min incubation at 65&#176;C for optimal primer hybridization to the template, 4&#160;&#181;l (10&#215;) first-strand buffer, 1&#160;&#181;l (200&#160;U/&#181;l) SuperScript III reverse transcriptase (Invitrogen, Bleiswijk, The Netherlands), 1&#160;&#181;l (0.1&#160;M) dithiothreitol (DTT), and 0.5&#160;&#181;l (20&#160;U) RNase inhibitor (Promega) were added to the mixture in a 20-&#181;l volume. To obtain cDNA, the reverse transcription mixture was sequentially incubated at 25&#176;C for 5&#160;min and at 42&#176;C for 1&#160;h. After 3&#160;min at 95&#176;C and 2&#160;min on ice, 1&#160;&#181;l Klenow DNA polymerase (5&#160;U) (New England BioLabs Inc., Ipswich, MA) was added and the mixture was sequentially incubated at 25&#176;C for 5&#160;min, 37&#176;C for 1&#160;h, and 75&#176;C for 20&#160;min to obtain double-stranded cDNA. The cDNA was purified using a MinElute PCR purification kit (Qiagen, Venlo, The Netherlands) according to the instructions of the manufacturer. To amplify the purified cDNA, a PCR with the individual circular permuted primers without the random hexamer was performed. The PCR mixture contained 2 &#181;l primer (40&#160;pmol), 2&#160;&#181;l purified cDNA, 1.25&#160;&#181;l (10&#160;mM each) deoxynucleoside triphosphate (Roche), 5&#160;&#181;l (10&#215;) PfuUltra II Rxn buffer, and 1&#160;&#181;l (2.5&#160;U) PfuUltra II DNA polymerase (Stratagene, Amsterdam, The Netherlands). Water was added to reach a final volume of 50&#160;&#181;l. The PCR mixture was incubated at 95&#176;C for 2&#160;min and then for 40 cycles of 95&#176;C for 20&#160;s, 56&#176;C for 1&#160;min, and 72&#176;C for 2&#160;min, followed by a final extension at 72&#176;C for 10&#160;min. Fragments were purified using a MinElute PCR purification kit (Qiagen) according to the instructions of the manufacturer.Amplified fragments were sequenced using a 454/Roche GS Junior sequencing platform. A fragment library was created according to the manufacturer&#8217;s protocol without DNA fragmentation (GS FLX Titanium rapid library preparation; Roche), selecting for fragments larger than 100&#160;bp. The emulsion-based PCR (emPCR) (amplification method Lib-L) and GS Junior sequencing run were performed according to the instructions of the manufacturer (Roche). The sequence reads were trimmed at 30 nt from the 3&#8242; and 5&#8242; ends to remove all primer sequences. Sequence reads were assembled into contigs using CLC Genomics 5.5.1 software (CLC Bio, Aarhus, Denmark). Using this deep-sequencing approach, approximately 90% of the virus genome sequence was obtained.As a second step, specific primers were designed to amplify overlapping fragments of approximately 800&#160;bp by RT-PCR. These PCR products were purified from agarose gels and sequenced using a BigDye Terminator v3.1 cycle sequencing kit (Applied Biosystems, Nieuwerkerk a/d IJssel, The Netherlands) and a 3130XL genetic analyzer (Applied Biosystems), according to the instructions of the manufacturers. The genomic 5&#8242;- and 3&#8242;-terminal sequences were determined using a FirstChoice RLM-RACE kit (Ambion, Bleiswijk, The Netherlands).\nVirus classification.\nNewly identified members of the family Coronaviridae are generally assigned by the ICTV to a subfamily and genus on the basis of rooted phylogeny and calculation of pairwise evolutionary distances for seven replicase polyprotein domains (1): the ADP-ribose 1&#8243;-phosphatase (ADRP) in nsp3, the coronavirus 3C-like (3&#160;CL) protease (3CLpro, or &#8220;main protease&#8221;) in nsp5, the RNA-dependent RNA polymerase (RdRp) in nsp12, the helicase (Hel) in nsp13, the exoribonuclease (ExoN) in nsp14, the nidoviral endoribonuclease specific for U (NendoU) in nsp15, and the ribose-2&#8242;-O-methyltransferase (O-MT) in nsp16. Amino acid sequence alignments were generated for each of these domains using ClustalW within the BioEdit (version 7.0.5.3) (47) program and concatenated, after which the sequence identity of HCoV-EMC/2012 with closely related strains was calculated. For this purpose, the full genomes of 9 strains, derived from 3 species, belonging to Betacoronavirus lineage C were available.To support virus classification, protein-based phylogenetic trees were generated. Multiple amino acid alignments, including sequences of HCoV-EMC/2012 and one representative of each of the 20 recognized species of the subfamily Coronavirinae, were produced for the following proteins, using the Viralis platform (48) followed by manual correction: ADRP, the N-terminal part of PLP2, TM1, Y domain, nsp4 to nsp16, and the C-terminal part of the spike (S) protein (S2), envelope (E) protein, membrane (M) protein, and nucleocapsid (N) protein. From each protein alignment, the most informative blocks (49) were extracted using the BAGG program (50), and only these strongly conserved alignment regions were used for further analyses. Two concatenated alignments were used. The first included replicase pp1ab protein regions (4,110 aa positions, gap content of 0.9%), and the second included regions in the C-terminal domain of the S protein (S2) and the E, M, and N proteins (1,127&#160;aa positions, gap content of 3.9%). ProtTest version 3.2 (51) was used to select the best-fitting model of protein evolution. For both datasets, the LG model with rate heterogeneity (4 categories) ranked top among 112 models tested, with a relative weight of 0.98 under the Bayesian information criterion (BIC) and 0.74 under the corrected Akaike information criterion (AICc) for the first data set and 0.97/0.75 (BIC/AICc) for the second data set. Hence, this model was applied for maximum likelihood phylogeny reconstruction using PhyML version 3.0 (52).\nPhylogenetic reconstruction.\nNucleotide sequences were aligned using the ClustalW software running within the BioEdit (version 7.0.5.3) (47) program and MAFFT version 6 (53). Maximum likelihood phylogenetic trees with 100 bootstrap replicates were estimated under the general time-reversible model (GTR) + I + &#915;4 and the transversion model (TVM) + I + &#915;4 (determined by ModelTest [54]), using PhyML 3.0 software (52). For both the 332-nt ORF1ab alignment that included isolate VM314/2008 and the alignment of the complete ORF1ab, the GTR + I + &#915;4 model ranked top among 65 models tested, with relative weights of 0.8185 and 1.000 under AIC, respectively.\nNucleotide sequence accession number.\nThe final HCoV-EMC/2012 consensus sequence was submitted to GenBank under accession number JX869059.\nSUPPLEMENTAL MATERIAL\n",
        "pmid": "23170002",
        "title": "Genomic Characterization of a Newly Discovered Coronavirus Associated with Acute Respiratory Distress Syndrome in Humans",
        "journal_title": "mBio",
        "authors": "Sander van Boheemen;Miranda de Graaf;Chris Lauber;Theo M. Bestebroer;V. Stalin Raj;Ali Moh Zaki;Albert D. M. E. Osterhaus;Bart L. Haagmans;Alexander E. Gorbalenya;Eric J. Snijder;Ron A. M. Fouchier",
        "doi": "10.1128/mBio.00473-12"
    },
    {
        "keywords": "",
        "abstract": "Canine respiratory coronavirus (CRCoV), identified in 2003, is a member of the Coronaviridae family. The virus is a betacoronavirus and a close relative of human coronavirus OC43 and bovine coronavirus. Here, we examined entry of CRCoV into human rectal tumor cells (HRT-18G cell line) by analyzing co-localization of single virus particles with cellular markers in the presence or absence of chemical inhibitors of pathways potentially involved in virus entry. We also targeted these pathways using siRNA. The results show that the virus hijacks caveolin-dependent endocytosis to enter cells via endocytic internalization.Electronic supplementary material\nThe online version of this article (10.1186/s13567-018-0551-9) contains supplementary material, which is available to authorized users.",
        "body": "Introduction\nCoronaviruses are enveloped, single-stranded, positive-sense RNA viruses belonging to the family Coronaviridae within the order Nidovirales [1]. Based on its properties, this family can be divided into four distinct genus: alpha, beta, delta, and gamma. Coronaviruses infect a wide variety of birds and mammals, including humans, livestock, and companion animals [1&#8211;3]. Human coronaviruses (HCoVs) are associated mainly with relatively mild upper and lower respiratory tract disease; however, emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in the winter of 2002&#8211;2003 in China, and more recently Middle East respiratory syndrome coronavirus (MERS-CoV) in the Middle East, demonstrates the potential threat posed by zoonotic coronaviruses [2&#8211;4].Canine respiratory coronavirus (CRCoV) was first identified in 2003 in samples obtained from the respiratory tracts of dogs with canine infectious respiratory disease (CIRD; also known as kennel cough) that were housed in animal shelters in the United Kingdom [5]. CIRD is a contagious disease with high morbidity but low mortality; it usually occurs in densely housed dog populations (e.g., rehoming centers, veterinary hospitals). Characterized by a dry, hacking cough, the disease is generally mild and self-limiting. However, it can progress to a potentially fatal bronchopneumonia [6, 7]. CIRD is considered a complex infection, with a multifactorial etiology in which a number of organisms (including Bordetella bronchiseptica, canine parainfluenza virus, canine adenovirus type 1 and 2, canine herpesvirus, Mycoplasma spp., canine pneumovirus, and influenza viruses) are involved [6, 8]. It is believed that CRCoV plays a role in the early stages of CIRD by limiting ciliary clearance of the upper airways. Consequently, infection leads to reduced respiratory clearance and sensitization to secondary infections [5&#8211;7].CRCoV is closely related to two other betacoronaviruses, bovine coronavirus (BCoV) and HCoV-OC43 (97.3% nucleotide identity in the spike gene for BCoV and 96.9% for OC43 as reported by Erles et al. [5]), but is clearly distinct from Canine Enteric Coronavirus (CECoV, previously known as Canine Coronavirus) [5, 7]. CRCoV is a difficult pathogen to work with because the only confirmed susceptible cell line is a human rectal tumor cell line (HRT-18) and its derivative HRT-18G. No canine cell line supports replication of the virus. Furthermore, CRCoV does not produce a cytopathic effect in HRT-18 cells [8].To initiate infection, enveloped viruses fuse with host cell membrane prior to delivering genetic material. This process may occur at the cell surface (e.g., human immunodeficiency virus, herpes simplex virus); otherwise prior internalization is required [2, 9]. To enter the cell, viruses hijack a number of different endocytic pathways, including macropinocytosis and clathrin-mediated, caveolin-mediated, and clathrin- and caveolin-independent routes [2, 9, 10]. For example, SARS-CoV uses clathrin-dependent, lipid raft-mediated, and clathrin- and caveolae-independent entry pathways [2, 11&#8211;13]. In addition, feline infectious peritonitis virus (FIPV) uses clathrin- and caveolin-independent endocytic routes [14], whereas HCoV-229E uses caveolae-dependent endocytosis [15]. Furthermore, some human respiratory coronaviruses may utilize protease activation to modulate the route of entry [16&#8211;18]. Generally, within each of these endocytic pathways, vesicles are formed through interaction of certain protein networks. Early vesicles provide a starting point for trafficking, which leads to endosome maturation and allows sorting of incoming cargo [19, 20]. Some internalized vesicles are recycled back to the cell surface, while others are converted, for example to lysosomes. Sorting of cargo is regulated by Rab GTPases, which serve as molecular hallmarks of different routes [19&#8211;21].Here, we studied internalization of CRCoV into HRT-18G cells. The results clearly demonstrated that CRCoV entry into HRT-18G cells requires endocytic internalization prior to membrane fusion, a process that requires caveolin-1 and dynamin. Furthermore, fusion of the viral and cellular membranes occurs before the endosome progresses to the late phase.\nMaterials and methods\n\nCells and viruses\nHRT-18G (ATCC&#160;CRL-11663) cells, derivative of HRT-18 (ATCC&#160;CCL-244, ileocecal colorectal adenocarcinoma) were maintained in Dulbecco&#8217;s MEM (Life Technologies, Poland) supplemented with 3% heat-inactivated fetal bovine serum (Life Technologies), penicillin (100&#160;U/mL), streptomycin (100&#160;&#956;g/mL), and ciprofloxacin (5&#160;&#956;g/mL). Cells were cultured at 37&#160;&#176;C under 5% CO2. Virus stock of canine respiratory coronavirus strain 4182 was prepared by infecting HRT-18G cells monolayers and collecting supernatant 5&#160;days post-infection (pi). Obtained stock was aliquoted and stored at &#8722;80&#160;&#176;C. The control from mock-infected cells was prepared in the same manner. Virus yield was estimated by titration on confluent HRT-18G cells according to the method of Reed and Muench [22]. As CPE is not visible, cells were infected at 37&#160;&#176;C for 5&#160;days, fixed and immunostained to detect virus-infected cells. For co-localization studies, stocks were concentrated using Amicon Ultra Centrifugal Filters (Merck, 10-kDa cutoff), aliquoted, and stored at &#8722;80&#160;&#176;C.\nChemical inhibitors\nDMEM supplemented with 100&#160;U/mL penicillin, 100&#160;&#956;g/mL streptomycin and one of the following chemical inhibitors: 1-aminoadamantane (100&#160;&#181;M, AMTD, Sigma-Aldrich, Poland), Pitstop (10&#160;&#181;M, Abcam), chlorpromazine (1.5&#160;&#181;M, Sigma-Aldrich), filipin&#160;III (2&#160;&#956;g/mL, Sigma-Aldrich), nystatin (50&#160;&#181;g/mL, Sigma-Aldrich), methyl-&#946;-cyclodextrin (5&#160;mM, M&#946;CD, Sigma-Aldrich), 5-(N-ethyl-N-isopropyl)amiloride (10&#160;&#181;M, EIPA, Sigma-Aldrich), 1,1&#8242;-Dithiobis-2-naphthalenol (20&#160;&#181;M, IPA-3, Sigma-Aldrich), wortmannin (5&#160;&#181;M, Calbiochem), dynasore (80&#160;&#181;M, Abcam), iminodyn&#160;22 (25 &#181;M, Abcam), mitmab (5&#160;&#181;M, Abcam), ammonium chloride (50&#160;mM, NH4Cl, Bioshop), bafilomycin A1 (10&#160;nM, Sigma-Aldrich), cytochalasin D (10&#160;&#181;M, Sigma-Aldrich), jasplakinolide (1.5&#160;&#181;M, Calbiochem), nocodazole (0.5&#160;&#956;M, Sigma-Aldrich), cell permeable Rho inhibitor (1&#160;&#181;g/mL, CT04 Cytoskeleton Inc.), N6-[2-[[4-(Diethylamino)-1-methylbutyl]amino]-6-methyl-4-pyrimidinyl]-2-methyl-4,6-quinolinediamine trihydrochloride (100&#160;&#956;M NSC23766), (R)-(+)-trans-4-(1-Aminoethyl)-N-(4-Pyridyl)cyclohexanecarboxamide dihydrochloride (10&#160;&#956;M, Y27632 Sigma-Aldrich), Decanoyl-Arg-Val-Lys-Arg-chloromethyl ketone (5&#8211;100 &#956;M, Santa Cruz Biotechnology) were used to pretreat HRT-18G cells for 1&#160;h at 37&#160;&#176;C.\nInhibition of viral replication\nCells pretreated with chemical inhibitors for 1&#160;h at 37&#160;&#176;C (full list at Additional file 1), were exposed to virus at a 50% tissue culture infectious dose (TCID50) of 400 in the presence of inhibitors. Two hours pi cells were washed with PBS twice to remove unbound virus, and medium with fresh inhibitors was added to each well. Five days pi cells were harvested for further analysis. Cell viability was tested at day five pi using XTT based Cell Proliferation Kit (Biological Industries), according to the manufacturer&#8217;s instructions.\nFACS analysis\nCells treated with inhibitors were harvested at 5&#160;day pi by trypsinization, centrifuged (5&#160;min, 300&#160;&#215;&#160;g), washed with PBS, fixed in 4% formalin in PBS (15&#160;min, room temperature (RT)) and permeabilized (0.5% Triton&#160;X-100 in PBS, 20&#160;min, RT, Bioshop). Unspecific binding sites were blocked (5% bovine serum albumin (BSA, Bioshop) in PBS, 2&#160;h RT) prior to staining. Further, cells were incubated with anti-coronavirus antibody OC43 strain (1&#160;&#181;g/mL, Merck) for 2&#160;h and with secondary Alexa Fluor 488 goat anti-mouse antibody (5&#160;&#181;g/mL, Thermo Scientific, Poland) for 1&#160;h. Cells were washed with 0.5% Tween-20, resuspended in PBS, and analyzed with flow cytometry (FACSCalibur, Becton Dickinson).\nConfocal microscopy\nCells cultured on coverslips in 6-well plate for 48&#160;h were washed with ice-cold PBS and incubated with concentrated CRCoV at 4&#160;&#176;C for 60&#160;min and subsequently incubated at 37&#160;&#176;C for 0&#8211;180&#160;min. Coverslips were washed thrice with PBS, fixed in 4% formalin for 15&#160;min, permeabilized with 0.5% Tween-20 (Bioshop) for 20 min (unless stated otherwise). Unspecific binding sites were blocked using 5% BSA in PBS (4&#160;&#176;C, overnight) prior to staining. For visualization of CRCoV particles anti-coronavirus antibody OC43 strain (1&#160;&#181;g/mL, 2&#160;h, RT, Merck) coupled with goat anti-mouse Alexa Fluor 488 antibody (5&#160;&#181;g/mL, 1&#160;h, RT, Thermo Scientific) was used. To visualize host cell proteins, cells were blocked using 10%&#160;FBS in PBS (4&#160;&#176;C, overnight) and incubated (2&#160;h, RT) with one of the following antibodies: caveolin-1 antibody (2&#160;&#181;g/mL, Santa Cruz Biotechnology), clathrin HC antibody (2&#160;&#181;g/mL, Santa Cruz Biotechnology), EEA1 antibody (2&#160;&#181;g/mL, Santa Cruz Biotechnology), Rab&#160;7 antibody (2&#160;&#181;g/mL, Santa Cruz Biotechnology), endophilin B2 antibody (2&#160;&#181;g/mL, Santa Cruz Biotechnology), Rab&#160;11 antibody (2.67&#160;&#181;g/mL, Proteintech), LAMP antibody (10&#160;&#181;g/mL, Thermo Scientific) coupled with secondary goat anti-rabbit Alexa Fluor 546 (10&#160;&#181;g/mL, Thermo Scientific). Signal specificity was verified using isotype control (normal rabbit IgG, normal goat IgG, Santa Cruz Biotechnology). Following this, cells were washed thrice with 0.5% Tween-20 in PBS. Nuclear DNA was stained with 4&#8242;,6&#8242;-diamidino-2-phenylindole (DAPI, 0.1&#160;&#956;g/mL, Sigma-Aldrich). Stained coverslips were mounted on glass slides in Prolong Diamond medium (Thermo Scientific). Fluorescent images were acquired using Zeiss LSM 710 confocal microscope (Carl Zeiss Microscopy&#160;GmbH).\nInhibitors influence on initial phase of infection\nCells pretreated with chemical inhibitors were incubated with concentrated CRCoV in the presence of inhibitors at 37&#160;&#176;C for 2&#160;h. Two hours pi cells were washed with PBS twice to remove unbound virus, fixed with 4% formalin in PBS (15&#160;min, RT) and permeabilized with 0.5% Tween-20. Unspecific binding sites were blocked using 5% BSA in PBS (4&#160;&#176;C, overnight) prior to staining. CRCoV was visualized using anti-coronavirus antibody OC43 strain (1&#160;&#181;g/mL, 2&#160;h, RT, Merck) coupled with goat anti-mouse Alexa Fluor 488 antibody (5&#160;&#181;g/mL, 1&#160;h, RT, Thermo Scientific). For this study Alexa-Fluor 647 Phalloidin (4&#160;U/mL, 1&#160;h, RT, Thermo Scientific) labeled actin cortex was assumed to indicate cell surface. Nuclear DNA was stained with DAPI (0.1&#160;&#956;g/mL, Sigma-Aldrich) and coverslips were mounted on glass slides in Prolong Diamond medium.\nPost-entry inhibitory effects\nCells were infected with CRCoV at TCID50 of 400/mL. After 2&#160;h incubation unbound virus was washed off with PBS and cells were overlaid with culture medium containing chemical inhibitors. Five days pi cells were harvested for further analysis.\nRole of furin during the infection\nCells were infected with CRCoV at TCID50 of 400/mL. After 2&#160;h incubation at 37&#160;&#176;C unbound virus was washed off with PBS and cells were overlaid with culture medium containing decanoyl-RVKR-chloromethyl ketone (dec-RVKR-CMK). Four days pi cells were fixed, stained and fluorescent images were acquired.\nsiRNA transfection\nHRT-18G cells were transfected with 25&#160;pmol of caveolin-1 siRNA (sc-29241, Santa Cruz Biotechnology) or scrambled negative control siRNA (sc-44237, Santa Cruz Biotechnology) using lipofectamine RNAiMAX reagent (Thermo Scientific) according to manufacturer&#8217;s protocol. Two consecutive transfections were performed 24&#160;h and 48&#160;h after cell seeding. Subsequently, cells were infected and prepared for imaging.\nComputer analysis\nAll graphs presented in this work were created using GraphPad Prism 6 software. Significance was estimated using one-way ANOVA with multiple comparisons to virus control. Images obtained from the confocal microscope were deconvolved using AutoQuant X3 and processed in ImageJ Fiji [23]. Co-localization analyses were performed in ImageJ using JACoP plugin [24] where Pearson&#8217;s and Manders&#8217; coefficient were calculated for 3D cell reconstructions. The bioinformatics analysis was conducted using arginine and lysine propeptide cleavage sites prediction algorithms ProP 1.0 server [25] using the following sequence data: AAO06124.1 (isolate 4182), ABG78748.1 (isolate 4182), ACX46840.1 (strain K9), AFW97360.1 (strain K37), ACX46860.1 (strain K39) and AQT26498.1 (strain BJ232).\nResults\n\nCRCoV enters HRT-18G cells via endocytosis\nWeak bases such as NH4Cl were previously described to hamper endosomal&#160;entry of viruses, by preventing pH-dependent activation of the fusion protein, subsequently blocking membrane fusion [26]. Treatment of HRT-18G cells with 50&#160;mM NH4Cl inhibited replication of CRCoV (5.9&#8201;&#177;&#8201;9.9% of control, Additional file 2), but did not inhibit virus entry into host cells (Figures&#160;1A&#160;and B). Bafilomycin A1 inhibits vacuolar H+-ATPases and blocks endosome acidification [27]. Similarly, bafilomycin&#160;A1 blocked virus replication (93.5%&#160;&#177;&#160;5.2% reduction in number of infected cells, compared to control) (Additional file 2), but not virus entry (Figures&#160;1A&#160;and B). Surprisingly, addition of bafilomycin&#160;A1 or NH4Cl 2&#160;h pi also resulted in decreased virus replication. Similar observations were made for the most of tested inhibitors of intracellular trafficking (Additional file 2). For this reason we decided to use for the analysis only the data on virus entry.Figure&#160;1Effect of endosome alkalization on CRCoV infection. A Analysis of inhibitors effect on CRCoV entry, cells were pretreated with inhibitors, infected in presence of them and incubated for 1&#160;h before they were washed off and fixed. CRCoV virions are presented in green, blue denotes DNA and red represents actin, each image is a single confocal plane while B represents maximum projections of their axial planes. Scale bar 10&#160;&#181;m.Consequently, to test whether the virus enters the cell by endocytosis, we examined co-localization of viral particles with early endosome antigen 1 (EEA1) at different times post-infection (pi). Co-localization was clearly visible (Figure&#160;2), peaking at 60&#8211;90 min pi (Mander&#8217;s coefficient 0.63&#8201;&#177;&#8201;0.18 at 60&#160;min and 0.64&#8201;&#177;&#8201;0.12 at 90&#160;min pi). No co-localization with the late endosome marker Rab 7, the lysosome marker LAMP1, or the endosome marker Rab&#160;11 was visible at any time (Additional files 3, 4).Figure&#160;2Co-localization of CRCoV with early endosome marker EEA1. Virus treated cells were synchronized on ice for 60&#160;min and incubated at 37&#160;&#8202;&#176;C before they were washed and fixed with 4% paraformaldehyde in PBS. Localization was analyzed by confocal microscopy after performing a double immunofluorescence staining to visualize virus nucleocapsid (left) and early endosomes (middle). Right panel shows merged image with virus presented as green and EEA1 as red. Cell nuclei are shown in blue. Control&#8212;negative control, cells incubated with mock sample (green) and stained with isotype antibodies (red). Scale bar 10&#160;&#181;m. Graph presents co-localization change in time.\nObtained results suggest that CRCoV undergoes fusion already in an early endosomal compartment, what may seem contradictory to some previous reports [2, 28&#8211;30]. However, some viruses are processed by furin in the producer cell, what makes the processing by cathepsins dispensable [31]. As in silico analysis predicted potential furin cleavage site in CRCoV spike glycoprotein gene (Additional file 5), effect of dec-RVKR-CMK (5&#8211;100&#160;&#181;M, Santa Cruz Biotechnology) on virus replication as well as cell-to-cell spread was tested. As visible on Figure&#160;3 no alteration of virus entry was observed.Figure&#160;3Effect of furin inhibitor&#8212;decanoyl-RVKR-chloromethyl ketone (CMK) on CRCoV replication (A) and cell to cell spread (B). A Graph shows number of virus positive cells at 4th day pi normalized to control. HRT-18G cells were pretreated with inhibitor before infection and it was present post-infection. B 3&#160;h pi cells were overlaid with CMK containing medium and were propagated in this conditions until fixation at 4th day pi. CRCoV virions are presented in green, blue denotes DNA and red represents actin. Scale bar 20&#160;&#181;M.\n\nInhibition of CRCoV entry into HRT-18G cells\nNext, we used inhibitors of endocytosis to examine the endocytic pathway utilized by CRCoV to enter HRT-18G cell. All inhibitors were used at the highest non-toxic concentration (determined in an XTT assay; data not shown). The validity of the obtained data was verified using positive controls (transferrin, cholera toxin subunit B (CHTxB), and dextran), which enter the cell via clathrin-dependent pathway, caveolin-dependent pathway, and macropinocytosis, respectively [9, 32].To determine whether CRCoV enters HRT-18G cells by clathrin-mediated endocytosis (CME), we treated cells with chlorpromazine, amantadine, or PitStop-2 [33, 34]. None of the compounds affected the virus entry to the cells (Figures&#160;4A&#160;and B).Figure&#160;4Effect of clathrin-mediated endocytosis inhibitors on CRCoV infection. A Analysis of inhibitors effect on CRCoV entry, cells were pretreated with inhibitors, infected in presence of them and incubated for 1&#160;h before they were washed off and fixed. CRCoV virions are presented in green, blue denotes DNA and red represents actin, each image is a single confocal plane while B represents maximum projections of their axial planes. Scale bar 10&#160;&#181;m.Caveolae, rich in cholesterol and sphingolipids, are disrupted by sterol-binding agents, as nystatin, filipin, or M&#946;CD. Filipin had no effect on virus entry; neither did it hamper internalization of CHTxB, implying ineffectiveness of this inhibitor on HRT-18G cell line. Nystatin and M&#946;CD blocked virus internalization to the cell (Figures&#160;5A&#160;and B), suggesting that caveolae are essential during CRCoV internalization. To ensure that the observed effect is not an artifact, caveolin-1 expression was silenced using siRNAs. As shown in Figure&#160;6, depletion of caveolin-1 resulted in reduction in number of virus particles entering the cell.Figure&#160;5Effect of caveolin-mediated endocytosis inhibitors on CRCoV. A Analysis of inhibitors effect on CRCoV infection, cells were pretreated with inhibitors, infected in presence of them and incubated for 1&#160;h before they were washed off and fixed. CRCoV virions are presented in green, blue denotes DNA and red represents actin, each image is a single confocal plane while B represents maximum projections of their axial planes. C Comparison of CRCoV and CHTxB co-localization with caveolin-1. Scale bar 10&#160;&#181;m.\nFigure&#160;6siRNA treatment. HRT-18G cells were transfected twice using scrambled negative control siRNA or caveolin-1 siRNA, infected, synchronized on ice for 60&#160;min and incubated at 37&#160;&#8202;&#176;C for 60 min before they were washed and fixed. CRCoV virions are presented in green, blue denotes DNA and red represents caveolin 1 (A) or actin (B), each image is a single confocal plane. C Negative control for staining. Bar 10&#160;&#181;m D Western blot signal of caveolin-1 and GAPDH expression in transfected cells. ctrl: control, sh: sham transfected cells, scRNA: scrambled RNA, siRNA: caveolin-1 siRNA. Scale bar 10&#160;&#181;m.To check whether the virus enters cells via macropinocytosis, we treated them with the NA+/H+ exchanger inhibitor EIPA, the Pak-1 inhibitor IPA-3, and the PI3K inhibitor wortmannin [35]. No effect on virus entry was, however, observed (Figures&#160;7A&#160;and B).Figure&#160;7Effect of macropinocytosis inhibitors on CRCoV infection. A Analysis of inhibitors effect on CRCoV entry, cells were pretreated with inhibitors, infected in presence of them and incubated for 1&#160;h before they were washed off and fixed. CRCoV virions are presented in green, blue denotes DNA and red represents actin, each image is a single confocal plane while B represents maximum projections of their axial planes. Scale bar 10&#160;&#181;m.\nCo-localization of virus particles with markers of endocytic pathways\nThe above results showed that only compounds that interfere with caveosome formation affect CRCoV entry. However, some reports show that these compounds have off-target and multi-target activity [36&#8211;39]. Therefore, to ensure the validity of the obtained data, we examined co-localization of virus particles with markers of different endocytic pathways. Cells were exposed to virus for 0&#8211;180&#160;min. As shown in Figure&#160;8, CRCoV co-localized with caveolin at 20&#160;min pi, and this was maintained for up to 120 min pi. Visual assessment and analysis of co-localization coefficients revealed that co-localization was more marked than for CHTxB, which utilizes the caveolin-1 mediated pathway to enter cells (Figures&#160;5C&#160;and 8D). We also examined whether the virus co-localizes with clathrin (Figure&#160;9) or endophilin (Additional file 6). No co-localization was detected.Figure&#160;8CRCoV co-localize with caveolin. Cells treated with virus were synchronized on ice for 60&#160;min and incubated at 37&#8202;&#160;&#176;C for 20, 40, 60, or 120&#160;&#8197;min before they were washed and fixed. Caveolin-1 are presented on A (red channel) while B shows stained virus nucleocapsid protein (green channel). C Depicts merged image with marked co-localization. D FITC conjugated CHTxB (green) co-localizing with caveolin-1 (red) after 40 (upper) and 60 (lower) min. E Negative controls for staining. Scale bar 10&#160;&#181;m.\nFigure&#160;9CRCoV (A) does not co-localize with clathrin in opposite to Alexa Fluor 488 conjugated transferrin (B). Cells treated with virus or transferrin were synchronized on ice for 60 min and incubated at 37&#160;&#176;C for 2, 5 or 20&#8197;min before they were washed and fixed. Clathrin HC are presented in red while stained virus nucleocapsid protein in green. C Negative controls for staining. D Co-localization changes in time. Scale bar 10&#160;&#181;m.\nDynamin is important for CRCoV internalization\nDynamin is essential for several endocytic pathways [14, 40, 41]. Therefore, we asked whether this GTPase plays a role in CRCoV entry into cells. For this, we used three inhibitors of dynamin, which inhibited GTPase activity (dynasore and iminodyn-22) [42, 43] or blocked the lipid binding (MiTMAB) [44]. All three inhibitors affected virus replication: dynasore by 87.8&#8201;&#177;&#8201;7.5%, iminodyn-22 by 60.0&#8201;&#177;&#8201;1.5%, and MiTMAB by 94.1&#8201;&#177;&#8201;3.7% (Additional file 2) when added before infection. What is even more important, their effect was limited when added only post-infection (Additional file 2), suggesting that dynamin is important at early stages of infection. All inhibitors caused a marked reduction in the number of virions entering the cell (Figures&#160;10A&#160;and B). Recently, Xu et al. [45] studied JEV entry into the cells and proposed a new model&#8212;actin- and dynamin-dependent caveolae-mediated endocytosis. Thus, we analyzed the effect of inhibiting Rho, Rac1, and ROCK kinases on virus infection. Only inhibition of Rac1 (by NSC23766) hampered viral replication showing significantly stronger effect when present prior to infection (7.9%&#8201;&#177;&#8201;9.3 of control) than after (51.9%&#8201;&#177;&#8201;3.6 of control) (Additional file 2) implying its involvement in entry process.Figure&#160;10Vesicle scission&#8212;dynamin dependence. A Analysis of inhibitors effect on CRCoV entry, cells were pretreated with inhibitors, infected in presence of them and incubated for 1&#160;h before they were washed off and fixed. CRCoV virions are presented in green, blue denotes DNA and red represents actin, each image is a single confocal plane while B represents maximum projections of their axial planes. Scale bar 10&#160;&#181;m.\nCytoskeleton\nThe results so far suggest that CRCoV enters HRT-18G cells via caveolin- and dynamin-dependent endocytosis, and that it is likely that fusion occurs early after internalization. Therefore, we examined the role of the cytoskeleton during virus entry. First, we evaluated the effect of an actin-disrupting agent (cytochalasin D), an actin stabilizing compound (jasplakinolide), and an inducer of microtubule depolimerization (nocodazole) [46, 47]. Additional file 2 shows that, while microtubules are not important for virus infection, actin plays a central role during the early stages of CRCoV infection. Disrupting actin filaments led to a reduction (by 84.5%&#8201;&#177;&#8201;8.9) in viral infection, although filament stabilization had no effect on virus yield. Lack of this effect after post-entry treatment (Additional file 2) imply that actin is required during early events of CRCoV infection. Studies of virus localization revealed that after treatment with cytochalasin&#160;D, multiple viral particles localize to actin aggregates (Figures&#160;11A&#160;and B); this implies that an intact actin cytoskeleton plays a role in intracellular transport of viruses.Figure&#160;11Effect of cytoskeleton reorganization on CRCoV entry. A Analysis of inhibitors effect on CRCoV entry, cells were pretreated with inhibitors, infected in presence of them and incubated for 1&#160;h before they were washed off and fixed. CRCoV virions are presented in green, blue denotes DNA and red represents actin, each image is a single confocal plane while B presents maximum projection of their axial planes. Scale bar 10&#160;&#181;m.\nDiscussion\nTo propagate, viruses need to deliver their genetic material into the host cell. To do this, enveloped viruses fuse with the cellular membrane prior to ejecting genomic RNA or DNA into the cytoplasm. This may take place on the cell surface or after endocytosis. Here, we identified the pathway by which CRCoV enters human epithelial cells (HRT-18G) [8].First, we asked whether CRCoV enters cells via endocytosis. Sensitivity to lysosomotropic agents is considered good evidence of endocytosis [48]; therefore, we treated cells with inhibitors of endosome acidification (bafilomycin A and ammonium chloride). Both compounds hampered virus infection, suggesting that CRCoV enters HRT-18G cells via endocytosis rather than via direct membrane fusion. However, inhibitory effect of inhibitors altering the endosomal pH was observed also on the replication stage. Similar phenomenon was previously reported for SARS-CoV [49]. It was suggested that these compounds interfere with glycosylation of the viral proteins and in such a way suppress their expression and assembly [49, 50]. Nonetheless, we decided to present the data on virus replication in supplementary material and focus on virus entry to the cell.No single pathway of entry has been reported for Coronaviridae. Indeed, studies have identified clathrin-dependent, caveolin-dependent, clathrin-and-caveolin-independent, and endocytosis-independent routes (see Table 1). To identify the route of entry used by CRCoV, we utilized inhibitors of several different pathways (see Additional file 1). However, it is worth remembering that chemical inhibitors may affect other phases of virus replication by interfering with viral and cellular proteins (e.g., kinases, GTPases) [32, 51]. Indeed, we observed such non-specific interaction and consequently it was not possible to test the effect of chemical inhibitors on virus replication, as virus yields were reduced by almost all compounds affecting intracellular trafficking. To further address this issue, we employed complementary approaches: siRNA-mediated knockdown of caveolin-1 and analysis of co-localization of CRCoV viral particles with markers of different intracellular compartments. The results revealed that CRCoV enters the cell via the caveolin-1 dependent pathway; indeed, CRCoV particles co-localized with endosomes coated with caveolin-1, but not with clathrin or endophilin (Additional file 6). Furthermore, siRNA-mediated depletion of caveolin-1 inhibited virus entry.Table 1Entry pathways described for CoronavirusesVirusGenusEntry routeCellsReferencesHCoV-229EAlphapH dependent endocytosisCaco-2 (human colon adenocarcinoma), MRC-5 (human lungs)[59]Caveolin-dependent endocytosis, lipid raft mediatedHuman primary fibroblasts, L132 (human embryonic lung cell line)[15]Endocytosis&#8212;laboratory strains, membrane fusion&#8212;clinical isolatesHeLa-229 (human cervix), HAE (human airway epithelium)[18]Membrane fusionHEK 293T (human embryonic kidney), Caco-2 (human colon adenocarcinoma)[16]HCoV-NL63AlphaClathrin-dependent endocytosisLLC-MK2 (rhesus monkey kidney) HAE (human airway epithelium)[60]Feline infectious peritonitisAlphaClathrin- and caveolin-independent endocytosisPrimary feline monocytes[14]Transmissible gastroenteritis virusAlphaLipid raft mediatedST (swine testicles)[61]Canine enteric CoronavirusAlphaLipid raft mediatedA72 (canine fibroma)[62]Mouse hepatitis virusBetaCell membrane fusionMouse 3T3-L2 fibroblast[63]pH dependent endocytosisMurine fibroblast[64]Lipid raft mediatedDBT cells (mouse astrocytoma)[65]SARS-CoVBetaClathrin-dependent endocytosisHepG2 (human hepatoma)[13]Membrane fusionVero E6 (African green monkey kidney)[66]Lipid raft mediatedVero E6 (African green monkey kidney)[11]Clathrin- and caveolin-independent endocytosisHEK 293E (human embryonic kidney) , Vero E6 (African green monkey kidney)[12]Membrane fusionVero E6 (African green monkey kidney), HEK 293T (human embryonic kidney)[67]MERS-CoVBetaVirus-cell fusionHEK 293T (human embryonic kidney), Caco-2 (human colon adenocarcinoma)[17]HCoV-OC43BetaCaveolin-dependent endocytosisHCT-8 (human adenocarcinoma)[68]Avian infectious bronchitis virusGammaLow-pH-dependent virus-cell fusionBHK (hamster kidney)[69]Lipid raft mediatedVero (African green monkey kidney)[70]Next, we focused on dynamin, an important GTPase involved in vesicle scission during endocytosis. While it is most commonly associated with clathrin-mediated endocytosis, it also plays a role in phagocytosis and caveolin-1, ILR-2-, and flotillin-dependent endocytosis [14, 40, 41]. To do this, we used three dynamin inhibitors, two that inhibit GTPase activity and one that blocks the lipid-binding domain of dynamin. The results led us to conclude that CRCoV entry into HRT-18G cells is dynamin-dependent.In order to monitor intracellular trafficking of CRCoV and to determine the specific point of virus-cell membrane fusion, we examined co-localization of the CRCoV nucleocapsid protein and endosomal vesicle markers. The results showed that CRCoV was present in early endosomes but absent from late and recycling endosomes and lysosomes. This indicates that fusion of CRCoV and cellular membranes occurs at an early stage, before the endosome recruit Rab&#160;7 and progresses to the late phase. This is different from the majority of coronaviruses, which travel far along the endocytic pathway to access high cathepsin activity (e.g., SARS-CoV, MHV, FIPV, porcine epidemic diarrhea coronavirus) [2, 28, 30]. The pathway used by CRCoV is more similar to that used by MERS-CoV reported to utilize furin as well as transmembrane protease serine 2 (TMPRSS2) [2, 17, 28, 29, 31]. We tested the influence of furin inhibitor decanoyl-RVKR-chloromethyl ketone in order to study the importance of furin for CRCoV entry. No inhibition of virus replication or cell to cell spread was noted, showing that furin processing is not relevant for CRCoV entry. One may however speculate that regardless of fusion with the host membrane occurring early after internalization, CRCoV may still utilize cathepsins similarly to most coronaviruses. Even if cathepsins is considered to require acidic pH for activation this dependency varies according to substrate and so far a few substrates were described to be cleaved in slightly acidic pH comparable to that achieved before endosome maturation [52&#8211;54]. Another potential explanation is cleavage by other proteases such as cell surface TMPRSS2 recently reported to be preferred over cathepsins by several coronaviruses [18, 29].Our last objective was to study the role of the cytoskeleton during CRCoV infection. We discovered that in cells treated with cytochalasin D prior to infection, virus replication was reduced and during entry the virions accumulated on actin aggregates. Surprisingly, another actin inhibitor, jasplakinolide, had no significant effect on virus replication or entry. Interestingly, cytochalasin D and jasplakinolide affect the actin cytoskeleton via opposing mechanisms: cytochalasin D inhibits polymerization of actin subunits, whereas jasplakinolide stabilizes filaments by inhibiting depolymerization [46, 55]. The results obtained using these two agents suggest that actin filaments, but not actin reorganization, are required for CRCoV entry.Coronaviruses are a family of viruses with high zoonotic potential. As such, they present a real threat to the global economy and to public health [4, 56&#8211;58]. Here, we show that CRCoV relies on dynamin-dependent, caveolin-1-mediated endocytosis to enter host cells. Moreover, we showed that fusion with the host membrane occurs early after internalization.\nAdditional files\n\nNone\n\nAdditional file 1.\nInhibitors of endocytosis and their modes of action. The table consists of a list of inhibitors used in this study with detailed information regarding their provider and applied concentration.\nAdditional file 2.\nChemical inhibitors effect on CRCoV infection. Graphs shows number of virus positive cells at 5th day pi normalized to control. HRT-18G cells were treated with acidification (A), dynamin (B), cell kinases (C), cytoskeleton (D), clathrin (E), macropinocytosis (F) and caveolin (G) inhibitors. Compounds were present prior and during the infection (white) or only after infection (gray). Cells were propagated in their presence until harvested.\nAdditional file 3.\nCo-localization of CRCoV with markers of late endosomes and lysosomes. A. CRCoV do not co-localize with late endosomes marker Rab7. B. CRCoV do not co-localize with lysosome marker LAMP1. C. Negative control D. Co-localization change in time. Cells treated with virus were synchronized on ice for 60 min and incubated at 37&#8202;&#160;&#176;C before they were washed and fixed. Rab7 and LAMP1 are presented in red and CRCoV nucleocapsid protein in green. Cell nuclei are blue. Scale bar 10&#160;&#181;m.\nAdditional file 4.\nCRCoV do not co-localize with recycling endosomes marker Rab11. Cells treated with virus were synchronized on ice for 60 min and incubated at 37&#160;&#8202;&#176;C before they were washed and fixed. Rab11 are presented in red and CRCoV nucleocapsid protein in green. Cell nuclei are blue. Scale bar 10&#160;&#181;m. Graph presents co-localization change in time.\nAdditional file 5.\nPotential furin cleavage site prediction Graphs show potential furin cleavage sites in the spike protein sequence of CRCoV isolate 4182 (A, B), K9 strain (C), K37 strain (D), K39 strain (E) and BJ232 strain (F).\nAdditional file 6.\nCRCoV do not co-localize with endophilin. Cells treated with virus were synchronized on ice for 60 min and incubated at 37&#8202;&#160;&#176;C before they were washed and fixed. Endophilin are presented in red and CRCoV nucleocapsid protein in green. Cell nuclei are blue. Scale bar 10 &#181;m. Graph presents co-localization change in time.\n",
        "pmid": "29970183",
        "title": "Canine respiratory coronavirus employs caveolin-1-mediated pathway for internalization to HRT-18G cells",
        "journal_title": "Veterinary Research",
        "authors": "Artur Szczepanski;Katarzyna Owczarek;Aleksandra Milewska;Zbigniew Baster;Zenon Rajfur;Judy A. Mitchell;Krzysztof Pyrc",
        "doi": "10.1186/s13567-018-0551-9"
    },
    {
        "keywords": "disease;pathogenesis;poultry;reservoir;SARS-coronavirus",
        "abstract": "SARS coronavirus injected intratracheally into chickens, turkeys, geese, ducks, and quail, or into the allantoic sac of their embryonating eggs, failed to cause disease or replicate. This finding suggests that domestic poultry were unlikely to have been the reservoir, or associated with dissemination, of SARS coronavirus in the animal markets of southern China.",
        "body": "",
        "pmid": "15200830",
        "title": "Domestic Poultry and SARS Coronavirus, Southern China",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "David E. Swayne;David L. Suarez;Erica Spackman;Terrence M. Tumpey;Joan R. Beck;Dean Erdman;Pierre E. Rollin;Thomas G. Ksiazek",
        "doi": "10.3201/eid1005.030827"
    },
    {
        "keywords": "coronavirus;HCoV-229E;spike protein;post-fusion core;X-ray structure;SARS;MERS",
        "abstract": "The complete post-fusion core structure of the Human coronavirus 229E spike protein was determined at 1.86&#8197;&#197; resolution. Comparison of the interactions between heptad repeats HR1 and HR2 in different human coronaviruses reveals some differences, which should be taken into consideration when designing pan-coronavirus HR2-mimicking inhibitors that target HR1.",
        "body": "Introduction \u00a0 \nCoronaviruses (CoVs) are enveloped, positive-sense, single-stranded RNA viruses that generally cause upper respiratory and enteric infections in mammals and birds (Enjuanes et al., 2006 &#9656;; Perlman & Netland, 2009 &#9656;). To date, six different strains of CoVs have been found to infect humans (hereafter referred to as HCoVs), and these include Severe acute respiratory syndrome-related coronavirus (SARS), Middle East respiratory syndrome-related coronavirus (MERS), Human coronavirus HKU1 (HCoV-HKU1), Human coronavirus OC43 (HCoV-OC43), Human coronavirus 229E (HCoV-229E) and Human coronavirus NL63 (HCoV-NL63) (de Wilde et al., 2017 &#9656;; Perlman & Netland, 2009 &#9656;). Of these HCoVs, SARS (Marra et al., 2003 &#9656;; Ksiazek et al., 2003 &#9656;; Peiris, Lai et al., 2003 &#9656;; Rota et al., 2003 &#9656;) and MERS (de Groot et al., 2013 &#9656;; Zaki et al., 2012 &#9656;) are notorious for their high mortality rates [10% for SARS (Peiris, Yuen et al., 2003 &#9656;; Stadler et al., 2003 &#9656;) and 35% for MERS (World Health Organization, 2016 &#9656;)] and pandemic potential. HCoV-229E, on the other hand, is endemic and is frequently associated with respiratory diseases ranging from the common cold to atypical pneumonia (van der Hoek, 2007 &#9656;). In healthy adults, HCoV-229E infection usually causes mild symptoms, including headache, cough, fever and rhinitis (Greenberg, 2016 &#9656;), but can lead to compromised productivity in infected individuals and result in a large impact on the economy. Moreover, HCoV-229E infection can result in severe or even lethal outcomes in infants, young children, seniors and those with weakened immune systems (Forgie & Marrie, 2009 &#9656;; Jartti et al., 2011 &#9656;). Therefore, it is important to investigate the structure and function of HCoV-229E to aid in the development of preventative or therapeutic strategies.As enveloped viruses, CoVs deliver their viral genomes into the host cytoplasm by fusing the viral membrane with either the host cell or endosome membrane (Belouzard et al., 2012 &#9656;; Hulswit et al., 2016 &#9656;). The spike glycoprotein (S) is an &#8764;500&#8197;kDa homotrimer on the envelope surface of CoVs that plays a fundamental role in cell entry, including receptor binding and membrane fusion (Li, 2016 &#9656;). By analogy to other class I viral fusion proteins, such as influenza virus haem&#173;agglutinin (Wilson et al., 1981 &#9656;) or the Env protein of HIV-1 (Julien et al., 2013 &#9656;; Lyumkis et al., 2013 &#9656;), the S protein of CoVs can be roughly divided into two structural and functional domains: an N-terminal S1 domain for engagement with cellular receptors and a C-terminal S2 domain for membrane fusion (Belouzard et al., 2012 &#9656;; Li, 2016 &#9656;). The S2 domain contains a fusion peptide (FP), two heptad repeats (HRs) and a transmembrane helix (TM), arranged as FP&#8211;HR1&#8211;HR2&#8211;TM (Liu et al., 2004 &#9656;).CoV S proteins are synthesized as a single polypeptide chain, and a two-step proteolysis of the S protein prior to viral entry is pivotal for viral&#8211;host membrane fusion (Belouzard et al., 2009 &#9656;; Millet & Whittaker, 2014 &#9656;; Park et al., 2016 &#9656;). Specifically, proteolytic cleavage at the S1&#8211;S2 domain junction removes the structural constraint of S1 on S2 (Li, 2016 &#9656;; Belouzard et al., 2012 &#9656;; Millet & Whittaker, 2015 &#9656;), a process in which binding of the CoV S1 domain to its cognate receptor may also contribute (Kirchdoerfer et al., 2016 &#9656;; Yuan et al., 2017 &#9656;; Zelus et al., 2003 &#9656;; Matsuyama & Taguchi, 2002 &#9656;). At the same time, cleavage at an additional S2&#8242; site also helps to untether the fusion peptide (FP; Li, 2016 &#9656;; Millet & Whittaker, 2015 &#9656;; Belouzard et al., 2012 &#9656;). Following this two-step cleavage, the otherwise buried hydrophobic FP becomes exposed and available to insert into the host cell membrane (Yao et al., 2016 &#9656;). Subsequent conformational rearrangements within S2 result in the formation of the 3HR1&#8211;3HR2 six-helical bundle, thus bringing viral and cellular membranes into close prox&#173;imity and facilitating membrane fusion (Baquero et al., 2013 &#9656;; Heald-Sargent & Gallagher, 2012 &#9656;).As illustrated previously, the six-helical bundles from SARS, MERS and HCoV-NL63 feature an extended triple-helical coiled coil (HR1) surrounded by three antiparallel HR2 &#945;-helices (Supekar et al., 2004 &#9656;; Xu et al., 2004 &#9656;; Duquerroy et al., 2005 &#9656;; Zheng et al., 2006 &#9656;; Gao et al., 2013 &#9656;; Lu et al., 2014 &#9656;). Such a conformation represents the post-fusion state of CoV spike proteins and is known as the fusion-core structure. More recently, the metastable pre-fusion states of Mouse hepatitis virus (MHV), HCoV-HKU1, HCoV-NL63, SARS and MERS have also been determined to high resolution by cryo-EM (Walls, Tortorici, Bosch et al., 2016 &#9656;; Kirchdoerfer et al., 2016 &#9656;; Walls, Tortorici, Frenz et al., 2016 &#9656;; Gui et al., 2017 &#9656;; Yuan et al., 2017 &#9656;). In the pre-fusion state, the S1 domains of CoVs all reside above the trimeric S2 stalk, thereby exerting structural constraints on S2 (Walls, Tortorici, Bosch et al., 2016 &#9656;; Walls, Tortorici, Frenz et al., 2016 &#9656;; Kirchdoerfer et al., 2016 &#9656;; Gui et al., 2017 &#9656;; Yuan et al., 2017 &#9656;). In contrast to their extended helical conformations in the post-fusion state (Xu et al., 2004 &#9656;; Gao et al., 2013 &#9656;; Duquerroy et al., 2005 &#9656;; Supekar et al., 2004 &#9656;; Zheng et al., 2006 &#9656;; Lu et al., 2014 &#9656;), the HR1 motifs within S2 from HCoVs form several shorter helices in their pre-fusion state (Yuan et al., 2017 &#9656;; Walls, Tortorici, Frenz et al., 2016 &#9656;; Gui et al., 2017 &#9656;). Together, these snapshots of the pre-fusion and post-fusion states of S proteins further consolidate the prevailing paradigm on CoV membrane fusion and enhance our knowledge about this complex membrane-fusion apparatus.Owing to its pivotal role in the host entry of CoVs, the spike protein has been a focus of anti-HCoV drug development. To date, a handful of viral inhibitors that target the spike proteins of SARS and MERS have been reported (Hu et al., 2005 &#9656;; Han et al., 2006 &#9656;; Yi et al., 2004 &#9656;; Bosch et al. 2004 &#9656;; Liu et al., 2004 &#9656;; Lu et al., 2014 &#9656;, 2015 &#9656;; Gao et al., 2013 &#9656;; Channappanavar et al., 2015 &#9656;). A subgroup of these inhibitors specifically target the interaction between the S1 domain of the SARS spike protein and its corresponding host receptors (Hu et al., 2005 &#9656;; Han et al., 2006 &#9656;). The remainder of these inhibitors target HR1s in the S2 domain of SARS and MERS by mimicking the corresponding HR2s, thereby inhibiting formation of the 3HR1&#8211;3HR2 helical bundle (Bosch et al., 2004 &#9656;; Liu et al., 2004 &#9656;; Lu et al., 2014 &#9656;, 2015 &#9656;; Gao et al., 2013 &#9656;; Channappanavar et al., 2015 &#9656;). Both groups of inhibitors are effective against specific HCoVs, at least in vitro. Nevertheless, the hypervariable nature of the S1 domain and the highly conserved features of the S2 domain render the latter a more suitable target for the development of pan-CoV inhibitors. As CoVs are rapidly evolving (Forni et al., 2016 &#9656;), pandemic outbreaks of newly emerging HCoVs become possible when zoonotic CoVs are able to accumulate sufficient mutations to cross the species barrier (Ge et al., 2013 &#9656;; Menachery et al., 2015 &#9656;, 2016 &#9656;). Thus, a broad-spectrum anti-HCoV drug or vaccine would be more effective than a specific one. To this end, structural profiling of the evolutionarily conserved S2 domain, especially the 3HR1&#8211;3HR2 six-helical bundle that drives the membrane-fusion process, would undoubtedly provide valuable information to help in the design of such a broad-spectrum anti-HCoV HR2 mimic.Of the six HCoVs, HCoV-NL63 and HCoV-229E belong to the alphacoronavirus (&#945;-CoV) family, while the other four HCoVs are classified as betacoronaviruses (&#946;-CoVs) (Woo et al., 2009 &#9656;). Among them, HCoV-229E was the first to be identified (Hamre & Procknow, 1966 &#9656;). However, its spike protein has been the least structurally studied. The host receptor of HCoV-229E was revealed to be human aminopeptidase N (APN) more than two decades ago (Yeager et al., 1992 &#9656;). Nevertheless, structural elucidation of the interaction between HCoV-229E and APN was only revealed very recently (Wong et al., 2017 &#9656;). More recently, a crystal structure of the HCoV-229E fusion core (HR1 residues 788&#8211;838 and HR2 residues 1060&#8211;1097) was reported at 2.45&#8197;&#197; resolution (Zhang et al., 2018 &#9656;) while we were preparing this manuscript. Here, we present a more complete fusion-core structure of HCoV-229E at 1.86&#8197;&#197; resolution (HR1 residues 785&#8211;873 and HR2 residues 1052&#8211;1105) with an eight-turn longer HR1 helix and at higher resolution. The fusion-core structure of HCoV-229E is similar overall to those of SARS, MERS and HCoV-NL63. Nevertheless, since HCoV-229E has a longer HR1 and HR2, the packing of its hydrophobic core and the buried interface between HR1 and HR2 differ from those in SARS and MERS (Supekar et al., 2004 &#9656;; Lu et al., 2014 &#9656;; Xu et al., 2004 &#9656;; Duquerroy et al., 2005 &#9656;; Gao et al., 2013 &#9656;) and are more similar to those in HCoV-NL63, another human &#945;-CoV (Zheng et al., 2006 &#9656;). Structural studies of the HCoV spike protein may be used to develop effective vaccines and drugs against HCoVs. We thus anticipate that our structure and the recently solved fusion-core structure of HCoV-229E, along with those of SARS, MERS and NL63, may serve as a template for the design of structure-based pan-coronavirus small molecules or peptides to inhibit viral fusion.\nMaterials and methods \u00a0 \n\nPlasmid construction \u00a0 \nTo express the HCoV-229E fusion core, the coding sequences for HR1 (residues 785&#8211;873) and HR2 (residues 1052&#8211;1105) were connected via a short linker (encoding L6; SGGRGG) by overlapping PCR. The resulting sequence HR1-L6-HR2 was then subcloned into the pET-28a vector with an N-terminal in-frame SUMO tag. The final construct thus contains an N-terminal His6 tag followed by the SUMO domain including the recognition sequence for Ulp1 and the coding sequence for HR1-L6-HR2.\nProtein production and purification \u00a0 \nFor protein production, the pET-28-SUMO-HR1-L6-HR2 plasmid was transformed into Escherichia coli strain BL21 (DE3) competent cells. A single colony was inoculated into 5&#8197;ml Luria&#8211;Bertani (LB) medium containing 100&#8197;&#181;g&#8197;ml&#8722;1 kanamycin (AMRESCO) and incubated overnight at 37&#176;C. The overnight culture was then seeded into 2&#8197;l fresh LB medium and cultured at 37&#176;C until the OD600 (optical density at 600&#8197;nm) reached 0.6. Target protein overexpression was then induced with 1&#8197;mM isopropyl &#946;-d-1-thiogalactopyranoside (IPTG; AMRESCO) at 16&#176;C for 16&#8197;h. After harvesting via centrifugation at 16&#8197;000&#8197;rev&#8197;min&#8722;1 for 30&#8197;min, the cell debris was resuspended in lysis buffer (50&#8197;mM Tris&#8211;HCl pH 7.5, 300&#8197;mM NaCl) supplemented with phenylmethylsulfonyl fluoride (PMSF; Biovision) and lysed using a sonicator (Thermo Fisher Scientific). The cell lysate was then centrifuged at 70&#8197;000g for 20&#8197;min at 4&#176;C. The supernatant was collected and loaded onto TALON metal-affinity resin (Clontech). After extensive washing, the HR1-L6-HR2 protein of interest was eluted with lysis buffer supplemented with 500&#8197;mM imidazole. The fractions eluted from the TALON column were then dialyzed against dialysis buffer (20&#8197;mM Tris&#8211;HCl pH 8.0, 5&#8197;mM &#946;-mercaptoethanol, 150&#8197;mM NaCl) overnight at 4&#176;C and then processed with Ulp1 [at a 1:100(w:w) ratio] to remove the SUMO tag from HR1-L6-HR2. Finally, the cleaved product was applied onto a Superdex 75 gel-filtration column (GE Healthcare). Fractions containing a homogeneous HR1-L6-HR2 trimer were collected and concentrated by ultrafiltration using Amicon Ultra-4 10&#8197;kDa centrifugal filter units (Millipore). All chemicals were from Sigma&#8211;Aldrich unless otherwise specified.\nCrystallization, data collection and structure determination \u00a0 \nThe HR1-L6-HR2 fusion protein was crystallized at 20&#176;C using the hanging-drop vapour-diffusion method. Crystals were grown on a siliconized cover slip by equilibrating a mixture consisting of 1&#8197;&#181;l protein solution (8&#8197;mg&#8197;ml&#8722;1 HR1-L6-HR2 trimer in 20&#8197;mM Tris&#8211;HCl pH 8.0, 150&#8197;mM NaCl) and 1&#8197;&#181;l reservoir solution (0.03&#8197;M citric acid, 0.07&#8197;M bis-tris propane pH 7.6, 20% PEG 3350) against 400&#8197;&#181;l reservoir solution. Single crystals grew after one week and were flash-cooled in liquid nitrogen for data collection after adding 20% glycerol as a cryoprotectant.Diffraction data were collected on beamline BL17U1 of the Shanghai Synchrotron Radiation Facility (SSRF) using a wavelength of 0.9795&#8197;&#197; and an ADSC Q315r CCD detector. 360 images were recorded at a distance of 250&#8197;mm with a 1&#176; oscillation angle and an exposure time of 1.0&#8197;s. Raw data were indexed and processed using HKL-3000 in space group R32:H. Molecular replacement was performed with Phaser in PHENIX (Adams et al., 2010 &#9656;) using the HCoV-NL63 fusion-core structure (PDB entry 2ieq; Zheng et al., 2006 &#9656;) as a search model. The initial model was further improved by cycles of manual building and refinement using Coot (Emsley et al., 2010 &#9656;) and PHENIX (Adams et al., 2010 &#9656;). Torsion-angle NCS restraints and TLS groups were used during refinement. The quality of the final model was analysed with MolProbity (Chen et al., 2010 &#9656;). Data-collection and refinement statistics are shown in Table 1 &#9656;. Atomic coordinates and structure factors have been deposited in the Protein Data Bank (PDB entry 5yl9). Figures were prepared with PyMOL (http://www.pymol.org). Electrostatic calculations were performed with PDB2PQR (Dolinsky et al., 2004 &#9656;). The volumes of cavities and pockets were calculated with the CASTp 3.0 online server (Dundas et al., 2006 &#9656;). Pitch values were calculated with TWISTER (Strelkov & Burkhard, 2002 &#9656;).\nResults \u00a0 \n\nOverall architecture of the HCoV-229E fusion core \u00a0 \nThe HR1 (residues 785&#8211;873) and HR2 (residues 1052&#8211;1105) regions of HCoV-229E were connected via a six-amino-acid linker (L6; SGGRGG), thereby generating an HR1-L6-HR2 fusion protein (Fig. 1 &#9656;\na). Similar linkers have been utilized in structural studies of MHV and HCoV-NL63 fusion cores and were found not to affect the intrinsic interaction (or packing) between HR1 and HR2 (Xu et al., 2004 &#9656;; Zheng et al., 2006 &#9656;). The HR1-L6-HR2 construct crystallized in space group R32:H, with unit-cell parameters a&#160;=&#160;b = 46.9, c = 402.1&#8197;&#197; and one molecule per asymmetric unit (Table 1 &#9656;). The structure was solved by molecular replacement using the crystal structure of the HCoV-NL63 fusion core (PDB entry 2ieq) as the search model, and was refined to a final resolution of 1.86&#8197;&#197; with an R\ncryst of 18.4% and an R\nfree of 20.6% (Table 1 &#9656;).HR1 of HCoV-229E forms a 24-turn &#945;-helix, while HR2 of HCoV-229E adopts a mixed conformation: residues 1067&#8211;1098 fold into a nine-turn &#945;-helix, while residues 1052&#8211;1066 and 1099&#8211;1105 on either side of the helix adopt an extended conformation (Fig. 1 &#9656;\nb). The HR1 regions of three HR1-L6-HR2 molecules are arranged around the crystallographic threefold symmetry axis and form the central hydrophobic core (Fig. 1 &#9656;\nc). The three companion HR2 regions pack on the outside into the hydrophobic grooves that are formed between adjacent HR1 helices (Figs. 1 &#9656;\nc and 1 &#9656;\nd) in an antiparallel manner, resulting in a six-helical bundle structure with dimensions of &#8764;128&#8197;&#197; in length and 32&#8197;&#197; in diameter with a left-handed supercoil (Fig. 1 &#9656;\nc). Such a structure is typical for the CoV fusion core and represents the post-fusion state of the HCoV-229E spike protein. As our structure is at higher resolution and the HR1 and HR2 motifs are more complete than in Zhang et al. (2018 &#9656;), we therefore used our HCoV-229E structure in the following comparisons of different HCoVs.\nHydrophobic core packing of the long HR1 from HCoV-229E \u00a0 \nSequence alignment of HCoVs indicates that a 14-amino-acid insertion arises in the HR1 regions of HCoV-NL63 and HCoV-229E (Fig. 1 &#9656;\nf) that would make the HR1 helix of HCoV-229E four turns (two heptad repeats) longer than those of MERS and SARS (Fig. 1 &#9656;\ne; insertion highlighted in red). To compensate for their longer HR1 regions, the HR2 region of HCoV-229E or HCoV-NL63 also contains a 14-residue insertion (Fig. 1 &#9656;\ng) that contributes to a longer HR2 helix (Fig. 1 &#9656;\ne; insertion highlighted in yellow).The 24-turn &#945;-helix of HCoV-229E HR1 is the longest among all HCoV fusion-core structures published thus far (Fig. 2 &#9656;\na). 26 hydrophobic residues as well as three polar residues (Gln791, Thr819 and Gln840) from each HR1 line the coiled-coil interface and pack in layers to form the hydrophobic 3HR1 core of HCoV-229E (Fig. 2 &#9656;\na, first panel). Besides the canonical &#8216;a&#8217; and &#8216;d&#8217; layers, which are a feature of heptad repeats, Zheng and coworkers introduced the concept of &#8216;x&#8217; and &#8216;da&#8217; (or &#8216;y&#8217;) layers to illustrate the noncanonical packing modes observed in the structure of the HCoV-NL63 fusion core (Zheng et al., 2006 &#9656;). Such &#8216;x&#8217; and &#8216;da&#8217; packing modes are also observed here for the 3HR1 core of HCoV-229E (Fig. 2 &#9656;\na, first panel). Interestingly, the four additional turns that are unique to &#945;-HCoVs (Fig. 1 &#9656;\nf) all adopt noncanonical packing geometry, with &#8216;d/a&#8217; layers alternating with &#8216;x&#8217; layers (Fig. 2 &#9656;\na, highlighted in red). Re-analysis of published SARS and MERS fusion cores reveals that &#8216;x&#8217; and &#8216;da&#8217; mode packing also exists in the 3HR1 cores of MER and SARS, albeit less prevalently than in &#945;-HCoVs (HCoV-229E and HCoV-NL63) (Fig. 2 &#9656;\na). In the standard heptad repeats, although the &#8216;a&#8217; or &#8216;d&#8217; residues (knobs) interact around the trimer threefold axis, they do so in such a way that their side chains are more laterally arranged and fit in between the &#8216;a&#8217; and &#8216;g&#8217; residues (holes) or the &#8216;d&#8217; and &#8216;e&#8217; residues (holes), respectively (Fig. 2 &#9656;\nb, panels 1 and 2). In the noncanonical &#8216;x&#8217; layer, each packing residue points its side chain towards the centre of the threefold symmetry axis (Fig. 2 &#9656;\nb, third panel); while in the &#8216;da&#8217; layer the &#8216;a&#8217; and &#8216;d&#8217; packing residues are arranged in a pseudo-hexagonal manner through hydrophobic interactions, thereby leaving a large unfilled space in the middle (Fig. 2 &#9656;\nb, fourth panel). As a consequence, a central cavity is always seen in the &#8216;da&#8217; layer (Fig. 2 &#9656;\na; cavity shown as a yellow surface). As calculated with the CASTp 3.0 server (Dundas et al., 2006 &#9656;), the molecular surface volume distribution of these cavities varies in different HCoVs. For HCoV-229E, the largest cavity seen in the Ala815/Ile816 &#8216;da&#8217; layer has a volume of 228&#8197;&#197;3, while the volumes of the cavities in the other &#8216;da&#8217; layers all fall between 20 and 50&#8197;&#197;3; for HCoV-NL63, the largest cavity was in the Ala996/Ile997 &#8216;da&#8217; layer, with a volume of 206&#8197;&#197;3, and the other cavities in &#8216;da&#8217; layers are all smaller than 55&#8197;&#197;3; for MERS, the largest cavity in the Asn1029/Ala1030 &#8216;d/a&#8217; layer has a volume of 266&#8197;&#197;3 and no other cavity larger than 50&#8197;&#197;3 was present in the structure. In sharp contrast to HCoV-229E, HCoV-NL63 and MERS, the cavities seen in the three &#8216;da&#8217; layers of the SARS 3HR1 core are all large, with volumes of 105, 232 and 285&#8197;&#197;3, respectively.The presence of multiple large cavities (>100&#8197;&#197;3) in the middle of the 3HR1 core of SARS may suggest that the supercoils of SARS are less stable compared with the supercoils of HCoV-229E and HCoV-NL63. We thus calculated the average pitch values of different HCoVs using TWISTER (Strelkov & Burkhard, 2002 &#9656;). The pitch values of the HCoV-229E, HCoV-NL63 and MERS supercoils are 188.0, 189.3 and 187.2&#8197;&#197;, respectively, which are all similar to each other and close to the most commonly seen pitch of trimeric coiled-coil structures (Seo & Cohen, 1993 &#9656;). Nevertheless, the 194.7&#8197;&#197; pitch value of the SARS supercoil is longer than those of MERS, HCoV-229E and HCoV-NL63, indicating that the 3HR1 core of SARS is less twisted compared with the others.Side-by-side electrostatic surface comparisons also reveal substantial differences between &#945;-HCoVs and &#946;-HCoVs (Fig. 2 &#9656;\nc). The electrostatic surface of SARS and MERS are strikingly similar to each other, except for some differences in the lower region (boxed with black dashed lines in Fig. 2 &#9656;\nc). The electrostatic surfaces of HCoV-229E and HCoV-NL63 are also alike to some extent, but the 3HR1 core of HCoV-229E is more hydrophobic. In fact, the 3HR1 core of HCoV-229E seems to be the most hydrophobic among the four HCoVs. It is worth noting that at certain positions the electrostatic surface potentials are opposite in &#945;-HCoVs and &#946;-HCoVs (Fig. 2 &#9656;\nc; boxed with red dashed lines). Considering that both hydrophobic and electrostatic inter&#173;actions between HR1 and HR2 contribute to the affinity between them (Aydin et al., 2014 &#9656;), such differences in electrostatic surface potentials should be taken into consideration for the design and development of a pan-CoV inhibitory HR2 mimic.\nInteraction between HR1 and HR2 of HCoV-229E \u00a0 \nThe HR2 residues 1067&#8211;1097 of HCoV-229E fold into a nine-turn amphipathic &#945;-helix (Fig. 1 &#9656;\nb), which fits snugly onto the 3HR1 core mainly through extensive hydrophobic inter&#173;actions (Fig. 3 &#9656;\na). Specifically, residues at the &#8216;d&#8217; position (Leu1067, Leu1074, Leu1081, Leu1088 and Ile1095) in this nine-turn &#945;-helix bury their side chains (Fig. 3 &#9656;\na; shown as orange stick models) into hydrophobic pockets on the surface of the 3HR1 core of HCoV-229E. Another kind of hydrophobic packing was also observed at locations where the hydrophobic surface on the 3HR1 core is relatively flat (Fig. 3 &#9656;\na; hereafter termed ridges). At these locations, residues at the &#8216;a&#8217; (Ile1071, Ser1078, Val1085, Ile1092 and Leu1099) and &#8216;g&#8217; (Glu1070, Lys1077, Thr1084, Leu1091 and Thr1098) positions on the HR2 helix pack about 50% of the solvent-accessible surface area (SASA) of their side chains (Fig. 3 &#9656;\na; shown as yellow stick models) against the HR1 residues constituting these ridges. Besides the nine-turn &#945;-helix, the rest of HR2 adopts an extended conformation which also makes substantial hydrophobic inter&#173;actions with the 3HR1 core. Interestingly, only the first type of hydrophobic inter&#173;action was seen in these regions (Fig. 3 &#9656;\na), where the side chains of Pro1053, Leu1055, Val1057, Tyr1060, Leu1065, Val1100, Leu1102 and Trp1104 (orange stick models in Fig. 3 &#9656;\na) are completely buried in the cavities on the 3HR1 core. Overall, the side chains of hydrophobic residues from the HR2 helix of HCoV-229E fit tightly into the cavities and ridges on the 3HR1 core, almost like chains onto a sprocket (Fig. 3 &#9656;\nb). It is worth noting that the HR2 residues that either fill the hydrophobic cavities or mask the hydrophobic ridges on the 3HR1 core are largely conserved across the different HCoVs (Fig. 1 &#9656;\ng; denoted by stars and filled squares, respectively), corroborating the essential role played by hydrophobic inter&#173;actions in the inter&#173;action between HR1 and HR2.Besides hydrophobic packing, electrostatic and polar inter&#173;actions also contribute to the affinity and specificity between HR1 and HR2 (Fig. 4 &#9656;). In the helical region, residues on HR2 mainly engage residues on HR1 through side-chain to side-chain inter&#173;actions that include hydrogen bonds and salt bridges (Fig. 4 &#9656;\na). In the extended region of HR2, extensive polar inter&#173;actions were observed between the main-chain atoms on HR2 and the side chains of residues from HR1 (Figs. 4 &#9656;\nb and 4 &#9656;\nc). It is worth noting that some of these interactions are highly conserved across different HCoVs (Fig. 4 &#9656;; conserved inter&#173;actions are shown as yellow stick models, with black dashed lines indicating hydrogen bonds or salt bridges). Interestingly, these conserved inter&#173;actions are mainly located in the regions adjacent to the end of the nine-turn helix, suggesting that they might play a critical role in fixing the register of the HR2 helical region on the HR1 surface.\nDiscussion \u00a0 \nWe have determined the fusion-core structure of HCoV-229E at 1.86&#8197;&#197; resolution, thus further completing the structural profiles of HCoV fusion cores. The HCoV-229E structure reveals that its HR1 folds into an unusually long 24-turn helix, which is the longest observed so far in published HCoV fusion-core structures (Fig. 1 &#9656;). Currently, multiple fusion-core structures are available for SARS and MERS and each has a different HR1/HR2 coverage (or completeness) (Supekar et al., 2004 &#9656;; Lu et al., 2014 &#9656;; Xu et al., 2004 &#9656;; Duquerroy et al., 2005 &#9656;; Gao et al., 2013 &#9656;). Among them, the most complete post-fusion structures are for SARS (PDB entry 1wyy; Duquerroy et al., 2005 &#9656;) and MERS (PDB entry 4njl; Lu et al., 2014 &#9656;), with a21-turn and 22-turn HR1 helix, respectively (Figs. 1 &#9656;\nf and 2 &#9656;\na). Both of these HR1s are shorter than the 24-turn HR1 reported here, as the HR1 helices of &#945;-HCoVs are intrinsically longer than those of &#946;-HCoVs owing to a 14-residue insertion in their HR1 domains (Fig. 1 &#9656;\nf). The 14-turn and 13-turn HR1 helices seen in previous post-fusion structures of HCoV-NL63 and HCoV-229E cover only around 56% of their annotated HR1 sequences (Fig. 1 &#9656;\nf; coverage highlighted with a yellow background; amino acids 981&#8211;1031 versus 964&#8211;1054 for HCoV-NL63 and amino acids 788&#8211;838 versus 783&#8211;873 for HCoV-229E; Zheng et al., 2006 &#9656;; Zhang et al., 2018 &#9656;). In contrast, for the structure presented here the HR1 coverage is 97% (Fig. 1 &#9656;\nf; coverage highlighted with a cyan background; amino acids 785&#8211;872 versus 783&#8211;873), which represents the most complete post-fusion conformation among all published &#945;-HCoV fusion-core structures (Figs. 1 &#9656;\nf and 1 &#9656;\ng).Owing to the 14-residue insertion in the HR1 regions, the HR2 regions of &#945;-HCoVs are also intrinsically longer than those of &#946;-HCoVs. The HR2 helix of HCoV-229E consists of nine turns (Fig. 4 &#9656;), whereas in &#946;-HCoVs such as MERS and SARS the HR2 helix is usually only 4&#8211;5 turns (Fig. 4 &#9656;). As a functional outcome, the melting temperatures (T\nm) of &#945;-HCoVs are significantly and consistently higher than those of &#946;-HCoVs, presumably owing to a larger buried interface and a higher affinity between the longer HR2s and 3HR1 cores of &#945;-HCoVs. In detail, the T\nm values for the HCoV-229E and HCoV-NL63 post-fusion cores are 90.1 and 98.0&#176;C, respectively (Xia et al., 2018 &#9656;; Zheng et al., 2006 &#9656;), while the T\nm values for SARS and MERS are 85.0 and 87.0&#176;C, respectively (Xu et al., 2004 &#9656;; Lu et al., 2014 &#9656;).The fusion-core structure of HCoV-229E and those of SARS, MERS and HCoV-NL63 are similar to each other overall (Fig. 2 &#9656;\na), despite differences in the packing of their 3HR1 cores. Specifically, &#945;-HCoVs seem to have more non&#173;canonical &#8216;x&#8217; and &#8216;da&#8217; layers than &#946;-HCoVs, especially in the region of the 14-residue insertion (Figs. 2 &#9656;\na and 2 &#9656;\nb). Compared with the 3HR1 cores of MERS, HCoV-NL63 and HCoV-229E, the SARS supercoils are less twisted, as shown by the longer pitch and larger hydrophobic voids in the middle of the 3HR1 core. Moreover, side-by-side electrostatic surface comparisons of the 3HR1 cores also reveal substantial differences between &#945;-HCoVs and &#946;-HCoVs (Fig. 2 &#9656;\nc). Notably, at certain positions, the electrostatic surface potentials are opposite in &#945;-HCoVs and &#946;-HCoVs (Fig. 2 &#9656;\nc).Similar to other HCoVs, the high-affinity interaction between HR1 and HR2 of HCoV-229E is mediated by both extensive hydrophobic and hydrophilic interactions (Figs. 3 &#9656; and 4 &#9656;). Besides the highly conserved hydrophobic interactions, some polar and electrostatic interactions are also well preserved across different HCoVs (Fig. 4 &#9656;). Together, these conserved interactions and the differences in the electrostatic surface potential of 3HR1 cores across different HCoVs should be taken into consideration when designing pan-CoV inhibitors that mimic HR2 for targeting HR1. Although previous studies have found that peptides derived from the HR2s of SARS or MERS can prevent the host entry of the corresponding viruses, these peptides lack cross-inhibitory activity with each other or with other HCoVs (Liu et al., 2004 &#9656;; Lu et al., 2014 &#9656;). Since most of the hydrophobic interactions between HR1 and HR2 are highly conserved across all HCoVs, this implies that hydrophilic interactions are also important for the adhesion and register of HR2s onto the 3HR1 cores. Indeed, it has been reported that both hydrophobic and hydrophilic interactions between HR1 and HR2 of SARS contribute to the affinity between them (Aydin et al., 2014 &#9656;). Hence, a good pan-CoV inhibitory peptide mimic of HR2 would preserve all of the conserved hydrophobic and hydrophilic interactions between HR1 and HR2 of all HCoVs and also accommodate the variation among different HCoVs. More specifically, at the positions where the electrostatic surface potentials of 3HR1 cores are opposite in &#945;-HCoVs and &#946;-HCoVs, residues on the HR2 inhibitory peptide mimic should be carefully chosen such that they could form contacts, but not electrostatic or steric clashes, with surrounding residues on the 3HR1 cores of different HCoVs.The deadly global outbreaks of SARS in 2002 and MERS in 2012 highlight the importance of developing vaccines or treatments for CoVs, for which there are still no effective drugs or vaccines available. Both SARS and MERS are zoonotic CoVs. Before inter-species jumping, humans rarely came into contact with zoonotic CoVs, and therefore lack neutralizing antibodies to zoonotic CoVs. As a consequence, zoonotic CoVs usually pose substantial health and pandemic threats to humans. Given the high genomic mutation rate of CoVs and ever-increasing global population shifts, future outbreaks of CoV pandemics such as SARS or MERS are very likely to occur. To this end, structural and functional characterization of HCoVs takes on greater significance, especially for the structure-based design of future therapies or vaccines.\nSupplementary Material\n",
        "pmid": "30198895",
        "title": "Crystal structure of the post-fusion core of the Human coronavirus 229E spike protein at 1.86&#8197;&#197; resolution",
        "journal_title": "Acta Crystallographica. Section D, Structural Biology",
        "authors": "Lei Yan;Bing Meng;Jiangchao Xiang;Ian A. Wilson;Bei Yang",
        "doi": "10.1107/S2059798318008318"
    },
    {
        "keywords": "outbreak;coronavirus;long-term care facility;respiratory infections;Louisiana;USA;United States;NL63;viruses;lower respiratory tract infection",
        "abstract": "We describe an outbreak of severe respiratory illness associated with human coronavirus NL63 in a long-term care facility in Louisiana in November 2017. Six of 20 case-patients were hospitalized with pneumonia, and 3 of 20 died. Clinicians should consider human coronavirus NL63 for patients in similar settings with respiratory disease.",
        "body": "",
        "pmid": "30226169",
        "title": "Severe Respiratory Illness Outbreak Associated with Human Coronavirus NL63 in a Long-Term Care Facility",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Julie Hand;Erica Billig Rose;Andrea Salinas;Xiaoyan Lu;Senthilkumar K. Sakthivel;Eileen Schneider;John T. Watson",
        "doi": "10.3201/eid2410.180862"
    },
    {
        "keywords": "canine respiratory coronavirus;bovine coronavirus;human coronavirus OC43;attachment receptor;entry receptor;sialic acids;sialic acids;coronavirus;entry;HLA",
        "abstract": "Despite high similarity of canine respiratory coronavirus (CRCoV), bovine coronavirus, (BCoV) and human coronavirus OC43 (HCoV-OC43), these viruses differ in species specificity. For years it was believed that they share receptor specificity, utilizing sialic acids for cell surface attachment, internalization, and entry. Interestingly, careful literature analysis shows that viruses indeed bind to the cell surface via sialic acids, but there is no solid data that these moieties mediate virus entry. In our study, using a number of techniques, we showed that all three viruses are indeed able to bind to sialic acids to a different extent, but these molecules render the cells permissive only for the clinical strain of HCoV-OC43, while for others they serve only as attachment receptors. CRCoV and BCoV appear to employ human leukocyte antigen class I (HLA-1) as the entry receptor. Furthermore, we identified heparan sulfate as an alternative attachment factor, but this may be related to the cell culture adaptation, as in ex vivo conditions, it does not seem to play a significant role. Summarizing, we delineated early events during CRCoV, BCoV, and HCoV-OC43 entry and systematically studied the attachment and entry receptor utilized by these viruses.",
        "body": "1. Introduction\nCoronaviruses are enveloped, single-stranded, positive-sense RNA viruses that cluster within the family Coronaviridae (order Nidovirales). They infect a wide variety of species, including humans, livestock, and companion animals. In humans, they usually cause mild to moderate respiratory symptoms, although more severe cases occur mostly in individuals with impaired immune responses [1]. The emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in China in the winter of 2002, and of Middle East Respiratory Syndrome coronavirus (MERS-CoV) in September 2012, reminds us that coronaviruses pose a significant threat, and that zoonotic transmission is common [2,3,4,5,6].The Coronaviridae family is divided into two subfamilies, Letovirinae and Orthocoronavirinae, and the latter one is further divided into four genera&#8212;alpha, beta, gamma, and delta&#8212;based on the degree of nucleotide similarity. The current phylogeny of the family is presented in Figure 1.Betacoronaviruses are highly variable and use a number of different attachment and entry receptors. Generally, it is believed that viruses from this genus attach to the glycan layer on the cell surface, (e.g., sialic acids (SAs) or heparan sulfate (HS)) [9,10,11,12,13,14]. The second step during the infection is interaction with the entry receptor. Several molecules were proposed to serve as such, including carcinoembryonic antigen-cell adhesion molecule (CEACAM1) for murine hepatitis virus (MHV), angiotensin-converting enzyme 2 (ACE2) for SARS-CoV and dipeptidyl peptidase 4 (DPP4) for MERS-CoV. Human leukocyte antigen class I (HLA-I) was suggested to serve as entry receptor for HCoV-OC43 [15,16] and HCoV-HKU1 [17], but these discoveries were not confirmed in subsequent studies. Interestingly, SAs were also reported to serve as entry receptors, but some speculate that they play a negligible role in cell entry [15,16,17].HCoV-OC43 was first isolated in 1967. It is (next to HCoV-NL63) the most common coronavirus in humans. Incidence of infection is the highest in winter and early spring [18,19,20]. It is thought that the virus emerged in the human population at the end of the 19th century, but the most recent common ancestor has not been identified. Of interest, a number of betacoronaviruses (e.g., bovine coronavirus (BCoV), canine coronavirus (CRCoV), and dromedary camel coronaviruses) are closely related to this human pathogen; it was suggested that these species recombine and new subspecies emerge [19,21,22,23].Here, we focused on three closely related betacoronaviruses; they are almost identical from a genetic perspective but cause disease in different hosts. HCoV-OC43 causes a common cold in humans, BCoV causes gastrointestinal and respiratory tract disease in cattle [24,25,26], and CRCoV is linked to kennel cough in dogs. Surprisingly high resemblance of their spike proteins (97.3% nucleotide identity for CRCoV and BCoV and 96.9% for CRCoV and HCoV-OC43 [27]) not common among betacoronaviruses [23], as well as cross-reactivity of BCoV antigen with canine antibodies [27] and infectivity of BCoV to pups [22,28] led us to ask whether these three viruses share receptor specificity. We found that both SAs and HS serve as attachment factors for all three viruses, albeit to different extent. However, SAs are not essential for CRCoV and BCoV infection and serve as entry receptors only for clinical strain of HCoV-OC43, whereas CRCoV and BCoV employ HLA-I.\n2. Materials and Methods\n\n2.1. Cells\nHRT-18G (ATCC CRL-11663) cells, derivative of HRT-18 (ATCC CCL-244, ileocecal colorectal adenocarcinoma) were maintained in Dulbecco&#8217;s MEM (Life Technologies, Warsaw, Poland) supplemented with 3% heat-inactivated fetal bovine serum (Life Technologies, Warsaw, Poland), penicillin (100 U/mL), streptomycin (100 &#956;g/mL), and ciprofloxacin (5 &#956;g/mL). Cells were cultured at 37 &#176;C under 5% CO2.To generate human airway epithelium (HAE) cultures, primary cells isolated from patients&#8217; tissue were cultured on plastic in Bronchial Epithelial Growth Media (BEGM) medium for 1&#8211;2 weeks. Next, passage 2 cells were seeded on permeable Transwell insert supports (12 mm) and after the confluence was reached, apical medium was removed, and cells were grown for another 5&#8211;8 weeks on air&#8211;liquid interface (ALI) in ALI medium. After this time, cells were fully differentiated, forming pseudostratified mucociliary epithelium.\n2.2. Viral Stocks\nHCoV-OC43, CRCoV, and BCoV stocks were prepared by infecting HRT-18G cells. HCoV-OC43 (ATCC: VR-1558) was cultured in Dulbecco&#8217;s MEM (Life Technologies, Warsaw, Poland) supplemented with 2% heat-inactivated fetal bovine serum (Life Technologies, Warsaw, Poland), penicillin (100 U/mL) and streptomycin (100 &#956;g/mL) at 32 &#176;C under 5% CO2. BCoV Mebus strain (NR-445, BEI Resources) was cultured in Dulbecco&#8217;s MEM (Life Technologies, Warsaw, Poland) supplemented with penicillin (100 U/mL) streptomycin (100 &#956;g/mL) and trypsin (1 &#181;g/mL) at 37 &#176;C under 5% CO2. CRCoV strain 4182 was cultured in Dulbecco&#8217;s MEM (Life Technologies, Warsaw, Poland) supplemented with penicillin (100 U/mL) and streptomycin (100 &#956;g/mL) at 37 &#176;C under 5% CO2. Stocks were collected at 5th day post infection (p.i.) by two freeze-thaw cycles. Mock-infected cells were used as a control. Virus&#8217; yield was estimated by titration according to the Reed and Muench formula [29]. Obtained aliquots were stored at &#8722;80 &#176;C.HCoV-NL63 stock (isolate Amsterdam 1) was generated by infecting LLC-MK2 cells cultured in MEM with Hanks&#8217; and Earle&#8217;s salts (two parts Hanks&#8217; MEM and one part Earle&#8217;s MEM) supplemented with 3% heat-inactivated fetal bovine serum (Life Technologies, Warsaw, Poland), penicillin (100 U/mL) and streptomycin (100 &#956;g/mL) at 32 &#176;C under 5% CO2. Stocks were collected at day 5 p.i. by two freeze-thaw cycles. Mock-infected cells were used as a control.Influenza A virus Kilbourne F108: A/Aichi/2/1968 (HA, NA) x A/Puerto Rico/8/1934 (H3N2), Reassortant X-31 was generated by infecting MDCK cells cultured in Dulbecco&#8217;s MEM (Life Technologies, Warsaw, Poland) supplemented with penicillin (100 U/mL) streptomycin (100 &#956;g/mL) and trypsin (1 &#181;g/mL) at 37 &#176;C under 5% CO2. Stocks were collected at day 2 p.i. by two freeze-thaw cycles. Mock-infected cells were used as a control. HCoV-OC43 isolate 0500 was propagated on fully differentiated HAE cultures. Prior to the infection, the apical surfaces of HAE were washed thrice with PBS and inoculated with 100 &#956;L of 1000&#215; diluted viral stock [30]. After 3 h incubation at 32 &#176;C, the unbound viral particles were removed by 3 washes with 500 &#956;L of PBS for 5 min at 32 &#176;C. HAE cultures were further maintained at 32 &#176;C for 2&#8211;5 days. The new viral stock was obtained by washing of the apical surface with PBS.\n2.3. Purification of CRCoV\nVirus-containing medium was concentrated using Amicon Ultra, 10 kDa cut-off (Merck, Warsaw, Poland) and subsequently overlaid on 15% iodixanol solution in PBS (OptiPrep medium; Sigma-Aldrich, Poznan, Poland) and centrifuged at 45,000&#215; g for 3 h at 4 &#176;C. Following centrifugation virus-containing fraction was overlaid on 10&#8211;20% iodixanol gradient and centrifuged at 45,000&#215; g for 18 h at 4 &#176;C. Fractions collected from gradient were analyzed by Western blotting to detect the CRCoV nucleocapsid protein. The virus-enriched fractions were aliquoted and stored at &#8722;80 &#176;C.\n2.4. Antibodies to CRCoV N Protein\nCRCoV nucleocapsid gene was codon optimized, synthetized, and cloned into pETDuet vector via BamHI and AatII sites. Plasmid identity was confirmed by sequencing. N protein was expressed in E. coli Origami cells following induction with 0.5 mM isopropyl-&#946;-d-thiogalactopyranoside (IPTG) at 20 &#176;C for 16 h. Protein was purified on Ni-nitrilotriacetic acid resin (IMAC Sepharose 6 Fast Flow).Mouse polyclonal antibodies specific to CRCoV N protein were developed according to the standard protocol [31]. Six-week-old Balb/C mice purchased from The Mossakowski Medical Research Centre, Polish Academy of Sciences, Warsaw, Poland. Mice were injected intraperitoneally with 100 &#181;g of purified recombinant protein diluted in PBS and mixed 1:1 with Complete Freund&#8217;s Adjuvant (Sigma-Aldrich, Poznan, Poland). Subsequent animal immunizations were performed weekly with 50 &#181;g of antigen mixed with Incomplete Freund&#8217;s Adjuvant (Sigma-Aldrich, Poznan, Poland). As soon as the anti-N serum titer reached 1:1,000,000 in ELISA, animals were euthanized and blood was collected by cardiac puncture. Antibodies were purified from serum by affinity chromatography on Capture Select LC-kappa (mur) Affinity Matrix according to the manufacturer&#8217;s instructions (Thermo Fisher Scientific, Warsaw, Poland) and then dialyzed into sterile PBS.\n2.5. Western Blot Analysis\nCell lysates were mixed with sample buffer (0.5 M Tris, pH 6.8, 10% SDS, 50 mg/mL dithiothreitol [DTT]) and boiled for 5 min. Afterwards, they were separated by SDS-PAGE electrophoresis alongside PageRuler prestained protein size markers (Thermo Fisher Scientific, Warsaw, Poland) and electrotransferred onto activated PVDF membrane. The membranes were then blocked overnight (4 &#176;C) with 5% skim milk in Tris-buffered saline supplemented with 0.5% Tween 20. An anti-CRCoV N protein antibodies (1:200, 2 h) followed by horseradish peroxidase labeled rabbit antimouse secondary antibodies (1:20,000, 1 h, Dako, Swarzewo, Poland) were used for detection of virus. The signal was developed using Immobilon Western Chemiluminescent HRP Substrate (Millipore, Warsaw, Poland).\n2.6. Hemagglutination Assay\nHemagglutination assay was performed in V-shaped 96-well plates. Mouse erythrocyte suspension was prepared by three washes of twelve-week-old mouse blood with PBS. The obtained 3% mouse erythrocyte suspension was mixed with viral stocks and incubated at room temperature for 1 h until developed. As controls served influenza A H3N2 reported to hemagglutinate erythrocytes [32] and HCoV-NL63 which does not bind to SAs and therefore does not trigger such effect [33]. Pictures were acquired using CANON EOS 40D camera. All viral stocks (except for influenza A) were used at TCID50 = 500,000. Influenza A virus was applied at TCID50 = 30,000, which was the highest possible titer to be obtained.\n2.7. Virus Attachment\nTo determine which cell surface molecules are responsible for virus&#8217; attachment, competition experiments were designed, or SAs residues were enzymatically removed from the surface of cells. Cells were grown on coverslips for 48 h, washed twice with PBS and incubated for 30 min at 37 &#176;C with type II neuraminidase (NA, 100&#8211;200 mU/mL) from Vibrio cholerae (N6514, Sigma-Aldrich, Poznan, Poland), N-acetylneuraminic acid (Neu5Ac 40&#8211;80 mM, A0812 Sigma-Aldrich, Poznan, Poland), heparan sulfate (HS, 10&#8211;100 &#181;g/mL), d-(+)-galactose (Gal, 50 mM), d-(+)-glucose (Glu, 50 mM), d-(+)-mannose (Man, 50 mM), or N-acetyl-d-glucosamine (NAc, 50 mM) in PBS. Following incubation, cells were cooled, washed thrice with ice-cold PBS and overlaid with viral stocks. In case of Neu5Ac virus was additionally preincubated with the compound for 1 h at 4 &#176;C before adsorption. Sugar moieties, Neu5Ac and HS but not NA were present during adsorption. Cells were incubated for 2 h at 4 &#176;C, washed twice with ice-cold PBS and fixed with 4% formaldehyde. Analysis was carried out with flow cytometry or confocal microscopy.\n2.8. Confocal Microscopy\nFixed cells were permeabilized using 0.5% Tween-20 (RT, 10 min, Bioshop) and unspecific binding sites were blocked using 5% bovine serum albumin in PBS (4 &#176;C, overnight) prior to staining. For visualization of viruses, anticoronavirus antibody OC43 strain (1 &#181;g/mL, 2 h, RT, Merck, Warsaw, Poland) coupled with goat antimouse Alexa Fluor 488 antibody (5 &#181;g/mL, 1 h, RT, Thermo Fisher Scientific, Warsaw, Poland) were used. After incubation with antibodies, cells were washed thrice with 0.5% Tween-20 in PBS. Nuclear DNA was stained with 4&#8242;,6&#8242;-diamidino-2-phenylindole (DAPI, 0.1 &#956;g/mL, Sigma-Aldrich, Poznan, Poland). Stained coverslips were mounted on glass slides in Prolong Diamond medium (Thermo Fisher Scientific, Warsaw, Poland). Fluorescent images were acquired using Zeiss LSM 710 confocal microscope (Carl Zeiss Microscopy GmbH, Poznan, Poland).\n2.9. Flow Cytometry\nFor flow cytometry analysis, cells were permeabilized (0.5% Tween-20 in PBS, 20 min RT) and blocked (5% bovine serum albumin in PBS, 2 h RT) prior to staining. Viruses were visualized using anticoronavirus antibody OC43 strain (1 &#181;g/mL, 2 h, RT, Merck, Warsaw, Poland) coupled with goat antimouse Alexa Fluor 488 antibody (5 &#181;g/mL, 1 h, RT, Thermo Fisher Scientific, Warsaw, Poland). Stained cells were scratched off the glass in PBS prior to analysis. A minimum of 10,000 cells per sample were analyzed. Cells were gated as shown in Figure S1.\n2.10. Virus Replication\nTo determine whether certain proteins or sugar moieties may serve as entry receptors, cell monolayers were washed with PBS and incubated with one of the following compounds: NA (200 mU/mL), HS (100&#8211;600 &#181;g/mL), Neu5Ac (5&#8211;80 mM), Gal (100 mM), Glu (100 mM), Man (100 mM), NAc (100 mM), polyclonal rabbit anti-HLA A/B/C antibodies (0.25&#8211;4 &#181;g/mL, sc-30204, Santa Cruz) for 60 min at 37 &#176;C. After three washes (with exception to HS treated cells), cells were infected with the viruses at TCID50 of 400 per milliliter (M.O.I. = 0.0007) in the presence of tested agent or control media. NA treatment was repeated every 24 h until the day of sample collection. After 2 h, unbound virions were washed off and the cells were further cultured in the presence of compounds at conditions appropriate for each virus (see Viral stocks section for details). Antibody specificity was confirmed with isotype control antibodies used at the same concentrations.\n2.11. Reverse Transcription Quantitative PCR (RT-qPCR)\nAt day 5 p.i., supernatants were collected and total RNA was isolated using Viral DNA/RNA Kit (A&A Biotechnology, Gdynia, Poland). This time point was selected based on replication curves for all three viruses (Figure S2). Reverse transcription was carried out with High Capacity cDNA Reverse Transcription Kit (Thermo Fisher Scientific, Warsaw, Poland), according to the manufacturer&#8217;s protocol. Serial dilutions of quantified DNA fragment corresponding to the viral N gene cloned with InsTAclone PCR cloning kit (Thermo Fisher Scientific, Warsaw, Poland) into pTZ57R/T plasmid were used as standards. Concentration of the linearized form of the plasmid was assessed using a spectrophotometer and its purity was confirmed by gel electrophoresis.Subsequently, PCR was performed with using KAPA PROBE FAST qPCR Master Mix (Kapa Biosystem). Rox was used as a reference dye. The amplification program, specific probes and primers for each virus are listed in Table 1.\n2.12. Resialylation\nTo test whether recovery of the SAs after neuraminidase-mediated desialylation affects the virus attachment, appropriate experiments were designed. Cells grown on coverslips were washed twice with PBS, incubated for 30 min at 37 &#176;C with 500 mU/mL NA. Subsequently cell cultures were washed with PBS and treated with 1 mM cytidine-5&#8242;-monophospho-N-acetylneuraminic acid (CMP-Neu5Ac, C8271 Sigma-Aldrich, Poznan, Poland) and either &#945;-2,3-sialyltransferase (&#945;2,3-ST) from Pasteurella multocida (S1951, Sigma-Aldrich, Poznan, Poland) or &#945;-2,6-sialyltransferase (&#945;2,6-ST) from Photobacterium damsel (S2076, Sigma-Aldrich, Poznan, Poland) at varying concentrations for 2 h at 37 &#176;C. Following treatment, cells were washed thrice with ice-cold PBS and infected with iodixanol concentrated CRCoV, BCoV orHCoV-OC43. After 2 h at 4 &#176;C, unbound virions were washed off with PBS and cells were fixed with 4% formaldehyde. Activity of neuraminidase and sialyltransferases was verified with &#945;-2,6-SAs specific fluorescein labeled Maackia Amurensis lectin I (Vector Labs) and &#945;-2,3-SAs specific Cy3 labeled Sambucus Nigra lectin (Vector Labs)\n2.13. Data Analysis\nAll experiments were performed at least three times in triplicate. All graphs presented in this work were created using GraphPad Prism 6 software. Distribution of values was tested using Shapiro&#8211;Wilk normality test and equality of group variances was examined with Browne&#8211;Forsythe test. Depending on the results of these tests, one-way ANOVA with Dunnett&#8217;s multiple comparisons test or Kruskal&#8211;Wallis test with Dunn&#8217;s multiple comparisons test or unpaired t test with Welch&#8217;s correction were used. p values < 0.05 were considered significant. One asterisk (*) identifies adjusted p values between 0.01 and 0.05, two asterisks (**) identify adjusted p values between 0.01 and 0.001, three asterisks (***) identify adjusted p values between 0.001 and 0.0001, four asterisks (****) identify adjusted p values below 0.0001. Images obtained from the confocal microscope were deconvolved using AutoQuant X3 software and processed in ImageJ Fiji [34]. Data obtained from FACS analysis were collected using BD Cell Quest Pro software and analyzed using FlowJo V10.\n2.14. Ethics Statement\nThe HRT-18G cell line and canine respiratory coronavirus (CRCoV) strain 4182 were provided by Judy A. Mitchell. BCoV, Mebus, NR-445 was obtained through the NIH Biodefense and Emerging Infections Research Resources Repository, NIAID. Influenza virus Kilbourne F108: A/Aichi/2/1968 (HA, NA) x A/Puerto Rico/8/1934 (H3N2), Reassortant X-31 (Derived from Mouse-adapted X-31b), NR-3483 was obtained through the NIH Biodefense and Emerging Infections Research Resources Repository, NIAID. Human coronavirus OC43 (HCoV-OC43) VR-1558 and MDCK cell line (ATCC CCL-34) were obtained from American Type Culture Collection (ATCC). LLC-MK2 cells were kindly provided by Lia van der Hoek. HCoV-NL63 (isolate Amsterdam 1) was obtained as described [35]. The authors are grateful to Volker Thiel for providing HCoV-OC43 isolate 0500.HAE cells were obtained from airway specimens resected from patients of Silesian Center for Heart Diseases. Following written informed consent of all subjects to participate in this study, the procedure of acquisition was carried out according to the protocol approved by the Bioethical Committee of the Medical University of Silesia in Katowice, Poland (approval no: KNW/0022/KB1/17/10 dated on 16 February 2010).All animal procedures were in agreement with the guidelines of the Institutional Animal Care and Use Committee (IACUC) and the whole study was approved by 2nd Local IACUC in Krak&#243;w (Institute of Pharmacology Polish Academy of Sciences). Further, all animal procedures complied with the Act on the Protection of Animals used for scientific or educational purposes dated on 15th of January 2015 (D20150266L), which implements the Directive of the European Parliament and the Council (2010/63/EU) dated on 22nd of September 2010.\n3. Results\n\n3.1. HCoV-OC43, BCoV, and CRCoV Use SAs to Attach to Target Cells\nA number of betacoronaviruses, including HCoV-OC43 and BCoV, are thought to use SAs as entry receptors [9,10,11]. Our previous observations suggest that the role(s) of these molecules may be different from that reported in the literature; therefore, we performed a series of experiments to examine receptor usage. First, we performed a hemagglutination assay to verify whether these two viruses, and closely related CRCoV interact with these moieties. As shown in Figure 2, all three viruses agglutinated mouse erythrocytes, which are rich in SAs. Next, to determine the importance of SAs for CRCoV, BCoV, and HCoV-OC43 VR-1558 attachment, we treated cells with neuraminidase (NA) (to remove SA residues) prior to infection and then examined viral attachment. Removing SAs reduced attachment of all three viruses, with the strongest effect observed for CRCoV and HCoV-OC43 VR-1558 (59.4 &#177; 14.0% and 48.9 &#177; 6.5 decrease, respectively; Figure 3).\n3.2. SAs Did Not Facilitate Entry of CRCoV, BCoV, and HCoV-OC43 VR-1558 to HRT-18G Cells\nTo determine the role of SAs during CRCoV, BCoV, and HCoV-OC43 entry, we treated cells with neuraminidase (NA) prior to infection and then examined viral replication. Removal of SAs with NA had no effect on viral replication (Figure 4).To ensure that obtained results are valid and that reconstitution of the SAs on cell surface or incomplete scission of the SAs by NA does not affect the outcome, the infection was carried out in the presence of soluble SAs (Neu5Ac) prior to infection. The role of SAs during virus attachment and lack of effect on replication was confirmed for all three viruses (Figure 5).\n3.3. Restoration of SAs on the Cell Surface Rescues Attachment of CRCoV, BCoV, and HCoV-OC43 VR-1558\nTo confirm that the observed effect was SA-specific, we stripped and then restored SAs to the cell surface. First, HRT-18G cells were treated with NA to remove SAs. Then, the cells were incubated for 2 h at 37 &#176;C with &#945;-2,3-sialyltransferase (&#945;2,3-ST) or &#945;-2,6-sialyltransferase (&#945;2,6-ST) in the presence of 1 mM of cytidine-5&#8242;-monophospho-N-acetylneuraminic acid sodium salt (CMP-Neu5Ac). Compounds&#8217; activity was verified using &#945;-2,3-SAs and &#945;-2,6-SAs specific lectins (Figure 6).We then examined virus attachment to control cells, to cells stripped of SA, and to cells on which the SA shield was restored. Restoration of the SAs rescued cell surface binding by all viruses (Figure 7, Figure 8 and Figure 9). CRCoV showed a clear preference for &#945;2,3-SA moieties (Figure 7), while HCoV-OC43 bound preferentially to &#945;2,6-linked moieties (Figure 8). Similar to HCoV-OC43 VR-1558, BCoV bound preferentially to &#945;2,6-SA, but to much lower extent (Figure 9).These results confirm that all three viruses require SAs for attachment: CRCoV shows a preference for &#945;2,3-SA, BCoV for &#945;2,6-SA, and HCoV-OC43 VR-1558 for &#945;2,6-SA as previously reported for MDCK I cell line [37].\n3.4. Interaction with HLA-I Molecules\nAs we discovered that SAs do not serve as entry receptors, we examined some other candidates. One protein reported to facilitate entry of HCoV-OC43 and HCoV-HKU1 is the HLA molecule. Therefore, we blocked the interaction between the virus and HLA-I molecules using polyclonal antibodies. This had no effect on replication of HCoV-OC43 VR-1558. However, it did block infection by CRCoV and BCoV, suggesting that HLA-I serves as an entry receptor for these viruses (Figure 10).\n3.5. Involvement of HS and Lectins\nEven though removal of SAs from the cell surface resulted in potent inhibition of viral attachment to the cell surface, the effect was not complete. Therefore, we examined the role(s) of HS and other sugar moieties (d-(+)-galactose, d-(+)-glucose, d-(+)-mannose, or N-acetyl-d-glucosamine) on virus attachment and entry. Only HS had a marked effect on CRCoV, BCoV, and HCoV-OC43. Inhibition was most pronounced for BCoV (49.8 &#177; 20.1%; 30 &#181;g/mL HS) and HCoV-OC43 (44.4 &#177; 16.0%; 30 &#181;g/mL HS), and to the lesser extent for CRCoV (25.7 &#177; 16.5%; 100 &#181;g/mL HS) (Figure 11A). There was no significant effect on replication of any of the viruses (Figure 11B). Among the sugars tested, NAc alone showed a slight inhibitory effect on HCoV-OC43 VR-1558 attachment but had no effect on that of CRCoV or BCoV (data not shown [38]).\n3.6. Attachment Receptors and Adaptation during Cell Culture\nTo test the possibility that the observed specificity of HCoV-OC43 resulted from adaptation during cell culture, we examined differences in cell attachment and entry for laboratory and clinical strains of HCoV-OC43. It is worth remembering that the laboratory strain has lost the ability to infect human airway epithelium [39], while the clinical strain does not replicate in cell lines. We performed a series of experiments in which HCoV-OC43 0500 (the clinical strain) was used to infect HAE cultures. Neither HS competition, nor specific blockade of HLA-I or sugar binding moieties affected viral replication, suggesting that these molecules do not play a role in cell entry by HCoV-OC43 0500. However, supplementation with soluble SAs resulted in dose-dependent reduction in replication, implying SAs role in entry process also for the clinical strain (Figure 12A).Furthermore we found that only SAs but not HS had effect on HCoV-OC43 0500 adhesion to HAE cultures (Figure 12B,C).\n4. Discussion\nTo date, studies on betacoronavirus entry show high variability in terms of receptors used for cell attachment and entry [9,10,11,12,13,15,16,17]. To make an example, SARS-CoV uses angiotensin-converting enzyme 2 (ACE2), a type I integral transmembrane protein as entry receptor [40,41], but it may also utilize c-type lectin receptor expressed by dendritic cells (DC-SIGN), a DC-SIGN-related molecule (L-SIGN), and vimentin [42,43,44,45,46,47]. On the other hand, MERS-CoV, which also infects human respiratory tract epithelium, employs dipeptidyl peptidase 4 to enter the cell [48]. Other, nonhuman betacoronaviruses exploit diverse molecules, such as CEACAM1 (MHV), neural cell adhesion molecule (porcine hemagglutinating encephalomyelitis coronavirus) [49], or SAs (HCoV-OC43) [37].HCoV-OC43 is believed to use SAs as attachment and entry receptors, but it was also reported to employ HLA-I to enter the cell [15,16]. The BCoV has been extensively studied in the context of SAs usage. Models based on chicken, mouse, and rat erythrocytes (hemagglutination) and MDCK I, CaCo-2, and LLC-PK1 cell lines (entry and replication) confirmed the role of SAs in attachment of BCoV, but no information on the role of these moieties for the cell entry has been available [9,10,11,12,37,50]. CRCoV receptors remain unknown.Here, we checked whether HCoV-OC43, CRCoV, and BCoV interact with SAs. First, the hemagglutination assay yielded positive results in all the cases; we made an effort to establish the exact role of these moieties during virus&#8211;cell interaction. In all three cases, depletion of SAs from target cells as well as blockade of virus and SAs interaction using soluble neuraminic acid had a strong effect on virus binding, proving that these moieties serve as attachment receptors. Reconstitution of the SAs on cell surface after enzymatic removal revealed that CRCoV binds preferentially to &#945;-2,3-linked SAs but not to &#945;-2,6-linked SAs. By contrast, HCoV-OC43 VR-1558 used &#945;-2,6-SAs rather than &#945;-2,3-SAs, consistently with the published literature [37]. Although the preference of BCoV was similar to that of HCoV-OC43, it was less pronounced, suggesting a possible role of other molecules in the virus attachment to HRT-18G cells.Apart from N-acetylneuraminic acid (Neu5Ac), N-glycolylneuraminic acid (Neu5Gc) is also present on most mammalian cells surface [27,51,52,53]. Neu5Gc is a derivative of Neu5Ac as a consequence of the activity of citidine monophosphate N-acetylneuraminic acid hydroxylase (CMAH). Due to deletion in CMAH gene, human cells are deprived of Neu5Gc [53]. Although it is commonly believed that Neu5Gc prevails in nonhuman animal cells, currently available literature shows that dog ocular mucins and erythrocytes of most western breeds are rich in Neu5Ac rather than Neu5Gc [52,54,55]. Besides it is also known that bovines express both Neu5Ac and Neu5Gc in their tissues [51,53].As &#945;-2,6-SAs dominate in human respiratory tract [56,57], the HCoV-OC43 preference for them is not surprising. It is worth stressing that &#945;-2,3-SAs are abundant on ciliated epithelial cells [56,57], which is in line with CRCoV role in CIRD development, as disruption of ciliary clearance is one of the early hallmarks of the disease [27]. Of interest, Bakkers et al. [58] recently reported loss of hemagglutinin-esterase (HE) mediated SAs binding for HCoV-OC43 and HCoV-HKU1 during adaptation to human tissue, which suggests that the observed attachment is mediated by the spike protein.The role of SAs in adhesion of these three betacoronaviruses is unquestionable, but we made an effort to check whether other moieties may also serve as attachment receptors for the viruses, as blockade of virus&#8211;SAs interaction in some cases inhibited virus attachment pronouncedly, but incompletely. Of the tested ones, soluble HS hampered attachment of all three betacoronaviruses but did not severely affect their replication. The most pronounced effect was noted for HCoV-OC43 and the weakest for CRCoV. Affinity for HS has been previously described to emerge during adaptation to cell culture [59,60]; therefore, one may assume that the observed effect would not be valid for a wild-type virus. As we had a unique opportunity to test this hypothesis on HCoV-OC43 0500 virus (which does not replicate in vitro), we used fully differentiated HAE cultures and a standard cell line to show that attachment of the clinical strain does not depend markedly on HS, and that the ability to bind to HS most likely is related to the cell culture adaptation [61]. HRT-18G cells constitute the only existing model for the CRCoV infection in vitro and we decided to study CRCoV and BCoV in vitro on these cells [62]. Consequently, one cannot rule out the possibility that attachment receptor specificity may also be altered due to the cell culture adaptation. While SAs appeared to be important for virus binding to the cell surface, they are not essential for entry of HCoV-OC43 VR-1558, CRCoV, and BCoV. Importantly, the clinical strain of HCoV-OC43 0500 appear to employ this molecule as an adhesion and entry receptor, as soluble SAs block the attachment and entry of the virus to the human airway epithelium. Depletion of SAs with neuraminidase only mildly affected virus attachment and entry, but we believe that this is due to incomplete removal of SAs, the ability of cells to reconstitute SAs on the surface and the experimental settings.Because the results showed that SAs do not serve as entry receptors for HCoV-OC43 VR-1558, CRCoV, or BCoV, we aimed to identify the molecules that facilitate cell entry by these viruses. Studies conducted in the mid-1990s showed that HLA served as an HCoV-OC43 entry receptor on RD and HCN-lA cells [15,16,63]; more recent studies show that it also acts as an entry receptor for HCoV-HKU1 [17]. These findings, however, have not been confirmed by other research groups until now [64]. Here, we examined the role of HLA-I as a receptor for all three betacoronaviruses and found out that it does not play a role in cell entry by the laboratory and clinical strains of HCoV-OC43. However, replication of BCoV and CRCoV was hampered in the presence of HLA-specific antibodies, showing that these viruses employ HLA-I as entry receptors. HLA class I molecules structurally consist of three &#945; chains linked with a &#946;2-microglobulin molecule. The membrane proximal end is formed by the &#945;3 and &#946;2M domains, which adopt a standard immunoglobulin-like fold [65]. Consequently, they belong to the immunoglobulin superfamily. Interestingly, this superfamily also includes CAECAM1, which serves as a receptor for MHV [16,17]. It will not be too farfetched a presumption to assume that cousins of MHV&#8212;CRCoV and BCoV&#8212;use structurally similar molecules as their entry receptors.Despite similarities between betacoronaviruses on the level of protein sequence and the range of surface molecules recognized by them, the prevailing opinion is that they form separate genera characterized by very narrow host ranges [59]. However, a recent report regarding HCoV-OC43 outbreak among wild chimpanzees suggests that betacoronaviruses may undergo recurrent interspecies transmissions [66]. Considering previous reports, one may conclude that these three betacoronaviruses frequently cross species boundaries, leading to recombination and emergence of intermediate forms [27,67,68,69]. As all three viruses can replicate in human cells, this scenario is realistic; however, epidemiological studies are needed to confirm the hypothesis. One may also speculate that separation of BCoV and OC43 has not really occurred, and that some lineages belonging to a single species are able to infect a number of different hosts. On the other hand, it is also possible that HLA-I specificity is related to the cell culture adaptation.To summarize, we show here that three closely related betacoronaviruses (HCoV-OC43 VR-1558, BCoV, and CRCoV) utilize SAs and HS to attach to the cell surface of target cells. However, they show different preferences for these moieties, and the importance of these interactions is debatable. HLA-I molecules serve as entry receptors for CRCoV and BCoV, but not for HCoV-OC43 0500 for which SAs carry both roles.",
        "pmid": "30959796",
        "title": "Canine Respiratory Coronavirus, Bovine Coronavirus, and Human Coronavirus OC43: Receptors and Attachment Factors",
        "journal_title": "Viruses",
        "authors": "Artur Szczepanski;Katarzyna Owczarek;Monika Bzowska;Katarzyna Gula;Inga Drebot;Marek Ochman;Beata Maksym;Zenon Rajfur;Judy A Mitchell;Krzysztof Pyrc",
        "doi": "10.3390/v11040328"
    },
    {
        "keywords": "swine;diarrhea;coronavirus;Coronaviridae;bat-HKU2;porcine enteric alphacoronavirus;PEAV;viruses;zoonoses;China",
        "abstract": "We identified from suckling piglets with diarrhea in China a new bat-HKU2&#8211;like porcine coronavirus (porcine enteric alphacoronavirus). The GDS04 strain of this coronavirus shares high aa identities (>90%) with the reported bat-HKU2 strains in Coronaviridae-wide conserved domains, suggesting that the GDS04 strain belongs to the same species as HKU2.",
        "body": "",
        "pmid": "28654418",
        "title": "A New Bat-HKU2&#8211;like Coronavirus in Swine, China, 2017",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Lang Gong;Jie Li;Qingfeng Zhou;Zhichao Xu;Li Chen;Yun Zhang;Chunyi Xue;Zhifen Wen;Yongchang Cao",
        "doi": "10.3201/eid2309.170915"
    },
    {
        "keywords": "",
        "abstract": "Background\nA new respiratory infectious epidemic, severe acute respiratory syndrome (SARS), broke out and spread throughout the world. By now the putative pathogen of SARS has been identified as a new coronavirus, a single positive-strand RNA virus. RNA viruses commonly have a high rate of genetic mutation. It is therefore important to know the mutation rate of the SARS coronavirus as it spreads through the population. Moreover, finding a date for the last common ancestor of SARS coronavirus strains would be useful for understanding the circumstances surrounding the emergence of the SARS pandemic and the rate at which SARS coronavirus diverge.Methods\nWe propose a mathematical model to estimate the evolution rate of the SARS coronavirus genome and the time of the last common ancestor of the sequenced SARS strains. Under some common assumptions and justifiable simplifications, a few simple equations incorporating the evolution rate (K) and time of the last common ancestor of the strains (T0) can be deduced. We then implemented the least square method to estimate K and T0 from the dataset of sequences and corresponding times. Monte Carlo stimulation was employed to discuss the results.Results\nBased on 6 strains with accurate dates of host death, we estimated the time of the last common ancestor to be about August or September 2002, and the evolution rate to be about 0.16 base/day, that is, the SARS coronavirus would on average change a base every seven days. We validated our method by dividing the strains into two groups, which coincided with the results from comparative genomics.Conclusion\nThe applied method is simple to implement and avoid the difficulty and subjectivity of choosing the root of phylogenetic tree. Based on 6 strains with accurate date of host death, we estimated a time of the last common ancestor, which is coincident with epidemic investigations, and an evolution rate in the same range as that reported for the HIV-1 virus.",
        "body": "Background\nA new respiratory infectious epidemic, severe acute respiratory syndrome (SARS), broke out and spread throughout the world, affecting over 8,000 individuals in 32 countries[1,2]. In response to this outbreak, a global network of international collaborating laboratories was immediately sponsored and established by World Health Organization (WHO) to facilitate the identification of the causative agent of SARS. By now the putative pathogen of SARS has been identified, by experimental proof and by Koch's postulates, as a new coronavirus, a single positive-strand RNA virus [3-5]]. The whole genome of SARS coronavirus was first sequenced by the British Columbia Centre for Disease Control (CDC) in Canada on 23, April 2003 [6], and subsequently a total of 16 SARS coronavirus strains isolated from Hanoi, mainland China, Hong Kong, Singapore, and Taiwan were sequenced within short time[7,8]. Phylogenetic analysis and comparative genomic studies based on these genomic sequences indicate that the SARS coronavirus is distinct from any of the previously characterized coronaviruses. Epidemiological investigations further indicate the SARS coronavirus strains may be divided into two different genotypes[9].RNA viruses commonly have a high rate of genetic mutation, by which the viruses escape from host defence and evolve into novel viral strains. It is therefore important to know the mutation rate of the SARS coronavirus as it spreads through the population. Moreover, finding a date for the last common ancestor of SARS coronavirus strains would be useful for understanding the circumstances surrounding the emergence of the SARS pandemic and the rate at which SARS coronavirus diverge.Many attempts have been made to extrapolate the age of the common ancestor of sequenced genomes. Most of them are based on accurate phylogenetic tree reconstructions, which demand a large amount computation, because of their application of the maximum likelihood strategy. Common for these methods is that it is critical to choose a sequence as the root of the phylogenetic tree. Korber et al. [10] implemented a parsimonious strategy, which used the consensus sequence including the most common bases appearing in strains as the ancestral sequence.\nMethods\nAmong the 16 full-length SARS coronavirus genomes, we selected 6 strains for which the accurate date of host death is known, and on which our modelling was based. Our model performed calculation under two hypotheses, which are commonly adopted and have lead to accurate prediction in the study of HIV-1 virus [10]: first, nucleotide variation of these strains occurred by independent mutations at random positions in a single ancestral sequence; second, there exist a molecular clock and a constant rate of evolution. In addition, we simplified the calculation by neglecting trivial non-linear effects of multi-mutation for a base, i.e. there has only been one mutation for a base at a specific position of all the sequences during SARS infection time. This simplification can be justified by further discussion (see Additional file: 1).For an ancestor sequence S0 of a strain S, we can deduce for the assumptions above thatE(D(S0, S)) &#8776; K(T - T0),where D(S0, S) is difference of the two sequences (as depicted by Hamming distance), T0 is the date of the last ancestor, T is the date of host death (as an estimate of sampling date), and K is the evolution rate constant. The formula gives the expectation of sequence differences in proportion to the time of evolution.If S0 is the last common ancestor of S and S', then we have E(D(S,S')) = E(D(S0,S)) + E(D(S0,S')) (Fig. 1(a)). The equation takes this form under the simplification that along the total of the infection paths of the two sequences, mutation at any specific point of the sequences could, at most, only occur once. Thus E(D(S,S')) &#8776; K(T + T' - 2T0).The last common ancestor S0 of all the sequences is the root of the hidden phylogenetic tree with the strains as nodes. From the time T0, the sequences should at least evolve along two different routes. Therefore, there should be a partition of the strains into B and B' such that every pair of strains S &#8712; B and S' &#8712; B' should share the root of the tree as their last common ancestor (Fig. 1(b)), i.e., for each pair , E(D(S,S')) should be linear to (T + T') with same parameter T0. Therefore, we can implement the least square method to estimate K and T0 from the dataset.Since the real partition cannot be known in advance, we carried out calculations for all of the possible partitions of these 6 strains. For each division we use the estimated K to calculate the possible T0(S,S') of each sequence pair. The division with the minimum variance of T0(S,S') is taken as our best solution to the problem, and the corresponding K as an estimation of the mutation rate.To analyze how the parameters affect the results and support our fitting method, the Monte Carlo method was employed. At first, we produced a phylogenetic tree (See Fig. 3(c)) and a table of parameters (See Table 3) including the evolution rate and the times of the sequences. From the time of the last common ancestor S0 of the other sequences, every base of a given sequence has the possibility to mutate over time according to the given evolution rate. So the other sequences, included intermediate sequences (I) and final sequences (F), can be obtained in steps in the stimulation according to the given phylogenetic tree and the time parameters. After the sequences were obtained, we used our fitting method to get the evolution rate, without including the hidden parameters. By analysis of the estimated K from the data, we can get to know how the parameters affect our fitting results and the quality of our method.\nResults\nOf the 16 SARS coronavirus strains submitted to Genbank before June, 2003, 6 had accurate date of host death recorded. We chose these 6 to estimate the last common ancestor and the mutation rate of the SARS coronavirus (Table 1). We performed the calculation, and the fitting result of the best division (See Table 2) is shown in Fig. 2, including the differences between sequences D(S,S') versus the time factor (T + T'). The evolution rate K was estimated to be 0.16 base/day, which is similar to the reported evolution rate of HIV-1 virus [10]. The date of the last common ancestor T0 was found to be about August or September, 2002, which is also in accordance with the epidemic investigations finding that the first verifiable SARS case was reported as early as on November 11, 2002.We validated our estimation of the evolutionary rate by grouping strains according to the estimated date of their pair wise last common ancestor. Applying the estimated K = 0.16, we can determine a date T0*(i, j) of the last common ancestor for each pair  0, while every two members in G2 have corresponding T0*(i, j) < 0. This would imply that the strains in G1 have a more recent last common ancestor than those in G2. This partition of strains was supported by Ruan et al [9].\nDiscussion\nAnalysis by Monte Carlo Method was employed to test our fitting method and explain why the error of the evolution rate and time of last common ancestor was so large in our prediction. In a simulation of the simplified evolution model, sequences were generated according to a given phylogenetic tree, with parameters including evolution rate and times for each sequence. Two sets of parameters were used for a common phylogenetic tree, the evolution rate kept constant while time parameters differed (See Fig. 3(c) and Table 3). In model 1 there is a narrow time distribution two month of final sequences, while model 2 had a wider time distribution of five months.Hundreds of iterations of sequence data from the stimulation were given according to the parameters. For each result, we could get estimated parameters by our fitting method. The estimated K distribution of the results (shown in Fig 3) is in support of our fitting method, as in both models the estimates for the evolutionary rate converged on the set parameter (0.2). Model 2 with wide time distribution had a narrower distribution of K, which indicates the fitted parameter has a smaller error. The difference between the two models hints a narrow sampling time window as a partially explanation of the large error on the estimated K for the real data.Ideally, an estimation of evolution rate and the date of last ancestor for the SARS coronavirus should be based on sampling dates, with possible adjustments for culturing time and conditions. As such data were neither included in the submissions to Genbank, nor obtainable by direct contacts to the sequencing labs, we were left to choose between less ideal age estimates for the strains, such as date of host death, sequencing date, or submission date to Genbank. Sequencing dates were no more available than sampling dates, and for some groups several sequences were submitted to Genbank on the same date. In addition large part of the GenBank sequence were submitted long after June 2003, when no or very few SARS patient were available for sampling, also rendering submission date a not very accurate estimate for strain time. This basically left us with little other choice than to accept the date of host death as the most accurate available estimate for the age of each strain. Assuming that in most cases samples were taken a few days before to just after the death of the host, we think these dates represent acceptable, though not ideal, estimates of the endpoint of strain time.In summary, certain inherent features of the situation around the SARS epidemic prevented our method from rendering more accurate estimates. First, as national and international efforts fortunately succeeded in stemming the spread of fledgling epidemic by summer 2003, all the samples used to obtain the 16 sequences were collected within a relatively short period of time (two months), which makes the error of D(S0, Si) is relative large. Second, because the date of host death is not good reflection of real time of sequences, the error of time is quite large. Third, as useful time data for the submitted sequences were scarce, the subset of sequences available for modelling was too small. Finally, as data on pre-sequencing culturing times and conditions have not been made available, differences in evolutions rates between in vivo and in vitro conditions cannot be estimated, and the basic assumption, that only a constant evolution rate may not be completely valid. A more accurate model considering two evolution rate parameters may produce a more accurate estimation, particularly on a larger dataset with accurate sampling and sequencing times.\nConclusions\nWe have proposed a mathematical model to estimate the evolution rate of the SARS coronavirus genome as well as the time of the last common ancestor of the various SARS coronavirus strains. The method is simple to implement and avoids the difficulty and subjectivity of choosing the root of phylogenetic tree. Based on 6 strains with accurate dates of host death, we estimated a time of the last common ancestor, which is coincident with epidemic investigations, and an evolution rate in the same range as that reported for the HIV-1 virus.\nCompeting interests\nNone declared.\nAuthors' contributions\nLu and Bu built the model including proposing the assumptions, deriving the system of equations, programmed and analyzed the data. Zhao, Wang, Li, Zhu, Sun, Cai collected data and analyzed them. Zhang, Xu programmed and prepared for the paper, Chen, Bu, Ling led the group to complete work related to the paper.\nPre-publication history\nThe pre-publication history for this paper can be accessed here:\nSupplementary Material\n",
        "pmid": "15028113",
        "title": "Date of origin of the SARS coronavirus strains",
        "authors": "Hongchao Lu;Yi Zhao;Jingfen Zhang;Yuelan Wang;Wei Li;Xiaopeng Zhu;Shiwei Sun;Jingyi Xu;Lunjiang Ling;Lun Cai;Dongbo Bu;Runsheng Chen",
        "doi": "10.1186/1471-2334-4-3"
    },
    {
        "keywords": "",
        "abstract": "Background\nDespite the passing of more than a year since the first outbreak of Severe Acute Respiratory Syndrome (SARS), efficient counter-measures are still few and many believe that reappearance of SARS, or a similar disease caused by a coronavirus, is not unlikely. For other virus families like the picornaviruses it is known that pathology is related to proteolytic cleavage of host proteins by viral proteinases. Furthermore, several studies indicate that virus proliferation can be arrested using specific proteinase inhibitors supporting the belief that proteinases are indeed important during infection. Prompted by this, we set out to analyse and predict cleavage by the coronavirus main proteinase using computational methods.Results\nWe retrieved sequence data on seven fully sequenced coronaviruses and identified the main 3CL proteinase cleavage sites in polyproteins using alignments. A neural network was trained to recognise the cleavage sites in the genomes obtaining a sensitivity of 87.0% and a specificity of 99.0%. Several proteins known to be cleaved by other viruses were submitted to prediction as well as proteins suspected relevant in coronavirus pathology. Cleavage sites were predicted in proteins such as the cystic fibrosis transmembrane conductance regulator (CFTR), transcription factors CREB-RP and OCT-1, and components of the ubiquitin pathway.Conclusions\nOur prediction method NetCorona predicts coronavirus cleavage sites with high specificity and several potential cleavage candidates were identified which might be important to elucidate coronavirus pathology. Furthermore, the method might assist in design of proteinase inhibitors for treatment of SARS and possible future diseases caused by coronaviruses. It is made available for public use at our website: .",
        "body": "Background\nIn the spring of 2003, the Severe Acute Respiratory Syndrome (SARS) caused numerous fatalities particularly in Southeast Asia and gravely affected the global economy. The causative agent was shown to be a human coronavirus [1], a virus type which normally causes mild cold symptoms in humans. The abrupt appearance raises concern of another break-out of an epidemic of SARS virus or similar strains in the future.Coronaviruses are found in different species ranging from chicken to cattle and humans. Currently, seven coronavirus genomes, including SARS coronavirus (CoV), have been fully sequenced and cluster into four main groups, of which SARS-CoV occupies its own [2,3]. Polyproteins encoded by the coronavirus RNA are processed by viral proteinases yielding mature proteins. The main proteinase 3CLpro performs at least eleven proteolytic cleavages within a single viral polyprotein [4,5]. Viral polyprotein processing is a common theme in viral molecular biology, e.g. as seen in picornaviruses and retroviruses like HIV. Therefore, essential viral proteinases have been suggested as potential targets for specific therapeutic approaches, e.g. by development of specific proteinase inhibitors [6-8].In the case of picornaviruses, virus-encoded proteinases are able to cleave specific cellular targets and thereby severely inhibit the cellular translational machinery (the \"host cell shut-off\" response) while still allowing for high translational activity of viral mRNA [9]. Earlier, we developed a computational approach for predicting potential cleavage sites of picornavirus proteinases 2A and 3C [10]. Badorff et al. successfully used this cleavage predictor to identify the cellular target dystrophin, which they experimentally showed to be cleaved both in vitro and in vivo [11]. However, preliminary studies revealed that this model is not compatible with coronavirus cleavage sites. The general approach is still valid though, and we decided to apply this method to the problem of predicting the 3CLpro proteinase cleavage sites and identifying potential host cell target proteins. We propose that a deeper understanding of coronavirus proteinase function and substrate specificity may benefit further research by: i) increasing the understanding of substrate specificity determinants which may direct studies focusing on the development of specific proteinase inhibitors and ii) providing a method for screening cellular target proteins for potential coronavirus proteinase cleavage sites.In this paper, we describe the development of a computational prediction method using artificial neural networks for predicting coronavirus 3CLpro proteinase cleavage sites. The method is based on known cleavage sites in seven members of the coronavirus family as the cleavage sites are believed to be sufficiently conserved among family members. This notion is supported by the fact that the SARS 3CLpro proteinase has recently been shown capable of catalysing the cleavage of peptide fragments from other coronaviruses at the expected cleavage sites [12].We discuss potential targets of 3CLpro proteinase, e.g. the cystic fibrosis transmembrane conductance regulator (CFTR) and translational and transcriptional factors, which may be involved in the molecular pathology of coronaviruses in general and SARS virus in particular.\nResults\n\nAnalysis of the proteinase cleavage site\nThe 77 annotated coronavirus polyprotein main proteinase cleavage sites were aligned without gaps by constraining the P1 position. Every site had a glutamine (Q) in position P1 (the position just before the cleavage site; the positions are named as suggested by Berger and Schechter [13] with P1, P2, ... etc., N-terminal to the cleavage site and P1', P2', ... etc., C-terminal to the cleavage site). From the sequence logo (Figure 1) a very strong consensus is evident around the cleavage site. As discussed by others [14,15], the coronavirus 3C-like proteinase shares many traits with its picornavirus 3C proteinase counterpart, hence the name. This is reflected in the cleavage site logo although differences between the two are also apparent. Positions P1', P1, and P4 have similar amino acid distribution in the 3C and 3CL proteinase cleavage sites. On the other hand, the coronavirus proteinase has a strong preference for leucine at position P2 while this position is relatively non-conserved among picornavirus proteinase cleavage sites [10]. A recently published study of the crystal structure of 3CLpro from the 229E strain of human coronaviruses indicates that residues at positions P5 to P3 form an anti-parallel &#946; sheet with part of the proteinase, signifying their importance in cleavage site recognition [7].It is clear from the above that a simple, position specific consensus sequence is difficult to define. With the present data set from seven different coronaviruses it is possible to classify correctly 60 (78%) of the 77 cleavage sites by matching an 'LQ' consensus pattern. However, an additional 196 sites in the viral polyproteins are incorrectly classified as cleavage sites, being random occurrences of this pair of amino acids. Classification is improved by using the consensus pattern 'LQ [S/A]', meaning Leu-Gln-(Ser OR Ala), but it is still far from being a useful classifier. The false positive rate is now down to 36 wrong sites, but at the same time only 48 (62%) of the correct cleavage sites are detected. As the pattern becomes more sophisticated, specificity increases (reducing the number of false positives) but at the same time sensitivity drops dramatically (i.e. fewer of the true sites are detected).\nNeural network training and performance\nTo overcome the limitations of simple consensus patterns, we trained an artificial neural network to identify the cleavage sites. The best model was obtained using a three-layered neural network with two hidden neurons and a sequence window encompassing nine amino acids centered on the P1 position, thus encompassing P5-P4'. The network evaluates and assigns a score between 0 and 1 to every glutamine to which it is presented, where a score above 0.5 is considered a positive answer (i.e. a cleavage site is predicted). This model was able to classify correctly 67 of 77 known cleavage sites (87.0%) and 1,358 of 1,372 (99.0%) sites assumed not to be cleaved by the proteinase when testing on independent sites not included when training. The neural network method could thus identify many more of the positive sites with fewer false positives than simple consensus-type methods thereby increasing the classification performance. The Matthews correlation coefficient reached 0.84 for the artificial neural network compared to 0.37, 0.53 and 0.51 for increasingly complex consensus patterns ('LQ', 'LQ [S/A]', ' [T/S/A]X [L/F]Q [S/A/G]' respectively) (Figure 2).To evaluate the predictive power of the neural network, we performed a basic bayesian analysis of the data set test results. The scoring range from 0 to 1 was divided into ten bins and the posterior probability of a positive prediction (a prediction indicating a cleavage) being true was calculated and plotted (Figure 3). The posterior probability in the range 0.5 to 0.8 cannot be determined accurately since relatively few examples score in this interval &#8211; only 3% of the test set (both positive and negative examples) scores between 0.4 and 0.8. However, results indicate that prediction scores can be classified into three categories, those that fall below 0.5 are most likely not cleaved, those that fall between 0.5 and 0.8 are possibly cleaved and those above 0.8 are most likely cleaved if available to the proteinase.\nAnalysis of selected human proteins\nAs mentioned above, there are several experimentally verified examples of host cell protein cleavage by virus proteinases. Thus, both these and other non-coronavirus proteins from Swiss-Prot [16] 41.0 were examined for potential cleavage sites. In total three groups of proteins were examined: i) proteins known to be cleaved by other viruses, ii) proteins which could be targets when considering the pathology of coronaviruses iii) proteins related to the expected immune response to a viral infection. Eukaryotic translation initiation factor 4 gamma (IF4G_HUMAN) has a potential cleavage site after Gln838 (0.822), but also at two other positions although with lower cleavage scores. Cleavage of this protein may lead to host cell shut-off in a similar way to what has been described for picornavirus 2A proteinase [17].Two subunits of the RNA polymerase III are predicted targets of the coronavirus proteinase 3CLpro. RNA polymerase (RPC1_HUMAN) has a predicted cleavage site after Glnl95 with a score (0.765) well above the 0.5 cut-off. The protein is the second largest subunit of the RNA polymerase III complex and if this protein is indeed a cellular proteinase target it might cause disruption of the RNA polymerase III complex upon infection with a coronavirus. A similar disruption would be expected in case of a cleavage of the largest subunit of the complex (RPA1_HUMAN) which also has a predicted cleavage site (at position 329, score 0.704). It agrees with findings that poliovirus disrupts RNA polymerase III function, although this occurs through cleavage of transcription factor IIIC and not the polymerase subunits themselves [18-20]. Several well-known transcription factors contain potential cleavage sites. The highest scoring is CREB-RP (AT6B_HUMAN) with a predicted cleavage site at Gln358 (0.916) close to the DNA binding leucine zipper motif. This is in agreement with findings from picornavirus 3Cpro proteinase although at a different position in the sequence [21]. OCT-1 (PO21_HUMAN) is also predicted to be cleaved by the 3CLpro proteinase with high confidence (0.874) following Gln62 again corresponding to experimental evidence from picornavirus [22]. Several subunits of the transcription initiation factor TFIID, which is a verified target in poliovirus infections [23], have predicted cleavage sites; the 250 kDa subunit (T2D1_HUMAN), the 135 kDa subunit (T2D3_HUMAN), and the 105 kDa subunit (T2DT_HUMAN).The tumor-suppressor protein P53 is known to be cleaved by picornavirus 3Cpro proteinase [24] but this protein is not predicted to contain any coronavirus 3Cpro proteinase cleavage sites. However, P53-binding protein 1 (P531_HUMAN) and P53-binding protein 2 (P532_HUMAN), which stimulate p53-mediated transcriptional activation [25], have several potential cleavage sites.Another known target for viral infections is the microtubule-associated protein 4 (MAP-4) which is cleavable in HeLa cells by the poliovirus 3Cpro proteinase [26,27]. MAP-4 (MAP4_HUMAN) might also be cleavable by 3CLpro albeit with a low score (after Gln 1005 with a score of 0.519) and furthermore microtubule-associated protein RP/EB member 1 and 3 (MAE1_HUMAN and MAE3_HUMAN) have sites which obtain scores above 0.5. The position of the possible cleavage site in MAP-4 is different from that observed with poliovirus 3Cpro reflecting the different specificity of this proteinase.Lung related proteins were examined as early symptoms of SARS could indicate a relation. The cystic fibrosis transmembrane conductance regulator (CFTR_HUMAN) is an ATP-dependent chloride channel. It has a predicted cleavage site with a high score (0.842) following Gln762 in the human sequence. This part of the membrane protein is cytoplasmic and contains several phosphorylation sites (residues 660 &#8211; 813) indicating an accessible region.The epithelial sodium channels play an important role in lung liquid homeostasis [28] and the amiloride-sensitive sodium channel &#948;-subunit (SCAD_HUMAN) has a predicted cleavage site in the cytoplasmic C-terminus (after residue 22) scoring 0.828. A number of proteins involved in the ubiquitin pathway which targets proteins to the proteasome, a necessary step to generate an immune response, have predicted cleavage sites (Swiss-Prot entries UBP1_HUMAN, SOC6_HUMAN, UBPD_HUMAN, UBP4_HUMAN, UBP5_HUMAN, UBPQ_HUMAN, FAFY_HUMAN, FAFX_HUMAN). Cleavage of one or more of these proteins may lead to reduced presentation of viral peptides to cytotoxic T lymphocytes thereby inhibiting the cellular immune response. IRAK-1 (IRA1_HUMAN) which is involved in IL-1 induced activation of cells has a predicted cleavage site after Gln457 scoring 0.859.Interferon-induced protein 6&#8211;16 precursor (INI2_HUMAN) is a membrane protein and was predicted to possess a cleavage site following Gln97 (0.890) which is located in the cytoplasmic part of the mature protein. Protein 6&#8211;16 has been shown to enhance interferon-&#945; antiviral efficacy [29]. Interferon-&#945;, -&#946;, and -&#947; are known to be involved in antiviral defence and have been employed for treatment of SARS [30], but the interferons themselves do not seem to possess cleavage sites.We have listed the human proteins analysed in this work in a table (Table 1).\nDiscussion\nWe have developed a neural network capable of identifying the cleavage site of the coronavirus proteinase 3CLpro and use this model to predict potential cleavage sites in host cell proteins. The predictor is highly specific which means that few false positives are expected, in fact on independent test sets we observed a false positive rate around 1%. The optimal network window size of nine residues agrees well with available structural information about the proteinase from human coronavirus 229E which indicates that the active site makes contact with at least four residues N-terminal to the glutamine [7].The ten sites known to be cleaved but failed to be recognised by the neural network are not dramatically different from the remainder of the sites (Table 2). We therefore do not suspect these to be sites of a different hitherto unknown proteinase, but it would be interesting to see if the lower prediction score reflects a lower cleavage efficiency in vivo. Of the fourteen negative examples wrongly predicted as cleavable (Table 3), the highest scoring were examined more closely. The selected examples all show some resemblance to real cleavage sites but also some resemblance to negative examples which are not predicted as cleavable. They may represent sites in-between which are cleavable to a certain extent but are shielded from cleavage due to conformational issues.Predicted sites even with high scores which are inaccessible to the proteinase (like extracellular domains, transmembrane domains, or buried domains in globular proteins) should be disregarded, as accessibility information is not available to the neural network. Cleavage sites probably exist that are not cleaved because they are not exposed to the solvent sufficiently for the proteinase to work.Others have attempted recognising the cleavage sites of the 3CL proteinase as a component of a coronavirus gene prediction server using different methods [31]. As the goal was different, that predictor is not publicly available and no performance values have been published.\nConclusions\nOur method can be employed by researchers suspecting a possible viral proteinase cleavage but may also prove useful for researchers working with coronavirus function. Finally, the method might facilitate proteinase blocking based drug discovery by providing hints about proteinase affinity to various non-cleavable peptide ligands, which is a possible strategy for drug development [7,32].\nMethods\n\nData Set Preparation\nSeven full-length coronavirus genomes were retrieved from the GenBank database [33] with the following accession numbers: NC_001451 (Avian infectious bronchitis virus, AIBV), NC_001846 (Murine hepatitis virus, MHV), NC_002645 (Human coronavirus 229E, HCoV-229E), NC_003436 (Porcine epidemic diarrhea virus, PEDV), NC_003045 (Bovine coronavirus, BCoV), NC_002306 (Transmissible gastroenteritis virus, TGV), and the TOR2 strain of SARS NC_004718. Deduced polyprotein sequences were aligned and cleavage sites identified from the annotation in NC_004718. Each sequence contained eleven 3CLpro proteinase cleavage sites, thus a total of 77 of these sites were identified. For training a neural network classifier, a number of negative examples (presumed non-cleavage sites) are required. For this purpose, all other glutamines in the viral polyproteins were treated as non-cleavable sites.Three test sets were created for three-fold cross-validation and the training set for one was created by combining the two other test sets. Every test set thus contained 483 examples of which 25 or 26 were positive examples. All testing and results reported are combined values of the three test sets, which are run individually with three separate neural networks to avoid testing on sequences included in training sets.\nSequence logos\nAmino acid conservation in multiple sequence alignments may be visualised using sequence logos. The height of the amino acid one-letter abbreviations reflect the Shannon information content [34] in units of bits at that specific position in the multiple sequence alignment [35]. The basic idea behind the visualisation technique is that the height of each letter in a given position reflects its probability pk(i). The total height of the column reflects the total information content (D(i)) at that specific position in the alignment given by (for proteins):Very conserved positions will then get tall columns with the height of individual residue symbols reflecting the amino acid distribution.\nTraining the neural networks\nThe artificial neural networks used in this work were of the standard feed-forward type. Sparse encoding was used for translating the amino acids to data input for the networks as has been described previously [36,10,37].Training was done with three-fold cross-validation and Matthews correlation coefficients [38] were calculated by summing up true positives, false positives, true negatives, and false negatives in all combinations of training and test sets. Using an architecture with two hidden neurons and a symmetric window of nine amino acids centered on the glutamine in the P1 position it was possible to obtain a correlation coefficient of 0.84 on cross-validated test sets. Care was taken to ensure that all cleavage sites were equally distributed in every cross-validated set.\nBayesian statistics\nThe validity of the statistics depends on the expected fraction of cleavage sites in a given data set, which we only know in the data set at hand. Statistics was thus done on the data set test results in order to create a histogram of prediction probabilities. Statistics was done using Bayes' Theorem:The prediction outcome (0&#8211;1) was divided into 10 bins (Xl) with increments of 0.1. The posterior probability P(Cpos|Xl) gives the probability of a positive prediction (that is, a cleavage) being true given the bin. This can be calculated from the prior probability P(Cpos), which is the fraction of positive examples in the data set, and the class-conditional probability P(Xl|Cpos) for positive examples, which is the fraction of positive examples in the bin Xl. P(Xl) is the fraction of prediction outcomes in bin Xl.\nSearching for potential cleavage sites\nAn averaged sum of the score of all three networks arising from the three-fold cross-validation was used for prediction. Each network outputs a score in the range [0.000&#8211;1.000], where scores below 0.5 indicate non-cleavage and scores above 0.5 indicate potential cleavage. This method is also employed by the prediction web server mentioned below. The Swiss-Prot database [16] release 41.0 (February 2003) was downloaded and proteins from this database were used as targets for the neural network predictions.\nAvailability\nOur neural network based prediction method, NetCorona, for prediction of potential cleavage sites of the SARS-3CLpro proteinase is publicly available by following the link 'CBS prediction servers' from  or at this specific URL: \nList of abbreviations\nAIBV &#8211; Avian Infectious Bronchitis Virus, BCoV &#8211; Bovine Coronavirus, CFTR &#8211; cystic fibrosis transmembrane conductance regulator, CoV &#8211; Coronavirus, HCoV-229E &#8211; Human Coronavirus 229E, MHV &#8211; Murine Hepatitis Virus, NN &#8211; neural network, PEDV &#8211; Porcine Epidemic Diarrhea Virus, SARS &#8211; severe acute respiratory syndrome, TGV &#8211; Transmissible Gastroenteritis Virus.\nAuthors' contributions\nLK carried out sequence retrieval, alignment, neural network training, prediction on potential proteins and drafted the manuscript. OL provided input on virus pathology and suggested human proteins for prediction. SB provided general inputs and improvements to the manuscript. Finally, NB conceived of and supervised the study in addition to assisting with the drafting of the manuscript.",
        "pmid": "15180906",
        "title": "Coronavirus 3CLpro proteinase cleavage sites: Possible relevance to SARS virus pathology",
        "authors": "Lars Kiemer;Ole Lund;S\u00f8ren Brunak;Nikolaj Blom",
        "doi": "10.1186/1471-2105-5-72"
    },
    {
        "keywords": "Bats;SARS;MERS;Coronavirus;Emerging infectious diseases",
        "abstract": "Bats have been recognized as the natural reservoirs of a large variety of viruses. Special attention has been paid to bat coronaviruses as the two emerging coronaviruses which have caused unexpected human disease outbreaks in the 21st century, Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV), are suggested to be originated from bats. Various species of horseshoe bats in China have been found to harbor genetically diverse SARS-like coronaviruses. Some strains are highly similar to SARS-CoV even in the spike protein and are able to use the same receptor as SARS-CoV for cell entry. On the other hand, diverse coronaviruses phylogenetically related to MERS-CoV have been discovered worldwide in a wide range of bat species, some of which can be classified to the same coronavirus species as MERS-CoV. Coronaviruses genetically related to human coronavirus 229E and NL63 have been detected in bats as well. Moreover, intermediate hosts are believed to play an important role in the transmission and emergence of these coronaviruses from bats to humans. Understanding the bat origin of human coronaviruses is helpful for the prediction and prevention of another pandemic emergence in the future.",
        "body": "Background\nBats, with extensive geographical distribution and capability of flight, constitute the second largest group of mammalian species and have been documented as natural hosts of a large number of diverse viruses such as lyssaviruses, paramyxoviruses and filoviruses [1, 2]. In the past decade, numerous novel coronaviruses have been discovered in a wide variety of bat species throughout Asia, Europe, Africa and America [3]. Within the coronavirus genera Alphacoronavirus and Betacoronavirus, which mainly infect mammals, 7 out of the 15 currently assigned viral species have only been found in bats [4]. It is proposed that bats are major hosts for alphacoronaviruses and betacoronaviruses and play an important role as the gene source in the evolution of these two coronavirus genera [5]. Among the coronaviruses harbored by bats, some have drawn particular research interests, as they have been found to be associated with two high profile human disease outbreaks, Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS).In this review, we focus on the emerging coronaviruses putatively linked to a zoonotic origin from bats, represented by SARS coronavirus (SARS-CoV) and MERS coronavirus (MERS-CoV). We present an overview of current evidence for bat origin of these two viruses and also discuss how the spillover events of coronavirus from animals to humans may have happened. Considering that bats have been known to harbor more coronaviruses than any other species, it is likely that SARS-CoV and MERS-CoV won&#8217;t be the only bat coronaviruses to jump among species and cause human infections. Bat coronaviruses should be seriously regarded in light of their potential risks to public health.\nEmergence of SARS and MERS\nSARS first emerged in late 2002 in Guangdong Province, southern China, as a novel clinical severe disease (termed &#8220;atypical pneumonia&#8221;) marked by fever, headache and subsequent onset of respiratory symptoms including cough, dyspnea and pneumonia. Being highly transmissible among humans, SARS rapidly spread to Hong Kong and other provinces across China and then to other 28 countries [6, 7]. By July 2003, it had caused 8096 confirmed cases of infection in 29 countries, 774 (9.6&#160;%) of which were fatal (http://www.who.int/csr/sars/country/table2004_04_21/en/). The second outbreak in 2004 only caused 4 infections with no mortality nor further transmission [8].The MERS epidemic emerged in the Kingdom of Saudi Arabia (KSA) since June 2012, with a similar clinical syndrome to SARS but seemingly less transmissible. In addition to respiratory illness, renal failure was identified in some severe cases [9&#8211;11]. Unlike SARS which had numerous super-spreader events, most MERS cases were independent clusters and limited to countries in the Middle East, particularly in KSA. Limited MERS cases have been reported in African and European countries and the United States of America, but exclusively in individuals travelling back from the Middle East. Some patients were reported to have a history of contact with camels while many other cases lacked this epidemiological link [9&#8211;11]. The MERS pandemic in the Republic of Korea in 2015 was caused by a single person who returned from travel in the Middle East. This made the Republic of Korea to be home to the second largest MERS epidemic with a total of 185 confirmed cases and 36 deaths [11, 12]. By 18 August 2015 a total of 1413 laboratory-confirmed cases of MERS have been reported worldwide with a median age of 50&#160;years, including 502 related deaths. The mortality of MERS (approximately 35&#160;%) is much higher than that of SARS (around 10&#160;%).\nSARS-CoV and MERS-CoV represent two different species in the genus \n\nGenomic structure and taxonomic classification\nSARS-CoV and MERS-CoV share similar genome organization with other coronaviruses, but display unique genomic structures and evolutionary lineages. The coronavirus genome possesses 6-to-7 major open reading frames (ORFs) in the characteristic gene order in the 5&#8217; to 3&#8217; direction: ORF1a and 1b which comprise two-thirds of the genome and encode the nonstructural polyproteins, and four ORFs downstream that encode structural proteins: spike protein (S), envelope protein (E), membrane protein (M) and nucleocapsid protein (N). Some coronaviruses have a hemagglutinin-esterase (HE) gene between ORF1b and S. Besides the coronavirus-conserved genes, the SARS-CoV genome contains a number of specific accessory genes including ORF3a, 3b, ORF6, ORF7a, 7b, ORF8a, 8b and 9b [13&#8211;15]. Comparably, MERS-CoV encodes five unique accessory genes, designated ORF3, ORF4a, ORF4b, ORF5 and ORF8b. None of these genes have been shown to be related to other known coronavirus genes at the time of discovery [16, 17]. MERS-CoV was found to have 75 and 77&#160;% amino acid (aa) sequence identity in 7 conserved replicase genes with two previously identified bat coronaviruses: BtCoV-HKU4 and BtCoV-HKU5. Based on the classification criteria of the the International Committee on Taxonomy of Viruses (ICTV), SARS-CoV and MERS-CoV represent two novel distinct coronavirus species in the genus Betacoronavirus (Fig.&#160;1a and Table&#160;1) [10, 18, 19]. Members of betacoronaviruses are separated into four lineages, A, B, C and D. SARS-CoV and MERS-CoV are clustered in lineage B and C, respectively [18].Fig. 1Phylogenetic analysis of bat coronaviruses with other coronaviruses. The phylogenetic tree was constructed based on 816-nt partial RdRp sequences (a) and full-length spike protein sequences (b). Available sequences were retrieved from GenBank and aligned using ClustalW. The alignment was used to construct tree by MEGA (Version 5.1) with the neighbor-joining statistical method. Bootstrap values were calculated from 1000 replicates (values &#8805;50 are shown). Bat coronaviruses are drawn in bold and named following bat species, plus BtCoV, strain name, and GenBank accession numberTable 1Comparison of bat coronaviruses with SARS-CoV or MERS-CoV in conserved replicase domains and structural proteinsCoV strainBat speciesCountry% amino acid sequence identitya with SARS-CoV or MERS-CoVADRP3CLproRdRpHelExoNNendoUOMTConcatenated domains@\nSEMNHKU3\nRhinolophus sinicus\nChina92.099.398.699.298.198.098.396.079.710098.696.7Rp3\nRhinolophus sinicus\nChina95.499.799.599.799.297.498.397.780.310097.398.1Rm1\nRhinolophus macrotis\nChina91.099.399.399.397.997.198.095.680.698.797.397.6Rf1\nRhinolophus ferrumequinum\nChina92.399.798.699.597.997.796.396.078.496.197.795.5Rs672\nRhinolophus sinicus\nChina97.099.399.899.399.198.699.098.480.210098.698.6Rs3367\nRhinolophus sinicus\nChina97.010099.699.899.298.398.098.492.310098.2100RsSHC014\nRhinolophus sinicus\nChina96.999.799.699.899.298.897.798.490.098.798.299.5WIV1\nRhinolophus sinicus\nChina97.099.799.599.899.298.898.098.492.210098.299.8Cp/Yunnan 2011\nChaerephon plicata\nChina97.610099.198.598.198.697.398.281.110099.198.1Rp/Shaanxi 2011\nRhinolophus pusillus\nChina93.510099.299.798.997.799.096.981.197.496.898.1YNLF_31C\nRhinolophus ferrumequinum\nChina97.299.799.699.799.498.397.798.479.210098.698.3BM48-31\nRhinolophus blasii\nBulgaria76.894.498.098.195.691.991.688.375.992.191.488.5HKU4-1\nTylonycteris pachypus\nChina55.58189.892.185.47682.878.46756.17965.8HKU5-1\nPipistrellus abramus\nChina56.482.691.893.891.779.785.780.16453.67961.4NeoCoV\nNeoromica capensis\nSouth Africa86.796.79898.498.294.196.3956487.794.291SC2013\nVespertilio superans\nChina53.57988.593.485.676.688.185.76984.584.774.4\naCalculated with MEGA5.1 using a pairwise deletion option; Bat SL-CoVs are listed in the upper part of the table while camel MERS-CoV and bat CoVs related to MERS-CoV in the lower part@Seven domains were series connected and calculated with MEGA5.1 using a pairwise deletion optionADRP, ADP-ribose 1-phosphatase; 3CLpro, coronavirus NSP5 protease; RdRp RNA-dependent RNA polymerase; Hel, helicase; ExoN, exoribonuclease; NendoU, endoribonuclease; OMT, 2&#8217;-O-methyltransferaseGenBank accession numbers: Tor2, NC_004718; HKU3, DQ022305; Rp3, DQ071615; Rm1, DQ412043; Rf1, DQ412042; Rs672, FJ588686; Rs3367, KC881006; RsSHC014, KC881005; WIV1, KF367457; Cp/Yunnan2011, JX993988; Rp/Shaanxi2011, JX993987; YNLF_31C, KP886808; EMC/2012, JX869059; HKU5-1, NC_009020; HKU4-1, NC_009019; BetaCoV/SC2013, KJ473821; NeoCoV, KC869678\nReceptor usage\nThe S protein of coronaviruses is a surface-located trimeric glycoprotein consisting of two subunits: the N-terminal S1 subunit and the C-terminal S2 subunit. The S1 subunit specializes in recognizing and binding to the host cell receptor while the S2 region is responsible for membrane fusion. Compared with the S2, the S1 subunit shows much higher variability [20]. Owing to its function of receptor binding, the variation in S protein defines in large part the tissue tropism and host range of different coronaviruses [21].Angiotensin-converting enzyme 2 (ACE2) was identified to be the functional receptor of SARS-CoV [22&#8211;24]. A 193 aa fragment (aa 318&#8211;510) of SARS-CoV S protein was demonstrated to bind ACE2 more efficiently than the full S1 domain and was defined as the receptor-binding domain (RBD) of SARS-CoV [25]. A loop subdomain (aa 424&#8211;494) that directly contacts with ACE2 was further identified as the receptor-binding motif (RBM) by crystal structure analysis [26]. In the RBM, several aa residues were found to be critical for receptor binding and changes in these key residues resulted in different binding efficiency among different SARS-CoV isolates [26&#8211;28].Dipeptidyl peptidase 4 (DPP4, also known as CD26) was identified as a functional receptor for MERS-CoV [29] and it is relatively conserved among mammalian species. Published results indicated that MERS-CoV can infect and replicate in most cell lines derived from human, non-human primate, bat, swine, goat, horse, rabbit, civet, and camel, but not from mice, hamster, dog, ferret, and cat [29&#8211;36]. DPP4 from camel, goat, cow, and sheep can be also recognized by MERS-CoV and can support MERS-CoV replication [30, 35]. Resolved crystal structures demonstrate that DPP4-recognizing RBD is localized to the S1 C-terminal portion of S protein of MERS-CoV [37&#8211;39]. The RBD of MERS-CoV consists of ~240 residues, spanning aa 367&#8211;606, which fold into a structure consisting of two subdomains, the core subdomain and the external subdomain. The core subdomain of MERS-CoV RBD is structurally similar to that of the SARS-CoV RBD, but the external subdomain (also named as RBM) is different to that of the SARS-CoV [37&#8211;39].\nBat origin of SARS-CoV\n\nCivets are intermediate and trasnmission host of SARS-CoV\nEpidemiological survey showed that early cases of SARS in 2002&#8211;2003 and all 4 cases in 2003&#8211;2004 had a history of animal contact through animal trade in wet markets or in restaurants where live animals were kept in Guangdong Province. Molecular detection and virus isolation studies suggested that the pandemic-causing SARS-CoV originated from traded civets in wet markets. This was indirectly confirmed by the massive culling of market civets, which was believed to play a major role in efficiently containing the SARS pandemics and no further SARS case was reported after 2004 [40&#8211;42].However, subsequent extensive epidemiology studies did not find SARS-CoV in farmed or wild-caught civets, indicating that other animal(s) was involved in SARS-CoV transmission in the animal market or other trading activities and civets are unlikely the natural reservoir of SARS-CoV [43&#8211;45].\nDiscovery of diverse SARS-like coronaviruses in bats\nSeveral years before the outbreak of SARS, two other zoonotic viruses, Nipah virus and Hendra virus, emerged in Asia and Australia and they were both known to be originated from bats [46, 47]. These led scientists to consider bats in the search of reservoirs of SARS-CoV. In 2005, a breakthrough was made as two independent research groups reported, almost simultaneously, the discovery of novel coronaviruses related to SARS-CoV in horseshoe bats (in the genus Rhinolophus) in China, which were termed SARS-like coronavirus (SL-CoV) [48, 49]. These bat SL-CoVs from both mainland China and Hong Kong manifested genome sequence identity of 88&#8211;90&#160;% among themselves and 87&#8211;92&#160;% identity to human or civet SARS-CoV isolates. The unique set of ORFs exclusively found in SARS-CoV was also present in bat SL-CoVs, demonstrating the close phylogenetic relationship between SARS-CoV and SL-CoV. The discovery of bat SL-CoV boosted researchers&#8217; interest in coronavirus surveillance studies in bats. In following years, SL-CoV RNA was detected in Rhinolophus species of a wider geographic range in China. The provinces or regions where SL-CoV-positive bats were captured included Hong Kong, Guangxi, Hubei, Shandong, Guizhou, Shaanxi and Yunnan [50&#8211;53]. 7 conserved replicase domains in orf1ab of these SL-CoVs found in China were compared with those of SARS-CoV (Table&#160;1). They all shared higher than 95&#160;% aa sequence identity with SARS-CoV in the concatenated domains and therefore can be considered to belong to SARS-CoV species [54].SL-CoVs were also discovered in rhinolophids from Slovenia, Bulgaria and Italy in Europe [55&#8211;57]. These European SL-CoVs exhibited significant genetic variation from Chinese isolates. The strain BM48-31 from Rhinolophus blasii in Bulgaria was highly divergent from Chinese isolates, displaying major sequence differences in several genes including ORF3b and ORF6 and lacking the coding region of ORF8 in its genome [55]. In Africa, novel betacoronaviruses related to SARS-CoV have been detected in Hipposideros and Chaerophon species from Ghana, Kenya and Nigeria. However, compared with Asian and European SL-CoVs, these viruses of non-rhinolophid origin were phylogenetically distant to SARS-CoV. The Western African isolates even formed a potential new lineage of Betacoronavirus in the phylogenetic tree (Fig.&#160;1a) [58&#8211;60].\nMost related ancestor of SARS-CoV in bats\nAlthough the aforementioned bat SL-CoVs showed high sequence identity to SARS-CoV, two deletions were present in the RBM of their S proteins [48, 49]. The differences in RBM substantially changed the receptor usage. In a study using an HIV-based pseudovirus system and cell lines expressing human, civet, and horseshoe bat ACE2 molecules, the bat SL-CoV Rp3 S protein demonstrated its inability to use ACE2 as cell receptor [61]. However, the chimeric Rp3 S protein carrying the RBD of SARS-CoV S protein was conferred the capability of cell entry via human ACE2 [61]. These results suggested that bat SL-CoVs such as Rp3 were unlikely to cause human infection. Therefore, they may not be considered as the direct progenitor of SARS-CoV. Besides, the theory of bat origin of SARS-CoV lacked a powerful support due to the failure of direct isolation of SL-CoV from bats, despite numerous trials by our group as well as many others around the world.During our longitudinal surveillance at a Rhinolophus sinicus colony in Yunnan Province over the years, a major breakthrough came in 2013 when diverse SL-CoVs were discovered in the single colony [53]. In this colony, there were at least 7 different strains related to SARS-CoV, HKU3, Rs672 or Rf1, based on analysis of the region corresponding to SARS-CoV RBD. Intriguingly, unlike all previously described SL-CoVs, two strains, designated Rs3367 and RsSHC014, did not contain the deletions in this region. Rs3367 showed a particularly high sequence identity to SARS-CoV in RBD and was identical to SARS-CoV in several key amino acid residues known to be important for receptor binding [53]. Whole genome sequencing revealed that Rs3367 and RsSHC014 shared more than 95&#160;% genome sequence identity with human and civet SARS-CoV, which was remarkably higher than that of any other bat SL-CoV (76 to 92&#160;%). Regarding individual genes, the amino acid sequence identity between Rs3367 or RsSHC014 and SARS-CoV was higher than 96&#160;% in ORF1a, 1b, 3a, 3b, E, M and N genes [53]. Most importantly, a live SL-CoV was isolated for the first time from bat fecal samples [53]. This virus, termed WIV1, had almost identical sequence (99.9&#160;%) to Rs3367 and was demonstrated to use ACE2 molecules from humans, civets and Chinese horseshoe bats for cell entry. It also displayed infectivity in cell lines from a broad range of species including human, pig, and bat. Furthermore, the close relatedness between WIV1 and SARS-CoV was confirmed by neutralization effect of convalescent SARS patient sera on WIV1 [53]. The isolation of a bat SL-CoV genetically closely resembling SARS-CoV and having a functional S protein capable of using the same ACE2 receptor as SARS-CoV provided robust and conclusive evidence for the bat origin of SARS-CoV.\nPossible origin of SARS-CoV from recombination of different SL-CoVs\nDespite the fact that Rs3367 or WIV1 is unprecedently close to SARS-CoV in terms of RBD region and genome identity, still there are gaps between them and the immediate ancestor of SARS-CoV. ORF8 is a highly variable gene and remarkable differences can be observed among SARS-CoVs and SL-CoVs of different host origins. Isolates from civets and from early phase of the 2002/2003 pandemic contained a single long ORF8, while in the human SARS-CoV isolates from the middle and late phase of the pandemic the ORF8 was disrupted into two ORFs, ORF8a and ORF8b, as a result of the acquisition of a 29-nt deletion after interspecies transmission to humans [8, 40, 62]. The SL-CoVs from Rhinolophus sinicus, including Rs3367, however, had a single ORF8 with only 32&#8211;33&#160;% amino acid identities to that of civet SARS-CoV. In contrast, the ORF8 of two novel SL-CoV strains recently reported in Yunnan from another rhinolophid species, Rhinolophus ferrumequinum, exhibited exceptionally high (81.3&#160;%) amino acid identity to civet SARS-CoV SZ3 [63]. This is consistent with isolate Rf1, a SL-CoV reported earlier from R. ferrumequinum in Hubei Province, of which the ORF8 shared 80.4&#160;% amino acid identity to SZ3 [48]. Potential recombination sites were identified around the ORF8 region between SL-CoVs from R.sinicus and R.ferrumequinum and it has been suggested that the ancestor of civet SARS-CoV probably acquired ORF8 from R.ferrumequinum SL-CoVs by recombination [63].\nAnimal origins of MERS-CoV\nAs with SARS-CoV, most early MERS cases had contact history with animals, e.g., dromedary camels [64, 65]. MERS-CoV RNA was detected in camels from Saudi Arabia, Qatar and Egypt and showed high similarities (>99&#160;%) to human MERS-CoV in genomic sequences [66&#8211;71]. Serological evidence further confirmed a high prevalence of MERS-CoV infections in camels in the Middle East [72&#8211;77], Africa [78&#8211;80] and Europe (Spain) [73]. The neutralization antibodies in camels could be traced back to 1983 [73, 80]. These results strongly suggested that MERS-CoV infection in humans were transmitted through close contact with infected camels [66, 76, 81&#8211;83].\nBat viruses related to MERS-CoV\nPrior to the emergence of MERS-CoV, a group of bat coronaviruses had been reported including Tylonycteris bat coronavirus HKU4 (BtCoV-HKU4) in Tylonycteris bats and Pipistrellus bat coronavirus HKU5 (BtCoV-HKU5) in Pipistrellus bats in China [50, 84, 85], E.isa/M/Spain/2007 in Eptesicus isabellinus bats in Spain [86] and N.noc/VM366/2008/NLD in Pipistrellus pipistrellus bats in the Netherlands [87]. Based on genomic sequence analysis, these bat coronaviruses were grouped into lineage C of the genus Betacoronavirus. After the outbreak of MERS, MERS-CoV related coronaviruses were found in more bat species and countries [88&#8211;96]. Among these viruses, full-length or near full-length genomes of BtCoV-HKU4, BtCoV-HKU5, SC2013 and NeoCoV have been characterized. By genomic analysis of lineage C betacoronaviruses, MERS-CoV derived from camels show high similarities to human MERS-CoV with >99.5&#160;%&#160;nt identities, confirming that the human and camel isolates belong to the same coronavirus species. Bat HKU4, HKU5, NeoCoV and SC2013, shared 69.8, 70, 85.6 and 75.6&#160;%&#160;nt identities with MERS-CoV at genomic level, respectively. Seven conserved replicase domains in orf1ab of MERS-CoV related viruses were compared with MERS-CoV (Table&#160;1). The concatenated translated domains of NeoCoV shared 95&#160;% aa sequence identity with MERS-CoV and it could be classified as the same MERS-CoV species [54]. Other bat coronaviruses, HKU4, HKU5 and SC2013, could be considered as different coronavirus species. The most recent ancestor analysis speculated that MERS-CoV may have jumped from bats to camels approximately 20&#160;years ago in Africa, with camels then being imported into the Arabian Peninsula [92], while HKU5 and MERS-CoV may have diverged from their common ancestor about 400 to 500&#160;years ago [85].Although NeoCoV is closer to MERS-CoV than other bat coronaviruses at genomic level, the phylogenetic analysis of the spike protein showed that HKU4 is the most closely related to MERS-CoV among all currently known bat coronaviruses, sharing 67&#160;% sequence identity (Fig.&#160;1b). This is correlated with the capability of HKU4 of using DPP4 as its functional receptor. However, HKU4 preferred bat DPP4 over human DPP4, whereas MERS-CoV showed the opposite trend [97]. It was suggested that MERS-CoV ancestors had been circulating in bats for very long time. MERS-CoV has evolved to adapt to use human receptor and the DPP4-recognizing bat coronaviruses like HKU4 may follow up, thereby posing a serious risk to human health [97, 98].\nComparison of transmission of MERS-CoV and SARS-CoV\nBoth SARS-CoV and MERS-CoV are emerging zoonotic pathogens that crossed the species barriers to infect humans [10, 53, 99]. Evidence showed that SARS-CoV and MERS-CoV originated from bats, the nature reservoirs, then transmitted to human via intermediate hosts civets and camels, respectively [10, 40, 53, 81, 100]. Human SARS-CoV infection originated from the direct contact between humans and civets in markets or restaurants. Closing wet markets and cleaning civet cut off the spread chain of SARS-CoV and effectively ended the SARS epidemic [40, 42, 101]. In contrast, MERS-CoV is believed to have existed in camels for a very long time and camels are widely distributed in Middle East and African countries, serving as important transport vectors and sources of meat and milk for the local population. Therefore, it is difficult to adopt the same strategy of SARS-CoV control in the prevention of future MERS-CoV outbreaks. Until a comprehensive approach is found, which most likely will involve the effective vaccination of camels against MERS-CoV among other measures, it is envisaged that sporadic human infection will persist for some time in the future [11, 70].\nBat coronaviruses and human coronavirus 229E (HCoV-229E) and NL63 (HCoV-NL63)\nHCoV-229E was found in the 1960s and causes comparatively mild common colds worldwide [102]. A bat coronavirus detected in Hipposideros caffer ruber in Ghana termed Hipposideros/GhanaKwam/19/2008 was genetically related to HCoV-229E. Its RdRp fragment shared 92&#160;% nucleotide sequence identity with HCoV-229E and they were predicted to share a most recent common ancestor (MRCA) only 200&#160;years ago [58]. A recent study characterized more 229E-related coronaviruses discovered in hipposiderid bats from Ghana on full genome level. These bat coronaviruses were more diversified and formed a single viral species with HCoV-229E. Interestingly, phylogenetic analysis revealed the intermediate position of a 229E-related alpaca virus between bat and human viruses. These findings suggested the ancestral origin of HCoV-229E in hipposiderid bats and the role of camelids as potential intermediate hosts was hypothesized [103].HCoV-NL63 was first isolated from babies suffering of pneumonia and bronchiolitis in 2004 [104]. To date, HCoV-NL63 has been found worldwide with up to 9.3&#160;% detection rate in hospitalized respiratory tract samples [105]. In 2010, a bat coronavirus termed ARCoV.2 (Appalachian Ridge CoV) detected in North American tricolored bat (Perimyotis subflavus) in the US showed close relationship with HCoV-NL63. The MRCA for HCoV-NL63 and ARCoV.2 was predicted to have existed 563 to 822&#160;years ago [106, 107]. Further analysis indicated that HCoV-NL63 can replicate in cell lines derived from the lungs of tricolored bats [107]. These results suggest that prototypes of HCoV-NL63 may also exist in bats and there may also be a bat origin of this human coronavirus.\nConclusions\nAlthough the study of bat-borne coronaviruses has only started just about 10&#160;years ago, the scientific community has already learnt a great deal of useful lessons which will be instrumental in mitigating, predicting, and preventing future zoonotic coronavirus outbreaks. Some of these lessons are summarized below.Bats harbor coronaviruses with great genetic diversity. It is believed that most, if not all, currently circulating alphacoronaviruses and betacoronaviruses in different mammals are evolutionally linked to ancestral coronaviruses originated from bats. Different species of rhinolophid bats in China carry genetically diverse SARS-like coronaviruses, some of which are direct ancestors of SARS-CoV and hence have the potential to cause direct interspecies transmission to humans. Meanwhile, different coronavirus species closely related to MERS-CoV are circulating in bats. Bats are likely natural reservoirs of MERS-CoV or an ancestral MERS-like CoV. It is hypothesized that bat MERS-like CoV jumped to camels or some other as yet unidentified animal several decades ago. The virus evolved and adapted with accumulating mutations in camels and then was transmitted to humans very recently. It took almost a decade from the first discovery of SL-CoV in bats to the final isolation of the SARS-CoV ancestral virus from bats, so continuing surveillance is vital to uncover the origin of MERS-CoV and bats should certainly be a priority of research. Besides, as the spike protein and host receptor are key factors of cross-species transmission of coronaviruses, characterization of the receptor and key binding sites of the spike protein will be important in estimating host tropism of bat coronaviruses and predicting spillover risk.With human activity increasingly overlapping the habitats of bats, diseases outbreaks resulted from spillover of bat coronaviruses will continue to occur in the future despite the fact that direct transmission of bat coronaviruses to humans appears to be rare. To better prepare ourselves in predicting and preventing the next emergence of a coronavirus disease, it is necessary to maintain our vigilance in long-term coronavirus surveillance studies in bats as well as in other wildlife and livestock. Combined with other laboratory-based studies such as receptor specificity, pathogenesis and animal infection, a focus on continued surveillance will help us to improve risk assessment as well as to reveal the potential intermediate hosts that may play an important role in the interspecies transmission of various known and as yet unknown bat coronaviruses.",
        "pmid": "26689940",
        "title": "Bat origin of human coronaviruses",
        "journal_title": "Virology Journal",
        "authors": "Ben Hu;Xingyi Ge;Lin-Fa Wang;Zhengli Shi",
        "doi": "10.1186/s12985-015-0422-1"
    },
    {
        "keywords": "",
        "abstract": "As cell-invading molecular machinery, coronavirus spike proteins pose an evolutionary conundrum due to their high divergence. In this study, we determined the cryo-EM structure of avian infectious bronchitis coronavirus (IBV) spike protein from the &#947;-genus. The trimeric IBV spike ectodomain contains three receptor-binding S1 heads and a trimeric membrane-fusion S2 stalk. While IBV S2 is structurally similar to those from the other genera, IBV S1 possesses structural features that are unique to different other genera, thereby bridging these diverse spikes into an evolutionary spectrum. Specifically, among different genera, the two domains of S1, the N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD), diverge from simpler tertiary structures and quaternary packing to more complex ones, leading to different functions of the spikes in receptor usage and membrane fusion. Based on the above structural and functional comparisons, we propose that the evolutionary spectrum of coronavirus spikes follows the order of &#945;-, &#948;-, &#947;-, and &#946;-genus. This study has provided insight into the evolutionary relationships among coronavirus spikes and deepened our understanding of their structural and functional diversity.",
        "body": "Introduction\nAs large enveloped RNA viruses, coronaviruses are capable of adapting to new hosts with relative ease through mutations and recombinations [1&#8211;3]. As a result, coronaviruses infect a wide range of mammalian and avian species, and have genetically evolved into four major genera: &#945;, &#946;, &#947;, and &#948; [4]. Coronaviruses from the four genera all contain envelope-anchored spike proteins that mediate viral entry into host cells [5, 6]. During viral entry, the spikes bind to host receptors through their S1 subunits and then fuse viral and host membranes through their S2 subunits. On the one hand, the spikes interact with host receptors and other host factors, hence needing to evolve for better adaptation to these host factors [7&#8211;10]. On the other hand, they are exposed to the host immune system, thereby needing to evolve to evade the host immune surveillance [11&#8211;14]. Consequently, the spikes are the most divergent among all coronavirus proteins [6]. The S1 subunits are particularly divergent, with little or low sequence similarities across different genera [15]. How coronavirus spikes have evolved to their current diverse structures imposes a major evolutionary conundrum.Traces of protein evolution can often be found more reliably in their tertiary structures and related functions than in their primary structures, because proteins generally need to evolve within certain structural and functional constraints [16, 17]. To decipher the evolutionary puzzles surrounding coronavirus spikes, extensive structural studies have been carried out using both X-ray crystallography and cryo-electron microscopy (cryo-EM) [12&#8211;14, 18&#8211;28]. These studies have resulted in structure determinations of S1 from the &#945;- and &#946;-genera and spike ectodomains from the &#945;-, &#946;-, and &#948;-genera. Coronavirus spikes exist in two distinct conformations: the pre-fusion structures are present on mature virions and have a clove-like shape with three S1 heads sitting on top of a trimeric S2 stalk [12&#8211;14, 18&#8211;20]; the post-fusion structures are the membrane-fusion state and have a dumbbell-like shape with three S2 subunits forming a six-helix bundle structure [29&#8211;35]. Whereas the structures of S2 from different genera are similar to each other in both the pre- and post-fusion states, the S1 subunits from different genera diverge structurally and they also recognize a variety of host receptors [6, 7]. S1 contains two domains, N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD), either or both of which can function as the receptor-binding domain. The S1 domains from different genera contain structural features that are unique to their genus. S1-CTDs are particularly diverse, with low or no structural similarity across different genera [15]. Overall, these previous studies have provided structural snapshots of coronavirus spikes from &#945;-, &#946;-, and &#948;-genera. However, because the structures of &#947;-coronavirus spikes were still missing, we lacked a clear picture of the evolutionary relationships among coronavirus spikes from different genera.In the current study, we determined the cryo-EM structure of avian infectious bronchitis coronavirus (IBV) spike, the first such structure from the &#947; genus. The IBV spike possesses structural features that are unique to different other genera, suggesting that it falls in the middle of an evolutionary spectrum of coronavirus spikes. We also discuss how the structural evolution of coronavirus spikes has affected their functions as cell-invading molecular machinery. Overall, our study has filled in a critical gap in the structural, functional and evolutionary studies of coronavirus spikes, and deepened our understanding of viral evolutions in general.\nMaterials and methods\n\nExpression and purification of IBV spike ectodomain\nIBV spike gene (virus strain M41; GenBank number ABI26423.1) was synthesized (Genscript) with codons optimized for insect cell expression. Its ectodomain (residues 20&#8211;1084) was cloned into pFastBac vector (Life Technologies Inc.) with a N-terminal honeybee melittin signal peptide and C-terminal GCN4 and His6 tags. It was expressed in Sf9 insect cells using the Bac-to-Bac system (Life Technologies Inc.) and purified as previously described [21]. Briefly, the protein was harvested from cell culture medium, and purified sequentially on Ni-NTA column and Superdex200 gel filtration column (GE Healthcare). IBV S1-CTD (residues 248&#8211;495) was expressed and purified in the same way as the IBV spike ectodomain, although it only contains a C-terminal His6 tag and does not contain the GCN4 tag.\nCryo-electron microscopy\nFor sample preparation, aliquots of IBV spike ectodomain (3 &#956;l, 0.35 mg/ml, in buffer containing 2 mM Tris pH7.2 and 20 mM NaCl) were applied to glow-discharged CF-2/1-4C C-flat grids (Protochips). The grids were then plunge-frozen in liquid ethane using a FEI MarkIII Vitrobot system (FEI Company).For data collection, images were recorded using a Gatan K2 Summit direct electron detector in the direct electron counting mode (Gatan), attached to a FEI Titan-Krios TEM, at Arizona State University. The automated software SerialEM [36] was used to collect ~2,000 total movies at 37,700x magnification and at a defocus range between 1 and 3 &#956;m. Each movie had a total accumulated exposure of 53.66 e/&#197;2 fractionated in 50 frames of 200 ms exposure. Data collection statistics are summarized in S1 Table.For data processing, whole frames in each movie were corrected for beam-induced motion and dose compensation using MotionCor2 [37] and ~1,400 best images were manually selected (we manually discarded micrographs with only carbon field of view or thick ice after motion correction as well as micrographs with defocus parameter higher than 4.5 &#956;m after CTF estimation). The final image was bin-averaged to lead to a pixel size of 1.02 &#197;. The parameters of the microscope contrast transfer function were estimated for each micrograph using GCTF [38]. Particles were automatically picked and extracted using RELION [39] with a box size of 320 pixels. Initially, ~802,000 particles were subjected to 2D alignment and clustering using RELION, and the best classes were selected for an additional 2D alignment. ~5,000 best particles were applied for creating the initial 3D model using RELION. ~170,000 particles selected from 2D alignment were then subjected to 3D classification and the best class with ~100,000 particles were subjected to 3D refinement to generate the final density map. The final density map was sharpened with modulation transfer function of K2 operated at 300kV using RELION post-processing. Reported resolutions were based on the gold-standard Fourier shell correlation (FSC) = 0.143 criterion, and Fourier shell correction curves were corrected for the effects of soft masking by high-resolution noise substitution [40]. Data processing statistics are summarized in S1 Table.\nModel building and refinement\nThe initial model of IBV spike ectodomain was obtained by fitting the seven parts (S1-NTD, S1-CTD, two parts of SD1, two parts of SD2, and S2) of the porcine delta coronavirus spike structure (PDB ID: 6B7N) individually into the cryo-EM density map of IBV spike using UCSF Chimera [41] and Coot [42]. Manual model rebuilding was carried out using Coot based on the well-defined continuous density of the main chain; the side chain assignments were guided by the densities of N-linked glycans and bulky amino acid residues. The structural model of the IBV spike in the pre-fusion state was refined using Phenix [43] with geometry restrains and three-fold noncrystallographic symmetry constraints. Refinement and model rebuilding in Coot were carried out iteratively until there were no further improvements in geometry parameters and model-map correlation coefficient. The quality of the final model was analyzed with MolProbity [44] and EMRinger [45]. The validation statistics of the structural models are summarized in S1 Table.\nIBV pseudovirus entry assay\nIBV pseudovirus entry assay was carried out as previously described [46]. Briefly, full-length IBV spike gene was inserted into pcDNA3.1 (+) plasmid. Retroviruses pseudotyped with IBV spike and expressing a luciferase reporter gene were prepared through co-transfecting HEK293T cells (source: American Type Culture Collection) with a plasmid carrying Env-defective, luciferase-expressing HIV-1 genome (pNL4-3.luc.RE) and the plasmid encoding IBV spike. The produced IBV pseudoviruses were harvested 72 hours post transfection, and then used to enter DF-1 cells (source: American Type Culture Collection) and HEK293T cells. After incubation for 5 hours at 37&#176;C, the medium was changed and cells were incubated for an additional 60 hours. Cells were then washed with PBS and lysed. Aliquots of cell lysates were transferred to Optiplate-96 (PerkinElmer), followed by addition of luciferase substrate. Relative light units (RLUs) were measured using EnSpire plate reader (PerkinElmer). All the measurements were carried out in quadruplicates.\nFlow cytometry cell-binding assay\nRecombinant IBV S1-CTD was assayed for its cell-binding capability using flow cytometry as previously described [13]. Briefly, HEK293T and DF-1 cells were incubated with recombinant IBV S1-CTD containing a C-terminal His6 tag (40 &#956;g/ml) at room temperature for 30 minutes, followed by incubation with phycoerythrin (PE)-labeled anti-His6 antibody for 30 minutes. The cells were then analyzed for the binding of IBV S1-CTD using flow cytometry.\nCalculation of buried surface area of coronavirus S1-CTDs\nThe total surface area and buried surface area of coronavirus S1-CTDs were calculated using the PISA server at the European Bioinformatics Institute (http://www.ebi.ac.uk/pdbe/prot_int/pistart.html) [47]. Specifically, for each trimeric spike protein, a PDB file containing all of the six S1 domains (including three copies of S1-CTDs and three copies of S1-NTDs) was submitted to the PISA server, and the total surface area and buried surface area for each S1-CTD were calculated. For the spike proteins used for the above analysis, all their S1-CTDs were in the &#8220;lysing down&#8221; state. The structures of MERS-CoV and HKU1 spikes were not included in the above analysis because the former contain at least one S1-CTD in the &#8220;standing up&#8221; state and the latter contains long stretches of missing residues in its S1 domains, both of which would interfere with the above analysis.\nResults\n\nOverall structure of IBV spike\nWe constructed the IBV spike ectodomain (from IBV strain M41) in the pre-fusion state by replacing its transmembrane anchor and intracellular domain with a C-terminal GCN4 trimerization tag, followed by a His6 tag (Fig 1A). We expressed the protein in insect cells and purified the protein to homogeneity. We collected cryo-EM data on IBV spike ectodomain, calculated a density map at 3.93&#197; resolution (Fig 1B; S1 Fig), built an atomic model of the structure and refined it (Fig 1C and 1D). The final structural model contains all of the residues from 21 to 1022 (except residues 702&#8211;710) as well as glycans N-linked to 20 sites. Data collection and model statistics are shown in S1 Table.The overall structure of IBV spike ectodomain resembles the pre-fusion structures of coronavirus spikes from the &#945;-, &#946;-, and &#948;-genera [12&#8211;14, 18&#8211;20]. It has a clove-like shape, with three S1 heads forming a crown-like structure and sitting on top of a trimeric S2 stalk. Each monomeric subunit of S1 contains two major domains, S1-NTD and S1-CTD, and two subdomains, SD1 and SD2 (Fig 2A and 2B). The S1-CTDs from three different subunits sit on the top and center of the spike trimer, whereas the three S1-NTDs are located on the lower and outer side to S1-CTDs (Fig 2C). SD1 and SD2 connect S1 to S2. The interface of trimeric S2 contains three central helices; each subunit of S2 contains one (Fig 2D and 2E). Each subunit of S2 also contains two heptad repeat regions, HR1 and HR2, and a fusion peptide (FP) (Fig 2D and 2E). In the post-fusion structure of trimeric S2, three copies of HR1 and three copies of HR2 would refold into a six-helix bundle structure, and FP would insert into the target membrane [29&#8211;35]. As in the structures of other coronavirus spikes, the HR2 region (residues 1022&#8211;1076) in the pre-fusion IBV spike is disordered (Figs 1A and 2D). The exact residue range of coronavirus FP remains unknown, although biochemical studies have identified a region in coronavirus S2 that associates with membranes and likely corresponds to FP (Fig 2D) [48, 49]. In the following sections of this paper, we will compare the structures and functions of IBV spike to those of the spikes from the other three genera, and discuss the evolution of coronavirus spikes.\nStructural and functional evolution of coronavirus spike S1-NTDs\nIBV S1-NTD takes the same galectin fold as the S1-NTDs from the other three coronavirus genera, but it contains unique structural features (Fig 3A&#8211;3D). Its core structure is a twelve-stranded &#946;-sandwich, which consists of two six-stranded antiparallel &#946;-sheet layers stacked together through hydrophobic interactions (Figs 2B and 3C). The topology of the &#946;-sandwich core is identical to that of human galectins (S2 Fig). Underneath the core structure is another &#946;-sheet and an &#945;-helix, which are also present in the S1-NTDs from the other three coronavirus genera. Above the core structure are some loops that form a partial ceiling-like structure. This structure is not present in human galectins or S1-NTDs from &#945;- or &#948;-genus, but becomes a more extensive ceiling-like structure in &#946;-coronavirus S1-NTDs (Fig 3A, 3B and 3D). Based on the structure and function of &#946;-coronavirus S1-NTDs, we previously predicted that S1-NTDs from all of the genera have a galectin fold, and further correlated the galectin fold to their functions as viral lectins [15]. Recent structural studies, including the current one, have confirmed our previous structural predictions (S2 Fig). These studies also have unexpectedly revealed that the presence and extent of the ceiling-like structure on top of the core structure are unique structural features for S1-NTDs from different genera.It has been known that IBV spike binds sugar [50]. A recent study further confirmed that the sugar-binding domain in IBV spike is its S1-NTD [51]. To date no structural information is available for the complexes of coronavirus S1-NTDs and their sugar ligand. Mutagenesis study showed that in the S1-NTD from &#946;-genus bovine coronavirus (BCoV), the sugar-binding site is located in the pocket formed between the core structure and the ceiling [24]. In the structure of host galectins, despite no ceiling, the sugar-binding site is located in the same place [52]. Based on the structural similarity between the S1-NTDs from different coronavirus genera, the sugar-binding site in IBV S1-NTD might also be located in the pocket formed between the core structure and the partial ceiling (Figs 2B and 3C).The structure determination of IBV S1-NTD provides insight into the structural and functional evolution of coronavirus S1-NTDs. We hypothesized that coronavirus S1-NTDs originated from host galectins based on the structural similarities between coronavirus S1-NTDs and host galectins [23, 24]. As host proteins, galectins are not recognized by the host immune system. In comparison, coronavirus S1-NTDs are under the host immune pressure to evolve. The gradual structural evolution of the ceiling on top of the core structure may have led to three functional outcomes. First, the ceiling could provide better protection to the sugar-binding site from host immune surveillance, which appears to be a common feature of viral lectins [53]. This hypothesis on protected sugar-binding sites in viral lectins is also consistent with the &#8220;canyon hypothesis&#8221; which states that receptor-binding sites on viral surfaces are hidden from the host immune surveillance [54]. Second, the ceiling is also involved in the quaternary packing of S1, which will be discussed later in this paper. Third, in the structure of S1-NTD from &#946;-genus mouse hepatitis coronavirus (MHV), the outer surface of the ceiling has further evolved the capability to bind a protein receptor CEACAM1, while the presumed sugar-binding pocket has lost its capability to bind sugar [23]. Hence, the structural development of the ceiling is a possible indicator for the evolution of S1-NTDs in the direction of &#945;- and &#948;-genera, then the &#947;-genus, and finally the &#946;-genus. Furthermore, we performed quantitative structural comparisons of S1-NTDs from different genera by calculating the Z-score and r.m.s.d. between each pair of the proteins (Fig 3E). The result confirmed that S1-NTDs are relatively conserved among different genera, as reflected by the generally high Z-scores and low r.m.s.d. In terms of structural distances to &#945;-coronavirus S1-NTDs, &#948;-coronavirus S1-NTDs are the closest, &#946;-coronavirus S1-NTDs are the farthest, and &#947;-coronavirus S1-NTDs fall in the middle. Moreover, the structural similarity between &#945;- and &#948;-coronavirus S1-NTDs is slightly higher than that between two &#946;-coronavirus S1-NTDs, suggesting that S1-NTDs within &#946;-genus have diverged slightly more than those between the &#945;- and &#948;-genera. Taken together, S1-NTDs from the four genera form an evolutionary spectrum in the order of &#945;-, &#948;-, &#947;-, and &#946;-genus, with &#945;-coronavirus S1-NTDs probably being the most ancestral (Fig 3A&#8211;3D).\nStructural and functional evolution of coronavirus spike S1-CTDs\nThe structure of IBV S1-CTD is significantly different from the structures of S1-CTDs from the other genera (Fig 4A&#8211;4D; S3 Fig). Its core structure is a &#946;-sandwich containing two &#946;-sheet layers: one is five-stranded and antiparallel, and the other is two-stranded and parallel (Figs 2B and 4C; S3 Fig). The interactions between the two &#946;-sheet layers are present but minimal. In contrast, the core structures of &#945;-coronavirus and &#948;-coronavirus S1-NTDs are both standard &#946;-sandwich folds with extensive interactions between the two &#946;-sheet layers: one is three-stranded and antiparallel, and the other is three-stranded and mixed (Fig 4A and 4B; S3 Fig). Even more drastically different are the &#946;-coronavirus S1-CTDs, which contain only one five-stranded antiparallel &#946;-sheet layer with the other layer turning into an &#945;-helix and a coil (Fig 4D; S3 Fig). Despite these dramatic structural differences, the S1-CTDs from all genera share the same structural topology (i.e., connectivity of secondary structural elements) (S3 Fig). Moreover, the additional structural motifs on the edge of the core structure are also diverse among different genera (S3 Fig). In the IBV S1-CTD, two extended loops on the edge of the core structure function as putative receptor-binding motifs (RBMs) by potentially binding to an unknown receptor (see below) (Figs 2B and 4C). In both the &#945;- and &#948;-coronavirus S1-CTDs, three short discontinuous loops are located in the same spatial region; they function as the RBMs in &#945;-coronavirus S1-CTDs and putative RBMs in &#948;-coronavirus S1-CTDs (Fig 4A and 4B). In &#946;-coronavirus S1-CTDs, a long continuous subdomain is located in this spatial region and functions as the lone RBM (Fig 4D). Structural variations of the RBMs in the S1-CTDs within each of the genera further lead to different receptor specificities [7]. In sum, IBV S1-CTD contains a weakened &#946;-sandwich core structure and two extended RBM loops; the former structural feature falls between the &#946;-sandwich cores of &#945;- and &#948;-genera and the &#946;-sheet core of &#946;-genus, whereas the latter structural feature falls between the three short discontinuous RBM loops of &#945;- and &#948;-genera and a single long continuous RBM subdomain of &#946;-genus.To investigate the function of IBV S1-CTD, we performed two assays to detect possible interactions between IBV S1-CTD and its potential receptor on the host cell surface. First, we carried out an IBV-spike-mediated pseudovirus entry assay in the presence of recombinant IBV S1&#8211;CTD (S4A Fig). To this end, retroviruses pseudotyped with IBV spike (i.e., IBV pseudoviruses) were used to enter host cells. In the absence of recombinant IBV S1-CTD, IBV pseudoviruses entered DF-1 cells (chicken fibroblast) efficiently, which was consistent with a previous report showing that DF-1 cells are permissive to live IBV (strain M41) infections [55]. As a negative control, their entry into HEK293T cells (human kidney) was inefficient. Recombinant IBV S1-CTD reduced the efficiency of IBV pseudovirus entry into DF-1 cells in a dose-dependent manner, likely because it competed with IBV pseudoviruses for an unknown receptor on the host cell surface. Second, we examined the binding of recombinant IBV S1-CTD to the host cell surface using a flow cytometry assay (S4B Fig). To this end, recombinant IBV S1-CTD was incubated with DF-1 cells, and subsequently cell-bound S1-CTD was detected using flow cytometry. Recombinant IBV S1-CTD bound to the surface of DF-1 cells efficiently, but not the surface of HEK293T cells. Taken together, IBV S1-CTD binds to a yet-to-be-identified receptor on the surface of chicken cells and hence functions as a receptor-binding domain (RBD). Thus, the S1-NTD and S1-CTD of IBV spike may both function as RBDs. Because coronavirus S1-CTDs from the &#945;- and &#946;-genera all use the additional structural features on the edge of their core structure as their RBMs, it is likely that the two extended loops in the same spatial region in IBV S1-CTD function as the RBMs.Coronavirus S1-CTDs represent remarkable examples of divergent evolution of viral proteins. The core structures and the RBM regions of S1-CTDs are both divergent among different genera (Fig 4A&#8211;4D; S3 Fig). The core structures are &#946;-sandwiches for &#945;- and &#948;-coronavirus S1-CTDs, weakened &#946;-sandwiches for &#947;-coronavirus S1-CTDs, and single &#946;-sheet layer for &#946;-coronavirus S1-CTDs. The RBMs are three short discontinuous loops for &#945;- and &#948;-coronavirus S1-CTDs, two reinforced loops for &#947;-coronavirus S1-CTDs, and a single continuous subdomain for &#946;-coronavirus S1-CTDs. Hence the S1-CTDs form an evolutionary spectrum, with &#945;- and &#948;-coronavirus S1-CTDs on one end, &#946;-coronavirus S1-CTDs on the other end, and &#947;-coronavirus S1-CTDs in between. We performed quantitative structural comparisons of S1-CTDs from all four genera (Fig 4E). The result confirmed that S1-CTDs are relatively poorly conserved among different genera, as reflected by the generally low Z-scores and high r.m.s.d. In terms of structural distances to &#945;-coronavirus S1-CTDs, &#948;-coronavirus S1-CTDs are the closest, &#946;-coronavirus S1-CTDs are the farthest, and &#947;-coronavirus S1-CTDs fall in the middle. The functional outcomes of the core structure evolution are not clear, but the evolution of the RBMs may have led to the following two functional outcomes. First, the diversity of the RBMs from three short loops to two extended loops and then to a long subdomain may allow coronaviruses to explore a wider variety of receptors. Second, the reinforced RBM regions in both &#946;- and &#947;-coronavirus S1-NTDs facilitate quaternary packing of S1, which will be discussed later in this paper. Taken together, the S1-CTDs from different genera form an evolutionary spectrum in the order of &#945;-, &#948;-, &#947;-, and &#946;-genus, although the evolutionary direction could go either way (Fig 4A&#8211;4D).\nEvolution of quaternary packing of coronavirus S1\nCuriously, coronavirus S1 from different genera take two types of quaternary packing modes (Fig 5A&#8211;5D) [12&#8211;14, 18&#8211;20]. IBV S1 takes a cross-subunit quaternary packing mode where the S1-NTD and S1-CTD from different subunits pack together (Fig 5C). Specifically, in the trimeric IBV spike, one S1-CTD packs against two S1-CTDs from the other subunits as well as one S1-NTD from another subunit. The putative RBMs of IBV S1-CTD and the partial ceiling of IBV S1-NTD are both involved in the cross-subunit packing. As a result, the putative RBMs of IBV S1-CTD are partially concealed, disallowing their full access to the host receptor. Hence IBV S1-CTD in the current structure was captured in a &#8220;lying down&#8221; state, and would need to &#8220;stand up&#8221; on the spike trimer for efficient receptor binding. This potential conformational change of IBV S1 can minimize the exposure of the putative RBMs in its S1-CTD to the immune system, thereby functioning as a possible strategy for viral immune evasion. &#946;-coronavirus S1 also takes the cross-subunit packing mode, with the RBM of its S1-CTD and the ceiling of its S1-NTD both involved in the cross-subunit packing (Fig 5D) [18&#8211;20]. In contrast, &#945;- and &#948;-coronavirus S1 both take an intra-subunit packing mode where the S1-NTD and S1-CTD from the same subunit pack together (Fig 5A and 5B) [12&#8211;14]. The RBMs of &#945;- and &#948;-coronavirus S1-CTDs are involved in the intra-subunit packing. Whether S1 packs in the intra-subunit or cross-subunit mode, the RBMs of S1-CTDs are concealed or partially concealed in their &#8220;lying down&#8221; state, and would need to switch to the &#8220;standing up&#8221; state for receptor binding. Overall, &#946;- and &#947;-coronavirus S1 both take the cross-subunit quaternary packing mode, whereas &#945;- and &#948;-coronavirus S1 both take the intra-subunit quaternary packing mode.We examined whether the quaternary structures of coronavirus S1 can lead to functional differences of coronavirus spikes. First, in both &#946;- and &#947;-coronavirus spikes, the RBMs of their S1-CTDs and the ceiling of their S1-NTDs have evolved to facilitate the cross-subunit packing. These additional structural features further evolved to gain other functions: the RBMs of S1-CTDs recognize diverse protein receptors, whereas the ceiling of the S1-NTDs either protects the sugar-binding site or recognizes a new protein receptor [7]. Second, to investigate the structural restrain on S1-CTDs that may hinder their potential conformational change, we calculated the total and buried surface areas of the S1-CTD on the spikes from different genera. The result did not reveal systematic difference between intra-subunit S1 packing and cross-subunit S1 packing in the buried surface area of S1-CTDs. However, it is worth noting that the S1-CTD from &#946;-genus SARS-CoV has the smallest buried surface area (in both absolute value and percentage) (S2 Table). The relative small buried surface area of SARS-CoV S1-CTD indicates less structural restraint on the S1-CTD from other parts of the spike S1, possibly allowing the S1-CTD to switch to the &#8220;standing up&#8221; and receptor-accessible conformation more easily. The &#8220;standing up&#8221; conformation of SARS-CoV S1-CTD may also weaken the structural restraint of S1 on S2 (discussed in more detail later), potentially allowing membrane fusion to proceed more easily [56]. Indeed, frequent &#8220;standing up&#8221; of SARS-CoV S1-CTD has been observed [19]. Overall, compared to the intra-subunit quaternary packing of &#945;- and &#948;-coronavirus S1, the cross-subunit quaternary packing of &#946;- and &#947;-coronavirus S1 may have allowed their S1 to evolve additional functions in receptor recognition; moreover, the S1-CTD from &#946;-genus SARS-CoV spike has a relatively small buried surface area, which may be responsible for its dynamic receptor-binding conformation.\nStructural and functional evolution of coronavirus spike S2\nThe structure and function of IBV S2 are highly similar to those of S2 from the other coronavirus genera. In the pre-fusion structure of IBV S2, HR2 is disordered, whereas HR1 and FP each consist of several &#945;-helices and connecting loops (the exact residue range of FP is not clear) (Fig 2D). In the post-fusion structure, HR1 would refold into a long &#945;-helix, HR2 would refold into a mixture of &#945;-helices and coils, three copies of HR1 and HR2 would pack into a six-helix bundle structure, and FP would also refold and insert into the target membrane (S5A Fig) [6, 29]. IBV S2 is locked in the pre-fusion state because of the structural restraint from S1. Specifically, because of the cross-subunit quaternary packing of trimeric IBV S1, HR1 and FP of IBV S2 are structurally restrained by two S1-CTDs from the other subunits and SD1 from another subunit, respectively (S5B Fig). The structural restraints from S1 on S2 can be weakened by the standing up of S1-CTDs (which allows receptor binding) and can be lifted completely upon proteolysis removal of S1. The packing between S1 and S2 in IBV spike is the same as those in &#946;-coronavirus spikes [18&#8211;20]. However, in &#945;- and &#948;-coronavirus spikes, the packing between S1 and S2 becomes different due to the intra-subunit quaternary packing of their trimeric S1: HR1 and FP are restrained by one S1-CTD and one SD1 from another subunit, respectively (S5C Fig) [12&#8211;14]. Other than the differences in S1/S2 packing, the structural and functional similarities of coronavirus S2 from different genera suggest evolutionary conservation of coronavirus S2.\nDiscussion\nThe fast evolutionary rates of viruses, particularly RNA viruses, make it difficult to trace their evolutionary history [1&#8211;3]. Envelope-anchored coronavirus spike proteins guide viral entry into cells; they are the fastest evolving coronavirus proteins due to viral needs to engage diverse host receptors, maximize membrane-fusion efficiency, and evade host immune surveillance [7&#8211;14]. Coronavirus spikes from four different genera are divergent, and their evolutionary relationships pose a major puzzle in the virology field [6]. Because viral proteins need to function under certain structural and functional constraints, evolutionary information of viral proteins can be more reliably found in their tertiary structures and related functions than in their primary structures [16, 17]. Although extensive structural studies including both X-ray crystallography and cryo-EM have been done on coronavirus spikes, a critical piece that was still missing is the structure of &#947;-coronavirus spikes, preventing a clear understanding of the evolutionary relationships among coronavirus spikes [12&#8211;14, 18&#8211;28]. In this study, we determined the cryo-EM structure of IBV spike ectodomain, the first such structure from the &#947;-genus, which bridges the divergent structures of coronavirus spikes into an evolutionary spectrum and provides insight into the evolutionary relationships among coronavirus spikes.Our study compares the structures and functions of coronavirus spikes from the four genera, and illustrates the structural and functional evolution of these proteins. First, coronavirus S1-NTDs from all genera share the same structural fold and possibly evolutionary origins with host galectins. From &#945;- and &#948;- genera to &#947; genus and then to &#946; genus, the S1-NTDs have evolved from simple galectin-fold core structure with an exposed sugar-binding site, to having a partial ceiling on top of the core structure, and to having an extensive ceiling to protect the sugar-binding site from host immune surveillance (the outer surface of the ceiling in one &#946;-coronavirus can even bind to a novel protein receptor). The partial ceiling in &#947;-coronavirus S1-NTDs and the extensive ceiling in &#946;-coronavirus S1-NTDs are also involved in the quaternary packing of S1. Second, coronavirus S1-CTDs from different genera are very diverse, but still form an evolutionary spectrum with &#945;- and &#946;-coronavirus S1-CTDs at two ends and &#948;- and &#947;-coronavirus S1-CTDs in the middle. The core structures of S1-CTDs have diverged from &#946;-sandwich to weakened &#946;-sandwich and then to &#946;-sheet, whereas the RBMs have diverged from short loops to extended loops and then to a long subdomain. The functional significance of the core structure evolution is not clear, but the RBM evolution may allow the viruses to expand receptor recognition and also contributes to the quaternary packing of S1. Third, from &#945;- and &#948;- genera to &#946;- and &#947;-genera, the quaternary packing of S1 has diverged from simple intra-subunit packing to more complex cross-subunit packing. The cross-subunit quaternary packing of &#946;- and &#947;-coronavirus S1 may have allowed their S1 to evolve additional functions in receptor recognition. Moreover, the relatively small buried surface area of the S1-CTD from &#946;-genus SARS-CoV may allow the S1-CTD to be more dynamic for receptor binding. Finally, the S2 from all four genera are structurally and functionally conserved, although there are some differences in their S1/S2 packing. Quantitative structural comparisons also demonstrate that in terms of structural distances to &#945;-coronavirus S1, &#948;-coronavirus S1 is the closest, &#946;-coronavirus S1 is the farthest, and &#947;-coronavirus S1 is the intermediate. We also calculated the phylogenetic tree using the amino acid sequences of 29 coronavirus spikes from different genera, and the result showed that in terms of amino acid sequence distances to &#945;-coronavirus spikes, &#948;-coronavirus spike is the closest, &#946;-coronavirus spike is the farthest, and &#947;-coronavirus spike is the intermediate (S6 Fig). Taken together, coronavirus spikes from different genera form an evolutionary spectrum, with &#945;-coronavirus spikes on one end, followed by &#948;-coronavirus spikes and &#947;-coronavirus spikes, and &#946;-coronavirus spikes on the other end.Because of their fast evolutionary rates, viruses are perfect model systems for studying evolution. Our study has demonstrated that despite structural divergence among coronavirus spikes, particularly in their S1 where low or little structural similarities can be detected, we can still trace the evolutionary relationships among these viral proteins through detailed analyses of their structures and functions. Our study also reveals that coronavirus spikes have evolved to remarkable diversity to expand their receptor recognition, facilitate membrane fusion, and evade host immune surveillance, while conserving basic membrane-fusion mechanisms. The evolutionary approaches that coronaviruses take and the evolutionary edges that they gain are good examples of viral evolution and deepen our understanding of evolution in general.\nSupporting information\n",
        "pmid": "29684066",
        "title": "Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins",
        "journal_title": "PLoS Pathogens",
        "authors": "Jian Shang;Yuan Zheng;Yang Yang;Chang Liu;Qibin Geng;Chuming Luo;Wei Zhang;Fang Li",
        "doi": "10.1371/journal.ppat.1007009"
    },
    {
        "keywords": "Middle East respiratory syndrome coronavirus;severe acute respiratory syndrome coronavirus;coronavirus spike glycoprotein;vaccine platforms;correlates of immunity;animal models",
        "abstract": "Efforts towards developing a vaccine for Middle East respiratory syndrome coronavirus (MERS-CoV) have yielded promising results. Utilizing a variety of platforms, several vaccine approaches have shown efficacy in animal models and begun to enter clinical trials. In this review, we summarize the current progress towards a MERS-CoV vaccine and highlight potential roadblocks identified from previous attempts to generate coronavirus vaccines.",
        "body": "1. Introduction\nIn recent years, viral zoonotic diseases have caused outbreaks marked by rapid spread and high mortality, including the 2002 emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) [1], the 2009 H1N1 swine flu pandemic [2], and the 2013 Ebola outbreak in West Africa [3]. Such outbreaks are difficult to predict as new strains emerge or reemerge from zoonotic reservoirs [4]. Coronaviruses (CoVs), large positive-stranded RNA viruses of the order Nidovirales [5], were considered minor human pathogens, causing cold-like symptoms and occasionally associated with pneumonia and more severe disease [6]. However, the emergence of SARS-CoV in 2002 and Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012, members of &#946;CoV lineages B and C, respectively, marked a shift in our understanding of the pathogenic potential of coronaviruses [7]. As these more virulent viruses are genetically similar to those currently circulating in bats [8,9], CoVs may pose a threat for future zoonoses [10].Since the emergence of MERS-CoV in Saudi Arabia in 2012, over 2200 confirmed cases have been reported in at least 27 countries, with an overall mortality rate of 35% (https://www.who.int/emergencies/mers-cov). Additionally, more severe disease has been noted in the aged, immunocompromised, and those with chronic health conditions [11]. Camels, which show seropositivity to MERS-CoV in archived sera dating back to 1983 [12], serve as intermediate hosts and are able to spread the virus to humans [13], who may then spread the infection person-to-person [14]. While a range of therapeutics have been explored for CoV disease [15,16,17], a MERS-CoV vaccine remains the most scalable, cost-effective prophylactic measure. Currently, a vaccine for MERS-CoV is not available, although several candidates have been developed using a variety of approaches. Vaccine studies were initially hampered by a lack of small animal models of MERS-CoV disease [18]. While rodents possess homologues for dipeptidyl peptidase 4 (DPP4), the human receptor for MERS-CoV [19], rodent DPP4 homologues are incompatible with MERS-CoV infection [20,21,22]. However, several in vivo approaches have been developed to overcome these barriers and facilitate MERS-CoV vaccine testing in small animal models [23,24,25,26,27].Preclinical vaccine development for both SARS-CoV and MERS-CoV has largely aimed to stimulate a robust immune response against the viral envelope-protruding spike (S) glycoprotein [28,29], a class I fusion protein, and/or the nucleocapsid (N) protein [30,31]. MERS-CoV S is proteolytically cleaved by host furin [32] during maturation into an S1 domain responsible for binding to DPP4 as well as an S2 domain containing two heptad-repeat regions that facilitate membrane fusion (Figure 1). The S1 domain can be further divided into the N-terminal domain (NTD), or S1A, associated with binding sialic acid [33], and the receptor binding domain (RBD), comprising the majority of the C-terminal domain of S1. Cryo-electron microscopy studies have shown that the RBD is flexible and opens upward or away from the viral envelope in order to establish contact with DPP4, which may expose S2&#8217; [34], a second protease cleavage site within S2. Cleavage at S2&#8217; is necessary for membrane fusion upon viral entry [32]. The centrality of S to viral entry helps explain why antibodies that target it are potently neutralizing [35]. On the other hand, while CoV N proteins are abundantly expressed during infection [36], immunization with SARS-CoV N did not induce strongly neutralizing antibodies [37], likely because N is not displayed on the viral surface. However, N is more conserved than S within CoV lineage [38], and vaccination with SARS-CoV N was shown to induce cytotoxic T cell responses in mice [39]. Therefore, N may help induce cell-mediated immunity to CoV infection [40], as may S [41]. CD4+ and CD8+ T cell responses from recovered MERS-CoV patients were particularly strong towards N peptides [42]. However, vaccination with N-based immunogens may carry risks associated with Th2-related eosinophilic immune enhancement [43], as may S-based vaccines [44]. Notwithstanding, because of the protection afforded by the robust immune response it generates, S has been the target of most vaccine candidates for MERS-CoV. In this review, we summarize the current state of MERS-CoV vaccine candidates and also describe potential barriers to MERS-CoV vaccine efficacy that first surfaced during research on developing a SARS-CoV vaccine.\n2. Subunit Vaccines: Immunogenically Focused\nSubunit vaccines comprise one or more immunogenic components derived from a pathogen [47]. They have gained popularity in recent decades due to the relative ease of their production and their reduced risks in vivo compared to vaccine types that involve live virus, namely live attenuated vaccines, viral vector vaccines, and even improperly prepared inactivated vaccines. \n2.1. Receptor Binding Domain\nIt is known from studies of recovered SARS-CoV patients that antibodies generated against the receptor binding domain (RBD) are both long-lasting (>3 years) and neutralizing [48]. The RBD in the MERS-CoV S glycoprotein was initially mapped to a region spanning residues 358 to 662 [49,50]; antisera from RBD protein-immunized mice or rabbits protected against in vitro infection with MERS-CoV. In further studies exploring immune correlates of protection, intranasal administration of RBD protein induced S1-specific immunoglobulin (Ig) G1, IgG2a, IgG3, lung IgA, and neutralizing antibodies (NAb, that is, untyped antibodies shown to functionally inhibit free virus from infecting cells) [51], as well as cell-mediated responses as measured by IL-2 and IFN-&#947; production in antigen-stimulated CD4+ and CD8+ splenocytes [52]. In this and subsequent studies, the RBD protein was fused to a fragment crystallizable (Fc) region of human IgG1 to increase the in vivo half-life of the immunogen [53]. Profiling of the immunogenic region of the RBD indicated residues 377-588 bound with the highest affinity to soluble DPP4 and induced the highest NAb titers, when administered to both mice and rabbits [54]. This range-refined RBD protein vaccine has been stably expressed in a modified high-yield CHO cell line [55]. Purified and adjuvanted with AddaVaxTM (MF59-like), the RBD protein vaccine was protective when administered intramuscularly to transgenic mice expressing human DPP4 (hDPP4), with no evidence of immunological toxicity or eosinophilic immune enhancement. Since the S glycoprotein exists in trimeric form on the virion, a quality lost in shortened forms of the protein, a trimer of this RBD protein vaccine has been generated containing a foldon trimerization motif [56]. This RBD protein trimer has been shown to elicit long-lasting NAb and be protective in challenged hDPP4-transgenic mice [57]. Independently, a monomeric RBD protein vaccine has been developed and tested in rhesus macaques, where it reduced MERS-associated lung pathology and reduced viral loads when adjuvanted with alum and administered intramuscularly in a three-dose regimen prior to challenge [58]. Finally, RBD proteins encoding sequences from different strains of MERS-CoV have been shown to induce cross-neutralizing antibodies against divergent human and camel MERS-CoV strains as well as monoclonal antibody (mAb) escape mutants, confirming the promise of the RBD as a valid vaccine target [59]. \n2.2. Full-Length S\nTargeting the entire S glycoprotein has the advantage of including non-RBD neutralizing epitopes, including those in the more conserved domains. S protein &#8220;nanoparticles,&#8221; protein aggregates containing full-length S, have been proposed as a subunit vaccine, as nanoparticle vaccination adjuvanted with Matrix-M1TM elicited NAb in mice [60] protected adenovirally hDPP4-transduced mice [24] from MERS-CoV challenge [61]. Measurements of viral titer and viral RNA were near the limit of detection in these vaccinated mice. However, despite these promising results, antibody-dependent enhancement (ADE) of infection was previously noted in the context of vaccination with full-length SARS-CoV S protein vaccine [62]. While ADE has not been demonstrated with full length S from MERS-CoV, further studies must consider this as a potential issue.A variation of the full-length S glycoprotein vaccine is a trimer of S ectodomain (all but the transmembrane domain) conformationally locked in the prefusion state by the substitution of two proline residues in the S2 domain [34]. Work on related fusion proteins such as the F glycoprotein of respiratory syncytial virus has shown that stabilizing the glycoprotein in its prefusion state helps elicit a stronger neutralizing antibody response [63]. This &#8220;prefusion&#8221; S administered to mice elicited sera with greater neutralization activity against a panel of pseudoviruses bearing strain-specific variants of MERS-CoV S as compared to wild-type S. Since the S2 domain of CoVs is more conserved than the S1 domain [64], targeting epitopes in S2 may provide broader protection against different MERS-CoV strains and other lineage C &#946;CoVs.\n2.3. N-Terminal Domain\nRBD- and full-length S-based protein vaccines build upon prior vaccine efforts for SARS-CoV. In contrast, vaccines targeting the N-terminal domain of S1 (NTD) offer a novel target. The NTD of S1 does not contain the RBD for MERS-CoV; however, the NTD binds sialic acid and is key to infecting certain cell types [33]. Immunization with NTD protein protected against MERS-CoV challenge in adenovirally hDPP4-transduced mice, inducing cell-mediated responses in splenocytes (CD8+ IFN-&#947; production, CD4+ IL-2 production, and IL-17A production) as well as humoral responses (IgG and NAb), although NAb titer was lower compared to that of an RBD protein vaccine [65]. Overall, these results suggest that targeting S1 domains outside the RBD may be a viable strategy for MERS-CoV vaccines.\n3. DNA Vaccines: Efficient Protection\nDNA vaccines offer a rapid platform to design and deliver immunogenic proteins, typically encoded on plasmid vectors and injected into tissue with accompanying electroporation [66]. DNA vaccine administration via electroporation has been tested in clinical trials with immunogenicity comparable to other vaccine types and predominantly low-grade adverse events reported [67]. The in vivo expression of plasmid-encoded proteins recapitulates native post-translational modifications while maintaining the capacity to stimulate both humoral and cell-mediated immunity [68]. While concerns about the safety of DNA vaccines and their potential to integrate into host cell chromosomes were voiced early in their development [69], integration using various plasmids and inserts appears to be extremely rare [70]. Several DNA vaccines for MERS-CoV have been reported to date.\n3.1. Full-Length S\npVax1TM is a proprietary, optimized plasmid vaccine vector that has been developed as a MERS-CoV vaccine by encoding a consensus MERS-CoV S glycoprotein containing codon and other proprietary optimizations, as well as an IgE leader sequence to promote expression and mRNA export [71]. Intramuscular administration of this construct with electroporation induced antibodies with cross-MERS-CoV-strain neutralization and antigen-specific, polyfunctional T cell responses in rhesus macaques. These humoral and cell-mediated immune responses correlated with minimal lung pathology and reduced lung viral loads upon MERS-CoV challenge. The same study reported NAb induction in dromedary camels, which indicates that the vaccine could be used in zoonotic reservoirs. Building upon these preclinical results, the pVax1TM vaccine (GLS-5300) has completed a Phase I clinical trial (clinicaltrials.gov/ct2/show/NCT02670187).A second vaccine platform utilizes pVRC8400 [72,73], a plasmid vector engineered for high transgene expression and enhanced cell-mediated responses. A vaccine regimen consisting of intramuscular administration of MERS-CoV strain England1 full-length S encoded on pVRC8400, with electroporation, and an AlPO4-adjuvanted S1 protein booster, induced NAb in rhesus macaques up to 10 weeks following booster [74]. This vaccine resulted in lower lung pathology upon challenge with the MERS-CoV strain JordanN3. In the same study, the full-length S DNA/S1 protein vaccine induced higher NAb titer in mice than other prime/boost combinations involving constructs encoding either S1 or S with the transmembrane domain (TMD) deleted. Consistent with this finding, the study reported that mAbs induced against domains outside the RBD were able to neutralize MERS-CoV pseudovirus. Together, these results reiterated the immunogenic potential of non-RBD epitopes including those derived from S quaternary (trimer) structure, which could help generate immune responses able to minimize escape variants derived from immunization targeting either the RBD or the S1 domain alone.\n3.2. S1 Domain\nThe S1 domain from MERS-CoV strain Al-Hasa_15_2013 encoded on pcDNATM3.1(+), a proprietary plasmid from which pVax1TM is derived, has also been tested as a vaccine platform. This vaccine induced NAb in mice when given intramuscularly and antigen-specific cytokine production including CD4+ and CD8+ production of both IL-4 and IFN-&#947; in murine splenocytes [75]. In addition, the vaccine protected adenovirally hDPP4-transduced mice against challenge with MERS-CoV strain EMC/2012. Moreover, this S1 vaccine elicited more NAb than did full-length S in the same vector. This result was attributed to increased secretion of the S1 protein, which lacked a TMD, and greater uptake by antigen-presenting cells [76]. A separate group&#8217;s study comparing full-length S- and S1-encoding pcDNATM3.1 vaccines found that the S1 vaccine elicited a more balanced IgG2a/IgG1 ratio in mice compared to that elicited by full-length S [77] suggesting a balanced Th1/Th2 response [78]. Together, these results show that multiple plasmid vaccine vectors encoding either full-length S or the S1 domain induce adaptive immunity and protect against MERS-CoV challenge.\n4. Viral Vector Vaccines: Optimized Delivery\nViral vector vaccines contain one or more immunogenic proteins of the pathogen of interest in the context of an attenuated virus backbone. This approach takes advantage of cellular entry by the virus as well as adjuvantation from viral components, and induces both humoral and strong cell-mediated responses [79]. Early studies into viral vector vaccines for MERS-CoV built upon established platforms and have subsequently transitioned to newer viral vector approaches.Venezuelan equine encephalitis (VEE) virus replicon particles (VRPs), an alphavirus-based platform that replaces the VEE structural genes with a foreign transgene, has been shown to induce strong humoral and cellular immune responses [80,81]. A VRP encoding MERS-CoV S elicited NAb in both young and aged mice [38]. Additional studies with this vector have shown that immunization with an N protein-expressing VRP protected adenovirally hDPP4-transduced mice from MERS-CoV challenge in a CD4+ T cell- and IFN-&#947;-dependent manner [82]. Moreover, a specific N protein epitope was stimulatory in mice transgenic for human leukocyte antigen DR2 and DR3, highlighting the relevance of this epitope to human antigen recognition and to promoting cell-mediated immunity in humans.Modified vaccinia virus Ankara (MVA) [83,84], a well-established vaccine platform, has been developed to encode full-length MERS-CoV S. This vaccine induced NAb and CD8+ T cell responses in mice [85] and also protected against MERS-CoV-induced histopathology in adenovirally hDPP4-transduced mice before challenge [86]. Moreover, minimal inflammation and lymph node hyperplasia was observed at the site of injection [87]. This same MVA-MERS-CoV S vaccine injected intramuscularly into dromedary camels was shown to induce NAb and to limit excretion of infectious virus upon intranasal challenge with MERS-CoV [88]. A Phase I clinical trial is underway (clinicaltrials.gov/ct2/show/NCT03615911).Adenoviruses compose a third platform of viral vectors for MERS-CoV vaccines. Adenovirus-vectored vaccines have been tested in clinical trials for a wide variety of diseases, notably HIV [89]. However, their efficacy may be hampered by pre-existing immunity to prevalent adenovirus serotypes [90,91]. For example, pre-existing immunity to human adenovirus serotype 5 (Ad5) was shown to result in reduced CD8+ T cell responses against an Ad5-vectored transgene [92]. To take advantage of the shared respiratory route of infection of both MERS-CoV and adenovirus, Ad5-vectored full-length S and S1 vaccines have been developed [93]. These elicited antigen-specific IgG and NAb when administered intramuscularly to mice with subsequent intranasal boosting. Importantly, this study did not detect immunity against the Ad5 vector in dromedary camels, the intended vaccination population. Moreover, camel peripheral blood mononuclear cells and a camel-derived fibroblast cell line were able to be infected with Ad5. Another Ad5-MERS-CoV S vaccine has been separately developed, as has a human adenovirus type 41 (Ad41)-MERS-CoV S vaccine [94]. Adenovirus type 41 (Ad41) is an enteric pathogen with potential use as an orally administered vaccine [95]. Both of these vaccines, Ad5-MERS-CoV S and Ad41-MERS-CoV S, were reported to induce humoral responses when administered intragastrically in mice. In addition to humoral responses, they also induced long-lasting cell-mediated responses in the lung and spleen when administered intramuscularly. One final Ad5-based MERS-CoV immunization regimen has been reported [96]. Immunizing with Ad5-vectored S followed by boosting with S nanoparticles induced S-specific IgG, NAb, and both Th1 and Th2 cell-mediated responses in mice, and also protected adenovirally hDPP4-transduced mice from MERS-CoV challenge.To circumvent the seroprevalence of circulating human adenoviruses, chimpanzee adenoviruses have also been developed as viral vaccine vectors [97] and have entered clinical trials [98]. A MERS-CoV S-encoding vaccine based on a chimpanzee adenoviral vector (ChAdOx1) was shown to induce high levels of NAb and cell-mediated responses (CD8+ IFN-&#947;, TNF&#945;, and IL-17 production) in mice 4 weeks post-immunization [99]. This vaccine was constructed with a codon-optimized S glycoprotein sequence and the tissue plasminogen activator (tPA) gene leader sequence to promote secretion [100,101]. The ChAdOx1-MERS-CoV S vaccine protected against lethal challenge in a transgenic hDPP4 mouse model [102]. Based on previous work with the ChAdOx1 vector demonstrating its safety in humans, the ChAdOx1-MERS-CoV S vaccine is undergoing a Phase I clinical trial (clinicaltrials.gov/ct2/show/NCT03399578).Several additional viral vectors have been employed as MERS-CoV vaccines. Measles virus vector platforms have been developed over the past two decades [103]. A full-length or soluble form of S encoded in measles vaccine strain MVvac2 induced NAb, proliferation of T cells, S-specific IFN-&#947; production, and cytotoxic activity [104]. The vaccine also protected against MERS-CoV challenge in adenovirally hDPP4-transduced mice that were transgenic for a measles virus receptor. Further characterization of the T cell responses induced by this vaccine has been performed [105]. Of note, 5-fold higher numbers of reactive T cells were induced by vaccination with MVvac2-S than those induced by N protein using the same vector. Additionally, antigen-specific IFN-&#947; production by T cells could be induced in older mice (7 months old) at levels near those induced in younger mice (6&#8211;12 weeks old). Newcastle disease virus (NDV) has been explored as a vaccine vector as it infects the respiratory tract and can induce systemic and mucosal immunity in non-human primates [106]. An NVD vector expressing MERS-CoV S was shown to induce long-lasting (up 14 weeks post-immunization) NAb titers in camels [107]. The research group behind this study also examined vesicular stomatitis virus (VSV) [108] as a viral vector. S expressed from a VSV reverse genetics system was shown to incorporate onto the surface of virions rescued in cell culture. Purified vaccine was able to infect cells in an hDPP4-dependent manner, induced S-specific IgG and NAb in mice, and stimulated humoral and cell-mediated (IFN-&#947;-production) responses in rhesus macaques [109]. Similar to the VSV platform, a rabies virus (RABV) vector has been explored. Inspired by studies combining rabies and Ebola vaccine platforms [110], a &#946;-propiolactone-inactivated dual rabies/MERS vaccine has been proposed which incorporates the MERS-CoV S1 domain fused to rabies virus G protein on the RABV virion [111]. This vaccine elicited S-specific IgG and NAb and fully protected adenovirally hDPP4-transduced mice from MERS-CoV challenge. The VSV and RABV approaches described here are unique in that they encode S (or S1) in the vector genome and also display it on the virion surface.Finally, virus-like particles (VLPs), which comprise self-assembling immunogenic proteins, but no genome [112], have also been used as viral vectors. A baculovirus VLP containing S as well as MERS-CoV envelope and matrix proteins elicited RBD-specific IgG and IFN-&#947; responses in rhesus macaques [113]. A subsequent baculovirus VLP vaccine was developed that focused only on a fusion of the RBD from S and the immunogenic VP2 protein of canine parvovirus. This vaccine induced RBD-specific IgG, NAb, and cell-mediated responses including IFN-&#947;, IL-2, and IL-4 production in mice, and also activated dendritic cells in inguinal lymph nodes [114]. In summary, a variety of viral vector vaccines for MERS-CoV induce promising immune responses in animal models and often demonstrate protection from challenge.\n5. Live Attenuated and Inactivated Vaccines: Situationally Useful\nA final approach to developing MERS-CoV vaccines delivers the whole virus, either inactivated or live but attenuated. Both of these vaccine types resemble the original virus, preserving structural features and a full or nearly-full repertoire of immunogenic components. Inactivated viruses may contain structural deformations introduced by inactivation, but, unlike attenuated viruses, they pose no risks, if properly inactivated, either of reversion to a virulent state or persistent infection in immunocompromised patients. Fewer examples of whole virus vaccines, compared to the other vaccine types, have been developed for MERS-CoV.\n5.1. Inactivated\nDevelopment of inactivated vaccines for MERS-CoV has been stymied by prior concerns with SARS-CoV inactivated vaccines. Eosinophil-related lung pathology was observed for a SARS-CoV vaccine doubly inactivated with both formalin and UV irradiation [115]. This response was particularly notable in aged mice versus young mice, and following heterologous versus homologous challenge. Similarly, immunization with a gamma-irradiated MERS-CoV vaccine adjuvanted with either alum or MF59 elicited NAb and reduced viral titer upon challenge in hDPP4-transgenic mice, but induced eosinophil-related lung pathology in vaccinated mice after challenge [116].A different inactivation method was tried for a second MERS-CoV inactivated vaccine. Formalin-inactivated MERS-CoV adjuvanted with alum and oligodeoxynucleotides containing unmethylated CpG motifs was shown to elicit levels of NAb on par with those elicited by an S glycoprotein-only vaccine [117]. Moreover, the vaccine offered better protection than S alone based on reduction of lung viral titer in adenovirally hDPP4-transduced mice after MERS-CoV challenge. Remarkably, eosinophil-mediated vaccine-related pathology was not observed in this animal model. Interestingly, it has also been shown that including Toll-like receptor agonists in a UV-inactivated SARS-CoV vaccine reduced Th2-associated pathology in lungs after challenge [118]. These results suggest that inactivated CoV vaccines may remain viable options for further development with the right inactivation method and adjuvants.\n5.2. Live Attenuated\nLive attenuated vaccines for MERS-CoV show efficacy in animal models, but so far have not been pursued in subsequent studies. While riskier than other vaccine types, live attenuated vaccines have historically offered protection against a variety of threatening illnesses [119] and may be reserved for outbreak scenarios where they offer an immunogenically robust solution. The CoV envelope (E) protein is important in virion assembly and egress, and has also been shown to inhibit the host cell stress response [120]. An E-deletion mutant of SARS-CoV was previously found to be protective in vivo against SARS-CoV challenge [121]. An initial study into a MERS-CoV reverse genetics system reported on a replication-competent, but propagation-defective mutant lacking the E protein that could be rescued in cell culture with E expressed in trans [122]. However, the E-deletion mutant was rescued at 100-fold lower titer compared to wild-type MERS-CoV, perhaps explaining why this mutant has not been further developed as a live attenuated vaccine candidate. Other CoV components have been targeted in live attenuated vaccine development. Nonstructural protein 14 (nsp14) contains an exoribonuclease (ExoN) essential to replication fidelity that is found in all known nidoviruses with genome sizes greater than 20 kb [5]. A stable deficiency in nsp14 attenuated SARS-CoV in young, aged, and immunocompromised mice, and was able to induce protection following vaccination [123]. However, ExoN mutants for MERS-CoV have not been reported.CoV gene nonstructural protein 16 (nsp16) is a 2&#8242;-O-methyl-transferase involved in viral mRNA capping [124] and was previously inactivated in a SARS-CoV live attenuated vaccine [125,126]. A live nsp16-deficient MERS-CoV vaccine was similarly attenuated in a type I interferon- and IFIT1-dependent manner. Immunization with the MERS-CoV nsp16 mutant induced NAb and protected CRISPR-engineered hDPP4-transgenic mice [23] from challenge with a mouse-adapted MERS-CoV strain [127]. Finally, live attenuated vaccines lacking CoV accessory proteins have also been considered. CoV accessory proteins are dispensable for viral replication but have been shown to modulate interferon signaling and pro-inflammatory cytokine production [128]. A MERS-CoV strain lacking accessory open reading frames (ORFs) 3, 4, and 5 was attenuated in vivo, induced NAb, and like the nsp16 mutant, protected CRISPR-engineered hDPP4-transgenic mice from challenge with a mouse-adapted MERS-CoV strain [129]. Overall, whole vaccine approaches to MERS-CoV vaccination appear both protective and safe in animal models.\n6. Conclusions and Future Directions\nAided by knowledge gained from vaccine development against SARS-CoV and other contemporary viral diseases, MERS-CoV vaccine development efforts have multiplied since its emergence, yielding promising vaccine candidates spanning multiple platforms (Table 1). Nevertheless, key barriers to vaccine efficacy first noted for SARS-CoV may also hold true for MERS-CoV. As with SARS-CoV [130], mortality from MERS-CoV has disproportionately affected the aged. Additionally, immunocompromised individuals and those with chronic conditions are at greater risk of mortality from MERS-CoV infection [11]. A universal MERS-CoV vaccine must offer protection to these vulnerable classes of people. More studies on the effectiveness of the proposed MERS-CoV vaccines in models of immunosenescence, immunocompromise, and chronic conditions are needed. In this regard, vaccination studies with SARS-CoV have indicated that vaccines may be capable of inducing protection in young animals while failing to protect aged animals [115]. In light of the threat of related coronavirus strains emerging, MERS-CoV vaccine studies must also consider heterologous challenge models to ensure safety from vaccine-induced immunopathology, especially in older individuals [115]. In short, vaccine-induced immunopathology, especially Th2-related eosinophilic immune enhancement [131], from both homologous and heterologous challenge should be specifically monitored in vulnerable populations. Interestingly, it was shown that pathology resulting from MERS-CoV challenge in the lungs of immunosuppressed rhesus macaques was lower compared to that of non-immunosuppressed macaques, underscoring the immunopathogenic component of respiratory disease caused by CoVs [132]. The different vaccine platforms described herein have unique advantages and disadvantages. Since severe CoV disease maintains an immunopathogenic component, a successful vaccine must strike a balance between protection and excessive immune activation. As seen with full-length S [44] as well as inactivated virus [116], vaccination may produce immunopathology under certain conditions. Alternatively, protection must be thorough enough to prevent NAb escape, a phenomenon inversely correlated with the number of immunogenic epitopes. While antibodies induced against S of either SARS-CoV or MERS-CoV poorly cross-neutralize across their respective lineages [38], a vaccine that contains multiple immunogenic epitopes would perhaps also offer greater cross-protection against heterologous strains within lineage as they emerge, especially if conserved epitopes are included in vaccine design. A greater understanding of MERS-CoV pathology will also help guide future vaccine development efforts by illuminating possibly critical differences in vaccine responses between MERS-CoV and SARS-CoV, the latter of which has largely influenced vaccine development against the former.Overall, MERS-CoV vaccines have shown encouraging results in preclinical studies and we hope these vaccines stand up to safety considerations in order to proceed through clinical trials. While development of therapeutic treatment is critical, vaccination carries the promise of mitigating future outbreaks and alleviating disease burden from the most vulnerable populations including the aged, the immunosuppressed, healthcare workers, family members of infected patients, and those in endemic areas.",
        "pmid": "30658390",
        "title": "Middle East Respiratory Syndrome Vaccine Candidates: Cautious Optimism",
        "journal_title": "Viruses",
        "authors": "Craig Schindewolf;Vineet D. Menachery",
        "doi": "10.3390/v11010074"
    },
    {
        "keywords": "coronaviruses;cats;spike protein;ELISA;virus neutralization;cross-species transmission;cross-reaction",
        "abstract": "Coronaviruses (CoVs) are widespread among mammals and birds and known for their potential for cross-species transmission. In cats, infections with feline coronaviruses (FCoVs) are common. Several non-feline coronaviruses have been reported to infect feline cells as well as cats after experimental infection, supported by their ability to engage the feline receptor ortholog for cell entry. However, whether cats might become naturally infected with CoVs of other species is unknown. We analyzed coronavirus infections in cats by serological monitoring. In total 137 cat serum samples and 25 FCoV type 1 or type 2-specific antisera were screened for the presence of antibodies against the S1 receptor binding subunit of the CoV spike protein, which is immunogenic and possesses low amino acid sequence identity among coronavirus species. Seventy-eight sera were positive for antibodies that recognized one or more coronavirus S1s whereas 1 serum exclusively reacted with human coronavirus 229E (HCoV-229E) and two sera exclusively reacted with porcine delta coronavirus (PDCoV). We observed antigenic cross-reactivity between S1s of type 1 and type 2 FCoVs, and between FCoV type 1 and porcine epidemic diarrhea virus (PEDV). Domain mapping of antibody epitopes indicated the presence of conserved epitope(s) particularly in the CD domains of S1. The cross-reactivity of FCoV type 1 and PEDV was also observed at the level of virus neutralization. To conclude, we provide the first evidence of antigenic cross-reactivity among S1 proteins of coronaviruses, which should be considered in the development of serological diagnoses. In addition, the potential role of cats in cross-species transmission of coronaviruses cannot be excluded.",
        "body": "1. Introduction\nCoronaviruses (CoVs) are enveloped viruses with a positive-stranded RNA genome and classified into four genera (alpha-, beta-, gamma- and deltacoronavirus) within the subfamily Orthocoronavirinae in the family Coronaviridae of the order Nidovirales. CoVs are found in a variety of mammals and birds, in which they can cause respiratory, enteric and systemic infections [1,2,3]. Additionally, CoVs have proven ability for cross-species transmission, exemplified by the emergence of severe acute respiratory syndrome (SARS) coronavirus in 2002/2003, and of the Middle-East respiratory syndrome (MERS) coronavirus in 2012 [4]. Both viruses belong to the Betacoronavirus genus and have an animal origin. SARS coronavirus crossed over from bats via intermediate hosts to humans, became human-adapted and quickly spread worldwide before its containment. MERS coronavirus recurrently enters the human population via its dromedary camel reservoir host, with limited, non-sustained human-to-human transmission particularly in healthcare settings [5,6,7]. Apart from SARS- and MERS-CoV, all four globally endemic human CoVs (HCoV-OC43, HCoV-NL63, HCoV-229E and HCoV-HKU1) originate from animals [8,9,10,11]. In addition, cross-species transmission potential of CoVs is also illustrated by the occurrence of chimeric coronaviruses that resulted from recombination events between feline CoVs (FCoV) and canine CoVs (CCoV) [12,13].In order to get insight into the frequency of interspecies transmission of coronaviruses within and between animal and human populations and the risk of subsequent development of a pandemic, it is useful to screen for coronavirus infections in animal species; especially those that are in close contact with humans. Serological assays that can detect virus-specific antibody responses against infection play an important role in these epidemiological studies [14].Cats live in close contact with humans and often roam around freely in the environment. Hence cats are an interesting species to study for infections with coronaviruses. Infections with feline coronaviruses (FCoVs) are recognized and widespread [15,16]. FCoVs are classified into two types, type 1 and type 2, based on the genetic and antigenic difference of their spike (S) protein [17]. In the field, the majority of FCoV infections are caused by FCoV type 1, while FCoV type 2, derived from recombination events of type 1 FCoVs and CCoVs obtaining the S gene and some flanking regions of CCoVs, is less prevalent [18,19]. Depending on the virulence of the FCoV strain and the immune response of the cat, the clinical presentation can range from apparently asymptomatic, through diarrhea, to full-blown feline infectious peritonitis [20]. FCoVs are members of the genus alphacoronavirus, to which also HCoV-229E, porcine transmissible gastroenteritis virus (TGEV), and CCoV belong. The latter three viruses and FCoV type 2 have been proven to use feline aminopeptidase N (fAPN) as a functional receptor in vitro [21]. The receptor for type 1 FCoV has still not been identified [22]. Notably, previous studies have shown that HCoV-229E and CCoV could infect cats after experimental inoculation, causing an asymptomatic infection [23,24]. Thus, cats might potentially become naturally infected with CoVs of other species which may lead to virus-host adaptation e.g., mutation or recombination, resulting in emergence of novel coronaviruses and potentially new diseases [19,25]. The extent to which infections with CoVs of other species occur in the field, has not been explored in previous epidemiological studies of CoV infections in cats [15,26,27,28].Being the main envelope protein of coronaviruses, the spike (S) protein mediates cell attachment and membrane fusion to allow viral entry. S functions as the main determinant of cell-, organ- and host-tropism. Additionally, it is also the major target of neutralizing antibodies. Spike comprises two functionally interdependent subunits, S1 and S2, with S1 responsible for receptor binding and S2 for membrane fusion [3,29]. The S1 subunit is the least conserved and the most variable immunogenic antigen between coronavirus species [30]. Therefore, the S1 subunit is well suited as an antigen to screen for coronavirus type specific antibodies [31].In this study, CoVs infection in cats were detected through profiling antibody presence in serum samples from cats. Recombinant CoV spike S1 subunits of different animal and human CoVs were expressed in a mammalian expression system and used for screening of cat sera for the presence of antibodies against the respective proteins. Positive samples were also tested by virus neutralization assays to support the specificity of the reaction [32,33,34]. This investigation intends to extend our knowledge of CoV epidemiology, potential reservoirs, and cross-species transmission.\n2. Materials and Methods\n\n2.1. Serum Samples\nSpecific FCoV type 1 and FCoV type 2 sera were obtained from specific pathogen free (SPF) cats previously infected with strain UU2 or RM and FIPV-1146 respectively [35,36]. In addition, for the serological survey, 137 feline sera were retrieved from the serum bank in our lab. These had all been collected from cats in the Netherlands. Most of the samples (>80%) were from a study on antibody titer testing for feline panleukopenia virus. The other samples were send to our lab for FIP or FeLV-FIV diagnostics. Sera of uninfected SPF cats were included as negative controls. All samples were stored at &#8722;20 &#176;C until analysis.\n2.2. Cells and Viruses\nAfrican green monkey kidney cells (Vero-CCL81), human hepatoma cells (Huh7), pig kidney epithelial cells (LLC-PK1), human embryonic kidney 293 cells stably expressing the SV40 large T antigen (HEK-293T) were maintained in Dulbecco modified Eagle medium (DMEM, Lonza, Basel, Switzerland) supplemented with 10% fetal bovine serum (FBS, Bodinco, Alkmaar, The Netherlands). Virus strains used in this study have been described previously [37,38,39]. Briefly, recombinant porcine epidemic diarrhea virus (PEDV) (rPEDV-SDR13-GFP) was propagated and titrated in Vero cells, and HCoV-229E in Huh7 cells. PDCoV was propagated and titrated in LLC-PK1 cells, but supplemented with 1 &#956;g/mL TPCK-treated trypsin (Sigma-Aldrich, Inc., St Louis, MO, USA) in DMEM.\n2.3. Plasmids Constructs and Recombinant Protein Expression\nSynthetic sequences of 12 coronavirus spike S1 subunits (HCoV-HKU1 (GB: YP_173238.1), MERS-CoV (GB:YP_009047204.1), SARS-CoV (GB: AAX16192.1), HCoV-OC43 (GB: AAR01015.1), HCoV-229E (GB: NP_073551.1), HCoV-NL63 (GB: YP_003767.1), TGEV (GB: ABG89325.1), PEDV (GB: AOG30832.1), BCoV (GB: P15777.1), PDCoV (GB: AML40825.1), FCoV type 1 (GB: FJ938060.1), FCoV type 2 (GB: AY994055.1)) and different domains of PEDV S1 subunit (S10 and S1A-D, as identified and described in [40]) were cloned into pCAGGS expression plasmids as described previously [41]. Similarly, the expression constructs encoding chimeric proteins in which S1s were fused to the Fc domain of mouse IgG2a. For protein production, HEK-293T cells were transfected with plasmid DNA conjugated to polyethyleneimine (Polysciences, Inc., Warrington, PA, USA). At 8&#8211;16 h post transfection, inoculum was removed and the transfection mixture was replaced by 293 SFM II expression medium (Gibco&#174;, Life Technologies Inc., Grand Island, NY, USA). At 6&#8211;7 days post transfection, cell supernatants were harvested and proteins were collected by Protein A Sepharose beads (GE Healthcare Bio-Sciences AB, Uppsala, Sweden). Proteins were then eluted with 0.1 M citric acid, pH 3.0 and neutralized with 1 M Tris-HCl, pH 8.8. Concentrations of proteins were assessed by Nanodrop spectrophotometry (ThermoFisher Scientific Inc., Waltham, MA, USA) and confirmed by sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE) with bovine serum albumin (BSA, BioIVT, West Sussex, UK) as standard. Typical yields for proteins were 0.2&#8211;0.5 mg/mL. For long term storage, proteins were stored at &#8722;80 &#176;C upon usage.\n2.4. FCoV S Structure Modelling and S1 Domain Expression\nTo study the potential cross-reaction between FCoV type 1 and 2 in more detail, models of FCoV type 1 (strain: UU2; GenBank accession no.: FJ938060.1) and FCoV type 2 (strain: 79-1146; GenBank accession no.: AY994055.1) S proteins were generated via the automated protein structure SWISS-MODEL Homology Modelling server (https://swissmodel.expasy.org/) [42] using the elucidated HCoV-NL63 Cryo-EM structure (PDB code: 5SZS) as the input model. Figures were made with PymoL (The PyMOL Molecular Graphics System, Version 1.0 Schr&#246;dinger, LLC.). FCoV S1 domains of both type 1 and 2, namely S10-CD, were expressed as murine Fc fusion proteins in HEK-293T cells as described above.\n2.5. Enzyme-Linked Immunosorbent Assay (ELISA)\nHigh binding microtiter plates (Greiner Bio-one BV, Alphen aan den Rijn, The Netherlands) were coated overnight at 4 &#176;C with equal molar amount of protein (0.25 pmol per well, diluted in phosphate buffered saline (PBS, pH 7.4)). After three washes with washing buffer (PBS containing 0.05% Tween-20), the plates were blocked for 2 h at 37 &#176;C with blocking buffer (PBS containing 5% milk powder (Protifar, Nutricia, Zoetermeer, The Netherlands), 0.05% Tween-20). Protein coating efficiency was assessed by binding of anti-mouse IgG antibodies in a direct ELISA, and confirmed the equimolar coatings of all proteins. To detect antigenic reaction with serum samples, sera were tested in duplicate at a 1:200 dilution in blocking buffer, and then incubated in the plates at 37 &#176;C for 1 h. After washing, plates were incubated with a 1:4000 diluted horseradish peroxidase (HRP)-conjugated goat anti cat IgG (Rockland Immunochemicals, Inc., Pottstown, PA, USA) at 37 &#176;C for 1 h. The peroxidase reaction was then visualized via adding TMB Super Slow One Component HRP Microwell Substrate (BioFX&#174;, Surmodics IVD, Inc., Eden Prairie, MN, USA) for 10 min. Reaction was stopped with 12.5% sulfuric acid and optical densities (OD) were measured at 450 nm. Negative sera (from uninfected SPF cats) were included to determine the ELISA cut-off values; sera with OD values higher than 5-fold the OD of negative sera were considered positive. All 12 S1 proteins were coated on the same ELISA plates making it easy to screen and compare the OD values of individual sera in one assay. Hereby we excluded the sera that give high background OD values against all proteins being considered false positive.\n2.6. Virus Neutralization Assay\nNeutralization assays were performed with some of the CoVs to support the specificity of ELISA results. Cat sera were serially diluted 2-fold in DMEM and mixed 1:1 with rPEDV-SDR13-GFP, HCoV-229E or PDCoV (2000 50% tissue culture infective doses [TCID50]/mL). These mixtures were then incubated at 37 &#176;C for 1h, and 100 &#181;L of each mixture was used for inoculation with Vero, Huh7 and LLC-PK1 cell monolayers in 96-well plates, respectively. For PDCoV infection, TPCK-treated trypsin (Sigma-Aldrich, Inc., St Louis, MO, USA) was supplied to LLC-PK1 tissue culture medium at a final concentration of 1 &#956;g/mL. At 2&#8211;5 days post infection, cytopathic effect (CPE) could be observed via microscopy. Virus neutralization titers (VNT) were expressed as the highest serum dilution resulting in 90% reduction of cytopathic effect (HCoV-229E and PDCoV) or virus-induced fluorescent cells (PEDV). Before virus neutralization, sera were inactivated through incubation at 56 &#176;C for 30 min. Experiments were performed in triplicate. \n3. Results\n\n3.1. CoVs Seroprevalence in Cats\nFeline sera (n = 137) were screened by indirect ELISA for antibody reactivity against 12 CoV S1 antigens. The OD values against these 12 antigens are shown in Figure 1. In total, 78 of the 137 sera (56.9%) contained anti-CoV antibodies, while 43 sera showed reactivity against more than one CoV S1 antigen. None of the samples had to be discarded because of reactivity against all of the proteins indicating a potential false positive result. The frequency of different combinations of CoV-S1 reactive samples is summarized in Table 1. Reactivity against eight out of 12 CoV S1 antigens could be observed, whereas none of the sera recognized the S1 protein of HCoV-HKU1, MERS-CoV, SARS-CoV and HCoV-OC43.As expected, many sera were positive for FCoV S1, with 75 sera (54.7%) positive for FCoV type 1 and 26 sera (19.0%) for FCoV type 2 S1 (Figure 1). All of the FCoV type 2 S1 positive sera also tested positive for FCoV type 1 S1, while 15 of 26 FCoV type 2 S1 positive sera also reacted with TGEV S1. The FCoV type 2 S1 and TGEV S1 ELISA reactivities showed a strong nonparametric Spearman correlation (Spearman r = 0.84, p < 0.0001). With respect to this, we suggest that the TGEV S1 positivity was due to cross-reactivity of FCoV type 2 S1, as FCoV type 2 shows close antigenic and genetic relationship with TGEV (S1 shares 70.4% amino acid sequence identity). The remaining 11 FCoV type 2 S1 positive but TGEV S1 negative sera do react with FCoV type 1 S1. An explanation might be the cross-reactivity between FCoV type 1 and type 2. Remarkably, 40 feline sera were reactive with S1 proteins from human, porcine and bovine CoVs (Table 1), including HCoV-229E (16/137), HCoV-NL63 (2/137), PEDV (27/137), PDCoV (8/137) and BCoV (1/137). OD values of feline sera positive for HCoV-NL63 S1 and BCoV S1 were relatively low (Figure 1A). ELISA reactivity towards non-feline CoV S1 proteins might be explained by infection with the respective or related CoVs or by the presence of cross-reacting antibodies, although there was low sequence identity (< 32.8%) between S1 proteins of FCoV type 1 and related non-feline coronaviruses (for the complete comparison of S1 sequence identities, see Table 2). Yet, all of PEDV-S1 positive sera were also positive for FCoV type 1 S1 (Figure 1B, Table 1). The ELISA results of FCoV type 1 S1 and PEDV S1 showed a strong nonparametric Spearman correlation (Spearman r = 0.83, p < 0.0001). Thus, this might indicate the occurrence of antibody cross-reactivity against FCoV type 1 and PEDV S1 antigens. Many of the HCoV-229E and PDCoV S1 positive sera also reacted with FCoV type 1 S1, but no strong nonparametric Spearman correlation was observed (HCoV-229E, r = 0.216; PDCoV, r = 0.307). One feline serum only reacted with HCoV-229E S1, and two feline sera only recognized PDCoV S1. (Figure 1B, Table 1). This observation led us to hypothesize that cross-reactivity may not play a role in ELISA reactivity of these three sera, but that the three cats had been infected with these viruses or related viruses. \n3.2. Assessment of Cross-Reactivity for CoV S1s\nIn our screening, 43 samples were shown to be positive for two or more S1 proteins including FCoV type 1. The data prompted us to test different hypotheses which may explain this phenomenon: specific reaction through natural virus infection or reaction due to cross-reactivity with FCoV-S1 antigens. To explore this further, we employed 25 FCoV type 1 specific sera derived from specific pathogen free (SPF) cats that had been experimentally infected with FCoV type I strain RM (n = 9) or strain UU2 (n = 16). These sera were tested for their ELISA reactivity against seven CoV S1 proteins (excluding TGEV S1) that showed positive reactivity in the previous serological screening. As expected, all 25 sera were positive for FCoV type 1 S1 in our ELISA; interestingly, four samples also reacted with FCoV type 2 S1, and five samples with PEDV S1. No positive ELISA-reactivity was detected with S1 of HCoV-229E, PDCoV, HCoV-NL63 or BCoV (Table S1). Thus, FCoV type 1 infection could lead to the generation of antibodies that cross-react in the S1-ELISA with FCoV type 2 and PEDV S1 proteins. \n3.3. S1 Domain Mapping of Conserved Epitopes Shared between Type 1 and Type 2 Feline Coronaviruses \nThe ELISA cross-reactivity of FCoV type 1 specific sera with FCoV type 2 S1 antigens prompted us to map the domains responsible for cross-reaction within the S1 subunit. Hence, to identify domain borders within S1, we built homology-based models of both FCoV type 1 and type 2 spike using the related elucidated HCoV-NL63 cryo-EM structure as the template model. As shown in Figure 2A, continuous structural domains can be identified for the S1 subunit of both spikes, namely S10, and S1A through S1D. Amino acid sequence identities of these domains between FCoV type 1 and type 2 differ, ranging from 22.4% to 57.4% (Figure 2B). Several S1 proteins for both type 1 and type 2 FCoV-S1 comprising one or two domains were expressed and purified (Figure 2C). FCoV type 1 specific sera (n = 4 for strain RM and n = 3 for strain UU2) and type 2 (n = 6, strain 79-1146) specific sera were then tested against these proteins in ELISA format. The four FCoV type 1 specific sera that cross-reacted with S1 of FCoV type 2 again showed binding to FCoV type 2 S1. But the FCoV type 2 specific sera showed little to no reactivity against FCoV type 1 S1. As shown in Figure 3, the type specific antisera reacted with all of the homologous S1 domains, with S1 and S1B of both type 1 and type 2 displaying the strongest reaction. Interestingly, the CD domain showed the highest level of cross-reactivity between FCoV type 1 and 2, in agreement with its highest sequence identity among S1 domains (Figure 3). The other three domains showed little to no cross-reactivity.\n3.4. Assessment of Cross-Reactivity between FCoV Type 1 and PEDV \nBecause the FCoV type 1 specific cat sera also showed ELISA reactivity with PEDV-S1 (Table S1), we analyzed the reaction of the five PEDV-S1 positive cats in more detail. Samples were analyzed via ELISA using antigens comprising different PEDV-S1 domains, as described in our previous study [40]. Cat sera taken pre- and post- FCoV infection were collected and tested. As indicated in Figure 4, all five cats had developed PEDV-S1 reactivity to different extent after FCoV type 1 inoculation. Noticeably, all sera showed the highest OD values with the CD domain, while the other domains, including the S1B containing the presumed receptor binding domain (RBD) [40], were non-reactive (Figure 4). On the other hand, the swine PEDV positive control serum exhibits strong reactivity against all PEDV-S1 domains. The next question we asked was whether FCoV type 1 specific sera could neutralize PEDV infection in tissue culture, as they showed no reactivity with the S1B of PEDV spike. As shown in Figure 5, PEDV neutralizing antibodies were detected in three out of five FCoV type I specific cat sera. \n3.5. Evaluation of Virus Neutralization Capacity of HCoV-229E and PDCoV Positive Serum Samples\nSeveral serum samples from field cats, but not virus-specific serum samples from FCoV inoculated SPF cats, were found to be ELISA positive for HCoV-229E (n = 16) and PDCoV S1 (n = 8) (Figure 1A). Also, a few feline sera displayed unique ELISA positivity for S1 of HCoV-229E (n = 1) or PDCoV (n = 2) (Figure 1B, Table 1). This could indicate that these antibodies were induced upon infection with these specific viruses. To corroborate the possibility of a natural infection in these cats with HCoV-229E or HCoV-229E-like viruses, we tested sera neutralization antibody titers. The results showed that one of the HCoV-229E S1 reactive feline sera was able to neutralize HCoV-229E infection (VNT = 32); no neutralization of PDCoV was detected for the all PDCoV-S1 positive sera. \n4. Discussion \nCoronavirus infections are endemic and ubiquitous in feline populations. Two viral types, type 1 and 2, are distinguished and both of them could well sustain themselves in the cat reservoir [15,43]. Both have been shown to have worldwide distribution, with the seropositivity rate up to 90% among animal shelter populations and in multi-cat households [20,44]. The majority of natural infections are caused by type 1 FCoVs, while in the field type 2 FCoVs are less common and mainly occur in Asia [15,28,45,46,47]. CoVs are generally considered to be host-specific; however, cross-species transmission does occur which may lead to incidental infections like the spillover of MERS-CoV from dromedary camel to humans, where humans function as an incidental and ultimately dead-end host [4]. But CoVs might also adapt to the new host exemplified by the animal origin of all four endemic human CoVs (HCoV-OC43, HCoV-NL63, HCoV-229E and HCoV-HKU1) [8,9,10,11]. Whereas in cats infections with FCoV are well recognized, studies regarding possible natural infections with other animal and human coronaviruses are lacking to the best of our knowledge. Knowing the genetic variability of coronaviruses and the use of orthologous receptors by non-feline CoVs, studies on cross-species transmission are desirable. This may provide insight regarding whether cross-species transmission does occur. In the present study we used the highly immunogenic S1 antigens to screen cat sera for the presence of antibodies against feline and non-feline coronaviruses, as a first indication of possible infections with these viruses. In our study, 78 of the 137 cat sera were shown to be seropositive for coronaviruses. The seropositive rate (54.7%) against S1 of FCoV type 1 is consistent with previous studies [15,47]. All of the FCoV type 2 S1 positive sera of naturally infected cats were also positive for FCoV type 1 S1, which might be the result of cross reaction between the two proteins, despite their low amino acid identity. ELISA with specific antisera from experimentally FCoV type 1 and type 2 infected cats showed that sera of several FCoV type 1 infected cats could cross-react with FCoV type 2 S1. Domain mapping ELISA results showed that FCoV type 1 specific sera react to different levels with the S1 domains of FCoV type 1 S1 protein, and also reacts with FCoV type 2 S1CD. Vice versa, FCoV type 2 specific sera also reacted with S1CD of FCoV type 1. These observations pose a potential two-way cross-reactivity between S1CD domains. Interestingly, in parallel with our findings on feline coronaviruses, we identified a number of samples that were seropositive against the S1 of PEDV, a viral pathogen that mainly replicates in the porcine intestinal epithelium. To study the possibility of cross-reaction, samples derived from pre- and post- FCoV infected cats were screened against PEDV S1 in ELISA. The reactivity found against PEDV S1 with FCoV specific sera shows that cross-reaction can occur at the level of domain S1CD; the other PEDV S1 domains showed no reaction with the FCoV positive sera. Judging from these observations, it seems that S1CD plays an important role in cross-reaction between FCoV type 1 and 2, and also FCoV and PEDV. As S1CD is the most conserved domain among FCoV and also between FCoV and other alphacoronaviruses (for a systematic assessment of sequence identities, see Table 3), it is reasonable to hypothesize that antibodies can develop against conserved epitopes within this region and subsequently cause cross-reaction. This should be taken into account when developing and interpreting serological assays.Noticeably, ELISA reactivity among cat sera towards the N-terminal FCoV S1 domains 0 and A was less consistent and generally lower compared to whole S1, which seems to correlate with the higher antigenic variation in those domains found among FCoV type 1 strains [48] (Figure 3). Especially the sera from FCoV-RM infected cats (cat 91, 93, 95 and 115) showed lower OD values against S1A. This phenomenon could be explained as the samples displaying higher reactivity were from cats inoculated with FCoV-UU2 (cat 089, 131 and 129), the particular strain from which the S1 region was used as an antigen in the ELISA studies. In the meantime, the possibility of the variable ELISA reactivity might be due to the difference in individual antibody levels. In principle, the distinct antigenic reactivity of S10 and S1A between the two FCoV types might facilitate the development of a specific ELISA method which allows the serological discrimination of FCoV type 1 and type 2 infections in cats. In order to provide further insight regarding cross-reactivity between FCoV type 1 and PEDV, we performed virus neutralization assays. Cross-neutralization of PEDV infection could be observed for some of the feline FCoV type 1 post-infection sera, in contrast to the pre-infection serum counterparts. Since FCoV specific PEDV neutralizing sera did not react with PEDV S10, S1A or S1B, it is likely that the cross-neutralizing antibodies are targeting conserved epitopes in the S1CD domain or the S2 subunit of the PEDV spike protein [49]. Given the unknown TGEV infection background of the PEDV positive pigs, the cross-reaction of PEDV specific sera against FCoV type 1 could not be explored in our study, as TGEV positive pig samples would certainly influence the outcome [12,18,50]. Of note, our findings cannot exclude the possibility that field cats might incidentally get naturally infected with PEDV or PEDV-like viruses, as there had been one report showing the detection of PEDV in one stray cat via PCR assay [51]. It would be interesting to include more sera of cats from pig farms in future studies.Considering the fact that cats play an important role in human society and have constant interaction with humans, it is of interest to conduct serological surveys for possible reverse zoonosis of human pathogens. In our study S1 antigens of several human coronaviruses were included and this led us to identify HCoV-229E seropositive feline samples in our ELISA survey (Table 1); one serum in particular reacted solely with HCoV-229E S1 but not with any other coronavirus. Of the HCoV-229E S1 reactive feline sera one showed low neutralizing activity against HCoV-229E infection. This might suggest that positive cats were indeed exposed to HCoV-229E or related viruses. MERS seropositivity is also seen in other species besides the dromedary host [52]. Rare cases of seropositivity might be considered as spill-over infections from the dromedary camel reservoir. Similar (perhaps dead-end) spill-over infections of 229E from the human reservoir to cats might also occur. A similar principle could also apply for PDCoV, a porcine pathogen that emerged rather recently. Both HCoV-229E and PDCoV use APN as their receptor and have been reported to also be able to use feline APN for cellular entry [21,37]. Although reports are lacking regarding the natural infection of these two viruses in cats, HCoV-229E was shown to cause a priming effect of FCoV antibody in experimentally FCoV infected cats suggesting that infection occurred [24]. Therefore, the detection of antibodies against S1 of HCoV-229E in a portion of the cats might be specific and due to the exposure to HCoV-229E through daily interaction with humans. Eight cats were seropositive for PDCoV of which two cats were seropositive only for PDCoV and not for any other CoVs. This could be caused by infection with PDCoV or PDCoV-related viruses through avian sources, considering the fact that cats are natural avian predators and the presumed avian origin of PDCoV [2,37]. Our findings emphasize the potential role of cats as incidental hosts for non-feline coronaviruses and the need of in-depth study of naturally infected pathogens in cats. Besides serological studies, efforts should also focus on isolation and identification of these viruses in cats.In conclusion, we presented a thorough serological survey in cats using S1 proteins of different animal and human coronaviruses. We demonstrated, despite the low amino acid identity, cross-reactivity between S1 proteins of FCoV type 1 and 2, and between that of FCoV type 1 and PEDV. This should be considered when developing FCoV serological assays as well as interpreting the results. Our observation that some feline sera displayed antibody reactivity exclusively against non-feline CoV S1 proteins warrant further research into the epidemiology and cross-species transmission of coronaviruses in cats and other animals that are in close contact with humans. Further large scale serological studies regarding coronaviruses infection across animal species using arrays of CoV S1 antigens can shed light into the hitherto unresolved host promiscuity of coronaviruses and the risk of cross-species transmission.",
        "pmid": "31412572",
        "title": "Serological Screening for Coronavirus Infections in Cats",
        "journal_title": "Viruses",
        "authors": "Shan Zhao;Wentao Li;Nancy Schuurman;Frank van Kuppeveld;Berend-Jan Bosch;Herman Egberink",
        "doi": "10.3390/v11080743"
    },
    {
        "keywords": "",
        "abstract": "AbstractBackground\nHuman Coronaviruses (CoV) have been long recognized as a common cause of respiratory tract disease including severe respiratory tract illness, yet there are few recent studies characterizing disease among adults in the United States. Here, we describe CoV infections and clinical characteristics among adults (>18&#160;years) presenting with respiratory illness in Cleveland,&#160;Ohio.Methods\nBetween February 1, 2016 and April 30, 2017, 2949 nasopharyngeal swab specimens were analyzed by NxTAG Respiratory Pathogen Panel in adults presenting with respiratory illness at MetroHealth Medical Center. Clinical data were collected on adults whose samples screened positive for CoV-HKU1, CoV-OC43, CoV-229E or CoV-NL63.Results\nCoronaviruses were detected in 192 (6.5%) adults including 105 (3.5%) OC43, 67 (2.3%) 229E, 13 (0.4%) HKU1 and 7 (0.2%) NL63. The majority of adults with coronavirus infection were females (66.2%) with a median age of 53&#160;years. Common comorbidities included smoking (40.0%), asthma (38.0%), COPD (35.4%), and inhaled corticosteroid use (28.6%). Eighty-five (46.4%) required admission to the hospital. Common presenting symptoms included shortness of breath (42.7%) and cough (31.0%) whereas fever was uncommon (12.5%). Gastrointestinal symptoms were more common in HKU1 and NL63 infected adults. Seventy-three percent of coronavirus disease occurred between the months of January and March. Despite the recognition of coronavirus infection, 70 (36.5%) received antibiotics for their disease.Conclusion\nThis study provides needed insight into clinical characteristics and severity associated with coronavirus infection in adults. Coronavirus infection should be considered in differential diagnosis of respiratory tract illness in adults including those that require hospitalization, have a history of smoking and have pulmonary comorbidities.Disclosures\n\nAll authors: No reported disclosures.",
        "title": "Human Coronavirus (HCoV) Infection Among Adults in Cleveland, Ohio: An Increasingly Recognized Respiratory Pathogen",
        "journal_title": "Open Forum Infectious Diseases",
        "authors": "Anubhav Kanwar;Suresh Selvaraju;Frank Esper",
        "doi": "10.1093/ofid/ofx163.729"
    },
    {
        "keywords": "ATP hydrolysis;DNA;RNA;coronavirus;enzyme kinetics;helicase",
        "abstract": "Coronaviruses are known to cause a wide range of diseases in humans and animals. Middle East respiratory syndrome coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans in the Middle East, Europe, North Africa, and the United States of America. Helicases are motor proteins that catalyze the processive separation of double-stranded nucleic acids into two single-stranded nucleic acids by utilizing the energy derived from ATP hydrolysis. MERS-CoV helicase is one of the most important viral replication enzymes of this coronavirus. Herein, we report the first bacterial expression, enzyme purification, and biochemical characterization of MERS-CoV helicase. The knowledge obtained from this study might be used to identify an inhibitor of MERS-CoV replication, and the helicase might be used as a therapeutic target.",
        "body": "INTRODUCTION\nMiddle East respiratory syndrome coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans in the Middle East (Saudi Arabia, Jordan, Qatar, and the United Arab Emirates), Europe (the United Kingdom, France, Italy, and Germany), North Africa (Tunisia and Egypt), and the United States of America (1, 2). Globally, as of 16&#160;May 2016, the WHO has been notified of 1,733 laboratory-confirmed cases of infection with MERS-CoV, including at least 628 related deaths, in 27 different countries (http://www.who.int/csr/don/16-may-2016-mers-saudi-arabia/en/). While gastrointestinal symptoms, including diarrhea and queasiness, are also occasionally observed (3, 4), MERS-CoV causes mainly respiratory diseases.MERS-CoV is a positive single-stranded RNA (ssRNA) virus with one of the largest known RNA genomes (30.119&#160;kb) (5). Following infection, there is translation of two large replicative polyproteins, pp1a (the ORF1a polyprotein) and pp1ab (the polyprotein made from ORF1a and ORF1b through a &#8722;1 ribosomal frameshift during translation). These polyproteins are processed by the virus-encoded papain-like proteinase (PLpro) and nonstructural protein 5 (nsp5), a 3C-like protease (3CLpro). This auto-proteolysis leads to the formation of 16 nonstructural proteins, including an RNA-dependent RNA polymerase (RdRp) and a nucleoside triphosphatase (NTPase) and helicase that are known as nsp12 and nsp13, respectively (3, 6&#8211;8). These are likely to form the core of membrane-bound replication-transcription complexes in double-membrane vesicles at perinuclear regions (9).Helicases are motor proteins that catalyze the processive separation of double-stranded nucleic acids (ds) into two single-stranded nucleic acids (ss) by utilizing the energy derived from ATP hydrolysis (10&#8211;17). Helicases are known to unwind nucleic acids during replication, recombination, and DNA repair (14), and recent reports have shown that they are also involved in other biological processes, including displacement of proteins from nucleic acid, movement of Holliday junctions, chromatin remodeling, catalysis of nucleic acid conformational changes (18&#8211;23), and several aspects of RNA metabolism (transcription, mRNA splicing, mRNA export, translation, RNA stability) and mitochondrial gene expression (9). A more recent biochemical study revealed a dual function of human enterovirus nonstructural protein 2CATPase as both an RNA helicase and an ATP-independent RNA chaperone which may destabilize RNA duplexes and assist in the formation of more globally stable RNA structures (24). Defects in helicase function have been associated with some human diseases, including Bloom&#8217;s syndrome, Werner&#8217;s syndrome, and xeroderma pigmentosum (25&#8211;28).Previously, biochemical characterization of severe acute respiratory syndrome CoV (SARS-CoV) helicase (S-nsp13) demonstrated that S-nsp13 can unwind both double-stranded DNA and RNA in a 5&#8242;-to-3&#8242; direction, and it can hydrolyze all deoxyribonucleotide triphosphates (dNTPs) and ribonucleotide triphosphates (29, 30), Additional study also showed that nsp12, the SARS-CoV RdRp, enhances the catalytic efficiency of S-nsp13 by increasing the step size of nucleic acid (RNA/RNA or DNA/DNA) unwinding by 2-fold (31). However, there are no reports on the biochemical characterization of MERS-CoV helicase (M-nsp13).Herein, we report the first bacterial expression, enzyme purification, and biochemical characterization of M-nsp13. With double-stranded RNA (dsRNA) as a substrate, M-nsp13 unwound the substrate in a 5&#8242;-to-3&#8242; direction, with the helicase requiring more ATP for optimal unwinding of RNA substrates with short 5&#8242; loading strands. We also report that the rate of unwinding (ku) of M-nsp13 is directly proportional to the length of the 5&#8242; loading strand of the partially duplex RNA substrate. While M-nsp13 required a single-strand 5- to 20-nucleotide (nt) 5&#8242; overhang for efficient unwinding of the dsRNA substrate, M-nsp13 was still able to partially unwind (~10% to 30%) a dsRNA substrate with a 0-nt (blunt-ended) to 2-nt 5&#8242; loading strand.\nRESULTS\n\nExpression and purification of MERS-CoV helicase (M-nsp13).\nTo obtain sufficient amounts of M-nsp13 for biochemical studies, an Escherichia&#160;coli expression system was used. MERS-CoV pp1ab residues 5311 to 5881 were fused at the N terminus to the E.&#160;coli Strep tag. As Fig.&#160;1A shows (lanes E1 to E4), fusion protein Strep&#8211;M-nsp13 of sufficient purity was obtained with a two-step purification protocol involving Strep-Tactin affinity and size exclusion chromatography. The same method was used to express and purify a Strep&#8211;M-nsp13 control protein, M-nsp13_K288A, in which the conserved lysine residue (5598 in pp1ab) of the Walker A box (32) (helicase motif I in Fig.&#160;2) was replaced with Ala. The identities of the proteins were confirmed by Western blotting (Fig.&#160;1B).The amino acid sequence of M-nsp13 was aligned with the helicase amino acid sequences of other coronaviruses, including SARS-CoV isolate Frankfurt 1 (GenBank accession no. AY291315), mouse hepatitis virus (MHV) strain A59 (accession number NC_001846), porcine transmissible gastroenteritis virus (TGEV) strain Purdue 46 (accession number AJ271965), and avian infectious bronchitis virus (IBV) strain Beaudette (accession number M95169). The protein alignment shows that all helicases, along with M-nsp13, have the conserved helicase motifs I to VI, with motif I (Walker A box) containing the Lys288 that is recognized as the NTP binding site and the core ATPase site of superfamily 1 to 3 (SF1 to -3) helicases (32).\nM-nsp13 can unwind both RNA and DNA helices.\nTo evaluate the RNA helix unwinding activity of M-nsp13, various concentrations of the enzymes (1 to 40&#160;nM) were incubated with a 5&#160;nM concentration of the partially duplex RNA substrate with a 20-nt overhang (5&#8242;-RNA-20 [20&#160;ss, 22&#160;ds]) for 30&#160;min. Unless stated otherwise, for this specific experiment, we used the 5&#8242;-RNA-20 partially duplex substrate (see Fig.&#160;S1 in the supplemental material), since previous coronavirus helicase, i.e., SARS-CoV nsp13, was shown to have a 5&#8242;-to-3&#8242; directionality (29). As shown in Fig.&#160;3A, the substrate was completely unwound at a 20 nM enzyme concentration in a 5&#8242;-to-3&#8242; direction.Some reports have shown that some helicases have a clear preference for either RNA or DNA (9, 33&#8211;36). To determine whether M-nsp13 prefers one substrate over the other, we used the 5&#8242;-overhang 20-nt (20&#160;ss, 22&#160;ds) RNA and DNA substrates (Fig.&#160;S1) and monitored M-nsp13&#8217;s helicase activity. As shown in Fig.&#160;3B, M-nsp13 did not exhibit any preference for either RNA or DNA substrates. The results are similar to what was previously reported for SARS-CoV helicase (29). As such, our experimental analyses were carried out using RNA substrates to mimic the natural substrate.\nMetal and pH dependence of M-nsp13.\nTo determine the metal requirements of M-nsp13 for its helicase activity, a 20 nM enzyme concentration was incubated with the 5&#8242;-RNA-20 (20&#160;ss, 22&#160;ds) substrate in the presence of 2&#160;mM MnCl2, MgCl2, ZnCl2, or CaCl2. The results showed that M-nsp13 had the best helicase activity with MgCl2, followed by MnCl2, with more than 85% of the substrate unwound. The enzyme showed lesser activity with ZnCl2 and no observable activity in the presence of CaCl2 (Fig.&#160;4A).To determine the optimal concentration of MgCl2 required for M-nsp13 activity, various concentrations of MgCl2 (1 to 6&#160;mM) were incubated with 20&#160;nM M-nsp13 and the 5&#160;nM 5&#8242;-RNA-20 (20&#160;ss, 22&#160;ds) substrate for 30&#160;min. The result showed that M-nsp13 had optimal activity with 2&#160;mM MgCl2 (Fig.&#160;4B).To evaluate the optimal pH for M-nsp13 unwinding activity, reactions similar to those described above were performed at various pHs (4.0 to 9.0). As shown in Fig.&#160;5, M-nsp13 has optimal unwinding activity from pHs&#160;7.0 to 8.5.\nMinimum 5\u2032 loading strand length requirement of M-nsp13 for efficient RNA unwinding.\nTo determine whether nucleic acid unwinding by M-nsp13 depends on the loading strand length and to determine the minimum length of the 5&#8242; single-stranded overhang required for efficient helicase activity, we designed five partially duplex RNA substrates with single-stranded 5&#8242; ends of various lengths (between 0 and 20 bases) (see Fig.&#160;S1 in the supplemental material; Fig.&#160;6) but with dsRNA duplex regions of the same length. Using these substrates, we monitored the unwinding activity of M-nsp13 in the presence of 2&#160;mM ATP. As shown in Fig.&#160;6A, the efficiency of RNA unwinding by M-nsp13 increased as the length of the 5&#8242; loading strand increased. While M-nsp13 required a 5- to 20-nucleotide (nt) single-strand 5&#8242; overhang for efficient unwinding of the dsRNA substrate, M-nsp13 was still able to partially unwind (~10% to 30%) a dsRNA substrate with a 0- (blunt) to 2-nt 5&#8242; loading strand.\nThe optimal helix unwinding activity of M-nsp13 requires NTPs.\nNTP hydrolysis is known to provide the energy required for translocation of RNA helicases along ssRNA and duplex-RNA during unwinding. To determine whether ATP is required for the unwinding activities of M-nsp13, a standard unwinding assay was conducted with all the RNA substrates described above in the absence of ATP. As shown in Fig.&#160;6B, while M-nsp13 was partially able to unwind the partially duplex RNA substrate with a 20-nt overhang (5&#8242;-RNA-20) in the absence of ATP, M-nsp13 could not unwind other RNA substrates with shorter loading strands.A recently identified human enterovirus helicase, nonstructural protein 2CATPase, was shown to possess an ATP-independent RNA chaperone activity (24). To investigate whether M-nsp13 has an underlying ability to unwind its substrate in an ATP-independent manner, M-nsp13_K288A (the control protein that lacks the ability to hydrolyze ATP) was incubated with the 5&#8242;-RNA-20 partially duplex substrate in the presence of 2&#160;mM ATP. The results showed that M-nsp13_K288A could not unwind the 5&#8242;-RNA-20 partially duplex substrate, indicating that M-nsp13 does not have an ATP-independent RNA chaperone activity (Fig.&#160;6C). Overall, these results suggest that M-nsp13&#8217;s ability to unwind the 5&#8242;-RNA-20 partially duplex substrate is likely due to residual cellular ATP purified with the helicase as well as to a long overhang length, which enhances M-nsp13&#8217;s unwinding activity.Given the observation that M-nsp13 was partially able to unwind the 5&#8242;-RNA-20 partially duplex substrate when ATP was not included in the reaction mixture, we sought to investigate the ATP requirement of M-nsp13 for the different RNA substrates with 5&#8242; loading strands of various lengths. We assessed the unwinding activity of M-nsp13 with the various 5&#8242;-RNA partially duplex substrates with different overhang lengths in the presence of various ATP concentrations from 0.25 to 5&#160;mM. Using the Michaelis-Menten equation, the unwinding activities under different ATP concentrations were plotted as the fraction of the released RNA from the total RNA helix substrate at each ATP concentration. The results showed that increasing ATP concentrations up to ~2&#160;mM ATP apparently enhanced the helix unwinding of each RNA substrate (Fig.&#160;7A), after which no significant difference in substrate unwinding was noticeable. Moreover, M-nsp13 could not completely unwind the 5&#8242;-RNA-0 (blunt substrate), 5&#8242;-RNA-2 (2-nt overhang), and 5&#8242;-RNA-5 (5-nt overhang) partially duplex RNA substrates even at 5&#160;mM ATP. Furthermore, the Km and Vmax values for each substrate (Table&#160;1) indicate that the amount of ATP required by M-nsp13 to unwind the substrates is inversely proportional to the length of the loading strand.To further investigate the (d)NTP requirements of M-nsp13, a standard unwinding assay was conducted as described above, using the 5&#8242;-RNA-10 (10&#160;ss, 22&#160;ds [or 10-nt overhang]) partially duplex RNA substrate in the presence or absence of individual (d)NTPs (2&#160;mM). The results showed that M-nsp13 was able to hydrolyze all the nucleotides for the purpose of nucleic acid unwinding, with slightly more specificity for the NTPs (Fig.&#160;7B). These results further confirmed that the helix unwinding activity of M-nsp13 needs the participation of ATP or other NTPs. It should be noted that since M-nsp13 can partially unwind the 5&#8242;-RNA-20 (20&#160;ss, 22&#160;ds [or 20-nt overhang]) partially duplex RNA substrate in the absence of ATP, we decided not to use this substrate for this experiment to allow for proper assessment of the effect of NTPs on M-nsp13&#8217;s unwinding activity.\nM-nsp13 rates of unwinding of the partially duplex RNA substrates that have various 5\u2032 overhang lengths.\nTo determine the rates of unwinding of M-nsp13 with all the partially duplex RNA substrates with various 5&#8242; overhang lengths used in this study, the enzyme was incubated with the RNA substrates 5&#8242;-RNA-0 (the blunt substrate), 5&#8242;-RNA-2 (2-nt overhang), 5&#8242;-RNA-5 (5-nt overhang), 5&#8242;-RNA-10 (10-nt overhang), and 5&#8242;-RNA-20 (20-nt overhang) in the presence of 2&#160;mM ATP at different time points (0 to 45&#160;min). The change in the fractions of unwound ssRNA product with time is shown in Fig.&#160;8. The experimental data for the substrates were fit to the following equation: fraction (ssDNA) = A&#160;&#8901;&#160;(1 &#8211; e&#8211;kt) + n, where A is the amplitude that corresponds to the maximum fraction of ssRNA that can be generated enzymatically from the substrates, k is the pseudo first-order rate constant of RNA unwinding, t is the reaction time, and n is an additive constant representing the amount of ssDNA present before the reaction started. The results showed that the amplitudes and the rates of RNA unwinding of M-nsp13 were directly proportional to the length of the 5&#8242; loading strand (Table&#160;2).\nM-nsp13 is a unidirectional helicase.\nPrevious reports have shown that SARS-CoV helicase has a 5&#8242;-to-3&#8242; directionality and, as such, does not show any activity with a substrate that has a 3&#8242; overhang (29, 31). To determine whether M-nsp13 can also unwind in a 3&#8242;-to-5 direction, the standard helicase assay was conducted with the RNA substrates that have various lengths of the 3&#8242; loading strand in the presence of 2&#160;mM ATP. As shown in Fig.&#160;9, M-nsp13 could not unwind any of the 3&#8242; RNA (3&#8242;-overhang) substrates, confirming that M-nsp13 is a unidirectional 5&#8242;-to-3&#8242; helicase.\nDISCUSSION\nSequence alignment of M-nsp13 with other helicases suggests that the MERS-CoV helicase belongs to the SF1 family of helicases (29). However, the biochemical aspects of the enzymatic mechanism of MERS-CoV nsp13, including ATP hydrolysis, translocation along the nucleic acid, and the unwinding rate, have not been studied. The present study fills this knowledge gap.One of the unique findings of this study is the observation that the length of the 5&#8242; loading strand of the partially duplex RNA can determine the amount of ATP required by the helicase for optimal helix unwinding. The longer the length of the loading strand, the lower the amount of ATP required for optimal unwinding. As noted in Fig.&#160;6B, M-nsp13 was able to partially unwind the substrate with the longest 5&#8242; loading strand without inclusion of ATP in the reaction mixture. A recently characterized human enterovirus helicase, nonstructural protein 2CATPase, was shown to possess an ATP-independent RNA chaperone activity (24). The RNA chaperone activity was observed along the 3&#8242;-to-5&#8242; direction in the absence of ATP. Given the presence of an ssRNA product in the absence of ATP, when M-nsp13 was incubated with the 5&#8242;-RNA-20 partially duplex RNA substrate, we wanted to confirm a possible RNA chaperone activity by M-nsp13. As shown in Fig.&#160;6C, M-nsp13_K288A (the control protein that lacks the ability to hydrolyze ATP) did not unwind either substrate, thereby ruling out an RNA chaperone activity by M-nsp13. The small amount of the ssRNA product observed with the 5&#8242;-RNA-20 partially duplex RNA substrate in the absence of ATP may be due to residual cellular ATP purified with the helicase as well as the longer overhang length of 5&#8242;-RNA-20 substrate relative to other substrates that enhances M-nsp13 unwinding activity.Another important finding in this study is that the rate of unwinding of M-nsp13 is directly proportional to the length of the 5&#8242; loading strand of the partially duplex RNA substrate. This observation may be due to the ease of binding the long loading strand by M-nsp13.Previous reports have shown that SARS-CoV nsp12 (RNA polymerase) and nsp13 (helicase) physically interact, with nsp12 enhancing the activity of nsp13 (31, 37), thereby forming part of a larger SARS-CoV replication complex. The 5&#8242;-to-3&#8242; helicase activity of M-nsp13 may be similar to what was previously proposed for SARS-CoV (31). It was suggested that SARS-CoV nsp12 and nsp13 may work together to carry out one of the replication complex functions, i.e., synthesis of subgenomic transcripts containing the leader sequence derived from the 5&#8242; end of the genome (29, 30). To achieve this, nsp12 would have to transcribe RNA in a 3&#8242;-to-5&#8242; template direction, followed by pausing at transcription regulatory element sites. Such pauses may activate motion in the opposite direction, allowing nsp13 (aided by nsp12 and possibly other viral and host proteins) to unwind the 3&#8242; end of the nascent RNA primer and facilitate transfer to the complementary region of the 5&#8242; leader genomic sequence.Previous studies revealed that SARS-CoV and human coronavirus 229E helicases required at least a 5-nt 5&#8242;-end single-strand overhang for efficient unwinding (31, 38). In this study, while up to 10 to 30% of ssRNA products were observed with the 0- to 2-nt 5&#8242;-end single-strand overhang partially duplex RNA substrates, M-nsp13 exhibited efficient unwinding activities only with partially duplex RNA substrates that have a &#8805;5-nt-long 5&#8242; loading strand.While some helicases exhibit a clear preference for either RNA or DNA (9, 33&#8211;36), some helicases, such as SARS-CoV helicase, worked well with either RNA or DNA (31). Our findings with M-nsp13 also support the fact that coronavirus helicases do not exhibit a preference for either RNA or DNA substrates.Comparison of the NTPase and dNTPase activities of M-nsp13 showed moderate variations, as M-nsp13 clearly exhibited better enzymatic activities with the NTPs than with the dNTPs. Nonetheless, the data suggested that the nucleotide-binding site of M-nsp13 has a low specificity among the NTPs. Similar data were also obtained for SARS-CoV nsp13 (30). Other RNA virus helicase-associated NTPase activities were also shown to lack any noticeable specificity for specific nucleotides (30, 35, 39&#8211;41). This may suggest that the lack of selectivity for specific nucleotide cofactors is a general feature of RNA virus helicases. Due to the high nucleotide consumption necessary for maximum viral RNA synthesis in the host cell, it may be beneficial for the helicase&#8217;s duplex-unwinding activity not to depend strictly on a specific nucleotide. As such, there is lower risk for depletion of cellular NTP pools.Other areas for future investigation include establishing the kinetic step size and the natural state of the active protein, .i.e., the monomer of multimers, and characterizing replication complex functions that involve M-nsp13, along with other MERS-CoV nonstructural proteins.In conclusion, we report the first expression, purification, and biochemical characterization of MERS-CoV helicase. We showed that M-nsp13 required more ATP for optimal unwinding of partially duplex RNA substrates with short 5&#8242; loading strands. Our results also revealed that the rate of unwinding of M-nsp13 increased with a partially duplex RNA substrate with a longer 5&#8242; loading strand. These results provide insights that enhance our understanding of the biochemical properties of M-nsp13. As such, M-nsp13 might be used as a target to identify inhibitors of MERS-CoV replication.\nMATERIALS AND METHODS\n\nCloning, expression, and purification of M-nsp13 helicase.\nM-nsp13 is encoded by nucleotides 16208 to 18001 of the MERS-CoV genome (human betacoronavirus strain 2c EMC/2012; GenBank accession no. JX869059.2). The coding region of M-nsp13 was amplified by PCR using 5&#8242;-BamHI-nsp13-CGGGATCCTGCTGTAGGCTCTTGTGTTG-3&#8242; as the forward primer and 3&#8242;-EagI-nsp13-ATGCGGCCGCTATTGCAGCTTGTAGTTGGTAAAGCTC-3&#8242; as the reverse primer. The PCR amplicons were digested with BamHI and EagI, followed by ligation into expression vector pET52b (Novagen-EMD Millipore), which fused M-nsp13 with a Strep tag at the N terminus. After confirmation of the amplicon nucleotide sequences, the construct was transformed into E.&#160;coli BL21 cells. Protein expression and purification were performed as described previously (1, 31, 42) using Strep-Tactin beads (IBA). The protein was eluted with 100&#160;mM Tris, pH&#160;8.0, 150&#160;mM NaCl, 10&#160;mM desthiobiotin, 0.1% Triton X-100, 5&#160;mM &#946;-mercaptoethanol (BME), and 5% glycerol. Using dialysis, the buffer was exchanged with (100&#160;mM Tris, pH&#160;8.0, 150&#160;mM NaCl, 0.1% Triton X-100, 5&#160;mM BME, and 5% glycerol). Fractions containing the desired protein were concentrated and stored at &#8722;80&#176;C.The M-nsp13_K288A mutant was prepared using the QuikChange II XL site-directed mutagenesis kit (Agilent) as described by the manufacturer and expressed and purified as described above for the wild-type enzyme.\nExpression and purification of foot-and-mouth disease virus (FMDV) 3Dpol.\nExpression and purification were as previously described (43). Briefly, the pET-28a-FMDV 3Dpol plasmid was transformed into the Rosetta 2 expression strain (Novagen). Kanamycin-resistant colonies were grown at 37&#176;C, and protein expression was induced at an A600 of 0.9 to 1.0 by the addition of 1&#160;mM isopropyl &#946;-d-1-thiogalactopyranoside (IPTG). The cells were allowed to grow for 3 more h after induction. The cells were harvested by centrifugation (4,500&#160;&#215; g, 20&#160;min) and stored at &#8722;20&#176;C. Frozen cell pellets were resuspended in buffer A (25&#160;mM Tris-HCl, pH&#160;8.0, 500&#160;mM NaCl, and 5% glycerol). The protein was purified by nickel affinity chromatography with a gradient of 25&#160;mM to 500&#160;mM imidazole in buffer A. Fractions containing pure protein (95%) were pooled and dialyzed against the storage buffer containing 12.5&#160;mM Tris-HCl, pH&#160;8.0, 100&#160;mM NaCl, and 50% glycerol.\nWestern blotting.\nPurified M-nsp13, M-nsp13K288A, and FMDV polymerase were separated by 10% SDS-polyacrylamide gel electrophoresis (PAGE), followed by transfer to nitrocellulose membranes (Schleicher & Schuell, Dassel, Germany). M-nsp13 was detected with anti-M-nsp13 rabbit antiserum as the primary antibody, which was commercially developed upon request (Abcam). A horseradish peroxidase-conjugated swine anti-rabbit immunoglobulin G (Dako) was used as the secondary antibody.\nNucleic acid substrates.\nSynthetic oligonucleotides were purchased from Integrated DNA Technologies (Coralville, IA). Sequences of the RNA and DNA substrates are shown in Fig.&#160;S1 in the supplemental material. Concentrations were determined spectrophotometrically using absorption at 260&#160;nm and their extinction coefficients. Unlabeled oligonucleotides were annealed to corresponding 5&#8242;-Cy3-labeled 22-mers in 50&#160;mM Tris, pH&#160;8.0, 50&#160;mM NaCl at a ratio of 1 to 1.2 by heating them at 95&#176;C for 5&#160;min and cooling them slowly to room temperature. Unlabeled 22-mers were used as traps for the helicase assay.\nCharacterization of helicase activity.\nHelicase activity was measured by incubating 20&#160;nM M-nsp13 with a 5&#160;nM concentration of an RNA substrate (Fig.&#160;S1) in a reaction buffer containing 20&#160;mM HEPES, pH&#160;7.5, 20&#160;mM NaCl, 1&#160;mM dithiothreitol (DTT), 0.1&#160;mg/ml bovine serum albumin (BSA), 5&#160;mM MgCl2, and 2&#160;mM ATP at 30&#176;C for various times. Reactions were quenched by the addition of an equal volume of loading buffer (0.2% SDS and 20% glycerol). Unless otherwise mentioned, reactant concentrations refer to the final concentration in the reaction mixture. The released single-stranded RNA (ssRNA) product and unwound double-stranded RNA (dsRNA) were resolved by 8% nondenaturing PAGE using a running buffer containing 89&#160;mM Tris-borate, pH&#160;8.2, and run for 45&#160;min at 4&#176;C and 110&#160;V. The controls for measuring maximum unwinding were dsRNAs denatured by heat for 5&#160;min at 95&#176;C and loaded immediately on the gel. In this and subsequent assays, the gels were scanned with a Bio-Rad multipurpose imager. When necessary, the band intensities representing ssRNA and dsRNA were quantitated using the Image Lab 5.1 software (Bio-Rad). The fraction of unwound RNA was plotted against time, and the kinetic parameters described in the mechanism were determined by nonlinear regression using GraphPad Prism (GraphPad Inc.).\nAnalyses of RNA unwinding.\nTo obtain the kinetic parameters associated with RNA unwinding by M-nsp13, the fraction of unwound RNA was plotted against time. By nonlinear regression, data were fit to the single-exponential equation in Results using Prism 7.0 (GraphPad Inc.).To obtain the kinetic parameters associated with the Km and Vmax of M-nsp13 with regard to ATP hydrolysis, the fraction of unwound RNA was plotted against different ATP concentrations. Data fitting was carried out by nonlinear regression to the Michaelis-Menten equation using Prism 7.0 (GraphPad Inc.).",
        "pmid": "27631026",
        "title": "Biochemical Characterization of Middle East Respiratory Syndrome Coronavirus Helicase",
        "journal_title": "mSphere",
        "authors": "Adeyemi O. Adedeji;Hilary Lazarus",
        "doi": "10.1128/mSphere.00235-16"
    },
    {
        "keywords": "Idiopathic thrombocytopenia;intracranial bleeding;coronavirus",
        "abstract": "Immune thrombocytopenic purpura (ITP) is an autoimmune disorder that causes isolated thrombocytopenia. Many viruses have been identified as triggering the autoimmune process, including HIV, MCV, EBV, parvovirus, rubella and measles. However, ITP in association with coronavirus infection has not previously been reported. We describe the case of a healthy man who presented with severe ITP complicated by intracranial haemorrhage following upper respiratory tract infection. An infection screen revealed coronavirus infection.LEARNING POINTS",
        "body": "INTRODUCTION\nImmune thrombocytopenic purpura (ITP) is a rare haematological disorder formerly known as idiopathic thrombocytopenic purpura before the autoimmune mechanism was identified and found to be triggered by many, mostly viral, infectious agents. Most cases of ITP are benign with only minor mucosal bleeding as the main presentation. We present a case of severe thrombocytopenia complicated by intracranial haemorrhage following infection with a coronavirus, which has not previously been reported.\nCASE DESCRIPTION\nA 24-year-old healthy man presented to the emergency department with prolonged gum bleeding and a 2-day history of a diffuse skin rash on his extremities and abdomen. He was in his usual state of health until 1 week before presentation when he developed fever and a runny nose. He reported no joint pain, oral ulcers or Raynaud&#8217;s phenomenon. He denied intake of any medications. His family history was not contributory.Upon arrival at the emergency department, the patient was awake and oriented, with blood pressure 127/72 mmHg, heart rate 90 bpm, temperature 37.3&#176;C and respiratory rate 18 breaths/min. Physical examination revealed signs of gum bleeding. Examination of the skin was remarkable for petechiae over the chest and limbs. The remainder of the systemic examination was unremarkable.Laboratory findings were as follows: haemoglobin: 13 g/dl; WBC: 6.8&#215;103/mm3; platelets: 1,000/&#956;l; PTT: 28.2 sec; PT: 11.8 sec; INR: 1.02. A peripheral blood smear was normal. Respiratory viral panel PCR was positive for coronavirus HKU1. Autoimmune work-up including ENA and ANCA serology was negative. Frequent platelet transfusions, intravenous immunoglobulin 1 g/kg for 3 days and intravenous dexamethasone 40 mg for 4 days were initiated.The next day, the patient developed a headache and vomiting, and his level of consciousness deteriorated to a Glasgow Coma Scale score of 10/15. Urgent CT of the head showed a large left frontal intracerebral haemorrhage extending into the ventricles with a midline shift of 12 mm.After 2 days, the platelet count rose to 100,000/&#956;l and then 400,000/&#956;l. The patient&#8217;s level of conscious and muscle power gradually recovered. After 1 month of rehabilitation, he was discharged in a good state of health.\nDISCUSSION\nITP is characterized by isolated low levels of circulating platelets (PLT <100,000/&#956;l) secondary to autoimmune destruction of platelets or inhibition of synthesis. It varies from mild to severe disease with lethal sequelae. The severe form includes bleeding requiring treatment which usually occurs with a platelet count <20,000/&#956;l. ITP may present acutely or chronically; the chronic form is defined as thrombocytopenia of more than 6 months&#8217; duration since initial clinical presentation. Acute ITP is common in children (<10 years) in contrast to the chronic form which is more common in adults.The exact mechanism of ITP is poorly understood, with many hypotheses claiming that viral infection triggers the disease after which preformed antibodies cross-react with platelet antigens[1,2].Clinical presentation varies from the more common petechiae, purpura and mucous membrane bleeding (epistaxis or gum bleeding) to the rare severe gastrointestinal or intracranial bleeding, which has been reported in 1.4% of patients[3].Viruses thought to cause ITP include HIV, HCV, CMV, EBV, herpes viruses and VZV[4&#8211;6]. In 1992, Wright[7] reported a case of severe thrombocytopenia secondary to asymptomatic CMV infection. In 2004, Hamada et al. [8] described a patient with severe thrombocytopenia associated with varicella zoster infection whose platelet count returned to normal after antiviral treatment. Interestingly, Zea-Vera and Parra[9] reported a case of ITP exacerbation that was secondary to Zika virus infection.An association between ITP and some bacterial infections such as tuberculosis and Helicobacter pylori has been documented[10]. However, a connection between ITP and infection with coronavirus, even though the virus is common, has not previously been reported in the literature.Infection with coronavirus (CoV) has been associated with severe acute respiratory syndrome (SARS). Haematological changes in patients with SARS are common and notably include lymphopenia and thrombocytopenia. The development of thrombocytopenia may involve a number of mechanisms. Although the development of autoimmune antibodies or immune complexes triggered by viral infection may play a significant role in inducing thrombocytopenia, SARS-CoV may also directly infect haematopoietic stem/progenitor cells, megakaryocytes and platelets, inducing their growth inhibition and apoptosis[11]. In contrast, we report severe thrombocytopenia following mild coronavirus upper respiratory tract infection.Coronavirus infections were considered benign until the SARS outbreak of 2003 when the their virulence attracted increased attention and new group members like Cov.HKU1 were identified. Cov.HKU1 was discovered in 2005 in Hong Kong in an adult with chronic pulmonary disease[12] and is now considered to be associated with acute respiratory infections.\nCONCLUSION\nCoronavirus is not a well-known cause of immune thrombocytopenia even though it may possibly cause severe ITP. Therefore, a respiratory viral panel test should be used in the initial assessment of a patient with immune thrombocytopenia.",
        "pmid": "31410357",
        "title": "Severe Immune Thrombocytopenia Complicated by Intracerebral Haemorrhage Associated with Coronavirus Infection: A Case Report and Literature Review",
        "journal_title": "European Journal of Case Reports in Internal Medicine",
        "authors": "Mohamed Magdi;Ali Rahil",
        "doi": "10.12890/2019_001155"
    },
    {
        "keywords": "human coronavirus;MERS-CoV;clinical features;upper respiratory tract infections;lower respiratory tract infections;respiratory viruses",
        "abstract": "Human coronaviruses cause both upper and lower respiratory tract infections in humans. In 2012, a sixth human coronavirus (hCoV) was isolated from a patient presenting with severe respiratory illness. The 60-year-old man died as a result of renal and respiratory failure after admission to a hospital in Jeddah, Saudi Arabia. The aetiological agent was eventually identified as a coronavirus and designated Middle East respiratory syndrome coronavirus (MERS-CoV). MERS-CoV has now been reported in more than 27 countries across the Middle East, Europe, North Africa and Asia. As of July 2017, 2040 MERS-CoV laboratory confirmed cases, resulting in 712 deaths, were reported globally, with a majority of these cases from the Arabian Peninsula. This review summarises the current understanding of MERS-CoV, with special reference to the (i) genome structure; (ii) clinical features; (iii) diagnosis of infection; and (iv) treatment and vaccine development.",
        "body": "1. Introduction\nGiven the diversity of animal coronaviruses, it was not surprising when another human coronavirus was isolated from a patient presenting with severe respiratory illness in June 2012. The 60-year-old man died as a result of renal and respiratory failure 11 days after admission to a hospital in Jeddah, Saudi Arabia [1]. The novel etiological agent was subsequently named Middle East Respiratory syndrome coronavirus (MERS-CoV) [2]. MERS-CoV is one of six known human coronaviruses that cause respiratory disease in humans and, with a mortality rate >35% [3], it is the first highly pathogenic human coronavirus to emerge since the global scare caused by the severe acute respiratory syndrome coronavirus (SARS-CoV) in 2003.With the Kingdom of Saudi Arabia, the focal point of an ongoing MERS-CoV outbreak, the large number of religious pilgrims congregating annually in Saudi Arabia drastically increases the potential for the uncontrolled global spread of MERS-CoV infections [4]. In fact, infections have already been reported in more than 27 countries across the Middle East, Europe, North Africa and Asia [5,6,7,8].This review focusses on the current information of MERS-CoV, with special reference to the genome structure, clinical features, diagnosis of infection and treatment and vaccine development. We also look at future prospects for MERS-CoV spread and prevention.\n2. Genome Structure and Gene Functions\nMERS-CoV, a lineage C Betacoronavirus (&#946;CoVs), has a positive-sense single-stranded RNA (ssRNA) genome about 30-kb in size [9,10]. As of 2016, phylogenetic analysis of MERS-CoV has been done on 182 full-length genomes or multiple concatenated genome fragments, including 94 from humans and 88 from dromedary camels [11,12]. The MERS-CoV genomes share more than 99% sequence identity, indicating a low mutation rate and low variance among the genomes. MERS-CoV genomes are roughly divided into two clades: clade A, which contains only a few strains, and clade B, to which most strains belong [12].As with other CoV genomes, the first 5&#8242; two-thirds of the MERS-CoV genome consist of the replicase complex (ORF1a and ORF1b). The remaining 3&#8242; one-third encodes the structural proteins spike (S), envelope (E), membrane (M), and nucleocapsid (N), as well as five accessory proteins (ORF3, ORF4a, ORF4b, ORF5 and ORF8b) that are not required for genome replication (Figure 1), but are likely involved in pathogenesis [9,13,14,15,16,17] . The flanking regions of the genome contain the 5&#8242; and 3&#8242; untranslated regions (UTR) [13,14]. Typical of the coronaviruses, the MERS-CoV accessory proteins do not share homology with any known host or virus protein, other than those of its closely related lineage C &#946;CoVs [12].MERS-CoV structural and accessory protein-coding plasmids transiently transfected into cells showed that, while ORF4b was localised mostly in the nucleus, all of the other proteins (S, E, M, N, ORF3, ORF4a and ORF5) localised to the cytoplasm [18]. Furthermore, studies with MERS-CoV deletion-mutants of ORFs 3 to 5 are attenuated for replication in human airway-derived (Calu-3) cells [19], and deletion-mutants of ORFs 4a and 4b are attenuated for replication in hepatic carcinoma-derived (Huh-7) cells [16,20]. This clearly points to important putative roles for the MERS-CoV accessory proteins in viral replication, at least in an in vitro setting [21].The principal response of mammalian cells to viral infection is the activation of the type I interferon (IFN)-mediated innate immune response through the production of type I IFNs (IFN-&#945; and IFN-&#946;). On the other hand, evasion of host innate immunity through IFN antagonism is a critical component of viral pathogenesis and is mediated by virus-encoded IFN antagonist proteins. Each protein blocks one or more key signaling proteins in the IFN and NF-&#954;B pathways to enhance viral replication and pathogenesis [22,23,24,25]. Coronaviruses have similarly evolved these mechanisms to impede or bypass the innate immunity of their hosts at various levels, which ultimately contribute to coronavirus virulence. Various coronavirus proteins have previously been implicated in the disruption of signal transduction events required for the IFN response [26], often by interfering with the host&#8217;s type I interferon induction.Evidence of MERS-CoV inducing type I IFN only weakly and late in infection (9&#8211;15) suggests that MERS-CoV has also evolved mechanisms to evade the host immune system. In fact, MERS-CoV M, ORF4a, ORF4b and ORF5 proteins are reported to be strong IFN antagonists [18]. Further studies, using the transient overexpression of MERS-CoV accessory protein ORF4a, ORF4b, and ORF5, show that the MERS-CoV accessory proteins inhibit both type I IFN induction [18,27,28] and NF-kappaB signaling pathways [28]. MERS-CoV ORF4a, a double-stranded RNA (dsRNA) binding protein [27], potentially acts as an antagonist of the antiviral activity of IFN via the inhibition of both the interferon production (IFN-&#946; promoter activity, IRF-3/7 and NF-&#954;B activation) and the ISRE promoter element signaling pathways [18]. MERS-CoV ORF4b, on the other hand, is an enzyme in the 2H-phosphoesterase (2H-PE) family with phosphodiesterase (PDE) activity. Even though MERS-CoV ORF4b is detected primarily in the nucleus of both infected and transfected cells [18,27,28], the expression levels of cytoplasmic MERS-CoV ORF4b are still sufficient to inhibit activation of RNase L, an interferon-induced potent antiviral activity [18,28]. MERS-CoV ORF4b is the first identified RNase L antagonist expressed by a human or bat coronavirus and provides a possible MERS-CoV mechanism for evasion of innate immunity by inhibiting the type I IFN and NF-kappa&#946; signaling pathways [16,28]. The MERS-CoV replicase proteins, including nsp1, nsp3 and nsp14, were also shown to interfere with the innate immune response signaling pathways through different mechanisms [21,29,30]. Evidently, MERS-CoV has developed various mechanisms to evade the host immune system.\n3. Clinical Features\nThe median age of persons with laboratory-confirmed MERS-CoV infection is 49 years (range, <1&#8211;94 years); 65% of patients are males. The median time from illness onset to hospitalization is approximately four days, resulting in a median length of stay of 41 days [31]. Currently, among all patients, the morbidity rate is approximately 36% [3], with the median time from the onset of symptoms to death 11.5 days [32]. Chest radiography and computed tomography findings are generally consistent with viral pneumonitis and acute respiratory distress syndrome [33]. Laboratory findings include lymphopenia, thrombocytopenia and elevated lactate dehydrogenase levels [1,31,34,35,36,37,38,39], with some cases with a consumptive coagulopathy and elevations in creatinine, lactate dehydrogenase and liver enzymes [31,33,40].The clinical spectrum of MERS-CoV infection ranges from asymptomatic infection [41,42,43] to rapidly progressive, acute respiratory distress syndrome, septic shock and multi-organ failure and death (see [32,44] for review of clinical spectrum). Initial symptoms are often nonspecific and patients report general malaise, including low grade fever, chills, headache, nonproductive cough, dyspnea, and myalgia [38,45]. Other symptoms can include sore throat and similar to SARS-CoV, MERS-CoV patients can also present with gastrointestinal symptoms such as anorexia, nausea and vomiting, abdominal pain and diarrhea [46,47,48]. Atypical presentations, including mild respiratory illness without fever and diarrheal illness, preceding the development of pneumonia have been documented [49]. Up to 50% of adult symptomatic patients require intensive care unit (ICU) treatment. These patients often show no sign of improvement and 40&#8211;70% typically require mechanical ventilation within the first week [32,41,50]. Renal replacement therapy is required for between 22&#8211;70% of critically ill patients [31,34,35,40,51], with the higher-end of the estimation possibly due to over-estimation as a result of hospital-acquired infections in patients with pre-existing renal disease [32,35].MERS-CoV is linked with more severe disease in older people, people with weakened immune systems, and those with chronic diseases such as cancer, chronic lung disease and diabetes. The majority of patients who are hospitalized with MERS-CoV infection had chronic co-morbidities such as obesity, diabetes, hypertension, cardiovascular diseases or end-stage renal disease [40,52,53,54]. In fact, about 75% of patients testing positive for MERS-CoV have at least one co-morbid disease; fatal cases are more likely to have an underlying condition (86% among fatal cases vs. 42% among recovered or asymptomatic cases) [33].Interestingly, MERS-CoV cases have been reported mainly in adults [55], with children rarely affected [56,57]. Even so, a recent case study of a MERS-CoV infected a 9-month-old child, newly diagnosed to have infantile nephrotic syndrome, showed complications that resulted in severe respiratory symptoms, multi-organ dysfunction and death [58]. In another study of 11 pediatric cases that tested positive for MERS-CoV, the two symptomatic patients had Down&#8217;s syndrome and cystic fibrosis, respectively, indicating that severe disease could potentially occur in children with serious underlying conditions [43]. Even with these reported pediatric cases, data on infection in children remain scarce, making it difficult to ascertain whether MERS-CoV is really a predominantly adult disease, or whether it often presents differently in children.Simultaneous infection of the respiratory tract with at least two viruses is common in hospitalized patients, and although it is not clear whether these infections are more, or less, severe than single virus infections [59], mixed clinical features are commonly observed [60]; this makes clinical diagnosis unreliable and severely limit epidemiological studies of etiological agents. Similar to other respiratory viruses, MERS-CoV has been found in combination with a second respiratory virus, such as influenza A virus [48,61] respiratory syncytial virus, human parainfluenza virus 3 or human metapneumovirus [62,63,64]. MERS-CoV infected patients requiring mechanical ventilation also exhibited a similar co-infection profile with nosocomial bacterial infections including, Klebsiella pneumoniae, Staphylococcus aureus, Acinetobacter species and Candida species [47,65]. Preceding or concurrent viral respiratory tract infections can predispose the host to secondary co-infections from other microorganisms throughout the airway. The mechanisms by which viruses promote these superinfections are diverse and replete [66]. As yet, not much is known as to how MERS-CoV damages the airway and dysregulate the lung barrier function, which, in turn, supports the adherence and invasion of other pathogens into normally sterile sites within the respiratory tract.Neuromuscular complications are not rare during MERS treatment, and could simply have been underdiagnosed previously [67]. The first cases of severe neurological syndrome, characterized by varying degrees of disturbed consciousness, ataxia, focal motor deficit and bilateral hyper-intense lesions were reported from a retrospective study of patients in ICU [68]. Another subsequent small retrospective study in Saudi Arabia reported that 25.7% of MERS patients developed confusion and 8.6% experienced some kind of seizure [69]. To date, other cases with central nervous system involvement, including one case of intracerebral haemorrhage as a result of thrombocytopenia, disseminated intravascular coagulation and platelet dysfunction, one case of critical illness polyneuropathy [70] and four cases that included Bickerstaff&#8217;s encephalitis overlapping with Guillain&#8211;Barre syndrome, intensive-care-unit-acquired weakness, or other toxic or infectious neuropathies [67], have been reported. Neurological complications in the latter study did not appear concomitantly with respiratory symptoms, but were delayed by 2&#8211;3 weeks [67].MERS-CoV can be detected in respiratory tract secretions, with tracheal secretions and broncho-alveolar lavage specimens containing a higher viral load than nasopharyngeal swabs. The virus has also been detected in feces, serum and urine [48,71,72,73]. Virus excretion peaks approximately 10 days after the onset of symptoms [48], but viable viruses can be shed through respiratory secretions for up to 25 days from clinically fully recovered patients. In the healthcare setting, MERS-CoV has been isolated from environmental objects such as bed sheets, bedrails, IV fluid hangers and X-ray devices [7]. Another study also reported that MERS-CoV could survive for longer than two days at 20 &#176;C and 40% relative humidity, confirming the risk of contact or fomite transmission in healthcare settings [74]. Viral RNA, on the other hand, is detected for up to five days on environmental surfaces following the last positive PCR from patients&#8217; respiratory samples; RNA was detected in samples from anterooms, medical devices and air-ventilating equipment [7], but this is not necessarily indicative of viable virus.\n4. Diagnosis of Infection\nWith no specific, reliable antiviral drug or vaccine approved for clinical use in MERS-CoV infections, rapid diagnostic tests are required to manage outbreaks of this virus. The first probe and primer sets for MERS-CoV detection by real-time RT-PCR were developed shortly after the initial reports of the disease [75,76]. Other early diagnostic tools included virus culture in Vero and LLCMK2 cells [1,77], but isolation and identification of viruses in cell culture is a slow, specialized and insensitive method [78].Laboratory detection and confirmation of MERS-CoV infections has broadly included (i) molecular detection of MERS-CoV RNA; (ii) MERS-CoV antigen detection; or (iii) assays to identify a humoral response to prior MERS-CoV infection among humans [79] (Table 1). These assays have been used with varying degrees of success in terms of specificity, sensitivity, etc.Currently, according to the WHO case definition, a positive real-time RT-PCR assay, targeting at least two different genomic regions, is used to confirm MERS-CoV infection (http://www.who.int/csr/disease/coronavirus_infections/case_definition/en/index.html) [80]. Of the different assay probes and primers sets used, those targeting ORF1a and upstream of the E gene show the highest sensitivity and remain the most widely used targets for MERS-CoV detection [75,81]. Additionally, a single positive assay result, confirmed by gene sequencing, can also be considered positive for MERS-CoV infection. A stumbling block here, though, is the fact that, when compared to real-time PCR, conventional RT-PCR typically generates lower quality sequence-ready templates [80,82,83,84,85,86], thereby limiting the usefulness of conventional RT-PCR in these single positive-sequencing assays.Molecular tests can detect nucleic acids derived from MERS-CoV in clinical respiratory, serum, and stool specimens [81,87]. However, a major obstacle of conventional nucleic acid-based tests is that they require specialized molecular techniques and equipment, and are therefore not appropriate for point-of-care testing or bedside diagnosis. For this reason, for effective diagnosis and treatment of MERS-CoV infection, it is necessary to develop alternative methods that can be adapted to rapid and reliable clinical detection of MERS-CoV antigens. Here, the most appropriate tests would be assays detecting viral antigens or antibodies in the infected host [87].\n5. Animal Models\nNot only are laboratory animal species often used as models for human disease progression, they are also needed to study and evaluate novel therapies against emerging viruses [105]. Studies have shown that rabbits [106], ferrets [107], Syrian hamsters [108] and wild-type mice [109] are not suitable as models of MERS-CoV infection. More recently, four transgenic mouse models for MERS-CoV infection have been developed. In the first, a modified adenovirus expressing human DPP4 (huDPP4) is introduced intranasally to mice, which results in the expression of huDPP4 in all cells of the lung, not just those that natively express DPP4. In this model, mice show transient human DPP4 expression and mild lung disease. A concern with this model is that cells constitutively expressing DPP4 will be infected and the role of a broader infection of all cell types may change pathogenesis [110]. In the second model, a transgenic mouse was produced that expresses huDPP4 systemically. In this model, MERS-CoV infection leads to high levels of viral RNA and inflammation in the lungs, but, unfortunately, significant inflammation and viral RNA are also detected in the brains of infected mice, which represent a non-physiological expression pattern [111]. In the third model, a novel transgenic humanized mouse model was generated by replacing the mouse DPP4 coding sequence with that encoding huDPP4, ensuring correct physiological expression of huDPP4. Mice in this model show lung pathology consistent with the radiographic findings of interstitial pneumonia and significant lung disease as seen in humans infected with MERS-CoV. This suggests that this mouse model recapitulates pathological sequelae that are seen in MERS-CoV infection of humans. Importantly, unlike what is seen in other mouse models of MERS-CoV infection, virus replication and pathology in the huDPP4 mice are localized in the lungs and no inflammation develops in the brain, ensuring a more physiological accurate model of the human disease [112]. Finally, in 2016, Cockrell et al. generated a mouse model permissive for MERS-CoV infection, but with functional DPP4 immune function. Infecting this DDP4-chimeric mouse with a mouse-adapted strain of MERS-CoV, mimics MERS-CoV-induced respiratory disease without bystander neurologic disease [113]. Non-human primate models, including the rhesus macaque [114,115,116] and common marmoset [117] have also been reported as suitable animal models of MERS-CoV infection. Even though both species are susceptible to MERS-CoV infection, the extent of virus replication and severity of disease vary [105]. Rhesus macaques infected with MERS-CoV via intra-tracheal inoculation show clinical signs of disease, virus replication, histological lesions and neutralizing antibody production, indicating that this monkey model is suitable for studies of MERS-CoV infection [116]. On the other hand, the common marmoset reproduces several, but not all, features of MERS-CoV infection, and can potentially be used to evaluate novel therapies for human use [105,117].\n6. Treatment and Vaccine Development\nWhen no vaccines or specific antiviral drugs are available during an outbreak, nonspecific therapeutic interventions are often introduced to prevent severe morbidity and mortality. However, for this to be done effectively, a basic understanding of the pathogenesis of the disease is required and interventions are implemented based on observations of the clinical course of disease and complications. Due to the nature of many diseases, however, it is often not possible to assess, or systematically compare, different therapeutic approaches during an outbreak [118]. Similarly, in the case of MERS-CoV, it is necessary to monitor epidemic patterns and investigate the spread of infections to efficiently identify, control and prevent possible epidemics. For MERS-CoV infections, supportive care, which includes rest, fluids and analgesics are used, and mainly depends on the provision of organ support and management of complications [49,119,120]. Broad-spectrum antimicrobials, antivirals [121,122], interferon-&#945;2b (96) and antifungals can be used to minimize the risk of co-infection with opportunistic pathogens [49,119].Interestingly, combination treatment with ribavirin and interferons inhibits MERS-CoV replication in vitro, and it was also shown to improve clinical outcomes in MERS-CoV-infected non-human primates. However, this treatment in the rhesus macaques was initiated very soon after viral challenge (~8 h), resulting in reduced disease severity in the rhesus macaque model. This appears to simulate mild-to-moderate human MERS-CoV cases, making it difficult to extrapolate the outcome of this early intervention in severe human cases. Even though the authors recommended that combined IFN-&#945;2b and ribavirin therapy should be considered as an early intervention therapy for MERS-CoV [115], we also need to keep in mind that, due to the limited effective therapeutic window of opportunity, broad spectrum antivirals might not be sufficient to treat severe MERS-CoV patients [122].Resveratrol has been shown to inhibit various human viruses in vivo and in vitro, including influenza virus, Epstein&#8211;Barr virus, herpes simplex virus, respiratory syncytial virus, HIV-1, varicella zoster virus, enterovirus 71, human metapneumovirus, human rhinovirus 16, polyomavirus and cytomegalovirus ([123,124] for review). The antiviral effects of resveratrol are mainly associated with the inhibition of viral replication, protein synthesis, gene expression, and/or nucleic acid synthesis [123,124,125]. In an in vitro study, resveratrol was shown to significantly inhibit MERS-CoV infection, most likely due to the observed inhibition of MERS-CoV nucleocapsid (N) protein expression [126], a multifunctional protein essential for CoV replication [127]. Furthermore, resveratrol downregulated apoptosis induced by MERS-CoV, thereby prolonging cellular survival post-infection [126]. Although the beneficial roles of resveratrol in several viral diseases have been well documented, adverse effects have been also been reported, including increasing viral RNA replication during Hep-C virus infection in vitro (OR6 cells) [128], strong cytotoxicity in cultured cells [129], as well as enhanced HBV transcription and replication in vitro and in vivo [130]. Clearly, the antiviral potential of resveratrol in MERS-CoV infections needs to be studied more extensively, but, based on the various unintended negative effects, this needs to proceed with caution.More recently, de Wilde et al. [131] reported that in an in vitro test, low-micromolar concentrations of alisporivir, a non-immunosuppressive cyclosporin A-analog, inhibit the replication of four different coronaviruses, including MERS-CoV. In this study, ribavirin was found to further potentiate the antiviral effect of alisporivir in the in vitro infection models, which warrants the further exploration of cyclophilin inhibitors as potential host-directed, broad-spectrum inhibitors of coronavirus replication [131].3C-like protease (3CLpro)&#8212;analogous to picornavirus 3C protease (3Cpro)&#8212;is functionally important in the CoV replication cycle [132] and is thus regarded as a validated drug target. Peptidomimetic inhibitors of enterovirus 3Cpro (6b, 6c and 6d) inhibited MERS-CoV 3CLpro and in MERS-CoV-infected cells, the inhibitors showed antiviral activity by downregulating viral protein production in cells, as well as reducing release of infectious viral particles into culture supernatants. These compounds exhibited good selectivity index and should be investigated further as, not only an inhibitor of MERS-CoV replication and infections, but also as broad-spectrum antiviral activity drugs against other CoVs and picornaviruses [133]. Our laboratory has also previously screened the ZINC drugs-now library for candidates with potential anti-3CLpro activity with a consensus high-throughput pharmacophore modelling and molecular docking approach. Molecular dynamics was used to confirm results obtained from structure-based techniques, resulting in a highly defined hit-list of 19 compounds, which represent valuable scaffolds that could be used as a basis for future anti-coronaviral inhibitor discovery experiments [50,134]. Even with all of these potential anti-MERS-CoV candidates, no experimental interventions have demonstrated significant benefit in acutely ill patients in a consistent or controlled manner. Therefore, supportive management, adapted from guidelines developed for SARS-CoV, has thus far been the mainstay of MERS-CoV treatment [135].Because of the highly sophisticated immune evasion mechanisms of viral pathogens, human vaccine development remains a major challenge [136]. In addition, the development of safe and effective coronavirus vaccines has been even more challenging, being curtailed by major obstacles, including (1) coronavirus immunity often wanes rapidly; (2) individuals needing to be protected include the elderly; and (3) vaccines may exacerbate rather than prevent coronavirus lung immunopathology [137,138]. Various vaccines against MERS-CoV have been designed, one of which are currently being tested in clinical trials (Table 2). All of the MERS-CoV structural proteins could potentially induce neutralizing antibodies and protective responses. However, prior to identification of the major neutralizing antibody-inducing epitopes, inactivated viruses could be used in the production of first-generation vaccines; this is an easy first-response approach since it is relatively simple to produce whole killed virus particles [139]. With the many safety concerns associated with the production of inactivated vaccines [140,141,142], these types of vaccines must preferably be replaced by safer and more effective neutralizing epitope-based vaccines, as soon as the fragments containing the neutralizing epitopes are identified [139]. Current MERS-CoV vaccines provide effective protection in a few animal models [143,144,145,146,147].\n7. Future Perspectives\nThe emergence of Middle East respiratory syndrome (MERS) and the discovery of the MERS coronavirus (MERS-CoV) in 2012 suggest that another SARS-like epidemic is occurring. Unlike the severe acute respiratory syndrome (SARS) epidemic, which rapidly disappeared in less than one year, MERS has persisted for over three years. More than 2000 cases of MERS have been reported worldwide, and the disease carries a worryingly high fatality rate of >30% [12]. While this number seems low, the virus remains a global threat due to its propensity to cause severe disease in patients with underlying medical conditions and its apparent ability to readily spread within hospital settings [148]. In addition, the pattern of MERS-CoV lineages is more consistent with the movement of infected livestock or animal products [171] and epidemiological evidence suggests that it is periodically introduced into human populations [172,173], which increases the risk for various future pandemics.Even though the clinical outcomes of MERS-CoV infections are well documented, more comprehensive population-based studies are required to determine the involvement of MERS-CoV in other body systems. In addition, the continued development of technologies to routinely and accurately identify asymptomatic MERS-CoV infections will shed light on the true incidence of this virus in the human population. It would appear the MERS-CoV has been circulating in the human population for greater than one year without detection and suggests independent transmission from an unknown source. However, as discussed previously with regard to the emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002, other evolutionary aspects, such as mutation rates and selection pressure, should be considered to understand the evolutionary dynamics of MERS-CoV [174,175,176,177]. Possibly different molecular clock rates of MERS-CoV in animal hosts and humans may also have to be taken into account. Similarly to the genomic evolution of influenza A viruses [178], MERS-CoV might experience different evolutionary courses in different hosts. To better understand these dynamics, the chain of MERS-CoV zoonotic transmissions should be further clarified [174].As with other HCoVs, a detailed manipulation of the MERS-CoV genome to understand the role of the MERS-CoV viral genes in pathogenesis and replication, and for the subsequent development of MERS-CoV as a vaccine vector, is needed. The development of MERS-CoV full-length infectious clones [19,20,179] already allows for the systematic experimental study of the roles of the various corresponding MERS-CoV proteins, which should lead to a better understanding of the role of the viral genes in infectivity and pathogenicity [180]. This manipulation of the virus genome also provides a reverse genetics platform that could lead to the future development of MERS-CoV-based vector vaccines [181].As a result of the increase in MERS spread, the WHO (World Health Organisation) and CDC (Center for Disease Control) have released various case definitions to allow for the likelihood of a pandemic threat to be reduced. Fever, pneumonia, and acute respiratory distress syndrome with a history of travel to the Arab Peninsula are some of the symptoms that are used to diagnose a MERS-CoV infection. Due to the increase in nosocomial infections, health care workers are also advised to be aware of any upper respiratory tract infections and exposure to MERS-CoV-positive individuals [182]. For the foreseeable future, important measures to prevent nosocomial outbreaks should include good compliance with appropriate personal protection equipment by health-care workers when managing patients with suspected and confirmed MERS-CoV infection, early diagnosis, prompt isolation of infected patients, and improvement of ventilation in health-care facilities [183,184].",
        "pmid": "29495250",
        "title": "MERS-CoV: Understanding the Latest Human Coronavirus Threat",
        "journal_title": "Viruses",
        "authors": "Aasiyah Chafekar;Burtram C. Fielding",
        "doi": "10.3390/v10020093"
    },
    {
        "keywords": "MERS-Coronavirus;Infection control;Respiratory",
        "abstract": "Middle East respiratory syndrome coronavirus (MERS-CoV) Infection, has caused recurrent outbreaks worldwide. It is associated with severe morbidity and mortality, and is not treatable with the currently available antiviral therapies. We present a case of a 43 year-old male healthcare provider, who admitted with productive cough, dyspnea, myalgia, pleuritic chest pain and fever. Computed tomography (CT) showed bilateral ground glass opacities and consolidation. Sputum polymerase chain reaction (PCR) for MERS-coronavirus was positive.",
        "body": "INTRODUCTION\nMiddle East respiratory syndrome coronavirus (MERSCoV) Infection, has caused recurrent outbreaks worldwide. It is associated with severe morbidity and mortality, and is not treatable with the currently available antiviral therapies (1). Since its emergence in 2012, about 1595 laboratory-confirmed cases of MERS have been reported with a fatality rate of more than 30% (global deaths 571) (2).Clinical manifestation of MERS-CoV infection is variable, ranging from asymptomatic to very severe pneumonia with acute respiratory distress syndrome, septic shock and multi-organ failure resulting in death. There is growing evidence that dromedary camel is a reservoir for the virus and a source of human infection (3).Although it is likely that zoonotic transmission is the starting point of most clusters, person-to-person transmission is the most common form of transmission for MERS-CoV. Nosocomial transmission amplified the spread of MERS, and the majority of cases so far have been reported from hospital outbreaks in Saudi Arabia, the United Arab Emirates and South Korea (4). The MERS situation in the republic of Korea can be highlighted as evidence. To date, a total of 186 MERS-CoV cases, including 36 deaths, have been reported in the Republic of Korea (5). The outbreak in the Republic of Korea has been associated with health care facilities and all cases reported so far include healthcare workers caring for confirmed cases, patients who were being cared for at the same healthcare facilities as confirmed patients, and family members of confirmed patients (6, 7). Thus, infection prevention and control measures are critical to prevent the possible spread of MERS-CoV in the health care facilities. In this article we describe the first health care&#8211;acquired MERS-CoV infection in Iran.\nCASE SUMMARY\nA 43 year-old male anesthesiologist who was also a cigarette smoker from Kerman province of Iran presented with productive cough, dyspnea, myalgia, pleuritic chest pain and fever since 10 days prior to admission on June 16, 2014.He was well up until 14 days prior to admission. He gave a history of intubating a patient in an emergency situation without standard protective measures. The intubated patient was later diagnosed with MERS-CoV. Four days later, he developed dyspnea, fever, myalgia, pleuritic chest pain and cough (first dry and then became productive). The patient was referred to Dr. Masih Daneshvari Hospital &#8211; NRITLD in Tehran. Upon admission, the patient&#8217;s body temperature was 38.1&#176;C, blood pressure was 95/50 mmHg, pulse rate was 106 beats/min, respiratory rate was 32/min and percutaneous oxygen saturation was 63% in room air. Pulmonary auscultation revealed fine crackles over the lung bases and wheezing in the middle zone. Physical examination was otherwise normal. The patient&#8217;s white blood count was 3,600 cells/mm3 (lymphocytes: 26%, neutrophils: 61%, monocytes: 13%), platelets: 104,000, urea: 20mg/dL, Cr: 0.9 mg/dL and AST: 61 IU/L, ALT: 76 IU/L and alkaline phosphatase: 163 IU/L. Arterial blood gas showed pH: 7.43, PO2: 53 mmHg, PCO2: 42 mmHg, bicarbonate: 28.1 mmol/lit.Computed tomography showed bilateral ground glass opacities and consolidation with air bronchogram (Figure 1). He was admitted to the ICU and diagnostic workups were done. With the impression of severe pneumonia, vancomycin, meropenem, ciprofloxacin (when he developed hyper-sensitivity reaction to ciprofloxacin, it was changed to co-trimoxazole) and oseltamivir were started.During the hospitalization period, liver enzymes raised (two-three folds, AST: 145, ALT: 195) but renal function test was normal. Blood cultures, urine culture and sputum smear and culture were negative for bacterial infections. Viral agent screening results for influenza viruses A and B, parainfluenza viruses 1, 2 and 3, coronavirus OC43, 229E and NL63, RSV, hMPV, CMV and Adenovirus were negative in sputum by real-time RT-PCR/PCR assays. But molecular tests became positive for MERS-CoV using upstream of E gene region real-time RT-PCR assay (upE assay) (8) with a threshold cycle of 31.34. The positive result was confirmed by real-time RT-PCR assay of 1A gene region (9) with a threshold cycle of 32.76. The RdRpSeq and NSeq gene sequences were amplified (9) (Figure 2) and then the sequence of RdRpSeq PCR product was determined with the Big-Dye terminator cycle sequence kit and an ABI 377A sequencer (Applied Biosystems Inc.). Figure 3 shows the phylogenetic tree of the MERS-CoV isolate.With supportive care and antimicrobial agents he became better and no longer needed oxygen therapy at the third day of admission, and chest X ray revealed significant improvement at the 4th day of admission (Figure 4). Complete blood count changed to normal (WBC 7100 cells/mm3, lymphocytes: 35%, neutrophils: 60%, monocytes: 5%, and platelets: 297,000 cell/mm3) and liver enzymes significantly decreased. The patient was discharged without any sequel after four days.\nDISCUSSION\nMiddle East respiratory syndrome is a viral respiratory illness that was first identified in Saudi Arabia in September 2012. It is caused by a novel coronavirus (called MERS-CoV) (1). Coronaviruses are a large family of viruses that produce a wide spectrum of illnesses in humans, from the common cold to the severe acute respiratory syndrome (SARS). The MERS-CoV genetically and epidemiologically differs from the previously recognized coronaviruses such as the SARS coronavirus (SARS-CoV), which caused the SARS pandemic in south east of Asia in 2003&#8211;2004. A number of viruses in this family also cause disease in animals.The World Health Organization (WHO) was informed of a total of 1595 laboratory-confirmed cases of MERS-CoV infection, including 571 deaths (current outbreak situation in the Republic of Korea and China as of 11 September 2015 with 186 confirmed causes and 36 deaths) (2). The main source of MERS-CoV has not yet been clearly understood. Although not established, the infection could be primarily zoonotic in nature, with limited human-to-human transmission. Molecular investigation indicated that bats in Saudi Arabia might play a role in human infection (10). Previous experience with animal and human coronaviruses and seroepidemiological studies from dromedary camels demonstrated a high prevalence of antibody reactive to MERS-CoV (3). Strains of MERS-CoV that contest human strains have been isolated from camels in Egypt, Qatar, and Saudi Arabia. These investigations supported a scenario of a virus reservoir in bats with a peridomestic animal such as the camel as intermediate host, which may in fact be the immediate source of human infection (11). Genetic sequencing data revealed a close link association between the virus found in camels and that found in human (12). These studies supported the hypothesis that camels or camel related products are the expected source of infection in humans (3).Most MERS-CoV cases have been reported in adults (median age approximately 50 years, and male predominance), although children and the elderly have also been infected (range 9 months to 94 years) (13).Clinical features of MERS-CoV infection are variable, ranging from asymptomatic infection to acute upper respiratory illness, and rapidly progressive pneumonitis, respiratory failure, septic shock and multi-organ failure resulting in death (1,13, 14). Most hospitalized MERS-CoV patients have had underlying medical disorders such as diabetes, hypertension, chronic cardiac disease, and chronic renal disease or immunosuppression but MERSCoV infections were seen in previously healthy cases (14).Data suggest that MERS-CoV presents primarily with respiratory symptoms (similar to those found in SARSCoV) as a flu-like illness with signs and symptoms of pneumonia. Common symptoms in patients with MERSCoV include fever, chills/rigors, headache, non-productive cough, dyspnea, shortness of breath, myalgia and gastrointestinal symptoms. However, SARS infections did not cause renal failure, unlike MERS-CoV (13, 14).Among confirmed MERS-CoV cases reported to date, the case fatality ratio is approximately 28&#8211;30% (2, 13). Although MERS-CoV has low overall person-to-person transmission potential, most MERS-CoV cases are acquired through person-to-person transmission (15), including the most recent South Korean outbreak of MERS in 2015 (\n4\n, \n5\n, \n6\n, \n7\n, \n16\n). Human-to-human transmission of MERS-CoV can occur in health care providers and may be associated with substantial morbidity and occasionally mortality (10). A significant fraction of MERS cases were linked to the healthcare setting, ranging from 49% for the nosocomial outbreak in Jeddah and Saudi Arabia in 2014 (17) to 100 % for both the outbreak in Al-Hasa, Saudi Arabia, in 2013 (10) and the recent outbreak in South Korea in 2015 (16).An analysis showed that 59% of cases were secondary cases with an epidemiologic relation to other confirmed cases and infection in these cases has occurred more frequently in healthcare settings (63.2%) than due to family contacts (13.7%) (18). In a recent update by the WHO, it was noted that the number of laboratory-confirmed MERS-CoV cases increased sharply in relation to the occurrence of health-care-associated outbreaks (6).One screening of contacts was done in Kingdom of Saudi Arabia and three patterns of transmission were described (sporadic, within families, nosocomial transmission to healthcare workers). The true incidence of the disease in the community is not known. Of the 1,695 healthcare contacts and 462 family contacts of confirmed MERS cases, a positive result was obtained from 19 (1.12%) healthcare workers and 14 (3.6%) cases of family contacts. As a result, the rate of secondary transmission among family contact is higher than other contacts (19). Thus, Infection in the healthcare setting is an important risk factor for human-to-human transmission and close and prolonged contact with the index case seems to be a necessity for the disease transmission (20). Potential routes of transmission included respiratory droplets (was the most likely mode of transmission), with the possibility of direct or indirect contact and airborne transmission during aerosol-generating procedures as other possible routes of transmission (20). One study showed that many healthcare workers were involved in aerosol generating procedures, such as intubation (71.4%), airway suctioning (57%) and sputum induction (28.6%) (21).Base on the WHO disease outbreak news, 6 cases of MERS-CoV infection have been reported from Iran; 5 cases in Kerman Province during May&#8211;July 2014 and 1 case in Kahnooj city in March 2015 (22). Since most cases of disease have been reported in the Middle East especially Saudi Arabia, the risk of returning Hajj pilgrims contracting and spreading MERS-CoV from Saudi Arabia is high (23) and owing to the nosocomial nature of the last outbreak, it is particularly important to identify cases and stop similar preventable tragic outbreaks, with training of the epidemiological determinants, molecular diagnostic tests and management in health care workers and reporting suspected and confirmed cases. The center for disease control recommend that travelers to countries in or near the Arabian Peninsula protect themselves from respiratory diseases, including MERS, by washing their hands often and avoiding contact with persons who are ill. If travelers to these regions have fever and symptoms of respiratory illness during their trip or within 14 days of their return, they should seek medical care (24).Prevention and control measures are critical to prevent the possible spread of MERS-CoV Infection in health care services. It is not always possible to detect patients with MERS-CoV early enough because, like other respiratory infections, the early symptoms of MERS-CoV are non-specific. Therefore, health-care workers should always follow standard precautions for all patients, regardless of their diagnosis. Droplet precautions should be added to the standard precautions when providing care to patients with symptoms of acute respiratory infection; contact precautions and eye protection should be added when caring for probable or confirmed cases of MERS-CoV infection; airborne precautions should be applied when performing aerosol generating procedures (25).At this time there is no definite therapy or effective vaccine for MERS-CoV. The clinical experience from SARS suggests that a number of interventions including ribavirin with and without corticosteroids, interferon alpha with corticosteroids, ribavirin with lopinavir and ritonavir, and convalescent plasma may advance the outcome in patients but the data are not conclusive (26).\nCONCLUSION\nMERS outbreaks are significantly augmented in the healthcare facilities, and infection in the healthcare setting is an important risk factor for person to-person transmission. It is not easy to recognize patients with MERS-CoV during the early stages of the disease based on clinical presentations alone. Thus, enhanced infection control practices in medical facilities should be attempted to protect those at risk of MERS. Since no specific treatment or effective vaccine is available, development of an effective vaccine is an important public health priority.",
        "pmid": "27114729",
        "title": "Health Care Associated Middle East Respiratory Syndrome (MERS): A Case from Iran",
        "journal_title": "Tanaffos",
        "authors": "Afshin Moniri;Majid Marjani;Payam Tabarsi;Davood Yadegarynia;Seyed Alireza Nadji"
    },
    {
        "keywords": "Coronavirus;water buffalo",
        "abstract": "We report the complete genome sequences of a buffalo coronavirus (BufCoV HKU26) detected from the faecal samples of two domestic water buffaloes (Bubalus bubalis) in Bangladesh. They possessed 98&#8211;99% nucleotide identities to bovine coronavirus (BCoV) genomes, supporting BufCoV HKU26 as a member of Betacoronavirus 1. Nevertheless, BufCoV HKU26 possessed distinct accessory proteins between spike and envelope compared to BCoV. Sugar-binding residues in the N-terminal domain of S protein in BCoV are conserved in BufCoV HKU26.",
        "body": "Conflict of Interest\nNone declared.",
        "pmid": "27274850",
        "title": "First genome sequences of buffalo coronavirus from water buffaloes in Bangladesh",
        "journal_title": "New Microbes and New Infections",
        "authors": "S.K.P. Lau;A.K.L. Tsang;S. Shakeel Ahmed;M. Mahbub Alam;Z. Ahmed;P.-C. Wong;K.-Y. Yuen;P.C.Y. Woo",
        "doi": "10.1016/j.nmni.2016.02.011"
    },
    {
        "keywords": "s2m;coronavirus;secondary structure;mobile genetic element",
        "abstract": "The mobile genetic element s2m has been described in several families of single-stranded RNA viruses. The function remains elusive, but an increasing number of s2m-containing sequences are being deposited in publicly available databases. Currently, more than 700 coronavirus sequences containing s2m can be found in GenBank, including the severe acute respiratory syndrome (SARS) coronavirus genome. This is an updated review of the pattern of s2m in coronaviruses, the possible functional implications and the evolutionary history.",
        "body": "1. Introduction\nThe genomes of single-stranded RNA (ssRNA) viruses have complex secondary structures, maintained both through conventional and non-Watson-Crick base-pairing [1,2]. Coronaviruses have the largest genomes of RNA viruses and have been shown to have several highly conserved secondary structures both in the 5&#8242; and 3&#8242; end of their genomes [1]. These structural motifs mainly function through protein binding or through direct RNA-RNA interactions. Some of these elements function merely as &#8216;spacer elements&#8217;, where neither the primary nor the secondary structure are critical [3]. In other cases, the stem-loop structure itself seems essential and base-pairing nucleotides may be substituted [4]. More complex structures, such as the 54-nucleotide hairpin-type pseudoknot, appear to be conserved both in structure and location, but not in sequence [5]. For most of these elements, the exact mechanism of function is unknown, but virus replication, viability and transcription are the main functional categories.A 43-nucleotide genetic element with a highly conserved secondary (as well as primary and tertiary) structure has been described in four different families of positive-sense ssRNA viruses, Astroviridae, Caliciviridae, Picornaviridae and Coronaviridae [6]. The presence of this element in four distantly related groups of viruses and the scattered distribution within these groups has led to the conclusion that viruses have the ability to acquire it horizontally [6]. The 3&#8242; stem-loop structure, referred to as s2m, was originally described in astroviruses where it appears to have been present in the last common ancestor [6,7,8], but the element has been most thoroughly studied in the SARS coronavirus (SARS-CoV) where the three-dimensional crystal structure has been resolved to 2.7 A resolution [9]. The function remains obscure, but current hypotheses include hijacking of host protein synthesis through interactions with ribosomal proteins [9] and RNA interference (RNAi) via processing of the s2m elements into a mature microRNA [6]. In coronaviruses, the highly conserved nature of the element has also allowed the development of a PCR-based virus discovery strategy [10]. The presence of s2m near the 3&#8242; end of some coronavirus genomes has been documented previously [6,11], but due to the exponential growth of gene sequence data available through public databases, this review is an update on the current status of s2m in this virus family. The evolutionary history is discussed and different hypotheses for the possible function of s2m are presented in light of recent progress in virus genomics.\n2. Coronavirus Phylogeny and Distribution of s2m\nA total of 20,068 coronavirus nucleotide sequences have been deposited in the current version of Genbank, and when translated into amino acid sequences, the complete ORF1ab polyprotein sequence could be found for 1113 of these entries. A phylogenetic analysis was performed using aligned ORF1ab sequences (Figure 1), and numerous clusters of nearly identical accessions could be identified. The largest group comprised 328 sequences from the porcine epidemic diarrhea virus (328 OTUs). Many smaller clusters were also found, encompassing two to 183 sequences, and all serogroups were monophyletic with 100% bootstrap support (Figure 1).To get an overview of the current status for s2m in coronaviruses, all coronavirus nucleotide sequences were screened for the presence of the s2m motif using the strategy described by Tengs et al. [6]. The consensus sequence CGNGG(N)CCACGNNGNGT(N)ANNANCGAGGGT(N)ACAG was used as bait, allowing for possible insertions (N) and/or a single mismatch. Further, 708 of 20,068 (3.5%) of the sequences were found to contain the s2m motif (Table 1). However, s2m was not found in alpha coronaviruses and also appears to be absent in the Torovirus genus as well as the recently described Bafinivirus genus [12], albeit only a small number of sequences were available from the two latter groups. In all instances, the s2m sequence was found in the non-coding, 3&#8242; end of the genome. Looking at coronavirus accessions annotated as &#8216;complete genome&#8217;, more than a third of the genomes appeared to be s2m-containing (Table 1). Many sequences have been generated using targeted (PCR-based) approaches that might not include the non-coding parts of the genome. This could explain why s2m appears to be approximately 10 times more common in complete genome sequences than in shorter GenBank submissions (Table 1).Three phylogenetic clusters were found to contain s2m (Figure 1). All of these groups were supported by 100% bootstrap values and thus are genetically distinct from related viruses not containing the element. Multiple hypothesis can be proposed in order to explain the distribution of s2m within the coronaviruses, including a model where s2m was present in the last common ancestor of this virus group and subsequently lost multiple times. However, given the mobile nature of the motif, a more parsimonious explanation includes two (or more) independent gains and a small number of losses. The gain of s2m at the base of the SARS-cluster (edge 1 in Figure 1) would not assume any losses in the beta coronaviruses, but for the gamma/delta coronavirus cluster, the evolutionary history seems to be more complicated. A gain at edge 4 would require multiple independent losses. There was no significant bootstrap support separating the Wigeon coronavirus and the Night-heron coronavirus sequences, so a monophyletic origin for these two strains could not be excluded. Loss in the common ancestor of this group combined with loss prior to the divergence of the Bottlenose dolphin coronavirus and the Beluga Whale coronavirus (edge 3) would thus indicate one gain and two independent losses. One could also hypothesize two independent gains (edge 2 and 5) in order to explain the s2m distribution in the gamma/delta cluster (Figure 1).Looking at topology and branch lengths, branch 1 (Figure 1) represents the shortest edge connecting an s2m-containing cluster with a virus strain not containing s2m. s2m is absent in Rousettus bat coronavirus HKU9, which is a cluster of eight nearly identical sequences basal to the SARS(-like) group. Rousettus bat coronavirus is thus far the only member of lineage D within the betacoronaviruses [13]. The most likely mechanism for transfer of the s2m element from one viral genome to another is through genetic recombination [6]. Recombination events are very common in single-stranded RNA viruses and have played an important role in the evolution of coronaviruses [14]. Nucleotide dot plot analyses of the Rousettus bat coronavirus versus the closest relatives in the SARS(-like) group revealed a high degree of sequence similarity in central parts of the genome, but did not identify any specific locus for recombination.\n3. Sequence and Secondary Structure of s2m\nExcluding sequences with ambiguous bases, a total of 37 different s2m genotypes were found within the coronavirus sequences (Figure 2). The great majority of the sequences could be folded into the canonical s2m stem-loop structure (Figure 3), albeit a single sequence had an insertion in the second stem-forming motif (Figure 1; Infectious bronchitis virus DPI) and some of the sequences require non-Watson-Crick base-pairing, such as the wobble base-pair G-U [15], in order to obtain the correct secondary structure. All clusters of identical s2m sequence groups were derived from viruses representing the same serogroups (beta, gamma or delta), indicating no recent transfers of the s2m element between these coronavirus clusters.\n4. Function of s2m\nFor a mobile genetic element to be successful, especially in the context of rapidly evolving genomes such as those of RNA viruses, it must offer an immediate selective advantage. s2m viruses infect higher vertebrates, making the permissive host species a lot more closely related than the viruses. The observation that s2m can apparently be transferred between unrelated viruses and remain functional (under selection to maintain sequence and structure) suggests strongly that the target for s2m is host-specific and not viral. As there are no conserved elements flanking the s2m motif in viral genomes, it is also plausible that the element has an &#8216;autonomous&#8217; function, independent of other viral genes, transcripts or genome secondary/tertiary structures. Somewhat surprisingly, the acquisition of s2m only has a subtle effect; exchanging the (non-s2m-containing) 3&#8242;-end of a murine coronavirus (MCV) with the 3&#8242;-end of a SARS virus did not appear to have any dramatic consequences [17], but 3&#8242;-ends from other coronaviruses were not able to replace the 3&#8242;-end of MCV and render it viable. There appears to be only a single sequence in GenBank with a deleted version of s2m (Infectious bronchitis virus strain ck/CH/LHLJ/07VII; accession number JF274479). This strain was discovered as an escape mutant in a vaccine development project, but did not appear phenotypically different from closely related viruses in culturing experiments [18].The folding of s2m is quite similar to the hairpin structures formed by microRNAs that are involved in RNAi-associated gene regulation, and s2m could potentially be involved in gene silencing. In vertebrates, pre-microRNA transcripts are generated in the nucleus and processed by the nuclear protein DGCR8 and the enzyme Drosha [19]. Processed pre-microRNAs are then exported from the nucleus in a process involving Exportin-5. The RNA is subsequently incorporated into the RNA-induced silencing complex (RISC) and eventually a functional RISC-bound small interfering RNA (siRNA) can serve as template for base-pairing recognition of messenger RNA (mRNA). The target mRNA is degraded, leading to a reduction in gene activity. ssRNA viruses generally replicate in the cytoplasm and earlier hypotheses on s2m function have thus been based on the assumption that the cellular components involved must be available in the cytosol. Recently, a paper was published documenting the presence of a non-canonical machinery for microRNA processing in the cytoplasm of human cells [20]. It has been well described that DNA viruses replicating in the nucleus can hijack the cellular machinery for RNAi activity for the regulation of both viral and host genes [21], but these new results using a recombinant Sindbis virus show that this is also possible for RNA viruses. The presence of multiple copies of s2m in some virus genomes [6] can be explained by there being an additive effect: more copies of the RNA will enable the formation of more siRNA/RISC complexes and give a more profound effect on target genes. A hypothesis where s2m is involved in RNAi-based gene regulation is also compatible with the target being host-specific (homologous genes in the infected species) and s2m functionality being independent of the rest of the virus genome.Observations arguing against an RNAi-like function would be that the stem region of s2m is a bit short compared to standard processed RNAi molecules (albeit mRNA binding regions as short as 17 base pairs have been reported [22]). We were also unable to find any good candidate target genes using sequence similarity searches to look for potential microRNA binding sites, and even when looking at viruses infecting relatively closely related species (such as bats, see Figure 2), there was significant variability in the s2m stem regions that could be predicted to be involved in mRNA binding. These observations make it less likely that s2m is involved in conventional RNAi-based gene silencing.Another hypothesis that would be compatible with the general assumptions regarding s2m functionality is that s2m is involved in the protection of the virus genome from being degraded by host ribonucleases. Coronaviruses are positive-sense ssRNA viruses with non-segmented genomes that are polyadenylated and capped. The genomes may serve as mRNA for the translation of viral polyproteins and contain two overlapping reading frames that encode precursor polyproteins pp1a and pp1ab through a frameshifting mechanism [23]. After infection, the viral genome is replicated and subgenomic RNAs are generated via negative-strand RNA intermediates [1,24]. In flaviviruses, another group of positive-sense ssRNA viruses, the presence of secondary structure elements has been shown to inhibit RNA decay through the formation of nuclease-resistant noncoding RNAs [25,26]. The small stem-loop structures described confer resistance to enzymatic degradation by the XRN1 5&#8242;-3&#8242; exoribonuclease. As the s2m element is near the 3&#8242; end of the coronavirus genome, inhibition of, for instance, XRN1 would only render the negative-strand versions of the genome (or subgenomic elements originating from the 3&#8242; flank) protected from decay, but resistance could also be gained through other mechanisms. Full-length coronavirus messenger RNAs share many features with endogenous transcripts and any interference with mechanisms involved in cellular mRNA turnover could also affect virus RNA stability. The exosome contains multiple proteins involved in 3&#8242;-5&#8242; degradation of RNA [27], and inhibition or stalling of, for instance, the Dis3 or Rrp6 polypeptides by s2m could also protect full-length sense copies of the genome.For some positive-strand ssRNA viruses, genome circularization has been shown to be essential for virus replication [28]. RNA circularization would require s2m to bind either directly or indirectly (mediated by RNA binding proteins) to the 5&#8242; end of the genome. The primary binding 5&#8242; site could either be a five-prime cap (5&#8242; cap) or an RNA structure (see discussion in [29]). It is possible that there are conserved structural RNA elements with such a binding affinity near the 5&#8242; end in all s2m viruses, but it seems unlikely given the complete lack of sequence similarity when comparing data from the 5&#8242; flank of these genomes. The direct involvement of a 5&#8242; cap also seems implausible, as, for instance, s2m-containing picornaviruses lack 5&#8242; capping.In order to elucidate the function of s2m, it seems imperative to establish a reverse genetics system for an s2m-containing viral strain. A good system could be an astrovirus-derived infectious construct, as there already are protocols described for the design of astrovirus cDNA clones that lead to the production of infectious viral particles [30]. Targeted mutations may be introduced and comparative studies performed using different permissive cell lines. Any changes in transcriptional activity, virus replication rates and/or viral RNA stability can be correlated with mutations interfering with the structure of s2m. Though s2m is present in some coronaviruses associated with severe infections in humans, such as SARS-CoV, there are other highly pathogenic viruses where s2m is absent (for instance the Middle East respiratory syndrome coronavirus; MERS-CoV). The significance of having s2m for a viral strain remains to be resolved, but this mobile element still seems to be in play and it is likely to be found in newly emerging coronaviruses in the future.",
        "pmid": "28933407",
        "title": "Distribution and Evolutionary History of the Mobile Genetic Element s2m in Coronaviruses",
        "journal_title": "Diseases",
        "authors": "Torstein Tengs;Christine M. Jonassen",
        "doi": "10.3390/diseases4030027"
    },
    {
        "keywords": "alpaca;human;coronavirus;reverse zoonosis;anthroponosis;respiratory;molecular evolution",
        "abstract": "In 2007, a novel coronavirus associated with an acute respiratory disease in alpacas (Alpaca Coronavirus, ACoV) was isolated. Full-length genomic sequencing of the ACoV demonstrated the genome to be consistent with other Alphacoronaviruses. A putative additional open-reading frame was identified between the nucleocapsid gene and 3'UTR. The ACoV was genetically most similar to the common human coronavirus (HCoV) 229E with 92.2% nucleotide identity over the entire genome. A comparison of spike gene sequences from ACoV and from HCoV-229E isolates recovered over a span of five decades showed the ACoV to be most similar to viruses isolated in the 1960&#8217;s to early 1980&#8217;s. The true origin of the ACoV is unknown, however a common ancestor between the ACoV and HCoV-229E appears to have existed prior to the 1960&#8217;s, suggesting virus transmission, either as a zoonosis or anthroponosis, has occurred between alpacas and humans. ",
        "body": "1. Introduction\nThe isolation of a novel coronavirus associated with an acute respiratory disease in alpacas (Alpaca Coronavirus, ACoV) has previously been described [1]. The virus was initially characterized as a Alphacoronavirus (CoV) based on gene sequencing of a small conserved region of the polymerase gene in the CoV genome. Sequence analysis phylogenetically grouped the alpaca respiratory CoV with several important animal and human coronaviruses, including transmissible gastroenteritis virus and porcine epidemic diarrhea virus of swine, feline coronavirus (FeCoV), canine coronavirus, ferret enteric virus and two common causes of upper respiratory disease in humans, HCoV-229E and HCoV-NL63. The genotype classification additionally differentiated the respiratory alpaca isolate from the CoV previously associated with New World camelids, a Betacoronavirus causing diarrhea and gastrointestinal disease [2]. While coronaviruses have been well documented for decades as important respiratory and enteric pathogens of avian [3] and mammalian species [4], and coronavirus-like particles have additionally been described in association with reptiles [5], there has been a notable increase in the number of new coronaviruses detected in recent years, with more than a dozen reported since the zoonotic SARS virus, a Betacoronavirus, was identified in 2003 [6,7]. The apparent emergence of new coronaviruses is believed, in part, to be associated with their genetic mutability and resultant adaptability to new host species [8] combined with better diagnostic tools. Among the RNA viruses, the coronaviruses possess the largest genomes, ranging from 26.4 to 31.7 kb. They are also among the most prone to homologous RNA recombination due to their unique use of random template switching during RNA replication, as well as the innate infidelity of RNA replication [6,7]. These virus characteristics combine to generate significant diversity within genotypes of single CoV species, as well as allow coronaviruses to jump host-species and adapt to new ecological niches [7,9,10]. While Alphacoronavirus transmission tends to be species-specific, experimental exposures have demonstrated that viral replication and seroconversion can occur across species [11,12,13,14,15], theoretically providing evidence of the first step for adaptation to and emergence as a pathogen in a new host species. The extent and role of the novel CoV in alpaca respiratory disease is not well understood, though strong epidemiologic associations have been made between the virus and the emergence of an acute respiratory syndrome in alpacas [1]. The purpose of this study was to complete genetic characterization of the novel alpaca respiratory CoV, through full genome sequencing and phylogenetic analysis, in order to better understand the emergence, possible origins and evolution of the newly detected virus. \n2. Results and Discussion\n\n2.1. Characteristics of ACoV Genome\nThe ACoV genome (GenBank Accn. JQ410000) is comparable to other coronaviruses in genome length (ACoV consists of 27,362 bases), the presence of a poly-A tail and an overall 38.5% G+C content (coronaviruses range between 37%&#8211;42% G+C [16]). The genome organization consists of six to sevenmajor ORFs and includes short untranslated regions (UTR) at both termini (Figure 1). The major genes are ordered 5'-replicase polyprotein (ORF1a and ORF1ab), spike gene, envelope gene, membrane gene and nucleocapsid gene. As previously reported for HCoV-229E, the sequence UUUAAAC, was also present within the ACoV genome (nt 12,526&#8211;12,532) [17]. This sequence is the putative region for a ribosomal frame shift, which creates ORF 1ab. Regions corresponding to previously described ORFs between the spike and envelope genes of other coronaviruses were present, however the ORFs were truncated and/or disrupted. A putative additional open-reading frame found between the nucleocapsid and 3'UTR, designated ORF &#8220;X&#8221;, was partially created by a 146 nucleotide insertion not present in HCoV-229E (Figure 1). A query of the ORF &#8220;X&#8221; sequence within the NCBI nucleotide database using a BLAST analysis [18] returned similarities corresponding to proteins found in Anopheles gambiae, a malaria-transmitting mosquito (34.3% similarity), and Candida tropicalis, a yeast present in the normal human flora (29.0% similarity). Most accessory ORFs do not show homology to other viral or cellular sequences when queried in public databases. Their origin is unknown, and accessory ORFs do not seem to be essential for viral replication in tissue culture [10].\n2.2. Full-Length Genome Sequence Comparison with Other Coronaviruses\nNucleotide similarity analysis of the ACoV full-length genome clearly placed the virus within the Alphacoronavirus (Figure 2). Within this group, the ACoV was most similar to HCoV-229E at the nucleotide level (92.2%) and less than 70% similar to HCoV NL63. Also, the ACoV had less than 50% genome similarity when compared to the previously reported coronavirus [2] associated with severe diarrhea in alpacas (GenBank Accn. DQ915164). Additional characteristics common to the ACoV and other alphacoronaviruses include the length of the genome, gene organization, a potential small ORF downstream of the nucleocapsid gene and an transcription-regulating sequence (TRS) with a core sequence (CS) sequence (AACUAAA) that was found throughout the genome [19]. For example, TRS was found upstream of the spike, membrane and truncated ORF4a genes. The same sequence was also found overlapping the beginning of the ORF &#8220;X&#8221; gene, while the sequence AUCUAAA was found upstream of the nucleocapsid gene. This transcription-regulating sequence with a core sequence coupled with the lack of a hemagglutinin/esterase (HE) gene clearly supports the inclusion of ACoV into the alphacoronavirus group.\n2.3. Comparison of ACoV and HCoV-229E Genes/Proteins\nDue to the overall similarity, the ACoV genome was compared with the genes from the previously described HCoV-229E (GenBank Accession no. AF304460) (Figure 1, Table 1). The nucleotide pairwise % identities ranged from 86.7%&#8211;94.2% (nucleocapsid = lowest; spike = highest). The amino acid pairwise identities ranged from 87.1%&#8211;95.0%, with the nucleocapsid having the lowest and the pol 1ab having the highest identity. \n2.4. Characteristics of the ACoV Spike Gene/Protein and Comparison with HCoV 229 E Spike Gene/Protein\nThe ACoV spike gene is 3,510 nucleotides in length and is preceded by the common transcription-regulating sequence (TRS) with a core sequence (CS) sequence for Alphacoronaviruses AACUAAA [20]. The spike gene is predicted to encode a protein of 1,170 amino acids, a class I viral fusion protein [21] containing 30 potential N-glycosylation sites, of which only 22 are predicted to be glycosylated (Figure 3) (NetNGlyc 1.0 server [22]). The pattern found in ACoV is essentially identical to the HCoV-229E spike protein. Since HCoV-229E utilizes the Aminopeptidase N protein of their natural host as receptor, a careful assumption is made that ACoV also uses the same receptor mechanism [8]. A basic amino acid cleavage site was not detected within the ACoV sequence, as has been reported for other coronaviruses [23].Due to the striking genomic similarity between the ACoV and HCoV-229E, a nucleotide comparison of spike gene sequences from different coronaviruses was performed. Again, the ACoV was most similar to HCoV-229E (Figure 4). Nucleotide comparison of spike gene sequences from the ACoV and isolates of HCoV-229E covering a span of five decades was analyzed, and the ACoV was observed to be most similar to HCoV-229E viruses isolated in the 1960&#8217;s to early 1980&#8217;s (Figure 5). \n2.5. Characteristics of the ACoV Nucleocapsid Gene/Protein and Comparison with HCoV 229 E Nucleocapsid Gene/Protein\nThe nucleocapsid protein, one of the most abundant viral proteins, is involved in replication processes, assembly and immunity during viral infection [24,25]. The ACoV nucleocapsid gene is 1,158 nt long encoding a protein of 368 amino acids. A 13.3% nucleotide difference resulting in 12.9% amino acid differences were detected between ACoV and HCoV-229E. It is interesting to note that more differences were detected between the nucleocapsid genes of the two viruses than between the spike genes. \n3. Experimental Section\n\n3.1. Cells and Viruses\nACoV was isolated out of lung tissue of an alpaca with respiratory symptoms using a continuing feline renal cell line. Cytopathogenic effects were detected as viral plaques six days post inoculation in the first passage [1]. Virus was stored at &#8722;80 &#176;C until use in small aliquots. \n3.2. Viral RNA Preparation\nViral RNA was extracted from cell culture fluid containing the ACoV using the MagMax-96 Viral RNA Isolation kit (Life Technology, Grand Island, NY, USA) or High Pure RNA Isolation kit (Roche, Indianapolis, IN, USA), as described in the manufacturer&#8217;s instructions.\n3.3. RT-PCR Amplification\nA two-step reverse transcriptase polymerase chain reaction RT-PCR was used to produce overlapping DNA fragments from the ACoV RNA (Table 1). Reverse transcriptase reactions were performed using the MonsterScript&#8482; 1st-Strand cDNA Synthesis kit (Epicentre Biotechnologies, Madison, WI, USA). Briefly, 5 &#956;L of ACoV RNA was mixed with 9 &#956;L of water and 1 &#956;L of ACoV genome specific primer (20 &#956;M) and then incubated at 65 &#176;C for 1 minute, followed by placing the tube on ice for 1 minute. Next, 4 &#956;L of 5&#215; premix was added to the mixture along with 1 &#956;L of RT enzyme. The mixture was incubated at 42 &#176;C for 5 minutes and then 60 &#176;C for 1 hour. The reaction was terminated by incubation at 90 &#176;C for 5 minutes. The cDNA was used immediately as a template for PCR or stored at &#8722;20 &#176;C for later use. For PCR, the Failsafe PCR System (Epicentre Biotechnologies, Madison, WI) was used. Briefly, 1 &#956;L of cDNA from a RT reaction was mixed with 9.25 &#956;L of water, 1 &#956;L of ACoV fragment specific forward primer (20 &#956;M) (Supplementary Table 1), 1 &#956;L of ACoV fragment specific reverse primer (20 &#956;M) (Table 1), 0.25 &#956;L of Taq enzyme and 12.5 &#956;L of the predetermined best premix buffer for each reaction. The PCR was performed using the thermocycling protocol of 95 &#176;C for 2 minutes, followed by 35&#8211;45 cycles of 95 &#176;C for 30 seconds, 50&#8211;60 &#176;C for 30 seconds, 72 &#176;C for 1&#8211;3 minutes, followed by a 7 minute elongation step at 72 &#176;C. The specific annealing temperature and elongation times were fragment-specific and varied based on the calculated melting temperature for each primer set and the size of the fragment being amplified. \n3.4. 5' RACE and 3' RACE\nDNA fragments representing the 5' and 3' ends of the ACoV genome were amplified using the FirstChoice&#174; RLM-RACE kit (Life Technology, Grand Island, NY, USA), per the manufacturer&#8217;s recommended procedure. \n3.5. Primers\nThe primer sets used to amplify the ACoV overlapping DNA fragments are listed in the Supplementary Table 1. The specific primer sets were designed to the full-length human coronavirus 229E sequence (GenBank accession # AF304460). Some of the primer sequences contain mismatches when compared to the resultant ACoV genome. The RT-PCR amplification and internal sequencing primers (sequences not shown) were designed using freely available primer selection programs available through Integrated DNA Technologies [26].\n3.6. TOPO TA Cloning\nSome RT-PCR fragments were cloned into pCR&#174; XL-TOPO (Life Technology, Grand Island, NY, USA) following the manufacturer&#8217;s recommendations to facilitate DNA sequencing and subsequent genome assembly.\n3.7. DNA Sequencing and Analysis\nThe overlapping DNA fragments corresponding to the ACoV full-length genome were sequenced by automated Sanger sequencing (Functional Biosciences, Madison, WI, USA). The genome coverage for the sequencing of the ACoV genome was approximately three-times coverage using Sanger sequencing. All DNA sequence data assembly and phylogenetic analyses were performed with Geneious Pro 5.0.3 (Biomatters, Newark, NJ, USA).\n4. Conclusions\nThe novel alpaca coronavirus described here was isolated during an outbreak of severe respiratory disease and associated abortions in alpaca in 2007, following co-mingling of a large number of animals at a national exposition and sale [1]. Subsequent large outbreaks of the disease have not been reported, leaving questions remaining as to the origins of the outbreak virus, as well the persistence of ACoV in the national alpaca population. Similarities between the whole genome sequences of ACoV and HCoV-229E, a coronavirus first isolated around 1960, [27,28] are evident. Though speculative, the similarities suggest a transmission event between humans and alpacas. Molecular studies suggest the possibility that HCoV-229E was acquired by humans from a host switching event a few hundred years ago and most likely involved bats [29]. Therefore, it is not surprising to find bat CoV isolated from Hipposideros bat species so closely grouped with the HCoV-229E and ACoV viruses in the phylogenetic analysis of the nucleocapsid gene. Although it is possible that the speculative transmission event between humans and camelids was recent, the evolutionary ancestor appears more closely related to HCoV-229E viruses isolated around 1960, and ACoVs may more realistically have been circulating for decades at low levels in alpaca populations. New world camelids were introduced into the U.S. as recently as 1985, meaning associated health management and diagnostic medicine practices are still relatively new. It is not unreasonable to assume that ACoV could have caused respiratory disease in individual animals and herds, without accurate diagnosis for decades. In the United States, alpacas are generally maintained in small herds where exposure to new infectious agents would occur primarily when unrelated naive and actively shedding animals are co-mingled. This occurred prior during the 2007 outbreak and helps to explain the large number of severely affected animals at a single point in time. Most Alphacoronaviruses, including AcoV, utilize Aminopeptidase N of their natural host as a receptor, and host tropism differences of viruses are caused by the ability of their spike proteins to detect small species-specific amino acid differences in the aminopeptidase N [8,13]. Despite the host specificity of this receptor, all of those viruses are known to bind and utilize feline Aminopeptidase N, suggestive of a common coronavirus ancestor infecting feline species [30]. This would explain the ability to grow ACoV on feline origin cell lines, such as Crandell-feline kidney cells (CRFK) [31], and the failure to propagate virus on numerous other mammalian cell lines [1].Due to the severe clinical signs associated with ACoV, the Camelid industry has expressed interest in the use of vaccination as a possible disease control option. The question was raised if a potential use of the bovine coronavirus vaccine might protect camelid species from the respiratory ACoV. Evaluating the sequence data due to potential binding mechanism of the ACoV as a Alphacoronavirus shows a completely different pattern compared to the enteric alpaca coronavirus and the bovine coronavirus, both belonging in the Betacoronavirus group. Betacoronaviruses are known for using mostly sialic acid binding activity for viral entry [32], whereas alphacoronaviruses can utilize Aminopeptidase N binding as a mechanism. Based on genome sequencing information, it is presumed that vaccines targeting Betacoronaviruses will likely not prevent infection with Alphacoronaviruses, such as AcoV, due to the different cell entry mechanism.The spike gene has the critical role of virus entry and membrane fusion processes in the host cell, however, work with chimeric viruses also shows the spike protein can influence tropism and pathogenesis of certain coronaviruses [33,34,35]. It is interesting to note that the spike genes, which tend to contain hypervariable regions, for the ACoV and HCoV-229E are more similar to each other than the nucleocapsid genes. Evolutionary studies monitoring HCoV-229E found only limited variation within the S gene nucleotide sequences in contrast to the evolutionary pattern of spike genes in other human coronaviruses [36,37], suggesting the spike protein is evolutionarily stabile. Nucleocapsid sequence comparisons among feline coronavirus isolates suggest the mutation patterns can be associated with geographic origin, rather than with virulence patterns [38]. It would be interesting to compare nucleocapsid sequence of South American HCoV-229E strains to the ACoV nucleocapsid strain.The ACoV described in this sequence analysis has several unique characteristics. The virus has a truncated or disrupted ORF 4a/4b, which has been reported for other, but not all HCoV-229E isolates [39]. Whether this ORF truncation is real or a laboratory artifact (in vitro growth in cell culture) remains to be elucidated. Also, as compared to HCoV-229E, the ACoV has an extra putative ORF &#8220;X&#8221; overlapping the 3' end of the nucleocapsid gene. Further laboratory experiments will be required to determine if this gene is expressed and, if so, the protein&#8217;s in vivo function. It is critical to characterize and understand new and emerging viruses, particularly in the context of zoonosis, whether transmitted from animal to man or man to animal. Coronaviruses, considered the most frequent source of the common cold in humans, are also widely dispersed throughout mammalian and avian populations. They have shown an ability to jump species, resulting in mild to serious zoonotic disease threats. The current ACoV genomic data provides strong evidence of an evolutionary relationship between HCoV-229E and AcoV. Whether the virus represents a recent cross-species transmission or a more distant transmission between humans and alpacas is yet to be determined. It would be interesting to analyze South American HCoV-229E isolates that were obtained in different years to determine the genetic similarities to the ACoV. This line of investigation might shed more light on the possibility of human to alpaca or alpaca to human transmission events and help to calculate when such an event might have occurred. ",
        "pmid": "23235471",
        "title": "Identification and Characterization of a Novel Alpaca Respiratory Coronavirus Most Closely Related to the Human Coronavirus 229E",
        "journal_title": "Viruses",
        "authors": "Beate M. Crossley;Richard E. Mock;Scott A. Callison;Sharon K. Hietala",
        "doi": "10.3390/v4123689"
    },
    {
        "keywords": "FCoV I;FCoV II;Seroprevalence",
        "abstract": "Background\nFeline coronavirus is comprised of two pathogenic biotypes consisting of feline infectious peritonitis virus (FIPV) and feline enteric coronavirus (FECV), which are both divided into two serotypes. To examine the prevalence of Korean cats infected with feline coronavirus (FCoV) type I and II, fecal samples were obtained from 212 cats (107 pet and 105 feral) in 2009.Results\nFourteen cats were FCoV-positive, including infections with type I FCoV (n = 8), type II FCoV (n = 4), and types I and II co-infection (n = 2). Low seroprevalences (13.7%, 29/212) of FCoV were identified in chronically ill cats (19.3%, 16/83) and healthy cats (10.1%, 13/129).Conclusions\nAlthough the prevalence of FCoV infection was not high in comparison to other countries, there was a higher prevalence of type I FCoV in Korean felines. The prevalence of FCoV antigen and antibody in Korean cats are expected to gradually increase due to the rising numbers of stray and companion cats.",
        "body": "Background\nFeline coronavirus (FCoV) is an enveloped, positive-sense, single-stranded RNA virus of the family Coronavirideae within the order Nidovirales. FCoVs are comprised of two pathogenic biotypes [1] consisting of feline infectious peritonitis virus (FIPV) and feline enteric coronavirus (FECV), which are both divided into two serotypes (Type I and II). The serotypes differ in their growth characteristics in cell culture and antigenicity, relative to canine coronaviruses (CCoVs) [2].The prevalence of type I and II FCoV have been surveyed in many countries, including Japan [3], United States [4], United Kingdom [5], Austria [6], Switzerland [7], and Taiwan [8]. The majority of field isolates in these countries are of type I, regardless of the assay method [4].FCoVs are associated with mild or subclinical enteric infections [9]. However, in a small proportion of cats, FCoV infection leads to the development of a lethal, immune-mediated condition known as feline infectious peritonitis (FIP) [10,11]. FIP is a complex immune disease involving virus or viral antigen, antiviral antibodies, and complement. Cats that do not develop anti-FCoV antibodies do not develop FIP.Serological surveys of FCoV infection have involved detection of antibody by indirect fluorescent antibody assay (IFA) or enzyme-linked immunosorbent assay (ELISA) [12-15]. A plaque-reduction neutralization test (PRNT) was developed to serologically distinguish FCoV type I and II infections in cats [16].In a previous study, the number of cats raised in Seoul, the capital city of South Korea, was reported to be around 30,000 in 2004 [17]. The most common breed in Korea is the Korean short hair cat, but this has been changing the recent years due to the increasing number of cats that are being raised as companion cats. Previously, there have been case reports of FIP in a Persian chinchilla (2 years age, female) and a Korean short hair cat (3 months age, male) [18], but no nationwide survey for FCoV has been carried out in Korea.Therefore, the objectives of the current study were to identify the seroprevalence of FCoV and to classify the FCoV serotypes in Korean cats in comparison with prevalence in other countries.\nMethods\n\nCat specimens\nFeline serum and fecal swab samples were collected from 212 cats consisting of 107 samples from six local animal hospitals (four in Seoul and two in Kyunggi) and 105 samples from two animal shelters (Incheon and Daejeon) in 2009. Of the 212 samples, 129 were from clinically healthy cats and 83 were from cats displaying symptoms of illness that included inappetence, anorexia, weight loss, lethargy, icterus, fever, diarrhea, and thoracic effusion.\nRNA extraction and reverse transcription-polymerase chain reaction (RT-PCR)\nViral RNA was extracted from fecal samples using TRIzol LSb (Invitrogen, Valencia, CA) according to the manufacturer's instructions. Two RT-PCR methods were used: one enabled the highly sensitive detection in fecal samples and the other distinguished type I from type II. The former method was based on nested PCR using primers for the amplification of the highly conserved 3'-untranslated region (3'-UTR) of the FCoV genome [19]. The latter method [8], which was based on the spike gene region of the FCoV genome, is a multiplex-nested PCR using 2 &#956;L of the first PCR product as the template and nested primers [5] for discrimination of Type I and II FCoVs. In addition, FCoV positive samples were further tested for feline panleukopenia virus (FPLV) using a specific FPLB primer set and conditions previously described [20] to identify possible relationships between the two viruses.\nIFA detection of anti-FCoV antibody\nAnti-FCoV antibody titers in feline serum samples were determined by IFA using porcine kidney-15 (PK-15) cells and porcine transmissible gastroenteritis virus (TGEV) [19]. In brief, PK-15 cells were inoculated with TGEV at a multiplicity of infection of 10 plaque forming units (PFU)/cell and incubated for 8 h at 37&#176;C. PK-15 cells were trypsinized, mixed approximately half-and-half with uninfected cells, and seeded into wells of eight-well glass plates (Nutacon, Amsterdam, The Netherlands). Cells were incubated for 60 min at 37&#176;C with serial two-fold dilutions of serum samples in phosphate buffered saline (PBS), washed three times for 5 minutes, and incubated for 1 hour with fluorescein isothiocyanate-conjugated goat anti-feline immunoglobulin G (KPL, Gaithersburg, MD).\nPhylogenetic analysis\nKorean FCoV sequences from this study and 30 sequences from global FCoV isolates obtained from GenBank were aligned using the Clustal X 1.83 sequence alignment program [21]. Nucleotide and amino acid sequence identities among the Korean FCoV isolates were calculated using BIOEDIT 7.053 [22]. The phylogenetic tree was created by a neighbor-joining method using MEGA 4.0 software [23].\nResults\nThe total number of positive fecal samples for FCoV 14 cats, which were mostly Korean short hair breeds over one year of age. Of these FCoV positive cats, three were identified to be co-infected with FPLV (Table 1).Among the 212 feline fecal swab samples, eight were positive for FCoV type I, four were positive for type II, and two were positive for types I and II (Table 2). Eleven of the positive samples were from cats displaying symptoms of infection. Three of the positive samples were from asymptomatic cats (Table 2). Among the ill cats, the infections were due to type I (n = 6), type II (n = 3) and both types I and II (n = 2). Of the three positive samples from asymptomatic cats, two were type I and one was type II (Table 2). Of the 107 specimens collected from six local animal hospitals, the nine FCoV-infected cats included type I (n = 5), type II (n = 2), and types I and II (n = 2) (Table 1). The 105 fecal swab samples obtained from two animal shelters included five that were FCoV positive (three type I and two type II; co-infection was not evident).Alternatively, canine coronavirus (CCoV) or TGEV can be used as a target antigen because of their close serological identity with FCoV. The prevalence of anti-FCoV antibody in the feline samples was low (29/212; 13.7%) and comprised 20-fold (n = 5), 40-fold (n = 6), 80-fold (n = 5), 160-fold (n = 3), 320-fold (n = 2), 640-fold (n = 4), 1280-fold (n = 3), and 2560-fold (n = 1) (Figure 1). The 29 cats having FCoV antibodies included 16 of the 83 ill cats (19.3%) and 13 of the 129 healthy cats (10.1%), and, according the location of sample collection, represented animal hospital cats (14.0%, 15/107) and shelter cats (13.3%, 14/105).A phylogentic analysis was performed using spike gene partial sequences from 46 CoVs including Korean FCoV from the current study (n = 16), FCoVs obtained globally (n = 23), CCoVs (n = 5), raccoon dog CoV (RDCoV; n = 1), and Chinese fetter badger CoV (CFBCoV; n = 1). The phylogenetic tree data set was resampled 1000 times to generate bootstrap percentage values. The phylogenic tree clearly delineated types I and II FCoVs. The analysis showed that of the ten Korean FCoVs in the type I group, nine type I FCoVs belonged to a different lineage when compared to strains isolated from other parts of the world, and only the Korean FCoV 08K-1991 was grouped with UCD1 (Japan). All six Korean type II FCoVs (08K-609, 08K-656, 08K-1154, 08K-1553, 08K-559/II and 08K-1429/II) was shown to be originated from the same lineage as F20-54-II isolated from Japan (Figure 2).\nDiscussion\nThe prevalence of type I and II FCoVs has been surveyed in many countries [3-8,16]. Similar to these other countries, the majority of FCoV infections in Korea was shown to be of type I (66.7%), with type II constituting approximately 33.3%. The type I virus is more genetically diverse than type II, perhaps reflecting its global distribution and a long period of persistence of the virus in asymptomatic cats [5,24].Previous studies have reported the proportion of infected cats that are co-infected with type I and II viruses were as high as 15.8% (6/38) in Taiwan [8] and 6.8% (5/74) in Austria [6]. In the current study, there was no incidence of co-infection with type I and II FCoV in healthy cats, whereas the prevalence of co-infection in ill Korean cats was somewhat higher (14.3%, 2/14) than that reported in cats from Austria. Investigation of both of the co-infected Korean cats revealed that they dwelled in a house that also had a dog as a companion animal. This indicates that type II FCoV may have originated from a double recombination between type I FCoV and type II CCoV [25].Previous research conducted in several countries has revealed that 69-91% of seropositive cats are suspected of having a type I infection [3,4,7]. In the present study, although the evaluation for seroprevalence by the virus neutralization test between types I and II FCoV could be not performed, FCoV seroprevalence was presently somewhat lower than the previously reported prevalences from other countries.In a study conducted in Japan in the early 1990s, a low prevalence of FCoV antibodies was detected in chronically diseased cats (21.3%) and non-diseased cats (13.4%) by IFA [3]. A more recent study in that country that utilized PRNT reported a substantially greater (three times) seroprevalence [16]. The latter authors suggested that, despite the differing analysis regimens and samples, type II FCoV infection has been decreasing in Japanese domestic cats in recent years. Accordingly, we anticipate that the use of the PRNT array for Korean feline specimens would produce a higher seroprevalence than was evident using IFA.Phylogenic analysis of the spike gene of FCoV types I and II clearly discriminated between the two types (Figure 2), in agreement with previous results [8]. The 16 Korean FCoVs in our study were classified into two serotypes, type I (n = 10) and II (n = 6) on the neighbor joining tree which was inferred using the Kimura 2 parameter model assuming uniform rates of change among sites. Geographical origin or sub-lineage classification of the FCoVs was not evident due to low nodal support value, but the perfect bootstrap percentage on the node (CCoV C8-45) indicates that our strains may have originated from this type II prototype.In this study, we examined the prevalence of Korean cats infected with type I and II FCoV and identified the seroprevalence of FCoV in Korean cats. Type I and II FCoV are not highly prevalent in comparison to the prevalence of other countries. However, in recent years Korean feline population, has been rising and it's possible their occurrence may increase in the future.\nCompeting interests\nThe authors declare that they have no competing interests.\nAuthors' contributions\nDJA: design of the experiments and writing the manuscript, JYP: help in writing the manuscript, WJ and MHL: conducted data analysis, HYJ: conducted PCR for FCoV, BKP: final correction of the manuscript. All authors read and approved the final manuscript.",
        "pmid": "21951835",
        "title": "Prevalence of Korean cats with natural feline coronavirus infections",
        "journal_title": "Virology Journal",
        "authors": "Dong-Jun An;Hye-Young Jeoung;WooSeog Jeong;Jee-Yong Park;Myoung-Heon Lee;Bong-Kyun Park",
        "doi": "10.1186/1743-422X-8-455"
    },
    {
        "keywords": "",
        "abstract": "Efficient vaccination against infectious agents and tumors depends on specific antigen targeting to dendritic cells (DCs). We report here that biosafe coronavirus-based vaccine vectors facilitate delivery of multiple antigens and immunostimulatory cytokines to professional antigen-presenting cells in vitro and in vivo. Vaccine vectors based on heavily attenuated murine coronavirus genomes were generated to express epitopes from the lymphocytic choriomeningitis virus glycoprotein, or human Melan-A, in combination with the immunostimulatory cytokine granulocyte-macrophage colony-stimulating factor (GM-CSF). These vectors selectively targeted DCs in vitro and in vivo resulting in vector-mediated antigen expression and efficient maturation of DCs. Single application of only low vector doses elicited strong and long-lasting cytotoxic T-cell responses, providing protective antiviral and antitumor immunity. Furthermore, human DCs transduced with Melan-A-recombinant human coronavirus 229E efficiently activated tumor-specific CD8+ T cells. Taken together, this novel vaccine platform is well suited to deliver antigens and immunostimulatory cytokines to DCs and to initiate and maintain protective immunity.",
        "body": "Introduction\nVaccination against viral infections has saved millions of lives by protecting many individuals from diseases such as measles, rubella, mumps, and polio. However, there is a growing need not only to develop improved vaccines against such acute infections but also to generate therapeutic vaccines which can stimulate specific immune responses to persistent viruses such as the human immunodeficiency virus or the human hepatitis C virus (1, 2). Likewise, novel approaches for vaccination against tumors which counteract the immunosuppression associated with cancer are needed (3). There is compelling evidence that CD8+ cytotoxic T cells are crucial players in the protective immune response against viral infections and tumors (4). Novel vaccine approaches should thus be rigorously evaluated for their ability to maximally expand antigen-specific CD8+ T cells, to induce their optimal differentiation into effector CD8+ T cells, and to elicit long-lasting protective memory (4).A major bottleneck in the development of new and effective vaccines is the delivery of antigens to dendritic cells (DCs) (5, 6), which sample the antigen, transport immunogenic components to secondary lymphoid organs, and initiate and maintain T and B cell responses. The excellent capacity of DCs to prime protective antiviral T cell responses has been shown in vivo (7&#8211;9). Likewise, several preclinical studies demonstrated that efficient antitumor immunity can be induced using adoptive transfer of DCs (10&#8211;12). Although individualized adoptive transfer of antigen-loaded DCs is feasible and, to a certain extent, efficient in clinical applications to humans (13), off-the-shelf vaccines that permit targeted delivery of antigens to DCs in vivo have become a major focus in vaccine development. Consequently, the description of cell surface molecules that, for example, exhibit a rather specific expression by DCs has fueled the development of antibody-based targeting strategies (14&#8211;16). These protein-based vaccines generate CD4+ T cell and B cell responses against a range of different antigens. However, antigen coupling to antibodies is a major limitation for the induction of CD8+ T cell responses which are strictly dependent on cross-presentation (5, 17). In contrast, viral vectors encoding immunogenic antigens can deliver their genetic cargo directly into DCs, thus generating antigenic peptides in infected cells and allowing for efficient loading of major histocompatibility complex (MHC) class I molecules. Among the currently most exploited virus systems that facilitate antigen delivery to DCs are adenoviral (18, 19), lentiviral (20), arenaviral (21), and alphaviral (22, 23) systems. However, major impediments of these vectors are frequent off-target transduction, resulting in antigen presentation by parenchymal cells outside secondary lymphoid organs, and limited cloning capacity for the insertion of multiple or large antigens. For example, the strong tropism of adenoviral vectors for hepatocytes, with >95% of the genetic material being deposited in the liver, leads to generation of functionally impaired CD8+ T cells (24, 25). Major efforts are thus required to engineer adenoviral vectors with improved specificity for the relevant antigen-presenting cells (12, 19). Likewise, lentiviral vectors preferentially infect cells other than DCs, and redirection of their target cell tropism is warranted (26). An additional potential impediment for the use of DNA-based viral vectors in clinics is their potential to integrate genomic material into the host genome (27).Coronaviral vectors display a number of features that clearly overcome these limitations. First, replication of these positive-stranded RNA viruses is restricted to the cytoplasm, without a DNA intermediary, making insertion of viral sequences into the host cell genome unlikely. Second, there is accumulating knowledge on how to attenuate coronaviruses in order to provide biosafe vectors (28, 29). Third, coronavirus genomes with sizes of 27 to 31&#160;kb represent the largest autonomously replicating RNAs known to date and thus offer a cloning capacity of more than 6&#160;kb. Fourth, the unique transcription process generates 6 to 8 subgenomic mRNAs encoding the four canonical structural proteins and various numbers of accessory proteins, which can be replaced to encode multiple heterologous proteins (30). Finally, and certainly most intriguing, cell surface receptors of human and murine coronaviruses are expressed on human and murine DCs, respectively (31, 32).The present study describes the generation and evaluation of rationally designed coronavirus-based vectors that efficiently target antigens and immunostimulatory molecules to DCs. We show that murine-coronavirus-based vectors can deliver multiple antigens and immunostimulatory cytokines almost exclusively to CD11c+ DCs within secondary lymphoid organs. Immunization with only low numbers of particles elicited potent CD8+ T cell responses that provided long-lasting protection against viral challenge. Moreover, single application of these novel viral vectors generated prophylactic and therapeutic immunity against metastatic melanoma. Induction of cytotoxic T lymphocytes (CTL) directed against the human Melan-A (Mel-A) antigen in HLA-A2-transgenic mice and efficient human-DC-mediated stimulation of Mel-A-specific CTL through Mel-A-recombinant human coronavirus 229E (HCoV-229E) indicates that coronavirus-mediated gene transfer to DCs represents a versatile approach for murine and human immunization against both viral infection and cancer.\nRESULTS\n\nDesign and propagation of biosafe coronavirus-based vaccine vectors.\nTo assess immunogenicity and to ensure maximal safety of coronavirus-based multigene vaccine vectors, we have rationally designed a series of prototype vectors based on the mouse hepatitis virus strain A59 (MHV-A59) (Fig.&#160;1A). With regard to safety, we introduced three basic modifications to obtain replication-competent, but attenuated and propagation-deficient, MHV-based vectors. First, we deleted all MHV accessory genes (NS2, HE, gene4, gene5a), a strategy that attenuates MHV in the natural host (29) and provides space for the introduction of heterologous genes. Second, we removed 99 nucleotides within the replicase gene-encoded sequence of nonstructural protein 1 (nsp1), because this deletion has been shown to greatly reduce MHV pathogenicity while retaining immunogenicity (28). Finally, we deleted structural gene E to restrict proper particle formation (33). Indeed, although replication of wild-type MHV to maximal titers occurred around day 5 postinfection, in vivo replication of MHV vectors could not be detected (not shown). Consequently, there was also no MHV vector replication detectable in livers or brains (not shown). The high safety profile of MHV-based vectors is further illustrated by the fact that the health status of C57BL/6 (B6) or different immunodeficient mice was not affected, even if the vectors were applied at high doses (see Fig.&#160;S1 in the supplemental material).In order to test the coronavirus-based vaccine concept and to develop a vaccine that provides strong CTL responses, in terms of both magnitude and longevity, we used the CTL epitope gp33-41 (the region from residues 33 to 41), derived from the lymphocytic choriomeningitis virus (LCMV) glycoprotein, and the Mel-A26-35 analog peptide, derived from the human Melan-A/melanoma antigen recognized by T cells 1 (MART1) protein (Fig.&#160;1A). The CTL epitopes were cloned as fusion proteins with the enhanced green fluorescent protein (EGFP) (34, 35), and the corresponding genes, designated GP-EGFP and EGFP&#8211;Mel-A, were cloned between the MHV vector-borne spike and membrane genes. Since appropriate stimulation of DCs is critical for the generation of efficient T cell responses (36) and hence an indispensable component of rationally designed vaccines, we inserted the gene encoding the murine granulocyte-macrophage colony-stimulating factor (GM-CSF) between the MHV vector-borne replicase and spike genes (Fig.&#160;1A). To propagate the MHV-based viral vectors to high titers, we produced a packaging cell line based on murine 17clone1 cells that express the MHV envelope (E) protein under the control of the Tet-off system (Fig.&#160;1B). This strategy enabled us to restore efficient vector particle formation and propagation in order to obtain vector stocks of >106/ml (Fig.&#160;1C).\nDC-specific delivery of coronavirus vector-encoded antigens and cytokines.\nTo assess MHV vector growth kinetics on cells that do not express the E protein, we transduced L929 cells, bone marrow-derived CD11c+ DCs, or peritoneal macrophages with the MHV vectors and compared growth kinetics to that of wild-type MHV-A59. In accordance with reports on E gene-deficient MHV-A59 (37), the growth of the MHV-based vector was greatly impaired in L929 cells (Fig.&#160;1D). Most importantly, however, MHV vector transduction did not result in particle release in peritoneal macrophages and DCs, demonstrating that the lack of the E protein prevents MHV propagation in primary cells (Fig.&#160;1D). MHV vectors elicited significant EGFP expression in transduced L929 cells and peritoneal macrophages (Fig.&#160;1E) and in CD11c+ DCs (Fig.&#160;1F). Notably, vector-mediated GM-CSF expression resulted in DC activation and maturation, as shown by upregulation of CD86 and CD40 on CD11c+ cells (Fig.&#160;1G). The rapid production of GM-CSF in L929 cells, peritoneal macrophages, and DCs following exposure to the cytokine-encoding vectors (Fig.&#160;S2) shows that MHV vectors can be used to simultaneously express antigens and immunostimulatory cytokines in target cells.To further determine the cellular tropism of MHV-based vectors, we transduced splenocytes from C57BL/6 (B6) mice with MHV-GM/GP in vitro and analyzed EGFP expression by flow cytometry. Neither CD4+ nor CD8+ T cells were susceptible to MHV-GM/GP transduction (not shown), whereas antigen-presenting cells, such as B cells (CD19+), macrophages (F4/80+ or CD11b+), and DCs (CD11C+), displayed green fluorescence indicative of MHV vector-mediated EGFP expression (see Fig.&#160;S3A in the supplemental material). In order to evaluate whether targeting of DCs and macrophages is also achievable in vivo, we injected (intravenously [i.v.]) 106 PFU of MHV-GP or MHV-GM/GP into B6 mice and analyzed EGFP expression of splenocytes by flow cytometry. EGFP expression was detectable mainly in CD11c+ CD11b+ DCs and CD11c+ CD8&#945;+ DCs (Fig.&#160;2A and B and see Fig.&#160;S3B in the supplemental material). Finally, we assessed the effects of MHV vector-mediated GM-CSF expression on DC stimulation and activation in vivo. Slightly elevated GM-CSF levels in spleens could be observed following application of the MHV-GM/GP vector (Fig.&#160;S4A), whereas GM-CSF levels in serum remained below the limit of detection (not shown). Concomitant immunization with MHV-GP and MHV-GM/Mel-A vectors revealed that the supply of GM-CSF in trans, i.e., via the MHV-GM/Mel-A vector was not sufficient to achieve the maximal activation of gp34-specific CD8+ T cells. Only when GM-CSF was encoded by the same vector that expressed the GP-EGFP protein (MHV-GM/GP), was the maximal immune response induced (Fig.&#160;S4B). Furthermore, MHV-GM/GP-induced GM-CSF expression did not alter the cellular composition in spleens (see Fig.&#160;S5 in the supplemental material). These results support the notion that vector-encoded GM-CSF acts locally in the microenvironment of transduced DCs. However, in vivo transduction with both MHV-GP and MHV-GM/GP mediated DC maturation, with upregulation of CD86 and CD40 (Fig.&#160;2C). Notably, application of MHV-GP vectors led to a less efficient transduction of DCs in vivo, which lasted only for roughly 18&#160;h (Fig.&#160;2A), suggesting that GM-CSF supplied by MHV vectors fosters DC survival and, to a lesser extent, DC maturation and thereby potentiates their antigen presentation function.\nAntiviral CTL responses following coronavirus vector immunization.\nInfection with LCMV is characterized by a vigorous expansion of antiviral CTL and persistence of protective memory CTL (38), which are directed against several epitopes. Two different epitopes are present in the gp33-41 region of the LCMV GP that was used in two of our constructs: the H2-Db-binding gp33-41 (39) and the H2-Kb-binding gp34-41 (40). On day 7 after LCMV infection, significant numbers of CD8+ T cells could be detected by MHC tetramer analysis and intracellular cytokine secretion (ICS) assay (Fig.&#160;3A). It is important to note that the gp33 ICS records both gp33- and gp34-specific CD8+ T cells. It appears that the processing of the GP-EGFP transgene in the MHV-GP and MHV-GM/GP vectors permitted the preferential generation of gp34-specific CD8+ T cells to a magnitude comparable to that seen during acute LCMV infection (Fig.&#160;3A). The GM-CSF-encoding MHV vector proved to be highly efficient in the induction of antiviral CTL, even at rather low doses of 103 and 104 PFU (Fig.&#160;3B). Since the intermediate dose of 105 PFU of MHV-GM/GP led to optimal induction of gp34-specific CTL, we used this dose to assess whether application via different routes would influence the induction of transgene-specific CTL. As shown in Fig.&#160;3C, all routes of immunization elicited robust CD8+ T cell responses, with intraperitoneal (i.p.), subcutaneous (s.c.), intramuscular (i.m.), and i.v. injection being the most efficient means of application. Both MHV-GP- and MHV-GM/GP-induced CD8+ T cells displayed full effector function because gp34 peptide-loaded target cells were rapidly eliminated in immunized hosts (see Fig.&#160;S6A in the supplemental material). MHV vector-induced CD8+ T cell responses were detectable for more than 65&#160;days after immunization (Fig.&#160;3D) and, most importantly, could be restimulated in vivo by immunization with the same vector (Fig.&#160;3E).Protection against LCMV challenge requires high levels of appropriately activated CD8+ T cells (38). In order to assess the efficacy of MHV vector-based immunization for protection against viral challenge, B6 mice were immunized with graded doses of the MHV-GP or MHV-GM/GP vector and challenged 7&#160;days later with LCMV. Comparable to the high level of CD8+ T cell induction (Fig.&#160;3), complete protection was observed in mice after immunization with a dose of 105 PFU (Fig.&#160;4A). Remarkably, as few as 103 PFU of MHV-GM/GP still led to a >3-log reduction of LCMV titers in spleens on day 4 postchallenge (Fig.&#160;4A). Protection against LCMV was long-lasting because mice were still protected after more than 2&#160;months following immunization with 105 PFU MHV-GM/GP via the i.v. (Fig.&#160;4B), s.c., or i.m. route (see Fig.&#160;S6B in the supplemental material). Moreover, the MHV-GM/GP vaccine elicited complete protection against i.p. challenge with LCMV GP recombinant vaccinia virus (VV) (Fig.&#160;4C). Likewise, immunization with MHV-GP provided substantial protection against this heterologous viral infection (Fig.&#160;4C). It is important to indicate that the MHV-based vaccine, even when GM-CSF was encoded by the vector, provided specific protection because replication of the unrelated recombinant vaccinia virus expressing vesicular stomatitis virus (VSV) Indiana glycoprotein (VV-INDG) was not affected (Fig.&#160;4C). Taken together, these results revealed that the MHV-based vector system is highly efficient in generating protective antiviral immunity and that the incorporation of GM-CSF into the vaccine significantly augmented its protective capacity.\nProphylactic and therapeutic antitumor immunization.\nIn order to evaluate whether MHV vector-based vaccination elicits prophylactic and therapeutic tumor immunity, we resorted to a rapidly growing B16 melanoma model which provides compatibility with the LCMV GP system through expression of a gp33 minigene (B16F10-GP cells) (41). I.v. injection of 5 &#215; 105 B16F10-GP or parental B16F10 cells in control B6 mice resulted in metastatic growth of tumor cells in lungs (Fig.&#160;5A). Immunization with either 105 PFU MHV-GP or 105 PFU MHV-GM/GP completely blocked the growth of B16F10-GP tumor cells, whereas metastasis formation of the parental B16F10 cells was not affected (Fig.&#160;5A). Application of graded doses of MHV-GP or MHV-GM/GP revealed the high efficacy of this vaccination approach in the prophylactic setting, i.e., 104 MHV-GP or only 103 PFU MHV-GM/GP were sufficient to prevent the growth of the melanoma cells (Fig.&#160;5B). A long-lasting memory response that provided significant protection against B16F10-GP challenge had been generated following MHV-GM/GP immunization (Fig.&#160;5C). Moreover, the potent CD8+ T cell response elicited through MHV-GM/GP immunization mediated therapeutic tumor immunity (Fig.&#160;5D), i.e., the tumor burden in lungs of B6 mice was significantly reduced even when the vaccine was applied after the tumors had started to form metastatic nodules, indicating that CD8+ T cell responses elicited by the novel MHV vectors can exert forceful antitumor activity.To further substantiate the finding that MHV vectors can induce potent and relevant antitumor CD8+ T cell responses, we employed the human HLA-A2-restricted Melan-A/MART1 system, in which immune responses against the Mel-A26-35 analog peptide can be monitored in HLA-A2.1-transgenic (A2DR1) mice (42). A2DR1 mice were immunized i.v. with either 105 PFU MHV&#8211;Mel-A or 105 PFU MHV-GM/Mel-A, and Mel-A-specific CD8+ T cell responses were recorded using tetramer analysis and ICS. As shown in Fig.&#160;6A, both vectors elicited substantial CD8+ T cell responses. Time course experiments following i.v. application of MHV-GM/Mel-A revealed a strong global expansion of CD8+ T cells (nearly 10-fold) and a massive expansion of Mel-A-specific CD8+ T cells, with 2 &#215; 106 cells per spleen being tetramer positive and 6 &#215; 106 cells per spleen secreting gamma interferon (IFN-&#947;) (Fig.&#160;6B). The Mel-A-specific CD8+ T cell population showed a typical contraction after day 7, and a stable memory population had been established by day 28 postimmunization (Fig.&#160;6B). During the acute phase following MHV-GM/GP immunization, >90% of the Mel-A-specific CD8+ T cells had downregulated CD62L (Fig.&#160;6C). As expected, memory CD8+ T cells reacquired CD62L expression, indicating establishment of a central memory CD8+ T cell population. Overall, these results underline that the coronavirus-based vaccination approach, particularly in combination with the immunostimulatory cytokine GM-CSF, provides efficient means for the induction of CD8+ T cell responses against a human tumor antigen.\nDelivery of tumor antigen to human DCs.\nHuman coronavirus-based vectors permit specific transfer of multiple genes to DCs because the receptor of HCoV-229E, human aminopeptidase N (hAPN or CD13) (43), is expressed on human DCs (31). Furthermore, we have demonstrated previously that HCoV-229E-based vectors can transduce immature and mature human DCs with reporter genes such as the firefly luciferase gene (30). Here we assessed whether HCoV-229E-mediated delivery of the Mel-A26-35 analog peptide to HLA-A2+ human DCs would specifically stimulate human Mel-A-specific CD8+ T cells. To this end, we generated two recombinant human coronaviruses, designated HCoV&#8211;Mel-A and HCoV&#8211;GP-EGFP, through replacement of HCoV-229E accessory gene 4 with the gene for the EGFP&#8211;Mel-A26-35 or gp33-EGFP fusion protein, respectively (Fig.&#160;7A). As shown in Fig.&#160;7B, EGFP expression was readily detectable in both human immature and mature DCs after HCoV&#8211;Mel-A infection. Time course analysis of human mature DCs infected with HCoV&#8211;Mel-A or HCoV&#8211;GP-EGFP revealed that high levels of EGFP expression can be reached as early as 9&#160;h postinfection (see Fig.&#160;S7 in the supplemental material). Finally, mature DCs infected with HCoV&#8211;Mel-A or HCoV&#8211;GP-EGFP were employed to stimulate Mel-A26-35-specific human T cells. As shown in Fig.&#160;7C, we observed efficient stimulation of the tumor-specific T cells by HCoV&#8211;Mel-A, indicating that HCoV-229E, with its pronounced tropism for human CD13-expressing DCs, represents a particularly promising tool for genetically delivering antigens to human antigen-presenting cells.\nDISCUSSION\nThis study describes a novel vaccine approach that facilitates delivery of viral or tumor antigens to DCs in vivo. Concomitant immunostimulation&#8212;here via the cytokine GM-CSF&#8212;was achieved through targeted transduction by the same viral vector. Single immunization with only 104 to 105 MHV-based particles was sufficient to elicit (i) vigorous expansion and optimal differentiation of CD8+ T cells, (ii) protective and long-lasting antiviral immunity, and (iii) prophylactic and therapeutic tumor immunity. Moreover, highly efficient antigen delivery to human DCs with recombinant HCoV-229E and specific stimulation of human CD8+ T cells revealed that this approach is exceptionally well suited for translation into human vaccine studies.Targeting of antigen to DCs in vivo can be achieved by several means (5). The use of viral vectors appears to be the superior strategy for eliciting innate activation of the immune system and optimal induction of CD8+ T cells (24). However, many virus vector systems are still limited in their ability to induce broad and long-lasting immune responses. For example, recombinant adenoviruses have been studied intensively as vaccine candidates mainly because they can be produced to high titers. Nevertheless, high doses of recombinant adenovirus vectors have to be applied to induce an immune response, most probably because they target antigens mainly to nonlymphoid organs, such as the liver (25, 44). In contrast to viral vectors based on DNA viruses (45), positive-stranded RNA virus-based vectors that replicate in the cytoplasm are considered safe vectors. The safety is well documented for vectors based on widely used vaccine strains, such as poliovirus (46), or virus-like particles (VLPs) that contain replicon RNAs devoid of structural genes (47). It should be noted that application of the MHV-based vectors described in this study was always well tolerated and did not result in any sign of disease or adverse side effect irrespectively of dosage and route of MHV-based vector application, even in immunodeficient mouse strains. Finally, it is noteworthy that although some RNA virus-based vectors are able to target DCs (21), their cloning capacity is, in sharp contrast to that of coronavirus-based vectors, generally restricted and the expression of multiple antigens and/or immunostimulatory cytokines is limited.The novel coronavirus-based vaccine platform described here displayed a high immunogenicity. It is very likely that it is the pronounced tropism of the MHV-based vectors for DCs and macrophages within secondary lymphoid organs that guarantees efficient activation of primary CD8+ T cell responses. In vivo imaging studies have shown that CD169+ CD11c+ DCs at the subcapsular sinus of lymph nodes efficiently present viral antigen to na&#239;ve T cells (48). Likewise, CD169+ macrophages at the same location are able to collect viral antigen from the lymph and present antigen to follicular B cells (49). Moreover, a recent study from our laboratory has revealed that type I IFN-mediated protection of DCs and macrophages from cytopathic effects of MHV infection is essential to buy time for mounting a protective CD8+ T cell response (50). It remains to be resolved which factors&#8212;besides the presence of the MHV receptor on DCs (32)&#8212;confers the preferential infection of the relevant antigen-presenting cells within secondary lymphoid organs. For further adaptation of the coronavirus-based vectors to the human system, some of the essential parameters have been clarified; i.e., the receptor of HCoV-229E is expressed mainly on monocytes and DCs within secondary lymphoid organs (31), and recombinant HCoV-229E efficiently targets human DCs irrespective of their maturation status. It is important to add that HCoV-229E is one of at least four human coronaviruses that are transmitted via mucosal surfaces and are associated with mild upper respiratory tract infections (common cold). Furthermore, the low pathogenic potential of HCoV-229E is corroborated by the fact that HCoV-229E was passaged in volunteers in the 1960s without any serious side effects. These features, together with the pronounced DC-targeting capabilities demonstrated in this report, should encourage the development of coronavirus-based vectors for human use.A second major advantage of the coronavirus-based vaccination strategy is the large cloning capacity of the vectors, which offers the possibility of incorporating immunostimulatory cytokines. GM-CSF-encoding MHV vectors led to strong production of this cytokine in both macrophages and DCs in vitro. Interestingly, GM-CSF expression in vivo appeared to be largely restricted to locally transduced cells, as no gross differences in GM-CSF levels in spleen or serum were observed after MHV-GM/GP vector immunization. It is most likely that GM-CSF-induced changes within the microenvironment of transduced macrophages and DCs are decisive for the optimal induction of effector and memory CD8+ T cell responses. Indeed, it is the optimally stimulated expression of costimulatory molecules on antigen-presenting cells, together with sufficient innate immune stimulation, that determines the primary expansion and the maintenance of antiviral CD8+ T cells (51). Accordingly, such non-TCR signals (&#8220;signals 2 and 3&#8221;) are considered key components of rationally designed vaccines (4). The lack of such optimally composed stimuli in a vector-based vaccine most likely requires very elaborate prime-boost immunization regimes, as was recently shown for a combined adenovirus/MVA vaccination approach (52), and/or substantially increased vector doses to achieve efficacy, as for example in vaccination with the latest versions of DC-adapted lentivirus vectors, where application of 5 &#215; 107 to 10 &#215; 107 particles was required to achieve significant expansion of ovalbumin-specific CD8+ T cells in mice (53).In conclusion, we provide here a versatile vaccine platform based on coronaviruses that achieved the efficient generation of protective immunity, shown as long-lasting memory against viral challenge and induction of both prophylactic and therapeutic tumor immunity. The biology of coronaviruses and the rational modification of these viral RNA vectors harbor significant potential for future development.\nMATERIALS AND METHODS\n\nEthics statement.\nExperiments were performed in accordance with federal and kantonal guidelines under permission numbers SG07/62, SG07/63, 07/64, and 07/71 following review and approval by the Kantonal Veterinary Office (St. Gallen, Switzerland).\nMice, cells, and viruses.\nC57BL/6 mice were obtained from Charles River Laboratories (Sulzfeld, Germany). A2DR1 mice were kindly supplied by F. A. Lemonnier (Pasteur Institute, Paris, France) (42). All mice were maintained in individually ventilated cages and were used at between 6 and 9&#160;weeks of age. L929 and CV-1 cells were purchased from the European Collection of Cell Cultures. MC57 and BSC40 cells were obtained from R. M. Zinkernagel (University of Z&#252;rich, Switzerland). D980R cells were a kind gift from J. F. Smith (Imperial College London, London, United Kingdom). 17clone1 cells were a kind gift from S. G. Sawicki (Medical University of Ohio, Toledo, OH). Huh7 cells were a kind gift from V. Lohmann (University of Heidelberg, Germany). The LCMV WE strain was obtained from R. M. Zinkernagel (Universit&#228;t Z&#252;rich, Switzerland). Recombinant vaccinia virus expressing LCMV glycoprotein 2 (VV-G2) was originally obtained from B. H. Bishop (University of Oxford, United Kingdom), and recombinant vaccinia virus expressing vesicular stomatitis virus Indiana glycoprotein (VV-INDG) was originally obtained from B. Moss (National Institutes of Health, Bethesda, MD). Titration and determination of antiviral protection assays have been performed as described previously (7). Further details can be found in Text S1 in the supplemental material.\nCloning and generation of recombinant MHV-based vectors and human coronaviruses.\nGenomic RNA of recombinant MHV-based vectors and recombinant HCoVs were produced from cloned cDNA using purified vaccinia virus DNA as the template for in vitro transcription as described previously (54). A detailed description of all cloning steps, sequence information, and production of recombinant coronavirus particles can be found in Text S1 in the supplemental material.\nIsolation of dendritic cells and macrophages, flow cytometry, immunofluorescence.\nGeneration of bone marrow-derived DCs and flow cytometric analysis were performed as described previously (50). Further information on these procedures can be found in Text S1 in the supplemental material. For immunofluorescence analysis of viral replication, 2 &#215; 105 L929 cells or peritoneal macrophages were seeded in slide chambers and transduced with MHV-GP or MHV-GM/GP (MOI = 1). At the time points indicated in Fig. 1, the slides were acetone fixed, blocked with the Fc-blocking antibody 2.4G2, and stained with anti-EGFP Alexa 488 (BioLegend). Images were acquired using a Leica DMRA microscope and processed using Adobe Photoshop (Adobe Systems).\nTetramer analysis and intracellular cytokine staining.\nPeptide-specific CD8+ T cells and ex vivo production of IFN-&#947; or tumor necrosis factor alpha (TNF-&#945;) were determined by tetramer staining, and intracellular cytokine staining was performed as described previously (24, 28). Detailed information on these procedures can be found in Text S1 in the supplemental material.\nMelanoma model.\nB16F10-GP melanoma cells expressing the LCMV gp33 epitope (41) and parental B16F10 cells were kindly provided by H. Pircher (University of Freiburg, Germany). Further information on tumor protection assays can be found in Text S1 in the supplemental material.\nGeneration of human DCs and ICS.\nDCs were generated from monocytes isolated from peripheral blood mononuclear cells as described previously (55). Mature DCs were infected with Mel-A/EGFP-recombinant HCoV-229E or GP-EGFP-recombinant HCoV-229E and incubated with Mel-A-specific T cell clones as responder cells. Further details on in vitro human CD8+ T cell activation by transduced human DCs can be found in Text S1 in the supplemental material.\nStatistical analysis.\nAll statistical analyses were performed with Prism 4.0 (GraphPad Software Inc.). Data were analyzed with the nonpaired Student t&#160;test under the assumption that the values followed a Gaussian distribution. A P&#160;value of <0.05 was considered significant.",
        "pmid": "20844609",
        "title": "Dendritic Cell-Specific Antigen Delivery by Coronavirus Vaccine Vectors Induces Long-Lasting Protective Antiviral and Antitumor Immunity",
        "journal_title": "mBio",
        "authors": "Luisa Cervantes-Barragan;Roland Z\u00fcst;Reinhard Maier;Sophie Sierro;Jozef Janda;Frederic Levy;Daniel Speiser;Pedro Romero;Pierre-Simon Rohrlich;Burkhard Ludewig;Volker Thiel",
        "doi": "10.1128/mBio.00171-10"
    },
    {
        "keywords": "SARS-CoV;MERS-CoV;coronavirus;acute respiratory distress syndrome;ARDS;acute lung injury;type II pneumocytes",
        "abstract": "Respiratory viruses can cause a wide spectrum of pulmonary diseases, ranging from mild, upper respiratory tract infections to severe and life-threatening lower respiratory tract infections, including the development of acute lung injury (ALI) and acute respiratory distress syndrome (ARDS). Viral clearance and subsequent recovery from infection require activation of an effective host immune response; however, many immune effector cells may also cause injury to host tissues. Severe acute respiratory syndrome (SARS) coronavirus and Middle East respiratory syndrome (MERS) coronavirus cause severe infection of the lower respiratory tract, with 10% and 35% overall mortality rates, respectively; however, >50% mortality rates are seen in the aged and immunosuppressed populations. While these viruses are susceptible to interferon treatment in vitro, they both encode numerous genes that allow for successful evasion of the host immune system until after high virus titres have been achieved. In this review, we discuss the importance of the innate immune response and the development of lung pathology following human coronavirus infection.",
        "body": "Introduction\nAcute lung injury (ALI) and its more severe form, acute respiratory distress syndrome (ARDS), can arise after many types of injury to the lung, including sepsis, mechanical and chemical injury and bacterial and viral infections 1. In ALI, the mortality rate is in the range 20&#8211;30%, with about 55% of the cases progressing to ARDS within a few days. ARDS causes significant morbidity and approximately 40% mortality, resulting in &#8764;75 000 deaths/year in the USA alone 2. In the past two decades, five emerging viruses have been known to cause significant ARDS-related mortality, including influenza H1N1 2009 and, in particular, the highly pathogenic avian influenza H5N1 and H7N9 viruses and the SARS and MERS coronaviruses. In this review we focus on mechanisms of coronavirus-induced lung pathogenesis and ARDS.Human coronavirus (CoV) infections have traditionally caused a low percentage of annual upper and lower respiratory infections 3, including severe disease outcomes in the elderly, immunocompromised patients and infants. HCoV-OC43 (OC43) and HCoV-229E (229E) were the first documented human CoVs but, more recently, HCoV-NL63 (NL63) 4 and HCoV-HKU1 (HKU1) 5 were identified as a consequence of increased viral surveillance efforts in the early twenty-first century (Table&#8201;1). These four viruses usually cause acute infection of the upper respiratory tract and less frequently are associated with lower respiratory tract 6,7 diseases as well. Severe disease is both rare and typically associated with co-morbidities and/or immunosenescence. The past 15 years have also seen the emergence of two new human coronaviruses that cause significant disease and mortality. SARS-CoV was identified in 2003 and caused an acute, atypical pneumonia and diffuse alveolar damage (DAD) in roughly 8000 patients 8,9. Those over 65 years of age often developed ARDS, resulting in mortality rates that exceeded 50%. Overall, SARS-CoV infection caused nearly 800 fatalities, representing a nearly 10% mortality rate. More recently, in 2012, a new human coronavirus, designated MERS-CoV, was identified. MERS-CoV continues to circulate in camels and humans, with over 857 official cases and 334 deaths, representing an approximately 35% case fatality rate to date in humans 10,11. MERS-CoV-induced disease is particularly severe in aged patients and those with pre-existing co-morbidities. MERS-CoV does not appear to be highly pathogenic or virulent in camels.SARS-CoV and MERS-CoV have clear zoonotic origins, although their exact paths from animal reservoir to human infection are not yet clear. Viruses with high nucleotide identity to SARS-CoV were found in key amplifying hosts such as palm civets and raccoon dogs in Guangdong Province China during the 2002&#8211;2003 SARS epidemic 12. Later studies identified highly conserved viruses circulating in horseshoe bats, including some strains that are able to bind to, and infect, human cells 13&#8211;16. The existence of novel bat SARS-like coronaviruses that also use bat, civet and human angiotensin 1 converting enzyme 2 (ACE2) receptors for entry, such as SARS-CoV, strongly suggests an opportunity for further zoonotic disease outbreaks in human and animal populations.SARS causes an atypical pneumonia characterized by cough, fever and infiltrates with a ground-glass appearance on X-ray 17,18. Early-stage disease was characterized by acute DAD, with oedema, fibrin and hyaline membranes in the alveolar spaces, typical of ALI 19. Other patients predominantly showed an acute fibrinous and organizing pneumonia pattern or a mixture of the two patterns 20,21. Longer-term disease courses typically progressed to organizing phase DAD and eventual deposition of fibrous tissue. Autopsy of fatal SARS-CoV cases also revealed denuded airways, haemorrhage and increased macrophage populations in the lung 22,23. During the SARS epidemic, researchers noted that late-term disease progression was unrelated to viraemia but was more likely to be associated with immunopathological damage 24.MERS-CoV has caused sporadic infections, along with several local outbreaks throughout the Middle East since its discovery in 2012 25,26. Although much remains unknown, closely related viruses have been isolated from camels 27 and highly homologous MERS-like bat CoVs have been identified in African Neoromicia capensis bats 28. Local surveillance efforts have detected high levels of antibodies that recognize MERS-CoV in dromedary camels 29; furthermore, sampling of archived camel serum samples has revealed MERS antibodies from as early as 1992 30. These data suggest that bat to camel to human transmission routes may have seeded the 2012 outbreak in human populations, perhaps associated with the expanding camel trade that has emerged between equatorial Africa and Saudi Arabia over the past 20 years.Animal models of human disease should recapitulate many of the pathological and immune outcomes seen in human infections. Numerous models have been established to better enable our understanding of the mechanics of SARS-CoV infection and pathogenesis, although few recapitulate the human disease phenotypes (Table&#8201;2). Initial studies utilized late epidemic strains in non-human primates 31&#8211;33, where mild to severe disease was observed, depending on the study location and animal age. To date, the differences in disease severity noted in primates have not been reconciled, but may reflect differences in virus strains or infection conditions. Although still under development, MERS-CoV replication and disease have been reported in both rhesus macaques and common marmosets 34,35. SARS-CoV replication resulted in limited disease in young models of immunocompetent mice 36&#8211;38; however, mild clinical disease was noted in 1 year-old mice 39. A mouse-adapted SARS (MA-SARS) strain was also developed that provides a model for moderate to lethal disease, depending on infectious dose, animal age and genetic background of the host 40&#8211;42 (Table&#8201;2). The MA-SARS model faithfully replicates the age-dependent susceptibility observed in human patients, as well as key features of human lung pathology, including virus tropism to airway epithelial cells and type II pneumocytes, pneumonia, hyaline membrane formation, development of DAD and denudation of airway epithelial cells 40,43. A limitation may be the rapid clearance of virus titres that is seen in younger and, to a much lesser extent, in aged animals. Development of the MA-SARS model has allowed for in-depth studies of viral pathogenesis and the host immune response, taking advantage of immunological tools and reagents for the mouse as well as the existence of knockout mouse strains. Use of these tools has greatly added to our understanding of SARS-CoV pathogenesis, far beyond what could be learned in in vitro experiments or observational studies of human cases. Because of receptor incompatibilities, MERS-CoV does not replicate in mice unless the animals are first transduced with adenovirus vectors encoding the receptor for entry, human dipeptidyl peptidase-4 (DPP4) 44.In this review we focus solely on hCoV interactions within the context of the respiratory system and infection of relevant cell types. More specifically, we review some CoV&#8211;host interactions that alter cell-intrinsic antiviral defence programmes and other host pathways that contribute to pathological findings of ARDS, with its associated exudative and organizing phase diffuse alveolar damage and pulmonary fibrosis.\nInnate immune response\nNF-&#954;B signalling is an important component of numerous cellular responses, including stress, cytokine signalling, response to bacterial or viral infection and apoptosis 45,46. The SARS-CoV envelope (E) protein stimulates NF-&#954;B signalling 47, leading to lung cytokine signalling and inflammatory cell recruitment. The SARS-CoV papain-like protease (PLP) has also been shown to antagonize NF-&#954;B signalling 48\nin vitro. Chemical inhibitors of NF-&#954;B signalling reduce lung pathology and inflammation following MA-SARS infection, demonstrating the importance of this pathway 47 in pathogenesis. While the SARS-CoV E protein is not required for viral replication, it is important for inhibition of the host cellular stress response, apoptosis and unfolded protein response 49,50. The E protein, along with the SARS-CoV ORF3a and ORF8a proteins, has ion channel activity 50 and may contribute to vascular permeability and fluid accumulation in the lung following SARS-CoV infection. SARS-CoV lacking E has been shown to be an effective vaccine 51,52 and a MERS-CoV clone lacking E has been generated 53, although replication requires expression of E in trans.SARS-CoV, and to a greater extent MERS-CoV, are highly sensitive to interferon treatment in cell culture. Interestingly, SARS-CoV pathogenesis does not significantly change in various type I interferon (IFN) knockout mouse models, except for a slight increase in overall virus titres 54&#8211;56. Despite this, STAT1- and Myd88-deficient mice are significantly more vulnerable to lethal outcomes following infection 56,57. Like many viruses, CoVs encode a suite of genes that antagonize cell-intrinsic innate immune defence programmes in the infected host cell (reviewed in 58). Numerous in vitro studies have demonstrated the IFN antagonist activity of both SARS-CoV and MERS-CoV proteins 59&#8211;61, and a detailed review of SARS-CoV evasion of the innate immune response was recently published by Totura and Baric 62.Analysis of IFN-stimulated gene (ISG) expression in Calu-3 human airway epithelial cells highlighted the ability of SARS-CoV and MERS-CoV to avoid detection by the host 63. As compared with influenza A viruses, ISG transcripts and proteins are not induced until late after SARS-CoV and MERS-CoV infection, when peak titres have already occurred in culture (&#8764;18&#8211;24&#8201;h). Late in infection, ISGs showed nearly universally increased expression following SARS-CoV infection, except for ACE2 and Serping1. However, a much larger subset of ISGs had significantly decreased expression following MERS-CoV infection. Like MERS-CoV, H5N1 VN1203 infection also resulted in significant down-regulation of subsets of ISGs. No consistent pattern in up-regulation or down-regulation of gene expression correlated with transcription factor usage, suggesting that a novel mechanism may be responsible for expression of the ISG subsets. Cells infected with MERS-CoV and H5N1 avian influenza were shown to have specifically altered open and closed chromatin structures, potentially limiting the ability of transcription factors to access and bind certain ISG promoter regions. The mechanism by which MERS-CoV induces this chromatin structural alteration is as yet unknown. In contrast, the NS1 protein of H5N1 was responsible for the chromatin changes in influenza-infected cells. Although speculative, it seems likely that many RNA viruses may encode strategies to epigenetically alter host chromatin structure, influencing host gene expression. This newly identified method of ISG control requires additional study.SARS-CoV further evades the host immune response by masking its RNA genome. This mechanism may be partially mediated by the production of double-membrane vesicles, which could sequester RNA replication intermediates away from the host-sensing machinery 64,65. MDA5 and IFIT1 are important host antiviral sensor or antiviral defence ISGs that detect viral RNAs. IFIT1 recognizes unmethylated 2&#8242;-O RNA 66 and alters efficient translation/stability of uncapped viral mRNAs 67. SARS-CoV and other coronavirus RNAs are protected from IFIT recognition because they encode a 2&#8242;-O-methyltransferase (2-OMT) activity in the viral replicase protein, nsp16 68,69. SARS-CoV is much more sensitive to interferon treatment in the absence of functional nsp16 methyltransferase activity, and mutant viral titres drop rapidly in both infected epithelial cells and mice. Deletion or knockdown of either MDA5 or IFIT1 restored mutant SARS-CoV viral loads demonstrated the essential role of these host proteins in detecting pathogen-associated molecular patterns. Ablation of the 2-OMT activity may provide a universal strategy to rationally design live attenuated mutants of contemporary and newly emerging CoV.Both in vivo and in vitro studies have addressed the role of specific proteins in the innate immune system, often ISGs, in SARS-CoV pathogenesis. Transcriptional analysis on autopsy tissue from SARS-CoV-infected patients revealed increased expression of STAT1 along with other IFN-induced cytokines 70. The SARS-CoV accessory protein ORF6 was identified as an interferon antagonist important for viral replication in low multiplicity of infection (MOI) in vitro infections 71,72. ORF6 was subsequently found to sequester Karyopherin 2&#945;, a nuclear import factor, and block the nuclear translocation of STAT1 after SARS-CoV infection. Interestingly, STAT1 translocation to the nucleus is not blocked in MERS-CoV-infected cells, so it remains uncertain whether antagonists of nuclear import are encoded in the viral genome 73. Transcriptional profiling of SARS-CoV-infected macaques revealed robust IFN signalling, including STAT1 translocation to the nucleus, in the lung but not in the cells that stained positive for viral antigen 74. These data highlight the importance of in vivo studies versus high MOI in vitro studies. Significantly, they also highlight the need to examine, or at least consider, expressing and signalling differences in specific cell types instead of global transcriptomic studies in those in vivo experiments.STAT1 knockout mice have been studied extensively in the context of viral infection, typically showing a heightened susceptibility to disease, due to the lack of a type I IFN response 75. These knockouts were first tested for SARS-CoV susceptibility using the Tor2 strain in a sublethal model; animals deficient in STAT1 were unable to clear virus from the lung and developed a more severe and longer-lasting pneumonia than the control mice 76. Frieman et al showed that STAT1 knockout mice are highly susceptible to infection with MA-SARS in a novel, IFN-independent mechanism 56. MA-SARS-infection causes massive inflammatory cell influx in the lungs of STAT1 knockout mice, including large numbers of macrophages, neutrophils and eosinophils. STAT1 knockout mice have gross pathological changes in their lungs, including massive haemorrhage as well as increased lung size and stiffness. As seen in some humans, these mice develop severe pulmonary fibrosis and succumb to disease at late time points after infection. Stained lung sections revealed the presence of collagen protein in alveolar exudates in STAT1 knockouts, indicating development of early-stage pulmonary fibrosis. Subsequent studies demonstrated that STAT1 knockout animals developed a Th2-skewed immune response and had significant numbers of alternatively activated or M2 macrophages in their lungs 77. These macrophages were characterized by positive CD11c, arginase and mannose receptor staining. STAT6 is required for the development of alternatively activated macrophages, and STAT1/STAT6 double knockout mice do not develop the severe lung disease and pro-fibrotic lesions observed in the STAT1 single knockout 78, thus demonstrating that these macrophages are essential for development of the pulmonary fibrosis phenotype. Further elegant experiments showed that it is the STAT1 deficiency in monocyte/macrophage cells, not the infected epithelial cells, that drives alternatively activated macrophage production and induction of fibrotic lung disease following SARS-CoV infection. Alternatively activated macrophages are typically induced by the Th2 cytokines IL-4 and IL-13; they have an anti-inflammatory role and play an important role in wound-healing processes 151.ACE2 is expressed on well-differentiated airway epithelial cells 79 and its expression increases following type I interferon treatment 63. Both ACE2 protein levels and RNA expression are down-regulated after either in vitro or in vivo SARS-CoV infection 63,80 and NL63 also down-regulates ACE2 expression following in vitro infection 81. It has previously been reported that ACE2 and angiotensin-2 protect mice from sepsis- and acid aspiration-induced ALI 82. Additionally, histopathological lung disease worsens when spike-Fc is inoculated into mice with ALI 80. The normal function of ACE2 is to inactivate angiotensin-2, a negative regulator of the renin&#8211;angiotensin system 83,84. This system controls blood pressure and is involved in the development of pulmonary hypertension and pulmonary fibrosis. The renin&#8211;angiotensin system is involved in lipopolysaccharide (LPS)-induced neutrophil recruitment to the lung 85. Multiple genome-wide association studies have investigated an association between genetic variation in ACE and susceptibility to ARDS, with mixed results 86. The role of DPP4 in MERS-CoV infection is discussed in detail by Haagmans et al in a separate review in this issue 152.High ISG expression has been linked to development of ARDS 87. Furthermore, it has been suggested that unregulated IFN responses contributed to development of immunopathology and severe disease following SARS-CoV infection 88. The data discussed above support this hypothesis and suggest that early control of ISG signalling may be a means of preventing or controlling the development of severe lung disease. Expression of the ISG Serping1 is also decreased following SARS-CoV infection of epithelial cells; it functions by inhibiting the complement system as well as several proteases in the coagulation pathway. The role of the coagulation, fibrinolysis and wound healing in ARDS development are discussed below.\nAcute respiratory distress syndrome (ARDS), coagulation, fibrinolysis and respiratory function\nThe alveoli of the lung are where gas exchange occurs, providing oxygen to blood flowing through capillaries in the alveolar membrane. The alveolar walls are composed of type I and type II pneumocytes, along with alveolar macrophages 89. Type I pneumocytes cover 95% of the alveolar surface area and allow for gas exchange with blood in the capillaries of the lung. Type II pneumocytes are the progenitors of type I pneumocytes and are also responsible for generating pulmonary surfactant 90, a mixture of lipids and surfactant proteins that is crucial in reducing surface tension in the lung. SARS-CoV infection causes desquamation of pneumocytes in humans and mice, contributing to alveolar dysfunction, oedema and haemorrhage. Alveolar macrophages play an essential role in surveillance of the local environment and inhibit an excessive immune response, although this inhibition can also block an effective response to SARS-CoV infection 91. The functions of these cell types are critical in maintaining balance between inflammation, coagulation and wound repair, especially following lung injuries such as viral infection 92.In ARDS patients, uncontrolled inflammation, fluid accumulation and developing fibrosis severely compromise gas exchange and lead to respiratory failure. SARS-CoV and influenza infect type I and type II pneumocytes in the lung 93,94. ARDS patients exhibit decreased surfactant levels 95 and MA-SARS infection results in decreased surfactant transcript and protein levels 43. Decreased surfactant, and the consequent increase in surface tension, reduces the ability of the lung to expand and contract during normal respiration; it also heightens the risk of lung collapse during expiration. Respiratory dysfunction occurs when the alveolar membranes are obstructed, or when the ability of the lung to expand and contract, circulating oxygenated air, is compromised. Lethal SARS-CoV infection in the mouse and human is characterized by a breakdown of alveolar membrane integrity, resulting in accumulation of fluid exudates in the alveolar spaces. Virus infection also results in an overwhelming cytokine response, severe lung tissue damage and respiratory failure 96&#8211;99. The progression from initial disease to diffuse alveolar damage and the exudative and organizing stage of DAD is often independent of high-titre viral replication 24, indicating that this severe disease outcome is primarily driven by an immunopathological response, including inflammatory cell recruitment and viral damage to type II pneumocytes. This conclusion is further supported by non-human primate and mouse models of SARS-CoV infection, where lethal disease is more often associated with severe pulmonary lesions, alveolar exudates and respiratory dysfunction than with high viral load 43,100. MA-SARS infection results in peak viral titres at 1&#8211;2 days post-infection, along with airway denudation and resulting debris, which can occlude the small airways. Severe lung disease, including inflammatory cell infiltrates, haemorrhage, alveolar oedema and hyaline membrane formation typical of the exudative stage of DAD (Figure&#8201;1), occurs at days 4&#8211;7 post-infection, when virus loads in the lung are dropping rapidly and/or are below the limit of detection. Many ISGs stimulated by SARS-CoV infection are involved in wound-healing responses and thus may contribute to SARS-induced ALI and ARDS.While SARS-CoV evades detection by the host immune system and causes minimal changes in transcript and protein levels for the first 24&#8201;h of infection 43, it ultimately induces a massive signalling response in infected lungs. Pro-inflammatory cytokines and chemokines, including IL-6, TNF&#945;, IL-1&#946; and CCL2 57, recruit inflammatory cells to the site of infection. Neutrophils and cytotoxic T cells, along with these cytokines, can induce tissue damage, including vascular leakage, and stimulate pulmonary fibrosis 101. Pro-fibrotic genes, including Tgf&#946;1, Ctgf and Pdgfa and numerous collagen transcripts, have increased expression following MA-SARS infection. Fluid exudates, haemorrhage and fibrin are all observed in the alveolar spaces of SARS patients, as well as in animal models of disease 17,43,102, increasing in severity as a function of age. In response, the coagulation cascade is activated, including increased factor 10 (FX), F2, F3 (tissue factor), F11, F12 and F7 transcript levels. Activation of the coagulation cascade results in F10 cleavage of prothrombin into thrombin and subsequent thrombin cleavage of fibrinogen into fibrin 103. Fibrin clots in the alveoli are a prominent feature of SARS-CoV infection in humans and mice. The goal of this coagulation response is likely to protect the host by sealing the alveoli, preventing alveolar flooding and haemorrhage, which limit oxygen exchange and endanger patient survival. Collagen expression is also increased following SARS-CoV infection 43. Collagen accumulation, fibrin and fibrin clots all contribute to a developing fibrotic lung state, while at the same time stimulating the infected host to up-regulate fibrinolytic pathways 92.Profibrinolytic genes include members of the urokinase pathway, such as urokinase (plau), tPA and plasmin (plg) 104. The urokinase signalling pathway leads to cleavage and activation of plasmin into plasminogen; this protease then cleaves fibrin clots. Serpine1 and Serpine2 are negative regulators of urokinase pathway and inhibit urokinase and tissue plasminogen activator (tPA) activity. Urokinase signalling is highly active in the absence of Serpine1, and this imbalance often results in haemorrhage in knockout mice. Serpine1 is highly expressed in SARS patients, non-human primates and small animal models 43,74,105. ARDS studies, independent of coronavirus infection, have attributed this Serpine1 expression to alveolar macrophages and type II pneumocytes 106,107. MA-SARS infection in a Serpine1 knockout mouse model results in lethal disease with extreme lung pathology 43. Conversely, MA-SARS-infected mice with a genetic deficiency in tPA have increased exudates in the lung. The dysregulation of these coagulation/anti-coagulation cascades can result in worsening end-stage lung disease conditions, resulting in death.Profibrotic and profibrinolytic signalling are part of the wound-healing response, along with other extracellular matrix (ECM) remodelling pathways 101. SARS-CoV infection causes massive tissue remodelling through urokinase and coagulation pathways activity, as discussed above. Other important wound-healing pathways and ECM proteins with altered signalling following SARS-CoV infection include matrix metalloproteinases, EGFR and collagens 43. Successful recovery from ALI requires a delicate balance of pro-inflammatory, profibrotic and profibrinolytic responses. By altering ISG expression, including ACE2, STAT1 and Serpine1, SARS-CoV infection of alveolar epithelial cells sets the stage for the development of severe lung disease, including ARDS.SARS and MERS patients with severe lung disease exhibited lung consolidation, decreased blood oxygen saturation and often required intubation and ventilation 99,108,109. Small animal models of severe lung disease typically lack physiological readouts of respiratory function that can be directly correlated back to human signs of disease. Whole-body plethysmography captures respiratory data in unrestrained animals, allowing for longitudinal measurement of pulmonary function; these data can also be directly related to some human respiratory metrics 110,111. SARS-CoV infection causes increased Penh, a calculated measure of airway resistance and increased EF50 (mid-breath exhalation force), indicating that respiratory function is compromised and animals must do more work to breathe 69. Unpublished data (Gralinski and Menachery) indicate that it is the exhalation portion of each breath that is impacted by SARS-CoV infection, likely due to extensive debris clogging the conducting airways. Further experiments have shown that Stat1 knockout mice have increased Penh levels early after infection, and that at late timepoints they have reduced lung capacity, corresponding with the profibrotic histopathological changes observed in the lung (Gralinski, unpublished data).\nConcluding thoughts\nARDS is a devastating end-stage lung disease with no cure. Despite numerous clinical trials, improved clinical outcomes have remained marginal at best 112. Several highly pathogenic emerging virus infections cause ARDS with high frequency, underscoring the critical public health importance of understanding the virological components and molecular mechanisms that drive this devastating end-stage lung disease. Furthermore, a portion of ARDS cases may progress to pulmonary fibrosis, another clinically devastating end-stage lung disease with few treatment options. Consequently, understanding the development of ARDS following viral infection remains a high-priority research topic that is germane to global health and pandemic disease control. The twenty-first century has demonstrated that zoonotic events will continue to introduce coronaviruses and other viruses into the human population, and that these viruses have the potential to spread rapidly, cause significant disease in communities and disrupt the global economy. An emerging theme is the connectivity between virus infection, complement and coagulation cascade activation, pro-inflammatory and profibrotic cytokine responses and disease severity. More studies are needed to unravel the complex interactions between these pathways that can interact to promote or dysregulate wound recovery after life-threatening respiratory virus infection. In particular there is a need for including well-articulated animal models that faithfully recapitulate disease processes across species. Only through a better understanding of the interplay between a dysregulated host immune response and ALI and ARDS can more effective treatments and therapeutics be developed.",
        "pmid": "25270030",
        "title": "Molecular pathology of emerging coronavirus infections",
        "journal_title": "The Journal of Pathology",
        "authors": "Lisa E Gralinski;Ralph S Baric",
        "doi": "10.1002/path.4454"
    },
    {
        "keywords": "",
        "abstract": "As of January 2016, 1,633 laboratory-confirmed cases of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) infection and 587 MERS-related deaths have been reported by the World Health Organization globally. Middle East Respiratory Syndrome Coronavirus may occur sporadically in communities or may be transmitted within families or hospitals. The number of confirmed MERS-CoV cases among healthcare workers has been increasing. Middle East Respiratory Syndrome Coronavirus may also spread through aerosols generated during various dental treatments, resulting in transmission between patients and dentists. As MERS-CoV cases have also been reported among children, pediatric dentists are at risk of MERS-CoV infection. This review discusses MERS-CoV infection in children and healthcare workers, especially pediatric dentists, and considerations pertaining to pediatric dentistry. Although no cases of MERS-CoV transmission between a patient and a dentist have yet been reported, the risk of MERS-CoV transmission from an infected patient may be high due to the unique work environment of dentists (aerosol generation).",
        "body": "MERS-CoV infection in healthcare workers\nThe 3 patterns of MERS-CoV transmission suggested in the literature include the following: a) sporadic cases occurring in communities; b) transmission within families; and c) nosocomial transmission.20 Healthcare-related MERS-CoV transmission is associated with high morbidity, extended use of mechanical ventilation and fatality rates of up to 65%; this type of transmission can be attributed to shortcomings in observing stringent infection control protocols.4,6,15,21 Furthermore, MERS-CoV has been reported to be viable in hospital-like environments for up to 48 hours with a stability that is unaltered during aerosolization.22 Among 200 healthcare workers&#8217; contacts identified in Al-Hassa, Saudi Arabia, MERS-CoV infection was confirmed in 2 cases.4 A Saudi study20 compared healthcare worker and family contact with laboratory-confirmed MERS-CoV patients and reported a lower rate (1.12%) of infection among the healthcare workers than among the families (3.4%).Healthcare workers could be infected with MERS-CoV through exposure in the community or at their workplace,1,6,20 they could be diagnosed late,1,6 and they could remain asymptomatic or mildly symptomatic.6,8,16 Furthermore, unsuspected cases entering healthcare facilities have been considered the main source of MERS-CoV.4 Considering these aspects and that healthcare workers may continue to work regardless of being symptomatic,15 the possibility of transmitting the infection to their patients is also high.\nMERS-CoV infection in children\nMemish et al1 reported that the age of 11 Saudi pediatric patients who presented clinically with symptoms of MERS-CoV infection ranged from 2-16 years with a female-to-male ratio of 2.7 to 1. Of the 11 cases, 2 were symptomatic, and 9 were asymptomatic with no underlying comorbidities. Another report by Thabet et al17 concluded that although MERS-CoV infection presents with a wide range of clinical manifestations, the mortality rate in children is lower than that in adults. A review on the effects of Coronavirus infection in children concluded that Human Coronavirus infections may be associated with respiratory symptoms and may also involve central nervous system. The authors suggested that the clinical and genetic traits of Human Coronavirus infection among children should be monitored closely for early prevention.23\nDental considerations. Bioaerosols in dental practice\nBioaerosols are defined as a suspension of biological particles in gaseous media.24 Apart from radiation exposures, dermatitis, eye injuries, and musculoskeletal and respiratory disorders, other occupational health risks exist in dentistry. These risks include percutaneous exposure incidents and exposure to infectious diseases, such as those that may be present in bioaerosols.25 Subgingival scaling of periodontally involved teeth with ultrasonic scalers may produce aerosols containing blood.26 One study reported that ultrasonic scalers and tips produced significantly more aerosol and splatter than a handheld curette, regardless of the scaler type used.27 Miller et al28 studied the characteristics of blood-containing aerosols generated by common powered dental instruments. The author concluded that all the recovered particles could contain the hepatitis B virus and be inhaled and retained (20-100%) in the human respiratory system.The repeated and chronic exposure to bioaerosols generated during certain dental procedures and the relatively small particle size contribute to an increased risk of infection among dental professionals.29 Furthermore, the protection provided by surgical masks worn by dental professionals may be low for small particles; in addition, these masks may not fit perfectly in clinical practice.30 A study reported that 15-83% of plasma aerosol particles ranging from 0.06-2.5 microns in size passed through the filters of 9 makes of surgical masks used by dentists.28 These factors may play a role in the airborne route of viral infections among dentists and clinical dental auxiliaries.The likelihood of detecting, reporting, documenting and publishing dentistry-associated infections is relatively less.31 Although no definitive evidence of an extensive public health hazard from exposure to dental unit waterlines has been reported.32 Few case reports have suggested a definitive link between exposure to contaminated dental unit waterlines and Legionella infection.33,34\nInfection control in dental practice\nA review by Scully and Samaranayake35 on the emerging and fluctuating viral diseases in the new millennium concluded that infection control plays an equally important role in the practice of dentistry as do the understanding of oral manifestations and the diagnosis and management of viral infections. While differences exist in the virology, epidemiology, and clinical outcomes of MERS-CoV and SARS-CoV infections,36 the clinical symptoms of MERS-CoV resemble those of SARS-CoV, apart from acute renal failure.37 Although these results cannot be directly interpreted for a definitive conclusion, the management protocols for different MERS-CoV infection-associated with clinical scenarios for dental professionals may be similar to those of SARS-CoV.The implications of SARS-CoV for general dental practitioners, the significance of droplets and aerosols in disease transmission, the recommended management protocols for SARS-CoV infection and specific infection control measures have been well described by Li et al.38 A comprehensive review on universal and standard precautions has been published elsewhere39 and is beyond the scope of this article, infection control measures pertaining only to pediatric dental practice in the context of MERS-CoV infection are discussed here.In pediatric dental practice, effective infection control measures for the prevention or minimization of viral infection transmission can be implemented by a) controlling the gag or cough reflex; b) reducing aerosol/splatter generation; c) managing contaminated air and; d) improving personal protection. The gag or cough reflex may be stimulated by certain procedures, such as posterior intraoral and bite-wing radiographs and taking impressions. Orthopantomographs or oblique lateral views may be considered instead of intraoral radiographs for screening,40 whereas oral mucosa in very sensitive patients may be anesthetized before taking impressions.41 Sedation may also be considered to control gag reflex.42Varying levels of physical, intellectual, emotional and social development of children and adolescents are major challenges for dentists especially attending to their psychological needs within adequate infection control regimen. Toys provided for pediatric patients may be a potential source of cross-infection. Soft toys are more likely to be contaminated, difficult to disinfect and may re-contaminate quickly compared to hard-surfaced toys. Furthermore, restraining devices used to control movements of pediatric patients such as Velcro fasteners may also be contaminated and should be disinfected accordingly.43 Extra-oral evacuation devices and special aerosol reduction devices may be used in combination with ultrasonic scalers to reduce the amount of aerosol produced.44,45 In high-risk cases, chemomechanical caries removal46 or the atraumatic restorative technique47,48 may be utilized to prevent or reduce aerosol and splatter generation. In addition, high-volume evacuation removes infectious droplets at the source as they are emitted; thereby, minimizing, or preventing their dispersion in the air. To maintain their efficacy, the filters in the suction apparatus should be cleaned every day, and the exhaust air should be vented outside to prevent the recirculation of contaminated air.38Contaminated air can be managed by improving dental clinic ventilation and/or by disinfecting the air. An ideal airflow pattern combined with a minimum of 3 air changes per hour has been recommended for dental clinics.49-51 Moreover, although its use in dental clinics is unconfirmed, ultraviolet germicidal irradiation may be installed and is effective against fungi, viruses, and bacteria, namely, tubercle bacilli and anthrax.38,49 On the other hand, measures of improving personal protection include washing hands frequently before and after treatment, using disposable barriers, dispensing instruments and materials just before treatment (thereby preventing particles from settling on the surfaces), and sterilizing soiled instruments.38 After each patient visit, surfaces may be disinfected using hospital-grade disinfectants, which are effective against coronavirus.52 Personal protective equipment, such as gowns, hair covers, masks, gloves, shielded face masks and shoe covers, should be used as appropriate.51 A higher level of respiratory protection should be considered, especially with aerosol-generating procedures. The use of N95 respirators offers a certain level of protection against the airborne transmission of SARS-CoV, although the exact level of protection offered by these respirators for individuals may vary.53 Examples of personal protective equipment such as face mask with plastic shield (Figure 1) and N95 respirator (Figure 2). These respirators may also provide some level of protection against MERS-CoV and may be utilized instead of surgical masks in dental clinics. Apart from these measures, vaccination of pediatric dentists and staffs working in pediatric dental clinics against measles, mumps, varicella and rubella may be necessary for their protection from these viral infections.54In conclusions, considering the unique work environment of dentists, which involves close patient contact and aerosol production, the risk of MERS-CoV transmission from an infected patient is high. Children are also prone to MERS-CoV infection. As the number of MERS-CoV cases may increase in future, pediatric dentists should be well informed and educated about not only the signs and symptoms of the condition but also how to follow stringent infection control measures in these cases.",
        "pmid": "28397938",
        "title": "Middle East respiratory syndrome in children",
        "journal_title": "Saudi Medical Journal",
        "authors": "Fares S. Al-Sehaibany",
        "doi": "10.15537/smj.2017.4.15777"
    },
    {
        "keywords": "Coronavirus;COVID-19;Pneumonia;Tertiary Infection;Viral Load;Real-Time Reverse-Transcriptase Polymerase Chain Reaction",
        "abstract": "Since mid-December of 2019, coronavirus disease 2019 (COVID-19) has been spreading from Wuhan, China. The confirmed COVID-19 patients in South Korea are those who came from or visited China. As secondary transmissions have occurred and the speed of transmission is accelerating, there are rising concerns about community infections. The 54-year old male is the third patient diagnosed with COVID-19 in Korea. He is a worker for a clothing business and had mild respiratory symptoms and intermittent fever in the beginning of hospitalization, and pneumonia symptoms on chest computerized tomography scan on day 6 of admission. This patient caused one case of secondary transmission and three cases of tertiary transmission. Hereby, we report the clinical findings of the index patient who was the first to cause tertiary transmission outside China. Interestingly, after lopinavir/ritonavir (Kaletra, AbbVie) was administered, &#946;-coronavirus viral loads significantly decreased and no or little coronavirus titers were observed.",
        "body": "Ethics statement\nMyongji Hospital Institutional Review Board (IRB) approved this study (No. IRB 2020-01-027) and written informed consent was given by the patient.",
        "pmid": "32056407",
        "title": "Case of the Index Patient Who Caused Tertiary Transmission of Coronavirus Disease 2019 in Korea: the Application of Lopinavir/Ritonavir for the Treatment of COVID-19 Pneumonia Monitored by Quantitative RT-PCR",
        "journal_title": "Journal of Korean Medical Science",
        "authors": "Jaegyun Lim;Seunghyun Jeon;Hyun-Young Shin;Moon Jung Kim;Yu Min Seong;Wang Jun Lee;Kang-Won Choe;Yu Min Kang;Baeckseung Lee;Sang-Joon Park",
        "doi": "10.3346/jkms.2020.35.e79"
    },
    {
        "keywords": "2019-nCoV;Wuhan pneumonia;metagenomic next-generation sequencing;phylogenetic analyses;virus evolution",
        "abstract": "ABSTRACT\nFrom December 2019, an outbreak of unusual pneumonia was reported in Wuhan with many cases linked to Huanan Seafood Market that sells seafood as well as live exotic animals. We investigated two patients who developed acute respiratory syndromes after independent contact history with this market. The two patients shared common clinical features including fever, cough, and multiple ground-glass opacities in the bilateral lung field with patchy infiltration. Here, we highlight the use of a low-input metagenomic next-generation sequencing (mNGS) approach on RNA extracted from bronchoalveolar lavage fluid (BALF). It rapidly identified a novel coronavirus (named 2019-nCoV according to World Health Organization announcement) which was the sole pathogens in the sample with very high abundance level (1.5% and 0.62% of total RNA sequenced). The entire viral genome is 29,881&#8197;nt in length (GenBank MN988668 and MN988669, Sequence Read Archive database Bioproject accession PRJNA601736) and is classified into &#946;-coronavirus genus. Phylogenetic analysis indicates that 2019-nCoV is close to coronaviruses (CoVs) circulating in Rhinolophus (Horseshoe bats), such as 98.7% nucleotide identity to partial RdRp gene of bat coronavirus strain BtCoV/4991 (GenBank KP876546, 370&#8197;nt sequence of RdRp and lack of other genome sequence) and 87.9% nucleotide identity to bat coronavirus strain bat-SL-CoVZC45 and bat-SL-CoVZXC21. Evolutionary analysis based on ORF1a/1b, S, and N genes also suggests 2019-nCoV is more likely a novel CoV independently introduced from animals to humans.",
        "body": "Introduction\nThe epidemic of emerging infectious diseases worldwide poses a great threat to public health. It is noted that most epidemic is caused by viral cross-species transmission from animals to human. Surveillance by fast and accurate diagnostic methods is crucial for the disease control and patient treatment. Thanks to the development of metagenomic next-generation sequencing (mNGS) methodology, the direct investigation of infectious microorganism from original clinical samples is currently achievable [1]. Particularly, RNA based mNGS approach could simultaneously reveal the entire &#8220;infectome&#8221; (i.e. RNA viruses, DNA viruses, bacteria and eukaryotes) present within an organism, because all except for prion express RNA [2]. Furthermore, RNA sequencing goes beyond pathogen identification to reveal relevant data on pathogen abundance, genome sequence, and gene expression, providing important insight into the cause of disease such that it represents an avant-guard diagnostic tool in the information age. Recently, in December 2019, an outbreak of unusual pneumonia caused by unknown infection was reported in Wuhan, China [3]. The earlier cases (before January) were all linked to Huanan Seafood Market in Wuhan before the disease was further spread to other cities of China and even overseas. Since this is an outbreak with unknown etiology, we report the use of RNA based mNGS approach for a rapid identification and characterization of a potential pathogen, which is therefore of great importance for disease control and prevention.\nMaterials and methods\n\nEthics statement\nThis study was approved by the Ethics Committee of the Zhongnan Hospital of Wuhan University. The mNGS analyses of BALF samples were performed on existing samples collected during standard diagnostic tests, posing no extra burden to patients.\nSequence of events\n2nd January 2020. Obtained BALF samples from two patients with unusual pneumonia.3rd January 2020. Performed SARS-specific RT-PCR assay, yielded partial RdRp fragment, and revealed potential pathogen.4th January 2020. Extended RdRp fragments and obtained more genome fragments, and started mNGS RNA library preparation5th January 2020. Completed mNGS RNA library preparation.6th January 2020. Started mNGS sequencing on Miseq platform.7th January 2020. Received sequencing data, started pathogen identification pipeline, obtained virus genome, corrected the genome end with mapping, identified 2019-nCoV as sole pathogen, and the final CoV genome was 29,881&#8197;nt.8th January 2020. Performed genome comparisons and evolutionary analyses.Since 3rd January 2020, instant progress reports have been sent to Chinese Center for Disease Control and Prevention (CDC), keeping pace with every advancement we made in pathogen identification and characterization.\nLibrary preparation and sequencing\nTotal RNA extracted from BALF samples (collected on 2nd January 2020) were subject to metagenomic next-generation sequencing (mNGS) testing. The concentration of RNA samples were low (<0.5&#8197;ng/ul) based on measurement by Qubit RNA HS Assay Kit (Thermo Fisher Scientific), and therefore the library preparation was performed with Trio RNA-Seq kit (NuGEN Technologies, USA) which targeted low concentration RNA samples and contained AnyDeplete probe that removes human ribosomal RNA. The resulting libraries were subject to 150&#8197;bp pair-end sequencing with an Illumina Miseq platform. The sequencing results were obtained in less than 24&#8197;h.\nPathogen discovery and characterization\nTo identify potential pathogens from the mNGS sequencing results, a pathogen discovery pipeline was carried out on sequenced data. Briefly, reads containing adaptor sequences and low-complex regions were removed from the dataset. Human reads were also removed by mapping against the reference human genome. All non-human and non-repeat sequence reads were then compared to a reference virus database (downloaded from https://ftp.ncbi.nih.gov/blast/db/ref_viruses_rep_genomes.tar.gz) and the non-redundant protein database (nr) using blastn and diamond blastx programs [4], respectively. Taxonomy lineage information was obtained for each blast hits by matching the accession number with the taxonomy database, which was subsequently used to identify reads of virus origin. Bacterial pathogen identification was carried out by using the Metaphlan2 program [5].Reads were also assembled de novo using Megahit [6], with the virus genome identified based on the blast procedure described above. To validate the assembled genome sequences, reads were subsequently mapped to the genomes and a majority consensus sequences were determined for each sample. Minor variation calling was performed after mapping using Genious software package, with a minimum coverage set to 20 and minimum variant frequency set to 0.05. In addition to mapping, the virus genomes were also confirmed with Sanger sequencing using primers designed based on the NGS sequences.\nPhylogenetic and recombination analyses\nReference sequences associated with CoVs were downloaded from GenBank and aligned using mafft program. Phylogenetic trees (both amino acid and nucleotide alignment) were reconstructed using the maximum likelihood method in PhyML 3.0 [7], employing a best fit substitution model and a SPR branch swapping algorithm. Recombination event were discovered from phylogenetic analyses and confirmed with similarity plot implemented in the Simplot program [8].\nResults and discussion\nOn 2nd January 2020, samples were collected from two unusual pneumonia patients from Zhongnan Hospital of Wuhan University. Patient 1 was a 39-year-old male staff at Huanan Seafood Market who experienced fever (up to 37.7&#176;C) and aggravated cough with frothy white sputum for 5 days before admitted to the hospital on 25th December 2019. Patient 2 was a 21-year-old female who developed an intermittent febrile cough, chills, fever (up to 40&#176;C), and frothy&#8194;white&#8194;sputum after having a contact with Huanan Seafood Market staff on 22nd December 2019. She was admitted on 28th December after unsuccessful outpatient treatment. The results of clinical laboratory test on the first day of hospitalization are listed in Table 1. Chest CT scan of both patients showed patchy pulmonary opacities below the pleura in the bilateral lung field (Figure S1), which suggests viral infections may occur in both lungs. However, the subsequent routine anti-viral and anti-infection treatment did not alleviate their symptoms. On 31st December 2019, patient 1 had more severe symptoms, including poor mental states, shortness of breath, and 86% SpO2 without oxygen inhalation. A CT re-examination showed mild pleural effusion in the left lung, an increase in the density of ground-glass opacities, and an extension of the patchy area. The patient later experienced Type I respiratory failure on the same day. On 2nd January 2020, both patients were transferred to Wuhan Infectious Diseases Hospital for continuing treatment. To the date this manuscript was prepared, patient 1 and patient 2 were later discharged from the hospital in stable condition on 12th January and 11th January 2020, respectively.\nTable 1.Clinical laboratory test on the first day of hospitalization.ItemsCase 1Case 2Normal range of lab testWBC, &#215;109/L5.232.893.5&#8211;9.5Neutrophils, &#215;109/L/L3.581.921.8&#8211;6.3T lymphocyte, &#215;109/L1.320.461.1&#8211;3.2Hb, g/L138.6127.5115&#8211;150Platelet, &#215;109/L170117125&#8211;350Albumin, g/L65.94740&#8211;55AST, U/L92337&#8211;45ALT, U/L303013&#8211;45CK, U/L3635<171CK-MB, U/L11100&#8211;25LDH, U/L313247110&#8211;245UREA, mmol/L2.812.72.8&#8211;7.60CREA, &#956;mol/L73.957.249&#8211;90Definition of abbreviations: ALT&#8201;=&#8201;alanine aminotransferase; AST&#8201;=&#8201;aspartate aminotransferase; CK&#8201;=&#8201;creatine kinase; CK-MB&#8201;=&#8201;creatinine kinase&#8211;MB isoenzyme; CREA&#8201;=&#8201;creatinine; UREA&#8201;=&#8201;Urea nitrogen; Hb&#8201;=&#8201;haemoglobin; LDH&#8201;=&#8201;lactate dehydrogenase; WBC&#8201;=&#8201;white blood count.On 3rd January 2020, respiratory and blood samples obtained from the patients were subjected to routine clinical laboratory tests for respiratory pathogens, including Influenza virus, Respiratory syncytial virus, Adenovirus, Metapneumovirus, Mycoplasma pneumonia, Chlamydophila pneumonia, and Legionella, all yielding negative results. The remaining RNA samples were first subjected to SARS-CoV specific RT-PCR assays recommended by World Health Organization (WHO). However, only one set yielded positive results (Figure 1A). Further sequencing of the corresponding PCR product surprisingly suggested that the virus discovered is more closely related to BtCoV/4991 (97.35%) but not SARS-CoV (Figure 1B).\nFigure 1.1st-round of RT-PCR assay, amplification and sequence analysis of unusual pneumonia outbreak in Wuhan. (A) RNA samples were subjected to SARS-CoV specific RT-PCR primer sets as indicated, only the SAR1-s/as set showed obvious band. Lane 1, 6, 11, 16, 21 are samples of patient 1. Lane 2, 7, 12, 17, 22 are samples of patient 2. Other lanes are samples of other patients who are irrelevant to this study. (B) The Blast result of PCR products of patient 1 and 2.On 4th January 2020, in 2nd-round RT-PCR assay, extended RdRp fragments and more genome fragments were identified, amplified, sequenced and analysed using new set of primers that were designed based on the 1st-round Blast analysis (Figure 2). These data further suggest that the pathogen of unusual pneumonia might be a coronavirus but not SARS-CoV. Meanwhile, total RNA extracted from BALF samples (collected on 2nd January 2020) were subject to metagenomic next-generation sequencing (mNGS) library construction.\nFigure 2.2nd-round of identification of unusual pneumonia. (A) RNA samples were subjected to multiple primer sets for different genes as indicated. Lane 1, 3, 5, 7, 9, 11, 12, 13, 14, 15, 21, 23, 25, 27 are samples of patient 1. Lane 2, 4, 6, 8, 10, 16, 17, 18, 19, 20, 22, 24, 26, 28 are samples of patient 2. (B) The PCR product of patient 1 and 2 were sequenced and the Blast result is shown.On 5th January 2020, the mNGS library construction was completed.On 6th January 2020, the resulting libraries were subject to 150&#8197;bp pair-end sequencing with an Illumina Miseq platform.On 7th January 2020, the sequencing results were obtained in less than 24&#8197;h, with 7,369,020 and 4,522,558 reads generated for the samples of patient 1 and 2, respectively. To identify potential pathogens from the mNGS sequencing results, a pathogen discovery pipeline based on individual reads was carried out on sequenced data. Aside from those belonged to PhiX genome (in-library control), a majority of the viral reads (99.9% and 99.7% respectively for sample 1 and 2) were associated with coronaviruses. The raw sequence data minus human genomic information was uploaded to Sequence Read Archive (SRA)&#160;database (Bioproject accession PRJNA601736). On the other hand, bacterial pathogen identification was carried out by using the Metaphlan2 program, which revealed Capnocytophaga sp and Veillonella sp in sample 2 and none in sample 1, and both bacteria identified were not known for their pathogenicity. Collectively, coronavirus is likely to be the main microbial pathogen within these samples. The reads were assembled de novo using Megahit to form a &#8764;30&#8197;kb contigs with sequence homology to CoV. After confirmation with read mapping, the final CoV genome was 29,881&#8197;nt.On 8th January 2020, the genome comparisons and evolutionary analyses were performed. Although some single nucleotide polymorphism (SNP) profiles were identified in the mNGS data (Table 2), the consensus genome sequences obtained from the patient 1 and 2 were identical (GenBank MN988668 and MN988669, respectively). These results indicated that these two individual patients were infected by the same CoV at separate times. We named the two clinical isolates as 2019-nCoV strain WHU01 and WHU02, respectively, according to WHO announcement. Based on the results of genome mapping, our data revealed extremely high viral abundance within the samples: the average genome coverage was 523.6X and 133.7X and the estimated abundance level were 1.5% and 0.62% of total reads sequenced for patient 1 and 2, respectively, suggesting active coronaviral replication in the lungs of both patients.\nTable 2.Minor nucleotide variant identified from WHU01 and WHU02 genomes.StrainRegionVariantStart PoisitonEnd PositionLengthChangeCoveragePolymorphism TypeVariantFrequency (%)P-valueWHU011aT2212211C&#8201;&#8594;&#8201;T27SNP (transition)14.806.70E-07WHU011aA110311031T&#8201;&#8594;&#8201;A119SNP (transversion)5.005.40E-14WHU011aA182018201G&#8201;&#8594;&#8201;A97SNP (transition)11.302.00E-27WHU011aG391639161A&#8201;&#8594;&#8201;G113SNP (transition)5.303.90E-14WHU011aTT391939202AA&#8201;&#8594;&#8201;TT110Substitution5.501.30E-13WHU011aT392339231C&#8201;&#8594;&#8201;T108SNP (transition)5.603.00E-14WHU011aT570157011C&#8201;&#8594;&#8201;T247SNP (transition)5.305.70E-29WHU011aG889288921A&#8201;&#8594;&#8201;G69SNP (transition)5.805.40E-10WHU011aA889588951T&#8201;&#8594;&#8201;A65SNP (transversion)6.204.20E-10WHU011aG897589751A&#8201;&#8594;&#8201;G59SNP (transition)5.106.50E-08WHU011aC911491141T&#8201;&#8594;&#8201;C43SNP (transition)7.007.70E-07WHU011a&#160;11,08111,0811(T)8&#8201;&#8594;&#8201;(T)778Deletion (tandem repeat)12.801.20E-20WHU011aC13,07413,0741T&#8201;&#8594;&#8201;C110SNP (transition)5.503.30E-14WHU011aTT13,28213,2832AA&#8201;&#8594;&#8201;TT78&#8201;&#8594;&#8201;79Substitution5.109.40E-10WHU011bA15,07915,0791C&#8201;&#8594;&#8201;A57SNP (transversion)8.801.30E-13WHU011bT18,25218,2521A&#8201;&#8594;&#8201;T192SNP (transversion)6.305.50E-23WHU011bT19,16319,1631C&#8201;&#8594;&#8201;&#8594;&#8201;T89SNP (transition)19.101.90E-47WHU011bA20,23420,2341C&#8201;&#8594;&#8201;A67SNP (transversion)6.001.20E-09WHU01SA22,31522,3151G&#8201;&#8594;&#8201;A182SNP (transition)6.604.70E-28WHU01SA22,44722,4471C&#8201;&#8594;&#8201;A54SNP (transversion)5.602.00E-07WHU01SC24,32224,3221A&#8201;&#8594;&#8201;C325SNP (transversion)38.500WHU01Other ORFA26,31326,3131G&#8201;&#8594;&#8201;A29SNP (transition)10.301.50E-08WHU021aT110011001C&#8201;&#8594;&#8201;T390SNP (transition)6.701.50E-56WHU021aA110311031T&#8201;&#8594;&#8201;A391SNP (transversion)5.903.10E-51WHU021aA182018201G&#8201;&#8594;&#8201;A382SNP (transition)5.201.00E-41WHU021aC682368220+C129Insertion5.402.50E-16WHU021aA10,77810,7781T&#8201;&#8594;&#8201;A323SNP (transversion)5.302.40E-32WHU021aT11,36611,3661A&#8201;&#8594;&#8201;T250SNP (transversion)6.004.40E-31WHU021aT11,56211,5621C&#8201;&#8594;&#8201;T397SNP (transition)13.601.30E-138WHU021bT13,69213,6921A&#8201;&#8594;&#8201;T356SNP (transversion)7.001.60E-57WHU021bC14,30614,3061T&#8201;&#8594;&#8201;C279SNP (transition)7.909.20E-50WHU021bA14,31514,3151G&#8201;&#8594;&#8201;A244SNP (transition)10.706.90E-57WHU02Other ORFA26,50426,5041G&#8201;&#8594;&#8201;A63SNP (transition)6.301.50E-10Since 3rd January 2020, instant progress reports have been sent to the Chinese Center for Disease Control and Prevention (CDC), keeping pace with every advancement we made in pathogen identification and characterization.The genomes of the 2019-nCoV were further analysed to determine its origin and evolutionary history. Full genome comparisons indicated that 2019-nCoV is close to CoVs circulating in Rhinolophus (Horseshoe bats). For example, it shared 98.7% nucleotide identity to bat coronavirus strain BtCoV/4991 (GenBank KP876546, only 370&#8197;nt sequence of RdRp gene) and 87.9% nucleotide identity to bat CoV strain bat-SL-CoVZC45 and bat-SL-CoVZXC21, indicating that it was quite divergent from the currently known human CoV, including SARS-CoV (79.7%). To put 2019-nCoV in the context of whole Coronaviridae family, we aligned ORF1b protein sequences from representative CoVs diversity for phylogenetic analyses (Figure 3A). It revealed that the 2019-nCoV is grouped under genus &#946;-coronavirus, subgenus Sarbecovirus, and a cluster that is known to harbour bat-SL-CoVs, many of which were associated with Rhinolophus sp. (horseshoe bats).\nFigure 3.Origin and evolutionary history of newly identified CoVs. A. the position of 2019-nCoV in the context of all reference CoVs. The phylogeny is constructed based on ORF1b protein alignment. For clarity, names were only shown for human-associated viruses. Bat associated diversity is shaded with blue and green boxes for alpha- and beta-CoVs respectively. B. genome structure of newly identified viruses and its sequence similarity against bat-SL-CoVZC45 and SARS-CoV in a 1000bp sliding window across the entire genome. Recombination breakpoints are shown as dashed vertical lines. C. the relationship of WHU viruses with the other SARS-like CoVs. Phylogeny is reconstructed based on the nucleotide sequence of four genes: namely 1a, 1b, S, and N. Those grouped with WHU at S gene are marked red, and those grouped with SARS CoVs at S gene are marked blue.To reveal a more detailed relationship between 2019-nCoV and other CoVs, we reconstructed phylogenies based on nucleotide alignment of key viral genes, including ORF1a/b, S, and N. Within this cluster, the 2019-nCoV also shared close relationship with CoVs originated from Rhinolophus bat. For ORF1b gene, the closest relative is BtCoV/4991 (KP876546, 98.65% nucleotide identity, based on partial RdRp gene comparisons) identified from Rhinolophus affinis from Yunnan; whereas for the rest of the genes analysed, the closest are bat-SL-CoVZXC21 (76.5&#8211;91.2% nucleotide identity) and bat-SL-CoVZC45 (76.9&#8211;91.2% nucleotide identity) identified from Rhinolophus sinicus. The close relationship with BtCoV/4991 is quite essential in tracing the potential reservoir host of 2019-nCoV. Unfortunately, the BtCoV/4991 sequence was only partial (373bp in length) and thus no comparisons can be made for the rest of genomes. However, the presence of such close relatives in bat viruses strongly suggests that it might be originated from a recent and independent introduction from bats to humans, although its immediate hosts remain to be identified.Through gene-specific phylogenetic analyses, we also identified phylogenetic incongruence for 2019-nCoV compared with other bat-SL-CoVs at different genes, suggesting potential recombination event. Specifically, 2019-nCoV was closely related to strains bat-SL-CoVZXC21 and bat-SL-CoVZC45 at ORF1a, S, and N genes, but not at ORF1b gene. At ORF1b gene, bat-SL-CoVZXC21 and bat-SL-CoVZC45 were related to strains Longquan-140 and HKU3-10 (Figure 2C). Simplot analyses based on genome alignment of 2019-nCoV, bat-SL-CoVZC45, Longquan-140, and SARS-CoV suggest that the recombinant strain was not likely to be 2019-nCoV, but bat-SL-CoVZC45 (Figure 3B). And it also revealed at least four recombination breakpoints at positions 11,754, 20,664, 22,321, and 24,134&#8197;nt of the genome alignment, respectively (Figure 3B).In conclusion, we have identified a novel CoV from two patients with unusual pneumonia. Although the direct association with the disease is yet to be confirmed with more experimental data, our results provide several lines of evidence that the virus is most likely associated with this disease: (i) the viral titre is very high, with the abundance level reaching 1.5% and 0.62% of total reads sequenced, surpassing the highest expressed host genes to be one of the most dominant RNA molecules in the host transcriptome, an important sign that the virus is then under active replication [9]; (ii) since our RNA mNGS approach targets the total infectome (except for prion) [10], the fact that no other pathogens were identified from the infected sample underlines the unique role played by 2019-nCoV; (iii) the virus is grouped within the notorious CoV clade (i.e. SARS-like) with history of cross-virus transmission to humans [11] and has been demonstrated to have strong zoonotic potential [12]; and while this manuscript was under preparation, we noticed another case report from Wuhan which identified a same virus as the one found in this study [13]. Collectively, these results use the rich information present in the RNA metagenomics to evaluate potential pathogens, which highlights a future trend of viral diagnosis in the age of information.\nSupplementary Material\n",
        "pmid": "32020836",
        "title": "RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak",
        "journal_title": "Emerging Microbes & Infections",
        "authors": "Liangjun Chen;Weiyong Liu;Qi Zhang;Ke Xu;Guangming Ye;Weichen Wu;Ziyong Sun;Fang Liu;Kailang Wu;Bo Zhong;Yi Mei;Wenxia Zhang;Yu Chen;Yirong Li;Mang Shi;Ke Lan;Yingle Liu",
        "doi": "10.1080/22221751.2020.1725399"
    },
    {
        "keywords": "",
        "abstract": "Coronaviruses raise serious concerns as emerging zoonotic viruses without specific antiviral drugs available. Here we screened a collection of 16671 diverse compounds for anti-human coronavirus 229E activity and identified an inhibitor, designated K22, that specifically targets membrane-bound coronaviral RNA synthesis. K22 exerts most potent antiviral activity after virus entry during an early step of the viral life cycle. Specifically, the formation of double membrane vesicles (DMVs), a hallmark of coronavirus replication, was greatly impaired upon K22 treatment accompanied by near-complete inhibition of viral RNA synthesis. K22-resistant viruses contained substitutions in non-structural protein 6 (nsp6), a membrane-spanning integral component of the viral replication complex implicated in DMV formation, corroborating that K22 targets membrane bound viral RNA synthesis. Besides K22 resistance, the nsp6 mutants induced a reduced number of DMVs, displayed decreased specific infectivity, while RNA synthesis was not affected. Importantly, K22 inhibits a broad range of coronaviruses, including Middle East respiratory syndrome coronavirus (MERS&#8211;CoV), and efficient inhibition was achieved in primary human epithelia cultures representing the entry port of human coronavirus infection. Collectively, this study proposes an evolutionary conserved step in the life cycle of positive-stranded RNA viruses, the recruitment of cellular membranes for viral replication, as vulnerable and, most importantly, druggable target for antiviral intervention. We expect this mode of action to serve as a paradigm for the development of potent antiviral drugs to combat many animal and human virus infections.",
        "body": "Introduction\nPrior to the emergence of the highly pathogenic severe acute respiratory syndrome-associated coronavirus (SARS-CoV) in 2003 [1]&#8211;[3] only two circulating human coronaviruses (HCoVs), HCoV-229E [4] and HCoV-OC43 [5] causing relatively mild common cold-like respiratory tract infections, were known, and coronaviruses have not been regarded as significant threat for human health. Now, more than ten years later, the emergence of another highly pathogenic coronavirus of zoonotic origin, the Middle East respiratory syndrome coronavirus (MERS-CoV) [6]&#8211;[8], boosted community awareness towards the pending need to develop effective therapeutic options to combat coronavirus infections.Coronaviruses are enveloped viruses and their positive strand RNA genome, the largest of all RNA viruses, encodes for as many as 16 non-structural proteins (nsps), 4 major structural proteins, and up to 8 accessory proteins (reviewed in [9]). Many of these proteins provide essential, frequently enzymatic, functions during the viral life cycle and are therefore attractive targets for antiviral intervention. Antiviral strategies are mainly proposed for targeting coronavirus entry and essential enzymatic functions, such as coronavirus protease or RNA-dependent RNA polymerase (RdRp) activities. For example, the spike (S) protein mediates binding of different HCoVs to their specific cellular receptors [10]&#8211;[14], an event associated with preferential virus tropism for either ciliated or non-ciliated cells of the airway epithelium [15]. The S protein also mediates fusion between lipids of the viral envelope and the host cell plasma membrane or membranes of endocytic vesicles to promote delivery of viral genomic RNA into the cytoplasm. Virus binding and cell entry events can be inhibited by antibodies directed against the S protein, antibodies or small molecules interfering with the virus receptors, or synthetic peptides derived from the fusion-triggering heptad repeat regions of the S protein (reviewed in [16]). Following virus entry, the coronavirus genome, a positive sense, capped and polyadenylated RNA strand, is directly translated resulting in the synthesis of coronavirus replicase gene-encoded nsps. Coronavirus nsps are translated as two large polyproteins harboring proteolytic enzymes, namely papain-like and chymotrypsin-like proteinases that extensively process coronavirus polyproteins to liberate up to 16 nsps (nsp 1&#8211;16) [9], [17]&#8211;[20]. These proteolytic functions are considered essential for coronavirus replication and, consequently, a number of candidate drugs were reported to inhibit coronavirus polyprotein processing [21]&#8211;[26]. Likewise, the coronavirus RdRp activities, which reside in nsp8 [27] and nsp12 [28], are considered essential for coronavirus replication and attractive targets for antiviral intervention. In addition to these classical drug targets, coronaviruses encode an array of RNA-processing enzymes representing additional candidate targets. These include a helicase activity linked to an NTPase activity in nsp13, a 3&#8242;-5&#8242;-exonuclease activity linked to a N7-methyltransferase activity in nsp14, an endonuclease activity in nsp15, and a 2&#8242;-O-methyltransferase activity in nsp16 (reviewed in [28]).Like all positive strand RNA viruses, coronaviruses synthesize viral RNA at organelle-like structures in order to compartmentalize this critical step of the viral life cycle to a specialized environment that is enriched in replicative viral and host-cell factors, and at the same time protected from antiviral host defense mechanisms [29]&#8211;[31]. There is now a growing body of knowledge concerning the involvement, rearrangement and requirement of cellular membranes for RNA synthesis of a number of positive-strand RNA viruses, including coronaviruses [30], [32]&#8211;[35]. Three coronaviral nsps, i.e., nsp3, nsp4, and nsp6 [9], [36], [37] are thought to participate in formation of these sites for viral RNA synthesis. In particular, these proteins contain multiple trans-membrane domains that are thought to anchor the coronavirus replication complex through recruitment of intracellular membranes to form a reticulovesicular network (RVN) of modified, frequently paired, membranes that includes convoluted membranes [32] and double membrane vesicles (DVM) [38] interconnected via the outer membrane with the rough ER [32]. Indeed, Angelini and colleagues [39] have recently shown that co-expression of all three transmembrane domain-containing SARS-CoV nsps (nsp3, nsp4, and nsp6) is required to induce DMVs that are similar to those observed in SARS-CoV-infected cells. Such organelle-like compartments harboring membrane-bound replication complexes show remarkable parallels amongst a broad range of positive-strand RNA virus families, and are potentially evolutionary linked to similar mechanisms in the life cycle of double-strand (ds)RNA, reverse-transcribing, and cytoplasmic replicating DNA viruses [29]. Coronavirus ER-derived DMVs are induced early after virus entry into the host cell cytoplasm [9], [32], [34], [38]&#8211;[43], and display striking similarities to DMVs induced by hepatitis C virus [33]. The evolutionary conservation of engaging host cell-derived organelle-like membranous structures for virus RNA synthesis and genetic evidence that impairment of coronavirus DMV integrity is associated with severe reduction of virus replication [44], [45] suggest that antiviral intervention by targeting membranes involved in virus replication represents an attractive, however yet largely unexplored approach.In this work, we describe a novel inhibitor of coronavirus replication that specifically interferes with membrane-bound coronaviral RNA synthesis. This novel mode-of-action is characterized by severe impairment of DMV formation that results in near-complete inhibition of RNA synthesis. Notably, the inhibitor displayed antiviral activity against a broad range of animal and human coronaviruses, including the recently emerging MERS-CoV.\nResults\n\nIdentification of anti-HCoV-229E hit compound K22\nTo identify novel inhibitors of coronavirus infectivity we screened the ChemBioNet collection of 16671 compounds for antiviral activity against HCoV-229E. To this end, MRC-5 cells growing on 384-well plates were supplemented with a specific library compound (20 &#181;M) and then inoculated with HCoV-229E. Compounds that reduced or abolished viral cytopathic effect were re-tested in 24-well plate format for more precise evaluation of their antiviral potential. This two-step screening procedure resulted in several hits including two structurally similar compounds referred to as K22 (Figure 1A) and J15 (Figure S1A). The former compound, K22, whose structural name is (Z)-N-(3-(4-(4-bromophenyl)-4-hydroxypiperidin-1-yl)-3-oxo-1-phenylprop-1-en-2-yl)benzamide was examined in detail. The compound was completely soluble in medium up to 50 &#181;M. The concentration of K22 that inhibited the number of HCoV-229E plaques by 50% (IC50) was 0.7 &#181;M (Figure 1B). K22 did not reduce viability of MRC-5 cells by >50% (CC50) at a concentration range of 0.032&#8211;500 &#181;M (Figure 1C). However this compound decreased proliferation of MRC-5 cells with a CC50 value of 110 &#181;M (Figure 1C). Hence, using the CC50 value determined in cell proliferation assay, the selective index for K22, i.e. the CC50/IC50 quotient, was 157. Compound J15, although showing anti-HCoV-229E activity similar to that of K22 exhibited a somewhat less favorable cytotoxicity profile in the cell viability assay (Figure S1B).\nK22 inhibits HCoV-229E during the early, post entry phase of the viral life cycle\nTo assess which step of the HCoV-229E life cycle is affected by K22, a time-of-addition/removal experiment was performed. K22 (4 &#181;M) was incubated with cells for a period of only two hours either prior to, during, or after infection with HCoV-229E. As shown in Figure 1D, K22 treatment prior to infection resulted in only marginal reduction of virus infectivity thus excluding blockade of cellular receptor(s) for HCoV-229E as its mode-of-action. Simultaneous addition of K22 and virus resulted in &#8764;50% reduction of virus infectivity suggesting that the compound may interact with viral particles thus inactivating their binding or cell-entry activity. To clarify this possibility, the virus was incubated with &#8764;70 IC50 doses of K22 or DMSO solvent for 15 min at 37&#176;C, followed by virus dilution and its titration at non-inhibitory compound concentrations. Similar titers were observed for the virus treated with K22 (7.2&#215;105/ml&#177;8.9%) and DMSO (7.5&#215;105/ml&#177;4.7%) (n&#8202;=&#8202;2; two experiments), indicating that K22 exhibited no virus particle-inactivating activity. Thus, the &#8764;50% reduction in plaque number (Figure 1D) observed by simultaneous addition of K22 and virus is likely due to cellular uptake of K22 and inhibitory activity of probably not yet metabolically processed compound during a very early step of virus replication rather than the drug binding to viral particles and interference with their penetration into cells. This idea is further corroborated by the most pronounced inhibition of HCoV-229E replication when K22 was added after infection (Figure 1D). To more precisely determine the time window of efficient K22-mediated inhibition of HCoV-229E, K22 (10 &#181;M) was added to infected cells at different time points post infection (p.i.), and intra- and extracellular viral RNA, and infectious particles were quantified at 24 hours p.i.. As shown in Figures 1E-F, K22 addition within the first 6 hours p.i. resulted in near complete inhibition of viral RNA synthesis and &#8764;1000-fold reduction of produced infectious virus, suggesting that K22 inhibits most potently post virus entry during the early phase of the HCoV-229E life cycle.\nK22 resistant mutants contain substitutions in nsp6\nTo obtain further insight concerning the target of K22 inhibition we aimed to generate K22-resistant mutants and therefore subjected plaque purified HCoV-229E to 10&#8211;13 consecutive passages on MRC-5 cells in presence of increasing concentrations of K22 (2&#8211;16 &#181;M). In two independent experiments we isolated and plaque purified several variants displaying moderate (&#8764;2-fold) to strong (&#8764;12-fold) K22 resistance (IC50 of 1.6&#8211;8.5 &#181;M; Table 1). Whole genome sequencing analysis of wild type (wt) HCoV-229E, mock passaged virus, and K22 passaged virus revealed two amino acid substitutions within nsp 6 (H121L; M159V) that were associated with strong K22 resistance (Table 1). Sequence alignment and prediction of potential transmembrane regions of nsp6 homologs of HCoV-229E and other coronaviruses used in this study, revealed presence of 7 potential membrane-spanning domains (Figure 2) 6 of which are proposed to be used as membrane anchors in other coronaviruses [36], [37], and that mutations conferring resistance to K22 are located in or near these regions (Figure 3A). Subsequent generation of recombinant mutants, designated HCoV-229EH121L, HCoV-229EM159V, and HCoV-229EH121L/M159V, carrying the nsp6 mutations individually or combined by reverse genetics confirmed that these mutations confer resistance to K22 inhibition as revealed by plaque inhibition (Table 1) and the time-of-addition (Figures 3B-C) assays. Thus, as expected from the previous experiment (Figure 1E), K22 addition within the first 6 hours p.i. with the wt HCoV-229E resulted in near complete inhibition of viral RNA synthesis (Figure 3C), an effect completely abrogated in the drug-resistant recombinant mutant viruses (Figure 3B). Notably, although the amount of intracellular (Figure 3D) and extracellular (Figure 3E) viral RNA was comparable between K22-resistant mutants and parental wt HCoV-229E, production of infectious particles during infection with K22-resistant mutant viruses was greatly reduced (up to 34 fold at 48h p.i.) (Figure 3F). This difference cannot be attributed to the presence of free viral RNA in preparations of extracellular virus, since the treatment of K22-resistant HCoV-229EM159V mutant virus with ribonuclease A did not reduce the quantity of viral RNA (Figure S2). This observation suggests that K22 resistance-conferring mutations in nsp6 are associated with a fitness cost (reduced specific infectivity).\nK22 treatment results in loss of DMVs\nThe observation that amino acid substitutions in nsp6 confer K22 resistance strongly suggests a mode-of-action based on interference with host cell membranes required for coronavirus replication. Nsp6 is expressed as a membrane-spanning integral component of the viral replication complex, and is, together with nsp3 and nsp4, implicated in anchoring the coronavirus replicase complex to DMVs or related membrane structures [9], [36], [37], [39], [43]. Indeed, there is genetic and experimental evidence concerning nsp4-mediated alterations of coronavirus DMVs [44], [45], and that ectopic expression of nsp6 results in the formation of ER-derived vesicles [46]. We therefore assessed if K22 may impact the formation of coronavirus-induced DMV by electron microscopy (Figure 4). As expected, perinuclear DMV clusters as well as viral particles were readily detectable in wt HCoV-229E-infected cells (Figure 4A). In sharp contrast, no DMV clusters or viral particles were detectable in wt HCoV-229E-infected and K22-treated (4 &#181;M) cells (Figure 4A). Since double-stranded (ds) RNA is indicative of coronavirus replication and has been shown to reside predominantly within the inner lumen of coronavirus-induced DMVs [32] we also performed immunofluorescence analysis and stained HCoV-229E-infected cells for viral replicase complex (nsp8) and dsRNA. Strikingly, the characteristic perinuclear immunofluorescence staining pattern for viral replicase complexes and dsRNA visible in wt HCoV-229E-infected cells was completely absent under K22 treatment (Figure 5), confirming the remarkable efficacy of K22-mediated inhibition of viral replication and supporting the notion that K22 blocks the formation of DMVs. In contrast to parental wt HCoV-229E and irrespectively whether K22 was applied, recombinant K22 escape mutants were still capable of inducing the formation of DMVs (Figure 4B) and displayed the characteristic staining pattern for replicase complexes and dsRNA (Figure 5). Likewise, compound J15 efficiently blocked replication (Figure S1B) and DMV formation of wt HCoV-229E but not K22 resistant nsp6 recombinant HCoV-229EM159V (Figure S3) suggesting that J15 may have the same target and mode-of-action. Notably, in cells infected with K22 escape mutants the overall number of DMVs per cell was reduced (30.3&#177;29.7 in HCoV-229EM159V versus 65&#177;50.1 in wt HCoV-229E infected cells; P<0.05; n&#8202;=&#8202;20), similar as previously described for mouse hepatitis virus (MHV) nsp4 mutants [44], [45], while the number of intracellular viral particles that were often packed in tubular vesicle-like structures (Figures 4A-B) was comparable to that of wt virus (471.8&#177;212.6 in HCoV-229EM159V versus 438.3&#177;96.8 in wt virus infected cells; n&#8202;=&#8202;10). We could also frequently detect DMVs displaying partially collapsed inner membranes in cells infected with K22 escape mutants (irrespectively whether or not K22 was applied; Figure 4B), again similarly as reported for MHV nsp4 mutants [45], suggesting that nsp6, like nsp4, has a pivotal role in coronavirus DMV formation. Overall, these findings demonstrate that the antiviral activity of K22 (and that of the structurally similar compound J15) results in complete loss of DMVs. This efficient block in replication can be overcome by resistance mutations in nsp6, and DMVs induced by nsp6 mutant viruses are reduced in numbers and structurally impaired &#8211; both findings concurring with the established function of nsp6 in DMV formation.\nK22 does not impact cellular autophagy\nOur data show that K22 targets a very early step in the HCoV-229E life cycle, and the appearance of resistance-conferring mutations in nsp6 suggests that K22 impairs DMV formation. We therefore assessed if K22 treatment may, independent of virus infection, impact autophagy, a cellular process displaying similarities to coronaviral DMV formation. To this end we first transfected Huh7 cells with a plasmid encoding LC3B-GFP in order to trace rapamycin-induced autophagsomes by life imaging. This analysis revealed that three to six hours after adding rapamycin to the culture medium green fluorescent autophagocytic vesicles become apparent, irrespectively if K22 (20 &#181;M) was added or not (data not shown). We corroborated this result by immunofluorescence analysis of Huh7 cells that were stained for endogenous LC3B at six hours post addition of rapamycin. As shown in supplementary Figure S4 rapamycin-incuced autophagocytic vesicles were again readily detectable in the presence of K22 (20 &#181;M), suggesting that K22 does not impact cellular autophagy.\nK22 inhibits a number of diverse coronaviruses\nSince K22 inhibits a crucial step in the HCoV-229E life cycle, we assessed the antiviral activity of K22 against a panel of diverse coronaviruses representing the major phylogenetic lineages of &#945;-, &#946;- and ???-coronaviruses. As shown in Figure 6A-D and supplementary Figure S5, K22 indeed displayed antiviral activity against recombinant MHV (strain A59 [47]) expressing Gaussia luciferase as marker for virus replication, recombinant type-I feline coronavirus (FCoV; strain Black [48]) expressing Renilla luciferase as marker for virus replication, avian infectious bronchitis virus (IBV; strain Beaudette [49]), and SARS- CoV (strain Frankfurt-1 [50]), suggesting that K22 targets a broad range of coronaviruses. Furthermore, there was no cytotoxicity detectable in cells of feline (FCWF cells), murine (L929 cells), and primate (Vero cells) origin in the K22 concentration range assessed, and analysis of K22 cytostatic activities in the cell proliferation assay revealed CC50 values &#8805;40 &#181;M (Table S1), i.e., the highest drug concentration used in antiviral assays. Notably, the efficacy of K22-mediated inhibition varied amongst different coronaviruses, however whether this is related, as in HCoV-229E, to nsp6 function would require generation and analysis of K22 resistant variants for all coronaviruses tested. In contrast, K22 exhibited little or no effect on replication of poliovirus (Figure S6), a pathogen that like coronaviruses induces rearrangement of cellular membranes to assist RNA replication.\nInhibition of HCoV-229E and MERS-CoV in primary human airway epithelia cultures\nFinally, we assessed the efficacy of K22 inhibition in the primary target cells of respiratory virus infection, the human airway epithelium. Fully differentiated primary human airway epithelia (HAE) cultures [15], [51] derived from three different donors and grown under air-liquid interphase conditions were infected with a recombinant HCoV-229E expressing Renilla luciferase as marker for virus replication [52], and with MERS-CoV [8], [51]. MERS-CoV was first described in 2012 and was isolated from a 60-year old man with acute pneumonia, renal failure and fatal outcome in Saudi Arabia [8]. The virus is most likely of zoonotic origin [7], [53] and by February 2014 the number of laboratory-confirmed cases of MERS-CoV infection reported to the World Health Organization exceeded 182, including more than 79 cases with fatal outcome. We have previously shown that MERS-CoV can readily replicate on primary HAE cells [51] by infecting non-ciliated cells expressing the cellular receptor dipeptidyl peptidase 4 [14]. As shown in Figure 6, HCoV-229E and MERS-CoV infections were inhibited by K22 treatment with remarkable efficacy, illustrated by reduction of viral replication by several orders of magnitude (Figure 6E-F) and substantial reduction of dsRNA in MERS-CoV-infected primary HAE cultures (Figure 6G-H). This result demonstrates that the broad anti-coronaviral activity of K22 makes this compound particularly promising for the development of efficacious treatment options for emerging coronaviruses, such as MERS-CoV.\nDiscussion\nHere we describe the discovery of a novel class of inhibitor and propose a mode-of-action that targets membrane-bound viral replication. Like all positive strand RNA viruses, coronaviruses employ host cell membranes to assemble the viral replicase complex. This evolutionary conserved strategy provides a compartment for viral RNA synthesis that is enriched in replicative viral and host cell-derived proteins and believed to protect from antiviral host cell defense mechanisms. The remarkable efficacy of K22-mediated inhibition of coronavirus replication confirms that the employment of host cell membranes for viral RNA synthesis is a crucial step in the coronavirus life cycle, and importantly, demonstrates that this step is extremely vulnerable and also druggable for antiviral intervention.The observation that K22 resistance is mediated through mutations in nsp6 defines transmembrane domain-containing nsps implicated in anchoring viral replicase complexes to host cell-derived membranes, as novel targets for anti-coronaviral intervention. Moreover, we expect this mode-of-action to serve as a paradigm for the development of similar antiviral drugs to combat infections caused by many other positive strand RNA viruses. Notably, resistance conferring mutations in nsp6 emerged only after 10&#8211;13 consecutive passages of HCoV-229E under K22 selection, and we were so far not successful in obtaining K22-resistant MHV-A59 mutants (data not shown). This suggests that escape mutations in membrane domain-containing coronavirus nsps compatible with maintaining efficient RNA synthesis are limited. In addition, the nsp6 escape mutants we have obtained for HCoV-229E display a remarkable reduction of specific infectivity. Thus, although RNA synthesis appears to be unaffected and viral RNA detected in preparations of extracellular virus was ribonuclease insensitive implying its adequate package in viral particles, mutations in nsp6 seem to reduce virus fitness. Thus, it is conceivable that the nsp6 mutants may be functionally impaired during an early step in the viral life cycle. Since dsRNA is localized in DMVs and nsp6 escape mutants induced decreased number of DMVs that are structurally impaired, it is possible that the reduced specific infectivity of these viruses could be related to dsRNA-triggered innate immune responses.SARS-CoV nsp6 was recently found to contribute to the establishment of the virus-induced RVN by promoting vesicle formation in transfected cells [39], and our observation that K22 resistant mutants generated decreased number of DMVs implies that specific alterations may adversely affect the vesicle-forming capability of nsp6. Nsp6 of HCoV-229E (this report), MHV, and SARS-CoV [36], [37] is predicted as a hexaspaning protein comprising a conserved C-terminal cytoplasmic tail. The latter domain may serve as a wedge-like amphipathic helix which upon insertion into the lipid membrane can trigger its bending due to induction of positive membrane curvature (reviewed in [54]). The vesicle formation would also require a putative ion channel activity that depolarizes curved membranes thus facilitating their fusion and vesicle scission. The question as to whether nsp6 or other components of the coronavirus replicase complex exhibit such activities would require further investigation.Although our data reveal that the K22 escape mutations occur in nsp6, further binding experiments are required to clarify whether K22 targets nsp6 directly. We observed that K22 is most active in inhibiting replication of the tested &#945;-coronaviruses (HCoV-229E, FCoV) and the &#947;-coronavirus IBV, whereas amongst &#946;-coronaviruses K22 was highly active in inhibiting MERS-CoV, but only moderately against MHV or SARS-CoV (Figure 6). It is conceivable that K22 may strong inhibit &#945;-coronaviruses, since K22 has been identified by screening for anti-HCoV-229E activity. However, the limited nsp6 sequence similarity between coronaviruses (Figure 2) does not allow predicting the strength of K22-mediated inhibition of replication based on nsp6 homology. We also like to address in future studies a question of how the moderately resistant virus variant L (containing mutations in nsp15 and nucleocapsid) can escape K22-mediated inhibition of replication. This variant, in contrast to these containing resistance mutations in nsp6, exhibited only moderate resistance to K22 (&#8764;2-3-fold) and was not consistently selected in separate selection experiments. Although nsp15 and nucleocapsid protein have not yet been described as being directly involved in DMV formation, these proteins are components of the replicase complex that may somehow affect/modulate nsp6 functions, and compensatory mutations in these proteins may partially relieve K22 blockade of nsp6. An alternative possibility is that the actual K22 target may be a cellular protein or a process of recruitment of a cellular protein that participates in coronavirus-induced membrane rearrangements by interacting with nsp6. While we could not observe any detectable impact of K22 on the formation of autophagosomes, further studies are required to address if K22 may target similar vesicles, such as EDEMosomes [41]. Both possibilities are compatible with the observed phenotype of DMV impairment and the detection of resistance mutations at regions of HCoV-229E nsp6 that are structurally conserved while displaying only limited sequence similarity. It is thus conceivable that membrane domain-containing nsp3 and nsp4 may represent additional drug targets. Similar as described for the related arteriviruses, where co-expression of membrane-spanning nsp2 and nsp3 results in membrane alterations and DMV formation similar to those observed during arterivirus infection [55], [56], co-expression of coronavirus nsp3, nsp4 and nsp6 is required to produce coronavirus-like membrane rearrangements including DMVs [39]. Expression of nsp3, nsp4 or nsp6 alone or in combinations of two induces aberrant membrane rearrangements that only partially mimic membrane structures known from coronavirus infection [39]. Thus, there is growing evidence that nsp3, nsp4, nsp6, and possibly ER membrane-resident host cell proteins [41], [57], orchestrate critical events that lead to the development of suitable membrane structures facilitating coronavirus RNA synthesis. Since K22 apparently interferes with these processes, inhibitors like K22 and corresponding escape mutants will likely become valuable tools to further our understanding on the induction of membrane alterations and DMV formation that take place during the early phase of the coronavirus life cycle. For example, co-expression of nsp3, nsp4 and native or mutated nsp6 in the absence of virus replication, similar as described by Angelini and colleagues [39], may help to clarify whether presence of K22 would affect formation of DMV by directly targeting nsp6 or cellular protein(s) required and recruited for DMV formation.We emphasize that the identification of K22 and its proposed mode-of-action is only the very first step towards an approved drug for therapeutic use in animals or humans. Specifically, we are currently focusing on the structure-activity relationship analysis of K22 analogs, with the aim to identify compounds with improved antiviral and cytotoxic profiles prior to their assessment in vivo. However, one important lesson of the past SARS-CoV and recent MERS-CoV outbreaks is that zoonotic transmission of coronaviruses into the human population can pose considerable threat to human health and that it is warranted to eventually invest significant efforts to developing efficacious and approved drugs to increase preparedness and combat coronavirus infections. The antiviral activity against a number of diverse coronaviruses makes K22 an ideal candidate for further development towards an efficacious &#8220;pan-coronavirus inhibitor&#8221;. Broad anti-coronaviral activity has been proposed for inhibitors targeting highly conserved enzymatic functions, such as coronavirus proteinase activities [26], [58], or more recently, for compounds targeting host cell factors required for efficient replication, such as cyclophilins [59], [60]. The concept of targeting multiple key functions of viral replication led to the development of efficacious treatment regimens against HIV and hepatitis C virus by combining multiple antiviral drugs [61], [62] and it is tempting to speculate that this concept will be applicable to combat coronavirus infections in the future. Moreover, with the identification of K22, we demonstrate that there are yet additional critical steps in the life cycle of positive strand RNA viruses to explore as targets for antiviral intervention.\nMaterials and Methods\n\nEthics statement\nHuman bronchial epithelial cells were isolated from patients (>18 years old) who underwent bronchoscopy and/or surgical lung resection in their diagnostic pathway for any pulmonary disease and that gave written informed consent. This was done in accordance with local regulation of the Kanton St. Gallen, Switzerland, as part of the St. Gallen Lung Biopsy Biobank (SGLBB) of the Kantonal Hospital, St. Gallen, which received approval by the ethics committee of the Kanton St. Gallen (EKSG 11/044, EKSG 11/103).\nCells and viruses\nHuman embryonic lung diploid fibroblasts (MRC-5), African green monkey kidney cells (Vero), baby hamster kidney cells (BHK-21), felis catus whole fetus 4 cells (FCWF-4), were purchased from the American Type Culture Collection (ATCC), murine fibroblast cells (L929), African green monkey kidney cells (CV-1) were purchased from the European Collection of Cell Cultures. D980R cells were a kind gift from G. L. Smith, Imperial College, London, United Kingdom. African green monkey kidney (GMK AH1) cells were obtained from the Swedish Institute for Infectious Disease Control, Stockholm. Cells were grown in Eagle's minimum essential medium (EMEM) (MRC-5, CV-1, D980R, L929, BHK-21, GMK AH1 cells) or in Dulbecco's modified EMEM (DMEM) (FCWF-4, Vero cells), supplemented with 5&#8211;10% heat-inactivated fetal calf serum, (HI-FCS), 1% L-glutamine, penicillin (60 &#181;g/ml) and streptomycin (100 &#181;g/ml) (PEST). Isolation and cultivation of primary human bronchial epithelial cells to form pseudostratified/differentiated human airway epithelial (HAE) cultures was performed as described previously [15], [63].Human CoV strain 229E [4] (HCoV-229E) was obtained from ATCC (VR-740). HCoV-229E stocks were prepared from virus passages 6&#8211;8 in MRC-5 cells growing in EMEM supplemented with 2% HI-FCS, 1% L-glutamine, HEPES (10 mM) and PEST (EMEM-FP). In some experiments, the virus was concentrated by centrifugation of infectious culture fluid of MRC-5 cells over a 1.5 ml cushion of 20% sucrose for 2 h at 22000 rpm (SW28.1 rotor, Beckman). The pellet was covered with PBS (137 mM NaCl, 2.7 mM KCl, 8.1 mM Na2HPO4, 1.5 mM KH2PO), left overnight at 4&#176;C, and then gently suspended by pipetting. The following viruses and their propagation were described previously: recombinant HCoV- 229E [64], recombinant HCoV-229E-Ren expressing Renilla luciferase [52], recombinant feline coronavirus (strain Black) expressing Renilla luciferase (recFCoV-RL) [48], SARS-CoV strain Frankfurt-1 [50], recombinant avian infectious bronchitis virus (IBV, strain Beaudette) [49], MERS-CoV [8], [51]. Recombinant MHV strain A59 expressing Gaussia luciferase (MHV-Gluc) was generated based on the previously described reverse genetics system [47], [65]. Briefly, the MHV-A59 accessory gene 4 was replaced by the gene encoding the codon-optimized Gaussia luciferase [66] (hGLuc) using vaccinia-virus-mediated homologous recombination essentially as described for the generation of MHV-GP33-GFP [67]. The plasmid DNA used for recombination contained MHV-A59 nucleotides (nts) 27500&#8211;27967, the hGLuc Gaussia luciferase gene, and MHV-A59 nts 28265&#8211;28700. Recombinant HCoV-229E containing mutations conferring K22 resistance in nsp6 were generated based on the previously described reverse genetics system [64], [65]. Briefly, vaccinia virus HCoV-inf1 (containing the full-length HCoV-229E cDNA) [64] was used to recombine with a plasmid based on pGPT1 [68] where the Escherichia coli guanine phosphoribosyltransferase (GPT) gene was flanked by HCoV-229E nts 9398&#8211;10098 and 10930&#8211;11580. The resulting GPT-positive vaccinia virus was then used to recombine with plasmids containing the HCoV-229E nts 9398&#8211;11580 with modification of nucleotide 10455 (A to T; HCoV-229EH121L), or nt 10568 (A to G; HCoV-229EM159V), or both nts 10455 and 10568 (HCoV-229EH121L/M159V). The resulting vaccinia viruses were then used to rescue HCoV-229EH121L, HCoV-229EM159V, and HCoV-229EH121L/M159V as described previously [64], [65]. The identity of plasmid DNA and recombinant vaccinia viruses and recombinant coronaviruses was confirmed by sequencing. In some experiments poliovirus 1 strain Sabin (obtained from the Swedish Institute for Infectious Disease Control, Stockholm) was used.\nReagents\nThe ChemBioNet diversity library of 16671 compounds was obtained from the Leibniz Institute for Molecular Pharmacology (Berlin, Germany). Library was provided in a 384 well plate format, each well containing 5 &#181;l of a compound solubilized in DMSO to a final concentration of 10 mM. Hit compound K22 was purchased from ChemDiv (San Diego, CA; catalog number 4295&#8211;0370). The correct structure and purity of K22 (>95%) was confirmed in our laboratory by NMR and LCMS analyses.\nImmunofluorescence analysis\nMRC-5 cells were infected at a multiplicity of infection (moi) of 0.05 with wtHCoV-229E and K22-resistant recombinants HCoV-229EH121L, HCoV-229EM159V, and HCoV-229EH121L/M159V with or without the presence of K22 (4 &#181;M). The cells were fixed at 18 h p.i. with 4% paraformaldehyde (PFA) and immunostained [69] using the mouse monoclonal anti-dsRNA (J2, English & Scientific Consulting Bt.) and rabbit anti-HCoV-229E nsp8 [70] (kindly provided by John Ziebuhr, University of Giessen, Germany) as primary antibodies for detection of double-stranded (ds) RNA and viral replication complexes. Donkey derived, Dylight 488 labeled, anti-mouse IgG (H+L) and Dylight 647 labeled, anti-rabbit IgG (H+L) (Jackson Immunoresearch) were applied as secondary antibodies. Cells were counterstained with DAPI (4',6-diamidino-2-phenylindole; Invitrogen) to visualize nuclei. HAE cell cultures were inoculated with 40000 plaque forming units (PFU), with or without the presence of K22 (50 &#181;M) and fixed with 4% PFA 48 h p.i. Staining was performed with the mouse monoclonal antibody directed against dsRNA (J2) and goat polyclonal anti-ZO1 (tight junctions; ab99462, Abcam) as primary antibodies. Dylight 488-labeled donkey anti-mouse IgG (H+L), Dylight 546-labeled donkey anti-goat IgG (H+L) (Jackson Immunoresearch) were applied as secondary antibodies, followed by two separate incubation steps with Alexa Fluor647-conjugated rabbit monoclonal anti-beta-Tubulin antibody (ciliated cells; 9F3, Cell Signal) and DAPI (Invitrogen). Images were acquired using EC-plan Neofluar 20x/50 M27 or EC Plan-Neofluar 40x/1.30 Oil DIC M27 objectives on a Zeiss LSM 710 confocal microscope. Image capture, analysis and processing were performed using the ZEN 2010 (Zeiss) and Imaris (Bitplane Scientific Software) software packages.\nAnti-coronavirus compound screening assay\nThe screening assay was performed as described previously for respiratory syncytial virus [71]. Briefly, MRC-5 cells were seeded in 384 well plates (CLS-3701; Costar-Corning, NY, USA) to become &#8764;70&#8211;90% confluent after one day of culture. The growth medium was removed, and the cells supplemented consecutively with 25 &#181;l of EMEM-FP medium, 1 &#181;l volumes of library compounds at 1 mM concentration, and &#8764;350 PFU of HCoV-229E in 25 &#181;l of EMEM-FP. The last two columns of the 384 well plate received either virus or EMEM-FP medium to serve as controls. The cells were observed under the microscope for their protection from the virus-induced cytopathic effect after 3 and 6 days of incubation at 37&#176;C.\nAntiviral assays\nPlaque reduction assay to determine the antiviral effect of K22 on HCoV-229E was done as follows. MRC-5 cells were seeded in 12-well plates to become nearly confluent after one day of culture. Serial fivefold dilution of K22 (0&#8211;100 &#181;M) and 100 PFU of HCoV-229E virus in 0.5 ml of EMEM-FP medium were added to and incubated with cells for 3 h at 37&#176;C, 5% CO2. Subsequently, the virus-compound mixtures were removed from cells, and 1.5 ml volumes of 1% methylcellulose (MC) solution in EMEM-FP medium supplemented with the same concentration of K22 were added. The plates with cells were further incubated at 37&#176;C, 5% CO2 for 2&#8211;3 days, and then stained with 0.2% solution of crystal violet to visualize the viral plaques.Viral yield reduction assays were done to determine the antiviral effect of K22 on HCoV-229E-Ren, recFCoV-RL, MHV-Gluc, SARS-CoV, IBV, MERS-CoV, and poliovirus replication. Briefly, K22 or its DMSO solvent in medium was added at the indicated concentrations to nearly confluent monolayers of corresponding cell lines or to HAE cultures at the basolateral side and incubated for 4 h at 37&#176;C, 5% CO2. The cells were then inoculated with recFCoV-RL (moi&#8202;=&#8202;0.1 on FCWF-4 cells), MHV-Gluc (moi&#8202;=&#8202;0.001 on L929 cells), SARS-CoV (moi&#8202;=&#8202;0.001 on Vero cells), IBV (moi&#8202;=&#8202;1 on Vero cells), HCoV-229E-Ren (4&#215;103 PFU on HAE cultures apically), MERS-CoV (4&#215;103 PFU on HAE cultures apically) or poliovirus (moi&#8202;=&#8202;0.001 on GMK AH1 cells). After 2 h the viral inoculum was removed, cells were rinsed three times with PBS, and fresh medium containing the same concentrations of K22 or DMSO was added. Coronavirus replication was assessed from cell culture supernatant by determining titer as TCID50 (tissue culture infectious dose that will produce pathological change in 50% of cell cultures inoculated) for IBV or poliovirus at 48 h p.i., by determining the amount of viral genome RNA produced by qRT-PCR specific for SARS-CoV and MERS-CoV at 48 h p.i. as described previously [51], or by determining the level of Renilla expression at 48 h p.i. (HCoV-229E-Ren) or 72 h p.i. (recFCoV-RL) using Renilla Luciferase Assay System (Promega, E2820), or Gaussia luciferase expression (MHV-Gluc) at 24 h p.i. using the BioLux Gaussia Luciferase Assay Kit (NEB,E3300), respectively.For the virucidal assay, 200 &#181;l of HCoV-229E suspension (&#8764;3&#215;104 PFU) in EMEM-FP medium was mixed with 50 &#181;M K22 and incubated for 15 min at 37&#176;C. In the control sample, virus was incubated with the DMSO solvent at a final concentration corresponding to that present in the test compound. Then, both mixtures were diluted serially tenfold in EMEM-FP medium and the residual virus infectivity determined by the viral plaque assay.\nCell toxicity and proliferation assays\nThe toxicity of K22 or its solvent (DMSO) for MRC-5 cells was evaluated using the tetrazolium-based CellTiter 96 AQueous One Solution cytotoxicity assay (Promega; G3580). The effect of K22 or its solvent on proliferation of MRC-5 cells was studied as follows. The cells were seeded in 48 well plates to become &#8764;50% confluent after one day of culture. The growth medium was removed, and cells incubated with specific concentrations of K22 or its solvent in EMEM-FP medium for 72 h at 37&#176;C. The cells were then dissociated with trypsin/EDTA solution and counted. The effect of K22 or DMSO on viability of Vero, L929, and FCFW-4 cells was assessed using the CytoTox-Glo Cytotoxicity Assay kit (Promega, G9291) while the toxicity of test compound for differentiated HAE cultures was evaluated with CellTiter-Glo Luminescent Cell Viability Assay kit (Promega, G7571).\nTime-of-addition assay\nMRC-5 cells growing in 12 well plates were precooled for 15 min at room temperature and for another 15 min at 4&#176;C. The cells were rinsed once with 500 &#181;l of cold EMEM-FP and inoculated with HCoV-229E at moi of 0.05. Following virus adsorption to cells for 45 min at 4&#176;C, the cells were rinsed twice with 500 &#181;l of cold EMEM-FP, and 990 &#181;l of warm EMEM-FP medium was added. Subsequently 10 &#181;l of 1 mM K22 was added at specific time points relative to the end of the virus adsorption period, and the infectious cell culture medium and cells harvested at the time point 24 h. The cell culture supernatant medium was clarified by centrifugation at 1000&#215;g for 5 min while the pelleted cells were suspended in RNase-free water and stored at &#8722;80&#176;C until quantification in RT-PCR assay. To study the effect of K22 on early virus-cell interaction the &#8220;time-of-addition&#8221; assay was modified as follows. MRC-5 cells were rinsed once with 1 ml of EMEM-FP and 500 &#181;l of EMEM-FP supplemented with 4 &#181;M K22 was added. The compound was incubated with cells for 2 h at 37&#176;C either prior to, during or after a 2 h period of infection of cells with &#8764;100 PFU of 229E virus in 500 &#181;l of EMEM-FP. The cells were washed once with 1 ml of EMEM-FP after each 2 h period of their incubation with compound and/or virus. Finally, the cells were overlaid with the MC solution, and after incubation for 2 days at 37&#176;C stained with crystal violet to visualize the viral plaques.\nRT-PCR\nThe RT TaqMan PCR was carried out as described by Brittain-Long et al. [72]. Briefly, the extraction of RNA was conducted in the Magnapure LC robot using the MagNA Pure LC Total Nucleic Acid Isolation Kit (Roche Applied Science, Mannheim, Germany), and amplification was performed using a TaqMan 7300 Real Time PCR system (Applied Biosystems, Foster City, CA), with a pair of forward 5&#8242;-CAGTCAAATGGGCTGATGCA-3&#8242; and reverse 5&#8242;-AAAGGGCTATAAAGAGAATAAGGTATTCT-3&#8242; primers as well as a probe 3&#8242;CCCTGACGACCACGTTGTGGTTCA 5&#8242; specific for HCoV-229E genome fragment coding for nucleocapsid protein [73]. The number of HCoV-229E RNA copies was determined by relating the detected cycle threshold values to a standard curve prepared based on five tenfold dilutions of the specific plasmid (pUC57) comprising a 94 bp insert from the nucleocapsid sequence of HCoV-229E. qRT-PCR assays to quantify SARS-CoV and MERS-CoV genomic RNA have been described previously [51].\nPreparation of drug-resistant variants of HCoV-229E and sequencing analysis\nA procedure described previously for respiratory syncytial virus [71] was used. Briefly, plaque purified HCoV-229E was subjected to 10&#8211;13 consecutive passages in MRC-5 cells in the presence of increasing concentrations (2&#8211;16 &#181;M) of K22. For control purposes, the same virus was also passaged in MRC-5 cells in the absence of inhibitor. The virus was then subjected to two rounds of plaque purification in the presence of inhibitor, and its relative drug-resistance tested using the viral plaque reduction assay. Genomic RNA of original, mock-passaged, and the K22-resistant virus from passage 10&#8211;13 was extracted from extracellular fluid of the 229E-infected MRC-5 cells using the QIAamp viral RNA purification kit (Qiagen). Overlapping DNA fragments covering the entire coding sequence were produced by reverse transcription PCR and subjected to nucleotide sequencing using the ABI PRISM Big Dye Terminator v3.1 Cycle Sequencing Ready Reaction kit (Applied Biosystems). Nucleotide sequence analysis was performed using Sequencher 4.9 software (Gene Codes Corporation).\nHCoV-229E replication kinetics\nMRC-5 cells growing in 12 well plates were precooled for 15 min at room temperature and for another 15 min at 4&#176;C. The cells were rinsed once with 500 &#181;l of cold EMEM-FP and inoculated with concentrated preparation (see the Cells and Viruses section) of HCoV-229E (moi&#8202;=&#8202;0.05). Following virus adsorption to cells for 1 h at 4&#176;C, the cells were rinsed thrice with 500 &#181;l of cold EMEM-FP, and 500 &#181;l of warm EMEM-FP medium was added. The supernatant fluid and infected cells were harvested at specific time points relative to the end of the virus adsorption period, and processed for determination of viral RNA and infectivity as described under the &#8220;time-of-addition&#8221; assay.\nRibonuclease treatment of HCoV-229E\nThe infectious culture medium comprising HCoV-229E or recombinant nsp6 mutant HCoV-229E M159V were clarified by centrifugation at 1000&#215;g for 5 min, and then 100 &#181;l volumes of the supernatant were supplemented with 2 &#181;l (20 &#181;g) of ribonuclease A (Thermo Fisher Scientific; EN0531) or its solvent. All samples were spiked with &#8764;7 &#181;g of RNA purified from human respiratory syncytial virus (RSV) to serve as an internal control of ribonuclease activity. Following incubation of the virus-enzyme mixture for 30 min at 37&#176;C, the coronaviral and RSV RNA were quantified by RT TaqMan PCR as described by Brittain-Long et al. [72] while coronavirus infectivity was determined by plaque titration.\nAutophagy\nTo assess the time-frame where autophagy vesicle formation occurs we seeded Huh-7 cells (100.000 cells) on glass bottom 12-well cluster plates (MatTek). Forty-eight hours prior to stimulation cells were transfected with LC3B-GFP plasmid [74] using lipofectamine2000 (Invitrogen), according to manufactures protocol. Hereafter cells were exposed to 100 nM of rapamycin (Invivogen) alone or in presence of either 20 &#181;M of K22 or an equal volume of DMSO for the duration of 18 hours at 37&#176;C. Fluorescent and differential interference contrast (DIC) images were acquired with 30 minute interval using EC Plan Neo-fluar 40x/1.30 Oil DIC M27 objective on a Zeiss LSM 710 confocal microscope. Image capture, analysis and processing were performed using the ZEN 2010 (Zeiss). To determine whether K22 inhibits endogenous autophagy vesicle formation we stimulated Huh-7 cells (40.000 cells) with 100 nM of rapamycin alone or in presence of either 20 &#181;M of K22 or an equal volume of DMSO for duration of six hours at 37&#176;C. Unstimulated cells were used as mock control. Cells were fixed and immunostained as previously described [69]. Rabbit polyclonal anit-LC3B (L7543, Sigma Aldrich) was applied as primary antibody for the detection of autophagy vesicles. Goat derived, Cy3 labeled, anti-rabbit IgG (H+L; Jackson ImmunoResearch) was applied as secondary antibody. Thereafter cells were counterstained with DAPI (Invitrogen). Fluorescent images were acquired using a PLAPON 60xO/1.42 objective on an Olympus FV-1000 confocal microscope. Image capture, analysis and processing were performed using the Olympus Fluoview software.\nElectron microscopy\nMRC-5 cells growing on a Melinex polyester film (Agar Scientific Ltd., Stansted, U.K.) in 24 well cluster plates were infected with HCoV-229E (moi&#8202;=&#8202;0.04) in the presence of 10 &#181;M of K22. After 18 h of infection at 37&#176;C, the culture medium was removed, the cells rinsed twice with Eagle's medium, and a fresh Eagle's medium supplemented with 2.5% glutaraldehyde was added and incubated for 45 min at 37&#176;C. The cells were washed twice with 0.05 M Tris-HCl buffer (pH 7.4) supplemented with 2 mM CaCl2, and further processed for electron microscopy as described [75]. Experiments with recombinant nsp6 mutant viruses and original virus were carried out in a similar manner except that the cells were inoculated at a moi of &#8764;0.25 and incubated with or without the presence of 4 &#181;M K22.\nSupporting Information\n",
        "pmid": "24874215",
        "title": "Targeting Membrane-Bound Viral RNA Synthesis Reveals Potent Inhibition of Diverse Coronaviruses Including the Middle East Respiratory Syndrome Virus",
        "journal_title": "PLoS Pathogens",
        "authors": "Anna Lundin;Ronald Dijkman;Tomas Bergstr\u00f6m;Nina Kann;Beata Adamiak;Charles Hannoun;Eveline Kindler;Hulda R. J\u00f3nsd\u00f3ttir;Doreen Muth;Joeri Kint;Maria Forlenza;Marcel A. M\u00fcller;Christian Drosten;Volker Thiel;Edward Trybala",
        "doi": "10.1371/journal.ppat.1004166"
    },
    {
        "keywords": "Middle East Respiratory Syndrome;Acute renal failure;Human Coronavirus;Renal epithelial cells;Dipeptidyl-peptidase-4;Angiotensin-converting-enzyme-2",
        "abstract": "Background\nThe Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes symptoms similar to Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV), yet involving an additional component of acute renal failure (ARF) according to several published case reports. Impairment of the kidney is not typically seen in Coronavirus infections. The role of kidney infection in MERS is not understood.Findings\nA systematic review of communicated and peer-reviewed case reports revealed differences in descriptions of kidney involvement in MERS versus SARS patients. In particular, ARF in MERS patients occurred considerably earlier after a median time to onset of 11&#160;days (SD &#177;2,0&#160;days) as opposed to 20&#160;days for SARS, according to the literature. In-situ histological staining of the respective cellular receptors for MERS- and SARS-Coronavirus showed highly similar staining patterns with a focus of a receptor-specific signal in kidney epithelial cells. Comparative infection experiments with SARS- and MERS-CoV in primary human kidney cells versus primary human bronchial epithelial cells showed cytopathogenic infection only in kidney cells, and only if infected with MERS-CoV. Kidney epithelial cells produced almost 1000-fold more infectious MERS-CoV progeny than bronchial epithelial cells, while only a small difference was seen between cell types when infected with SARS-CoV.Conclusion\nEpidemiological studies should analyze kidney impairment and its characteristics in MERS-CoV. Virus replication in the kidney with potential shedding in urine might constitute a way of transmission, and could explain untraceable transmission chains leading to new cases. Individual patients might benefit from early induction of renoprotective treatment.",
        "body": "None\nCoronaviruses (CoVs) cause human disease with symptoms ranging from mild respiratory symptoms to severe pneumonia [1]. In September 2012, a novel CoV termed the Middle East Respiratory Syndrome (MERS)-CoV emerged on the Arabian Peninsula with 163 laboratory-confirmed cases including 71 deaths so far (World Health Organization, December 2nd, http://www.who.int/csr/don/2013_12_02/en/index.html). Due to its distribution in several countries of the Arabian Peninsula there is a risk of global spread through travel and pilgrimage, posing a potential threat to global public health.The clinical picture of MERS-CoV infection is characterized by acute atypical pneumonia and respiratory failure, resembling symptoms caused by SARS-CoV [2-5]. Acute renal failure (ARF) was described in a number of MERS cases, with potential influence on disease severity [2,3,5-7]. ARF has neither been a typical feature of SARS, nor has it been commonly observed in infections caused by any other CoV, including human HCoV-NL63, -229E, &#8211;OC43, and -HKU1. Of note, the majority (76%) of MERS-CoV patients were reportred to have underlying medical conditions such as diabetes, chronic cardiac disease and chronic renal disease [4,8].It has been shown recently that MERS-CoV enters target cells not via the SARS-CoV receptor Angiotensin-converting-enzyme-2 (ACE-2), but via binding to Dipeptidyl-peptidase 4 (DPP-4) [9,10]. Both, ACE-2 and DPP-4 are expressed in several human tissues, including the kidney [11,12]. However, kidney infection has not been compared between MERS-CoV and SARS-CoV, and no post-mortem investigations on patients who succumbed to MERS-CoV have been done. To assess the potential role of kidney affection in MERS-CoV infection, we reviewed all peer-reviewed or communicated reports on MERS cases for information on kidney involvement. The search was performed in PubMed and Promed.Mail (http://www.promedmail.org) using the search term &#8220;coronavirus&#8221; on items published between 15th September 2012 and 16th September 2013. The broad search term was chosen due to inconsistencies of nomenclature before the consensus name MERS-CoV was announced [13].In total, the search revealed 655 publications, including 508 articles in PubMed, and 147 reports in ProMed (Figure&#160;1). All identified publications were reviewed in order to identify clinical descriptions of MERS-CoV infection that included information on kidney function, irrespective of whether they were pointing to normal or impaired function. In cases where two reports on the same patient were available, only the report with the more detailed clinical description was selected. From all identified reports, we extracted information on patient sex and age, country of patient origin and travel route, onset of illness, presence or absence of ARF, date of onset of ARF, day of illness on which ARF occurred, as well as clinical outcome (death or survival). Among 21 publications describing a total of 111 MERS-CoV patients (double-reporting of the same patients could not be excluded), a total of 7 publications were identified that specifically addressed kidney function [2,3,5-7,14,15]. In these 7 publications, a total of 12 MERS patients were described with disease onset dates ranging from April 2012 to March 2013 (Table&#160;1). All patients were male with a mean age of 47&#160;years (range 25&#8211;73&#160;years), originating from Jordan, Saudi Arabia, and Qatar, including imported cases to the United Kingdom (UK), Germany and France. Nine of those 12 patients (75%) developed ARF during the acute course of MERS-CoV infection. For 6 of those 9 patients, dates of onset of ARF were available, suggesting a median time of 11&#160;days (SD &#177;2,0&#160;days) from first symptoms to onset of ARF. Six of 9 (67%) patients with ARF had a fatal outcome, while one of three patients without ARF died. However, the differences found for the fatality rates in cases with and without ARF during MERS-CoV infection were not statistically significant.To investigate a laboratory surrogate for susceptibility of the kidney to MERS-CoV in comparison to SARS-CoV, we analyzed cryoslides of a healthy human kidney for ACE-2 and DPP-4 expression by immunofluorescence. Staining using polyclonal goat-anti-human ACE-2 immunoglobulin (R&D systems) and polyclonal goat-anti-human DPP-4 immunoglobulin (R&D systems) showed that both receptors are abundantly expressed in the healthy human kidney, and both receptors are predominantly found in the epithelial layer of the renal ducts (Figure&#160;2A), as well as in human primary epithelial cells derived from these ducts and in epithelial cells from the human airway tract (Figure&#160;2B).To test whether kidney epithelial cells can actually be infected in-vitro, we performed comparative infection experiments with MERS- and SARS-CoV, using defined cultures of primary kidney epithelial cells (HREpC, Promocell, Heidelberg). As the bronchoalveolar epithelium of the lung constitutes the primary target compartment for both viruses, infection of human primary bronchial epithelial cells (HBEpC, Promocell, Heidelberg) was studied in parallel. As summarized in Figure&#160;2, primary cells up to passage 4 were seeded at densities of 4x105/mL and inoculated with either MERS-CoV strain EMC/2012, or SARS-CoV strain Frankfurt-1 at multiplicities of infection (MOI) of 0.5 for 1&#160;hour. Cells were washed twice and supernatants were harvested immediately after washing, as well as 20 and 40&#160;hours later. Cells were checked regularly for appearance of cytopathic effects (CPE) by light microscopy. Replication of both viruses was quantified by titration of supernatants in duplicate plaque assays on Vero cells as described previously, as well as by real-time RT-PCR [9,16].Pronounced CPE consisting of visible and strong cell lysis occurred only in HREpC, and only upon infection with MERS-CoV (Figure&#160;2C). Neither in HBEpCs upon MERS-CoV infection, nor in HBEpCs or HREpCs upon SARS-CoV infection any CPE was observed. Virus replication was nevertheless detectable by real time RT-PCR in all experiments, confirming successful infection (Figure&#160;1D). The CPE in HREpCs consisted of rounding and detachment of cells, leading to cell death in the majority of cells already after 20&#160;hours. Comparative quantification of infectious virus production between SARS- and MERS-CoV showed that MERS-CoV replicates to high titers in HREpC with almost 1000-fold higher concentrations of infectious MERS-CoV progeny than in HBEpCs. In SARS-CoV infection, only a small difference between primary kidney and bronchial epithelial cells was seen (Figure&#160;1E).Our results from in-vitro infection experiments suggest differences in kidney cell involvement might exist between SARS- and MERS-CoV infection. In a retrospective analysis of 536 SARS cases, only in 36 (6.7%) patients acute renal impairment was diagnosed. ARF occurred rather late in the course of disease after a median of 20&#160;days from the appearance of first symptoms [17]. In our literature review of MERS cases, ARF occurred much earlier, namely in 6 of 9 sufficiently-described patients after a median of 11&#160;days. To date, it is difficult to compare these clinical data with experiences from animal models. The only animal that exhibits clinical signs similar to human MERS-CoV infection is the rhesus macaque, but none of the available studies reported kidney failure in macaques [18-20]. Only very small amounts of viral RNA were detected in kidney tissue, and none in urine. Macaques might be of limited value for the study of MERS-CoV-specific kidney pathology as seen in humans.While no autopsy data have become available for patients infected with MERS-CoV, post-mortem studies have been done systematically in SARS victims. Histopathological findings revealed mainly acute tubular necrosis without evidence of glomerular pathology, which is considered to be a consequence of systemic inflammatory response in the context of multi-organ failure, rather than a specific effect of viral infection of the kidney [17]. SARS-CoV has never been successfully isolated from post mortem kidney tissue of infected patients [21]. In-vitro studies of SARS-CoV with immortalized human proximal tubular epithelial cells showed replication without cell impairment as seen in our study, while no infection of podocyte cell lines and only low-level replication in glomerular mesangial cells (MC) was seen, providing further evidence against specific involvement of the renal tract in SARS-CoV infection [22]. In contrast, MERS-CoV was shown to efficiently replicate in a broad range of bat, primate and also human kidney epithelial cell lines that are commonly used as laboratory models [9]. The abundant expression of both viruses&#8217; entry receptors in kidney epithelium argues against receptor-dependent limitations to viral kidney tropism. Specific interference of MERS-CoV with the induction of the interferon response provides one of many possible explanations for an increased capacity of MERS-CoV to replicate in kidney cells [23]. The clear differences in viral permissiveness of primary kidney epithelial cells suggest these cells to be appropriate models for the identification of host-specific restriction factors in the future. We are aware that cell culture resembles a limited model of viral infection which will not reflect the complexity of infection in-vivo. Nevertheless, our results have some congruence with clinical observations in a well-investigated MERS case [5]. Virus in the urine has been detected as late as day 12 of illness, albeit at low concentrations. Virus was detected in the urine but not in the blood, which could indicate autonomous virus replication in the kidneys. Of note, by using a model to assess the ability of CoVs to persist in the environment, it was shown that MERS-CoV has similarities to viruses that are transmitted via the fecal-oral route [24]. The authors suggest that this could hint for oral-urine transmission of MERS-CoV. Further data on virus shedding in the urine over the course of disease are urgently needed. If the kidney should indeed constitute a site of primary virus replication, shedding of virus in the urine might provide a possible source for human-to-human transmission, especially in health-care settings or among close family contacts [3,6,7]. Clinical guidelines should consider the possibility that MERS patients may benefit from early induction of renoprotective treatment.\nCompeting interests\nThe authors declare that they have no competing interests.\nAuthors\u2019 contributions\nIE, MAM designed the experiments, IE, MAM, SK performed experiments and analysis, IE, CD wrote the manuscript, IE, DNG, CD contributed to the final version of the manuscript. All authors read and approved the final manuscript.",
        "pmid": "24364985",
        "title": "In-vitro renal epithelial cell infection reveals a viral kidney tropism as a potential mechanism for acute renal failure during Middle East Respiratory Syndrome (MERS) Coronavirus infection",
        "journal_title": "Virology Journal",
        "authors": "Isabella Eckerle;Marcel A M\u00fcller;Stephan Kallies;Daniel N Gotthardt;Christian Drosten",
        "doi": "10.1186/1743-422X-10-359"
    },
    {
        "keywords": "COVID-19;2019-nCoV, outbreak;Second derivative;Infectious disease epidemic;Dynamic modeling",
        "abstract": "Background\nSimilar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively.Methods\nIn this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2&#160;months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases.Results\nResults from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14&#8201;days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14&#8201;days on February 18, 2020.Conclusions\nThe coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.",
        "body": "Introduction\nThe epidemic of COVID-19 is caused by a novel virus first detected in Wuhan, China. This virus was previously named as 2019-nCoV and it is a positive, enveloped, single-strand RNA virus. It also shares a lot of similarities with two other coronaviruses, the MERS-CoV (Middle East Respiratory Syndrome) and SARS-CoV (Severe Acute Respiratory Syndrome). Outbreak of the COVID-19 started with the report of a first suspected case on December 8, 2019 in Wuhan. The first&#160;two months of the epidemic covered three significant holidays, including the New Year of 2020, the Chinese New Year&#8217;s Day with vacations from January 24 to February 2, 2020, and the Lantern Festival on February 8, 2020. During this period, one study by the Chinese Center for Disease Prevention and Control (CDC) and Hubei Provincial CDC with data collected by Wuhan CDC documented the details of the epidemic day by day from December 8, 2019 to January 21, 2020 [1]. Data in this study showed that detected and confirmed cases with COVID-19 infection declined from the peak of 44 on January 8 to only 2 on January 19, 2020, suggesting that the epidemic was likely under control.China officially declared the epidemic as an outbreak on January 20 when obvious human-to-human transmissions were ascertained with reagent probes and primers distributed to local agencies on that day. Immediately following the declaration, massive actions were taken the next day to curb the epidemic at Wuhan, and soon spread to the whole country from central to local government, including all sectors from business to factories and to schools. On February 23, 2020, Wuhan City and other cities along with the main traffic lines around Wuhan were locked down. Rigorous efforts were devoted to 1) identify the infected and bring them to treatment in hospitals for infectious diseases, 2) locate and quarantine all those who had contact with the infected, 3) sterilize environmental pathogens, 4) promote mask use, and 5) release to the public of number of infected, suspected, under treatment and deaths on a daily basis.On January 24, 2020, the New Year&#8217;s Eve and 25, the Chinese New Year&#8217;s Day, President Xi Jinping held a special meeting at the Central Chinese Government and decided to implement massive national efforts to curb the epidemic. An Anti- COVID-19 Group headed by Premier Li Keqiang was established to lead the massive national efforts. Vice Premier Sun&#160;Chunlan was sent to Hubei and Wuhan to directly lead the local efforts. A massive number of detection kits were made available to all locations to test all susceptible patients for final diagnosis. People in other cities and provinces who either traveled to or out of Wuhan were quarantined, with suspected patients being diagnosed and treated.The sudden escalation of the control and the spread of the number of infected and deaths, however, ignited strong emotional responses of fear and panic among people in Wuhan. The negative emotional responses soon spread from Wuhan to other parts of China, and further to the world via almost all communication channels, particularly social media. The highly emotional responses of the public were fueled by (1) sudden increases in the number of detected new cases after the massive intervention measures to identify the infected; (2) massive growing needs for masks; (3) a large number of suspected patients waiting to confirm their diagnose; (4) a large number of diagnosed COVID-19 patients for treatment; and (5) a growing number of deaths, despite national efforts to improve therapy, including the decision to build two large hospitals within a period of days. The emotional responses, mostly stimulated by the daily release of data have created a big barrier for effective control of the epidemic as has been observed in other epidemics of similar nature [2, 3].It is a paradox that during the early period of an epidemic, little is known or available about the new infections; while the need for such information is at the highest level. This is particularly true for the COVID-19. The occurrence of this epidemic may follow a nonlinear, chaotic and catastrophic process, rather similar to the epidemic of SARS that occurred in Hong Kong in 2003 [2], the Ebola epidemic in West Africa during 2013&#8211;16 [4, 5], the pandemic of 2009 H1N1 epidemic started [6&#8211;8] and the recent measles outbreaks in the United States (US) [9]. Similar to an eruption of a volcano or occurrence of an earthquake, no matter how closely it is monitored, how much research we have done, how much we&#160;know about it, no one knows&#160;for sure if and when&#160;the virus infection will become an outbreak. Therefore, there is no so-called rational responses, no standard-operating-procedure (SOP) to follow, no measures to take without negative consequences [2].However, defining the COVID-19 as nonlinear and chaotic does not mean that we cannot do anything after we knew it was an outbreak, but simply waiting. On the contrary, defining it as nonlinear and chaotic will better inform us to make right decisions and to take appropriate actions. (1) During the early stage of an infection, which we cannot tell whether it will be growing into an outbreak, we must closely monitor it using limited data and to find the early signs of change and to predict if&#160;and when it will become an outbreak; (2) After it is declared as an outbreak, it is better to take actions as soon as possible since infectious diseases can be controlled even without knowledge of the biology [10]; and evaluate if the control measures work.The ultimate goal of this study is to&#160;attempt to provide some solutions to this paradox by providing early messages to inform control measures, to be optimistic and not panic, to ask right questions, and to take right actions.\nMethods\n\nDaily detected and confirmed cases\nData for this study were daily cumulative cases with COVID-19 infection for the first two months (63&#8201;days) of the epidemic from December 8, 2019 to February 8, 2020. These data were derived from two sources: (1) Data for the first 44&#8201;days from December 8, 2019 to January 20, 2020 were derived from published studies that were determined scientifically [1]. Since no massive control measures were in place during this period, these data were used as the basis to predict the underlying epidemic, considering the overall epidemic. The best fitted model was used to predict the detectable cases and was used in assessing detection rate at different periods for different purposes.Data for the remaining 19&#8201;days from January 21 to February 8, 2020 were taken from the daily official reports of the National Health Commission of the People&#8217;s Republic of China (http://www.nhc.gov.cn/xcs/yqfkdt/gzbd_index.shtml). These data were used together with the data from the first source to monitor the dynamic of COVID-19 on a daily basis to 1) assess whether the COVID-19 epidemic was nonlinear and chaotic, 2) evaluate the responsiveness of the epidemic to the massive measures against it, and 3) inform the future trend of the epidemic.\nUnderstanding of the detected cases on a daily basis\nIn theory, the true number of persons with COVID-19 infection can never be known no matter how we try to detect it. In practice, of all the infected cases in a day, there are some who have passed the latent period when the virus reaches a detectable level. These patients can then be detected if: a) detection services are available to them, b) all the potentially infected are accessible to the services and are tested, and c) the testing method is sensitive, valid and reliable. When reading the daily data, we must be aware that the detected and diagnosed cases in any day can be great, equal, or below the number of detectable. For example, a detectable person in day one can be postponed to next day when testing services become available. This will result in reduction in a detection rate&#8201;&#8201;100% in the testing day.\nModel daily change in the epidemic\nWe started our modeling analysis with data of cumulative number of diagnosed COVID-19 infections per day. Let xi =diagnosed new cases at day i, i =(1, 2, &#8230;t), the cumulative number of diagnosed new cases F(x) can be mathematically described as below:\n1\\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ F(x)={\\int}_{i=1}^t{x}_i=\\sum \\limits_{i=1}^t{x}_i. $$\\end{document}Fx=&#8747;i=1txi=&#8721;i=1txi.Results of F(x) provide information most useful for resource allocation to support the prevention and treatment; however F(x) is very insensitive to changes in the epidemic. To better monitor the epidemic, the first derivative of F(x) can be used:\n2\\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ F^{\\prime }(x)={\\int}_{i=1}^{\\left(t+1\\right)}{x}_i-{\\int}_{i=1}^t{x}_i=\\sum \\limits_{i=1}^{t+1}{x}_i-\\sum \\limits_{i=1}^t{x}_i $$\\end{document}F&#8242;x=&#8747;i=1t+1xi&#8722;&#8747;i=1txi=&#8721;i=1t+1xi&#8722;&#8721;i=1txiInformation provided by the first derivative F&#8201;&#8242;&#8201;(x) will be more sensitive than F(x), thus can be used to gauge the epidemic. Practically, F&#8201;&#8242;&#8201;(x) is equivalent to the newly diagnosed cases every day. A further analysis indicates that F&#8201;&#8242;&#8201;(x), although measuring the transmission speed of the epidemic, provides no information about the acceleration of the epidemic, which will be more sensitive than F&#8201;&#8242;&#8201;(x). We thus used the second derivative F&#8243;(x):\n3\\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ {F}^{{\\prime\\prime} }(x)={F}^{\\prime}\\left({x}_{\\mathrm{i}+1}\\right)-{F}^{\\prime}\\left({x}_i\\right) $$\\end{document}F&#8243;x=F&#8242;xi+1&#8722;F&#8242;xiMathematically, F&#8242;&#8242;(x) measures the acceleration of the epidemic or changes in new infections each day. Therefore, F&#8242;&#8242;(x)&#8201;&#8776;&#8201;0 is an early indication of neither acceleration nor deceleration of the epidemic; F&#8242;&#8242;(x)&#8201;>&#8201;0 presents an early indication of acceleration of the epidemic; while F&#8242;&#8242;(x)&#8201;<&#8201;0 represents an early indication of deceleration.\nModeling the epidemic with assumption of no intervention\nWith a close population assumption and continuous spread of the virus, the number of detected cases can be described using an exponential model [10]. We thus estimated the potentially detectable new cases every day for the period by fitting the observed daily cumulative cases to an exponential curve:\n4\\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ F\\left(\\overline{x}\\right)=\\left(\\alpha \\right){\\mathit{\\exp}}^{\\beta (t)},\\mathrm{t}=\\left(12/8/2019,12/9/2019,\\dots, 1/20/2020\\right), $$\\end{document}Fx&#175;=&#945;exp&#946;t,t=12/8/201912/9/2019&#8230;1/20/2020,where, &#945; =number of expected cases at the baseline and &#946; = growth rate per day.\nEstimation of daily detection rate\nTo assess the completeness of the diagnosed new cases on a daily basis, we used Eq (4) first to obtain a time series of \\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ F\\left(\\overline{x}\\right) $$\\end{document}Fx&#175; to represent the estimates of cumulative number of potentially detectable cases; we then used the first derivative \\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ F^{\\prime}\\left(\\overline{x}\\right) $$\\end{document}F&#8242;x&#175; to obtain another time series of observed new cases each day; finally, with the observed F&#8201;&#8242;&#8201;(xi) and model predicted \\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ F^{\\prime}\\left(\\overline{x}\\right) $$\\end{document}F&#8242;x&#175;, we obtained the detection rate Pi for day i as:\n5\\documentclass[12pt]{minimal}\n\t\t\t\t\\usepackage{amsmath}\n\t\t\t\t\\usepackage{wasysym} \n\t\t\t\t\\usepackage{amsfonts} \n\t\t\t\t\\usepackage{amssymb} \n\t\t\t\t\\usepackage{amsbsy}\n\t\t\t\t\\usepackage{mathrsfs}\n\t\t\t\t\\usepackage{upgreek}\n\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n\t\t\t\t\\begin{document}$$ {P}_i=F^{\\prime}\\left({x}_i\\right)/{F}^{\\prime}\\left({\\overline{x}}_i\\right),\\mathrm{i}=\\left(12/8/2019,12/9,2019\\dots, 2/8/2020\\right) $$\\end{document}Pi=F&#8242;xi/F&#8242;x&#175;i,i=12/8/201912/92019&#8230;2/8/2020We used these estimated Pi in this study in several ways.\nBefore January 20, 2020 when the massive intervention was not in position, an estimated Pi&#8201;>&#8201;1 was used as an indication of detecting more than expected cases, while an estimated Pi&#8201;<&#8201;1 as an indication of detecting less than expected cases.During the early period of massive intervention, an increase trend in Pi over time was used as evidence supporting the effectiveness of the massive intervention in detecting and quarantining more infected cases.During the period 14&#8201;days (latent period) after the massive intervention, Pi&#8201;<&#8201;1 was used as evidence indicating declines in new cases rather than under-detection; thus,&#160;it was used as a sign of early declines in the epidemic.The modeling analysis was completed using spreadsheet. As a reference to assess the level of severity of the COVID-19 epidemic, the natural mortality rate of Wuhan population was obtained from the 2018 Statistical Report of Wuhan National Economy and Social Development.\nResults\n\nCumulative number of detected and diagnosed cases\nThe COVID-19 epidemic was initiated in Wuhan, the Provincial Capital of Hubei Province with a total population of 14.2 million, including 5.1 million mobile population. The mortality rate was 5.5/1000 for Wuhan residents with most available data in 2018. Assuming all diagnosed cases in China were infected in Wuhan (an exaggerated scenarios for illustration purpose), the two-month incidence rate of COVID-19 was 2.6/1000 among Wuhan residents. Based on reported case mortality of 2.3%, the population-based mortality of COVID-19 was 0.6/1000, or 1/9th of the mortality of Wuhan residents.Figure 1 presents the cumulative diagnosed cases F(x) and major events during the study period from December 8, 2019 to February 8, 2020. During the period, a total of 37,198 cases were diagnosed and reported. The daily cases varied from 0 to 3886 with the median cases of 199 (January 8, 2020), and inter-quarter range (IQR) of 24 (December 23), and 830 (January 23, 2020).\nFig. 1Cumulative number of diagnosed COVID-19(2019-nCoV) infection F(x) and key events before, during and after declaration of the outbreak in the first 2&#160;months of the Epidemic in China\nDynamics of the epidemic and response to massive interventions\nThe dynamic changes based on the observed F(x) in Fig.&#160;1 were presented in Fig.&#160;2 using the first derivatives F&#8201;&#8242;&#8201;(x) (top panel of the figure) and the second derivative F&#8201;&#8242;&#8201;&#8201;&#8242;&#8201;(x) (bottom panel of the figure), respectively. Before the declaration of outbreak, information provided by the two dynamic measured was similar: not much variations were revealed relative to the changes after the outbreak. These findings suggest the nonlinear and chaotic character of the COVID-19 outbreak.\nFig. 2The first F&#8242;(x) and second derivative F&#8243;(x) of diagnosed COVID-19 (formally 2019-nCoV infection) F(x) before, during and after declaring the outbreak in first 2&#8201;months of the EpidemicAfter declaring the outbreak on January 20, information revealed by F&#8201;&#8242;&#8201;&#8201;&#8242;&#8201;(x) differed much from F&#8201;&#8242;&#8201;(x). Based on information from F&#8201;&#8242;&#8201;(x), the newly diagnosed F&#8201;&#8242;&#8201;(x) cases increased progressively with some fluctuation, then peaked on February 4, 2020, and followed by a decline. The increases in the diagnosed cases could be either due to the natural growth of the epidemic in itself, or due to the interventions to detect the infected or both. Furthermore, F&#8201;&#8242;&#8201;(x) provided no sign of epidemic decline until February 4, 2020. In other words, we have to wait for at least 14&#8201;days after the massive anti-COVID-19 epidemic without using information derived from F&#8201;&#8242;&#8201;&#8201;&#8242;&#8201;(x).Quite different from F&#8201;&#8242;&#8201;(x), F&#8201;&#8242;&#8242;&#8201;(x) removed the time trend of F&#8201;&#8242;&#8201;(x) to show the acceleration/deceleration of diagnosed COVID-19. Consequently, F&#8201;&#8242;&#8201;&#8201;&#8242;&#8201;(x) was much more sensitive than F&#8201;&#8242;&#8201;(x) to gauge the intrinsic dynamics of the epidemic in response to the massive anti-COVID-19 action. Since January 21, 2020 after the massive anti-COVID actions, the F&#8201;&#8242;&#8201;&#8201;&#8242;&#8201;(x) suddenly became very active, as indicated by the alternative accelerations and decelerations. F&#8201;&#8242;&#8201;&#8201;&#8242;&#8201;(x) reached the peak on January 27 after the distribution of large number of test kits on January 26, which was an action based on the decision at the central government level in a meeting held by Chinese President Xi Jinping on January 24 and 25, the Chinese New Year&#8217;s Eve and New Year&#8217;s Day.In addition, the estimated F&#8201;&#8242;&#8242;&#8201;(x) captured three significant decelerations on January 28, February 5 and 6 (two days in a row), and 8, 2020 respectively; corresponding to the intensified massive actions in locating and treating the infected, locking down more communities, plus mask use and massive pathogen sterilization in neighborhood environment. In addition to informing whether the epidemic was responsive to the massive interventions, information from&#160;F&#8201;&#8242;&#8242;&#8201;(x) signaled an overall downturn of the epidemic since the beginning of the massive anti-COVID-19 action on January 21, 2020. This was further pronounced by the band region between the two dotted lines in Fig. 2. Despite zigzags, an overall downward trend in F&#8201;&#8242;&#8242;&#8201;(x) was clearly revealed by the downward and progressively narrowing down band region. This trend strongly indicates that the epidemic could be brought under control soon with the current interventions in place.\nExponential growth and detection rate\nThe observed F(x) fit the exponential model of Eq. 4 well with R2&#8201;=&#8201;0.9778. The estimated &#945; =1.1070, representing the first person who was infected and ignited the epidemic. The estimated &#946; =0.1716, representing the growth rate. Using this estimated growth rate, it takes only 4&#8201;days for the diagnosed COVID-19 to double.Figure 3 presents the daily detection rates, estimated with the fitted exponential model from day one of the epidemic to the last day of the study period. Based on findings in this figure and data from Figs.&#160;1 and 2, we divided the COVID-19 epidemic during the first two months of the epidemic into five phases.\nFig. 3Estimated daily detection rate Pi of COVID-19 (2019-nCoV) infection before, during and after declaration of the outbreak, the first 2 months of the Epidemic in ChinaPhase 1 was from December 8 to 25, 2019. During this period, the detection rate Pi was high overall, with fluctuations around and above 100%. This was corresponding to the early period after the first suspected case was identified and diagnosed.Phase 2 was from December 26, 2019 to January 8, 2020, covering the New Year&#8217;s Day. The detection rate Pi fluctuated at around 50% with the lowest of 17% on December 31, 2019 and the highest of 108% on January 8.Phase 3 was from January 8 to 20, 2020, and it was featured with a progressive decline in the estimated Pi from 105% on January 8, 2020 to 1% on January 20, 2020. This progressive declining period was the time for the Chinese to prepare for the traditional Chinese New Year&#8217;s with the longest and highest level of celebration. Unfortunately, the COVID-19 as an outbreak was silently stepping in during this period.Phase 4 was from January 20 to 27, 2020 with the estimated Pi increased from 1% on January 20, 2020 to surpass 100%, and reached the peak of 170% on January 27, 2020. This period was corresponding to the initiation and progressive intensifying of the massive intervention organized and coordinated by the Central Government of China.Phase 5 started from January 27, 2020 to the end of the study period, corresponding to the sustained massive national efforts, plus frequent emphases. Different from the previous four phases, reductions in the estimated Pi during this phase were not an indication of under-detection but an indication of declines in the epidemic reflected by the detected and confirmed cases of COVID-19. This is because the model predicted Pi did not consider any interventions but natural growth of the epidemic.Based on Figs. 2 and 3 (Phase 4 and 5), three pieces of information can be derived: (1) The epidemic was highly sensitive to external interventions, supporting the nonlinear and chaotic characters revealed by the long latent period in the first three phases; (2) the massive national efforts were highly effective in detecting the detectable COVID-19; (3) signal for the COVID-19 in China to decline appeared on January 21 in 2020, 14&#8201;days before the start of eventual declines on February 4, as indicated by F&#8201;&#8242;&#8201;&#8201;&#8242;&#8201;(x) and F&#8201;&#8242;&#8201;(x) in Fig. 2 and Pi in Fig. 3.\nDisscussion\nIn this study, we used a novel approach to distill information from the cumulative number of diagnosed cases of COVID-19 infection. Among various types of surveillance data, this data often reported the earliest and on a continuous basis with high completeness and are most widely available. In addition, patients with a diagnosed infection are those with high likelihoods to spread the virus to others. Findings from this study provided useful information in a real time manner to monitor, evaluate and forecast the COVID-19 epidemic in China. The methods used in this study although somewhat mathematical, are easy to follow while information extracted from the commonly used data with the methods are highly useful and more sensitive than the daily new and cumulative cases.\nNonlinear and chaotic nature of the COVID-19 outbreak\nAlthough an analytical demonstration of the COVID-19 outbreak as nonlinear, chaotic and catastrophic requires more time to wait till the epidemic ends, evidence in the first 2&#8201;months suggests that the COVID-19 outbreak in China is nonlinear and chaotic. The epidemic emerged suddenly after a long latent period without dramatic changes as revealed from the cumulative cases, and their first and second derivatives. The high responsiveness of the epidemic to interventions adds additional evidence supporting the chaotic and catastrophic nature, and demonstrating the selection of a good timing to start intervention. Many of these characters are similar to those observed in the 2003 SARS epidemic started in Hong Kong [2], the 2013&#8211;16 Ebola spread in the West Africa [4, 5], the 2009 pandemic of H1N1 started in the US [6&#8211;8], and the measles outbreaks over 80 cities in the US recently [9]. Even the seasonal common flu has been proved to have a nonlinear component [11, 12].The significance of nonlinear and chaotic nature of COVID-19 means that no methods are available to predict exactly at what point in time the epidemic will emerge as an outbreak, just like volcanoes and earthquakes. Therefore, practically there is no so-called a best time or missed the best time to take actions. There will also no so-called rational analysis and rational responses. There is no silver bullet to use, no standard-operating-procedure (SOP) to follow, and no measures without negative consequences to control the epidemic [2]. For example, it took more than 6&#8201;months for both the US and the WHO to determine the 2009 H1N1 pandemic as an outbreak [13, 14]. Therefore, knowing the nonlinear and chaotic nature of an epidemic outbreak, like COVID-19, for all stockholders will be essential to the mobilization of resources, working together, taking all actions possible to control the epidemic, and minimizing the negative consequences.Specifically, what we can do to deal with an outbreak like COVID-19 would be to (1) collect information as early as possible, (2) monitor the epidemic as close as possible just like we do for an earthquake and make preparations for a hurricane and (3) communicate with the society and use confirmed data appropriately reframed not causing or exacerbating fear and panic in the public, stress and distress among medical and public health professionals, as well as administrators to make right decisions and take the right strategies at the right time in the right places for the right people.Knowing the nonlinear and chaotic nature is also essential for taking actions to control the outbreak of an epidemic like the COVID-19 infection. As soon as an outbreak is confirmed, the follow measures should be in position immediately 1) closely and carefully monitor the epidemic; 2) take evidence-based interventions to control the epidemic, 3) actively assess responses of the epidemic to the interventions; 4) allow errors in the intervention, particularly during the early period of the epidemic, 5) always prepare for alternatives.Another confusion is, when an epidemic starts, everyone asks what it is? How does it happen? How should I do to avoid infection? Is there any effective treatment? Answering these questions takes time, but there is no need to wait till all these questions are resolved before taking actions. We can take actions to prevent COVID-19 immediately while waiting for answers to these questions. This is because we have the evidence-based strategy for control and prevention of any infectious disease without complete understanding of an infection. That is so-called Tri-Component Strategy: locating and controlling the sources of infection, identifying and blocking the transmission paths, and protecting those who are susceptible [10].This was just what China has done, is doing, and will continue to do this time. Typical examples of control and prevention measures include locking down of cities, communities, and villages with potential of large scale transmission, massive environment sterilization, promotion of mask use, efforts to locate, isolate and treat the infected. More importantly, most of these actions are initiated, mobilized, coordinated and supported by the government from central to local, and enhanced by volunteers and international support.\nHighly effective of the national effort\nAnother important piece of findings is that we detected the effect of the national efforts taken by China from the beginning when they were in position till the end of this study. For example, from the second derivative, we observed increases in the infected through the action on January 22, 2020, the next day after the massive intervention started on January 21, 2020. This result was also picked up by the exponential modeling. From day one on January 21, 2020 when the massive intervention measures activated to February 4, 2020 is the latent period of COVID-19 infection. The second derivative precisely recorded the change in newly diagnosed cases in response to the massive measures, reflected as the rapid increase in detection rate, consistent with the result from the exponential modeling analysis.The detected responsiveness of the epidemic to the intervention provided data to predict the occurrence of deceleration of the epidemic on February 4, 2020 if the same measures persist, which was exactly what we observed from the second derivative. Based on the findings from our analysis, the COVID-19 in China may end up soon. Despite a delay of 43&#8201;days from the first confirmed cases on December 8, 2019 to January 20, 2020, the COVID-19 epidemic was highly responsive to massive interventions, supporting the effectiveness of these interventions. It is our prediction that the outbreak of the COVID-19 infection will be brought under control by the end of February 2020, given the effective control measures known to everyone, increases in immune level in the total population due to latent infections, and most widely spread of knowledge and skills for infectious disease control and prevention among the 1.4 billion people in China.\nEffective methods for surveillance\nThere are a number of advantages of methods we developed and used in this study. First, framing the diagnosed cases as the cumulative, the first and the second derivative constructs a system to gauge the epidemic, with the cumulative cases showing the overall level of the epidemic, the first derivative to reflect the change of the epidemic, and the second derivative to monitor the speed of change. By inclusion of the mortality rate as a reference, results from our approach will be (1) comprehensive to inform the public to be prepared, not scared and not to blame others; (2) useful for administrators to make decisions; (3) valuable for medical and health professionals to take actions.Second, we conceptually separated (1) the true number of infections, which will never be practically detected, from (2) the infections that are practically detectable if services are available and accessible and detection technologies are sensitive and reliable, and (3) the actually detected cases of infections. This classification greatly improved our understanding of the observed data as well as findings from the two derivatives, and aided us in assessing the responsiveness to the massive interventions, and predicting of the epidemic over time. The clarification also enhanced our analytical approach by adding an exponential model to evaluate the detection rate and to bring more data assessing the responsiveness of the epidemic to the massive interventions. We highly recommend the inclusion of the methods as a part of routine surveillance in disease control and prevention institutions.\nLimitations and future plan\nThere are limitations. First, this study covered only the first 2&#8201;months of the epidemic. We will continue to evaluate the utility of this method as we follow the development of the epidemic. Second, the methods used in this study was based on a close population. This hypothesis may not be true because of a large number of people with potential history of exposure in China traveled to other countries. Up to February 8, 2020, the total cases diagnosed were 37,552 worldwide (Worldometer on Coronavirus) with 37,198 in China, which accounted for 99.1% of the total number of the world. Therefore, the impact of close-population assumption would be rather limited. Third, there was a lack of individual patient-level data for detailed analysis. Fourth, our model can be further improved with other data, such as cases by severity, number of the suspected, number of those who received treatments and treatment results. We will follow the epidemic closely and prepare for further research on the topic when more data become available.Despite the limitations, this study provided new data to encourage those who are infected to better fight against the infections; to inform and encourage the general public, the medical and health professionals and the government to continue their current measures and to think of more measures that are innovative and effective to end the COVID-19 epidemic. One of the greatest motivations for this study is to attempt to provide right information at the population level in a real manner to complement the data from micro-organism centered and laboratory-based biological, molecular, pharmacological and clinical information in both the academic and the mass media that often scare rather than encourage people, even health professionals. Of the diagnosed COVID-19 cases, less than 20% are severe. Findings from our study indicated that there is no need to be panic from a public health population perspective. Although the total cases COVID-19 reached to big numbers, but the 2-month incidence rate was about a half of the natural death rate for Wuhan residents.",
        "pmid": "32158961",
        "title": "First two&#160;months of the 2019 Coronavirus Disease (COVID-19) epidemic in China: real-time surveillance and evaluation with a second derivative model",
        "journal_title": "Global Health Research and Policy",
        "authors": "Xinguang Chen;Bin Yu",
        "doi": "10.1186/s41256-020-00137-4"
    },
    {
        "keywords": "porcine deltacoronavirus;nucleocapsid;monoclonal antibodies",
        "abstract": "Porcine deltacoronavirus (PDCoV), first identified in 2012, is a swine enteropathogen now found in many countries. The nucleocapsid (N) protein, a core component of PDCoV, is essential for virus replication and is a significant candidate in the development of diagnostics for PDCoV. In this study, monoclonal antibodies (mAbs) were generated and tested for reactivity with three truncations of the full protein (N1, N2, N3) that contained partial overlaps; of the five monoclonals chosen tested, each reacted with only the N3 truncation. The antibody designated 4E88 had highest binding affinity with the N protein and was chosen for in-depth examination. The 4E88 epitope was located to amino acids 308-AKPKQQKKPKK-318 by testing the 4E88 monoclonal for reactivity with a series of N3 truncations, then the minimal epitope, 309-KPKQQKKPK-317 (designated EP-4E88), was pinpointed by testing the 4E88 monoclonal for reactivity with a series of synthetic peptides of this region. Homology analysis showed that the EP-4E88 sequence is highly conserved among PDCoV strains, and also shares high similarity with sparrow coronavirus (HKU17), Asian leopard cat coronavirus (ALCCoV), quail coronavirus (UAE-HKU30), and sparrow deltacoronavirus (SpDCoV). Of note, the PDCoV EP-4E88 sequence shared very low similarity (<22.2%) with other porcine coronaviruses (PEDV, TGEV, PRCV, SADS-CoV, PHEV), demonstrating that it is an epitope that can be used for distinguishing PDCoV and other porcine coronavirus. 3D structural analysis revealed that amino acids of EP-4E88 were in close proximity and may be exposed on the surface of the N protein.",
        "body": "1. Introduction\nThe genus Deltacoronavirus is a relatively new member of the Coronavirus subfamily, that consists of avian and mammalian CoVs [1]. Among these is porcine deltacoronavirus (PDCoV), originally discovered from fecal samples of pigs in Hong Kong in 2012 [2]. Since then, PDCoV has been reported in multiple states of the United States and Canada [3,4,5,6], South Korea [7], mainland China [8,9] and Thailand [10] causing economic losses to each country&#8217;s swine industry. Clinically, Porcine deltacoronavirus (PDCoV) is indistinguishable from porcine epidemic diarrhea virus (PEDV) and transmissible gastroenteritis virus (TGEV), both Alphacoronaviruses, it is characterized by severe diarrhea, vomiting, and dehydration in piglets, and histopathological lesions typical of atrophic enteritis [11]. The clinical and epidemiological similarities between PDCoV and other porcine intestinal pathogenic coronaviruses make diagnosis and treatment of these viruses a challenge, highlighting the need for discriminating diagnostic methods [12].PDCoV is an enveloped, single-stranded, positive-sense RNA virus with a 25 kb genome [13]. In the genome opening reading frames(ORFs), ORF1a and ORF1b account for two-thirds of its genome, which encode two polymerase proteins, pp1a and pp1ab [14]. The last one-third of the genome encodes four structural proteins: spike (S protein), envelope (E protein), membrane (M protein), nucleocapsid (N protein), and three accessory proteins (NS6 and NS7/NS7a) [15,16]. NS7 ORF is included into N gene sequence. Moreover, NS7a is contained into NS7 ORF [16]. The N protein is a phosphoprotein and binds to RNA genome, which supplies a structural basis to the helical nucleocapsid [17,18]. The common characteristics for all CoVs N proteins are high expression levels early in the infection and high anti-N antibody levels. N protein also owns multiple functions in pathogenesis, viral replication, and immune system interference [17]. These characteristics make the N protein an ideal target for development of serological methods based on purified protein [19] or antigenic epitopes [20].PDCoV N protein is highly conserved among PDCoV strains but had low sequence identity with other porcine coronavirus, such as PEDV, TGEV, and PRCV [21]. Although CoV N proteins have low sequence identity, all share the same domain and structure organization [18,22]. For diagnosis of PDCoV, serological assays based on N protein, such as indirect ELISA and fluorescent microsphere immunoassay, have proven to be highly sensitive [23]. Monoclonal antibodies of PDCoV N protein have also proven useful in fluorescent antibody and immunohistochemistry staining methods for identification of PDCoV-infected cells or intestinal tissues [23]. However, the cross-reactivity between porcine coronaviruses in these assays makes accurate diagnoses difficult [24,25,26], thus development of discriminate diagnostic assays for PDCoV is essential. In this study, the N protein of PDCoV was expressed in E. coli, purified, then used to produce mouse monoclonal antibodies. The epitope (EP-4E88/309-KPKQQKKPK-317) of the antibody with the highest N protein binding affinity was extensively investigated. Sequence alignment analysis revealed that the sequence of EP-4E88 is highly conserved among porcine deltacoronavirus strains, but has very low sequence similarity to other porcine coronavirus (PEDV, TGEV, PRCV, SADS-CoV, PHEV). Among them, TGEV, PRCV N protein are identical, because PRCV is a S gene deletion of TGEV [27]. Besides, SADS-CoV also known as swine enteric alphacoronavirus (SeACoV) [28] and porcine enteric alphacoronavirus (PEAV) [29]. The results of I-TASSER server for 3D structure prediction showed that amino acids of EP-4E88 was in close proximity and may be exposed on the surface of the N protein. Our findings on this PDCoV N protein epitope added insight for developing epitope-associated diagnostics and vaccines.\n2. Results\n\n2.1. Expression, Purification, and Characterization of Full-Length Porcine Deltacoronavirus (PDCoV) Recombinant N Protein (rNP)\nAs seen in the SDS-PAGE results (Figure 1A), the expressed PDCoV-rNP was soluble and found predominantly in the supernatant post sonication (lane 4), and lanes 6 show the Ni-NTA purified rNP at the expected size, 44 kDa. The Western blot (Figure 1B), demonstrates that the recombinant N protein was specifically recognized by the pig anti-PDCoV hyperimmune serum at a 1:200 dilutions.\n2.2. Production and Screening of PDCoV rNP mAbs\nPurified PDCoV rNP was used to immunize BALB/c mice; the mouse with the highest antibody titer was sacrificed for hybridoma production. Hybridomas were screened four times by indirect ELISA, and ultimately five hybridomas (11E, 7D2, 4E88, 6A5, and 3B5) were chosen for further testing. The hybridomas were identified as IgG1 &#312; light chain isotype using a mouse monoclonal antibody isotyping ELISA kit (Proteintech Group, Inc.). As can be seen from the representative indirect immunoinfluscent assay (IFA) images (Figure 2A) the monoclonal 4E88 showed the greatest reactivity with the N protein in infected cells. The reactivity of infected cells detected by each monoclonal was quantitated by Imagine Pro plus 6.0 and the results are shown in the bar graph in Figure 2B. ELISA results (Figure 2C) also demonstrated that monoclonal 4E88 had greater reactivity with the N-protein than did the other mAbs. PDCoV NS6 was used as a negative control. To determine if the binding of the mAbs to the N protein required its conformational integrity, native and denatured N was dotted onto PVDF membrane and reacted with each mAb. The results (Figure 2D) show that 4E88, 3B5, 11E, and 7D2 react well with denatured N protein, indicating they recognize linear epitopes. Monoclonal 6A5 did not react with denatured N protein, indicating that it may recognize a conformational epitope.Next, IEDB Analysis Resource online analysis software was used to predict the B-cell epitope of N protein. Based on the analysis results, we divided the N protein into three parts: N1 (1-124aa), N2 (113-240aa), N3 (221-342aa). The ELISA results (Figure 2E), showed that all the five mAbs recognized the N3 fragment but did not react with N1 and N2. \n2.3. Epitope Mapping of PDCoV NP\nTo locate the epitope on N3 recognized by mAb 4E88, three truncations of N3 were constructed: N3-1 (221-267 aa), N3-2 (261-307 aa), and N3-3 (301-342 aa) (Figure 3A). ELISA (Figure 3D) and dot blot (Figure 3E) results showed that mAb 4E88 reacted only with the N3-3 fragment. From here, four truncations of the N3-3 fragment were constructed: N3-3-1 (301-330 aa), N3-3-2 (308-342aa), N3-3-3 (221-328 aa), and N3-3-4 (221-318 aa). N3-3-1 was not expressed successfully, and although mAb 4E88 did react with N3-3-2 the signal was weak. Both N3-3-3 and N3-3-4, which contain most of the N3-3-1 region, reacted with mAb 4E88, but the N3-3-3 region reacted strongly while N3-3-4 reacted somewhat weakly. These results demonstrated that the amino acids 308-318 are at a minimum necessary for the 4E88 interaction. Peptides spanning these amino acids were synthesized (Sangon Biotech, China) (Figure 4A,B) for subsequent experiments.\n2.4. Identification of the Minimal Epitope\nThe synthesized peptides were used as antigens in ELISA (Figure 4C) and dot blot assays (Figure 4D), the results showed that mAb 4E88 reacted most strongly with P1, reacted about half as well with P2, P3, and P6, and did not react with P4 or P5, demonstrating that the 4E88 minimal epitope is amino acids 309-317. \n2.5. Cross-Reactivity Analysis \nIFA was performed to investigate whether PDCoV cross-reacts with PEDV and TGEV on the epitope of EP-4E88 (aa 309-KPKQQKKPK-317). The mAb 4E88 was used as primary antibody. Figure 5 shows that no fluorescent signals were observed in TGEV infected ST cells and PEDV-infected Vero cells. This result shows that PDCoV EP-4E88 does not cross-react with TGEV and PEDV.\n2.6. Homology Analysis \nTo explore the level of conservation of the 4E88 epitope, 25 PDCoV strains from GenBank were selected for sequence alignment (Figure 6A), which was done using DNA Star. The results show that the EP-4E88 (aa 309-KPKQQKKPK-317) is highly conserved among the PDCoV strains analyzed, with shared sequence similarity of 100% (Figure 6A). Sixteen strains in the genus Deltacoronavirus were chosen for further sequence alignment. The results revealed that PDCoV, sparrow coronavirus (HKU17) and Asian leopard cat coronavirus (ALCCoV) share 100% sequence similarity in the position of epitope 4E88 (Figure 6B). The newly discovered Quail deltacoronavirus (UAE-HKU30) and Sparrow deltacoronavirus (SpDCoV) share 90% sequence similarity with PDCoV in epitope 4E88, having one amino acid deletion at lys314. The other deltacoronavirus have 30&#8211;60% sequence similarity with PDCoV in epitope 4E88. PDCoV is the only porcine deltacoronavirus; sequence alignment of the 4E88 epitope with other porcine coronaviruses was performed to determine conservation among coronavirus genera. The results showed very low sequence similarity, the highest being 22.2% (Figure 6C). \n2.7. Distructure of EP-4E88\nA 3-dimensional model of the PDCoV N-protein was constructed based on seven templates in the I-TASSER server website(Figure 7). Top five final models predicted by I-TASSER and its C-score. In I-TASSER, C-score has the function of estimating the quality of predicted models. Our models had C-scores in the range of [&#8722;4.26, &#8722;3.58], which is nomal for predicted models. The amino acids of EP-4E88 are located in close proximity to one another and are predicted to be exposed on the surface of the N protein, suggesting that EP-4E88 is highly likely to be a linear epitope. \n3. Discussion\nPDCoV, a novel swine enteropathogen, has spread quickly since first discovered in 2012 in Hong Kong. Its distribution is now nearly worldwide and is causing increasing economic losses to commercial pig industries. Co-infections of PDCoV and other enteric viruses complicates diagnosis and leads to increased morbidity and mortality [12,30]. A discriminate diagnostic tool would be invaluable for efforts in the control of PDCoV and similar viruses. Coronavirus nucleocapsid protein is one of the most abundant viral proteins and is the major antigen recognized by convalescent antisera [31]. Compared with other coronavirus proteins, the N protein is a preferred candidate for serological diagnostics because it is abundant and highly conserved among coronavirus species [26]. Leung et al. found [32] that in patients with SARS, IgG most frequently dominated the antibody response and predominantly targeted the viral nucleocapsid. With respect to PDCoV, the S and N proteins have frequently been the focus for the development of diagnostics [15]. The CoV spike protein however, varies due to mutation to a much greater extent that does the N protein, the result of this variation is a lack of diagnostic accuracy [33].In this study, monoclonal antibodies were produced against purified recombinant N-protein and their epitopes investigated. The five antibodies we tested, all recognized the 122 C-terminal amino acids of the N-protein (aa 221-341), indicating that this section contains the main antigenic epitope of N protein. IEDB (Immune Epitope Database and Analysis Resource) prediction analysis software (http://www.iedb.org/) also proved our point, the results indicated that three epitopes located at the C-terminal of the N-protein sequence, and the largest of 10 predicted epitopes is in amino acids 221-342, this fragment we termed N3. B-cell epitopes are classified as linear or conformational, though it has been reported that 90% of B cell-recognizing epitopes are conformational epitopes, this is due to most of the B-cells epitopes can form conformation in the three dimensional structure, the antigen internalizing process and antigen recognizing ability [34,35,36]. To determine whether our monoclonals recognized conformational epitopes, the N protein was denatured and then tested for reactivity with the mAbs by dot-blot. One mAb (6A5) did not react with the denatured N protein, so we speculated that it recognizes a conformational epitope, the remaining mAbs did react with the denatured N protein so we speculated that these recognize linear epitopes. One mAb (4E88) from our panel was selected for further study of the antigenic epitope of N protein; we chose this antibody because it had the highest reactivity with N-protein in ELISA and dot blot, as well as in PDCoV-infected cells. Moreover, the dominant epitope of the N protein has a greater potential use for early diagnostics than the others.Identification of PDCoV B-cell epitopes is fundamental for the development of epitope-based diagnostic tools, vaccines, and therapeutic antibodies. Approaches for epitope identification are structural and functional. Structural methods include X-ray crystallography, which can precisely locate the epitope position but it is limited to small soluble proteins, the method is also time consuming and costly, together these impede its widespread use [37]. Functional methods are used to detect the binding activity of antibody with antigen fragments, synthetic peptides or recombinant antigens, and have the advantage of being simpler to conduct [37]. Here, the epitope region recognized by 4E88 (308-AKPKQQKKPKK-318 aa) was determined by serially truncating the N3 portion of the N-protein. From there a series of peptides were synthesized and used as antigen in ELISA and dot blot assays, thus we determined that 309-KPKQQKKPK-317 was the minimal epitope.Woo et al. [1] found that avian and mammalian deltacoronaviruses share similar genome characteristics and structure, and that avian coronaviruses are the gene source of gamma- and deltacoronaviruses. In our study alignment analysis revealed that the sequence of epitope 4E88 is highly conserved among PDCoV strains; the sequence similarity was also extremely high with Sparrow coronavirus (HKU17), Asian leopard cat coronavirus (ALCCoV), Quail coronavirus (UAE-HKU30), and Sparrow deltacoronavirus (SpDCoV). PDCoV may come from a host jumps event between mammals and birds [38], also supporting this view is genome analysis showing that PDCoV shares a close relationship with HKU17 (96.8% of sequence identity in N), HKU30 (90.9% of sequence identity in N), and SpDCoV (95.3% of sequence identity in N) [1,39]. The shared sequence at position 309-317aa (epitope 4E88) among PDCoV, HKU17, ALCCoV, HKU30, and SpDCoV may contribute to the understanding of the evolutionary relationship between birds and mammals. Similar phenomenon was also found in other virus. Li et al. [40] found a shared epitope, EXE/DPPFG, among six flaviviruses and verified the cross-reactivity by positive sera detection. Chen et al. [41] found that PDCoV N gene-based PCR cross-reacts with SpDCoV because of their relatively conserved regions. The relationship between PDCoV and other animal-originated deltacoronavirus needs further exploration and whether PDCoV EP-4E88 cross-reacts with HKU17, ALCCoV, HKU30, or SpDCoV should be analyzed using two-way serum cross-reactivity.To date, six porcine coronavirus diseases (PEDV, TGEV, PRCV, SADS-CoV, PHEV, PDCoV) have been reported [42,43]. Because they present with similar clinical symptoms, diagnosis is a challenge, a sensitive and discriminate serologic method is clearly needed. The PDCoV epitope, 4E88 (aa 309-KPKQQKKPK-317), found in this study shares very low sequence similarity (<22.2%) with other swine coronavirus (Figure 6C), such as PEDV (aa 309-NKRETTLQQ-317) and TGEV (aa 309-SRSKSAERS-317), PHEV (aa 309-QKNGQVEND-317), PRCV (aa 309-SRSKSAERS-317), SADS-CoV (aa 309-SQSQDLNA-317) indicating the EP-4E88 is unique to PDCoV and could be used to distinguish PDCoV and from other swine coronaviruses. Cross-reactivity among swine coronaviruses has been reported and is usually associated with the epitope of N protein [24,26,44]. Lin et al. [24] observed that one-way cross-reaction between TGEV Miller hyperimmune pig antisera and various PEDV strains with the TGEV Miller N protein mAb (14G9.3C) was based on an N protein epitope. Xie et al. [26] found two N-terminal epitopes (58-RWRMRRGERIE-68 and 78-LGTGPHAD-85) of the PEDV N protein contribute to the cross-reaction between PEDV and TGEV. Ma et al. [44] found that N protein is responsible for two-way cross-reactivity between PEDV and PDCoV; four regions (47-GYW-49, 67-FYYTGTGPRGNLKY-82, 194-PKG-197, and 329-EWD-332) are highly conserved and likely account for the cross-reaction [44]. To our knowledge, this is the first report to identify the PDCoV N-protein antigenic epitope EP-4E88 (aa 309-NKRETTLQQ-317). It is a linear B-cell epitope and highly conserved among PDCoV strains and other porcine coronaviruses, EP-4E88 also shares high sequence identity with four non-swine deltacoronavirus. Our findings provide valuable insight into the evolutionary and serological relationship among PDCoV and non-swine deltacoronavirus, and for the development of PDCoV epitope-associated diagnostics and vaccine design.\n4. Materials and Methods\n\n4.1. Ethics Statement\nAll animal experiments were approved by the Institutional Animal Care and Use Committee of Sichuan Agricultural University (IACUC#RW2016-090, approval date: 8 September 2016), and were performed in strict accordance with the Care and Use of Laboratory Animals guidelines and regulations of the Ministry of Science and Technology of the People&#8217;s Republic of China. \n4.2. Virus and Cells\nST cells (ATCC CRL-1746), Sp2/0-Ag14 (ATCC CRL-1581) cells and Vero cells (ATCC CCL-81) were maintained at 37 &#176;C in a humidified 5% CO2 atmosphere in Dulbecco&#8217;s modified Eagle medium (DMEM; Gibco, Carlsbad, CA, USA) supplemented with 10% heat-inactivated fetal bovine serum (PAN-Biotech, Aigenbach, Germany) and 1% antibiotic-antimycotic (Solarbio, Beijing, China). The PDCoV strain CHN-SC2015 (GenBank accession No.MK355396), PEDV-CV777 strain (GenBank accession No. AF353511.1), TGEV-H strain was preserved by the Laboratory of Research Center of Swine Disease in Sichuan Agricultural University. For PDCoV propagation, ST cells at 90% confluence were washed three times with DMEM supplemented with 5 &#956;g/mL trypsin-EDTA (maintenance medium) and then inoculated with 1 mL of PDCoV. The virus was removed and 6 mL of maintenance medium was added. When CPE was observed, generally at 2 days post-infection, the cells were harvested then stored at &#8722;80 &#176;C until further use. The titers of CHN-SC2015 obtained were up to 106.64 TCID50/mL. For PEDV propagation, Vero cells at 90% confluence were washed three times with DMEM supplemented with 10 &#956;g/mL trypsin-EDTA (maintenance medium) and then inoculated with 1 mL of PEDV. When CPE was observed, generally at 3 days post-infection, the cells were harvested then stored at &#8722;80 &#176;C until further use. For TGEV propagation, ST cells at 90% confluence were washed three times with DMEM supplemented with 5 &#956;g/mL trypsin-EDTA (maintenance medium) and then inoculated with 1 mL of TGEV. When CPE was observed, generally at 2 days post-infection, the cells were harvested then stored at &#8722;80 &#176;C until further use. \n4.3. Construction of Full-Length and Truncated Recombinant N-Protein\nPDCoV CHN-SC2015 genomic RNA was extracted using TRIzol Reagent (Sangon Biotech, Shanghai, China) according to the manufacturer&#8217;s instructions. The N gene was amplified by RT-PCR with primers (Table 1) designed with Primer 5.0. The N gene amplicon (1029 bp) was inserted into the pET-28a (+) expression vector (Novagen, Madison, WI, USA) between the EcoR I and Xho I sites. The integrity of the resulting construct (pET28-N) was verified by restriction enzyme digestion, PCR, and DNA sequencing. pET28-N was transformed into Transetta (DE3) Escherichia coli. Protein expression was induced with 0.8 mM isopropyl-&#946;-galactopyranoside (IPTG) for 3 h at 30 &#176;C. Bacteria were collected by centrifugation and lysed by ultrasonication, then centrifuged again; the recombinant N protein (rNP) was purified from the supernatant post sonication by Ni-NTA His-Bind Resin (Bio-Rad, Hercules, CA, USA) according to the instructions. The concentration of the purified rNP was analyzed by SDS-PAGE. Using the same methodology, a series of truncated N-proteins were produced: His-N1, His-N2, His-N3, His-N3-1, His-N3-2, His-N3-3, His-N3-3-1, His-N3-3-2, His-N3-3-3, His-N3-3-4 (Figure 3A).\n4.4. Western Blotting\nWestern blotting was used to test the reactivity of hyperimmune pig anti-serum against the recombinant N protein. Purified N protein and pET-28a (+) empty vector were subjected to SDS-PAGE then transferred to a PVDF membrane. The membrane was blocked with 5% skim milk in PBST (PBS/0.05% Tween-20) for 1.5 h then incubated with anti-PDCoV pig hyperimmune serum (1:200) overnight at 4 &#176;C overnight. The membrane was washed three times with PBST then incubated with HRP-goat anti-pig IgG (1:5000) for 1 h at 37 &#176;C. The membrane was washed four times with PBST and the proteins were visualized using enhanced chemiluminescence reagents (ECL; Bio-Rad, Hercules, CA, USA).\n4.5. Production of Anti-rNP mAbs\nSix-week-old female BALB/c mice purchased from Chengdu Dossy Experimental Animal Co, Ltd., were inoculated via subcutaneous injection with purified rN protein (100 &#956;g/mouse) mixed with an equal volume of Montanide Gel 01 PR adjuvant (Montanide, SEPPIC, Puteaux, France). Mice were boosted twice, at 2-week intervals, with the same immunogen and adjuvant. Pre-immune serum samples were taken from all mice and tested for reactivity against purified rNP by indirect ELISA. Two weeks after the final boost, mice were sacrificed and spleens removed. Spleen cells were fused with Sp2/0 Ag14 cells, and hybridomas were selected in HAT and HT medium. Culture supernatants from individual hybridoma clones were screened for reactivity with rNP by indirect ELISA. Reactive hybridomas were subcloning three times by limiting dilution, stable hybridomas were injected into the abdominal cavity of sensitized mice to produce ascites. Antibodies were purified twice, first using the octylic acid ammonium sulfate method (CA-AS), then by HPLC through a diethylaminoethyl column. The identification of isotype in prepared mAbs by mouse monoclonal antibody isotyping ELISA kit (Proteintech Group, Inc., Wuhan, China).\n4.6. Denatured Protein \nTo identify mAbs that recognize conformational or linear epitopes, the N3 proteins were denatured as previously described [34]. Briefly, the purified rN proteins were mixed with 6&#215; protein-loading buffer with DTT (TransGen Biotech, Beijing, China) then heated for 8 min at 95 &#176;C; this procedure fully denatures secondary structure. The denatured proteins were tested with the mAbs using a dot-blot assay. \n4.7. IFA\nST cells were grown in 12-well plates until 80% confluent. Half the wells were infected with PDCoV (MOI = 0.1) and half were mock infected. After 36 h post-infection, cells were then washed twice with PBS, fixed with 4% formaldehyde in PBS for 30 min, then permeabilized with 0.5% Triton-X-100 for 30 min at room temperature. Cells were washed again and blocked with 2% BSA in PBS for 1.5 h then incubated with each of the five anti-N mAb for 1 h in PBS with 1% BSA. Cells were washed three times and incubated with FITC labeled goat anti-rabbit IgG (1:500 in PBST). Nuclei were stained with DAPI (Solarbio, Beijing, China). The Image-Pro Plus 6.0 was used to analyze number of immunfluoresnce from six different stained images. The FITC-positive cells of mAb 6A5 was set to one-fold.\n4.8. ELISA \nIndirect ELISA was performed as described in Guo et al. 2006 [45]. Briefly, the wells of a 96-well ELISA plate were coated with 1 &#956;g/well of protein overnight at 4 &#176;C. The wells were rinsed, then blocked with 5% non-fat dry milk in TBST for 1 h at room temperature. Wells were rinsed three times with TBST then incubated with mAb (1:500) for 1 h at 37 &#176;C. Wells were rinsed again and incubated with HRP-conjugated goat anti-mouse IgG, (1:5000) for 1 h at 37 &#176;C. After final rinsing wells were incubated with tetramethylbenzidine (TMB) for 15 min, color development was stopped with 3M H2SO4. The OD450 was read by ELISA plate reader (Bio-Rad, Hercules, CA, USA).\n4.9. Dot-Blot Analysis\nDot-blot hybridization for identification of linear epitopes was based on the method described by Chen et al. 2017 [46]. Briefly, PVDF membranes were soaked in dimethyl sulfoxide then in methanol. Total of 1 &#956;L (0.5 &#956;g) of protein was spotted onto the treated membrane, then air-dried for 10 min. The membrane was blocked with 2% BSA in TBST for 30 min at room temperature, then incubated with mAb (1:200) in 2% BSA/TBST for 1 h at 37 &#176;C. The membrane was rinsed then incubated with goat anti-mouse IgG (1:5000) for 1 h at 37 &#176;C. The signal was developed using enhanced chemiluminescence reagents (ECL; Bio-Rad, Hercules, CA, USA). \n4.10. Identification of the Minimal 4E88 Epitope\nTo determine the minimal B-cell epitope recognized by mAb 4E88, five peptides spanning various lengths of aa308-318 were commercially synthesized (Sangon Biotech, Shanghai, China). The amino acid sequence of these peptides (P1&#8211;P6) are shown in Figure 3A,B. The reactivity of 4E88 with the peptides was tested by dot-blotting and ELISA as previous described [47]. Specially, peptides were dissolved in DMSO (Solarbio) to a concentration of 10 mg/mL, for dot-blotting 1 &#956;L (10 &#956;g) was spotted onto PVDF membrane, and for ELISA, 100 &#956;L (1 mg) were aliquoted into wells and incubated at 4 &#176;C for 15 h. \n4.11. Sequence Homology\nTo determine whether the epitope recognized by mAb 4E88 was conserved among PDCoV strains, the amino acid sequences of 25 PDCoV strains in GenBank (Table 2) were aligned with the sequence in EP-4E88 using MAFFT v7.037 and analyzed by DNASTAR. The EP-4E88 sequence was also aligned with other deltacoronavirus and porcine coronavirus (Table 3 and Table 4) as described. The results were rendered in figure form with ESPript 3.\n4.12. Three-Dimensional Structure Prediction\nI-TASSER (https://zhanglab.ccmb.med.umich.edu/I-TASSER/) was used to predict the three-dimensional structure of the full-length PDCoV N-protein as described [48,49,50]. Seven threading templates in Protein Data Bank (PDB) were selected for construction by this program (2gecB, 5gaoE, 1sskA, 4n16A, 4j3kA, 1ssk, 2gecA). Figures were generated using the PyMOL molecular visualization system.\n4.13. Statistical Analysis\nThe experiments were repeated three times. All statistical data were analyzed by GraphPad Prism version 7.0 and expressed as mean &#177; SD. The differences among the five monoclonals were analyzed using the one-way ANOVA. Statistical changes marked by * p value < 0.05, ** p value < 0.01, *** p value < 0.001, **** p value < 0.0001.\n5. Conclusions\nIn this study, five monoclonals of PDCoV N protein were produced. Of the five monoclonals, mAB 4E88 had highest binding affinity with the N protein and was chosen for identifying epitope. EP-4E88 (aa309-KPKQQKKPK-317) was the minimal epitope which recognized by mAb 4E88. Homology analysis showed that the EP-4E88 sequence is highly conserved among PDCoV strains and several deltacoronavirus but shared very low similarity with other porcine coronaviruses. Therefore, EP-4E88 lays a good foundation for applied research associated with PDCoV diagnosis.",
        "pmid": "31963776",
        "title": "Identification of a Novel Linear B-Cell Epitope on the Nucleocapsid Protein of Porcine Deltacoronavirus",
        "journal_title": "International Journal of Molecular Sciences",
        "authors": "Jiayu Fu;Rui Chen;Jingfei Hu;Huan Qu;Yujia Zhao;Sanjie Cao;Xintian Wen;Yiping Wen;Rui Wu;Qin Zhao;Xiaoping Ma;Xiaobo Huang",
        "doi": "10.3390/ijms21020648"
    },
    {
        "keywords": "respiratory viruses;human coronaviruses;hMPV;bocavirus;SARS-CoV;MERS.",
        "abstract": "The rapid advancement of molecular tools in the past 15 years has allowed for the retrospective discovery of several new respiratory viruses as well as the characterization of novel emergent strains. The inability to characterize the etiological origins of respiratory conditions, particularly in children, led several researchers to pursue the discovery of the underlying etiology of disease. In 2001, this led to the discovery of human metapneumovirus (hMPV) and soon following that the outbreak of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) promoted an increased interest in coronavirology and the latter discovery of human coronavirus (HCoV) NL63 and HCoV-HKU1. Human bocavirus, with its four separate lineages, discovered in 2005, has been linked to acute respiratory tract infections and gastrointestinal complications. Middle East Respiratory Syndrome coronavirus (MERS-CoV) represents the most recent outbreak of a completely novel respiratory virus, which occurred in Saudi Arabia in 2012 and presents a significant threat to human health. This review will detail the most current clinical and epidemiological findings to all respiratory viruses discovered since 2001.",
        "body": "1. Introduction\nViral infections of the upper and lower respiratory tract are among the most common illness in humans. Children and infants bear the major burden of infection, typically presenting with 5 to 6 episodes annually [1]. These infections are often associated with significant patient morbidity and related mortality. For this reason, URTIs and LRTIs represents the leading cause of death in children younger than five years of age worldwide [2,3]; this accounts for approximately 4 million deaths annually [4]. Acute respiratory tract disease is the leading cause of hospitalization in children and febrile episodes in infants younger than three months of age [5,6].Bacteria only represent approximately 10% of all upper respiratory tract infections with the subsequent 90% of infections caused by respiratory viruses [7]. Despite the viral aetiological origin of most respiratory infections, antibiotics are often prescribed in the treatment of such diseases [8], exacerbating antibiotic abuse. The morbidity and fiscal implications associated with respiratory infections are significant, with approximately 500 million cases reported in the United States alone each year with subsequent direct and indirect costs to the US economy estimated at $40 billion annually [2]. The burden of respiratory tract infections is increased in patients with chronic comorbidities or clinical risk factors including asthma [9], chronic obstructive pulmonary disease (COPD) [10], young, elderly [11] and immunocompromised [12,13].The viruses primarily associated with upper respiratory tract infections commonly include rhinoviruses, enteroviruses, adenoviruses, parainfluenza viruses (PIV), influenza viruses, respiratory syncytial viruses (RSV) and coronaviruses [3,8,14,15]. In recent years six new human respiratory viruses have been reported including human metapneumovirus (hMPV) [16], bocavirus and four new human coronaviruses including Severe Acute Respiratory Syndrome coronavirus (SARS-CoV), human coronavirus NL63 (HCoV-NL63), HCoV-HKU1 and Middle East Respiratory Syndrome coronavirus (MERS-CoV). This review will detail these newly discovered and emerging respiratory viruses.\n2. Human Coronaviruses\nCoronaviruses affect a diverse group of animal hosts, and cause a plethora of diseases in animals including progressive peritonitis, acute and chronic hepatitis, gastroenteritis, nephritis, and encephalitis [17]. In humans coronavirus infection results in respiratory tract complications with varying degree of severity and have been associated with gastroenteritis. Four human coronaviruses (HCoV-229E, HCoV-OC43, HCoV-NL63 and HCoV-HKU1) are endemic in the human population and are mainly associated with mild, self-limiting respiratory illnesses. Another two human coronaviruses, namely SARS-CoV and MERS-CoV cause severe respiratory syndromes and present a significant threat with their high fatality rates.The first human coronaviruses were identified in the 1960s by Tyrell and Bynoe who passaged a virus, named B814, in human embryonic tracheal organ cultures. When this virus was inoculated intranasally into human volunteers, common cold-like symptoms were produced [18,19]. Hamre and Procknow (1966) later isolated a virus, which they were able to grow in tissue culture, from subjects presenting with symptoms of the common cold. The virus was later named human coronavirus 229E (HCoV-229E) [20].All of these viruses, including B814 and HCoV-229E, were ether-sensitive, indicating the presence of a lipid covering. McIntosh et al. (1967) later recovered several ether-sensitive viruses from human respiratory tracts, which were grown in organ culture. These viruses were termed &#8220;OC&#8221; viruses to indicate passage in organ culture. Under electron microscopy all of these viruses had morphologies similar to several other animal viruses previously identified, namely IBV, MHV and TGEV [21]. These viruses were later classified as coronaviruses, which was accepted as a new genus in 1975. Unfortunately, many of the clinical samples collected and stored in the 1960s, which were positive for coronavirus-like particles, were subsequently lost. Therefore, until the identification of SARS-CoV in 2003, the study of human coronaviruses was restricted to research of HCoV-229E and HCoV-OC43 [17] despite the clear evidence of further coronavirus strains [22,23].\n2.1. Severe Acute Respiratory Syndrome Coronavirus: Emergence of a Novel Coronavirus\nAn outbreak of atypical pneumonia was first reported to the WHO in February 2003. By the 15th of March the disease was named Severe Acute Respiratory Syndrome (SARS) [24,25,26] and by the 27th of March the causative agent was identified as a completely novel coronavirus, termed SARS-CoV [27,28,29]. From November 2002 to July 2003 a total of 8098 patients, in 25 countries, were affected by the atypical pneumonia which resulted in 774 deaths globally. The mortality rates were drastically increased in certain population [30] and age [31] groups to as high as 40%&#8211;55%. Further local outbreaks were reported in Singapore, Taiwan and Beijing from accidental laboratory exposure and animal to human transmission in Guangzhou in late 2003 and early 2004 [32]. This was directly related to the lifting of a ban on serving palm civets in wet markets and restaurants, which was implemented during the SARS outbreak [33].No antiviral agents were available during the outbreak to combat infection with SARS-CoV [34] and supportive care and administration of antibiotics to combat secondary bacterial infection was the primary treatment regimen [35]. With this it is clear that the control of the SARS-CoV outbreak is thanks, almost entirely, to a highly effective global public health response [36], where intense contact tracing and quarantine of suspected and infected patients were key in the control of the SARS outbreak [35]. SARS-CoV is no longer in human circulation but the discovery of SARS-like-CoV in bats and other mammals does present a potential threat of re-emergence [37].The outbreak of SARS-CoV was most-likely linked to a zoonotic event in the live animal markets of China. Himalayan palm civets and raccoon dogs were the first animals identified to carry a SARS-like-CoV with 99.8% nucleotide homology to human SARS-CoV [38]. Further studies of wild animals in the areas around Hong Kong identified a further SARS-like-CoV in Chinese horseshoe bats (genus Rhinolophus), which shared a sequence identity of 87%&#8211;92% with human SARS-CoV [39,40]. Horseshoe bats therefore appear to be the natural reservoir of the ancestral SARS-CoV, with civets providing the intermediate amplification host with subsequent spread to animal handlers in the wet markets of Guangzhou. Under positive selection pressures within the human host SARS-CoV became readily transmissible and allowed for fast and effective spread from human-to-human with subsequent international dissemination by the index case in Hong Kong [41]. Analysis of genomic sequences from viral isolates obtained in the early, mid and late stages of the epidemic clearly indicate positive and purifying selective pressures followed by stabilization [33]. Seroepidemiological studies have shown that 40% of wild animal traders and 20% of people responsible for the slaughtering of animals in the region where human SARS was thought to originate, were seropositive for SARS-CoV, although all cases were asymptomatic. This indicates that these people were previously exposed to a SARS-like-CoV, which resulted in asymptomatic infection [38].SARS presents as an atypical pneumonia [42,43], with pneumocytes being the primary target of infection. Infection results in haemorrhagic inflammation in most pulmonary alveoli with alveolar thickening, diffuse alveolar damage, desquamation of pneumocytes, formation of hyaline membranes and multinucleated pneumocytes with capillary engorgement and microthrombosis [44]. Approximately 60% of patients deteriorated in the second week of infection, presenting with persistent fever, dyspnoea and oxygen desaturation [29]. Approximately 20%&#8211;30% of patients were subsequently admitted to intensive care, where mechanical ventilation was necessary [45]. A surprising finding with the SARS outbreak was that it was not as great a threat to infants and children [33,46]. Clinical presentation was less severe in infants and no children aged between 1 and 12 required intensive care or mechanical ventilation [47,48]. This is in sharp contrast to the age related burden of other respiratory infections and the underlying biological mechanism remains unclear [49].A family of viruses that were previously understood to cause mild, self-limiting upper respiratory tract infections was showcased by the SARS-CoV outbreak to be a significant threat to global public health. The economic losses brought on by the SARS pandemic was estimated to be in the region 40 billion dollars [50] with Hong Kong bearing a significant proportion of the losses. Tourism, entertainment and restaurant industries in the area recorded up to 80% loss in business. The pressure on healthcare facilities in affected areas was substantially exacerbated by the spread to healthcare workers. Several hospitals were forced to close to new admissions as large numbers of staff became infected with SARS-CoV. To view this as an isolated incidence would be na&#239;ve and the potential for the emergence and re-emergence of new and existing infectious agents poses a probable risk. Understanding the SARS-CoV outbreak has provided immense knowledge and an excellent model to replicate in the event of further outbreaks of communicable diseases.\n2.2. Human Coronavirus NL63 and HKU1: Discovery of Existing Coronaviruses\n\n2.2.1. Human Coronavirus NL63\nIn 2003 a 7 month old child presenting with bronchiolitis and conjunctivitis was screened for several respiratory viruses to identify the causative agent, with all diagnostics yielding negative results. The group led by Lia van der Hoek then used a modified cDNA amplified restriction fragment-length polymorphism (cDNA-AFLP) technique (Virus-Discovery-cDNA-AFLP or VIDISCA), to identify the causative agent. Briefly, the technique utilizes reverse transcription-PCR of viral RNA with subsequent restriction digest of the cDNA using frequently cutting restriction enzymes. Since the restriction sites are selected and therefore known, the resultant &#8220;sticky ends&#8221; can be ligated into anchors for amplification and sequencing with specific primers. The results showed highest sequence similarities with known coronaviruses, but with significant sequence divergence indicating the discovery of a new coronavirus species, later named human coronavirus NL63 [51].At about the same time, two other independent groups identified essentially the same virus [52,53]. Shortly after the van der Hoek paper [51], a novel coronavirus that replicated efficiently in tertiary monkey kidney and Vero cells, was retrospectively isolated from a nose swab sample collected in 1988 from an 8 month-old boy presenting with pneumonia. This virus was reported to be similar to HCoV-NL63 and named HCoV-NL [52]. In 2005, Esper et al. (2005) also reported the identification of a novel coronavirus isolated in New Haven, Connecticut, which was named HCoV-NH. This novel virus was identified by PCR which was adapted to amplify a conserved region within the replicase 1a or pol gene [53]. Subsequent sequence analysis showed that these three viruses were essentially the same virus or variants thereof [53,54].It is obvious that HCoV-NL63 has been circulating in the human population since before 1988 [52]. In fact, molecular clock analyses have shown that HCoV-NL63 and HCoV-229E diverged from a most recent common ancestor, in a zoonotic event, approximately 563 to 822 years ago [55]. HCoV-NL63 later diverged into two lineages with subsequent recombination of the two lineages during co-infection. This frequent recombination has given HCoV-NL63 a mosaic structured genome with multiple recombination sites [56].\n2.2.2. Human Coronavirus HKU1\nA 71 year old male patient from China was admitted to hospital with pneumonia in January 2004. Viral cultures, RT-PCR and direct antigen detection from nasopharyngeal aspirate were all negative for respiratory viruses. A pan-coronavirus RT-PCR targeting a conserved region of the pol gene confirmed the presence of a coronavirus however attempts to culture the causative agent were all unsuccessful. Sequencing the gene segment amplified by the pan-coronavirus assay indicated a high homology to other viruses of the &#946;CoV genus including HCoV-OC43 but of novel origin. The human coronavirus, termed HCoV-HKU1, was later isolated from another female patient. Efforts to culture the virus had posed complicated and the complete genome was isolated, amplified and sequenced directly from RNA extracted from a nasopharyngeal aspirate [57]. Successful propagation of the virus was achieved recently in human ciliated airway epithelial cells [58] but culturing of HCoV-HKU1 still remains a daunting task. The presence of the HE gene further characterizes HCoV-HKU1 as belonging to the &#946;CoV genus.Since the discovery of HCoV-HKU1, its prevalence has shown a global distribution and retrospective analysis on stored nasopharyngeal aspirates have confirmed its existence since 1995 [59]; however, phylogenetic analysis suggests a much earlier divergence.\n2.3. MERS-CoV: 10 Years on from SARS\nJune 2012 saw the most recent emergence of a completely novel strain of human coronavirus. A sputum sample was collected from a 60 year old male patient presenting with severe respiratory disease in the Dr. Soliman Fakeeh Hospital in Jeddah, Saudi Arabia. Viral assays frequently used could not identify any aetiological agent responsible for the disease. The sputum sample was sent to Dr. Ron Fouchier at the Erasmus Medical College in Rotterdam, Netherlands, where the virus was identified as a novel coronavirus, provisionally termed HCoV-EMC (human coronavirus Erasmus Medical College). The patient later succumbed to the disease with acute pneumonia and subsequent renal failure [60]. A retrospective study further traced the virus back to April 2012 where an outbreak of pneumonia, resulting in two fatalities, occurred in health care workers in an intensive care unit in Zarqa, Jordan [61].Since the initial discovery, several new isolates were identified and described in scientific literature, databases and media under various names. This provoked the convening of the Coronavirus Study Group to introduce a naming convention and avoid confusion within the research field, health care authorities, governments and general public. The name Middle East Respiratory Syndrome coronavirus (MERS-CoV) was coined and has been widely accepted by the discoverers of the virus and pioneers of the field, the WHO and Saudi Ministry of Health [61].Primary infections, to which all cases were linked directly or indirectly, occurred in Middle Eastern countries including Saudi Arabia, Qatar, Jordan, Oman and United Arab Emirates and subsequently spread to United Kingdom, Tunisia, France, Italy and Germany with Egypt and the United States recently reporting their first laboratory confirmed cases [62,63]. The potential of widespread pandemic outbreak is thought to be low as human-to-human transmission is largely inefficient [64] and only reported in close, sustained contact [65] such as within families [66], healthcare workers [67] and as nosocomial transmission [68], especially if the patient represents with comorbidities. There has been no evidence to suggest sustained community transmission [62]. The ever increasing case numbers and countries affected by the disease however suggests MERS-CoV does represent potential for widespread, global outbreak [69].From the first reported case in June 2012, until 7 February 2014, a total of 182 cases had been reported with 79 fatalities. According to the latest figures available from the WHO, as of 11 June 2014, the virus had resulted in a total of 699 laboratory confirmed cases with a total of 209 fatalities [70]. These statistics indicate that within a 4 month period case numbers had more than trebled suggesting the virus is far from controlled. The fatality rate of 30% is also substantially increased in patients with comorbidities, with a large number of reported cases being identified in immunocompromised patients or those with underlying disease [68,71] and a severe threat of nosocomial transmission has been demonstrated [72]. Clinical presentation is similar to SARS and includes a spectrum of respiratory diseases with the most common symptoms including cough, fever and gastrointestinal symptoms [69] before progressing to pneumonia [68]. Acute respiratory distress syndrome (ARDS), renal failure, pericarditis and disseminated intravascular coagulation have also been reported [69]. The potential of widespread pandemic outbreak is thought to be low as human-to-human transmission is largely inefficient [64] and only reported in close, sustained contact [65] such as within families [66], healthcare workers [67] and as nosocomial transmission [68], especially if the patient presents with comorbidities. There has been no evidence to suggest sustained community transmission [62]. The ever increasing case numbers and countries affected by the disease however suggests that MERS-CoV does represent potential for widespread, global outbreak [69].The inefficient spread between infected patients strongly suggests zoonotic transmission, however over two years on from the initial discovery of MERS-CoV it is still unclear where it originated from and how it infects humans. During the outbreak of SARS-CoV the identification of the wet markets of China as the probable source greatly contributed to the control of the disease [38]. This highlights the importance in identifying the origins of disease outbreak. Phylogenetic analysis places it in the same genus as SARS-CoV but within a new lineage, 2c. The closest detectable relatives suggests origins from bat coronavirus species with relative high sequence homology between Bat-CoV HKU4 and HKU5 in China [73], VM314 in the Netherlands [74] and a recently discovered isolate from South Africa [75]. Assuming that bats also form the immediate source is improbable and as with SARS-CoV an intermediate animal host species for transmission to humans is of greater likelihood [61]. High levels of neutralizing antibodies, viral RNA and infectious viruses have been discovered in dromedary camels suggesting a potential for camels to form the source for human transmission [76,77,78,79]. A recent study identified identical single nucleotide polymorphisms in a sequence of MERS-CoV isolated from an infected patient and camel, which was cared for by the patient. Phylogenetic analysis of these two sequences supported the conclusion that transmission occurred between patient and camel, however the direction of transmission could not be confirmed [80]. It has since been proven that a patient who succumbed to MERS-CoV infection obtained the virus directly from his camel herd. The direction of transmission was confirmed as camel-to-human by serological analysis [81]. These facts strongly contribute to an increasingly popular hypothesis that MERS-CoV infections in humans are directly acquired from camels, however very few patients have had reported contact with camels. Many residents of the Arabian Peninsula frequently consume unpasteurized camel milk and this has been suggested as a potential source as the virus has been detected and can remain viable for prolonged periods in milk [82,83]. The WHO, Saudi Arabia and Qatar have recently issued warnings and recommended against consuming unpasteurized camel milk and to be cautious when interacting with dromedary camels [84].The outbreak of SARS-CoV in 2003 and MERS-CoV less than 10 years later highlights the significant threat of coronaviruses to humans and confirms that the SARS outbreak was not an isolated incident. With the ever increasing diversity of animal coronavirus species, especially within bats, the likelihood of recombination leading to future outbreaks is high and the threat of potential pandemics is real as highly pathogenic coronaviruses continue to spill over from zoonotic sources into the human population. Misdiagnosis of these outbreaks pose a further substantial threat to healthcare workers with nosocomial spread to other patients putting further pressure on an already strained healthcare system. Understanding the dynamics and molecular characteristics of human coronaviruses currently in circulation and how they emerge, infect and cause disease, we can be better prepared for future pandemics allowing for improved response, management and treatment of related conditions. Table 1 summarizes the human coronaviruses discovered since 2003.\n2.4. Clinical Importance of Non-Severe Human Coronaviruses\nThe clinical presentation of human coronaviruses clearly follows two distinct lines of progression. SARS-CoV and MERS-CoV present with severe respiratory complications and often multisystem involvement with renal failure and enteric symptoms. The high fatality rates associated with these infections is not reflected in the remaining four human coronaviruses, HCoV-229E, HCoV-OC43, HCoV-NL63 and HCoV-HKU1. The clinical importance of SARS-CoV and MERS-CoV are evident and discussed in detail in previous sections allowing for the presentation of the involvement of the remaining non-severe human coronaviruses and there implications in human health.\n2.4.1. Clinical Presentation of Human Coronavirus Infection\nThe clinical presentation of these four non-severe human coronaviruses is largely identical and indistinguishable symptomatically, commonly presenting with rhinorrhoea, sore throat, cough and fever [85,86]. Majority of infections are associated with self-limiting upper respiratory tract disease or &#8220;the common cold&#8221; but can also present with high morbidity outcomes of the lower respiratory tract including bronchiolitis, pneumonia, [87,88,89], asthmatic exacerbations [9] acute exacerbations of chronic obstructive pulmonary disease (COPD) [10] and croup in HCoV-NL63 infected patients [90]. A report by Esper et al. found a correlation between HCoV-NL63 infections and Kawasaki disease [91], although other studies could not replicate the association [92,93,94]. Febrile seizures have been reported for most human coronaviruses but the significance of HCoV-HKU1 is alarming with one study indicating that 50% of patients infected with HCoV-HKU1 experience febrile seizures [9].\n2.4.2. Prevalence and Distribution of Human Coronaviruses\nHuman coronaviruses affect all age groups [85,86] but elicit more serious disease in young, elderly and immunecompromized [87,95,96], frequently resulting in hospitalization. Reports on the prevalence of human coronaviruses and their association with upper and lower respiratory tract disease vary but range between 3.3% and 16% [85,86,95,97]. Over 70% of the general public has seroconverted towards all four non-severe human coronaviruses with primary infection shown to occur in childhood [98] and reinfection occurring throughout life [95]. Given the high prevalence of respiratory infections, human coronaviruses represent a substantial disease burden which is exacerbated by the high implications of healthcare workers in coronavirus outbreaks [97].\n2.4.3. Human Coronaviruses Frequently Present with Co-Infection\nHigh rates of co-infection with other respiratory viruses are commonly reported [85,86,95,99]. Viruses frequently associated with co-infection include enterovirus, rhinovirus and PIV [100] however reports of co-infection with two human coronaviruses are limited. Dijkman et al. recently demonstrated that HCoV-OC43 and HCoV-NL63 may elicit immunity that protects against HCoV-HKU1 and HCoV-229E, respectively [101]. Clinical progression and outcomes of disease in patients presenting with co-infection are however similar to patients presenting with mono-infection [85,95]. There is also no substantial difference in coronaviral load between co-infected and mono-infected patients. No substantial difference in disease progression in co-infected versus mono-infected patients has been reported and therefore understood to have little impact; however, the role in facilitation of infection of one respiratory virus by another is still speculative [95].\n2.4.4. Seasonal Distribution\nAll four human coronaviruses are endemic worldwide but the prevalence, regional distribution and pathogenicity of individual human coronaviruses is unclear and highly subjective on study conducted with parameters including population sampled, respiratory sample collected, sensitivity of diagnostic assay used, region where study was conducted and duration of the study playing a substantial role in epidemiological findings. A common finding in majority of studies conducted is the increased prevalence of human coronaviruses during winter months in temperate climates [95,97]. However even this has shown to be geographically dependent with a spring/summer predominance in subtropical and tropical climates [9,102]. Biennial outbreaks are frequently reported [85,95,99,100] for all strains of non-severe human coronaviruses.\n3. Human Metapneumovirus (hMPV)\nIn 2001 a previously undiscovered virus was identified in 28 epidemiologically distinct patients in the Netherlands. Patient symptoms were similar to those infected with RSV and, several patients required hospitalization and mechanical ventilation. Viral isolates were cultured in tertiary monkey kidney (tMK) cells and cytopathic effects caused by the virus were largely identical to those caused by RSV. Electron microscopy of infected cell supernatants revealed paramyxovirus-like particles; however, RT-PCR assays to detect known paramyxoviruses were all negative. The low stringency of the assays used indicated a currently unknown, genetically distinct virus. A RAP-PCR assay was then utilized to obtain sequence information of the unknown virus and fragments amplified by the RAP-PCR allowed for further sequencing of the 3&#8217;-end of the genome. Based on the sequence homology and gene organization, the unidentified virus displayed closest homology with avian pneumovirus, but to be a tentative new member of the Metapneumovirus genus and the first virus in the genus to infect humans, provisionally termed human metapneumovirus (hMPV) [16].\n3.1. Clinical Presentation of hMPV Infection\nSymptomatic differentiation between hMPV and other respiratory viruses cannot be made as there is a significant overlap in clinical presentation [103,104]. The most common presentation of hMPV in children includes complications of the upper respiratory tract with rhinorrhoea, cough and fever [105]. Acute otitis media is also frequently reported [106,107] and conjunctivitis, rash, diarrhea and vomiting are reported but infrequently [103]. Bronchiolitis, pneumonia, croup and asthmatic exacerbations are the most frequently associated lower respiratory tract complications [108] and viral load is directly associated with disease severity [109]. hMPV infection in the young and elderly frequently requires hospitalization and fatalities have been reported in the elderly [110,111]. An increased morbidity in elderly patients with a delayed clearance of symptoms has been reported and is likely related to the age related impairment of the innate and adaptive immunity [103] or an over stimulated immune response leading to inflammation [112]. Elderly patients requiring hospitalization most frequently present with acute bronchitis, COPD exacerbations, pneumonia and congestive heart failure [113]. In healthy adults asymptomatic infections or cold- and flu-like symptoms are the most prevalent presentation [114].\n3.2. Coinfections with hMPV\nThe pathogenesis of hMPV infection is strongly affected by bacterial coinfections with pneumococcus. One study has shown that administration of a conjugate pneumococcal vaccine is sufficient to reduce the incidence of hMPV infection of the lower respiratory tract and the incidence of clinical pneumonia in both HIV positive and negative patients [115]. These finding suggest that the incidence of hospitalizations in hMPV infections may be decreased by vaccination with a conjugate pneumococcal vaccine. Another case report of severe respiratory failure was found to be caused by coinfection with hMPV and Streptococcus pneumonia in a 64 year old patient [116]. Both in vitro and in vivo studies have shown that infection with hMPV facilitates adhesion of pneumococcal bacteria, which may provide an explanation for the coinfection with pneumococcal strains and hMPV [117].Viral coinfections between hMPV and RSV have been reported, but remain a contentious issue. The typical seasonal overlap of the two viruses has been suggested to promote viral coinfection. One study reported a 10-fold increase in risk of admission to an intensive care unit in pediatric patients coinfected with RSV and hMPV and associated the dual infection as capable of augmenting severe bronchiolitis [118]. Other studies do not support this finding and further report a decreased correlation between hMPV-RSV coinfections and hospitalization and additionally lists dual infection, along with breastfeeding, as having protective effects [119].\n3.3. Epidemiology of hMPV\nAlthough hMPV was only discovered in 2001, it has been shown by phylogenetic analysis to have been in existence for approximately 50 years [120,121]. Soon after the discovery of hMPV, it was evident that two lineages, A and B, existed. These two lineages were further subdivided into two sublineages per lineage, A1-A2 and B1-B2 [16]. A recent report analyzing sequence divergence of the attachment and fusion surface glycoproteins indicates the presence of five sublineages, namely A1, A2a, A2b, B1 and B2 [122]. From long term retrospective studies it was evident that these lineages are not restricted to certain locations or times and that multiple lineages can exist in the same location and period [108,123]. It has also become evident that old sublineages may be replaced by new variants [103]. Disease progression or varying clinical outcomes related to different lineages of hMPV has become a contentious issue. Several studies have reported that lineage A presents with more severe clinical outcomes [124,125,126] where the same is reported for lineage B by other groups [127,128]. It has been further reported that there is no difference in disease outcomes related to the two lineages [108,129,130].Between 7% and 19% of all cases of respiratory infections in children are caused by hMPV, in both hospitalized and outpatients [108,131,132] and has been reported to be the second most frequently identified virus in respiratory tract infections [133]. Extrapolation of consensus data suggests a total of 20 000 hospital and one million clinic visits annually in the US among children younger than 5 [131]. Children hospitalized with hMPV infections are also more likely to present with pneumonia or asthma and required longer stay in intensive care units with supplemental oxygen, when compared to other respiratory viruses [131]. Seroprevalence studies indicates that 100% of young adults are seropositive for hMPV with stable neutralizing titres, which further suggests that reinfection occurs throughout life [16], with a potential for genetic variation between clades promoting reinfection [108].hMPV has a worldwide distribution and affects all age groups but predominantly affects young, elderly and immunocompromised patients [111], with children younger than five years of age being most susceptible to infection [134]. Children and adults with underlying or chronic conditions such as asthma, chronic lung disease, congenital heart disease, cancer or COPD are more likely to be hospitalized with hMPV infection [131]. Infection with hMPV occurs throughout the year but seasonal prevalence in late winter and spring has been observed and coincides with the peak of RSV infection [131,135,136,137].\n4. Human Bocavirus\nThe first human bocavirus (hBoV) was discovered in 2005 from nasopharyngeal aspirates of 282 patients with unresolved lower respiratory tract infections in Sweden. Researchers utilized a novel technique which included steps of DNase treatment to exclude contaminating, or non-viral, nucleic acids followed by PCR amplification by nonspecific primers. The PCR-products were subsequently cloned with large-scale sequencing of the clones. Bioinformatic analysis of generated sequence data yielded the discovery of a new parvovirus with a high homology to bovine and canine minute parvoviruses. The genus name Bocavirus was in fact derived from the species infected by the known virus strains, namely bovine and canine. The new virus was named hBoV1 and was the first virus to be discovered by molecular virus screening [138]. Three additional species of hBoV were later discovered in 2010 and added to the genus; these were named hBoV2, hBoV3 and hBoV4 [139,140,141].\n4.1. Clinical Significance of hBoV\nHBoV1 is a respiratory pathogen affecting all regions of the globe and is associated with approximately 2%&#8211;19% of all upper and lower respiratory tract conditions [142,143,144]. HBoV1 productively infects human airway epithelium cell cultures and leads to damage of airway epithelial cells [145,146,147], which supports clinical observations that infection does result in respiratory disease. In contrast, hBoV2&#8211;4, are found in the gastrointestinal tract and hBoV2, and possibly hBoV3, are associated with gastroenteritis [139,148,149,150]. Interestingly, HBoV2 is the only enteric bocavirus to be isolated from nasopharygeal aspirates and may, therefore, also be associated with respiratory disease [151,152]. HBoV1 is detected in all age groups, but predominantly in young children between the ages of 6&#8211;24 months [143,153,154] and is rarely detected in adults [155,156,157,158,159]. Transmission and infection occurs throughout the year, but predominantly during winter and spring months [158,160,161,162]. Seroprevalence studies suggest that maternal antibodies, which provide protection, are present in infants younger than 2 months of age [163,164], after which seropositivity decreases with low levels of detection until 6&#8211;12 months. Virtually 100% of children aged 6 are seroconverted for hBoV1 and as reinfection occurs throughout life this remains into adulthood [143,163,164,165,166]. The presence of the three enteric bocaviruses does however complicate the findings of seroconversion as cross-reactivity does exist [166].As with many respiratory viruses, clinical differentiation with hBoV1 infection is not possible by symptomatic presentation [8]. Common features of infection of the upper respiratory tract include common cold-like symptoms with cough, rhinorrhoea and acute otitis media [167]. Infection of the lower respiratory tract in children is associated with pneumonia, acute wheezing, asthmatic exacerbations and bronchiolitis [160,162,168,169,170], but life-threatening complications are rare with hBoV1 infection [143]. Although hBoV1 has been isolated from stool samples, there is no statistical evidence to associate hBoV1 with gastrointestinal disease [139]. HBoV1 has not only been found in the upper and lower respiratory tract and gastrointestinal specimens, but also in urine samples, serum, saliva, and tonsils [143]. Rather than having a role in disease pathogenesis, this viraemia and systemic spread may be a feature common to all Parvoviruses as they require proliferating host cells for replication [138].Interestingly, hBoV appears to be more than just a respiratory or gastrointestinal virus. In a recent study, hBoV was identified in 18.3% of lung (n = 11/60) and 20.5% of colorectal (n = 9/44) tumors screened. Unfortunately, the study did not investigate whether the hBoV genomes were in fact incorporated into the host genome as reported for other known Parvoviruses. Therefore, based on their observations as well as previous studies on other parvoviruses, the authors speculate that hBoV could contribute to the development of some lung and colorectal tumors. However, they do also acknowledge that these tumors could simply be providing an optimal environment for hBoV replication and more conclusive studies are required to resolve this issue [171].\n4.2. Coinfection with hBoV1\nHBoV1 has been associated with a prolonged period of persistence in the mucosa of the respiratory tract. This prolonged presence has possibly led to a high frequency of coinfection found with hBoV1 infections of the both the upper and lower respiratory tract [142,157,158]. The high rate of detection of multiple respiratory viruses within up to 83% of respiratory specimens, and the presence of asymptomatic hBoV1 infections, does complicate the determination of the actual pathogenic role of HBoV [143]. High viral load is statistically associated with symptoms [172] and this may therefore be a better indication of coinfections which are related to disease severity or symptomatic presentation. It has been further suggested that patients presenting with viraemia are better candidates to assess the symptoms of disease when compared to investigations of respiratory secretions [172]. The effects and mechanisms of latency, persistence, reactivation and reinfection are however poorly understood and therefore its effects on coinfection and its contribution to active disease cannot be accurately stated [173].\n5. Conclusions\nThe etiological agents of 12%&#8211;39% of lower respiratory tract infections still remain to be identified [138,158]. These results may vary significantly depending on the sensitivity of the diagnostic assay used, respiratory site sampled and even geographical location of the study but it does suggest that many uncharacterized respiratory pathogens could still remain elusive awaiting discovery. The vast improvements in molecular techniques within the past decade have however led to the discovery of four previously circulating respiratory viruses and also the rapid characterization of two completely novel viruses, namely SARS-CoV and MERS-CoV. All these viruses have varying but significant impact on human health and the potential for outbreak of completely novel, emergent respiratory viruses, seen with SARS and MERS, poses their own unique threats. Lessons learned from these viruses, and others currently in circulation, provide health care authorities and scientists with suitable expertise and knowledge to rapidly identify and combat novel respiratory viruses and as our techniques improve we will be in a position to characterize those viruses that are currently difficult to isolate and identify.",
        "pmid": "25757061",
        "title": "Identification of New Respiratory Viruses in the New Millennium",
        "journal_title": "Viruses",
        "authors": "Michael Berry;Junaid Gamieldien;Burtram C. Fielding",
        "doi": "10.3390/v7030996"
    },
    {
        "keywords": "",
        "abstract": "Severe acute respiratory syndrome-coronavirus (SARS-CoV) and SARS-like coronavirus are a potential threat to global health. However, reviews of the long-term effects of clinical treatments in SARS patients are lacking. Here a total of 25 recovered SARS patients were recruited 12 years after infection. Clinical questionnaire responses and examination findings indicated that the patients had experienced various diseases, including lung susceptibility to infections, tumors, cardiovascular disorders, and abnormal glucose metabolism. As compared to healthy controls, metabolomic analyses identified significant differences in the serum metabolomes of SARS survivors. The most significant metabolic disruptions were the comprehensive increase of phosphatidylinositol and lysophospha tidylinositol levels in recovered SARS patients, which coincided with the effect of methylprednisolone administration investigated further in the steroid treated non-SARS patients with severe pneumonia. These results suggested that high-dose pulses of methylprednisolone might cause long-term systemic damage associated with serum metabolic alterations. The present study provided information for an improved understanding of coronavirus-associated pathologies, which might permit further optimization of clinical treatments.",
        "body": "Introduction\nBetween 2002 and 2003, severe acute respiratory syndrome (SARS) caused by coronavirus (SARS-CoV) affected over 30 countries on five continents. Soon after, HCoV-NL63 was identified in a seven-month-old infant with bronchiolitis and conjunctivitis in 20041, and followed by HCoV-HKU1 in 20052. Later, the Middle East Respiratory Syndrome coronavirus (MERS-CoV) rapidly spread worldwide with fatality rates up to 30%3. Several reports have recently demonstrated the existence of a SARS-like coronavirus, which indicates that coronavirus-associated respiratory tract infections continue to have a significant epidemic potential4&#8211;6. Therefore, there remains an urgent requirement for standardization of treatments in order to develop more effective interventional strategies for coronavirus-associated diseases.Clinical treatments implemented following the sudden outbreak of SARS-CoV included empirical and experimental types, although their effectiveness and side effects remain controversial. Antiviral drugs, including ribavirin, lopinavir, and ritonavir, were prescribed soon after SARS-CoV was identified as the causative agent7, 8. Type I interferon was also used owing to its inhibitory effect on SARS-CoV replication9. Furthermore, the use of convalescent plasma or immunoglobulin was reported for the treatment of SARS10. More frequently, corticosteroids, including methylprednisolone alone or in combination with antiviral drugs, were administered to SARS patients to prevent immunopathological lung damage11, although the timing and dosage regimens for steroid administration were controversial12, 13. The administration of high-dose pulses of methylprednisolone has been associated with psychic derangements, acute myopathy, and osteonecrosis14&#8211;16. Considering the fact that these disorders usually progress to a chronic state, we propose that the SARS survivors may suffer from metabolic disturbances, resulting in systemic pathologies in the long-term. Metabolites, including phosphatidylinositol (PI) and lysophosphatidylinositol (LPI), are associated with cellular entry and/or egress of respiratory viruses17, 18. However, it remains unknown whether or not serum levels of PI and LPI are disregulated during recovery of SARS infections.In the present study, metabolomics based on ultra-high-performance liquid chromatography-mass spectrometry (UPLC-MS) and gas chromatography-mass spectrometry (GC-MS) was adopted to assess the comprehensive metabolite profile of serum or urine samples from SARS patients 12 years following their clinical recovery. It was observed that the recovered SARS patients were affected by various serum metabolic disorders predominantly associated with lipid metabolism, including hyperlipidemia (HL), cardiovascular abnormality (CVA), and abnormal glucose metabolism (AGM).\nResults\n\nDemographics of recovered SARS patients at 12 years following recovery\nOf the 31 recruited SARS-CoV survivors, 25 ultimately completed all study procedures, while 6 discontinued participation in the study (three dropouts, one protocol deviation, and two missing comparable healthy controls). The 25 patients that completed the study included 5 men and 20 women, with an average age of 47 years, an average body weight of 67&#8201;kg, and an average body mass index of 24&#8201;kg/m2, which were not significantly different from healthy volunteers (Table&#160;1). None of the recovered SARS patients had smoking or drinking habits. During SARS-CoV infection, these patients had been treated with methylprednisolone at an average of 5,454&#8201;mg for a mean period of 36 days, with a maximum daily dose of 381&#8201;mg (Table&#160;1).Table 1General participant characteristics (means&#8201;&#177;&#8201;SD).Recovered SARS patients (n&#8201;=&#8201;25)Healthy volunteers (n&#8201;=&#8201;25)\nP\nMale/female (n)5/205/20Age (years)47&#8201;&#177;&#8201;1247&#8201;&#177;&#8201;110.486Body weight (kg)67&#8201;&#177;&#8201;1363&#8201;&#177;&#8201;120.107BMI (kg/m2)24&#8201;&#177;&#8201;324&#8201;&#177;&#8201;40.230Smoking08%Alcohol drinking08%Methylprednisolone (mg)5454&#8201;&#177;&#8201;31530&#160;Duration of treatment (days)36&#8201;&#177;&#8201;70&#160;Max. daily pulse (mg)381&#8201;&#177;&#8201;3060Hospitalization since recovery (%)4812 (8% deliveries)Lung infection since recovery (%)6412Osteonecrosis (%)600Hyperlipidemia (%)6840Cardiovascular abnormality (%)440Abnormal glucose metabolism (%)6016Red blood cells in urine (%)334Urine protein (%)80Below normal range (%)&#160;Neutrophil ratio2012&#160;Triiodothyronine80Above normal range (%)&#160;TGs4416&#160;VLDL-C160SARS, severe acute respiratory syndrome; BMI, body mass index; TGs, triglycerides; VLDL-C, very low-density lipoprotein concentration.\nThe responses to the questionnaires indicated that, since recovery, 12 of the 25 recovered SARS patients (48%) had been readmitted to hospital at least once: four had experienced osteonecrosis of the femoral head; one due to fetal death; and two had been diagnosed with hysteromyoma, one with osphyarthrosis, one with cervical spondylosis, one with a spinal cord tumor, and one with breast cancer (data not shown). Sixteen of the recovered SARS patients (64%) had suffered lung infections during the past 12 years (Table&#160;1). The susceptibility of respiratory tract was also suggested by the fact that 13 of the 25 subjects (52%) reported having had a cold at least five times in the past year, as compared with the healthy volunteers, in which all patients reported having had less than five colds in the past year (data not shown). In addition, the recovered SARS patients had a tendency to have HL (68%), CVA (44%), and AGM (60%) (Table&#160;1). Urine screening detected red blood cells in 8 of the 24 SARS subjects (33%); of these, 2 patients (8%) exhibited abnormal urine protein levels. More subjects in the recovered SARS group had a neutrophil ratio and triiodothyronine levels below the normal ranges (20% and 8%, respectively) than in the healthy volunteers (12% and 0%, respectively). Similarly, as compared with the healthy volunteers, more subjects in the recovered SARS group had triglyceride (TG) levels and very low-density lipoprotein concentrations (VLDL-C) above the normal ranges (Table&#160;1 and S1). Together with the SF-36 Health Survey (Fig.&#160;1), these data demonstrated that the recovered SARS patients had a poor quality of life 12 years following recovery, and were susceptible to inflammation, tumors, and glucose and lipid metabolic disorders.Figure 1MOS 36-item Short Form Health Survey (SF-36) results for severe recovered respiratory syndrome patients 12 years after recovery.\n\nAltered serum metabolomes\nAlthough no significant differences were observed in the abundance of metabolites in the urine samples of the SARS and healthy volunteers, partial least squares-discriminant analysis (PLS-DA) showed significant differences in the serum metabolome between recovered SARS patients and healthy controls (Fig.&#160;2A). These data points were clustered into two distinct groups in the plot map, obviously separating the recovered SARS patients and healthy controls (Fig.&#160;2A). The majority of the differentially expressed metabolites, including 2,3,4-trihydroxybutyric acid, cysteine, long-chain acylcarnitine (carnitines C18:0 and C18-OH), fructose, 1,2,4-trihydroxybenzene, pyroglutamic acid, and myo-inositol phosphate, were upregulated in the serum samples from recovered SARS patients compared with the healthy volunteers (Fig.&#160;2B). Metabolites that were decreased in the recovered SARS patients compared with healthy volunteers included methyl esters such as 9,12-octadecadienoic acid and hexadecanoic acid, free carnitine, and arginine (Fig.&#160;2B).Figure 2Differential serum metabolic profiles between recovered SARS and healthy volunteers. (A) OPLS-DA score plot separating the recovered SARS and healthy volunteers (Control). (B) Heat map of significantly altered metabolites detected by gas chromatography-mass spectrometry and liquid chromatography-mass spectrometry.\nTo investigate the potential metabolic disruptions in recovered SARS patients, the alterations in metabolite abundance were further associated with metabolic pathways. The levels of lactic acid derived from glycolysis were elevated in recovered SARS patients compared with healthy volunteers (Fig.&#160;3). Similarly, the levels of cysteine, aspartic acid, and alanine, which can be converted to pyruvic acid, as well as the levels of succinic acid, which is an intermediate of the tricarboxylic acid cycle (TCA) cycle, were elevated in the recovered SARS patients compared with healthy volunteers. In addition, the serum levels of proline, histine, and pyroglutamic acid, which can be converted into glutamic acid and enter the TCA cycle for further energy production or substance synthesis, were increased in the SARS patients compared with healthy volunteers. While the levels of free carnitine were decreased in the SARS patients compared with healthy volunteers, the levels of carnitines C18:0 and C18-OH were increased (Fig.&#160;3). The ratio of carnitine (C16&#8201;+&#8201;C18) to free carnitine ((C16&#8201;+&#8201;C18)/C0) was also increased (Fig.&#160;3), which suggested that fatty acids could easily enter the mitochondria for further &#946;-oxidation in the recovered SARS patients. The serum levels of sarcosine and spermidine were increased in recovered SARS patients compared with healthy volunteers, whereas the levels of citrulline and arginine, which are intermediates of the urea cycle, were decreased (Fig.&#160;4), thus suggesting that there was an increased requirement for arginine in the urea cycle in the recovered SARS patients.Figure 3Mapping of differential metabolites related to energy metabolism (A). Relative serum levels of differentially expressed free carnitine, acylcarnitines with long chains, and the ratio of carnitines (C16&#8201;+&#8201;C18) to carnitine (B). Data shown as mean&#8201;&#177;&#8201;SEM. *&#8201;<&#8201;0.05, **&#8201;<&#8201;0.01.\nFigure 4Mapping of differential metabolites related to arginine and proline metabolism. Data shown as mean&#8201;&#177;&#8201;SEM. *&#8201;<&#8201;0.05, **&#8201;<&#8201;0.01.\n\nDisturbed lipid metabolism\nThe elevated serum lipids included free fatty acids (FFAs 10:0, 15:0, 17:0, 17:1, 20:4, 22:1, and 22:2, accounting for 6.4% of all detected FFAs), lysophosphatidylcholines (LPCs 18:2, 20:2, 21:3, and 22:6, accounting for 23.7% of all detected LPCs), lysophosphatidylethanolamines (LPEs 18:1 and 22:6, accounting for 19.4% of all detected LPEs), and phosphatidylglycerol (PG 36:0) (Table&#160;2). Of note, there were comprehensive elevations of LPIs and PIs in recovered SARS patients. Among 3 LPIs and 14 PIs examined, LPIs 18:0, 18:2, and 20:4 (accounting for 100% of all detected LPIs) and PIs 34:1, 34:2, 36:1, 36:2, 36:3, 36:4, 38:3, 38:4, 38:5, 38:6, 40:5, and 40:6 (accounting for 95.7% of all detected PIs) were increased (Table&#160;2). The lipids that were decreased in recovered SARS patients compared with the healthy volunteers included long-chain FFAs (FFAs 25:0 and 26:0, accounting for 0.026% of all detected FFAs), ether phosphatidylethanolamines (PE O-s 37:4, 38:6, and 40:7), PG 36:2, phosphatidylserines (PSs 38:4 and 40:6, accounting for 3.8% of all detected PSs), sphingomyelins (SMs 34:1;3, 36:0;2, 37:4;2, and 38:0;2, accounting for 1.9% of all detected SMs), and TGs (TGs 44:2, 44:3, and 47:0, accounting for 0.69% of all detected TGs).Table 2Differential lipids in serum from recovered SARS patients versus Healthy volunteers.LipidsRatio\nP\nLipidsRatio\nP\nLipidsRatio\nP\nFFA 10:01.140.039LPE 22:61.270.041PI 38:41.61<0.001FFA 15:01.110.008LPI 18:02.13<0.001PI 38:51.470.001FFA 17:01.080.032LPI 18:21.75<0.001PI 38:61.550.001FFA 17:11.100.005LPI 20:41.74<0.001PI 40:51.460.001FFA 20:41.290.025PE O-37:40.63<0.001PI 40:61.360.001FFA 22:11.290.001PE O-38:60.610.017PS 38:40.850.010FFA 22:21.150.037PE O-40:70.500.001PS 40:60.780.002FFA 25:00.860.029PG 36:01.380.002SM 34:1;30.860.020FFA 26:00.900.006PG 36:20.580.003SM 36:0;20.62<0.001FFA (odd)1.090.019PI 34:11.49<0.001SM 37:4;20.61<0.001FFA (odd)/FFA(even)1.080.023PI 34:21.78<0.001SM 38:0;20.60<0.001LPC 18:21.180.026PI 36:11.64<0.001TG 44:20.61<0.001LPC 20:21.280.008PI 36:21.57<0.001TG 44:30.64<0.001LPC 21:31.060.003PI 36:31.390.002TG 47:00.850.019LPC 22:61.300.021PI 36:41.760.002total SM/total (SM&#8201;+&#8201;PC)0.920.019LPE 18:11.260.035PI 38:31.530.001FFA, free fatty acids; FFA(odd), fatty acids with odd number carbon chains; FFA (odd)/FFA(even), ratio of fatty acids with odd number carbon chains to fatty acids with even number carbon chains; LPC, lysophosphatidylcholine; LPE, lysophosphatidylethanolamine; LPI, lysophosphatidylinositol; PE O-, ether phosphatidylethanolamine; PG, phosphatidylglycerol; PI, phosphatidylinositol; PS, phosphatidylserine; SM, sphingomyelin; TG, triglyceride; total SM/total (SM&#8201;+&#8201;PC), ratio of total SM to the sum of total SM and total phosphatidylcholine.\nSeveral pathways exist for the synthesis of phospholipids and neutral lipids, including fatty acid rearrangement via a remodeling process (Fig.&#160;5A). Serum levels of inositol-3-phosphate and PIs were significantly increased in recovered SARS patients compared with the healthy volunteers (Fig.&#160;5A). Additionally, the PI levels were positively correlated with the levels of very low-density lipoprotein (VLDL) in recovered SARS patients (Fig.&#160;5B and C). Furthermore, VLDL was positively correlated with total TG in both the control and recovered SARS groups (Fig.&#160;5B and C). These results suggested that de novo synthesis of PIs was promoted during or after SARS-CoV infection.Figure 5Major pathways of phospholipid and neutral lipid synthesis. (A) Dashed lines show fatty acid rearrangement through the remodeling process. Correlation analysis of very low-density lipoprotein, phosphatidylinositol, lysophosphatidylinositol, and total triglycerides in (B) recovered SARS and (C) healthy volunteers (Control).\nA cluster analysis was performed to further assess the associations among the differentially expressed lipids. In recovered SARS patients, the three LPIs (LPIs 18:0, 18:2, and 20:4) and FFA 20:4 were most closely clustered, followed by the PIs and LPC 22:6, LPE 22:6, LPC 18:2, and LPE 18:1 (Fig.&#160;6A). Products of phospholipase A2 (PLA2), including FFA 20:4, LPIs, LPCs, and LPEs, showed strong positive correlations (Fig.&#160;6A). LPI 18:0, LPI 18:2, and LPI 20:4 were associated with their respective precursor PIs 18:0, 18:2, and 20:4 (cij&#8201;=&#8201;0.68, 0.6, and 0.67, respectively). Nevertheless, FFA 20:4 was correlated with LPI 18:0/PIs (18:0/20:4) and LPI 18:0/PI 18:0 (Cij&#8201;=&#8201;0.72 and 0.74, respectively), further illustrating the enhanced levels of PI remodeling in recovered SARS patients. More LPIs were produced, which was associated with the release of FFA 20:4, in recovered SARS patients compared with healthy volunteers, which may have been due to the elevated levels of PIs; thus, SARS patients may be more sensitive to PLA2 activity (Fig.&#160;6B). The LPI 18:0/PI 18:0 ratio was elevated, while the FFA 20:4/PI 20:4 ratio was significantly decreased, in recovered SARS patients, which indicated that more LPIs remained following PLA2 cleavage, whereas the released FFA 20:4 from PIs was rapidly used for eicosanoid synthesis (Fig.&#160;6B).Figure 6(A) Cluster analysis of differentially expressed lipids. (B) Fatty acids as a result of phosphatidylinositol remodeling in recovered SARS patients sensitive to phospholipase A2.\n\nEffect of steroid treatment on lipid metabolism\nHigh-dose steroid treatment could contribute to the alterations of lipid metabolism in serum of recovered SARS patients. Next we investigated the short-term effects of methylprednisolone treatment on non-SARS patients with severe pneumonia. Twenty three non-SARS patients with severe pneumonia treated with steroid included 13 men and 10 women (67&#8201;&#177;&#8201;14 years old, 301&#8201;&#177;&#8201;235&#8201;mg of methylprednisolone for a mean period of 9 days). Twenty three non-SARS patients with severe pneumonia without steroid treatment included 15 men and 8 women (68&#8201;&#177;&#8201;8 years old). In coincidence with the former findings, the levels of detected PIs were significantly higher in steroid-treated group than those without steroid treatment (Table&#160;3). Although the doses of steroid used for these non-SARS patients were much less than those for SRAS patients, these results, to some extent, suggested that steroid treatment could increase levels of serum PIs.Table 3Differential phosphatidylinositols in serum from steroid-treated non-SARS patients with severe pneumonia versus non-SARS patients with severe pneumonia without steroid treatment.Lipidssteroid-treated patients vs. non-SARS patients with severe pneumoniaRatio\nP\nPI 32:13.16<0.001PI 34:11.83<0.001PI 34:21.54<0.001PI 35:21.55<0.001PI 36:11.730.003PI 36:21.370.012PI 36:31.570.013PI 36:41.77<0.001PI 37:41.66<0.001PI 38:31.490.021PI 38:41.420.013PI 38:51.600.001PI 38:61.340.022PI 40:41.400.001\n\nDiscussion\nAlthough SARS-CoV survivors benefited from clinical treatments that targeted the virus and inflammation, the recovered patients in the current study continued to experience a poor quality of life 12 years after their infections. Our previous study reported that the pulmonary ventilation function of recovered SARS survivors was normal19. A significant proportion of these recovered patients were reported as having lung infections, HL, CVA, AGM, or other sequelae. Serum metabolomics analyses identified systemic metabolic alterations in these patients. The predominant metabolic dysregulations were observed in the serum, with serum levels of PIs, LPIs, FFAs, and LPCs showing the greatest disruptions. Compared with the other differentially expressed metabolites, LPIs and PIs were markedly upregulated in the recovered SARS patients. The serum levels of LPIs and PIs were not influenced by sequelae observed in these survivors (Tables&#160;S2&#8211;S5). Subjects were regrouped based on metabolic syndromes. Statistical analyses of such subgroups indicated that the identified metabolic alterations were not associated with HL, CVA, or AGM (Figs&#160;S1&#8211;S3). But, our results suggested that early steroid treatment could have an effect on the serum levels of PIs or LPIs in recovered SARS patients. However, in rat experiments, the levels of the detected PIs in rats intraperitoneally injected with methylprednisolone groups (10 or 30&#8201;mg/kg) were comparable to those of the untreated rats (Table&#160;S6), suggesting that the effect of steroid treatment on serum lipid metabolism is more complex than what we thought.PIs have previously been related to viral infections associated with lungs. For example, the membrane binding and penetration of the Ebola virus fusion peptide requires the presence of Pl in the target membrane17. Downstream elements of the PI signaling pathway have been shown to be involved in viral entry. The Ebola virus matrix protein, VP40, was recently shown to require Pl 4,5-bisphosphate for extensive oligomerization and viral egress18. Pl 4-kinase III&#946; (PI4KB) is hypothesized to have a role in the cellular entry of pseudoviruses bearing the SARS-CoV spike protein, and cell entry mediated by the SARS-CoV spike protein was strongly inhibited by knockdown of PI4KB20. Therefore, upregulated PIs in SARS patients may promote SARS-CoV infection and entry. However, Pls were also shown to suppress respiratory syncytial virus infection in mice21.LPIs are thought to have a critical role in glucose homeostasis22. A large proportion of the recovered SARS patients reported glucose metabolic disorders, including hyperinsulinemia, insulin resistance, hyperglycemia, and type 1 or 2 diabetes, a few years prior to this study. LPI stimulates insulin release23, and activation of the G-protein coupled receptor 55 (GPR55) can increase plasma insulin levels and glucose tolerance24. Lipidomic analyses also demonstrated increasing LPI levels in a pre-type 1 diabetic mouse model25. In the present study, increased rates of glycolysis with few effects on the TCA cycle were observed in the recovered SARS patients, suggesting an association between LPI and glucose metabolism in recovered SARS patients. As a result of the broad expression of GPR55 across various tissues, LPI exerts diverse regulatory roles, including in the excitability of dorsal root ganglion neurons26, intestinal inflammation27, and the migration and orientation of human breast cancer28. This may explain the various symptoms observed in the recovered SARS patients. LPI-associated disorders may persist as long as the high serum LPI levels remain.There were a number of limitations associated with the present study. First, we cannot rule out the possibility that some of the patients with glucose metabolic disorders had taken medication prior to the analyses, which may have influenced their physiological status. Second, although we endeavored to recruit as many recovered patients as possible, the sample size was small. Third, the best controls for the investigation of steroid treatment effects were to recruit SARS patients without steroid treatment. Because of lack of those controls, non-SARS patients with severe pneumonia with or without steroid treatment were collected in present study, but they may not exactly mimic the two key elements of SARS infection: steroid treatment and SARS virus infection. Therefore, the reasons for the marked increases in serum levels of LPIs and PIs in recovered SARS patients require further investigation. Changes in metabolite levels observed in our study may not reflect the real situation exactly.As the primary organ damaged by SARS infection, lung function was improved gradually over years. The profound metabolic disruptions may also be directly or indirectly related to initial damages in lungs, which were resulted from SARS-CoV infection. Or the lung damage may be resulted from complex organism response to infection and high-dose pulse of steroid treatment. More importantly, these findings may improve the understanding of coronavirus-induced pathological mechanisms, and may aid in the development of improved measures for the treatment and control of coronavirus infections that continue to threaten human health worldwide.\nMaterials and Methods\n\nEthics Statement\nWritten informed consent was obtained from all study participants. The institutional review board of Tianjin Haihe Hospital provided written approval and judged that all methods in the study met relevant ethical guidelines and regulations (ethical # 2013HHKT-01).\nSubjects\nThirty-one SARS survivors of the 2002&#8211;2003 outbreak, as well as 31 age-matched, consenting healthy volunteers who were medical staff at the Tianjin Haihe Hospital, were recruited in 2014. General participant information, including age, disease duration, and medication, was collected using a standard form. Student&#8217;s t-test (two-tailed) was used to evaluate the differences between the two groups. Furthermore, each participant completed a symptom questionnaire regarding his or her medical history since SARS infection. After fasting overnight, serum and urine samples were collected in the morning from all patients for routine clinical measurements and metabolomics analysis.For further lipid metabolism investigation, serum samples were also collected from 46 non-SARS patients with severe pneumonia, half of these patients were treated with methylprednisolone.\nClinical measurements\nEvaluation of the general physical ability of the patients included quality of life scoring (Medical Outcomes Study Short Form-36, SF-36), Harris Hip score, the six-min walk test, assessment of pulmonary ventilation function, and the bronchial dilation test. Routine clinical chemical analyses were conducted using fasting blood samples at the Tianjin Haihe Hospital.\nLC-MS-based serum metabolic profiling\nSerum samples were prepared as described before29. Briefly, protein precipitation was performed on the serum samples. Aliquots of 200&#8201;&#956;L were drawn for each sample, and four volumes of acetonitrile (with internal standard mixture of carnitine C2:0-d3, carnitine C10:0-d3, carnitine C16:0-d3, LPC 19:0, free fatty acid (FFA) 16:0-d3, FFA 18:0-d3, chenodeoxycholic acid-d4, cholic acid-d4, leu-d3, Val-d8, tryptophan-d5, phenylalanine-d5 and leucine-enkephalin) were added. After thorough vortex and centrifugation (13,000&#8201;&#215;&#8201;g for 10&#8201;min at 4&#8201;&#176;C), two aliquots of 300&#8201;&#956;L supernatant were pipetted for drying in CentriVap Centrifugal Vacuum Concentrators (Labconco, MO). The dried residues were stored at &#8722;80&#8201;&#176;C until analysis. LC-MS-based metabolic profiling was performed on an ultra-high-performance liquid chromatograph (Waters, USA) coupled to a tripleTOF&#8482; 5600 plus mass spectrometer (Applied Biosystems, Foster City, CA) equipped with an electrospray source. The dried serum samples were dissolved in acetonitrile (ACN)/water (1:4&#8201;v/v) and, following centrifugation at 4&#8201;&#176;C, 5&#8201;&#956;L supernatant was loaded. For the positive mode, a BEH C8 (100 mm&#8201;&#215;&#8201;2.1 mm&#8201;&#215;&#8201;1.7 &#956;m; Waters, Milford, MA) column was used. The gradient was initiated with 10% B mobile phase (0.1% formic acid in ACN), which was maintained for 1&#8201;min, followed by linearly increasing to 40% B within 4&#8201;min, and then to 100% B at 17&#8201;min, which was maintained for 5&#8201;min. The mobile phase was rapidly changed to 90% A (0.1% formic acid) within 0.1&#8201;min, and the total run time for each injection was 25&#8201;min, including a post-equilibration of 2.9&#8201;min.\nGC-MS-based serum metabolic profiling\nThe derivatization procedure and data acquisition were performed as described previously, with some modifications29. Briefly, the temperature for both oximation and silylation reactions was changed to 37&#8201;&#176;C and the oximation reaction time was reduced to 1.5&#8201;h. Serum metabolic profiling was performed in a pseudotargeted mode using a GCMS-QP 2010 Plus quadrupole mass analyzer coupled to an AOC-20i autosampler (Shimadzu, Kyoto, Japan). Chromatographic separation was performed on a DB-5 MS capillary column (30&#8201;m&#8201;&#215;&#8201;250 &#956;m&#8201;&#215;&#8201;0.25 &#956;m; J&W Scientific, Folsom, CA). First, the quality control (QC) sample was analyzed in full scan mode with a mass-to-charge ratio (m/z) scan range of 33&#8211;600 Dalton to obtain the target ion for selected ion monitoring (SIM) scanning. The parameters for the full scan were as follows: the flow rate of the carrier gas (helium) was constant at 1.2&#8201;mL/min; the oven temperature began at 70&#8201;&#176;C, which was maintained for 3&#8201;min, followed by a linear increase to 310&#8201;&#176;C at 5&#8201;&#176;C/min, and finally holding for 6&#8201;min at 310&#8201;&#176;C; the temperature of the inlet and interface was 280&#8201;&#176;C, and that of the ion source was 240&#8201;&#176;C; the detector voltage was set at 1.05&#8201;kV; the solvent delay was 5.1&#8201;min; and the event time was initiated at 0.5&#8201;s. With the integrated SIM scan table, batch analysis for all samples was conducted in pseudotargeted mode with all MS parameter settings the same as those used in full scan mode, but with the event time adjusted to 0.2&#8201;s.\nLipidomic analysis of serum samples\nFor lipidomics analysis, 30-&#181;L serum aliquots were pipetted for each sample and 200&#8201;&#181;L methanol containing 8 lipid internal standards (PC38:0, PE34:0, LPC19:0, SM12:0, triglyceride 45:0 (TG45:0), Cer17:0, FFA16-d3 and FFA18-d3) was added, followed by a 30-sec vortex. Then, 400&#8201;&#181;L chloroform was added and the mixture was vortexed for another 30&#8201;seconds. After that, 120&#8201;&#181;L water was added and the extraction mixture, vortexed, and centrifuged (8,000 x g for 10&#8201;min at 4&#8201;&#176;C) to form aqueous-organic separation phases. Two, 200&#8201;&#181;L aliquots of the upper organic phase were drawn and dried, respectively, in a vacuum centrifuge and stored at &#8722;80&#8201;&#176;C until analysis. Quality control (QC) samples were produced by pooling equal serum aliquots from each sample, containing the average metabolome information from the real samples included in this study. For LC-MS, GC-MS metabolic profiling and lipidomics, QC sample pretreatment was performed using the same method as that used for the actual patient samples. The QC sample was run every 10 real samples to evaluate the repeatability of sample pretreatment and for batch analysis of each analytical platform. The lipidome was analyzed using UPLC (Waters) and triple-quadrupole time-of-flight mass spectrometry (AB Sciex) systems. Prior to the analysis, the dried serum samples were reconstituted in 20&#8201;&#181;L CHCl3-MeOH solution (2:1&#8201;v/v) and then diluted in a mixed solvent containing ACN-MeOH-H2O (65:30:5&#8201;v/v/v). Subsequently, 5&#8201;&#181;L of the diluted samples were separated using a C8 ACQUITYTM column (150&#8201;&#215;&#8201;2.1 mm, 1.7&#8201;&#181;m internal diameter). The column temperature and elution rate were set at 55&#8201;&#176;C and 0.26&#8201;mL/min, respectively. The mobile phases A and B, which were ACN:H2O (6:4&#8201;v/v) and isopropanol: ACN (9:1&#8201;v/v), respectively, both contained 10&#8201;mM ammonium acetate. The initial gradient started at 32% B, which was maintained for 1.5&#8201;min, followed by a linear increase to 85% B in 14&#8201;min, and then to 97% B within 0.1&#8201;min. This condition was maintained for 2.4&#8201;min at 97% B, followed by a decline to 32% B within 0.1&#8201;min. The total run time was 20&#8201;min, including a post-equilibration time of 1.9&#8201;min. The mass spectrometry signal was scanned from a m/z of 400&#8211;1500 Dalton in both positive and negative ion modes. The capillary voltages were set at 4.0&#8201;kV and &#8722;3.8&#8201;kV for positive and negative modes, respectively. The capillary temperature was 350&#8201;&#176;C.\nData preprocessing for serum GC-MS, LC-MS, and lipidomics analysis\nFor the GC-MS analysis, unique ions for metabolites in the final SIM scan table were imported to GC-MS solution software for batch integration. For the LC-MS analysis, raw data were first imported to Peak-view software for automated peak detection, alignment, and integration. After checking the derived peak table, known metabolites that had been missed or had a relative standard deviation of greater than 30% during the automated peak picking were extracted and integrated manually. Further unknown ions detected in blank samples were deleted if their intensity proportion in the blank sample was more than 1% compared with the average intensity levels in the experimental samples. The 80% rule was applied to remove peaks with missing values surpassing 20% in both groups. For mass spectrometry response normalization in both GC-MS and LC-MS analyses, the raw peak areas for each ion or metabolite were normalized to that of adjacent QCs, with the local linear regression of QC intensities selected. For the lipidomics analysis, peak picking was performed using Lipid-View software (AB Sciex). Owing to their complexity, the lipid compounds were detected by extracting the ion chromatograms to avoid the &#8220;black box&#8221; associated with automatically pretreated data. Adjustment of the integration parameters was performed in certain cases. The peak areas of each detected lipid compound were normalized to those of the spiked lipid internal standards according to the different lipid classes.\nStatistical analyses\nMann-Whitney U tests and hierarchical cluster analyses were performed using the Multi Experiment Viewer. The significance level for the univariate analysis was set at p&#8201;<&#8201;0.05. Corresponding q values for each significant metabolite were calculated to evaluate false discovery rates in multiple significant tests. An overall safer bootstrap method was used for estimation with the step 0.05 set for lambda. For robust q value calculation from small p values with finite samples, the &#8220;use robust method&#8221; was further selected. The critical value for q was set as 0.2. The column plots were drawn using Vanted, with metabolic pathways based on Kyoto Encyclopedia of Genes and Genomes (KEGG) classifications30.\nElectronic supplementary material\n\nNone\n\nSupplemental data\n",
        "pmid": "28831119",
        "title": "Altered Lipid Metabolism in Recovered SARS Patients Twelve Years after Infection",
        "journal_title": "Scientific Reports",
        "authors": "Qi Wu;Lina Zhou;Xin Sun;Zhongfang Yan;Chunxiu Hu;Junping Wu;Long Xu;Xue Li;Huiling Liu;Peiyuan Yin;Kuan Li;Jieyu Zhao;Yanli Li;Xiaolin Wang;Yu Li;Qiuyang Zhang;Guowang Xu;Huaiyong Chen",
        "doi": "10.1038/s41598-017-09536-z"
    },
    {
        "keywords": "",
        "abstract": "Feline infectious peritonitis virus (FIPV) is highly virulent and responsible for the highly fatal disease feline infectious peritonitis (FIP), whereas feline enteric coronavirus (FECV) is widespread among the feline population and typically causes asymptomatic infections. Some candidates for genetic markers capable of differentiating these two pathotypes of a unique virus (feline coronavirus) have been proposed by several studies. In the present survey, in order to search for markers that can differentiate FECV and FIPV, several clones of the 3a&#8211;c, E, and M genes were sequenced from samples obtained from cats with or without FIP. All genes showed genetic diversity and suggested the presence of FCoV mutant spectrum capable of producing a virulent pathotype in an individual-specific way. In addition, all the feline coronavirus FIPV strains demonstrated a truncated 3c protein, and the 3c gene was the only observed pathotypic marker for FCoVs, showing that 3c gene is a candidate marker for the distinction between the two pathotypes when the mutant spectrum is taken into account.",
        "body": "1. Introduction\nFeline coronavirus (FCoV) (Nidovirales; Coronaviridae; Coronavirinae; Alphacoronavirus 1) is a widespread pathogen among the domestic cat population. FCoV occurs as two pathotypes, feline enteric coronavirus (FECV) and feline infectious peritonitis virus (FIPV), both of which with strains of serotypes I and II [1].Phenotypically, the main difference between the pathotypes is that FIPV is able to infect monocytes and macrophages, causing systemic infection and fatal disease, while FECV is limited to replication in the mature intestinal epithelium, which mainly results in asymptomatic infection. However, FECV may undergo a short systemic viremia phase that also involves monocytes [2, 3].The FCoV genome (single-stranded positive-sense linear RNA, 29&#8201;kb) encodes proteins directly involved in viral replication (ORF1), the structural spike protein (S), envelope protein (E), membrane protein (M), and nucleocapsid protein (N), as well as the accessory proteins 3a&#8211;c and 7a-b [4].There is a major controversy regarding the existence of genetic markers to differentiate between FECV and FIPV pathotypes [5, 6]. Numerous markers that correlate with the manifestation of FIP have been proposed for S [7&#8211;12], 3a&#8211;c [8, 13&#8211;16], 7a-b [16&#8211;18], and M [5] genes.The most studied gene is 3c [7, 9, 13&#8211;15, 19, 20], which is present in the truncated form in the majority of all strains obtained from animals with FIP. However, previous studies on 3c genes focused only on the dominant sequence and not on the mutant spectrum or the clonal diversity and the information obtained from a dominant sequence may not be suitable to detect mutations [10], with the exception of a study [7] in which clones were used to assess the presence of alterations in the gene 3c in ascites and/or faeces from FIP and healthy cats, and again 3c truncated protein was not observed in 100% of FIPV samples.Considering (i) the current discussion on molecular markers capable of distinguishing the pathotypes of FCoV and (ii) the lack of knowledge of the diversity of such markers in the mutant spectrum of RNA, the present study was designed to investigate the clonal diversity of the 3a&#8211;c, E, and M genes in FCoV strains present in tissue and/or faecal samples from cats with and without FIP.\n2. Materials and Methods\n\n2.1. Animals and Samples\nFrom 2010 to 2013, tissue (eye, cerebrum, cerebellum, lung, heart muscle, thoracic lymph node, thymus, liver, spleen, stomach, mesenteric lymph node, peripancreatic lymph node, kidneys, large and small intestines, and urinary bladder), abdominal, thoracic, and pericardial effusions, aqueous humor, and faecal samples were collected during the necropsies of 6 cats with clinical manifestations and histopathological changes consistent with FIP, such as localized inflammation with macrophages, neutrophils, lymphocytes, and plasma cells. Vascular lesions were found surrounded by a proliferation of inflammatory cells in heart, lung, omentum, kidneys, cerebrum, and cerebellum. Focal accumulations of inflammatory cells and necrotic-proliferative lesions were observed in granulomatous lesions. Intestinal samples were collected including mucosa, submucosa, muscular layer, and serosa. Only 2 animals (USP6, and USP7) showed signs of dry-form FIP (neurological changes), while the remaining 4 (USP8, USP16, USP17, and USP18) showed an accumulation of effusion in body cavities (effusive FIP).All FIP-positive cats were sampled from different geographic locations, with the exception of 2 cats (USP17 and USP18) that lived in the same house. USP17 was euthanized 1 month prior to USP18 due to FIP related reasons.In parallel, faecal samples were also obtained from 2 cats (USP2, and USP14) without clinical signs of FIP and from two different locations not related to the locations from the FIP+ cats.This study has been approved by Commission on Ethics on Animal Use from School of Veterinary Medicine, University of S&#227;o Paulo (CEUA N. 2055211113).\n2.2. Messenger RNA Detection for the M Gene of FCoVs\nAll tissues were complete homogenized after tissue maceration and before RNA extraction. Total RNA extraction and RT-PCR to detect mRNA of the M gene of FCoVs were performed in all samples as previously described [21] as a screening test for FCoV replication.\n2.3. Amplification of the 3a\u2013c, E, and M Genes of FCoVs\nAll samples from 6FIP+ and 2FIP&#8722; cats were submitted to amplification of the 3a&#8211;c, E, and M genes of FCoVs, even if the mRNA detection was negative. To amplify all these five genes of FCoVs in only one amplicon, a single pair of previously described primers [14] was used (nucleotides 24,380 to 26,996 corresponding to the FCoV-WSU 79-1146; genome GenBank DQ010921).cDNA synthesis was performed with 6&#8201;&#956;L total RNA extracted with TRIzol&#8482; Reagent (Life Technologies Corporation, Carlsbad, CA, USA) and ThermoScript&#8482; (Life Technologies Corporation, Carlsbad, CA, USA) using Random Primers&#8482; (Life Technologies Corporation, Carlsbad, CA, USA) according to the manufacturer's instructions.For PCR, 5&#8201;&#956;L of cDNA was added to 1x PCR buffer (Life Technologies Corporation, Carlsbad, CA, USA), 2&#8201;mM MgSO4, 0.2&#8201;mM of each dNTP, 0.25&#8201;&#956;M of each primer, 1.5&#8201;U Platinum&#8482; Taq DNA Polymerase High Fidelity (Life Technologies Corporation, Carlsbad, CA, USA), and DEPC water for a final reaction volume of 50&#8201;&#956;L. The cycling conditions were as follows: 94&#176;C/2&#8201;min; 10 cycles of 94&#176;C/15&#8201;s, 50&#176;C/15&#8201;s with an increase of 0.5&#176;C per cycle, and 68&#176;C/4&#8201;min; 10 cycles of 94&#176;C/15&#8201;s, 55&#176;C/15&#8201;s, and 68&#176;C/4&#8201;min; 25 additional cycles of 94&#176;C/15&#8201;s, 55&#176;C/15&#8201;s, and 68&#176;C/4&#8201;min with an increase of 10&#8201;s per cycle; and a final extension of 72&#176;C/10&#8201;min.Specific bands were excised from the gels and purified using the Gene Jet&#8482; Gel Extraction kit (Thermo Fisher Scientific Inc., Vilnius, Lithuania) according to the manufacturer's instructions.\n2.4. Molecular Cloning\nThe purified amplicons (2,617&#8201;bp containing 3abc-E-M) were cloned using pGEM&#8482;-T Easy Vector Systems (Promega Madison, WI, USA) and E. coli JM109 competent cells according to the manufacturer's recommendations. After the transformation protocol, transformation culture was plated onto duplicate LB/ampicillin/IPTG/X-Gal plates and the plates were incubated overnight (16 hours) at 37&#176;C. Then, the transformed colonies were transferred to LB liquid medium and were incubated overnight (16 hours) at 37&#176;C and 180&#8201;rpm. Recombinant plasmid extraction was performed with the Illustra&#8482; plasmidPrep Mini Spin Kit (GE Healthcare, Buckinghamshire, UK) as per manufacturer's recommendations.For an easier identification of the clones, the following nomenclature was used: cat identification/sample source/presence (+) or absence (&#8722;) of FIP/cat origin/clone number.\n2.5. DNA Sequencing\nSense primers were designed at approximately every 500 nucleotides to obtain full coverage of genes 3a&#8211;c, E, and M (Table 1). Primers for the plasmid (pUC/M13F and R) were also used to reach the full length of the amplicon sequence.DNA sequencing with each primer in duplicate was performed using the BigDye&#8482; Terminator v3.1 Cycle Sequencing Kit (Life Technologies Corporation, Carlsbad, CA, USA) with a Genetic Analyzer 3500&#8482; capillary sequencer (Applied Biosystems Foster City, CA, USA), according to the manufacturers' instructions.\n2.6. Sequence Editing\nChromatograms were generated in duplicate for each segment and evaluated for their base quality using the online application Phred (http://asparagin.cenargen.embrapa.br/phph/). Only those chromatograms with scores greater than 20 (probability lower than 1 error per 100 nucleotides) were used. The chromatograms were analyzed and manually edited with the software FinchTV version 1.4 (http://www.geospiza.com/Products/finchtv.shtml). The final sequences of each sample were obtained using the CAP contig application with the software BioEdit version 7.0.9.0 [22].\n2.7. Phylogenetic Analyses\nThe nucleotides and amino acids corresponding to the 3a&#8211;c, E, and M genes were aligned using the application Clustal/W in BioEdit 7.0.9.0 [22]. Clones were classified in serotypes based on comparative analysis of the amino acid sequences encoded by the 3a gene with the sequences available in GenBank (serotype I: DQ160294 and DQ848678 and serotype II: DQ010921). Nucleotide phylogenetic trees (neighbor-joining, MCL model, 1,000 bootstrap replicates) were constructed using the computer software Mega version 5.05 [23].\n3. Results\n\n3.1. Messenger RNA Detection for the M Gene and Amplification, Sequencing, and Cloning of the 3a\u2013c, E, and M Genes of FCoVs\nmRNA was not obtained from all samples; nevertheless sequences for the 3a&#8211;c, E, and M amplicons were obtained from a total of 292 clones originating from 8 animals [6FIP+ (USP6, USP7, USP8, USP17, USP16, and USP18) and 2FIP&#8722; (USP2 and USP14)]. Sequences from extraintestinal tissue samples and faecal/intestines samples were obtained simultaneously for 3 animals (USP6, USP7, and USP17) with FIP (Table 2). Due to poor sequence quality for some clones and/or regions in a clone, not all full length amplicons could be sequenced or used in the analyses. The sequences obtained in the present study were deposited in GenBank under access numbers KJ879334 to KJ879438.\n3.2. Phylogenetic Analysis of the 3a\u2013c, E, and M Genes of FCoVs\nAll sequences of the 3a genes obtained from a total of 79 clones belong to serotype I. Phylogenetic trees generated for the 3a, 3c, E, and M genes showed similar topologies and thus only the 3c and 3b trees are shown (Figures 1 and 2, resp.). Sample from cat USP14 (non-FIP) showed a remarkable clonal diversity as their clones are present in five different clusters of the 3b tree. High clonal diversity was observed in samples from FIP+ (USP8, USP16, USP17, and USP18) and FIP&#8722; cat (USP14); however FIP+ cats USP6 and USP7 and USP2 FIP&#8722; cat showed a lower clonal diversity. In the 3c gene tree, for cats USP6 and USP7, differences can be noticed between the FCoV strains obtained from tissue and faecal clones; that is, for each animal, there was a distinction between systemic (FIPV) and enteric (FECV) strains. However, no generic grouping per pathotype (FECV or FIPV) was observed for all samples and trees generated from this study.Conversely, in the 3b gene tree samples obtained from cat USP7 (FIP+) could not be separated into systemic (cerebrum) and enteric (faeces) strains.The overall nucleotide identity of all clones (FIP+ and FIP&#8722;) obtained from the 5 genes was &#8805;92.09% for faecal samples and &#8805;90.09% for tissue samples (Table 3). The greatest nucleotide diversity when all available sequences for each gene were compared together was observed for the 3b gene, for which a 90.09% identity was found, with the exception of comparisons between cats living in the same house, for which the identity between the 3b gene sequences was 97.09%.For tissue and faecal samples, 3a was the most conserved gene (nucleotide identities of 99.38% and 97.70%, resp.) considering each individual cat. The mean identity between clones from the two cohabitating FIP+ cats (USP17 and USP18) ranged from 96.17% (3c gene) to 99.83% (E gene).Premature stop codons were observed in the 3a (USP18, 3/3 cerebrum clones), 3b (USP17, 1/8 cerebrum; USP7, 1/10 faeces), 3c (USP17, 8/8 cerebrum and 2/2 large intestine; USP18, 2/2 cerebrum; USP7, 2/2 cerebrum; USP16, 1/9 cerebrum; USP8, 1/1 stomach; USP6, 6/6 stomach), and M (USP7, 1/5 cerebrum) genes from FIP+ cats. No premature stop codon in the 3a&#8211;c or E genes was observed in faecal samples from cats without FIP, with exception of M gene (USP2, 1/6 faeces).Stop codons in 3c were evident in at least one of the clones from tissue samples from cats with FIP. The faecal samples from cats with or without FIP did not demonstrate this truncated gene in any of the clones, and no other molecular marker capable of differentiating the biotypes (FECV and FIPV) was observed in the sequenced genes.\n4. Discussion and Conclusions\nThe existence of genetic markers to differentiate these two biotypes is currently a topic of the highest interest on the FIP field not only from the clinical and epidemiological points of view but also as a basis for an in-deep comprehension of the pathogenesis of this highly complex disease. Considering that, in the present study, clones of structural (E and M) and accessory (3a&#8211;c) genes were obtained from cats with and without FIP, and the results demonstrate a highly likely candidate FCoV pathotypic marker in the 3c gene.Amplicons obtained from the cerebrum and large intestine of the cat USP17 (FIP+) were also cloned and sequenced. When assessing the 46 clones (32 clones from cerebrum and 14 from large intestine) obtained from this cat, the nucleotide identity among the five genes was greater than 99%, and according to the phylogenetic trees generated, clones from this cat resulted in a single cluster. In the 3c gene, a single nucleotide transversion (G &#8594; T) was observed at position 10 in all clones from samples from this cat, resulting in premature stop codons and truncated proteins. The observation of truncated proteins in large intestine observed herein is in contrast to what has been observed for FECV in other studies [13, 14, 20].Possibly because it serves as a preferential organ for viral replication and therefore for pathological changes in FIP cases, the intestine of the cat USP17 (FIP+) was also affected by the systemic strain. Chang et al. [20] suggested that viruses with an inactivated 3c gene rarely replicate in the intestine, which helps to explain the rare incidence of FIP outbreaks due to a less frequent faecal-oral transmission. A premature stop codon in the 3c gene was previously reported in a strain of FCoV from the jejunum of a cat with FIP, and the same intestinal strain showed 100% identity with a strain obtained from a liver sample of the same cat [6].In recent studies [13, 15], 3c genes with premature termination were detected in faecal and extraintestinal samples from cats with FIP, although it was not determined whether the viruses were replicating. Also, after an experimental inoculation of cats with truncated and complete 3abc, the role of the truncation in this region on FIP pathogenesis could not be clearly determined, despite it led to lower viral replication in the gut [24]. However, in the present study, the sample from the large intestine of the cat USP17 (FIP+), from which the studied clones originated and which demonstrated a truncated 3c gene, was positive for FCoV mRNA. This result indicated the presence of viral replication in that sample.When comparing the sequences obtained from the extraintestinal organs and faeces of the FIP+ cats USP7 and USP6, the presence of a premature stop codon in the 3c gene was not observed in faecal clones from both cats, in contrast to what was observed for tissue clones. Moreover, the nucleotide identity of the 3c gene for tissue and faecal samples was 99.09% and 95.58% for the cats USP7 and USP6, respectively. In a study comparing FCoV from extraintestinal FIP lesions and the faeces of the same cat, the identity between the sequences of the structural and accessory genes evaluated (S, M, N, E, 3a&#8211;c and 7a-b) was always higher than 99%, corroborating the hypothesis of in vivo mutation [14].As FIPV lineages may be derived from FECV mutations, it was expected that nucleotide diversity between faecal and tissue samples from the cat USP6 (FIP+) would be lower. However, the difference was 8.84% for the 3b genes, consisting exclusively in point mutations, indicating that this cat could be coinfected with an FCoV strain that is distinct from the systemic strain. Furthermore, the high nucleotide identity between samples from the cerebrum and faeces of the cat USP7 (FIP+), along with the topology of the generated trees, showed that the intestinal viral replication of the enteric pathotype may be maintained even during FIP in some cases, as demonstrated by the presence of mRNA.As previously described and according with results from the present study, FIPV can both emerge endogenously and be transmitted to different cats as a &#8220;ready&#8221; virulent pathotype, which has major implications for the understanding of the dynamics of viral transmission [5, 15, 21].Among the structural proteins genes (E and M) studied herein, premature stop codons were detected only in the M gene. The M protein of FCoVs is the most abundant structural protein [25]. The presence of a premature stop codon in a constitutive gene such as the M gene is incompatible with the functionality of the virion. However, this defective virus is no less important than the others, as it provides genetic background for recombination with other FCoVs, resulting in greater genetic diversity and possibly increased virulence of some viral quasispecies. Recent studies suggest that the 3c gene, which encodes a small protein (238aa) of unknown function, is associated with most FIPV isolates when present in truncated form, although not all FCoV strains in FIP+ samples present this 3c form [13, 14, 20]. In the present study, the 3c truncated protein was observed in all clones obtained from tissue samples from FIP+ cats, with the exception of those obtained from the cat USP16, where only 11% (1/9) of the clones demonstrated the truncated protein.It is essential to note that if the predominant viral population in the present study had been assessed by direct amplicon sequencing only, as in studies conducted by other authors [13&#8211;15, 19, 20], without using the previous cloning step, the truncated 3c gene could have been undetected as it was only present in a minority of the sequences observed in the cerebrum of the cat USP16 (FIP+).The transversions observed at position 10 of the 3c gene, which resulted in premature stop codons, were the same as those observed in the clone sequences obtained from the cerebrum and stomach of the FIP+ cats USP17 and USP6, respectively. With the exception of this finding and in agreement with the results of Pedersen et al. [14, 15], the 3c gene mutations were unique to each cat, including the sequences obtained from the cohabitating FIP+ cats (USP17 and USP18).It can be concluded that the presence of a truncated 3c protein is a suggestive factor for differentiation between the FECV and FIPV pathotypes and therefore serves as a genetic marker of virulence. It still remains to be answered whether 3c is expendable for FCoV replication in phagocytic cells and a nontruncated 3c is thus selected-against during the FECV/FIPV transition. Moreover, the 3a&#8211;c, E, and M genes showed a level of genetic diversity indicative of the constitution of FCoV quasispecies or mutant spectrum and the likelihood of emergence of a highly virulent, individual-specific biotype.",
        "pmid": "27243037",
        "title": "Feline Coronavirus 3c Protein: A Candidate for a Virulence Marker?",
        "journal_title": "BioMed Research International",
        "authors": "A. S. Hora;P. O. Tonietti;S. A. Taniwaki;K. M. Asano;P. Maiorka;L. J. Richtzenhain;P. E. Brand\u00e3o",
        "doi": "10.1155/2016/8560691"
    },
    {
        "keywords": "coronavirus;COVID-19, SARS-CoV-2, 2019-nCov;Surveillance;contact tracing;containment;France",
        "abstract": "A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.",
        "body": "Background\nA novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020 [1]. Twenty-seven patients with pneumonia had initially been reported, with an epidemiological link to a live animal market that was closed and disinfected on 1 January [1]. From 20 January, the number of notifications of cases rose dramatically, and as at 12 February 2020, 45,179 cases of SARS-CoV-2 have been confirmed, including 1,116 deaths [2]. Most of the cases (n&#8239;=&#8239;44,665) were reported in 31 provinces and autonomous regions of China and 514 cases were reported in 25 other countries in Asia, Australia, Europe and North America [2]. To date, the primary source of infection remains unknown and could still be active. Human-to-human transmission was observed early after the emergence of this new virus in China and abroad, including family clusters and healthcare settings. The current outbreak dynamics strongly indicate sustained human-to-human transmission.Strengthened surveillance of COVID-19 cases was implemented in France on 10 January 2020. The objective of the surveillance is to identify imported cases early and to prevent secondary transmission whether in the community or among healthcare workers (HCW). Investigations are carried out among contacts immediately upon illness onset and a follow-up procedure is initiated according to the evaluated level of infection risk.Here we describe the real-time implementation of this surveillance scheme for the first three imported cases of COVID-19 identified in France, who were confirmed on 24 January 2020 in persons with a recent stay in Wuhan. Two cases were diagnosed in Paris and one in Bordeaux. We present data until 12 February on the follow-up of the cases&#8217; contacts initiated immediately upon confirmation of infection.\nMethods\n\nFrench surveillance system\nIn France, according to the COVID-19 surveillance protocol, physicians suspecting a COVID-19 case have to contact immediately either the emergency hotline (SAMU-Centre 15), if the patient is seeking medical attention from a general practitioner, or a referring infectious diseases specialist at hospital level. Together, they evaluate whether the patient matches the case definition criteria for a possible case (see below). If they do, the case has to be reported immediately through a 24/7 available phone line to the Regional Health Agency (Agence r&#233;gionale de sant&#233;, ARS), which informs without delay the hospital infection control teams involved in the management of the patient, the French Public Health Agency (Sant&#233; publique France, SpFrance) and the Ministry of Health.A standardised investigation form collecting socio-demographical information, clinical details and history of exposure (history of travel to or residence in Wuhan, China or contact with a confirmed case) is completed for each possible case at regional level, in collaboration between the clinicians, the ARS and SpFrance. Data are entered into the secure web-based application Voozanoo (Epiconcept, Paris).Possible cases have to be hospitalised, isolated and cared for in one of the 38 French referral hospitals designated by the Ministry of Health, according to the guidelines for the management of patients with Middle East respiratory syndrome (MERS) [3].For each possible case, respiratory samples from the upper respiratory tract (nasopharyngeal swabs or aspirates) and when possible from the lower respiratory tract (bronchoalveolar lavage fluid, when indicated, or induced sputum) are collected and sent to one of the laboratories accredited to perform SARS-CoV-2-specific real-time RT-PCR. Until 27 January, only the National Reference Centre for respiratory viruses (Institut Pasteur, Paris) was able to test for the presence of the SARS-CoV-2.\nCase definition\nFrom 17 to 29 January 2020, a possible case was defined either as a patient with a severe acute lower respiratory infection requiring admission to hospital and with a history of travel to or residence in Wuhan, China in the 14 days before symptom onset, or a patient with an acute respiratory illness whatever the severity and with a history of at-risk exposure, mainly to a confirmed case. A confirmed case was defined as a possible case with a positive SARS-CoV-2 RT-PCR on respiratory samples, performed by an accredited laboratory. Testing relied on the real-time RT-PCR procedure developed by the Charit&#233; [4] as well as on the use of real-time RT-PCR specific for the RdRp gene (four targets) designed at Institut Pasteur (RdRp-IP).The case definition was first set up on 10 January and adapted over time. The detailed case definition used for the cases presented here as well as the most up-to-date case definition are available in the Supplement.\nContact and co-exposure tracing\nCo-exposed persons are defined as people who shared the same risks of exposure as a possible or confirmed case of COVID-19. Contact and co-exposure identification is done for all identified possible cases. Contacts are traced from the date of onset of clinical symptoms in a case. If the diagnosis of SARS-CoV-2 infection is confirmed in the index case, active surveillance of contacts/co-exposed persons is initiated immediately.Three levels of risk of infection are defined for contacts/co-exposed persons of a possible/confirmed COVID-19 case (Table). Co-exposed persons of a confirmed case are followed-up according to the same procedure as a moderate-/high-risk contact. The follow-up procedure for the contacts/co-exposed persons differs according to the evaluation of the level of risk of infection (Table).During the initial implementation phase of the procedure, owing to the limited number of contacts involved, it was decided to also implement an active follow-up for low risk contacts.Patients are interviewed by the clinicians, with the help of a translator if needed, who recover relevant information on their contacts since onset of clinical symptoms and the nature and intensity of exposure. The involved regional health agencies work closely with the regional entities of Sant&#233; publique France (cellules r&#233;gionales) in order to implement contact tracing and follow-up. Sant&#233; publique France coordinates the surveillance at national level in liaison with the national Health Authorities.\nEthical statement\nThe investigations were carried out in accordance with the General Data Protection Regulation (Regulation (EU) 2016/679 and Directive 95/46/EC) and the French data protection law (Law 78&#8211;17 on 06/01/1978 and D&#233;cret 2019&#8211;536 on 29/05/2019). Informed consent to disclosure of information relevant to this publication was obtained from the three patients confirmed with 2019-nCoV infection.\nResults\n\nDetected confirmed cases\nBetween 10 January and 24 January (period until confirmation of the first cases in France), nine possible cases were identified in France; among them, three cases were confirmed with COVID-19.Case 1 was a 48-year-old male patient living in France. He was travelling for professional reasons in China in various cities including Wuhan when he experienced his first symptoms (fever, headaches and cough) on 16 January. He flew back to Bordeaux, France on 22 January via Shanghai, Qingdao and Paris Charles de Gaulle airports. He reported wearing a mask during the flights. He sought medical attention from a general practitioner on 23 January, where he was suspected of COVID-19, and was subsequently transferred to the regional referring hospital in Bordeaux, isolated and sampled for laboratory confirmation of SARS-CoV-2 infection. Infection was confirmed on 24 January by the National Reference Centre (Figure). Case 1 tested positive only for the E gene target when using the Charit&#233; procedure [4] and was positive for all four RdRp-IP targets with threshold cycles (Ct) in good agreement with those obtained for the E gene target.The patient arrived in Wuhan on 13 January, did not report any visit to markets, exposure to live animals or contact with sick persons during his stay. No detailed information is available about the circumstances of exposure, apart from a visit to family members and friends on 15 January.Case 2 was a 31-year-old Chinese male tourist who had left Wuhan on 18 January and arrived in Paris on 19 January. He developed fever, chills, fatigue, conjunctivitis and cough on 19 January. Case 3 was a 30-year-old Chinese female tourist who travelled with Case 2. She developed fever, chills, fatigue and cough on 23 January. On 24 January, they were advised by the Chinese embassy to seek medical attention at the national hotline (SAMU-centre 15) and were immediately transferred to a regional referring hospital, isolated and sampled for laboratory confirmation of COVID-19. Infection with SARS-CoV-2 was confirmed on 24 January for both of them by the National Reference Centre (Figure). Cases 2 and 3 were positive by RT-PCR for all targets of the Charit&#233; procedure [4] (RdRp Pan Sarbeco and 2019-nCov probes; E; N) as well for the four RdRp-IP targets with Ct values in good agreement with those obtained for the E gene target.The condition of the male patient deteriorated on 29 January and he was admitted to the intensive care unit (ICU) the same day. He stayed 72 h in the ICU for non-invasive oxygen therapy and was transferred back to infectious diseases ward on 31 January.Neither of the two cases reported any visit to markets, exposure to live animals or contact with sick persons during the 14 days before symptom onset. Both visited a hospital in Wuhan on 16 January for an unrelated medical condition in Case 3 (Figure).As at 12 February, Case 1 was afebrile and symptomatic with a persistent cough. Cases 2 and 3 were not symptomatic any more and were discharged from hospital on 12 February.As soon as the infection with SARS-CoV-2 was confirmed for the three cases on 24 January, this information was immediately released through a press conference held by the French Minister of Health and the Chief Medical Officer. Daily public communication on the state of the investigations around the cases was subsequently implemented by the Ministry of Health. Daily updates were also published on the SpF website.The three cases were notified to the European Commission via the Early Warning and Response System (EWRS) on 26 January, and to the European Center for Disease Prevention and Control (ECDC) via the European Surveillance System (TESSy) on 28 January.\nContact and co-exposition tracing\nNo co-exposed person was identified for Case 1. Two contacts were evaluated at low risk of infection, the taxi driver who drove the case from the airport to his home (30-min drive) and the general practitioner who took care of the patient before wearing appropriate personal protection equipment (3-min non-close contact). Seventeen contacts were evaluated at moderate/high risk of infection. Four of them shared the same waiting room in the general practitioner&#8217;s office while Case 1 was coughing, seated ca 1&#8211;1.5&#8201;m away from the case during 5&#8211;30 min. The other 13 contacts were the persons sitting in the two seats around Case 1 in the Shanghai&#8211;Paris and Paris&#8211;Bordeaux flights (Figure). They were considered at moderate risk of exposure despite the fact that Case 1 reported wearing a mask during the whole flight; this was based on the length of one of the flights (>&#8201;6 h) and the fact that it was unclear whether or not Case 1 removed his mask during short periods (e.g. meals) and kept the same mask during the whole flights. None of the contacts of the Shanghai&#8211;Paris flight were French nationals and their contact tracing was referred to their home countries&#8217; health authorities. All other identified contacts were evaluated at negligible risk of infection because the contacts were short and/or distant in public settings and did not imply face-to-face conversations or because appropriate personal protective equipment (PPE) was worn by the healthcare personnel who took care of the patient, including those involved in the transfer from the general practitioner to the referring hospital.Cases 2 and 3 stayed together and shared the same activities during their stay in Paris, and therefore shared the same contacts from 23 January (date of illness onset for Case 3). Three contacts were evaluated at low risk of infection: the two owners of the apartment rented by the couple and a department store employee with whom Case 2 reported a distant (>&#8201;1&#8201;m) contact during around 20 min on 22 January. The apartment owner&#8217;s child who visited Cases 2 and 3 and was hugged by them was evaluated at moderate/high risk of infection (Figure). All other identified contacts were evaluated at negligible risk of infection, as contacts were short and distant in public settings such as department stores and did not imply face-to-face conversations or because appropriate PPE was worn by the healthcare personnel who took care of the patients.Follow-up of the identified contacts was initiated according to the COVID-19 procedure (Table). As at 2 February, two contacts have been classified as possible cases since the implementation of the follow-up: A person sitting two seats away from Case 1 during the Paris&#8211;Bordeaux flight, and therefore identified as a moderate/high risk contact, developed respiratory symptoms on 27 January and was classified as a possible case on 31 January and was subsequently excluded following negative RT-PCR results. Infection with SARS-CoV-2 was excluded on the same day. A radiology assistant who took care of both Cases 2 and 3 developed respiratory symptoms on 30 January and was classified as a possible case on 2 February. This person had been classified as at negligible risk of exposure, because she wore appropriate PPE during the whole procedure. Infection with SARS-CoV-2 was excluded on 2 February.Follow-up of the contacts ended on 6 February. No identified contact of the three cases has been confirmed with COVID-19.\nDiscussion\nSpecific COVID-19 surveillance has been in place in France since 10 January 2020, 3 days after the identification of the SARS-CoV-2 in China. The first three imported cases of COVID-19 in France, the first ones in Europe, were diagnosed 14 days later, on 24 January. Rapid and effective collaboration between the clinicians (general practitioners attending the cases, emergency hotline clinicians (SAMU-centre 15) and infectious diseases specialists), the National Reference Centre and the regional and national health authorities has played a crucial role in the system&#8217;s capacity to quickly detect, isolate and investigate those cases in order to implement adequate control measures. The surveillance system as well as the control measures were adapted from those implemented during past emerging infections that occurred after 2003 (severe acute respiratory syndrome (SARS), MERS, influenza A(H1N1)pdm09, Ebolavirus disease), and all involved parties were already familiar with the system, which probably favoured its responsiveness.The case definition of a possible case in use on 24 January was slightly adapted from the one provided by the World Health Organization (WHO), based on an epidemiological link to Wuhan, China and a severe lower acute respiratory disease. It is noteworthy that the first nine possible cases identified in France, including the three confirmed cases described here, displayed mild respiratory symptoms with no sign of severity at the time of diagnosis. Increasing evidence suggests that mild clinical symptoms could be more frequent in cases of COVID-19 than with SARS-CoV and MERS-CoV [5]. Therefore, the case definition in effect on 24 January lacked sensitivity. This was counter-balanced by a tendency from the infectious diseases specialists in charge of classification of suspected cases to privilege the exposure to Wuhan over the clinical presentation in their decision. However, we cannot exclude that some COVID-19 cases remained undetected in France because of the lack of sensitivity of our case definition. The clinical criteria were expanded on 4 February to include any lower acute respiratory disease and the epidemiological criterion was extended to the whole of China. At that time, the French laboratory capacities were reinforced from one to five laboratories able to perform the diagnostics for COVID-19. Further extension to all 38 referring hospital laboratories is expected by early to mid-February 2020. Sant&#233; publique France will deploy in early February the outbreak investigation tool developed by the WHO (Go.Data [6]) in order to facilitate case data management and contact tracing at the national and local level in France.Contact and co-exposure identification of the three confirmed cases had been initiated as soon as they were classified as possible cases, which facilitated investigations upon confirmation of COVID-19. Confirmation of the diagnosis was made in the evening of 24 January and the investigation to retrieve as exhaustively as possible contacts and co-exposed individuals and evaluate their level of risk of transmission was started immediately overnight. Complete transparency of the investigations was ensured through daily press conferences held by the French health authorities.Although the follow-up procedure for the contacts/co-exposed persons used in France slightly differ from the ECDC and WHO guidelines [7,8], which were not available at the time of this investigation, it relies on the same general principles. Contact tracing of the passengers seated near Case 1 during the two flights Shanghai&#8211;Paris and Paris&#8211;Bordeaux was adapted from the ECDC guidelines for infectious diseases transmitted on aircraft [9]. Even though Case 1 was wearing a face mask during those flights, we could not exclude breaches and subsequent risk of transmission to the persons sitting in the two seats around him.Because of the current uncertainties about the capacity of SARS-CoV-2 to easily spread from human to human, the decision to consider a contact as close if the case&#8211;contact distance was between 1&#8239;m and 2&#8239;m was made on a case-by-case basis, depending on the type and length of interaction. Through the extensive interviews made with the cases and their high compliance to cooperate to the investigation, we believe that the contacts most at risk have been satisfactorily identified. All of them could be rapidly contacted and informed about measures to be taken, which they all agreed to. However, some contacts were either impossible to trace back (e.g. co-travellers on public transportation) or evaluated as at negligible risk of exposure because of short and/or distant contacts (e.g. restaurant, contacts with cashiers while running errands, visiting museums), although accidental events carrying the risk of transmission on such occasions, such as an episode of cough of sneezing, cannot be ruled out.Moreover, the contact tracing was limited to the period after onset of illness. However, should the transmission of SARS-CoV-2 occur during the asymptomatic phase, we cannot exclude that secondary transmission events initiated from the three confirmed cases remained undetected during the investigations.Case 3 developed symptoms 4 days after her husband and 5 days after the couple had left Wuhan. The incubation period of SARS-CoV-2 is currently estimated at around 3&#8211;7 days [5,10,11]. Therefore, she may have acquired the infection from her husband, although this cannot be proved.The active surveillance of close contacts of confirmed COVID-19 cases and the implementation of control measures, including home quarantine for those evaluated at moderate/high risk of exposure, decrease the risk of human-to-human transmission originating from imported cases and subsequently delay propagation of the virus in the general population. This allows our healthcare system to prepare for any further spread of the epidemic. Besides, the epidemiological and clinical data collected about the confirmed cases and their contacts will increase our knowledge of COVID-19.The rapid and collaborative management of the first imported COVID-19 cases in France highlights the fact that the French healthcare system is adequately prepared to respond to such emerging diseases threats. However, this surveillance system is extremely time-consuming and requires considerable manpower. The data available on 12 February strongly suggest that human-to-human transmission of SARS-CoV-2 is frequent, with the reproduction number estimated at 2 to 3 [5,10-14]. Twenty-five countries have already reported imported cases from China, and several of them have described autochthonous transmission events [15]. In the case of further spread of SARS-CoV-2 worldwide, it would soon become impossible to detect all imported cases and trace their contacts. Especially the occurrence of large clusters in the same region would strongly impact on the local health authorities&#8217; capacities. The surveillance objectives would then need to evolve from containing the epidemic to mitigating its medical and societal impact.As at 12 February, the contacts of the three first confirmed cases of COVID-19 in France have been followed up for the whole 14 days follow-up time after the cases isolation. No secondary transmission event has been detected so far despite active follow-up. Given the first estimations of the SARS-CoV-2 incubation period, the probability of secondary cases originating from those three cases is negligible.",
        "pmid": "32070465",
        "title": "First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020",
        "journal_title": "Eurosurveillance",
        "authors": "Sibylle Bernard Stoecklin;Patrick Rolland;Yassoungo Silue;Alexandra Mailles;Christine Campese;Anne Simondon;Matthieu Mechain;Laure Meurice;Mathieu Nguyen;Cl\u00e9ment Bassi;Estelle Yamani;Sylvie Behillil;Sophie Ismael;Duc Nguyen;Denis Malvy;Fran\u00e7ois Xavier Lescure;Scarlett Georges;Cl\u00e9ment Lazarus;Anouk Taba\u00ef;Morgane Stempfelet;Vincent Enouf;Bruno Coignard;Daniel Levy-Bruhl",
        "doi": "10.2807/1560-7917.ES.2020.25.6.2000094"
    },
    {
        "keywords": "",
        "abstract": "Similar to eukaryotic mRNA, the positive-strand coronavirus genome of ~30 kilobases is 5&#8217;-capped and 3&#8217;-polyadenylated. It has been demonstrated that the length of the coronaviral poly(A) tail is not static but regulated during infection; however, little is known regarding the factors involved in coronaviral polyadenylation and its regulation. Here, we show that during infection, the level of coronavirus poly(A) tail lengthening depends on the initial length upon infection and that the minimum length to initiate lengthening may lie between 5 and 9 nucleotides. By mutagenesis analysis, it was found that (i) the hexamer AGUAAA and poly(A) tail are two important elements responsible for synthesis of the coronavirus poly(A) tail and may function in concert to accomplish polyadenylation and (ii) the function of the hexamer AGUAAA in coronaviral polyadenylation is position dependent. Based on these findings, we propose a process for how the coronaviral poly(A) tail is synthesized and undergoes variation. Our results provide the first genetic evidence to gain insight into coronaviral polyadenylation.",
        "body": "Introduction\nPosttranscriptional modifications occurring in the nucleus of eukaryotic cells include cleavage of the 3&#8242; end of nascent mRNAs and the addition of a poly(A) tail [1&#8211;5]. The polyadenylation process involves two discrete phases [6]. In the first phase, synthesis of a short poly(A) tail of nearly 10 nucleotides (nts) depends on interaction between polyadenylation-related proteins and the polyadenylation signal (PAS) hexamer AAUAAA or its variant (AGUAAA, AUUAAA or UAUAAA) located 10&#8211;30 nts upstream of the poly(A) cleavage site [1, 7&#8211;13]. The rapid addition of a poly(A) tail of nearly 200 nts that occurs in the second phase requires the nearly 10 adenosine residues synthesized in the first phase. The synthesized poly(A) tail is important for the nuclear export of mature mRNAs and has been demonstrated to be involved in the control of mRNA stability and translation efficiency [14&#8211;17].As opposed to mRNAs used only for translation, polyadenylation of viral RNA in RNA viruses may be involved in both translation and replication [16, 18]. RNA viruses have developed several mechanisms for synthesizing a poly(A) tail based on genetic features. It has been demonstrated that influenza virus utilizes a stretch of short U residues, instead of the hexamer AAUAAA, located at the 5&#8217; terminus of the negative-strand genomic RNA as a signal for poly(A) synthesis by the viral RNA polymerase with a stuttering mechanism during positive-strand synthesis [19&#8211;21]. A similar mechanism is also used by paramyxoviruses to generate a poly(A) tail during transcription [22]. On the other hand, poliovirus uses homopolymeric stretch on negative-strand as template for the addition of poly(A) tail during positive-strand synthesis [23]. Moreover, the cis-acting element required for replication may also be used for polyadenylation in RNA viruses. For example, the hexamer AAUAAA in bamboo mosaic virus and a domain immediately upstream of the poly(A) tail in coxsackievirus B3 have been shown to function as cis-acting elements involved in both negative-strand RNA synthesis and polyadenylation [24, 25].Bovine coronavirus (BCoV), a betacoronavirus, subfamily Coronavirinae, family Coronaviridae and order Nidovirales, is a 5&#8217;-capped and 3&#8217;-polyadenylated positive-strand RNA virus. Although the mechanism for coronaviral polyadenylation remains unknown, a stuttering mechanism based on the short poly(U) stretch found in the negative-strand genome has been postulated [26]. Moreover, a regulated poly(A) tail length during the coronaviruses life cycle has been suggested, whereby the viral poly(A) tail length is increased in the early stage of infection but gradually decreases after the peak tail length (~65 nts) in the later stage of infection in both cell culture and animals [16, 27]. Such regulated poly(A) tail length may function in translation regulation, as it has been experimentally demonstrated that a longer coronavirus poly(A) tail is associated with better translation efficiency [16]. However, the mechanism by which the coronaviral poly(A) tail is regulated remains unclear.In the current study, we determined that both the poly(A) tail and hexamer AGUAAA are important elements responsible for the polyadenylation of coronavirus. The efficiency of poly(A) tail elongation during infection depends on the initial poly(A) tail length at the time of infection. Based on these findings, we propose a process for how the coronaviral poly(A) tail is synthesized and undergoes variation. The results presented here provide the first genetic evidence that will help in elucidating coronaviral polyadenylation.\nMaterials and Methods\n\nViruses and cell line\nA plaque-purified Mebus strain of bovine coronavirus (BCoV) (from Dr. David A. Brian, University of Tennessee, USA) (GenBank accession no. U00735) at 3 &#215; 107 PFU/ml was used as a helper virus throughout the study. Human rectal tumor (HRT)-18 cells (from Dr. David A. Brian, University of Tennessee, USA) were maintained in DMEM supplemented with 10% fetal bovine serum and used for BCoV infection as described [28&#8211;30].\nPlasmid constructs\nConstruction of defective interfering (DI) RNA pW-25A (formerly called pBM25A) (S1A Fig, lower panel) in which the 288-nt 3&#8217; UTR of BCoV-Mebus in BCoV DI RNA pDrep1 (S1A Fig, upper panel) was replaced with the 301-nt 3&#8217; UTR and 25-nt poly(A) tail of mouse hepatitis virus (MHV)-A59 (GenBank accession no. NC_001846) has been described [16, 31, 32]. To construct pR-25A, an overlap PCR mutagenesis procedure was performed as previously described [31, 33, 34], but using oligonucleotides TGEV 7(&#8722;) and R(+) and pW-25A DNA in the first PCR, oligonucleotides R(&#8722;) and BM25A(+) and pW-25A DNA in the second PCR, and oligonucleotides TGEV 7(&#8722;) and BM25A(+) and the products of the first two reactions in the third PCR. The resulting PCR product was cloned into the TOPO XL vector (Invitrogen) and digested with SpeI and MluI. The digested fragment was then cloned into SpeI- and MluI-linearized pW-25A to generate the mutant pR-25A. Mutants of pM1-25A, pM2-25A, pM3-25A, pM4-25A, pM5-25A, pPAS-R-25A and pPAS-PAS-25A were similarly constructed, except for the corresponding oligonucleotides used in the first and second reactions, as described in S1 Table.To generate the constructs pW-0A, pW-5A, pW-8A, pW-12A, pW-15A, pW-18, pW-20A, pW(C)-25A, pW-25U, pW-25C, pW-25G, pW-random and pW-polyCC, PCR was performed using pW-25A DNA as the template with the oligonucleotide TGEV 7(-) and the appropriate oligonucleotide that binds to the terminal sequence of the 3&#8217; UTR, as described in S1 Table, to create DI RNA constructs with various patterns of sequences at their 3&#8217; ends. Each PCR product was cloned into the TOPO-XL vector (Invitrogen) and digested with SpeI and MluI, and the digested fragments were cloned into SpeI- and MluI-linearized pW-25A to create the aforementioned constructs. For constructs pR-5A, pR-8A, pR-12A, pR-15A, pR-18, pR-20A, pR(C)-25A, pR-25U, pR-25C, and pR-25G, PCR was also performed using a similar method but with pR-25A DNA as the template. This strategy was also applied to generate mutants pM1-15A, pM2-15A, pM3-15A, pM4-15A, pM5-15A, pPAS-R-15A and pPAS-PAS-15A, but the DNA templates for these mutants were pM1-25A, pM2-25A, pM3-25A, pM4-25A, pM5-25A, pPAS-R-25A and pPAS-PAS-25A, respectively, and the oligonucleotides were TGEV 7(-) and BM15A(+).\nNone\nTo synthesize transcripts in vitro, all DNA constructs (except W-0A, W-25U, W-25C, W25G, R-25U, R-25C and R-25G, which were linearized with BsmBI to accurately synthesize transcript with no poly(A) tail, or with only poly (U), poly(C) or poly(G) tail) were linearized with MluI. The linearized DNA was transcribed in vitro with the mMessage mMachine T7 transcription kit (Ambion) according to the manufacturer's instructions and passed through a Biospin 6 column (Bio-Rad), followed by transfection [35]. For transfection, HRT-18 cells in 35-mm dishes at ~80% confluency (~8 &#215; 105 cells/dish) were infected with BCoV at a multiplicity of infection of 5 PFU per cell. After 2 hours of infection, 3 &#956;g of transcript was transfected into mock-infected or BCoV-infected HRT-18 cells using Lipofectine (Invitrogen) [31, 36].\nPreparation of RNA from infected cells\nTo prepare RNA for the identification of DI RNA poly(A) tail length, RNA was extracted with TRIzol (Invitrogen) at the indicated times after transfection of DI RNA constructs into BCoV-infected HRT-18 cells; the virus within the transfected cells is referred to as virus passage 0 (VP0) (S1B Fig). Supernatants from BCoV-infected and DI RNA transfected HRT-18 cells at 48 hours posttransfection (hpt) (VP0) were collected, and 500 &#956;l was used to infect freshly confluent HRT-18 cells in a 35-mm dish (virus passage 1, VP1) (S1B Fig). RNA was extracted with TRIzol (Invitrogen) at the indicated time points.\nDetermination of poly(A) tail length\nAmong the PCR-based methods for the determination of poly(A) tail length [37&#8211;41], a head-to-tail ligation method using tobacco acid pyrophosphatase (TAP) and RNA ligase followed by RT-PCR and sequencing was employed in this study (S1C Fig). This method has been previously used to identify the terminal features of histone mRNA [42] and influenza virus [43] as well as the poly(A) tail length of cellular mRNAs [44] and coronavirus RNAs [16, 27]. In brief, 10 &#956;g of extracted total cellular RNA in 25 &#956;l of water, 3 &#956;l of 10X buffer and 10 U of (in 1 &#956;l) TAP (Epicentre) were used to de-block the 5&#8242; capped end of genomic RNA. Following decapping, RNA was phenol-chloroform-extracted, dissolved in 25 &#956;l of water, heat-denatured at 95&#176;C for 5 min and quick-cooled. Head-to-tail ligation was then performed by adding 3 &#956;l of 10X ligase buffer and 2 U (in 2 &#956;l) of T4 RNA ligase I (New England Biolabs); the mixture was incubated for 16 h at 16&#176;C. The ligated RNA was phenol-chloroform-extracted and used for the RT reaction. SuperScript II reverse transcriptase (Invitrogen), which is able to transcribe poly(A) tails greater than 100 nts with fidelity [19, 45], was used for the RT reaction with oligonucleotide BCV29-54(+), which binds to nts 29&#8211;54 of leader sequence of the 5&#8217; UTR of the BCoV positive strand, as previously described. A 5-&#956;l aliquot of the resulting cDNA was used in a 50-&#956;l PCR with AccuPrime Taq DNA polymerase (Invitrogen) and oligonucleotides BCV29-54(+) and MHV3UTR3(-), the latter of which binds to nts 99&#8211;122 counted from the poly(U) track on the MHVA59 negative strand. The resulting PCR product was subjected to sequencing to determine the poly(A) tail length. At least three independent experiments were carried out for determining the poly(A) tail length of DI RNA mutants.\nNorthern blot analysis\nDetection of the reporter-containing DI RNA was performed essentially as described previously [35, 36]. In brief, HRT-18 cells were seeded in 35-mm dishes at ~80% confluency (~8 &#215; 105 cells/dish). For RNA stability assay, 3 &#956;g of DI RNA transcript was transfected into the HRT-18 cells. HRT-cells were incubated with RNase A (final concentration 0.5 mg/ml) for 15 min prior to extraction of cellular RNA. For a replication assay, after 2 h of infection with BCoV at a multiplicity of infection of 5 PFU per cell, 3 &#956;g of DI RNA transcript was transfected into the BCoV-infected HRT-18 cells. Tthe supernatant was harvested at 48 hpt (VP0), and 500 &#956;l was used to infect freshly confluent HRT-18 cells in a 35-mm dish (VP1). Total cellular RNA was extracted with TRIzol (Invitrogen) at 48 hpi of VP1, and 10 &#956;g was electrophoresed through a formaldehyde-agarose gel. The RNA was transferred from the gel to a Nytran membrane by vacuum blotting, and the blot was probed with 5&#8217;-end 32P-labeled oligonucleotide TGEV8(+) for 16 h. The probed blot was washed and autoradiographed at -80&#176;C for 24 h.\nQuantitation of DI RNA synthesis by qRT-PCR\nTo determine the replication efficiency of DI RNA, the cDNA prepared as described above from head-to-tail ligation RNA collected at 1 and 24 hpi of VP1 was used for real-time PCR amplification with TagMan&#174;Universal PCR Master Mix (Applied Biosystems) using primers MHV3&#8217;UTR3(-) and BCV23-40(+). The real-time PCR amplification was performed according to the manufacturer&#8217;s recommendations in a LightCycler&#174; 480 instrument (Roche Applied Science).\nResults\n\nDetermination of the minimum length of poly(A) tail required to initiate poly(A) tail lengthening of coronavirus defective interfering (DI) RNA\nIn a previous study, we demonstrated that the length of the coronaviral poly(A) tail on both viral RNAs and DI RNA is regulated during infection; that is, the poly(A) tail length is increased in the early stage of infection and then decreased after the peak tail length in the later stage of infection [16, 27]. To test whether the increase in coronaviral poly(A) tail length requires a minimum tail length in the initial viral RNA, as with the requirement of nearly 10 adenosine residues for the second phase of polyadenylation in eukaryotic mRNA, a series of bovine coronavirus (BCoV) DI RNAs with various poly(A) tail lengths were constructed and tested (Fig 1A). The 2.2-kb and helper virus-dependent BCoV DI RNA (S1A Fig, upper panel) is a naturally occurring DI RNA [36, 46], and it has been extensively exploited for analyzing the cis-acting elements required for replication in coronaviruses [16, 33, 35, 36, 47&#8211;49]. To differentiate the origin of the poly(A) tail between the helper virus BCoV genome and BCoV DI RNA, the latter was engineered to carry the mouse hepatitis virus (MHV) 3&#8217; UTR (S1A Fig, lower panel) [16, 31, 32] with which an MHV-specific primer can be used for RT-PCR to determine the length of the DI RNA poly(A) tail [16, 27]. It should be noted that both BCoV and MHV-A59 belong to the genus betacoronavirus and that the replication efficiency of this MHV 3&#8217; UTR-containing BCoV DI RNA is similar to that of wild-type BCoV DI RNA [32]. After transfection of DI RNA constructs into BCoV-infected HRT-18 cells, the virus within the transfected cells is referred to as virus passage 0 (VP0) (S1B Fig). Supernatants from BCoV-infected and DI RNA transfected HRT-18 cells at 48 hours posttransfection (hpt) (VP0) were collected, and 500 &#956;l was used to infect freshly confluent HRT-18 cells in a 35-mm dish (virus passage 1, VP1) (S1B Fig). After RNA extraction and head-to-tail ligation (S1C Fig), RT-PCR products with the size of less than 200 bp (S2 Fig), which are expected to contain the sequence from 3&#8217; UTR, poly(A) tail and 5&#8217;UTR of DI RNA (S1C Fig) [16], were detected and subjected to sequencing to determine the poly(A) tail length. The RT-PCR products with the size of less than 100 bp were also sequenced and were determined to be primer-dimer. Note that the amounts of DI RNAs in cells were various at different time points of infection and thus various cycles of PCR were applied to amplify sufficient amounts of products for sequencing. Accordingly, the intensity of RT-PCR product shown in S2 Fig may not represent the replication efficiency of the DI RNAs. As shown in Fig 1B, the tail length of DI RNAs W-0A and W-5A remained the same (0 and 5 nts, respectively) throughout infection. To ensure that the identified poly(A) tail is from the replicating DI RNA rather than input DI RNA, qRT-PCR was applied for the evaluation of the replication efficiency for DI RNAs W-0A and W-5A at different time points of VP1. As shown in the upper panels of S3A and S3B Fig, the intensity of RT-PCR product was increased with the time of infection in VP1, suggesting that these DI RNAs is able to replicate. The results were subsequently confirmed by qRT-PCR (S3A and S3B Fig, lower panels) and thus suggest that the detected poly(A) tails of W-0A and W-5A were not from input DI RNAs but from replicating DI RNAs. Conversely, the poly(A) tail length of DI RNA W-15A (with an initial poly(A) tail length of 15 nts) was decreased (8 nts) at 48 hpt (VP0), gradually increased (10 and 13 nts, respectively) at 8 and 24 hours postinfection (hpi) of VP1 and then decreased (11 nts) at 48 hpi of VP1. Although the poly(A) tail length of DI RNA W-20A (with an initial poly(A) tail length of 20 nts) was also shortened (19 nts) at 48 hpt (VP0), the tail length was increased (22 nts) at 8 hpi of VP1 and then gradually decreased (19 nts and 15 nts at 24 and 48 hpi of VP1, respectively). Similar results were obtained for DI RNA W-25A (with an initial poly(A) tail length of 25 nts), whereby the poly(A) tail length was also shorter (21 nts) than the initial length at 48 hpt (VP0) but increased to 31 nts at 8 hpi of VP1 and then gradually decreased to 24 nts and 19 nts at 24 and 48 hpi of VP1, respectively. Because it has been shown that the poly(A) tail length of BCoV DI RNA within infected cells at 48 hpt (VP0) is similar to that of packaged BCoV DI RNA in inoculum collected at the same time point [16], the poly(A) tail length of DI RNA in infected cells may represent that in inoculum at 48 hpt (VP0). Under this criterion and based on the results that (i) the coronaviral poly(A) tail length of DI RNA W-15A increased from 8 nts at 48 hpt (VP0) to 10 nts at 8 hpi of VP1 and (ii) the tail length of W-0A and W-5A remained the same during VP0 and VP1, we conclude that the minimum poly(A) tail length required to initiate tail lengthening of coronavirus DI RNA may lie between 5 and 9 nts during the natural infection of VP1, regardless of the length of the input DI RNA transcript. Moreover, the level of lengthening was found to be correlated to the initial length of the poly(A) tail; that is, DI RNA with a longer poly(A) tail (for example, W-25A) showed a better lengthening than that with a shorter poly(A) tail (for example, W-15A), as evidenced by the comparison of poly(A) tail lengths synthesized at 48h of VP0 and 8h of VP1 for these DI RNA constructs during infection (Fig 1B).\nEffect of hexamer AGUAAA and poly(A) tail length on the efficiency of coronaviral polyadenylation\nInteractions between consensus polyadenylation signal (PAS) hexamer AAUAAA or its variant (AGUAAA, AUUAAA or UAUAAA) located10&#8211;30 nts upstream of the poly(A) cleavage site [5, 10] and related proteins are integral aspects of eukaryotic mRNA polyadenylation [6, 50]. Although we in the previous study (Fig 1) have identified the minimum requirement to initiate lengthening of the coronaviral poly(A) tail during infection, little is known with regard to the detailed mechanism of how poly(A) tails are synthesized in coronaviruses. As with most eukaryotic mRNAs, both the coronavirus genome and subgenomic mRNAs are 3&#8217; polyadenylated. Moreover, the PAS hexamer AGUAAA is also found in the 3&#8217; UTR of genome and subgenomic mRNAs in BCoV and MHV-A59 between 37 and 42 nts upstream of the poly(A) site. To determine whether the hexamer AGUAAA, as with the eukaryotic PAS, serves as a cis-element involved in coronaviral polyadenylation, the hexamer in BCoV DI RNA W-25A was mutated from AGUAAA to UCAUUU; the resulting DI RNA was designated R-25A (Fig 2A). RNA was collected at 24 hpi of VP1, and the RT-PCR product was detected and subjected to sequencing analysis. As shown in Fig 2B, the lengths of the W-25A and R-25A poly(A) tails were 24 and 22 nts, respectively, suggesting that the hexamer AGUAAA only had a minor effect on coronavirus polyadenylation when the tail length of the input R-25A was 25 nts. However, under the similar RNA stability (Fig 2C), the replication efficiency of R-25A was impaired in comparison with that of W-25A, as determined by Northern blot analysis (36% vs 100%) (Fig 2D), suggesting that the replication efficiency may not be a major factor determining the poly(A) tail length. Besides, to exclude the possibility that the detected poly(A) tail is from the potential recombination between the DI RNA and BCoV genome, the primer MHV3&#8217;UTR2(+), which anneals to the 3&#8217; UTR of DI RNA and primer BM3(-),which anneals to the BCoV M protein gene were used for RT-PCR to identify the potential recombinant [31, 32, 51]. However, no RT-PCR product was observed (Fig 2E, lanes 2&#8211;3), suggesting there is no potential DI RNA-BCoV genome recombinant synthesized during infection. Furthermore, because the last 21 nts of 3&#8217; UTR between DI RNA and BCoV genome of helper virus are identical [52], it is also possible that the synthesized poly(A) tail may originate from genome of the helper virus BCoV via homologous recombination in this region either during negative- or positive-strand RNA synthesis. To test this possibility, the nt A at the position 2 upstream of poly(A) tail in DI RNAs R-25A and W-25A was mutated to C to create R(C)-25A and W(C)-25A as shown in Fig 2F, upper panel. As shown in Fig 2F, lower panel, the mutated nt C was still maintained at 24 hpi of VP1, suggesting that there is no homologous recombination between helper virus and DI RNA in this region and thus the detected poly(A) tail on DI RNAs R-25A and W-25A may not acquire from a potential DI RNA-BCoV genome recombination. It is noteworthy that the UCAUUU sequence in DI RNA mutant R-25A did not revert back to the wild-type AGUAAA at 24 hpi of VP1 (data not shown). Accordingly, the RNA stability (Fig 2C), replication efficiency (Fig 2D) and recombination between DI RNA and BCoV genome (Fig 2E and 2F) may not be the major factors in determining the poly(A) tail length on R-25A or W-25A.Since the hexamer AGUAAA may not be required for coronaviral polyadenylation when the poly(A) tail on input DI RNA is 25 nts long, we hypothesize that the AUGAAA hexamer may be critical when the tail length is short. That is, it may be possible that both the hexamer AGUAAA and poly(A) tail may contribute to coronaviral polyadenylation through concerted action when the tail is at a certain length shorter than 25 nts on input DI RNA. This hypothesis is based on the results that (i) a poly(A) tail was not synthesized for W-0A (Fig 1B), even though this DI RNA construct contains a hexamer AGUAAA, and (ii) the level of poly(A) tail synthesis for hexamer AGUAAA-deficient R-25A was similar to that for hexamer AGUAAA-containing W-25A (Fig 2B). Therefore, to test our hypotheses and further elucidate the role of the AGUAAA hexamer in coronaviral polyadenylation, we created a series of DI RNA constructs in which the hexamer was substituted with UCAUUU and with various poly(A) tail lengths (Fig 3A, left panel) or the hexamer was intact but with various poly(A) tail lengths (Fig 3A, right panel). According to previous study, the W-65A (with 65 nts of poly(A) tail), which is structurally the same as W-25A (with 25 nts of poly(A) tail) except poly(A) tail length, is almost not detected at 1 and 2 hpi of VP1 but is steadily identified in the later stage (e.g. 24 hpi) of infection using head-to-tail ligation and RT-PCR [16] under the same amplification condition, suggesting the detected DI RNA in the later infection is newly synthesized; therefore, to ensure that the detected DI RNA is not from the input DI RNA which may be carried over by supernatant of VP0 but from the replicating DI RNA, total cellular RNA was collected at 24 hpi of VP1. As shown in Fig 3B (for uncropped gel images, see S4 Fig), RT-PCR products were observed for W-5A- or W-8A-transfected BCoV-infected cells at the same time point with poly(A) tail lengths of 5 and 8 nts, respectively (Fig 3C). Although RT-PCR products were observed for AGUAAA-deficient R-12A, no clear poly(A) tail was identified; instead, sequencing analysis revealed a mixed population at the 3&#8217;-terminal end. Nonetheless, an RT-PCR product was detected for W-12A, and after sequencing, the poly(A) tail length of W-12A was found to be 9 nts (Fig 3C). Interestingly, an RT-PCR product was also observed for AGUAAA-deficient R-15A, but subsequent sequencing revealed a poly(A) tail length of 3 nts (Fig 3C). On the other hand, the length of the W-15A poly(A) tail was determined to be 13 nts (Fig 3C). For R-18A, R-20A and R-25A, the poly(A) tail lengths were 10, 18, and 22 nts, respectively, whereas those for W-18A, W-20A and W-25A were 17, 19 and 24 nts, respectively (Fig 3C). Based on a comparison of synthesized poly(A) tail lengths between R-5A and W-5A, R-8A and W-8A, R-12A and W-12A, R-15A and W-15A, and R-18A and W-18A (Fig 3C), the poly(A) tail in AGUAAA-deficient DI RNA is shorter than that in AGUAAA-containing DI RNA. Accordingly, it was concluded that when the poly(A) tail length for the input DI RNA transcript is 18 nts or less, the hexamer AGUAAA is required for coronaviral polyadenylation. However, according to the results for R-20A, W-20A, R-25A and W-25A (Fig 3C), once the poly(A) tail length for the input DI RNA transcript reached 20 nts, the synthesized poly(A) tail length for the AGUAAA-deficient DI RNA was similar to that of AGUAAA-containing DI RNA. Thus, it was concluded that the hexamer AGUAAA is not required for polyadenylation when the tail length on input DI RNA transcript is 20 nts (for example, R-20A) or more (for example, R-25A). Because the poly(A) tail length of DI RNA became varied after transfection (Fig 1B) and the DI RNA poly(A) tail length at 48 hpt (VP0) in infected cells was similar to that of packaged DI RNA in inoculum [16], we also applied RT-PCR and sequencing to identify the poly(A) tail length of R-20A at 48 hpt (VP0). The length of poly(A) tail for R-20A was determined to be 18 nts at 48 hpt (VP0) (data not shown) and therefore it was also concluded that when the initial poly(A) tail length for coronavirus genome is 18 nts or more, coronaviral polyadenylation during natural infection is independent of hexamer AGUAAA. Taken together, the results suggest that (i) the poly(A) tail length plays an important role in the efficiency of coronaviral polyadenylation and (ii) the hexamer AGUAAA is also involved in coronaviral polyadenylation and may function in concert with the poly(A) tail to accomplish the subsequent polyadenylation when the initial length of poly(A) tail is shorter than 18 nts. This conclusion, therefore, supports our hypothesis.To further determine whether, besides hexamer AGUAAA and poly(A) tail length, the DI RNA stability, replication efficiency and recombination are also factors affecting the synthesis of poly(A) tail on DI RNA, DI RNAs W-25A, R-5A, R-15A and R-20A with various length of synthesized poly(A) tail (24, 0, 3 and 18 nts, respectively) at 24 hpi of VP1 (Fig 3C) were selected and tested. As shown in Fig 3D, the stability of selected DI RNA variants is almost the same, suggesting the stability may not the main determinant affecting polyadenylation. For the factor of replication efficiency, DI RNA was not detectable by Northern blot assay from R-15A and R-20A, suggesting that the replication efficiency for both constructs is low (Fig 3E); however, the poly(A) tail length for both constructs was different (3 and 18 nts, respectively) (Fig 3C), also suggesting replication efficiency may not play a major role in poly(A) tail synthesis (see Discussion for more details). In addition, the unidentified poly(A) tail from R-5A may be attributed to the undetected RT-PCR product due to the poor replication efficiency. For the factor of recombination, no RT-PCR product was observed (Fig 3F, lanes 2&#8211;5) using the primers specifically binding to DI RNA and BCoV genome as described above (Fig 2E), suggesting that the detected poly(A) tail on the selected DI RNAs may not acquire from a potential DI RNA-BCoV genome recombinant. Furthermore, given that the poly(A) tail in DI RNAs originates from the genome or subgenome of the helper virus BCoV, the length in DI RNAs is expected to be longer than 40 nts because poly(A) tail in genome or subgenome of the helper virus BCoV at 8, 24 and 48 hpi is ~68, ~50 and ~40 nts, respectively [16]. However, the lengths of poly(A) tail in all DI RNAs used in the current study at these time points of infection are all shorter than 40 nts (the longest length is ~31 nts in W-25A at 8 hpi of VP1, Fig 1B). Therefore, consistent with the results shown in Fig 2 for R-25A and W-25A, DI RNA stability, replication efficiency and recombination may not the major factors in determining the synthesis of poly(A) tail on DI RNA.\nSynthesis of a poly(A) tail from poly(A) tail-lacking DI RNA\nTo test further the role of hexamer AGUAAA and poly(A) tail in coronaviral polyadenylation, 25 nts of the W-25A poly(A) tail was first replaced with the same length of a poly(U), poly(C) or poly(G) tail to create mutant W-25U, W-25C or W-25G, respectively (Fig 4A). Moreover, 25 nts of the W-25A poly(A) tail were also substituted with random sequences to generate mutants W-random and W-polyCC (Fig 4A). These mutants were then transfected into BCoV-infected cells to examine whether a poly(A) tail is synthesized and where on the DI RNA it is added if the tail is identified. As shown in Fig 4A, poly(A) tails with lengths of 18 and 21 nts were found on W-25U and W-25C. Moreover, the position where the tail was added was not at the 3&#8217; terminus of the poly(U) or poly(C) tail but at the 3&#8217; terminus of the 3&#8217; UTR. For W-25G, a poly(A) tail was not found until 48 hpi of VP1, with an 8-nt tail also added to the 3&#8217; terminus of the 3&#8217; UTR. For W-random and W-polyCC, RT-PCR products were detected but no poly(A) tail or clear sequence of 3&#8217; UTR was identified during VP1 infection. Although the mechanism remains to be elucidated, it was surprising that the poly(A) tail can be synthesized from DI RNA without poly(A) tail sequence such as W-25U, W-25C and W-25G. Regardless, since the poly(A) tail can be synthesized from DI RNA constructs W-25U, W-25C and W-25G, which contain a hexamer AGUAAA, they may be suitable candidates for further determining the requirement of the hexamer AGUAAA in poly(A) tail synthesis. That is, if the hexamer AGUAAA is mutated, lack of poly(A) tail synthesis for these hexamer AGUAAA-deficient constructs may reinforce the important role of the hexamer in coronaviral polyadenylation. To this end, the AGUAAA in W-25U, W-25C and W-25G was substituted with UCAUUU to create mutants R-25U, R-25C and R-25G. As shown in Fig 4B, the poly(A) tail was not found in R-25G because the RT-PCR product was not obtained. In this case, it was speculated that the poor replication efficiency may be the reason which leads to the result. However, when compared with W-25U and W-25C, a poly(A) tail was not synthesized for R-25U and R-25C at 24 and 48 hpi of VP1, further emphasizing the significant role of the AGUAAA hexamer in coronaviral polyadenylation.\nDissection of hexamer AGUAAA by mutagenesis to further determine its role in coronavirus polyadenylation\nTo further characterize the role of hexamer AGUAAA in coronaviral polyadenylation, the hexamer was dissected by mutagenesis (Fig 5A), and the effect on poly(A) tail synthesis was evaluated. RT-PCR products were detected for W-15A and other W-15A-derived mutants at 24 hpi of VP1, as displayed in Fig 5B, left panel. Sequencing analysis revealed that 1 (M1-15A) or 2 (M2-15A) substitutions from the 3&#8217; end of the hexamer did not alter the efficiency of poly(A) tail synthesis, and the resulting length (13 nts) for these two mutants was the same as that for W-15A, which has an intact AGUAAA motif. However, decreased poly(A) tail synthesis efficiency was found for M3-15A (8 nts) and M4-15A (10 nts), with 3 and 4 substitutions, respectively, from the 3&#8217; end of the hexamer. A severe impact on poly(A) tail synthesis occurred for M5-15A, in which 5 nts were substituted from the 3&#8217; end of the hexamer, and the resulting poly(A) tail length was 3 nts, the same as that obtained for R-15A in which the entire hexamer was substituted. Therefore, the length of the poly(A) tail gradually decreases with the increased mutations within the hexamer. Based on the results that these DI RNAs were able to replicate (S5B Fig) and still retained the mutated hexamer sequence (S5C Fig), these results further support our argument that the AGUAAA hexamer functions as a cis-acting element in coronaviral polyadenylation. Additionally, different levels of substitutions within the hexamer AGUAAA were also performed in W-25A (Fig 5A, right panel). RT-PCR products were detected (Fig 5B, right panel), and the overall poly(A) tail lengths for the mutants M1-25A, M4-25A and M5-25A were not altered, whereas the lengths for M2-25A, M3-25A and R-25A are slightly decreased (Fig 5C, right panel), suggesting that the alternations in the poly(A) tail length do not occur in the DI RNAs with 25 nts of poly(A) tail and mutated AGUAAA. Taken together, these results further support our finding that the hexamer AGUAAA is an important cis-element in coronavirus polyadenylation.\nThe function of hexamer AGUAAA in polyadenylation is position dependent\nBased on the data presented here, this hexamer AGUAAA is involved in coronaviral polyadenylation. To determine whether the function of the hexamer AGUAAA in polyadenylation is position dependent, the sequence in W-15A between 37 and 42 nts upstream of the poly(A) tail site, where the original hexamer AGUAAA is located, was replaced with UCAUUU, and the sequence at 49 to 54 nts upstream of the poly(A) tail site was substituted with hexamer AGUAAA to create mutant PAS-R-15A (Fig 6A, left panel). In addition, mutant PAS-PAS-15A was also constructed in which the sequence between 49 and 54 nts upstream of the poly(A) tail site was replaced with the hexamer AGUAAA, but the original hexamer AGUAAA between 37 and 42 nts upstream of the poly(A) tail site was retained (Fig 6A, left panel). Wild-type DI RNA W-15A, mutant R-15A, in which the original hexamer AGUAAA was replaced with UCAUUU, and mutant PAS-PAS-15A were then used as controls to evaluate the position dependency of the hexamer AGUAAA in polyadenylation. Moreover, to ensure that the altered polyadenylation efficiency indeed results from sequence changes in the aforementioned positions of DI RNA with 15 nts of poly(A) tail, we also created constructs PAS-R-25A and PAS-PAS-25A, with poly(A) tail lengths of 25 nts (Fig 6A, right panel). We predicted that the sequence changes may not alter the polyadenylation efficiency for DI RNA with 25 nts of poly(A) tail according to the previous data shown in Fig 2. As shown in Fig 6B, left panel, RT-PCR products were observed for all constructs at 24 hpi of VP1, and the poly(A) tail lengths were determined to be 5, 15, 4 and 13 nts for PAS-R-15A, PAS-PAS-15A, R-15A and W-15A, respectively (Fig 6C, left panel). Note that the RT-PCR product for PAS-R-15A with the size less than 100 bp was sequenced and determined to be primer-dimer. These results suggest (i) the sequence substitution between 49 and 54 with hexamer AGUAAA in PAS-PAS-15A (the original hexamer AGUAAA was retained) did not affect the efficiency of polyadenylation when compared with the poly(A) tail length of W-15A (15 vs 13 nts) and (ii) the efficiency of poly(A) tail synthesis was still low (5 nts) for PAS-R-15A (the sequence between 49 and 54 was replaced with hexamer AGUAAA and the original hexamer AGUAAA was mutated). Note that these DI RNAs were able to replicate (S6B Fig) and the mutated sequences were still retained (S6C Fig). Consequently, since the position change of hexamer AGUAAA in PAS-R-15A did not restore the efficiency of poly(A) tail synthesis, it is concluded that the function of the hexamer AGUAAA in polyadenylation is position dependent. In addition, by RT-PCR (Fig 6B, right panel) and sequencing, the poly(A) tail length for constructs PAS-R-25A, PAS-PAS-PAS, R-25A and W-25A was identified to be similar (Fig 6C, right panel), suggesting that for DI RNA with poly(A) tail of 25 nts, substitution mutation between 49 and 54 nts or 37 and 42 nts upstream of the poly(A) site has only a minor or no effect on the efficiency of polyadenylation. Taken together, it is concluded that the function of the hexamer AGUAAA in coronaviral polyadenylation is position dependent.\nThe hexamer AGUAAA or its variants are found among coronaviruses\nThe consensus PAS hexamer AAUAAA, located 10&#8211;30 nts upstream of the poly(A) cleavage site, is one of the cis-acting elements responsible for eukaryotic mRNA polyadenylation [1, 7, 8]. Variants, including AGUAAA, AUUAAA and UAUAAA, have also been identified as important for poly(A) tail synthesis in certain eukaryotic mRNA populations [9&#8211;11]. In the current study, we demonstrated that the hexamer AGUAAA, located between 37 and 42 nts upstream of the poly(A) site in BCoV, is involved in coronaviral polyadenylation. Moreover, the AGUAAA motif or its variants are also found in other coronaviruses, as summarized in Table 1. Among these hexamers, AAUAAA, AGUAAA and AUUAAA, which function in eukaryotic mRNA polyadenylation, are found in betacoronavirus C, betacoronavirus A and deltacoronavirus, respectively. Other hexamer variants, which differ from hexamer AAUAAA, AGUAAA or AUUAAA by one nucleotide, have also been identified in alphacoronavirus, betacoronavirus B, betacoronavirus D, and gammacoronavirus. Interestingly, although the positions of these hexamers in coronaviruses can be from 24 to 57 nts upstream of the poly(A) tail, the motifs are at similar positions within the same genus or lineage. However, it remains to be determined whether the hexamer has similar functions in coronaviruses other than BCoV.\nDiscussion\nIn this article, we provide genetic evidence of how coronavirus lengthens its poly(A) tail during the infection cycle and how the hexamer AGUAAA and poly(A) tail act in concert to accomplish coronaviral polyadenylation. The characterization of two polyadenylation-related elements, AGUAAA and the poly(A) tail, will help in a better understanding of the mechanism of coronaviral polyadenylation.\nFactors involving in the regulation of coronaviral poly(A) tail length during coronavirus infection\nIn the previous study [16], it was found that the poly(A) tail length of W-65A (DI RNA with 65 nts of poly(A) tail) at 48 hpt (VP0) is 28 nts and was increased to be 62 nts at 8 hpi of VP1. The results led us to ask the question of what the the minimum poly(A) tail length required to initiate tail lengthening of coronavirus DI RNA is. In order to precisely identify the minimum requirement to initiate the lengthening of poly(A) tail, instead of constructing DI RNA with long ply(A) tail, for example, W-65A with 65 nts of poly(A) tail [16], several DI RNAs with a little variation in poly(A) tail length were constructed and therefore only a small variation in poly(A) tail length from these DI RNA constructs was expected in the process of infection as shown in Fig 1. In terms of the identification of the minimum length for the initiation of poly(A) lengthening, the small increase in the poly(A) tail length of the same DI RNA construct (for example, 2 nts increase in the poly(A) tail length of W-15A between 48 hpt (VP0) and 8 hpi of VP1) is therefore critical for the interpretation of the results although the level of length change is not dramatic. Regardless, the results shown in Fig 1 along with the previous findings from W-65A [16] collectively reinforce the critical nature of the initial poly(A) tail length in the elongation of coronaviral poly(A) tail in the early stage of infection. Interestingly, after 8 h of infection in VP1, the poly(A) tail length on DI RNAs W-25A and W-20A decreased from 31 and 22 nts to 24 and 19 nts, respectively, whereas that of DI RNA W-15A increased from 11 to 13 nts. We speculate that DI RNA with a short enough poly(A) tail may not be efficiently detected by deadenylases [53, 54], thus avoiding efficient tail degradation mediated by the deadenylation mechanism. After initial lengthening of the poly(A) tail, it is also feasible that the polyadenylation may occur again based on the previous lengthening poly(A) tail possibly by viral polymerase or cytoplasmic poly(A) polymerase during infection. On the other hand, in terms of the shortening of coronavial poly(A) tail, it is still not clear at this point what mechanism is responsible for the coronaviral deadenylation. The regulation of tail length in coronaviruses, therefore, may be complicated because it occurs in a dynamic process and in addition to poly(A) tail length, other factors such as deadenylase and viral polymerase or cytoplasmic poly(A) polymerase may be collectively involved in the decision of the eventual length of coronavirus poly(A) tails at a given time during infection through unidentified mechanisms.\nContribution of hexamer AGUAAA for virus survival\nIn the present study, we demonstrated that both the poly(A) tail and hexamer AGUAAA are involved in coronaviral polyadenylation and the two elements appear to be able to functionally compensate for each other when one is missing or modified, as exemplified by DI RNA mutants R-25A and W-25U. Based on present results, we propose that both the hexamer AGUAAA and poly(A) tail contribute to virus survival under diverse environments. For example, a poly(A) tail length shorter than 20 nts was consistently found for coronaviral RNA collected from (i) BCoV-infected HRT-18 cells at 2 hpi, (ii) W-25A-transfected BCoV-infected HRT-18 cells at 72 hpi of VP1 and (iii) mouse brain at 5 days postinfection with MHV-A59 (S7 Fig). Moreover, in persistent infection, a MHV-A59 poly(A) tail length of less than 20 nts was also frequently found (S7 Fig); therefore, when such persistent-infection coronavirus infects fresh cells, the AGUAAA hexamer is also required in concert with the poly(A) tail for efficient polyadenylation. Thus, the hexamer AGUAAA along with the poly(A) tail may be required to restore efficient polyadenylation for subsequent translation and replication, contributing to virus survival under these conditions.\nThe importance of the minor changes in poly(A) tail length for gene expression of coronavirus\nUnlike most mammalian mRNAs with ~250 nts of poly(A) tail, positive-strand RNA viruses such as piconaviruses, have heterogeneous lengths of natural poly(A) tail ranging from about 10 to 120 nts long [55&#8211;57]. Functional analysis reveals that 12 nts of poly(A) tail on positive-strand genome of poliovirus is the minimum length to initiate negative-strand synthesis of genome and that increasing the poly(A) tail length from 12 to 13 nts results in about a ten-fold efficiency in negative-strand synthesis [58], suggesting that minor changes in viral poly(A) tail length exert significant effect on viral replication. Similar results have also been obtained in sindbis virus during negative-strand synthesis in which increasing the poly(A) tail length from 10 to 15 nts leads to a nearly nine-fold increase in negative-strand RNA synthesis [59]. The size of poly(A) tail in BCoV varies at different time points of infection and its distribution during infection is from about 13 to 68 nts (S7 Fig and [16]). As with aforementioned positive-strand RNA viruses, small alternations in poly(A) tail size from 5 to 10 nts and from 25 to 45 nts in coronavirus dramatically increase the efficiency of viral replication [18] and translation [16], respectively, further demonstrating the small changes in the poly(A) tail length play an important role in the biological function of coronavirus. Therefore, the lengthening of poly(A) tail, for example, in W-15A from 8 nts at 48 hpt of VP0 to 13 nts at 24 hpi of VP1 and W-25A from 21 nts at 48 hpt of VP0 to 31 nts at 8 hpi of VP1 (Fig 1B) may increase the efficiency of gene expression. Since the minor changes have an important effect on viral replication and translation, we speculate that for coronavirus with short poly(A) tail length ranging between ~13 to ~68 nts (in comparison with mammalian mRNA with ~250 nts of poly(A) tail) the minor changes in the poly(A) tail length may be sensitive and thus critical in the regulation of gene expression during infection.\nEvaluation of the effect of replication efficiency on polyadenylation of DI RNA\nIn the present study, we found that the replication efficiency of DI RNA mutants R-25A (36%) (Fig 2) and W-25C (<1%, S8 Fig) was much lower than that of W-25A (100%). However, the synthesized poly(A) tail lengths of these two mutants was similar to that of W-25A, suggesting that the replication efficiency may not be a major factor affecting coronaviral polyadenylation. For DI RNA with shorter poly(A) tail and AGUAAA mutation, for example, R-15A, its replication efficiency was similar to that of W-15A at the stage of 24 hpi of VP1,and RT-PCR products were detected for sequencing (S5A and S5B Fig); however, the synthesized poly(A) tail length (3 and 13 nts for R-15A and W-15A, respectively) was different (Fig 3C), also suggesting that the replication efficiency may not be a major factor in the poly(A) tail synthesis. Accordingly, this may be applied to account for the results shown in Fig 3E. The replication efficiency for both R-15A and R-20A was similar because they were not detected by Northern blot assay (Fig 3E); however, both were detected by RT-PCR and sequencing results showed that the poly(A) tail length for both constructs was different (3 and 18 nts, respectively) (Fig 3C), also suggesting replication efficiency may not play a major role in poly(A) tail synthesis. For R-5A in Fig 3E, the unidentified poly(A) tail from R-5A may be attributed to the undetected RT-PCR product due to the poor replication efficiency. The aforementioned arguments may also account for the results shown in Fig 4. For W-25U and W-25C, the replication efficiency for both DI RNAs is low because they were not detected by Northern blot assay (S8A Fig) when compared with that for W-25A. Subsequent study showed that the replication efficiency was similar between W-25U, W-25C, R-25U and R-25C as determined by qRT-PCR (S8B Fig); however, poly(A) tail was synthesized from W-25U and W-25C but not from R-25U and R-25C (Fig 4B), suggesting replication efficiency may not be the main determinant in the poly(A) tail synthesis. For R-25G, similar to R-5A, the unidentified poly(A) tail may be attributed to the undetected RT-PCR product. Taken together, we speculate that the replication efficiency may not be a major factor affecting coronaviral polyadenylation and that replication and polyadenylation in coronaviruses are separate processes but proceed by a similar theme during viral RNA synthesis.\nRole of hexamer AGUAAA in coronavirus replication\nIt has been shown that with reverse genetic approaches the deletion of nts 30&#8211;170 upstream of poly(A) tail do not affect the growth function in tissue culture for MHV [60, 61]. It is reasonable to speculate that the hexamer AGUAAA, located between 37 and 42 nts upstream of the poly(A) site, may not play a role in the replication. However, in the current study, the results suggest that the replication efficiency is decreased in DI RNA R-25A with mutated hexamer AGUAAA in comparison with that in W-25A with intact AGUAAA (36% vs 100%). We speculate that the system (reverse genetics with a full-length cDNA vs DI RNA) and mutagenesis (deletion of entire region between nts 30&#8211;170 vs replacement of hexamer within the undeleted region of nts 30&#8211;170) employed for the analysis may lead to the discrepancy of the results. Nevertheless such discrepancy does not affect the role of hexamer AGUAAA in the coronaviral polyadenylation concluded in this study because the polyadenylation appears not to be influenced by replication efficiency as evidenced by the results of DI RNA constructs discussed above.\nProposed mechanism for coronaviral polyadenylation\nBased on the evidence shown in this study and in others, we propose a model for coronaviral polyadenylation (Fig 7). First, coronavirus polymerase utilizes positive-strand viral RNA as a template to synthesize negative-strand viral RNA in which the length of the poly(U) tract is similar to that of the poly(A) tail at the same time point after infection [16]. Subsequently, the negative-strand viral RNA serves a template for synthesizing the positive-strand viral RNA. During positive-strand viral RNA synthesis, once the nascent hexamer AGUAAA is copied, cytoplasmic polyadenylation-associated factors such as CPSF and other accessory factors may interact with the hexamer on the positive strand. This hexamer-protein complex may interact with (an)other complex(es) formed by interactions between proteins and the poly(U) tract on the negative strand to generate a stable RNA-protein complex that directs the viral RdRp to synthesize the poly(A) tail. Alternatively, the RNA-protein complex formed may recruit cytoplasmic poly(A) polymerase instead of viral RdRp for poly(A) tail synthesis [62]. This model emphasizes the important role of the AGUAAA hexamer on the nascent positive-strand viral RNA as well as the 5&#8217; terminal sequence (i.e., poly(U) tract) on the negative-strand viral RNA, with which polyadenylation-associated proteins interact.Accordingly, under the argument of constitution of a stable complex for efficient coronavirus polyadenylation, this model may explain (i) why a poly(A) tail was synthesized from poly(A)-deficient DI RNA constructs W-25U, W-25C, and W-25G, rather than poly(A)-deficient DI RNA W-0A, even though they all contain a hexamer AGUAAA; and (ii) why a poly(A) tail was found for R-15A and R-18A, but not R-25U, R25C and R25G, despite the fact that they all lack the hexamer AGUAAA. In the cases of W-25U, W-25C and W-25G, which contain the hexamer AGUAAA but lack a poly(A) tail, it is speculated that in spite of different affinity the binding of proteins (e.g., PABP [63, 64]) to the 5&#8217; end of poly(A), poly(G) or poly(C) on the W-25U, W-25C or W-25G negative strand, respectively, and then to proteins binding to the AGUAAA motif on the nascent positive-strand RNA to form a protein complex is a key step in coronaviral polyadenylation. In contrast, in the case of W-0A, the aforementioned protein complex is not formed because there is no such RNA element at the 5&#8217; end of negative-strand W-0A for protein binding and thus no polyadenylation occurs. Regarding constructs R-15A, R-18A, R-25U, R25C and R25G, in which the hexamer AGUAAA was mutated, we reason that if polyadenylation-related protein binding to RNA elements at the 5&#8217; end of negative-strand viral RNA is not sufficient to form a stable RNA-protein complex, assistance from interactions between the AGUAAA motif and proteins may be required for polyadenylation. Accordingly, although proteins were able to bind to poly(A), poly(G) or poly(C) on the negative strand of R-25U, R-25C or R-25G, respectively, polyadenylation was unable to occur without the help of the hexamer AGUAAA to form a stable RNA-protein complex. Furthermore, under the same argument of constitution of a stable complex for efficient coronavirus polyadenylation, this model may be extended to explain why different levels of hexamer AGUAAA mutation (Fig 5) and the alternation of hexamer AGUAAA position (Fig 6) decreased the efficiency of polyadenylation on DI RNA.In conclusion, we for the first time determined the viral RNA elements involved in coronaviral polyadenylation. Future works may elaborate on the identification of the cellular and viral proteins participating in the synthesis of the coronaviral poly(A) tail as well as the detailed mechanism of coronaviral polyadenylation and its regulation.\nSupporting Information\n",
        "pmid": "27760233",
        "title": "Characterization of the Role of Hexamer AGUAAA and Poly(A) Tail in Coronavirus Polyadenylation",
        "journal_title": "PLoS ONE",
        "authors": "Yu-Hui Peng;Ching-Houng Lin;Chao-Nan Lin;Chen-Yu Lo;Tsung-Lin Tsai;Hung-Yi Wu",
        "doi": "10.1371/journal.pone.0165077"
    },
    {
        "keywords": "",
        "abstract": "The severe acute respiratory syndrome coronavirus (SARS-CoV) and recently emerged Middle East respiratory syndrome coronavirus (MERS-CoV) epidemics have proven the ability of coronaviruses to cross species barrier and emerge rapidly in humans. Other coronaviruses such as porcine epidemic diarrhea virus (PEDV) are also known to cause major disease epidemics in animals wiith huge economic loss. This special issue in Virology Journal aims to highlight the advances and key discoveries in the animal origin, viral evolution, epidemiology, diagnostics and pathogenesis of the emerging and re-emerging coronaviruses in both humans and animals.",
        "body": "Editorial\nIn the coming months, Virology Journal will publish a thematic series of reviews and original articles on emerging and re-emerging coronaviruses.Several coronaviruses have caused serious problems in humans and animals in the past two decades. The best known examples are severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV) and porcine epidemic diarrhea virus (PEDV). Urbanization and the increasingly frequent mixing of different animals in densely populated areas may have facilitated the emergence and re-emergence of some of these viruses. On the other hand, coronaviruses are known to have high mutation and recombination rates, which may allow them to cross species barriers and adapt to new hosts.The SARS epidemic in 2003 has awakened scientists and the world on the ability of coronaviruses for animal-to-human transmission. It was one of the worst epidemics in our city&#8217;s history, with 1,755 people infected and 299 died from the dreadful infection. The animal sources of SARS-CoV was later traced back to civets in wild life markets in China as the intermediate host, and ultimately to horseshoe bats as the primary reservoir [1, 2]. Although SARS-related coronaviruses are continuously found in bats from China and worldwide, it is believed that the immediate ancestor of human SARS-CoV has disappeared. Nevertheless, we still have to be vigilant about the possibility of re-emergence of SARS and uphold measures to avoid mixing of wild animals especially bats and civets in markets.The SARS epidemic has boosted interests in hunting for other novel coronaviruses. Before the SARS epidemic, only two coronaviruses were known to infect humans, namely human coronavirus (HCoV)-229E and HCoV-OC43. Two additional human coronaviruses, HCoV-NL63 and HCoV-HKU1, were discovered soon after the SARS epidemic, although these viruses have likely been circulating in humans for a long time prior to their discovery. On 23 September 2012, the World Health Organization (WHO) announced the discovery of a new coronavirus, MERS-CoV, in two patients who died from a mysterious, rapidly-fatal disease in the Middle East [3]. The virus was, at that time, found to be most closely related to the prototype lineage C betacoronaviruses, Tylonycteris bat coronavirus HKU4 and Pipistrellus bat coronavirus HKU5, previously identified in Hong Kong [3&#8211;5]. Subsequently, dromedary camels were identified as the source of some human cases [6].Apparently, MERS has an even higher mortality rate (>35&#160;%) than SARS (9.6&#160;%), which may be partly explained by the high prevalence of medical comorbidities among infected patients [7]. Fortunately, in the past three years, MERS has not been transmitted from person to person as efficiently as SARS was. However, a number of large healthcare-associated outbreaks in the Middle East and the recent epidemic in the Republic of Korea have highlighted the possibility of sustained person-to-person transmission of MERS-CoV. In the recent outbreak in Korea, more than 180 people were infected as a result of second-, third-, or fourth generation infection from a single index case who acquired MERS after traveling to Qatar. Although these large-scale outbreaks of MERS were eventually controlled with enhanced infection control measures, they have highlighted the importance of rapid diagnosis and coordinated public health response. A lot more research efforts on MERS-CoV are needed, especially at the epicenter, the Middle East, where continuous animal-to-human transmissions of MERS-CoV are still being regularly reported. This is in contrast to SARS which was rapidly contained after removal of civets from wild life markets in southern China. In particular, more research on the pathogenesis, treatment and prevention of this new coronavirus with pandemic potential are urgently needed [7].Besides humans, coronaviruses also cause various diseases in animals with some resulting in significant economic losses. PEDV, a member of the genus Alphacoronavirus, is the etiological agent of porcine epidemic diarrhea in swine populations. The disease was endemic in Asia and European countries in the past decades. In 2013, the virus has emerged or re-emerged to cause large-scale epidemics in the Americas and Asia, causing millions of fatal cases among piglets and representing one of the most serious concerns among the global pig industries [8, 9]. The ability of epidemic PEDV strains to emerge despite current vaccination schemes suggests that the virus is able to evolve and escape from immune protection.The special issue to be published in Virology Journal will emphasize advances and key discoveries in the animal origin, viral evolution, epidemiology, diagnostics and pathogenesis of different emerging and re-emerging coronaviruses.Two reviews in this thematic series will focus on the evolution and pathogenesis of MERS-CoV. While both SARS and MERS can cause similar severe respiratory illnesses, considerable differences exist in terms of patient demographics, clinical outcomes as well as transmission dynamics. The review, MERS coronavirus: epidemiology, transmission and diagnostics by I. Mackay et al. discusses the current understandings on the epidemiology and transmission of MERS, and highlights the keys to rapid diagnosis [10]. While intensive research has been conducted on this virus, histopathological and immunopathogenesis studies using clinical samples are scarce and autopsy findings are lacking, partly due to cultural reasons. As a result, current understandings on the pathogenesis and immune response of MERS rely mainly on in vitro or in vivo studies. The review by J. Zhou et al. (MERS coronavirus infection in human cells: virus-host interaction and implication for pathogenesis) summarizes the most up-to-date findings on virus-host interaction of MERS-CoV based on cellular and animal studies and their implications on disease pathogenesis [11].Bats are important reservoir for many emerging viruses. Similar to SARS-CoV, the finding of MERS-CoV-related bat coronaviruses suggests that bats may also be the ultimate animal origin of MERS-CoV, with camels being the intermediate hosts before the virus crossed species barriers to infect humans [7]. It is now known that bats are important reservoirs of diverse alphacoronaviruses and lineage B, C and D betacoronaviruses [12]. While horseshoe bats in China are the natural reservoir of SARS-CoV-like viruses, the bat species that may harbor the ultimate origin of MERS-CoV is yet to be identified. In a review entitled Bat origin of coronaviruses, Z. Shi et al. summarize the role of bats as the origin of emerging coronaviruses in humans, with special focus on SARS-CoV and MERS-CoV [13]. The review will also emphasize the importance of field studies to elucidate the animal origins of coronaviruses in predicting and preventing future pandemics.Beside SARS-CoV and MERS-CoV which have only emerged recently, the other four human-pathogenic coronaviruses, namely, HCoV-229E, HCoV-OC43, HCoV-NL63 and HCoV-HKU1, have been continuously circulating in human for at least decades or centuries. Although they are often thought to be associated with mild respiratory illnesses, increasing reports have suggested that they may cause severe infections, especially in people at the extremes of age or those with comorbidities. However, pathogenesis studies are often limited by the difficulties in isolating these coronaviruses in cell cultures. The human airway serves as the first entry point for these respiratory viruses. We also plan to include a review that discusses the establishment of human airway epithelial cells for isolation of human coronaviruses and latest understandings on virus-host interactions based on cell culture studies.Apart from humans, animals are also susceptible to different coronaviruses and PEDV represents a notable example how coronaviruses can cause serious epidemics in animal population worldwide. In the review entitled Emergence and re-emergence of PEDV, C. Lee et al. discuss the latest advances in understanding the epidemiology, pathogenesis and control measures of PEDV, focusing especially on the recent epidemics affecting America and Asia [14].Virology Journal is taking a leading role in facilitating the dissemination of new information in various emerging infectious diseases including coronaviruses. It is hoped that the present series will stimulate more research and collaborative efforts to control the ongoing and prevent future epidemics caused by coronaviruses. We are grateful to Linfa Wang, Ph.D., Editor in Chief of Virology Journal and Professor at Duke-National University of Singapore for his guidance during the preparation of this thematic series.",
        "pmid": "26690088",
        "title": "Coronaviruses: emerging and re-emerging pathogens in humans and animals",
        "journal_title": "Virology Journal",
        "authors": "Susanna K. P. Lau;Jasper F. W. Chan",
        "doi": "10.1186/s12985-015-0432-z"
    },
    {
        "keywords": "MERS-CoV;Middle East respiratory syndrome;MERS;CoV;coronavirus;nCoV;coronavirus infections;viruses;Germany;respiratory infections;United Arab Emirates",
        "abstract": "No evidence was found for nosocomial transmission of this coronavirus.",
        "body": "Methods\n\nContact Investigation\nFor the investigation, the City Health Department assessed all contact persons (contacts) retrospectively and monitored them prospectively. All contacts received a questionnaire for retrospective documentation and prospective daily self-monitoring of symptoms, exposure to the patient, and infection control measures applied. For every day from March 19 through April 5, information was collected about the contacts&#8217; distance from the patient (2 meters); type of contact with the patient (aerosol-producing procedures, non&#8211;aerosol-producing procedures, care of patient, handling of urine catheter, handling of respiratory samples in the laboratory, handling of urine samples in the laboratory); type of protection used (surgical mask, FFP1 mask, FFP2 mask, FFP3 mask, gown, gloves, protective glasses); and symptoms experienced by the contacts (cough, fever, temperature, sore throat, diarrhea, shortness of breath). An aerosol-producing procedure was defined as respiratory suction, bronchoalveolar lavage, intubation, or bronchoscopy.On the basis of the self-reported information in the questionnaires and personal interviews with the contacts, we divided contacts into 2 groups. Close-distance contacts had face-to-face contact with the patient (<2 meters from the patient) or direct contact with secretions or body fluids of the patient, irrespective of protective measures worn. All other contacts were classified as less-close-distance contacts. According to WHO recommendations on the duration of follow-up at that time, close-distance contacts were asked to contact the City Health Department daily for 10 days after the last exposure to the patient. Those who failed to do so were contacted by the City Health Department, supported by the occupational health service of the hospital. Less-close-distance contacts were asked to report to the City Health Department only in case of onset of symptoms.Respiratory illnesses in contacts that occurred 1&#8211;10 days after exposure to the patient were assessed through the City Health Department by telephone contact with the contact; a respiratory tract sample was taken from any contact with respiratory illness. In addition, attempts were made to obtain paired serologic samples from all contacts, the first taken immediately after contact and the second >28 days after the last exposure.\nPatient Questionnaire\nBecause the MERS-CoV patient was on mechanical ventilation and could not be interviewed, family contacts who had accompanied him to Germany were interviewed about the onset of his symptoms and possible exposures in the 10 days before disease onset. For the interview, a structured questionnaire was used, and information collected was documented on paper.\nLaboratory Methods\nPCR testing and serologic testing were done as described (17,20). Serum samples from contacts were tested for MERS-CoV antibodies if a serum sample was taken >28 days after last exposure. In addition, serum samples were tested for antibodies against influenza A, B, and C; rhinovirus A, B, and C; parainfluenzavirus 1, 2, 3, and 4; respiratory syncytial virus A and B; human metapneumovirus; coronavirus 229E, NL63, OC43, and HKU1; and adenovirus. All samples were analyzed at the Institute for Virology of the University of Bonn.\nData Analysis\nData from the City Health Department&#8217;s contact monitoring, the contacts&#8217; questionnaires, and the laboratory findings were integrated in 1 database. Results were validated and analyzed by using Stata version 12.0 (StataCorp, College Station, TX, USA).\nResults\n\nContact Investigation\nThe City Health Department identified 83 contacts. Of these, 69 (83%) were classified as close-distance contacts and 14 (17%) as less-close-distance contacts (Table). Four (5%) of the contacts were members of the patient&#8217;s family, 16 (19%) were physicians, 25 (30%) were nursing staff, 20 (24%) were laboratory personnel, and 18 (22%) were part of other professional groups. Clinical follow-up was available for 81 (98%) contacts.A respiratory symptom or fever developed in 10 (12%) contacts. Of these, swab specimens were collected from 9 (90%) and blood samples from 7 (70%). All 9 swab specimens were negative for MERS-CoV; 1 (11%) was positive for CoV NL-63, and 2 (22%) were positive for rhinovirus. All 7 serum samples were negative for MERS-CoV antibodies. All symptomatic contacts had >1 sample type (respiratory swab or serum) collected for laboratory testing; results of PCR and serologic testing were available from 6 (60%), PCR only from 3 (30%), and serologic testing only from 1 (10%). In addition, serologic test results were available for 53 (75%) of the 71 nonsymptomatic contacts; all were negative for MERS-CoV antibodies. Overall, persons for whom serologic testing results were available were more likely to be close-distance contacts than were persons without available serologic results (p = 0.007; Table).The 4 family members who accompanied the patient were his wife, daughter, son, and son-in-law. Their ages were 35&#8211;37 years, and none reported symptoms. The patient&#8217;s children and son-in-law had their last contact with the patient on March 20 and his wife on March 23. Because no protection measures had been used until after March 20, the family members were considered at high risk for infection. All 4 provided respiratory swab and serum samples on March 24; all samples had negative results. Serum samples taken >28 days after last exposure to the patient were not available.MERS-CoV infection was added to the differential diagnosis for the patient on March 21. The daily numbers of HCWs who had any contact with him (regardless of protection measures) and of those who had aerosol exposure were lower after that date than before (Figure 2): 4.4 HCW per illness day vs. 7.5 HCW per illness day (p = 0.05) and 2.8 HCW per illness day vs. 6 HCW per illness day (p = 0.03). Among HCWs with aerosol exposure, 1 (8%) of 12 daily exposures occurred while FFP2 or FFP3 masks were being used before March 21; after that date, 11 (79%) of 14 daily exposures occurred while FFP2 or FFP3 masks were being used (p<0.01).\nPatient Questionnaire\nThe patient was a 73-year-old married man from Abu Dhabi, United Arab Emirates; he had a medical history of multiple myeloma. At the time of his MERS-CoV infection, he was receiving corticosteroid therapy. His profession was camel breeding; in the 2 weeks before his onset of illness, 1 of his camels was reported to have had a respiratory illness. In the questionnaire, we did not differentiate between dromedary (Camelus dromedaries) and Bactrian (C. bactrianus) camels. His neighborhood had palm trees, and bats were known to dwell in the area. The patient had no known contact with other MERS-CoV patients, had no personal contacts in Qatar or Jordan, and had no travel history in the 10 days before illness onset. He consumed different types of fruit juices and cooked goat meat, beef, and sheep meat, but no raw meat. He ate dates from his region, but he reportedly did not consume date or palm syrup. Other than the camels on his farm, he had no contact with animals; he did not practice falconry and did not visit camel racetracks or animal markets.\nDiscussion\nWe describe the case and contact investigation of a confirmed case of MERS-CoV infection that was imported to Germany. We did not identify person-to-person transmission from the patient to any of the contacts. As with the previous imported case in this country, the patient was already on mechanical ventilation when he was transferred to Germany. However, whereas the previous case was in the late third week of illness, this patient was in the second week of illness. Sample from this patient taken from different body locations and at different times were positive for MERS-CoV by PCR, and the viral load detected was several logs higher than in samples from the patient with the previous imported case (18,20). These results indicate that this patient may have been more infectious than the previous patient.Nosocomial transmission from MERS patients to HCWs has been documented (13,21,22). In our study, the patient was isolated during the first 2 days of his hospital stay (before MERS was suspected), although the reason for this intervention was the hospital&#8217;s policy to isolate every patient from the Middle East, irrespective of the assumed diagnosis, because of perceived increased risk of carrying drug-resistant pathogens, rather than any special measures taken because of the patient&#8217;s respiratory illness. After MERS was suspected, HCWs used FFP2 masks significantly more frequently than they had before, and fewer HCWs had daily contact with the patient. Our result suggest that, in the later stages of this disease, the combination of standard protection measures (use of surgical masks for potentially aerosol-generating procedures), cautious handling of the patient (because of his potential to harbor drug-resistant bacteria), and possible decreased infectiousness compared with the first week of illness may have prevented transmission to HCWs. These findings also underline the importance of following WHO recommendations on infection prevention and control when managing a patient who may be infected with a pathogen that could lead to nosocomial transmission (23).Regarding possible sources of infection, an extensive interview was conducted with family members because the patient could not be interviewed. The patient&#8217;s illness was likely a primary case, and possible exposures that might have caused the MERS-CoV infection were explored. Of note were the presence of bats in the neighborhood of his residence, the patient&#8217;s profession as camel breeder, and his contact with a camel that was reported to have had a respiratory illness before his own illness onset. Bats are a likely reservoir for MERS-related CoV (5,8), and serum samples from Omani racing camels have shown to have neutralizing antibodies against MERS-CoV (24). These findings suggest these animals&#8217; possible relevance (e.g., as intermediate hosts) for human acquisition of MERS-CoV.Two complementary monitoring instruments for contact persons were used: active follow-up with daily telephone contact and a self-administered monitoring questionnaire. Both methods have merits, and a combination of both is likely to ensure the most thorough contact follow-up. Advantages of personal interviews on the telephone are immediacy and the possibility for the interviewer to receive intangible information, such as the self-assessment of symptoms, as well as the opportunity to answer questions from the contacts. This process enables a more specific way to judge a person&#8217;s health status. On the other hand, a daily monitoring questionnaire provides detail in clinical information, exposure, and protection measures that might be used for more in-depth analyses (e.g., when a few contacts have become infected). Such a questionnaire could be expanded to include a section for contact persons to fill in the names of persons with whom they had face-to-face contact during each day. This information might become crucial for second-generation contact tracing when contacts under observation become infected. Rapid availability of this type of information is essential for efficient investigation of clusters or outbreaks similar to those that have been reported already (13).In conclusion, we conducted a contact investigation of an imported case of MERS-CoV infection in Germany. Laboratory testing of symptomatic and asymptomatic contacts of the index case-patient did not indicate transmission of the virus. Furthermore, we documented the change from standard hygiene to infection control measures after MERS-CoV was suspected, an adaptation that may have prevented nosocomial transmission. Exposure to camels as a possible etiologic mechanism for human MERS-CoV infection requires further evidence from other studies.",
        "pmid": "24655721",
        "title": "Contact Investigation for Imported Case of Middle East Respiratory Syndrome, Germany",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Annicka Reuss;Annette Litterst;Christian Drosten;Michael Seilmaier;Merle B\u00f6hmer;Petra Graf;Hermann Gold;Clemens-Martin Wendtner;Arina Zanuzdana;Lars Schaade;Walter Haas;Udo Buchholz",
        "doi": "10.3201/eid2004.131375"
    },
    {
        "keywords": "",
        "abstract": "The positive-strand coronavirus genome of ~30 kilobase in length and subgenomic (sg) mRNAs of shorter lengths, are 5&#8217; and 3&#8217;-co-terminal by virtue of a common 5&#8217;-capped leader and a common 3&#8217;-polyadenylated untranslated region. Here, by ligating head-to-tail viral RNAs from bovine coronavirus-infected cells and sequencing across the ligated junctions, it was learned that at the time of peak viral RNA synthesis [6 hours postinfection (hpi)] the 3&#8217; poly(A) tail on genomic and sgmRNAs is ~65 nucleotides (nt) in length. Surprisingly, this length was found to vary throughout infection from ~45 nt immediately after virus entry (at 0 to 4 hpi) to ~65 nt later on (at 6 h to 9 hpi) and from ~65 nt (at 6 h to 9 hpi) to ~30 nt (at 120-144 hpi). With the same method, poly(U) sequences of the same lengths were simultaneously found on the ligated viral negative-strand RNAs. Functional analyses of poly(A) tail length on specific viral RNA species, furthermore, revealed that translation, in vivo, of RNAs with the longer poly(A) tail was enhanced over those with the shorter poly(A). Although the mechanisms by which the tail lengths vary is unknown, experimental results together suggest that the length of the poly(A) and poly(U) tails is regulated. One potential function of regulated poly(A) tail length might be that for the coronavirus genome a longer poly(A) favors translation. The regulation of coronavirus translation by poly(A) tail length resembles that during embryonal development suggesting there may be mechanistic parallels.",
        "body": "Introduction\nThe~30 kilobase positive-strand coronavirus genome and subgenomic (sg) mRNAs of shorter lengths are 5&#8217; capped and 3&#8217; polyadenylated as are most eukaryotic mRNAs [1]. They also share a common leader sequence (65 to 90 nucleotides (nt) in length, depending on the coronavirus species) and a common 3&#8217; untranslated region (~300 nt). In the group 2 mouse hepatitis coronavirus (MHV), the poly(A) tail on the genome has been determined to be about 90 nt in length using RNase T1 followed by ion-exchange chromatography and gel electrophoresis [2&#8211;5], whereas for MHV no measurement of the complementary poly(U) on the negative strand has yet been reported. In the bovine coronavirus (BCoV) genome, a 3&#8217;-terminal poly(A) tail of 60 nt and an oligo(U) tract of 8 to 20 nt have been identified by primer extension followed by head-to-tail ligation of synthesized cDNA molecules, PCR amplification and sequencing [5]. How the long poly(A) tract is generated from a short oligo(U) tract has not been described, although it has been postulated to result from a stuttering mechanism by the viral RNA-dependent RNA polymerase during synthesis of genomic or sgmRNA [5], or by a cellular cytoplasmic poly(A) polymerase [5].Here, we report studies using a previously-described technique of head-to-tail ligation of intracellular coronaviral positive-strand RNAs and negative-strand RNAs which demonstrate that both the poly(A) on the positive-strand RNAs and poly(U) on the negative-strand RNAs can reach a length of ~65 nt, and that both, surprisingly, vary in length during infection suggesting there is a regulatory mechanism [6]. To our knowledge, the regulation of poly(A) and poly(U) lengths during infection has not been previously documented in any positive-strand poly(A)-containing RNA virus. A well-studied example of cytoplasmic regulation of poly(A) tail length occurs in oocyte mRNAs. In the cytoplasm of oocytes, certain mRNAs have relatively short poly(A) tails (20-40 nt) and are translationally inactive. During maturation, the poly(A) tail is elongated to nearly 150 nt by cytoplasmic polyadenylation and the mRNAs become translationally active [7&#8211;9]. Experimental evidence from the current study suggests that coronavirus poly(A) tail length is causally related to RNA translation efficiency. Based on these findings, we propose a process for how coronaviral poly(A) and poly(U) lengths may undergo variation and contribute to the regulation of translation and replication during infection possibly by different mechanisms.\nResults\n\nPoly(A) tail length on total BCoV positive-strand RNA varies during infection\nThe ability to ligate positive-strand viral RNAs head-to-tail and determine poly(A) tail length soon after viral infection has enabled us to measure poly(A) tail length differences as a function of time post-infection (Figure 1A) [6,10]. To characterize these changes, we first determined poly(A) tail length by the same method in the genomes of virus used for inoculum. For this, virus inoculum was prepared in confluent human adenocarcinoma (HRT-18) cells using an MOI of ~1 virus per cell and harvesting supernatant fluids at 48 hours postinfection (hpi). RNA extracted from pelleted BCoV was decapped with tobacco acid pyrophosphatase and the 3&#8217; end of the positive-strand viral RNA was ligated to the 5&#8217; end of the positive-strand viral RNA with T4 RNA ligase I as described in Materials and Methods. With the use of a PCR primer set that specifically binds within 5&#8217; and 3&#8217; UTRs of BCoV genome (Figure 1A), RT-PCR was carried out and the resulting products were sequenced to determine the precise viral poly(A) tail lengths. The results show that the poly(A) tail length in the virus contained in the inoculum was ~45 nt (Figure 1B).To determine poly(A) tail length on total intracellular viral RNA as a function of time post-infection, the characterized inoculum containing an infectious titer of ~106 PFU/ml was used to infect freshly-confluent HRT-18 cells with an MOI of ~10 viruses per cell and total cellular RNA was extracted at the time points indicated in Figure 2A. Total RNA was used for head-to-tail ligation and viral poly(A) tail length was determined as described above. As shown in Figure 2A, the length of RT-PCR products ranged from ~250 to ~300 base pairs (bp) and displayed an increasing then decreasing size pattern throughout the 144 h period of infection. Sequencing showed the length of viral poly(A) represented in the major population of viral RNA varied from ~45 nt early in infection (0-2 hpi) to ~65 nt later (4-12 hpi) and then gradually decreased to ~40 nt over time (12-48 hpi) (Figure 2B). The length of viral poly(A) tail became ~<30 nt after 144 h of infection. The data together suggest that the poly(A) tail length on total positive-strand BCoV RNA is regulated during infection.\nThe 5\u2019-terminal poly(U) length on total BCoV negative-strand (antigenome and sg negative strand) RNA co-varies nearly in size with the 3\u2019 poly(A) tail on total positive-strand RNA\nThe mechanisms by which the coronaviral poly(A) tail and poly(U) tract are made are unknown. Considering that a difference in the kinetics of appearance between them might yield a clue as to how they are synthesized, the extent and timing of antigenomic poly(U) synthesis were measured and compared with the appearance of poly(A). For this, the same primers were used as for poly(A) measurement but in reverse order (Figure 3A). A major RT-PCR product of between 250 and 300 nt was found throughout infection (Figure 3B), but was not found in a control reaction containing the same primers with a mixture of in vitro-ligated components from mock-infected cell RNA mixed with in vitro synthesized positive-strand subgenomic mRNA 7 transcripts and in vitro synthesized transcripts representing full-length BCoV genomic RNA (Figure 3C, left panel, lane 2). Nor was a product of this size found from a control reaction mixture containing RNA from infected cells at 9 hpi but no exogenously-added primer for the RT-PCR (Figure 3C, right panel, lane 1), or a reaction mixture containing RNA from infected cells at 9 hpi but only primer that binds cellular &#946; actin mRNA for the RT-PCR (Figure 3C, right panel, lane 2). Samples from a complete reaction with RNA from infected cells at 9 hpi served as a size reference (Figure 3C, left panel, lane 1, and Figure 3C, right panel, lane 3). In the experiment depicted in Figure 3B, a faint band migrating with an apparent size of ~400 nt was also found from 4 through 144 hpi, but when gel-purified, cloned and sequenced, it was found to contain the same sequence as the major band and thus appears to be a multimer of unknown structure. These results together suggested the ~250 to 300-nt RT-PCR product was specifically derived from negative-strand viral RNA. Interestingly, the poly(U) sequence length pattern was remarkably similar to that for poly(A) on the positive strand except that the peak length of 65 nt began to decline earlier (9 hpi for poly(U) vs 12 hpi for poly(A)) (compare Figure 2B with Figure 3D). Therefore, the poly(U) tract length was ~40 nt during the first 2 h of infection, increased to ~65 nt from 4&#8211;9 h of infection, and then decreased to ~50 nt by 24 hpi, and ~25 nt by 144 hpi. These results suggest that the length of coronaviral poly(U), like that of poly(A), is regulated during BCoV infection.\nThe poly(A) tail length on subgenomic mRNA 7 and the poly(U) tract length on subgenomic mRNA 7 negative strand co-vary in kinetics and size with those on total viral positive and negative strands\nTo determine the poly(A) and poly(U) sequence lengths on subgenomic mRNA, advantage was taken of the greater abundance and shorter length of subgenomic mRNA 7 from its leader to the poly(A) tail (a distance of 1.8 kb vs 2.4 kb for the next-larger subgenomic mRNA, subgenomic mRNA 6, because of the coronavirus 3&#8217; nested set arrangement of subgenomic mRNAs) [6]. RT-PCR products made in this way, specific for subgenomic mRNA 7 positive and negative strands, were separately made and sequenced [6]. This strategy to characterize the lengths of the poly(A) and poly(U) is depicted in Figure 4A and Figure 4B, respectively. RT-PCR products of ~1.8 kb for subgenomic mRNA 7 positive strand were detected (Figure 4C, left panel). RT-PCR products of ~1.8 kb for subgenomic mRNA 7 negative strand were also detected (Figure 4C, right panel). In the experiment depicted in Figure 4C, the smear of faint bands above and below the 1.8 kb fragment were of unidentifiable mixed sequence as determined by cloning and sequencing. In addition, RT-PCR products of ~1.8 kb for subgenomic mRNA 7 negative strand were found from BCoV-infected cell RNA at 8 hpi (Figure 4D, left panel, lane 1, and Figure 4D, right panel, lane 3), but not from a control reaction which used an in vitro-ligated mixture of mock-infected cell RNA mixed with positive-strand subgenomic mRNA 7 transcript and full-length transcripts representing full-length BCoV genomic RNA (Figure 4D, left panel, lane 2). Nor were they found from RT-PCR reaction mixtures that contained RNA from infected cells at 8 hpi but no exogenously-added primer (Figure 4D, right panel, lane 1), or that contained cellular &#946; actin mRNA-specific primer (Figure 4D, right panel, lane 2). These results indicate that the RT-PCR product shown in Figure 4C, right panel, arose specifically from negative-strand subgenomic mRNA 7. From direct sequencing of the RT-PCR products, the lengths of poly(A) on positive-strand subgenomic mRNA 7 were determined to be ~30, ~64, ~38 and ~34 nt at 2, 8, 24 and 48 hpi, respectively (Figure 4E, left panel), and of poly(U) on subgenomic mRNA 7 negative strands to be ~32, ~61, ~41 and ~29 nt at 2, 8, 24 and 48 hpi, respectively (Figure 4E, right panel). These results show that the length of the poly(A) tail on subgenomic mRNA 7 and of the poly(U) tract on subgenomic mRNA 7 negative strands co-vary closely in kinetics and size with those on total viral positive and negative strands.\nThe poly(A) and poly(U) lengths on replicating defective interfering RNA and its antigenome co-vary in length during infection\nSince it is not technically feasible to apply the same strategy as was used for subgenomic mRNA 7 analysis to determine the lengths of the poly(A) tail and poly(U) tract on the viral genome and antigenome (an analysis of an RT-PCR product of >7 kb would be required), an alternative approach was used. For this, the BCoV defective interfering (DI) RNA, a helper-virus-dependent replicon of 2.2 kilobases in length (Figure 5A) [11] was used. This BCoV DI RNA is a naturally-occurring DI RNA that has been modified with a reporter and extensively analyzed with regard to required cis-acting elements for replication [6,12&#8211;16]. To analyze the poly(A) and poly(U) tail lengths on this molecule, the BCoV DI RNA was modified to carry the MHV 3&#8217; UTR (which was named DI RNA-M, Figure 5A) with which a MHV-specific oligonucleotide primer for RT-PCR analysis could be used [6]. For analysis, BCoV was used as helper virus and DI RNA-M was transfected into BCoV-infected HRT-18 cells. To mimic the kinetics of BCoV natural infection, supernatant fluids containing packaged DI RNA progeny were collected from BCoV-infected DI RNA-M-transfected HRT-18 cells at 48 h posttransfection (hpt) and used as inoculum to infect fresh HRT-18 cells (designated virus passage 1, VP1) [17]. The poly(A) tail length of packaged DI RNA&#8211;M in the VP1 virus used for inoculum was determined to be ~26 nt (Figure 5B) by the same strategy as depicted in Figure 1A except that primer 1 was replaced with primer 5 which specifically binds MHV 3&#8217; UTR (Figure 5A). After inoculation, total cellular RNA was extracted at the indicated time points for VP1 and VP2 as shown in Figure 5C. The length of the poly(A) was determined by sequencing the RT-PCR products from the ligated positive strand RNAs (Figure 5C and E, left panel). To determine poly(U) length, the same primers were used as for poly(A) measurement but in reverse order. RT-PCR products were detected (Figure 5C, right panel) and sequenced to determine poly(U) length (Figure 5E, right panel). Three control reactions were used to determine the specificity of the negative-strand RT-PCR product. Whereas a ~200-nt negative-strand RT-PCR product was found from RNA extracted from BCoV-infected cells at 24 h of VP1 (Figure 5D, left panel, lane 1 and Figure 5D right panel, lane 3), a product of this size was not found from RT-PCR reactions with an in vitro-ligated mixture of BCoV-infected cell RNA with positive-strand input DI RNA-M (Figure 5D, left panel, lane 2) [6]. Nor was it found in an RT-PCR reaction mixture containing RNA from cells infected with DI RNA-M-containing BCoV and no exogenous primer (Figure 5D, right panel, lane 1), or in an identical RT-PCR reaction mixture but with primer that binds cellular &#946; actin mRNA (Figure 5D, right panel, lane 2), indicating that the measured poly(U) length (Figure 5E, right panel) was specifically from negative-strand DI RNA. The overall results, therefore, reveal that the length of both poly(A) and poly(U) was &#65374;27 nt at early times after infection (0-2 hpi), which increased to &#65374;62 nt at 8 hpi, and then decreased to &#65374;28 nt at 48 hpi. Although using a BCoV DI RNA with MHV 3&#8217; UTR could possibly alter the poly(A) and poly(U) lengths when compared with wt BCoV DI RNA, these results still indicate that the length of the poly(A) tail on the hybrid DI RNA positive strand and of the poly(U) tract in the complementary strand, which are presumably the same as on the replicating viral genome, are regulated throughout replication. Furthermore, the kinetics of the poly(A) and poly(U) changes parallel those as measured for the total RNA and sgmRNA 7 described above.\nThe coronaviral poly(A) tail length on BCoV DI RNA, a surrogate for the viral genome, correlates positively with translation efficiency in BCoV-infected HRT-18 cells\nThe link between coronaviral poly(A) tail and translation has not been previously described. On eukaryotic mRNAs, a widely-documented behavior of the 3&#8217;-terminal poly(A) tail is that an increase in its length positively correlates with an increased translation efficiency [18,19]. Such polyadenylation in oocytes, for example, activates translation of dormant mRNAs with short poly(A) tails. Therefore, one possible function of the increasing poly(A) tail length on the coronavirus genome during replication is a condition-dependent enhancement of translation. To test the hypothesis that increasing poly(A) tail length on the viral genome results in increased translation efficiency, BCoV DI RNA constructs with a replication-blocking 5&#8217;-terminal 50-nt deletion [17] but with differing poly(A) tail lengths (Figs. 6A and 6B) were tested. The rationale for this experiment was twofold: (i) Poly(A) tails with distinct lengths of 25, 45, and 65 nt were found on replicating DI RNAs in infected cells (Figure 5E, left panel) (ii). Using replication-incompetent DI RNAs enables an assessment of translation only from transfected DI RNAs with defined poly(A) tail lengths. To determine whether poly(A) tails of differing lengths affect translation efficiency of DI RNA in infected cells, replication incompetent His-tagged BCoV DI RNAs with lengths of 25, 45 and 65 nt were produced in vitro and studied in vivo. For this, equal amounts (3&#181;g) of transcript were transfected into BCoV-infected HRT cells and accumulation of the DI RNA-encoded protein was measured by Western blot analysis (Figure 6C, row 1) and compared with the steady-state expression of cellular &#946; actin (Figure 6C, row 2). The amount of His-tagged protein increased markedly over time with poly(A) tails of 45 and 65 nt, and less so with a poly(A) tail of 25 nt (Figure 6C, row 1, and summarized in Figure 6D). To relate these quantities to the abundance of DI RNA template and to three other RNA species, Northern analyses were carried out on electrophoretically-separated RNA in samples from the same experiment (Figure 6C, rows 3 through 6). The His-tag-encoding replication-defective BCoV DI RNA was measured with a probe specific for the 30-nt reporter sequence in BCoV DI RNA (TGEV(+)) (Figure 6C, row 3) and was found to decrease in abundance for DI RNAs with all three poly(A) tail lengths (Figure 6C, row 3), whereas the N-encoding sgmRNA 7 from helper virus, as measured by an N-specific probe, steadily increased over time in the presence of all three DI RNA constructs (Figure 6C, row 4), and 18S rRNA remained relatively steady throughout (Figure 6C, row 5).One interpretation of these results is that the protein is synthesized more efficiently from constructs with longer poly(A) tails, and although the RNA template from the non-replicating DI RNA degraded with time, the accumulating His-tagged protein persisted in the cell because of a longer half-life. Another interpretation is that the His-tag coding sequence could have recombined with the helper virus genome and become expressed from helper virus sgmRNAs containing this sequence. Recombination of this nature has been demonstrated when experimentally-applied selective pressures are used [20]. To seek evidence of recombination, Northern analysis after infection and transfection was carried out to detect the His-tag-coding sequence in the helper virus genome and subgenomic mRNAs. No evidence of His-tag coding sequence was found in any of the viral sgmRNAs or genome within the 21 h infection period that could explain the expression of the His-tagged protein from this source (Figure 6C, rows 6 and 7, and data not shown). An increase in accumulated His-tagged protein in the presence of a decreasing DI RNA abundance is, therefore, apparently due to a longer half-life of the protein molecule. Therefore, the enhancement of viral translation efficiency by a longer coronavirus poly(A) tail suggests that poly(A) tail length plays a role in the regulation of coronaviral genome translation.\nDiscussion\nIn this study we report that the coronaviral 3&#8217;-terminal poly(A) tail length in total viral RNA, sgmRNA7, and DI RNA is relatively short (~26-45 nt) in infected cells at 0-2 hpi, increases to peak length (~65 nt) at ~6-10 hpi, and gradually decreases in size (~30-45 nt) after ~10 h of infection. An analysis of poly(A) sequences in 16 isolated bacterial colonies at 39 hpi showed tail lengths to range from 39 to 45 nt, suggesting the mean sizes measured by group sequencing is a good reflection of lengths in the major population at the times measured. This finding appears to represent a regulation of polyadenylation in a coronavirus that mimics aspects of cytoplasmic regulation of poly(A) tail length in certain mRNAs during the maturation of oocytes. It suggests also that there may be a regulated control of poly(A)-influenced translation. This behavior, to our knowledge, has not been previously documented in coronaviruses nor in other poly(A) tail-containing positive-strand RNA viruses, although poly(A) addition following transfection of non-polyadenylated MHV MIDI-C defective interfering (DI) RNAs into infected cells has been reported [21]. We also report that the length of 5&#8217;-terminal poly(U) on coronaviral negative-strand RNA co-varies similarly in length in a regulated fashion with the poly(A) tail. These findings have led us to reexamine previously proposed mechanisms for coronaviral poly(A) and poly(U) synthesis [5] and to speculate on the role of these differences in coronavirus replication.The coronaviral poly(U) length at the 5&#8217; end of negative-strand RNA in BCoV as reported here differs from two previous reports [5,6]. (i) In the first [5], the poly(U) on BCoV negative-strand RNA was measured at 8-20 nt in length at 24 hpi vs ~45 nt at 24 hpi as described here. We suggest that the shorter sequence resulted from an experimental approach that favored selection of shorter sequences. In the method used, a radiolabeled oligodeoxynucleotide primer was annealed near the common 5' end of the negative-strand RNAs, extended with reverse transcriptase, ligated head-to-tail, amplified by the PCR, cloned, and sequenced. This method would have favored enrichment of the shorter extended products owing to the secondary structures in the RNA template that contribute to reverse transcriptase pausing, and perhaps also RNA degradation [22&#8211;24]. (ii) In the second it was reported that the poly(U) tract was 30 nt at 12 hpt on the negative-strand of replicating DI RNA in virus-infected cells [6]. In the current study, we used this latter method but with modifications. Most notably, the poly(U) of >40 nt was observed prior to 12 hpi (Figure 3A) and was measured on a replicating DI RNA that had gained entry into cells as a packaged DI RNA in the infecting virion, not as a transfected RNA. We speculate that gaining entry via infection may have presented the DI RNA a more favorable microenvironment for replication.It has been suggested that influenza virus employs a stuttering mechanism in which viral RNA polymerase pauses at a stretch of short U residues located at the 5&#8217; terminus of the (-)-strand viral genomic RNA and moves back and forth over this stretch of U residues to synthesize the poly(A) tail [25&#8211;27]. It has been proposed that the coronavirus also may exploit a stuttering mechanism to generate a viral poly(A) tail from a short poly(U) tract [5]. It might also be possible that the coronaviruses uses an AAUAAA signal or its variations in the 3&#8217; UTR for cytoplasmic polyadenylation [28,29], or alternatively a Musashi protein-dependent polyadenylation as the result of a element of (G/A) U1&#8722;3AGU [30]. However, the current findings that (i) the length of coronaviral poly(U) tract is almost the same as the poly(A) tail (~65 nt), and that (ii) the poly(U) and poly(A) tracts display the same pattern of increased-then-decreased length during infection (Figure 2B, and Figure 3D) lead us to propose an alternate mechanism for polyadenylation and uridylation which is as follows: (i) As with poliovirus [31], coronaviruses use full-length homopolymeric stretches in the positive- and negative-strands for the addition of poly(A) tail and poly(U) tracts at the same times during infection (ii). Lengthening of the poly(A) and poly(U) stretches prior to 12 hpi may be carried out by a stuttering mechanism, although at this time we have no evidence of such a mechanism (iii). The shortening of the poly(A) or poly(U) tracts after 12 hpi may be a function of BCoV replicase or a cellular deadenylase [32]. Taken together, the evidence indicates that coronaviruses may employ more than one mechanism for the regulation of poly(A) and poly(U) lengths during infection.It has been demonstrated that the regulation of polyadenylation in the cytoplasm for specific mRNAs during oocyte maturation is associated with the regulation of translation [7&#8211;9]. In this study, our experimental evidence also shows that coronaviral poly(A) tail length is positively correlated with translation efficiency suggesting translation may be likewise regulated by coronaviral poly(A) tail length. However, unlike cellular mRNA which is only used as a template for translation, the coronavirus positive-strand RNA genome must serve as a template not only for translation but also for negative-strand RNA synthesis. Since (i) translation and negative-strand RNA synthesis cannot occur simultaneously on the same template and (ii) the longer coronaviral poly(A) tail length is preferred for translation, we speculate that one benefit of the regulated poly(A) tail length during coronavirus infection might be that it functions to regulate the processes of viral genome translation and negative-strand RNA synthesis. On the other hand, since BCoV DI RNA with the longer poly(A) tail replicated better [21] and the length of the poly(U) tract is essentially the same as that of the poly(A) tail at the same time (this study), we suggest that the regulated poly(U) length on negative-strand RNA is a critical factor regulating replication. Accordingly, regulation of poly(A) and poly(U) length may function in regulating viral translation and replication as required by the virus for replication. Further studies are needed to test these possibilities.\nMaterials and Methods\n\nPlasmid constructs\nConstruction of pDrep1 and pDI RNA-M (formerly called pDI RNA-2) [6] which encodes BCoV DI RNA and DI RNA-M, respectively, and were kindly provided by David Brian (University of Tennessee, Knoxville, TN), has been described [6,17]. In brief, pDI RNA-M was made by replacing the 288-nt 3&#8217; UTR of BCoV-Mebus in pDrep1 with the 301-nt 3&#8217; UTR of MHV-A59 (GenBank accession no. NC_001846) from fragment G DNA [33].To construct pHisBM-25A, p&#916;50HisBM-25A, p&#916;50HisBM-45A and p&#916;50HisBM-65A, an overlap PCR mutagenesis procedure was used as described [34], but with the appropriate sets of oligonucleotides for poly(A) tail lengths of 25, 45, or 65 As. The overlapping PCR product was cloned into TOPO-XL vector (Invitrogen) and digested with XbaI and MluI. The digested fragment was cloned into XbaI- and MluI-linearized pDrepI to make pHisBM-25A or pD50 to make p&#916;50HisBM-25A, p&#916;50HisBM-45A and p&#916;50HisBM-65A [17].Six cDNA clones representing the full length of the BCoV-Mebus genome, and pNrep1 which encodes the entire subgenomic mRNA 7 [17] were kindly provided by David Brian (University of Tennessee, Knoxville, TN).\nPreparation of viral RNA and DI RNA from infected cells\nThe Mebus strain of BCoV (GenBank accession no. U00235) was plaque-purified three times and grown on a HRT-18 cell line as described [4,35]. BCoV at 3 &#215; 107 PFU/ml to be used for inoculum was prepared by passaging virus three times at a multiplicity of infection 1 PFU per cell. To prepare RNA for determining coronaviral poly(A) and poly(U) lengths, HRT-18 cells at approximately 5 &#215; 106 per 35-mm-diameter dish when 100% confluent were infected with BCoV at a multiplicity of 10 PFU per cell. Total cellular RNA was extracted with TRIzol (Invitrogen) at the different time points post-infection as indicated in each experiment. To prepare coronaviral RNA from purified virus, supernatant from BCoV-infected HRT-18 cells at 48 hpi. was collected and extracted with TRIzol (Invitrogen).To identify the poly(A) tail length on DI RNA-M in packaged virus, supernatant fluids from BCoV-infected and DI RNA-M transfected HRT-18 cells at 48 hpt were collected and extracted with TRIzol (Invitrogen). To prepare RNA for identifying poly(A) and poly(U) lengths on DI RNA-M in infected cells, HRT-18 cells at approximately 4 &#215; 106 per 35-mm-diameter dish at 80% confluency were infected with BCoV (as helper virus) at a multiplicity of 10 PFU per cell and 500 ng of DI RNA-M transcript, obtained as T7 RNA polymerase transcripts of pDI RNA-M linearized at the MluI site [immediately downstream of the poly(A) tail], was transfected into BCoV-infected cells with the use of Lipofectin (Invitrogen) as previously described [36]. Supernatant fluids were harvested at 48 hpt and 500 &#181;l was used to infect freshly confluent HRT-18 cells in a 35-mm dish (virus passage 1, VP1). Total cellular RNA was extracted with TRIzol (Invitrogen) at the indicated time points.\nHead-to-tail ligation of viral RNA and RT-PCR reactions to determine poly(A) and poly(U) tail lengths\nA head-to-tail ligation method previously used to identify terminal features on influenza virus [10] and coronavirus [6] RNAs was employed here with some modifications to determine poly(A) and poly(U) tail lengths. In brief, to determine the poly(A) tail length, 10 &#181;g of extracted total cellular RNA in 25 &#181;l of water, 3 &#181;l of 10X buffer and 10 U of (in 1 &#181;l) tobacco acid pyrophosphatase (Epicentre) were used to de-block the 5&#8242; capped end of the genomic RNA. Following decapping, phenol-chloroform-extracted RNA in 25 ul of water was heat-denatured at 95&#176;C for 5 min and quick-cooled. Then 3 &#181;l of 10X ligase buffer and 2U (in 2 &#181;l) of T4 RNA ligase I (New England Biolabs) were added, and the mix was incubated for 16h at 16&#176;C. For the RT reaction, SuperScript II reverse transcriptase (Invitrogen), which transcribes poly(A) tails and poly(U) tracts of greater than 100 nt with fidelity was used as previously described [37,38]. For RT-PCR, phenol-chloroform-extracted ligated RNA was used for the RT reaction and the resulting cDNA was used for PCR with AccuPrime Taq DNA polymerase (Invitrogen). To determine the length of poly(A) tail on positive-strand coronaviral RNA, primer 2, which binds nt 107&#8211;129 from the 5&#8217; end of BCoV positive strand, was used for RT, and for the PCR, 5 &#956;l of the resulting cDNA mixture was used in a 50-&#181;l PCR with primers 2 and 1, which binds nt 87&#8211;110 from the poly(U) tail on the negative strand of the BCoV 3&#8217; UTR. To determine the length of the poly(U) tails on coronaviral RNA, the same primers were used but in reverse order. To determine the poly(A) and poly(U) tail lengths on subgenomic mRNA 7 from BCoV-infected cells, BCoV leader positive-strand-specific primer 4, which binds nt 8-27 of leader sequence from 5&#8217; end of BCoV positive-strand subgenomic mRNA 7, and BCoV leader negative-strand-specific primer 3, which binds nt 29&#8211;54 of leader sequence from 3&#8217; end of BCoV negative-strand subgenomic mRNA 7, respectively, were used for RT. Five &#181;l of resulting cDNA was used in a 50-&#181;l PCR with primer 3 and primer 4 (Figure 4A and 4B). To examine the DI RNA-M poly(A) and poly(U) tail lengths, primer 6, which binds nt 29&#8211;54 of leader sequence from 5&#8217; UTR of BCoV positive strand, and primer 5, which binds nt 99&#8211;122 from the poly(U) tail on the negative strand of the MHVA59, respectively, were used for RT. Five &#181;l of resulting cDNA was used in a 50-&#181;l PCR with primers 5 and 6. Non-viral RNA target primer used for control reactions was one that binds &#946; actin mRNA (&#946; actin(+): 5&#8217; CAAAGGCGAGGCTCTGTGCTCGC3&#8217;). The resulting 50-&#181;l PCR mixture was heated to 94&#176;C for 2 min, then subjected to 34 cycles of 30 s at 94&#176;C, 30 s at 55&#176;C, and 30 s at 72&#176;C. The predicted poly(A) or poly(U)-containing PCR product was sequenced to determine the length of poly(A) or poly(U) tail.\nWestern blot analysis for \n\nMluI-linearized plasmid DNA constructs &#916;50HisBM-25A, &#916;50HisBM-45A and &#916;50HisBM-65A were transcribed in vitro with the mMessage mMachine T7 transcription kit (Ambion) in a total reaction volume of 50 &#181;l supplemented with 7.5 &#181;l of 30 mM GTP. In vitro transcription was done at 37&#176;C for 60 min, treated with 5 &#181;l Turbo DNAse (Ambion) at 37&#176;C for 30 min, and chromatographed through a Biospin 6 column (Bio-Rad) before use in transfection [16]. For transfection, HRT-18 cells in 35-mm dishes at ~80% confluency (~8 &#215; 105 cells/dish) were first infected with BCoV at a multiplicity of infection of 5 PFU per cell and then transfected at 2 hpi with 3 &#181;g of transcript RNA using Lipofectin (Invitrogen) [17]. Protein samples from cell lysates were harvested from HRT-18 cells at the time points indicated in Figure 6B, electrophoresed on 12% SDS-PAGE gels, and electrotransferred to a nitrocellulose membrane (Amersham Biosciences). Proteins of interest were detected by Western analysis using primary antibody specific to Histidine tag (Serotec) or &#946; actin (Serotec) as primary antibody, and Goat anti mouse IgG conjugated HRPO as secondary antibody (Jackson). Proteins were visualized by Western Lightning&#8482; Chemiluminescence Reagent (PerKinElmer NEL105) and X-ray film (Kodak) [39].\nNorthern assay for intracellular DI RNA, helper virus RNA, N subgenomic mRNA, His-tag-encoding sequence, and 18S rRNA\nThe Northern assay for detecting reporter-containing DI RNAs was performed as described previously [16,17]. Briefly, 3 &#181;g of DI RNA transcript was transfected into HRT cells in 35-mm dishes at ~80% confluency (~8 &#215; 105 cells/dish) which were then infected with BCoV at a multiplicity of infection of 5 PFU per cell. For replication assay, supernatant fluids were harvested at 48 hpt and 500 &#181;l was used to infect freshly confluent HRT-18 cells in a 35-mm dish (VP1). Total cellular RNA was extracted with TRIzol (Invitrogen) at 48 hpi of VP1 and 10 &#181;g of extracted RNA was used for Northern assay. For internal controls, 3 &#181;g of DI RNA transcript was transfected into BCoV-infected HRT cells and total cellular RNA was extracted with TRIzol at 4, 8, and 21 hpt of virus passage 0 (VP0) as indicated in Figure 6C and 10 &#181;g of extracted RNA was used for electrophoresis in a formaldehyde-agarose gel. RNA was transferred from the gel to Nytran membrane by vacuum blotting and blots were probed with 5&#8217;-end 32P-labeled oligonucleotides. For detecting DI RNA specifically, the reporter-detecting oligonucleotide probe named TGEV(+), 5&#8217; CATGGCACCATCCTTGGCAACCCAGA3&#8217;, was used. For detecting viral N subgenomic mRNA, viral M subgenomic mRNA, and the viral RNA portion of DI RNA the oligonucleotide named BCoVN(+), 5&#8217; CCAGAACGATTTCCAAAGGACGCTCT3&#8217;, was used. For detecting the His-tag-encoding sequence, the oligonucleotide His(+), 5&#8217; GTGGTGGTGGTGGTGGTG3&#8217;, was used. For detecting 18S rRNA, the oligonucleotide 18S rRNA(+), 5&#8217; GCCTGCTGCCTTCCTTGGATCTGGTAGCC3&#8217;, was used. The probed blot was exposed to Kodak XAR-5 film at -80&#176;C. For quantitating DI RNA, N subgenomic mRNA and 18S rRNA, probed blots were read with a Packard InstantImager Autoradiography System.",
        "pmid": "23923003",
        "title": "Regulation of Coronaviral Poly(A) Tail Length during Infection",
        "journal_title": "PLoS ONE",
        "authors": "Hung-Yi Wu;Ting-Yung Ke;Wei-Yu Liao;Nai-Yun Chang",
        "doi": "10.1371/journal.pone.0070548"
    },
    {
        "keywords": "",
        "abstract": "ABSTRACT\nMiddle East respiratory syndrome coronavirus (MERS-CoV) is the first highly pathogenic human coronavirus to emerge since severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002. Like many coronaviruses, MERS-CoV carries genes that encode multiple accessory proteins that are not required for replication of the genome but are likely involved in pathogenesis. Evasion of host innate immunity through interferon (IFN) antagonism is a critical component of viral pathogenesis. The IFN-inducible oligoadenylate synthetase (OAS)-RNase L pathway activates upon sensing of viral double-stranded RNA (dsRNA). Activated RNase L cleaves viral and host single-stranded RNA (ssRNA), which leads to translational arrest and subsequent cell death, preventing viral replication and spread. Here we report that MERS-CoV, a lineage C Betacoronavirus, and related bat CoV NS4b accessory proteins have phosphodiesterase (PDE) activity and antagonize OAS-RNase L by enzymatically degrading 2&#8242;,5&#8242;-oligoadenylate (2-5A), activators of RNase L. This is a novel function for NS4b, which has previously been reported to antagonize IFN signaling. NS4b proteins are distinct from lineage A Betacoronavirus PDEs and rotavirus gene-encoded PDEs, in having an amino-terminal nuclear localization signal (NLS) and are localized mostly to the nucleus. However, the expression level of cytoplasmic MERS-CoV NS4b protein is sufficient to prevent activation of RNase L. Finally, this is the first report of an RNase L antagonist expressed by a human or bat coronavirus and provides a specific mechanism by which this occurs. Our findings provide a potential mechanism for evasion of innate immunity by MERS-CoV while also identifying a potential target for therapeutic intervention.",
        "body": "INTRODUCTION\nMiddle East respiratory syndrome coronavirus (MERS-CoV) infections range from mild upper respiratory infections to severe acute respiratory distress syndrome, with a global case fatality rate of 36% (1, 2). MERS-CoV has predominantly affected the Kingdom of Saudi Arabia and neighboring countries with sporadic cases arising in Europe and North America as the result of travel to and from the Middle East (3). A recent outbreak of MERS-CoV in South Korea has raised the specter that unrecognized infections combined with potential superspreaders may pose a much greater risk of significant travel-associated outbreaks of MERS-CoV than previously suspected, particularly in health care settings (4, 5). The lethality of MERS-CoV and the ease of global travel necessitate further study and understanding of the mechanisms of MERS-CoV pathogenesis.MERS-CoV, a lineage C Betacoronavirus, has a 30-kb positive-sense single-stranded RNA (ssRNA) genome. As with all CoVs, the first 5&#8242; two-thirds of the genome consists of the replicase encoded in open reading frame 1a (ORF1a) and ORF1b. The remaining 3&#8242; one-third encodes the structural proteins spike (S), envelope (E), membrane (M), and nucleocapsid (N) as well as accessory proteins (encoded in ORF3 to ORF5 and ORF8b) that are not required for genome replication but likely act as immune antagonists that may be critical for pathogenesis (6&#8211;8). Indeed, MERS-CoV and other CoVs induce type I interferon (IFN) only weakly and late in infection (9&#8211;15), suggesting that coronaviruses have evolved mechanisms of immune evasion. Furthermore, there are reports that MERS-CoV accessory protein NS4a (encoded by ORF4a), NS4b (encoded by ORF4b), and NS5 (encoded by ORF5), when overexpressed from plasmids, antagonize type I IFN induction and signaling at various points in the pathways (16&#8211;19). NS4a is a double-stranded RNA (dsRNA) binding protein (17). NS4b is detected primarily in the nucleus (16&#8211;19), making NS4b and the severe acute respiratory syndrome coronavirus (SARS-CoV) ORF3a-encoded proteins (20, 21) the only known coronavirus proteins thus far to be detected in the nucleus (20, 21). A MERS-CoV mutant with deletions of ORFs 3 to 5 is attenuated for replication in human airway-derived Calu-3 cells (22), and a mutant with deletions of ORFs 4a and 4b is attenuated for replication in hepatic carcinoma-derived Huh-7 cells (23).The type I IFN response is an essential component of antiviral innate immunity and is initiated when host sensors initiate signaling pathways resulting in transcription of type I IFN genes (9, 10, 24). IFN activates transcription of numerous IFN-stimulated genes (ISG); among these and most relevant to this report are the OAS genes (25). Oligoadenylate synthetase (OAS), upon detection and binding of dsRNA, synthesizes 2&#8242;,5&#8242;-oligoadenylate (2-5A) [px5&#8242;A(2&#8242;p5&#8242;A)n; x = 1 to 3; n &#8805; 2] from intracellular ATP that induces the homodimerization of latent RNase L, leading to its subsequent activation (24, 26, 27). Activated RNase L cleaves both viral and host ssRNA preferentially at UU and UA dinucleotide sequences, leading to translational arrest and apoptosis, and limits viral replication and spread in vitro and in vivo (24, 28, 29). In addition, RNA cleavage products can be recognized by RNA sensors, leading to further augmentation of IFN production and signaling (30).We have shown previously that lineage A Betacoronavirus mouse hepatitis virus (MHV) NS2 is a determinant of cellular and organ tropism. MHV NS2 is a 2&#8242;,5&#8242;-phosphodiesterase (PDE) that antagonizes the type I IFN response by blocking activation of the OAS-RNase L pathway and is a critical determinant of MHV hepatovirulence (7, 29). Here we report that by structural homology, biochemistry, and biological measures, MERS-CoV NS4b and homologs encoded by related bat lineage C Betacoronavirus, BtCoV-SC2013 (SC2013) and BtCoV-HKU5 (HKU5), are also 2&#8242;,5&#8242;-PDEs that can antagonize the IFN-inducible OAS-RNase L pathway.\nRESULTS\n\nAlignment and modeling of NS4b proteins.\nTo elucidate the function of MERS-CoV NS4b, we queried the primary amino acid sequence with the National Center for Bioinformatics (NCBI) Basic Local Alignment Search Tool (BLAST). MERS-CoV NS4b was found to have no primary amino acid sequence homology outside of lineage C Betacoronavirus. To broaden the search for homologous proteins, we used tertiary structural prediction and homology search by Phyre2 (31) and I-TASSER (32, 33). The top scoring structure identified by both servers was the central domain of Rattus norvegicus A kinase anchoring protein 7 isoform gamma or delta (AKAP7&#947;/&#948;) (PDB: 2VFK), a 2H-phosphoesterase (2H-PE) superfamily member with 2&#8242;,5&#8242;-PDE activity (34) (Fig.&#160;1A). These enzymes are characterized by two H-&#934;-[ST]-&#934; motifs (where &#934; is a hydrophobic residue) separated by an average of 80 residues (35). To generate a more accurate predicted structure, the 2H-PE domain of MERS-CoV NS4b was modeled directly on AKAP7 using one-to-one threading on Phyre2 followed by loop and side chain refinement in Modeller (Fig.&#160;1B) (29, 31). For comparison, the recently solved structure of lineage A Betacoronavirus mouse hepatitis virus (MHV) NS2, a 2H-PE with 2&#8242;,5&#8242;-PDE activity, is also shown (Fig.&#160;1C) (36).MERS-CoV NS4b shares significant homology with closely related bat coronavirus (BtCoV) proteins encoded by SC2013 and HKU5, 47% and 31% amino acid sequence homology, respectively. While both BtCoV homologs have the 2 [H-&#934;-(S/T)-&#934;] predicted catalytic motifs, SC2013 NS4b (previously referred to as NS3c [37]) had low-confidence structural homology (<45%) with AKAP7, and HKU5 NS4b had no significant structural homology with any structures in the Phyre2 database (38). However, alignment of the regions surrounding the H-&#934;-(S/T)-&#934; motifs with cellular AKAP7 from rat, mouse (Mus musculus, NP_061217.3), and virus-encoded 2&#8242;,5&#8242;-PDEs illustrates the conservation of these two motifs (Fig.&#160;2A). We have previously shown that the central domain of the M. musculus AKAP7&#947; and simian rotavirus A (RVA), strain SA11, VP3 protein carboxy-terminal domain (CTD) are catalytically active 2&#8242;,5&#8242;-PDEs (28, 34, 35, 39). Additionally, MERS-CoV NS4b and BtCoV homologs, like the rat and mouse AKAP7 but unlike any of the known viral PDEs, including MHV NS2 or rotavirus VP3 CTD, contain an amino-terminal nuclear localization signal (NLS) (Fig.&#160;2B) (16, 18, 19). In addition to a PDE domain, AKAP7&#947; also contains a carboxy-terminal binding domain for the regulatory subunit II (RII) of cyclic AMP (cAMP)-dependent protein kinase A (PKA) (PKA-RII-&#945;-BD) (39), and VP3 contains amino-terminal guanylyltransferase (Gtase) and methyltransferase (Mtase) domains (E) (28), neither of which are found adjacent to coronavirus PDE domains.\nNS4b from MERS-CoV and both BtCoVs are enzymatically active 2\u2032,5\u2032-phosphodiesterases that cleave 2-5A.\nTo determine whether MERS-CoV, BtCoV-SC2013, and BtCoV-HKU5 NS4b encode enzymatically active PDEs, NS4b genes were expressed in Escherichia&#160;coli as maltose binding protein (MBP) fusion proteins and purified by affinity chromatography followed by ion exchange and size exclusion chromatography. Purified proteins were incubated with various 2-5A substrate concentrations and subsequently used to assess enzyme kinetics via a fluorescence resonance energy transfer (FRET)-based RNase L activation assay (29). Substrate steady-state kinetic studies were performed (see Materials and Methods for details), and the quantity of cleaved 2-5A was determined as a function of time from 180-min progress curves conducted in triplicate. The best-fit parameter estimates are shown in Table&#160;1. All three proteins were capable of cleaving 2-5A and preventing the activation of RNase L with similar kinetic parameters to MHV NS2. As expected, mutant proteins containing a His-to-Arg mutation in the second catalytic motif were incapable of cleaving 2-5A (Fig.&#160;3A). Although the activities of all three NS4b proteins were similar to that of MHV NS2 (Fig.&#160;3B), velocities plateaued at lower concentrations of 2-5A compared with MHV NS2, suggesting that the lineage C proteins become saturated more quickly at higher 2-5A concentrations (Fig.&#160;3C to E).\nExpression of MERS-CoV NS4b and BtCoV homologs from chimeric MHV.\nTo investigate OAS-RNase L antagonism in cell culture, using the backbone of catalytically inactive NS2 with the H126R substitution (NS2H126R) (MHVMut), we constructed chimeric MHVs that express one of the NS4b proteins, and for each chimeric MHV, we constructed a corresponding catalytically inactive mutant, each with a carboxy-terminal Flag tag (18, 34). In addition, we constructed chimeric MHVs expressing MERS-NS4b with a deletion of the amino-terminal 52&#160;amino acids including the NLS (MHV-MERS&#916;1-52WT) as well as its catalytically inactive mutant MHV-MERS&#916;1-52Mut. Thus, wild-type (WT) or mutant NS4b genes were cloned into the MHVMut genome in place of MHV ORFs 4a and 4b which encode protein(s) with no known function in MHV replication or pathogenesis (40) (Fig.&#160;4). To assess expression of NS4b proteins from chimeric viruses, bone marrow-derived macrophages (BMM) derived from mice genetically deficient in RNase L (RNase L&#8722;/&#8722;) were infected with each of the chimeric viruses in addition to WT MHV and MHVMut expressing catalytically inactive mutant NS2. (RNase L&#8722;/&#8722; BMM were used rather than B6 BMM because an active PDE is required for robust replication in the latter [29].) BMM were lysed at 12&#160;h postinfection, and protein lysates were analyzed by Western immunoblotting with antibodies directed against Flag to detect NS4b proteins, antibodies against viral nucleocapsid to assess viral replication, and antibodies against glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a loading and transfer control. All NS4b proteins were expressed in RNase L&#8722;/&#8722; BMM but to different extents, and mutant NS4b proteins were expressed at lower levels than WT NS4b proteins were, possibly reflecting reduced protein stability as a result of the mutations (Fig.&#160;5A). Similar findings were obtained with chimeric virus infection of 17Cl-1 cells (data not shown).All three NS4b proteins have an amino-terminal tripartite NLS; previous studies reported that MERS-CoV NS4b is predominantly detected in the nucleus (41) and that overexpressed BtCoV-HKU5 NS4b is solely detected in the nucleus (16). Previously, we found that murine AKAP7 was exclusively nuclear and that it could not act as a PDE to rescue replication of chimeric mutant MHV unless the NLS was removed and cytoplasmic expression was achieved (34), so it was important to determine the subcellular localization of NS4b protein during chimeric virus infection. Thus, immunofluorescence staining was carried out to assess expression and subcellular localization of expression of each NS4b protein. Murine L2 cells were infected with chimeric viruses expressing both MERS-CoV NS4b and corresponding CoV homologs, fixed at 8&#160;h postinfection, and stained with anti-Flag antibodies (NS4b) and antibodies against the MHV nucleocapsid protein, which is expressed in the cytoplasm (Fig.&#160;5B). Interestingly, MERS-CoV and BtCoV-SC2013 NS4b proteins localized to the nucleus with expression in the cytoplasm as well, while the BtCoV-HKU5 NS4b homolog was almost completely nuclear, similar to reports of overexpressed NS4b in the literature (16&#8211;18). As expected, MHV-MERS&#916;1-52WT expressing an NS4b with deletion of the NLS was localized to the cytoplasm.\nNS4b proteins rescue replication of MHV\nNS2 expression is necessary for robust replication of MHV in BMM derived from B6 BMM (WT) but is not required in RNase L&#8722;/&#8722; BMM (7, 29). To elucidate the roles of the NS4b proteins in the context of viral infection, we inoculated B6 BMM or RNase L&#8722;/&#8722; BMM with wild-type MHV (MHVWT) or MHV expressing a catalytically inactive NS2 (MHVMut) as well as chimeric MHVMut expressing either WT or mutant MERS-CoV (both full-length and with amino acids 1 to 52 deleted) and BtCoV-SC2013 NS4b proteins (see Fig.&#160;4 for schematics of the chimeric viruses). All three WT NS4b proteins, but not mutant NS4b proteins, were able to rescue MHVMut replication in WT BMM (Fig.&#160;6A to D). Similar results were obtained for the pair of viruses expressing BtCoV HKU5 NS4b proteins (data not shown). MHV expressing MERS-CoV WT NS4b (MHV-MERSWT) replicated to a titer equivalent to that of WT MHV. Viruses expressing the BtCoV homologs replicated well in B6 BMM but not quite to the same extent as WT MHV or MHV-MERSWT, similar to findings with the chimeric viruses expressing PDE of RVA VP3 CTD and cellular AKAP7 protein (28, 34) (Fig.&#160;6D and data not shown). As expected, viruses expressing mutant as well as WT NS4b proteins replicated to high titer in RNase L&#8722;/&#8722; BMM. Although localization of NS4b proteins was primarily nuclear, these results demonstrate that the level of expression of NS4b PDE at low levels in the cytoplasm was sufficient to rescue replication in MHV-MERS- and MHV-SC2013-infected B6 BMM.rRNA degradation during infection is indicative of RNase L activation, and we have previously used this read out as an indirect measure of RNase L activation (29). In addition, we previously demonstrated that enzymatically active PDEs of RVA VP3 and AKAP7, but not catalytically inactive mutant proteins, were capable of preventing rRNA degradation in B6 BMM during MHV chimeric virus infection as determined by analyzing total RNA levels postinfection on an Agilent bioanalyzer. Similarly, MHV-MERSWT, both full-length and MHV-MERS&#916;1-52WT mutant and MHV-SC2013WT clearly prevented RNase L-mediated RNA degradation, while viruses expressing the inactive mutant proteins activated the OAS-RNase L pathway. Surprisingly, BtCOV-HKU5WT (expressed during MHV-HKU5WT infection) was unable to efficiently prevent rRNA degradation despite the fact that it did confer efficient replication in B6 BMM (data not shown). This may be due to the mostly nuclear localization of MHV-HKU5WT, which is discussed further below.\nMERS-CoV NS4b rescues replication of MHV\nTo determine whether MERS-CoV NS4b was capable of rescuing replication of MHVMut\nin vivo, we infected B6 and RNase L&#8722;/&#8722; mice with chimeric viruses expressing either MHV-MERSWT or enzymatically inactive virus (MHV-MERSMut) and compared replication to MHVWT. NS4b, but not NS4bMut, rescued MHVMut replication to MHVWT levels in B6 mice (Fig.&#160;7). Additionally, gross pathology was restored for MHV-MERSWT, which is typically fully abrogated without an enzymatically active NS2, but not for MHV-MERSMut (data not shown). This confirms that MERS-CoV NS4b can act as an RNase L antagonist and contribute to pathogenesis in vivo.\nMERS-CoV inhibits rRNA degradation in a human airway cell line.\nWe constructed a MERS-CoV NS4b deletion mutant (MERS-&#916;NS4b) and mock inoculated or inoculated a human airway epithelial cell line, Calu-3, with MERS-CoV, MERS-&#916;NS4b, or MERS-CoV lacking NS3 to NS5 (MERS-&#916;NS3-5) (22). We measured replication and rRNA degradation through the course of a one-step growth curve. The characteristic signature of RNase L activation in human cells is present during MERS-&#916;NS4b and MERS-&#916;NS3-5 infections at late time points (Fig.&#160;8A) and can be quantified by Agilent RNA integrity number (RIN) and plotted over time (Fig.&#160;8B). The characteristic rRNA degradation signature is not present during MERS-CoV infection or mock infection. We observed no reduction in titer for MERS-&#916;NS4b but nearly a 10-fold reduction in titer for MERS-&#916;NS3-5 compared with wild-type MERS-CoV infection (Fig.&#160;8C). Thus, while MERS-CoV deletion mutants activate RNase L in Calu-3 cells, rRNA degradation was not very extensive compared to that observed in Sindbis virus infection of human A549 cells.The more robust rRNA degradation seen with MERS-&#916;NS3-5 than with MERS-&#916;NS4b suggested that there may be additional mechanisms of RNase L antagonism by MERS-CoV. In fact, NS4a is a dsRNA binding protein that may help sequester viral dsRNA and prevent stimulation of OAS and subsequent activation of RNase L (17, 18). In addition, since NS4a and NS4b are expressed from the same mRNA by an unknown mechanism of initiation of translation, it is possible that deletion of ORF4b could result in increased expression of the upstream ORF encoding NS4a and increased dsRNA binding and inhibition of RNase L, negating the need for a PDE to antagonize RNase L. Thus, we constructed MERS-NS4bH182R, expressing NS4b with an inactive PDE, and assessed its replication and ability to active RNase L in Calu-3 cells. While MERS-NS4bH182R replicated similarly to WT MERS (data not shown), it did promote RNase L degradation and like the deletion mutants only at late times postinfection (Fig.&#160;8D).Thus, MERS-CoV NS4b and BtCoV homologs have 2&#8242;,5&#8242;-PDE activity that antagonizes RNase L activation in vitro and is dependent on a catalytic His residue. In addition, like MHV NS2, lineage C Betacoronavirus NS4b activity is crucial for protection of rRNA integrity and for efficient replication of chimeric MHV in cell culture and in the case of MERS-CoV NS4b, in mice. While RNase L activation in Calu-3 cells is observed with both MERS-CoV deletion mutants, it is clearly less extensive than observed in the chimeric MHV infections of BMM and has less effect on viral replication as discussed below.\nDISCUSSION\nHere we demonstrate that MERS-CoV and related BtCoV NS4b proteins inhibit the OAS-RNase L pathway by enzymatic cleavage of 2-5A, the activator of RNase L, potentially antagonizing IFN signaling. The type I IFN response is an essential component of the antiviral innate immune response and is activated when host sensors detect viral dsRNA, leading to production of type I IFN (9, 10, 24). IFN activates transcription of IFN-stimulated genes (ISG), leading to the development of an antiviral state in the infected cells and in neighboring cells, thereby restricting viral spread. Based on overexpression experiments, it was previously reported that MERS-CoV NS4b and HKU5 NS4b proteins antagonize type I IFN signaling by several different methods, including interaction with Tank binding kinase 1 (TBK1)/I&#954;B kinase &#949; (IKK&#949;), leading to inhibition of IFN regulatory factor 3 (IRF3) phosphorylation, nuclear localization, and subsequent beta interferon (IFN-&#946;) transcription (16, 18, 19). It was recently shown by overexpression of NS4b and an IFN-&#946; promoter reporter plasmid that MERS-CoV NS4b interacts with as yet unknown nuclear targets inhibiting IRF3- and IRF7-induced IFN-&#946; transcription (19). It is important to note that there is little if any information on IFN antagonism during MERS-CoV infection, and none of these reports recognized the enzymatic activity of NS4b proteins. Nevertheless, the previous publications in combination with our current study of the enzymatic activity of NS4b proteins suggest that NS4b proteins are multifunctional with activities in both the cytoplasm and nucleus, consistent with its localization in both cellular compartments.Searches for sequence homology between MERS-CoV NS4b in protein databases were unsuccessful. However, NS4b was predicted by tertiary structural modeling to be a 2H-PE and the fact that it is conserved in all sequenced lineage C Betacoronavirus genomes suggested that NS4b is an enzymatically active 2&#8242;,5&#8242;-PDE (29), as confirmed in this study. Furthermore, NS4b protein antagonizes the OAS-RNase L pathway by cleaving 2-5A and blocking the subsequent activation of RNase L (7, 29). Additionally, closely related lineage C Betacoronavirus BtCoV-SC2013 and BtCoV-HKU5 NS4b proteins are enzymatically active PDEs with similar activity to MERS-CoV NS4b. We have demonstrated that these lineage C proteins can cleave 2-5A in vitro and that their kinetic parameters are similar to those of NS2.Limitation of RNase L activation can have a profound effect on enhancing virus replication and spread as well as leading to decreased IFN production in myeloid cells (7, 29, 42). Indeed, we previously found that the 2&#8242;,5&#8242;-PDE of MHV NS2 is a determinant of myeloid cell tropism and a critical liver virulence factor in mice (29). Thus, to initially assess the effectiveness of NS4b proteins, we utilized the chimeric MHV NS2 mutant virus system that was previously used to demonstrate activity for RVA and AKAP PDEs (28, 34). Lineage C PDEs were expressed from chimeric MHV with an NS2Mut background, encoding an inactive PDE. MERS-CoV NS4b and BtCoV homologs were detected by immunoblotting to different extents, as normalized to the amount of MHV nucleocapsid protein detected, with the mutant proteins less well expressed compared to WT proteins. The different levels of expression of WT proteins and the more efficient expression of the WT PDEs compared to the corresponding mutant proteins was observed in both BMM and murine 17Cl-1 fibroblasts (Fig.&#160;5 and data not shown), suggesting that there may be differences in stability in the WT proteins and that the mutant proteins may be less stable. Nevertheless, we were able to rescue replication in murine BMM with wild-type MHV-MERS, wild-type BtCoV-2013 (Fig.&#160;6), and wild-type BtCoV-HKU5 (data not shown) but not with viruses expressing corresponding mutant PDEs and in the case of MHV-MERSWT, but not MHV-MERSMut, in vivo in the mouse liver as well. Since the NS4b catalytic mutants used here are poorly expressed, they do not provide the ideal control and we cannot rule out the possibility that either one or more would rescue at a higher expression level or that factors other than the PDE activity contribute to rescue. This, however, is highly unlikely, as we know from several previous studies (28, 29, 34) that efficient expression of catalytic mutant PDE does not rescue replication of MHV NS2 mutant virus, while supplying with heterologous PDE does (28, 29, 34).We have previously demonstrated that cytoplasmic localization of PDE activity is necessary to antagonize the OAS-RNase L pathway. Murine AKAP7, which contains an NLS, can rescue only chimeric PDE-deficient MHV in the absence of the NLS (34). Interestingly, it is unnecessary to remove the NLS from the MERS and SC2013 NS4b proteins to achieve rescue of chimeric MHV, and in fact, removal of the N-terminal 52&#160;amino acids, including the NLS, of MERS-CoV NS4b confers no replication enhancement (Fig.&#160;6), suggesting that the NLS neither hinders nor improves OAS-RNase L antagonism. Recombinant HKU5 NS4b clearly has 2&#8242;,5&#8242;-PDE activity (Fig.&#160;3), and MHV-HKU5WT (but not MHV-HKU5Mut) clearly replicates efficiently in B6 BMM (data not shown), demonstrating that HKU5 NS4b can replace the function of NS2 in the MHV chimeric background. However, there was still detectable RNA cleavage in BMM infected with both MHV-HKU5WT and MHV-HKU5Mut (data not shown). The most likely explanation is that BtCoV-HKU5 NS4b does not antagonize RNase L activation as well as MERS-CoV or BtCoV SC2013 NS4b does, because it is more localized to the nucleus, consistent with our findings with the PDE of AKAP7 (34). It is also possible the rRNA is more susceptible to cleavage than viral RNA; therefore, the less effective BtCoV-HKU5 NS4b cannot antagonize RNase L effectively enough to spare rRNA while still protecting viral and/or cellular mRNA.We have demonstrated that MERS-CoV antagonizes OAS-RNase L activation in Calu-3 cells and that mutant viruses with deletions of either NS3 to NS5 or NS4b alone as well as the catalytic mutant MERS-NS4bH182R activate this pathway. Although all mutants induced rRNA degradation, only MERS-&#916;NS3-5 had a reduction in viral titer by 24&#160;h postinfection. Thus, the lack of effect on replication of MERS-&#916;NS4 or MERS-NS4bH182R correlates with the weak rRNA degradation observed and indicates a lack of biological effect in this cell type of deleting or inactivating the NS4b PDE. This may be due in part to the very late times postinfection (24 to 48&#160;h) that we observe rRNA degradation; virus replication is already quite robust by that time, and it may be too late for restriction of replication to occur.We have found previously that cell type is critical for robust activation of RNase L and reduction in viral titers during infection of murine cells and have observed activation of RNase L by MHVMut only in myeloid cells (43) and endothelial cells (unpublished data). The use of Calu-3 epithelial airway cells may explain the lack of restriction of MERS-&#916;NS4b replication and only 10-fold reduction for MERS-&#916;NS3-5, when in murine BMM, abrogation of RNase L antagonism by MHV, which is observed by 9&#160;h postinfection, leads to 100- to 1,000-fold reduction in viral titers. Indeed, RNase L is not readily activated in Calu-3 cells compared to other cell types; the amount of rRNA degradation observed in Sindbis virus-infected A549 cells is clearly more than in Calu-3 cells (Fig.&#160;8A), and we did not observe robust activation of RNase L Calu-3 cells even when transfected with the dsRNA surrogate poly(I&#160;&#8901;&#160;C) (data not shown). Further studies are being directed at identifying a cell type in which PDE activity enhances viral replication.\nMATERIALS AND METHODS\n\nCell lines and mice.\nMurine L2 (L2) and baby hamster kidney cells expressing MHV receptor (BHK-MHVR) were cultured as described previously (7, 44). Calu-3 clone 2B4 cells (a kind gift from Chien-Te Tseng, University of Texas Medical Branch, Galveston, TX) were cultured with Gibco minimum essential medium (MEM) (Thermo Fisher Scientific, Grand Island, NY) supplemented with 20% fetal bovine serum (FBS) (HyClone, GE Healthcare Bio-Sciences, Pittsburg, PA), penicillin-streptomycin (Gibco), and amphotericin B (Fungizone) (Gibco). Vero CCL-81 cells (ATCC, Manassas, VA) were cultured, and the MERS-CoV plaque assay was performed as previously described (22). Human A549 cells were cultured in RPMI 1640 medium (Gibco) supplemented with 10% FBS, 100&#160;U/ml of penicillin, and 100&#160;&#181;g/ml streptomycin. All cells were maintained at 37&#176;C and 5% CO2. B6 mice and RNase L&#8722;/&#8722; mice were bred and maintained in the University of Pennsylvania animal facility, and the protocols were approved by the Institutional Animal Care and Use Committee at the University of Pennsylvania. Primary BMM were derived from bone marrow harvested from the hind limbs (tibia and femur) of 4- to 6-week-old B6 or RNase L&#8722;/&#8722; mice and described previously (29, 45). Cells were cultured in Dulbecco modified Eagle medium (DMEM) (Gibco) supplemented with 10% FBS (HyClone) and 20% L929 cell-conditioned media for 6&#160;days before infection.\nPlasmids and recombinant viruses.\nNS4b sequences for MERS-CoV (GenBank accession no. AFS88939.1), SC2013 (GenBank accession no. AHY61340.1), and HKU5 (accession no. YP_001039965.1) were obtained from NCBI. NS4b cDNA sequences were synthesized with the addition of a 5&#8242; SalI site and 3&#8242; Flag epitope and NotI site and then cloned into pUC57 (Bio Basic, Markham, Ontario, Canada). Site-directed mutagenesis was performed on each NS4b plasmid to generate point mutants (CAC to CGC) containing histidine-to-arginine mutations within the second catalytic motif yielding H182R, H186R, and H183R for MERS-CoV, SC2013, and HKU5 NS4b proteins, respectively, and subsequently sequence verified. Each NS4b gene was then PCR amplified and subcloned in frame with maltose binding protein into pMAL protein expression vector. MHV NS2 and NS2H126R were previously cloned into pMAL parallel-2 vector (29).MHVWT, MHVMut, and all chimeric viruses were constructed with the MHV strain A59 infectious clone (44). Each WT or mutant NS4b cDNA was subcloned into open reading frame 4 (ORF4) of the infectious clone plasmid GEGFP (EGFP stands for enhanced green fluorescent protein), and sequences were confirmed as described previously (28). The full-length cDNAs were assembled with A to E (MHVWT), F (MHVMut), and G (NS4b) fragments, and RNA was in vitro transcribed and recovered as previously described (44). Briefly, infectious clone fragments were excised from their respective plasmids by restriction enzyme digestion. The fragments were gel purified and ligated in vitro to create a full-length cDNA. cDNA was in vitro transcribed with mMessage mMachine T7 transcription kit (Ambion; Thermo Fisher, Grand Island, NY), generating full-length genomic RNA. Each genomic RNA transcript was split into two distinct pools and electroporated into BHK-MHVR along with N-protein transcript using Gene Pulser II (Bio-Rad, Hercules, CA). Electroporated cells were incubated until cytopathology was evident throughout, freeze thawed three times, and plaque purified, and one plaque from each pool was utilized in parallel for all assays.MERS-&#916;NS4b was constructed using the MERS-CoV infectious clone (22). To delete NS4b, sequence was synthesized (Bio Basic) in which mutations were made in the overlap region between NS4a and NS4b to abrogate putative start codons (T2C and T17C; all positions relative to the NS4b start) and to insert premature stop codons (A91T, C97A, and C102G). Additionally, residues 106 to 669 were deleted to remove the majority of NS4b but leave in place transcriptional regulatory sequence 5. The truncated NS4b cassette and MERS-CoV infectious clone F plasmid were digested with PacI and SanDI, and truncated NS4b was ligated into the MERS-CoV F plasmid. To construct the MERS-NS4bH182R mutant, two PCRs were performed using the F plasmid as the template to synthesize overlapping DNA fragments and introducing an H182R substitution. These two templates were then joined in an overlapping extension PCR. The resultant product was digested with PacI and SanDI and cloned into the MERS F plasmid. Assembly of the infectious clone and recovery of infectious virus were performed as described previously (22). The potential gain-of-function (GOF) concerns of MERS-&#916;NS4b and MERS-NS4bH182R were evaluated and reviewed by NIH under grant U19AI107810 and approved for continued study. The MERS-&#916;NS3-5 mutant (22) was generated prior to GOF regulations. The MERS-&#916;NS3-5 and catalytic mutant display attenuated growth on select interferon-competent human cell lines.\nFRET assay and enzyme kinetics.\nNS4b of MERS-CoV, SC2013, and HKU5 were cloned into pMal parallel-2 vector (46) and expressed in BL21 T7 express competent E.&#160;coli (NEB, Inc., Ipswich, MA) as maltose binding protein (MBP) fusion proteins and purified by affinity chromatography followed by ion exchange chromatography on MonoQ GL10/100 using a NaCl gradient from 0 to 1 M in 20&#160;mM HEPES (pH&#160;7.2) and gel filtration in 20&#160;mM HEPES (pH&#160;7.2) containing 100&#160;mM NaCl as described earlier for purification of MHV NS2 (29). Briefly, purified proteins (10&#160;&#181;M MBP as control or 1&#160;&#181;M concentration of MBP fusion proteins with NS4b of MERS-CoV, SC2013, and HKU5 or their catalytically inactive mutants described in the previous section) in 150&#160;&#181;l of assay buffer (20&#160;mM HEPES [pH&#160;7.2], 10&#160;mM MgCl2, 1&#160;mM dithiothreitol) were incubated at 30&#176;C with various concentrations of (2&#8242;-5&#8242;)p3A3. Aliquots of 20&#160;&#181;l were withdrawn at different times, and the reactions were stopped by heat inactivation at 95&#176;C for 3&#160;min followed by 30-min centrifugation at 20,000 &#215;&#160;g (4&#176;C) and supernatants were carefully removed. A fluorescence resonance energy transfer (FRET)-based RNase L activation assay was used to determine enzyme kinetics by measuring the uncleaved, intact (2&#8242;,5&#8242;)p3A3 as previously described (47). The amounts of 2-5A cleaved were determined by the relative fluorescence units (RFU) using standard curves with different concentrations of authentic 2-5A. The reaction velocities were determined by nonlinear fit of the data. The kcat and Km were determined in GraphPad Prism 5.0 for Windows by plotting velocity against substrate concentrations, and data were analyzed by the following equation: Y = Et &#215; kcat &#215; [X/(Km + X)], where Y is the velocity of the reaction in micromolar per second, X is the substrate concentration in micromolar, and Et is the concentration of enzyme catalytic sites. The kinetics were determined in triplicate reactions.\nNone\nBMM were mock inoculated or inoculated with MHVWT, MHVMut, MHV-MERSWT, MHV-MERSMut, MHV-SC2013WT, MHV-SC2013Mut, MHV-HKU5WT, or MHV-HKU5Mut. Virus was added to cells at a multiplicity of infection (MOI) of 1&#160;PFU/cell and allowed to adsorb for 1&#160;h at 37&#176;C. Cultures were washed with phosphate-buffered saline (PBS) (three times) and fed with medium. At the times indicated in the figures, cells were fixed and analyzed for protein expression by immunofluorescent staining, lysed, and analyzed for protein expression by immunoblotting or analyzed for degradation of RNA, or supernatants were harvested for quantification of viral titers by plaque assay on L2 cells (48).\n(ii) MERS-CoV infections of Calu-3 cells.\nPrior to infection, Calu-3 cells were washed once with PBS (Gibco) to remove residual fetal bovine serum (FBS). Cells were inoculated under biosafety level 3 (BSL3) conditions with MERS-CoV, MERS-&#916;NS4b, MERS-NS4bH182R, or MERS-&#916;NS3-5 at an MOI of 1&#160;PFU/cell and allowed to absorb for 40&#160;min at 37&#176;C. The inoculum was then removed, and the cells were washed twice with PBS prior to adding media containing low FBS (4%). At the time points indicated in the figures, 100&#160;&#181;l of medium was collected and subsequently analyzed by plaque assay in Vero CCL-81 cells as previously described (22), and total RNA was harvested in 1&#160;ml Trizol.\nImmunoblotting.\nAt 12&#160;h postinfection, cells were lysed in Nonidet P-40 (NP-40) buffer (1% NP-40, 2&#160;mM EDTA, 10% glycerol, 150&#160;mM NaCl, and 50&#160;mM Tris [pH&#160;8.0]) containing protease inhibitors (Roche). Protein concentrations were measured using a DC protein assay kit (Bio-Rad). Supernatants were mixed 3:1 with 4&#215; SDS-PAGE sample buffer. Samples were boiled, separated by 4 to 15% SDS-PAGE, and transferred to polyvinylidene difluoride (PVDF) membranes. Blots were blocked with 5% nonfat milk and probed with the following antibodies: anti-Flag M2 mouse monoclonal antibody (Agilent) (1:1,000), anti-N mouse monoclonal antibody (a gift from Julian Leibowitz) (1:400), and anti-GAPDH mouse monoclonal antibody (Thermo Fisher Scientific) (1:1,000). Anti-mouse secondary antibodies labeled with horseradish peroxidase (HRP) (Santa Cruz; 1:5,000) were used to detect the primary antibodies. The blots were visualized using SuperSignal West Dura extended-duration substrate (Thermo Scientific). Blots were probed sequentially with antibodies with the blots stripped between antibody treatments.\nIndirect immunofluorescence assay.\nL2 cells were infected at an MOI of 1&#160;PFU/cell for 8&#160;h with MHVWT, MHV-MERSWT, MHV-SC2013WT, or MHV-HKU5WT. Cells were washed with Dulbecco&#8217;s phosphate-buffered saline (DPBS) (Gibco) and then fixed with 4% paraformaldehyde in DPBS (Gibco). After fixation, cells were permeabilized with 0.1% Triton in DPBS (Gibco) and then blocked with bovine serum albumin. Cells were then stained with mouse monoclonal anti-nucleocapsid (1:200) and rabbit anti-Flag polyclonal (Sigma) (1:2,000) antibodies. After the cells were washed, secondary staining with Alexa Fluor 488-labeled goat anti-mouse (1:400) and Alexa Fluor 594-labeled goat anti-rabbit (1:1,000) antibodies was performed. The cells were washed once more, and the nuclei were then stained with 4&#8242;,6&#8242;-diamidino-2-phenylindole (DAPI). Stained cells were imaged at a magnification of &#215;400 on an Eclipse TE2000-U fluorescence microscope (Nikon Instruments, Inc.). Images were acquired using NIS-Elements Basic Research microscope imaging software (Nikon Instruments, Inc.). This was performed at least twice for all viruses.\nQualitative and quantitative analyses of RNase L-mediated rRNA degradation.\nRNA from B6 BMM infected with MHVWT, MHVMut, MHV-MERSWT, MHV-MERSMut, MHV-SC2013WT, MHV-SC2013Mut, MHV-HKU5WT, or MHV-HKU5Mut was harvested with an RNeasy kit (Qiagen, Valencia, CA) 15&#160;h postinoculation. Alternatively, Calu-3 cells infected with MERS-CoV, MERS-&#916;NS4b, MERS-&#916;NS3-5, or MERS-NS4bH182R, and at the time points indicated in the figures, RNA was harvested with Trizol (Sigma Aldrich, St. Louis, MO), extracted with chloroform (Sigma), and precipitated with isopropanol (Sigma). RNA was then separated and analyzed on RNA nanochips with an Agilent Bioanalyzer 2100 (Agilent Technologies, Santa Clara, CA), and the pseudogels were normalized for concentration (49). RNA integrity numbers (referred to as RIN values) (50) are measurements of RNA integrity produced by the Agilent Bioanalyzer. RNA from A549 cells infected with Sindbis virus (51) was used as a marker for the pattern of human RNase L-dependent rRNA cleavage products.\nInoculation of mice.\nFour-week-old B6 or RNase L&#8722;/&#8722; mice were anesthetized with isoflurane (Abbott Laboratories, Chicago, IL) and inoculated intrahepatically with MHVWT, MHVMut, MHV-MERSWT, or MHV-MERSMut in 50&#160;&#181;l of DPBS (Gibco) containing 0.75% bovine serum albumin (Sigma). Mice were humanely euthanized with CO2 and perfused with DPBS (Gibco), and their livers were harvested at day 5 postinoculation. Viral titers were determined by plaque assay of lysates (48). This study was carried out in strict accordance with the Guide for the Care and Use of Laboratory Animals (41) and the Public Health Service Policy on Humane Care and Use of Laboratory Animals (52).",
        "pmid": "27025250",
        "title": "Middle East Respiratory Syndrome Coronavirus NS4b Protein Inhibits Host RNase L Activation",
        "journal_title": "mBio",
        "authors": "Joshua M. Thornbrough;Babal K. Jha;Boyd Yount;Stephen A. Goldstein;Yize Li;Ruth Elliott;Amy C. Sims;Ralph S. Baric;Robert H. Silverman;Susan R. Weiss",
        "doi": "10.1128/mBio.00258-16"
    },
    {
        "keywords": "convoluted membranes;electron tomography;membrane structure;nidoviruses;nonstructural proteins;replication complex;replication organelle biogenesis;replication structures;viral factory;viral protein",
        "abstract": "ABSTRACT\nBetacoronaviruses, such as Middle East respiratory syndrome coronavirus (MERS-CoV), are important pathogens causing potentially lethal infections in humans and animals. Coronavirus RNA synthesis is thought to be associated with replication organelles (ROs) consisting of modified endoplasmic reticulum (ER) membranes. These are transformed into double-membrane vesicles (DMVs) containing viral double-stranded RNA and into other membranous elements such as convoluted membranes, together forming a reticulovesicular network. Previous evidence suggested that the nonstructural proteins (nsp&#8217;s) 3, 4, and 6 of the severe acute respiratory syndrome coronavirus (SARS-CoV), which contain transmembrane domains, would all be required for DMV formation. We have now expressed MERS-CoV replicase self-cleaving polyprotein fragments encompassing nsp3-4 or nsp3-6, as well as coexpressed nsp3 and nsp4 of either MERS-CoV or SARS-CoV, to characterize the membrane structures induced. Using electron tomography, we demonstrate that for both MERS-CoV and SARS-CoV coexpression of nsp3 and nsp4 is required and sufficient to induce DMVs. Coexpression of MERS-CoV nsp3 and nsp4 either as individual proteins or as a self-cleaving nsp3-4 precursor resulted in very similar DMVs, and in both setups we observed proliferation of zippered ER that appeared to wrap into nascent DMVs. Moreover, when inactivating nsp3-4 polyprotein cleavage by mutagenesis, we established that cleavage of the nsp3/nsp4 junction is essential for MERS-CoV DMV formation. Addition of the third MERS-CoV transmembrane protein, nsp6, did not noticeably affect DMV formation. These findings provide important insight into the biogenesis of coronavirus DMVs, establish strong similarities with other nidoviruses (specifically, the arteriviruses), and highlight possible general principles in viral DMV formation.",
        "body": "INTRODUCTION\nCoronaviruses are positive stranded RNA viruses that can pose serious zoonotic threats to human health, as evidenced by the emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002 (1, 2) and, more recently, the Middle East respiratory syndrome coronavirus (MERS-CoV). Since the start of the outbreak in 2012, MERS-CoV has continued to circulate in the Arabian Peninsula (3, 4), which to date has led to over 2,000 laboratory-confirmed human infections with a lethality rate of about 35% (http://www.who.int/emergencies/mers-cov/en/). Coronaviruses, members of the order Nidovirales, have the largest known positive stranded RNA genomes, ranging from 26 to 33.5&#160;kb (5&#8211;7). The 5&#8242;-proximal two-thirds of the genome contains the replicase gene that consists of two open reading frames (ORF1a and ORF1b). ORF1a translation yields polyprotein 1a (pp1a; roughly 4,000 to 4,500&#160;amino acid [aa] residues long), which, following a &#8722;1 ribosomal frameshift, can be extended with the ORF1b-encoded polyprotein to yield pp1ab (6,700 to 7,200&#160;aa residues in total). The pp1a and pp1ab polyproteins contain the enzymes of the RNA-synthesizing complex that drives viral genome replication and subgenomic mRNA synthesis (8). The replicase polyproteins are co- and post-translationally processed into 15 or 16 nonstructural proteins (nsp&#8217;s) by two or three ORF1a-encoded proteases (9&#8211;13). Depending on the coronavirus, one or two papain-like proteases (PLpro) that reside in nsp3 process the part of the polyproteins upstream of nsp4. In all coronaviruses, the region downstream of nsp4 is cleaved by the 3C-like cysteine protease or main protease (Mpro) located in nsp5 (Fig.&#160;1A) (9&#8211;13).Coronaviruses, like all positive stranded RNA viruses of eukaryotes, hijack intracellular membranes to form their replication organelles (ROs) (14&#8211;18). These generally reside in the perinuclear region of the cell and are assumed to constitute microenvironments that promote viral RNA synthesis while possibly shielding replicative intermediates, specifically double-stranded RNA, from detection by the innate immune system. The most prominent membrane structures induced after coronavirus infection are double-membrane vesicles (DMVs) (19&#8211;28), which appear to contain double-stranded RNA, a frequently used marker of positive stranded RNA virus replication (19, 29). DMVs are not only formed during the replication of coronaviruses but are also a central component of the ROs induced by several other plus-stranded RNA viruses such as hepatitis C virus (HCV) and enteroviruses like poliovirus and coxsackievirus (30&#8211;32). Most of our current knowledge of coronavirus ROs has been gained through electron microscopy (EM) studies of members of the genus Betacoronavirus, which include SARS-CoV, MERS-CoV, and mouse hepatitis virus (MHV) (19&#8211;24). Electron tomography (ET) studies of SARS-CoV-infected cells showed that DMV outer membranes are often interconnected and also connect to the endoplasmic reticulum (ER) and/or another virus-induced structure called convoluted membranes (CM) (19). Together, they form an elaborate reticulovesicular network (RVN), for which the ER probably serves as the membrane donor (19).Given their membrane-spanning features, the nsp3, nsp4, and nsp6 subunits of the coronavirus replicase are the prime candidates for directing RO formation (Fig.&#160;1A) (17, 33). Each of these proteins spans the membrane multiple times (2, 4, and 6 times, respectively), and they have 1, 2, and 3 luminal loops, respectively, with both nsp3 and nsp4 having a large luminal domain (34&#8211;37). Mutagenesis studies showed that the first luminal loop of MHV nsp4 is critical for viral replication (38, 39). Furthermore, nsp4s of both MHV and SARS-CoV contain sites (2 and 1, respectively) for N-linked glycosylation in the first luminal loop of nsp4 (34, 37, 40). When both these sites were mutated in MHV nsp4 (38, 40), the virus was attenuated in cell culture and DMV formation was impaired, suggesting that nsp4 plays a critical role in coronaviral RO formation. The combined membrane-spanning regions of these proteins (i.e., including all luminal loops and flanking transmembrane domains) are commonly referred to as TM1, TM2, and TM3, respectively. nsp3, nsp4, and nsp6 are nonconventional transmembrane proteins in the sense that they are derived from a polyprotein and do not contain N-terminal signal sequences for cotranslational membrane insertion. It is currently unknown how their membrane insertion is accomplished and whether polyprotein cleavage precedes (or is perhaps required for) translocation across the ER membrane. To a certain extent, nsp2, nsp3, and nsp5 of the distantly related arteriviruses (also members of the order Nidovirales) can be considered equivalent to coronavirus nsp3, nsp4, and nsp6, in terms of their relative position in the replicase polyprotein and their membrane-spanning properties. For arteriviruses, expression of nsp2 and nsp3 alone was necessary and sufficient for the formation of double-membrane structures strikingly resembling the DMVs observed in infected cells (41). Coexpression of nsp5 reduced the size of the induced DMVs but did not change their overall architecture (18). In the case of coronaviruses, it was recently reported that the transient coexpression of SARS-CoV nsp3, nsp4, and nsp6 led to the formation of DMVs (42). Cells coexpressing nsp3 and nsp4 alone contained so-called maze-like bodies (MLBs), consisting of paired ER membranes (zippered ER) and some circular profiles that were interpreted as cross sections of double-membrane tubules. Therefore, it was concluded that nsp6 is essential for the biogenesis of SARS-CoV DMVs, whereas nsp3 and nsp4 can mediate the pairing of membranes that are likely an intermediate in DMV formation (42).In the current study, we examined the role of MERS-CoV nsp&#8217;s in betacoronavirus RO biogenesis. Using EM and ET, we found that MERS-CoV nsp3 and nsp4, either coexpressed from separate plasmids or expressed as a self-cleaving polyprotein fragment (nsp3-4), are essential and sufficient for the formation of DMVs that assemble into an RVN. Addition of the third transmembrane subunit of the MERS-CoV replicase, nsp6, did not alter the overall morphology of the induced DMVs. When nsp3-4 polyprotein processing was prevented by mutagenesis, this blocked the formation of DMVs while membrane pairing did still occur, strongly suggesting that proteolytic processing coordinates DMV formation in time and/or space. To compare our results for MERS-CoV with the previous work on SARS-CoV (42), we used ET to analyze the three-dimensional (3D) structure of the maze-like bodies induced upon coexpression of SARS-CoV nsp3 and nsp4 and were thus able to conclude that the circular profiles observed in that setting in fact correspond to DMVs rather than tubules. This established that, also in the case of SARS-CoV, coexpression of nsp3 and nsp4 suffices to induce DMV formation. Together, our results provide important new insights regarding the biogenesis of coronavirus ROs and demonstrate the conservation of certain principles underlying RO formation, both among the coronaviruses and in comparison to more distantly related members of the order Nidovirales.\nRESULTS\n\nMERS-CoV nsp3 and nsp4 colocalize in the perinuclear region of the cell.\nTo study whether the transmembrane nsp&#8217;s of MERS-CoV are able to induce DMV formation, we expressed nsp3 and nsp4 from a CAG promoter (43) either by cotransfection of cells with plasmids encoding individual proteins or by transfection with a single plasmid encoding a self-cleaving nsp3-4 polyprotein fragment (Fig.&#160;1A; Table&#160;S1). Constructs were codon optimized for expression in human cells, potential splice sites were eliminated, and the encoded proteins were equipped with hemagglutinin (HA), myc, or V5 tags at their termini. The constructs were transfected into 293T cells to verify protein expression and processing (Fig.&#160;1B). The wild-type nsp3-4 polyprotein was fully cleaved into mature nsp3 and nsp4, as was previously described (44). As a control, a mutant in which the nsp3/nsp4 cleavage site was inactivated (G2739A/G2740A; GG>AA) (45) was included to generate the noncleaved precursor. Interactions between nsp3 and nsp4 were previously shown to occur for MHV and SARS-CoV (46, 47), and we assessed whether this was also the case for the corresponding MERS-CoV proteins. To this end, 293T cells were transfected with a construct expressing HA-nsp3-myc or nsp4-V5 or cotransfected with both constructs. Expression products were labeled metabolically with [35S]methionine and [35S]cysteine and subsequently immunoprecipitated with either HA- or V5-specific antibodies (Fig.&#160;1C). Upon immunoprecipitation with the HA-specific antiserum, nsp4-V5 was brought down when HA-nsp3 was present (left panel). Conversely, when using the V5-specific antibody, HA-nsp3 was coimmunoprecipitated when nsp4-V5 was present (right panel). These findings demonstrated that these two MERS-CoV proteins interact and further supported the notion that this is a common feature of coronaviruses.When using immunofluorescence microscopy, separate expression of nsp3 or nsp4 in HuH-7 cells yielded a reticular labeling pattern, with some more-intense foci in the perinuclear region of the cell, suggesting that&#8212;in the absence of the other&#8212;either protein localized at least partially to the ER (Fig.&#160;1D). This reticular pattern (but without the foci) has been described previously upon transient expression of MHV and SARS-CoV nsp4 (34, 48), whereas full-length SARS-CoV nsp3 was reported to localize to foci similar to those that we observed (42). When coexpressing MERS-CoV nsp3 and nsp4 or when expressing the self-cleaving nsp3-4 polyprotein, the reticular pattern was much less pronounced and the two proteins mainly colocalized in foci in the perinuclear region (Fig.&#160;1D, lower panels). This was in agreement with the finding that MERS-CoV nsp3 and nsp4 interact and suggested that this interaction strongly promotes their recruitment to the foci in the perinuclear region.\nMERS-CoV nsp3 and nsp4 are required and sufficient to induce DMV formation.\nThe next step was to determine whether nsp3 and nsp4 could induce the formation of double-membrane structures similar to those observed during infection. As a reference, MERS-CoV-infected HuH-7 cells were analyzed by EM. The membrane structures that were previously described in high-pressure frozen and freeze-substituted Vero cells infected with MERS-CoV (21) were readily apparent at 10&#160;h postinfection (p.i.) in chemically fixed HuH-7 cells (Fig.&#160;2A). Numerous DMVs were found (red asterisks), often adjacent to areas containing CM. The DMV interior appeared electron translucent, a difference from cryofixed samples (21) that can likely be attributed to the different sample preparation method, as the contents of CoV-induced DMVs are easily lost upon chemical fixation (22, 24, 28). Occasionally some smaller circular profiles were observed that seemed similar in size to the spherules recently described for the gammacoronavirus infectious bronchitis virus (IBV) (red arrows) (28). None of these structures was found in mock-infected control samples (Fig.&#160;2B).When HuH-7 cells expressed either nsp3 or nsp4, areas containing modified membranes were observed, which likely corresponded to the foci observed in fluorescence microscopy (Fig.&#160;1D). In nsp3-expressing cells (Fig.&#160;2C), we detected large regions, usually several micrometers in diameter, of disordered membrane bodies (DMBs), which were similar to those previously observed after SARS-CoV nsp3 expression (42). The membrane structures clustering in these DMBs were reminiscent of the surrounding ER cisternae, with which they were frequently connected, suggesting that DMBs consisted of clustered ER-derived membranes. Upon expression of MERS-CoV nsp4, large clusters of modified single membranes (MSM) were observed (Fig.&#160;2D), but these structures seemed more irregular than those induced by nsp3 (Fig.&#160;2C). The expression of SARS-CoV nsp4 did not result in changes in intracellular membrane morphology (42), in contrast with our present observations following MERS-CoV nsp4 expression. Whether this reflects differences between the experimental setups used or an actual difference between these viral proteins remains to be determined.When MERS-CoV nsp3 and nsp4 were expressed in the same cell, either by cotransfection or by expression of the self-cleaving nsp3-4 polyprotein, a remarkably different set of membrane structures was observed (Fig.&#160;2E and F). A combination of circular double-membrane profiles (red asterisks) and paired membranes (red arrows) was present in both cases, suggesting that the combined expression of MERS-CoV nsp3 and nsp4 is sufficient to induce DMV formation. There was no apparent difference between the structures resulting from coexpression of nsp3 and nsp4 and those resulting from expression of the self-cleaving nsp3-4 polyprotein (Fig.&#160;2E and F), but in both cases the circular profiles were significantly smaller than the ones observed in MERS-CoV-infected cells (average diameters of 146 and 148 nm, respectively, versus 252&#160;nm in infection) (see Fig.&#160;S1 in the supplemental material).These membrane modifications were frequently found in all the samples analyzed. In order to further investigate how the frequency of EM-positive cells compared to the transfection efficiency, a quantitative analysis was carried out on samples of cells expressing MERS-CoV nsp3-4. Immunofluorescence microscopy showed that approximately 40% of the transfected cells were positive for expression of MERS-CoV nsp3 and nsp4 (n = 174), while around 19% of the cell sections (n = 288) contained double-membrane structures. Both DMVs and zippered ER clustered together in all the EM positive cell sections, although at slightly different ratios (see Fig.&#160;S2 for a gallery). As the EM analysis was based on one random section per cell (~100&#160;nm thick) that may not always capture the region with membrane modifications, it was not surprising that the fraction of positive cells observed in EM was smaller than that observed in whole cells using light microscopy. The numbers above in fact strongly suggest that the formation of DMVs and zippered ER is induced in most, if not all, cells expressing MERS-CoV nsp3 and nsp4.While the circular profiles observed were suggestive of double-membrane vesicle formation, they could also correspond to cross sections of double-membrane tubular structures. To resolve this issue, we obtained 3D reconstructions of these membrane structures using ET (Fig.&#160;3; Movies&#160;S1 and S2), which confirmed that genuine DMVs were indeed formed upon expression of either nsp3 plus nsp4 or nsp3-4 of MERS-CoV. The distinctive feature that unambiguously identifies a vesicle in a tomogram is a circular profile that is largest at the vesicle&#8217;s equator and decreases in diameter when moving up or down from that plane through successive tomographic slices until, if the vesicle is fully contained in the section, it disappears. Indeed, many profiles like this were observed in the tomograms (Fig.&#160;3A, red asterisks; Movies&#160;S1 and S2, green dots). We found no openings connecting the DMV interior and the cytosol, similar to what was observed previously upon tomographic analysis of coronavirus-infected cells (19, 28).The tomograms corroborated the structural similarity between the membrane structures induced by cotransfection with nsp3 and nsp4 constructs and by expression of the self-cleaving nsp3-4 polyprotein. The electron density of the DMV interior seemed similar to that of the surrounding cytoplasm, and in this sense, it was different from that of DMVs in MERS-CoV-infected cells (compare to Fig.&#160;3A), which is likely due to the absence of other viral proteins and double-stranded RNA. In some cases, DMVs appeared to be contained in a larger double-membrane structure (Fig.&#160;3B, red asterisks). Such structures have not been observed in coronavirus-infected cells. The paired membranes were often continuous with ER cisternae (Fig.&#160;3B) and resembled the so-called zippered ER that has also been observed in IBV-infected cells (28), although they have not been documented so far for betacoronavirus-infected cells. These paired membranes may represent an intermediate of DMV biogenesis. Further supporting this explanation, structures in which the zippered ER seemed to transform into a nascent DMV could readily be observed in the tomograms (Fig.&#160;3B, red arrows). We also observed DMV-DMV, DMV-zippered ER, and DMV-ER connections (Fig.&#160;3C, red arrows), whereas completely isolated DMVs were in fact rare.In summary, while the described differences between nsp3-4-expressing and MERS-CoV-infected cells suggest that other viral components may modulate the process of DMV formation and would be required to form the full array of membrane structures observed during infection, our results establish that MERS-CoV nsp3 and nsp4 are sufficient to trigger all the membrane-remodeling steps required for inducing DMV formation, likely through the transformation of ER membranes into an RVN consisting of DMVs and modified ER.\nMERS-CoV nsp6 does not alter DMV morphology.\nThe DMVs induced by expression of MERS-CoV nsp3 and nsp4 largely mimicked those observed during infection. However, the additional RVN elements that have been observed in this and previous studies of coronavirus-infected cells (CM and spherules) were not detected. To investigate whether nsp6, the third transmembrane subunit of the coronavirus replicase, plays a role in their formation or affects DMV formation, we aimed to extend the expressed polyprotein fragment to include nsp5 and nsp6. In addition to PLpro cleaving the nsp3/nsp4 site, this should lead to processing of the nsp4/nsp5 and nsp5/nsp6 junctions by the nsp5-based Mpro, an assumption based on sequence conservation and studies performed with other coronaviruses (3), as the kinetics of MERS-CoV polyprotein processing in cell-based assays have not been documented in any detail.Remarkably, however, when the &#8220;regular&#8221; nsp3-6 polyprotein was expressed, efficient processing of the nsp3/4 site was achieved, but nsp&#8217;s located downstream of this junction were retained in processing intermediates due to poor cleavage of the nsp4/5 and nsp5/6 junctions, as observed by immunoprecipitation (IP) analysis (Fig.&#160;4A and B). This prompted us to design a set of alternative polyprotein constructs to investigate and optimize the proteolytic autoprocessing of the nsp3-6 region (see Table&#160;S1). Efficient cleavage at all sites was observed only for an engineered polyprotein (nsp3-5-GFP-6) in which green fluorescent protein (GFP) had been inserted between two copies of the nsp5/6 cleavage site (Fig.&#160;4A). Immunoprecipitation analysis established that this nsp3-5-GFP-6 polyprotein was processed into four separate nsp&#8217;s and GFP (Fig.&#160;4B). Consequently, this construct could be used to evaluate the effect of expressing nsp6 in addition to nsp3 and nsp4.When HuH-7 cells expressed the &#8220;regular&#8221; nsp3-6 polyprotein, which was barely cleaved at the nsp5/6 junction (Fig.&#160;4B), we no longer detected the DMVs previously observed upon nsp3-4 expression. Instead, large areas of highly organized and curved membrane structures were seen (Fig.&#160;4C), which were connected to surrounding ER cisternae (Fig.&#160;4C, red arrows). In contrast to the large single-membrane clusters observed in nsp3- or nsp4-expressing cells (Fig.&#160;2C and D), they consisted of double membranes (Fig.&#160;4C, black arrow in the inset). The geometric pattern in these large areas containing double-membrane structures is typical of cubic membranes (49), which can result from overexpression and/or misfolding of ER proteins, leading to protein and membrane aggregation. In contrast, when HuH-7 cells expressed the engineered nsp3-5-GFP-6 polyprotein, which was almost fully processed (see above), cubic membranes were not observed and we found instead putative DMVs together with zippered ER (Fig.&#160;4D), structures very similar to the ones found in cells expressing just nsp3 and nsp4 (compare with Fig.&#160;2E and F). Also, the average size of these DMVs (146&#160;nm) was comparable to that of DMVs induced by nsp3-4 expression (148&#160;nm) (Fig.&#160;S1). Circular profiles (putative DMVs) were detected in 33 out of 642&#160;cell sections analyzed; however, none of these regions contained CM or spherules. This suggests that, while nsp3 and nsp4 are necessary and sufficient to induce the rearrangement of intracellular membranes into DMVs, the presence of (cleaved) nsp6 does not suffice to trigger the formation of the additional membrane structures typical of MERS-CoV infection. Other viral components that are present during MERS-CoV replication, such as viral RNA or other viral proteins, might thus be required for the formation of convoluted membranes and spherules.\nCleavage of the MERS-CoV nsp3/nsp4 junction is essential for DMV formation.\nTo gain more insight into the biogenesis of coronavirus DMVs, we set out to determine the role of the nsp3/nsp4 cleavage event. We surmised that the membrane modifications induced by an uncleaved nsp3-4 polyprotein could differ from those triggered by the (cleaved or coexpressed) nsp3 and nsp4 subunits. We transfected HuH-7 cells with plasmids encoding nsp3-4 carrying either a mutated nsp3/nsp4 cleavage site (GG>AA) or a catalytic site mutation in the nsp3 PLpro domain (C1592A) that inactivates the protease (50). In both cases, only the uncleaved nsp3-4 precursor was observed (Fig.&#160;5A). Interestingly, DMVs were no longer found and instead we detected concentric structures consisting of zippered ER that mostly lacked the pronounced curvature present in DMVs (Fig.&#160;5B and C). Cotransfection of the cells with a plasmid encoding the active PLpro domain restored the nsp3/nsp4 cleavage in the nsp3-4 C1592A mutant polyprotein but not in the nsp3-4 polyprotein with the mutated cleavage site (Fig.&#160;5A). Accordingly, expression of PLpro together with the nsp3-4 cleavage site mutant (Fig.&#160;5D) did not alter the structures observed. In contrast, when transcleavage of the nsp3/nsp4 site, by coexpression of PLpro with the nsp3-4 C1592A polyprotein, was achieved, DMV formation was at least partially restored and resulted in a mixture of abundant DMV and zippered ER profiles (Fig.&#160;5E), as observed before (Fig.&#160;2E and F). Expression of PLpro by itself did not have a membrane-remodeling effect. These results clearly showed that the nsp3-4 precursor is able to induce the membrane pairing required to form zippered ER but that cleavage of the nsp3/nsp4 junction is essential for the formation of DMVs.\nSARS-CoV nsp3 and nsp4 are also sufficient to induce DMV formation.\nRecently, Angelini et al. reported that, in the case of SARS-CoV, nsp3, nsp4, and nsp6 are all required for the formation of DMVs when these proteins are transiently expressed as individual subunits (42). In their two-dimensional (2D) imaging study, coexpression of SARS-CoV nsp3 and nsp4, in the absence of nsp6, led to the formation of so-called maze-like bodies (MLBs), large clusters of double-membrane structures that were interpreted as closely packed double-membrane tubules, not vesicles. Since our MERS-CoV tomography data (Fig.&#160;3) established that DMV formation can be triggered just by coexpression of nsp3 and nsp4, the interpretation of Angelini et al. suggested that these subunits of MERS-CoV and SARS-CoV differ in their ability to induce DMV formation in the absence of nsp6.To address this issue, we coexpressed nsp3 and nsp4 of either virus, using the same experimental setup previously used for SARS-CoV by Angelini et al. (293T cells transfected using lipofection), and employed ET for a comparative analysis in 3D. Coexpression of SARS-CoV nsp3 and nsp4 led to the formation of MLBs very similar to those observed by Angelini et al. (42), with areas of zippered ER, often clustered as regularly spaced profiles, and circular double-membrane profiles (Fig.&#160;6A and B). The latter were postulated to be cross sections of double-walled tubules, of which the regularly spaced zippered ER profiles would then represent longitudinal sections (42). The fact that the spacing between clustered zippered ER profiles roughly coincided with the diameter of the circular profiles supported this interpretation; however, Angelini et al. also acknowledged that ET would be required for its validation. To determine whether the circular profiles in the MLBs represented tubular or vesicular structures, we now used ET to analyze several MLBs, two of which are shown in Fig.&#160;6A and B. In one of those images, zippered ER is the dominant structure (Fig.&#160;6A; Movie&#160;S3), whereas the other mainly contained circular double-membrane profiles (Fig.&#160;6B; Movie&#160;S4). In both tomograms, we could detect multiple double-membrane profiles that increase and decrease in diameter when progressing through the tomogram and ultimately disappear (marked with green dots in the tomogram movies), indicating that they represent vesicles rather than tubules. In fact, no tubular structures were observed in the tomograms. The presumed longitudinal views of tubular structures turned out to consist of zippered ER winding through the MLB.For MERS-CoV, coexpression of nsp3 and nsp4 in 293T cells led to the formation of numerous circular double-membrane profiles together with some zippered ER (Fig.&#160;6C and D), which strongly resembled what we had observed in HuH-7 cells previously. Taken together, our ET results make it clear that, in the case of SARS-CoV as well, coexpression of nsp3 and nsp4 suffices for the induction of DMV formation and strongly suggest that this is a common feature among betacoronaviruses.\nDISCUSSION\nThe generation of membranous organelles that support their replication machinery is a universal mechanism among positive stranded RNA viruses infecting eukaryotes. The formation of these ROs is induced by viral proteins (32, 51, 52), which are largely uncharacterized in most instances, and appears to be also reliant on host factors, some of which have been identified as important players (53, 54). In this study, focusing on betacoronaviruses, we sought to identify the viral proteins required to induce the formation of DMVs, the most prominent membrane structure formed during coronavirus infection. Using ET, we found that coexpression of nsp3 and nsp4 of either SARS-CoV or MERS-CoV was required and sufficient to trigger the formation of ER-derived DMVs. Moreover, the 3D architecture of these membrane structures was similar to what has been observed during betacoronavirus infection (19). The DMVs formed upon coexpression of nsp3 and nsp4 were closed, with no detectable opening connecting the DMV interior and the surrounding cytosol, whereas their outer membrane generally was continuous with those of other DMVs and/or with (modified) ER. Our data importantly alter the conclusions of an earlier SARS-CoV study (42), which was based on the transient coexpression of SARS-CoV nsp3 and nsp4 from separate plasmids and the 2D imaging of the resulting membrane structures. The observation of maze-like bodies and circular double-membrane profiles, which were interpreted to represent tubular structures, led these authors to conclude that coexpression of SARS-CoV nsp3 and nsp4 was not sufficient for DMV formation. Using ET, we could now show that the circular profiles observed in these maze-like bodies are in fact DMVs (Fig.&#160;6A and B), suggesting that the basic capability of nsp3 and nsp4 to induce DMV formation probably is a common feature of betacoronaviruses. These findings also highlight the importance of 3D analysis as a tool to ascertain and characterize the ultrastructure of membranous viral ROs.Interestingly, in the case of the arterivirus equine arteritis virus (EAV), nsp2 and nsp3 were found to be required and sufficient for DMV formation (18, 41). At least to a certain extent, these proteins can be considered the functional equivalents of coronavirus nsp3 and nsp4, respectively, as they share a number of features like the presence of multiple membrane-spanning domains and a papain-like protease in the upstream protein that cleaves the junction between the two subunits. They also occupy comparable positions in the replicase polyproteins, suggesting that there may also be similarities in terms of the relative order in which these subunits are synthesized, released, and targeted to the membranes that they transform (17). These functional similarities and the potential to trigger DMV formation may thus be shared by these proteins of all coronaviruses and arteriviruses and possibly extend to other branches of the order Nidovirales, like the poorly studied ronivirus and mesonivirus families.Our findings also shed more light on DMV biogenesis, for which two models have been proposed that are not mutually exclusive. The first has been termed &#8220;double budding,&#8221; where a vesicle would first bud into the ER lumen and then bud out again to acquire a second membrane. The alternative model is based on &#8220;wrapping&#8221;: membranes would first pair or &#8220;zipper&#8221; and then curve and finally form a closed DMV after a membrane fission event (18, 55). The frequent observation of zippered ER after coexpression of nsp3 and nsp4 and multiple EM images in which zippered ER seemed to wrap into a DMV (e.g., Fig.&#160;3B) suggest that this structure is a DMV precursor. Interestingly, whereas the uncleaved nsp3-4 precursor was able to induce the pairing of ER membranes, DMV formation occurred only upon cleavage of the nsp3/nsp4 junction, which strongly suggests that membrane pairing is an early step in DMV formation. Our findings contrast with what was observed previously for arteriviruses, where cleavage of the nsp2/3 junction was not required for the formation of DMVs in an expression system (56). Together, our observations favor the wrapping model for DMV formation, and even though the existence in parallel of a double budding mechanism cannot be formally ruled out, the current data add to the mounting evidence pointing toward double-membrane wrapping as the central mechanism for DMV formation. For the distantly related arteriviruses (18, 57) and the unrelated picornaviruses (30, 31), putative wrapping intermediates have been also described, indicating that this might be a common mechanism of DMV biogenesis among positive stranded RNA viruses.Several steps are required for DMV biogenesis: pairing of membranes, membrane curvature (both positive and negative), and fission (18, 55). In the wrapping model for DMV biogenesis, membrane pairing is an early step that may be mediated directly by interactions between the viral proteins inducing DMV formation. The interaction(s) between nsp3 and nsp4 that we described here for MERS-CoV may be sufficient to facilitate membrane pairing. Similar observations have been made for SARS-CoV and MHV (42, 48, 58). The most likely candidate regions for this kind of interactions are the luminal loops of nsp3 and nsp4 (33, 35) that are located in the TM1 and TM2 regions, respectively, as these could interact with their counterparts on the opposite side of the ER cisterna, thus inducing membrane pairing. This view is partly supported by studies on MHV and SARS-CoV for which a truncated nsp3 lacking the region upstream of TM1 coexpressed with nsp4 was sufficient to induce membrane pairing but not the formation of DMVs (48, 58). This suggests that, although the cytosolic N-terminal region of nsp3 is required for complete DMV formation, the TM1 region (together with nsp4) may be sufficient to induce membrane pairing.In principle, the liberation (by PLpro-mediated cleavage of the nsp3/nsp4 junction) and presumed membrane insertion of the hydrophobic N-terminal domain of nsp4 may be an important determinant of the ultimate transmembrane configuration of this protein, potentially with direct implications for the transformation of zippered ER into DMVs. However, we found no major differences between DMVs induced by expression of self-cleaving MERS-CoV nsp3-4 and those induced by coexpression of nsp3 and nsp4, suggesting that nsp4 is properly inserted in the membrane when individually expressed. The concentric zippered ER observed after expression of the uncleaved nsp3-4 polyprotein could then reflect an intermediate stage in which the lack of nsp3/nsp4 cleavage prevents proper membrane remodeling. The proximal (and largest) luminal loop of nsp4 contains an N-linked glycosylation site (N2985 in MERS-CoV), similar to the glycosylation site(s) in SARS-CoV and MHV (34, 37, 40). Analysis of the use of that site in proteolytically processed nsp4 and the uncleaved nsp3-4 precursor could provide insight into the sequence of events leading to the membrane insertion of MERS-CoV nsp4.Although our data establish that expression of nsp3 and nsp4 suffices for coronaviral DMV formation, the precise role in this process&#8212;if any&#8212;of the nsp6 transmembrane subunit remains unclear. Our MERS-CoV data suggest that, compared to cells expressing nsp3 and nsp4 only, coexpression of cleaved nsp6 does not affect DMV formation, nor does it lead to the formation of additional structures like CM or spherules. However, coexpression of SARS-CoV nsp3, nsp4, and nsp6 was previously reported to induce CM formation in addition to DMVs (42). Additionally, expression of nsp6 alone resulted in the formation of small single-membrane vesicles near the microtubule organizing center (42), which suggested that nsp6 may have membrane proliferation and vesiculation abilities that could play a role in RO formation.When MERS-CoV nsp6 was retained in an unprocessed nsp4-6 precursor, DMVs were no longer formed and membrane clusters that resemble cubic membranes appeared (Fig.&#160;4C). It has been proposed that the CM formed by coronaviruses are in fact a form of cubic membranes (59). This might be related to observations that, compared to DMVs, CM are mostly formed relatively late in infection (19, 21, 24) when viral proteins or polyprotein fragments accumulate. It is conceivable that such accumulation could lead to aggregation, misfolding, and/or impaired polyprotein processing resulting in the formation of cubic membranes. In other words, there could be a link between the status of polyprotein processing in the nsp4-6 region and the membrane structures formed. Along the same lines, the observation that blocking the cleavage at the nsp3/4 junction impedes DMV formation (see above) directly implicates polyprotein processing in the control of membrane remodeling, possibly by facilitating conformational changes required for specific interactions with the membrane and/or between nsp3 and nsp4. For an unrelated DMV-forming virus, hepatitis C virus (HCV), it was recently shown that DMV formation became less efficient when the proteolytic cleavage of the NS4B/5A site in the viral polyprotein was accelerated, which similarly suggests a role for the polyprotein processing in DMV formation (60).The existence of different proteolytic processing intermediates containing nsp5 is well documented for arterivirus infection (61), although the role of the different precursors in DMV formation has not been studied in depth so far. Unfortunately, the kinetics of polyprotein processing by Mpro in MERS-CoV and SARS-CoV are still largely unknown. Mpro&#8217;s enzymatic activity has mainly been assessed using recombinant nsp5 and peptide substrates in vitro (9, 11, 13, 50), but an analysis of the kinetics in a large (or larger) polyprotein setting is mostly lacking. Most information on the processing of the coronavirus nsp4-to-nsp10 region is derived from studies on other coronaviruses, such as MHV, IBV, and human coronavirus 229E (HCoV-229E) (the latter two being a gamma- and an alphacoronavirus, respectively) (9, 62, 63). An in-depth analysis of the kinetics of polyprotein maturation during coronavirus infection, the identification of nsp6-containing processing intermediates, and the investigation of their possibly distinct roles in membrane remodeling could help to further unravel the mechanisms underlying the formation of the coronavirus ROs.\nMATERIALS AND METHODS\n\nCells, viruses, and antibodies.\nHuH-7 cells (kindly provided by Ralf Bartenschlager, Heidelberg University) were grown in Dulbecco&#8217;s modified Eagle&#8217;s medium (DMEM; Lonza) supplemented with 8% (vol/vol) fetal calf serum (FCS; Bodinco), 2&#160;mM l-glutamine (PAA Laboratories), and nonessential amino acids (PAA Laboratories). 293T cells (kindly provided by the Virgin lab, Washington University School of Medicine in St. Louis, MO) were cultured in DMEM with 10% (vol/vol) FCS. All cell culture media contained 100&#160;U/ml penicillin and 100&#160;&#181;g/ml streptomycin. Infection of HuH-7 cells with MERS-CoV (EMC/2012 strain kindly provided by Ron Fouchier, Erasmus Medical Center, The Netherlands [3, 4]) was performed as previously described (21).Primary antibodies used were mouse anti-HA (clone HA.C5; Abcam), mouse anti-&#946;-actin (clone AC-74; Sigma), and mouse anti-V5 (clone 2F11F7; Thermo Fisher). A rabbit serum recognizing SARS-CoV-nsp3 that cross-reacts with MERS-CoV nsp3 has been previously described (21, 22). A polyclonal rabbit serum was used against a combination of two MERS-CoV nsp5 peptides, SGLVKMSHPSGDVEAC (amino acids 3248 to 3263 of pp1a) and CPADQLSDPNYDALLI (amino acids 3291 to 3306), which was produced by Eurogentec.\nPlasmid construction and transfection.\nHuman codon-optimized coding sequences of MERS-CoV nsp3-6 were designed using GeneArt, ordered from Thermo Fisher in four fragments, and subsequently assembled in low-copy-number vector pACNR1180 (64) using conventional cloning. The precise parts of MERS-CoV pp1a used for polyprotein constructs are outlined in Table&#160;S1&#160;in the supplemental material. The nsp4 construct included the 21 C-terminal aa of nsp3 to prevent the N-terminal hydrophobic region of nsp4 from acting as a signal sequence, which could result in improper membrane insertion. In all constructs with a C-terminal myc or V5 tag, the C-terminal glutamine of the viral sequence was omitted to prevent the removal of the tag by Mpro. The SARS-CoV nsp3 gene (Frankfurt 1 strain, pp1a amino acids 819 to 2740) was synthesized by Bio Basic Inc. (Ontario, Canada). Coding sequences were transferred to the pCAGGS expression vector (Addgene) for expression. pCAGGS-SARS-nsp4 was described previously (42). 293T cells were transfected using Lipofectamine 2000 (Thermo Fisher) according to the manufacturer&#8217;s instructions. HuH-7 cells were transfected using a Nucleofector 2b device (Lonza) with Nucleofector kit T (Lonza) in 6 &#215; 106 cells and 12&#160;&#181;g of plasmid DNA per transfection. Cotransfections were carried out with equimolar amounts of plasmids.\nWestern blotting.\nCells were lysed in 2&#215; Laemmli sample buffer (50&#160;mM Tris-HCl, pH&#160;6.8, 20% [vol/vol] glycerol, 4% [wt/vol] sodium dodecyl sulfate [SDS], 20&#160;mM dithiothreitol, 0.02&#160;mg/ml bromophenol blue) and separated by electrophoresis on SDS-polyacrylamide gels. Proteins were transferred to polyvinylidene fluoride membranes (Amersham) using a Trans-Blot Turbo transfer system (Bio-Rad). Blots were blocked with 5% (wt/vol) ELK skimmed milk powder (Campina) in phosphate-buffered saline (PBS) supplemented with 0.05% (vol/vol) Tween 20. Secondary horseradish peroxidase (HRP)-conjugated antibodies (Dako) and ECL Plus Western blotting substrate (Thermo Fisher) were used to visualize protein signal.\nImmunofluorescence microscopy.\nAfter electroporation, HuH-7 cells were seeded on coverslips and fixed 24&#160;h later with 3% (wt/vol) paraformaldehyde in PBS. Samples were permeabilized with 0.2% (vol/vol) Triton X-100 and incubated with antibodies, including fluorescent conjugates, diluted in 5% (wt/vol) bovine serum albumin (BSA) in PBS. Nuclei were stained with 1&#160;&#181;g/ml Hoechst 33258. After embedding with Prolong Gold (Thermo Fisher), samples were analyzed with a Leica TCS SP8 confocal laser scanning microscope, which was equipped with a 63&#215; objective (numerical aperture [NA] 1.40; 1 Airy unit) and a Leica HyD hybrid detector.\nMetabolic labeling and IP.\n293T cells were metabolically labeled with 100&#160;&#181;Ci/ml [35S]methionine and [35S]cysteine (EXPRE35S35S protein labeling mix; PerkinElmer) from 4&#160;h posttransfection onward. Cells were lysed at 18&#160;h posttransfection in 20&#160;mM Tris-HCl (pH&#160;7.6), 150&#160;mM NaCl, 0.5% (wt/vol) deoxycholic acid, 1% (vol/vol) Nonidet P-40, and 0.1% (wt/vol) SDS. Lysates were diluted in immunoprecipitation (IP) buffer (20&#160;mM Tris-HCl [pH&#160;7.6], 150&#160;mM NaCl, 5&#160;mM EDTA, 0.1% [wt/vol] deoxycholate [DOC], 0.5% [vol/vol] NP-40, and 0.5% [wt/vol] SDS) and incubated with antibody overnight. Antibody-protein complexes were then pulled down using protein A and protein G Sepharose beads (GE Healthcare), which were first blocked with 2% (wt/vol) BSA in PBS, and incubated for several hours. After repeated washing of the beads with IP buffer, proteins were eluted by heating in 2&#215; Laemmli sample buffer. After separation on large 10% polyacrylamide gels and gel drying, signal was visualized using an Imaging Screen-K (Bio-Rad) and a Typhoon 9410 scanner (GE Healthcare).\nElectron microscopy.\nTransfected HuH-7 or 293T cells were fixed 24&#160;h posttransfection in 1.5% (wt/vol) glutaraldehyde in 0.10 M cacodylate buffer (pH&#160;7.4) for 1&#160;h at room temperature. After washing in 0.14 M cacodylate buffer, samples were postfixed and stained at 4&#176;C with 1% (wt/vol) osmium tetroxide in 0.10 M cacodylate buffer for 1&#160;h. After washing with 0.14&#160;M cacodylate and Milli-Q water, cells were scraped and stained with 1% (wt/vol) tannic acid in Milli-Q water on a 3D rotator for 1&#160;h at room temperature. Following washing with Milli-Q water, cells were spun down in heated 3% (wt/vol) agar in PBS, and after solidification, pellets were excised, cut into small blocks, and dehydrated in increasing concentrations of ethanol. Samples were embedded in epoxy resin (LX-112; Ladd Research), and after polymerization, 100-nm sections were placed on mesh-100 copper EM grids covered with a carbon-coated Pioloform layer. Following poststaining with 7% (wt/vol) uranyl acetate and Reynolds lead citrate, samples were analyzed on an FEI Tecnai 12 BioTwin microscope equipped with an Eagle cooled slow-scan charge-coupled device (CCD) camera (FEI) and operated at 120&#160;kV. Measurements of circular profiles from 2D EM images were done with ImageJ software and Aperio Imagescope software (Leica). Circular profiles were measured over their longest and shortest axes, and the geometric mean of those values was used as the diameter. One hundred circular profiles were measured for each condition.\nElectron tomography.\nSections of 150-nm thickness were cut from the resin-embedded blocks of transfected HuH-7 or 293T cells prepared as described above. Prior to poststaining, colloidal gold particles of 10&#160;nm were applied to both sides of the EM grid to serve later as fiducial markers for alignment. Tomography data were recorded on an Eagle CCD camera (FEI) in an FEI Tecnai 12 BioTwin (HuH-7 samples) or a Twin (293T samples) electron microscope operated at 120&#160;kV, with the grids mounted on a 2040 Fischione tomography holder. Dual-axis tilt series of the regions of interest were collected using Xplore3D software (FEI) at magnifications that resulted in a pixel size of 1.7&#160;nm (BioTwin data) or 1.4&#160;nm (Twin data). The angular coverage for each single-axis tilt series was 130&#176; sampled in increments of 1&#176;. Alignment of the tilt series and tomogram reconstruction by weighted back-projection were performed in IMOD (65).",
        "pmid": "29162711",
        "title": "Expression and Cleavage of Middle East Respiratory Syndrome Coronavirus nsp3-4 Polyprotein Induce the Formation of Double-Membrane Vesicles That Mimic Those Associated with Coronaviral RNA Replication",
        "journal_title": "mBio",
        "authors": "Diede Oudshoorn;Kevin Rijs;Ronald W. A. L. Limpens;Kevin Groen;Abraham J. Koster;Eric J. Snijder;Marjolein Kikkert;Montserrat B\u00e1rcena",
        "doi": "10.1128/mBio.01658-17"
    },
    {
        "keywords": "",
        "abstract": "Background\nCoronaviruses are an important cause of infectious diseases in humans, including severe acute respiratory syndrome (SARS), and have the continued potential for emergence from animal species. A major factor in the host range of a coronavirus is its receptor utilization on host cells. In many cases, coronavirus-receptor interactions are well understood. However, a notable exception is the receptor utilization by group 3 coronaviruses, including avian infectious bronchitis virus (IBV). Feline aminopeptidase N (fAPN) serves as a functional receptor for most group 1 coronaviruses including feline infectious peritonitis virus (FIPV), canine coronavirus, transmissible gastroenteritis virus (TGEV), and human coronavirus 229E (HCoV-229E). A recent report has also suggested a role for fAPN during IBV entry (Miguel B, Pharr GT, Wang C: The role of feline aminopeptidase N as a receptor for infectious bronchitis virus. Brief review. Arch Virol 2002, 147:2047&#8211;2056.Results\nHere we show that, whereas both transient transfection and constitutive expression of fAPN on BHK-21 cells can rescue FIPV and TGEV infection in non-permissive BHK cells, fAPN expression does not rescue infection by the prototype IBV strain Mass41. To account for the previous suggestion that fAPN could serve as an IBV receptor, we show that feline cells can be infected with the prototype strain of IBV (Mass 41), but with low susceptibility compared to primary chick kidney cells. We also show that BHK-21 cells are slightly susceptible to certain IBV strains, including Ark99, Ark_DPI, CA99, and Iowa97 (<0.01% efficiency), but this level of infection is not increased by fAPN expression.Conclusion\nWe conclude that fAPN is not a functional receptor for IBV, the identity of which is currently under investigation.",
        "body": "Background\nThe family of Coronaviridae is composed of group 1&#8211;3 coronaviruses (CoVs) [1]. These viruses are able to infect human, canine, feline, murine, bovine, porcine, rat, and avian species. The etiological importance and zoonotic characteristics of coronaviruses have received much attention since the discovery of the newly emerged severe acute respiratory syndrome associated coronavirus (SARS-CoV) in 2003 [1,2]. Coronaviruses have a high frequency of viral genome recombination and polymerase infidelity, which may have contributed to the increase of viral pathogenesis, inter-species transmission, and tissue tropism [3-5]. In the case of SARS-CoV, its ancestral origin remains undetermined, but some evidence suggests that Chinese horseshoe bats may be the natural reservoirs, while Himalayan palm civets harbor and support inter-species transmission to humans [5,6]. Other examples of extended tissue tropisms can also be found in some group 2 CoVs. It is speculated that the acquisition of hemagglutinin esterase (HE) activity from influenza C virus gives rise to the ability of sialic acid recognition and the extended tissue tropism and pathogenesis for some group 2 CoVs [7-9]. Furthermore, bovine coronavirus (BCV) is thought to have jumped to human hosts, possibly by recombining with influenza C virus, thus giving rise to human coronavirus-OC43 (HCoV-OC43) around 1890 [4,8].Receptor interaction between the virus and its host is the first step leading to a successful entry and productive replication. Viruses increase fitness by adapting to environmental pressure through mutation and recombination. In contrast to other families of viruses that utilize a universal receptor to gain entry into host cells, members in the coronavirus family use a variety of cellular proteins and/or co-factors. Group 1 CoVs &#8211; including human coronavirus-229E (HCoV-229E), feline infectious peritonitis virus (FIPV), transmissible gastroenteritis virus (TGEV) and canine coronavirus (CCV) &#8211; utilize human, feline, porcine, and canine aminopeptidase N (APN) as functional receptors during virus entry [10-13]. The only notable exception is HCoV-NL63, which utilizes angiotensin-converting enzyme 2 (ACE2). In group 2 CoV, mouse hepatitis virus (MHV) of group 2a and SARS-CoV of group 2b independently utilize carcinoembryonic antigen-cell adhesion molecule (CEACAM1) and ACE2 to mediate infection [14,15]. However, other group 2a CoVs, including HCoV-OC43 and BCoV recognize N-acetyl-9-O-acetylneuraminic acid as a functional receptor [9]. While the cellular receptors for both groups 1 and 2 CoVs have been identified and independently confirmed, group 3 CoV receptors remains undetermined.The avian CoVs, such as turkey CoV and infectious bronchitis viruses (IBV), have been classified in group 3, with IBV the most extensively studied. Recently, Winter and colleagues suggested that sialic acids are responsible for IBV strain Massachusetts 41 entry [16]. However, group 3 CoVs lack HE as a key viral protein regulating sialic acid binding, and the use of sialic acid would not explain the dependence on chicken cells for infection. Therefore IBV is unlikely to use sialic acids as a functional entry receptor, but rather as a non-specific attachment factor. Heparan sulfate may also serve as an attachment factor for the IBV strain Beaudette (IBV_Bdtt) [17]. IBV_Bdtt is a highly chicken embryo-adapted strain [18,19], which has an extensive tropism in cell culture and efficiently infects various cell types, including BHK-21 cells [17,19,20]. In contrast, clinical isolates and field strains of IBV typically only infect chicken cells.In the effort to identify the receptor for group 3 CoVs, feline APN (fAPN) was reported to allow entry of the IBV strain Arkansas 99 (IBV_Ark99) [21]. This could therefore be the first indication of a more universal receptor for the CoV family. APN belongs to a family of metalloproteases [22]. It is a type II membrane-bound glycoprotein, and it is expressed on a variety of cell types, including granulocytes, monocytes, and fibroblasts. APN can also be found on the synaptic membranes of the central nervous system neurons, and on epithelial cells in the proximal convoluted tubules, intestinal brush border, and respiratory tract [11,23]. For coronaviruses in general, there is a cross-species restriction that permits cells of a certain species to be infected only by its own complimentary CoV. However, several studies on FIPV and HCoV-229E, CCV and TGEV have identified fAPN as a universal entry receptor for group 1 coronaviruses [11-13,23]. The demonstration that fAPN can allow infection by the IBV strain Ark_99 prompted us to examine both prototype and field isolates of IBV and test them for the potential use of fAPN as a receptor.In this study, we first verified the use of the expressed fAPN as a receptor for FIPV and TGEV by transient and constitutive expression of fAPN in non-permissive BHK-21 cells. We also cultured seven strains of IBV, including Arkansas 99, Arkansas_DPI, California 99, Connecticut 46, Holland 52, Iowa 97, and Massachusetts 41 (designated as Ark99, Ark_DPI, CA99, Conn46, H52, Iowa97, and Mass41) as candidates to test for fAPN utilization by group 3 avian CoVs. Surprisingly, expression of fAPN did not increase viral infection in any of the strains tested. As a consequence, we conclude that fAPN is not a functional receptor during IBV entry. The authentic receptor is still under investigation.\nResults\nIn order to determine fAPN receptor usage by IBV, our goal was to rescue IBV infection in non-permissive cells by expressing fAPN on the cell surface. To authenticate fAPN expression, we first transiently transfected BHK-21 cells with fAPN/pcDNA3.1D/TOPO plasmid DNA for 24 hours. Samples were then subjected to immunofluorescence staining to detect fAPN protein expression (Fig. 1A). We found that nearly 40% of the BHK-21 population were efficiently transfected and stained positive for fAPN expression. Next, we verified the fAPN expression on the BHKexp.fAPN cell line that constitutively expresses fAPN and its negative control cells, BHKexp.pCiNeo (Fig. 1B and 1C). 100% of BHKexp.fAPN cells also stained positive, while the negative control showed no fAPN expression (Fig. 1).To verify the functionality of fAPN as a coronavirus receptor, we first tested its ability to rescue FIPV-1146 and TGEV infection of non-permissive cells, as reported in previous studies [11]. Feline kidney CRFK cells and canine fibroblast A72 cells are able to support FIPV-1146 and TGEV infection respectively (Dr. Edward Dubovi, Cornell University, personal communication). Therefore, these cell lines served as positive controls for FIPV-1146 and TGEV infection. We observed that FIPV-1146 and TGEV efficiently infected CRFK and A72 host cells respectively, but they were unable to mediate infection in BHK-21 cells (Fig. 2A and 2B), presumably due the lack of a functional receptor on the cell surface. However, transient transfection of fAPN in BHK-21 cells rescued FIPV and TGEV infection to 34% and 21% respectively (Fig. 2C). We therefore confirm that fAPN can function as an entry receptor for both FIPV-1146 and TGEV.Group 3 CoVs such as infectious bronchitis viruses (IBV) can infect primary chicken kidney cells at high efficiency, but they generally restrict interspecies infections [19]. However, previous studies have shown that feline kidney cells (FEK) can support IBV_Ark99 infection [21], suggesting the presence of a feline receptor on the FEK cell surface that enables IBV_Ark99 entry. Since IBV_Mass41 has served as a prototype virus to study CoV entry [24,25], we first examined the potential utilization of a feline receptor by IBV_Mass41. We found that whereas BHK-21 cells were resistant to IBV_Mass41 infection, and IBV_Mass41 infected CK cells at 45% efficiency, IBV_Mass41 was able to also infect feline CRFK cells at 6% efficiency (Fig 3B). As indicated by Miguel et al. [21], the IBV_Mass41 infection found in CRFK cells suggests the potential use fAPN as a receptor.To determine the potential role of fAPN as a receptor for the prototype IBV (Mass_41), we transiently transfected fAPN into BHK-21 cells. Both untransfected and fAPN-transfected BHK-21 cells failed to show any detectable infection with IBV_Mass41, whereas primary CK cells were efficiently infected (Fig. 4). To rule out the potential inhibitory effect of viral infectivity caused by transient transfection, we also examined IBV infection in cell lines constitutively expressing fAPN. We found that expression of fAPN in BHKexp.fAPN cells could not rescue IBV_Mass41 infection (Fig. 4A and 4B).To examine if fAPN might act in a strain specific manner, seven field IBV isolates: Ark99, Ark_DPI, CA99, Conn46, H52, Iowa97, and Mass41 were obtained and cultured in 10-day old specific pathogen free (SPF) chicken embryonic eggs. To verify virus infectivity, monolayers of CKC were individually incubated with all seven strains of IBV for 12 h. Figure 5 shows that CKC monolayers were found to be efficiently infected by all seven field strains of IBV tested. To determine the role of fAPN as a functional receptor for entry of these seven fields strains of IBV, as well as the prototype Mass 41, were inoculated onto BHKexp.pCiNeo or BHKexp.fAPN cell monolayers in triplicate. We found that BHK cells expressing empty vector alone (BHKexp.pCiNeo) were completely resistant to infection by IBV strains Conn46, H52 and Mass41. However, IBV strains Ark99, Ark_DPI, CA99, and Iowa97 showed limited infection in the same cells, with a level of infection of less than 0.01% (Fig. 6A). However, fAPN expression did not significantly enhance IBV infectivity in any of the virus strains tested (Fig. 6B). Consequently, our data show that IBV does not utilize fAPN as a functional receptor during virus entry.\nDiscussion\nViruses make use of a variety of receptors to gain entry into their target cells. The ability to recognize sialic acids that are ubiquitously present on the cell surface gives influenza viruses the ability to indiscriminately infect varied tissue or cell types. In contrast, retroviruses require more specialized receptor interactions between viral glycoprotein 120 (gp120) and CD4 as well as other chemokine receptors on the T-helper lymphocytes during invasion [26]. Despite some subtle differences among different species or strains of viruses in the same family, they may still retain a trace of evolutionary similarity in terms of receptor utilization. However, receptor usage appears to have very little consensus among coronaviruses across different groups and species. Coronaviruses seem to be able to devise various strategies to interact with specific target host at the level of receptor binding.In order to gain a more comprehensive understanding of IBV receptor utilization, we cultured several clinical strains of IBV for this study. Surprisingly, we found that the BHK-21 cells are in fact, weakly permissive to IBV_Ark99, IBV_DPI, IBV_CA99, and IBV_Iowa97 infections. However, fAPN expression on BHK-21 cell surface did not increase viral infectivity for any IBV strain. We therefore conclude that fAPN could not be a functional receptor for IBV entry. It is important to note that one, highly chick-embryo- adapted, IBV strain (IBV Beaudette) gives efficient infection in BHK cells [20]; combined with the data presented here, that a low level of infection of BHK cells could be obtained with some clinical strains, this suggests that there are no post-entry restrictions to IBV replication and gene expression that might account for a lack of infection of BHK cells.How do we explain the discrepancy between these data and previous studies that show that fAPN is the receptor for IBV_Ark99 entry? According to our observations, IBV_Ark99 viruses were able to infect BHK-21 cells to a limited degree (<0.01%), although the infection profile was not always consistent among samples. The relevance of this very limited infection of hamster cells is unclear. As a consequence, previous studies may have shown only localized and under-represented populations of cells, which were infected by IBV_Ark99, since there was no further quantification of viral infectivity. In contrast, our present study was performed in a quantitative manner by measuring infection frequency of nearly 106 cells and we provide numerical virus infection data between fAPN-expressing and non-expressing cells.Various studies have shown several potential receptor candidates for group 3 coronaviruses, which include fAPN, sialic acids, and heparan sulfate [16,21]. However, there is still little consensus on which of these might be the functional entry receptor in vivo. Sialic acid utilization as a functional entry receptor by IBV remains controversial since IBV generally has a highly restricted species tropism, and does not possess a HE glycoprotein that would normally be responsible for sialic acid destruction and hence efficient virus spread. Unlike influenza viruses that are established to use cell surface sialic acids for entry into a wide variety different cells, IBV is generally considered only to infect cells of chicken origin. The infected chickens often show lesions or lymphocyte infiltration in the ciliated epithelia cells, mucus secreting cells, and sub-epithelial cells along the respiratory tract [27]. IBV also infects lung, ovary, and kidney tissues [24]. In cell culture, IBV_Mass41 infects CKC but not chick embryo fibroblast cells or the chicken DF-1 fibroblast cell line (data not shown), suggesting possible cell type or tissue specificity for IBV infection.Feline APN has been previously reported to serve as a functional receptor for IBV_Ark99 entry, a conclusion based in part on the ability of the virus to infect feline kidney cells (CRFK, also known as FEK) [21]. We have independently confirmed the ability of feline kidney cells to support IBV_Mass41 infection (Fig. 3A) with relatively high efficiency. As a consequence, cats could potentially serves as an intermediate host during an extension of coronavirus tissue tropism. In the case of SARS-CoV, viral transmission is thought to originate from Chinese horseshoe bats to Himalayan palm civet cats before jumping to humans [5,6]. The use of a feline receptor for IBV could potentially be another example of coronaviruses using feline cells as an interspecies reservoir to cross species boundaries. The reason why cats may serve as such an intermediate host is unclear, although it may be noteworthy that the endogenous feline coronavirus (FECV) can undergo distinct recombination events within feline cells that result in the emergence of the clinically different virus FIPV [28]. Also of potential relevance is the suggestion that the SARS-CoV S gene may be a mosaic of feline and avian sequences [29].\nConclusion\nWe conclude here that IBV cannot utilize fAPN as a functional receptor. It is however important to note that although IBV does not utilize fAPN as an entry receptor, there is still a possibility that chicken APN (cAPN) may serve as the key receptor. Future work will address the potential usage of cAPN, as well as other chicken cell-specific proteins, as IBV receptors.\nMethods\n\nViruses and cells\nIBV Conn46, Iowa97, H52, and Mass41 were obtained from Dr. Benjamin Lucio-Martinez, Unit of Avian Health, Cornell University, and IBV Ark_DPI was obtained from Dr. Mark W. Jackwood, University of Georgia. IBV Ark99 and CA99 were obtained Dr. Shankar P. Mondal (University of California, Davis). Viruses were propagated in 11 day-old embryonated chicken eggs and purified as described previously [20]. Virus stocks were titered by infection of primary chicken kidney cells, followed by immunofluorescence microcopy with monoclonal anti-S antibodies after 8&#8211;12 h of infection. Typically, the highest titer stocks were able to infect between 50% and 70% of the CK cells present, and were used at this concentration for all infections, unless otherwise indicated.FIPV-1146 and TGEV were obtained from Dr. Edward J. Dubovi, Animal Health Diagnostic Center, Cornell University, and were propagated in CRFK (feline kidney cell) and A72 (canine fibroblast cells) respectively. BHK-21 cells were purchased from ATCC and propagated twice weekly in DMEM, supplemented with 10% fetal bovine serum (FBS) 1% penicillin-streptomycin, and 10 mM HEPES (Cellgro). CRFK and A72 were grown in L:M media. BHKexp.fAPN or and BHKexp.pCiNeo cells were kindly provided by Dr. Kathryn V. Holmes (University of Colorado Health Sciences) and grown in 10% DMEM + 300 &#956;g/ml G418 (Sigma).Primary chicken kidney cells (CKC) were prepared as follows: in short, SPF White Leghorn Chicks (11&#8211;14 days) were sacrificed in a CO2 chamber, and kidneys were removed via aseptic techniques. Kidney cells were washed in 50 ml sterile phosphate buffer saline (PBS) twice via gentle stirring to remove red blood cells. The cells were then treated with 25 ml of trypsin-EDTA (Cellgro) for two minutes to separate large chunks of kidney tissues. Cell supernatant was mixed with 1 ml of FBS and centrifuged at 1,000 rpm for 2 min. Media were removed and kidney cells were resuspended in 25 ml of M20 media. Cell population was determined on a hemocytometer. For plating, 1&#8211;1.5 &#215; 106 cell/ml was prepared in M25 media (M20 media with 5% FBS) and incubated in 5% CO2 at 37&#176;C. For infection, 60% confluent cell monolayers were inoculated with virus stocks and incubated for 8, 12, or 24 hours before fixation.\nPlasmids and transfections\nfAPN/pcDNA3.1D/TOPO and hAPN/pCiNeo plasmid DNAs were provided by Dr. Kathryn V. Holmes (University of Colorado Health Sciences). For standard transfection, 0.6 &#956;g of plasmid DNA were premixed with 2 &#956;l of Lipofectamine 2000 (Invitrogen) in 300 &#956;l of Opti-MEM (Gibco) at room temperature according to manufacture's instructions. Cell monolayers were transfected at 37&#176;C for 24 hours before fixation or virus infection.\nVirus infection and immunofluorescence microscopy\nInfection and immunofluorescence microscopy were essentially performed as described previously [30] except with methanol fixation for detecting viral antigens. For fAPN staining, cell monolayers were fixed in 3% paraformaldehyde at room temperature and labeled with monoclonal antibody R-G-4 (provided by Dr. Tsutomu Hohdatsu, Kitasato University, Towada, Aomori, Japan). For viral antigen staining, anti-S1 monoclonal antibody 15:88 was used for IBV Massachusetts 41 and Iowa 97. The anti-S1 monoclonal antibody 13:18 was used for IBV Arkansas_DPI. The anti-M monoclonal antibody 9:19 was used for IBV Arkansas 99, Connecticut 46, California 99, and Holland 52. Monoclonal antibody 17B71 and rabbit polyclonal antibody 367 were provided by Dr. Edward Dubovi, Cornell University, and were used detect FIPV and TGEV infection respectively. The secondary antibodies Alex-fluor 488 goat anti-mouse IgG and Alex-fluor 488 goat anti-rabbit IgG were purchased from Molecular Probes. Cell nuclei were counter-stained with Hoechst 33258 (Molecular Probes). Cells were viewed on a Nikon Eclipse E600 fluorescence microscope. Images were captured with a SPOT RT Monochrome camera and SPOT Advanced v.4.0.9 software and processed with Adobe Photoshop v.7.\nCompeting interests\nThe author(s) declare that they have no competing interests.\nAuthors' contributions\nVC participated in the design of the study, carried out the experiments presented, and drafted the manuscript. LM, JM, TO and CH carried out preliminary experiments on which the manuscript is based. BB contributed to the conception and design of the experiments. GW conceived of the study, participated in its design and coordination, and wrote the final manuscript. All authors read and approved the final manuscript.",
        "pmid": "17324273",
        "title": "Feline aminopeptidase N is not a functional receptor for avian infectious bronchitis virus",
        "authors": "Victor C Chu;Lisa J McElroy;Jed M Aronson;Trisha J Oura;Carole E Harbison;Beverley E Bauman;Gary R Whittaker",
        "doi": "10.1186/1743-422X-4-20"
    },
    {
        "keywords": "Zika virus;Ebola virus;WHO;SARS;MERS.",
        "abstract": "Abstract\nThe World Health Organization (WHO) published 2 alcohol-based formulations to be used in healthcare settings and for outbreak-associated infections, but inactivation efficacies of these products have not been determined against (re-)emerging viruses. In this study, we evaluated the virucidal activity of these WHO products in a comparative analysis. Zika virus (ZIKV), Ebola virus (EBOV), severe acute respiratory syndrome coronavirus (SARS-CoV), and Middle East respiratory syndrome coronavirus (MERS-CoV) as (re-)emerging viral pathogens and other enveloped viruses could be efficiently inactivated by both WHO formulations, implicating their use in healthcare systems and viral outbreak situations.",
        "body": "MATERIAL AND METHODS\n\nCell Culture and Viral Strains\nAn overview of the viruses and cell culture systems used in this study is given in Supplementary Table 1. Hepatitis C virus (HCV) chimeric Jc1 virus was generated in the human hepatoma cell line (Huh7.5) as previously described [7]. The African lineage ZIKV strain (MP1751), isolated in Uganda in 1962, was propagated by using Vero-B4 cells like MERS-CoV strain EMC and SARS-CoV strain Frankfurt 1. Bovine CoV (BcoV) was produced in the human glioblastoma astrocytoma cells U373, human influenza A virus (H1N1) was produced in Madin-Darby canine kidney epithelial cells (MDCK) and modified vaccinia Ankara strain (MVA) was produced in baby hamster kidney cells (BHK-21). Ebola virus was propagated in Vero E6 cells as previously described [8]. Ebola virus&#8211;like particles encoding a luciferase were generated using 239T cells as previously reported [9]. In general, cell lines were cultured in Dulbecco&#8217;s modi&#64257;ed minimal essential medium or Eagle&#8217;s minimum essential medium supplemented with 10% fetal calf serum and other additions (Supplementary Table 1).\nQuantitative Suspension Test and Virus Titrations\nOne part by volume of the test virus suspension and 1 part by volume of the organic load were mixed with 8 parts by volume of 1 of the 2 WHO formulations at different concentrations. Additional information is provided in the Supplementary Data.\nStatistical Analyses\nConcentrations at which the formulations reached the half maximal virus inactivation effective concentration (EC50) were determined using nonlinear regression using the robust fitting method on the normalized 50% tissue culture infectious dose (TCID50) data implemented in GraphPad Prism version 6.07 for Windows. The mean TCID50 of 2 individual experiments and standard deviations of means were also calculated using GraphPad Prism. Significance of differences in mean EC50 obtained for the viruses between WHO formulations I and II was tested using 2-tailed Wilcoxon matched-pairs signed rank test (P < .01).\nRESULTS\n\nVirucidal Activity of World Health Organization Formulations Against Hepatitis C Virus and Zika Virus\nHepatitis C virus and ZIKV both belong to the family of Flaviviridae (Supplementary Table 1) but are transmitted in the environment by different routes. Whereas HCV is a blood-borne virus [10], transmission of ZIKV occurs mainly through mosquitos, with the most important and common vectors being the Aedes genus. However, other modes of transmission, including sexual transmission, have been reported. To determine the efficacy of WHO formulations I and II against HCV and ZIKV, we incubated the 2 viruses for 30 seconds with the formulations at final concentrations ranging from 10% to 80% (Figure 1). In the case of HCV, viral titers started to decline at a concentration of 30% with WHO formulation II and 40% with WHO formulation I and were reduced to background levels at a concentration of 60% with WHO formulation I and at 40% with WHO formulation II, respectively (Figure 1A). As depicted in Figure 1B, a dose-dependent reduction of viral titers was also observed for ZIKV (Figure 1B). Importantly, viral titers of 106 TCID50/mL in the control decreased to undetectable levels with WHO formulation I at a concentration of 40%, whereas a concentration of only 30% was required for complete inactivation with WHO forumulation II.\nSusceptibility of Bovine Coronoavirus, Middle East Respiratory Syndrom Coronavirus, and Severe Acute Respiratory Syndrome Coronoavirus to World Health Organization Formulations\nNext, we investigated the susceptibility of emerging respiratory CoVs against the WHO formulations in the same experimental suspension assay setup. As reference for CoVs, which can be cultivated under lower biosafety levels, we included BCoV that naturally infects cattle. As depicted in Supplementary Figure 1A, WHO formulation II at a 30% concentration was sufficient to completely inactivate BCoV, whereas for WHO formulation I higher concentrations of at least 40% were required (Supplementary Figure 1A). Similar inactivation profiles could be observed for MERS-CoV (Figure 1C) and SARS-CoV (Figure 1D), demonstrating a high susceptibility of these emerging CoVs to WHO formulations. Furthermore, these results implicate BCoV as a valid surrogate virus for inactivation studies with MERS-CoV and SARS-CoV.\nVirucidal Activity of World Health Organization Formulations Against Ebola Virus, Human Influenza A Virus, and Modified Vaccinia Ankara Strain\nWork with infectious EBOV is restricted to biosafety level 4 laboratories, significantly limiting studies with these viruses. In 2014, Watt et al reported a novel life cycle modelling approach for EBOV, which can be performed at biosafety level 2 laboratories [9]. Inactivation of these transcription- and replication-competent virus-like particles (trVLPs) with WHO formulations showed a dose-dependent reduction of trVLP reporter activity with increasing WHO formulation I and II concentrations (Figure 2A). Next, we tested full infectious EBOV cultured at biosafety level 4 for its susceptibility to WHO formulations for potential usage in outbreak situations. Interestingly, viral titers of 107 TCID50/mL in the control were reduced to background levels at concentrations of 40% with WHO formulation II and 60% with WHO formulation I, showing again a superior virucidal activity of WHO formulation II compared with WHO formulation I (Figure 2B). We also included the influenza A virus H1N1 in these inactivation experiments because of its importance in causing viral respiratory epidemics and pandemics. H1N1 could be inactivated at concentrations of 60% with WHO formulation I and 40% with WHO formulation II (Supplementary Figure 1B). Furthermore, MVA was studied for its susceptibility to WHO formulations because it is the chosen test virus for all enveloped viruses in the European Guideline. In line with EBOV and H1N1, similar inactivation profiles could be observed with increasing WHO formulation I and II concentrations (Supplementary Figure 1C).\nComparative Inactivation Profiles for World Health Organization Formulations Against Enveloped Viruses\nBased on the obtained virucidal activities of the WHO formulations against the different enveloped viruses, we next analyzed the inactivation profiles in a comparative analysis (Figure 2C and 2D). The most susceptible viruses to the WHO formulation I were the bovine and emerging CoVs (BCoV, SARS-CoV, MERS-CoV) and ZIKV (Figure 2C). With a shift to increasing WHO forumation I concentration, the more stable viruses included the full infectious EBOV (trVLPs excluded in this analysis) and HCV (Figure 2C). The highest alcohol-based concentrations of WHO formulation I (>40%) were required for H1N1 and MVA, which displayed nearly identical inactivation response curves (Figure 2C). The results for the isopropanol-based WHO formulation II are depicted in Figure 2D; it demonstrated a similar pattern of susceptibility for the different enveloped viruses with an obvious shift toward lower concentrations (Figure 2D). The CoVs and ZIKV showed the highest susceptibility to WHO formulation II, whereas HCV, EBOV, H1N1 and MVA demonstrated a more resistant inactivation profile (Figure 2D). To also directly compare the performance of the 2 WHO formulations, we determined the concentrations at which the products reached the EC50 (Supplementary Figure 2). World Health Organization formulation II showed a significantly higher virucidal activity against the different viruses compared with WHO formulation I (P = .008). In summary, CoVs and ZIKV showed the highest susceptibility to WHO formulations. Ebola virus and HCV were observed to be less susceptible than the CoVs, whereas H1N1 and MVA were the most stable viruses. In addition, WHO formulation II demonstrated a higher virucidal effect compared with WHO formulation I.\nDISCUSSION\nThe WHO has recommended 2 formulations in Guidelines on Hand Hygiene in Health Care, a document proposing the use of cheap alcohol-based hand rubs to reduce the transmission of pathogens [1]. We aimed in this study to analyze the virucidal efficacies of these products, particularly against emerging or re-emerging viruses that caused severe epidemics in the recent past [2]. Importantly, both WHO formulations inactivated all tested viruses, including ZIKV, EBOV, and emerging CoVs, in a suspension test with 30-second exposure time, implicating the usability of these formulations in viral outbreak situations. In the case of ZIKV, specific viral inactivation data are lacking, and consequently disinfection guidelines are based on data obtained from other members of the flaviviruses. So far, 1 recent study by M&#252;ller et al reported that ZIKV was inactivated by classical inactivation methods including ultraviolet light [11]. Zika virus was readily reduced in viral titers by the WHO formulations, similar to the other member of the family of Flaviviridae, HCV. These findings are supported by earlier analyses of the environmental stability and inactivation profiles of HCV, which showed strong virucidal effects of the main WHO formulation ingredients ethanol and isopropanol [12]. For EBOV, limited data on the efficacy of virucidal products are available because these viruses require high biosafety level laboratories. The Centers for Disease Control and Prevention advises &#8220;suitable disinfectant solutions include 0.5% sodium hypochloride as well as 2% glutaraldehyde and phenolic disinfectants (0.5&#8211;3%)&#8221; for EBOV inactivation [13]. The comparative inactivation analyses of all viruses tested revealed that the CoVs, in particular SARS-CoV, were the most susceptible viruses to WHO formulation treatment. The degree of susceptibility of the different viruses to the WHO formulation likely depends on the specific surface properties of the lipophilic envelope of the respective virus. We could show by a comparative inactivation analysis that H1N1 and MVA showed the highest stability against alcohol-based inactivation, with higher concentrations of WHO formulation I and II being required compared with CoVs, ZIKV, and EBOV. These results confirm MVA as the model surrogate virus for all enveloped viruses for testing chemical disinfectants and antiseptics in human medicine [8]. When testing nonenveloped viruses like noro-, polio-, or adenovirus, a far higher level of resistance to both WHO formulations was observed, probably due to the more hydrophilic character of these viruses [3, 6]. Interestingly, WHO formulation I was superior compared with WHO formulation II in inactivating these nonenveloped viruses, whereas in this study the opposite effect occurred, with WHO formulation II showing a higher virucidal activity against enveloped viruses. This discrepancy can be explained by the presence of the virus envelope, which likely renders enveloped viruses more susceptible to the isopropanol-based WHO formulation II compared with the ethanol-based WHO formulation I [14]. Furthermore, isopropanol has 1 more carbon than ethanol, giving it greater lipophilic properties and higher virucidal activities against lipophilic viruses [15].In conclusion, WHO-recommended alcohol-based formulations were validated with different enveloped viruses. A strong virucidal effect against emerging pathogens, including ZIKV, EBOV, SARS-CoV, and MERS-CoV, could be demonstrated, implicating the usability of these WHO formulations in healthcare and outbreak-associated viral infections.\nSupplementary Data\nSupplementary materials are available at The Journal of Infectious Diseases online. Consisting of data provided by the authors to benefit the reader, the posted materials are not copyedited and are the sole responsibility of the authors, so questions or comments should be addressed to the corresponding author.\nSupplementary Material\n",
        "pmid": "28453839",
        "title": "Virucidal Activity of World Health Organization&#8211;Recommended Formulations Against Enveloped Viruses, Including Zika, Ebola, and Emerging Coronaviruses",
        "journal_title": "The Journal of Infectious Diseases",
        "authors": "Anindya Siddharta;Stephanie Pfaender;Nathalie Jane Vielle;Ronald Dijkman;Martina Friesland;Britta Becker;Jaewon Yang;Michael Engelmann;Daniel Todt;Marc P. Windisch;Florian H. Brill;Joerg Steinmann;Jochen Steinmann;Stephan Becker;Marco P. Alves;Thomas Pietschmann;Markus Eickmann;Volker Thiel;Eike Steinmann",
        "doi": "10.1093/infdis/jix046"
    },
    {
        "keywords": "Coronavirus;Severe acute respiratory syndrome;Severe acute respiratory syndrome coronavirus 2;Coronavirus disease 2019;Outbreak;Fear;Discrimination;Prejudice",
        "abstract": "The current corona virus disease 2019 outbreak caused by severe acute respiratory syndrome coronavirus 2 started in Wuhan, China in December 2019 and has put the world on alert. To safeguard Chinese citizens and to strengthen global health security, China has made great efforts to control the epidemic. Many in the global community have joined China to limit the epidemic. However, discrimination and prejudice driven by fear or misinformation have been flowing globally, superseding evidence and jeopardizing the anti-severe acute respiratory syndrome coronavirus 2 efforts. We analyze this phenomenon and its underlying causes and suggest practical solutions.",
        "body": "INTRODUCTION\nThe corona virus disease 2019 (COVID-19) outbreak caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) appears to have started in Wuhan, China in December 2019[1-10]. Subsequently, it has spread within and outside China[10-12] and has become an unprecedented global public health issue[12-19]. Most cities in China heavily affected by the epidemic have been put on lockdown[1,2]. As of 18 February 2020, there have been 72528 confirmed infections, 1870 deaths in China (Figure 1 and Figure 2), and 804 confirmed cases with 3 deaths in 25 other countries[20]. To safeguard Chinese citizens and strengthen global health security, medical staff and government in China have made great efforts to control the epidemic[6-9,21]. It was confirmed that 1716 medical workers had been infected with SARS-CoV-2, and 6 died in hospitals for the control of epidemic on 14 February 2020[22]. Many in the global community have joined China in this effort[14,23,24].However, discrimination and prejudice driven by fear or misinformation have been flowing within and outside China[19,25-31], triggering panic and jeopardizing the response efforts of health workers and health authorities[25,32]. Importantly, discrimination, prejudice, and stigma make sick people reluctant to get medical help[19]. Nurses are prevented from accessing the areas where their rented houses are located[33]. This should be stopped[23,29].\nWHEN SOME PEOPLE FEAR, THEY MAY STIGMATIZE OTHER PEOPLE\nPsychologically, when the living environment changes, people feel unsafe, uneasy, and anxious[19]. When the cause of an epidemic is not clear, close-minded attitudes and rumors often flourish. Innocent people in Wuhan are targeted and unfairly blamed at a national level, and the same happens to Chinese people internationally[31].Attempts to control the epidemic have involved isolation and quarantine, lockdown of an entire city, cancellation of flights, and evacuation of foreign nationals from Wuhan. As the World Health Organization&#8217;s (WHO&#8217;s) Director of Global Infectious Hazard Preparedness Sylvie Briand said, \"Fear and stigma go together and when people fear, they tend to stigmatize some groups and what we try to do is to reduce this fear\"[34].\nFORMAL NOMINATION OF SARS-COV-2 AND COVID-19\nThe International Committee on Taxonomy of Viruses (ICTV) is responsible for developing the official classification of viruses and taxonomy of the Coronaviridae family. The Coronavirus Study Group (CSG) of ICTV evaluated the novelty of the new virus causing human infection in Wuhan, China tentatively named 2019-nCoV[35,36]. Based on phylogeny, taxonomy, and established practice, the CSG formally recognized this virus as a sister to SARS-CoVs of the species SARS-related coronavirus and designated it &#8220;SARS-CoV-2&#8221;[35]. China&#8217;s National Health Commission gave the virus a temporary name, the Novel Coronavirus Pneumonia[37]. The term &#8220;novel&#8221; may refer to that 2019-nCoV is only emerging and requires further studies and an incomplete match between the genomes of this and other previously known coronaviruses[35,38]. On February 11, 2020, WHO formally named the disease caused by SARS-CoV-2 as &#8220;COVID-19&#8221;[39]. WHO Director-General Tedros Adhanom Ghebreyesus said, &#8220;the name COVID-19 did not refer to a geographical location, an animal, an individual or group of people, and which is also pronounceable and related to the disease. Having a name matters to prevent the use of other names that can be inaccurate or stigmatizing. It also gives us a standard format to use for any future coronavirus outbreaks&#8221;[39].\nHARMFUL EFFECTS OF FEAR, STIGMA, OR DISCRIMINATION\nSome people believe that viruses are present in the air for quite a long time, and fear being infected, thus becoming very anxious and angry, targeting people from Wuhan, and blaming them for spreading the virus[31]. Anecdotal evidence has shown Wuhan's families being \"rejected and even hunted\"[40]. Travel bans have been imposed, and when some people see a car with a Wuhan plate, they hit the car even when the driver is inside[40]. In some rural areas, vigilantes have even dug a ditch across the road to their villages to stop &#8220;outsiders&#8221; from entering[40]. An Indian person who was mistakenly thought to have SARS-CoV-2 infection, was so scared of transmitting the virus, that he committed suicide to protect his villagers[41]. Some posts have accused the United States of introducing the virus into China[31,42]. The SARS-CoV-2 outbreak is a perfect metaphor that illustrates the negative side of human societies[31].Prejudice and bigotry have occurred with each huge epidemic in history. There is a link between imagining disease and imagining foreignness. For instance, when syphilis started to spread through Europe in the last decade of the fifteenth century, it was named a dreaded foreign disease, such as the Chinese disease by the Japanese, Morbus Germanicus by the Parisians, &#8220;French pox&#8221; by the English, Naples sickness by the Florentines. In 2003, Chinese were targeted for the epidemic of SARS-CoVs; in 2009, Latinos were widely targeted for the outbreak of swine flu; and in 2014, people of African origin were targeted for the Ebola crisis[31].Indeed, as far as the SARS-CoV-2 is concerned, the virus is named the \"China virus\" or \"Wuhan virus\" by some westerners[30,31]. At the end of January of 2020, a newspaper in Denmark published a cartoon of the Chinese national flag that replaced the five symbolic stars with virus-like figures[30,31]. On February 3, 2020, the Wall Street Journal published an opinion column headlined \"China Is the Real Sick Man of Asia\"[25,31,32]. The Australian Herald Sun made reference to the virus as to a \"Chinese virus\" in a subtitle under a headline and blazoned \"China Kids Stay Home\" on its front page[25,30]. French papers ran articles headlined \"New Yellow Peril\" and \"Yellow Alert\"[25,30]. Some Chinese people in the United States and in European countries have been verbally or physically insulted simply because they were wearing masks[30,43]. In some parts of Europe, some people of non-Chinese Asian heritage have had to make clear that they are not Chinese[30,31,43].\nSOLUTIONS TO STOP FEAR, STIGMA, AND DISCRIMINATION\nFear, prejudice, and discrimination can considerably impede anti-SARS-CoV-2 efforts[34]. A recent study of 138 hospitalized COVID-19 patients shows that SARS-CoV-2 can be transmitted from person to person through droplets and hand contact during an incubation period as well as on acute attack[2]. The closer we are to the truth, the farther we are from fear or panic. We think that open and transparent information on the outbreak to the public in China and the world, and administrative warning from authorities in every country are necessary to reduce fear and discrimination[34]. The health authorities should make the SARS-CoV-2 epidemic and essential preventive measures to the public in China and the world openly and transparently in time through radio, television, newspaper, WeChat, and the internet, i.e. washing hands and wearing a mask. A 24-h free hotline 7 d a week should be set up for public to respond to all the questions and concerns, relive the worries and panic, and clear the rumor or misinformation. The public hospitals should open COVID-19 clinics and mental health clinics to consult with people to clear panic and misinformation.Likewise, government authorities in every country should cooperate with China to oppose stigma, language abuse and discrimination caused by the SARS-CoV-2 epidemic and strictly abide by disciplines and laws stress that our enemy is SARS-CoV-2 rather than the fear and stigma.As Singapore's Prime Minister Lee Hsien Loong said, \"The virus may have started in China but it doesn't respect nationality or race. It doesn't check your passport before it goes into your body and anybody can be infected\"[30]. Indeed, the SARS-CoV-2 outbreak is not a Chinese issue, but a worldwide one. People can discriminate against each other, yet the SARS-CoV-2 never will. No boundary can stop the SARS-CoV-2 from interrupting the regular life of all of us on earth, reminding us that we are indeed a global community.Stigma and discrimination are more harmful than the SARS-CoV-2 itself for our global efforts to control the outbreak. As WHO Director-General Tedros Adhanom Ghebreyesus said, \"This is a time for facts, not fear. This is the time for science, not rumors. This is the time for solidarity, not stigma. This outbreak is a test of solidarity - political, financial and scientific. We need to come together to fight a common enemy that does not respect borders\"[23].\nCONCLUSION\nAlthough the COVID-19 outbreak caused by SARS-CoV-2 started in Wuhan, China, it is now a global public health issue rather than a Chinese issue. As global community, we need solidarity instead of stigma or discrimination to contain the spread of SARS-CoV-2 outbreak. An open and transparent information on the outbreak to the global public, and administrative warning from authorities in every country are necessary to reduce fear and discrimination.",
        "pmid": "32149049",
        "title": "Fear can be more harmful than the severe acute respiratory syndrome coronavirus 2 in controlling the corona virus disease 2019 epidemic",
        "journal_title": "World Journal of Clinical Cases",
        "authors": "Shi-Yan Ren;Rong-Ding Gao;Ye-Lin Chen",
        "doi": "10.12998/wjcc.v8.i4.652"
    },
    {
        "keywords": "WIV1-CoV;coronavirus;emerging infectious diseases;animal model",
        "abstract": "Severe acute respiratory syndrome (SARS)-like WIV1-coronavirus (CoV) was first isolated from Rhinolophus sinicus bats and can use the human angiotensin converting enzyme 2 (ACE2) receptor. In the current study, we investigate the ability of WIV1-CoV to infect Rousettus aegyptiacus bats. No clinical signs were observed throughout the experiment. Furthermore, only four oropharyngeal swabs and two respiratory tissues, isolated on day 3 post inoculation, were found positive for viral RNA. Two out of twelve bats showed a modest increase in coronavirus specific antibodies post challenge. In conclusion, WIV1-CoV was unable to cause a robust infection in Rousettus aegyptiacus bats.",
        "body": "1. Introduction\nEmerging infectious diseases form a significant threat to the human population. The natural history of viruses in animal reservoirs and their ability to infect the human host is important to predict the risk of spillover events and subsequent epidemics. The World Health Organization has identified a list of top emerging diseases likely to cause major epidemics. The list includes severe acute respiratory syndrome coronavirus (SARS-CoV), which caused a pandemic in 2002&#8211;2003 [1]. SARS-CoV is hypothesized to originate from Rhinolophus sinicus bats since several SARS-like viruses have been detected in this host [2,3]. However, most SARS-like viruses from bats do not bind the human receptor, angiotensin-converting enzyme 2 (ACE2). For example, a recombinant virus based on a consensus sequence of SARS-like viruses from bats could not be cultured unless a part of the spike protein was exchanged with SARS-CoV spike protein [4]. This suggests that a substantial portion of the SARS-like viruses circulating in bats cannot infect humans directly.In 2013, Ge et al. isolated a SARS-like virus from the Chinese horseshoe bat (Rhinolophus sinicus): WIV1-CoV [5], which can use human angiotensin-converting enzyme 2 (ACE2), the receptor for SARS-CoV, in vitro and transgenic mice [6]. This suggests WIV1-CoV may be able to directly transmit from bats to humans. In order to understand the relationship between coronaviruses and their bat hosts, experimental animal models are needed. Unfortunately, most bat species are difficult to obtain for laboratory experiments and most of these species have poorly annotated genomes. These limitations have posed a challenge in the field and explain a current deficit in the literature. In 2010, one group inoculated Leschenault&#8217;s rousette bats (Rousettus leschenaultia) with a homogenate of Betacoronavirus (clade b)-positive intestinal tissues obtained from a lesser short-nosed fruit bat (Cynopterus brachyotis). No clinical signs were observed, but viral RNA was detected in both fecal samples and intestinal tissue [7]. Likewise, no clinical signs were observed following intranasal and intraperitoneal inoculation of Jamaican fruit bats (Artibeus jamaicensis) with Middle East respiratory syndrome (MERS)-CoV. However, shedding of viral RNA was detected in oral and rectal swabs, and a variety of different tissues tested positive for viral RNA [8]. Together, these studies suggest that bats might show limited clinical signs when infected with coronaviruses but will continue to shed the virus.In this study, we investigate the ability of WIV1-CoV to infect and replicate in Rousettus aegyptiacus bats.\n2. Materials and Methods\n\n2.1. Ethics Statement\nAnimal experiments were approved by the Institutional Animal Care and Use Committee of the Rocky Mountain Laboratories (ASP 2016-021E, 05/2016). Experiments were performed following all guidelines and basic principles of the United States Public Health Service Policy on Humane Care and Use of Laboratory Animals and the Guide for the Care and Use of Laboratory Animals. Experiments with infectious WIV1-CoV under BSL3 conditions was approved by the Institutional Biosafety Committee (IBC). IBC-approved standard operating procedures were followed for inactivation and removal of samples from high containment.\n2.2. Virus and Cells\nAn infectious clone of WIV1-CoV was generated as previously described [6]. Virus propagation was performed in VeroE6 cells in DMEM (Sigma, St. Louis, MO, USA) supplemented with 2% fetal bovine serum (Gibco, Grand Island, NY, USA), 1 mM l-glutamine (Lonza), 50 U/mL penicillin, and 50 &#956;g/mL streptomycin (Gibco, Grand Island, NY, USA) (2% DMEM). VeroE6 cells and baby hamster kidney (BHK) cells were maintained in DMEM supplemented with 10% fetal bovine serum, 1 mM l-glutamine, 50 U/mL penicillin, and 50 &#956;g/mL streptomycin (10% DMEM). The virus was titrated by inoculating VeroE6 cells with tenfold serial dilutions of virus in 2% DMEM. Five days after inoculation, cytopathic effect (CPE) was scored and TCID50 was calculated from four replicates using the method of Spearman-Karber.\n2.3. Spike Incorporation into VSV Reporter Particles\nEquivalent volumes of VSV reporter particles encoding luciferase and WIV1-spike, MERS-CoV spike or SARS-CoV spike were concentrated over an OptiPrep cushion (10% in PBS; Sigma, St. Louis, MO, USA) at 20,000&#215; g for 2 h at 4 &#176;C. Particle pellets were resuspended in lysis buffer (1% SDS, 1&#215; ThermoFisher NuPage LDS (Waltham, MA, USA), 1&#215; ThermoFisher NuPage DTT), boiled for 10 min at 100 &#176;C and analyzed for FLAG expression on a 10% Bis-Tris PAGE gel (NuPage; Thermofisher).\n2.4. VSV Pseudotype Entry Assay\nPseudotyped luciferase VSV reporter particles were produced as previously described [9]. ACE2-transfected BHK cells were then infected with equal volumes of VSV reporter particles pseudotyped with MERS-CoV, SARS-CoV, or WIV1-CoV spike.\n2.5. ACE2-Dependent Replication Kinetics of WIV1\nBHK cells were transfected in 6-well plates with 4 &#956;g pcDNA3.1(+) containing ACE2 from Rousettus aegyptiacus (XM016118926) or homo sapiens (AB193259) using 7.5 &#956;L of Lipofectamine 3000 and 8 &#181;L 3000 reagent (Thermo Fisher, Waltham, MA, USA). Replication kinetics were determined by inoculating cells with a multiplicity of infection (MOI) of 0.01 WIV1-CoV in triplicate 24 h post-transfection. One hour after inoculation, cells were washed twice with PBS, and fresh media was placed on the cells. Supernatants were sampled at 0 and 48 h after inoculation. Virus titers in supernatants were determined as described.\n2.6. Animal Experiment\nTwelve male adult Egyptian fruit bats (Rousettus aegyptiacus) were obtained from the Catoctin Wildlife Preserve and Zoo in Thurmont, Maryland. All bats were inoculated with 105 TCID50 of WIV1-CoV via the intranasal (25 &#181;L per nare), intratracheal (100 &#181;L) and intra-esophageal route (100 &#181;L). Exams were performed daily and included measuring body weight and temperature, obtaining oropharyngeal, rectal and urogenital swabs, and scoring for clinical signs of disease. On day 3, 7, and 28, four animals were euthanized and nasal turbinates, larynx, pharynx, trachea, lung, brain, eye, conjunctiva, heart, liver, spleen, kidney, bladder, reproductive organs, stomach, proximal and distal intestinal tract, cervical lymph nodes, adrenal glands, skin, and skeletal muscle were collected for virological and histopathological analyses. Lung weights were measured upon necropsy.\n2.7. Viral RNA Detection\nTissues were homogenized in RLT buffer, and RNA was extracted using the RNeasy method on the QIAxtractor (Qiagen, Venlo, The Netherlands) according to the manufacturer&#8217;s instructions. RNA was extracted from swab samples using the QiaAmp Viral RNA kit on the QIAxtractor. For one-step real-time qPCR, 5 &#956;L RNA was used in the QuantiFast Probe PCR kit (Qiagen), according to instructions by the manufacturer. Standard dilutions of a virus stock with known titer were run in parallel in each run, to calculate TCID50 equivalents in the samples. Primers and probe were designed for detection of WIV1-CoV (Forward primer: TCAGGCTGGAAATGCTACAG, reverse primer: GTCCTCCACTTGCTAGGTAATC, probe: TGTGCTTTCCTTCTGTGCCTTTGC). Primers and probes specific for Rousettus aegyptiacus GAPDH and HPRT were designed as extraction controls (GAPDH: forward primer: GGTTGTCTCCTGCGACTTTA, reverse primer: CCTGTTGCTGTAGCCAAAT TC, probe: AAAGTGGTCATTGAGGGCAATGCC. HPRT forward primer: AGATGGTGAAGGTCGCAAG, reverse primer: CCTGAAGTATTCATTATAGTCAAGGG, probe: ACTTTGTTGGATTTGAAATTCCAGACA AGTTTG. All qRT-PCR cycles were as follows: 15 min at 50 &#176;C, 5 min at 95 &#176;C, then 40 cycles of 15 s at 95 &#176;C and 15 s (WIV1) or 30 s (Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) and Hypoxanthine-guanine phosphoribosyltransferase (HPRT)) at 60 &#176;C.\n2.8. Hematology\nEDTA blood was obtained from bats pre-challenge (D-95 and D-2) and day of necropsy. The total white blood cell, lymphocyte, neutrophil, monocyte, eosinophil, and basophil count were determined with the IDEXX ProCyte DX analyzer (IDEXX Laboratories, Westbrook, ME, USA).\n2.9. Histopathology\nTissues were fixed in 10% neutral buffered formalin for a minimum of seven days. Sagittal-cut skulls were decalcified in 20% EDTA in sucrose (Newcomer Supply, Middleton, WI, USA), changed weekly for six weeks. Tissues were then processed using a Sakura VIP-6 Tissue Tek tissue processor and embedded in Ultraffin paraffin polymer (Cancer Diagnostics, Durham, NC, USA). Samples were sectioned at 5 &#181;m, and resulting slides were stained with hematoxylin and eosin. For WIV1-CoV immunohistochemistry (IHC), tissues were stained with SARS NP rabbit polyclonal antibody (Novus Biologicals, NB100-56576, 1:250, Centennial, CO, USA). For ACE2 IHC, tissues were stained with ACE2 rabbit polyclonal antibody (Abcam, ab15348, 1:1500). Hereafter, antibodies were detected using ImmPress VR Polymer HRP anti-rabbit IgG (Vector Laboratories, MP-6401-15, no dilution, Burlingame, CA, USA) followed by Discovery ChromoMap DAB kit (Ventana Medical Systems, 760-159, Tucson, AZ, USA).\n2.10. Serology and Microneutralization\nEnzyme-linked immunosorbent assay (ELISA) was performed by coating SARS-CoV spike S1 or nucleoprotein (N) antigen (NR686 and NR48761, BEI resources, 0.5 &#181;g/mL in 50 mM bicarbonate binding buffer (4.41 g KHCO3 and 0.75 g Na2CO3 in 1 L water)) on to MaxiSorb plates (Nunc, Rochester, NY, USA). Plates were blocked with 5% non-fat dried milk in PBS-0.1% Tween (5MPT). Serum samples were diluted in 5MPT and incubated on plates for 1 h. Detection of antibodies was performed with HRP-conjugated IgG (H+L) secondary antibody and developing solution (Seracare, Milford, KS, USA) followed by a measurement at 450 nm. Sera was termed seropositive if the optical density (OD) value was higher than 0.2. To determine microneutralization titers, heat-inactivated sera (30 min at 56 &#176;C) was diluted in 2% DMEM, then incubated 1 h at 37 &#176;C and 5% CO2 with 100 TCID50 of WIV1-CoV. Serum:virus mixture was added to VeroE6 cells and five days after inoculation, CPE was scored. Neutralizing titer was determined from two replicates.\n2.11. Statistical Analysis\nStatistical analysis was performed using Student&#8217;s t-test on Graphpad Prism 7 for Windows. p-values < 0.05 were considered as significant.\n3. Results\nBecause variation in ACE2 has been shown to act as a host species barrier [10], we first tested if WIV1 spike could interact with Egyptian fruit bat ACE2. Single-cycle VSV luciferase reporter particles were generated with FLAG-tagged MERS-CoV, SARS-CoV or WIV1 spike (Figure 1A). All three coronavirus spike proteins incorporated into VSV reporter particles (Figure 1B). While MERS-CoV spike could only mediate entry into cells expressing DPP4, both SARS-CoV and WIV1 spikes mediated entry into cells expressing human and Egyptian fruit bat ACE2 (Figure 1C). Furthermore, transfection of BHK cells with either human or Egyptian fruit bat ACE2 and subsequent infection with WIV1-CoV resulted in virus replication as measured in supernatant at 48 h post infection (hpi), whereas no virus replication could be found in BHK cells not transfected with ACE2 (Figure 1D). Taken together, these data show that WIV1 spike can effectively utilize ACE2 from Egyptian fruit bats for entry and infection.To confirm ACE2 expression in the organs of the Egyptian fruit bats, specific staining for ACE2 was performed on nasal turbinates, trachea, lung, kidney, stomach, and intestinal tissues. ACE2 expression was identified multifocally within the nasal turbinates, apical olfactory epithelium, serous submucosal glands, and pars nervosa of the pituitary gland. Cytoplasmic endothelial cell immunoreactivity was found in lung and stomach tissue. The kidney showed expression multifocally within interstitial capillaries and parietal epithelium. Intestinal tissue showed a bright diffuse immunoreactivity at the brush border (Figure 2). Together, the staining suggests necessary receptor expression to produce infection with WIV1-CoV in Egyptian fruit bats.Twelve Egyptian fruit bats were inoculated via the intranasal, intratracheal and intra-esophageal route with 105 TCID50 of WIV1-CoV. These bats were observed daily for 14 days for any clinical signs. Their body weight and temperature were measured every day. None of the bats showed signs of disease such as respiratory distress, anorexia or lethargy. No significant body weight loss or changes in body temperature were detected throughout the experiment (Figure 3A,B).Rectal, urogenital and oropharyngeal swabs were taken daily for six days and tested for the presence of viral RNA. Viral RNA could not be detected in rectal or urogenital swabs at any time point. However, on days 1 and 3 post inoculation (dpi) viral RNA was detected in the oropharyngeal swabs of three bats and one bat, respectively (between 100&#8211;300 copies/mL, Figure 3C).Four bats each were necropsied on 3, 7, and 28 dpi. No postmortem gross pathological lesions were identified during necropsy. For each bat, the weight of the lungs was measured, and lung/body weight ratios were calculated. When comparing the lung/body weight ratios between animals taken on different days, no significant differences could be found (Student&#8217;s t-test, p &#8805; 0.05, Figure 3D).Multiple organ systems were examined histopathologically and were essentially normal with a few exceptions. All 12 bats had marked severe vacuolar change (glycogen-type) within the liver. The cause of this liver change was not identified but may be related to diet or stress. Six of the bats had minimal&#8211;moderate numbers of Kupffer cells in the liver containing a brown granular intracytoplasmic pigment predominately in the portal areas which are positive for Prussian Blue staining and consistent with excessive iron deposition. Seven bats had minimal&#8211;moderate numbers of lymphohistiocytic nodules within the pulmonary parenchyma. None of these histopathological changes were associated with coronavirus replication or immunohistochemistry suggesting that this was background pathology.RNA was extracted from harvested tissues and analyzed for the presence of the host and viral RNA. Host RNA could be detected in all samples. Viral RNA was only detected in tissues obtained from animals necropsied on 3 dpi and was limited to the pharynx (N = 1) and nasal turbinates (N = 1) (Figure 3E). Subsequently, the full skull, as well as cervical lymph nodes, lung, spleen, and liver tissue, were stained for WIV1-CoV antigen using a SARS NP-specific antibody. All tissues were negative as determined by immunohistochemistry.Complete blood cell counts of four bats on each day of necropsy were compared to values obtained pre-challenge (two values per bat, 23 values in total). No abnormal values were detected in the levels of white blood cells, neutrophils, lymphocytes, monocytes, eosinophils or basophils (Figure 4). We hypothesize that the observed lymphopenia is stress-related, and not associated with viral infection.The presence of antibodies was investigated using an ELISA, based on SARS-CoV S or N protein, and expressed as the highest dilution of sera that resulted in an OD > 0.2. Interestingly, sera obtained before challenge from several bats showed reactivity with SARS S or N (Figure 5). We compared the ELISA titer in sera obtained before and after challenge. Only two bats (bat #6 and #11) showed a significant increase in ELISA titer. No WIV1-CoV-neutralizing antibodies were detected in the sera as determined via virus neutralization assay. \n4. Discussion\nIn the present study, we investigated whether Egyptian fruit bats (Rousettus aegyptiacus) are susceptible to infection by SARS-like WIV1-CoV, originally detected in Chinese rufous horseshoe bats. The first step in host susceptibility to coronaviruses is associated with the ability to bind to host-specific receptors and the availability of these receptors on target organs. SARS-CoV S1 binds less efficiently to murine ACE2 than human ACE2, and thus wildtype mice are a suboptimal SARS-CoV animal model [10]. Upon expression of Egyptian fruit bat ACE2 on non-susceptible BHK cells, WIV1-CoV was able to enter and replicate, demonstrating that ACE2 from Egyptian fruit bats is a suitable receptor. However, while ACE2 was expressed in bat intestine and respiratory tract tissue, similar to humans [11], we observed very limited evidence of virus replication and seroconversion. Taken together, these data demonstrate that WIV1-CoV restriction in the Egyptian fruit bat was not at the receptor level. However, in this study very limited evidence of virus replication or seroconversion was detected. It is currently unclear why WIV1-CoV seems unable to replicate efficiently in Egyptian fruit bats. It is possible that the WIV1-CoV S glycoprotein is not processed by surface or intracellular proteases, which have been shown to be important host restriction factors during coronavirus entry [12,13]. Furthermore, a specific host factor may be required for virus transcription and replication, which could be lacking in the Egyptian fruit bats [14]. A recent study generated an annotated full genome for Egyptian fruit bats and used this to show greatly expanded natural killer cell receptors, MHC class I genes and type I interferons. It has been hypothesized that these factors allow greater tolerance to viral infection [15], resulting in prolonged infection but limited inflammation which has been described for Egyptian fruit bats inoculated with Marburg virus [16,17]. However, as we found very limited evidence of viral RNA, it is more likely that either an enhanced antiviral response or an inability of WIV1-CoV to dampen the innate response abrogated viral replication completely. Interestingly, some of the sera collected from bats before challenge contained antibodies which cross-reacted against full-length SARS-CoV spike protein as well as nucleoprotein. As most bats were negative for SARS-specific antibodies, we were unable to detect neutralizing antibodies. This does not directly explain why we did not see extensive viral replication. A small increase in the presence of SARS-CoV-specific antibodies was observed in 2 out of 12 bats; bat #6 showed an increase in N-specific antibodies, bat #11 showed an increase in S-specific antibodies. These findings suggest that some low-level virus replication might have occurred in these bats, resulting in an adaptive immune response.We observed cleavage of MERS S glycoprotein in our pseudotypes, whereas neither SARS S or WIV1 S glycoprotein were cleaved. This matches closely to what has been reported on proteolytic activation of MERS S and SARS S glycoprotein and further supports the validity of the use of pseudotypes. Whereas MERS S glycoprotein is processed by host proprotein convertases in virus-producing cells before virus-cell entry [18], SARS S glycoprotein processing by these convertases is absent or inefficient [19]. Our results suggest that like SARS S glycoprotein, WIV1 S glycoprotein does not get cleaved by host proprotein convertases.The obtained results suggest that the ability to infect and replicate efficiently is bat species-specific, and limited inferences can be made from non-matched host-pathogen systems. This has been observed with filoviruses as well. Marburg virus replicates efficiently in Egyptian fruit bats, which are the natural reservoir. However, other filoviruses, such as Ebola virus, hardly replicate in the Egyptian fruit bats [16]. Therefore, it would be interesting to perform an experimental inoculation study using Chinese horseshoe bats, from which WIV1-CoV was originally isolated, to investigate if more efficient virus replication and shedding can be observed in these animals.In conclusion, we were unable to detect efficient replication of WIV1-CoV in Egyptian fruit bats and observed modest seroconversion in 2 out of 12 bats. Our findings add to the current understanding of the intricate relationship between specific bat species and their coronaviruses.",
        "pmid": "30572566",
        "title": "SARS-Like Coronavirus WIV1-CoV Does Not Replicate in Egyptian Fruit Bats (Rousettus aegyptiacus)",
        "journal_title": "Viruses",
        "authors": "Neeltje van Doremalen;Alexandra Sch\u00e4fer;Vineet D. Menachery;Michael Letko;Trenton Bushmaker;Robert J. Fischer;Dania M. Figueroa;Patrick W. Hanley;Greg Saturday;Ralph S. Baric;Vincent J. Munster",
        "doi": "10.3390/v10120727"
    },
    {
        "keywords": "",
        "abstract": "The complete genome of a novel coronavirus was sequenced directly from the cloacal swab of a Canada goose that perished in a die-off of Canada and Snow geese in Cambridge Bay, Nunavut, Canada. Comparative genomics and phylogenetic analysis indicate it is a new species of Gammacoronavirus, as it falls below the threshold of 90% amino acid similarity in the protein domains used to demarcate Coronaviridae. Additional features that distinguish the genome of Canada goose coronavirus include 6 novel ORFs, a partial duplication of the 4 gene and a presumptive change in the proteolytic processing of polyproteins 1a and 1ab.",
        "body": "Introduction\nViruses belonging to the Coronaviridae family have a single stranded positive sense RNA genome of 26&#8211;31&#8201;kb. Members of this family include both human pathogens, such as severe acute respiratory syndrome virus (SARS-CoV)1, and animal pathogens, such as porcine epidemic diarrhea virus2. Currently, the International Committee on the Taxonomy of Viruses (ICTV) recognizes four genera in the Coronaviridae family: Alphacoronavirus, Betacoronavirus, Gammacoronavirus and Deltacoronavirus. While the reservoirs of the Alphacoronavirus and Betacoronavirus genera are believed to be bats, the Gammacoronavirus and Deltacoronavirus genera have been shown to spread primarily through birds3. The first three species of the Deltacoronavirus genus were discovered in 20094 and recent work has vastly expanded the Deltacoronavirus genus, adding seven additional species3.By contrast relatively few species within the Gammacoronavirus genus have been identified. There are currently two recognized species in the Gammacoronavirus genus: avian coronavirus (ACoV) and beluga whale coronavirus SW1 (SW1). ACoVs infect multiple avian hosts and include several important poultry pathogens, such as infectious bronchitis virus (IBV) and turkey coronavirus (TCoV)5. IBV was first described in the United States6 but has since been described around the globe7. Turkey Coronavirus is the cause of acute enteritis in domestic turkeys8. The second species in the Gammacornavirus genus SW1 was first discovered in beluga whales9 but has since been detected in other cetaceans, such as Indo-Pacific bottlenose dolphins10. Despite IBV being the first discovered coronavirus and the impact it has on the poultry industry11, the number of identified species within the Gammacoronavirus genus remains small in comparison to the other coronavirus genera. Coronaviruses from several other avian hosts for which partial sequences are available suggest relatedness to IBV and TCoV. These viruses, which include goose coronavirus (GCoV), were tentatively classified as part of the ACoV species. An approximately 3&#8201;kb region, including the nucleocapsid gene and several accessory genes, of GCoV were previously sequenced from a greylag goose in Norway12.Here we present the full genome of Canada goose coronavirus (CGCoV) sequenced directly from the cloacal swab of a Canada goose, which expired in a mass die-off in a remote region near the arctic in Nunavut, Canada. Our analyses demonstrate that it should be classified as a novel species in the Gammacoronavirus genus.\nResults and Discussion\nDue to the remote location of the die off, samples from the dead birds were not collected immediately and sent to a diagnostic laboratory until severe predation and decomposition had occurred. The poor sample quality, in addition to the difficulty of coronavirus isolation, led to the failure to isolate infectious virus using standard methods. However, the complete genome of a novel gammacoronavirus was assembled from high throughput sequencing reads derived from the cloacal swab of a single Canada goose. The assembled genome of the novel Canada goose coronavirus (CGCoV) is 28,539 nts in length (excluding the poly(A) tail) and has 38.4% GC-content. The genome of&#160;CGCoV is approximately 1000 nts longer than the reference genomes for ACoV available in GenBank. The genome organization of CGCoV is presented in Fig.&#160;1. The 5&#8242; UTR of CGCoV is 553 nt in length and contains a higher GC content (48.3%) relative to the genome as a whole. The 5&#8242; UTR of CGCoV shares only 68% pairwise identity with that of duck coronavirus (DCoV) and 47.5% pairwise identity to that of SW1. Like all coronavirus genomes reported to date, CGCoV&#8217;s genome is dominated by the coding regions for the large polyproteins 1a and 1ab, followed by the structural and accessory genes. The heptanucleotide slippery sequence UUUAAAC, associated with the ribosomal slippage that produces polyprotein 1ab, was present at nt positon 11,995. CGCoV&#8217;s genome contains genes for all four structural proteins common to coronaviruses; spike (S), envelope (E), membrane (M) and nucleocapsid (N). In addition, CGCoV contains 10 open reading frames (ORFs) predicted to encode accessory proteins. The order of the structural and accessory protein-coding ORFs in CGCoV resembles that of ACoV, but there are notable differences. The general genome organization of ACoV is 1ab-S-3a-3b-E-M-4b-4c-5a-5b-N-6b13. However, there is some variance in the genome organization within the ACoV species. For example, Australian IBV strains lack ORFs 4a, 4b and 5b14. Overall, CGCoV contains a larger number (n&#8201;=&#8201;14) of ORFs coding for predicted accessory and structural proteins downstream of the polyprotein 1ab coding region. Two additional ORFs (7a and 7b) are found between the CGCoV M and N ORFs. There are also two additional ORFs (10 and 11) following the N gene. While some ACoVs do have ORFs following the N gene, ORFs 10 and 11 in CGCoV do not share obvious homology to those of IBV and TCoV. The 3&#8242; UTR of CGCoV is 301 nucleotides in length and contains the stem loop-like motif 113&#8201;bp upstream from the poly(A) tail. This stem loop-like motif was first identified in astroviruses15 but is also present in ACoVs and SARS-CoV13. Further downstream in the 3&#8242; UTR, the octanucleotide motif (GGAAGAGC) is found 71&#8201;bp upstream of the poly(A) tail. The 3&#8242; UTR of CGCoV shares 98% pairwise identity to the partially sequenced GCoV and 84% pairwise identity to IBV.Figure 1Genome organization of Canada goose coronavirus. Purple indicates untranslated regions, blue indicates putative proteins, green indicates coding region of mature non-structural proteins (NSP) and red indicates transcription regulatory sequences (TRS). The stem loop-like motif and octamer motif are contained within the 3&#8242; UTR. Genome organization figure was constructed using GeneiousTM (Biomatters, v 9.1.8). *Indicate ACoV 4b homologues. Proteins are named numerically from the 5&#8242; end of the genome, with the exception of the structural genes, which are denoted by their common names.A trait suggesting common ancestry between CGCoV and ACoV is the canonical ACoV transcription regulatory sequence (TRS) found at the end of the leader sequence in CGCoV. The TRS of CGCoV is identical to that identified by Cao et al. (2008) as the TRS of TCoV (CTTAACAAA). Body TRS&#8217;s regulate viral gene expression by forming a complex with the leader TRS, causing discontinuous transcription of mRNA16. Ten putative body TRSs were found in the 3&#8242; end of the CGCoV genome (Fig.&#160;1). Four of the ten putative TRSs (4, 6, 8, 9) were exact matches to the canonical leader TRS. Three TRSs (2, 7, 11) contained one mismatch and the remaining three TRSs (3, 5, 10) contained two mismatches to the leader TRS. The functionality of these TRSs would need to be experimentally determined; however, previous studies have shown that TRSs of ACoVs are subject to some variation13,17. CGCoV contains twice the number of TRS&#8217;s as ACoVs and a similar number compared to the nine contained in SW19. Table&#160;1 demonstrates the nucleotide distances between the TRS and the start codon of ORFs found in CGCoV&#8217;s, which are comparable to those of TCoV13.Table 1Putative viral proteins of Canada goose coronavirus.ProteinTop Match in NCBITop match - aa % identity*Size (aa)Distance between TRS and start codon (nt)1a1a-Infectious bronchitis virus strain B16484338254801ab1ab-Infectious bronchitis virus strain ck/CH/LJL/05I576510480SpikeSpike-Infectious bronchitis virus strain N2-75531184823n/an/a5304an/an/a553EnvelopeEnvelope-Infectious bronchitis virus strain IS-149469100n/aMembraneMembrane-Duck Coronavirus isolate DK/GD/201472235745b4b-Infectious bronchitis virus strain Georgia 1998 Vaccine4188n/a6n/an/a6357a4b-Duck Coronavirus isolate DK/GD/2014239237bn/an/a69n/a8a5a-Duck Coronavirus isolate DK/GD/2014376548b5b-Duck Coronavirus isolate DK/GD/20144685n/aNucleocapsidNucleocapsid-Goose Coronavirus944149410ORFxg-Goose Coronavirus9297011ORFyg-Goose Coronavirus8118091*Matches below 20% coverage not shown.The start codon of CGCoV&#8217;s polyprotein 1ab is located 567 nucleotides downstream of the leader TRS. The coronavirus polyprotein 1ab is cleaved into 15&#8211;16 non-structural proteins (NSPs) by two viral proteases18. Putative cleavage sites for these proteases are present in CGCoV&#8217;s 1a and 1ab polyproteins, with the exception of the NSP 10/11 (polyprotein 1a) and NSP 10/12 (polyprotein 1ab) cleavage sites. The missing cleavage site would be located near the end of polyprotein 1a, producing the NSPs 10 and 11, and also in the alternatively transcribed polyprotein 1ab, producing NSPs 10 and 12. The absence of the NSP10/11 and 10/12 protease recognition site was confirmed with Sanger sequencing. With the exception of the missing cleavage sites, the putative cleavage sites would produce NSPs of sizes congruent with other Gammacoronavirus species (Table&#160;2). No Gammacoronavirus species to date, including CGCoV, have a papain-like protease cleavage site between NSP 1-219.Table 2Non-structural proteins size and cleavage site of gammacoronaviruses.ProteinCGCoVTCoVIBVSW1Cleavage siteSize aaCleavage siteSize aaCleavage siteSize aaCleavage siteSize aaNSP1/2AG^GH609AG^GK673AG^GK673VD^GD636NSP3AG^GV1532AG^GV1594AG^GI1592LG^GV1586NSP4LQ^AG503LQ^AG514LQ^SG514LQ^AG537NSP5LQ^SN307LQ^SS307LQ^SS307LQ^SN303NSP6VQ^SK295VQ^SK297VQ^AK293VQ^SK303NSP7LQ^AV83LQ^SV83LQ^SV83LQ^AV83NSP8LQ^NN212LQ^NN210LQ^NN210LQ^NN198NSP9LQ^GK111LQ^SK111LQ^SK111LQ^HG112NSP10\nSRFV*\n\n173\n\nVQ^SA\n\n145\n\nVQ^SV\n\n145\n\nLQ^SV\n\n189\nNSP11\n&#8212;\n\n&#8212;\n\n&#8212;\n\n23\n\n&#8212;\n\n23\n\n&#8212;\n\n17\nNSP12\nSRFV*\n\n1101\n\nVQ^SA\n\n941\n\nVQ^SV\n\n940\n\nLQ^SV\n\n926\nNSP13LQ^SC599LQ^SC601LQ^SC600LQ^AS601NSP14LQ^SN522LQ^GT521LQ^GT514LQ^SQ528NSP15LQ^SI338LQ^SI338LQ^SI338LQ^SL349NSP16LQ^SG298LQ^SA302LQ^SA302LQ^SD312*Amino acids present in CGCoV where putative protease cleavage sites were observed in TCoV, IBV and SW1.While the genome structure of CGCoV resembles that of ACoV, there are some notable differences. For example, there are no homologues to ACoV&#8217;s 3a or 3b accessory proteins in CGCoV, a trait shared with SW1. Furthermore, CGCoV has a number of ORFs that do not appear to have homologues in other sequenced Gammacoronavirus species, such as the ORFs for putative proteins 3 and 4a (Fig.&#160;1). These two ORFs are found in CGCoV in the corresponding location of ACoV&#8217;s 3a and 3b ORFs (between the S and E ORFs) and are also similar in size to ACoV&#8217;s 3a and 3b proteins. However, they share no obvious sequence similarity with any 3a or 3b gene, or any other entry in NCBI (Table&#160;1). ACoV&#8217;s 3a and 3b proteins have been shown to be unnecessary for replication20, however knock-out mutants for these accessory genes are attenuated21. The IBV&#8217;s 3 gene is functionally tricistronic, meaning the 3a, 3b and E proteins are under the control of a single TRS22,23. This is not the case in CGCoV, as the E ORF of CGCoV shares a TRS with only the 4a ORF in CGCoV and 3 ORF is preceded by a separate TRS (Fig.&#160;1).An additional TRS is also found in between CGCoV&#8217;s M and N ORFs, preceding the proteins 7a and 7b (Fig.&#160;1). Commonly ACoV&#8217;s have two ORFs between the M and 5 genes, coding for the 4b and 4c accessory proteins. CGCoV contains 4 ORFs between the M and 8 gene (ACoV 5 gene homologue). Two of these ORFs (5b and 7a) are ACoV 4b homologues, likely the result of gene duplication. This area in IBV has been identified as a hotspot for recombination24. The region between the ACoV M and 5 gene was formally called the intergenic region because of the lack of a TRS. However, it was later shown that gene 4 is expressed using an alternative TRS in IBV17. Notably, one of the 4b homologs (i.e. 5b) in CGCoV does have a TRS (Fig.&#160;1). The use of template switching at TRSs is thought to lend to recombination in coronaviruses25. The two CGCoV 4b homologs are not identical to each other (Table&#160;1). Amino acid sequence identity to other 4b proteins is low for both CGCoV 4b homologues, 41% to IBV and 23% to DCoV respectively. The gene 4 duplication was also confirmed by Sanger sequencing of the genomic region between the M ORF to the 8 gene.The ACoV 5a and 5b accessory proteins (8a and 8b in CGCoV) appear to be the only accessory proteins conserved in all 3 Gammacoronavirus species, although gene order differs. ORFs encoding putitive proteins 5a and 5b belong to the bicistronic gene 5 of ACoVs and are also unnecessary for replication21. To date, all publically available sequence information suggest that Gammacoronavirus species have lost the NSP1 cleavage site. The function of NSP1 in alphacoronaviruses and betacoronaviruses is the inhibition of host protein production. Accessory protein 5a is shown to have adopted this function in place of NSP1 in IBV19.The majority of structural proteins of CGCoV also share low amino acid sequence identity (53&#8211;72%) with IBV and DCoV. Phylogenetic analysis of the spike gene show that the CGCoV spike gene clusters with the IBV spike gene, separate from the TCoV cluster (Fig.&#160;2a). Figure&#160;2b also demonstrates the nucleocapsid gene of CGCoV is distantly related to those of ACoVs. However the CGCoV nucleocapid protein does share 94% amino acid sequence identity with the nucleocapsid protein encoded in the partially sequenced graylag GCoV genome13. In addition, ORFs 10 and 11, which are preceded by the nucleocapsid gene, also share high amino acid identity with graylag GCoV proteins, 92% and 81% respectively. It should be noted that, among full and partial genomes of gammacoronaviruses sequenced to date, ORFs 10 and 11 seem to be unique to CGCoV and GCoV and are both preceded by a TRS, suggesting that these ORFs are very likely expressed. The fact that some CGCoV proteins share higher amino acid sequence similarity with the partial GCoV sequences available suggest these two viruses are more closely related to each other than to other gammacoronaviruses known to date.Figure 2The phylogeny of gammacoronavirus spike and nucleocapsid proteins. A maximum likelihood tree built, using the amino acid sequences of the spike protein (a) and nucleocapsid protein (b) domains aligned with ClustalW31, in MEGA X using the Jones-Taylor-Thornton (JTT) substitution model and 1000 bootstraps32. IBV Infectious Bronchitis virus, TCoV Turkey Coronavavirus, PCoV Pigeon Coronavirus, DCoV Duck Coronavirus.The phylogenetic tree built using the coding regions for the conserved replicase and helicase domains demonstrates that CGCoV clusters with gammacoronaviruses and shares a more recent common ancestor with ACoV than with the cetacean gammacoronaviruses (Fig.&#160;3). Further comparisons suggest that CGCoV is a separate species from ACoV. Current taxonomy of Coronaviridae is determined using pairwise comparisons of the amino acid sequence of seven conserved domains in the 1ab polyprotein. Members of the same species share over 90% amino acid identity in these seven conserved domains5. Percent identity of CGCoV falls well below the 90% threshold set by ICTV with ACoV and SW1, suggesting CGCoV is a separate species (Table&#160;3). Within Coronaviridae, CGCoV shares the highest homology (68%) in the 7 conserved domains to the gammacoronaviruses TCoV and DCoV.Figure 3The phylogeny of Canada goose coronavirus. A maximum likelihood tree built, using the concatenated&#160;amino acid sequences of the replicase and helicase protein domains aligned with ClustalW31, in MEGA X using the Jones-Taylor-Thornton (JTT) substitution model and 1000 bootstraps32. Numbers at nodes indicate the bootstrap value.Table 3Comparison of the amino acid pairwise identity of 7 conserved coronavirus domains in the poly1ab protein of Canada goose coronavirus to other gammacoronaviruses.Domainaa % identity to IBVaa % identity to TCoVaa % identity to DCoVaa % Identity to SW1ADP-ribose-1&#8243;-phosphatase424338233C-like Protease56585749RdRp80808369Helicase 189909278Exonuclease78727756Endoribonuclease53535441Ribose-2&#8242;-O methyltransferase74777665Average67686854As the full genome was sequenced from only the cloacal swab of a single Canada goose, a screening PCR was designed based on the 4b duplication region unique to CGCoV and performed on all samples. The Sanger sequencing primers of the region between the M and 8 gene were used, as this area of the genome is specific to CGCoV. All samples were found to be positive, with the exception of the pharyngeal swab of the snow goose and the lung tissue of the second Canada goose which could not be tested as the sample was exhausted. Amplicons were Sanger sequenced and confirmed to match the CGCoV genome. High throughput sequencing conducted on RNA extracted from cloacal swabs from the second Canada goose and the snow goose also resulted in partial (64 and 18%) genomes of the CGCoV. While this does not confirm the virus&#8217;s presence in all animals that perished in the die off, this shows CGCoV was present in all birds that were available for testing. Further studies will require the availability of an infectious virus to determine the pathogenicity of CGCoV and its ability to cause mortality in Canada geese and snow geese.To summarize, the complete genome of CGCoV, a novel Gammacoronavirus species was sequenced directly from the cloacal swab of a Canada goose associated with a mass die-off. The CGCoV genome was also detected in samples derived from a second Canada goose and a snow goose that perished in the die-off, using PCR, Sanger and high throughput sequencing. Comparative genomics and phylogenetic analysis indicate CGCoV clusters with ACoV but is a distinct Gammacoronavirus species. Interesting features of this new species include the presence of two 4b homologues, a putative change in the proteolytic processing of the polyproteins 1a and 1ab, and six novel accessory genes.\nMethods\n\nSource of samples\nA large die off of Canada and snow geese occurred in the fall of 2017 near the arctic in Cambridge Bay, Nunavut, Canada. Due to poor carcass quality and remote location, samples were only collected from two dead Canada geese and one Snow goose, all of which had undergone predation and decomposition. Cloacal and pharygenal swabs were collected from all three birds, lung tissue was collected from one Canada goose. Other organs were not present or were in extremely poor condition. Detection of both common avian pathogens, such as avian influenza and avian paramyxovirus by the National Reference Laboratory, by routine laboratory testing gave negative results. Virus isolation was performed by two serial passages in SPF chicken eggs using protocols prescribed by the World Organization for Animal Health (OIE) for the most closely related gammacoronavirus, infectious bronchitis virus (IBV). Samples were then subjected to targeted sequence enrichment26 and next-generation sequencing on an Illumina MiSeq platform.\nSample pre-treatment\nTissues were homogenized using a Precellys Evolution homogenizer (Bertin Instruments) according to the manufacturer&#8217;s instructions. Following a clarification by centrifugation at 3000&#8201;rpm for 10&#8201;minutes, nucleic acids were extracted using the MagMAX Pathogen RNA/DNA Kit (Ambion) according to the manufacturer&#8217;s instructions.cDNA synthesis was then performed using SuperScript&#8482; IV First-Strand Synthesis System (SSIV) (ThermoFisher) according to the manufacturer&#8217;s recommendation. A total of 11 uL of extracted total nucleic acid was mixed with dNTPS (10&#8201;mM) and a tagged random nonamer primer (40 uM) (GTT TCC CAG TCA CGA TAN NNN NNN NN). Samples were incubated at 65&#8201;&#176;C for 5&#8201;minutes, and then placed on ice for 1&#8201;minute. A reagent mixture of 5x SSIV Buffer, Ribonuclease Inhibitor (40 U/&#956;L), DTT (100&#8201;mM) and SuperScript&#8482; IV Reverse Transcriptase was then added. The samples were incubated for 10&#8201;minutes at 23&#8201;&#176;C, 10&#8201;minutes at 50&#8201;&#176;C and 10&#8201;minutes at 80&#8201;&#176;C.Second strand synthesis was performed using Sequenase Version 2.0 DNA Polymerase (ThermoFisher) according to the manufacturer&#8217;s recommendation. The first strand synthesis product was incubated with 10 uL of Sequenase Version 2.0 DNA Polymerase diluted in 5x reaction buffer and nuclease free water. Samples were then heated to 37&#8201;&#176;C over five minutes and incubated at 37&#8201;&#176;C for 12&#8201;minutes, followed by 2&#8201;minutes at 95&#8201;&#176;C. Samples were then cooled to 10&#8201;&#176;C and 1.2 uL of Sequenase DNA polymerase in dilution buffer was added. Samples were again ramped to 37&#8201;&#176;C over five minutes and incubated at 37&#8201;&#176;C for 12&#8201;minutes, followed by 8&#8201;minutes at 95&#8201;&#176;C. A total of 6 uL of the second strand synthesis product was then used as template for amplification. AccuPrime&#8482; Taq DNA Polymerase (Thermofisher) was mixed with 10X AccuPrime&#8482; PCR Buffer I, nuclease free water and a primer for the nonomer&#8217;s tag (100 uM). 30 cycles of PCR were then performed with the following parameters: 30&#8201;seconds at 94&#8201;&#176;C, 30&#8201;seconds at 40&#8201;&#176;C, 30&#8201;seconds at 50&#8201;&#176;C and 1&#8201;minute at 72&#8201;&#176;C. cDNA/DNA mixtures were then cleaned with Genomic DNA Clean & Concentrator columns (Zymo Research) and eluted in 20&#8201;mM Tris (ThermoFisher).\nLibrary preparation and sequencing\nSequence libraries were prepared with the KAPA HyperPlus library kit (Roche). Sequence library construction and capture were carried out according to Nimblegen&#8217;s SeqCap EZ HyperCap Workflow User&#8217;s Guide v1. Samples were pooled in equal amounts by weight prior to capture. Sequencing was performed on an Illumina Miseq instrument in the National Centre for Foreign Animal Disease biocontainment level 3 sequencing facility. A V2 flow cell was used with a 500 cycle reagent cartridge (Illumina).\n5\u2032 Race and Sanger sequencing\n5&#8242; RACE was used to obtain the missing leader sequence (52&#8201;bp). The SMARTer 5&#8242; RACE and 3&#8242; RACE kit (Takarabio) was used according to the kit instructions. The gene specific primer used for 5&#8242; RACE was TCAGCTACAGTAGAGGGAGATGTCATAGGTGC. For Sanger sequencing, amplicons was performed using KAPA HiFi HotStart ReadyMixPCR Kit (KAPABiosystems). The primers CTAAAGAGAAGGTGGACACTGGT and CTAAGAATGCGAACTTCACAGAGC were used to amplify the gene 4b homologue region. The primers GTTGTTGTGTTACAAGGCAAGGG and GGATTATGATCAAACCATGAACCTGG were used to amplify the NSP 10/12 region. Cycling conditions used to generate amplicon for Sanger sequencing were: 1 cycle: 95&#8201;&#176;C for 3&#8201;minutes, 40 cycles: 98&#8201;&#176;C for 20&#8201;seconds, 65&#8201;&#176;C for 15&#8201;seconds, 72&#8201;&#176;C for 2.5&#8201;minutes, and 1 cycle: 72&#8201;&#176;C for 3&#8201;minutes. Amplicons were cleaned using AMPure XP beads (Beckman Coulter) according to the manufacturer&#8217;s directions. Sanger sequencing was performed on the ABI Genetic Analyzer 3130XL platform using the BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems) according to the user manual.\nBioinformatics\nRead quality was assessed using FastQC and trimmed using Trimmamatic27 (Version 0.36). Host reads were then filtered with RAMBO- K, using the only complete genome of a goose species (Anser cygnoides) currently available and DCoV28. The near complete genome sequence of CGCoV was assembled from NGS derived sequences from a cloacal swab of one Canada goose using SPAdes29. Sanger reads were aligned to the draft genome in GeneiousTM (Biomatters, v 9.1.8). Annotations were performed using Geneious and protein domains were identified using PFAM30. The Canada goose coronavirus genome is available under accession number MK359255 on NCBI.",
        "pmid": "30976080",
        "title": "Genome Organization of Canada Goose Coronavirus, A Novel Species Identified in a Mass Die-off of Canada Geese",
        "journal_title": "Scientific Reports",
        "authors": "Amber Papineau;Yohannes Berhane;Todd N. Wylie;Kristine M. Wylie;Samuel Sharpe;Oliver Lung",
        "doi": "10.1038/s41598-019-42355-y"
    },
    {
        "keywords": "bovine coronavirus;equine coronavirus;horses;vaccine",
        "abstract": "A vaccine for equine coronavirus (ECoV) is so far unavailable. Bovine coronavirus (BCoV) is antigenically related to ECoV; it is therefore possible that BCoV vaccine will induce antibodies against ECoV in horses. This study\ninvestigated antibody response to ECoV in horses inoculated with BCoV vaccine. Virus neutralization tests showed that antibody titers against ECoV increased in all six horses tested at 14 days post inoculation, although the\nantibody titers were lower against ECoV than against BCoV. This study showed that BCoV vaccine provides horses with antibodies against ECoV to some extent. It is unclear whether antibodies provided by BCoV vaccine are effective\nagainst ECoV, and therefore ECoV challenge studies are needed to evaluate efficacy of the vaccine in the future.",
        "body": "",
        "pmid": "28993568",
        "title": "Antibody response to equine coronavirus in horses inoculated with a bovine coronavirus vaccine",
        "journal_title": "The Journal of Veterinary Medical Science",
        "authors": "Manabu NEMOTO;Toru KANNO;Hiroshi BANNAI;Koji TSUJIMURA;Takashi YAMANAKA;Hiroshi KOKADO",
        "doi": "10.1292/jvms.17-0414"
    },
    {
        "keywords": "MERS-COV;Pregnancy;ARDS;Perinatal outcome",
        "abstract": "Background\nMiddle East Respiratory Syndrome coronavirus (MERS-CoV) is a viral respiratory disease. Most people infected with MERS-CoV develop severe acute respiratory illness. It was first reported in Saudi Arabia in 2012 and has since spread to several other countries. We report the clinical course of MERS-CoV infection in a pregnant woman who acquired the infection during the last trimester.Case presentation\nThe patient is a 33-year-old female working as a critical care nurse. She was 32&#160;weeks pregnant when she presented with respiratory symptoms after direct contact with a MERS-COV patient. Although the patient was in respiratory failure, necessitated mechanical ventilation, and intensive care (ICU) admission, a healthy infant was delivered. The mother recovered. To the best of our knowledge, this is the first reported case of a laboratory-confirmed Middle East Respiratory Syndrome Coronavirus in a pregnant woman.Conclusions\nMiddle East Respiratory Syndrome coronavirus (MERS-CoV) known to cause severe acute respiratory illness associated with a high risk of mortality Various factors may have contributed to the successful outcome of this patient such as young age, presentation during the last stages of pregnancy, and possible differences in immune response.",
        "body": "Background\nMiddle East Respiratory Syndrome coronavirus (MERS-CoV) is a novel coronavirus known to cause severe acute respiratory illness associated with a high risk of mortality. As of August 17 2015, 1432 laboratory-confirmed cases of infection with MERS-CoV, including at least 507 deaths, have been confirmed worldwide [1]. In pregnant women, the risk of viral pneumonia is significantly higher than for the rest of the population according to data collected from the previous 1957&#8211;1958 pandemics, and the H1N1 influenza pandemic of 2009 [2, 3]. Pregnant women with severe acute respiratory syndrome (SARS) appear to have a worse clinical outcome and a higher mortality rate compared to non-gravid women [4, 5]. Rates of maternal mortality, stillbirth, spontaneous abortion, and preterm delivery have all been elevated in viral pneumonia such as influenza-A, virus subtype H1N1, and SARS. While there are no clinical or serologic reports suggesting transmission of SARS coronavirus to the fetus, vertical transmission has been reported for H1N1 and Respiratory Syncytial Virus (RSV) [4, 6]. Data on the effects of MERS-CoV on pregnancy are limited; two cases of MERS-CoV in pregnancy have been reported to this day. The first report involved a stillbirth at 5&#160;months of gestation in a woman with MERS-CoV infection in Jordan [7]. The other involved a woman in the United Arab Emirates with MERS-CoV infection during the 3rd trimester who died after giving birth to a healthy baby with no evidence of MERS-CoV infection [8]. We report the clinical course of MERS-CoV infection in a pregnant woman who acquired the infection during the last trimester of pregnancy during a large hospital outbreak.\nCase presentation\nThe patient was a 33-year-old female working as a critical care nurse, with a prior history of hypothyroidism and primary infertility. She had undergone successful in vitro fertilization prior to presentation. She had been exposed to a patient with a respiratory illness, later identified as MERS-CoV infection, and was using standard contact and droplet precautions while under her care. Three days after exposure, she began to exhibit a dry cough and fever prompting her to visit a family medical clinic, where she was prescribed antipyretics. Over the following week, her symptoms gradually worsened, with increasing cough, shortness of breath, and persistent fever prompting her to come to the emergency department. She was hospitalized and diagnosed with pneumonia. Two nasopharyngeal swabs were taken, and the patient was admitted to a negatively pressurized room with contact, droplet, and airborne transmission precaution protocols in place.On admission, her highest recorded temperature was 39.2&#160;&#176;C, and her oxygen saturation was maintained via room air (97&#8201;&#8722;&#8201;99&#160;%). Chest radiography showed diffuse bilateral lower lobe infiltrates (Fig.&#160;1). Other laboratory findings included a leukocyte count of 6600/mm3 comprised of 81&#160;% polymorphonuclear cells, 13&#160;% lymphocytes, and 5&#160;% monocytes; a platelet count of 127,000/mm3; alanine aminotransferase concentration of 254 U/L; and aspartate aminotransferase concentration of 258 U/L. The patient displayed normal renal function. A fetal ultrasound, performed upon admission, demonstrated a live intrauterine fetus of approximately 31&#160;weeks of gestational age, and a posteriorly located placenta. MERS-CoV RNA was not detected in a nasal swab taken at that time.Fig. 1Chest Radiography (postero-anterior view) showing bilateral infiltrationTreatment with intravenous imipenem, vancomycin, azithromycin and oseltamivir were initiated. Despite antibiotic therapy, the patient became increasingly dyspneic over the following 3&#160;days. Arterial blood gas analysis showed pH&#160;7.34, PaCO2 36&#160;mmHg, and PaO2 73&#160;mmHg while on room air. She also exhibited radiographic evidence of progressive pulmonary infiltrates. Five hours later, the patient developed respiratory failure. She required mechanical ventilation, and dexamethasone was administered twice as prophylaxis for the fetus. A cesarean section was performed at 32&#160;weeks of gestation, and a healthy 1.79&#160;kg preterm male infant was delivered without complications. Following an uneventful delivery, the patient was transferred to the intensive care unit (ICU). Her oxygen requirement increased to 100&#160;%, chest radiography revealed bilateral consolidation, and her condition progressed to acute respiratory distress syndrome (ARDS). Nine days after the initial onset of infection, polymerase chain reaction (PCR) analysis, using real-time reverse transcription PCR (rRT-PCR) targeting genes E and 1a open reading frame, of tracheal aspirate confirmed the presence of MERS-COV infection [9]. Combination therapy with IFN-alpha-2b and ribavirin were initiated, as was methylprednisolone for ARDS. Subsequent septic screening samples including urine, blood, and sputum cultures were all negative for bacterial super infections.Over the next 4&#160;days, the patient began to improve. Chest radiography showed interval improvement, her fever resolved, her oxygen requirement decreased. Repeated blood tests showed normal leukocyte and platelet counts, and liver function test came back to normal. Five days later, the patient was weaned from mechanical ventilation, and transferred to the medical ward. She continued to recover uneventfully, and was discharged 28&#160;days after hospital admission.All health care workers taking care of her were asymptomatic with negative nasopharyngeal swab for MERS-Cov PCR, except for a 50-year old female nurse, with no comorbidities, from the obstetrics and gynecology ward. The patient was under her care prior to being transferred to the ICU with progressive and severe pneumonia. She had unprotected exposure for 2&#160;min with 50&#160;cm distance separation between her and the patient. The nurse clearly mentioned that the patient was coughing in her direction during that transient exposure. She was later admitted for 3&#160;days with self-limiting MERS-Cov pneumonia, based on positive MERS-Cov PCR nasopharyngeal swab and bilateral lung infiltrate. She was discharged and sent home in a stable condition.The male infant in the case reported herein was delivered by emergency cesarean under contact, droplet, and airborne transmission precautions. He was kept in the neonatal unit for observation, and was fed artificial formula in replacement of breast milk. Repeated nasopharyngeal swabs PCR analyses showed a consistently negative result for MERS-CoV. All healthcare workers in contact with the infant during his delivery, and subsequent care, have remained asymptomatic.\nDiscussion\nThe severity of viral pneumonia in pregnancy is evidently related to physiological and immunological changes that result in a shift from cell-mediated to humoral-mediated immunity [10]. Different outcomes in pregnancy have been reported in association with different viral respiratory illnesses, including pandemic influenza, H1N1, and SARS [4]. Among the 12 gravid women who presented with SARS in Hong Kong, between February 1st and July 31st in 2003, 50&#160;% required ICU admission, 33&#160;% required mechanical ventilation, 57&#160;% who presented during the first trimester had spontaneous miscarriages, and 80&#160;% of those presenting late in pregnancy underwent preterm deliveries. Over 80&#160;% of the women gave birth via emergency cesarean secondary to failure at maintain adequate blood oxygen saturation, despite being on 100&#160;% oxygen [4].In the present report, we have described a serious case of MERS-CoV during the 3rd trimester of pregnancy, requiring mechanical ventilation. This case differs from those reported previously that were associated with stillbirth in the 2nd trimester, during the MERS-CoV outbreak that occurred in Jordan from February through April in 2012 [8]. Our patient had an uneventful perinatal course, and a successful outcome. Various factors may have contributed to this, including the timing of MERS-CoV exposure during pregnancy, her young age, the use of steroids, and potential differences in immune responses. While combination therapy with ribavirin and interferon was administered, it is unlikely that either contributed to the successful outcome; because both were given after delivery, as MERS-COV was not confirmed until later in the course of her illness.There is limited information available on pregnancy, during antenatal, birth, or postnatal period, and MERS-CoV infection. Antiviral therapy has not yet been approved for the treatment of gravid patients with MERS-CoV infection. A retrospective cohort study in patients with severe MERS-CoV infection showed that the combined administration of ribavirin and IFN-alpha-2a seemed to significantly improve survival rate at day 14 but not day 28 [11]. However, other retrospective studies have failed to show any improvement in the mortality rate following this combination therapy [12]. During the SARS outbreak of 2003, ribavirin was used in pregnant women with the most severe forms of the illness. However, ribavirin therapy increases the risk of teratogenic effects in newborns [13]. Therefore, the use of this drug is not recommended during pregnancy or lactation. Drug treatment of MERS-CoV in pregnancy requires further investigation in the clinical setting.This report provides an initial view of the outcome associated with pregnancy-related MERS CoV infection. Further data on larger numbers of gravid women infected with MERS-CoV will facilitate a better understanding of the impact of MERS-CoV infection on perinatal outcome.\nConclusion\nMiddle East Respiratory Syndrome coronavirus (MERS-CoV) known to cause severe acute respiratory illness associated with a high risk of mortality Various factors may have contributed to the successful outcome of our patient infected with MERS-COV, including young age, presentation in late trimester, and differences in immune responses.\nConsent\nWritten informed consent was obtained from the patient for publication of this case report and any accompanying images. A copy of the written consent is available for review by the Editor of this journal.",
        "pmid": "26936356",
        "title": "Impact of Middle East Respiratory Syndrome coronavirus (MERS&#8208;CoV) on pregnancy and perinatal outcome",
        "journal_title": "BMC Infectious Diseases",
        "authors": "Haleema Alserehi;Ghassan Wali;Abeer Alshukairi;Basem Alraddadi",
        "doi": "10.1186/s12879-016-1437-y"
    },
    {
        "keywords": "Codon usage;coronavirus;spillover;coevolution",
        "abstract": "Abstract\nThis paper reports on an investigation of the role of codon usage evolution on the suggested bovine-to-human spillover of Bovine coronavirus (BCoV), an enteric/respiratory virus of cattle, resulting in the emergence of the exclusively respiratory Human coronavirus OC43 (HCoV-OC43). Analyses based on full genomes of BCoV and HCoV-OC43 and on both human and bovine mRNAs sequences of cholecystokinin (CCK) and surfactant protein 1 A (SFTP1-A), representing the enteric and respiratory tract codon usage, respectively, have shown natural selection leading to optimization or deoptimization of viral codon usage to the human enteric and respiratory tracts depending on the virus genes under consideration. A higher correlation was found for the nucleotide distance at the 3rd nucleotide position of codons and codon usage optimization to the human respiratory tract when BCoV and HCoV-OC43 were compared. An MCC tree based on relative synonymous codon usage (RSCU) data integrating data from both viruses and hosts into a same analysis indicated three putative host/virus contact dates ranging from 1.54E8 to 2.44E5 years ago, suggesting that an ancestor coronavirus might have followed human evolution.",
        "body": "Introduction\nHuman coronavirus OC43 (Nidovirales: Coronaviridae: Coronavirinae: Betacoronavirus: Betacoronavirus 1, HCoV-OC43) is an epitheliotropic respiratory virus widespread in human populations and involved in common cold (M&#228;kel&#228; et al., 1998), while Bovine coronavirus (BCoV), another host-type of Betacoronavirus 1, is commonly found infecting both the respiratory and enteric tracts of cattle and might lead to respiratory disease and diarrhea/dysentery (Dea et al., 1995; Saif, 2010). A suggested bovine-to-human spillover of BCoV resulting in HCoV-OC43 has been proposed around year 1890, based on the spike (S) gene sequences of BCoV and HCoV-OC43 (Vijgen et al., 2005b; Bidokhti et al., 2013).The Betacoronavirus 1 genome is a ca. 32 kb single-stranded positive-sense 5&#8217; capped RNA coding for subgenomic mRNAs (sgmRNAs) in the order ORF1(replicase)-HE (hemagglutinin-esterase)-S-(spike glycoprotein)-E (envelope protein)-M (membrane protein)-I (internal protein)-N (nucleocapsid protein). A 32kDa accessory protein (ns2) is found in both BCoV and HCoV-OC43 where the gene (ns2) is located before the HE gene (Masters, 2006; Labont&#233; et al., 1995). The replicase polyprotein is cleaved into 16 non-structural proteins (nsps) with multiple roles in sgmRNA synthesis and genome replication (Ziebuhr and Snijder, 2007).Betacoronaviruses have a history of spillover to humans leading to the emergence of pathogens, such as the Middle East Respiratory Syndrome Human Coronavirus (MERS-CoV) and the Severe Acute Respiratory Syndrome Human Coronavirus (HCoV-SARS) (Li et al., 2005; Gossner et al., 2016). Such a pathogen emergence is limited by ecological and genetic factors (Gandon et al., 2013), and codon usage, i.e., the deviation from the random use of different codons for the 2 to 6-fold degenerate codons (Hershberg and Petrov, 2009; Roth et al., 2012), is one genetic factor that might help to explain this process.Codon usage evolution has a measurable role on the adaptation of viruses to hosts (Chantawannakul and Cutler, 2008) due to natural selection based on translation efficiency and also drift according to the genomic mutation pressure (Nei and Kumar, 2000; Hershberg and Petrov, 2009). Nonetheless, codon usage studies meet limitations on plausible indicators and dating methods to estimate the coevolution patterns after a virus meets a new host species. If the dating of a spillover event based solely on virus nucleotide sequence data would agree with codon usage dating, based on both virus and host data, is hitherto unknown.The aim of this study was to analyze the BCoV/HCoV-OC43 spillover to humans based on codon usage data for codon selection regime, fitness and virus/host relationship dating estimates.\nMaterials and Methods\n\nSequences\nComplete genome sequences were retrieved from GenBank for BCoV (strain BCoV R-AH187, EF424620.1), detected in 2000 in the USA (Zhang et al., 2007), and HCoV-OC43 (strain 19572, AY903460.1), detected in 2004 in Belgium (Vijgen et al., 2005a). These two sequences were considered as representatives of the diversity of each virus, and the inclusion criteria were based on genome completeness and annotation.Further human coronaviruses complete genome sequences included HCoV-HKU1 (KF686341.1), HCoV-NL63 (DQ445911.1), HCoV-229E (JX503061.1), HCoV-SARS (AY291315), and two HCoV-MERS (KJ156949 from a strain detected in a human patient and KJ713299.1 detected in a dromedary camel).The eight coronavirus genomes were split into each coding region/mRNA for the analyses. Nsps 1-16 sequences were checked based on nsps 3 and 5 cleavage sites (Ziebuhr and Snijder, 2007; Wojdyla et al., 2010).As representatives of highly expressed, tissue-specific proteins for the respiratory and enteric tracts of H. sapiens sapiens and B. taurus taurus, complete mRNA sequences were retrieved from GenBank for the surfactant protein A1 SFTPA1 (NM_001077838.2 and NG_021189.1) and cholecystokinin CCK (NM_001046603.2 and NM_000729.4), respectively.\nCodon adaptation index (CAI) limits for human coronaviruses and \u0394CAI for HCoV-OC43 and BCoV\nCAI is an indicator of translational fitness of an mRNA regarding a reference translational system, ranging from 0 (no fitness) to 1 (highest fitness) (Lee et al., 2010). To determine the lower and upper limits for HCoVs in the respiratory and enteric tracts of humans, the eight HCoV sequences had their CAIs calculated for each coding region/mRNA using human SFTPA1 and CCK sequences as references in CAI Calculator 2 (Wu et al., 2005) based on the equation by Sharp and Li (1987).CAI differences (&#916;CAI) were calculated as HCoV-OC43 CAI - BCoV CAI (calculated as mentioned above) for each coding region/mRNA regarding human respiratory and enteric tracts in order to access the codon optimization (&#916;CAI>1) or deoptimization (&#916;CAI<1) for the bovine-to-human spill over.\nCodon usage selection regimes\nFor each HCoV-OC43 and BCoV coding region/mRNA, the observed effective number of codons (Nc) and the frequency of G or C at the 3rd codon positions in synonymous codons (%GC3s) (Wright, 1990) was calculated using ACUA 1.0 software (Vetrivel et al., 2007) and CAI Cal (Puigbo et al., 2008), and both indicators were plotted in the expected number of codons (ENC)/ expected %GC3 graph (Wright, 1990). Dots from observed values outside the expected values curve are an indication of natural selection, while those on the curve indicate drift/ mutation pressure.\nViruses/hosts codon usage co-evolution analysis\nFor each HCoV-OC43 and BCoV coding region/mRNA and human and bovine CCK and SFTPA1, the values of RSCU (relative synonymous codon usage) were estimated for the 59 nonstop degenerate codons using Mega 7 software (Kumar et al., 2016). Codons with RSCU 1 are preferred, while an RSCU=1 indicates a neutral codon (Su et al., 2009).Next, continuous RSCU values were assigned the binary values 0 (RSCU&#8804;1) and 1 (RSCU>1), and data from both hosts and both HCoV-OC43 and BCoV assembled into a single alignment were used to build an MCMC MCC tree with the simple model. This included estimated frequencies, burn in=10% states, uncorrelated exponential relaxed clock (which showed a lower standard deviation when compared to lognormal clock) and constant population size (due to the lack of consensus priors for an exponential growth coalescent analysis for H. sapiens sapiens, B. taurus taurus and coronaviruses) and was built using Beast v. 1.8.3 (Drummond and Rambaut, 2007; Drummond et al., 2012).Calibration times to estimate branch lengths were based on dates with 2004 (HCoV-OC43 strain 19572 detection date) as the reference year and were as follows: 200,000 years ago (y.a.) for H. sapiens sapiens (Weaver, 2012), 10,000 y.a. for B. taurus taurus based on the domestication dates for this species (reviewed by Ajmone-Marsan et al., 2010), 114 y.a. for HCoV-OC43 (Vijgen et al., 2005b) and 602 y.a. for BCoV based on the Betacoronavirus 1 split (Lau et al., 2015).The RSCU binary distance between human and bovine CCK and SFTPA1 was calculated as the total difference for each of these two datasets and used as a measure of codon usage distance for the enteric and respiratory tracts, respectively, for these two host species.\nResults\n\nCodon adaptation index (CAI) limits for human coronaviruses and \u0394CAI for HCoV-OC43 and BCoV\nCAI upper and lower limits for the seven human coronaviruses included in this study in human respiratory and enteric tracts were 0.244-0.611 (corresponding to HCoV-SARS nsp11 and nsp10, respectively) and 0.244-0.472 (corresponding to HCoV-SARS ORF7b and nsp11, respectively).CAI optimization (&#916;CAI>1) was found for nsp2-5, nsp8, nsp11, nsp15, ns2, HE, S, M, I and N and nsp2, nsp4-6, nsp11, nsp14, nsp16, ns2, M and N proteins on the enteric and respiratory tracts, respectively. Deoptimization (&#916;CAI<1) was found nsp1, nsp6, nsp9-10, nsp12-14, nsp16 and E and nsp1, nsp3, nsp8-10, nsp12-13, HE, S, E and I proteins for the enteric and respiratory tracts, respectively.A &#916;CAI=0 was found for nsp7 on both respiratory and enteric human tracts and for nsp15 on the respiratory tract. &#916;CAI values for each coding region/mRNA of HCoV-OC43 on the human enteric and respiratory tracts are represented in Figure 1.For both BCoV and HCoV-OC43 nsp7, the lowest CAI distance (-0.039) was found for both the human and respiratory and enteric tracts regarding the lower CAI limit calculated for all seven human coronaviruses, while the highest CAI distances for the lower human coronaviruses CAI was found for BCoV and HCoV-OC43 N for both the human and respiratory and enteric tracts (-0.282 and -0.302, respectively) and BCoV nsp15 (-0,282) for the human respiratory tract.Correlation analysis of &#916;CAI and nucleotide identities amongst the 23 BCoV and HCoV-OC43 homologous coding regions/ mRNAs based on 1st, 2nd and 3rd and on the 3rd nucleotide position only showed the highest r2 (correlation coefficient) value (0.27) for the 3rd nucleotide position regarding the human respiratory tract, while r2 values for &#916;CAI and 1st, 2nd and 3rd regarding the human enteric and respiratory tracts were both 0.05 and, regarding the 3rd positions only and the human enteric tract, 0.07.\nCodon usage selection regimes\nAll Nc x %GC3s plots were found either above or below the ENC x %GC3 expected curve for all HCoV-OC43 and BCoV coding regions/mRNAs and for human and bovine CCK and SFTPA1 (Figure 2), an indication that codon usage in these cases was ruled by natural selection.In Figure 2, the two closest dots to bovine and human SFTPA1 dots represent the internal I protein of BCoV (upper) and HCoV-OC43 (lower), while the two dots at the bottom of the graph refer to BCoV and HCoV-OC43 nsp11.\nViruses/hosts codon usage co-evolution analysis\nAll 95% HPDs (Highest Posterior Densities) are presented in years. In the MCC tree shown in Figure 3, the first split event (node A, 95% HPD 2.44E5-1.54E8) resulted in two major clusters, the largest one containing all HCoV-OC43 and BCoV coding regions/ mRNAs data except for I protein and a minor cluster containing both human and bovine CCK and SFTPA1 and HCoV-OC43 and BCoV I protein. For this minor cluster containing both hosts and coronaviruses codon usage statuses, a second split was found (node B, 95% HPD 2.07E5-1.55E8), resulting in a cluster with SFTPA1 only and another cluster with CCK and HCoV-OC43 and BCoV I, and for this last one a third split event (node C, 95%HPD 2.04E5-3.54E6) led to CCK and HCoV-OC43/ BCoV I exclusive clusters. The RSCU distance of human and bovine CCK and SFTPA1 mRNAs were 0.136 and 0.221, respectively.\nDiscussion\nCodon usage optimization and deoptimization based on &#916;CAI values for ORF1 nsps, observed for both the human enteric and respiratory tracts, might be a consequence of a balance between synthesis efficiency and fine-tuning codon usage adaptation to the new host codon usage after a bovine-to-human coronavirus spillover. Though these proteins are coded in the same ORF, the distinct roles they play during RNA replication and sgmRNAs transcription might demand not only different synthesis efficiencies but also, in some cases, compensatory or concerted codon usage evolution, as in the case of the proteases PLpro and 3C-like in nsps 3 and 5, respectively, which can process ORF1 polyprotein and release from it all subunits (Ziebuhr and Snijder, 2007).The analysis of coronaviruses non-structural proteins of the replicase class allows deep phylogenies to be estimated (Snijder et al., 2003) and results, thus, in a more representative range of evolutionary data to assess ancient virus/hosts relationship when associated to structural proteins data as herein.Regarding the structural proteins, the different degrees of optimization and deoptimization found based on human enteric and respiratory tracts, besides the translation efficiency, might also be due to both an immune escape efficiency, as in the case of HE and S, as a lower CAI might lead to lower protein synthesis and consequently lower exposure to the immune system (Bahir et al., 2009) and a fine-tuning codon adaptation leading to a more efficient receptor binding to the human tissues due to the primary S and accessory HE roles on this function (Popova and Zhang, 2002).As seen in the MCC tree (Figure 3), the first split (Node A) of hosts (H. sapiens sapiens/B. taurus taurus) and HCoV-OC43/BCoV showed a 95% HPD from 1.54E8 to 2.44E5 years ago, ranging from the Kimmeridgian age of the Late Jurassic to the Middle Pleistocene.Taking node A as a first split and thus as a consequence of a first contact between the codon usage of an ancestor coronavirus with the codon usage of an ancestor host, the lower limit (1.54E8 ya) brings the ancestor coronavirus codon usage status to an age compatible with the proposed ancient origin of coronaviruses as being 2.93E8 y.a. (Wertheim et al., 2013), while the upper limit (2.44E5) is related to a time compatible with early humans, in agreement with the suggested interspecies transmission of a betacoronavirus prior to the HCoV-OC43/BCoV split (Vijgen et al., 2006).Such a large time span might be due to the lack of data from hosts and coronaviruses in between these upper and lower limits, but it places an ancestor betacoronavirus as coevolving with a diversity of dinosaurs (Langer et al., 2010) in the Late Jurassic and reaching early humans with until unknown intermediate hosts during this large time span. It is worthy of note that this time span overlaps with the one found for node B (95% HPD 2.07E5-1.55E8), meaning that the first ancestor host/ ancestor betacoronavirus contact might have been stable for circa 150 million years before reaching early humans.As for node C, the 95% HPD 2.04E5-3.54E6 embraces human evolution from Australopithecus spp to H. sapiens sapiens (McHenry, 1994), what could finally represent the first sign of BCoV spillover from an ancestor ruminant host to the human lineage after a first contact with the respiratory tract (represented by SFTPA1 in Figure 3).The discrepancy of HPDs values, when compared to previous dates on the HCoV-OC43/BCoV split and the emergence of all coronaviruses, might be a consequence of both the use of full genomes data and the selection unit used in this survey, i.e., codon usage, instead of subgenomic data based on nucleotide evolution as proposed by others (Vijgen et al., 2005b; Vijaykrishna et al., 2007; Munir and Cortey, 2015).All coding regions/mRNAs from an ancestor coronavirus (except for HCoV-OC43 nsp7 in both respiratory and enteric human tracts, and for nsp15 on the human respiratory tract, &#916;CAIs=0) experienced optimization or deoptimization, as suggested in Figure 1, probably after Node A (Figure 3). This process of codon usage evolution resulted in CAIs approaching the CAI limits for human coronaviruses as calculated herein (0.22-0.611 for the respiratory and 0.244-0.472 for the enteric tract) during codon usage evolution by natural selection, as shown in the Nc x %GC3s analysis (Figure 2). The association of data on fluctuations in codon usage optimization with analysis of the selection regime and a temporal analysis, both based on codon usage, as used in this investigation, might be of value for a deeper understanding of tempo and modes of viruses and hosts coevolution.Having crossed the longer codon usage distance from the bovine to human respiratory tract (0.221) when compared to the enteric tract (0.136), HCoV-OC43 became a highly respiratory-specialized virus with high fitness to this new replication site and predating the proposed event around the year 1890 (Vijgen et al., 2005b; Bidokhti et al., 2013).Though nsp14 is a coronavirus 3&#8217;-5&#8217; exonuclease (Denison et al., 2011), whose proofreading activity lowers the mutation rate of these viruses when compared to other RNA viruses, the mutant spectrum phenomenon is well documented in HCoV-OC43 and BCoV (Vabret et al., 2006; Borucki et al., 2013), and as a result, a plethora of synonymous mutations that power codon usage diversity is available for the optimization or deoptimization of codon usage in different genes via natural selection or drift as well.An in important limitation to these arguments is that codon usage studies only allow speculations after virus attachment and entry, two processes intimately related to membrane receptor specificities that cannot be assessed in organisms for which at least gene data are not available. Also, the full set of interspecies jumps for the HCoV-OC43 ancestors has not been assessed here, as the focus was the proposed recent bovine-to-human spillover (Vijgen et al., 2005b), and this might have limited the detection of further nodes of codon usage status split with coronaviruses and different hosts.As a conclusion, via codon usage through natural selection resulting in immune escape balanced with protein synthesis efficiency, an ancestor coronavirus might have followed human evolution with no codon usage barrier fitness deep in the human lineage.",
        "pmid": "30004106",
        "title": "Could human coronavirus OC43 have co-evolved with early humans?",
        "journal_title": "Genetics and Molecular Biology",
        "authors": "Paulo Eduardo Brand\u00e3o",
        "doi": "10.1590/1678-4685-GMB-2017-0192"
    },
    {
        "keywords": "",
        "abstract": "Cell surface aminopeptidase N (APN) is a membrane-bound ectoenzyme that hydrolyzes proteins and peptides and regulates numerous cell functions. APN participates in tumor cell expansion and motility, and is a target for cancer therapies. Small drugs that bind to the APN active site inhibit catalysis and suppress tumor growth. APN is also a major cell entry receptor for coronavirus, which binds to a region distant from the active site. Three crystal structures that we determined of human and pig APN ectodomains defined the dynamic conformation of the protein. These structures offered snapshots of closed, intermediate and open APN, which represent distinct functional states. Coronavirus envelope proteins specifically recognized the open APN form, prevented ectodomain progression to the closed form and substrate hydrolysis. In addition, drugs that bind the active site inhibited both coronavirus binding to cell surface APN and infection; the drugs probably hindered APN transition to the virus-specific open form. We conclude that allosteric inhibition of APN functions occurs by ligand suppression of ectodomain motions necessary for catalysis and virus cell entry, as validated by locking APN with disulfides. Blocking APN dynamics can thus be a valuable approach to development of drugs that target this ectoenzyme.",
        "body": "Results\n\nThe APN ectodomain structure\nAs the APN protein is a type II membrane protein, ectodomain expression required deletion of the N-terminal cytoplasmic and transmembrane domains, and introduction of a secretion signal sequence, as well as a hemagglutinin (HA) tag to allow protein detection and purification (Supplementary Fig. S1a). As the N-terminal and middle portions of the hAPN and pAPN ectodomains are heavily glycosylated, we produced them in CHO cells (see Methods). The purified proteins generated distinct crystal forms under different crystallization conditions (Table 1 and Methods).In the past we reported a pAPN ectodomain crystal structure in complex with a CoV spike (S) fragment (PDB code 4F5C)16, here we show three new structures for APN (Table 1). In the four structures, the N-terminal HA tag and ~30 ectodomain residues were very disordered, indicating a large degree of flexibility in the membrane proximal polypeptide. The ectodomains have a hook-like conformation formed by domain I to IV and contained a zinc ion at the active site in domain II (Fig. 1). The exposed convex side of domain IV mediates similar protein dimerization in the distinct crystals. Approximately 950&#8201;&#197;2 of each monomer is buried at the dimer interface (Table 2), indicative of a stable protein-protein interaction. Domain IV is the largest APN domain and the most divergent in the M1 aminopeptidase family. In APN, domain IV has seven helix-turn-helix HEAT repeats and a single ARM repeat formed by three alpha helices (&#945;25-&#945;27). The ARM repeat is the most variable domain IV region in the hAPN and pAPN structures, and can contact the peptide substrate bound to the active site (see below).\nDynamic conformation of the APN ectodomain\nAlthough the dimeric assembly of human and pig APN ectodomains was preserved in various crystals, the conformation of each monomer differed among crystal forms, such that the distance between the N-terminal region of the ectodomains that formed the dimer varied from 95 to 131&#8201;&#197; in the structures (Table 2 and Supplementary Fig. S1b). Each crystal captured a single APN conformation, with all the monomers in the same form. These structures identified three distinct APN conformations, based on active site accessibility, which we termed closed, intermediate and open forms (Fig. 1a). As reported for other M1 family members2324, the observed APN structural diversity indicated ectodomain dynamics in solution and on the cell surface.The active site accessibility at domain II differed among crystal forms because of interdomain adjustments in the APN. The contacts between domain IV and other domains in the monomers varied among the structures, whereas the domain IV-IV buried surfaces in each monomer at the dimerization interfaces were preserved (Table 2). Domain IV contacts with domain I or III changed markedly less (~100&#8211;200&#8201;&#197;2 buried surface) than with domain II (1000&#8201;&#197;2); domain II-IV interaction thus mainly stabilized the closed APN conformation. There were no notable differences in the other interdomain contacts in the distinct APN forms (Table 2). Domains I-II are distant from domain IV in the open conformation of the APN monomer (Fig. 1a,b), where the zinc ion at the catalytic site is more accessible to the solvent (Fig. 1c). The domain I to III module swings 15&#176; toward domain IV, closing the active site (Fig. 1b,c). The hAPN structure adopts an intermediate conformation in the crystals (Fig. 1a); the distance between the N terminus of each monomer in the dimer is 116&#8201;&#197;, and the angle difference of domain II with the closed conformation is ~6&#176; (Table 2). Other crystal structures of human and pig APN showed closed and intermediate conformations1921, respectively, similar to the forms reported here.APN dimerization mediated by domain IV-IV interactions is preserved among open, intermediate and closed ectodomains (Fig. 1a and Table 2). On the cell surface, the domain I to III module must swing over domain IV, which is fixed by dimerization (Fig. 1b, Supplementary Video S1). The module movement must be facilitated by the flexible ~30-residue polypeptide that links domain I to the transmembrane domain (Supplementary Fig. S1a), although polypeptide length probably limits the interdomain movement shown here with APN (15&#176;), which is less pronounced than that reported for ERAP-1 (22&#176;, determined as in Table 2). The type of interdomain movement also differs between ERAP-1 and the APN. The domain III-IV module moves together relative to domain I-II in ERAP-1, whereas domain I to III swings over domain IV in the APN. In addition, the ERAP-1 hinge region is at the domain III N terminus, whereas that of APN is in the domain IV N terminus. Domains I, II and III can pivot at the beginning of the first (&#945;13) or third (&#945;15) domain IV helix, which are perpendicular to the swing angle (Fig. 1b). These differences in APN motion compared to other aminopeptidases are probably related to dimer formation, which is not observed in other M1 family members.\nAPN dynamics in catalysis and CoV binding\nThe domain II buried surface increases due to its interaction with domain IV when the conformation changes from open to closed (Table 2), thus reducing accessibility of the active site cavity (Fig. 1c). M1 aminopeptidase dynamics is thought necessary for catalysis, and the closed conformation is considered the active form2024, although it remains unclear how this structural switch connects to peptide processing. Conformation of the domain II residues that participate in zinc coordination and peptide hydrolysis was conserved among the different APN structures shown here (Supplementary Fig. S2).Crystal structures are reported for mammalian APN in complex with substrates, in both intermediate (pAPN) and closed conformations (hAPN)1921. To determine how domain movement contributes to peptide processing, we modeled a pAPN-bound poly-Ala peptide in the active site of closed, intermediate and open APN (Fig. 2a). In the closed pAPN, the side chain of a phenylalanine (Phe893) at domain IV was placed at about 4.5&#8201;&#197; from the hydrolyzable peptide bond, whose carbonyl group is coordinated to the zinc ion. The phenylalanine was located in the loop that connects &#945;26 and &#945;27 in the single domain IV ARM repeat of human and pig APN (Fig. 2a); it penetrated the active site groove in the closed conformation and locked the peptide, ready for hydrolysis. Domain IV residues that precede Phe893 in the &#945;26-&#945;27 loop contacted domain II in the closed pAPN. A similar loop conformation is seen in a closed hAPN structure (PDB code 4FYS)19.The phenylalanine side chain in closed APN probably hinders peptide release or translocation for further processing after P1 hydrolysis. It is likely that binding of the P1&#8242; residue to the zinc ion required removal of the phenylalanine plug by domain II displacement away from domain IV. The phenylalanine adopted a distinct conformation in the intermediate and open APN conformations (Fig. 2a). Domain II movement was accompanied by a conformational change of the &#945;26-&#945;27 loop in domain IV (Supplementary Video S1), which became more solvent-exposed; the phenylalanine side chain faced into domain IV in the intermediate and open conformations and the peptide plug was removed from the active site. These changes would facilitate release of the N-terminal residue after hydrolysis. The small interdomain movement of the intermediate APN structure would be sufficient for peptide processing (Fig. 2a).We previously described in detail the CoV spike RBD-pAPN binding interface16. The porcine CoV spike RBD binds to a pAPN region that is distant from the catalytic site (Supplementary Fig. S1b). A critical CoV receptor-binding motif, which bears an exposed tryptophan, penetrates a narrow cavity formed by domain II and IV (Fig. 2b)16. The tryptophan aromatic side chain stacks onto pAPN domain IV residues His786-Pro787, and is trapped by domain IV residues Asn783-Pro787 on one side and domain II residues Gln367-Ser368 on the other (Fig. 2b). The main chain of domain II residues is in close contact (3.9&#8201;&#197;) with the tryptophan side chain, and its imino nitrogen forms a hydrogen bond with the domain IV Asn783 main chain carbonyl. Domain IV-based superposition of the open pAPN with bound RBD and that of closed pAPN showed a shift in the domain II main chain region that contacts the RBD; this region collides (<3.0&#8201;&#197;) with the CoV tryptophan (Fig. 2b). Closing of the ectodomain would hinder penetration of the viral tryptophan between the pAPN domain II and IV.\nAllosteric inhibition of APN catalysis by CoV binding to the ectodomain\nOur data suggested that CoV binding to APN would lock the protein in its open conformation (Fig. 2b), preventing the ectodomain movement probably necessary for peptide hydrolysis (Fig. 2a). We analyzed the catalytic activity of soluble human and pig APN ectodomains in the presence of porcine CoV S fragments bearing the RBD (Fig. 3). The soluble S proteins specifically inhibited pAPN-mediated catalysis, measured as the hydrolysis of the L-pNA substrate, and had no effect on hAPN activity. The TGEV (transmissible gastroenteritis coronavirus) spike does not bind hAPN because it lacks the N-linked glycan recognized by porcine CoV in pAPN1516.The isolated RBD was sufficient to inhibit pAPN catalysis (Fig. 3a); inhibition was dependent on RBD concentration. A high RBD:pAPN ratio was needed to achieve maximum inhibition (50&#8211;60%; Fig. 3b), which decreased slowly after 30&#8201;min (Fig. 3c), probably due to substrate hydrolysis during RBD dissociation and binding to APN over time. The RBD can thus prevent catalysis in only a fraction of pAPN molecules, probably those in the open form. The conformation of catalytically active pAPN would not allow RBD binding. The results indicate that the open APN is catalytically inactive, and that blocking APN dynamics impairs catalysis.\nDrugs that bind the catalytic site inhibit CoV binding to APN\nNon-hydrolyzable drugs that bind the APN catalytic site inhibit catalysis and prevent angiogenesis and tumor growth7102627. They appear to restrict ectodomain conformational changes, as shown by reduction in the number of some APN conformation-specific mAb epitopes25. On the cell surface, active site epitopes recognized by the MY7 mAb decrease in the presence of actinonin, which indicates APN closure. Crystal structures of M1 aminopeptidases in complex with these drugs show preferential adoption of a closed state19202428. Drug binding would thus not only compete with substrates for active site binding, but might also restrict the aminopeptidase dynamics needed for peptide processing.The structure of the pAPN-RBD complex indicates that porcine CoV would be specific for the open conformation (Fig. 2b). Restriction of APN ectodomain opening by active site-binding drugs would thus have an allosteric effect on CoV binding. To test this hypothesis, we studied TGEV RBD binding to cell surface pAPN in the presence of drugs that bind to the active site (Fig. 4). In flow cytometry, we determined the binding of an RBD-Fc fusion protein to cells that expressed pAPN or an active site mutant (pAPN-HH/AA), alone or with various drugs (Fig. 4a,b). We analyzed the effect of the natural APN inhibitors actinonin and bestatin29; both reduced RBD-Fc binding to cell surface pAPN (Fig. 4a, left). We then evaluated four synthetic amino-benzosuberone (ABS) derivatives that bind with high affinity and selectivity to APN (Supplementary Fig. S3a)30; all four ABS molecules prevented RBD binding to pAPN and its effectiveness increased with APN-binding affinity (Fig. 4b, left). The bulkier ABS2 and ABS4 compounds, which contain a phenyl group and bind with the highest affinity to APN, more efficiently blocked binding of the TGEV RBD to pAPN on the cell surface.The inhibitory molecules bind to the APN active site1921, which is distant from the APN region recognized by CoV (Supplementary Fig. S1b). To further determine whether the inhibitory effect was linked to drug binding to the pAPN active site, we analyzed RBD binding to the pAPN-HH/AA mutant, which lacks the two histidines (H383 and H387) that coordinate the zinc ion (Supplementary Fig. S2). Staining for the RBD-Fc protein was similar in cells expressing the mutant, alone or with the drugs (Fig. 4a right and b right), which showed that compound binding to the pAPN active site was necessary to prevent RBD binding to a distant site. In addition, inhibition of RBD binding to pAPN was drug concentration-dependent (Fig. 4c), and the amount of compound needed to reach 50% inhibition (IC50) decreased with compound affinity for APN (~30&#8201;&#956;M for bestatin (Ki ~4&#8201;&#956;M), ~1&#8201;&#956;M for actinonin (Ki ~1&#8201;&#956;M), ~0.1&#8201;&#956;M for ABS4 (Ki ~0.06&#8201;nM)). These results show that drugs that bind to the active site cause allosteric inhibition of TGEV RBD binding to pAPN, probably by restricting APN ectodomain dynamics and its transition to the CoV-specific open form.\nAnti-tumor APN-binding drugs inhibit CoV cell infection\nAPN catalytic activity is not necessary for CoV cell entry and infection31. We nonetheless found that active site-binding molecules hindered CoV S protein binding and might inhibit virus infection. Studies with low affinity binding drugs such as bestatin show no reduction in TGEV infection31. Virus particles have high receptor-binding avidity, and these drugs might not have sufficient affinity to maintain most APN molecules closed. The selective compounds ABS1-4 have high affinity for APN and, at 1&#8211;10&#8201;&#956;M concentrations, inhibit capillary tube formation in cell cultures, with no cytotoxicity27. In our cultures, we observed no toxicity at ABS concentrations <100&#8201;&#956;M (not shown). We therefore analyzed the TGEV-mediated cytopathic effect for each of the four ABS molecules and actinonin at a 50&#8201;&#956;M concentration and monitored inhibition of virus infection with ABS4 (2&#8201;log) and ABS2 (1&#8201;log) (Fig. 5a); at the same concentration, the lower-affinity ABS1 and ABS3 compounds or actinonin did not inhibit. ABS4 has a bromo substituent that is predicted to interact with the phenylalanine that plugs the substrate in the closed conformation32; this interaction likely helped maintain the closed ectodomain and efficiently prevented virus binding.TGEV is a representative, extensively studied animal CoV that use pAPN for cell entry414. To further determine whether ABS4 inhibition of virus infection was linked to cell entry, we analyzed TGEV replication at 6&#8201;h post-infection and found that virus entry decreased with the ABS4 concentration (Fig. 5b). ABS4 addition after virus absorption at 4&#8201;&#176;C did not inhibit virus growth (not shown), which indicates that it prevented virus binding to cells. In addition, we studied the effect of ABS4 concentration on TGEV cell infection, and we observed that the TGEV cytopathic effect was reduced and cell survival increased at higher ABS4 concentrations (Fig. 5c). ABS compounds are selective for APN molecules and designed to inhibit APN catalytic activity and tumor growth; here we show that they also prevent CoV cell infections.\nRestricting APN dynamics inhibits function\nTo further analyze the importance of APN dynamics, we engineered disulfide bonds to bridge domains II and IV and restrict ectodomain motion. We replaced the pAPN domain II Ser464 and domain IV Ser892 and/or Ser929 with cysteine to lock the ectodomain in the closed form with interdomain disulfide bridges. The Ser464 main chain C&#945; is ~5 and ~6&#8201;&#197;, respectively, from Ser892 and Ser929 in the closed form, but Ser464 moves ~12&#8201;&#197; away in the open form (Supplementary Fig. S4). Disulfide bond formation between pAPN Cys464 and Cys892 or Cys929 should thus prevent ectodomain motion. We expressed the pAPN-cysteine mutants (C2-C4) on the 293T cell surface and compared their catalytic and CoV binding activity with that of the wild type pAPN (Fig. 6).The pAPN cysteine mutants showed reduced catalytic activity (Fig. 6a) and TGEV RBD binding (Fig. 6b) relative to the wild type protein in 293T transfectants that express similar protein amounts. The higher activity of the pAPN C2 than the C3 or C4 mutants suggested that the Cys464-Cys892 disulfide bond was more labile than the Cys464-Cys929 bond, probably because Cys892 is in a flexible loop (Supplementary Fig. S4). Treatment with a reducing agent restored catalysis and RBD binding in the cysteine mutants and did not affect wild type pAPN binding activity (Fig. 6). Reducing the disulfide bonds fully restored RBD binding, but catalysis was partially recovered in the pAPN C3 and C4 mutants. Substrate hydrolysis is proposed to close the ectodomain (see above), which would facilitate rebuilding the disulfides. Locking the closed form and the phenylalanine in the domain IV ARM repeat inside the active site probably impeded substrate processing (Fig. 2a). The pAPN cysteine mutants bound markedly less RBD than the wild type protein, which confirmed that CoV S protein binding to the closed pAPN was sterically hindered (Fig. 2b), and that CoV recognized the open form. Overall, these results validate the functional relevance of the APN ectodomain conformations and its motion.\nDiscussion\nStructural dynamics is an intrinsic property of aminopeptidases. The APN crystal structures reported here indicate the dynamic conformation of its ectodomain, and functional studies show its relevance in catalysis and virus infection. Distinct ectodomain regions mediate these functions, but agents that bind to one region prevent activities linked to the other. These allosteric effects with ligands are probably caused by restrictions in APN conformational dynamics, as confirmed with disulfide bond mutants. They demonstrated that preventing ectodomain motion and locking APN forms inhibits its functions.APN ectodomain movement is less pronounced and differs from that reported for other M1 aminopeptidases. These differences could be due to the APN dimeric conformation and its linkage to the cell surface. Dimerization only engages the domain IV region, and we found that the dimer is conserved in all APN structures, closed, intermediate and open. APN domain IV thus does not move as described for ERAP-1 or F32324, proteins that do not form dimers. The fixed conformation of the APN dimer determines that the domain I to III module swings over domain IV (Supplementary Video S1), with the hinge at the domain IV N-terminal region. The length of this movement is less marked in APN (15&#176;) than in ERAP-1 (22&#176;), although the two proteins have very similar closed conformations. Displacement of the APN domains I, II and III must be limited by the length of the flexible polypeptide that links domain I to the transmembrane region, whose movement is restricted by membrane fluidity. The extent of APN movement nonetheless appears to be sufficient for release of the hydrolyzed peptide N-terminal residue, which is not plugged by domain IV in the open or in the intermediate APN conformations (Fig. 2a). It is not clear how each monomer in the dimer moves, whether their movement is random or synchronized in the same or inverse directions. Experiments with hAPN antibodies25 and those shown here with the TGEV RBD (Fig. 3) suggest that ~50% of the molecules adopt different forms; these data imply that each APN monomer maintains a distinct conformation (Supplementary Video S1).The crystal structures reported here provide snapshots of APN dynamic conformation, and also guided experiments that demonstrate its role in virus entry into cells and catalysis. The switch between a proteolytic active (closed) and an inactive (open) conformation has been proposed for several M1 aminopeptidases20232428. This dynamics is thought to be important for peptide hydrolysis and release from the aminopeptidase active site. The region that joins &#945;26 and &#945;27 in the domain IV ARM repeat penetrates the active site groove in closed pig and human APN structures reported here and elsewhere (Fig. 2a)19, and a conserved phenylalanine in this region locks the substrate coordinated to the zinc ion, permitting hydrolysis. Further peptide processing likely requires removal of the phenylalanine lock by opening the APN ectodomain, which facilitates N-terminal residue release and peptide translocation, both sterically hindered in the closed conformation (Fig. 2a). The inherent flexibility in the domain IV ARM repeat that we demonstrate here is linked to interdomain arrangements might also enable substrate processing, and indicate how ectodomain movements participate in peptide hydrolysis. Local changes in a conserved tyrosine (Tyr472 in pAPN) at the active site of M1 aminopeptidases are also suggested to be important202428. Among APN forms, the absence of conformational switches in active site residues at domain II (Supplementary Fig. S2) nonetheless indicates that tyrosine movement is not linked to interdomain motion.Disulfide bonds that lock the APN closed conformation or drugs that prevent opening of the ectodomain inhibited CoV protein binding and cell infection, whereas porcine CoV S proteins probably hinder APN transition to the closed form and peptide hydrolysis. Our results verify the critical role of APN dynamics in CoV infection and catalysis, and demonstrate that the open APN structure is inactive in peptide hydrolysis. Anti-APN antibodies that inhibit APN activity and reduce tumor growth2533 likely block ectodomain movements. The allosteric inhibition of APN functions shown here using viral proteins and drugs is likely to be due to suppression of APN transient conformational states, as shown for other enzymes34. Blocking APN movement prevents its functions, and suggests a new approach for the development of drugs that target this protein. Small molecules or conformation-specific antibody inhibitors of ectodomain motions can bind to the active site or interact with distant sites, as shown here with CoV spike fragments. High affinity drugs designed to inhibit catalysis and tumor growth prevent CoV infections, which indicates that targeting APN ectodomain dynamics can be a valuable approach to block APN functions related to cancer progression and virus infections.\nMethods\n\nRecombinant proteins\nThe engineered hAPN and pAPN ectodomains contain extracellular residues 36 to 967 and 36 to 963, respectively. They have a hemagglutinin (HA; YPYDVPDYA) peptide at the N terminus. Soluble ectodomains were produced in stably transfected CHO-Lec 3.2.8.1 (CHO-Lec) cells with selective culture media35. A pAPN protein with Met residues replaced by seleno-Met (Se-Met pAPN) was produced using methionine- and glutamine-free DMEM (Invitrogen) supplemented with 10% dialyzed fetal calf serum (FCS) and L-seleno-methionine (both from Sigma). APN proteins secreted to culture supernatants were purified by affinity chromatography with anti-HA 12AC5 mAb (Roche), followed by size exclusion chromatography in HEPES-saline buffer (20&#8201;mM HEPES, 150&#8201;mM NaCl) pH 7.5.Preparation of most soluble CoV S proteins used here has been described1635. S1H and S3H proteins were derived from the HOL87 porcine CoV S glycoprotein and bear the pAPN-binding domain. Soluble TGEV RBD was derived from the S glycoprotein of the TGEV SC11 strain (GenBank acc. n&#176; AJ271965). It contains S residues 505 to 657, an N-terminal HA peptide, and a FLAG sequence (monovalent RBD variant) or human IgG1 Fc (bivalent RBD-Fc variant) at the C-terminal end. CoV S proteins were produced in CHO-Lec or 293&#8201;T cells and purified as described35.\nAnalysis of APN catalysis\nAPN catalytic activity was determined using leucine p-nitroanilide (L-pNA) (Sigma) in a standard spectrophotometric assay in 96-well plates with soluble proteins or transfected cells.To study CoV protein inhibition of APN catalysis, soluble APN ectodomains (5&#8201;&#956;g/ml; ~40&#8201;nM) were added to duplicate wells, alone or with soluble CoV S protein variants, followed by the L-pNA substrate (1&#8201;mM) in 100&#8201;&#956;l final volume (4&#8201;&#176;C). Plates were incubated at room temperature and OD at 405&#8201;nm was measured at different times. Background OD of wells without APN was subtracted to determine specific catalytic activity. Similar procedure was used with 293T cells (3&#8201;&#215;&#8201;104) expressing pAPN 36&#8201;hr after transfection. OD of well with mock-transfected cells were taken as background. Cell samples expressing various amounts of pAPN at the membrane were used to normalize the activity of the pAPN cysteine mutants. Relative activity of the mutant to wild type was determined as the ratio of the pAPN mutant to the wild type OD from samples with the same protein expression, as monitored by flow cytometry (see below).\nCoV protein binding to APN\nStably transfected BHK21-pAPN, CHO-pAPN and CHO-pAPN mutant cells or transiently transfected 293T cells were used. The pAPN contained the HA peptide at the C terminus to monitor cell surface expression in CHO and 293T cells. In the pAPN-HH/AA mutant, the two active site histidines were replaced with alanines, whereas in the pAPN cysteine mutants, the domain II Ser464 and the domain IV Ser892 and/or Ser929 were substituted by cysteine. We analyzed the effect on RBD binding to pAPN of two natural inhibitors of APN enzyme activity, bestatin and actinonin (Sigma)29, as well as four synthetic ABS compounds30. Bestatin and actinonin were dissolved at 25&#8201;mM in PBS, whereas ABS compounds were used at 20&#8201;mM in DMSO.In wild type or mutant pAPN-expressing cells, we used flow cytometry to monitor TGEV RBD binding to cell surface pAPN, essentially as reported1635. Cells were washed three times with cold PBS and resuspended (106 cells/ml) in PBS supplemented with 0.3% heat-inactivated FCS and 0.125% bovine serum albumin (BSA; binding buffer); 200&#8201;&#956;l of cell suspension were added to 96-well plates (Nunc), cells were sedimented and resuspended in 20&#8201;&#956;l of 1&#8211;2&#8201;&#956;g/ml RBD-Fc solution alone or with inhibitors at indicated concentrations (30&#8201;min, 4&#8201;&#176;C). An unrelated Fc fusion protein was used as control. Cells were washed and incubated with anti-human IgG fluorescein isothiocyanate (FITC)-labeled secondary antibody (30&#8201;min, 4&#8201;&#176;C). The mean fluorescent intensity was determined in a Beckman Coulter EPICS XL1; background cell staining with the Fc protein was subtracted to determine the specific RBD-Fc binding to cell surface pAPN. In parallel, the amount of cell surface pAPN expression was determined by flow cytometry with the anti-HA 12AC5 mAb (Roche) and an anti-mouse FITC-labeled secondary antibody (Invitrogen). Analysis of the pAPN cysteine mutants binding activity was normalized by the cell surface protein amounts as explained above for the catalytic activity.\nInhibition of CoV cell entry and infection\nTGEV cell entry was quantified by determination of viral RNA by qRT-PCR (quantitative reverse transcription polymerase chain reaction). Stable transfected BHK21-pAPN or BHK21 cells (5&#8201;&#215;&#8201;104 cells/well) in DMEM (Dulbecco&#8217;s modified Eagle&#8217;s medium) with 10% FCS were plated in 24-well plates (18&#8201;h). Plates were transferred to 4&#8201;&#176;C, medium was removed and 200&#8201;&#956;l binding buffer alone or with APN-binding drugs or RDB protein were added to wells; after 10&#8201;min, the solution was replaced with 200&#8201;&#956;l virus inoculum at a multiplicity of infection (m.o.i) of 1, alone or with inhibitors in binding buffer. After virus adsorption at 4&#8201;&#176;C, cells were washed three times with binding buffer, and incubated in DMEM with 5% FCS (6&#8201;h, 37&#8201;&#176;C, 5% CO2). Cells were detached and lysed with 100&#8201;&#956;l Tri Reagent (Sigma) for RNA extraction, and cDNA was generated from 1&#8201;&#956;g RNA using the High Capacity cDNA Reverse Transcription Kit (Applied Biosystems). Real-time PCR reactions (10&#8201;&#956;l) were performed in triplicate using 5&#8201;&#956;l cDNA sample, 2&#8201;&#956;l of 5x HOT FIREPol EvaGreen qPCR Mix Plus (ROX) (Solis Biodyne) and 0.3&#8201;&#956;l of specific primers for mouse &#946;-actin or for the TGEV S gene, in a 7500 Real Time PCR system (Applied Biosystems) using a standard protocol. Data were analyzed with 7500 Software using the Comparative Ct Method (&#916;&#916;Ct). TGEV S expression relative to &#946;-actin was determined and the ratio of values alone and with inhibitor used as relative cell entry.Infection or cytopathic effect of TGEV was inhibited in porcine ST cells. One day after seeding (2.5&#8201;&#215;&#8201;104 cells/well) in 96-well plates, cells were transferred to 4&#8201;&#176;C and pre-incubated with 50&#8201;&#956;l binding buffer alone or with inhibitors, in duplicate. Solutions were replaced with 50&#8201;&#956;l of serial 10-fold dilutions of virus inoculum with inhibitors or with DMSO (&#8804;0.25%) as control. After incubation (1&#8201;h, 4&#8201;&#176;C), cells were washed three times with DMEM with 5% FCS and incubated alone or with inhibitors for two days at 37&#8201;&#176;C. To determine cell survival after infection, medium was removed, cells were formalin-fixed, stained with crystal violet and viability determined by optical density (OD) at 590&#8201;nm. Ratios from wells with and without virus were determined to calculate cell survival (see Supplementary Fig. S3b).\nCrystallization and diffraction data collection\nThe endoglycosidase H-treated (16&#8201;h, 30&#8201;&#176;C) pAPN (pAPNeh) ectodomain was crystallized by the sitting drop technique with a crystallization solution of 5% polyethylene glycol (PEG)-1000 and 10% PEG-8000 (pH ~6) and a 15&#8201;mg/ml protein sample. Alternatively, native glycosylated pAPN ectodomain crystals were prepared with a crystallization solution of 20% PEG-3350 and 100&#8201;mM sodium acetate pH 5.6. The hAPN ectodomain (15&#8201;mg/ml) was crystallized with a solution of 20% PEG-6000, 50&#8201;mM imidazole-HCl pH 8.0. Crystals were frozen in crystallization solutions containing 20% ethylene glycol for diffraction data collection at the European Synchrotron Radiation Facility (ESRF; ID14 and ID23) and Swiss Light Source (SLS; PXII) beamlines. Diffraction data were processed with XDS36 and scaled with SCALA programs37. For statistical data, see Table 1.\nStructure determination\nThe structure of Se-Met pAPNeh protein was solved by a combination of molecular replacement (MR) and single-wavelength anomalous dispersion (SAD) methods. The crystals contained two molecules in the asymmetric unit (Table 1). A partial structure was obtained by MR using the PHASER program38 and domains I to III of the tricorn-interacting factor F3 (PDB code 1Z1W), which share ~30% residue identity with pAPN. The PHASER LLG value for the best MR solution was 161, whereas RFZ values were 6.5 and 5.0, and TFZ values of 3.0 and 9.9. We then used the MRSAD protocol in the Auto-Rickshaw server39 to determine the complete pAPNeh structure, starting from the partial MR structure and using Se-Met pAPNeh crystal diffraction data collected at the selenium peak wavelength. The final structure included the two pAPN molecules of the asymmetric unit, which were adjusted manually and refined with phenix.refine40 using data extending to 2.5&#8201;&#197; resolution (for statistics, see Table 1). The pAPNeh structure comprises residues 60 to 963 and the zinc atoms coordinated in the enzyme active site.The other APN ectodomain structures (Table 1) were determined by the MR method using the pAPNeh structure as search model. Two ensembles including domain I, II and III or isolated domain IV were used for MR structure determination with PHASER. Structures were refined with phenix.refine (statistics in Table 1). In all structures, the engineered tags and 25&#8211;30 residues of the N-terminal ectodomains were very disordered and are not included in the final models. Electron density maps of active site residues and of N-linked glycans are shown in Supplementary Figs S2 and S5, respectively. Structure representations prepared with PyMOL (pymol.org).\nAdditional Information\nHow to cite this article: Santiago, C. et al. Allosteric inhibition of aminopeptidase N functions related to tumor growth and virus infection. Sci. Rep.\n7, 46045; doi: 10.1038/srep46045 (2017).Publisher's note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\nSupplementary Material\n",
        "pmid": "28393915",
        "title": "Allosteric inhibition of aminopeptidase N functions related to tumor growth and virus infection",
        "journal_title": "Scientific Reports",
        "authors": "C\u00e9sar Santiago;Gaurav Mudgal;Juan Reguera;Rosario Recacha;S\u00e9bastien Albrecht;Luis Enjuanes;Jos\u00e9 M. Casasnovas",
        "doi": "10.1038/srep46045"
    },
    {
        "keywords": "",
        "abstract": "Feline infectious peritonitis (FIP), caused by a mutated feline coronavirus, is one of the most serious and fatal viral diseases in cats. The disease remains incurable, and there is no effective vaccine available. In light of the pathogenic mechanism of feline coronavirus that relies on endosomal acidification for cytoplasmic entry, a novel vacuolar ATPase blocker, diphyllin, and its nanoformulation are herein investigated for their antiviral activity against the type II feline infectious peritonitis virus (FIPV). Experimental results show that diphyllin dose-dependently inhibits endosomal acidification in fcwf-4 cells, alters the cellular susceptibility to FIPV, and inhibits the downstream virus replication. In addition, diphyllin delivered by polymeric nanoparticles consisting of poly(ethylene glycol)-block-poly(lactide-co-glycolide) (PEG-PLGA) further demonstrates an improved safety profile and enhanced inhibitory activity against FIPV. In an in vitro model of antibody-dependent enhancement of FIPV infection, diphyllin nanoparticles showed a prominent antiviral effect against the feline coronavirus. In addition, the diphyllin nanoparticles were well tolerated in mice following high-dose intravenous administration. This study highlights the therapeutic potential of diphyllin and its nanoformulation for the treatment of FIP.",
        "body": "Introduction\nFeline coronaviruses (FCoVs) belong to the genus alphacoronavirus in the family Coronaviridae. As a ubiquitous viral pathogen among domestic and wild feline populations, FCoV can be horizontally transmitted via the fecal-oral route with persistent infection, and the prevalence and seropositivity in multi-cat households or shelters may achieve 90%1,2. With regard to pathogenicity, FCoV exists as two biotypes, namely feline enteric coronavirus (FECV) and feline infectious peritonitis virus (FIPV)3. Genetic mutations occurred in FECV within a persistently infected cat allow it to acquire macrophage tropism and change the virus into the virulent FIPV4. Infections caused by the former show no clinically apparent or mild, self-limiting gastroenteritis in kittens. In contrast, FIPV infections develop high fatality and progressive, multi-systemic disorders in young cat populations. Feline infectious peritonitis remains one of the most devastating and lethal viral diseases among Felidae, not only because of the high mortality rate, but also because of the lack of effective options in diagnosis, prevention, and treatment5. The disease is further complicated by antibody-dependent enhancement (ADE), which can exacerbate disease symptoms4.Virus fusion with host cell endosomal membranes, facilitated by a low pH, is a major event for certain virus infection cascade6. Vacuolar ATPase (V-ATPase) activity, which is responsible for pumping protons into endosomal compartments, has been identified as a requirement for feline coronavirus replication in previous studies7. The endosomal acidification process is also necessary for the establishment of ADE in FIPV infection8. Therefore, the endosomal V-ATPase presents an attractive target for antiviral intervention. In the past decades, abundant studies indicated that V-ATPase inhibitors could be applied to treat viral infections as well as other diseases including osteoporosis, lytic bone disease, renal acidosis, and cancer9,10. For example, the most commonly used V-ATPase inhibitor, bafilomycin, showed a broad-spectrum antiviral effect against several pathogens. However, the drug&#8217;s cytotoxicity compromises its clinical applicability11. More recently, diphyllin, a natural compound extracted from the plant Cleistanthus collinus, was discovered as a potent V-ATPase inhibitor with a promising safety profile. Studies have shown that diphyllin at nanomolar concentrations can inhibit V-ATPase activity in bovine chromaffine granules and lysosomal acidification in human osteoclasts with no cytotoxic effect on bone formation12. In a study on treating gastric adenocarcinoma, diphyllin decreased V-ATPase activity and lowered tumor cell infectivity for improved disease outcome13. A previous study also demonstrated diphyllin&#8217;s antiviral effect in vitro against influenza and dengue viruses14, highlighting the compound&#8217;s potential as a broad-spectrum &#8220;host-targeted&#8221; antiviral. The present study thus aims to investigate the compound&#8217;s effect against FIPV.As drug safety and delivery are critical factors that determine an antiviral&#8217;s translational potential, a nanoformulation of diphyllin is herein developed with the aim of improving diphyllin safety and efficacy using poly(ethylene glycol)-block-poly(lactide-co-glycolide) (PEG-PLGA)15. The block-copolymer is highly biocompatible and frequently employed for drug delivery applications, and the nanocarrier may benefit the diphyllin compound in two ways. Firstly, the hydrophobic cores of PEG-PLGA nanocarriers offer an ideal medium for carrying and delivering the hydrophobic diphyllin compound, obviating the need for organic solvents. Secondly, the intracellular uptake of nanoparticles via the characteristic endocytosis mechanism may enhance diphyllin efficacy by facilitating compound colocalization with endosomal V-ATPase, thereby reducing the drug&#8217;s off-target effect and enhancing its antiviral activity. To examine the benefits of the diphyllin nanoformulation, cellular cytotoxicity and antiviral activity between free diphyllin and diphyllin nanoparticles were compared. In addition, an in vitro model of FIPV infection was established to assess the viral inhibitory effect of diphyllin nanoparticles in the context of ADE. Finally, safety of the diphyllin nanoparticles were assessed following intravenous injections in mice. Blood chemistry analysis and body weight monitoring were performed to evaluate the drug&#8217;s safety in vivo.\nResults\n\nDiphyllin blocked endosomal acidification in fcwf-4 cells\nUpon acridine orange labeling, intracellular acidic endosomes were stained red whereas non-acidic compartments appeared green under the fluorescence microscope. As shown in Fig.&#160;1, in comparison to the extensively red-stained endosomes in untreated cells, the addition of bafilomycin A1 or diphyllin showed a decreased number of acidic endosomes and an increased number of non-acidic endosomes in cells. The degree of inhibition in endosomal acidification was shown to correlate with diphyllin concentration (Fig.&#160;1A). To quantitatively analyze the endosomal acidification, green and red fluorescence data collected from diphyllin-treated wells were compared, and the green/red fluorescence ratio was indicated in Fig.&#160;1B. A positive correlation was observed between the green/red ratio and diphyllin concentration, validating inhibition of endosomal acidification in fcwf-4 cells by the diphyllin compound.Figure 1Dose-dependent inhibition of endosomal acidification caused by diphyllin. Fcwf-4 cells were incubated with bafilomycin A1 or various concentrations of diphyllin at 37&#8201;&#176;C for 20&#8201;min. Untreated cells were used as controls. Acridine orange dye (1&#8201;&#956;g/ml), which adopts a green fluorescence in non-acidic endosomes and a red fluorescence in acidic endosomes, was added to each well and incubated for 10&#8201;min. (A) Acidic endosomes in cells were stained red by acridine orange and non-acidic endosomes were stained green. Representative images are shown (magnification: 400&#215;). (B) Fluorescence data was collected from diphyllin-treated wells and the green/red fluorescence ratio was presented. Data in the plot present the mean&#8201;&#177;&#8201;SEM out of four replicates.\n\nPretreatment of diphyllin exhibits the maximal antiviral effect\n2&#8201;&#956;M of diphyllin was added to fcwf-4 cells at three different time points relative to virus infection (Fig.&#160;2A). Compared to the untreated control, diphyllin treatment before, during, and after infection all resulted in cytopathic effect (CPE) inhibition (Fig.&#160;2B) and reduced viral titers in culture supernatants (Fig.&#160;2C). The inhibitory effect against the FIPV infection was most pronounced when the cells were exposed to diphyllin prior to viral exposure. Other incubation schedules also resulted in significant reduction in FIPV titer, thereby validating diphyllin&#8217;s antiviral ability. To further demonstrate the viral inhibition activity by diphyllin, immunofluorescence staining was performed. FITC-labeled antibodies were used to detect FIPV nucleocapsid protein under fluorescence microscopy. As seen in Fig.&#160;2D, with the reference of the DAPI-stained nuclei, cytoplasmic fluorescent signal was reduced with pretreatment of bafilomycin A1 and varying concentrations of diphyllin. A dose-dependent inhibitory effect against FIPV was observed for the diphyllin compound.Figure 2Diphyllin alters cellular susceptibility against FIPV. (A) Schematic presentation of the time-of-addition assays for diphyllin treatment. 2&#8201;&#956;M of diphyllin was added to fcwf-4 cells at three different time points relative to FIPV virus infection (MOI&#8201;=&#8201;0.0001): 1&#8201;h prior to infection (pre-treat), same time as infection (simultaneous) or 1&#8201;h after infection (post-treat). After a 1&#8201;h infection period, all test cells were washed and incubated with fresh media containing 2&#8201;&#956;M of diphyllin and incubated for 24&#8201;h. (B) Cells were then harvested for CPE observation. Virus-induced CPE are indicated by arrows. Infected cells without diphyllin treatment (untreated) or 0.2&#8201;&#956;M bafilomycin-treated cells were used as controls. (C) Extracellular viral titers in culture supernatant were determined with plaque assays. Viral titers between each treated group and the untreated control group were compared by one-way ANOVA followed by Dunnett&#8217;s multiple comparisons test (**p&#8201;<&#8201;0.01). Data in the plot present the mean&#8201;&#177;&#8201;SEM out of three replicates. (D) 0.2&#8201;&#956;M of bafilomycin A1 or various concentrations of diphyllin (0.25, 0.5, 1, 2&#8201;&#956;M) were added to fcwf-4 cells one hr before FIPV infection (MOI&#8201;=&#8201;0.0001). Infected cells without diphyllin treatment were used as controls. After a 1-hr period of infection, cells were washed, overlaid with fresh media containing the same concentrations of diphyllin as in previous step, and incubated for another 24 hr. Fluorescence images of FITC (green), which represents expression of viral nucleocapsid protein, and nucleus (DAPI, blue) were acquired. Representative images are shown (magnification: 400&#215;).\n\nCharacterization of diphyllin-loaded nanoparticles\nThe composition of diphyllin-loaded PEG-PLGA nanoparticles is illustrated in Fig.&#160;3A. To examine the formation of nanoparticles, TEM analysis with negative staining was performed. Under the TEM visualization, diphyllin-load nanoparticles (Fig.&#160;3B) were observed to be uniform in size with an average diameter of 40&#8201;nm. To further characterize the nanoparticles, DLS was used to analyze the size distributions of the nanoparticles. The results from DLS demonstrate that diphylin-loaded nanoparticle exhibited an average particle diameter of 36.19&#8201;&#177;&#8201;1.15&#8201;nm (Fig.&#160;3C). Quantification of diphyllin loading in the PEG-PLGA nanoparticles was performed using HPLC to determine the drug loading yield and efficiency. As presented in Fig.&#160;3D (arrowed peak), the elution of diphyllin was observed at 8.8&#8201;min in the chromatogram. The standard curve was constructed with serially diluted samples of diphylin (1.56, 6.25, 25 and 100&#8201;&#956;g/mL) that separately corresponded to each area value 116.19537, 489.75905, 1997.97016 and 5392.25782 (mAU*second) (Fig.&#160;3D inset). With the reference of standard curve, the content of diphyllin in nanoparticles for each batch was determined. In this study, the loading efficiency of diphyllin was 13.52% with a drug loading yield of 0.67&#8201;wt%.Figure 3Preparation and characterization of diphyllin-encapsulated nanoparticles. (A) Illustration of diphyllin-encapsulated PEG-PLGA nanoparticles. (B) Diphyllin nanoparticles were visualized using transmission electronic microscopy under negative staining with uranyl acetate. Bars&#8201;=&#8201;50&#8201;nm. (C) The nanoparticle size measurement by DLS. (D) The HPLC analysis of diphyllin (inset, standard curve of diphyllin).\n\nSafety and antiviral effect of diphyllin and diphyllin-loaded nanoparticles\nCell viability of fcwf-4 cells treated with various concentrations of diphyllin was measured using an MTT assay. After 24 hr of incubation, the CC50 of diphyllin was determined as 5.99&#8201;&#177;&#8201;0.68&#8201;&#956;M in fcwf-4 cells, whereas the CC50 of diphyllin nanoparticles was 77.26&#8201;&#177;&#8201;13.14&#8201;&#956;M (p&#8201;<&#8201;0.0001) (Fig.&#160;4A), indicating a 13-fold reduction in cellular toxicity with the nanoformulation. No reduction in cellular viability was observed with the empty nanoparticle control at concentrations up to 20&#8201;mg/mL (Supplementary Fig.&#160;S1), which is more than 4 times higher than the highest diphyllin nanoparticle formulation examined in the study (80&#8201;&#956;M of diphyllin; 4.5&#8201;mg/mL of nanoparticles). The result demonstrates a significantly improved safety profile with the diphyllin nanoparticles as compared to the free diphyllin formulation.Figure 4Dose-dependent cytotoxicity and anti-FIPV activity of diphyllin and diphyllin nanoparticles. (A) Various concentrations of diphyllin were added to fcwf-4 cells and incubated for 24 hr. An MTT assay was performed and cell viability was normalized to the value of untreated vehicle controls (100%). Data in the plot present the mean&#8201;&#177;&#8201;SEM out of four test replicates. (B) Various concentrations of diphyllin or diphyllin nanoparticles were added to FIPV (NTU156; MOI: 0.0001)-infected fcwf-4 cells and incubated for 24 hr. An MTT assay was performed and cell viability was normalized to the value of untreated vehicle controls (100%). Data in the plot present the mean&#8201;&#177;&#8201;SEM out of four test replicates. (C) Various concentrations of diphyllin or diphyllin nanoparticles were added to FIPV (NTU204; MOI: 0.01)-infected fcwf-4 cells and incubated for 24 hr. An MTT assay was performed and cell viability was normalized to the value of untreated vehicle controls (100%). Data in the plot present the mean&#8201;&#177;&#8201;SEM out of four test replicates.\nTo assess the IC50 of diphyllin, various non-toxic concentrations of diphyllin were used to treat FIPV infection using an FIPV NTU156 strain at an MOI of 0.0001. An MTT assay was performed to examine the CPE protection. As compared to the untreated control, the presence of diphyllin significantly inhibited the number of CPE foci induced by the viral infection. Based on the MTT assay result, the IC50 value of diphyllin against NTU156 at an MOI of 0.0001 in fcwf-4 cells was 590 &#177; 91&#8201;nM. The diphyllin nanoparticle, on the other hand, showed an IC50 of 9.751&#8201;&#177;&#8201;2.35&#8201;nM (p&#8201;<&#8201;0.0001) (Fig.&#160;4B), which corresponds to a 60-fold enhancement in antiviral activity as compared to the free drug. Empty PEG-PLGA nanoparticles showed no antiviral effect at 10&#8201;mg/mL (Supplementary Fig.&#160;S1). Calculation of therapeutic index (TI&#8201;=&#8201;CC50/IC50) yielded values of 10 and 7923 for the free diphyllin and diphyllin nanoparticles, respectively. The antiviral activities of diphyllin and diphyllin nanoparticles were further assessed against a different strain of FIPV (NTU204). At an MOI of 0.01, the free diphyllin and the diphyllin nanoparticles exhibited IC50 values at 348 &#177; 53.6&#8201;nM and 44.3 &#177; 38.3&#8201;nM respectively (p&#8201;<&#8201;0.0001) (Fig.&#160;4C). Enhancement of diphyllin antiviral efficacy by the nanoparticles was also observed at an MOI of 0.1 (1034 &#177; 49.5&#8201;nM vs 585 &#177; 61.9&#8201;nM in IC50 values; p&#8201;<&#8201;0.001) (Supplementary Fig.&#160;S2), validating that the nanoformulations greatly broadened the therapeutic window of the compound.\nAntiviral activity of diphyllin-loaded nanoparticles in ADE model of FIPV infection\nIn light of the promising therapeutic index of the diphyllin nanoparticles, the nanoparticles&#8217; antiviral activity was further examined against FIPV-infected cells under antibody-dependent enhancement. Plaque assays were conducted to examine the viral load in culture supernatants derived from two different infection conditions in the presence and absence of anti-FIPV mAb. As indicated in Fig.&#160;5, the presence of mAb during viral infection resulted in a significant increase of viral titer from 6&#8201;&#215;&#8201;104 PFU/ml to 8&#8201;&#215;&#8201;105 PFU/ml in the culture supernatant. Fcwf-4 cells were infected with FIPV in the context of ADE, and the antiviral activity of diphyllin at various concentrations was tested by plaque assays after 48 hr of incubation. It was observed that diphyllin reduced antibody-enhanced FIPV infection in a dose-dependent manner, and 2&#8201;&#956;M of diphyllin nanoparticle treatment reduced the viral load by 5 orders of magnitude to 6 PFU/ml. The result highlights the promise of diphyllin nanoparticles in addressing a serious immunopathological complication in FIPV infection.Figure 5Various concentrations of diphyllin loaded PEG-PLGA nanoparticles were added to fcwf-4 cells in a direct infection (white open bar) or ADE infection (black bars) for 48 hr incubation (NTU156, MOI&#8201;=&#8201;0.0035). The viral titers from the collected culture supernatants were titrated by plaques assays. Viral titers between each diphyllin-treated group and the untreated control group were compared by one-way ANOVA followed by Dunnett&#8217;s multiple comparisons test (**p&#8201;<&#8201;0.01). Data in the plot present the mean&#8201;&#177;&#8201;SEM out of three replicates.\n\nDiphyllin nanoparticles are well tolerated \nTo assess the safety of the diphyllin nanoparticles in vivo, blood chemistry analysis and body weight monitoring were performed on mice treated with the nanoformulation. 100&#8201;&#956;L of 80&#8201;mg/mL diphyllin nanoparticle (5&#8201;&#956;g of diphyllin) were intravenously given to 8-week-old BALB/c mice twice on day 0 and day 1, and blood draws were performed on day 0 before the nanoparticle administration, day 3, and day 9. Blood chemistry analysis showed that many parameters, including total protein, blood urea nitrogen (BUN), and creatinine were not affected by the nanoparticle administration (Fig.&#160;6A&#8211;C). Aspartate transaminase (AST) and alanine transaminase (ALT) were observed to be elevated following the treatment but remained within normal ranges16 (Fig.&#160;6D,E). The diphyllin nanoparticles were found to decrease the level of alkaline phosphatase (ALP) on day 3, but the effect was transient as the ALP level recovered by day 9 (Fig.&#160;6F). Monitoring of body weight showed little weight change among the treated animals over 9 days (Fig.&#160;6G). Overall, the diphyllin nanoparticles were well tolerated as the tested animals showed no sign of distress over the 9-day observation.Figure 6Mice intravenously injected twice with 100&#8201;&#956;L of 80&#8201;mg/mL diphyllin nanoparticles (5&#8201;&#956;g of diphyllin) were evaluated for changes in blood chemistry parameters, including (A) total protein, (B) blood urea nitrogen (BUN), (C) creatinine, (D) aspartate transaminase (AST), (E) alanine transaminase (ALT), and (F) alkaline phosphatase (ALP), and (G) body weight. Dashed lines represent normal ranges for BALB/c mice. Data in the plot present the mean&#8201;&#177;&#8201;SEM out of three mice.\n\nDiscussion\nFCoV infection is common in young cats, and up to 12% of infected cats may develop FIP17. To this date, FIP remains fatal in most cases and lacks effective prophylactic and therapeutic treatment options. In clinical settings, recommended disease management during FIP manifestation includes a variety of supportive treatments, such as immunomodulatory drugs, antibiotics, anticoagulants, fluids and supplements3. Studies have shown that disease severity correlates with the systemic viral load18, hence, antiviral therapeutics that reduce the viral spread may be beneficial towards disease control. Previously, a coronaviral protease inhibitor GC376 demonstrated a promising advance in direct inhibition of virus replication in a clinical trial19. Other pathogen-targeted antiviral approaches have been tested in vitro, awaiting further efficacy validation in vivo\n20&#8211;25. Among different antiviral strategies, host-targeted therapeutics are increasingly recognized as an attractive antiviral treatment approach as they do not exert direct selective pressures on viral pathogens. Advantages of host-targeted therapies include broad applicability against multiple viruses and insusceptibility to drug-resistant mutations26,27. Examples of host-targeted treatments against FIP include the use of cyclosporine A to suppress the replication of FIPV in cell culture via the inhibition of cyclophilin28,29. As cyclophilin is a critical host factor responsible for the replication of many members of the Coronaviridae family, cyclosporine A was suggested to be a pan-coronavirus inhibitor30,31. In another example, chloroquine was shown to have anti-FIPV and anti-inflammatory activities in vitro and further relieved clinical symptoms in FIP-infected cats. The compound, however, poses safety concerns and it may inflict liver damage32. Therefore, the present study explores a novel compound and an alternative delivery approach towards facilitating safe and effective anti-FIPV drug development.Diphyllin, a new class of V-ATPase inhibitor, is herein applied for the inhibition of endosomal acidification for FIPV treatment. V-ATPase is a major class of cation translocating enzyme that is involved in a variety of vital processes, including endocytosis, protein trafficking, and metabolites transport. The compound has been previously applied as therapeutics, including treatments against cancer and leishmaniasis. It has also been demonstrated as a host-targeted treatment against infections by influenza and dengue viruses, reducing cellular entry by the viruses by intercepting the endosomal acidification process. The present study further validates the compound&#8217;s antiviral effect against FIPV, affirming the compound&#8217;s broad applicability as an antiviral agent. As V-ATPases are present diverse subunit isoforms in different organs and are involved in different disease pathogenesis, transient, tissue-specific inhibition of V-ATPases by the compound may open up new therapeutic opportunities.In spite of reported complexity of virus internalization33,34, it has been recognized that acidity-mediated endosomal escape and cytosolic entry are essential pathways in the FCoV life cycle7,8,35,36. Medium to low sensitivity to the inhibitor of endosomal acidification (NH4Cl) exhibited by serotype II FCoV 79-1146 cultured in A-72 cells was reported by Regan et al.7, whereas our serotype II FCoV NTU156 and NTU204 cultured in fcwf-4 cells has shown significant reduction when treated with bafilomycin A1, consistent with Takano et al.8, or diphyllin. Differential role for acidic endocytic compartments in cellular entry between these two FIPV strains suggests that the low-pH entry may be strain-dependent and biotype-independent. While diphyllin also exhibited inhibitory effect on NTU156 cultured in U937 cells under the ADE infection (Supplementary Fig.&#160;S3), we speculate the sensitivity to dipyllin exhibited by NTU156 is cell type-independent. In addition, type II FCoV 79-1146 cultured in fcwf-4 cells was found less sensitive to the treatments of viroporin inhibitors and cholesterol24,37.Towards applying diphyllin for therapeutic purposes, the compound&#8217;s insolubility and potential adverse effect are two concerns that can be addressed using nanocarrier technology. Nanoparticle-based drug delivery presents unique advantages that arise from the carriers&#8217; nanoscale morphology and drug loading capacity. The approach has been applied extensively in anticancer applications, yielding formulations with improved delivery efficiency and therapeutic efficacy15,38&#8211;40. More recently, nanomedicine platforms have begun to draw attention as an antiviral delivery platform. For example, acyclovir, a common antiviral drug against herpes simplex virus type-1 (HSV-1), was delivered by nanocarriers for enhanced efficacy and reduced side effects, exhibiting a three-fold improvement in therapeutic effect in comparison to free drugs41. Saliphenylhalamide (Saliphe), a V-ATPase inhibitor, has also been shown to display enhanced viral inhibition when delivered via nanoparticles42. The virus-like dimension of nanoparticles may favor drug delivery to sites of viral infections as many similarities regarding cellular entry and in vivo biodistributions have been observed between viruses and nanocarriers43. In the present study, PEG-PLGA, a biocompatible block co-polymer widely used for drug delivery applications44&#8211;47, was applied for the encapsulation and delivery of diphyllin. The nanoparticles significantly improved the safety and efficacy of the diphyllin compound, increasing the therapeutic index by approximately 800-fold in one of our infection models. This pronounced enhancement can be attributed to multiple characteristics of the nanomaterial. Firstly, the hydrophobic nature of the PEG-PLGA nanoparticle cores facilitates diphyllin incorporation and obviates the need of organic solvents for compound dissolution. As a result, rather than permeating through the cells with the aid of DMSO, nanocarrier-encapsulated diphyllin relies on nanoparticle-mediated endocytosis for cellular entry. V-ATPase, the target of diphyllin, are ubiquitous among intracellular compartments and govern a multitude of physiological cellular functions10. Enhancing diphyllin localization inside endosomes may thus reduce the compound&#8217;s potential side effects. Higher drug efficacy and improved cell protection by the nanoformulations can also be directly attributed to enhanced endosomal drug delivery by the nanoparticles. A higher effective dosage can efficiently reach the endosomal protein target upon nanoparticle delivery.As an immune-related disease, FIPV possesses an ADE characteristic that contributes to immunopathology and complicates disease prognosis. The recognition of FIPV immune-complexes via the FcR by immune cells can enhance disease progression through altered cross-reactive cellular immune response, inefficient lysis of infected cells, and altered T cell-cytokine responses that collectively promote ADE. Although the relationship between serotypes or strains and the ADE phenomenon remains unclear48, the requirement of endosomal acidification for ADE has been well established8. Our study has demonstrated that diphyllin nanoparticles possess dose-dependent antiviral activity in an in vitro model of ADE of FIPV infection, which mimics the disease&#8217;s clinical manifestation. By attenuating cellular organelle acidification, the nanoformulation effectively inhibits endosomal acidification responsible for virus uncoating and cytoplasmic entry (Fig.&#160;7). As the diphyllin nanoparticles are shown to be well tolerated in vivo upon high-dose intravenous administration, the formulation presents a promising treatment option against the lethal feline disease. Future studies with in vivo animal models of viral infections are warranted.Figure 7Schematic diagram of diphyllin&#8217;s antiviral mechanism. Diphyllin demonstrated prominent inhibitory activity in anti-FIPV infection by attenuating cellular organelle acidification, thus blocking virus entry. In addition, diphyllin treatment displayed suppression in viral load in the context of ADE of FIPV infection. By using a nanoparticulate drug delivery system, diphyllin exhibited improved safety and enhanced antiviral activity.\n\nConclusions\nDiphyllin, a V-ATPase inhibitor, is herein demonstrated to reduce FIPV infection by inhibiting the endosomal acidification process that is required for virus uncoating and cellular entry. Diphyllin pretreatment to cells prior to viral infection altered the cellular susceptibility to the virus and maximized the compound&#8217;s antiviral activity. To enhance the translational potential of the antiviral compound, PEG-PLGA nanoparticles were applied for drug encapsulation and delivery. The nanoformulation significantly reduced the toxicity and increased the antiviral effect of diphyllin, enhancing the compound&#8217;s therapeutic index by up to 800-fold in in vitro analysis. The diphyllin nanoparticles potently inhibits FIPV infection in an in vitro ADE model. In an animal study, the diphyllin nanoparticles were well tolerated following intravenous administration. The present work highlights diphyllin and diphyllin nanoparticles as effective host-targeted antivirals. It presents a promising treatment modality against lethal infectious diseases.\nMethods\n\nCells\nFcwf-4 (felis catus whole fetus-4) cell line was purchased from ATCC and maintained at 37&#8201;&#176;C with 5% CO2 in Dulbecco&#8217;s modified Eagle&#8217;s medium supplemented with 10% fetal bovine serum and 1% penicillin/streptomycin/amphotericin B. U937 cell line was purchased from the Bioresource Collection and Research Center (Hsinchu, Taiwan) and maintained at 37&#8201;&#176;C with 5% CO2 in RPMI-1640 medium supplemented with 10% fetal bovine serum and 1% penicillin/streptomycin/amphotericin B. All the culture reagents were purchased from Gibco.\nViruses, plaque assays, and quantitative RT-PCR\nThe serotype II FIPV strains NTU15649 and NTU204 isolated at National Taiwan University were used in this study. The viruses were propagated and titrated in fcwf-4 cells. The titrated viral stocks were stored in &#8722;80&#8201;&#176;C. For plaque assays, fcwf-4 cells were seeded in 12-well plates at 2&#8201;&#215;&#8201;105 cells/well one day prior to experiment. Right before experiment, media were removed and cells were washed. Virus-containing samples were ten-fold serially diluted with media, and 500&#8201;&#956;l of the diluted samples were added to cells. After incubation at 37&#8201;&#176;C for 1 hr, media were again removed and cells were washed. 1&#8201;mL of DMEM containing 1.8% methylcellulose (Sigma) and 2% FBS was added to each well. After 3 days, the plates were fixed with 10% formalin and stained with 1% crystal violet at room temperature for 10&#8201;min. Upon drying, the viral plaques were counted, and the plaque-forming unit (PFU) per ml of virus was determined. For FIPV infection in U937 cells, quantitative RT-PCR (qRT-PCR) was used for virus quantification. Briefly, viral RNA was extracted using a viral nucleic acid extraction kit (Geneaid Biotech Ltd., Taipei, Taiwan). cDNA was synthesized using a previously described P1b primer, and qPCR was performed using the primer set P009 and P010, and the fluorescence probe P9/1050. The cycling protocol in a thermal cycler (Biorad CFX Connect) consisted of 3&#8201;min of preheating at 95&#8201;&#176;C, and followed by 35 cycles of 15&#8201;s of denaturation at 95&#8201;&#176;C, 1&#8201;min of primer annealing and extension at 60&#8201;&#176;C. 2&#8201;&#956;L of plasmid DNA containing various copy numbers of membrane-nucleocapsid protein gene of FIPV was used for standard curve construction in qRT-PCR for viral load quantification. Two replicates were performed for each samples or standards.\nDiphyllin and diphyllin-encapsulated nanoparticles\nDiphyllin (ChemDiv, San Diego, CA) and bafilomycin A1 (Sigma) were dissolved in dimethyl sulfoxide (DMSO, Sigma). For nanoparticle preparation, diphyllin was dissolved in chloroform at 1&#8201;mg/ml with 20&#8201;mg/mL of PEG-PLGA (Sigma). 1&#8201;mL of the solvent mixture was added to 5&#8201;mL of deionized water and probe sonicated for 1&#8201;min and 45&#8201;sec. The resulting emulsion was then placed in a fume hood for chloroform evaporation for 3 hr. For the vehicle control, empty PEG-PLGA nanoparticle was prepared in the absence of diphyllin. Particles were washed for 3 times using the Amicon Ultra 15&#8201;mL Centrifugal Filters (100k MWCO; Millipore) at 2000&#8201;&#215;&#8201;g for 4&#8201;min. Nanoparticle size distribution was measured by DLS (Malvern Instruments Inc., UK) based on the manufacturer&#8217;s instructions. The washed particles were then resuspended in 10% sucrose and stored at &#8722;80&#8201;&#176;C.\nAnalysis of endosomal acidification with acridine orange labeling\nFcwf-4 cells (4&#8201;&#215;&#8201;104 cells/well) were cultured onto an 8-well chamber slide (Nunc) one day before the experiment. Each well was incubated with various concentrations of diphyllin or 0.2&#8201;&#956;M of bafilomycin A1 at 37&#8201;&#176;C for 20&#8201;min. Bafilomycin A1 was used as a positive control, and 2% DMSO was applied as vehicle controls. Acridine orange dye (1&#8201;&#956;g/ml; Biotium) was added to each well and incubated at 37&#8201;&#176;C for 10&#8201;min before wash. Fluorescence images were obtained by fluorescence microscopy, and the red and green fluorescence colors were acquired using Texas Red and FITC filters respectively. Fluorescence intensity (green/red ratio) in each well was analyzed by ImageJ.\nTime-of-addition assay\nFcwf-4 cells were seeded in a 12-well plate at a density of 2&#8201;&#215;&#8201;105 cells/well 1&#8201;day before the experiment. 2&#8201;&#956;M of diphyllin was added to the cells at three different time points with respect to virus infection. The time points include 1 hr prior to infection, same time as infection, and 1 hr after infection. Cells treated with 2% DMSO were prepared as controls. Virus (NTU156) was inoculated at an MOI of 0.0001. After 1hr of virus incubation, all cells were washed and resuspended in fresh media containing 2&#8201;&#956;M of diphyllin. After 24&#8201;h, the cellular supernatants were collected, and virus titers in the supernatants were assessed by plaque assays.\nImmunofluorescence staining of intracellular viral protein\nThe immunofluorescence staining was performed to validate the antiviral effect of diphyllin. Fcwf-4 cells were seeded onto an 8-well chamber slide (4&#8201;&#215;&#8201;104 cells/well) one day before the experiment. After being treated with various concentrations of diphyllin for 1 hr, the cells were infected with FIPV-NTU156 at an MOI of 0.0001. Each well was fixed by 80% acetone at &#8722;20&#8201;&#176;C for 20&#8201;min and air-dried at room temperature. Mouse anti-FCoV monoclonal antibody (clone: FIPV3-70; Biorad) was used as the primary antibody stain. FITC-conjugated goat anti-mouse IgG (Jackson Lab) was used as the secondary antibody stain. DAPI-containing mounting media (Vector shield) was then added to the antibody-stained cells, and the cells were observed using a fluorescence microscope.\nTransmission electronic microscopy\nFor transmission electron microscopy, 10&#8201;&#956;L of nanoparticle sample was added onto a glow-discharged grid. The grid was then washed with ddH2O. Subsequently, the grid was stained with 1% uranyl acetate for 15&#8201;sec and excessive staining was removed by filter paper. The grid was air-dried overnight and stored in the desiccator until TEM observation using an FEI 120&#8201;kV Sphera microscope (FEI Tecnai F20).\nQuantification of diphyllin by HPLC\nThe content of diphyllin in the PEG-PLGA nanoparticles was quantified by HPLC (Agilent 1100 series). 200&#8201;&#956;l of control PEG-PLGA nanoparticles and particles loaded with diphyllin were dissolved with 800&#8201;&#956;l of acetone, which was then air-dried with nitrogen. The samples were then dissolved in mobile phase solvent and filtered prior to HPLC. Ascentis C18 column was equipped while mobile phase was prepared using methanol and water in proportion of 60% and 40%. The optimal detection wavelength was 254&#8201;nm and column temperature was 25&#8201;&#176;C. Serially diluted samples of diphylin (1.56, 6.25, 25, 100&#8201;&#956;g/mL) dissolved in the mobile phase solvent were prepared as standards for the HPLC analysis. The encapsulation efficiency (EE) of diphyllin was calculated using the formula: EE&#8201;=&#8201;(encapsulated drug/drug input)&#8201;&#215;&#8201;100%. The drug loading yield was calculated using the formula: loading yield&#8201;=&#8201;(weight of encapsulated drug/total polymer weight)&#8201;&#215;&#8201;100%.\nNone\nThe in vitro cytotoxicity of diphyllin and diphyllin nanoparticles was measured by an MTT assay. To obtain 80% confluent cell monolayers, fcwf-4 cells (2&#8201;&#215;&#8201;104/well) were seeded in 96-well plates with 100&#8201;&#956;l of medium one day before the experiment. Cells were treated with different concentrations of diphyllin dissolved in 2% DMSO or diphyllin nanoparticles. After 24 hr, the culture supernatants were removed. 20&#8201;&#956;l of 5&#8201;mg/ml MTT solution (3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide) was added to each well and the cells were cultured at 37&#8201;&#176;C for 4 hr. The supernatant was removed and 100&#8201;&#956;l of DMSO was added to each well to solubilize the purple formazan crystals. After 20&#8201;min, the absorbance was measured at 540&#8201;nm and 650&#8201;nm using a microplate reader. Untreated cells containing 2% DMSO were prepared to represent 100% cell viability. When testing the diphyllin nanoparticles, empty PEG-PLGA nanoparticles were used as the vehicle control. The concentration of diphyllin that reduced cell viability by 50% (CC50) was analyzed using GraphPad Prism (GraphPad Software, San Diego, CA).\nCell cytopathic effect (CPE) inhibition assay and determination of IC\nTo examine the antiviral effect of diphyllin, IC50 of diphyllin (the concentration of diphyllin showing 50% inhibition of virus-induced CPE) against FIPV-NTU156 was assessed using the cell viability assay. Briefly, fcwf-4 cells were seeded on 96-well microplates (2&#8201;&#215;&#8201;104 cells/well) one day prior to experiment to obtain an 80% confluent cell monolayer. Diphyllin was twofold diluted and added to fcwf-4 cells for 1 hr before virus inoculation. After 1 hr period of infection with a multiplicity of infection (MOI) of 0.0001 with FIPV NTU156, each well was washed and overlaid with media containing diphyllin of the same concentrations as in the initial treatment step. An alternative FIPV-NTU204 strain was tested using the same infection protocol with an MOI of 0.01 and 0.1. After another 24 hr of incubation, cellular viability was examined by an MTT assay. Signals from cells without infection and diphyllin treatment were used to represent 100% cell viability. The concentration of free diphyllin and diphyllin nanoparticles that inhibited virus-induced CPE by 50% (CC50) was determined by fitting the data onto a non-linear regression curve using GraphPad Prism (GraphPad Software).\nEstablishment of \nUpon addition of the mAb 6-4-2, the FIPV infection can be enhanced via Fc receptor (FcR)8. To set up the ADE infection condition, 1&#8201;mL of NTU156 viral suspension (1.4&#8201;&#215;&#8201;104 PFU) was mixed with 1&#8201;mL of the mAb 6-4-2 (1:10 diluted hybridoma supernatant) in a 15-mL centrifuge tube at 4&#8201;&#176;C for 1 hr on a rotator. Subsequently, 0.1&#8201;mL of the resulting mixture was inoculated to fcwf-4 or U937 cells (2&#8201;&#215;&#8201;105 cells/well in a 12-well plate), which were pretreated with diphyllin for 1 hr. Bafilomycin A1 was used as a positive control. After the virus-antibody complex was added to cells and co-cultured for 3 hr (MOI&#8201;=&#8201;0.0035), the cells were washed 3 times and re-incubated in diphyllin-containing fresh media. 48 hr later, supernatant from each well was collected and viral titers were determined by a plaque assay or qRT-PCR.\nDiphyllin nanoparticle safety in mice\n8-week-old female BALB/c mice were intravenously administered with 100&#8201;&#956;L of 80&#8201;mg/mL diphyllin nanoparticle (5&#8201;&#956;g of diphyllin) twice on day 0 and day 1. Blood were withdrawn from the mice on day 0 before the nanoparticle administration, day 3, and day 9 for blood chemistry analysis. The mice were monitored for signs of distress and change in body weight for 9 days. The animal protocol was reviewed and approved by the Institutional Animal Care and Use Committee (IACUC) of the National Taiwan University (approval no. NTU105-EL-00181). All animal experiments were carried out in accordance with the approved guidelines.\nStatistical analyses\nData were analyzed by unpaired t tests or ANOVA by Dunnett&#8217;s multiple comparison tests using GraphPad Prism (GraphPad Software). The p values smaller than 0.05 were considered significant.\nData availability\nThe datasets generated during the current study are available from the corresponding author on reasonable request.\nElectronic supplementary material\n\nNone\n\nSupplementary information\n",
        "pmid": "29026122",
        "title": "Nanoparticulate vacuolar ATPase blocker exhibits potent host-targeted antiviral activity against feline coronavirus",
        "journal_title": "Scientific Reports",
        "authors": "Che-Ming Jack Hu;Wei-Shan Chang;Zih-Syun Fang;You-Ting Chen;Wen-Lin Wang;Hsiao-Han Tsai;Ling-Ling Chueh;Tomomi Takano;Tsutomu Hohdatsu;Hui-Wen Chen",
        "doi": "10.1038/s41598-017-13316-0"
    },
    {
        "keywords": "Keywords: phylogenetic analysis;coronavirus NL63;molecular detection;pneumonia;bronchiolitis;multiplex assay",
        "abstract": "Coronavirus NL63 was found in hospitalized children with upper and lower respiratory infections.",
        "body": "Materials and Methods\nFrom November 2002 to April 2003, the virology laboratory (University Hospital, Caen, France) received 1,427 respiratory samples (nasal aspirates and swabs) from patients <20 years of age. All specimens were tested for influenza virus A and B; respiratory syncytial virus (RSV); parainfluenza virus 1, 2, 3, and 4; and adenovirus by direct or indirect immunofluorescence or virus isolation. Samples were also tested for human metapneumovirus (HMPV), rhinovirus, enterovirus, and HCoV 229E and OC43 by virus isolation and RT-PCR. A total of 556 samples (39%) were positive for any of these viruses. Symptoms indicated viral infection, as judged by the clinical department; therefore, samples were not tested for bacterial pathogens. Of the 556 positive samples, the following respiratory viruses were detected: RSV (37%, n = 205), rhinovirus (18%, n = 101), influenza virus A and B (15%, n = 86), HMPV (9.7%, n = 54), and HCoV-OC43 (1.2%, n = 7); no HCoV-229E were detected.Of the 871 negative samples, 300 (50 per month) were tested for HCoV-NL63. These 300 samples represented 191 patients <2 years of age (64%), 46 patients 2&#8211;5 years of age (15%), and 63 patients 6&#8211;20 years of age (21%). All patients were hospitalized with acute respiratory tract illness. Data for 18 patients with recorded HCoV-NL63 infection were available and were examined retrospectively for specific respiratory symptoms. All patients consented to having their samples tested for respiratory viruses, including coronaviruses.Two RT-PCR assays were used to detect HCoV-NL63 in respiratory samples. RNA was extracted by using the QIAamp Viral RNA Mini Kit (Qiagen, Hilden, Germany) according to manufacturer's instructions. The first RT-PCR assay was a 1-step simple RT-PCR that amplified a 255-bp fragment of the nucleocapsid (N) gene of HCoV-NL63 by using the following primers: N5-PCR2 (5&#180;-GATAACCAGTCGAAGTCACCTAGTTC-3&#180;) and N3-PCR2 (5&#180;-ATTAGGAATCAATTCAGCAAGCTGTG-3&#180;). The second assay was a 1-step multiplex RT-PCR that amplified the same 255-bp fragment of the N gene of HCoV-NL63, a 574-bp fragment of the membrane (M) gene of HCoV-229E, and a 334-bp fragment of the M gene of HCoV-OC43 by using previously described primers (4,5). These assays (OneStep RT-PCR kit, Qiagen) were undertaken in 25-&#956;L reaction volume containing 2.5 &#956;L RNA extract, 5 &#956;L 5&#215; Qiagen OneStep RT-PCR buffer, 1 &#956;L 10 mmol/L deoxynucleoside triphosphate (dNTP), 1 &#956;L Qiagen OneStep RT-PCR Enzyme Mix, 1.2 &#956;L of 10 mmol/L each primer, 3 &#956;L Qiagen OneStep RT-PCR kit Q solution, and RNase-free water to 25 &#956;L. The reaction was carried out in a GeneAmp PCR system 2700 thermal cycler (Applied Biosystems, Foster City, CA, USA) with an initial reverse transcription step at 50&#176;C for 30 min, followed by PCR activation at 95&#176;C for 15 min, 40 cycles of amplification (30 s at 95&#176;C, 30 s at 58&#176;C, 1 min at 72&#176;C), and a final extension step at 72&#176;C for 10 min. Each RT-PCR test included water controls that were treated identically to the virus samples throughout and was performed with usual precautions to avoid contamination.RT-PCR products were subject to electrophoresis on an agarose gel, stained with ethidium bromide, and visualized under UV light. The comparative analytical sensitivities of these simple and multiplex RT-PCR assays were previously studied on prototype strains by analyzing serial 10-fold dilutions of positive control for HCoVs NL63, OC43, and 229E. The analytical sensitivity was equivalent to detect HCoV-OC43, the multiplex assay was more sensitive (by one 10-fold dilution) to detect HCoV-229E, and less sensitive (by one 10-fold dilution) to detect HCoV-NL63 (data not shown). No cross-reaction of these tests was observed between these coronaviruses. Samples that were positive for HCoVs NL63, 229E, and OC43 were confirmed by using a DNA enzyme immunoassay (GEN-ETI-K DEIA, Sorin, Saluggia, Italy) carried out as recommended by the manufacturer with original probes previously described for HCoVs 229E and OC43 and the following probe defined in the N gene for HCoV-NL63: 5&#180;-(Biotin)CCTCTTTCTCAACCCAGGGCTGATA-3&#180; (4). A third RT-PCR assay was carried out on 12 HCoV-NL63&#8211;positive samples amplifying a 523-bp fragment with spike (S) gene&#8211;specific primers NL63-S-sens (position 22557&#8211;22582: 5&#180;-ACCGCTGTTAATGAGTCTAGATATG-3&#180;) and NL63-S-antisens (position 23043&#8211;23063: 5&#180;-GTCCTGCTATACGGCTTGAA-3&#180;). This assay was performed essentially as described above. The RT-PCR products were purified by using ExoSAP-IT (USB Corporation, Cleveland, OH, USA) and sequenced with the primers by using the CEQ Dye Terminator Cycle Sequencing Quick Start Kit on a CEQ 8000 Genetic Analysis System (Beckman Coulter, Fullerton, CA, USA). The nucleotide sequences of the partial S gene (GenBank accession nos. AY994243&#8211;AY994254) were compared with the 2 prototype HCoV-NL63 sequences available in GenBank (NL63-Amsterdam1: NC005831 and NL: AY518894). Both nucleotide and predicted amino acid sequence alignments were prepared by using ClustalX version 1.83. The phylogenetic trees were constructed by using HCoV-229E as an outgroup.\nResults\nHCoV-NL63 was detected in 28 (9.3%) of the 300 samples evaluated from November 2002 to April 2003. Twenty-two samples were positive for HCoV-NL63 by both simple and multiplex RT-PCR (Figure 1). Discordant results were found for the remaining 6 samples: 3 were positive only in simple RT-PCR, and 3 were positive only in multiplex RT-PCR. These discordant samples were controlled by using the same methods from the RNA extraction product; the results obtained were identical. The specificity of the RT-PCR products under UV was confirmed by hybridization. Multiplex RT-PCR identified 3 samples with HCoV-OC43 and 1 sample with both HCoV-NL63 and HCoV-OC43. The 28 HCoV-NL63&#8211;positive samples were obtained from 18 patients <2 years of age (65%), 4 patients 2&#8211;5 years of age (14%), and 6 patients 6&#8211;15 years of age (21%). The age distribution of the patients infected by HCoV-NL63 was identical to the age distribution of the sample. Positive specimens were collected throughout the study period; no epidemic was observed. The temporal distribution of HCoV-NL63 infection is shown in Figure 2.All patients included in this study had a respiratory tract illness. The medical reports of 18 patients with HCoV-NL63&#8211;positive samples were retrospectively examined, and the following symptoms were noted: fever (61%, n = 11), rhinitis (39%, n = 7), lower respiratory tract illness (bronchiolitis, pneumonia [39%, n = 7]), digestive problems (diarrhea and abdominal pain [33%, n = 6]), otitis (28%, n = 5), pharyngitis (22%, n = 4), and conjunctivitis (17%, n = 3). One patient had severe underlying disease (congenital immunodeficiency) and had upper respiratory tract illness with fever, another had a family history of atopic allergy, and pneumonia was diagnosed. Overall, more than one third of the patients infected by HCoV-NL63 had severe lower respiratory tract infection (6 bronchiolitis and 1 pneumonia). All of them recovered completely.To determine if the isolates from France contain different genetic markers, we sequenced a part of the S protein gene of 12 isolates for molecular analysis. The phylogenetic analysis shows that the isolates from France are a divergent group containing sequences with different markers. One isolate (23034101) had characteristics of an outlier (Figure 3A). The phylogenetic analysis of the predicted amino acid sequence also shows that this isolate is an outlier (Figure 3B). However, the branches of this tree are based on only 1 amino acid difference. Care should be taken because of the limited informative sites in the sequence.\nDiscussion\nA number of viruses cause respiratory infections, and many infections cannot be attributed to any known pathogen (6). This fact may be because some detection methods lack sensitivity, because some respiratory viruses are not systematically tested for, or because some pathogens are not yet identified. Of the 4 novel agents, HMPV, SARS-CoV, HCoV-NL63, and CoV-HKU1, identified recently, 3 were coronaviruses (1,3,7&#8211;9). Coronaviruses infect many species of mammals and birds, they possess the largest genome of all RNA viruses (&#8776;30 kb), and they have a high frequency of recombination. In addition, the potential to infect other species has been described for bovine coronavirus and is suspected to have caused the SARS outbreak (10,11). Therefore, coronaviruses represent a potential major infectious agent in humans. Based on genotypic and serologic characteristics, coronaviruses were divided into 3 distinct groups: with HCoVs 229E and NL63 in group 1 and HCoVs OC43 and HKU1 in group 2. SARS-CoV is not definitively assigned to any of these groups. However, an early split-off of SARS-CoV from the group 2 lineage was suggested (12). Until recently, only the HCoVs 229E and OC43 and SARS-CoV have been thoroughly studied. As suggested by epidemiologic surveys conducted in the 1970s, human coronaviruses are distributed worldwide and circulate during seasonal outbreaks (13&#8211;15). In this study, we determined whether this is also the case for HCoV-NL63. Furthermore, we looked at the symptoms of a HCoV-NL63 infection and the heterogeneity of the virus isolate circulating in France. In this study, 9.3% of the samples were positive for HCoV-NL63. These results suggest that HCoV-NL63 can frequently cause infections, particularly in young children.Because of availability, only samples that tested negative for other respiratory viruses were included in this study. Consequently, we could not identify co-infections. Nevertheless, by using multiplex RT-PCR, which simultaneously detected the classical human coronaviruses (OC43, 229E, and NL63), we detected 1 co-infection by HCoVs NL63 and OC43 and 3 HCoV-OC43&#8211;positive samples. These additional cases of HCoV-OC43 infection in our patients can be explained by the fact that multiplex RT-PCR was performed directly on respiratory samples, whereas in previous routine tests, samples were first instilled into a cell culture system (HUH7 cell line) and RT-PCR was then performed on cell culture supernatant.Detecting human respiratory coronaviruses requires molecular techniques because of difficulties in virus culture and lack of an assay to detect intracellular antigens or others serologic assay. A multiplex RT-PCR is therefore a useful tool to simultaneously test for various HCoVs from a clinical sample. This study showed that the clinical sensitivity of multiplex RT-PCR was equivalent to simple RT-PCR, allowing clinical studies and routine testing.No HCoV-229E was detected in samples. HCoVs 229E and NL63 both belong to antigenic group 1. The percentage amino acid sequence identities between the S, M, and N proteins of HCoVs NL63 and 229E are 54.7%, 61.5%, and 43.2%, respectively (2). A cross-protective immune response could explain why these 2 human coronaviruses do not circulate at the same time. We detected HCoV-NL63 in respiratory specimens in February with a frequency of 18%. These results correlate with the fact that human coronaviruses circulate primarily in the winter. However, HCoV-NL63 was found in nasal aspirates each month of our study.The clinical symptoms associated with HCoV-NL63 still need to be determined. In the patients in our study, symptoms included not only respiratory symptoms but also lower respiratory tract diseases such as bronchiolitis, bronchitis, and pneumonia. Human coronaviruses, except SARS-CoV, generally cause disease much like the common cold, but they have also been associated with more severe lower respiratory tract conditions, especially in frail patients (4,16). Whether HCoV-NL63 is also responsible for coldlike illnesses in healthy adults, as has been described for HCoVs 229E and OC43, must be determined. The same can be said about the very recently described coronavirus CoV-HKU1. This virus was identified in a 71-year-old patient with chronic obstructive airway disease who was hospitalized with pneumonia (3).Digestive problems were noted in approximately one third of patients. No clear evidence exists that human coronaviruses, except SARS-CoV, cause enteric illness, but previous studies have suggested that these viruses may be involved in enteric diseases (17&#8211;20). Further studies must be conducted to detect coronaviruses in stool samples and clarify the origin of these digestive symptoms.The S protein of coronaviruses is a major determinant of cell tropism and pathogenicity and a major inducer of neutralizing antibodies (21). Furthermore, heterogeneity of the S gene has been observed for different HCoV-NL63 strains. We amplified part of the S gene to study the variability of our isolates. Phylogenetic analysis showed that several different isolates are cocirculating in France, similar to the situation in the Netherlands, Australia, Canada, and Belgium.In conclusion, HCoV-NL63 can be found in patients with upper and lower respiratory tract illness, particularly in hospitalized children. This observation is the first of HCoV-NL63 infection in France, and several isolates of HCoV-NL63 were found to circulate in our country. The sensitive multiplex RT-PCR for the HCoVs NL63, 229E, and OC43 that we developed is a useful tool to facilitate the routine detection of these pathogens.This project was supported by the European Commission EPISARS contract (No. SP22-CT-2004-511063) and the Programme de Recherche en R&#233;seaux Franco-Chinois (&#233;pid&#233;mie du SRAS: de l'&#233;mergence au contr&#244;le).",
        "pmid": "16102311",
        "title": "Human Coronavirus NL63, France",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Astrid Vabret;Thomas Mourez;Julia Dina;Lia van der Hoek;St\u00e9phanie Gouarin;Jo\u00eblle Petitjean;Jacques Brouard;Fran\u00e7ois Freymuth",
        "doi": "10.3201/eid1108.050110"
    },
    {
        "keywords": "Bats;coronavirus;viruses;severe acute respiratory syndrome;human coronavirus;molecular clock;Ghana;research",
        "abstract": "Hipposideros spp. bats harbor a coronavirus that shares common ancestry with human viruses.",
        "body": "Materials and Methods\n\nCapturing and Sampling\nIn the locations identified in Figure 1, mist netting and sampling were conducted as described (11). In Kumasi Zoo, fecal samples were collected with plastic foil under trees occupied by Eidolon helvum bats (estimated colony size 300,000). For all capturing and sampling, permission was obtained from the Wildlife Division of the Ministry of Lands, Forestry, and Mines in Ghana. Research samples were exported under a state agreement between the Republic of Ghana and the Federal Republic of Germany, represented by the City of Hamburg. Additional export permission was obtained from the Veterinary Services of the Ghana Ministry of Food and Agriculture.\nProcessing and Analysis of Samples\nSamples (1&#8211;4 fecal pellets or swabs suspended in RNA stabilization solution [RNAlater Tissue Collection; Applied Biosystems, Foster City, CA, USA]) were tested at the Kumasi Centre for Collaborative Research in Tropical Medicine as described (11,18). After initial sequencing, specific primers were designed for each group of CoV found. Nested reverse transcription&#8211;PCR (RT-PCR) primer sets used for sequencing of longer fragments of representative viruses are available upon request. The following sequences were derived from this study and were submitted to GenBank under the listed accession numbers: RNA-dependent RNA polymerase (RdRp) sequences: BtCoV/Hip/GhanaBoo/348/2008, FJ710043; BtCoV/Hip/GhanaBoo/344/2008, FJ710044; BtCoV/Hip/GhanaKwam/8/2008, FJ710045; BtCoV/Hip/GhanaKwam/19/2008, FJ710046; BtCoV/Hip/GhanaKwam/20/2008, FJ710047; BtCoV/Hip/GhanaKwam/13/2008, FJ710048; BtCoV/Hip/GhanaKwam/31/2008, FJ710049; BtCoV/Hip/GhanaKwam/27/2008, FJ710050; BtCoV/Hip/GhanaKwam/26/2008, FJ710051; BtCoV/Hip/GhanaKwam/24/2008, FJ710052; BtCoV/Hip/GhanaKwam/10/2008, FJ710053; BtCoV/Hip/GhanaKwam/22/2008, FJ710054/nucleocapsid sequences; BtCoV/Hip.sp/GhanaBoo/344/2008, FJ710055; BtCoV/Hip.sp/GhanaKwam/19/2008, FJ710056.\nPhylogenetic Analysis\nNucleic acid alignments were conducted based on amino acid code by using the ClustalW algorithm (www.ebi.ac.uk/clustalw) in the Molecular Evolutionary Genetics Analysis version 4.0 software package (www.megasoftware.net) (19). Two gap-free nucleotide alignments (817 bp and 1,221 bp) were generated. Tree topologies were determined on both datasets by using MrBayes version 3.1 (20). The analysis used a general time reversible (GTR) substitution model with 6 rate categories to approximate a gamma-shaped rate distribution across sites and an invariant site assumption (GTR + &#915;6 + I). Markov chain Monte Carlo (MCMC) chains of 107 iterations were sampled every 500 generations, resulting in 20,000 sampled trees. Two Metropolis-coupled chains (1 cold and 3 heated chains each) were run in parallel, compared, and pooled. Convergence of chains was confirmed by the potential scale reduction factor statistic in MrBayes (21) and by visual inspection of each cold chain using the TRACER program (22). Phylogenetic dating was conducted by using Bayesian evolutionary analysis sampling trees (BEAST) (22). Chain lengths in BEAST were at least 20,000,000 generations with sampling every 500 generations. Convergence of the model was checked visually and by the effective sample size statistic with TRACER.\nResults\n\nVirus Detection\nDuring February 2008, bats were sampled in the described locations around Kumasi, Ghana. Initially, 7 fecal samples tested positive by pan-CoV PCR. Products (440 bp, RdRp gene) were sequenced and aligned with prototype CoV. Neighbor-joining phylogenies indicated 2 distinct groups of sequences that belonged to CoV group 1 (n = 4) and group 2 (n = 3), respectively. Specific primer pairs for the group 1 and group 2 sequences were designed and applied again to all samples. Five additional viruses were found, resulting in a total CoV prevalence of 9.76% in insect-eating bats (n = 123). No virus was found in any oral swab. All virus findings in fecal samples are listed by capture site in Table 1.Notably, all CoV findings were in insect-eating leaf-nosed bats of the genus Hipposideros. Within the genus, the species H. abae could be discriminated unambiguously by morphology (Table 1). The remaining Hipposideros species were assigned to the complex of forms related to currently recognized species H. caffer and H. ruber. Because 2 morphotypes were present (Figure 2), the mitochondrial cytochrome b gene was sequenced as described (23). Both morphotypes belonged to phylogenetic lineages distinct from H. caffer and possibly represented 2 distinct species (P. Vallo, personal ongoing investigation). Both are collectively referred to as H. caffer (cf.) ruber in this study. A fraction of 15.4% of H. cf. ruber specimens yielded CoV, without a difference between sexes (14%/19%, n = 57/21 [M/F], respectively). Only adult males and nonlactating adult females, but no lactating females, juveniles, and subadults of H. cf. ruber were encountered.\nVirus Concentrations\nTo estimate the quantity of CoV genomes in bat feces, we did end-point dilution experiments with the nested pan-CoV RT-PCR (18). The previously determined sensitivity limit of the PCR assay was 5&#8211;45 copies/PCR (18). In the assay, the equivalent of 1 mg feces was tested per PCR tube (100 mg feces collected, 1:10 dilution extracted, 1:10 dilution tested). The highest dilution factor that still yielded an amplification signal in any of the samples was 1:10, which suggested a maximal concentration of 50 to 450 CoV RNA copies/mg of feces.\nVirus Classification\n\nGroup 1 CoV\nIn H. cf. ruber bats in the Kwamang and Booyem caves, a diverse group 1 CoV was found. Further analysis was complicated by the low RNA content in samples. Based on alignments of prototype group 1 viruses, 5 different nested RT-PCRs were designed and the RdRp fragment could finally be extended by 441 bp to the 5&#8242; end, providing an 817-nt fragment for phylogenetic analysis. All methods of phylogenetic inference placed this virus next to a common ancestor with human coronavirus 229E, which circulates worldwide in humans (Figure 3). Bootstrap support of the hCoV-229E/GhanaBt-CoVGrpI root point in neighbor-joining analysis was 100%. The corresponding Bayesian posterior probability was 1.0. The most closely related member of the GhanaBt-CoVGrp1 clade shared 91.90% nucleotide identity with hCoV-229E in the analyzed fragment. The most distant member was 86.50% identical. The next phylogenetic neighbor, the human CoV hCoV-NL63, was only 74.70%&#8211;78.60% identical in the analyzed fragment.\nGroup 2 CoV\nWith the pan-CoV screening assay, a group 2 CoV was initially found in the Kwamang cave. Sequences from 3 bats were identical. The secondary group-specific PCR identified 4 additional samples of this virus, 1 of them from Booyem Cave B and the remaining from Kwamang. Nucleotide identity among these sequences was 97.2%&#8211;100%. Phylogenetic analysis with different methods of inference (neighbor-joining nucleotide-based, neighbor-joining amino acid&#8211;based, Bayesian) yielded variable tree topologies suggesting basal associations with either the 2a, 2d, or 2b subgroups (data not shown) (24). Based on alignments of prototype group II viruses, 8 additional nested RT-PCR primer sets were designed and 2 of the samples could be amplified. Sequences could be extended 520 bp upstream and 383 bp downstream of the initial fragments, yielding 1,221-bp fragments for phylogenetic analysis. Bayesian phylogenetic inference with different substitution models and parallel analysis using Metropolis coupling now placed the virus reliably next to a common ancestor with the 2b group of CoV (SARS-like viruses, Figure 3). The Bayesian posterior probability of the CoV 2b/GhanaBt-CoVGrp2 clade being monoyphletic was 1.0. A maximum of 72.2% nucleotide identity was shared with SARS CoV.\nMolecular Dating\nReliable isolation dates were researched in the literature for each employed virus. Because a reliable molecular clock dating existed for the most recent common ancestor (MRCA) of the hCoV-OC43/bovine CoV pair (25), this date was set as a normal-distributed probabilistic prior within the published ranges (25) for calibration of all analyses. A first analysis was conducted on the 1,225-bp dataset that did not include the novel GhanaBt-CoVGrp1. All virus sequences were assumed to be contemporary. Phylogeny was inferred using a GTR + &#915;4 + I model. The resulting MRCA date of the CoV2b (SARS-like)/GhanaBt-CoVGrp2 clade was 260 ad and that of the hCoV-NL63/-229E pair was 981 ad (see Table 2 for details). To include the novel GhanaBat-CoVGrp1, we repeated the same analysis by using the 817-bp dataset. The resulting MRCA date of the hCoV-NL63/229E pair was 816 ad in this analysis, which was in good concordance with results from the 1,221-bp dataset (Table 2) and also with previously published data (26). The diversification estimate for the novel group 1 bat-CoV and hCoV-229E then was 1803 ad.Because it has been suggested that codon-based evolutionary models may be preferred for Bayesian phylogenetic inference from protein-coding datasets (27), analyses on the 817-bp dataset were repeated by using the SRD06 substitution model in BEAST. This analysis did not yield a different substitution rate, but resulted in older resulting MRCA dates (Table 2). A Bayes factor test conducted in TRACER yielded a strong estimate of superiority of the codon-based model over the GTR + &#915;4 + I model (log10 Bayes factor 139 [20 is highly significant]). To further optimize the prediction of MRCA dates, the constant population size assumption used in all analyses was exchanged against expansion growth or exponential growth assumptions. Both assumptions were predicted to fit the data better than the constant size model (Bayes factors 13.5 and 13.9). There was no difference between the expansion and exponential models (Bayes factor 0.34 in favor of expansion). The MRCA date of hCoV-229E and the GhanaBt-CoVGrp1 was 1686 (expansion) or 1800 (exponential growth). Table 2 summarizes the results. Figure 3 shows a dated phylogeny of coronaviruses with MRCAs according to the 2 last mentioned analyses.\nRecombination\nTo determine whether CoV recombination might play a role in the studied virus population, the structural nucleocapsid gene was amplified using 8 nested RT-PCR primer sets that had been designed on alignments of all available CoV group 1 nucleocapsid sequences. Using a similar approach, we also tested the same samples for CoV group 2 nucleocapsid sequences. Only group 1 RT-PCRs yielded fragments. These fragments could be combined into contiguous 1,030-nt sequences for Bt-CoV GhanaKwam 19 and 1,176 nt for Bt-CoV GhanaBoo 344. As shown in Figure 3, panel B, the resulting phylogenetic placement was exactly matching that of the RdRp fragments, giving no evidence of recombination between the RdRp region located in the middle of the genome and the nucleocapsid gene located at the extreme downstream end. Sequencing of the nucleocapsid gene of the GhanaBt-CoVGrp2 was not successful when we used 15 nested RT-PCRs designed on alignments of all available CoV 2b nucleocapsid sequences. Amplification with above mentioned nested RT-PCRs for CoV group 1 was also unsuccessful.\nDiscussion\nIn the aftermath of the SARS epidemic, bats have been identified as carriers of CoV in China (3&#8211;7). Furthermore, in addition to our earlier finding of antibodies against CoVs in various African bats (10), we have confirmed the presence of CoV in bats of Ghana. Together with recent data from Germany, North America, Trinidad, and Kenya (11,12,28), these findings suggest that the association of CoV with bats is a worldwide phenomenon. The prevalence of CoV in insect-eating bats (9.76%) matched our previous findings in Germany. However, in that study we sampled during the breeding season and showed that CoVs are most likely amplified in maternity roosts (11). The composition of the catch in this study (no lactating females, no young bats) suggests sampling outside the breeding season and may not be directly comparable. Future studies relating to risks of exposure should address whether virus prevalence may change over time.The risk of exposure was also addressed by investigations of virus concentration. Several groups have shown that CoVs are almost exclusively detected in bat feces and not, as hypothesized earlier, in saliva (3,4,28,29). Surprisingly, little virus was found in all fecal samples tested in our study. We estimated the RNA concentration per full sample (100 mg feces = 2&#8211;4 fecal pellets) to be only up to 4.5 &#215; 104 RNA copies. Human pathogenic viruses transmitted by the fecal-oral route generate much higher virus concentrations in stool, up to &#8776;1012 RNA copies/mg, e.g., for different picornaviruses (30&#8211;32). Based on these data it would be difficult to postulate that humans can acquire CoV from bat feces. However, studies in other locations and at different times are needed to address virus concentration in bat droppings in more detail. Because virus in this study was only observed in insectivorous bats and not in frugivorous bats, future studies should investigate whether insects might constitute a source of CoV infection for bats.To achieve a direct prediction of the potential of bat CoVs to infect human cells, it would be highly relevant to conduct virus isolation studies on bat feces. However, in our study we sampled no more than 100 mg of feces per bat. All samples had to be collected in RNAlater solution (0.5 mL) (Applied Biosystems) for reasons of storage and transportation. Although it has been suggested that RNAlater solution may preserve virus infectivity (33,34), our observations showed that the solution has to be diluted at least 1:20 in cell culture medium to avoid cytotoxicity (data not shown). Because of the low virus RNA concentrations observed, we did not attempt to isolate the virus. However, the absence of successful virus isolation from bat feces in previous studies (3&#8211;6,8,11,12) may not reflect the incapability of bat CoV to infect human cells. Recently, a synthetic bat CoV complemented with an appropriate spike protein has shown potential to infect human cells (35).Reconstruction of phylogenetic and temporal relationships between bat CoV and other mammalian CoV is another way to obtain information on their zoonotic potential. Unfortunately, for CoV long sequence fragments must be analyzed before valid phylogenies can be inferred from the conserved nonstructural genome portion (28,36). Because of the low concentration of RNA in bat samples, generation of long sequences from novel bat CoV is tedious and technically demanding, which may be why some published phylogenies of bat CoV are based on short datasets, making it difficult to use these data for reference. For molecular clock dating, we have therefore relied on reference viruses mainly from other mammals that covered our 1,221-bp fragment in the conserved RdRp region. We assumed that the RdRp would be under less selective pressure than the structural genes and other nonstructural genes, and therefore could be used to infer nucleotide substitution rates over distantly related CoVs (7,25,26,36&#8211;38). We have confirmed all tree topologies using alternative methods of phylogenetic inference, including an MCMC algorithm implemented in MrBayes that eliminates artifacts contributed by fixation of MCMC chains in suboptimal prosterior probability maxima (20). Calibration was conducted on reliable isolation dates of prototype and novel bat CoV from the literature, as well as on the MRCA of the hCoV-OC43/Bovine CoV clade. For dating of only this specific CoV clade, a wide range of dated virus isolates has been available that covered as much as 34% (1965&#8211;2004) of the projected time of virus evolution from root to tip (1890&#8211;2004) (25). A probabilistic calibration prior was used, which is favorable for dating in combination with relaxed molecular clock assumptions (39). The determined mean substitution rates were in good concordance with earlier studies on non-bat&#8211;CoV that used maximum likelihood-based methods in addition to Bayesian inference (25,26,38,40).Although the exponential growth prior on the virus population seemed equivalent with an expansion growth model by the Bayes factor test and produced highly compatible MRCAs, the exponential model produced a better match with the previously determined MRCA of the HCoV-NL63/HCoV-229E pair (26). Because Pyrc et al. generated these data by 3 alternative approaches (Bayesian, serial unweighted pair group method with arithmetic mean, maximum likelihood [26]), we used their MRCA to validate our results, and consequently prefer the MRCA dating from the exponential growth population model (as presented in Figure 3 in plain type). One earlier study on bat and non-bat CoV suggested a much faster evolutionary rate for CoV than other studies (7). As Vijaykrishna et al. pointed out, their results were associated with large confidence intervals caused by the lack of available data on Bt CoV at the time the study was conducted (7). The increase of available sequence data now enables a better account of CoV evolutionary history.All CoVs in our study were found in members of the genus Hipposideros (family Hipposideridae). The genus Rhinolophus from the sister family Rhinolophidae was found to host SARS-like viruses in several studies in China. One of our Hipposideros CoVs was in a basal phylogenetic relationship with the SARS-like clade (group 2b); their most recent common ancestors date back to &#8776;400 bc. Tong et al. (17) have detected a sequence fragment of a bat CoV in Kenya that also belongs to the 2b clade but is associated with the genus Chaerephon, a free-tailed bat that is rather distantly related to the genus Hipposideros. Although these authors analyzed only a short sequence fragment, their 2b CoV seems to be related more closely to SARS CoV than the virus found in our study. In the many studies conducted in China, only closely related members of the 2b group were detected, with the most basal members dating back only to the 17th century, according to our analysis. The co-occurrence of basal and closely related viruses in Africa, as well as the existence of the same virus clade in bats other than those of the family Hipposideridae, may entail speculations about a possible origin of the SARS-like group of CoVs in Africa rather than in Asia.Another result that should be integrated with earlier findings is the surprisingly recent date of the MRCA of the novel Grp1 Bt CoV and the human common cold virus hCoV-229E. Further to the proven recent host switching of SARS CoV, Vijgen et al. have suggested that hCoV-OC43 entered the human population &#8776;120 years ago, causing a pandemic (25). This virus was most likely acquired by humans from domestic cattle. Results of our study show that it is not unlikely that hCoV-229E, which today is circulating worldwide in humans, resulted from a host switching event not more than 208&#8211;322 years ago. However, as with molecular clock dating of viruses, associated confidence limits should not be overlooked.Because H. cf. ruber bats are found only in sub-Saharan Africa and are not migratory (23), it would be relevant to know how tightly the associated CoV is restricted to its host. Despite the statistical limitations of our rather small sample size, the absence of CoV in bats of the closely related species H. abae that were tested in our study in two different caves speaks in favour of tight host restriction. Another supportive argument is the absence of CoV in C. afra, a bat species sampled in sufficient numbers at the Booyem cave. This cave was coinhabited by CoV-positive H. cf. ruber bats. If tight host restriction to nonmigratory H. cf. ruber bats existed, this would indicate an origin of hCoV-229E within the geographic range of its host, i.e., the rainforest belt and the wet forested savannahs of sub-Saharan Africa (23). Unfortunately, it will be difficult to reconstruct whether the projected host transition event might have been associated with human epidemic disease.",
        "pmid": "19788804",
        "title": "Distant Relatives of Severe Acute Respiratory Syndrome Coronavirus and Close Relatives of Human Coronavirus 229E in Bats, Ghana",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Susanne Pfefferle;Samuel Oppong;Jan Felix Drexler;Florian Gloza-Rausch;Anne Ipsen;Antje Seebens;Marcel A. M\u00fcller;Augustina Annan;Peter Vallo;Yaw Adu-Sarkodie;Thomas F. Kruppa;Christian Drosten",
        "doi": "10.3201/eid1509.090224"
    },
    {
        "keywords": "Coronavirus infections;Middle East respiratory syndrome coronavirus;Coronavirus;Korea;Disease outbreaks",
        "abstract": "Middle East Respiratory Syndrome coronavirus (MERS-CoV) was first isolated from a patient with severe pneumonia in 2012. The 2015 Korea outbreak of MERSCoV involved 186 cases, including 38 fatalities. A total of 83% of transmission events were due to five superspreaders, and 44% of the 186 MERS cases were the patients who had been exposed in nosocomial transmission at 16 hospitals. The epidemic lasted for 2 months and the government quarantined 16,993 individuals for 14 days to control the outbreak. This outbreak provides a unique opportunity to fill the gap in our knowledge of MERS-CoV infection. Therefore, in this paper, we review the literature on epidemiology, virology, clinical features, and prevention of MERS-CoV, which were acquired from the 2015 Korea outbreak of MERSCoV.",
        "body": "INTRODUCTION\nMiddle East respiratory syndrome coronavirus (MERSCoV) was first isolated from a patient with severe pneumonia in September 2012 [1]. Since then, as of 31 January 2018, the World Health Organization (WHO) has been notified of 2,143 laboratory-confirmed cases of MERSCoV infection, including at least 750 deaths, from 27 countries (Fig. 1) [2]. Most of MERS-CoV human infections have occurred in the Arabian Peninsula. Saudi Arabia reported 1,783 cases, including 726 deaths (fatality rate, 40.7%) [3]. A recent article summarized the current knowledge on the epidemiology, virology, human pathogenesis, clinical management, and prevention of MERS-CoV infection [4]. Another comprehensive review article on risk factors and determinants of transmission of MERS-CoV from human to human in the community and hospital settings has been published recently [5]. Public Health England also published an extensive review of literature for clinical decision-making support for treatment of MERS-CoV patients [6].The 2015 Korea outbreak of MERS-CoV is the largest outside the Arabian Peninsula and provides a unique opportunity to fill the gap in our knowledge about MERS-CoV infection. Since it has now been almost 3 years since the 2015 MERS outbreak in the Republic of Korea, we reviewed the literature to summarize the lessons we have learned from the Korea outbreak.\nSEARCH STRATEGY\nPubMed databases were searched for studies on MERS cases during the 2015 outbreak in the Republic of Korea. The search terms used were combinations of &#8216;Middle East respiratory syndrome&#8217; in all fields and &#8216;Korea&#8217; in all fields. As of 31 January 2018, a total of 221 articles were retrieved, and 216 of them were written in English. The full texts of the English articles were reviewed and critically assessed.\nEPIDEMIOLOGY\n\nSummary of the Korea outbreak\nThe 2015 Korea outbreak of MERS-CoV resulted in 186 cases including 38 fatalities [7]. The index case was a returning traveler from the Middle East. The infection had spread within the hospital, and subsequently to other hospitals because of patient movement, resulting in nosocomial transmission at 16 clinics and hospitals. The epidemic lasted for 2 months, with the government declaring a &#8220;virtual&#8221; end to the epidemic on July 6, 2015. In order to control the outbreak, the government quarantined 16,993 individuals for 14 days, and the economic loss was estimated at 9.311 trillion Korean won (8.5 billion US dollars) [8].\nThe first case of MERS\nThe first patient (index case) with MERS-CoV infection was a 68-year-old Korean man returning from the Middle East. He ran a commercial greenhouse business in Bahrain. He flew from Seoul to Bahrain on April 24, 2015, and traveled through the United Arab Emirate and the Kingdom of Saudi Arabia, and came back to Seoul on May 4, 2015. During his 10-day business trip, he did not visit a camel farm or a hospital. He did not eat camel meat or any other camel products. He did not remember meeting any persons with respiratory symptoms [9-12].Seven days after returning home, on May 11, he developed fever and myalgia. He visited a local clinic on May 12, 14, and 15. His symptoms, however, did not improve and a new non-productive cough developed on 15th May. He visited Pyeongtaek St. Mary&#8217;s (PTSM) Hospital (65 km south from Seoul), a secondary hospital in Gyeonggi Province. His chest X-ray showed a consolidation in the upper zone of right lung, and he was admitted to the hospital for pneumonia on May 15. Ceftriaxone and amikacin were started, but his pneumonia progressed despite the antibiotic treatment. He was transferred to the Samsung Medical Center (SMC), a university-affiliated tertiary hospital in Seoul on May 17 [9]. An infectious disease physician suspected MERS pneumonia and requested a laboratory test from the Korea Centers for Disease Control (CDC). MERS-CoV was confirmed on May 20.\nThe chain of transmission/hospital to hospital transmission\nThe chain of transmission was identified in all 186 MERS patients [13,14]. Of the 86 healthcare institutions that had taken in MERS-CoV-infected patients, nosocomial transmission occurred in 12 hospitals and four clinics (plus two ambulances) [13,15,16]. The epidemiological characteristics are summarized in Table 1 [17]. The epidemic curve is shown in Fig. 2.\nThe first wave of the epidemic: outbreak at PTSM Hospital\nDuring his admission from May 15 to 17 at PTSM Hospital, the first patient was unknowingly spreading MERSCoV. Patients, visitors, and healthcare workers (HCWs) were exposed to MERS-CoV. Therefore, by May 20, when the Korea CDC noticed the importation of MERSCoV into the Republic of Korea, individuals staying on the same floor as the first case had already been exposed to MERS-CoV. As a result, a total of 36 MERS cases occurred at PTSM Hospital. Of them, 26 patients developed symptoms within 14 days after exposure, suggesting they were first-generation cases; the remaining 10 patients may be second-generation cases [7,10].The mode of transmission of MERS-CoV at PTSM Hospital remains uncertain. In addition to family members of the first patient, only one patient shared the same room (#8104) with him; all the other cases stayed in other rooms, which were located > 2 m apart from room #8104. All patient rooms were equipped with an air ventilating system except for room #8104. A recent study using a multi-agent modeling framework suggested the transmission occurred via a long-range airborne route or via a combination of a long-range airborne route and direct contact transmission [18]. A chest computed tomography (CT) scan taken in the early stages (day 1 or 2) of illness showed very small, ground glass nodules in the periphery of the lungs, suggesting inhalation of airborne particles [19].\nThe index case for the outbreak at the SMC\nThe index case (superspreader-1) of the outbreak at the SMC was a 35-year-old man who had been exposed to MERS-CoV at PTSM Hospital during May 15 to 17 [19,20]. He developed a fever on May 21. Despite antibiotic treatment, his pneumonia became aggravated and tachypnea developed, so he eventually visited the emergency room (ER) at SMC on May 27. During his 58-hour stay at the ER, he experienced vigorous coughing and watery diarrhea. On May 29, the Korea CDC found he was staying there. A laboratory test confirmed MERS-CoV infection and he was transferred to the Seoul National University Hospital, a government-designated isolation facility on May 30.\nThe second wave of the epidemic: the outbreak at SMC\nMany individuals had been exposed to the superspreader-1 at the ER, and contact tracing identified 675 patients, an estimated 683 visitors, and 218 HCWs as contacts of superspreader-1. Of them, 278 were quarantined and 617 were under active monitoring. As a result of exposure to superspreader-1, 82 had laboratory-confirmed MERSCoV infection, and an additional 10 were secondarily infected from them [20,21].\nThe outbreaks in Daejeon: Daechung & Geonyang Hospitals\nThe index case (superspreader-2) for the Daejeon (100 km south from Pyeontaek city) outbreak was infected at PTSM Hospital. He developed symptoms on May 20 and was admitted to Daechung Hospital on May 22. As his pneumonia progressed despite the treatments, he was transferred to Geonyang Hospital on May 28. A total of 25 secondary cases (14 in Daechung Hospital and 11 in Geonyang Hospital) were detected. Although superspreader-2 caused the two hospital outbreaks, the median time of incubation (8.8 days vs. 4.6 days), secondary attack rates (4.7% vs. 3.0%), and case fatality rate (28.6% vs. 63.6%) were different between the two outbreaks. The high fatality rate was associated with pre-existing pneumonia or poor underlying pulmonary function [22-24].\nOne patient traveled to China\nA 44-year-old male, who had been exposed to the first index case on May 16, developed back pain on May 21. He is a son of the third MERS case. His sister had also been exposed to MERS-CoV and was confirmed to have an MERS-CoV infection on May 25. On May 26, against medical advice, he flew to Hong Kong and subsequently travelled to Huizhou through an entry point in Shenzhen City, Guangdong Province. The Korea CDC notified the WHO of his travel, and the Chinese health authority found him on May 27 and immediately isolated him. He was confirmed to have MERS-CoV infection on May 29. The Chinese authority identified 78 close contacts. After 14 days of quarantine, none of the contacts presented symptoms and all tested negative for MERS-CoV [25,26].\nEpidemiological characteristics\nEpidemiological characteristics have been reported by several groups (Table 2) [7,16,20,21,23,27-30]. The defining epidemiological characteristics were superspreading events at hospitals [30-32]. Early superspreading events generated a disproportionately large number of secondary infections, and the transmission potential decreased sharply in subsequent generations [30,33]. A total of 83% of transmission events were due to five superspreaders, and 44% of the 186 MERS cases were in-patients who had been exposed within the hospitals [7]. The spreaders transmitted MERS-CoV from days 1 to 11 of their illness (median, 7 days), and the number of patients infected by each spreader ranged from 1 to 84 (interquartile range, 1 to 12) [27].The median incubation period and serial interval was 7 days and 12.5 days, respectively [7,34]. R0 ranged from 2.5 to 7.2, higher than the previous estimate of < 1 [35]. It was estimated at 4.04 for the outbreak cluster in PTSM Hospital, and 5.0 for the outbreak in SMC [36]. A mathematical modeling study demonstrated that although R0 < 1 overall, cluster sizes of over 150 cases are not unexpected for MERS-CoV infection [37].Compared with the non-spreaders, the spreaders had a higher frequency of fever (&#8805; 38.5&#176;C) and chest infiltrates in more than three lung zones, and longer non-isolated in-hospital days [27]. The spreaders had higher viral load in sputum samples with the cycle thresholds for the upE and ORF1a genes of 22.7 vs. 27.2 and 23.7 vs. 27.9, respectively. The spreaders with more than three transmissions had higher numbers of contacts and ER visits [38]. A brief exposure of a 10-minute stay and 2 minutes of talking was enough for the transmission of MERS-CoV [39].\nVIROLOGY\n\nCharacterization of the MERS-CoV isolates\nThe whole genome sequences of MERS-CoV isolated from the Korea outbreak were reported [26,40-43]. The virus most similar to the Korean isolates was a camel virus (Camel/Riyadh/Ry159/2015) that belonged to lineage 5, a recombinant of lineage 3 and 4 [44,45]. Viruses from lineage 5 had been predominant in Saudi Arabian camels since November 2014, but human viruses of this lineage were reported from February 2015 [44]. A phylogenetic study of spike glycoprotein genes also showed the Korean strains were closely related to the viral strains from Riyadh, Saudi Arabia [46]. In a molecular study, 11 of 13 MERS-CoV genome had an I529T mutation and one D510G mutation in the receptor binding domain of viral spike protein, resulting in reduced affinity to the human cognate receptor, CD26 [47]. Heterogeneity analysis revealed the combined frequency of I529T and D510G was high (87.7%), although the frequency of both mutations varied greatly among specimens [48].Another genome sequence study also reported deletions of 414 and 419 nucleotides between ORF5 and the E protein, suggesting that the virus might be defective in its ability to package MERS CoV [49]. A microevolution study found no evidence of changes in the evolutionary rate [43]. However, whether the transmissibility and virulence of the Korean isolates are different from those of the previous isolates from the Kingdom of Saudi Arabia remains to be determined by phenotypic assays.\nKinetics of virus shedding\nA viral shedding study showed that the copies of MERSCoV RNA detected by real-time reverse transcription polymerase chain reaction (rRT-PCR) in respiratory samples peaked during week 2, and the median value was 7.21 log10 in the severe group and 5.54 log10 in mild cases [50]. The study also showed that viral titers were higher in throat samples than in nasopharyngeal samples. Another study also demonstrated higher viral loads were associated with severity and mortality [51].\nAntibody response kinetics\nAfter MERS-CoV infection, antibody responses developed by the third week of illness in most patients [52]. Seroconversion rates increased with the increase of disease severity. In a serologic study of 42 MERS patients, the seroconversion rate was 0% in asymptomatic infection, 60.0% in symptomatic infection without pneumonia, 93.8% in pneumonia without respiratory failure, and 100% in pneumonia with respiratory failure [53]. The serological response remained detectable for > 12 months in all survivors (11/11) who had severe disease. Antibody titers were not detectable in four of six patients who had mild pneumonia, suggesting that MERS-CoV seroepidemiological studies may underestimate the extent of mild and asymptomatic infection [54].\nCLINICAL FEATURES\n\nClinical manifestations\nThe Korean Society of Infectious Diseases compiled the clinical data of the 186 MERS patients [55]. The median age was 55 years (range, 16 to 86). The male to female ratio was 3:2. The most common coexisting medical conditions were hypertension (31.7%), diabetes (18.8%), solid organ malignancy (13.4%), and chronic lung disease (10.2%). Patients with MERS-CoV infection developed a spectrum of illnesses, ranging from asymptomatic to fulminant pneumonia with fatal outcome. The respiratory symptoms were similar to other acute viral respiratory infections. At the time of presentation, fever had developed in 81.7%, cough in 56.7%, and sputum in 39.8% of patients. Symptoms of upper respiratory-tract infections were infrequent: sore throat was present in 9.1% and rhinorrhea in 1.6% of patients. Gastrointestinal symptoms were also observed: diarrhea was present in 19.4%, nausea or vomiting in 14.0%, and abdominal pain in 8.1% of patients. Diarrhea may be due to the side effect of lopinavir/ritonavir, as 87% of the patients received antiviral drugs for MERS-CoV treatment [55]. In the early stages of illness, cough and sputum were not prominent even in patients who later developed overt pneumonia [56]. At admission, 68% (123/180) of patients had abnormalities on chest radiographs, while 80.8% (147/182) of them developed the abnormalities during the course of the disease [55]. Sudden progression of pneumonia occurred around day 7 of illness [50,57]. Predictive factors for development of pneumonia included older age, high fever, thrombocytopenia, lymphopenia, C-reactive protein (CRP) &#8805; 2 mg/dL, and a high viral load in sputum (threshold cycle value of rRT-PCR < 28.5) [58]. Forty-five patients (24.5%) were treated with mechanical ventilation and 15 patients (8.2%) needed extracorporeal membrane oxygenation [55]. The typical course of pneumonia is shown in Fig. 3.Acute kidney injury (AKI) developed in 14.0% of the patients, 42.7% (44/103) had proteinuria, and 35.0% (36/103) had hematuria. Fifteen patients were treated with renal replacement therapy [55]. Old age is an independent risk factor for the occurrence of AKI [59]. Neuromuscular manifestations were not uncommon; hypersomnolence, weakness and tingling in the extremities were reported during the treatment of MERS, suggesting Guillain-Barr&#233; syndrome or virus-related sensory neuropathy [60].\nRadiological findings\nSerial changes in chest radiographs was reported in five patients [56]. Chest CT scans revealed rapidly developed multifocal nodular consolidations with ground-glass opacity halo and mixed consolidation, mainly in the dependent and peripheral areas [61].\nLaboratory findings\nThere are few laboratory findings specific to MERS-CoV infection, although monocytosis with normal white blood cell count and low CRP level were more common in MERS patients at initial presentation [57,62].\nLaboratory diagnosis of MERS-CoV\nThe guidelines for the molecular diagnosis of MERSCoV infection have been published by the Korean Society for Laboratory Medicine [63,64]. Specimen type and quality is important in the laboratory diagnosis of MERS-CoV infection. Lower respiratory-tract samples, such as sputum and tracheal aspirates, have higher viral loads than upper respiratory-tract samples [50]. However, MERS patients may not shed the virus during the early stage of their illness. Therefore, initial negative results should not rule out the possibility of MERS [65], and patients suspected of having MERS-CoV infection should be retested using a lower respiratory-tract sample [63,66]. When sputum cannot be obtained, throat swabs may be an alternative source of diagnostic samples [50]. Special attention should be given to diagnosing MERS-CoV infection in immunocompromised patients, as they may present with atypical features, such as a longer incubation period, a longer period from initial PCR positivity to symptom onset, and persistent viral shedding [67].\nAntiviral treatment\nAn antiviral treatment guideline has been published by the writing committee with support from the Korean Society of Infectious Diseases and the Korean Society for Chemotherapy [68]. The guideline recommended a triple combination regimen of type 1 interferon, ribavirin, and lopinavir/ritonavir for 10 to 14 days. However, the guideline was mostly based on expert opinions and further study for the optimal antiviral treatment in MERS is warranted.\nRisk factors for mortality and prognosis\nThe median duration of fever was 8 days. The median time to negative conversion of MERS-CoV in sputum samples by rRT-PCR was 17 days (range, 4 to 45) [55].The median time from symptom onset to death was 14 days. The case fatality rate was 20.4% (38/186) (Fig. 4). The in-hospital mortality, 7-day mortality, and 28-day (from symptom onset) mortality were 19.4% (36/186), 3.8% (7/186), and 17.7% (33/186), respectively [55].Host factors associated with mortality were old age (> 60 years), smoking history, pre-existing pneumonia, abnormal renal function, and comorbid conditions [24,69-71]. Low albumin, altered mentality, and high pneumonia severity index score at admission were risk factors for mortality [69]. MERS-CoV RNA in blood samples was detected in 33% (7/21) of patients at presentation, and it was associated with a worse clinical outcome [72]. A shorter incubation period was also associated with an increased risk of death [24,73].A pregnant woman infected with MERS-CoV in her 35 weeks of gestational age developed dyspnea and her chest radiograph showed diffuse infiltrates in the left lower lung zone. She recovered from the infection and delivered a healthy full-term baby without vertical MERS-CoV transmission [74].A patient developed organizing pneumonia 7 days after the resolution of MERS-CoV infection. He was successfully treated with corticosteroids [75].In a mental health study, 7.6% of 1,656 patients who were quarantined showed symptoms of anxiety, and 6.4% reported feelings of anger during the 2 weeks of quarantine [76]. Mental health support, accurate information, and appropriate supplies, including food, clothes, and accommodation, should be provided to isolated persons [76]. A case of possible transfusion-related acute lung injury following transfusion of convalescent plasma was also reported [77].\nINFECTION CONTROL AND PREVENTION\n\nGuidelines for infection control and prevention\nA comprehensive &#8220;MERS infection prevention and control guideline for healthcare facilities&#8221; was drafted by the guideline development committee with generous support from Korean Academic Societies [78]. The guideline included practical aspects of infection control and prevention based on the experience from the Korea outbreak, such as the composition of members for the MERS emergency committee, a space plan for an anteroom for donning and doffing, isolation of in-patients and HCWs in a hospital affected by an outbreak, and management of the deceased and autopsy [78].The hemodialysis unit may become an epicenter for MERS-CoV outbreak because hemodialysis patients must receive renal replacement therapy even during the outbreak, and the risk of exposure to MERS-CoV continues. During the Korea outbreak, a hemodialysis unit was found to have a MERS patient, and a total of 104 patients and 18 HCWs were exposed to MERS-CoV. With support from the Korean Society of Nephrology, the dialysis unit could continue to operate while the exposed patients were isolated in the unit. Fortunately, no further transmission occurred at the unit [79]. After this successful experience, the Society published a clinical practice guideline for hemodialysis facilities dealing with MERS patients [80].The Korea outbreak was driven by the superspreaders who visited multiple healthcare facilities; thus, generating a large number of secondary cases. Limiting unnecessary contacts with patients with respiratory symptoms in healthcare settings, especially in emergency departments, is of critical importance [33].\nEnvironmental contamination\nMERS-CoV could survive for longer than 48 hours at 20&#176;C and 40% of relative humidity, suggesting contact or fomite transmission might occur in healthcare settings [81]. MERS-CoV was detected by rRT-PCR in specimens taken from the medical equipment [82-84]. MERSCoV was also isolated by cell culture of the air and swab samples taken from a MERS isolation unit, suggesting extensive contamination of the isolation unit [85,86]. Of the 186 cases, 23% were infected by undocumented contact between cases (i.e., indirect transmission of MERS-CoV via environmental contact) [87]. Therefore, fomites with possible MERS-CoV contamination should be sanitized, and a minimum room ventilation rate of six air changes per hour should be implemented to minimize recirculation of pathogen-bearing droplets. Meticulous environmental cleaning may be important for preventing transmission in healthcare settings.\nHealthcare workers and MERS-CoV infection\nThere was a case of a 39-year-old female nurse who was infected with MERS-CoV during a cardiopulmonary resuscitation lasting 1 hour for a MERS patient who had pneumonia and hemoptysis [88]. Serological surveillance conducted after the MERS outbreak for asymptomatic infection among HCWs involved in the direct care of MERS patients showed that 0.3% (2/737) of them were MERS-CoV IgG positive by an indirect immunofluorescent assay. Among the HCWs who did not use appropriate personal protective equipment (PPE), seropositivity was 0.7% (2/294) compared with 0% (0/443) in HCWs with appropriate PPE use [89]. Another serological survey also showed that none of the 285 HCWs were positive for MERS-CoV immunoglobulin G, although 38.2% (109/285) of the HCWs reported experiencing MERS-like symptoms while caring for the MERS patients [90]. In a third study, 0 of 189 HCWs showed seroconversion by a plaque reduction neutralization test, although 20% to 25% of HCWs reported MERS-like symptoms [91].During the outbreak in SMC, all HCWs assigned to MERS patients were screened for MERS-CoV, regardless of the presence or absence of symptoms. Of the 591 HCWs, three (0.5%) asymptomatic HCWs (two nurses and one physician) were found to be MERS-CoV (+), but they did not transmit the virus to others [21]. In another study, an asymptomatic nurse without PPE contacted 82 HCWs, including 33 close contacts who were exposed within 2 m from the index nurse and were not using PPE. None of the exposed HCWs were infected [92].However, the potential for transmission from asymptomatic rRT-PCR positive individuals is still unknown. Therefore, asymptomatic HCWs who are rRT-PCR-positive for MERS-CoV should be isolated and should not return to work until two consecutive respiratory-tract samples test negative on rRT-PCR [93].\nControl of the outbreak by private-public collaboration\nThe government requested the Korean Society of Infectious Diseases to participate in the control of the MERS outbreak. On June 8, 2015, the Society organized the Rapid Response Team, consisting of 15 infectious-disease doctors and two infection-control professionals [94]. Critical suggestions to prevent future epidemics were made regarding a rapid alerting system for index cases, provision of a sufficient airborne infection isolation facility, education and training of HCWs for important infectious diseases, and overcrowding and visitor control in the hospital [95].\nCONCLUSIONS\nThe 2015 Korea outbreak was the largest outbreak outside of the Middle East. Researchers had a unique opportunity to compare the nature of MERS-CoV infection with the Middle East experience. The outbreak unveiled the weak points of infrastructure in our medical system, especially of preparedness for emerging global infectious diseases. Nosocomial transmission was one of the core features of MERS-CoV infection. The introspection for loose medical referral systems, overcrowding at the ER, a lack of expert resources and infection control infrastructure, and lack of organized preparedness for medical crises prompted the government to reform the healthcare system, and healthcare sectors to invest further in infectious diseases and infection control. Although the improvement is still ongoing, the speed and content are still currently insufficient. International cooperation to prepare for and defeat emerging infectious diseases should also be emphasized. Lessons we learned from the outbreak are summarized in Table 3.",
        "pmid": "29506344",
        "title": "Middle East respiratory syndrome: what we learned from the 2015 outbreak in the Republic of Korea",
        "journal_title": "The Korean Journal of Internal Medicine",
        "authors": "Myoung-don Oh;Wan Beom Park;Sang-Won Park;Pyoeng Gyun Choe;Ji Hwan Bang;Kyoung-Ho Song;Eu Suk Kim;Hong Bin Kim;Nam Joong Kim",
        "doi": "10.3904/kjim.2018.031"
    },
    {
        "keywords": "",
        "abstract": "Dromedary camel heavy chain&#8211;only antibodies may provide novel intervention strategies against MERS coronavirus.",
        "body": "INTRODUCTION\nIn 2012, a novel virus, termed Middle East respiratory syndrome coronavirus (MERS-CoV), was identified in humans (1). Six years later, more than 2000 laboratory-confirmed MERS cases, including 36% with a fatal outcome, have been reported globally. Most cases thus far originated from the Arabian Peninsula, as a result of hospital outbreaks (2). There is convincing evidence that dromedary camels are the primary source of MERS-CoV infection in humans. The virus isolated from camels is similar to that isolated from humans and also replicates in human cells (3). In addition, epidemiological and phylogenetic analyses suggest multiple introductions of MERS-CoV into the human population (2, 4). This raises a great concern as MERS-CoV could continue to cause outbreaks in the near future. Effective prophylactic and therapeutic intervention strategies are therefore needed to combat this virus.Monoclonal antibodies (mAbs) are promising candidates for the treatment and prevention of viral infections. Recently, MERS-CoV&#8211;neutralizing mAbs have been identified and characterized by several research groups, using various approaches. These antibodies have been isolated from human na&#239;ve B cells (5), memory B cells of MERS-CoV&#8211;infected individuals (6), or transgenic mice expressing human antibody variable heavy chains (VHHs) and &#954; light chains (7). All these mAbs target the receptor binding domain (RBD) of the MERS-CoV spike protein. The MERS-CoV spike protein is a structural viral component that contains the RBD, located in the S1 subunit of the protein, which binds to the MERS-CoV entry receptor dipeptidyl peptidase-4 (DPP4) (8). Antibodies raised against the S1 or RBD block MERS-CoV infection in vitro (9, 10), and the most potent mAbs identified against MERS-CoV thus far recognize the RBD (5, 7, 11&#8211;14). However, production of these mAbs at a large scale is costly and requires a long developmental process, and relative large quantities might be needed to protect humans against a viral infection (15). Alternatively, antibody engineering technologies allow the cloning of variable regions of mAbs for expression in Escherichia coli or yeast to produce large amounts of recombinant antibody fragments (16). To date, 68 therapeutic mAbs have been licensed, of which 7 are chimeric antibodies (17).Heavy chain&#8211;only antibodies (HCAbs) are naturally produced in camelid species (18). These antibodies are dimeric and do not contain a light chain, and their antigen recognition region is solely formed by the VHH region termed single-domain antibody fragment. This fragment is about 14 kDa in size, is relatively stable, and can be produced with high yields in prokaryotic systems (18, 19). Camelid VHHs have long complementarity-determining region 3 (CDR3) loops, capable of binding to unique epitopes not accessible to conventional antibodies (20). Because of these beneficial properties, VHHs have been exploited for a range of biotechnological applications, including diagnostics, therapeutics, and fundamental research (21, 22). The recent preclinical success of a VHH that blocks von Willebrand factor&#8211;mediated platelet aggregation (23) shows their therapeutic potential. VHHs may also efficiently prevent entry of viruses into host cells (24). Chimeric HCAbs, which have the camel VHH and the human Fc portion (lacking the CH1 exon as in camel HCAbs), allow them to interact with human effector cells and complement cascade factors (15).Several studies have demonstrated the presence of high levels of MERS-CoV&#8211;specific neutralizing antibodies (mean virus neutralization titer &#8776; 1000) in dromedary camels in the Middle East and Africa (25&#8211;27). Therefore, next to human mAbs, characterization of MERS-CoV&#8211;neutralizing VHHs from dromedary camels could serve as an alternative strategy to develop neutralizing antibodies. Here, we report the identification and characterization of neutralizing VHHs against MERS-CoV from immunized dromedary camels and demonstrate the prophylactic activity of camel/human chimeric HCAbs in a MERS-CoV transgenic mouse model.\nRESULTS\n\nIdentification of MERS-CoV\u2013specific VHHs from a dromedary camel bone marrow complementary DNA library\nFirst, we identified MERS-CoV&#8211;neutralizing VHHs by direct cloning and screening of VHH complementary DNA (cDNA) libraries derived from bone marrow cells (given the high frequency of specific plasma cells) rather than using B cells from peripheral blood of immunized animals (Fig. 1). Bone marrow was obtained from two dromedary camels immunized with modified vaccinia virus encoding the MERS-CoV spike protein and subsequently challenged with live MERS-CoV (28). At the time of sampling, MERS-CoV&#8211;neutralizing antibodies were detected at very high levels (titer > 10,000) in the sera of these dromedaries (Fig. 2A). Subsequently, VHH-specific primers (29) were used to amplify a VHH library from the bone marrow cDNA by nested polymerase chain reaction (PCR) (fig. S1, A to C). After gel purification, PCR products were directly cloned into the dephosphorylated prokaryotic expression vector pMES4, tagging the VHHs with six histidine amino acids at the C terminus (29). To obtain a high-diversity repertoire of VHHs, we reduced the number of amplification cycles. The ligated VHH plasmid library was transformed into E. coli strain WK6 and plated on ampicillin nutrient agar plates without preculturing the bacteria in nutrient medium. A total of 560 VHH clones (225 from camel 1 and 335 from camel 2) were obtained in a single transformation event, and grown individually in 96-deep-well plates. Periplasmic expression of recombinant VHHs was induced by isopropyl-&#946;-d-thiogalactopyranoside (Fig. 1). VHHs were purified from the periplasm as a crude extract (30), and expression was verified using SDS&#8211;polyacrylamide gel electrophoresis (PAGE) analysis (fig. S1D).Next, we used formalin-fixed and permeabilized virus-infected cells [either MERS-CoV&#8211;infected or severe acute respiratory syndrome coronavirus (SARS-CoV)&#8211;infected] to select for MERS-CoV&#8211;specific VHHs using immunofluorescent staining. Crude periplasmic VHH extracts were incubated on the infected cells, and VHH cell binding was visualized with a fluorescently labeled anti-histidine antibody as a secondary antibody. All 560 VHH clones were screened by confocal microscopy (Fig. 2B). We obtained 204 MERS-CoV&#8211;reactive VHHs (41.7% from camel 1 and 58.3% from camel 2), of which none reacted to SARS-CoV&#8211;infected cells. To confirm the specificity of the VHHs for MERS-CoV, we randomly selected several clones for double staining of MERS-CoV&#8211;infected cells using a rabbit anti&#8211;MERS-CoV serum, revealing that these VHHs exclusively targeted MERS-CoV&#8211;infected cells (Fig. 2C).\nBlocking of RBD binding to receptor DPP4 by MERS-CoV spike-specific VHHs in vitro\nTo test whether the VHHs identified in our study recognized the RBD or other parts of the S1, we performed MERS-CoV S1&#8211; and MERS-CoV RBD&#8211;specific ELISAs. Out of 204 MERS-CoV&#8211;reactive VHHs, 188 (92.15%) were directed against the MERS-CoV S1 subunit, of which 46 VHHs (22.5%) blocked the binding of recombinant S1 to the MERS-CoV receptor present on Huh-7 cells (fig. S2). All these in vitro blocking VHHs were directed against the RBD (Fig. 2D).Next, we selected all blocking VHHs for further characterization. The VHH clone p2E6 (negative for immunofluorescent staining and S1 ELISA) was used as the negative control. Using a MERS-CoV plaque reduction neutralization test (PRNT), we estimated the virus neutralization capacity for each VHH. Except for the control VHH-p2E6, all tested VHHs inhibited MERS-CoV entry at varying concentrations ranging from 100 to 900 pM (PRNT90; table S1). VHHs with high neutralizing capacity (VHH-1, VHH-4, VHH-83, and VHH-101) were selected for further characterization.We obtained sequences from all 46 blocking VHHs. Because CDR3 is known to be of most importance for the interaction with the antigen, the assumption was made that VHHs with the identical CDR3 would recognize the same epitope. Overall, 33 VHHs had different CDR3 sequences ranging in length from 16 to 20 amino acids (fig. S3). Phylogenetic analysis of these sequences revealed considerable diversity among the different VHH clones and showed that the selected VHHs formed 14 different clusters with different CDR3 sequences (fig. S4). All sequences contained the characteristic VHH tetrad, except clone 10 that, at amino acid positions 37, 45, and 47, shows VH characteristics (valine, leucine, and tryptophan). The best MERS-CoV&#8211;neutralizing VHHs (VHH-1, VHH-4, VHH-83, and VHH-101) had different CDR3 sequences (fig. S4).\nVHHs bind to MERS-CoV spike protein with high affinity\nSubsequently, the best four neutralizing VHHs and the control VHH-p2E6 were selected for large-scale production and purification. We obtained high quantities (5 to 30 mg) of pure (>95%) His-tag affinity-purified VHHs from 1 liter of bacterial culture (fig. S5A). Mixing these VHHs with recombinant MERS-CoV spike S1 protein generated VHH-spike complexes, as observed by nonreducing PAGE analysis (fig. S5B). In addition, the equilibrium dissociation constant (Kd) between the VHH and spike protein of these four VHHs was relatively low, with Kd values ranging from 1 to 0.1 nM, indicating high-affinity binding (fig. S6, A and B).\nNeutralization of MERS-CoV by VHHs and camel/human chimeric HCAbs\nNext, we tested the neutralizing activities of these VHHs in vitro by PRNT. All four VHHs were confirmed to neutralize MERS-CoV with high efficiency, with PRNT50 values ranging from 0.0014 to 0.012 &#956;g/ml (93 to 800 pM), while no inhibition was observed using the control VHH-p2E6 at high concentration (>1.0 &#956;g/ml; 67 &#956;M; Fig. 3A). Because of their small size, VHHs are rapidly cleared from the circulation (30, 31). Therefore, we additionally produced the four VHHs as camel/human chimeric HCAbs by C-terminal tagging the VHHs with the Fc part of human immunoglobulin G2 (IgG2) (containing the hinge and CH2 and CH3 exons) (Fig. 1, right). These HCAbs (HCAb-1, HCAb-4, HCAb-83, and HCAb-101) form homodimers of about 78 kDa in size and exhibit approximately the same neutralizing capacity as the monomeric VHHs in vitro (Fig. 3B). Moreover, using an S1-specific ELISA, we could detect HCAb binding at even lower concentrations, down to 0.00019 &#956;g/ml (2.5 pM; Fig. 3C).\nEpitope mapping of four potent MERS-CoV\u2013neutralizing VHHs\nTo map the VHH binding epitopes, we first tested the binding of the four different VHHs to recombinant S1 protein using ForteBio&#8217;s Octet system. As shown in fig. S7, all four VHHs competed for a single epitope. Subsequently, we used a set of recombinant S1 proteins that contain single amino acid mutations present in spike proteins of MERS-CoV field isolates, located within the receptor binding subdomain (residues 483 to 566) of the RBD that engages DPP4. MERS-CoV polyclonal antibodies, an irrelevant VHH, and four VHHs were then tested for their ability to bind to these S1 variants. MERS-CoV polyclonal antibodies, but not the control VHH-p2E6, bound to all variants (Fig. 4, A and B), whereas the four MERS-CoV&#8211;specific VHHs did not bind to the D539N variant and differed in their binding to the other variants. VHH-1 also did not bind to variant E536K, whereas VHH-4 and VHH-101 showed partial binding to three additional variants (I529T, V534A, and E536K) (Fig. 4, C to F). These data show that all four VHHs bind an epitope in the receptor binding subdomain that is partially overlapping, consistent with the binding competition analysis (fig. S7). The RBD residues D537, D539, Y540, and R542 are important for the virus to bind to its cellular receptor DPP4 (32, 33). Because all four VHHs did not bind to the D539N variant, this suggests that these VHHs neutralize MERS-CoV most likely by blocking its binding to its cellular receptor. Despite several attempts, we were not able to identify MERS-CoV escape variants in vitro. Because of the best neutralizing capacity and epitope recognition, we selected VHH-83 and HCAb-83 for further in vivo testing.\nIn vivo efficacy of VHH-83 and HCAb-83\nTo test the prophylactic efficacy of VHH-83 or HCAb-83 in vivo, we used the K18 transgenic mouse model (34). In our first experiment, mice were given VHH-83 or an irrelevant VHH control (p2E6) at 20 or 200 &#956;g per mouse (nine mice per group) by intraperitoneal injection 6 hours before intranasal infection with a lethal dose of MERS-CoV (EMC isolate). Mice that received VHH-83 lost weight and died within 7 days post-inoculation (dpi), as well as those injected with the control VHH (fig. S8).Next, we tested HCAb-83 or the control HCAb-p2E6 using a similar experimental setup. Mice treated with 200 &#956;g of HCAb-83 gained weight (Fig. 5A), and all mice survived (Fig. 5B). In contrast, control HCAb-p2E6&#8211;treated groups lost weight and died within 7 dpi (Fig. 5, A and B). Gross pathological changes (Fig. 5C), mononuclear cell infiltration, and alveolar edema (Fig. 5E) were observed in the lungs of control HCAb-p2E6&#8211;treated mice on day 4 after inoculation. Whereas low doses (20 &#956;g) of HCAb-83&#8211;treated mice were only partially protected on the basis of the observed reduction of pathological abnormalities on 4 dpi (Fig. 5F), the lungs of high-dose HCAb-83&#8211;treated mice showed no pathological changes at any time point tested (Fig. 5G). In addition, no infectious virus could be isolated from the lungs of these mice, while high viral titers were observed in the low dose&#8211; and control HCAb&#8211;treated mice (Fig. 5H).\nPharmacokinetics of HCAb-83\nWe also evaluated the pharmacokinetics of VHH and HCAb in the sera of mice treated with either VHH-83 or HCAb-83. First, we estimated the presence of MERS-CoV&#8211;neutralizing activity in sera obtained 2 days after treatment. No neutralization of the virus was observed in the sera of VHH-83&#8211; or control VHH-p2E6&#8211;treated mice (Fig. 6A), consistent with the reported rapid clearance of small VHH domains from the circulation (31). Significant levels of neutralizing antibodies (mean titer, 1024) were observed in the sera of mice treated with 200 &#956;g of HCAb-83 and, to a limited extent, in low dose&#8211;treated mice (mean titer, 64; Fig. 6A). Second, we tested the presence of circulating HCAb-83 in the sera obtained at various time points after injection (0, 2, 4, and 8 days after treatment) by ELISA. As shown in Fig. 6B, 200 &#956;g of HCAb-83&#8211;treated mice still had high levels of HCAb-83 8 days after treatment, with an apparent serum half-life of approximately 4.5 days.\nDISCUSSION\nVHHs are small in size; have high stability, solubility, and affinity; and efficiently recognize antigens. They have many potential biomedical applications including the treatment of cancer, autoimmune diseases, and virus infections (18, 19, 22, 24, 31). Moreover, VHHs are gaining much attention in the field of diagnostics and therapeutics for viral diseases. They have been used for the detection of viruses, such as Marburg virus, human immunodeficiency virus (HIV), influenza virus, dengue virus, and norovirus (22, 24). VHHs also block virus attachment to the host cells in respiratory syncytial virus, influenza virus, hepatitis B virus, rotavirus, and HIV infections (22, 24). Some VHHs inhibit viral RNA transcription or nuclear import of viral ribonucleoproteins (35). Here, we have shown that MERS-CoV&#8211;neutralizing VHHs can be obtained from immunized dromedary camels that were challenged with MERS-CoV. The engineered camel/human chimeric HCAbs were highly stable in mice, and prophylactically treated mice were fully protected from MERS-CoV infection upon challenge with live virus.Naturally infected dromedary camels have remarkably high levels of neutralizing antibodies against MERS-CoV (25, 36). We used dromedaries that showed high levels of neutralizing antibodies in their sera and identified MERS-CoV&#8211;neutralizing VHHs by direct cloning from a VHH cDNA library using bone marrow, a major source of highly enriched long-lived antibody-producing plasma cells. After immunization, antigen-stimulated B cells undergo affinity maturation in germinal centers of secondary lymphoid organs, where they differentiate into plasma cells that secrete antibodies. Significant portion of long-lived plasma cells migrate to the bone marrow. A small portion of plasma cells reside in the lymphoid organs, but these are often short-lived (37, 38). In mice, 8 days after boost immunization with ovalbumin, about 10 to 20% of the antigen-specific plasma cells migrate from secondary lymphoid organs to the bone marrow (39). In particular, bone marrow plasma cells are long-lived and are thus suitable for maintaining antibody levels in the serum for an extended period, which plays a significant role in pathogen neutralization and humoral immune responses (40). The number of S1-specific clones found in the VHH library generated from vaccinated and infected camels (188 of 560 clones; 33.5%) was much higher compared to nonvaccinated infected camels (12 of 496 clones; 2.4%), suggesting that the vaccination and challenge protocol used in this study had a major impact on the frequency of S1-specific B cells detected in the bone marrow.Camelid species have naturally occurring HCAbs. These antibodies contain long CDR3 sequences (20), which allow them to interact with unique and even recessed epitopes that may not be recognized by conventional antibodies (20, 41). We identified 46 MERS-CoV&#8211;neutralizing VHHs, of which 4 bound to the RBD of the spike protein with high affinity and neutralized MERS-CoV infection at picomolar concentrations. VHH-83 showed a neutralizing capacity down to a concentration of 30 pM (PRNT50), making it more potent than the most potent mAbs described thus far (7, 42). However, direct comparisons between different antibodies would require determining the exact differences in vitro. Competition and spike protein binding assays showed that the four VHHs competed for binding to an overlapping epitope on the RBD, which partially overlaps with the RBD-DPP4 interface. Binding assays using variant recombinant spike proteins revealed that all four VHHs bound to wild-type spike protein but not to a D539N-mutant protein. Amino acids E536, D537, and D539 are negatively charged residues on the surface of the RBD, which interact with three positively charged residues on the outer surface of the DPP4 (32, 33). This indicates that the four VHHs can prevent virus attachment and entry. Given the critical role of these amino acids in the DPP4-virus interaction, viral escape mutants without loss of fitness are less likely to develop (32, 33). This could be the reason why we did not identify HCAb-83 escape variants in vitro. However, further (structural) studies are needed to pinpoint all RBD-VHH contact residues involved.Next, we produced the four VHHs as HCAbs, which showed threefold enhanced MERS-CoV&#8211;neutralizing capacity in comparison to the monomeric VHHs in vitro (PRNT50, 30 pM). In contrast to VHH-83, mice prophylactically treated with 200 &#956;g of HCAb-83 were fully protected from weight loss and death upon challenge with live virus. No infectious virus was detected in the lungs of these mice, and protection correlated with the presence of sustained high levels of HCAbs-83 in the sera of mice. In addition, most in vivo studies testing mAbs to MERS-CoV showed only reduced MERS-CoV replication (two to four log reductions in lung virus titer) or complete protection only at higher doses used (1000 &#956;g per mouse) (6, 7, 13). The high level of neutralizing activity of HCAb-83 (PRNT50, 30 pM) could be due to the different antigen recognition pattern of camelid HCAbs. Recent studies also revealed the importance of long CDR3 sequences from bovine antibodies raised against HIV in cross-neutralization against different viral serotypes (43). The therapeutic efficacy of HCAb-83 still needs further evaluation, but given the limited therapeutic efficacy of other mAbs against acute respiratory infections such as respiratory syncytial virus in humans (44), prophylactic administration of antibodies may also be preferred to contain outbreaks of MERS-CoV.Apart from direct neutralization, antibodies may also play a role in mediating effector functions such as complement-dependent cytotoxicity and antibody-dependent cell-mediated cytotoxicity (16). HCAb-83 has an Fc domain of human IgG2, which has limited effector function in vivo (17), suggesting that the observed protection in mice could be mainly due to the neutralizing activity. Therefore, additional studies need to evaluate whether the potency of HCAb-83 may be increased further by replacing the IgG2 Fc with the IgG1 Fc or by combination with other antibodies targeting different epitopes.In summary, we identified and characterized potent HCAbs that neutralized MERS-CoV in vitro and in vivo. Because of their high affinity, in vivo stability, and efficacy, these HCAbs may be used as a prophylaxis for MERS-CoV.\nMATERIALS AND METHODS\n\nImmunization\nOne female (6-month-old) and one male (8-month-old) healthy dromedary camels (Camelus dromedarius), negative for antibodies against MERS-CoV and modified vaccinia virus Ankara (MVA), were obtained from the Canary Islands and housed in biosecurity level 3 (BSL-3) facilities [Centre de Recerca en Sanitat Animal (CReSA)], as described previously (28). Experimental procedures were approved by the local Ethics Committee of the Autonomous University of Barcelona (number 8003). Both animals were immunized twice with a 4-week interval with 108 plaque-forming units of MVA-S via both nostrils and intramuscularly in the neck of the animals. After the second immunization, both animals were anesthetized with midazolam (5 mg/ml) and inoculated with 107 TCID50 MERS-CoV in 3 ml of phosphate-buffered saline (PBS) intranasally in both nostrils using a laryngo-tracheal mucosal atomization device (45). Blood samples were taken at different dpi. On day 14 after inoculation, both animals were anesthetized and femoral bone marrow samples (about 1 cm3) were collected next to the epiphysis and placed, each, in tubes containing 3.6 ml of ice-cold fetal calf serum. Specimens were gently crushed with a 1-ml tip and homogenized by slow up and down pipetting. After 10-min incubation on ice, 400 &#956;l of dimethyl sulfoxide was mixed into each tube, and the preparation was dispensed into 2-ml cryovials discarding debris and slurs and stored at &#8722;135&#176;C.\nProtein expression\nThe recombinant S1-Fc fusion proteins were produced as described previously (46). Briefly, plasmids encoding MERS-CoV S1-Fc or MERS-CoV RBD-Fc were generated by ligating a fragment encoding the S1 subunit (GenBank accession number AFS88936; residues 1 to 747) or RBD (residues 358 to 588) 3&#8242; terminally to a fragment encoding the Fc domain of human IgG1 into the pCAGGS expression vector. Plasmids encoding S1-Fc variants with single amino acid substitutions were generated by site-directed mutagenesis. S1-Fc fusion proteins were expressed by transfection of the expression plasmids into human embryonic kidney (HEK)&#8211;293T (CRL-11268, American Type Culture Collection) cells and affinity-purified from the culture supernatant using Protein A-Sepharose beads (GE Healthcare).\nEstimation of antibody/VHH titers\nMERS-CoV&#8211;specific antibody titers were measured by ELISA. First, 96-well plates were coated with MERS-CoV S1 or MERS-CoV RBD proteins at 1 &#956;g/ml in PBS (pH 7.4) and incubated overnight at 4&#176;C. Wells were then washed three times with PBS, blocked with 10% normal goat serum in PBS, and incubated at 37&#176;C for 30 min. Dromedary camel sera or VHHs were serially diluted in PBS, 100 &#956;l was added per well, and plates were incubated at 37&#176;C for 1 hour. Next, plates were washed three times in PBS containing 0.05% Tween 20 (PBST), after which they were incubated with biotin-conjugated goat anti-llama antibodies (1:2000, Abcore) or mouse anti-histidine antibodies (1:2000, Thermo Fisher Scientific) at 37&#176;C for 1 hour. After three washes with PBST, plates were incubated with streptavidin horseradish peroxidase (HRP; 1:10,000, Dako) or goat anti-mouse HRP (1:2000, Dako) at 37&#176;C for 1 hour. After this incubation, plates were washed three times in PBST and incubated at room temperature for 10 min in the presence of 3,3&#8242;,5,5&#8242;-tetramethylbenzidine substrate (eBioscience). Reactions were stopped with 2N H2SO4 (Sigma). The absorbance of each sample was read at 450 nm with an ELISA reader (Tecan Infinite F200).\nRNA isolation and cDNA synthesis\nFor RNA isolation, cryopreserved bone marrow cells were removed from the &#8722;135&#176;C freezer and transferred to a 37&#176;C water bath. The thawed cell suspension was quickly transferred to 40 ml of ice-cold RPMI 1640 (Lonza) medium. Cells were counted, and 107 cells were transferred to a new ribonuclease (RNase)&#8211;free falcon tube and centrifuged at room temperature at 300g for 10 min. The supernatant was completely removed, and cells were subsequently lysed with 1 ml of TRIzol reagent (Life Technologies) and 0.2 ml of RNase-free chloroform (Life Technologies). The mixture was vortexed for 15 s and incubated for 3 min at room temperature, followed by centrifugation at 13,000 rpm for 15 min at 4&#176;C. The aqueous phase was transferred to a new tube. Subsequently, RNA was isolated using the RNeasy MinElute Cleanup Kit (Qiagen) according to the manufacturer&#8217;s protocol. Total RNA was quantified at 260 nm using the NanoDrop 2000, and the quality of the isolated RNA sample was determined by measuring the A260/A280 ratio. cDNA was synthesized from 1.5 &#956;g of total RNA using a First-Strand cDNA Synthesis kit (Life Technologies). For a 20-&#956;l reaction mix, 10 &#956;l of RNA, 1 &#956;l of deoxynucleoside triphosphate (dNTPs; 10 mM each), 1 &#956;l of random hexamers (10 mM; Promega), and 1.5 &#956;l of distilled water (dH2O) were added to a microvial. The mixture was incubated at 65&#176;C for 10 min and then at 4&#176;C for 2 min. Next, 6.5 &#956;l of reverse transcriptase mix containing 4 &#956;l of 5&#215; SuperScript III reaction buffer, 1 &#956;l of dithiothreitol (100 mM), 0.5 &#956;l of RNase inhibitor (20 U/&#956;l), and 1 &#956;l of SuperScript III reverse transcriptase (200 U/&#956;l; Life Technologies) were added to the microvial and incubated at 25&#176;C for 5 min, 50&#176;C for 45 min, and 70&#176;C for 20 min. cDNA was stored at 4&#176;C until PCR amplification.\nPCR amplification and cloning of VHH\nThe amplification of VHH was performed using a nested PCR approach (29) that was adapted for use with a high-fidelity DNA polymerase (PfuUltra II Fusion HS DNA Polymerase, Stratagene). The first PCR mix (50 &#956;l of reaction volume) consisted of 5.0 &#956;l of 10&#215; PfuUltra II Fusion HS DNA polymerase buffer, 2.5 &#956;l of dNTPs (10 mM each), 1.5 &#956;l of gene-specific forward primer (CALL001, 5&#8242;-GTCCTGGCTGCTCTTCTACAAGG-3&#8242;; 10 mM), 1.5 &#956;l of gene-specific reverse primer (CALL002, 5&#8242;-GGTACGTGCTGTTGAACTGTTCC-3&#8242;; 10 mM), 1.0 &#956;l of PfuUltra II Fusion HS DNA polymerase, 36.5 &#956;l of dH2O, and 2.0 &#956;l of cDNA. PCR amplification was performed in a thermocycler with the following protocol: initial denaturation at 94&#176;C for 3 min, followed by 20 cycles at 94&#176;C for 20 s, 50&#176;C for 30 s, and 72&#176;C for 2 min, and a final extension at 72&#176;C for 10 min. The first PCR generated two amplified products: the heavy chain of conventional antibodies (~1000 bp) and the VHH heavy chain (~700 bp; fig. S1). The amplified VHH amplicon (~700 bp) was purified from the agarose gel using the QIAquick Gel Extraction Kit (Qiagen), according to the manufacturer&#8217;s instructions. The product was then subjected to the second round of PCR amplification using VHH inner primers that contained restriction sites for cloning: forward, 5&#8242;-CTAGTGCGGCCGCTGGAGACGGTGACCTGGGT-3&#8242; (Eco 91I); reverse, 5&#8242;-GATGTGCAGCTGCAGGAGTCTGGRGGAGG-3&#8242; (Pst I). PCR amplification continued for 12 to 14 cycles, after which the amplicons were purified with the QIAquick PCR Purification Kit (Qiagen) and digested using Eco 91I and Pst I. The vector pMES4 (GenBank accession number GQ907248) was digested with the same enzymes and dephosphorylated using alkaline phosphatase (New England Biolabs). VHH amplicons were ligated into pMES4 using a ratio of 100 ng of vector to 46 ng of VHH (~1:3 molar ratio). Next, we used E. coli strain WK6, prepared using the Mix & Go! E. coli Transformation Kit and Buffer Set (Zymo Research), for transformation according to the manufacturer&#8217;s instructions. After transformation, cells were directly plated onto an ampicillin (100 &#956;g/ml) nutrient agar plates. The following day, the insertion of VHH into the vector was confirmed by randomly picking 25 clones, screening by PCR for the insert, and Sanger sequencing, as described below.\nSequencing\nTo sequence the inserts, colony PCR was performed using PfuUltra II Fusion HS DNA Polymerase (Agilent Technologies) and primers (29) 5&#8242;-TTATGCTTCCGGCTCGTATG-3&#8242; (MP57) and 5&#8242;-CCACAGACAGCCCTCATAG-3&#8242; (GIII) under the following conditions: initial denaturation at 95&#176;C for 3 min, followed by 39 cycles of (95&#176;C for 20 s, 55&#176;C for 30 s, and 72&#176;C for 40 s), and a final extension at 72&#176;C for 5 min. The amplicons were gel-purified and sequenced in both directions using the BigDye Terminator v3.1 Cycle Sequencing Kit and an ABI PRISM 3100 genetic analyzer (Applied Biosystems). The obtained sequences were assembled and aligned using CLC Genomics Workbench (CLC Bio 4.9).\nSupplementary Material\n",
        "pmid": "30101189",
        "title": "Chimeric camel/human heavy-chain antibodies protect against MERS-CoV infection",
        "journal_title": "Science Advances",
        "authors": "V. Stalin Raj;Nisreen M. A. Okba;Javier Gutierrez-Alvarez;Dubravka Drabek;Brenda van Dieren;W. Widagdo;Mart M. Lamers;Ivy Widjaja;Raul Fernandez-Delgado;Isabel Sola;Albert Bensaid;Marion P. Koopmans;Joaquim Segal\u00e9s;Albert D. M. E. Osterhaus;Berend Jan Bosch;Luis Enjuanes;Bart L. Haagmans",
        "doi": "10.1126/sciadv.aas9667"
    },
    {
        "keywords": "",
        "abstract": "Background\nA recent publication reported that a tyrosine-dependent sorting signal, present in cytoplasmic tail of the spike protein of most coronaviruses, mediates the intracellular retention of the spike protein. This motif is missing from the spike protein of the severe acute respiratory syndrome-coronavirus (SARS-CoV), resulting in high level of surface expression of the spike protein when it is expressed on its own in vitro.Presentation of the hypothesis\nIt has been shown that the severe acute respiratory syndrome-coronavirus genome contains open reading frames that encode for proteins with no homologue in other coronaviruses. One of them is the 3a protein, which is expressed during infection in vitro and in vivo. The 3a protein, which contains a tyrosine-dependent sorting signal in its cytoplasmic domain, is expressed on the cell surface and can undergo internalization. In addition, 3a can bind to the spike protein and through this interaction, it may be able to cause the spike protein to become internalized, resulting in a decrease in its surface expression.Testing the hypothesis\nThe effects of 3a on the internalization of cell surface spike protein can be examined biochemically and the significance of the interplay between these two viral proteins during viral infection can be studied using reverse genetics methodology.Implication of the hypothesis\nIf this hypothesis is proven, it will indicate that the severe acute respiratory syndrome-coronavirus modulates the surface expression of the spike protein via a different mechanism from other coronaviruses. The interaction between 3a and S, which are expressed from separate subgenomic RNA, would be important for controlling the trafficking properties of S. The cell surface expression of S in infected cells significantly impacts viral assembly, viral spread and viral pathogenesis. Modulation by this unique pathway could confer certain advantages during the replication of the severe acute respiratory syndrome-coronavirus.",
        "body": "Background\nThe recent severe acute respiratory syndrome (SARS) epidemic, which affected over 30 countries, resulted in more than 8000 cases of infection and more than 800 fatalities (World Health Organization, ). A novel coronavirus was identified as the aetiological agent of SARS [1]. Analysis of the nucleotide sequence of this novel SARS coronavirus (SARS-CoV) showed that the viral genome is nearly 30 kb in length and contains 14 potential open reading frames (ORFs) [2-4]. These viral proteins can be broadly classified into 3 groups; (i) the replicase 1a/1b gene products which are important for viral replication, (ii) the structural proteins, spike (S), nucleocapsid (N), membrane (M) and envelope (E), which have homologues in all known coronaviruses, and are important for viral assembly, and (iii) the \"accessory\" proteins that are specifically encoded by SARS-CoV. Much progress have been made in characterizing these SARS-CoV proteins [5,6], but the molecular determinant for the severe clinical manifestations of SARS-CoV infection in contrast to the mild diseases caused by most coronaviruses, remains to be determined. In addition, the exact roles of \"accessory\" proteins of SARS-CoV are still poorly understood.The subject of this hypothesis relate to the S protein and one of the \"accessory\" proteins, the SARS-CoV 3a protein. The S protein, which forms morphologically characteristic projections on the virion surface, mediates binding to cellular receptor and the fusion of viral and host membranes, both of these processes being critical for virus entry into host cells [7,8]. As such, S is known to be responsible for inducing host immune responses and virus neutralization by antibodies [9,10]. 3a (also termed ORF3 in [2] and [11], as X1 in [3], and as U274 in [12,13]) is the largest \"accessory\" protein of SARS-CoV, consisting of 274 amino acids and 3 putative transmembrane domains. Three groups independently reported the expression of 3a in SARS-CoV infected cells [13-15] and it was also detected in a SARS-CoV infected patient's lung specimen [14]. Antibodies against 3a were also found in convalescent patients [11,12,14].This article hypotheses that the endocytotic properties of 3a allow it to modulate the surface expression of S and explores a functional significance for the interaction between S and 3a, which has been observed experimentally [13,15].\nPresentation of the hypothesis\nThe cellular fate of the S protein has been well mapped [16,17]: S is cotranslationally glycosylated and oligomerized at the endoplasmic reticulum. Its N-linked high mannose side chains are trimmed, modified and become endoglycosidase H-resistant during the transportation to the Golgi apparatus. Only this fully-matured form of S can be assembled into virions and/or transported to the cell surface. The latter could cause cell-cell fusion and the formation of syncytia. Recently, Schwegmann-Wessels and co-worker reported that a novel sorting signal for intracellular localization is present in the S protein of most coronaviruses, but absent from SARS-CoV S [18]. Site-directed mutagenesis studies confirmed that a Yxx&#934; motif (where x is any amino acid and &#934; is an amino acid with a bulky hydrophobic side chain) retains the S protein of TGEV intracellularly when it is expressed alone. On the other hand, SARS-CoV S is transported efficiently to the cell surface unless such a motif is introduced into its cytoplasmic tail by mutagenesis.The Yxx&#934; motif has been implicated in directing protein localization to various intracellular compartments [19-21]. Furthermore, most Yxx&#934; motifs are capable of mediating rapid internalization from the plasma membrane into the endosomes. Interaction between the adaptor protein complex 2 (AP-2) with the Yxx&#934; motif present in the cytoplasmic domain of the internalizing protein concentrated the protein in clathrin-coated vesicle, which then budded from the plasma membrane resulting in internalization. However, it appears that the Yxx&#934; motif can also bind other adaptor protein complexes, like AP-1, 3 and 4, and the differential binding to the different adaptors will determine the pathway of a cargo protein containing a particular Yxx&#934; motif [21]. Coincidently, a Yxx&#934; motif in the cytoplasmic domain of 3a has previously been identified [13]. Furthermore, the juxtaposition of the Yxx&#934; motif and a ExD (diacidic) motif was found to be essential for the transport of 3a to the cell surface, consistent with the role of these motifs in the transportation of other proteins to the plasma membrane [22]. 3a on the cell surface can also undergo internalization [13].Analyzing the experimental results present in these publications collectively, it is possible to postulate a functional role for the evolution of the SARS-CoV 3a protein. The SARS-CoV S protein lacks the Yxx&#934; motif but it can bind to the 3a protein which has internalization properties. In SARS-CoV infected cells, S is rapidly transported to the cell surface. But if 3a is expressed in the same cell, it is also transported to the cell surface where it can bind S. The interaction between 3a and S enables both proteins to become internalized, resulting in a decrease in the expression of S on the cell surface. Thus, this viral-viral interaction confers the functional role for the Yxx&#934; motif found in other coronaviruses to the SARS-CoV S. This hypothesis also implies that the precise mechanisms used by TGEV and SARS-CoV to reduce the expression of S are different although in both cases, the Yxx&#934; motifs will be crucial. In TGEV, the Yxx&#934; motif in S caused it to be retained intracellularly, while in SARS-CoV, S that is transported to the cell surface becomes internalized again after it interacts with the 3a protein.\nTesting the hypothesis\nUsing mammalian cell culture system and biochemical methods, it will be possible to determine the exact effects of 3a on the trafficking properties of S. Mutagenesis studies can be used to map the protein domains that are important for the interaction between 3a and S and for the defining the manner by which 3a contributes to the reduction of cell surface expression of S. Given that a full-length infectious clone of SARS-CoV has been assembled [23], the use of reverse genetics would certainly reveal more about the interplay between 3a and S during SARS-CoV infection.\nImplication of the hypothesis\nThis hypothesis, if proven, will indicate that the interaction between SARS-CoV-unique 3a protein and S results in a reduction of S on the cell surface through the endocytotic properties of 3a [13]. During SARS-CoV infection, expression of S on the cell surface of an infected cell mediates fusion with un-infected neighboring cells, leading to syncytium formation. It follows that reducing the cell surface expression of S will delay this cell-damaging effect and prevent the premature release of unassembled viral RNA. It may also enhance virus packaging as it appears that the assembly of coronavirus occurs intracellularly, probably in the intermediate compartments between the endoplasmic reticulum and Golgi apparatus [24]. Clearly, this has certain advantages for the virus at certain stages of its life cycle. In addition, a reduction in the cell surface expression of S may also help the infected cell evade the host defense system and reduce the production of anti-S neutralizing antibodies. Conversely, host or viral factors that disrupt the interaction between S and 3a would favor the expression of S on the cell surface and enhance cell-cell fusion, a process that is important for viral spreading.Table 1 shows a comparison of the amino acid sequences of the cytoplasmic tails of the S protein of different coronaviruses, including SARS-CoV, which is distantly related to the established group 2 coronaviruses [25], as well as two recently identified novel human coronaviruses, HCoV-NL63 [26] and HCoV-HKU1 [27]. The Yxx&#934; motifs are clearly present in all group 1 coronaviruses and also in IBV, which belongs to group 3. However, no Yxx&#934; motif is present in SARS-CoV and MHV, both group 2 coronaviruses. In addition, there is a YGGR motif in the S protein of RtCoV and YxxH motifs in the S proteins of the other group 2 coronaviruses, BCoV, HEV and HCoV-HKU1. However, these motifs may not be able to function as signaling motifs because both R and H are not hydrophobic amino-acids. Therefore, HCoV-OC43 is the only one of these group 2 coronaviruses that encodes a S protein with a Yxx&#934; motif. It is still unclear how the localization of S is modulated in those viruses that lack Yxx&#934; motifs in the S proteins and further studies will be needed to understand the different signaling pathways that are important for regulating the trafficking properties of S. Indeed, the dilysine endoplasmic reticulum retrieval signal, which is a different type of sorting signal from the Yxx&#934; motif, in the cytoplasmic tail of IBV was reported to be important for intracellular retention of S [28].It therefore appears that the cell surface expression of S protein of SARS-CoV can be reduced like that for other coronaviruses, but the mechanism may be different. The trafficking of SARS-CoV S may be mediated through 2 separate viral proteins, expressed from separate subgenomic RNA, and regulated by numerous complex cellular processes including the efficiency of transcription and translation, post-translation modification and stability of the viral proteins, as well as their interactions with host factors. Indeed, it is crucial to determine how this unique pathway benefits replication of the SARS-CoV. It is also interesting to note that sequence comparison of isolates from different clusters of infection showed that both S and 3a showed a positive selection during virus evolution [29,30], implying that these proteins play important roles in the virus life cycle and/or disease development and is consistent with the proposal that 3a has evolved to modulate the trafficking properties of the spike protein.\nCompeting interests\nThe author(s) declare that they have no competing interests.\nAuthor's contributions\nYee-Joo Tan is responsible for the entire manuscript.",
        "pmid": "15703085",
        "title": "The Severe Acute Respiratory Syndrome (SARS)-coronavirus 3a protein may function as a modulator of the trafficking properties of the spike protein",
        "authors": "Yee-Joo Tan",
        "doi": "10.1186/1743-422X-2-5"
    },
    {
        "keywords": "coronaviruses;epigenetics;systems biology",
        "abstract": "Coronaviruses (CoV) comprise a large group of emerging human and animal pathogens, including the highly pathogenic severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) strains. The molecular mechanisms regulating emerging coronavirus pathogenesis are complex and include virus&#8211;host interactions associated with entry, replication, egress and innate immune control. Epigenetics research investigates the genetic and non-genetic factors that regulate phenotypic variation, usually caused by external and environmental factors that alter host expression patterns and performance without any change in the underlying genotype. Epigenetic modifications, such as histone modifications, DNA methylation, chromatin remodeling, and non-coding RNAs, function as important regulators that remodel host chromatin, altering host expression patterns and networks in a highly flexible manner. For most of the past two and a half decades, research has focused on the molecular mechanisms by which RNA viruses antagonize the signaling and sensing components that regulate induction of the host innate immune and antiviral defense programs upon infection. More recently, a growing body of evidence supports the hypothesis that viruses, even lytic RNA viruses that replicate in the cytoplasm, have developed intricate, highly evolved, and well-coordinated processes that are designed to regulate the host epigenome, and control host innate immune antiviral defense processes, thereby promoting robust virus replication and pathogenesis. In this article, we discuss the strategies that are used to evaluate the mechanisms by which viruses regulate the host epigenome, especially focusing on highly pathogenic respiratory RNA virus infections as a model. By combining measures of epigenome reorganization with RNA and proteomic datasets, we articulate a spatial-temporal data integration approach to identify regulatory genomic clusters and regions that play a crucial role in the host&#8217;s innate immune response, thereby defining a new viral antagonism mechanism following emerging coronavirus infection.",
        "body": "1. Coronaviruses\nThe severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002/2003, most likely in the Guangdong Province, China. From the initial outbreak, SARS-CoV rapidly spread across the globe causing 8000 infections and ~800 deaths in 28 countries; mortality rates approached 50% in aged individuals [1,2,3]. From its animal reservoir in Chinese horseshoe bats (genus Rhinolophus), the SARS-CoV was thought to have adapted to Palm civets and raccoon dogs in open markets, before finally colonizing human populations [4]. More recent studies have shown that SARS-CoV, as well as a large reservoir of SARS-like bat CoV (SL-CoV) have the ability to efficiently utilize the human angiotensin-converting enzyme 2 (ACE2) receptor for docking and entry and replicate efficiently in primary human airway epithelial cells. These data document the presence of a large animal reservoir of prepandemic SARS-like bat CoV that supports the possibility of direct bat-to-human transmission and recurrent outbreaks in the future [5,6,7].In 2012, the antigenically distinct Middle East respiratory syndrome coronavirus (MERS-CoV) emerged and it is continuing to cause an ongoing outbreak in Saudi Arabia, the Arabian Peninsula and eastern Africa. Currently, 1626 infections in 26 countries have been reported with a mortality rate of ~30% [8]. Most of the cases are connected to the Arabian Peninsula, where camels, and perhaps bats, have been identified as the natural reservoir. While animal-to-human transmission has been responsible for a majority of cases, human-to-human transmission has also been described in hospital settings and in the home, most notably in May 2015 in South Korea, where a MERS-CoV-infected individual returned from Bahrain, causing more than 170 cases by human-to-human transmission [9]. Both pathogens cause severe lower respiratory tract infections, with the most severely impacted individuals developing acute respiratory distress syndrome (ARDS), a severe end-stage lung disease with poor treatment options and high fatality rates. Although asymptomatic infections were rare during the SARS-CoV epidemic, MERS-CoV infections frequently result in asymptomatic infections leading to asymptomatic spread. In general, the molecular mechanisms governing virus pathogenesis and disease severity remain understudied and vaccines and therapeutics are still not available [9,10].Coronaviruses are enveloped RNA viruses, containing the largest currently-known single-stranded, positive-sense RNA genome, which ranges in length from 25.5 to 32 kb. The viral particles range from 70 to 120 nm and are surrounded by &#8216;spike&#8217;-shaped glycoproteins, which give the viruses their characteristic &#8216;corona-like&#8217; appearance in the electron microscope. Coronaviruses encode for 7 to 14 open reading frames (ORFs). ORF1 compromises approximately two thirds of the genome at the 5&#8242;-end and consists of two overlapping ORFs, ORF1a and ORF1b. Both ORFs are translated into large polyproteins, the precursor of at least 16 nonstructural proteins (nsps) that encode the viral replication machinery, and other important functions in virus&#8211;host interaction, like innate immune antagonism and pathogenesis. By ribosomal frameshifting, ORF1 is expressed as two polyproteins designated ORF1a and ORF2b, which are processed by a papain-like protease (PLpro) and the viral main protease (3CLpro), into at least 16 proteins. Unique among RNA viruses, coronaviruses encode a proof-reading complex, consisting of nsp10/nsp14, nsp12 (replicase) and perhaps nsp16, which regulates fidelity [11]. Downstream of ORF1, the genome also encodes for four structural proteins, the S (spike), E (envelope), M (matrix), and N (nucleoprotein) protein. These genes are interspersed with several additional luxury ORFs, which differ significantly among coronavirus in number, nucleotide sequence, gene order, and function; for SARS-CoV, most of these additional ORFs are indispensable for viral replication but many have been shown to antagonize the innate immune response and influence disease severity (Figure 1) [12]. \n2. Innate Immunity and Coronavirus Infections\nInnate immunity is one of the earliest barriers to coronavirus infection. Following infection, pathogen recognition receptors (PRRs) such as retinoic acid-inducible gene I (RIG-I), melanoma differentiation-associated gene 5 (MDA5), Toll-like receptors (e.g., TLR 3, 4 and 7) and other sensing molecules recognize pathogen-associated molecular patterns (PAMPs) in viral components, such as viral structure proteins or viral nucleic acid. Successful recognition initiates a signaling cascade that activates an antiviral state in the host. Several main players of innate immunity, such as signal transducer and activator of transcription 1 (STAT1), myeloid differentiation primary response gene 88 (MyD88), TLR4, TLR7 and TLR3/TIR-domain-containing adapter-inducing interferon-&#946; (TRIF), function to dampen infection severity during coronavirus infection in vivo [13,14,15,16,17]. Similarly, interferons (IFN&#8722;alpha (&#945;) and IFN-beta (&#946;), IFN-gamma (&#947;)) also play critical roles in controlling SARS-CoV in vivo and in vitro [12,13,14,15,16,18,19]. Data from the 2002/2003 outbreak have demonstrated that differential IFN and interferon-stimulated gene (ISG) expression levels in patients correlated with SARS disease outcomes. Several mouse models for SARS-CoV pathogenesis confirmed protective roles of MyD88, TLRs and select ISGs [12,14,16,20].Like other viral pathogens, coronaviruses such as SARS-CoV and MERS-CoV have evolved genetic functions that delay and/or antagonize pathogen recognition as well as ISG effector functions. SARS-CoV encodes several proteins that modulate innate immune signaling through the antagonism of the induction of interferon. As mentioned above, several nsps that are encoded by ORF1 (ORF1a/b), like nsp1, nsp3 papain-like protease, nsp14 and nsp16 antagonize various sensing or signaling programs and NF&#954;&#946;, or function to cap viral messenger RNAs (mRNAs) and evade interferon-induced protein with tetratricopeptide repeats (IFIT) 1-3 ISGs [11,21,22,23,24,25] (Figure 1). These nsps show high homology to proteins of other human coronaviruses and are critical for efficient viral replication. Several downstream open reading frames like ORF3a, ORF3b and ORF6 also antagonize sensing or signaling pathways, or block karyopherin 2 nuclear import [13,21] (Figure 1). MERS-CoV also encodes several luxury functions with interferon antagonism activities, including ORF4a, ORF4b and perhaps ORF5, noting that ORF4b antagonizes phosphodiesterase activity and RNAse L activation [26,27,28,29,30]. However, how the exact underlying mechanisms allow these antagonistic molecules to interfere with the effector molecules that establish an antiviral state, assist in wound repair, or prime and enhance an adaptive immune response, which is critical for clearance, is still under study. Recent studies have suggested that RNA viruses like coronaviruses and influenza viruses are able to manipulate the host&#8217;s epigenome, potentially heralding entirely new mechanisms of viral antagonism and new targets for therapeutic intervention and control [18,31]. The purpose of this review is to summarize the available methodology to study the epigenetic mechanisms that allow study of the epigenome during infection with coronavirus and other RNA viruses. \n3. Epigenetics\nEpigenetic regulation bridges genotype and phenotype by changing the function of the gene locus without changing the sequence of the underlying DNA. Over the last decade, research efforts have revealed a dynamic range of epigenetic factors that shape and regulate chromatin status, leading to changes in host gene expression patterns, and therefore to alterations in phenotypes. Epigenetic modifications are significant in regulating cellular mechanisms and pathways during embryonic development, in memory function, in immunity and in disease [32,33]. While mutations directly affect the genetic material by changing the genetic code, epigenetic modifications change the chromatin structure or modify the nucleic acid without altering the genetic code. This makes epigenetic modifications reversible, flexible, and quickly responsive to changes in the environment and other exposures. Based on this ability, the study of epigenetic modifications is an important interface between the environment and the genome [32]. Over the last decade, epigenetics research has made rapid progress in understanding developmental biology, memory, and inheritability functions. More recently, it has become increasingly important in studies of oncology, adaptive and innate immunity, and infectious diseases [34,35]. It is becoming well established that many DNA viruses, and to some lesser extent RNA viruses, have evolved functions that antagonize the regulatory machine of the host epigenome, leading to regulated changes in host gene expression that lead to a favorable environment for virus replication and spread [36]. Over the last 20 years, the development of many biochemical and in particular high-throughput approaches have revolutionized our understanding of chromatin biology and function. Chromatin biology is now at the point where studies can be performed that use its tools to discover and validate new players and pathways in epigenetics and their role in a variety of biological disciplines, including developmental biology, oncology and infectious diseases, like bacterial and viral infections.The human genome project (HGP) was officially completed in 2003, providing the research community with a detailed map of the genetic organization and structure of the human genome as well as the epigenome [37]. Another benefit of the human genome project was the development of next-generation sequencing (NGS) technology. Epigenetics research adopted the NGS techniques early on by refining methods like ChIP-Seq, RNA-Seq, and MeDIP-Seq [38,39]. Today, these are routine methods for the investigation of genome-wide changes in DNA methylation, histone modification, and DNA&#8211;protein interactions. Similarly, in the field of infectious disease research, high-throughput DNA analyses have enabled the genome-wide examination of epigenetic modifications and DNA methylation, providing systematic, large-scale association testing with disease phenotypes. It is likely that many common diseases, cancers, and infectious disease outcomes in humans are mediated by genetic and environmental factors. Likewise, epigenome-wide association studies (EWAS) provide a systematic identification of genome-wide epigenetic variants associated with disease outcomes [40]. EWAS can collect information about variation of epigenetic markers, global epigenetic patterns, and genome-wide distribution of epigenetic markers which can provide functional correlation with genotypes and phenotypes associated with particular pathological or non-pathological outcomes, defining new disease-associated marks [40,41]. In particular, the Encyclopedia of DNA Elements (ENCODE) project has advanced our understanding of the principles of genome, epigenome and chromatin organization, discovering and identifying formerly unknown histone modifications, nucleosome positions, and chromosome-wide maps of regulatory chromatin structures [38,42,43]. GENCODE, a part of ENCODE, now contains an extensive catalogued transcript, and pseudogene and long noncoding RNA (lncRNA) resources, helping to develop and to identify histone modifications and variants from several combinatorial patterns that define active promoters/TSSs (transcription start sites), transcribed gene bodies, inactive regions, and enhancers [44,45]. Several techniques described below are now commonly used in studies that integrate different data types including transcriptomics, proteomics, and epigenomics [46]. These allow us to validate and discover new molecular pathways that could lead to new discoveries in developmental biology, memory, and disease.\n4. Chromatin\nThe genetic information in eukaryotic cells is encoded in the chromosomes and mitochondrial DNA. Chromosomes exist in deoxyribonucleoprotein complexes called chromatin. Chromatin is found in two variations: the euchromatin and the heterochromatin, which were originally distinguished cytogenetically by Giemsa staining procedures. Darker staining heterochromatin indicates tightly packaged protein and nucleic acid complexes found at centromers and telomers. These contain mostly inactive satellite DNA as opposed to the lighter-stained loosely-packed euchromatin, which defines genome regions under active transcription and gene expression [47]. Chromatin organization is complex and composed of a specialized set of proteins&#8212;the histones (H)&#8212;that organize the DNA into the nucleosome. The nucleosome is composed of a tightly-packed histone octamer consisting of the core histones H3, H4, H2A, and H2B with roughly 165 base pairs of DNA wrapped around it, much like beads on a string. This structure maintains stability and most importantly, protein and transcription factor accessibility to the DNA genome, allowing the chromatin to guarantee tight packaging of the genomic DNA, accurate replication, and distribution into the daughter cells during cell division, as well as transcriptional regulation of gene expression [48].The histone&#8217;s N-termini, the so-called histone tails, extend from the globular protein unit and as such are targets for post-translational modifications. At this time, several chemical modifications have been identified and characterized: lysine acetylation, lysine and arginine methylation, serine and threonine phosphorylation, and lysine ubiquitination and sumoylation. These modifications are found on H2A, H2B, H3, and H4 histone subunits [48,49]. In 2001, Jenuwein and colleagues described the histone code, hypothesizing that a coding mechanism within the chromatin structure is regulated by chemical modifications to the histone tail, a concept which is now well supported by the literature [50]. We now know that distinct modifications of the histone tails interact with different sets of chromatin-associated proteins (Table 1). As a result, modifications on the same or different histone tails may be interdependent and generate various combinations on any one nucleosome, thereby supporting the modification-induced recruitment of chromatin-associated proteins. Consequently, the specificity of the downstream information is guaranteed and a specific crosstalk between histone modifications is possible.The last decade has shown that this regulatory instance of nucleosomes and chromatin structure on the genome has emerged as a critically important determinant of cellular transcription, replication, and differentiation state.\n5. Epigenetic Regulation of Gene Expression\n\n5.1. Histone Modifications and Transcription\nThe histone tails of the nucleosome are subject to post-translational modulations. These modulations are covalently attached to the tails and include methylation of arginine residues, and methylation, acetylation, ubiquitination, phosphorylation, and sumoylation of serines (S) and threonines (T) (Table 1, Figure 2A). Over the last decade, many modifications have been associated with active or non-active transcription. Modifications like the acetylation (ac) of H3 and H4, as well as the di- or tri-methylation (me) of H3 on lysine 4 (H3K4), are associated with an active transcription state. On the other hand, methylation of H3K9 and H3K27 are now associated with transcriptional repression of the particular gene [52]. Based on their function within gene expression the particular histone marks can be found in distinct localizations within a gene region [53].Likewise, many modifications are almost uniquely associated with gene organization components, like promoters, TSSs, enhancers or gene bodies, slicing sites, and transcriptional end sites (TES). These histone marks help to organize the chromatin by modulating accessibility, thereby defining regulatory regions and elements, like promoters, enhancers and insulators, within the genome [54]. Some of the histone mark distributions are uniquely associated with the transcription rates of particular genes. As mentioned above, main histone marks for regulating the TSS are H3K4me3 and H3K27me3; both modifications are exclusively found at the TSS and in the appropriate promoter region of the particular gene [55]. H3K4me3 is the main modification for an active promoter region and therefore actively transcribed chromatin, while H3K27me3 on the other hand, is the main modification found at repressed promoters [56]. Despite these differences, both modifications perform a crucial function in bivalent or &#8216;poised&#8217; promoters [53]. A promoter occupied with both H3K4me3 and H3K27me3 can be rapidly activated or inactivated for transcription, making both histone modifications signature configurations for poising bivalent promoters for alternate fates: active and repressed gene transcription [51,57,58]. Other histone marks are preferentially located in enhancer regions or within the gene body. For example, H3K27ac and H3K4me1 are enriched at active enhancer sequences, active promoters are flanked by H3K27ac and H3K4me3, and gene bodies show enrichment of H3K36me3. The state of the chromatin is modulated by a large number of proteins which can be seen as &#8216;writers&#8217;, &#8216;readers&#8217;, and &#8216;erasers&#8217; [59]. &#8216;Writers&#8217; are responsible for encrypting the information capacity of nucleosomes by adding distinct post-translational modifications to the histone tails. Generally, &#8216;writers&#8217; are acetylases, methylases, and phosphorylases that specifically add the appropriate modification to the histone tails. &#8216;Erasers&#8217; antagonize the function of the &#8216;writers&#8217;, and remove the histone modifications. These enzymes include deacetylases, demethylases, and phosphatases. &#8216;Writers&#8217; and &#8216;erasers&#8217; modulate the assembly, placement, recognition, and modification. The recognition of histone modifications is mediated by &#8216;readers&#8217;; proteins which are tightly regulated by phosphorylation and dephosphorylation through signaling pathways, recruitment and binding of co-factors, like transcription factors and adaptor proteins [59]. Overall, histone modification patterns are dynamic and reflect the activation state of a gene, the elongation state and the splicing patterns of the pre-mRNA transcript. To match the complexity of the modification patterns, &#8216;readers&#8217; are often organized in protein complexes, containing a bromodomain, chromodomain, and tudor domains and harboring several putative modification-dependent binding sites. Many of the histone modifying enzymes belong to complex protein superfamilies that show stringent substrate, catalytic, and tissue specificity. This way, these proteins regulate DNA accessibility together with ATP-dependent chromatin remodeling complexes, which are mediating remodeling of nucleosomes, like moving, ejecting or restructuring nucleosomes. This mediates pioneer transcription factors, that are involved in recruiting transcription machinery complexes, like the RNA polymerase II (Pol II) complex, or insulator proteins, to bind at nucleosome-free DNA regions and initiate gene expression. This guarantees a specific modification and response to environmental stimuli [60].Based on the theory of the histone code, the same histone mark can have very different physiological outcomes depending on the location in the chromatin, the neighboring modifications, and the combination of modifications [50]. In other words, to understand the function of a single histone mark, the combination and the co-occurrence with other marks needs be considered. This complexity determines that a combination of multiple histone modifications can have a cascading effect with a variety of different outcomes ranging from transcription repression to transcription activation to transcription termination. These varying outcomes regulate combinatory and sequentially downstream functions, generating distinct signatures for every individual gene [50,61].This circumstance influences strongly the nature of the modification-binding proteins, the so-called readers and the following down-stream processes. This way, a rather small set of separate histone modifications results in a broad range of different outcomes for the cell. Overall, it is not surprising that these chromatin-modifying enzymes play an important role in maintaining chromatin structure and dynamics. It is important to note, however, that chromatin marks can be easily reversed. As a consequence, they can rapidly respond to external stimuli, thereby regulating the accessibility of the underlying DNA to the transcriptional machinery and ensuring the correct association of expressed genes in the appropriate situation [62,63]. \n5.2. DNA Methylation and De-Novo Methylation\nIn all mammalian cells, DNA methylation takes place post DNA replication. It occurs at the 5&#8242; position of the cytosine ring within CpG nucleotides by adding a methyl group to create 5-methylcytosine (5mC). The modification is mediated by a family of enzymes, the DNA methyltransferases (DNMTs) (Figure 2B). DNMT3a and DNMT3b have been described as de novo methyltransferases, preferentially targeting unmethylated CpG islands (CGIs) in the genome to initiate DNA methylation [64]. Studies have shown that DNMT1 functions as maintenance methyltrasferase, ensuring that the methylation status is maintained during DNA replication and following cell division [65]. However, DNA methylation as an epigenetic marker is highly dynamic, and therefore crucial in gene silencing and gene regulation, the establishment of heterochromatin, and in regulating the stability of the chromosome [66]. Hypermethylation of repetitive DNA sequences in combination with certain histone marks results in the condensation of chromatin and therefore in the establishment of heterochromatin [67]. Recently, DNA hydroxymethylation (5hmC) has been identified as another form of DNA methylation. Several studies have shown that enzymes of the ten-eleven translocation (TET)-family catalyze the modification and that it has a major role in embryonic neuronal development. [68,69].The presence of 5mC CGIs plays a critical role in regulation of gene expression. More than 60% of coding genes contain CGIs in promoter-associated regions. These CGIs are generally unmethylated, and therefore easily accessible to transcription factors and other chromatin-associated proteins for the expression of most housekeeping genes and other regulated genes [67]. However, de novo methylation of those promoter-associated CGIs will repress and silence promoter activity. Transcriptional inactivity at a methylated promoter region can be reversed by methyltransferases, rendering the DNA sequence into active chromatin, therefore demonstrating another instance of gene regulation [51,64,67,70].\n5.3. Non-Coding RNAs and micro RNAs\nStarting in the early 2000s, the field of non-coding RNAs (ncRNAs) evolved from its historic origins as &#8220;junk RNA&#8221; and quickly expanded into its own field of research. Based on their function and their genetic origin ncRNAs can be divided into long non-coding RNAs (lncRNAs) and small non-coding RNAs (sncRNAs), based on whether each RNA is greater than or less than 200 bp in length [71]. For many years, lncRNAs were considered to be unimportant junk byproducts of evolution and were ignored by most of the research community. However, this group of RNAs is now recognized as a critical regulator in chromatin remodeling, transcriptional regulation, and post-transcriptional processing [72]. Epigenetics and microRNAs (miRNAs) regulate whole gene expression patterns transcriptionally and post-transcriptionally, respectively (Figure 2C) [73]. At the same time, epigenetics and miRNAs control each other to form a regulatory circuit and to maintain normal physiological functions [74]. Several miRNAs have been identified that target genes that control epigenetic pathways, like DNMTs and histone methyltransferases (HMTs), thus controlling chromatin structure by regulating by regulating histone modifier molecules. The expression of miRNAs on the other hand is regulated by histone modification and DNA methylation, forming an epigenetics&#8211;miRNA regulatory circuit [73].\n6. Methods to Study Epigenetics\nA number of high throughput technologies have been developed to study the epigenetic landscape and epigenetic modifications genome-wide and on sequence-specific levels (Figure 3). \n6.1. Modifications of Histones and Localizations of Histone Marks within the Genome\n\n6.1.1. Formaldehyde-Assisted Isolation of Regulatory Elements (FAIRE)\nAs a broad strategy to identify modifications of histones and the localization of histone marks across the genome, FAIRE was developed and has been applied to understand the chromatin status of target cells and of DNA viruses under different conditions of infection [75]. FAIRE is a method to isolate regulatory elements from eukaryote chromatin, thereby taking advantage of the fact that DNA segments that actively regulate transcription in vivo are typically characterized by eviction of nucleosomes. The FAIRE method involves crosslinking the chromatin by adding formaldehyde, which preferentially targets heavily-condensed, transcriptionally-repressed chromatin over transcriptionally-active chromatin. The crosslinked chromatin is then sheared by sonication; phenol-chloroform is added to separate protein (nucleosome-depleted) DNA fragments from nucleosome-covered DNA. Downstream detection methods include microarrays, NGS, or quantitative PCR. The regions isolated and detected by FAIRE are largely coincident with the location of open chromatin, such as DNase hypersensitive sites, TSS, enhancers, and actively-transcribed promoters [51,76,77,78].\n6.1.2. Chromatin Immuno-Precipitation (ChIP) \nThis technique is used to determine whether a given protein binds to, or is localized to, a specific DNA sequence in vivo. Cross-linked chromatin is sheared and the DNA-binding protein of interest is precipitated by using a protein-specific antibody. The bound DNA is then isolated by reverting the cross-linking and can be analyzed by utilizing microarrays (ChIP-on-chip), next-generation sequencing (ChIP-Seq), and quantitative PCR (ChIP-PCR) [79,80]. The method is strictly dependent on the availability of high quality antibodies to the target protein. The availability of antibodies and the quality of the antibody used in the ChIP determines the quality of the data generated by the study. In general, only antibodies with high sensitivity and specificity should be considered for use, because this will allow the detection of enrichment peaks without substantial background noise [81].The combination of the ChIP technology with next-generation sequencing allows and improves the characterization of binding sites for transcription factors and other DNA-binding proteins and the identification and characterization of DNA sequence motifs across the entire genome. The advancement in high resolution is crucial in profiling nucleosome positioning, the systematic cataloguing of histone modification patterns, and the establishment of precise histone modification maps throughout the entire genome [51,80,82]. \n6.2. Whole-Genome Methylation Status\nTo quantify the global distribution of active and inactive states of chromatin across the genome, several methods and technologies have been developed to measure the methylation status across the genomic DNA [83]. Recently, the methylated DNA immunoprecipitation (MeDIP) technique was developed and has proven to be a versatile, unbiased approach to study the methylation status of either the whole genome or specific regions of interest. In brief, genomic DNA is sheared and precipitated with a monoclonal antibody that recognizes 5-methylcytidine. Another approach based on immune precipitation is the Methyl-DNA binding protein ChIP using the Methyl-CpG-binding domain protein 2 (MBD2), a member of the MBD protein family. The resulting enrichment of methylated DNA can be determined by PCR to assess the methylation state of CpG islands in individual promoters or gene regions of interest. Alternatively, precipitated methylated DNA can be combined with large-scale analysis using microarrays or next-generation-sequencing [84,85].There are other complementary approaches to study the genome-wide methylation status of chromatin based on methylation arrays and methylation-sensitive and methylation-insensitive restriction enzymes. The methylation array technology is based on the Infinium MethylationEPIC technology (formerly the Infinium Human Methylation 450 array) which allows low sample input and fast read-out but has the disadvantage of not covering all annotated genes and shows bias which is based on the array technology. Another approach uses restriction enzymes, like HpaII/MspI, which are blocked or not blocked by CpG methylation. After treatment of the total DNA with the enzymes the distribution and extent of DNA methylation can be analyzed by quantitative PCR targeting regions of interest. However, the enzymatic approach is prone to bias based on the sequence specificity of the utilized restriction enzymes. This limits the analysis to certain sequence 3 motifs, which can be unevenly distributed across the whole genome [86]. \n7. Immune System and Genetics\nBased on functional and spatial patterns, the immune system is broadly divided into two broad arms: the innate immune system and the adaptive immune system. Both systems include a wide range of cell types that communicate via direct cell&#8211;cell interactions or by the secretion of mediators such as interleukins, cytokines, and chemokines. The innate immune system not only regulates cell intrinsic defense programs in response to microbial attack but also has a critical role in activating and shaping the adaptive immune response. The innate immune system accomplishes this by being able to generate and drive a transcriptional response that is both cell- and stimulus-specific. Based on these mechanisms, the signal-specific induced response guarantees initiation of the appropriate innate and adaptive immune responses that have the greatest potential to successfully control a particular pathogen [87,88,89]. Much of the innate immune response is regulated by membrane-bound and intracellular PAMPs, like TLRs, RIG-I, MDA-5 and cyclic GMP-AMP synthase (cGAS)- stimulator of interferon genes (STING) and other sensors that detect invading pathogens [90,91]. These PAMPs use unique and overlapping signaling cascades to activate effector transcriptional programs that regulate antimicrobial defense pathways. Most of the research has focused on elucidating the exact signaling programs that regulate antimicrobial defense to different pathogens and the microbial countermeasures that inactive specific pathways [92,93]. More recently, a growing body of evidence has determined that chromatin modifications and epigenetic regulation play a crucial role in shaping the activated host response to a microbial invasion [35,94,95]. Advances in sequencing technologies have significantly increased our ability to sensitively and specifically measure the transcriptional state at a single-cell level. Systems biology approaches have revealed the more complex gene interaction networks that become activated or repressed. These mechanisms have been essential in understanding the functional specialization of cells as individual units of the innate immune system, the flexibility in mounting innate and inflammatory immune response, and in deciphering the mechanism of communication and interactions within specific cell populations.\n8. Epigenetic Regulation/Modulation of Host Response\nA basic feature of innate immune cells is the ability to start a transcriptional response program that is specific to the stimulus, and then mounting a signal with a high degree of cell type and stimulus specificity [96,97,98].Recent studies have involved epigenetic factors in every aspect of activation and shaping innate and adaptive immune responses. Major contributions are the:\n&#10132;Recruitment of transcription factors/machinery;&#10132;Prevention of unwanted expression of potent mediators; and&#10132;Repression or activation of secondary gene programs [98,99].The main players of the innate immune system are primary response genes like IFN and tumor necrosis factor (TNF), which are rapidly induced and whose promoters show the characteristics of a poised promoter. Often, the promoters of these genes also contain CpG islands which are resistant to epigenetic modifications like DNA methylation and histone tail modification. These common modifications can be found at promoters of highly active transcribed genes, which also show high levels of RNA Pol II occupancy [99]. To the contrary, ISGs usually display low levels of activating histone marks like H3K4me3, H4Ac, and low level RNA Pol II occupancy [100]. These genes often require additional transcription factors and chromatin remodelers, like recruitment of the ATP-dependent chromatin remodeling complex SWItch/sucrose non-fermentable (SWI/SNF) to initiate transcription [36,97]. Two cell types of the innate immune system, dendritic cells and macrophages, are the primary sensors of &#8216;danger&#8217; signals. Once these cells are activated, it is especially important that their signals are both cell-specific and stimulus-specific to ensure the initiation of a temporal and spatial response. These cell-specific signals can be mediated through cell&#8211;cell contact or by secretion of IFN and TNF. Thus, the ability of their epigenome to change within minutes after a stimulus is not just essential for initiating a rapid antiviral host response but is also essential to ensure a persistent and specific defense response. This way, epigenetic mechanisms are responsible for the priming and the memory of these responses and for guaranteeing a functional and highly regulated host response beyond the initial activation wave.Besides the histone marks for poised promoters H3K4me3 and H3K27me3, an important role as a major player in regulating the activation of IFN has been described for H3K9me2. Fang et al. correlated the levels of H3K9me2 modification with the level of interferon expression in vitro. H3K9me2 is a repressive histone mark that contributes to DNA methylation and heterochromatin formation and thereby prohibits histone tail acetylation by recruiting the transcriptional repressor of the heterochromatin protein 1 family [101]. However, in the above study, Fang et al. were able to demonstrate that the overall levels of H3K9me2 mark in the promoter region of the type I interferon and the expression of ISGs inversely correlates in dendritic cells, defining this histone modification as an important regulator of the IFN response [101,102].On the other hand, H3K4me3 is a histone modification exclusively found at active promoters and is therefore often enriched in promoter regions regulating TLRs. A recent study has shown that 60 min after lipopolysaccharide (LPS) stimulation of macrophages and dendritic cells, the overall histone acetylation and the binding of polymerase II (Pol II) at the specific promoters was tremendously increased, demonstrating an efficient and specific induction of the innate immune response by epigenetic control mechanisms [103]. \n9. Coronaviruses/Influenza Viruses: Viral Antagonism of Host Gene Expression by Altering Histone Modifications\nInterferons are important mediators of an antiviral state and initiators of pathogen-driven immune response by the inactivation of ISGs [104,105]. Therefore, it is likely that many viruses have evolved antagonistic mechanisms to overcome specific ISG effectors [106]. As discussed earlier, IFN and innate immune responses are subject to extensive epigenetic regulation, mediated by specific epigenetic marks, and the manipulation of histone modification enzymes, DNA methylases, and chromatin remodeling complexes. Viruses have evolved mechanisms to disturb and antagonize these epigenetic regulatory programs by (1) interfering with the host&#8217;s histone modification enzymes [107]; (2) interfering with the host&#8217;s chromatin remodeling machinery [108]; and (3) encoding for viral proteins that interact directly with the host&#8217;s modified histones [98,109]. Marazzi et al. have demonstrated that the highly pathogenic H3N2 influenza A virus inhibits the initiation of the host innate immune response in part by interfering with the epigenetic control of gene expression. Using histone mimicry, it has been proposed that the carboxyterminus of the H3N2 nonstructural protein NS1 shares homologue sequences with the aminoterminus of the histone H3 tail [31]. Essentially, the viral NS1 protein mimics the histone tail of the H3 histone and thereby interacts with the transcription complex, which usually docks to the H3K4 mark to initiate transcription [31,110]. Previously, we compared ISG profiles of pathogenic influenza viruses and coronaviruses in Calu3 cells, a human airway epithelial cell line, by using a transcriptomics and proteomics dataset [18]. The infection of Calu3 cells with the tested respiratory viruses resulted in diverse virus-specific ISG expression signatures. The highly pathogenic H5N1 avian influenza A (HPIA) virus showed a rapid manipulation of ISG with strong down- and up-regulation of specific ISG sets at 7 h post infection. In contrast, the 2009 pandemic H1N1 strain showed no ISG modulation, and the infected cells mounted a robust IFN-induced antiviral state starting at 3 h post infection. SARS-CoV infection of Calu3 cells also showed a strong induction of ISG effectors, but the response was significantly delayed with peak expression at 24 to 48 h post infection. In 2012, the newly emerged MERS-CoV showed delayed ISG production with effects visible at 18-h post infection (hpi), with significant inhibition of expression of specific ISG subsets. Overall, the viral manipulation of the host antiviral IFN response results in successful virus infection and viral replication, defining a viral antagonistic approach, which may be a mechanism to interfere with the host&#8217;s innate immune response [22]. In our laboratory, by using ChIP-PCR approaches, we could determine differential occupancy of histone marks at the promoters of ISG genes. We showed that the promoter regions of ISG genes contained more histones with active marks of H3K4me than the repressive H3K27me3 mark, therefore favoring open chromatin and promoting active transcription and ISG expression during H1N1-2009 and SARS-CoV infection. In contrast, infection of Calu3 with the highly pathogenic HPAI and MERS-CoV resulted in increased levels of H3K27me3 and decreased levels of H3K4me3 occupancy at the promoter regions of subsets of specific ISGs, which were not induced, demonstrating that these viruses have developed antagonistic mechanisms to specifically target the IFN arm of the innate immunity (Figure 4). We have expanded this dataset by using a genome-wide ChIP-Seq approach. This allows us to choose any set of cellular effector molecules and to study their histone modification profile during infection. Figure 5 shows as an example the expression profile of ISGs during the early phase MERS-CoV infection. As already described by Menachery et al., expression of ISGs effectors occurred rather late after infection at 12 hpi to 18 hpi. However, MERS-CoV infection in Calu3 induces the up-regulation and down-regulation of ISGs. We then applied ChIP-Seq data to the expression data. When we looked at the expression level of ISG transcripts at 18 hpi, we could see how the histone modification promoter profile at 12 hpi corresponded with the transcriptomic data. ISGs that were downregulated during infection showed increased occupancy of H3K27m3 modification in their promoter region (indicated in yellow) and ISGs that were upregulated during infection showed increased occupancy of H3K4me3 within their promoter region (blue).While histone mimicry has been identified for H3N2 virus, the responsible binding motif is not contained in NS1 protein encoded by H5N1-VN1203, although the NS1 protein may contribute to this phenotype [31]. This suggests that different NS1 proteins may mediate the downregulation of subsets of ISGs by different mechanisms depending on Influenza Type A virus (IAV) strain. For H5N1-VN1203, NS1 may inhibit ISG expression by mimicking different histones, targeting histone-modifying enzymes, or disrupting a histone adaptor protein complex [18]. Several IFN antagonists have been identified; however, the nature of the accessory protein mediating ISG downregulation by interfering with the host&#8217;s epigenome remains to be identified [18,111].\n10. Systems-Biology Approach\n\n10.1. A Model Platform for Epigenetic Research Following Coronaviruses and Other Respiratory Virus Infections\nCalu3 cells are a continuous human airway epithelial cell line that can be differentiated in ciliated cells and are commonly used to study respiratory cell function under different physiological stresses and conditions [112]. By utilizing Calu3 cells, we have developed a robust human model platform to study innate immune regulatory control and epigenetics following emerging coronavirus and influenza virus infections as well as other highly pathogenic viruses (Figure 6). The first step is to define expression changes following treatment with defined perturbations, like various cytokines, to identify smaller subsets of effector gene expression patterns (RNA and protein) that are downstream of a specific signaling pathway. In parallel, the Calu3 cells are then infected with different highly pathogenic emerging or contemporary respiratory viruses and global proteomic and transcriptomic expression patterns characterized at different times post infection (Figure 6). By filtering expression changes to specific subsets of cytokine specific gene sets after infection, novel patterns of virus-induced regulatory control are revealed while identifying novel gene sets for downstream epigenetic and virus studies. For example, the same set of ISGs are either globally induced rapidly or differentially induced following H1N1 and H5N1 infection. In contrast, ISG expression patterns are significantly delayed but either globally or differentially induced following SARS-CoV and MERS-CoV infection, respectively [18]. Expression pattern differences were independent of any specific transcription factor function, but rather were regulated primarily by epigenetic control mechanisms. Importantly, these studies seed future studies using other viruses and/or segwaying into primary human airway epithelial cells (HAE), primary type II pneumocytes (AT2), lung fibroblast (LF), pulmonary endothelial cells (MEV) and resident immune cell populations in the lung [18,113]. Utilizing these model systems, we aim to study genome-wide histone modifications, DNA methylation patterns, and the chromatin landscape after virus infection across different cell types in the lung, revealing cell type-specific regulatory features that function to regulate infection outcomes. The goals of recently performed studies in our laboratory sets the stage to determine if host epigenetic processes play a crucial role in controlling transcriptional regulatory networks that antagonize or promote MERS-CoV and H5N1 infection and pathogenesis. While the molecular mechanisms regulating epigenetic control remain elusive following emerging CoV infection, a growing body of evidence suggests that viruses, even RNA viruses that replicate in the cytoplasm, interfere with the host&#8217;s epigenome.\n10.2. Integration across Data Types\nHost&#8211;pathogen relationships are plastic and dynamic. In particular, selective pressure put on the pathogen by the host increases the degree of plasticity and adaptability of the pathogen. On the other hand, the host phenotype is altered by the pathogen and the pathogen&#8217;s virulence, which leads to a co-adaptation that influences pathogen and host equally. For a successful viral life cycle, viruses have co-evolved with the host, which often means that the virus has to adjust to the host&#8217;s immune system, resulting in distinct mechanisms to repress or evade the host&#8217;s immune response. Co-evolution with the host may well have expanded the capabilities of epigenetic regulation, adding additional mechanisms and sources of dynamic reversible phenotype variation [109]. The evolution of viral antagonistic mechanisms which interfere with the host&#8217;s gene expression ability by modifying histone marks and therefore chromatin dynamics, enabling viruses to attack entire immune gene clusters, has not been heavily investigated and may represent a productive arena for future study. Mechanisms by which viruses could modulate the host&#8217;s chromatin include sequestration and displacement of chromatin-associated proteins, interference with the chromatin remodeling machinery, DNA-binding transcription factors, histone modifying enzymes, and direct alteration of methylation and acetylation state of histones and histone mimicry [18,31,114,115].Several complementary approaches can be combined to study, define, and map the epigenetic landscape during coronavirus infection and other highly pathogenic human viruses (Figure 6). Combining systems biology approaches like epigenomic, transcriptomic and proteomic datasets provides a data integration approach to identify regulatory genomic clusters and regions that play a crucial role in the host&#8217;s antiviral response (Figure 7) [116,117]. Moreover, systems biology approaches allow the development of models to make comparisons of data across pathogens to better predict complex biological systems. For example, the integration across systems-based data types will allow us to predict gene expression and to infer new gene regulatory networks. Using comparatively generated data from multiple levels of biological systems will allow the association between phenotypic outcome and variation, and the prediction of gene expression using only a few epigenetic features (Figure 7) [117,118]. These genome-wide based predictions are essential to define and interpret gene regulatory networks (GRNs). By linking epigenetic features to gene activation, gene expression levels are integrated into enrichment profiles which can be interpreted as clusters of differential enrichment patterns and then used to map interactions of virus and host to infer the identity of genes that are implicated in disease and pathogenesis associated with the virus [117].\n11. Outlook\nRecent advancements through basic research in the understanding of the mechanisms involved in viral chromatin modification in lytic viruses have opened a new window into previously unknown mechanisms of viral antagonism and host&#8211;virus interactions, including genetic factors that influence both protective or pathogenic host responses. The subsequent identification and improved understanding of these mechanisms open new avenues for the development of antiviral drugs by illuminating new targets for specific inhibitors. Established model systems for latent and persistent viruses such as human immunodeficiency virus 1 (HIV-1) and herpesviruses have already demonstrated modulation of the viral infection by chromatin [119,120]. As described here, a growing number of studies also show modulation of viral infection by chromatin in lytic virus infections. The emerging parallels between the existing knowledge of chromatin&#8217;s effect on and interaction with latent and persistent viruses and the emerging understanding of its comparable interaction with lytic viruses suggest that a greater focus on chromatin-based therapies for a variety of virus families could reveal fundamental new landscapes of virus&#8211;host interaction that play critical roles in disease severity.",
        "pmid": "28212305",
        "title": "Epigenetic Landscape during Coronavirus Infection",
        "journal_title": "Pathogens",
        "authors": "Alexandra Sch\u00e4fer;Ralph S. Baric",
        "doi": "10.3390/pathogens6010008"
    },
    {
        "keywords": "coronavirus;wildlife;wild rabbits;hedgehogs;bats;rodents;France;genetic diversity",
        "abstract": "Coronaviruses are closely monitored in the context of emerging diseases and, as illustrated with Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome-coronavirus (MERS-CoV), are known to cross the species barrier and eventually to move from wildlife to humans. Knowledge of the diversity of coronaviruses in wildlife is therefore essential to better understand and prevent emergence events. This study explored the presence of coronaviruses in four wild mammal orders in France: Bats, rodents, lagomorphs, and hedgehogs. Betacoronavirus and Alphacoronavirus genera were identified. The results obtained suggest the circulation of potentially evolving virus strains, with the potential to cross the species barrier.",
        "body": "1. Introduction\nCoronaviruses (CoVs) infect a wide variety of animals and are common throughout the world. They cause respiratory, enteric, hepatic, and neurological diseases with variable severity, from asymptomatic to severe. Coronaviruses that infect mammals (except pigs) belong mainly to two genetic and serologic groups: The Alpha- and Betacoronavirus genera.Along with Paramyxoviridae and the influenza viruses, the Coronaviridae family is one of the three viral families closely monitored in the context of emerging diseases. With Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome (MERS-CoV), coronaviruses have shown their ability to move from wildlife to domestic animals or to humans. In this context, describing the coronaviruses circulating in wildlife in a specific area is the first step to being readily responsive in the event of emergence. However, data concerning wildlife species harboring coronaviruses in Europe and especially France are currently very limited. Bat coronaviruses have been studied because of the SARS-CoV and MERS-CoV epidemics. In Europe, both beta-CoVs and alpha-CoVs have been found in bats [1,2]. To our knowledge, only four bat species among the 35 species living in France have been screened for Coronaviridae [2].Since all these novel alpha- and beta-CoVs were found in insectivorous bats, the search in Western European hedgehogs (Erinaceus europaeus), as an insectivorous mammal belonging to a related order of Chiroptera, resulted in the identification of a novel Betacoronavirus, Eri-CoV, detected from animals raised in an animal shelter in northern Germany [3]. Other wildlife species appear to be relevant to identify the potential presence of Coronavirinae given, for instance, the recent description in wild rodents of infection by Alphacoronavirus [4]. Another species of interest is wild rabbits (Oryctolagus cuniculus) because of beta-CoV identification in domestic rabbits in China [5], which have wild European rabbits as ancestors.The aim of this study was to explore the presence of Coronaviridae in specific relevant species of wildlife in France, in order to produce an overview of the reservoir of potential emerging Coronaviridae from wildlife and of the species that host them.\n2. Materials and Methods\n\n2.1. Sample Collection\nSamples of intestinal content were used for the detection of CoVs in bats, rodents, and rabbits. Regarding studies about the detection of CoVs, intestinal samples were tissues with the highest CoV RNA concentration in hedgehogs [3] and were successfully used to detect CoV in bats [6] and rabbits [5]. Intestinal samples were collected from carcasses of these animals as follows:-From 2013 to 2015, intestinal samples from carcasses of bats were collected within the lyssaviruses surveillance in France and originated from 74 of the 101 French administrative department (with only one overseas French department concerned, French Guyana, Figure 1).-From 2014 to 2016, rodents were trapped in five areas of eastern France (Figure 1) in the context of other studies. Rodents were captured using pieces of carrot and sunflower seeds as bait in different kinds of traps adapted to species. Captured animals were euthanized in accordance with the French Animal Protection Law and Directive 2010/63/EU of the European Parliament and of the Council on the protection of animals used for scientific purposes (identification code of the approval (29 September 2016) project AP AFIS#2939-20160106142231, name of the ethics committee: Cometh).-From 2007 to 2009, intestinal samples from rabbits (Oryctolagus cuniculus) collected in 10 departments by the French National Hunting and Wildlife Agency for another research project were analyzed (Figure 1). Intestinal samples were taken from animals killed by hunters during rabbit hunting. Therefore, no wild animal was killed specifically for the purpose of this study.Intestinal samples were stored at &#8722;20 &#176;C without additives.Samples of hedgehogs (E. europaeus) were fecal samples collected in three animal shelters in 2015 and 2016 (Figure 1). Hedgehog droppings were sampled on the first or second day after their arrival at the animal shelter. Fecal samples were stored at &#8722;20 &#176;C in UTM-RT&#8482; (Universal Transport Medium, Copan diagnostics, Murrieta, CA, USA).\n2.2. RNA Extraction and RT-PCR of the Partial RNA-Dependent RNA Polymerase Protein (RdRp) Gene\nA Nucleospin Viral RNA isolation kit (Macherey-Nagel, Hoerdt, France) was used for RNA extraction from stools or from intestinal homogenate. Reverse transcription was performed from 8 &#181;L of RNA with RT Superscript III and random hexamer primers. Amplification by PCR was carried out as described previously [7], by using a semi-nested PCR targeting the RdRp conserved coding region using 5 &#181;L of cDNA in a final volume of 50 &#181;L. PCR products were sent for sequencing by private companies (GenoScreen and Genewizz, Lille, France). Nucleic sequences of 125 bases corresponding to the RdRp gene were recovered from all positive samples.\n2.3. Phylogenetic Analysis\nThe CoV sequences found in this study were deposited in the DNA Data Bank of Japan (DDBJ) under accession numbersLC334347 to LC334385 and LC334392 to LC334408. Sequence similarity to known sequences was determined using Basic Local Alignment Search Tool (BLAST) nucleotide analysis against the GenBank database. The phylogenetic tree (Figure 2) was inferred using sequences obtained from positive samples aligned with coronavirus sequences representing the whole diversity of the group. Sequence multi-alignments were performed in SEAVIEW v4.6 [8]. Phylogenetic reconstruction using maximum likelihood was done using PhyML v3.0 [9]. The GTR+G+I (General Time Reversible) substitution model was selected as the optimal model by a function implemented in MEGA7 [10] and a discrete gamma distribution with four categories was used for rate heterogeneity. Trees were also inferred using the Bayesian approach implemented in the Beast package, with both GTR (General Time Reversible) and TN93 (Tamura-Ne&#239; 1993) optimum models, gamma distribution, and invariable sites. The coalescent constant size model was used and the clock parameters were set to the uncorrelated lognormal relaxed clock. The length of the chain was then set to 30 million iterations in order to produce ESS (Effective Sample Size) superior to 200. The maximum credibility tree with the branch length in number of substitutions was defined from ten thousand trees after a discard of 10% and was edited using FigTree (version 1.4.3 on http://tree.bio.ed.ac.uk/software/figtree/).\n2.4. Statistical Analysis\nConfidence intervals for seroprevalence were estimated using the Wilson method [11] in the Epitools calculator (http://epitools.ausvet.com.au/).\n3. Results\n\n3.1. Bats\nIntestinal samples from 504 bats were analyzed (Table 1). Twelve bats were found to be infected by a coronavirus. Five sequences from Pipistrellus pipistrellus and two from Pipistrellus sp. had a 94% to 99% nucleic acid sequence identity with alpha-CoVs of Pipistrellus from France and Germany (GenBank KT345295, KT345296 and EU375868). Two Myotis emarginatus were found to be infected by coronaviruses with an identity of 100% with sequences of alpha-CoVs found in Myotis emarginatus in Luxembourg (GenBank KY502401). The coronavirus found in one Myotis nattereri exhibited a 95% identity with an unclassified coronavirus found in Myotis nattereri in Hungary (GenBank KJ652333). Concerning Miniopterus schreibersii, the nucleic sequences of the two infected animals had an identity of 99% and 98% with coronaviruses found in the same species in Bulgaria (GenBank GU190247 and GU190241).\n3.2. Hedgehogs\nFecal samples from 74 hedgehogs were analyzed and 37 samples were positive. The estimated prevalence for all the animal shelters was 50% (CI95%: 38.89; 61.11), with no statistical difference between the three animal shelters.All sequences were strongly homogeneous with a maximum of four differences at the nucleotide level. The nucleic sequences detected from hedgehogs had an identity from 96% to 98% with the sequence of coronavirus (KC545383) found in the hedgehogs in Germany.\n3.3. Rodents\nOf the 330 trapped rodents (Table 2), most of the animals belonged to Apodemus spp. (n = 206) and Myodes glareolus (n = 80), and were collected at four sites (Bitche, Forbach, La-Petite-Pierre, and Murbach) in mixed coniferous and deciduous forests. Other species of rodents were Arvicola terrestris (n = 35) and Microtus spp. (n = 9), and were mainly collected in an apple orchard in Wissembourg (Bas-Rhin).Twenty-one samples were positive, represented by five Myodes glareolus (6.25% (CI 95%: 2.70; 13.81)) and 16 Apodemus spp. (7.77% (CI 95%: 4.84; 12.24)). Positive samples were found each year and in each area (Table 2), except in Wissembourg. All the sequences were very close (identities from 94% to 98%) to five sequences of unclassified coronavirus detected in 2010 in Germany in Myodes glareolus and Apodemus spp. (KM888128, KM888131, KM888139, and KM888159), and in 2007 in the Netherlands in Microtus arvalis (KM888142). Aligned sequences were separated into two groups. Fifteen sequences with a maximum difference of two nucleotides formed the first group detected from Apodemus spp. samples. The other six sequences of the second group with 27 to 33 nucleotides different from the R152 sequence of the first group were detected from samples of Myodes glareolus, except one in Apodemus spp.\n3.4. Rabbits\nA total of 291 intestinal samples from rabbits were analyzed. Twenty-two animals were found to be infected, corresponding to a global estimated prevalence of 7.56% (CI 95%: 5.05; 11.18), which differed from one department to another from 0% (CI 95%: 0.00; 29.91) to 16.66% (CI 95%: 8.32; 30.60).Seventeen sequences were very close (with identities from 94% to 98%) to the beta-CoV RbCoV HKU14 (5) and to an alpha-CoV identified in hares (Lepus sp.) and rabbits in Spain (with identities from 97% to 98%) (GenBank Nos. KM888169 to KM888181). Five sequences were very close (with identities from 96% to 97%) to the alpha-CoV identified by Tsoleridis et al. [4] in rodents (KU739072). Both beta- and alpha-CoVs were identified in rabbits from the same geographical area and collected the same year. For each genus, nucleic sequences showed strong homogeneity, with a maximum of five differences at the nucleotide level.\n3.5. Phylogenetic Diversity\nPhylogenetic analyses (Figure 2) of the partial RdRp gene revealed that all coronavirus sequences from bats belonged to the Alphacoronavirus genus. Sequences of alpha-CoVs detected in France from Miniopterus schreibersii (CS131010) clustered with sequences of alpha-CoVs detected in the same species in Germany. Likewise, sequences of alpha-CoVs from Myotis emarginatus from France (CS130613) clustered with sequences of alpha-CoVs detected in the same species in Luxembourg. Similarly, sequences of alpha-CoVs from Myotis nattereri in France (CS130938) clustered with sequences from the same species in Germany and England. Sequences of alpha-CoVs from Pipistrellus sp. in France (CS130412, CS130747, CS130894, CS130459, CS130860, and CS130786) were found in two distinct groups, well supported by Likelihood Ratio Test analysis and posterior probabilities. The other sequences of European alpha-CoVs from Pipistrellus sp. (KT345294, KT345295, and KT345296 in France, KF500945 in Italy, and GQ259964 in the Netherlands) also clustered into these two distinct groups.Nucleic sequences from hedgehogs (HE on Figure 2) were highly homogeneous and clustered with one of the other sequences of beta-CoV detected from hedgehogs in Germany. The sublineage of hedgehog sequences, human MERS-CoV, camel MERS-CoV-like, and bat MERS CoV-like sister clade is different from the other beta-CoVs, as shown by its well-supported Likelihood Ratio Test analysis and posterior probabilities.All sequences of beta-CoV detected from rodents in France were in the same group of sequences as beta-CoV from wild rabbits detected in the same country (L103, L118, L139, L172, L276) and domestic rabbits in China (GenBank No. JN874559). Sequences of beta-CoV from Apodemus sp. in France (R49, R86, R90, R150, R152) were in the same subgroup, whereas sequences from Myodes glareolus were found in three distinct subgroups: Two (R42, R91, R214, R218) with sequences of beta-CoV from Myodes glareolus from Germany (KM888128, KM888131 and KM888139); and the third with a sequence (R75) from Microtus arvalis from the Netherlands (KM888142).The two sequences of alpha-CoV from rabbits from France (L8, L16) were in the same group of alpha-CoVs from rodents in the United Kingdom (GenBank Nos. KU739072, KU739073, KU739074, and KU739071).\n4. Discussion\nThis study described the presence of coronavirus infection in four wild mammal orders in France. Our results suggest that coronaviruses seem to be relatively common in these species, regardless of the place and time of sampling. The sequence length in our study was short because samples originated from other works for other purposes (except hedgehogs); however, interesting conclusions can be suggested. Studies about coronaviruses in bats are numerous in comparison with other wild mammals. In Europe, alpha-CoVs have been found in Pipistrellus sp. (France, Hungary, Romania, Germany, Spain, Italy, and the Netherlands), Myotis sp. (Hungary, Germany, Spain, the United Kingdom, and the Netherlands), Rhinolophus sp. (Bulgaria and Hungary), and Nyctalus sp. (Spain and Bulgaria) [2]. In this study, seven Pipistrellus (six P. pipistrellus and one Pipistrellus sp.) were found to be positive in several geographically distant parts of the country (East, West and North of France). Two Myotis emarginatus in the same area in the East of France, one Myotis nattereri in the North of France, and two Miniopterus schreibersii were found in two areas in the South and East of the country. The number of positive samples by site (except for Miniopterus with two positive samples out of five) was highly dependent on the number of collected samples by site. The collection concerned 23 species of bats, out of the 35 known in mainland France. The collection of dead bats through the French network for lyssaviruses surveillance is performed by volunteers with no minimum sample sizes in each department, explaining the heterogeneous distribution of samples and the respective abundance of populations in biotopes close to humans. The use of this network allowed for better distribution of the dead animal samples across the national territory, compared to colony by colony collection. Nevertheless, the detection of the RNA sequence of RdRp in the infected animals was highly dependent on the state of the carcasses when discovered, and on storage before being dispatched to the laboratory.The overall prevalence in bats of 2.4% (CI 95%: 1.37; 4.12) was consistent with other studies in European countries, which have reported figures from 0% ([12] in Belgium) to 17.1% ([13] in the UK).This was the second report of Myotis emarginatus infection by coronavirus inside the usual range of this bat; the first was in Luxembourg [14]. Interestingly, both cases reported here came from a department very close to Luxembourg. This species was rarely (about 20 samples) concerned in previous studies [15,16,17,18], but it would be relevant to study coronaviruses in these bats from other parts of its range to confirm the existence of a novel RdRp-grouping unit (RGU), as suggested in a recent study [14]. In addition, our new nucleic sequences of coronavirus identified from Myotis emarginatus belong to Alphacoronavirus, like all sequences previously detected from other Myotis bats in European countries [13,16,17,18,19,20,21,22], except in one more recent study [14] that focused on this species and found alpha-CoV and beta-CoV shed by bats. In the same way, alpha-CoVs were also identified in Miniopterus schreibersii, as described in other studies [15,16], but no beta-CoV was found, probably because of the small number of fecal samples tested (one in [15], two in [16], and five in our study).The situation is more contrasted with Pipistrellus pipistrellus. Our results reinforce the findings of the other study carried out in France [2] with the identification of alpha-CoV and a prevalence of 2.44% (CI 95%: 1.05; 5.58) in our study versus 5.4% (CI 95%: 0.6; 7.9) in Goffard et al. [2]. However, Pipistrellus pipistrellus may be infected by beta-CoV as reported in Romania [1], while both beta- and alpha-CoVs were found in this species in the Netherlands [18], but also when considering the Pipistrellus genus in general [1,6,21].This study demonstrated the circulation of beta- and alpha-CoVsin wild rabbits. Most of the detected nucleic sequences were very close to the beta-CoV of domestic rabbits in China [5], and the others were close to an alpha-CoV KU739072 detected from rodents in the UK [4]. Unfortunately, the lack of other studies targeting wild rabbits in its historical range prevents us from carrying out other comparisons. Continued surveillance of coronaviruses in rabbits would be useful to supplement our knowledge on infection in this species. Bearing in mind that wild rabbits are the ancestors of domestic rabbits, the study of coronaviruses in wild and domestic rabbits could be a useful model to better understand viral adaptive strategies. Indeed, human subtype OC43-related beta-CoV, Beta-CoV1 (lineage A), as identified is this study, is thought to have low host-specificity and to evolve by host-switching and recombination [5,14,23]. By contrast, alpha-CoVs are thought to be more specific to their hosts, at least in bats [21]. The identification in intestinal rabbit samples from twos department of alpha-CoVs very close to those identified previously (KU739072) in rodents [4] is not in favor of host specificity. Two hypotheses are possible. First, there is the circulation of non-specific alpha-CoVs within rodents, very close to viruses found in rabbits. The proximity of the Rodentia order to the Lagomorpha order, together forming the Glires clade [24], corroborates this hypothesis since rodents and rabbits share the same environment. Second, our sequences were short and were not sufficiently discriminant.All sequences from rodents in this study were identified as beta-CoV, whereas in the study in the UK and Poland [4], all identified coronaviruses from rodents belonged to alpha-CoVs. Species that tested CoV-positive in the 2016 study were Apodemus sylvaticus, Myodes glareolus, Microtus agrestis, and Rattus norvegicus. The CoV-positive species in our study were Myodes glareolus and Apodemus flavicollis. No Microtus sp. tested were positive, and no Rattus norvegicus were sampled. As for bats and rabbits, the circulation of both coronaviruses should be expected in rodents in Europe, but further investigations are needed to confirm this hypothesis.In addition, genetic segregation between coronaviruses from Apodemus and from Myodes was observed and suggested co-evolution with the host family [25]. The identification of one Apodemus infected by a coronavirus very close to that observed in Myodes may indicate potential host switching.The situation seemed to be very different for hedgehogs with the observation of only beta-CoV. The prevalence in hedgehogs was higher than in other species. The only other study to our knowledge concerning coronaviruses in hedgehogs in Europe [3] found results for Germany that are very similar to those in our study, with a beta-CoV close to those that identified in this study (96% to 98% identity), and with a prevalence in fecal samples of 58.9%. By contrast, in the same animal shelter, two groups of coronaviruses separated by a 3.1% to 3.4% nucleotide distance in an 816-nt RdRp fragment were found, but all viruses were in the same clade. In our study, our short nucleic sequences of the RdRp gene were highly homogeneous, despite the significant distance between the three animal shelters in France. Hedgehogs are territorial; they do not travel distances as long as those between the shelters and they do not migrate. The similarity between sequences could be the result of co-evolution between viruses and hedgehogs.This preliminary investigation of the presence of coronavirus in different wildlife species in France highlighted the presence of both alpha- and beta-CoVs in rodents, rabbits, and bats. The circulation of a range of coronaviruses within the same order or the same genus of mammals suggests the emergence of new coronaviruses by host switching, which is one of the main evolutionary mechanisms for coronaviruses [25].In addition, our study highlighted observations in French wildlife compatible with the second evolutionary mechanism for coronaviruses reported by Anthony et al. [25]: Co-speciation, like for hedgehogs, Apodemus spp. and Myodes glareolus.Our results showed the existence of potentially evolving virus strains, with possible crossing of the species barrier. Further investigations are needed in these species and other species of wildlife to assess the potential risk for domestic animals and humans of widely distributed CoVs by studying the spike region involved in the crossing of the species barrier [26,27].",
        "pmid": "29186061",
        "title": "Identification of Alpha and Beta Coronavirus in Wildlife Species in France: Bats, Rodents, Rabbits, and Hedgehogs",
        "journal_title": "Viruses",
        "authors": "Elodie Monchatre-Leroy;Franck Bou\u00e9;Jean-Marc Boucher;Camille Renault;Fran\u00e7ois Moutou;Meriadeg Ar Gouilh;G\u00e9rald Umhang",
        "doi": "10.3390/v9120364"
    },
    {
        "keywords": "bat;SARS",
        "abstract": "We report group B Betacoronavirus infection in little Japanese\nhorseshoe bats in Iwate prefecture. We then used reverse-transcription PCR to look for the\ncoronavirus RNA-dependent RNA polymerase gene in fecal samples collected from 27 little\nJapanese horseshoe bats and found eight were provisionally positive. We had a success in\nthe nucleotide sequencing of six of the eight positive samples and compared them with\nthose of authentic coronaviruses. We found that these six samples were positive in\ncoronavirus infection, and they belonged to the group B Betacornavirus by\nphylogenetic analysis. Virus isolation using the Vero cell culture was unsuccessful.\nPathogenic trait of these bat coronaviruses remained unexplored.",
        "body": "",
        "pmid": "24871548",
        "title": "Group B Betacoronavirus in Rhinolophid Bats, Japan",
        "journal_title": "The Journal of Veterinary Medical Science",
        "authors": "Jin SUZUKI;Ryota SATO;Tomoya KOBAYASHI;Toshiki AOI;Ry\u00f4 HARASAWA",
        "doi": "10.1292/jvms.14-0012"
    },
    {
        "keywords": "coronavirus;SARS;real-time RT-PCR;molecular diagnostics",
        "abstract": "We developed a set of three real-time reverse transcription&#8211;polymerase chain reaction (PCR) assays that amplify three different regions of the SARS-associated coronavirus (SARS-CoV), can be run in parallel or in a single tube, and can detect <10 genome equivalents of SARS-CoV. The assays consider all currently available SARS-CoV sequences and are optimized for two prominent real-time PCR platforms.",
        "body": "The Study\nRecently, a new coronavirus was identified as the suspected causative agent of an increased number of atypical pneumonia cases reported from Hong Kong, Singapore, Vietnam, and Canada (1&#8211;4). Subsequent publications demonstrated that this new coronavirus was detectable in patients with severe acute respiratory syndrome (SARS) (5,6), classified according to the World Health Organization's case definition (7). During the first 6 months of 2003, a total of 8,422 patients were affected. This fact, together with the reappearance of the SARS-associated coronavirus (SARS-CoV) in China in late 2003, makes it clear that rapid and reliable diagnostic tools are essential for accurate disease reporting and subsequent disease management.Because a defined treatment program and vaccination strategy are lacking, the main strategy to counteract the spread of this emerging virus is timely identification and isolation of infected persons. SARS patients' typical initial symptoms include fever, cough, and headache, similar to many acute viral respiratory infections. Therefore, molecular-based diagnostic methods are applied to rapidly identify SARS-CoV&#8211;infected persons. Recently, nested and real-time reverse transcription&#8211;polymerase chain reaction (RT-PCR) assays to detect SARS-CoV have been published (5,8). These assays&#8212;the first tools to detect SARS-CoV in patients with SARS&#8212;were based on the short stretches of viral sequence identified as the RNA-directed RNA polymerase of a new microbe.Subsequently, sequences from several SARS-CoV isolates were determined, and all of these sequences were closely related, as would be expected during the clustered outbreaks in 2003. However, the genomes of RNA viruses, including those of coronaviruses, tend to vary over time and with location (9&#8211;12). Recently, the sequence variations of SARS-CoV during the first epidemic phases in China in 2003 were reported. The neutral mutation rate for SARS-CoV was almost constant and similar to that of known RNA viruses; the S protein, responsible for virus-host receptor recognition, displayed the most extensive amino acid changes (13). In addition, the sequence analysis of isolates from recent SARS patients in China in 2004 has shown that 98.8%&#8211;99.4% of the 3,768 bases of S gene, 99% of 658 bases of M gene, and 99% of 1,068 bases of N gene are isogenous with those submitted to public databases, which date back to the first epidemic in spring 2003 (14). However, even these minimal changes could render existing PCR assays ineffective should SARS-CoV reemerge (15).To improve the ability to detect SARS-CoV safely and reduce the risk of eliciting false-negative results caused by genome sequence variations, we established three individual real-time RT-PCR assays. Target sequences were chosen by using the following criteria: 1) the regions are distributed over the whole genome, including the nonstructural polyprotein 1a and 1ab genes and the spike glycoprotein gene (Table 1); 2) the regions are highly conserved among the 89, 90, and 100 respective sequences available in public sequence databases; 3) the regions are suitable for the design of a real-time RT-PCR assay; and 4) the designed primers, 5&#180;-nuclease probes, and amplicons displayed no considerable homology to other viruses, including human CoV OC43 and 229E in BLAST searches (available from http://www.ncbi.nlm.nih.gov/BLAST/).These assays were based on the fluorogenic oligoprobe chemistry, which uses the 5&#180;-exonuclease activity of the DNA polymerase to generate a more specific signal than that produced by the use of SYBR Green I (8). The real-time RT-PCR assays were successfully run on the Applied Biosystems real-time PCR systems (SDS7700 and SDS7000; Applied Biosystems, Foster City, CA) as well as on the Roche LightCycler (Roche Diagnostics GmbH, Mannheim, Germany). All assays were designed as one-step RT-PCR reactions to be run under identical conditions on the respective PCR platform. This system allowed the simultaneous detection of different SARS-CoV regions in a single PCR run. Moreover, we could combine the three assays in a single tube, which might be important when clinical material is limited. Finally, the assays were compared to the 5&#180;-nuclease assay published recently (5) and to a commercially available real-time PCR kit (Real-Art HPA-Coronavirus LC RT PCR Reagents, Artus GmbH, Hamburg, Germany).After optimization of primer and 5&#180;-nuclease probe concentration and annealing temperature, reaction conditions for our 5&#180;-nuclease assay were as follows. For the RT-PCR performed on the Applied Biosystems platforms, each 25-&#181;L reaction contained 12.5 &#181;L of 2xQuantiTect Probe RT-PCR Master Mix (Qiagen, Hilden, Germany), 10 pmol of each primer, 3 pmol of 5&#180;-nuclease probe, and 0.25 &#181;L of QuantiTect Probe RT Mix. RNase-free water was added up to 23 &#181;L, and 2 &#181;L of RNA was used. Cycling conditions were 30 min at 50&#176;C for RT reaction, 15 min at 95&#176;C for inactivation of RT, activation of the Taq DNA polymerase, and cDNA denaturation, followed by 45 cycles of 15 s at 95&#176;C and 30 s at 60&#176;C. Total running time was 140 min.For LightCycler RT-PCR reactions, each 20-&#181;L reaction included 7.5 &#181;L of 2.7xLightCycler RNA Master Hybridization Probes mix (Roche Diagnostics GmbH), 10 pmol of each primer, 3 pmol of the 5&#180;-nuclease probe, and 1.3 &#181;L Mn (OAc)2 (50 mmol/L). RNase-free water was added up to 18 &#181;L, and 2 &#181;L of RNA was used. Cycling conditions were 20 min at 55&#176;C for the RT reaction, 30 s at 95&#176;C for initial denaturation, followed by 45 cycles of 1 s at 95&#176;C, 10 s at 55&#176;C, and 10 s at 72&#176;C. Total running time was 55 min. The combined assays were set up by adding all primers and probes in the same concentration; the amount of water was reduced accordingly. Protocols are also available from the Robert Koch-Institut homepage (available from www.rki.de/INFEKT/SARS/PCRPROTOCOL.PDF). The human L13 gene and the human cyclophilin gene (16) were amplified under identical reaction conditions as the SARS-CoV&#8211;specific assays on the ABI platforms and the LightCycler, respectively, to act as amplification controls.To evaluate the sensitivity of the SARS-CoV&#8211;specific assays, RT-PCRs were performed repeatedly on serial dilutions of RNA extracted with the Viral RNA Kit (Qiagen) from cultured SARS-CoV with defined amounts of genome equivalents (GE) by using the international standard of the European Network for the Diagnostics of Imported Viral Diseases (ENIVD), distributed through the Robert Koch-Institut (available from: http://www.rki.de/INFEKT/SARS/DATASHEET.PDF). Results are shown in Table 2. Detection limits of the three new assays were <10 GE. Comparison of the threshold cycle (CT) values showed that the new assays were at least as sensitive as the previously described assays (5,8) and the commercially available kit. When we combined the assays in a single tube targeting three different regions on the same RNA template, the CT was reduced by 1 to 2 cycles (SDS7700), which suggests either that sensitivity was unchanged or, when there was an increase, that it was attributable to the combination of all three signals (LightCycler). Subsequent agarose gel analysis during optimization steps of the PCR confirmed the simultaneous amplification of the three RT-PCR products (Figure).Using the single or combined assays, we analyzed 27 bronchoalveolar-lavage fluid samples from 19 suspected SARS case-patients and 8 probable SARS patients (according to the Robert Koch-Institut case definition, available from http://www.rki.de/INFEKT/SARS/AOLG-FALLDEF-ARSUU.PDF). All samples were positive for L13 and cyclophilin control sequences when amplified in parallel. In agreement with the previously published assay results (5), SARS-CoV was detectable in three samples from eight probable SARS patients, without explicit differences in the CT value of individual assays when the single or combined assays were used. These patients were seropositive and are regarded as confirmed SARS patients. Respiratory samples and stool samples taken 8 days later from the remaining 5 probable patients as well as the 19 persons with suspected disease were negative by RT-PCR. Moreover, these patients remained seronegative and are regarded as unconfirmed SARS patients. In addition, 35 serum samples from patients with SARS-CoV infection obtained 1&#8211;52 days after disease onset were analyzed. Between 20 and 1,000 GE/mL of SARS-CoV-specific RNA was detected in 21 of 35 serum samples, even when serum was obtained from patients 1 day after disease onset. (A detailed description of this study will be published later.)\nConclusions\nNone of the assays displayed cross-reactivity to clinical samples containing human cDNA from blood; human CoV 229E; influenza viruses A and B; parainfluenzaviruses 1, 2, and 3; respiratory syncytial virus; rhinoviruses; enteroviruses; adenoviruses 1&#8211;10; human metapneumovirus; Mycoplasma pneumoniae; or Chlamydia pneumoniae. For these pathogens, we obtained neither a fluorescent signal nor an amplification product in subsequent agarose gel analysis (selection shown in Figure). Although we focused on a one-step RT-PCR to decrease handling and total assay time, the three real-time RT-PCR assays can also be performed as two-step RT-PCR, including a separate cDNA synthesis step followed by PCR, and then finally using appropriate ready-to-use master mixes and the same cycling conditions, omitting the RT step.The single assays and the combined assay were also used in an external quality assessment to detect SARS-CoV organized by the ENIVD. All assays could detect SARS-CoV in 7 of 11 samples with virus loads ranging from 5x106 to 2x103 GE of two isolates of SARS-CoV per milliliter sample without false-positive or false-negative results. While the application of three single assays to detect SARS-CoV leads to a higher reliability of negative results, reflecting the negative outcome of three independent amplification reactions, it is a more expensive approach than combining the assays.In conclusion, the real-time RT-PCR assays we describe provide a fast and reliable tool that can complement and improve recently introduced techniques for SARS diagnostics. Parallel amplification of two human reference genes, L13 and cyclophilin, confirmed negative results in clinical samples by demonstrating amplifiable RNA. The separation of the control reaction was chosen to guarantee the high sensitivity of the SARS-CoV detection of <10 GE of SARS-CoV per reaction. An RT-PCR run is completed in <1 h, depending on the real-time PCR platform. In cases of small amounts of material or in an emergency situation with a high throughput of samples, the three SARS-CoV&#8211;specific assays can be combined into one RT-PCR reaction without loss of sensitivity. Furthermore, as the ambiguous diagnostic results in a hospital in Canada have recently shown (17), targeting three different regions distributed over the whole genome considerably reduces the risk for false-negative results caused by virus sequence modifications.",
        "pmid": "15324554",
        "title": "SARS Coronavirus Detection",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Andreas Nitsche;Brunhilde Schweiger;Heinz Ellerbrok;Matthias Niedrig;Georg Pauli",
        "doi": "10.3201/eid1007.030678 "
    },
    {
        "keywords": "",
        "abstract": "EK1 is a broad-spectrum human coronavirus fusion inhibitor for combating infection of current and emerging coronaviruses.",
        "body": "INTRODUCTION\nCoronaviruses (CoVs) are enveloped viruses with a positive-sense, single-stranded RNA and are associated with various natural hosts. CoVs are divided into alpha, beta, gamma, and delta groups, and the beta group is further composed of A, B, C, and D subgroups (Fig. 1A) (1). Among them, six CoVs can infect humans (HCoVs), including HCoV-229E (229E) and HCoV-NL63 (NL63) in the alpha group, HCoV-OC43 (OC43) and HCoV-HKU1 (HKU1) in beta subgroup A, severe acute respiratory syndrome CoV (SARS-CoV) in beta subgroup B, and Middle East respiratory syndrome CoV (MERS-CoV) in beta subgroup C (2).In this century, SARS-CoV and MERS-CoV have emerged in the human population and caused severe pulmonary disease with alarmingly high case-fatality rates. In 2002, SARS-CoV infections first appeared in China and then quickly spread as a global epidemic in more than 30 countries with 8273 infections and 775 deaths (nearly 10% mortality) (2). In 2012, MERS-CoV emerged in Saudi Arabia and spread throughout the Middle East. In 2015, the second pandemic of MERS-CoV occurred in South Korea, causing super-spreading events with third- and fourth-generation cases of infection. The World Health Organization has reported 2229 laboratory-confirmed cases of MERS-CoV infection, including 791 deaths (about 35% case fatality) in 27 countries as of August 2018 (www.who.int/emergencies/mers-cov/en/). Meanwhile, the remaining common HCoVs, such as 229E, OC43, and NL63, usually infect the human upper respiratory tract and cause the common cold, but they also are responsible for severe and even fatal diseases in children, elderly, and immunocompromised patients (3&#8211;5). These scenarios suggest that those common HCoVs might also pose a lethal threat to humans. Note that HCoVs are rapidly evolving. OC43 isolates with novel genomes are being continuously identified (6&#8211;8).The existence of SARS-like CoV (SL-CoV) and MERS-like CoV (ML-CoV) also pose great threats to public health worldwide. Recent studies identified some types of SL-CoV, such as SL-WIV1-CoV and SL-SHC014-CoV, whose spike (S) proteins highly resemble those of SARS-CoV. These SL-CoVs could use the same SARS-CoV receptor, i.e., angiotensin-converting enzyme-2 (ACE2), to directly enter permissive human cells without need for adaptation (9&#8211;11). In addition, the ML-CoV, Tylonycteris bat CoV HKU4, was shown to recognize the MERS-CoV receptor CD26 and infect human cells either after introduction of two single mutations (S746R and N762A) into its S protein or with the help of exogenous protease (12, 13). From a historical perspective, zoonotic CoVs harbor a strong ability to cross species barriers to infect humans rapidly and unpredictably, as illustrated by newly emerging HCoVs (2, 14). Thus, developing a specific drug that only targets a single HCoV would be ineffective against newly emerging HCoVs (9&#8211;11). Since no broad-spectrum anti-HCoV drug is currently available for clinical use, it is incumbent to search for a common or conserved target site based on existing HCoVs.The S glycoprotein is a type I transmembrane glycoprotein that plays an important role in mediating viral infection and is common to all HCoVs. The S proteins consist of two subunits, S1 and S2 (Fig. 1B). The S1 subunit binds the cellular receptor through its receptor-binding domain (RBD), followed by conformational changes in the S2 subunit, which allows the fusion peptide to insert into the host target cell membrane. The heptad repeat 1 (HR1) region in the S2 subunit forms a homotrimeric assembly, which exposes three highly conserved hydrophobic grooves on the surface that bind heptad repeat 2 (HR2). This six-helix bundle (6-HB) core structure is formed during the fusion process and helps bring the viral and cellular membranes into close proximity for viral fusion and entry (Fig. 1B) (15). Thus, the S protein is an important target protein for the development of specific drugs. In particular, the S1 RBD is a very good target site, and both RBD-specific antibodies and RBD-based vaccines have previously exhibited effective antiviral activity or protective effect in blocking binding of virus to host receptors (16). However, from an evolutionary perspective, the RBD of CoV is part of a highly mutable region and, thus, is not an ideal target site for broad-spectrum antiviral inhibitor development (14). The SARS-CoV RBD-specific antibody fm6 failed to block infection mediated by the S protein of SL-CoV&#8211;SHC014 (9&#8211;11, 17). In contrast, the HR region in the S2 subunit is conserved among various HCoVs and plays a pivotal role in HCoV infections by forming the 6-HB that mediates viral fusion (fig. S1A). Furthermore, the mode of interaction between HR1 and HR2 is conserved among CoVs such that residues located at the &#8220;e&#8221; and &#8220;g&#8221; positions in the HR1 helices interact with residues at the &#8220;a&#8221; and &#8220;d&#8221; positions in the HR2 helices (fig. S1B) (18). Previous studies have reported that peptides derived from the HR2 (or C-terminal HR) region of class I viral fusion proteins from some enveloped viruses, including HIV-1 (19&#8211;21), respiratory syncytial virus (RSV) (22), Ebola virus (23), paramyxoviruses SV5 (24), Nipah virus (25), and murine hepatitis virus (MHV) (26), could competitively bind the viral HR1 (or N-terminal HR) and effectively inhibit viral infection. Therefore, it is reasonable to speculate that HR1 could also be a good target for the development of fusion inhibitors against highly pathogenic HCoVs.We and others have reported that peptides derived from the HR2 regions of SARS-CoV and MERS-CoV S proteins can competitively inhibit viral 6-HB formation, thereby preventing viral fusion and entry into host cells (18, 27). For example, the peptide CP-1, derived from the SARS-CoV spike protein HR2 region, was able to inhibit SARS-CoV entry in a manner similar to that of MERS-HR2P against MERS-CoV infection (18, 27). However, we note that those peptides lack broad-spectrum antiviral activity against heterologous HCoVs. For example, CP-1 and MERS-HR2P peptides failed to cross-inhibit MERS-CoV and SARS-CoV infection, respectively. Furthermore, the CoV fusion core HR regions can be divided into two groups (28, 29): short HRs, such as MERS-CoV, SARS-CoV, and OC43 HRs; and long HRs, such as 229E and NL63 HRs. The difference between short and long HRs arises from an insertion of 14 amino acids in the long HRs, further increasing the difficulty in designing broad-spectrum peptide fusion inhibitors. To address this challenge, we herein report the successful screening of a peptide OC43-HR2P with broad-spectrum fusion inhibitory activity. Furthermore, a modified OC43-HR2P peptide, EK1, shows promising potency and breadth in inhibiting infection by multiple HCoVs. In vivo studies demonstrate that administration of EK1 via the nasal route exhibits highly protective effects and safety profiles, highlighting its clinical potential. Moreover, structural studies of EK1 in complex with HR1s from different HCoVs explain the conserved basis for the HR1-EK1 interaction, further indicating that HR1 region could serve as a promising target site for the development of broad-spectrum pan-CoV fusion inhibitors.\nRESULTS\n\nHR-derived peptides inhibit cell-cell fusion mediated by S proteins of multiple HCoVs\nOn the basis of the 6-HB fusion core structure, we previously reported on peptides MERS-HR1P and MERS-HR2P that were derived from MERS-CoV HR1 and HR2, respectively (fig. S1C), and these two peptides displayed good interaction with each other (27). Here, we located the conserved HR region of multiple HCoVs by sequence comparisons and then synthesized HR1- and HR2-derived peptides, termed HR1P and HR2P (Fig. 1C). Notably, we observed 14&#8211;amino acid insertions in both HR1P and HR2P in the HRs of two &#945;-HCoVs, i.e., 229E and NL63.To systematically assess the inhibitory activities of these peptides against different HCoVs, we developed multiple cell-cell fusion assays that were mediated by the S protein of various HCoVs (fig. S1D). Consistent with previous results, MERS-HR2P had inhibitory activity against cell-cell fusion mediated by MERS-CoV S protein with a concentration for 50% inhibition (IC50) of 1.01 &#956;M, whereas it showed little inhibitory activity in other HCoV S-mediated cell-cell fusion assays even with concentrations up to 5 &#956;M (Fig. 1D and table S1). Similarly, SARS-HR2P specifically inhibited SARS-CoV S-mediated cell-cell fusion with an IC50 of 0.52 &#956;M. On the other hand, HR2P peptides derived from the two &#945;-HCoVs, i.e., 229E and NL63, showed potent and broad inhibitory activity against &#945;-HCoV S-mediated cell-cell fusion with IC50 values ranging from 0.13 to 0.51 &#956;M or from 0.21 to 0.56 &#956;M, respectively, but no effective inhibitory activity against &#946;-HCoVs (including MERS-CoV, SARS-CoV, and OC43) S-mediated fusion (Fig. 1D and table S1). OC43-HR2P exhibited broad and potent fusion inhibitory activity with IC50 values of 0.39, 0.54, and 0.66 &#956;M against MERS-CoV, SARS-CoV, and OC43, respectively (Fig. 1D and table S1). Unexpectedly, OC43-HR2P, a &#946;-HCoV HR2-derived peptide, and thus 14 residues shorter than 229E-HR2P and NL63-HR2P, exhibited effective activity against &#945;-HCoVs with an IC50 of 0.84 &#956;M on 229E-S&#8211;mediated cell-cell fusion and an IC50 of 0.94 &#956;M on NL63-S&#8211;mediated cell-cell fusion. Among all HR1Ps, only 229E-HR1P exhibited moderate inhibitory effects on 229E-S&#8211; and NL63-S&#8211;mediated cell-cell fusion (Fig. 1D). Thus, compared to HR1Ps, all HCoV HR2-derived peptides exhibited excellent self-specific fusion inhibitory activity, whereas OC43-HR2P showed broad-spectrum and potent fusion inhibitory activity against both &#945;-HCoV and &#946;-HCoV S-mediated cell-cell fusion (Fig. 1D and table S1). We also measured the &#945;-helicity of OC43-HR2P and HCoV-HR1Ps by circular dichroism and found that the peptides alone exhibited limited &#945;-helicity ranging from 11.3 to 42.3% (fig. S1E). In contrast, the mixtures of OC43-HR2P and each of these HR1 peptides, respectively, exhibited high &#945;-helicity (70.9 to 86.7%) with melting transition temperature (Tm) values that ranged from 48.5&#176; to 91.5&#176;C (fig. S1E), further implying that OC43-HR2P can bind HR1s of different HCoVs and form stable complexes, thereby blocking S protein&#8211;mediated fusion.\nEK1, a modified OC43-HR2P peptide, has improved fusion inhibitory activity and increased solubility\nIt was previously reported that introduction of negatively and positively charged amino acids Glu (E) and Lys (K), at the i to i + 3 or i to i + 4 positions in a helix, into an HIV-1 fusion inhibitory peptide can form intramolecular E&#8211;K or K&#8211;E salt bridges that result in substantial enhancement of the peptide&#8217;s stability, solubility, and antiviral activity (30, 31). Using a similar design, we further optimized the sequence of OC43-HR2P by introducing Glu or Lys at appropriate sites in the peptide to increase the solubility and thereby the antiviral activity of the peptide. Moreover, on the basis of the structure of MERS-CoV S 6-HB, we also introduced mutations at some sites that are not expected to be involved in HR1 binding, such as 4Q, 14Y, 32D, and 36L, to further enhance the fusion inhibitory activity of the peptide (table S2). Among the series of optimized peptides, peptides EK0-1, EK0-2, EK0-3, and EK1 showed gradually increased solubility and excellent inhibitory activity in cell-cell fusion assays. The final peptide, EK1, exhibited the most potent pan-CoV antiviral fusion activity with IC50 values in the range of 0.19 to 0.62 &#956;M (table S2). In some HCoV cell-cell fusion assays, EK1 exhibited even more effective inhibitory activity than the autologous peptide, such as MERS-HR2P and SARS-HR2P. Akin to its ancestor OC43-HR2P, EK1, which is derived from the short 6-HB fusion core of OC43, was also able to potently inhibit the cell-cell fusion mediated by 229E and NL63 S proteins, both of which harbor the long fusion core. These results further underscore the broad-spectrum anti-HCoV potential of EK1 (Fig. 2, A to E, and table S2). Furthermore, EK1 also had superior pharmaceutical properties and solubility in phosphate-buffered saline (PBS) and water, which increased by 478-fold and 3.5-fold, respectively, as compared to the original OC43-HR2P. Together, these results establish an important framework and platform for subsequent development of EK1 as a potential therapeutic (table S2).\nEK1 peptide potently inhibits multiple CoV cell-cell fusion and blocks various pseudotyped and live CoV infection\nSL-CoVs, including WIV1, Rs3367, and SHC014 CoVs, all manifest potential for human infection (9&#8211;11). To further assess the breadth of fusion inhibitory activity, as demonstrated by EK1, we established cell-cell fusion assays mediated by the S protein of these SL-CoVs (fig. S2). Notably, many studies have suggested diversity in the receptor-binding motif (RBM) of SL-CoV&#8217;s RBD (9&#8211;11), which plays an important role in binding the cellular receptor and serves as an effective target site for the development of CoV-specific antibodies or vaccines (32). Specifically, the RBM of SHC014 CoV has only 80% similarity and 64% identity to that of SARS-CoV, and an antibody targeting the SARS-CoV RBD could not prevent infection mediated by the SHC014 S protein (9&#8211;11, 17). In contrast, the HR1 and HR2 sequences of SL-CoV are 100% identical to those of SARS-CoV (fig. S2B). Correspondingly, EK1 exhibited greater inhibitory activity than the autologous peptide SARS-HR2P against cell-cell fusion mediated by the S protein of the three SL-CoVs tested (Fig. 2, F to H), while an EK1-scrambled peptide showed no inhibitory activity on cell-cell fusion mediated by any of these CoV S proteins (Fig. 2, A to H).The pseudovirus assay is a good model to mimic the process of virus entry into the target cell and has been widely used in previous studies to assess the inhibitory activity of antiviral agents against related CoV infection (33). We also used here pseudovirus assays to assess the inhibitory activity of EK1 against different pseudotyped HCoVs. In a MERS-CoV pseudovirus infection assay, both EK1 and MERS-HR2P showed inhibitory activity with IC50 values of 0.26 and 1.06 &#956;M, respectively (Fig. 2I). Similarly, EK1 and SARS-HR2P also effectively blocked SARS-CoV pseudovirus infection with IC50 values of 2.23 and 2.81 &#956;M, respectively (Fig. 2J). Meanwhile, EK1 demonstrated antiviral activity against pseudotyped 229E, NL63, and OC43 infection with IC50 values of 3.35, 6.02, and 1.81 &#956;M, respectively (Fig. 2, K to M). Consistent with the results from cell-cell fusion assays, both EK1 and SARS-HR2P inhibited pseudotyped Rs3367 virus infection with IC50 values of 2.25 and 3.05 &#956;M, respectively, and prevented WIV1 pseudovirus infection with IC50 values of 2.10 and 2.73 &#956;M, respectively (Fig. 2, N and O). In contrast, EK1 exhibited no inhibitory activity against VSV-G&#8211;mediated viral infection (Fig. 2P). The EK1-scrambled peptide showed no inhibitory activity on infection by any of these pseudoviruses (Fig. 2, I to P). As previously reported (10), we failed to assemble a pseudotyped SHC014 and thus did not perform pseudovirus infection assays on this virus. Collectively, pseudovirus infection by all HCoVs and SL-CoVs could be efficiently blocked by EK1, further indicating that EK1 has broad-spectrum antiviral activity against infection by pan-HCoVs, including &#945;-HCoVs, &#946;-HCoVs, and SL-CoVs.Blam-Vpr assay is a sensitive model to characterize the inhibitory mechanism and activity of antiviral agents on virus&#8211;cell membrane fusion. This assay has been widely used for assessing the membrane fusion activity of various viruses, including HIV-1, Ebola virus, and henipavirus (34&#8211;37). Here, we developed the Blam-Vpr assays for MERS-CoV and SARS-CoV to further clarify the fusion inhibitory mechanism of EK1. We found that EK1 effectively inhibited the entry of MERS-CoV Blam-Vpr virions and SARS-CoV Blam-Vpr virions into Huh-7 cells and ACE2-293 T cells, respectively, in a dose-dependent manner (fig. S2D), thus confirming that EK1 does act as an HCoV fusion inhibitor.Subsequently, we further evaluated the inhibitory activity of EK1 against live HCoV infection, including MERS-CoV, OC43, 229E, and NL63. We found that EK1 could effectively inhibit the infection and replication of these HCoVs at cellular level in a dose-dependent manner (Fig. 2, Q to T). EK1 even surpassed the autologous OC43-HR2P (IC50 = 0.93 &#956;M) or MERS-HR2M2 [IC50 = 0.23 &#956;M; which is an optimized MERS-HR2P peptide (33)] against OC43 or MERS-CoV with IC50 values of 0.62 and 0.11 &#956;M, respectively. Meanwhile, EK1 also exhibited equally effective inhibitory activity against 229E and NL63 infections with IC50 values of 0.69 and 0.48 &#956;M, respectively, compared to 229E-HR2P (IC50 = 0.56 &#956;M) and NL63-HR2P (IC50 = 0.14 &#956;M) (Fig. 2, S and T). Unfortunately, because of the global prohibition of handling live SARS-CoV in most virology laboratories, we could not test the inhibitory activity of EK1 against SARS-CoV replication.\nEK1 exhibits broad anti-HCoV activity in vivo\nThe respiratory tract is the primary target tissue of HCoV infection. To investigate the preclinical potential of the EK1 peptide, we administered it via an intranasal route and assessed its distribution within the respiratory tract. We observed the fluorescent signals of Cy5-EK1 mainly in the upper and lower respiratory tract of mice (Fig. 3, A and B). Compared with those from PBS-treated mice, the lungs obtained from Cy5-EK1&#8211;treated mice exhibited significantly higher fluorescence signals (P < 0.0001, n = 3) (Fig. 3, C and D). Together, these results suggest that EK1 (intranasal) can widely be distributed in the whole respiratory tract and be enriched in the lung. We also noted that Cy5-EK1 administered intranasally could be detected in significant amounts in several extrapulmonary organs, including the liver, kidney, and spleen, in some animals, suggesting that EK1 could enter into the blood circulation and other organs (fig. S3, A and B). This observation suggests that intranasal administration of EK1 could also be beneficial for multiorgan infection or systemic infection of HCoVs, both of which are common in MERS-CoV infection.Next, we assessed the protective effect of EK1 in vivo on OC43 and MERS-CoV infection mouse models. In the OC43-infected mouse model, we treated newborn mice with EK1 at a dose of 5 mg/kg or with PBS 30 min before or after challenge with HCoV-OC43 at 102 TCID50 (50% tissue culture infectious dose). Body weight of mice in the viral control group decreased, starting from 5 days postinfection (dpi), and mice succumbed to infection by 10 dpi with 100% mortality (Fig. 3, E to G). In contrast, the final survival rate of mice in the EK1 prophylactic and therapeutic groups was 100 and 66.7%, respectively (Fig. 3E), and their body weight either appeared normal (prophylactic) or rapidly recovered at 16 dpi (therapeutic) (Fig. 3F). Meanwhile, we tested the viral titers in brains of mice of all groups at 5 dpi. Infectious virus was readily detectable in the viral control group, whereas infectious virus titers were below the limit of detection (2 log TCID50/g) in the EK1 prophylactic mice or very low in the EK1 therapeutic mice (Fig. 3G). However, the infectious virus titers in the brains of mice that died during EK1 therapeutic treatment were as high as those in the brains of viral control mice without EK1 treatment, while the viral titers in the brains of survival mice with EK1 therapeutic treatment and in those of normal control mice were both undetectable (fig. S2E). Consistently, the brains of mice that died during EK1 therapeutic treatment exhibited similar histopathological changes as those of the viral control mice, i.e., similar amount of vacuolation, degeneration, and infiltration (fig. S2F). In contrast, the brains of the survival mice with EK1 therapeutic treatment and those of the normal control mice showed no apparent histopathological changes (fig. S2F).To evaluate the prophylactic and therapeutic potentials of EK1 against MERS-CoV infection, we took advantage of the well-characterized transgenic (Tg) mice globally expressing human dipeptidyl peptidase IV (DPP4) viral receptor. We treated mice with 200 &#956;g of EK1 or with PBS 30 min before or after challenge with MERS-CoV at 104 TCID50. Likewise, the weight loss of Tg mice treated with EK1 before or after MERS-CoV challenge was insignificant and rapidly recovered at 16 dpi, while untreated mice progressively lost significant weight before succumbing to infection within 9 dpi with 100% mortality (Fig. 3, H to J). In contrast, the survival rates in the EK1 prophylactic and therapeutic groups were 100 and 75%, respectively (Fig. 3, H to J). When the yield of infectious viruses in lungs was used as the end point for assessing the efficacy of EK-1 on MERS-CoV infection, we were unable to recover any infectious virus from both EK1 prophylactic and therapeutic groups, in sharp contrast to the untreated controls (Fig. 3J). Together, these results indicated that the peptide EK1 has broad and potent prophylactic and therapeutic efficacy against HCoV infections.\nThe safety and low immunogenicity of EK1\nSafety is obviously very important for the development of EK1 in clinical applications, and therefore, we first tested its in vitro cytotoxicity on various target cells. EK1 is not cytotoxic at concentrations up to 1 mM, which is more than 200 times higher than its IC50 for inhibiting any HCoV S-mediated cell-cell fusion and pseudovirus entry (fig. S3C). We then further investigated its cytotoxicity in vivo. We continuously administered mice with PBS, low-dose EK1 (20 mg/kg), or high-dose EK1 (100 mg/kg) by intranasal route every day for 1 week, and we recorded the body weight changes for the following 2 weeks (fig. S3D). The EK1-treated mice in both high- and low-dose groups lived normally, with no apparent difference in weight gain/loss observed as compared to PBS-treated mice. Meanwhile, we used enzyme-linked immunosorbent assay (ELISA) to measure EK1-specific antibodies in sera of mice 2 weeks after intranasal administration. No EK1-specific antibodies were detected in either the EK1-treated group or in the PBS-treated group (fig. S3E).On the basis of the distribution of EK1 by intranasal administration (fig. S3, A and B), we postulated that EK1 could penetrate the air-blood barrier to enter blood circulation and become enriched in some important organs, such as the liver and kidney. Therefore, we further examined levels of alanine aminotransferase (ALT; fig. S3F) and creatinine (fig. S3G) in the sera of mice. ALT and creatinine showed no significant difference (P > 0.05) at all time points between EK1- and PBS-treated groups, suggesting that nasal application of EK1 at high or low doses did not affect mouse hepatic and renal function.We further compared the potential histopathological changes of EK1- and PBS-treated groups 4 weeks after administration. The hematoxylin and eosin staining of lung, liver, kidney, and spleen sections from mice treated with EK1 at different doses exhibited no pathological abnormality, when compared to control mice treated with PBS (fig. S3H). None of these organs showed evidence of cell degeneration, necrosis, or infiltration of inflammatory factors. Hence, EK1 appears to be generally safe via nasal application.\nStructural basis of EK1 pan-CoV inhibitory activity\nTo investigate the structural basis for the pan-CoV inhibitory effect of EK1 peptide, we crystallized EK1 in complex with HR1 peptides from three representative HCoVs, including the most pathogenic SARS-CoV and MERS-CoV in &#946;-HCoVs and 229E in &#945;-HCoVs, the HR1 of which is 14 amino acids longer than those of &#946;-HCoVs.Crystal structures of HR1(MERS)-L6-EK1, HR1(SARS)-L6-EK1, and HR1(229E)-L6-EK1 were solved by molecular replacement and refined to final resolutions of 3.3, 3.3, and 2.2 &#197;, respectively (table S3). In all three structures, EK1 snugly fits into the hydrophobic groove formed between two adjacent HR1 helices in an oblique and antiparallel manner (Fig. 4, A to C), producing a 6-HB structure resembling the 3HR1-3HR2 postfusion state of corresponding HCoVs (fig. S4). Notably, the binding sites of EK1 coincide with those of native HR2s (fig. S4). These results indicate that the presence of EK1 would preclude binding of HR2 onto their corresponding 3HR1 core, thereby blocking formation of the 3HR1-3HR2 6-HB, which is an indispensable step during host-viral membrane fusion. Hence, administration of the EK1 peptide would block cellular entry of those HCoVs (Fig. 2).In all three structures, the EK1 peptide adopts a mixed secondary structure conformation (Fig. 4, A to C). The central region of EK1 folds into a five-turn helix, which packs against two neighboring HR1 helices via extensive hydrophobic interactions (3HR1 cores of different HCoVs are illustrated as electrostatic surfaces, while residues on EK1 that are involved in hydrophobic packing are shown as stick models; residues L12, E15, M16, L19, A22, I23, L26, S29, and Y30 on EK1 locate within the five-turn helix of EK1 and form strong hydrophobic interactions with the 3HR1 cores; Fig. 5A) and a few electrostatic or polar interactions (residues E13, E15, K18, E20, E27, E28, and Y30 locate within the five-turn helix region of EK1 and interact with HR1 residues through side chain&#8211;to&#8211;side chain hydrophilic interactions; Fig. 6A). The rest of EK1 adopts an extended conformation, except that one extra turn is formed at the C-terminal end of EK1 (around E35) in the HR1(229E)-L6-EK1 structure (Fig. 6, right panel). In the extended region of EK1, polar interactions between side chain (HR1 residues) and main chain (EK1 residues) dominate (compare Fig. 6, B and A). Most of these side chain&#8211;to&#8211;main chain polar interactions cluster at either end of the EK1 helical region, which interweave into extensive H-bond networks and likely help secure the EK1 helical region in the correct register (Fig. 6B). Hydrophobic residues in the extended region of EK1 also insert their bulky side chains into hydrophobic pockets on the surface of the 3HR1 cores (residues L2, I5, V7, L10, I31, and L36; Fig. 5A), further strengthening the adhesion of the EK1 extended region onto the 3HR1 cores.Both hydrophobic pockets and ridges exist on the surface of 3HR1 cores (Fig. 5A). Correspondingly, we observed two kinds of hydrophobic interactions between the EK1 and 3HR1 cores. In particular, certain EK1 residues insert their hydrophobic side chains into pockets on the 3HR1 cores (Fig. 5A, shown as orange stick models and hereafter named as &#8220;burying residues&#8221;), and other EK1 residues pack their side chains against hydrophobic ridges on the 3HR1 cores (Fig. 5A, shown as yellow stick models and hereafter named as &#8220;ridge-packing residues&#8221;). Note that the residues on HR1 that mediate hydrophobic interactions with the burying and ridge-packing residues on EK1 are conserved across all HCoVs [Fig. 5B (bottom panel), HR1 residues shaded with orange and yellow], and most of the HR1 residues involved in polar and electrostatic interactions with EK1 also manifest high levels of similarity across all HCoVs [Fig. 5B (bottom panel), HR1 residues boxed in cyan and red]. Thus, EK1 would be able to form similar hydrophobic and hydrophilic interactions with HR1s from OC43, HKU1, and NL63, as we observed in the homology models of HR1(OC43)-L6-EK1, HR1(HKU1)-L6-EK1, and HR1(NL63)-L6-EK1 (fig. S5). Together, extensive and highly conserved hydrophobic and hydrophilic interactions between EK1 and 3HR1 cores endow EK1 with the ability to bind the 3HR1 cores from different HCoVs and, hence, the capability of blocking the association of different HR2s onto their corresponding 3HR1 cores (Fig. 2).The EK1 peptide outcompetes self-derived HR2 of MERS-CoV in both cell-cell fusion and pseudovirus infection assays. We therefore also compared the interactions between EK1 and MERS-HR1 (Figs. 5A and 6, left panel) with those between MERS-HR2 and MERS-HR1 (fig. S6A). Most hydrophobic interactions between EK1 and MERS-HR1 are similar to those between MERS-HR2 and MERS-HR1 [compare Fig. 5A (left panel) and fig. S6A (left panel)]. Nevertheless, EK1 formed more side chain&#8211;to&#8211;side chain hydrophilic interactions with MERS-HR1 [compare Fig. 6A (left panel) and fig. S6A (middle panel)] than did MERS-HR2, which likely accounts for its better IC50 in cell-cell fusion and pseudovirus infection assays (Fig. 2).Similar to HR1 from &#945;-HCoVs, sequence alignment across HCoVs revealed that a 14&#8211;amino acid insertion also exists in the HR2 region of &#945;-HCoVs (Fig. 1C). Such an insertion renders the HR2 helices of &#945;-HCoVs four turns longer than those of &#946;-HCoVs. The helical region of EK1 is only five turns long, much shorter than the nine-turn helix in 229E-HR2 (38). Nevertheless, the burying residues in the extended region of EK1 and their equivalents in 229E-HR2 all neatly insert their side chains into corresponding hydrophobic pockets (fig. S6C), although these residues are located in extended regions in EK1, while their equivalents are within &#945;-helical regions in 229E-HR2 (fig. S6C). As a consequence, the hydrophobic interactions between EK1 and 229E-HR1 are not compromised at all in comparison to those between 229E-HR2 and 229E-HR1 [compare Fig. 5A (right panel) and fig. S6B (left panel)]. Furthermore, a total of 12 polar and electrostatic interactions occur between EK1 and 229E-HR1, including five side chain&#8211;to&#8211;side chain and seven side chain&#8211;to&#8211;main chain interactions [Fig. 6, A and B (right panel)]. Together, these extensive hydrophobic and hydrophilic interactions preserve the high affinity of EK1 toward long HR1 HCoVs. That EK1 can form 6-HB structures with both short (SARS-CoV and MERS-CoV) and long (229E) HR1s highlights its broad structural compatibility in accommodating HR1s from different HCoVs, thus consolidating its broad-spectrum inhibitory effect against pan-CoVs.\nDISCUSSION\nNo effective and broad-spectrum anti-HCoVs drugs or vaccines are currently available in the clinic. Notwithstanding, the RBD of the S protein has been proposed as a promising target for the development of specific antibodies and vaccines (39, 40). For example, the RBD-specific antibody CDC2-C2 exhibits inhibitory activity against MERS-CoV infection. Nevertheless, administration of the single-antibody CDC2-C2 could result in the emergence of escape mutations in MERS-CoV RBD (41). Unfortunately, the CoV S RBD is hypervariable throughout evolution, which has led to marked difference in host receptor usage in different HCoVs. Even when the same host receptor is used by different HCoVs, they frequently target different binding sites on the host receptor (42). Therefore, specific RBD-targeting antibodies or vaccines inevitably lack broad-spectrum activity against HCoV infection. For example, Menachery et al. (9) found that a SARS-CoV RBD-specific antibody could not protect mice from infection by the chimeric virus with the SHC014 S proteins, although SHC014 has high homology with SARS-CoV and can bind the same host receptor ACE2. The lag time between emerging human CoV outbreaks and development of new prophylactic treatments or vaccines is of concern. Thus, there is an urgent need for the development of new, broad-spectrum drugs that target conserved sites in currently circulating and future emerging HCoVs so as to prepare for future outbreaks of yet unknown HCoVs.In the early 1990s, a series of peptides derived from the HIV-1 gp41 HR2 (or CHR) domain, such as SJ-2176 (19), DP-178 (later T20) (20), and C34 (21), were reported to have highly potent inhibitory activity against HIV-1 gp41-mediated membrane fusion and HIV-1 infection with IC50 values at low nanomolar levels. Subsequently, numerous virus fusion inhibitory peptides overlapping the HR2 sequences of class I membrane fusion proteins from other enveloped viruses, including RSV (22), Ebola virus (23), paramyxoviruses SV5 (24), and Nipah virus (25), were also reported. In 2003, Bosch et al. (26) discovered that a soluble HR2 peptide derived from the MHV S protein HR2 region inhibited virus cell entry, suggesting that the S glycoprotein of CoV is a class I fusogen with the ability to form 6-HB during the S protein&#8211;mediated membrane fusion process. In 2004, the Bosch (43) and Jiang (18) groups independently reported that peptides derived from the HR2 region of SARS-CoV S protein could interact with the peptides from its HR1 region to form 6-HB and inhibited SARS-CoV S protein&#8211;mediated membrane fusion and SARS-CoV infection with moderate potency. In 2014, Lu et al. (27) reported that peptide derived from the MERS-CoV S-HR2 could competitively inhibit 6-HB formation, thereby preventing fusion of the virus with host cells.In our previous studies, we found that the MERS-CoV&#8211;specific fusion inhibitor MERS-HR2P could not block SARS-CoV pseudovirus infection and did not display any broad-spectrum inhibitory activity. Thus, it was unclear whether 6-HB was a good target site for the development of a broad-spectrum anti-HCoV inhibitor. To identify a pan-HCoV fusion inhibition target site and preferably also to develop a potential pan-HCoV inhibitor against infection of multiple HCoVs in the human respiratory tract, we successfully established multiple HCoV S-mediated cell-cell fusion assays to determine the cross-inhibitory spectrum between HR1Ps and HR2Ps, which are derived from the HR1 and HR2, respectively, of HCoVs. Unexpectedly, we found that OC43-HR2P harbored broad-spectrum binding activity to all HR1Ps and manifested fusion inhibitory activity against all HCoV S-mediated cell-cell fusion. We then further optimized the OC43-HR2P to improve its antiviral activity and solubility. The optimized peptide EK1 exhibited broad and potent fusion inhibitory activity against infection by HCoVs and SL-CoVs on various in vitro models, i.e., cell-cell fusion assays and pseudotyped or live viral infection assays. We assessed the prophylactic and protective effects of EK1 through intranasal administration on OC43 and MERS-CoV mouse models. EK1 exhibited effective preventive and protective effects in these mouse models with an acceptable in vivo safety profile. Furthermore, the in vivo pharmacokinetic profiling and safety studies helped us acquire useful data for potential human clinical trials in the future.The potency of HR2-derived peptides in inhibiting corresponding virus-cell fusion varies significantly. For example, anti&#8211;HIV-1 peptide T20 is about 900-fold more potent than the anti&#8211;SARS-CoV peptide SC-1, yet only 30-fold more potent than the anti&#8211;MERS-CoV peptide HR2P (18, 27, 44). It is known that HIV-1 and SARS-CoV each enter their target cells mainly through plasma and endosomal membrane fusion, respectively, while MERS-CoV could infect its target cells via both plasma and endosomal membrane fusion (18, 27, 44). The vast differences seen in potency thus suggest that only a limited number of the EK1 peptides can get into the endosome to inhibit virus-cell fusion. Therefore, increasing the cell permeability of the fusion inhibitory peptides is expected to enhance the potency of these peptides. We have recently reported that addition of hydrocarbon stapling or a palmitic acid group (C16) to the MERS-CoV fusion inhibitory peptides significantly improves their antiviral potency and pharmacokinetic properties (45, 46). In the near future, we will use the similar approaches to improving the antiviral activity of EK1.Recent studies have reported that several emerging SL-CoVs, including WIV1, SHC014, and Rs3367, have the potential to infect humans with high virulence by further evolution or direct gene recombination with SARS-CoV (9, 11). We found that the HR sequences of these SL-CoVs are the same as those of SARS-CoV, and we predicted that EK1 could prevent infection by these SL-CoVs. As expected, EK1 efficiently inhibited SL-CoV S protein&#8211;mediated cell-cell fusion. Consistent with previous studies (9, 11), we herein also failed to establish a SHC014 pseudovirus using an HIV-1 backbone vector; therefore, we assessed the antiviral effect of EK1 only on WIV1 and Rs3367 pseudovirus infection and found that EK1 exhibited effective inhibitory activity against these two SL-CoVs. In addition, some animal CoVs within the CoV family are very close to HCoV, such as bovine coronavirus (BCoV) and MHV. The HR sequences of these CoVs are very similar to those of OC43, indicating that peptide EK1 will likely also be effective on these viruses.One intriguing question is why only EK1 and the HR2 from OC43 manifested &#8220;pan-CoV&#8221; inhibitory activity, while HR2 from other HCoVs did not. HR2s from &#945;-HCoVs are substantially longer than those from &#946;-HCoVs (Fig. 1C), and docking of HR2s from 229E and NL63 onto the 3HR1 cores of &#946;-HCoVs resulted in severe steric clashes (figure not shown). It is thus understandable why the HR2s from 229E and NL63 would not have broad-spectrum activity toward &#946;-HCoVs (Fig. 1D).At the same time, sequence alignment between EK1 and HR2s from OC43, SARS-CoV, and MERS-CoV provided clues on the very limited inhibition breadth of SARS-HR2 and MERS-HR2. Although most residues that mediated hydrophobic (shaded orange and yellow) and side chain&#8211;to&#8211;side chain hydrophilic interactions (boxed in cyan) are conserved between EK1 and different HR2s, sequence variability does exist at positions V7, Y30, and some other ridge-packing residues (fig. S7A). We thus focused our analysis on these residues and their surrounding environments in different HCoVs&#8217; 3HR1 cores. In both solved structures of MERS(HR1)-EK1 and SARS(HR1)-EK1, V7EK1 fits its side chain neatly into the hydrophobic pockets on these two 3HR1 cores (fig. S7B, second and fourth panels). Nevertheless, in the previously reported structure of the SARS-CoV fusion core, A1156SARS, the equivalent of V7EK1 in SARS-CoV, would occupy the same cavity loosely (fig. S7B, third panel), while T1257MERS, the equivalent of V7EK1 in MERS-CoV, could not fully bury its side chain owing to the polar nature of the Thr residue (fig. S7B, first panel). Thus, Val, in EK1 and OC43-HR2, is more effective in fitting the hydrophobic cavities at this position than Ala (seen in SARS-HR2) or Thr (seen in MERS-HR2) (fig. S7B).In total, we observed four ridge-packing interactions between the EK1 and 3HR1 cores. The ridge to which S29Y30EK1 binds is surrounded by polar and positively charged residues in MERS-CoV and SARS-CoV but surrounded by polar and negatively charged residues in 229E [fig. S7C (top panel), side chains of surrounding residues are shown as stick models]. At this position, the polar pair &#8220;Ser-Tyr&#8221; in EK1 and OC43-HR2 appears to be more suitable than the &#8220;Ser-Leu&#8221; pair from SARS-HR2 in accommodating the opposite electrostatic environments in different HCoVs. Moreover, the aromatic ring of Tyr could also form additional polar interactions with HR1 side chains in different HCoVs (Fig. 6A). Likewise, the ridges to which E15M16EK1 binds also exhibited better shape complementarity to Met (in EK1 and OC43-HR2) than to Ile (in SARS-HR2). Collectively, it seems that EK1 and OC43-HR2 have more optimal amino acid choices at all the abovementioned critical positions, which likely accounts for their unique pan-CoV inhibitory activity.In its natural state, the S protein that is present on the CoV surface is inactive. After receptor binding target cell proteases activate the S protein by cleaving the exposed enzyme target sites, leaving the S2 subunit free to mediate viral fusion and entry. CoV can enter the target cell via two pathways: one is the endocytosis pathway and the other is direct fusion on the cellular surface. For example, when SARS-CoV enters the target cell through the endocytosis pathway, its S protein can be cleaved and activated by the pH-dependent cysteine protease cathepsin L in the endosome. On the other hand, recent studies have consistently reported that the SARS-CoV S protein can also be cleaved and activated by transmembrane protease serine 2 (TMPRSS2) and human airway trypsin-like protease, which are located on the cell surface, thus activating and allowing the S protein to mediate SARS-CoV infection at the plasma membrane (9&#8211;11). In accord with such a finding, Matsuyama et al. (47) reported that the HR2 peptide efficiently inhibited SARS-CoV entry into cells, while lysosome-tropic reagents failed to inhibit at all. Similarly, TMPRSS2 also has the capacity to promote the entry of MERS-CoV through bypassing the endocytosis pathway and directly activating its S protein on the cellular surface (48). Recently, several studies have reported that TMPRSS2 is highly expressed on human respiratory epithelial cells surface and was even associated with several CoV receptors, such as ACE2 and DPP4 (48&#8211;50). Hence, the plasma membrane fusion pathway seems a preferred choice for HCoV infection in the human respiratory tract. Consistently, our previous study found that MERS-CoV&#8211;specific fusion inhibitor HR2PM2 effectively inhibited MERS-CoV infection in vivo by intranasal administration (33). Other studies have also reported that the current clinical isolates of 229E and OC43 are very sensitive to cell surface TMRRSS2 but not to endosomal cathepsins (51, 52). Overall, for current circulating HCoVs or emerging HCoVs, the cell surface fusion pathway in human respiratory tract would appear to be very important. Therefore, peptide fusion inhibitors and the strategy of intranasal administration are excellent choices for preventing HCoV infection via the airway, which is a key site for HCoV to rapidly establish infection and widely spread to other organs.Currently, circulating HCoVs pose a potential threat to humans; moreover, it is almost certain that other zoonotic CoVs will be transmitted to humans in the future. The availability of HCoV-specific drugs with broad-spectrum inhibitory activity is therefore important for the prevention and control of a future HCoV epidemic. The EK1 fusion inhibitor peptide targeting the conserved site in the spike HR1, but not the hypervariable RBD region, has potent and broad inhibitory activity against multiple HCoV infections. EK1 through nasal administration exhibited effective and broadly anti-HCoV activity with a satisfactory safety profile in vivo. Hence, EK1 is a promising candidate for further development as an antiviral agent against infection of multiple HCoVs, especially for use in infants and the elderly, as well as immunocompromised patients, who would be more vulnerable to HCoV infections (53&#8211;56). Meanwhile, this study provides clues and methods for the development of peptide fusion inhibitors with potency and breadth in inhibiting infections by other highly pathogenic enveloped viruses with class I membrane fusion proteins, such as Ebola and Marburg viruses, Hendra and Nipah viruses, and influenza viruses.\nMATERIALS AND METHODS\n\nCells, viruses, and peptides\nCell lines Huh-7, Vero E6, HCT-8, Calu-3, A549, LLC-MK2, 293 T, and 293 T/ACE2 were obtained from American Type Culture Collection (ATCC; Manassas, VA, USA). All cell lines were cultured in Dulbecco&#8217;s modified Eagle&#8217;s medium (DMEM) with 10% fetal bovine serum (FBS). The HCoV-OC43 (VR-1558) and HCoV-229E (VR-740) strains were obtained from the ATCC. MERS-CoV-EMC/2012 was originally provided by H. Feldmann (National Institutes of Health, National Institute of Allergy and Infectious Diseases, Rocky Mountain Laboratories, Hamilton, MT) and R. A. Fouchier (Erasmus Medical Center, Rotterdam, Netherlands). PMD2.G-VSV-G, pcDNA3.1-MERS-S, pcDNA3.1-SARS-S, pcDNA3.1-WIV1-S, pcDNA3.1-Rs3367-S, pcDNA3.1-SHC014-S, pNL4-3.Luc.R-E, and pAAV-IRES-EGFP plasmids and the NL63 strain (Amsterdam) were preserved in our laboratory. The DNA sequence of the S protein of OC43 with a deletion of 17 amino acids in the C terminus was synthesized. DNA encoding 229E-S or NL63-S-18 (containing a C-terminal 18&#8211;amino acid deletion) was provided by F. Li. All peptides were synthesized by KareBay Biochem with a purity of >95% (tested by high-performance liquid chromatography). The EK1-scrambled sequence was &#8220;LKVLLYEEFKLLESLIMEILEYQKDSDIKENAEDTK.&#8221;\nCoV phylogenetic analysis\nPhylogenetic trees were constructed using the MEGA6.06. Accession numbers used for phylogenetic analysis are as follows: BCoV (KX982264.1), PEDV (NC_003436), HCoV-229E (AAK32191.1), HCoV-NL63 (AVA26873.1), MHV (AC_000192.1), IBV (KY421672), HCoV-OC43 (CAA83661.1), HCoV-HKU1 (DQ415904), BtCoV-HKU4 (NC_009019.1), BtCoV-HKU5 (NC_009020), SW1 (EU111742.1), MERS-CoV (AID55097.1), SARS-CoV (ABD72979.1), BtSCoV-SHC014 (KC881005), BtSCoV-Rs3367 (KC881006), BtSCoV-WIV1 (KF367457), BtCoV-HKU9 (EF065516), HKU11 (FJ376620), HKU12 (NC_011549.1), HKU13 (NC_011550.1), and PDCoV (KX022605.1).\nExpression and purification of fusion protein HR1-L6-EK1\nThe coding sequence of EK1 peptide was individually fused to the 3&#8242; end of the HR1 domain from SARS-CoV, MERS-CoV, and HCoV-229E (residues 892 to 970, 984 to 1062, and 785 to 873, respectively) through a six&#8211;amino acid linker (L6: SGGRGG). The resulting sequences encoding different HR1-L6-EK1 fusion proteins were then subcloned into a modified pET-28a vector, which contains a His6-SUMO tag upstream of the multiple cloning site. The resulting constructs, pET-28a- His6-SUMO-HR1-L6-EK1, were then expressed in Escherichia coli BL21 (DE3) at 16&#176;C overnight in LB medium. We initially purified these fusion proteins using His-Talon resin (Clontech). Eluted fractions from the His-Talon column were then mixed with Ulp1 enzyme [1:100 (w/w)] and dialyzed against buffer A [20 mM tris-HCl (pH 8.0), 150 mM NaCl, and 1 mM dithiothreitol] at 4&#176;C overnight. After SUMO tag cleavage, the samples were reloaded onto the His-Talon column, and flow-through fractions containing untagged HR1-L6-EK1 were pooled, concentrated, and gel-filtered in buffer B [20 mM tris (pH 8.0) and 150 mM NaCl] on a HiLoad 16/60 Superdex 75 (GE Healthcare) column. Peak fractions that contain HR1-L6-EK1 trimer were pooled and concentrated to 10 mg/ml through centrifugation (EMD Millipore).\nCrystallization and structure determination\nInitial crystallization conditions were screened using the sitting-drop vapor diffusion method on a Mosquito crystallization robot (TTP Labtech) at 20&#176;C. Diffraction-quality crystals of the HR1(SARS)-L6-EK1 fusion protein were obtained using the hanging-drop vapor diffusion method by mixing equal volumes (0.5 &#956;l) of protein solution (5 mg/ml) and reservoir solution [0.2 M Li2SO4, 0.1 M bis-tris (pH 5.5), and 25% PEG3350]. Diffraction-quality crystals of the HR1(MERS)-L6-EK1 and HR1(229E)-L6-EK1 fusion proteins were obtained through similar methods from 0.2 M KSCN (pH 7.0), 20% PEG3350, and 0.05 M MgCl2 and 0.1 M HEPES (pH 7.5) and 30% PEG550MME, respectively. Diffraction data were collected at beamline BL19U1 of Shanghai Synchrotron Radiation Facility (SSRF), China, and processed with the HKL3000 program (57). A summary of the data collection statistics is provided in table S3. The structure of HR1(SARS)-L6-EK1 was solved by molecular replacement, as implemented in the PHASER program of PHENIX (58). The programs used a SARS-CoV fusion core structure [Protein Data Bank (PDB): 2bez] as the search model. For structure determination of HR1 (MERS)-L6-EK1 and HR1(229E)-L6-EK1, the MERS-CoV fusion core [PDB: 4mod] and SARS-CoV fusion core [PDB: 2bez] were used as search models. The structural models were further improved by cycles of manual building and refinement using the COOT (59) and PHENIX (58) programs. The quality of these models were analyzed with MolProbity (60). A summary of the structure refinement statistics is also given in table S3. The figures were all prepared using the PyMOL program (The PyMOL Molecular Graphics System, Version 2.1, Schr&#246;dinger LLC). The electrostatic calculations were performed with PDB2PQR (61).\nHomology model building\nModels for HR1(OC43)-L6-EK1, HR1(HKU1)-L6-EK1, and HR1(NL63)-L6-EK1 were derived by homology modeling using Swiss Model website (62). The template for HR1(OC43)-L6-EK1 and HR1(HKU1)-L6-EK1 were obtained from the crystal structures of HR1(SARS)-L6-EK1 and HR1(MERS)-L6-EK1, and the template for HR1(NL63)-L6-EK1 proteins was obtained from the crystal structure of HR1(229E)-L6-EK1. To relax and stabilize the interaction between EK1 and corresponding HR1, initial models were optimized by performing energy minimization, followed by a 5-ns molecular dynamics simulation using Schr&#246;dinger Suite 2017-4 (www.schrodinger.com). The simulation systems were solvated with full-atom TIP3P water, containing Cl&#8722; and Na+ ions at a concentration of 0.15 M to mimic physiological ionic strength. During the simulation, temperature T and pressure P were kept constant, at 310 K and 1 atm, respectively.\nCircular dichroism spectroscopic analysis\nCircular dichroism spectra (195 to 260 nm) were collected on a J-815 spectropolarimeter (JASCO Inc., Japan) to evaluate the secondary structure of the peptides and their complexes (63, 64), individual peptides or complexes dissolved in PBS with the final concentration at 10 &#956;M. Thermal denaturation of peptide complexes was monitored from 20&#176; to 100&#176;C at 222 nm with a thermal gradient of 5&#176;C/min. The midpoint of the Tm values was acquired by JASCO software utilities.\nInhibition of pseudotyped virus infection\nA pseudovirus bearing CoV S protein or VSV-G protein and a defective HIV-1 genome that expresses luciferase as reporter was produced in 293 T cells, as previously described (27), and its titer was quantitated by using HIV-1 p24 ELISA. The pseudovirus was then used to infect target Huh-7 cells (or ACE2/293 T cells for pseudotyped SARS-CoV) (10 4 per well in 96-well plates) in the presence or absence of the test peptide at the indicated concentration. Twelve hours after infection, culture medium was refreshed and then incubated for an additional 48 hours, followed by washing cells with PBS, lysing cells with lysis reagent (Promega), and transferring the cell lysates to 96-well Costar flat-bottom luminometer plates (Corning Costar) for the detection of relative light units using the Firefly Luciferase Assay Kit (Promega) and an Ultra 384 luminometer (Tecan).\nEstablishment of multiple HCoV S-mediated cell-cell fusion\nAs previously described (27), 293 T effector cells were transfected with plasmid pAAV-IRES-EGFP encoding the EGFP (293 T/EGFP cells) or plasmid pAAV-IRES-S-EGFP encoding the corresponding HCoV S protein (293 T/HCoV S/GFP cells) as the effector cells. Huh-7 cells, expressing various HCoV receptors on the membrane surface, were used as target cells, as described below.1) MERS-CoV S-mediated cell-cell fusion: Effector cells (293 T/MERS-CoV/GFP) and target cells (Huh-7 cells) were cocultured in DMEM containing 10% FBS, at 37&#176;C for 2 hours;2) 229E S-mediated cell-cell fusion: Effector cells (293 T/229E/GFP) and target cells (Huh-7 cells) were cocultured in DMEM containing 10% FBS, at 37&#176;C for 4 hours;3) SARS-CoV and SL-CoV S-mediated cell-cell fusion: Effector cells (293 T/SARS-CoV/GFP or 293 T/SL-CoV/GFP) and target cells (Huh-7 cells) were cocultured in the presence of trypsin (80 ng/ml) in DMEM without FBS, at 37&#176;C for 4 hours;4) OC43 or NL63 S-mediated cell-cell fusion: Effector cells (293 T/HCoV-OC43/GFP or 293 T/HCoV-NL63/GFP) and target cells (Huh-7 cells) were cocultured in the presence of trypsin (80 ng/ml) in DMEM without FBS, at 37&#176;C for 4 hours.Five fields in each well were randomly selected for counting the fused and unfused cells. The fused cells are at least twice as large as the unfused cells, and the fluorescence intensity in the fused cell became weak as a result of the diffusion of enhanced green fluorescent protein (EGFP) from one effector cell to target cells (see figs. S1D and S2C). The percentage of cell-cell fusion [(number of the fused cells/number of the fused and unfused cells) &#215; 100%] was then calculated.\nInhibition of HCoV S-mediated cell-cell fusion\nThe inhibitory activity of a test peptide on HCoV S-mediated cell-cell fusion was determined, as previously described (27). Briefly, effector cells (293 T/S/GFP) and target cells (Huh-7 cells) were cocultured in the presence or absence of a test peptide at the indicated concentrations for fusion. After counting the fused and unfused cells, the percentage of cell-cell fusion was calculated, as described above. The percent inhibition of cell-cell fusion was calculated using the following formula as described elsewhere (27): [1 &#8722; (E &#8722; N)/(P &#8722; N)] &#215; 100%. &#8220;E&#8221; represents the percentage of cell-cell fusion in the experimental group. &#8220;P&#8221; represents the percentage of cell-cell fusion in the positive control group, where 293 T/HCoV S/EGFP cells were used as effector cells, to which only PBS was added. &#8220;N&#8221; is the percentage of cell-cell fusion in negative control group, where 293 T/EGFP cells were used as effector cells. The IC50 was calculated using the CalcuSyn software provided by T. C. Chou (65). Samples were tested in triplicate, and all those experiments were repeated twice.\nHCoV virion\u2013based fusion assay\nAn HCoV virion&#8211;based fusion assay was performed, as described elsewhere (34, 35). Briefly, Huh-7 cells and ACE2-293 T cells were used as target cells for the entry of MERS-CoV Blam-Vpr virions and SARS-CoV Blam-Vpr virions, respectively. These target cells were cultured at 37&#176;C for 5 hours in six-well plates (with virions containing BlaM-Vpr equivalent to 80 ng of p24-Gag per well) in the presence or absence of EK1 at the indicated concentrations. The cells were washed with PBS, resuspended in 500 &#956;l of DMEM, and then incubated with CCF4-AM substrate at room temperature for 2 hours, as described by the manufacturer (Invitrogen, Germany). Last, the cells were monitored via flow cytometry. After the virion fusion with the target cell, the CCF4-AM (emission at 520 nm) substrate could be cleaved by BlaM-Vpr into CCF4 (emission at 447 nm). Flow cytometric data were collected with a BD FACSDIVA and analyzed with FlowJo.\nInhibition of live HCoV\u2019s replication\nThe inhibitory activity of peptides against OC43 replication in HCT-8 cells was assessed, as described elsewhere (66). Briefly, 100 TCID50 of OC43 was mixed with a test peptide at graded concentration and incubated at 37&#176;C for 30 min. The mixture was then applied in triplicate onto the monolayer of HCT-8 cells grown in a 96-well microtiter plate. On day 5 after infection, viral titer in the culture medium was tested, and TCID50 was calculated on the basis of the cytopathic effect (CPE) (67). The inhibitory activity of the tested peptide against 229E replication in A549 cells and NL63 replication in LLC-MK2 cells was evaluated in a similar way, as described above.The inhibitory activity of peptide against MERS-CoV replication was tested in Calu-3 cells using a modified standard microneutralization assay, as previously described (68). Briefly, 60 &#956;l of a serially twofold diluted peptide was incubated with 60 &#956;l (120 TCID50) of MERS-CoV in MEM medium supplemented with 2% FBS (M-2 medium) in duplicate wells of 96-well plates for ~60 min at room temperature. One hundred microliters of the peptide/MERS-CoV mixtures was then transferred into confluent Calu-3 cells grown in 96-well plates. Wells of Calu-3 cells cultured with M-2 medium with and without virus were included in these assays as positive and negative controls, respectively. While the advanced CPE of Calu-3 cells could develop within 24 to 36 hours in response to MERS-CoV infection at higher multiplicities of infection (MOIs) of 1 or 0.1 (69), we did not observe any prominent formation of CPE until ~ 60 to 72 hours after infection at an estimated MOI of ~ 0.001. Hence, to more accurately measure the efficacy of peptide inhibitors against MERS-CoV infection, we harvested the supernatants at 72 hours and quantified infectious virus titers by the standard Vero E6&#8211;based infectivity assays and expressed the titers as log10 TCID50/ml.\nMouse infection studies\nPregnant Balb/c mice (18 days) purchased from the Department of Laboratory Animal Science, Fudan University were separated into four groups after delivery of their offspring. Eleven newborn mice were chosen for each group. Mice in the prevention and treatment groups were intranasally administered peptide (5 mg/kg in 2 &#956;l of PBS) 30 min before or after intranasal challenge with a viral dose of 102 TCID50 (in 2 &#956;l DMEM). Mice in the viral control group and the normal control group were intranasally administered with 2 &#956;l of PBS 30 min before viral challenge or without viral challenge. Mouse survival rate and body weight variations were recorded up to 2 weeks after infection (70). On day 5 after infection, five mice in each group were randomly selected for euthanasia to collect and assess the viral titer in mouse brain.Briefly, mice expressing human DPP4 were randomly separated into three groups (n = 6): viral control, prophylactic group, and therapeutic group. In the viral control and therapeutic groups, each mouse was challenged intranasally with 104 TCID50 [i.e., 1000 LD50 (median lethal dose)] of MERS-CoV in a volume of 60 &#956;l, 30 min after viral challenge, and then treated via the intranasal route with 200 &#956;g of EK1 (in 50 &#956;l of PBS) or 50 &#956;l of PBS, respectively. Prophylactic group mice were pretreated via the intranasal route with 200 &#956;g of EK1 per mouse 30 min before viral challenge. Challenged mice were monitored daily for weight loss and mortality. In the interim, two mice were randomly euthanized (two of six) in each group at day 2 after challenge to determine the lung virus titers by CPE on Vero E6 and expressed as TCID50/ml.\nIn vivo fluorescence imaging\nSix Balb/c female mice (8 weeks old) were randomly assigned to two groups and nasally administered with 20 &#956;g of Cy5-EK1 in 50 &#956;l of PBS (n = 3) or 50 &#956;l of PBS without peptide (n = 3) as control for background fluorescence measurement. One hour later, mice were imaged for distribution of Cy5-EK1 by the IVIS Lumina K Series III in vivo imaging system (PerkinElmer, Waltham, MA, USA). Mice were euthanized by intraperitoneal injection of sodium pentobarbital to obtain the lungs, livers, kidneys, spleens, and hearts for imaging and measuring their fluorescence value. The relevant radiant efficiency (P s&#8722;1 cm&#8722;2 sr&#8722;1) (&#956;W&#8722;1 cm2) was then calculated by Living Image 4.4 software (56, 71, 72).\nCytotoxicity assay\nCytotoxicity of the peptides to the cells (293 T, 293 T/ACE2, Huh-7, A549, LLC-MK2, and Calu-3 cells) was tested by using the Cell Counting Kit-8 (CCK-8; Dojindo, Kumamoto, Japan). Briefly, each cell type was seeded into the wells of a 96-well microtiter plate (10 4 per well) and incubated at 37&#176;C for 12 hours, replacing medium with DMED containing EK1 at graded concentrations. After incubation at 37&#176;C for 2 days, CCK-8 solution (10 &#956;l per well) was added, followed by an additional incubation for 4 hours. The absorbance was measured at 450 nm.\nIn vivo safety of EK1 through intranasal administration\nAs previously described (27), female Balb/c mice (8 weeks old) were assigned randomly to three groups (n = 5) and continuously administered with PBS, low-dose EK1 (20 mg/kg), or high-dose EK1 (100 mg/kg) by intranasal route every day for 1 week. The body weight changes were monitored for the following 2 weeks. ALT and creatinine in the sera of each group of mice were measured using the ALT and creatinine assay kits (NJJCBIO, Nanjing, China) before the first treatment and 1, 3, and 5 days after the final treatment. The titer of immunoglobulin G sera to EK1 in each group of mice was evaluated by ELISA 2 weeks after the final treatment. Four weeks after administration, mice in each group mice were euthanized to harvest the lungs, livers, kidneys, and spleens for hematoxylin and eosin staining. The animal studies were approved by the Institutional Laboratory Animal Care and Use Committee at Fudan University (20180302-019).\nStatistical analyses\nAnalyses of independent data were performed by Student&#8217;s unpaired two-tailed t test. Statistical analyses were carried out using GraphPad Prism 6.0. P values less than 0.05 were considered significant; *P < 0.05, **P < 0.01, ***P < 0.001, and ****P < 0.0001. The IC50 was calculated using the CalcuSyn software (65).\nSupplementary Material\n",
        "pmid": "30989115",
        "title": "A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike",
        "journal_title": "Science Advances",
        "authors": "Shuai Xia;Lei Yan;Wei Xu;Anurodh Shankar Agrawal;Abdullah Algaissi;Chien-Te K. Tseng;Qian Wang;Lanying Du;Wenjie Tan;Ian A. Wilson;Shibo Jiang;Bei Yang;Lu Lu",
        "doi": "10.1126/sciadv.aav4580"
    },
    {
        "keywords": "gamma coronavirus;avian coronavirus;infectious bronchitis virus;genome;recombination",
        "abstract": "Recombination in the family Coronaviridae has been well documented and is thought to be a contributing factor in the emergence and evolution of different coronaviral genotypes as well as different species of coronavirus. However, there are limited data available on the frequency and extent of recombination in coronaviruses in nature and particularly for the avian gamma-coronaviruses where only recently the emergence of a turkey coronavirus has been attributed solely to recombination. In this study, the full-length genomes of eight avian gamma-coronavirus infectious bronchitis virus (IBV) isolates were sequenced and along with other full-length IBV genomes available from GenBank were analyzed for recombination. Evidence of recombination was found in every sequence analyzed and was distributed throughout the entire genome. Areas that have the highest occurrence of recombination are located in regions of the genome that code for nonstructural proteins 2, 3 and 16, and the structural spike glycoprotein. The extent of the recombination observed, suggests that this may be one of the principal mechanisms for generating genetic and antigenic diversity within IBV. These data indicate that reticulate evolutionary change due to recombination in IBV, likely plays a major role in the origin and adaptation of the virus leading to new genetic types and strains of the virus.",
        "body": "Introduction\nAvian infectious bronchitis virus (IBV) is a gamma-coronavirus in the family Coronaviridae, the order Nidovirales, and the genus Coronavirus that causes a highly contagious upper-respiratory disease of domestic chickens. In layer type birds it can cause a drop in egg production and some strains are nephropathogenic. Infectious bronchitis remains one of the most widely reported respiratory diseases of chickens worldwide despite the routine usage of attenuated live vaccines to control the disease. Control of IBV is difficult because there is little to no cross-protection between the numerous different serotypes of the virus.Infectious bronchitis virus is an enveloped, single-stranded, positive-sense RNA virus with a genome length of approximately 27 kb. The 3&#8242; end of the genome encodes four structural proteins; spike (S), envelope (E), membrane (M) and nucleocapsid (N) as well as several non-structural proteins [1]. The S glycoprotein of IBV forms projections on the surface of the virion. Spike is post-translationally cleaved into S1 and S2 subunits with the S1 subunit forming the outermost portion and S2 forming a stalk-like structure that is embedded in the viral membrane. The S1 subunit contains hypervariable regions that play a role in attachment to host cell receptors, and it contains conformationally-dependent virus-neutralizing and serotype-specific epitopes [2,3]. Spike is also involved in membrane fusion and viral entry into the host cell. The E and M proteins are integral membrane proteins involved in assembly of the virus. The N protein is closely associated with the viral genome and plays a role in replication. The 5&#8242; two-thirds of the genome, approximately 21 kb, encodes two polyproteins 1a and 1ab. A minus one frame-shift mechanism is used to translate the 1ab polyprotein. The polyproteins are post-translationally cleaved into 15 non-structural proteins (nsps), nsp 2&#8211;16 (IBV does not have an nsp1) that make up the replication complex. Key nsps encoded, include a papain-like protease 2 (PLP2) within nsp 3, a main protease (Mpro) within nsp 5, and the RNA-dependent RNA-polymerase (RdRp) within nsps 11 and 12.Genetic diversity in coronaviruses is due to adaptive evolution driven by high mutation rates and genetic recombination [4]. High mutation rates are attributed to minimal proof reading capabilities associated with the RdRp. Recombination is thought to be due to a unique template switching &#8220;copy-choice&#8221; mechanism during RNA replication [5]. Evidence of recombination among strains of IBV has been observed both experimentally and in the field [6&#8211;11]. The emergence of several alpha- and beta-coronaviruses has been attributed to recombination [12,13] but only recently was recombination shown to be the mechanism behind the emergence of a novel gamma-coronavirus, turkey coronavirus (TCoV) [14]. Although &#8220;hot spots&#8221; of recombination in the genome of IBV have been reported [9,15], a thorough study of recombination using multiple different strains across the entire genome has not been conducted.In this study we sequenced and analyzed the entire genome of eight IBV strains that represent different serotypes that have not been previously sequenced, and we compared these sequences with other gamma-coronavirus full-length genome sequences available in GenBank for evidence of recombination [16]. Different serotypes of field viruses and vaccine type viruses were selected to provide a wide variety of sequences potentially capable of contributing gene fragments to recombinants.\nResults and Discussion\n\nSequence Analysis\nThe full-length genomes of eight isolates of IBV were sequenced at 5&#215; to 10&#215; coverage, and the consensus sequences were assembled. The genome size (see the end of the 3&#8242;UTR in Table 1), organization of the genome, and the location and size of the open reading frames (ORFs) are listed in Table 1 for each of the viruses. The gene order is the same for all the viruses examined; 5&#8242;UTR-1a/ab-spike-3a-3b-Envelope-Membrane-4b-4c-5a-5b-Nucleocapsid-3&#8242;UTR. In addition, the genomes for CAV/CAV56b/91, DE/DE072/92, FL/FL18288/71, Mass/H120, Iowa/Iowa97/56 and JMK/JMK/64 contain ORF 6b between nucleocapsid and the 3&#8242;UTR.The full-length genomes were aligned and phylogenetic trees were constructed using the Neighbor-joining, Minimum Evolution, Maximum Parsimony and UPGMA programs in MEGA4 [17]. The trees all had similar topology and bootstrap support, and a representative tree is shown in Figure 1. The feline coronavirus FCoV/FIPV/WSU-79-1146 and the beluga whale virus BelugaWhaleCoV/SW1/08 were included as out-groups. The wild bird viruses isolated from a munia (MuniaCoV/HKUY13/09), thrush (ThrushCoV/HKU12/09) and bulbul (BulBulCoV/HKU11/09) formed a unique clade, which is not surprising as this group might represent a new coronavirus genus provisionally designated Deltacoronavirus [18]. The remaining viruses separated into clades consisting of IBV isolates from the US and vaccine viruses, TCoV isolates, an IBV isolate from West Africa and IBV isolates from China and Taiwan.Vaccines for IBV used in commercial poultry include the serotypes Mass, Conn, DE and Ark. The PeafowlCcV/GD/KQ6/03, CK/CH/LSD/051/06 and CK/CH/ZJ971/97 strains from China grouped with Mass type viruses indicating that they are closely related, which is not surprising since Mass type vaccines are used in China. The overall percent similarities between the various strains are listed in Supplemental Table 1. All IBV genomes examined are greater than 80% similar at the nucleotide level.\nRecombination Analysis\nRecombination among coronaviruses reduces mutat onal load, creates genetic variation, and can result in the emergence of new strains [19]. However, evolutionary history is traditionally represented using a strictly bifurcating phylogenetic tree, which implies that nce two lineages are created they subsequently never interact with each other. When evolutionary events such as reassortment, horizontal gene transfer or recombination occur, reticulations among the phylogenetic tree branches can result. Using the Neighbor-net analysis we observed evidence of networked relationships (represented by boxes, in Figure 2) among the analyzed sequences. Since the boxes only imply the possibility of recombination, we conducted a pairwise homoplasy index (PHI) test, which showed a significant difference in the compatibility between closely linked sites (p < 0.0001) supporting the occurrences of recombination among the viruses [20].The Recombination Detection Program 4 (RDP4) [21,22] w s used to identify recombination breakpoint positions in full-length IBV genome sequences and the data for 50 of a total 135 unique transferred fragments with statistical support of p &#8804; 1 &#215; 10&#8722;12 are listed in Table 2. The full-length genomes available in the database for CK/CH/EP3, CK/CH/p65, and Mass/Beaudette were excluded from the analysis because they are viruses not found in the field. The recombination programs can be used to detect recombination without reference sequences, and our analysis was conducted without regard to date of isolation because that information was not available for some of the viruses. Although the programs attempt to identify major and minor parent sequences contributing to each recombinant, the data reported herein only represents sequences in other viruses that are most closely related to the sequence surrounding the transferred fragment (major sequence) and the sequence closely related to the transferred fragment (minor sequences) and doesn&#8217;t imply origin or source of the transferred fragment. In many cases, the transferred fragment has undergone mutations making it difficult to identify all the endpoints for the major and minor sequences. In addition, some of the transferred fragments overlap suggesting that recombinations have occurred between recombinant viruses.Twenty-five IBV strains were examined and the viruses with the most transferred fragments in Table 2 are CAV/56b/91 and Mass/H52 both with 8 fragments, and CK/CH/LSD/051/06 and GA98/0470/98 both with 7 fragments. The strains with the fewest transferred fragments are Iowa/Iowa97/56 and TW/2575/98 with only 2 transferred fragments and the CK/CH/BJ/97, Holte/Holte/54, and NGA/A116E7/06 strains with only 1 transferred fragment. The Ark/Ark-DPI-p11/81 and Ark/Ark-DPI-p101/81 strains are the same virus that was passaged 11 and 101 times in embryonated eggs, respectively. Both viruses share identical transferred fragments indicating that they have identical recombination history. In addition, Conn/Conn46/66 and Conn/Conn46/91 share the same recombination history (4 identical transferred fragments). The Conn/Conn46/66 field virus was used to produce an attenuated live vaccine, which is currently used in commercial poultry. Viruses that share the same recombination history are likely derived from the same parent virus suggesting that Conn/Conn46/91 is reisolated Conn vaccine derived from the Conn/Conn46/66 virus. The FL/FL18288/71 virus also shares all 4 transferred fragments with the Conn viruses, however; FL/FL18288/71 and Conn viruses are different serotypes suggesting that FL/FL18288/71 is a field virus that emerged due to point mutations accumulating in spike over time rather than from recombination.All 6 of the transferred fragments in CK/CH/ZJ971/97 are identical to all 6 of the transferred fragments in vaccine strain Mass/H120, providing compelling evidence that CK/CH/ZJ971/97 is reisolated Mass/H120 vaccine. That observation was also reported by Zhang et al. [23]. It is interesting that Mass/H52 (8 transferred fragments) and Mass/H120 (6 transferred fragments) share only 5 identical transferred fragments. The Mass/H52 and Mass/H120 viruses were isolated circa 1955 in the Netherlands and it is widely accepted that H stands for Holland, but it actually stands for Houben, the owner of the broiler farm where the viruses were isolated [24]. It is thought that Mass/H120 was derived from Mass/H52 but the actual relationship between the viruses is not certain. Our data indicates that they are not necessarily parent and progeny but they are closely related.The Gray/Gray/60 and JMK/JMK/64 viruses share 99.7% nucleotide similarity across the entire genome and have 4 identical transferred fragments with JMK/JMK/64 having one additional fragment located in the 5&#8242;UTR, which is not found in Gray/Gray/60. It is well known that the Gray/Gray/60 virus is nephropathogenic, whereas the JMK/JMK/64 virus is strictly respirotropic. Perhaps sequence differences in the 5&#8242;UTR, which is involved in replication of the viral genome, play a role in the different pathobiologies observed for these viruses.There is evidence that some transferred fragments in field viruses come from vaccines. As an example, CK/CH/LSD/051/06 has 3 of 7 and 2 of 7 transferred fragments in common with vaccine strains Mass/H52 and Mass/H120, respectively. In addition, the only fragments that USA viruses have in common with the viruses from China and Taiwan are fragments also associated with Mass type vaccines, which are used in both regions, providing further evidence that some of the fragments in field viruses come from vaccines. That result and the observation in Figure 1 that the viruses separated into clades based on geographic location also supports the conclusion that USA viruses have not recombined with Asian viruses.A difference in the order of taxa in phylogenetic trees constructed from different regions of the genome is further evidence of recombination [25]. The ordering of taxa in sequential trees [26,27] was conducted and inconsistent phylogenetic relationships were observed for all of the examined virus strains across the entire genome, indicating a substantial amount of recombination (data not shown). There is a high number of breakpoints in the 1a region of the genome and immediately upstream of the S gene, which has been previously shown to be a &#8216;hot spot&#8217; for recombination [9]. A phylogenetic compatibility matrix constructed at the 70% bootstrap level for 250 bp sequence fragments at 100 bp intervals also showed that recombination breakpoints were distributed throughout the IBV genomes (data not shown).To determine recombination hot and cold spots, a recombination breakpoint distribution plot (Figure 3) was generated in RDP4 using a 200 nt window and 1,000 permutations [21]. No global hot-spot regions were observed in the 95% and 99% confidence thresholds (dotted lines at the top of the graph). The detectable recombination breakpoint positions are shown at the top of the figure and were distributed throughout the genome with a relatively high number clustered just upstream of the S gene. That region also had the highest breakpoint count within the 99% local hot/cold-spot confidence interval. A high number of breakpoints were also observed in the 1a region of the genome; nsp 2, nsp 3, and nsp 16, in the envelope and matrix protein genes and in a small area near the 3&#8242;UTR. Table 3 shows that nsp2, nsp3, nsp16 and spike genes were associated with the greatest number of transferred fragments, which is consistent with the location and number of breakpoints in Figure 3.Recombination in the 1ab ORF area, which encodes the nonstructural proteins involved in the viral replication complex, has the potential to alter the pathogenicity of the virus [28]. The nsp 2 contains hydrophobic residues that likely anchor the replication complex to the Golgi [29]. The nsp 3 encodes the protease PLP2 which cleaves nsps 2, 3, and 4 and an area with ADP-ribose 1&#8242;-phosphatase (ADRP) activity. The protease PLP2 has been shown to have deubiquinating-like activity [30] and also to be a type I interferon (IFN) antagonist [31]. Changes in the amino acid composition of this area could affect the ability of the virus to replicate in a variety of cell types. The ADRP region of nsp 3 is conserved among coronaviruses [32,33], and a recent study suggested a biological role for the coronavirus ADRP in modulating the expression of pro-inflammatory immune modulators such as tumor necrosis factor alpha and interleukin-6 [34]. Recombination in this area could alter the pathogenicity of the virus by modulating host cytokine expression. The nsp16 is reported to be an S-adenosyl-L-methionine (AdoMet)-dependent RNA (nucleoside-2&#8242;O)-methyltransferase (2&#8242;O-MTase) responsible for capping the viral mRNA nascent transcripts [32]. An alteration in the efficiency of this protein could profoundly decrease not only viral replication but also pathogenicity. The spike glycoprotein of IBV on the surface of the virus plays a role in attachment to host cell receptors, membrane fusion and entry into the host cell. It also contains conformationally-dependent epitopes that induce virus-neutralizing and serotype specific antibodies [2,3]. We and others [6&#8211;8,10] have observed a relatively high number of breakpoints in and immediately upstream of spike, and changes to this region of the genome can result in the emergence of new genotypes and serotypes of IBV as well as new avian coronaviruses (i.e., TCoV). The envelope and matrix proteins are as ociated with virus assembly, and changes in those proteins could reduce the efficiency of virus particle formation and subsequent transmission of the virus. The 3&#8242;UTR is involved with binding of the viral RdRp and viral genome replication. Changes to the 3&#8242;UTR could affect replication efficiency and thus virulence of the virus.\nConclusions\nIn this study, evidence was obtained that recombination is occurring among avian coronavirus IBV isolates across their entire genome. Every sequence included in the analysis was recognized as a potential recipient of horizontally acquired sequences at some point in its viral evolutionary past. The nsp2, nsp3, nsp16 were associated with the greatest number of transferred fragments. In addition, the area immediately upstream of the spike gene had the highest number of recombination breakpoints. Breakpoints in the 1ab polyprotein gene have the potential to alter pathogenicity of the virus, and breakpoints near or in spike have the potential to lead to the emergence of new serotypes of IBV or new coronaviruses. Although the spike region determines the serotype of the virus, the remainder of the genome may be a mosaic of sequence fragments from a variety of gamma-coronaviruses. The only evidence of a gamma-coronavirus possibly recombining with an alpha or beta-coronavirus was the discovery of the mosaic nature of the SARS-coronavirus genome [35]. Although this type of recombination is possible it appears to be rare in nature.In this study, we characterized recombination in the full-length genomes of avian gamma-coronavirus IBV strains from around the world. Our bioinformatic analysis was similar to a previous study on enteroviruses [36] and we found that recombination in IBV is more extensive than formerly thought, involving regions across the entire genome. Our data suggests that reticulate evolution due to a high frequency of recombination in IBV, likely plays a major role in the generation of new serotypes of the virus. The characterization, distribution and frequency of recombination breakpoints are important information that will further our understanding of the mechanisms behind the diversity and evolution of these viruses so that better control methods can be developed.\nMaterials and Methods\n\nViruses and Viral RNA Extraction\nAll of the viruses sequenced in this study (Table 4), were propagated in 9&#8211;11 day-old specific-pathogen-free (SPF) embryonated eggs as described [37]. Total RNA was isolated from 200 &#956;L of allantoic fluid collected from the infected eggs using the High Pure RNA Isolation Kit (Roche Applied Science, Mannheim, Germany) following the manufacturer&#8217;s instructions.\nRT-PCR Amplification and Sequencing\nThe amplification reactions were carried out using strand displacement RT-PCR or one step RT-PCR. Strand displacement RT-PCR uses a random (at the 3&#8242; end) primer and an amplification primer. The sequence of the random primer was (AGCGGGGGTTGTCGAATGTTTGANNNN) and the sequence of the amplification primer was (AGCGGGGGTTGTCGAATGTTTGA). The RT-PCR reaction was carried out using the TaKaRa RNA LA PCR kit (Takara Bio. Inc., Otsu, Shiga, Japan) according to the manufacturer&#8217;s protocol. A DNA Engine Peltier thermocycler (Bio-Rad Laboratories Inc., Hercules, CA, USA) was used for the RT reaction, which included an RNA denaturing step at 65 &#176;C for 10 min followed by 30 &#176;C for 10 min, 42 &#176;C for 60 min, 99 &#176;C for 5 min, and 5 &#176;C for 5 min. The PCR reaction was run on the same machine as the RT step and included a one-time initial denaturation step of 94 &#176;C for 2 min, followed by 30 cycles of 94 &#176;C for 30 s, 60 &#176;C for 30 s and 72 &#176;C for 3 min.The PCR products were agarose gel purified using the QIAquick gel extraction kit (Qiagen, Valencia, CA, USA) according to the manufacturer&#8217;s protocol. The PCR products were cloned into the TOPOXL vector using the TOPOXL cloning kit (Invitrogen, Carlsbad, CA, USA) according to manufacturer&#8217;s protocol to prepare cDNA libraries for sequencing.Plasmid DNA from the libraries of the cloned cDNA fragments for each virus was isolated using an alkaline lysis method modified for the 96-well format and incorporating both Hydra and Tomtek robots. Sequencing reactions were performed using the BigDye&#8482; Terminator&#174; Cycle Sequencing Kit Version 3.1 (Applied Biosystems, Foster City, CA, USA) and MJ Research (Watertown, MA, USA) thermocyclers. Sephadex filter plates were used to filter each reaction into Perkin-Elmer MicroAmp Optical 96-well plates. A 1/12-strength sequencing reaction on an ABI 3730 was used to sequence each clone from both the 5&#8242; and 3&#8242; ends.Primers for one-step RT-PCR were specifically designed for each virus (Supplemental Table 2). Viral RNA was amplified using the Titan One Tube RT-PCR kit (Roche Diagnostics, Indianapolis, IN, USA) following manufacturer&#8217;s instructions. A DNA Engine Peltier Thermocycler (Bio-Rad Laboratories, Inc., Hercules, CA, USA) was used for the RT-PCR reaction, which had the following steps: one cycle of 42 &#176;C for 60 min and 95 &#176;C for 5 min, followed by 10 cycles of 94 &#176;C for 30 s, 50 &#176;C for 30 s, and 68 &#176;C for 1 min 30 s, and then 25 cycles of 94 &#176;C for 30 s, 50 &#176;C for 30 s, 68 &#176;C for 1 min and 30 s adding 5 s with each cycle.The resulting PCR products were agarose gel purified using the QIAquick gel extraction kit (Qiagen, Valencia, CA, USA) according to the manufacturer&#8217;s protocol. The resulting cDNA was sequenced using ABI Prism BigDye Terminator Cycle Sequencing Ready Reaction Kit (Applied Biosystems, Foster City, CA, USA) following the manufacturer&#8217;s protocol. The reactions were prepared for sequencing by centrifugation through either a Centri-Sep column (Applied Biosystems, Foster City, CA, USA) or using the Edge system (EdgeBio, Gaithersburg, MD, USA) plate. The samples were sequenced at the Georgia Genomics Facility (University of Georgia, Athens, GA, USA).\nGenome Assembly and Analysis\nChromatogram files and trace data were read and assembled using SeqMan Pro, and genome annotation was conducted with SeqBuilder (DNASTAR, Inc., Madison, WI, USA). Each sequence was aligned to a representative genome; Mass/Mass41/41 (GenBank accession #AY851295), or CAL99/CAL99/99 (GenBank accession #AY514485) as a backbone for genome assembly.Whole genome analyses were generated and phylogenetic trees constructed with the Neighbor-Joining method with 1000 bootstrap replicates as well as with Minimum Evolution, Maximum Parsimony and UPGMA methods [17].\nGenBank Accession Numbers\nVirus genome sequences generated in this study were submitted to GenBank and assigned the following accession numbers: CAV/CAV56b/91 (GU393331), DE/DE072/92 (GU393332), FL/FL18288/71 (GU393333), Gray/Gray/60 (GU393334), Mass/H120 (GU393335), Holte/Holte/54 (GU393336), Iowa/Iowa97/56 (GU393337), JMK/JMK/64 (GU393338).GenBank accession numbers for full-length sequences used as reference in this study are: Mass/Mass41/41 (AY851295), Mass/H52 (EU817497), Ark/Ark-DPI-p11/81 (EU418976), Ark-DPI-p101/91 (EU418975), CAV/CAV99/99 (AY514485), CK/CH/EP3 (DQ001338), CK/CH/p65 (DQ001339), Mass/Beaudette (NC_001451), NGA/A116E7/06 (FN430415), ITA/90254/05 (FN430414), TW/TW2575/98 (DQ646405), CK/CH/SC021202/02 (EU714029), CK/CH/ZJ971/97 (EU714028), CK/CH/BJ/97 (AY319651), CK/CH/SAIBK (DQ288927), CK/CH/LSD/051/06 (EU637854), CK/CH/DY07/07 (HM245923), CK/CH/CQ04-1/04 (HM245924), GA98/GA98/98 (GQ504723), PeafowlCcV/GD/KQ6/03 (AY641576), PartridgeCoV/GD/S14/03 (AY646283), TCoV/IN-540/94 (EU022525), TCoV/MN-ATCC (EU22526), TCoV/VA-74/03 (GQ427173), TCoV/TX-GL/01 (GQ427174), TCoV/IN-517/94 (GQ427175), TCoV/TX-1038/98 (GQ427176), TCoV/Canada-MG10 (EU095850) BulbulCoV/HKU11/09 (FJ376619), ThrushCoV/HKU12/09 (FJ376621), MuniaCoV/HKU13/09 (FJ376622), BelugaWhaleCoV/SW1/08 (NC_010646), FCoV/FIPV/WSU-79/1146 (DQ010921).\nDetection of Networked Relationships and Recombination Break Points\nWe used Neighbor-net analysis to examine the IBV genomes for evidence of networked relationships and the pairwise homoplasy index (PHI) in SplitsTree (Version 4, Simmonics, University of Warwick, Coventry, UK) [20,38,39] to statistically determine the likelihood of recombination. In addition, the IBV genomes were examined for recombination breakpoints using the Recombination Detection Program (RDP4, Version 4, Simmonics, University of Warwick, Coventry, UK) [21,22]. Unless otherwise stated, default settings were used in all of the programs. The specific algorithms used were RDP [40], GENECONV [41], BOOTSCAN/RESCAN [40], MAXIMUM CHI SQUARE [42], CHIMAERA [43], SISCAN [44], and 3Seq [45]. We used more than one method to analyze the data because evaluation of these recombination detection methods using both simulated and empirical data showed that the results from only a single method were not very reliable [46]. Automasking was used for optimal recombination detection. The RDP analysis was run without a reference and a window size of 60, BOOTSCAN window size was increased to 500, MAXCHI and CHIMAERA number of variable sites per window was increased to 120, and the window size and step size for SISCAN was increased to 500 and 20, respectively. The window sizes were increased from their default settings because IBV has a high mutation rate, which can mask recombination signals. Increasing the window size was shown to increase the ratio of recombination signals relative to mutational &#8220;noise&#8221; [47].\nPhylogenic Analysis of Sequential Genome Fragments\nInconsistent phylogenetic relationships between different regions of the viral genome provide further evidence of genetic recombination. Herein, we examined the order of avian gamma-coronavirus IBV strains in phylogenetic trees generated from sequential genome fragments using TreeOrder Scan (Version 1.6, Simmonics, University of Warwick, Coventry, UK) [26,27]. Changes in the tree position of taxa supported at the 70% or greater bootstrap level for a 250 bp sequence window were examined at 100 bp intervals. In addition, a phylogenetic compatibility matrix was constructed and used to examine the frequency and location of recombinations across the entire genome.\nRecombination Site Detection\nPotential recombination sites were identified using the RDP4 software [22] and a breakpoint map was constructed. A breakpoint density plot was then created from this map by moving a 200 nt window 1 nt at a time along the length of the map. The number of breakpoints falling within a window was plotted at the central window position. A 99% (upper) and 95% (lower) confidence threshold for globally significant breakpoint clusters (defined as windows with more breakpoint positions than the maximum found in >95% of the 1,000 permuted plots) was calculated. In addition, 99% and 95% confidence intervals were calculated for local breakpoint clusters (defined as windows with more breakpoint positions than the maximum found in >99% of the windows at that location in 1,000 permuted plots).",
        "pmid": "21994806",
        "title": "Recombination in Avian Gamma-Coronavirus Infectious Bronchitis Virus",
        "journal_title": "Viruses",
        "authors": "Sharmi W. Thor;Deborah A. Hilt;Jessica C. Kissinger;Andrew H. Paterson;Mark W. Jackwood",
        "doi": "10.3390/v3091777"
    },
    {
        "keywords": "Animal model;Coronavirus (CoV);Emerging pathogen;Middle East respiratory syndrome (MERS);Reservoir;BSL, biosafety level;DPP4, dipeptidyl peptidase-4;FDA, Food and Drug Administration;HCoV, human coronaviruses;hDPP4, human dipeptidyl peptidase-4;MERS-CoV, Middle East respiratory syndrome coronavirus;NHP, Nonhuman primates;PI, post-inoculation;RDB, receptor binding domain;SARS-CoV, severe acute respiratory syndrome coronavirus;URT, upper respiratory tract;WHO, World Health Organization",
        "abstract": "Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013&#8211;2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.",
        "body": "Introduction\nOver the past years, outbreaks of zoonotic diseases and growing resistance against antibiotics have emphasized the need for interdisciplinary collaboration between human health, veterinary medicine and environmental sciences, a concept commonly known as &#8220;One health&#8221; [1]. Most of emerging diseases are zoonotic [2]. For instance, the human flu pandemics have originated in domestic animals and wildlife, and have been driven by ecological, behavioral, or socioeconomic changes [3]. In these cases, the reaction time between detection of a new outbreak and application of medical countermeasures are critical in terms of epidemic control. To understand the potential role that animal sources could play in virus dissemination and the epidemiology of the disease, surveillance studies, as well as experimental infections in potential target species, are required. Furthermore, after having identified the novel or re-emerged virus responsible of the outbreak, it is important to rapidly provide an accurate diagnosis as a basis for quarantine measures. It is also imperative to focus on the search for new vaccines and treatments for highly pathogenic viruses, especially for those that represent a threat to human and animal health, particularly livestock.Until the beginning of the last decade, human coronaviruses (HCoV) infections were considered to be restricted to the upper respiratory tract (URT), with low mortality rate, and recognized as the second ranked cause of the common cold after rhinoviruses [4]. However, in the late-2002, the severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in China. It rapidly spread worldwide with more than 8000 causalities and a lethality rate of 10% [5]. Ten years later, a novel HCoV associated with severe pneumonia emerged in the Kingdom of Saudi Arabia [6]. The new CoV was named Middle East respiratory syndrome coronavirus (MERS-CoV), and by March 2014, a total of 207 cases and 45% fatalities were recorded. One month later, only in April 2014, an increase in human cases was registered with at least 217 more infected people and 38 fatalities. More recently, as of December 2016, 1842 cases of MERS-CoV have been reported to the World Health Organization (WHO), including at least 652 deaths [7].Besides coronaviruses, highly pathogenic viruses belonging to other families represent a threat to either human or animal health, or both. One of the most recent examples is the outbreak of Ebola virus (Filoviridae) in West Africa, which started in December of 2013 in Guinea and evolved as the largest Ebola outbreak recorded with more than 28,600 cases [8]. Furthermore, during recent years, outbreaks caused by other emerging viral pathogens from Arenaviridae, Bunyaviridae, and Flaviviridae families among others, disturbed public and private health, social networks and the economies of the affected countries [9], [10]. Prevention and control of emerging and reemerging viral diseases is efficient when several actions are combined: i.e. creating diagnostic networks and surveillance programs, training medical and veterinary staff, informing the population about sanitary measures, and also promoting research on prophylaxis, treatments and on the causative agent pathogenesis. Regarding the last point, animal models are crucial to study the viral and host factors contributing to the disease as well as transmission outcomes of virus infection and to allow pre-clinical testing of antiviral drugs and vaccines. Non-human primates (NHP) are the preferred models for pathogenesis studies, and potential vaccine and treatment testing, as they better translate to humans [11]. However, working with NHP is costly, with limited availability, and raises ethical problems. Therefore small-animal models are usually the first choice for drug screening. The United States Food and Drug Administration's (FDA) Animal Rule provides guidelines concerning the appropriateness of animal models for licensing purposes [12]. Additionally, by controlling the disease in animal reservoirs and/or in intermediate hosts, virus transmission to humans can be significantly reduced [13], [14]. This is particularly true for domestic or feral animals for which efficient vaccines and vaccination strategies can be implemented [15]. Therefore, in cases of new pathogenic virus outbreaks, the search for natural hosts or potential target animals (as opposed to laboratory animals) seems to be relevant not only to implement prophylactic solutions but also to improve the preparedness for an eventual global extension of diseases. Nowadays, this task is rendered possible by the availability of improved biosafety levels 3 and 4 (BSL3 and 4) animal facilities, which can accommodate large animal experimentation with such highly virulent pathogens [16].In this article, the current situation of comprehension on potential hosts for MERS-CoV is reviewed. Based on the coronaviruses experience, benefits and limitations of these species as animal models and transmission studies are discussed.\nAnimal models for MERS research\nSeveral review articles have described and discussed animal models for MERS-CoV infection [17], [18], [19], [20]. In this section, the current status of animal models for MERS disease reproduction is briefly summarized.After the identification of MERS-CoV in 2012 [6], the efforts were directed to develop an animal model to study pathogenesis and to test the efficacy of vaccines and/or treatments in vivo. Similar to SARS-CoV, rhesus macaques have demonstrated susceptibility to MERS-CoV [21], [22], [23]. A work led by Munster demonstrated that the common marmoset is also suitable as a MERS-CoV model [24]. They showed that this model recapitulates the disease observed in humans; therefore, findings in the evaluation of potential therapeutic strategies might be implemented in humans. However, small animals are required for controlled, large and comprehensive studies. While, at first, experiences with SARS-CoV turned out to be very helpful for the research on MERS-CoV, the development of a small animal model for MERS was a more difficult task [18], [19]. Raj and collaborators rapidly identified dipeptidyl peptidase-4 (DPP4) as the functional receptor for MERS-CoV [25], and DPP4 is present in lung cells of many rodents. Thus, rodents were expected to be susceptible for MERS-CoV. However, and as predicted by the crystal structure analysis of the MERS-CoV receptor binding domain (RBD) with the human DPP4 (hDDP4) extracellular domain [26], so far, no rodent model is naturally permissive for MERS-CoV infection. In Syrian hamster, the DPP4 receptor was shown to be expressed on bronchiolar epithelium, but inoculation of MERS-CoV via aerosols or intratracheal routes with different doses did not lead to productive infection [27]. Wild type and immune-deficient mice were also tested for MERS-CoV infection without success [28]. Since then, several groups have been focused on new strategies to develop a small animal model susceptible to MERS-CoV infection. It was found that mouse cells could be made permissive for MERS-CoV when expressing hDPP4. Consequently, the hDPP4 was transduced into mouse lungs using an adenovirus vector, which resulted in animals susceptible to MERS-CoV infection. These mice exhibited pneumonia and extensive inflammatory-cell infiltration with the presence of virus in the lungs [29]. Recently, a transgenic mice model expressing hDPP4, highly susceptible to MERS-CoV infection and able to display systemic lesions, has been developed [30]. As demonstrated for several diseases, transgenic animal models have become an important tool to improve medical research [31]. On the other hand, glycosylation of the murine DPP4 is a major factor impacting the receptor function by blocking the binding to MERS-CoV [32]. Therefore, the modification of the mouse genome to match the sequence in the hDPP4 made this species susceptible to MERS-CoV infection [33]. Accordingly, these newly established mice models are useful to evaluate the efficacy of vaccines and therapeutic agents against MERS-CoV infection [30], [34], [35], [36]. VelocImmune and VelociGene technologies have been used to develop a humanized mouse model for MERS-CoV infection [36]; these methodologies can be also applied for other pathogens in future emerging epidemics.\nMERS-CoV animal reservoir and the role of domestic animals\nResearchers worldwide have identified several animal species which could have a role in the transmission of MERS-CoV to humans (summarized in Fig. 1). Bats have been suggested to be the reservoir for MERS-CoV, but RNA of MERS-like CoVs (and no MERS-CoV) has been found in several bat families (Vespertillionidae, Molosidae, Nycteridae and Emballonuridae) in Africa, America, Asia and Europe [37], [38]. Recently, an experimental infection with Jamaican fruit bats (Artibeus jamaicensis) confirmed that MERS-CoV can replicate in bats [39]. However, MERS-CoV strains causing disease in humans have not been so far identified in bats. The presence of MERS-CoV neutralizing antibodies has been reported in dromedary camels (Camelus dromedaries) [40], [41] and, more recently, a link between MERS-CoV infection in camels and a human infection in Qatar has been suggested [42], [43], [44], [45]. Most importantly, the MERS-CoV strain that caused the 2015 human outbreak was found in dromedary camels; in fact, phylogenetic analyses indicate that MERS-CoV was generated in this species by recombination [46]. In addition, Adney and collaborators have provided experimental evidence to support the role of dromedary camels as a MERS-CoV reservoir [47]. Recently, evidence was provided that other members of the Camelidae family such as alpaca (Vicugna pacos) and llama (Lama glama), are also susceptible to MERS-CoV infection [48], [49], [50]. Field studies with alpacas performed in Qatar confirmed this finding [51]. A recently published experimental study has demonstrated that domestic pigs are also susceptible to MERS-CoV infection, suggesting the possibility of MERS-CoV circulation in other unsuspected animal species such suidae [50]. While DPP4 in silico predictions and in vitro studies suggested that other livestock species such as goats, sheep, or horses could be susceptible to MERS-CoV infection, experimental data suggested the opposite [50]. Each natural and potential host for MERS-CoV infection is discussed below in detail and summarized in Table 1, Table 2.Fig. 1Illustration of the known and potential host range of Middle East respiratory syndrome coronavirus (MERS-CoV). Dromedary camel is a demonstrated reservoir of MERS-CoV, but other species might act as hosts. In red: reported MERS-CoV-seropositive species and/or species in which virus has been naturally detected. In orange: other animal species that might be considered potential hosts since they are experimentally susceptible to MERS-CoV infection. Alpacas fit into both scenarios (red and orange). In black: animals (bats, to date) in which RNA of different sequences of MERS-CoV-like viruses have been found. Continuous arrows represent already described intra- and inter-species transmission events. Discontinuous arrows represent potential inter-species transmission.Fig. 1Table 1Summary of MERS-CoV shedding and presence of virus in different tissues in the potential animal reservoirs for MERS-CoV after experimental inoculation.Table 1SpeciesRoute and dose of inoculationMERS-CoV sheddingMERS-CoV RNA in tissuesMERS-CoV antigen in tissuesInfectious MERS-CoV in tissuesReferencesCamelidsDromedary camels (Camelus dromedarius)IT, IN, conjunctival or IN only; 107 TCID50Viral RNA in NS (1 to 13 dpi)Infectious virus in NS (1 to 6 dpi)URT, LRT, tracheal LN, pulmonary LN, cervical LN, tonsil, PSG, intestine, liver, spleen, kidney, heart, adrenalNasal respiratory epithelial cellsURT, trachea, large bronchus, tracheal LN[47], [53]Alpacas (Vicugna pacos)IN; 107 PFU in 3&#160;mL saline solutionInfectious virus in NS (1 to 5 dpi)URT, tracheaNasal respiratory epithelial cellsND[48], [49]Llamas (Lama glama)IN; 107 TCID50 in 3&#160;mL saline solutionViral RNA in NS (1 to 15 dpi)Infectious virus in NS (1 to 7 dpi)URT, trachea and bronchusNasal respiratory epithelial cellsND[50]Non-camelid domestic speciesDomestic pig (Sus scrofa domesticus)IN; 107 TCID50 in 3&#160;mL saline solutionViral RNA in NS (1 to 10 dpi)Infectious virus in NS (1 to 4 dpi)URT, trachea and bronchusNasal respiratory epithelial cellsND[50]Rabbit (Oryctolagus cuniculus)IN (1&#160;&#215;&#160;106 TCID50), IT (4&#160;&#215;&#160;106 TCID50)Viral RNA in NS (1 to 10 dpi)Infectious virus in NS (1 to 7 dpi)Mainly in URT, LRTNasal respiratory and bronchiolar epithelial cellsND[54]NHPRhesus macaques (Macaca mulatta)IT, OC, oral, IN; 7&#160;&#215;&#160;106 TCID50/IT, 6,5&#160;&#215;&#160;107 TCID50Viral RNA in NS, BAL samples, and few OSURT, lung, mediastinal LNType I and II pneumocytes, alveolar M&#934;Lung[21], [22], [23]Common marmoset (Callithrix jacchus)OC, oral, IT, IN; 5&#160;&#215;&#160;106/IT, 5&#160;&#215;&#160;107 TCID50Viral RNA in NS and OSURT, lung, mediastinal LN, bloodType I pneumocytes, alveolar M&#934;Nasal mucosa, trachea, lung[24], [55]Abbreviations: BAL, bronchoalveolar lavages; dpi, days post inoculation; IN, intranasal; IT, intratracheal; LN, lymph node; LRT, lower respiratory tract; M&#934;, macrophages; ND, non-determined; NHP, non-human primates; NS, nasal swabs; OC, ocular; OS, oropharyngeal swabs; PFU, plaque-forming units; PSG, parotid salivary gland; RNA, ribonucleic acid; TCID50, 50% tissue culture infectious dose; URT, upper respiratory tract.Table 2Summary of clinical signs, pathological findings and target cells in tissues of natural and potential reservoir hosts for MERS-CoV infection as experimental animal models.Table 2SpeciesClinical signsGross findingsHistopathological lesionsTarget cells in tissuesReferencesCamelidsDromedary camels (Camelus dromedarius)Mild respiratory disease, nasal dischargeNot presentMultifocal moderate rhinitis, tracheitis and bronchitis with epithelial necrosis. Hyperplasia of lymph nodes and tonsilRespiratory epithelial cells in the URT[47], [53]Alpacas (Vicugna pacos)Not observableNot presentSquamous metaplasia of the epithelium of the turbinate. Hypertrophy and hyperplasia of lymph nodesRespiratory epithelial cells in the URT[48], [49]Llamas (Lama glama)Mild mucus secretion in one nostrilNot presentMild to severe rhinitisRespiratory epithelial cells in the URT[50]Non-camelid domestic speciesDomestic pig (Sus scrofa domesticus)Mild excretion of mucus in the noseNot presentMild to severe rhinitisRespiratory epithelial cells in the URT[50]Rabbit (Oryctolagus cuniculus)Not observableNot presentFocal mild to moderate rhinitis with necrosisRespiratory epithelial cells in the URT[54]NHPRhesus macaquesa(Macaca mulatta)Fever, mild to moderate respiratory diseaseLung congestion and nodules in lungNo extra pulmonary lesionsMultifocal mild-to-moderate interstitial pneumoniaType I and II pneumocytes and alveolar macrophages[21], [22], [23]Common marmoseta(Callithrix jacchus)Mild to severe respiratory diseaseCongestion of bronchiolesDiffuse interstitial infiltration in lower lung lobes, bronchointerstitial pneumoniaType I pneumocytes and alveolar macrophages[24]Abbreviations: NHP, non-human primates; URT, upper respiratory tract.aAlso animal models of disease (translation to human).\nCamelids\nDromedary camels are the main source of MERS-CoV zoonotic transmission (reviewed in [52]). In experimental intranasal inoculations with MERS-CoV, only mild clinical signs (i.e. nasal discharge) with URT infection were observed. Viral RNA was detected in nasal swabs, in upper and lower respiratory tracts, and also in extra-pulmonary tissues (i.e. lymph nodes, tonsil, intestine, liver, adrenal gland, etc.). In contrast, infectious virus was only detected in the URT, trachea, large bronchus and tracheobronchial lymph node. Gross lesions were not observed in dromedary camels, but inflammation in the nasal cavity, trachea and bronchus was present. The virus replication in dromedaries was only detected in epithelial cells in the URT [47], [53].Llamas and alpacas, also known as domestic new world camelids, developed a similar clinical-pathological picture to that of dromedaries after experimental MERS-CoV infection. In both species the virus was inoculated via intranasal route, and either no clinical signs (alpacas) or mild mucus secretion (llamas) was observed. MERS-CoV was detected in nasal swabs, and in the URT and trachea of both llamas and alpacas. None of the species showed lesions macroscopically, but microscopically mild to severe rhinitis was detected in alpacas as well as metaplasia of the epithelium of the turbinate in alpacas. Similar to dromedaries, the epithelial cells in the URT were the main target cells for virus replication. Concomitant to an antibody response, the virus was cleared from the URT 7 to 10&#160;days after experimental infection [48], [49], [50].\nNon-camelid domestic species\nAfter intranasal inoculation of MERS-CoV, only mild excretion of mucus was observed in 6 to 8-week old domestic pigs [50]. Viral RNA was detected in nasal swabs, in the URT, trachea and bronchus. Although gross lesions were not present in pigs, they showed mild to moderate rhinitis, with virus replication observed in the epithelial cells in the URT. Shedding of MERS-CoV was detected in nasal swabs from days 1 to 10 PI, but infectious virus was only detected until day 4 PI. Viral RNA was also detected in the URT, trachea and bronchus [50].New Zealand white rabbits did not exhibit clinical signs or significant gross lesions at necropsy after experimental MERS-CoV inoculation; thus, they were considered an animal model of asymptomatic infection [54]. Similar to pigs, mild to moderate rhinitis with necrosis was observed, and respiratory epithelial cells in the URT were identified as the target cells for MERS-CoV replication. Viral RNA was present in nasal swabs, and upper and lower respiratory tracts. Infectious virus was also detected in nasal swabs up to 7&#160;days PI [54].\nNon-human primates\nThe rhesus macaque was used as the first animal model developed for MERS-CoV infection, showing mild to moderate respiratory disease from day 1 to 4 PI after intratracheal inoculation [21]. Gross lesions were present only in the lung, consisting in congestion and presence of nodules, and the main observed microscopical lesion was interstitial pneumonia. Although MERS-CoV RNA was detected in nasal swabs, bronchoalveolar lavage samples, oropharyngeal swabs, and also in some upper and lower respiratory tract tissue samples, infectious virus was only isolated from the lungs. MERS-CoV replication occurred in type I and II pneumocytes, and viral antigen co-localized with sites of pneumonia [21]. Macaques represent a useful model to study mild MERS-CoV infection because they develop a transient respiratory disease similar to humans.On the other hand, common marmosets exhibited moderate to severe respiratory disease from 1 to 13&#160;days after inoculation of MERS-CoV through multiple routes (ocular, oral, intratracheal and intranasal) [24]. Similar to macaques, gross findings were present only in the lung and correlated with moderate to severe bronchointerstitial pneumonia. MERS-CoV antigen was detected by immunohistochemistry in both type I and II pneumocytes and alveolar macrophages, but the virus replicated only in type I pneumocytes and macrophages. Similar results were observed in marmosets after inoculation of MERS-CoV only via intratracheal route [55]. However, the outcome of MERS-CoV infection in marmosets has been controversial after the publication of a recent study by Johnson and collaborators, which demonstrated no lethality after intratracheal inoculation [56]. The outcome of experimental infections in both species is reviewed in recent publications [17], [18], [19], [20].\nAdvantages and disadvantages of animal hosts used for MERS-CoV experimental infection\nAs described in the previous section, a number of animal species have been described as either natural reservoir (dromedary camel) or potential intermediate hosts of MERS-CoV, each one with its benefits and limitations (Table 3) when used as experimental infection models. Camelids (dromedary camels, alpacas and llamas), non-camelid domestic species (pigs and rabbits), and NHP (rhesus macaques and common marmosets) have been experimentally demonstrated to be susceptible to MERS-CoV infection, but with differences among them [21], [22], [23], [24], [47], [48], [49], [50], [53], [54]. Experiments with dromedary camels, the natural MERS-CoV host, and probably the first target for controlling MERS through vaccination [53], are costly and represent a high security risk for animal caretakers because of the difficulty in handling these animals under appropriate biosafety conditions. The main advantages of using the llama or alpaca models are that both belong to the family Camelidae, have smaller size, more gentle behavior, and are more available at a commercial level than dromedary camels; importantly, specific reagents for immune monitoring have been developed for new world camelids [57]. Therefore, they may be useful surrogates for dromedaries under experimental conditions. However, both models are also quite expensive and require large and complex BSL3 facilities. In contrast to camelids, other domestic species such as pigs and rabbits are readily available, with lower cost and easier handling. Additionally, an extensive panel of specific-immunological reagents is available for these species. When compared to camelids, however, lower MERS-CoV titers were detected in nasal cavities and tissue samples of pigs and rabbits during the infection. Furthermore their usefulness as animal models for transmission studies has not yet been addressed.Table 3Advantages and limitations of natural and potential intermediate hosts for MERS-CoV infection as experimental animal models as well as for transmission studies.Table 3SpeciesAdvantagesLimitationsCamelidsDromedary camels (Camelus dromedarius)&#8226;Natural host for MERS-CoV&#8226;Efficient animal-to-animal transmission (field studies)&#8226;Expensive and limited availability&#8226;Complex husbandry requirements (large BSL3 facilities)Alpacas (Vicugna pacos)&#8226;Potential surrogates for dromedaries under experimental conditions (potential hosts for MERS-CoV)&#8226;Efficient animal-to-animal transmission (experimental)&#8226;Expensive and limited availability&#8226;Complex husbandry requirements (large BSL3 facilities)Llamas (Lama glama)&#8226;Potential surrogates for dromedaries under experimental conditions (potential hosts for MERS-CoV)&#8226;Expensive and limited availability&#8226;Complex husbandry requirements (large BSL3 facilities)&#8226;No animal-to-animal transmission studies availableNon-camelid domestic speciesDomestic pig (Sus scrofa domesticus)&#8226;Relative low cost and readily available&#8226;Ease handling&#8226;Potential surrogates for dromedaries under experimental conditions (potential hosts for MERS-CoV)&#8226;Animal-to-animal transmission (experimental)&#8226;Pig-specific immunological reagents are available&#8226;Complex husbandry requirements (large BSL3 facilities)&#8226;Limited transmission (only via direct contact, and during few days)&#8226;Lower viral titers in NS and tissues upon infectionRabbit (Oryctolagus cuniculus)&#8226;Low cost and readily available&#8226;Ease handling&#8226;Potential surrogates for dromedaries under experimental conditions (potential hosts for MERS-CoV)&#8226;Rabbit-specific immunological reagents are available&#8226;No animal-to-animal transmission studies available&#8226;Lower viral titers in NS and tissues upon infectionNHPRhesus macaques (Macaca mulatta)&#8226;Human-specific immunological reagents are widely available&#8226;Clinical disease similar to humans: translational research (for intra species transmission)&#8226;Expensive and limited availability&#8226;Complex husbandry requirements (large BSL3 facilities)&#8226;No animal-to-animal transmission studies availableCommon marmoset (Callithrix jacchus)&#8226;Some human-specific immunological reagents cross-react&#8226;Clinical disease similar to humans: translational research (for intra species transmission)&#8226;Expensive and limited availability&#8226;Complex husbandry requirements (large BSL3 facilities);&#8226;No animal-to-animal transmission studies availableAbbreviations: BSL3, biosafety level 3; MERS-CoV, Middle East respiratory syndrome; NHP, non-human primates; NS, nasal swabs.Contrary to the mentioned species, both macaques and common marmosets develop clinical disease relatively similar to humans. In that respect, phylogenetically-related species as baboons [58], which live in Africa and Arabic Peninsula, might also play a role in the transmission of the virus. However, little attention has been paid to these species since no sero-epidemiology has been documented. There are important limitations when working with NHP models, namely the complex husbandry requirements that lead to substantially increased costs, some controversy results among different groups. Besides practical considerations, human-specific immunological reagents cross-react with NHP species and are widely available.\nConclusion and future steps\nAs summarized in this review, several species of animals are susceptible to experimental MERS-CoV infection; thus, they might act as potential intermediate hosts of the disease. However, the presence of viral RNA and/or specific antibodies against the virus has been only demonstrated in the field in dromedaries and alpacas [41], [51]. At the light of recent experimental studies, it seems that the list of potential host targets for MERS-CoV is not closed. MERS surveillance programs should be implemented in endemic areas in animal species for which experimental evidence of susceptibility has been provided and species closely related to them.SARS and MERS outbreaks taught us many lessons, and one of the most important is that, even in the absence of an overt threat, there is the possibility of the re-emergence of a virus or other similar viruses. On the other hand, and since the first case of MERS, continuous new cases have been described in different countries around the world [59], [60], [61], [62]. This underlines the importance of the development of animal models closer to the natural host targets. The key role of domestic animals and wildlife in the transmission of MERS-CoV should be further elucidated; meanwhile, countermeasures against deadly coronaviruses must be further explored since the risk of a global outbreak is not negligible. Noteworthy, after more than a decade of SARS and five years of MERS epidemics, there are still no licensed preventive or therapeutic drugs available that could be used in case of an eventual re-emergence of SARS or MERS. This scenario is not the outcome of technical issues, since effective vaccine prototypes against those pathogens are already available [53], [63], [64]. In case of MERS-CoV, vaccination of dromedary camels, the main source of zoonotic transmission, might be useful to control the spread of MERS [53]. However, when developing a vaccine, besides testing the protection efficacy, researchers need to think about social problems such as the reticence of camel owners to vaccinate their animals. Thus, the development of a dual vaccine able to protect against both, MERS-CoV and camelpox virus (an endemic disease in the Middle East, Africa and Asia) might be an ideal solution [53]. Recently, another dual-vaccine for humans and animals against MERS-CoV and rabies virus has been designed [65]. Political aspects have also a key role in the release of a vaccine into the market. Unless the requirements and timings for vaccine licensing procedures are facilitated, pharmaceutical companies will unlikely invest in their development taking into account the current market demand. Moreover, fragmentation of intellectual property rights may also adversely affect the development of vaccines to combat those infections [66].\nConflicts of interest\nNone.",
        "pmid": "28616501",
        "title": "Searching for animal models and potential target species for emerging pathogens: Experience gained from Middle East respiratory syndrome (MERS) coronavirus",
        "journal_title": "One Health",
        "authors": "J\u00falia Vergara-Alert;Enric Vidal;Albert Bensaid;Joaquim Segal\u00e9s",
        "doi": "10.1016/j.onehlt.2017.03.001"
    },
    {
        "keywords": "genome space;graphical representation;moment vector;classification;phylogeny",
        "abstract": "A genome space is a moduli space of genomes. In this space, each point corresponds to a genome. The natural distance between two points in the genome space reflects the biological distance between these two genomes. Currently, there is no method to represent genomes by a point in a space without losing biological information. Here, we propose a new graphical representation for DNA sequences. The breakthrough of the subject is that we can construct the moment vectors from DNA sequences using this new graphical method and prove that the correspondence between moment vectors and DNA sequences is one-to-one. Using these moment vectors, we have constructed a novel genome space as a subspace in RN. It allows us to show that the SARS-CoV is most closely related to a coronavirus from the palm civet not from a bird as initially suspected, and the newly discovered human coronavirus HCoV-HKU1 is more closely related to SARS than to any other known member of group 2 coronavirus. Furthermore, we reconstructed the phylogenetic tree for 34 lentiviruses (including human immunodeficiency virus) based on their whole genome sequences. Our genome space will provide a new powerful tool for analyzing the classification of genomes and their phylogenetic relationships.",
        "body": "Introduction\nComparative genomics at the sequence level has existed for &#8764;20 years, since the time genome sequencing started in earnest. Already there have been many proposals to compare genomes. Boore and Brown1 used gene order to study the evolutionary relationships of metazoan mitochondrial genomes. Snel et al.2 later constructed the phylogenetic tree for completely sequenced prokaryotic genomes based on gene content. In Snel et al.'s method, the similarity between two genomes is defined as the number of genes that they have in common divided by their total number of genes. However, such techniques are time-consuming as they must first identify all the genes in one genome. These approaches, together with G + C content, edit distance, and reversal and rearrangement distances,3&#8211;5 compare genomes using only partial genomic information. Thus, these results are usually controversial because single-gene sequences generally do not contain enough information to construct an evolutionary history of organisms.In Koonin's editorial6 about the emerging paradigm and open problems in comparative genomics, he pointed out &#8216;&#8230; but within these superfamilies, there is nothing like a straight one-to-one correspondence between genomes, and in distant genomes, most of the members may not be orthologous&#8217;. This sentence enlightened us to get a novel idea of comparing genomes. We can construct a genome space. In this space, each point corresponds to a genome uniquely. The natural distance between two points in the genome space reflects the biological distance between these two genomes. Currently, there is no method to represent genomes by a point in a space without losing biological information.We introduce graphical representation of DNA sequence to construct the genome space. The graphical representation of DNA sequence provides a simple way of viewing, sorting, and comparing various gene structures. Thus, it is an attractive and promising research direction. The first important method in this direction is due to Hamori.7 He used a three-dimensional curve to represent a DNA sequence. Gates8 later constructed a two-dimensional graphical representation that is simpler than the Hamori curve. However, Gates' graphical representation has high degeneracy. Recently, we reported a new two-dimensional graphical representation of gene sequences9 which has no circuit or degeneracy, so that the correspondence between gene sequences and gene graphs is one-to-one. In this way, the original DNA sequence can be recovered from its graph mathematically without loss of biological information. In this paper, we make a minor modification of our previous method and obtain a new graphical representation approach for DNA sequences. The breakthrough of the subject is that we can construct the moment vectors from DNA sequences using this new graphical method, and we can prove that the correspondence between moment vectors and DNA sequences is one-to-one. The novelty and uniqueness of our approach is that by using these moment vectors of DNA sequences, we have constructed a genome space as a subspace in Euclidean space. Each genome sequence can be represented as a point in this space. Therefore, this genome space can be used to make comparative analysis to study the clustering and phylogenetic relationship among genomes. The biological (evolutionary) distance between two genomes can be obtained through the Euclidean distance among the corresponding points in the genome space.\nMaterials and methods\n\nGraphical representation of DNA sequence\nWe constructed a new DNA sequence graph in two quadrants of the Cartesian coordinate system, with pyrimidines (C and T) in the first quadrant and purines (A and G) in the fourth quadrant. As shown in Fig.&#160;1, the vectors corresponding to the four nucleotides G, A, T, and C are as follows:\nHere, we should emphasize that the specific ordering of the four nucleotides in the Cartesian coordinate system is related to the GC content of genomes (as it is explained in the &#8216;Discussion&#8217; section). Points in the graphical representation are obtained by the sum of vectors representing nucleotides in the sequence. In Fig.&#160;2, we give the graphical representation of whole mitochondrial genome sequences of human, common chimpanzee, Norway rat, and hedgehog, which are based on the vector system shown in Fig.&#160;1. Since human and chimpanzee belong to the same order (primates), their mitochondrial genome graphical representations are very similar visually. These four DNA sequence graphical curves have no circuits or degeneracy and the correspondence between the sequence and the graphical curve can be mathematically proved to be one-to-one.9\nMoment vector of DNA sequence\nMotivated by our previous work for protein sequence analysis,10 we use moment vectors to characterize a DNA graphical curve. Given the graphical curve of a DNA sequence, which can be represented by a sequence of points (1, y1), (2, y2), &#8230; , (n, yn), we can compute a sequence of numbers 1 &#8722; y1, 2 &#8722; y2, &#8230; , n &#8722; yn. Conversely, if we know the sequence of numbers 1 &#8722; y1, 2 &#8722; y2, &#8230; , n &#8722; yn, we can recover the graph (1, y1), (2, y2), &#8230; , (n, yn). Therefore, we want to find a sequence of numbers, each of which uses the global information of the sequence of numbers 1 &#8722; y1, 2 &#8722; y2, &#8230; , n &#8722; yn in such a way that this new sequence of numbers determines and is determined by the sequence of numbers 1 &#8722; y1, 2 &#8722; y2, &#8230; , n &#8722; yn. For this purpose, we consider the moments which are defined as follows:\nwhere n is the number of nucleotides contained in a DNA sequence, and (xi, yi) represents the position of the ith nucleotide in the DNA graphical curve. According to this definition, each DNA sequence has an n-dimensional moment vector (M1, M2, &#8230; , Mn) associated with it.The crucial point in this paper is that the correspondence between a DNA sequence and its moment vector obtained from its sequence graph is one-to-one. To obtain this conclusion, we need to prove the following theorem.\nConstruction of genome space\nWe have already obtained a good numerical characterization, moment vector, to represent a DNA sequence. Now, we will use this tool to construct a genome space. Here, we emphasize that the structure of genomes is complicated. It may be single-stranded or double-stranded and in a linear or circular structure. Thus, we should consider the different structures when constructing the genome space.For the simplest genome structures, linear single-strand forms, we can treat them as linear DNA sequences. That is, every genome corresponds to a general DNA sequence. Thus, we can utilize our moment vector to construct the genome space. In order to use whole genome information to make comparative analysis among genomes, we can use the first N components (M1, M2, &#8230; , MN) of the moment vector of a genome sequence graph to represent a genome as a point in N-dimensional space. Thus, we obtain an N-dimensional genome space as a subspace in RN. Using the Euclidean distance between two points as an index for comparison, we can perform phylogenetic and clustering analysis for genome sequences in this genome space.For the circular single-strand genomes, the construction of genome space is more complicated because we do not know which point is the start point in this circular DNA sequence. In this case, we treat every point as the start point in this circular sequence of length n, and then we get n linear single-strand genomes. For every linear single-strand genome sequence, we can compute its n-dimensional moment vector. Then, we take average by n for these n n-dimensional moment vectors to get a normalized moment vector (M1, M2, &#8230; , Mn). For circular single-strand genomes, we use the first N components (M1, M2, &#8230; , MN) of this normalized moment vector to represent a genome as a point in an N-dimensional space. Thus, we obtain an N-dimensional genome space as a subspace in RN.For the double-stranded genomes, we need to point out that the moment vector of reverse complementary sequence is not the same as the original sequence. Generally, when meeting the double-stranded genomes, we treat them as two single-stranded genomes. We use the above method (linear or circular) to get two n-dimensional moment vectors for these two single-stranded sequences, and then take average to get a general moment vector (M1, M2, &#8230; , Mn). By using the first N components (M1, M2, &#8230; , MN) of this general moment vector to represent a genome as a point in N-dimensional space. Thus, we obtain an N-dimensional genome space as a subspace in RN. Here, we need to point out that the two strands of some genomes (e.g. mitochondrial genomes, some bacterial genomes) are differentiated by their nucleotide content, which are called the heavy strand and the light strand, respectively. The two strands have different masses because one has a higher proportion of heavier nucleic acids and its complement a lower proportion. In this case, we just treat them as the single-stranded (by using the heavy strand) genomes to make the genome space.\nResults\nTo verify that the biological distance obtained in this way truly incorporates biological utility, we apply our new genome space to the phylogenetic analysis of organisms. Most existing methods for phylogenetic inference using biological sequences can be divided into two groups. The algorithms in the first group utilize various distance measures12&#8211;15 which are based on different models of nucleotide substitution or amino acid replacement, and then transform the distance matrix into a tree. In the second group of approaches, instead of building a tree, the tree that can best explain the observed sequences under the evolutionary assumption is found by evaluating of different topologies. This category includes parsimony16&#8211;18 and maximum likelihood methods.19&#8211;21 All these methods require a multiple alignment of the sequences and assume some sort of evolutionary model, which require human intervention. Thus, the results are usually controversial. However, our genome space does not need sequence alignment and any evolutionary model. It is totally automatically generated and avoids computation repetition.First, we consider the phylogeny of mammals. Mitochondrial DNA is not highly conserved and has a rapid mutation rate, thus it is very useful for studying the evolutionary relationships of organisms.22 We extracted 35 complete mammalian mitochondrial genome sequences from the GenBank, each of which has length of more than 16 000 nucleotides. Moreover, they have double-stranded and circular structures. As mentioned in the previous section, because we have already known the gene content of both strands of these genomes, we just treat them as the single-stranded (by using the heavy strand) circular genomes. For this case, we treat every point as the start point in this circular sequence of length n, and then we get n linear single-strand genomes. For every linear single-strand genome sequence, by using the nucleotide vector system shown in Fig.&#160;1, we can compute its n-dimensional moment vector. Then, we take average by n for these n n-dimensional moment vectors to get a normalized moment vector (M1, M2, &#8230; , Mn). Here, we use the first 60 components of the moment vector (M1, M2, &#8230; , M60) to characterize these 35 genome graphical curves and obtained 35 points in 60-dimensional genome space. By computing the Euclidean distances between these points, we got the distance matrix for these 35 organisms. The phylogenetic tree for them shown in Fig.&#160;3 is generated using UPGMA program in the MEGA 4 package.23 The last 10 mammals are grouped into a cluster because they are primates, and the phylogenetic relationship among them coincides with those found by Raina et al.24 We also found that Norway rat, vole, and squirrel are grouped into a cluster for the reason that they are rodent species.In order to further illustrate the efficiency of our genome space, we then focus on the origins and evolution of human immunodeficiency virus (HIV). HIV is a lentivirus that can lead to acquired immune deficiency syndrome (AIDS), a condition in humans in which the immune system begins to fail, leading to life-threatening opportunistic infections. To develop the anti-HIV drugs and vaccines, the research into the origins and evolution of this virus has become very important. Rambaut et al.25 reconstructed the phylogenetic tree of the primate lentiviruses including HIV-1, HIV-2, and the simian immunodeficiency viruses (SIVs). It was discovered that the two human viruses are related to different SIVs and therefore have different evolutionary origins. However, in Rambaut et al.'s paper, the tree was constructed using the maximum likelihood method on an alignment of the nucleotide sequences of polymerase gene in these lentiviruses. Generally, a single-gene sequence does not possess enough information to construct an evolutionary history of organisms, thus we use our new method to reconstruct the phylogenetic tree based on the whole genome sequences.The genomes of lentiviruses are single-stranded linear RNA. RNA has the base uracil (U) rather than thymine (T) that is present in DNA. In fact, these RNA genome sequences downloaded from GenBank have already been transformed into DNA sequences (change U by T). Thus, we treat them as linear DNA sequences. Using the nucleotide vector system shown in Fig.&#160;1, the sequence graphs of the genomes of the 33 lentiviruses were obtained. Here, we use the first 12 components of the moment vector to characterize these 33 genome graphical curves, and thus we obtained 33 twelve-dimensional vectors. These 33 vectors can be viewed as 33 points in a 12-dimensional genome space. By computing the Euclidean distance between these points, we reconstructed the phylogenetic tree of these primate lentiviruses (Fig.&#160;4) using UPGMA program in the MEGA 4 package.23 The figure&#160;illustrates that both the HIV-1 and HIV-2 lineages fall within that of the SIVs which are isolated from other primates, thus they represent the independent cross-species transmission events. In agreement with Rambaut et al.'s result, our phylogenetic tree shows that HIV-1 group N is a new HIV-1 type between the HIV-1 groups M and O (somewhat closer to the M group). It also indicates that HIV-1 groups M, N, and O are closely related to SIVs from chimpanzees (Pan troglodytes troglodytes and Pan troglodytes schweinfurthii) because there is a mixing of the HIV-1 and SIV lineages. Moreover, our result suggests that P. t. troglodytes (SIVcpz2 and SIVcpz3 in Fig.&#160;4) is the primary reservoir for HIV-1 group O, which is not clear in Rambaut et al.'s evolutionary tree. In Rambaut et al.'s evolutionary tree, the position of HIV-1 group O is just between P. t. troglodytes and P. t. schweinfurthii, but not closer to either of them. Our suggestion coincides with those found by Gao et al.26 In Gao et al.'s work, the authors concluded that the HIV-1 group O is closely related to the just one of these SIVcpz lineages, found in P. t. troglodytes. For HIV-2, we find that the SIVs that are closely related to HIV-2 are not only SIVsm (sooty mangabey monkey) and SIVmac (macaque) as shown by Rambaut et al.'s tree, but also SIVsun (sun-tailed monkey).In addition, we apply our genome space to another field of virology: the taxonomy of coronavirus. To study the classification and phylogeny of coronaviruses clearly, we apply our genome space to a large set of 30 complete coronavirus genomes from GenBank, including the two newly sequenced human coronaviruses, HCoV-NL63 and HCoV-HKU1, along with four genomes from Flaviviridae and Togaviridae which are not coronaviruses (outgroups). The coronavirus genomes are also single-stranded linear RNA. So, similar to the above lentivirus case, we treat these coronavirus genomes as linear DNA sequences. Their abbreviation, accession number, description, and classification are shown in Table&#160;1. First, we used our two-dimensional genome space (actually, it is a two-dimensional plane with the first two moments M1 and M2 being x-axis and y-axis) to characterize these &#8216;34&#8217; virus genomes and calculated &#8216;34&#8217; points in Fig.&#160;5A. Four groups, group 1, group 2, group 3, and outgroups, can be seen in this figure as four distinct clusters. To study the classification for coronavirus clearly, we expanded it and obtained Fig.&#160;5B.\nTable&#160;1Thirty coronavirus genomes and four outgroup genomes from the GenBankAccession No.AbbreviationLength (nt)Description/ClassificationAF304460HCoV-229E27 317Human coronavirus 229E, complete genome/Group 1AY391777HCoV-OC4330 738Human coronavirus OC43, complete genome/Group 2AF353511PEDV28 033Porcine epidemic diarrhea virus strain, complete genome/Group 1U00735BCoVM31 032Bovine coronavirus strain Mebus, complete genome/Group 2AF391542BCoVL31 028Bovine coronavirus isolate BCoV-LUN, complete genome/Group 2AF220295BCoVQ31 100Bovine coronavirus strain Quebec, complete genome/Group 2NC_003045BCoV31 028Bovine coronavirus, complete genome/Group 2AF208067MHVM31 233Murine hepatitis virus strain ML-10, complete genome/Group 2AF201929MHV231 276Murine hepatitis virus strain 2, complete genome/Group 2AF208066MHVP31 112Murine hepatitis virus strain Penn 97-1, complete genome/Group 2NC_001846MHV31 357Murine hepatitis virus, complete genome/Group 2NC_001451IBV27 608Avian infectious bronchitis virus, complete genome/Group 3EU095850TCoV27 657Turkey coronavirus isolate MG10, complete genome/Group 3AY278488BJ0129 725SARS coronavirus BJ01, complete genome/Group 4AY278741Urbani29 727SARS coronavirus Urbani, complete genome/Group 4AY278491HKU-3984929 742SARS coronavirus HKU-39849, complete genome/Group 4AY278554CUHK-W129 736SARS coronavirus CUHK-W1, complete genome/Group 4AY282752CUHK-Su1029 736SARS coronavirus CUHK-Su10, complete genome/Group 4AY283794SIN250029 711SARS coronavirus isolate SIN2500, complete genome/Group 4AY283795SIN267729 705SARS coronavirus isolate SIN2677, complete genome/Group 4AY283796SIN267929 711SARS coronavirus isolate SIN2679, complete genome/Group 4AY283797SIN274829 706SARS coronavirus isolate SIN2748, complete genome/Group 4AY283798SIN277429 711SARS coronavirus isolate SIN2774, complete genome/Group 4AY291451TW129 729SARS coronavirus TW1, complete genome/Group 4NC_004718TOR229 751SARS coronavirus TOR2, complete genome/Group 4AY297028ZJ0129 715SARS coronavirus ZJ01, complete genome/Group 4AY572034Civet00729 540SARS coronavirus civet007, complete genome/Group 4AY572035Civet01029 518SARS coronavirus civet010, complete genome/Group 4NC_005831HCoV-NL6327 553Human coronavirus NL63, complete genome/Group 1NC_006577HCoV-HKU129 926Human coronavirus HKU1, complete genome/Group 5NC_001564CellF10 695Cell fusing agent virus, complete genome/Flaviviridae (outgroup)NC_004102HepaCF9646Hepatitis C virus, complete genome/Flaviviridae (outgroup)NC_001512NyongT11 835O'nyong-nyong virus, complete genome/Togaviridae (outgroup)NC_001544RossT11 657Ross River virus, complete genome/Togaviridae (outgroup)Based on genotypic and serological characterization, coronaviruses are divided into three distinct groups, with human coronavirus 229E (HCoV-229E) being a group 1 coronavirus and human coronavirus OC43 (HCoV-OC43) being a group 2 coronavirus. There is one additional branch, the group 3 coronaviruses, which are found exclusively in birds. These three distinct groups of coronaviruses clearly form three distinct clusters in our two-dimensional genome space. We also find that the newly discovered human coronavirus NL63 is very close to the human coronavirus 229E, and thus it is classified into group 1. This result is the same as the identification by van der Hoek et al.27 Our two-dimensional genome space reveals that for human SARS-CoV, the most closely related coronavirus is from a small infected mammal, the palm civet (not a bird as initially suspected). This result coincides with those found by Guan et al.28 and Wang et al.29 In Guan et al.'s work, the authors also suggested that SARS viruses were isolated from Himalayan palm civets found in a live-animal market in Guangdong, China. Similarly, in Wang et al.'s work, the researchers found six palm civets at the restaurant were positive for SARS-associated coronavirus. They think that SARS cases at the restaurant were the result of recent interspecies transfer from the putative palm civet reservoir and not the result of continued circulation of SARS-CoV in the human population.For another newly discovered human coronavirus HCoV-HKU1, Woo et al.30 classified it into group 2 because HCoV-HKU1 contains certain features that are characteristic of group 2 coronaviruses. However, these authors also stated that the proteins of HCoV-HKU1 formed distinct branches in the phylogenetic trees, indicating that HCoV-HKU1 is a distinct member within the group and is not very closely related to any other known member of group 2 coronaviruses. In our two-dimensional genome space, we found that HCoV-HKU1 is an individual coronavirus between the SARS group (which we suggest as group 4) and the traditional group 2, thus we propose that HCoV-HKU1 belongs to a new group 5. By computing the distances between the genomes in Fig.&#160;5A, we also reconstructed the phylogenetic tree of these coronaviruses (Fig.&#160;6) using UPGMA program in the MEGA 4 package.23 Thus, the evolutionary relationship of these genomes is clearer.\nDiscussion\nIt should be pointed out that the construction of our genome space depends on four parameters (the y-coordinates of the A, C, T, and G in Fig.&#160;1). If we change these four parameters, we shall have different embedding of the genome space. Because the GC content of DNA molecule is found to be variable with different organisms, GC content should be considered when we assign the y-coordinate values of nucleotide vectors. The genomes of three groups in this paper have low GC content. Most of them have 30&#8211;50% GC content, thus we assign larger y-coordinate absolute values for G and C. However, the y-coordinate values of the four nucleotides must be between &#8722;1 and 1 to assure that the correspondence between a DNA sequence and its corresponding moment vector is one-to-one. Therefore, in order to obtain a universal genome space for all species, further studies will be needed to determine the universal y-coordinate values.In the study of mammalian mitochondrion, lentiviruses (including HIV), and coronavirus genomes, we used 60, 12, and 2 moments to construct the genome space, respectively. Here, we should emphasize that we do not need to calculate all the moments to determine the biological information of genomes. Remember that in the central limit theorem in probability and statistics, the limiting process is Gaussian. For Gaussian, the first two moments determine the density function. Thus, we just use the first N moments to get the results, where N is much less than n (the length of genome). Thus, for coronavirus genomes, we only used the first two components of the moment vector (M1, M2) because these two moments have allowed us to obtain the stable classified result&#8212;when higher moments are included the relationship of being close or farther away remains unchanged. To make this point clearer, we also use the first 20 components of the moment vector (M1, M2, &#8230; , M20) to construct the 20-dimmensional genome space. By computing the Euclidean distances between these points in this genome space, we reconstructed the phylogenetic tree of these coronaviruses (Fig.&#160;7). Comparing Figs&#160;6 and 7, we found that the classification relationship of these genomes are the same&#8212;group 1, group 2, group 3, group 4, group 5, and outgroups can still be seen in the two trees as six distinct clusters. This means that when using the higher moments, the relationship of being close or farther away remains unchanged. In other word, two moments are already enough to give the right classifying relationship for these genomes. For the same reason, we used the first 60 components of the moment vector (M1, M2, &#8230; , M60) and the first 12 components of the moment vector (M1, M2, &#8230; , M12) to generate the genome space for mammalian mitochondrion and lentiviruses genomes and obtained the stable phylogenetic analysis result.Gene rearrangements play important roles in evolution. The order of genes and transcription regions are changed during evolution by gene rearrangements such as DNA inversions and transpositions,31 which do not affect the gene content of the chromosomes. For example, inversions of large genomic fragments are often observed even between closely related species. The phylogenetic analysis based on the gene order is challenging as it requires detailed complex gene order data in genomes and intensive computation. In order to test whether our genome space is stable for genomic rearrangement, we consider the following simulated experiment. We choose the human mitochondrion genome and then invert its two genes ATPase 6 and Cytochrome oxidase III to get a simulated genome, which we called human AC genome. We can treat this new genome as the result of the inversion of genes from the human mitochondrion genome. Next, we randomly generate a genome sequence which has the same length and nucleotide content as the human mitochondrion genome, which we called random genome. Thus, we have three genomes of the same length and nucleotide content: human, human AC, and random. In addition, we also choose the common chimpanzee mitochondrion genome as comparison. By using the 60-dimmensional genome space, we calculate the distance among these four genomes and get a distance matrix in Table&#160;2. In Table&#160;2, we found that the distance between human and human AC is very small. This means that even some genomic rearrangement happened in a genome, the new produced genome still has very close distance from the original genome. Thus, in our genome space, the original genome and its genomic rearrangement genome represent different points but have very close distance. Large-scale genomic rearrangements in closely related organisms make the genome sequences very different from the original genome sequences. Although the moment vectors of the original and rearranged DNA sequences are different, if we treat double DNA genomes as two single-stranded genomes to get two n-dimensional moment vectors for these two single-stranded sequences and take the average to get a general moment vector (M1, M2, &#8230; , Mn), the computed distances among related species by gene rearrangements will be small. The genome spaces constructed by the moment vector have advantages over the gene order phylogenetic analysis since the genome spaces do not depend on gene orders and are capable of using all gene families. We will apply the moment vector method on the detailed phylogenetic analysis using the large-scale genomic rearrangement data.\nTable&#160;2The distance matrix of four genomes in the 60-dimmensional genome spaceHumanHuman ACRandomChimpanzeeHumanHuman AC1.778773Random223.018945224.777237Chimpanzee20.17107219.112686237.476133In this paper, we report a two-dimensional graphical representation for DNA sequences. A moment vector system to represent a DNA sequence is introduced, and the correspondence between a DNA sequence and its moment vector is mathematically proven to be one-to-one. With this moment vector system, each genome sequence can be represented as a point in a Euclidean space, and the genome space is constructed as a subspace of this Euclidean space. Genomes with close evolutionary relationship and similar properties plot close together in this genome space. Thus, it will provide a new powerful tool for analyzing the classification of genomes and their phylogenetic relationships. Our method is easier and quicker in handling whole or partial genomes than multiple alignment methods. There are two major advantages to our method. (i) Once a genome space has been constructed, it can be stored in a database. There is no need to reconstruct the genome space for any subsequent application, whereas in multiple alignment methods, realignment is needed for add-on new sequences. (ii) One can have global comparison of all genomes simultaneously, which no other existing method can achieve. Furthermore, in our method, the results in two-dimensional genome space can be displayed and viewed graphically; this is user-friendly and allows even non-expert to understand the relationship among different genomes via viewing the graph of genome space.",
        "pmid": "20360268",
        "title": "A Novel Construction of Genome Space with Biological Geometry",
        "journal_title": "DNA Research: An International Journal for Rapid Publication of Reports on Genes and Genomes",
        "authors": "Chenglong Yu;Qian Liang;Changchuan Yin;Rong L. He;Stephen S.-T. Yau",
        "doi": "10.1093/dnares/dsq008"
    },
    {
        "keywords": "SARS-coronavirus;accessory proteins;open reading frames;respiratory disease",
        "abstract": "A respiratory disease caused by a novel coronavirus, termed the severe acute respiratory syndrome coronavirus (SARS-CoV), was first reported in China in late 2002. The subsequent efficient human-to-human transmission of this virus eventually affected more than 30 countries worldwide, resulting in a mortality rate of ~10% of infected individuals. The spread of the virus was ultimately controlled by isolation of infected individuals and there has been no infections reported since April 2004. However, the natural reservoir of the virus was never identified and it is not known if this virus will re-emerge and, therefore, research on this virus continues. The SARS-CoV genome is about 30 kb in length and is predicted to contain 14 functional open reading frames (ORFs). The genome encodes for proteins that are homologous to known coronavirus proteins, such as the replicase proteins (ORFs 1a and 1b) and the four major structural proteins: nucleocapsid (N), spike (S), membrane (M) and envelope (E). SARS-CoV also encodes for eight unique proteins, called accessory proteins, with no known homologues. This review will summarize the current knowledge on SARS-CoV accessory proteins and will include: (i) expression and processing; (ii) the effects on cellular processes; and (iii) functional studies.",
        "body": "1. Introduction\nPrior to the outbreak of severe acute respiratory syndrome (SARS) caused by SARS human coronavirus (HCoV) in 2003 [1,2,3], only two human coronaviruses, HCoV-OC43 and HCoV-229E, were known [4,5]. In the following two years, two additional coronaviruses were identified, namely HCoV-NL63 [6] and HKU1 [7]. Then, very recently, in September 2012 another novel coronavirus was identified in two patients, of which one died [8]. Not much is known about this latest coronavirus and studies are underway to understand the virus better. Only HCoV-OC43, HCoV-229E, HCoV-NL63 and HKU1 are known to continuously circulate in the human population [9]. The International Committee for Taxonomy of Viruses (ICTV) recently reported that the three traditional coronavirus groups have been replaced by four genera, namely Alpha-, Beta-, Gamma- and Deltacoronaviruses [10].SARS eventually spread to more than 30 countries in 2003, infecting ~8000 people with a mortality rate of ~10% [11]. The etiological agent was eventually identified as the SARS human coronavirus (SARS-CoV) [12]. The virus has been identified as an enveloped, positive-sense RNA virus containing a ~27 kbs genome that encodes for proteins expressed from full-length and subgenomic (sg) mRNAs [13]. In addition to the replicase and structural open reading frames (ORFs), the SARS-CoV genome contains eight ORFs encoding for accessory proteins with no known homologues (Figure 1). These group-specific accessory proteins are interspersed among the structural genes at the 3&#8217;-end of the genome.Coronavirus accessory proteins have been shown to be dispensable for growth in vivo and in vitro [14,15,16,17,18]. On the other hand, deletion of the nonessential genes from the mouse hepatitis virus (MHV) genome results in attenuation of the virus when inoculated into the natural hosts [15], indicating a possible in vivo function. Continuous passage of infectious bronchitis virus (IBV) in cell culture, results in mutations in the IBV 3b gene that confers a growth advantage to the virus when cultured in cells and chicken embryos. These mutations also increase the virulence of the IBV in the chicken embryos [19]. Reports to date make it clear that the CoV accessory proteins are not essential for virus replication in vitro, but since these genes are maintained in the virus genomes under selective pressures, they have to confer a biological advantage to the virus in the natural environment of the infected host [20].Although recent evidence shows that the SARS-CoV accessory genes are expressed in the host during infection, their functions remain somewhat obscure. Currently, there are a range of proposed functions for these accessory proteins, including modulation of viral pathogenicity and replication, as well as acting as cell death inducers and interferon (IFN) antagonists, to name a few. In this review the current knowledge on SARS-CoV accessory proteins (ORFs 3a, 3b, 6, 7a, 7b, 8a, 8b and 9b) will be summarized and will include: (i) expression and processing; (ii) the effects on cellular processes; and (iii) functional studies. \n2. SARS Accessory Proteins\n\n2.1. ORF3a and ORF3b\nSARS-CoV 3a protein, also referred to as ORF3 [21], X1 [22], ORF3a [23] and U274 [24], was the first of the SARS-CoV accessory proteins to be characterized. ORF3a, located at the 3&#180;-end of the SARS-CoV genome between the S and E genes, encodes for a 274 amino acid (a.a.), 31 kDa accessory protein [21,22,25,26]. The ORF3a N-terminal ectodomain (a.a. 1-34), contains three transmembrane regions (a.a. 35-56; 77-99; 103-125), and is separated from the C-terminal endodomain (a.a. 209-264) by a central region containing a cysteine-rich domain (a.a 127-133), a Yxx&#934; domain (a.a. 160-163; where x is any a.a. and &#934; is an a.a. with a bulky hydrophobic side-chain) and a diacidic domain (a.a. 171&#8211;173) [24]. Although indirect immunofluorescence detects ORF3a partly in the rough endoplasmic reticulum [27], it is mainly found perinuclearly in the Golgi complex [24,27,28]. Antibodies to ORF3a also stain the plasma membrane [24,27,29], where it is either secreted out of the cell [30,31] or undergoes endocytosis [24]. Following ORF3a translation, the diacidic domain ensures that the protein is efficiently exported from the endoplasmic reticulum [32] to the Golgi apparatus, where ORF3a undergoes posttranslational modifications in the form of O-glycosylation [33] before being inserted into the plasma membrane. Transport of ORF3a to the cell surface requires two protein sorting and tracking signals, the centrally located Yxx&#934; and diacidic motifs [24]. Once in the plasma membrane, the side chains of cysteine in the cysteine rich domain allow ORF3a to form homo- and heterotetramers, which cluster together to form a potassium-permeable ion channel [34]. In transfected cells, co-immunoprecipitation experiments have shown that membrane-bound ORF3a interacts with structural proteins S, M and E, as well as with SARS-CoV ORF7a, another accessory protein [24,27,35,36].The presence of ORF3a in the cytoplasm of infected cells has been confirmed through immunohistochemical analysis of tissue biopsies and/or autopsies of SARS-CoV-infected patients. In lung sections, the expression of ORF3a is restricted to the cytoplasm of pneumocytes and to multinucleated giant cells, a rare feature of the lung pathology in SARS [37]. Moreover, sections taken from the small intestine show ORF3a localizes to the cytoplasm of surface enterocytes [37]. The co-localization of ORF3a together with the SARS-CoV genome in the cytoplasm of pulmonary pneumocytes and small intestinal enterocytes is not totally unexpected as the lungs and small intestine are primary targets of SARS-CoV infection. The lung and small intestines are, however, not the only tissue targets for SARS-CoV. One of the cellular receptors for SARS-CoV, angiotensin-converting enzyme 2 (ACE2) [38,39,40], is also expressed by a certain population of peripheral CD14+ monocytes, from which osteoclasts are derived [41]. Interestingly, the expression of ORF3a in a murine macrophage osteoclast precursor cell line enhances osteoclast differentiation, and may explain why many recovered SARS patients suffer from reduced bone density [41]. The first attempts to elucidate the functions of coronaviral group-specific genes involved the generation of recombinant MHV from which the group-specific genes had been deleted. These recombinant viruses could still replicate, demonstrating that these genes are dispensable for virus growth in vitro. The mutant viruses did, however, still lead to a significant attenuation of the MHV when inoculated into mice, suggesting that coronaviral group-specific genes play a role in virus-host interactions and therefore contribute to viral fitness [15]. Subsequently, much effort has been placed on elucidating the role of ORF3a in the life cycle of the SARS-CoV. The biological functions of ORF3a may be closely linked to those of the structural proteins with which they interact. The S protein is involved in viral assembly and packaging of coronaviruses (reviewed by [42]) and so the formation of inter-chain disulfide bonds between ORF3a and the S protein in the interior of the viral envelope may suggest that ORF3a is integrated into viral particles (reviewed by [20]). Indeed, ORF3a is incorporated into newly packaged mature SARS-CoV virions, suggesting that ORF3a is a structural protein [29,30,43]. Furthermore, the co-localization of ORF3a with M and E proteins, which are essential for viral assembly [44], suggests that ORF3a is important in SARS-CoV assembly or budding [27]. There seems to be some conflicting data in the literature as to the role that ORF3a plays in viral replication. In Vero E6 cells transfected with a SARS-CoV deletion mutant lacking the 3a gene, the virus still replicates as efficiently as the wild-type, suggesting a non-essential function of ORF3a in viral assembly in vitro [17]. Moreover, in a study that utilized small interference RNAs (siRNAs) to knockdown the expression of ORF3a in FRhK-4 cells, viral replication was also unaffected [34]. However, the transfection of Vero E6 cells with siRNAs specific to ORF3a significantly reduced the yield of progeny SARS-CoV, signifying an inhibition of SARS-CoV replication cycle in these cells [45]. This could have been a combinational effect due to the concurrent knockdown of ORF3b. In addition to a possible role in viral replication, ORF3a is likely to modulate viral release, as a study has shown that there is a significant reduction in viral release in FRhK-4 cells transfected with ORF3a specific siRNAs [34]. Taking into account that ORF3a forms a potassium-sensitive ion channel in the plasma membrane [29,34], and ion channels for viral proteins control virus entry or release [46,47], the mechanism whereby ORF3a influences viral release warrants further investigation [34]. ORF3a has also been shown to regulate various host cellular responses. Overexpression of ORF3a in several different cell culture models has demonstrated that it induces apoptosis [29,48,49,50], upregulates mRNA and protein levels of fibrinogen in lung epithelial cells [51], activates C-Jun N-terminal kinase (JNK) and the transcription factor nuclear factor kappa B (NF-kappaB), which is involved in the activation of pro-inflammatory genes [20,41,52] and downstream from that, up-regulates the production of pro-inflammatory cytokines and chemokines such as interleukin 8 (IL-8) and RANTES (CCL5) [52]. Taken together, these changes to host cellular homeostasis imply a role for ORF3a in the pathogenesis of SARS.The occurrence of both apoptosis and necrosis in host cells during SARS-CoV infection suggests that the regulation of cell death is important for viral replication and/or pathogenesis (reviewed by [53]). The pro-apoptotic function of ORF3a is reliant on its ion channel activity [29] and is induced via caspase-8 and -9 dependent pathways, i.e. through both the death receptor- and mitochondria-mediated pathways, respectively [23,29,48]. In addition, it has recently been reported that Bax, p53 and p38 MAP kinase also play roles in ORF3a-induced apoptosis [54]. The fact that ORF3a utilizes more than one caspase pathway to trigger cell death may explain why tissue responses to SARS-CoV infection are distinct in different organs; for instance whereas the lung pathology is dominated by diffuse alveolar damage, the small intestine remains relatively intact [37].Even though the natural animal reservoir for SARS-CoV has not been identified, the threat of another SARS outbreak is still a disturbing possibility as bats have been shown to be natural reservoirs for different SARS-like coronaviruses (SL-CoV) [55]. Like all accessory proteins, ORF3a is unique to the SARS-CoV, and shares an 83% homology to SL-CoV [56], making it a good candidate for the development of diagnostic assays, vaccines, and drugs. The immunogenic properties of ORF3a in SARS-CoV-infected patients has been shown to vary depending on: (i) whether or not the sera was collected from convalescent or recovered patients; (ii) whether or not full-length or truncated fusion proteins were used to detect immunoglobulin G (IgGs) to ORF3a; and (iii) the method used to detect these antibodies to ORF3a, namely western blot, ELISA or protein microarrays. To date, in general, IgGs to ORF3a could be detected in only ~60&#8211;73% of patient sera screened [35,57,58,59]. This could be due to frame-shift mutations within the 3a gene, resulting in a heterogeneous population of sgRNA3 transcripts in patients with acute SARS-CoV infection containing copies of both wild-type and mutant 3a genes. [24,60]. These mutant 3a genes encode for proteins with distinctively shorter N-termini than the wild-type forms [24]. The neutralizing activity of anti-3a sera has subsequently been assessed in vitro. The sera from rabbits immunized with recombinant ORF3a (a.a. 125&#8211;274) does not exhibit any significant neutralizing activity on SARS-CoV infected Vero E6 cells [59]. In another study, however, sera from rabbits immunized with a different recombinant ORF3a (a.a. 15&#8211;28) contains neutralizing antibodies that do inhibit SARS-CoV infection in Vero E6 cells [61]. These contrasting results could be explained by differences in the immunogenicity of the central region and C-terminal domain compared to the N-terminal ectodomain, respectively. Indeed, ectodomain-specific antibodies (a.a. 15&#8211;28) from convalescent-phase patient plasma recognize and destroy SARS-CoV-ORF3a-expressing cells [62]. A more recent study has showed that mice immunized with full-length 3a DNA elicit strong humoral and Th1-based cellular immune responses. The vaccine stimulated IFN-&#947; production mainly by CD8+ T cells and interleukin-2 (IL-2) mainly by CD4+ T cells [56]. Whether or not ORF3a will be useful as a subunit vaccine for humans, still remains to be seen though. ORF3b is predicted to express the 154 amino-acid 3b protein, also referred to as ORF4 [21], X2 [22] and U154 [57], via an internal ribosomal entry mechanism [21,22,25]. SARS-CoV ORF3b has been described as a multi-localized protein due to its shuttling behavior [63]. Although the protein initially accumulates in the nucleolus [64], it subsequently translocates to the outer membrane of mitochondria [63,65]. Nuclear export of other accessory proteins, namely SARS-CoV ORF9b, has also been previously reported [66] but how this spatio-temporal distribution impacts on ORF3b behavior and function is still unclear [67]. In fact, one study has reported that ORF3b is localized primarily in the nucleus with no evidence of mitochondrial localization [68]. Immunohistochemical analysis of tissue biopsies and/or autopsies of SARS-CoV-infected patients has failed to demonstrate the presence of ORF3b in vivo [37]. The antigenicity of ORF3b has also come into question as IgGs to ORF3b are detectable in SARS patient sera in some studies [69,70], but not in others [59]. To date, the presence of ORF3b in SARS-CoV-infected Vero E6 cells is the only evidence for the expression of this protein [37].In comparison to ORF3a protein, much less is known about the function(s) of ORF3b protein. SARS-CoV deletion mutants lacking the 3b gene replicate to levels similar to those of wild-type virus in several tissue culture cell types [17], suggesting that, like ORF3a, ORF3b is dispensable for viral replication in vitro. Although ORF3b may not be involved in viral replication, a role for this protein in viral pathogenicity has been inferred through its ability to modulate the host innate immune response to SARS-CoV infection [68]. ORF3b has been shown to inhibit the synthesis of type 1 IFNs (IFN-alpha/beta), major components of the innate immune response; as well as with IFN signaling [67,68]. Considering that SARS-CoV-infected cells do not produce IFN [71,72], it is possible that ORF3b provides the SARS-CoV with a mechanism to subvert the IFN response, and therefore the innate immune response [68]. However, when mice are infected with mutant SARS-CoV lacking 3b genes, the deletion viruses grow to levels similar to those of wild-type virus in the lungs of BALB/c mice at day 2 post infection [17]. This demonstrates that SARS-CoV is therefore able to inhibit the host IFN response without the 3b gene [17] and emphasizes the limitations of transfection studies. Recently, SARS-CoV ORF3b was shown to modulate the transcriptional activity of the host factor, RUNX1b, indicating a physiological role in upregulating cytokines and chemokine levels during SARS-CoV infection [73]. ORF3b also induces AP-1 transcriptional activity through activation of JNK and ERK pathways [74].SARS-CoV ORF3b has also been shown to affect cell cycle and apoptosis-regulation but again, there are discrepancies within the literature. When overexpressed in Vero E6 cells, ORF3b induces apoptosis and necrosis with an increase in caspase-3 activity [75]. Similarly, expression of ORF3b causes cell growth arrest in GO/G1 phase, inducing cell- or tissue-specific apoptosis [76]. Since data show massive necrosis in the lungs, spleen and lymph nodes, as well as increased apoptosis in cells from the spleen, liver, lung and lymph nodes of SARS patients [76,77,78], it may suggest that ORF3b contributes to these clinical observations. \n2.2. ORF6\nORF6, also referred to as p6 [79], is a small 63 amino acid membrane protein with no known homologues. This relatively small peptide has a ~44 amino acid residue amphipathic N-terminus and ~20 residue polar C-terminus [80]; the C-terminus contains intracellular protein sorting motifs [81]. The protein is found on cytoplasmic membranes in the endoplasmic reticulum [82] and Golgi-complex of cells overexpressing ORF6 [82,83]. The membrane topology of this protein is N-endo C-endo, with a membrane-spanning region ~36 a.a. in length [79,84]. Using immunohistochemical staining, ORF6 was detected in both SARS-CoV infected cells, as well as in the pneumocytes of the lung and surface enterocytes of the terminal ileum from 8 SARS autopsy cases [82]; the latter corresponds to the tissue-sites of infection reported by another group [37]. Recent work has hinted at a few possible functions for SARS-CoV ORF6. Sucrose gradient purification of SARS-CoV particles and virus capture assays show that ORF6 is associated with purified virions, with the presence of SARS-CoV proteins E, M and S sufficient for incorporation of ORF6 into virus-like particles (VLPs). When overexpressed alone in cells [79,85,86], as well as in virus infected cells [87], ORF6 is released in membrane-associated vesicular structures, most likely a subpopulation of endosomal/lysosomal vesicles [86]. The C-terminus portion ORF6 contains the sequence motif YSEL, which has been shown to target proteins for internalization from the cell membrane to endosomal/lysosomal vesicles [84]. Several groups have previously shown that coronaviruses use similar vesicular structures within infected cells, suggesting that these vesicles have a role in the viral replication cycle [86,88,89,90]. Recombinant studies with SARS-CoV lacking ORF6 remained viable in vitro and in vivo at a high multiplicity of infection (moi) [17]. In contrast, an enhancement of replication of the virus is observed at low moi, which is consistent with previous findings using recombinant studies [91]. Recombinant studies using attenuated MHV expressing SARS-CoV ORF6, show accelerated MHV replication [92] and lead to higher viral titers in both cell culture and the mouse central nervous system [83,93]. In fact, the presence of ORF6 leads to increased virulence of normally sub-lethal infections in mice [83]. The recombinant MHV studies by Tangudu and colleagues also showed that ORF6 associated with viral RNA, co-localized with replication viral RNA on cytoplasmic vesicles and increased MHV protein synthesis [20,92]. Lastly, not only did ORF6 partially co-localize with nonstructural protein 3 (nsp3), a marker for virus replication complexes [79], it also interacted with nsp8, another replicase protein [87], pointing to a putative role for ORF6 in replication. Collectively, these results provide strong evidence that ORF6 plays a role in enhancing SARS-CoV replication and pathogenicity by interacting with membrane-bound replication and assembly machinery [84].Interestingly, overexpression of ORF6 in CHO cells induces DNA synthesis, of which the biological significance is still not clear [82]. However, this was done in the absence of other viral proteins and needs to be verified in the context of the whole viral genome. On the other hand, ORF6 suppresses the expression levels of co-transfected expression constructs. The ability of ORF6 to interfere with the expression of co-expressed constructs has previously been linked to the blockage of nuclear translocation of these proteins [80,93]. Gunalan and co-workers also found that, even though ORF6 did not have a global effect on protein synthesis, it down-regulated the mRNA level of co-transfected nsp8. However, they linked the N-terminus diacidic cluster motif, DDEE which is similar to the putative diacidic motif DxE, which governs ER export and localization to different membranous compartments [32,86], to this suppression [86]. ORF6 is a type I IFN antagonist, and its expression suppresses the induction of both IFN as well as the IFN signaling pathways [68,94]. The protein inhibits IFN signaling by interacting via its C-terminal region to the nuclear import protein, karyopherin &#945;2, thereby sequestering it in the cytoplasm and indirectly preventing movement of signal transducer and activator of transcription factor 1 (STAT1) to the nucleus [94]; the 18 a.a. N-terminus is also required for maximal inhibition [79] and this inhibition probably plays a role in the immune evasion of SARS-CoV [94]. ORF9b immunoprecipitated with ORF6 from extracts of SARS-CoV infected cells and subsequent mass spectrometry analysis reveal an interaction of the two in biologically relevant conditions. This interaction has been validated by co-localization of both proteins in the cytoplasm of SARS-CoV infected cells [95]. Taken together, findings support the hypothesis that ORF6 could be a minor structural protein and could be an important virulence factor during SARS-CoV infection in vivo.\n2.3. ORF7a and ORF7b\nThe SARS-CoV ORF7a protein, also known as ORF8 [21], U122 [96] or X4 [22,97], is encoded by the first ORF of sgRNA7 [25,26]. The protein is a 122 a.a. long, type 1 transmembrane protein with an N terminal signal peptide sequence of 15 aa, an ectodomain of 81 aa, and a C terminus transmembrane domain containing 21 aa residues followed by a short cytoplasmic tail of 5 aa residues [98]. The ORF7a shares a 26% sequence homology with human CD62L protein, an adhesive molecule that plays important roles in immune and inflammatory responses, suggesting that ORF7a may adhere to other cells and influence their functions, such as leukocyte adhesion, migration and the inflammatory response [97]. In fact, the ectodomain bears high structural and similar sequence homology to members of the immunoglobulin superfamily, namely the D1 domains of ICAM-1 and ICAM-2 [98,99]. The ectodomain of ORF7a binds to the alpha integrin I domain of human lymphocyte function-related antigen 1 (LFA-1) on Jurkat cells, suggesting that LFA-1 may be an attachment factor or even the receptor for SARS-CoV on human leukocytes [99]. The C-terminus contains a motif (KRKTE) that typically serves as an ER retrieval signal but in the case of ORF7a, serves as a dibasic ER export signal that allows for the ER exit of ORF7a to the Golgi apparatus [96]. Expression of ORF7a in SARS-CoV-infected cells and lung tissue of SARS patients has been confirmed [96,97,98], but there are discrepancies in the literature as to the exact subcellular localization of the protein. It seems to localize predominantly to the intermediate compartments between the ER and Golgi apparatus [13,96,97,98,100], partially in the mitochondria [101], as well as in the cytoplasm [100]. The export of ORF7a from the ER to the Golgi apparatus is regulated via the ER export motif [96,102]. Therefore the presence of ORF7a in the ER-Golgi intermediate compartment, where coronaviruses are known to assemble [103], could explain how ORF7a becomes incorporated into purified SARS-CoV particles [31]. The interaction between ORF7a and the viral structural proteins M, E and S [31,96], which are important for viral assembly, implies that ORF7a is important in the viral replication cycle and that virion-associated ORF7a protein may exert its function early in infection [31]. ORF7a also interacts with the accessory ORF3a which itself interacts with M, E and S, suggesting that these viral proteins may form complexes in SARS-CoV-infected cells [24]. The incorporation of ORF7a in VLPs occurs however in the absence of S or ORF3a protein interaction [31]. The presence of 7a protein in the cytoplasm could explain why ORF7a is able to inhibit protein translation [13]. How this would be beneficial to the viral life cycle is unclear as the SARS-CoV relies on host cell protein synthesis to replicate viral proteins. Expressed ORF7a also interacts with human small glutamine-rich tetratricopeptide protein [100], human lymphocyte function-associated antigen 1 (LFA-1) on Jurkat cells and with human Ap4A-hydrolase [104].ORF7b is encoded by the second ORF of sgRNA7 and may be expressed by &#8220;leaky scanning&#8221; of ribosomes [25]. This protein has been partially characterized [105] and is a 44 amino-acid long integral membrane protein that is localized to the Golgi compartment in SARS-CoV-infected cells [102]. The protein is found in association with intracellular virus particles and also in purified virions [102].To date, there is very little experimental evidence to support a role for ORF7a or ORF7b in the replication of SARS-CoV. Monkey Vero E6 cells transfected with SARS-CoV deletion mutants lacking the 7a and 7b genes still release virions at levels similar to those of wild-type virus [102,106] and virion morphology remains unchanged [107]. In airway epithelial cultures generated from Golden Syrian hamster tracheas, SARS-CoV replication titers are comparable to infected human cultures [106]. Transgenic mice expressing the SARS-CoV receptor ACE2, one of the receptors for SARS-CoV, also still release virions when infected with mutant SARS-CoV [107]. Moreover, studies in immunosuppressed Syrian golden hamsters, an animal model for the study of SARS-CoV, have shown that infection with mutant SARS-CoV bearing deletions in gene 7 does not result in significant changes to replication kinetics, tissue tropism, morbidity or mortality [102]. One study utilizing siRNAs targeting specifically ORF7a and ORF7b has, however, demonstrated that the silencing of these proteins significantly reduces the yield of progeny virus [45]. The 70% decrease in progeny virus is not due to a disruption of the full-length genomic mRNA but reflects inhibition of both sgRNA 7 and 8, resulting in the subsequent knockdown of ORF7a, -7b, -8a and -8b [45]. Moreover, the limitations of in vitro studies must be taken into consideration. For example, although the 7b gene of feline coronavirus is easily lost upon virus adaptation to cell culture, it is strictly maintained in naturally occurring strains and its loss is correlated with reduced virulence [108]. The importance of ORF7a and ORF7b during the replication cycle of SARS-CoV therefore still remains uncertain. ORF7a may be important for viral host interactions and therefore contribute to viral pathogenesis in vivo by inducing inflammatory responses in SARS. Like ORF3a, ORF7a activates NF-kappaB and the IL-8 promotor [52], and downstream from that, augments the production of pro-inflammatory chemokines such as IL-8 and RANTES [52]. ORF7a may also play a role in the pathogenesis of SARS-CoV by way of the protein&#8217;s contribution to virus-induced apoptosis. Several studies have shown that overexpression of ORF7a induces apoptosis in a caspase-3-dependent manner [101,109] as well as independently via p38 MAPK [13]. Although deletion of gene 7 does not alter markers of early apoptosis or activation of caspase 3, viruses with gene 7 disruptions are not as efficient as wild-type virus in inducing DNA fragmentation, an apoptotic feature in later stages of infection [102]. Overexpression of Bcl-XL protein, a member of the pro-survival Bcl-2 family, blocks ORF7a-induced apoptosis in 293T cells, suggesting that ORF7a regulates host cell apoptosis at or upstream of the Bcl-2 family in the apoptotic pathway [101]. In addition, ORF7a forms complexes with Bcl-XL protein and binds to other pro-survival Bcl-2 family members, indicating that it may induce apoptosis by interfering with the pro-survival function of Bcl-XL protein [101]. It must be kept in mind though that ORF7a is not the only apoptosis-inducing factor during SARS-CoV infection (review by [53]) and that the actual levels at which ORF7a is expressed in infected tissues is still unknown bringing into the question the validity of the overexpression data in the in vivo context.Although ORF7a is expressed in SARS-CoV-infected cells [96,97] and in lung tissue of SARS patients, specifically in the bronchial epithelial cells, peripheral blood erythrocytes and leukocytes [37], it does not display strong immunogenicity. Protein microarrays fail to detect antibodies to ORF7a in SARS patient sera [59] and SARS patient sera has low to non-specific antibody responses to ORF7a epitopes when tested on a peptide chip platform [70]. The selected ORF7a epitopes are therefore poorly immunogenic, and so ORF7a is not likely to serve as a potential marker for SARS-CoV infection or be important for vaccine development. The ORF7b protein is likely to be expressed in vivo because anti-7b antibody has been detected in SARS convalescent patient serum [69]. \n2.4. ORF8a and ORF8b\nWhereas the SARS-CoV isolated from humans infected during the peak of the SARS epidemic, encodes two accessory proteins termed ORF8a and ORF8b, the SARS-CoV isolated from animals and some early human isolates, encodes for a single 122 a.a. protein, ORF8ab. Interestingly, this difference is due to a 29 nucleotide deletion in the mRNA8 of SARS-CoV from some animal species [110,111,112,113,114], that splits ORF 8 into ORF8a and ORF8b, encoding 39- and 84-residue polypeptides, respectively [110]. In fact, larger deletions were reported in sequenced ORF8 genes from a cluster of viruses that were isolated from humans later during the SARS epidemic [111,115]. Vaccinia virus expressed SARS-CoV ORF8ab, contains a cleavable signal sequence, which directs the movement of the precursor protein to the ER and then mediates its translocation into the ER lumen, where it is N-glycosylated on the N81 residue [110,116] and subsequently assembled into disulfide-linked homo-multimeric complexes, which remains stably in the ER [110]. This retention and accumulation of ORF8ab in the ER occurs in the absence of a recognized ER retention signal [110,117]. Not only do in vitro and in vivo studies show that ORF8ab is ubiquitinated, but both ORF8a and ORF8b also binds to monoubiquitin and polyubiquitin, suggesting the potential involvement of these proteins in the pathogenesis of SARS-CoV [116]. ORF8ab interacts with SARS-CoV proteins S, M, ORF3a and ORF7a [118], all of which have been shown to be structural proteins. Since ORF8a is too small for its signal sequence to function, it is most likely directly released from the ribosome and remains in the cytosol in its precursor form [110]. Antibodies to ORF8a have been detected in a small number of SARS patients, proving that the protein is indeed expressed during virus infection [119]. This group also reported that ORF8a not only enhances viral replication, but also induces apoptosis through a mitochondria-dependent pathway [119]. When overexpressed, the 9.6 kDa ORF8b localizes to both the nucleus and cytoplasm of Vero E6 and CHO cells [120]. In another study, however, ORF8b was not detected in SARS-CoV-infected cells or when expressed from mRNA's mimicking mRNA8. However, after it was cloned immediately downstream of a T7 promoter, a soluble, unmodified and monomeric ORF8b protein was expressed in the cytoplasm, which was extremely unstable and rapidly degraded [120]. Similarly, in the absence of the ORF8a region, ORF8b undergoes rapid degradation by proteasomes [116]. ORF8b down-regulates the expression of SARS-CoV E, a process that could be inhibited by the addition of proteasome inhibitors [116]. For this reason, E is not detectable in SARS-CoV-infected cells that are expressing high levels of ORF8b [118]. Similarly to ORF6, overexpression of ORF8b in mammalian cells also induces DNA synthesis. However, co-expression of SARS 8b and SARS 6 did not elicit synergistic effects on DNA synthesis [120]. The biological function of this induction is still to be elucidated.It is obvious that the 29-nucleotide deletion disrupts the proper expression of the SARS-CoV ORF8. Even though this mutation does not appear to adversely affect the survival of the virus, it has become clear that ORF8ab, -8a and -8b have different stabilities and varying functions, and could contribute differently to viral replication and pathogenesis in vivo. Taken together these findings suggest that ORF8ab may modulate viral replication and/or pathogenesis in a yet unknown manner. \n2.5. ORF9b\nBi-cistronic mRNA9 encodes for ORF9b which is translated via a leaky ribosomal scanning mechanism [121]. This 98 amino acid peptide is synthesized from an alternative complete reading frame within the N gene [37,121]. It has no homology with known proteins and, as yet, its function has not been confirmed. Both anti-9b antibody and ORF9b antigen have been detected in clinical samples from SARS patients [37,59], with the latter also detected in virus infected cells [37]. Analysis, using fluorescence microscopy of live transfected cells and indirect immunofluorescence of transfected fixed cells, shows that ORF9b is exported outside of the cell nucleus and localizes to the endoplasmic reticulum [66,122]; the authors suggest that the 46-LRLGSQLSL-54 amino acid sequence motif of ORF9b functions as a nuclear export signal (NES) [66]. More recently, it was confirmed that ORF9b interacts with the cellular protein Crm1 and gets exported out of the nucleus using an active NES; this nucleocytoplasmic export of ORF9b is linked to apoptosis [123]. Cellular CRM1 (Exportin1) protein functions as a nuclear export receptor for proteins carrying a Rev-like NES in a process that also requires the GTP bound form of the Ran GTPase [124]. The authors concluded that the interaction of ORF9b with Crm1 is essential for the proper degradation of ORF9b and blocking the nuclear export of the latter induces apoptosis. Also, ORF9b was shown to enter the nucleus in the absence of any nuclear localization signal (NLS) by passive transport, independent of the cell cycle [123]. The resolved crystal structure of ORF9b shows that the protein has a novel fold, a dimeric tent-like &#946; structure and a central hydrophobic cavity involved in lipid binding. Since the protein also co-localizes with intracellular vesicles, it has been speculated that the protein has a role in virus assemble via membrane association [125]. Interestingly, recent evidence shows that ORF9b is a structural protein incorporated into both purified virions and VLPs, with efficient incorporation dependent on the co-expression of viral structural proteins E and M [121]. The interaction of ORF9b with another SARS-CoV accessory protein, ORF6, was discussed earlier in this review. \n3. Conclusions\nViruses generally encode for: (i) proteins functioning in the replication and transcription of the viral genome; (ii) structural proteins of the virion; and (iii) accessory proteins that enable, facilitate or modulate the infection process. These accessory proteins normally act by interfering with cellular processes or by modulating virus-host interactions at the level of the organism (Table 1). It has also been shown that these proteins are often dispensable for virus replication in vitro, but required for optimal replication and virulence in the natural host [110]. SARS-CoV is the prototype human CoV and has been studied extensively. In addition, SARS-CoV contains eight encoded accessory genes interspersed amongst the viral structural genes [22,25,126], the most of the known CoVs. Interestingly, these accessory genes are present in SARS-CoVs isolated from bats, civet cats, raccoon dogs, and humans, suggesting important accessory functions in a variety of hosts [55,127,128]. Evidence confirms that these accessory proteins are indeed expressed during SARS-CoV infection, but for many, their functions are yet to be determined. Five of the SARS-CoV accessory proteins (ORF3a, -6, -7a, -7b and -9b) have been identified as minor viral structural components, and others induce ion channels, act as cell death inducers and IFN antagonists (Table 1). It would be interesting to determine whether the high SARS-CoV accessory gene number correlate with the increased virulence of SARS-CoV when compared to the other known human CoVs. For this reason, the role of the accessory proteins in pathogenesis must be elucidated. ",
        "pmid": "23202509",
        "title": "The Role of Severe Acute Respiratory Syndrome (SARS)-Coronavirus Accessory Proteins in Virus Pathogenesis",
        "journal_title": "Viruses",
        "authors": "Ruth McBride;Burtram C. Fielding",
        "doi": "10.3390/v4112902"
    },
    {
        "keywords": "",
        "abstract": "Attenuated viral vaccines can be generated by targeting essential pathogenicity factors. We report here the rational design of an attenuated recombinant coronavirus vaccine based on a deletion in the coding sequence of the non-structural protein 1 (nsp1). In cell culture, nsp1 of mouse hepatitis virus (MHV), like its SARS-coronavirus homolog, strongly reduced cellular gene expression. The effect of nsp1 on MHV replication in vitro and in vivo was analyzed using a recombinant MHV encoding a deletion in the nsp1-coding sequence. The recombinant MHV nsp1 mutant grew normally in tissue culture, but was severely attenuated in vivo. Replication and spread of the nsp1 mutant virus was restored almost to wild-type levels in type I interferon (IFN) receptor-deficient mice, indicating that nsp1 interferes efficiently with the type I IFN system. Importantly, replication of nsp1 mutant virus in professional antigen-presenting cells such as conventional dendritic cells and macrophages, and induction of type I IFN in plasmacytoid dendritic cells, was not impaired. Furthermore, even low doses of nsp1 mutant MHV elicited potent cytotoxic T cell responses and protected mice against homologous and heterologous virus challenge. Taken together, the presented attenuation strategy provides a paradigm for the development of highly efficient coronavirus vaccines.",
        "body": "Introduction\nCoronaviruses are vertebrate pathogens mainly associated with respiratory and enteric diseases [1]. They can cause severe diseases in livestock animals and lead thereby to high economic losses. In humans, coronavirus infections manifest usually as mild respiratory tract disease (common cold) that may cause more severe symptoms in elderly or immune-compromised individuals [2,3]. In 2002&#8211;2003, the appearance of severe acute respiratory syndrome (SARS), caused by a formerly unknown coronavirus (SARS-CoV), exemplified the potential of coronaviruses to seriously affect human health [4&#8211;7]. The frequent detection of SARS-like coronaviruses in horseshoe bats (Rhinolophus sp.) and the broad range of mammalian hosts that are susceptible to SARS-CoV infection may facilitate a potential reintroduction into the human population [8]. Therefore, the development of efficacious coronavirus vaccines is of high medical and veterinary importance.Effective vaccines controlling virus spread and disease are available for a number of infections, such as smallpox, poliomyelitis, measles, mumps, rubella, influenza, hepatitis A, and hepatitis B [9,10]. Some of these vaccines consist of virus subunits or inactivated virus preparations that mainly induce the production of pathogen-specific antibodies. In contrast, live attenuated vaccines consist of replication-competent viruses that induce broad cellular and humoral immune responses without causing disease [10]. The most prominent live attenuated vaccines are vaccinia virus [11], poliovirus [12], and yellow fever virus (YF-17D) [13]. Despite their documented efficacy, it is still not fully understood why and how successful vaccines work [10,14]. However, recent concepts in immunology provide a link between innate and adaptive immune responses and suggest that the quality, quantity, and longevity of adaptive immune responses is determined very early after infection or vaccination [14]. Of major importance are professional antigen-presenting cells (pAPCs) such as dendritic cells (DCs) and macrophages, which play a major role in (i) sensing pathogen-associated molecular patterns, (ii) inducing innate immune responses, and (iii) shaping the upcoming adaptive immune response. Efficient live attenuated vaccines should therefore not only lack significant pathogenicity, but should also deliver antigens to pAPCs and activate the innate immune system.Notably, the majority of currently available attenuated vaccines have been derived empirically. Given the recent proceedings in the areas of virus reverse genetics and virus&#8211;host interactions, the time should be ripe for more rational approaches in vaccine development. An attractive strategy is to target virally encoded pathogenicity factors, such as interferon (IFN) antagonists [15], to attenuate virulence while retaining immunogenicity. This concept has been proposed for the generation of live attenuated influenza virus vaccines encoding altered NS1 proteins [16,17].Our rudimentary knowledge on coronavirus-encoded pathogenicity factors is reflected by the fact that only a few putative coronaviral pathogenicity factors have been identified and that functional analyses are still limited to the description of in vitro effects [18&#8211;20]. For a number of reasons, the non-structural protein 1 (nsp1) is of particular interest in this context. First, coronaviruses are positive-stranded RNA viruses, and the replicase-encoded nsps are expressed from the viral genomic RNA immediately after virus entry by translation of two large polyproteins. nsp1 is encoded at the 5&#8242; end of the replicase gene and is therefore the first mature viral protein expressed in the host cell cytoplasm [21]. Second, a recent in vitro study suggests that SARS-CoV nsp1 may be associated with host cell mRNA degradation and may counteract innate immune responses [18]. Finally, nsp1 is encoded by all mammalian coronaviruses known to date (coronavirus groups 1, 2a, and 2b) [22], and recent structural data on SARS-CoV (group 2b) nsp1 suggest functional similarities to mouse hepatitis virus (MHV; group 2a) nsp1 [23].Using a reverse genetics approach, we show here that nsp1 is a major pathogenicity factor. Recombinant MHV mutants encoding a deletion in nsp1 replicated as efficiently as wild-type virus in cell culture, but displayed an unprecedented degree of attenuation in mice. Interference with the type I IFN system appears to be the dominant mode of action of murine coronavirus nsp1. Vaccination with the nsp1 mutant virus elicited efficient memory cytotoxic T cell responses and protected against homologous and heterologous virus infections. Our study will pave the way for the generation of novel coronavirus vaccines based on modified coronavirus replicase genes.\nResults\n\nTransient nsp1 Expression Affects Cellular Gene Expression\nWe assessed several replicase-encoded nsps of MHV (strain A59), SARS-CoV, and human coronavirus 229E (HCoV-229E) for their ability to interfere with host cell gene expression. Using transient gene expression studies, we found that MHV-A59, SARS-CoV, and HCoV-229E nsp1 significantly reduced luciferase reporter gene expression under the control of IFN-&#946;, IFN-stimulated response element (ISRE), and SV40 promoters (Figure 1). This is consistent with a recent report suggesting that SARS-CoV nsp1 induces general host cell mRNA degradation [18]. Nevertheless, it should be noted that the nsp1-mediated reduction in reporter protein expression appeared more robust for ISRE and SV40 than for IFN-&#946; promoter-driven expression. Our data also support the hypothesis that MHV encodes a SARS-CoV nsp1 homolog that displays the same function [23]. Although comparative sequence analyses suggest that nsp1 of group 2a/2b coronaviruses (e.g., MHV and SARS-CoV, respectively) and the nsp1 of group 1 coronaviruses (e.g., HCoV-229E) may belong to different protein families [22,23], we also observed reduced reporter gene expression in HCoV-229E nsp1-transfected cells (Figure 1). Whether functional similarities may exist between nsp1 molecules encoded by coronaviruses of different phylogenetic lineages remains to be established in future studies. Importantly, our data also revealed that reporter gene expression from all tested promoters was not affected when C-terminally truncated MHV nsp1 molecules were tested (Figure 1).\nGeneration of the MHV nsp1 Deletion Mutant\nTo assess the role of nsp1 in the context of virus replication, we constructed a recombinant MHV encoding a truncated nsp1 protein using our reverse genetic system [24]. Based on the results shown in Figure 1, we decided to delete MHV nucleotides (nts) 829&#8211;927 (99 nts). In the resulting mutant virus, MHV-nsp1&#916;99, the replicase gene start codon, the translational reading frame, and the residues required for proteolytic release of nsp1 from the replicase polyprotein were maintained (Figure 2A). As reported for a set of similar MHV mutants by Brockway et al. [25], viral growth and peak titers of MHV-nsp1&#916;99 in murine 17Clone1 cells were indistinguishable from that of wild-type virus (Figure 2B). To assess the stability of the recombinant MHV-nsp1&#916;99, we analyzed the nsp1-coding region by RT-PCR sequencing after seven passages in tissue culture and no nucleotide changes were detected (unpublished data).Infection of conventional DCs (cDCs) is an early and crucial event for the generation of protective antiviral immunity [26]. MHV productively infects cDCs and activates plasmacytoid DCs (pDCs) to generate a first wave of protective type I IFN [27]. To assess whether the mutant MHV-nsp1&#916;99 has retained the ability to infect pAPCs, peritoneal macrophages (Figure 2C), bone marrow&#8211;derived CD11c+ cDCs (Figure 2D), and splenic, FACS-sorted CD11c+ cDCs (Figure 2E) were exposed to MHV-nsp1&#916;99 and wild-type control virus. Similar to replication kinetics in cell lines (Figure 2B), MHV-nsp1&#916;99 showed no significant growth defect in primary pAPCs (Figure 2C&#8211;2E), indicating that the deletion of nsp1 did not alter the pronounced tropism of MHV for cDCs and macrophages.\nDeletion in nsp1 Confers Strong Attenuation In Vivo\nMHV-A59 is a hepatotropic and neurotropic virus that can cause acute hepatitis and encephalitis. Following intraperitoneal infection, virus replication is first detectable in spleen and liver, followed by virus spread to other organs, including the central nervous system. Hepatitis is the first clinical sign of disease, accompanied by elevated liver enzyme values in serum. Associated with the appearance of cytotoxic T cell responses approximately at day 5 post infection (p.i.), virus titers usually decline and are no longer detectable after day 7 p.i. Infections with a high dose (&#8805; 5 &#215; 106 pfu, intraperitoneal) may, however, occasionally result in fatality. To evaluate the importance of nsp1 for virus replication and viral pathogenicity in vivo, C57BL/6 mice were infected intraperitoneally with different doses of wild-type MHV or MHV-nsp1&#916;99. Both viruses replicated in the spleen, whereby MHV-nsp1&#916;99 titers were consistently lower than wild-type virus titers (Figure 3A). Furthermore, MHV-nsp1&#916;99 was rapidly cleared and not detectable after day 2 p.i. (Figure 3A). Wild-type, but not mutant virus, was detectable in the liver at low and intermediate dose (50 pfu and 5,000 pfu, respectively) (Figure 3B). When high virus doses (5 &#215; 106 pfu) were applied, MHV-nsp1&#916;99 eventually reached the liver at day 2 p.i., but was not detectable at later time points (Figure 3B). MHV-nsp1&#916;99 was not detectable in other non-hematopoietic organs, such as lung and central nervous system (unpublished data). Mice infected with wild-type virus showed acute liver disease with elevated liver enzyme values in serum. Furthermore, after high dose infection with wild-type virus (5 &#215; 106 pfu), a significant weight loss that peaked at approximately 10%&#8211;15% at day 4 was observed (Figure 3E). In contrast, mice infected with the nsp1 mutant virus remained healthy after low, intermediate, or high dose infections. Even at the highest dose applied (5 &#215; 106 pfu), MHV-nsp1&#916;99&#8211;infected mice did not lose weight (Figure 3E), and no elevated liver enzyme values were detected in the serum (Figure 3C). This observation correlated well with the absence of hepatocyte necrosis and parenchymal inflammation following MHV-nsp1&#916;99 infection (Figure 3D). To further assess the attenuation of the MHV nsp1 mutant, mice were infected intracranially with 200 pfu and 20,000 pfu of MHV-nsp1&#916;99 or MHV-A59. All mice infected with 200 pfu survived for at least 30 d (unpublished data). Mice infected with 20,000 pfu of MHV-A59 succumbed to the infection, whereas mice infected with 20,000 pfu of MHV-nsp1&#916;99 survived and showed no signs of clinical disease (Figure 3F). Collectively, these data demonstrate that MHV-nsp1&#916;99 is strongly attenuated in vivo, but has retained the ability to replicate in secondary lymphoid organs, such as the spleen.\nEffect of nsp1 on Innate Immune Responses\nWe have previously shown that pDCs are the major source of IFN-&#945; in the early stages of MHV infection and that type I IFN responses in CD11c+ cDCs are only weakly triggered by MHV [27]. To test whether nsp1 has an influence on the induction of IFN-&#945;, we infected both cDCs and pDCs with MHV-nsp1&#916;99 or wild-type MHV. Both viruses elicited rapid and high IFN-&#945; production in Flt3-L&#8211;differentiated bone marrow&#8211;derived pDCs (Figure 4A) and FACS-sorted primary pDCs (Figure 4B). Furthermore, both wild-type and mutant MHV elicited only a late and weak IFN-&#945; production in cDCs (Figure 4A and 4B). These results suggest that nsp1 does not affect the induction of type I IFN. To assess a potential impact of nsp1 on type I IFN signaling and antiviral effector mechanisms in target cells that efficiently support MHV replication, cDCs and macrophages were pretreated with different dosages of IFN-&#945; and infected with MHV-nsp1&#916;99 or wild-type MHV. In cDCs, IFN-&#945; treatment had a comparable effect on the replication of both MHV-nsp1&#916;99 and the wild-type control virus (Figure 4C). However, replication of MHV-nsp1&#916;99 was, in a dose-dependent manner, more vulnerable to IFN-&#945; treatment in macrophages (Figure 4D), suggesting that nsp1 might counteract IFN signaling and/or the antiviral activities of IFN-induced effector proteins.Indeed, in vivo experiments in type I IFN receptor-deficient (IFNAR&#8722;/&#8722;) mice [28] strongly support this interpretation. Infection of IFNAR&#8722;/&#8722; mice with wild-type MHV led to high titers in all tested organs (Figure 5A&#8211;5D), indicating that signals transmitted via the IFNAR are crucial for preventing uncontrolled spread of the virus [27]. Surprisingly, the severe attenuation of MHV-nsp1&#916;99 in wild-type 129Sv mice was not present in IFNAR&#8722;/&#8722; mice (Figure 5A&#8211;5D). Replication of MHV-nsp1&#916;99 in IFNAR&#8722;/&#8722; mice was largely restored and virus titers reached about 104&#8211;105 pfu/g tissue in several organs after only 36 h (figures 5A&#8211;5D). These data strongly suggest that nsp1 has a pivotal role in counteracting type I IFN host responses and provide an explanation for the rapid clearance of MHV-nsp1&#916;99 in wild-type mice. Interestingly, liver damage, measured as alanine 2-oxoglutarate-aminotransferase (ALT) levels in serum, was not yet detectable in MHV-nsp1&#916;99&#8211;infected IFNAR&#8722;/&#8722; mice at 36 h p.i. (Figure 5E). At 72 h p.i., MHV-nsp1&#916;99 reached titers and ALT levels in IFNAR&#8722;/&#8722; mice comparable to those observed in MHV-A59&#8211;infected IFNAR&#8722;/&#8722; mice at 36 h p.i., demonstrating that MHV-nsp1&#916;99 replication in IFNAR&#8722;/&#8722; is, although with slower kinetics, restored.\nImmunization with the MHV nsp1 Deletion Mutant Protects against Homologous and Heterologous Virus Challenge\nThe phenotypic analysis of MHV-nsp1&#916;99 revealed a number of features that are advantageous for live attenuated vaccines. MHV-nsp1&#916;99 grows to high titers in cell culture, infects pAPCs, replicates almost exclusively in secondary lymphoid organs, and is strongly attenuated in vivo. To assess the potential of MHV-nsp1&#916;99 as an attenuated live vaccine, we replaced accessory gene 4 of MHV-nsp1&#916;99 and wild-type MHV-A59 by a gene encoding a fusion protein of the immunodominant cytotoxic T lymphocyte (CTL) epitope (KAVYNFATC) of the lymphocytic choriomeningitis virus (LCMV) and the enhanced green fluorescent protein (GP33-GFP) [29]. The resulting recombinant viruses, MHV-nsp1&#916;99-GP33-GFP and MHV-GP33-GFP, were used to infect C57BL/6 mice with different doses (50 and 5,000 pfu, intraperitoneal), and CD8+ T cell responses were assessed using flow cytometry&#8211;based detection of intracellular IFN-&#947; following antigen-specific short-term in vitro restimulation. As shown in Figure 6A and 6B, infection with as few as 50 pfu of MHV-nsp1&#916;99-GP33-GFP elicited strong CD8+ T cell responses against both the H2-Db&#8211;restricted GP33 and the H2-Kb&#8211;restricted MHV S598 epitope.To assess the level of protection against homologous MHV-A59 challenge, groups of C57BL/6 mice were immunized (5,000 pfu) with MHV-nsp1&#916;99-GP33-GFP, MHV-GP33-GFP, or treated with PBS. Sixteen days p.i., mice were challenged with wild-type MHV (5,000 pfu) and viral titers were determined 5 d post challenge infection. Viral titers were below the limit of detection in MHV-nsp1&#916;99-GP33-GFP&#8211; and MHV-GP33-GFP&#8211;immunized mice (Figure 6C). Together with the absence of elevated liver enzyme values in immunized mice (Figure 6D), these data indicate that vaccination with the attenuated MHV nsp1 mutant provides complete protection against homologous virus challenge.The reverse genetic system facilitates incorporation of antigens derived from other infectious organisms. In order to determine whether the attenuated nsp1 mutant virus could confer protection against heterologous virus infection, MHV-nsp1&#916;99-GP33-GFP&#8211;immunized C57BL/6 mice were challenged after 4 wk with LCMV (200 pfu, intravenous). LCMV titers in spleens were significantly reduced both in mice vaccinated with MHV-GP33-GFP and the attenuated MHV-nsp1&#916;99-GP33-GFP virus (Figure 6E). Remarkably, only 50 pfu of nsp1 mutant virus expressing the GP33 epitope were sufficient to achieve a reduction of LCMV titers by more than 4 orders of magnitude, indicating that nsp1 mutant viruses are well-suited to serve as attenuated recombinant virus vectors against heterologous viral infections.\nDiscussion\nThe rational design of live attenuated viral vaccines is greatly facilitated by the identification and targeting of pathogenicity factors. This study demonstrates an unprecedented level of attenuation of a murine coronavirus through a 99-nt deletion in nsp1. The nsp1 mutant virus was rapidly cleared in mice and did not induce clinical signs of disease in immunocompetent mice. These findings in the murine coronavirus model demonstrate that nsp1 is a major pathogenicity factor. In a stepwise approach, we made use of these observations to provide a blueprint for the construction and evaluation of live attenuated coronavirus vaccines encoding a truncated nsp1.The presented results indicate that nsp1 plays a crucial role in the MHV life cycle by interfering with host innate immune responses. In accordance with the recent report by Kamitani et al. [18], we observed reduced reporter gene expression in transient nsp1 expression studies. The suggestion that SARS-CoV nsp1 may play a role in SARS-CoV pathogenesis by promoting host cell mRNA degradation [18] has now received support through the analysis of a coronavirus nsp1 mutant in a murine model. The MHV nsp1 mutant phenotype led us to conclude that nsp1 mainly affects IFN signaling pathways or other downstream events. The influence on IFN-&#945; induction appears to be limited. These conclusions are based on several observations. First, the analysis of IFN-&#945; production by pDCs and cDCs revealed no significant differences between wild-type and nsp1 mutant virus infections. Second, treatment of macrophages with IFN-&#945; revealed a more efficient reduction of MHV-nsp1&#916;99 replication compared with that of wild-type MHV. Finally, and most strikingly, IFNAR&#8722;/&#8722; mice were highly permissive for the mutant virus, and organ titers almost reached those of wild-type MHV-infected IFNAR&#8722;/&#8722; mice. Nevertheless, it should be noted that the nsp1 mutant virus replication was still slightly delayed in IFNAR&#8722;/&#8722; mice. Therefore, further studies are required to define molecular target(s) and the precise function(s) of coronavirus nsp1. Likewise, further studies are required to assess the impact of other coronaviral gene products on coronavirus pathogenicity. Recent reports indicate that coronaviruses most likely express a number of proteins, such as MHV and SARS-CoV nucleocapsid proteins, and SARS-CoV ORF3b and ORF6 proteins, that may interact with innate immune responses [19,20]. Also, the coronavirus replicase gene may harbor additional functions that play a role in virus&#8211;host interactions. It has been shown that the MHV and SARS-CoV nsp2 proteins [30], and the highly conserved ADP-ribose-1&#8242;&#8242;-monophosphatase activity [31] encoded in nsp3, are both dispensable for virus replication in tissue culture, and that a single point mutation in the MHV nsp14 confers a strong attenuation of MHV in mice [32]. Clearly, the murine model, with MHV as a natural mouse pathogen, will be highly advantageous in the examination of this issue, because it allows the use of well-characterized inbred and transgenic mice in combination with well-established immunological techniques required to assess the full range of coronavirus&#8211;host interactions.The most remarkable finding of this study is the level of attenuation of the nsp1 mutant virus and its restricted replication in secondary lymphoid organs. It may well be that other coronaviral nsp1 molecules exert similar functions as the MHV nsp1. The coronavirus nsp1 has been suggested as a group-specific marker to differentiate group 1 coronaviruses from group 2a/2b coronaviruses [22]. Our transient nsp1 expression data indeed support the notion that SARS-CoV and MHV may encode evolutionarily conserved nsp1 homologs [22,23]. Nevertheless, further in vivo studies are required to determine whether the group 2b SARS-CoV nsp1 is indeed a functional equivalent to the structurally highly conserved group 2a nsp1 molecules encoded by MHV, bovine coronavirus, porcine hemagglutinating encephalomyocarditis virus, HCoV-OC43, and HCoV-HKU1. Likewise, it will be important to clarify in vivo, whether, despite the apparent lack of any sequence homology [22,23], the nsp1 of group 1 coronaviruses (e.g., HCoV-229E) may represent a functional correlate to the nsp1 of group 2a/2b coronaviruses. Recent progress in the establishment of suitable mouse models for SARS-CoV [33&#8211;35] and HCoV-229E [36] will enable researchers to address these questions in future studies.The chosen attenuation strategy has resulted in the generation of a recombinant virus that fulfills important criteria of a live virus vaccine candidate: (i) growth to high titers in cell culture, which facilitates vaccine production, and (ii) generation of immunological memory that mediates efficient protection against viral challenge. One important aspect of protection against viral infections is the induction of specific cytotoxic T cells by pAPCs in secondary lymphoid organs [14]. A number of coronaviruses, such as MHV, HCoV-229E, feline infectious peritonitis virus, and SARS-CoV, have been shown to infect pAPCs and to replicate in the secondary lymphoid organs [27,37&#8211;42]. Because of their pronounced tropism for pAPCs and the induction of strong CTL responses, we propose that coronaviruses represent promising vectors for the expression of heterologous antigens. The identification of nsp1 as a major pathogenicity factor will significantly increase the safety of coronavirus-based vectors [43]. For example, the deletion of accessory genes (i.e., not replicase or structural genes) has been described for some coronaviruses to confer attenuation in the natural host [44&#8211;46], and the deletion of the structural envelope protein E has resulted in the development of replication-competent, but propagation-deficient, coronavirus vectors [47,48]. Now, with an accompanying deletion in the nsp1-coding sequence, such vectors would be considered &#8220;recombination proof&#8221;, because the deletions are located at opposite genomic regions (i.e., within the replicase gene at the 5&#8242; end and within the structural gene region at the 3&#8242; end of the coronavirus genome), which make the reconstruction of virulent viruses by recombination unlikely. We therefore suggest that accessory gene, E gene, and partial nsp1 gene deletions will result in particular safe vectors with the potential to express multiple heterologous antigens [40,49].Taken together, our results describe a novel type of coronavirus vaccines based on impaired function of a replicase gene product. We expect that our approach is applicable to most, if not all, mammalian coronaviruses and that it will enable the development of long-desired live attenuated vaccines for important coronavirus-induced diseases in humans and animals.\nMaterials and Methods\n\nMice and cells.\nC57BL/6 mice were obtained from Charles River Laboratories (http://www.criver.com/). 129Sv and type I IFN receptor-deficient mice (IFNAR&#8722;/&#8722;) [28] were obtained from the Institut f&#252;r Labortierkunde (University of Z&#252;rich) and bred in our facilities. All mice were maintained in individually ventilated cages and were used between 6 and 9 wk of age. All animal experiments were performed in accordance with the Swiss Federal legislation on animal protection.MC57, BHK-21, L929, 293, and CV-1 cells were purchased from the European Collection of Cell Cultures (http://www.ecacc.org.uk/). D980R cells were a kind gift from G. L. Smith, Imperial College, London, United Kingdom. 17Clone1 cells were a kind gift from S. G. Sawicki, Medical University of Ohio, Toledo, Ohio, United States. BHK-MHV-N cells, expressing the MHV-A59 nucleocapsid protein under the control of the TET/ON system (Clontech, http://www.clontech.com/), have been described previously [24]. All cells were maintained in minimal essential medium supplemented with fetal bovine serum (5%&#8211;10%) and antibiotics.\nIsolation of dendritic cells and macrophages, flow cytometry, and antibodies.\nMurine cDCs and pDCs were obtained from spleens of C57BL/6, 129Sv, or IFNAR&#8722;/&#8722; mice following digestion with collagenase type II (Gibco-BRL, http://www.invitrogen.com/) for 20 min at 37 &#176;C. Cells were resuspended in PBS supplemented with 2% FCS and 2 mM EDTA and overlaid on 20% Optiprep density gradient medium (Sigma-Aldrich, http://www.sigmaaldrich.com/). After centrifugation at 700g for 15 min, low density cells were depleted of CD3- and CD19-positive cells using DYNAL magnetic beads according to the instructions of the manufacturer (Invitrogen, http://www.invitrogen.com/). The DC-enriched preparations were stained with &#945;-PDCA-1, &#945;-CD11b, and &#945;-CD11c, and the distinct pDC and cDC populations were sorted using a FACS ARIA (BD Biosciences, http://www.bdbiosciences.com/) sorter. Purity of both cell preparations was always >98%.Murine bone marrow&#8211;derived cDCs or pDCs were generated by 6 to 7 d of culture with either granulocyte-monocyte colony stimulating factor (GM-CSF)-containing supernatant from the cell line X63-GM-CSF (kindly provided by Antonius Rolink, University of Basel, Basel, Switzerland) or Flt3-L (R&D Systems, http://www.rndsystems.com/) at 20 ng/ml, respectively. Bone marrow&#8211;derived cDCs were further purified using Optiprep density gradient centrifugation. Bone marrow&#8211;derived pDCs were purified using the mouse pDC isolation kit (Miltenyi Biotec, http://www.miltenyibiotec.com/) adapted for the isolation of in vitro&#8211;derived pDCs by adding CD11b-biotin to the negative selection cocktail. Antibodies used in this study were purchased from BioLegend (http://www.biolegend.com/): CD11c-PE, B220-APC, CD11b-FITC; or from Miltenyi Biotec: mPDCA-1-FITC and CD11c-APC. Thioglycolate-elicited macrophages were collected from the peritoneal cavity of mice and cultured at 4 &#215; 105 cells per well in DMEM (with 10% FCS, L-glutamine, and penicillin/streptomycin) for 2 h at 37 &#176;C. Non-adherent cells were removed by washing with cold PBS.\nRecombinant DNA and viruses.\nLCMV-WE strain, originally obtained from F. Lehmann-Grube (Hamburg, Germany), was propagated on L929 cells. MHV A59 was generated from a molecularly cloned cDNA [24] based on the Albany strain of MHV A59. Coronaviruses and recombinant vaccinia viruses were propagated, titrated, and purified as described [24,50,51].Mutant vaccinia viruses are based on the recombinant vaccina virus vMHV-inf-1 (containing the full-length MHV-A59 cDNA) and were generated using our reverse genetic system as described previously [24]. Briefly, the gene to be mutated was replaced by the Escherichia coli guanine-phosphoribosyltransferase (GPT) gene through vaccinia virus&#8211;mediated homologous recombination, and GPT-positive clones were selected by three rounds of plaque purification on CV-1 cells in the presence of xanthine, hypoxanthine, and mycophenolic acid (GPT-positive selection) [50]. In a second round, the GPT gene was replaced by the mutated gene, and GPT-negative clones, containing the mutated gene, were selected by three rounds of plaque purification on D980R cells in the presence of 6-thioguanine (GPT-negative seletion) [50]. To construct the recombinant vaccinia virus encoding the MHV-nsp1&#916;99 cDNA, the 5&#8242; end of the MHV-A59 cDNA in vMHV-inf-1 was replaced by GPT using the plasmid pRec4. This plasmid is based on pGPT-1 [50], and the GPT gene is flanked to its left by 500 bp of vaccinia DNA and to its right by an internal ribosomal entry sequence (IRES) followed by MHV-A59 nts 952-1315. The GPT-negative selection was carried out using the plasmid pRec15. This plasmid contains 250 bp of vaccinia DNA followed by the bacteriophage T7 RNA polymerase promoter, one G nucleotide, and MHV nts 1&#8211;828 linked to MHV nts 928-1315. To replace the MHV-A59 accessory gene 4 in vMHV-inf-1 and vMHV-nsp1&#916;99 by a gene encoding a fusion protein of EGFP and the LCMV-derived CTL epitope KAVYNFATC (GP33-GFP) [29], the plasmid pRec8 was used for recombination with vaccinia viruses vMHV-inf-1 and vMHV-nsp1&#916;99 . This plasmid contains the GPT gene flanked to its left by MHV nts 27500&#8211;27967 and to its right by MHV nts 28265&#8211;28700. GPT-negative selection was carried out using the plasmid pRec9. This plasmid contains MHV bp 27500&#8211;27967, the GP33-GFP gene, and MHV nts 28265&#8211;28700. Further cloning details, plasmid maps, and sequences are available from the authors upon request. Recombinant coronaviruses were rescued from cloned cDNA using purified vaccinia virus DNA as template for the in vitro transcription of recombinant MHV genomes as described [51].The firefly luciferase (FF-Luc) plasmid for monitoring IFN-&#946; promoter activation (p125-Luc) was kindly provided by Takashi Fujita, Tokyo Metropolitan Institute of Medical Science, Japan [52]. The FF-Luc reporter construct for monitoring ISRE activation (p(9&#8211;27)4tk&#916;(&#8722;39)lucter) [53] was kindly provided by Stephen Goodbourn, St. George's Hospital Medical School, London, UK. The control plasmid pRL-SV40 (Promega, http://www.promega.com/) encodes the renilla luciferase (REN-Luc) gene under control of the constitutive SV40 promoter. The negative control expression plasmid contained the reading frame of the N-terminus of the human MxA protein. To construct the coronavirus nsp1 expression plasmids, the MHV nts 1&#8211;951 (pMHV-nsp1), MHV nts 1&#8211;902 (pMHV-&#916;49), MHV nts 1&#8211;851 (pMHV-&#916;100), HCoV-229E nts 293&#8211;625 (pHCoV-229E-nsp1), and SARS-CoV nts 265&#8211;804 (pSARS-CoV-nsp1) were amplified by standard PCR techniques and cloned downstream of a CMV promoter between the SmaI and XhoI sites of the eukaryotic high-level expression plasmid pI.18 (kindly provided by Jim Robertson, National Institute for Biological Standards and Control, Hertfordshire, UK).\nTransient transfections and reporter gene assays.\nSubconfluent cell monolayers of 293 cells seeded in 12-well plates were transfected with 250 ng p125-Luc reporter plasmid, 50 ng pRL-SV40, and 1 &#956;g of expression plasmid in 200 &#956;l of OPTIMEM (Gibco-BRL) containing 3.9 &#956;l of Fugene HD (Roche, http://www.roche.com). At 8 h post transfection, cells were induced with either 0.2 &#956;g of viral ssRNA containing 5&#8242; triphosphates [54] (p125-Luc), 2.5 &#956;g of poly(I:C) (Sigma), or 500 U/ml IFN-&#945; (p(9&#8211;27)4tk&#916;(&#8722;39)lucter), or left uninduced. After an incubation period of 16 h, cells were harvested and lysed in 100 &#956;l of Reporter Lysis Buffer (Promega). An aliquot of 10 &#956;l lysate was used to measure luciferase activity as decribed by the manufacturer (Promega).\nVirus infections, determination of virus titers, and liver enzyme values.\nMice were injected intraperitoneally or intracranially with indicated pfu of MHV A59 or intravenously with the indicated pfu of LCMV and sacrificed at the indicated time points. Organs were stored at &#8722;70 &#176;C until further analysis. Blood was incubated at RT to coagulate, centrifuged, and serum was used for ALT measurements using a Hitachi 747 autoanalyzer (http://www.hitachi.com/). Peritoneal exudates cells (PECs) were isolated from the peritoneal cavity by flushing with 4 ml of ice-cold PBS. MHV titers were determined by standard plaque assay using L929 cells. LCMV titers in the spleens were determined 4 d after intravenous challenge in an LCMV infectious focus assay as previously described [55].\nHistology, IFN-\u03b1 ELISA, IFN-\u03b1 treatment.\nOrgans were fixed in 4% formalin and embedded in paraffin. Sections were stained with hematoxylin and eosin. Images were acquired using a Leica DMRA microscope (Leica, http://www.leica.com/) with a 25&#215;/0.65 NA objective (total magnification, &#215;162). Images were processed using Adobe Photoshop (Adobe Systems, http://www.adobe.com). Mouse IFN-&#945; concentration in cell culture supernatants was measured by ELISA (PBL Biomedical Laboratories, http://www.interferonsource.com/) according to the manufacturer's instructions. IFN-&#945; treatment of cells prior to MHV infection was performed using universal type I IFN (IFN-&#945;A/D, Sigma).\nIntracellular cytokine staining.\nSpecific ex vivo production of IFN-&#947; was determined by intracellular cytokine staining. Organs were removed at the indicated time points following infection with recombinant MHV. For intracellular cytokine staining, single cell suspensions of 1 &#215; 106 splenocytes were incubated for 5 h at 37 &#176;C in 96-well round-bottom plates in 200 &#956;l of culture medium containing 25 U/ml IL-2 and 5 &#956;g/ml Brefeldin A (Sigma). Cells were stimulated with phorbolmyristateacetate (PMA, 50 ng/ml) and ionomycin (500 ng/ml) (both purchased from Sigma) as positive control or left untreated as a negative control. For analysis of peptide-specific responses, cells were stimulated with 10&#8722;6 M GP33 peptide or 10&#8722;4  M MHV S598 peptide. The percentage of CD8+ T cells producing IFN-&#947; was determined using a FACSCalibur flow cytometer (BD Biosciences). Both S598 (RCQIFANI) and GP33 (KAVYNFATC) peptides were purchased from Neosystem (http://www.neomps.com/).\nStatistical analysis.\nAll statistical analyses were performed with Prism 4.0 (GraphPad Software, http://www.graphpad.com/). Data were analyzed with the paired Student's t-test assuming that the values followed a Gaussian distribution. A p-value of < 0.05 was considered significant.\nSupporting Information\n\nAccession Numbers\nThe GenBank (http://www.ncbi.nlm.nih.gov/Genbank/) accession numbers for the viruses and sequences discussed in this paper are HCoV-229E (AF304460), MHV-A59 (AY700211), and SARS-CoV Frankfurt-1 (AY291315).",
        "pmid": "17696607",
        "title": "Coronavirus Non-Structural Protein 1 Is a Major Pathogenicity Factor: Implications for the Rational Design of Coronavirus Vaccines",
        "journal_title": "PLoS Pathogens",
        "authors": "Roland Z\u00fcst;Luisa Cervantes-Barrag\u00e1n;Thomas Kuri;Gjon Blakqori;Friedemann Weber;Burkhard Ludewig;Volker Thiel",
        "doi": "10.1371/journal.ppat.0030109"
    },
    {
        "keywords": "virus classification;machine learning;random forest;protein domains;Pfam;profile Hidden Markov model",
        "abstract": "Abstract\nThe Coronaviridae family of viruses encompasses a group of pathogens with a zoonotic potential as observed from previous outbreaks of the severe acute respiratory syndrome coronavirus and Middle East respiratory syndrome coronavirus. Accordingly, it seems important to identify and document the coronaviruses in animal reservoirs, many of which are uncharacterized and potentially missed by more standard diagnostic assays. A combination of sensitive deep sequencing technology and computational algorithms is essential for virus surveillance, especially for characterizing novel- or distantly related virus strains. Here, we explore the use of profile Hidden Markov Model-defined Pfam protein domains (Pfam domains) encoded by new sequences as a Coronaviridae sequence classification tool. The encoded domains are used first in a triage to identify potential Coronaviridae sequences and then processed using a Random Forest method to classify the sequences to the Coronaviridae genus level. The application of this algorithm on Coronaviridae genomes assembled from agnostic deep sequencing data from surveillance of bats and rats in Dong Thap province (Vietnam) identified thirty-four Alphacoronavirus and eleven Betacoronavirus genomes. This collection of bat and rat coronaviruses genomes provided essential information on the local diversity of coronaviruses and substantially expanded the number of coronavirus full genomes available from bat and rats and may facilitate further molecular studies on this group of viruses.",
        "body": "1. Introduction\nThe Coronaviridae family comprises enveloped positive-sense single-stranded RNA viruses of the order Nidovirales with a genome of up to 32&#8201;kb in length. The family is divided into Coronavirinae and Torovirinae sub-families, which are further divided into six genera: Alphacoronavirus, Betacoronavirus, Gammacoronavirus, Deltacoronavirus, Torovirus, and Bafinivirus. While viruses in the genera Alphacoronaviruses and Betacoronaviruses infect mostly mammals, the Gammacoronavirus infect avian species and members of the Deltacoronavirus genus have been found in both mammalian and avian hosts (de Groot 2012; Drexler, Corman, and Drosten 2014).Coronaviruses (CoVs) cause a range of respiratory, enteric, and neurological diseases in human and animals. In human CoV infections, the severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) cause severe respiratory tract disease with high mortality rates, and there is strong evidence of zoonosis for both viruses (Hu et al. 2015; Dudas et al. 2018; Leopardi et al. 2018). Given such zoonotic movement, detailed descriptions of the Coronaviridae in broad animal reservoirs that may cross the host barriers to cause diseases in humans are important; many of these Coronaviridae strains in animal reservoirs could represent uncharacterized strains and be missed by conventional diagnostic assays.Advances in nucleic acid sequencing technology (commonly termed Next-Generation Sequencing, NGS) are providing large sets of sequence data obtained from a variety of biological samples and allowing the characterization of both known and novel virus strains. Algorithms that can accurately and rapidly detect and classify low-frequency virus sequences amidst a high-sequence background are useful. The desired features of these classification algorithms are the ability to rapidly process large number of sequences and to accurately identify more distantly related sequences. Use of such tools in the field during outbreak sequencing is common, thus methods that are stand-alone requiring no internet connection are desirable.All viruses encode a collection of proteins required to ensure self-replication and persistence of the encoding virus. Enzymes for genome mRNA production and genome replication, proteases for protein maturation, proteins for genome encapsidation, and proteins for undermining the host antiviral responses can all be identified conserved protein motifs or domains. Likely because of selective pressures, viral genomes are streamlined and the functional protein content encoded by viruses is much higher than for a cellular organisms. Thus, describing a viral genome by the collection of encoded protein domains is a potentially useful classification method that we would like to explore in more detail.Profile Hidden Markov Models (HMMs) provide a probabilistic framework for describing multiple sequence alignments that can reveal position-specific patterns (Krogh, Mian, and Haussler 1994; Eddy 1996, 1998; Durbin et al. 1998; Sonnhammer et al. 1998). The Pfam protein families database (Finn et al. 2016) of >16,000 protein domains is available (Pfam 31.0 at http://Pfam.xfam.org/). Within the Pfam collection, each domain family is defined by a manually selected and aligned set of protein sequences, which is used to construct a profile HMM of the domain. The HMM domain concept and search algorithms for generating and detecting profile HMMs have gone through a number of refinements and a current rapid implementation for finding profile HMMs in novel sequences is HMMER3 (http://hmmer.org/, Eddy 2011).A number of strategies have been developed to use protein domains for virus sequence classification. The Virus Pathogen Resource (ViPR) site has a useful compilation of Pfam domains found in specific virus families (https://www.viprbrc.org/brc/home.spg? decorator=vipr) however the catalog is currently limited to fourteen virus families while the International Committee on Taxonomy of Viruses (ICTV) currently recognizes ninty-six virus families (Lefkowitz et al. 2018). The use of profile Hidden Markov Models (HMMs) for virus classification and discovery was recently reviewed (Reyes et al. 2017). The use of an HMM structure as the basis for sequence classification has the potential to identify more distant members of protein domain family. Both Metavir (Roux et al. 2011) and VirSorter (Roux et al. 2015) make extensive use of protein domains as part of effective virus classification algorithms. The implementation VirSorter is primarily focused on identifying novel bacteriophage sequences. MetLab (Norling et al. 2016) and vFAM (Skewes-Cox et al. 2014) methods have demonstrated the utility of such a protein domain classification approach. ClassyFlu (Van der Auwera et al. 2014) builds influenza subtype specific profile HMM-defined protein domains for the HA coding region and then uses this database of HMMs to classify test influenza HA segments.We describe here a strategy using Pfam protein domains as the basis for identifying and classifying Coronaviridae genomic sequences. We show that the method can be used for rapid identification of Coronaviridae sequences in de novo assembled contigs, although sensitivity requires longer, ideally genome-length contigs. If sufficient sequence across the virus genome is available, the method can provide virus classification to the genus level. We then employ this method to identify fourty-five novel Coronaviridae genome sequences from random-primed deep sequencing data from bats and rats sampled from Dong Thap province of Vietnam.\n2. Materials and methods\n\n2.1 Study setting and design\nFecal pellets from Scotophilus kuhlii bats were collected from roosts on bat guano farms in the Dong Thap province in southern Vietnam, &#8764;150&#8201;km south west of Ho Chi Minh City as shown in the map (Fig. 1). Rat fecal pellets from Rattus argentiventer were collected from trapped rice-field rats or from rats purchased in wet markets in Dong Thap (locations are indicated in the map, Fig. 4A). Samples were stored at &#8722;80&#176;C until processed for NGS. Approvals for the study were obtained from the Oxford Tropical Research Ethics Committee (Approval No. 15&#8211;12) (Oxford, UK), the institutional ethical review board of Dong Thap Provincial Hospital and the Sub-Department of Animal Health, Dong Thap province (Dong Thap, Vietnam).\n2.2 Sample processing, library preparation, and NGS\nTotal nucleic acid was extracted as previously described (de Vries et al. 2012; Cotten et al. 2014). In brief, a volume of 110&#8201;&#181;l of each sample was centrifuged for 10&#8201;min at 10,000&#8201;&#215;&#8201;g. Unprotected (non-encapsidated) DNA in the samples was degraded by addition of 20&#8201;U TURBO DNase (Ambion). Remaining nucleic acid was subsequently extracted using the Boom method (Boom et al. 1990). Reverse transcription was performed using non-ribosomal random hexamers (Endoh et al. 2005), and second strand DNA synthesis was performed using 5&#8201;U of Klenow fragment (New England Biolabs) followed by phenol/chloroform extraction and ethanol precipitation. Illumina libraries were prepared for each sample, the material was sheared to 400&#8211;500&#8201;bp in length, separately indexed, and multiplexed at ninety-six samples per HiSeq 2500 run, generating two to three million 250-nt paired-end reads per sample.\n2.3 \nRaw sequencing reads were trimmed to remove residual sequencing adapters and trimmed from the 3&#8242; end to a median Phred score&#8201;>&#8201;35 using QUASR (Watson et al. 2013). The quality controlled reads were assembled into contigs using de novo assembly with SPAdes 3.10 (Bankevich et al. 2012). Coverage was estimated for contigs followed by additional filtering for minimum contig size cutoff (300&#8201;nt). Final details on the genomes including GenBank accession numbers, sample locations, and collection dates can be found in Table 1. For most of the samples, complete or nearly complete genomes were obtained from the original SPAdes assembly. However, in a subset of samples (usually those with mixed infections or with too high-sequence coverage), SPAdes yielded two or more subgenomic contigs that were manually checked by consulting short reads and re-assembled.\n2.4 Phylogenetic analyses\nGlobal coronavirus reference sequences sharing &#8805;80 per cent nt similarity to the reported CoVs in this study were retrieved from GenBank in addition to selected reference CoV sequences for comparison. The coding regions of spike protein from all reference and the assembled Coronaviridae sequences were extracted and aligned in MUSCLE (Edgar 2004), followed by manual check in AliView (Larsson 2014). The best-fitted nucleotide substitution models were determined in IQ-TREE v1.5.2 using the Akaike Information Criterion (Nguyen et al. 2015). Maximum likelihood (ML) phylogenetic trees were inferred in IQ-TREE employing GTR&#8201;+&#8201;I&#8201;+&#8201;&#915;4 model of substitution, bootstrapping for 1,000 pseudoreplicates. Bootstrap values of &#8805;70 per cent were considered as statistically significant, and resulting trees were visualized and edited in FigTree v1.4.3 (Rambaut 2016).\n2.5 Protein domain database\nUsing the HMMER3 hmmsearch function (Eddy 2011) and the Pfam collection of profile HMM protein domains, we examined all available Coronaviridae full genome sequences from GenBank (as of January 2018; N&#8201;=&#8201;2,255). A set of seventy-nine Pfam domains were found at least once in a Coronaviridae genome and these domains formed the basis for the classification methods explored here. The seventy-nine Pfam domains used for Coronaviridae classification and their frequencies in the set of 2,255 Coronaviridae genomes are listed in Table 2.A Random Forest (RF) classification using the Scikit-learn (Pedregosa et al. 2011) RandomForestClassifier module was performed on the initial triage contigs using a full genome Coronaviridae genera training set as follows. For all full genomes in each of the six Coronaviridae genera, all six open reading frames were translated and peptides&#8201;&#8805;&#8201;100 amino acids in length were collected. HMMER3 hmmsearch (Eddy 2011) was applied to the set of peptides, screening against the Coronaviridae Pfam domains ever found in the Coronaviridae (see Table 2). For domain hits with e-values&#8201;<&#8201;0.01, the Pfam domain scores were collected into an array organized by genera. An RF model using these six genera sets was then trained and built from 1,000 random trees.Classification of novel query sequences was performed as follows. For each query nucleotide sequence, all six open reading frames were translated and peptides&#8201;&#8805;&#8201;100 amino acids in length were collected. HMMER3 hmmsearch (Eddy 2011) was applied to the set of peptides, screening against the database of Coronaviridae Pfam domains and for domain hits with e-values&#8201;<&#8201;0.01, the domain scores were collected into an array. An initial triage was performed to retain only contigs with at least one of the five Coronaviridae Triage Domains (CTDs, see below) to yield the initial triage contigs. The RF model was then applied on each query contig and the genus level probability was calculated as the mean predicted probabilities of 1,000 random trees in the forest. A CSV table with the probabilities and a heatmap of the same data are the output from the process.To facilitate the use of this tool for virus discovery and classification, the process of HMMER3 Pfam domain identification, the encoding of the domain content into a matrix and the RF classification against all available Coronaviridae genomes was incorporated into a single, platform-independent Docker image (available for download at https://hub.docker.com/r/matthewcotten/cotten_myphan_coronavirus_classification_tool/). The tool can be downloaded and installed on any computing platform (Unix, Mac, Windows) and includes all required dependencies. Further details on installation and running the tool can be found at the Docker hub link.\n2.6 Forty-one virus mock contig set\nFor testing the specificity and sensitivity of the classification methods, a test set of random genome fragments from forty-one virus families including two of each of the six Coronaviridae genera was prepared. The set was derived from 492 full genomes (12 genomes from each of 40 virus families) plus 2 genomes from each of the 6 Coronaviridae genera. For each genome, 300 random fragments in the size range from 500&#8201;nt to full genome size were prepared, combined resulting in a total of 111,577 fragments including 3,316 Coronaviridae fragments.\n3. Results\nThe Pfam domain content of all 2,255 available Coronaviridae genomes was determined and a variety of distribution patterns were observed, from domains whose frequency across all Coronaviridae was >95 per cent, to rare domains present in only a few known genomes. To illustrate this variety, two genomes examples of Alphacoronavirus, Betacoronavirus, Gammacoronavirus, Deltacoronavirus, Torovirus, and Bafinivirus were selected and all domains encoded by the full genomes were identified and their positions in each virus genome is marked by colored rectangles indicating frequent, moderate, or rare occurrence (Fig. 2). The domain content is both extensive and varied by genus and this information might be used to identity and classify Coronaviridae sequences.A hierarchical-clustering of domain content of all genomes in each genus showed three distribution patterns (Fig. 3). Domains present in a high frequency in single genus (upper-third of the cluster map), domains present at high frequency in most or all genera (bottom-third of the cluster map), and domains with low frequency in some genomes or genera (middle of the cluster map). In particular, five domains (AAA_30, Macro, Viral_helicase1, AAA_11, NSP13) were found to be encoded by genomes from all six Coronaviridae genera in >95 per cent of the 2,255 Coronaviridae genomes examined. We define these domains as Coronaviridae Triage Domains (CTDs). Of the five CTDs, three were promiscuous (encoded in all Coronaviridae genomes as well as in other virus families), while two domains appeared specific for Coronaviridae (AAA_11 and NSP13, termed Coronaviridae Absolute Triage Domains (CATD)) and were encoded in all Coronaviridae genomes, but were not found in genomes from forty other common virus families infecting animals (results not shown).\nThe utility of domain content for Coronaviridae classification was first tested by developing a simple triage method to identify potential Coronaviridae sequence contigs. Preliminary work identified four triage conditions as useful for this purpose. These triage conditions were CTD_any (the contig encodes at least one of the five CTDs), CTD_all (the contig encodes all five CTDs), CATD_any (the contig encodes at least one of the two CATDs) CATD_all (the contig encodes both CATDs). The presence of these domains was combined with contig length cutoffs of 500&#8201;nt, 3,000&#8201;nt, 10,000&#8201;nt, and 20,000&#8201;nt.The performance of these triage conditions for identifying coronavirus contigs was examined. Using a mock contig set derived from forty-one virus families (see Section 2), the Pfam domain content of each fragment was determined using HMMER3 and used to sort the fragments into Coronaviridae groups based on fragment length plus CTD and CATD domain content. The accuracy of the classification was assessed in comparison to the classification of the original genome annotation in GenBank (Fig. 4). We ran each classification process five times to control for the random selection of features.The analyses were run with four size cutoff classes and sensitivity/specificity values for correct classification were color and shape coded for Triage method (&#8805;500&#8201;nt panel A, &#8805;3,000&#8201;nt panel B, &#8805;10,000&#8201;nt panel C, and &#8805;20,000&#8201;nt panel D, Fig. 4). The highest sensitivity/specificity values were obtained with the CTD_all triage for all four size classes (dark blue triangles, Fig. 4). Overall performance was observed in the order CTD_any&#8201;<&#8201;CATD_any&#8201;<&#8201;CATD_all&#8201;&#8201;0.975.We next applied this protein domain-based method to classify Coronaviridae genomic sequences generated from next-generation sequencing surveillance data. The NGS data were derived from bat and rat fecal samples collected in the Dong Thap province (Fig. 1) and processed for agnostic, random primed NGS as described in Section 2. All de novo assembled Coronaviridae contigs that passed the quality control and minimum length cutoff were subjected to Pfam domain content identification, triage by CATD content and length and RF classification. This process is summarized in Fig. 5. The process identified thirty-four potential Coronaviridae genomes from 177 bat fecal samples and 11 Coronaviridae genomes from 391 rat fecal samples. These forty-five genomes were classified to the Coronaviridae genus level using the Coronaviridae classification tool (Fig. 6).\nNote that screening using specific PCR targeting the conserved RNA-dependent RNA polymerase (RdRp) gene had previously identified CoV sequences in some of the same samples (29). The RdRp sequences generated in that study were closest to the bat Alphacoronavirus NC_009657 (Scotophilus bat coronavirus 512) and the rat Betacoronaviruses NC_026011 (Betacoronavirus HKU24) or KF294372 (Longquan Rl rat coronavirus). These are likely to be the same viruses described here at the full genome level.Two lineages of Alphacoronavirus were identified in the surveyed bat samples, one group showed some relationship to the Scotophilus bat coronavirus 512 (GenBank NC_009657, Tang et al. 2006) with 96 per cent shared identity across the genome. The other group of Alphacoronavirus was distant from any known Alphacoronavirus strains and may represent new species. Two groups of Betacoronavirus were identified in rat samples, the closest available virus genome in GenBank was the Betacoronavirus_HKU24 (KM349743) and the Longquan mouse coronavirus (KF294357) with 95 and 94 per cent shared identity across the entire genome. The next closest coronaviruses were the human coronavirus OC43 and the porcine hemagglutinating encephalomyelitis virus (PHEV) with &#8764;70 per cent shared amino acid identity across the entire genome. The genome organization for the new coronaviruses was similar to the closest reference genomes sharing similar open reading frame organization as well as similar Pfam domains (Fig. 7A). Furthermore, the expected ribosome slippage sequences between the ORF 1&#8201;A and 1AB, as well as the repeat sequences and protease cleavage sites were all present in the new CoVs genomes (results not shown).To examine the relationship between the reported viruses and known Alpha- and Betacoronaviruses, the spike protein encoding regions of these genomes were compared with the spike coding regions from the most closely related coronavirus genomes from GenBank. Consistent with observation at the full genome scale, phylogenetic analyses suggested that the Vietnamese bat Alphacoronaviruses belonged to two lineages; viruses in the one lineage are closely related (sharing 94&#8211;96 per cent nt identities) to Scotophilus bat CoVs strains A515 (DQ648719), A527 (DQ648791), CYCU-S1/TW/2013 (KT346372), and 512 (NC_009675), while viruses in the second lineage were more distantly related (sharing 75&#8211;76 per cent nt identities) to the four previously mentioned bat CoVs strains (Fig. 7B). The CoVs identified from Vietnamese rats were classified as Betacoronavirus and belonged to two distinct lineages as shown in phylogenetic tree (Fig. 7B); viruses from one lineage closely related to the CoV strain HKU24 from Hongkong (KM349743, Lau et al. 2015), while viruses in the second lineage are more related to the rat CoV strains Longquan-189 (KF294370) and -370 (KF294371) (Wang et al. 2015) and rat CoVs from China (KY370051, KY370049, KY37048, and KY370043).\n4. Discussion\nMembers of the Coronaviridae family of viruses cause health problems in a variety of animal hosts. MERS-CoV often moves from camels to humans (Memish et al. 2014; Dudas et al. 2018) and can spread in health care systems with serious consequences (Assiri et al. 2013). SARS-CoV moved from civet cats to humans and caused substantial morbidity and mortality before it was brought under control (Poon et al. 2004). Several porcine coronaviruses cause frequent problems including PEDV (Kocherhans et al. 2001) and swine acute diarrhea syndrome coronavirus (SADS-CoV) (Zhou et al. 2018). Given the frequent association of Coronaviridae members with severe diseases, a more comprehensive description of Coronaviridae diversity, especially in animals with frequent human contact, is an important objective.We describe a Coronaviridae sequence classification strategy based on the set of protein domains encoded by the genome sequence. The classification is not dependent on a single domain, but rather the composite score of all domains present in the query sequence. This is a strength of the method that can limit false positive identifications which might be due for example, to shorter regions of homology to a bacterial or host or repetitive sequence. The requirement for longer sequence contigs is also a weakness of the method as sufficient query sequence must be available to encode multiple protein domains. This also limits the tool to assembled contigs rather than short read data. In other words, the sensitivity of the classification is directly dependent upon the length of the genomic sequences, that is, higher sensitivity of genus assignment with longer or complete genome sequence. The classification tool provides a robust, rapid, and alignment-free method to classify large sets of more distantly related sequences. Once a database is generated, the algorithm can be used in the field or resource-limited settings and the classification can be performed with typical contig sets within minutes on a standard laptop. With the availability of the platform independent Docker version of the algorithm (see Section 2), scientists can easily run the analyses on any computing platform.Given the large number of genomes available for most of the Coronaviridae genera, this domain-based classification method can provide a sensitive measure of genome and annotation quality. One consideration is that the genus classification may be broad and the diversity within that genus includes genomes with more distant variations in the protein domains. An additional consideration is that the genus classification of individual Coronaviridae genomes in GenBank may not be correct (mis-annotation), that the genome sequences may include errors (machine errors, PCR errors, chimeric sequences) or have been assembled incorrectly or with sequence duplications or deletions (mis-assemblies). The domain method described here can help identify these patterns.Bats have been suggested to harbor great diversity of CoVs and play a key role in the emergence and transmission of pathogenic CoVs causing severe diseases in human (Menachery, Graham, and Baric 2017). Rats, on the other hand, represent the largest order of mammalian species and are potentially a major zoonotic source of human infectious diseases (Meerburg, Singleton, and Kijlstra 2009; Luis et al. 2013). As part of a large-scale zoonotic surveillance in Vietnam (Rabaa et al. 2015), we applied agnostic deep sequencing to 177 bat and 391 rat samples from a single location. The sample collection was from the Dong Thap province in southern Vietnam where humans and domestic and farm animals live in close proximity. The site is 154&#8201;km from Ho Chi Minh City, the largest city in south of Vietnam. From this modest sample size surveillance, forty-four complete or nearly complete genomes belonging to Coronaviridae family were identified, thirty-four of which were from bat samples belonging to the Alphacorovirus genus and eleven genomes from ten rat samples belonging to the Betacoronavirus genus. The bat fecal samples were pooled material from five to ten individuals, thus the total individual bats samples screened ranged from 885 to 1,770 and the frequency of full genome identification was 1.9&#8211;4.0 per cent (34/1,770 to 34/855). In comparison, the 391 screened rat samples were derived from individual fecal pellets and frequency of the coronavirus genome identification was 2.8 per cent (11/391). Given this small sample size, the frequency of CoVs identified was not strikingly different between bats and rats.While it is possible to use Blast methods to identify/classify the putative viral genomes, one advantage of the tool described here is that it rapidly provides a genus-level classification, which is not directly obtained from a Blast search. Certainly, one could use a Blast search to find the closest sequence in a database and then use this as a classification if (1) the closest homology sequence covers 100 per cent of the length of the query sequence, and (2) the closest entry contains sufficient classification annotation. Finally, we are not presenting this new tool as a replacement for Blast (which is a reliable and highly trusted tool). Instead, the domain-based classification method described here provides a rapid, alternative and complementary classification method that can help organize complex data sets.Of note, this is the first whole genome characterization of Alpha- and Betacoronaviruses from rats and bats from southern Vietnam. Although there are reports in the literature of enhanced CoV prevalence in various bat species (Anthony et al. 2017), at least in the current survey, when the same agnostic sequencing approach was applied to both bat and rat samples, similar virus frequencies were observed, suggesting that rats and in more general rodents may be as important a host for CoVs as bat species. Indeed surveys using similar agnostic NGS methods have identified a wide range of virus diversity in rats and other rodents (Li et al. 2015, 2016; Wang et al. 2015, 2017). The zoonotic potential of these reported viruses is yet to be defined. Prior to the current study, only single examples of these viruses were available and the availability of a larger number of full genomes for these viruses will facilitate more functional studies.Sensitive surveillance that is not dependent on known viral sequences for primer design is important for documenting circulating viruses and for identifying those that might cause problems in the future. This is particularly of great importance for virus discovery and to characterize potentially zoonotic viruses in exotic hosts that have not been described. The bat Alphacoronaviruses described here had only a single genome available in GenBank and we have enriched the public database with thirty-four additional genomes. The rat Betacoronaviruses also provide description of novel, more distant members of a recognized but poorly characterized group of CoVs. The disease potential of these viruses in their normal host and their zoonotic potential are yet to determine. Certainly, the reported sequences would add important information to the growing collection of coronavirus diversity and particular with great relevance in the context of south and east Asia where SARS-CoV first emerged. We hope they are useful for future studies, for monitoring disease of unknown origin and for increasing our understanding of the molecular diversity of these viruses.\nAccession numbers\nThe coronavirus sequences described in this study have been deposited in GenBank under the accession numbers MH687934&#8211;MH687978.",
        "pmid": "30568804",
        "title": "Identification and characterization of Coronaviridae genomes from Vietnamese bats and rats based on conserved protein domains",
        "journal_title": "Virus Evolution",
        "authors": "My V T Phan;Tue Ngo Tri;Pham Hong Anh;Stephen Baker;Paul Kellam;Matthew Cotten",
        "doi": "10.1093/ve/vey035"
    },
    {
        "keywords": "",
        "abstract": "Spillover of viruses from bats to other animals may be associated with increased contact between them, as well as increased shedding of viruses by bats. Here, we tested the prediction that little brown bats (Myotis lucifugus) co-infected with the M. lucifugus coronavirus (Myl-CoV) and with Pseudogymnoascus destructans (Pd), the fungus that causes bat white-nose syndrome (WNS), exhibit different disease severity, viral shedding and molecular responses than bats infected with only Myl-CoV or only P. destructans. We took advantage of the natural persistence of Myl-CoV in bats that were experimentally inoculated with P. destructans in a previous study. Here, we show that the intestines of virus-infected bats that were also infected with fungus contained on average 60-fold more viral RNA than bats with virus alone. Increased viral RNA in the intestines correlated with the severity of fungus-related pathology. Additionally, the intestines of bats infected with fungus exhibited different expression of mitogen-activated protein kinase pathway and cytokine related transcripts, irrespective of viral presence. Levels of coronavirus antibodies were also higher in fungal-infected bats. Our results suggest that the systemic effects of WNS may down-regulate anti-viral responses in bats persistently infected with M. lucifugus coronavirus and increase the potential of virus shedding.",
        "body": "Introduction\nBats are hosts for many viruses and are thought to be the source of some viruses that have spilled over to humans and other mammals, causing fatal disease. These include coronaviruses causing severe acute respiratory syndrome (SARS1), Middle East respiratory syndrome (MERS2&#8211;5), porcine epidemic diarrhoea (PED6) and swine acute diarrhoea syndrome (SADS7); paramyxoviruses such as Hendra8 and Nipah9; and filoviruses like Marburg10 and Ebola11. Four families of viruses that are pathogenic for other mammalian species (Coronaviridae12, Paramyxoviridae13, Rhabdoviridae14 and Filoviridae15) may also have originated in bats. These viruses often cause serious disease in their secondary hosts, but most do not appear to cause clinical signs or pathology in bats16&#8211;18, suggesting that uniquely benign relationships have co-evolved between the viruses and their primary bat hosts19,20. While relatively little is known about the dynamics of viral infections in bats, these viruses may be maintained in bat populations as a result of either persistently infected individuals, reinfection after waning immunity, or spatial transmission dynamics21,22.The rare spill-over of bat viruses to other animals may require a &#8220;perfect storm&#8221; of conditions that include increased contact between bats or fomites and other mammals, possibly due to human impacts on habitat quality23, and the ability of the virus to infect, replicate, and transmit in the secondary host. The rate of viral shedding and the amount of detectable virus associated with bat colonies fluctuates, with periodic increases often linked to parturition, waning maternal immunity, nutritional stress or increased energy consumption17,24&#8211;29. Increased shedding of virus by a colony of bats may reflect an increase in the proportion and number of susceptible individuals, or an increase in the replication of persistent or latent virus normally suppressed by the host. For herpesviruses, reactivation from latency is linked to perturbations caused by a variety of physiological, immunological and psychological stressors30. The mechanisms that trigger the reactivation of latent or persistently infecting viruses are not clearly understood, but the increased shedding of viruses is correlated with some incidents of spill-over of bat viruses to other animals31.The Canadian prairies are home to three species of bats, including the little brown bat (Myotis lucifugus), big brown bat (Eptesicus fuscus), and northern long-eared bat (Myotis septentrionalis). All three species hibernate from October to May, sometimes in shared hibernacula. We recently demonstrated that ~30% of hibernating M. lucifugus sampled over two years from hibernacula in Manitoba were infected with a coronavirus (Myl-CoV), which persisted at low levels in the intestine32. A closely related coronavirus also infects E. fuscus33.Whereas bats appear to be relatively resistant to viral infections, a cold-adapted fungus that was recently introduced to North America has caused widespread mortality in some species of bats in eastern United States and Canada34&#8211;37. The fungus (Pseudogymnoascus destructans) causes white-nose syndrome (WNS) in hibernating bats, which is characterized by the growth of white fungal mycelia on the face and exposed skin of the wings and tail membranes. The visual and microscopic effects of P. destructans on the skin of the wings are associated with increased expression of several genes devoted to innate immunity and inflammation in wing tissue38,39. Profound systemic effects include dehydration, hypovolemia, metabolic acidosis, and fat depletion, which can lead to death40&#8211;42. Other systemic effects of bat WNS include an accumulation of neutrophils in the lungs, which is accompanied by an increase in the expression of several cytokine genes43 suggesting that even the most severely afflicted hibernating bats are capable of at least some systemic immune response to fungal infection.Previous studies on other species have demonstrated that a fungus and a virus could interact during co-infection and affect each other44,45. Similar interactive impacts of co-infection with P. destructans and viruses on bat immune responses are not known. We used M. lucifugus experimentally-infected with P. destructans and/or naturally infected with Myl-CoV as a model to understand how co-infections influence bat-virus interactions. This system allows us to avoid confounding factors of direct pathogen-pathogen interactions, because the fungus affects the skin, while the coronavirus infections occur internally, almost exclusively in the ileum and lungs32. We hypothesized that co-infection would alter the molecular response of bats to a persistent viral infection, and that viral shedding would change as a result of the increased or disrupted host immune response. To test this prediction, we examined tissues collected from M. lucifugus at the termination of an earlier study that quantified the effects and pathogenesis of P. destructans in hibernating bats experimentally infected with the fungus37, some of which were naturally infected with Myl-CoV32. This combination of uninfected, virus-infected, fungus-infected and co-infected M. lucifugus allowed us to test our hypothesis that host responses to co-infection are synergistic and not simply additive.\nResults\nQuantitation of Myl-CoV and M. lucifugus RNA through reverse transcription quantitative PCR (RT-qPCR) and dual-RNA-sequencing indicated that co-infected bats had significantly higher levels of Myl-CoV RNA than bats infected with virus alone. The amount of Myl-CoV RNA correlated with the severity of WNS pathology in co-infected bats. This phenomenon was associated with specific molecular responses to co-infection, even in the intestines of bats where only one of the two pathogens was directly interacting with the host tissue. The levels of antibodies against Myl-CoV nucleocapsid (N) protein were also higher in co-infected bats. Each key result is discussed in detail below.\nBats co-infected with the fungus \nMyl-CoV genomic RNA was detected in bats infected only with Myl-CoV (virus-infected; 7/18), co-infected bats (European P. destructans (3/13), or with North American P. destructans (7/16)37). There was no difference in the frequency of Myl-CoV detected among these treatments (p-value&#8201;=&#8201;0.801). We pooled bats infected with the two P. destructans isolates for all further analyses and tested whether co-infection with P. destructans and Myl-CoV correlated with an increase in viral replication. Our RT-qPCR data showed that the co-infected bats contained 60-fold more Myl-CoV RNA on average than the virus-infected bats (Mann Whitney test; p-value&#8201;=&#8201;0.014; Fig.&#160;1).Figure 1Effect of white-nose syndrome on level of Myotis lucifugus coronavirus (Myl-CoV) RNA in hibernating little brown bats (M. lucifugus). Relative transcript levels for the coronavirus RNA polymerase gene for each bat are depicted as reciprocal of Cycle threshold (Ct) normalized separately (&#916;Ct) for levels of glyceraldehyde 3-phosphate dehydrogenase (GAPDH) transcripts in each sample. The horizontal bar represents the mean while the vertical bar indicates standard deviation from the mean. Significance (p value) is as calculated with an independent Mann-Whitney test. Virus-infected bats had lower 1/&#916;Ct values for coronavirus RNA than co-infected bats. The average fold-differences between virus-infected and co-infected bats were calculated from the difference between the average &#916;Ct values.Relative quantities of Myl-CoV RNA detected in the ileum of the virus-infected bats were low and showed low variation (Standard Deviation of 1/&#916;CT&#8201;=&#8201;0.005), compared to the relative quantities of Myl-CoV RNA in the co-infected bats (Standard Deviation of 1/&#916;CT&#8201;=&#8201;0.108; Fig.&#160;1). The severity of WNS fungal pathology varied in co-infected bats, and we therefore tested whether relative quantities of viral RNA in the ileum correlated with the severity of WNS symptoms. Levels of WNS severity were scored based on fungal hyphae on the wings, secondary bacteria in wing lesions, oedema, necrosis and inflammation in wing lesions, and levels of neutrophils in lung, spleen and liver interstitium. Severity scores for wing tissue, secondary bacteria in lesions, and neutrophils in the lung interstitium positively correlated with relative amounts of coronavirus RNA in hibernating bats (Table&#160;1).Table 1Correlation between level of Myotis lucifugus coronavirus RNA and disease severity of white-nose syndrome (WNS) in co-infected M. lucifugus, based on three measures of WNS severity and pathology.CorrelateLevel of coronavirus RNAPearson CorrelationaSignificanceNVirus-infected/Co-infected&#8722;0.610\n0.009\n17Average hyphae score&#8722;0.630\n0.016\n14Average bacterial score&#8722;0.680\n0.007\n14Lung interstitial neutrophils&#8722;0.618\n0.043\n11aPearson&#8217;s coefficients were calculated for the &#8710;Ct levels for cytokine transcripts for bats in each treatment class and lung interstitial neutrophil scores and mean bacterial and hyphae scores for 5 wing sections for each bat.\nBat responses to co-infection exceed the sum of responses to virus or fungal infection alone\nTo determine the extent to which Myl-CoV and P. destructans infection interact to influence gene expression in bat intestines, we performed a transcriptomic analysis on bat intestines comparing gene expression among the uninfected, virus-infected, fungus-infected, and co-infected treatments (Fig.&#160;2(A)). RNA sequencing resulted in ~700 million paired-end reads passing filters, 65% of which aligned to the M. lucifugus genome (Table&#160;S1). Pairwise differential gene expression varied widely among the four treatments with relatively low overlap in differentially expressed transcripts (Fig.&#160;2B,C), Supplementary Fig.&#160;1). Similar transcript expression occurred between the uninfected and virus-infected bats, and between the fungus-infected and co-infected bats.Figure 2Co-infection of little brown bats (Myotis lucifugus) with M. lucifugus coronavirus (Myl-CoV) and Pseudogymnoascus destructans results in non-additive patterns of gene expression compared to sole infection with the virus or fungus. (A) Experimental design, showing the four treatments of little brown bat (Myotis lucifugus) established by experimental inoculation with Pseudogymnoascus destructans and by qPCR detection of persistent Myl-CoV infections: uninfected, virus-infected, fungus-infected and co-infected. (B) Differential gene expression identified by DESeq2 among virus-infected, fungus-infected and Co-infected bats as compared to the change each exhibited relative to uninfected bats. (C) Differential gene expression among the four treatments, detected by DESEQ2 and visualized in volcano plots. The log of the adjusted p-value is plotted as a function of the log ratio of differential expression. Colored data points represent different groups of genes based on fold change and false discovery rate (FDR) cutoff; red (>2 fold change, FDR &#8201;0.05), light grey (<2 fold change, FDR&#8201;<&#8201;0.05), black (&#8201;0.05).The fungus-infected bats exhibited a much stronger response, differentially expressing 324 transcripts compared to the uninfected bats (Table&#160;S3). These transcripts were enriched for only two gene ontology (GO) terms (cell-cell junction and plasma membrane part; Table&#160;S8). The co-infected bats differentially expressed 634 transcripts relative to the uninfected bats (Table&#160;S4). These transcripts showed significant enrichment for 16 GO terms (Table&#160;S8). The co-infected and fungus-infected bats shared 108 similar differentially expressed transcripts and overlapped in one enriched GO term relative to the uninfected bats (plasma membrane part; Table&#160;S8).\nEffect of infection with the fungus \nWhen we directly compared responses of bats among the four treatments, response of the virus-infected bats differed strongly from the responses of fungus-infected or co-infected bats (virus-infected vs. fungus-infected: 461 differentially expressed transcripts and 9 significantly enriched GO terms; virus-infected vs. co-infected: 473 transcripts and 43 enriched GO terms; Tables&#160;S5, S6, S7; Supplementary Fig.&#160;1). These differences in gene expression patterns included genes that clustered in two processes relevant to host-pathogen interactions &#8211; the mitogen-activated protein kinase (MAPK) pathways and cytokine and innate immune responses. Table&#160;2 lists genes from the two processes that were significantly either up or down-regulated when virus-infected bats were compared to co-infected bats. For the MAPK pathway-related transcripts, genes such as RSU1 and RERG were up-regulated while those, such as STYK1, RRAD, MAP3K and SRC were down-regulated. For cytokine-related genes several transcripts were suppressed. When we compared the expression of the same genes for bats with WNS (combining fungus-infected and co-infected bats) and all bats without WNS (combining uninfected and virus-infected bats), we found similar differences (last two columns of Table&#160;2). This suggested that superficial infection with fungus, P. destructans, was the driving factor for altered gene expression in the bat intestines.Table 2RNA-sequencing identified differential expression of transcripts related to the MAPK pathway and to cytokine-related processes, comparing gene expression in the ileum of little brown bats (Myotis lucifugus) infected only with the M. lucifugus coronavirus (Myl-CoV; virus-infected) or co-infected with Myl-CoV and Pseudogymnoascus destructans. The last 2 columns show the same comparisons made after grouping bats that were not exposed to the fungus, and bats that were exposed to the fungus and exhibiting symptoms of WNS (irrespective of their viral infection status).Ensembl Gene NameEnsembl DescriptionVirus-infected vs. Co-infectedAll bats &#8220;without fungus&#8221; vs. all bats &#8220;with fungus&#8221;aLog2 Fold ChangebPadjLog2 Fold ChangebPadj\nMAPK pathway-related transcripts\nSTYK1serine/threonine/tyrosine kinase 1&#8722;1.2680.025&#8722;1.516<0.0001RSU1Ras suppressor protein 11.1020.004RRADRRAD, Ras related glycolysis inhibitor and calcium channel regulator&#8722;1.2970.028&#8722;1.30.025RERGRAS like estrogen regulated growth inhibitor1.5620.0181.4160.005MAP3K11mitogen-activated protein kinase 11&#8722;1.140.040&#8722;1.130.0002SRCSRC proto-oncogene, non-receptor tyrosine kinase&#8722;1.5390.013&#8722;1.2970.0037\nCytokine-related transcripts\nIRF1Interferon regulatory factor 1&#8722;1.5510.001&#8722;1.4440.0001IFI6Interferon alpha inducible protein 6&#8722;1.7980.014&#8722;1.3520.039IL22RA1Interleukin 22 receptor subunit alpha 1&#8722;1.4110.015&#8722;1.3140.002SOCS6Suppressor of cytokine signaling 6&#8722;1.2780.008&#8722;1.534<0.0001a(Uninfected&#8201;+&#8201;virus-infected) vs. (fungus-infected&#8201;+&#8201;co-infected)bPositive log2 fold-change values indicate higher expression in the second listed treatments relative to the first.To confirm the results of the RNA-seq analysis, we selected 4 genes from Table&#160;2, namely IRF1, RERG, SRC and IL22RA1, to be verified by RT-qPCR. We also included interleukin 10 (IL10) due to its biological relevance to immune regulation and because we had previously observed an increase in its expression related to fungal infection43. As we wanted to confirm whether WNS was driving gene expression in the intestines of bats, we performed a two-group analysis for the RT-qPCR data. We combined all the bats without WNS into a single group (uninfected&#8201;+&#8201;virus-infected) and all the bats with WNS into the other group (fungus-infected&#8201;+&#8201;co-infected; Fig.&#160;3(A)). Expression of Ras-like estrogen regulated growth inhibitor (RERG) increased while expression of Interleukin 22 receptor subunit alpha 1 (IL22 RA1) genes decreased in bats with WNS, irrespective of viral infection (Fig.&#160;3C,E). Expression of the immune modulatory cytokine IL10 tended to be higher in bats with WNS than in bats without WNS, but the difference was not statistically significant (p-value&#8201;=&#8201;0.07) (Fig.&#160;3(F)).Figure 3Effect of white-nose syndrome (WNS) on the levels of immune genes IRF1, RERG, SRC, IL22RA1 and IL10 expressed in the ileum of little brown bats (Myotis lucifugus). (A) Summary of the four treatments, with a red arrow indicating the two groups (&#8220;with fungus&#8221; and &#8220;without fungus&#8221;) that were compared. (B&#8211;F) The relative transcript levels of each gene for bats with and without WNS, depicted as reciprocal of Cycle threshold (Ct) normalized separately (&#916;Ct) for levels of transcripts for GAPDH in each sample. Statistical significance was calculated based on the independent Mann Whitney test. The difference in the two groups was significant for RERG and IL22RA1 genes.\nWhite-nose syndrome is associated with increased coronavirus antibody levels in the co-infected bats\nIn 2017, we performed a similar study, experimentally exposing 63&#8201;M. lucifugus to P. destructans as described in Warnecke et al.37. We performed IgG ELISA on blood plasma to detect Myl-CoV (coronavirus) N protein antibodies and found that 21/63 were positive for antibodies against the coronavirus. Of those 21 bats, 7 had detectable coronavirus RNA in their intestines suggesting an active infection, and 3 out of the 7 had been experimentally infected with P. destructans during the course of the study. We compared the ELISA optical density (O.D.) values of these virus-infected bats to co-infected bats (Fig.&#160;4(A)) and found that the presence of P. destructans was associated with increased levels of coronavirus antibodies (Mann Whitney test, p value&#8201;=&#8201;0.03; Fig.&#160;4(B)).Figure 4Little brown bats (Myotis lucifugus) coinfected with M. lucifugus coronavirus (Myl-CoV) and Pseudogymnoascus destructans produce more antibodies against Myl-CoV than bats infected only with Myl-CoV. (A) Diagram summarizes the four treatments; the red arrow shows the two groups between which antibody levels were compared. (B) Antibody levels against the Myl-CoV N protein detected by antibody capture ELISA expressed as optical density (O.D.) values at 405&#8201;nm. Co-infected bats had significantly higher antibody levels than bats infected only with Myl-CoV (independent Mann Whitney test; p value&#8201;=&#8201;0.03).\nDiscussion\nOur findings suggest that systemic responses of bats to WNS results in increased coronavirus replication and consequently, increased viral shedding, which may lead to subsequent infection of susceptible animals. Coronavirus infection may in turn increase the severity of WNS pathology. This is the first study to examine the systemic effects of co-infection on either bat coronavirus or WNS, and our results raise important questions in regard to zoonotic spillover events. Although events of successful viral spillover to distantly related species are thought to be extremely rare, in recent years several coronaviruses have spilled over, including SARS-CoV1, MERS-CoV2&#8211;5, PEDV-CoV6 and SADS-CoV7. These viruses are thought to have originated in bats. In addition, circumstantial evidence suggests that most alpha and beta coronaviruses that parasitize other mammals may have originated in bats as well46. If so, then understanding host-pathogen interactions between bats and coronaviruses could inform our ability to predict or manage the risk of spillover. In this study, we showed that a coronavirus exhibits low activity in its natural host, M. lucifugus, but that co-infection with a fungus increases the quantity of viral RNA in the intestines. We have no reason to expect zoonotic transmission of the coronavirus i.e. Myl-CoV, but similar co-infection mechanisms may operate in tropical bat species harbouring potentially zoonotic viruses.Our results suggest that secondary skin infection with the fungus, P. destructans, substantially increases the level of viral RNA in the intestine of hibernating bats. We showed that infection of the skin with P. destructans can cause profound changes in gene expression in the intestines, despite a lack of direct contact between intestinal tissue and the fungus. Infection with P. destructans causes modulation of a number of immune responses, including down-regulation of interleukin and cell proliferation genes which may compromise bats&#8217; ability to suppress viral activity (Fig.&#160;5). Taken together, our results have implications for epidemiological studies of P. destructans, the WNS fungus and for research into viral spillovers, which should consider the potential implications of co-infections that increase viral shedding.Figure 5Hypothesized model of pathways involved in increased coronavirus shedding and white-nose syndrome (WNS) severity in little brown bats (Myotis lucifugus) co-infected with M. lucifugus coronavirus (Myl-CoV) and Pseudogymnoascus destructans. Diagram summarizes the changes observed by comparing co-infected bats with virus-infected bats. Bats with persistent Myl-CoV infection exhibit relatively low viral shedding. When bats are also infected with P. destructans (shown in yellow arrow) and develop WNS, the level of coronavirus increases. There is a change in the level of some immune genes, such as IL22, RERG and possibly IL10, which may have an effect on immune response and cell proliferation. The increase in coronavirus levels in co-infected bats is possibly due to the bats&#8217; systemic response to WNS reducing innate anti-viral responses.Complex strategies allow viruses to remain endemic in populations. These include a continuously replenished source of susceptible hosts for viruses that cause short-term acute infections with long-lasting immunity (e.g. measles virus), antigenic drift of virus (e.g. influenza virus) or waning immunity (e.g. respiratory syncytial virus) that allows reinfection, and long-lasting latent (e.g. herpesviruses) or persistent infections (e.g. pestiviruses) with sustained or periodic shedding. It is not yet clear how bat viruses are maintained in their natural host populations, or how they avoid extinction as host populations become immune and less susceptible. Persistent infections can be established in cultured cells with viruses that may have originated in bats, including Ebola virus47 and SARS-CoV48&#8211;50, but whether these viruses persist in their primary hosts is not known. Studies of persistence of bat viruses in infected bats have produced equivocal results. The lack of direct evidence supporting specific models of persistence or transmission dynamics represents a major knowledge gap in bat-virus ecology21.We maintained M. lucifugus in controlled laboratory hibernation chambers for four months during these experiments, and we detected the coronavirus i.e. Myl-CoV, at the end of hibernation. These data imply that the coronavirus can persist in its host for at least the duration of hibernation, particularly as nucleotide variability among the detected coronavirus isolates showed that spread of coronavirus among bats within a chamber was unlikely32. In an extensive study of New World alphacoronaviruses, no target viruses were detected in the rectal swabs of individual bats sampled over time46, suggesting that persistence and intensity of shedding varies among species or viruses. The authors concluded that the targeted coronaviruses do not persist in their hosts but are maintained in populations by the introduction of new susceptible individuals. However, their results could also reflect viral persistence in individual animals, with low baseline levels of virus replication and undetectable shedding interspersed with periods of increased replication and shedding that did not occur during the sampling period.Periodic or seasonal increases in virus shedding associated with parturition, lactation, nutritional deprivation or environmental stress21,29 suggest persistent or latent viruses may be activated by hormonal or other systemic cues. Direct evidence linking a specific trigger to increased shedding has not yet been found. However, viral replication in rodent and bat cells persistently infected with Ebola virus increased greatly following modulation of the Ras/MAPK pathway with lipopolysaccharides or phorbol esters, and with the resulting suppression of the cells&#8217; interferon response47&#8211;49. In experimental systems, the immune modulatory cytokine IL10 also influences viral persistence and replication51&#8211;53, although more study is required to clarify the effects of circulating cytokines on the replication of persistently infecting viruses. Nevertheless, these results suggest that circumstances which induce anti-inflammatory cytokines or suppress anti-viral innate responses, may provide a trigger for increased shedding of persistently infecting virus.We discovered that bats with WNS (fungus-infected and co-infected) had significantly lower intestinal levels of transcripts for IL22RA1 and other interferon-related genes as compared to uninfected bats, and we observed the same trend in IL10 (although it was not significant; p value&#8201;=&#8201;0.07). IL22RA1 is the receptor present on host cells, including intestinal cells, which help in initiating cellular signalling in response to IL22 produced by T-cells54. IL-22 leads to an increase in anti-microbial peptide production, cellular protection against damage and increases cellular proliferation55. Therefore, reduced IL-22 signalling in the intestines of bats with WNS, might suppress the bat defences that control the coronavirus infection. Additionally, previous studies have shown that the anti-inflammatory gene, IL10, is expressed more in the lungs of bats with WNS than in bats without it43. We saw a similar trend with the levels of IL10 in the intestines which might play a role in suppressing the immune response against the coronavirus. Another altered cytokine gene which was of interest was the suppressor of cytokine signalling-6 (SOCS6) gene. Fungal-infected bats showed lower levels of SOCS6 transcripts, lack of which has been implicated in mild growth retardation in mice56. Overall, our results suggest that WNS triggers changes in gene expression in the ileum (Fig.&#160;5). These may influence expression of interferon-stimulated-genes (ISGs), thereby leading to increased viral replication at the site of viral persistence. Interferon-related transcripts were more highly expressed in the ileum of virus-infected bats that did not have WNS, suggesting that the bat&#8217;s response to WNS causes down-regulation of interferon activity. Interferons may control coronavirus replication, as seen in cases of SARS-CoV57 and MERS-CoV58. Therefore, a decrease in interferon activity might cause an increase in coronavirus (Myl-CoV) replication. In addition to interferon-related genes, we also found that RERG, which is related to growth inhibition, was upregulated in the fungus-infected bats when compared to virus-infected bats. Upregulation of RERG could affect the rate of cell proliferation in the intestines59. Finally, this cascade of responses is associated with increased severity of WNS symptoms.Bats with WNS experience a range of systemic disturbances including dehydration, hypovolemia, metabolic acidosis and fat depletion40,41, neutrophil infiltration of the lung interstitium, and increased expression of transcripts related to anti-microbial and pro- and anti-inflammatory cytokines43. Taken together, this evidence suggests that hibernating bats respond systemically to superficial fungal infection, and this hypothesis is further supported by our observations of altered gene expression in the ileum of fungus-infected bats.Based on our results, we propose a model for how secondary infections may increase the replication and subsequent shedding of persistently infecting virus (Fig.&#160;5). The establishment of WNS (or other secondary infection) impacts the tissue with which that pathogen interacts (in the case of P. destructans, the skin). Direct interactions between the host and the secondary pathogen are limited to the affected tissue, but the systemic response to the disease triggers a cascade of immune responses, including increased release of cytokines or neutrophils. Affected cells such as intra-alveolar macrophages in the lungs or cells lining the intestine, may produce pro- or anti-inflammatory molecules and influence cells that harbour viral genomes. This cascade of host responses disrupts the equilibrium between the persistently infecting virus and the cell&#8217;s innate immune response, leading to a dramatic increase in the expression of coronavirus (Myl-CoV) replication.Our assays were unfortunately limited to analysing viral and cytokine transcripts rather than protein, because reagents for detecting bat viral and host proteins are not yet available. We were not able to perform serial dilutions of the plasma to precisely quantify anti-viral titre due to the limitation in the amount of plasma obtained from each bat. The sample size was also small for this assay, because only 7 of the sampled bats had detectable levels of coronavirus in their intestines and were positive for viral antibodies. Despite these limitations, we demonstrated higher antibodies against the coronavirus in the plasma of co-infected bats when compared with virus-infected bats. This increased antibody level in co-infected bats might reflect an adaptive immune response to increased coronavirus replication in the intestines.Our proposed hypothesis for the mechanism driving increased viral replication following pathogenic co-infection was worth testing, but our results are also consistent with an alternative hypothesis. Increased viral replication or viral load may affect the severity and population-level impacts of WNS. Bat mortality following the arrival of WNS varies widely from site to site, with populations decreasing from 30% to 99%35. Variation in the microclimate, and other ecological factors may drive some of this variation60, but our data suggest that cryptic viral infections may also play a role in determining survival rates for bats hibernating in sites colonized by P. destructans. We recommend that future studies on population-wide impact of WNS incorporate viral sampling to help better understand the role of co-infections on bat populations in the wild.\nMaterials and Methods\n\nSample acquisition\nFifty-four male M. lucifugus were collected from a WNS-free cave in Manitoba, Canada in November 2010. Details of the experimental design as well as protocols for collecting and transporting bats, infection with P. destructans, maintenance of bats in hibernation and sample collection have been described previously37,43. Briefly, bats in groups of 18 were either sham-inoculated or inoculated with North American or European isolates of P. destructans. Bats were housed at 7&#8201;&#176;C and >97% relative humidity with ad libitum water. All bats were equipped with data loggers to monitor skin temperatures. Bats were euthanized during the experiment when humanely required or at the termination of the experiment 120 days after inoculation. Immediately following euthanasia samples from segments of wing as well as various tissues were preserved in RNAlater (Qiagen, 76016) or in formalin. Samples in RNAlater were kept at &#8722;20&#8201;&#176;C until they were processed. North American and European isolates of P. destructans caused similar disease outcomes37, so we did not differentiate between the strains in subsequent analysis. The procedures for care, handling and euthanasia of bats were approved by the University Committee on Animal Care and Supply of the University of Saskatchewan (Protocol #20100120). Bats were collected under the province of Manitoba Wildlife Scientific Permit WB11145.In 2017, a further 129&#8201;M. lucifugus were collected from a WNS-free cave in Manitoba, Canada in January under the Manitoba Sustainable Development Wildlife Scientific Permit No. SAR16009. Bats were euthanized during the experiment when humanely required or at the termination of the experiment 70 days after infection and a similar experiment was performed at the University of Winnipeg as described above (Protocol #AE08399).\nHistological classification\nDuring necropsy, we collected representative samples for histopathology from all major organ systems. In addition, representative samples were taken from all areas of the wing and rolled on dental wax before placing in 10% neutral buffered formalin. Tissues were processed routinely for histology. Five &#181;m sections were cut and stained with periodic acid-Schiff stain to highlight fungal hyphae. Liver and other tissues were processed routinely and stained with hematoxylin and eosin. Wings were scored on a scale of 0 to 5 with 5 being very severe with >50% of wing covered in fungal hyphae. We used a bacterial score from 0 to 5, with 5 indicating wide-spread and abundant bacteria being present in many areas within the dermis and underlying connective tissues. Average scores from 5 sections of wing were used for analysis. Interstitial lung neutrophil assessment was similarly evaluated on a scale of 0 to 5, with 5 being very severe. Average scores from the 5 sections were used for analysis.\nRNA Extraction\nTissues preserved in RNAlater were homogenized in 2&#8201;ml sealed vials with a 5&#8201;mm stainless steel bead, 0.1&#8201;g of 0.1&#8201;mm zirconia/silica beads and 350&#8201;&#956;l Buffer RLT Plus (with &#946;-mercaptoethanol, RNeasy Plus Mini Kit, Qiagen, 74136) using a Retsch MM400 Oscillating Mill at 30&#8201;Hz for 4&#8201;min. Total RNA was extracted following the manufacturers protocol. RNA integrity was assessed using RNA 6000 Nano Kit (Agilent, 5067-1511) with the Agilent 2100 Bioanalyzer.\ncDNA Synthesis\ncDNA was synthesized from 1&#8201;&#956;g of RNA (or less if concentrations were too low) per reaction using QuantiTect Reverse Transcription Kit (Qiagen 205313). cDNA samples were stored at &#8722;80&#8201;&#176;C until they were used for PCR.\nPolymerase Chain Reaction (PCR)\nTissue samples were identified by their submission numbers with no reference to treatment class prior to analysis with PCR, so that evaluation of the results could not be inadvertently biased by knowledge of the treatment. We used semi-nested PCR to detect Myl-CoV. Primers were designed from the partial sequence of Rocky Mountain bat coronavirus replicase (accession number EF544563) (Table&#160;S9). The primary reaction used primers MyCVF1 and MyCVR1 to yield a 441&#8201;bp product. The secondary or nested reaction used primers MyCVF2 and MyCV R1 to give a 273&#8201;bp product. PCR were performed in a MJ Research PTC-200 thermal cycler using TopTaq DNA Polymerase (Qiagen, 200205). Each reaction (50&#8201;&#956;l) contained 2&#8201;&#956;l cDNA (or 1&#8201;&#181;l primary reaction), 200&#8201;nM of each primer, 200&#8201;&#181;M of each dNTP (Invitrogen, 10297018), TopTaq PCR buffer and 0.25&#8201;&#956;l TopTaq. The thermal profile for the primary reaction was: 94&#8201;&#176;C for 3&#8201;min (denaturation), followed by 30 cycles of 94&#8201;&#176;C for 30&#8201;sec, 45&#8201;&#176;C for 30&#8201;sec (annealing), 72&#8201;&#176;C for 1&#8201;min and finally 72&#8201;&#176;C for 10&#8201;min. The thermal profile used for the secondary reaction was 94&#8201;&#176;C for 3&#8201;min (denaturation), then 30 cycles of 94&#8201;&#176;C for 30&#8201;sec, 55&#8201;&#176;C for 30&#8201;sec (annealing), 72&#8201;&#176;C for 1&#8201;min and finally 72&#8201;&#176;C for 10&#8201;min. PCR products were analyzed on ethidium bromide stained 1.0% agarose gels (Invitrogen 15510-027 in 0.5X TBE). PCR products were purified using MinElute PCR Purification Kit (Qiagen, 28006) and verified by sequencing (Macrogen, Korea). If more than one DNA band was present, the appropriate size band was cut out and purified using QIAquick Gel Extraction Kit (Qiagen, 28706) before sequencing.\nReverse-Transcription Quantitative PCR (RT-qPCR)\nThe Stratagene MX3005P qPCR System was used in conjunction with QuantiFast SYBR Green PCR Kit (Qiagen 204056). We quantified coronavirus with RNA primers MyCVF2 and MyCV R1 (Table&#160;S9). For initial experiments data were normalized to two transcripts &#8211; glyceraldehyde-3-phosphate dehydrogenase (GAPDH) and beta-actin43. As there were no differences in results, all subsequent experiments used only GAPDH as a normalizer using primers GAPDH US and GAPDH DS (designed for use in humans but also amplify M. lucifugus transcripts &#8211; Table&#160;S9). As well, a no-template (negative) control was included with every set of primers. Each 25&#8201;&#956;l reaction contained: 1&#8201;&#956;M of each primer set, 12.5&#8201;&#956;l SYBR Green Master Mix and 8.5&#8201;&#956;l of diluted cDNA.To verify the RNAseq data, cDNA from ileum samples in which coronavirus RNA had been detected via RT-qPCR were analysed using the following primers, IL22RA1, IRF1, RERG and SRC (for sequence of primers see Table&#160;S9). Primers were designed by aligning primers described for quantitating human cytokines (PrimerBank) with annotated transcripts of M. lucifugus genes: c-jun (Accession number: XM_006096110.1), cyclin D1 (XM_006098046.1), IL10 (XM_006094865.1) and TNF alpha (XM_006104644.1). The interferon beta primers were designed using the annotated transcript for the E. fuscus gene (XM_008145044.1), which also amplify transcripts from M. lucifugus. Primer efficiencies were determined from cycle threshold (Ct) values of purified PCR products serially diluted and re-amplified. Primers amplified targets with an efficiency of about 100% and in all cases the identities of the PCR products were confirmed by their specific dissociation temperature, specific sizes on agarose gels and by sequencing.We observed primer-dimers in some reactions in addition to the PCR product. The dimers dissociated at 77&#8201;&#176;C, while the specific coronavirus polymerase product dissociated at 83&#8201;&#176;C. To avoid false positives due to primer-dimers, the thermocycler was programmed to read at 80&#8201;&#176;C (in the cycle after the primer-dimer had dissociated, and before dissociation of the target product). The thermal profile used was 95&#8201;&#176;C for 5&#8201;min followed by 40 cycles of 95&#8201;&#176;C for 10&#8201;sec, 60&#8201;&#176;C for 30&#8201;sec (readings taken at 80&#8201;&#176;C), and a final cycle of dissociation of product 95&#8201;&#176;C for 1&#8201;min, 55&#8201;&#176;C for 30&#8201;sec and 95&#8201;&#176;C for 30&#8201;sec (readings taken at every degree between 55&#8201;&#176;C and 95&#8201;&#176;C). Only results from reactions that yielded unambiguous results were used for analysis.\nRNA-seq Analysis\nTo explore the mechanisms driving high virus load in bats with WNS, we performed RNA-seq analysis which could potentially screen all targets in the bat intestinal cells. We targeted the ileum transcriptome because this is the tissue in which Myl-CoV is present in detectable concentrations32. Extraction of RNA from ileum tissue, which includes the ileum and potential gut contents have been described in previous sections. Bats were screened for Myl-CoV using RT-qPCR, and bats were assigned post hoc to treatment groups representing four infection histories (Fig.&#160;1(A)): 1) Uninfected (bats were not infected with virus or fungus; n&#8201;=&#8201;5), 2) Virus-infected (bats were na&#239;ve to the fungus but had a persistent Myl-CoV infection; n&#8201;=&#8201;4), 3) Fungus-infected (bats were experimentally infected with P. destructans and no virus was detected; n&#8201;=&#8201;3), or 4) Co-infected (bats with persistent Myl-CoV infections that were also experimentally infected with P. destructans; n&#8201;=&#8201;4). All samples had adequate RNA quality for sequencing (i.e. RIN value >7).\nRNA isolation\nTissues were homogenized in 2&#8201;ml sealed vials with a 5&#8201;mm steel bead, 0.1&#8201;g of 0.1&#8201;mm zirconium silica beads, 350&#8201;&#181;L of RLT buffer (with &#946;-mercaptoethanol) (RNeasy Plus Kit, Qiagen) using a Retsch MM400 tissue homogenizer at 30&#8201;Hz twice for 2&#8201;minutes each. Total RNA from tissues was extracted using the procedure provided with the RNeasy Plus Kit.\ncDNA library preparation and RNA-sequencing\nTotal RNA was sent to The Centre for Applied Genomics at The Hospital for Sick Children (Toronto, Canada). RNA quality was assessed using a Bioanalyzer (Agilent Technologies). We retained all samples with a DV200 (percentage of RNA fragments greater than 200 nt) greater than 85% (Table&#160;S1), discarding one Co-infected sample with a DV200&#8201;=&#8201;42%. Poly(A) mRNA was enriched using oligo dT-beads, and cDNA libraries were prepared using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (New England BioLabs). Barcoded libraries were pooled in equimolar quantities, and the sixteen libraries were sequenced on three lanes of a HiSeq. 2500 System (Illumina Inc.), which generated 126&#8201;bp paired-end reads.\nRNA-sequencing read alignment and analysis\nWe used FastQC v0.11.561 to assess sequence quality and Trimmomatic v0.3662 to remove the adapter sequences and low-quality bases from reads with the following settings: Illumina clop:2:30:10, leading:3, tailing:3, slidingwindow:4:15, minlength:36. We used TopHat v2.1.163 to align the trimmed paired-end reads from each library, separately, to the Ensembl M. lucifugus genome sequence (Myoluc2.064) in strand-specific mode (fr-firststrand) with mate-inner-dist values specific for the insert size of each library. We used featureCounts65 to count reads mapped to the Myoluc2.0 genome annotation in strand-specific mode (reversely stranded), counting paired-end reads as fragments, counting only those fragments where both reads aligned successfully, counting multi-mapping fragments, and excluding chimeric fragments. We assessed the variability within and between the treatments using the R package SARTools v.1.3.066. The featureCount-estimated gene counts were transformed by a variance stabilizing method (VST) using SARTools.We identified differentially expressed genes between each of the treatments using DESeq2&#8201;v.1.12.3, run in SARTools. Custom SARTools-based DESeq settings included: cooksCutoff&#8201;=&#8201;TRUE (perform outliers detection), independentFiltering&#8201;=&#8201;TRUE, alpha&#8201;=&#8201;0.05 (threshold of statistical significance), pAdjustMethod&#8201;=&#8201;BH (benjamini hochberg p-value adjustment method), and locfunc&#8201;=&#8201;median (estimate size factors). Differentially expressed genes were identified as having a fold-change >2 and false discovery rate (FDR)-corrected p-values&#8201;<&#8201;0.0567. We produced volcano plots representing the differential expression comparisons by plotting the log of the adjusted p value as a function of the log ratio of differential expression. We used the Ensembl gene IDs identified by DESeq2 as input for the web-based g:Profiler68 to test for gene ontology (GO) term enrichment among the differentially expressed genes, using a FDR significance threshold <0.05. These GO-terms and their corresponding p-values were used in REViGO69 to visualize significant enrichment of biological processes.\nIgG capture ELISA against \nPurified, glutathione-s-transferase (GST)-tagged Myl-CoV N protein expressed in infected E. coli BL21 cells was used as positive antigen, and GST-tagged protein expressed in uninfected BL21 cells was used as negative antigen. 96-well Costar high-binding round-bottom assay plates were coated with 0.05&#8201;&#181;g/well of either antigen diluted in 0.1&#8201;M phosphate buffered saline (pH 7.4) in a total volume of 100&#8201;&#181;l. Plates were covered and incubated overnight at 4&#8201;&#176;C and washed three times with 300&#8201;&#181;l of PBS-Tween 20 (0.1%) immediately prior to use. Serum samples were diluted to 1:100 in PBS-Tween 20 (0.2%) supplemented with 5% fetal bovine serum (Gibco, Thermofisher). 100&#8201;&#181;l of each sample was added in parallel to a positive and negative antigen plate and incubated at 37&#8201;&#176;C for one hour and washed as above. A peroxidase-labelled goat anti-bat IgG secondary antibody (0.05&#8201;&#181;g in 100&#8201;&#181;l per well, Bethyl labs) was added, incubated for one hour at 37&#8201;&#176;C and washed as above. Peroxidase substrate (2,2&#8217;-azino-bis (3-ethylbenzthiazoline-6-sulfonic acid)) was added to each well and colour development was quantified 30&#8201;minutes later by measuring the optical density at 405&#8201;nm using an ELISA microplate reader. The ELISA cut-off value (0.39) was calculated as the [(mean bat plasma O.D. values for bats that were PCR-negative for Myl-CoV in the ileum)&#8201;+&#8201;(3x standard deviations of those O.D values)].\nStatistical analysis\nData from RT-qPCR and histopathological scores were analysed with SPSS Statistics version 23. The relative levels of a transcript for each bat were calculated as RT-qPCR Cycle threshold (Ct) normalized separately (&#916;Ct) to the &#8220;house-keeping&#8221; gene GAPDH. A &#916;Ct reduction of one (1) indicates an approximately two-fold higher concentration of RNA. The significance of differences of mean values of &#916;Ct between co-infected bats and virus-infected bats were determined using an independent-samples Mann-Whitney U test. We calculated Pearson&#8217;s coefficients to test the correlation between &#916;Ct levels for coronavirus polymerase cDNA for bats in each treatment class, and average scores for fungal hyphae, secondary bacteria, oedema, necrosis and inflammation in wing lesions, as well as bacteremia and levels of neutrophils in lung, spleen and liver interstitium.\nEthical Statement\nBat studies were carried out in strict compliance with Canadian Council on Animal Care guidelines and the procedure for care, handling, and euthanasia of bats were approved by the University Committee on Animal Care and Supply of the University of Saskatchewan (protocol s#20100120).\nElectronic supplementary material\n\nNone\n\nSupplementary Fig 1\nDatasets S1-S9\n",
        "pmid": "30341341",
        "title": "White-nose syndrome is associated with increased replication of a naturally persisting coronaviruses in bats",
        "journal_title": "Scientific Reports",
        "authors": "Christina M. Davy;Michael E. Donaldson;Sonu Subudhi;Noreen Rapin;Lisa Warnecke;James M. Turner;Trent K. Bollinger;Christopher J. Kyle;Nicole A. S.-Y. Dorville;Emma L. Kunkel;Kaleigh J. O. Norquay;Yvonne A. Dzal;Craig K. R. Willis;Vikram Misra",
        "doi": "10.1038/s41598-018-33975-x"
    },
    {
        "keywords": "",
        "abstract": "Analyses of viral protein-protein interactions are an important step to understand viral protein functions and their underlying molecular mechanisms. In this study, we adopted a mammalian two-hybrid system to screen the genome-wide intraviral protein-protein interactions of SARS coronavirus (SARS-CoV) and therefrom revealed a number of novel interactions which could be partly confirmed by in vitro biochemical assays. Three pairs of the interactions identified were detected in both directions: non-structural protein (nsp) 10 and nsp14, nsp10 and nsp16, and nsp7 and nsp8. The interactions between the multifunctional nsp10 and nsp14 or nsp16, which are the unique proteins found in the members of Nidovirales with large RNA genomes including coronaviruses and toroviruses, may have important implication for the mechanisms of replication/transcription complex assembly and functions of these viruses. Using a SARS-CoV replicon expressing a luciferase reporter under the control of a transcription regulating sequence, it has been shown that several viral proteins (N, X and SUD domains of nsp3, and nsp12) provided in trans stimulated the replicon reporter activity, indicating that these proteins may regulate coronavirus replication and transcription. Collectively, our findings provide a basis and platform for further characterization of the functions and mechanisms of coronavirus proteins.",
        "body": "Introduction\nInteractions between viral proteins play pivotal roles in many processes during the viral infection cycle. This is the case in the formation of virus replication complexes, coordinated functions between different viral proteins, assembly of virions, and counter-defense of host immune responses. Analysis of protein-protein interactions is essential to understand protein functions and the molecular mechanisms underlying biological processes. As the viral genomes are of limited sizes, they are particularly well suited for genome-wide analysis of all possible protein-protein interactions. However, the viral protein interaction maps have been generated until now only for a limited number of viruses, including T7 bacteriophage [1], vaccinia virus [2], potato virus A [3], pea seed-borne mosaic virus [3], wheat steak mosaic virus [4], hepatitis C virus [5], [6], porcine teschovirus [7], Kaposi sarcoma-associated herpesvirus [8], and very recently severe acute respiratory syndrome coronavirus (SARS-CoV) [9], [10].Although a large variety of methods have been developed to detect protein-protein interactions, only a few of them are suited for large-scale and high throughput protein interaction analysis. Until now, all the genome-wide analysis of protein interaction networks for viruses and cells have been carried out mainly with the yeast two-hybrid systems, in combination with glutathione-S-transferase (GST) pull-down assays to verify major interactions [8], [9]. Considering that protein modifications that are likely to influence protein interactions may be different for certain proteins in the context of yeast and mammalian cells, the mammalian two-hybrid system may better reflect genuine protein interactions for human viruses. Accordingly, we adopted the mammalian two-hybrid system for detecting genome-wide protein-protein interactions of SARS-CoV.The coronaviruses are classified into the family Coronaviridae in the order Nidovirales and possess the largest RNA genomes known. The genome of SARS-CoV contains a single-stranded, plus-sense RNA of approximately 29.7 kb in length. Fourteen open reading frames (ORFs) have been identified, of which 12 are located in the 3&#8242; end of the genome [11], [12]. The two large ORFs (1a and 1b) in the 5&#8242;-proximal two-third of the genome encode the viral replicase and are translated directly from the genomic RNA, while ORF 1b is expressed by &#8722;1 ribosomal frameshifting. The large polypeptides encoded by 1a and 1b are considered to be cleaved into 16 functional replicase proteins by two proteinases, a papain-like proteinase 2 encoded by nsp3 and a 3C-like proteinase (or main proteinase) encoded by nsp5 [12].A number of functions or characteristics have been identified for the 16 non-structural proteins (nsps). Nsp1 was proved to be able to suppress host gene expression by promoting host mRNA degradation and was involved in cellular chemokine deregulation [13], [14]. Nsp2 seems not to play a crucial role in the generation of infectious viruses in cell culture [15]. Nsp3 is involved in many activities including papain-like proteinase activity, deubiquitinating activity, and ADP-ribose-1&#8243;-phosphatase activity [16]&#8211;[18], [60], which are essential for viral replication and transcription. Nsp5 encodes a 3C-like proteinase which is considered to be an important target for antiviral drug design [19]. Coronavirus nsp4 and nsp6 are transmembrane proteins that could anchor the replication complexes to double membrane vesicles [20]. Based on the structural analysis, hexadecamer of nsp7 and nsp8 may possess dsRNA-binding activity [21]. Nsp8 was shown to have RNA-dependent RNA polymerase (RdRp) activity that could be involved in producing primers utilized by nsp12 which is normally accepted to be the RdRp for SARS-CoV [22], [23]. Nsp9 is a single-stranded RNA-binding protein [24]. Though the structure of nsp10 is resolved, its function is still poorly understood, except that of nsp10 of MHV, homologous to that of SARS-CoV, which regulates viral RNA synthesis [25]&#8211;[27]. Nsp13 is shown to be RNA helicase and 5&#8242;-triphosphatase that may play a crucial role in the viral RNA capping [28], [29]. Nsp14 of coronaviruses possesses a 3&#8242;->5&#8242; exoribonuclease activity which may be involved in the proof-reading ability during the viral RNA replication and transcription [30]&#8211;[32]. Besides the exoribonuclease activity, SARS-CoV also possesses the endoribonuclease activity that is rendered by nsp15 [33]. According to the bioinformatic prediction, nsp16 and SUD domain of nsp3 of SARS-CoV may function as 2&#8242;-O methyltransferase and guanine N7 methyltransferase, respectively [34], [35]. Very recently, the 2&#8242;-O methyltransferase activity was confirmed experimentally for nsp16 of feline coronavirus [36]. Nsp14, 15 and 16 were shown to be essential for efficient replication of coronavirus [30], [37], [38].The 3&#8242; one-third of genome encodes 12 viral proteins including the structural and accessory proteins, which are translated from about 10 subgenomic RNAs [39], [40]. Proteins S, E, M, and N are four well described structural proteins, and 3a, 6, 7a, and 7b were also reported to be virion-associated proteins or viral structural proteins [41]&#8211;[44]. Multiple functions and activities have been identified for the structural and accessory proteins, including apoptosis induction, interference with the innate immunity response, and regulating the cellular protein expression [45]. However, these proteins are not essential for viral replication and transcription at least in cell culture and tested animal models, except nucleocapsid (N) protein [46]&#8211;[49].Previous studies on SARS-CoV focused mostly on the molecular characterization and functional analysis of individual proteins encoded by SARS-CoV and limited information is available on viral protein-protein networks of coronaviruses [50]. To provide more insights into the functions of individual proteins, we analyzed interactions between all SARS-CoV-encoded proteins. By using mammalian two-hybrid assays, 40 different interactions between viral proteins have been identified, and six novel interactions could be confirmed in vitro by biochemical assays. Moreover, a sensitive replication and transcription reporter system of SARS-CoV was established in this study, and, based on this system, we examined the impacts of all the individual viral proteins on the viral replication and transcription and found that the N protein played an important role at the early stage of SARS-CoV genome replication.\nResults\n\nIdentification of protein-protein interactions of SARS-CoV based on mammalian two-hybrid assays\nIn this study, a mammalian two-hybrid system was adopted to analyze the protein-protein interactions of SARS-CoV as it was assumed that the viral proteins expressed in mammalian cells were prone to be in their native conformations and therefore the interactions detected were more likely to be biologically relevant in comparison with other in vitro biochemical methods and assays performed in yeast cells [51].For analysis of genome-wide protein interactions of SARS-CoV, all known ORFs were amplified by PCR from viral cDNAs of SARS-CoV isolate WHU [39] and cloned into pGEM-T vector (Table 1). The large polyprotein encoded by ORF 1a/b was split into 18 domains according to the proteinase cleavage sites, except for nsp3 which was divided into 3 parts based on predicted functional domains. For structural and accessory proteins, the intact ORFs including start and stop codons were amplified, except for S protein which was divided into S1 and S2 as these domains were proved to be two separate domains with distinct functions [52]. All the primers used for amplification were designed by inserting appropriate restriction sites which could be used for subcloning all the fragments from pGEM-T cloning vectors into mammalian two-hybrid vectors pM (bait) and pVP16 (prey) and other protein expression vectors (see below) in correct reading frames.In total, 1024 interaction combinations between all SARS-CoV proteins were examined in a pairwise matrix. As a result, 40 different interactions were detected using the mammalian two-hybrid assays (Fig. 1A). All the interactions shown (Fig. 1) were most likely specific for the viral protein domain of the fusion proteins as the activities of the reporter genes were reduced to background levels when one interacting partner in any of the combinations was replaced with non-relevant fusion protein (negative control) (data not shown). To further show the specificity of the interactions (Fig. 1), a quantitative assay for a typical positive interaction exemplified by nsp10&#8211;nsp14 and various controls were shown (Fig. 1B). To test the specificity of individual interactions, competitive assays were performed by co-expression of viral proteins using a different vector within the cells transformed with the bait and prey constructs. The co-expressed partner proteins interfered with the nsp10&#8211;nsp14 interaction, leading to reduced reporter activity (Fig. 1C), indicating that the same proteins with different fusion domains competed with each other and impaired the specific interactions needed for activation of the reporter genes.Most interactions showed directionality, indicating the influence of fusion domains on the interacting sites. Nevertheless, three pairs of interactions were detected in both directions: nsp10 and nsp14, nsp10 and nsp16, and nsp7 and nsp8. Nsp11 is a small polypeptide containing only 13 amino acids and no interaction was detected with it in various assays but nsp11 in the fusion with nsp10 (nsp10/11) could significantly enhance the binding capability of nsp10 with either nsp14 or nsp16, indicating that the small nsp11 may also play important roles in viral protein interactions and replication. Self-interactions of nsp3, nsp5, nsp8, nsp15 and N were observed, suggesting that these proteins could form dimeric or multimeric complexes by interacting with themselves. Furthermore, 8 interactions between non-structural proteins and accessory proteins (3b, 7b, 8b, 9b) and one interaction between non-structural protein nsp16 and structural protein N were detected, indicating that some of the accessory proteins might be involved in viral replication and transcription processes. Nevertheless, that being the case, these interactions could have a minor effect on modulating virus replication as the virus-specific proteins are not strictly essential for viral replication at least in cell culture and mouse models [48], [49].\nConfirmation of SARS-CoV protein interactions by pull-down assays\nTo confirm the interactions which were newly detected by mammalian two-hybrid assays, pull-down assays were performed. Proteins that were related to the interactions not reported previously were expressed in different bacterial systems including pET30a (His-tagged), pGEX-6P-1 (GST fusion) and pMAL-c2X (MBP fusion). Using pull-down assays that were described in Materials and Methods section, six protein-protein interactions were confirmed, including those between: nsp10 and nsp14, nsp10 and nsp16, nsp13 and 3b, nsp8 and 3b, nsp16 and N, and 8b and N (Fig. 2). Due to difficulties in bacteria expression, the interactions involving nsp12, nsp3.2 and nsp3.3 could not be confirmed.\nEstablishment of a reporter gene-containing replicon and analysis of the impacts of viral proteins provided in \nAlthough a large number of protein-protein interactions were detected for SARS-CoV in virtue of the large-scale screening analysis in mammalian two-hybrid system, the roles of these interactions in the viral replication and transcription were still not clarified. To obtain more clues to the general roles of individual proteins, we constructed a SARS-CoV replicon (Rep-SCV-luc/neo) that expresses the firefly luciferase gene, a sensitive reporter, under the control of M gene transcription regulatory sequence (TRS) as described in the Materials and Methods section (Fig. 3A). The effects of individual proteins or protein domains provided in trans on the replication/transcription of the SARS-CoV replicon were evaluated. Any effect of the protein provided in trans on the luciferase expression levels could in principle be due either to changes in the extent of the replication, the transcription, or a combination of both.To evaluate the reporter gene expression of the replicon generated in this work, the BAC plasmid encoding Rep-neo/luc (pBAC-Rep-SCV-neo/luc) was transfected into BHK21 cells. A significant increase in luciferase activity, in comparison with that of the parental replicon plasmid pBAC-SARS-CoV-REP was observed (Fig. 3B). As the luciferase gene is located in the 3&#8242;-end of the genome and under the control of a viral TRS, it can only be expressed from subgenomic RNAs but not from the viral genome RNA. Therefore, expression of luciferase was expected to reflect the genome replication and transcription of SARS-CoV replicon. To further clarify whether the luciferase expression of Rep-SCV-luc/neo derived from the viral replication and transcription processes but not from nuclear splicing products of the large viral replicon RNA during the CMV promoter-driven transcription in nucleus, the positive and negative subgenomic RNAs were examined according to the strategy described in our previous work [39] and the existence of correct subgenomic RNAs was confirmed (data no shown). Moreover, when the gene segment between two MluI restriction sites in viral genome, which encodes nsp4&#8211;nsp11 and part of nsp12, was deleted, the construct pBAC-Rep-SCV-&#916;Mlu-luc/neo produced much lower levels of luciferase expression although it still retained detectable level of luciferase activity (Fig. 3B). To analyze the luciferase expression from the MluI deletion mutant, the identity of the luciferase mRNA was sequenced. The results showed that a low level of luciferase mRNA was generated either by cryptic promoters or by splicing of cryptic introns within the replicon transcripts in nucleus but not in the mRNAs derived from the cytoplasmic replication and transcription of the replicon (data not shown). Taken together, these results showed that the reporter-containing SARS-CoV replicon could be used as a model system for analyzing the functions of viral proteins.To study the effect of SARS-CoV proteins on replication and transcription, individual viral proteins were co-expressed in trans with the SARS-CoV replicon. Renilla luciferase was employed as an internal control reporter to normalize the transfections among different wells (Fig. 4). Though most of the viral proteins did not exert significant impact on the replication and transcription of the replicon, the nsp3.1 containing X and SUD domains, and the RNA polymerase nsp12 (Fig. 4A), and nucleocapsid N (Fig. 4B) significantly increased the luciferase activity, whereas nsp3.2 containing the papain-like (PL) proteinase domain reduced the viral replication and transcription. Other proteins, such as nsp10/11, also showed a minor but significant increased expression. These results confirmed that the function of N gene fragment was not resulted from protein 9b that is internally nested in N sequence, as 9b showed no obvious impact on the replication and transcription of replicon when it was co-expressed (Fig. 4B). In addition, these results indicate that nucleocapsid protein, nsp3 and RNA polymerase may possess a dominant function in trans, while the PL proteinase is involved in important cis functions like cis-processing of the large viral polyprotein.\nTrans-activation activity of N protein at the early stage of genome replication or transcription of SARS-CoV\nTo make further analysis on the enhancement effect of N protein on the viral replication and transcription, we examined the dynamic changes of the luciferase activity expressed from the reporter replicon Rep-SCV-luc/neo. When the replicon plasmid pBAC-Rep-SCV-luc/neo was transfected alone, the luciferase activity could be detected 13 h post transfection (more than 10 times compared with background levels) while the highest value of activity was reached around 32 h post transfection (Fig. 5A). The luciferase activity became stable 72 h post transfection and the activity level was as low as that of 14 h post transfection (Fig. 5A).To study the role of N protein provided in trans, we next transfected cells with equal amount of the reporter replicon together with either N protein expression construct (pcDNA3.0-N) or the empty vector pcDNA3.0. Luciferase activity became detectable at 7 h post transfection when N protein was provided in trans (Fig. 5B), which was 9 h earlier than similar transfection without additional N protein (Fig. 5B). It was also observed that the luciferase activity was significantly enhanced in the first 40 h post transfection (Fig. 5B). The ratios of luciferase activities with and without co-expression of N protein at different time points post transfection were shown in Fig. 5C. The ratios became lower step by step from over 50 times at 16 h post transfection to around 4 times at 38 h post transfection (Fig. 5C). The corresponding ratios for nsp3.2, nsp3.1 and nsp12 were also investigated and no similar phenomena were observed (data not shown). Collectively, these data showed that the nucleocapsid N protein provided in trans could enhance the efficiency of SARS-CoV genome replication and transcription at early stages.\nDiscussion\nCoronaviruses have the largest RNA genome known, which encodes a large number of proteins that are involved in viral replication, assembly, and other important functions that are essential to viral amplification in host cells. Except for some viral proteins that might perform their activities individually, most of the viral proteins could associate with other proteins or themselves to carry out their functions, indicating that the interactions between these proteins may play a crucial role during the viral life cycle. For SARS-CoV, at least 17 proteins, including 16 non-structural proteins and the structural nucleocapsid protein, are most likely involved in the replication process [12], [53].In this study, a mammalian two-hybrid system was used to determine the genome-wide matrix-based protein-protein interactions of SARS-CoV. In total, 40 different interactions for 28 predicted mature proteins were observed, and most of them have not been previously reported. Interestingly, 32 percent of the interactions tested could be confirmed by pull down assays. To our knowledge, this is the first genome-wide protein-protein analyses for an intact viral ORFeome by using mammalian two-hybrid system. In our screen, 1.4 protein interactions per viral protein on the average was detected, and this result is in the upper range of the detection rates of viral protein interactions obtained by yeast two-hybrid systems [58]. Therefore, this study indicates that mammalian two-hybrid system can also serve as a convenient system to detect viral protein interactions of the whole ORFeomes. In addition, mammalian two-hybrid system may have some advantages over yeast two-hybrid system for detecting genuine viral protein interactions due to the native posttranslational modifications and folding of the proteins.Very recently, two large-scale analyses of protein interactions of SARS-CoV were carried out by using yeast two-hybrid systems [9], [10]. In the report by von Brunn et al, 70 pairs of interactions were revealed, among which 30% could be verified by co-immunoprecipitation [9]. In the work by Imbert et al, 17 pairs of interactions were observed for non-structural proteins, about half of which were related to nsp3 [10]. Surprisingly, none of the interactions revealed in the two studies was overlapping although they both employed the yeast two-hybrid system for the screening. In contrast, overlapping interactions could be identified between that detected by mammalian two-hybrid system in current work and that by different yeast two-hybrid systems. There are 9 pairs of interactions found in our work that are overlapping with that of von Brunn et al, and they are nsp5 with nsp5, nsp7 and nsp8, nsp7 with nsp7, nsp8 and nsp9, nsp8 with nsp8, nsp9 and 9b. There are 4 pairs of interactions found in this study that are identical to that of Imbert et al. and they are nsp10 with nsp14 and nsp16 in both directions. Interestingly, all the overlapping interactions represented strong interactions in the mammalian two-hybrid assays and could be verified by biochemical assays.In the two previous studies with yeast two-hybrid system, four proteins (nsp2, nsp3, nsp8 and 9b) were shown to have a wide range of interactions with other viral proteins, but these phenomena could not be observed in mammalian two-hybrid assays. In contrast, a number of interactions including N-N and nsp15&#8211;nsp15 were detected in current study and proved previously by other studies [54]&#8211;[57] but they could not be revealed by the yeast two-hybrid systems [9], [10]. Intriguingly, the two strong interactions nsp10&#8211;nsp14 and nsp10&#8211;nsp16 identified in mammalian two-hybrid system was also revealed in a yeast two-hybrid system by Imbert et al. [10] but not by von Brunn et al [9]. Although it is difficult to judge which system offers more biologically relevant results, great caution should be taken when interpreting the interaction data as both two-hybrid screenings as well as biochemical methods may generate false positive and false negative interactions. Nevertheless, the above comparative analysis of overlapping interactions of different systems may suggest that mammalian two-hybrid system could provide more reliable assays for detecting human viral protein interactions. In other scenario, the profiles of SARS-CoV protein interactions revealed by mammalian two-hybrid system (in this work) and yeast systems [9], [10], respectively, may be complementary to each other and jointly serve as a framework for further characterization of protein-protein interactions and their biological functions on coronavirus replication cycle.Similar to that of yeast two-hybrid system, the detectable protein interactions all take place in the nucleus whereas, on the contrary, all positive-stranded RNA viruses replicate in cytoplasm, the natural location for viral proteins. Thus, the two-hybrid systems may have obvious limitation on detecting proteins that contain transmembrane domains or change to abnormal conformations in acidic conditions, and this may represent one reason for generating false negative and false positive interactions. In current screen, only few interactions were detected for the SARS-CoV membrane proteins such as nsp4, M, E and 3a proteins, and no interactions could be detected for nsp1, nsp6, spike and the small transmembrane protein 6, possibly reflecting the limitation of the two-hybrid system. Therefore, using shorter or random cDNA fragments by removing the transmembrane and other inhibitory domains may help detect more interactions in this system.In this study, we have detected several interactions between accessory proteins and replicase proteins, for example, 3b with nsp8, nsp12, nsp13 or nsp14, 7b with nsp9 or nsp16, 8b with nsp9, and 9b with nsp8. Interestingly, among these interactions those between 3b with nsp8 and nsp13 were confirmed by pull-down assays. However, previous studies showed that deletions of ORFs 3b, 7b, 8b and 9b alone or in combination with other virus-specific proteins did not significantly influence the level of viral RNA and replication efficiency in cell culture and mouse models [48], [49], suggesting either that these interactions may not play significant roles in viral replication or that these systems are not sensitive enough to detect minor differences. In any case, it would be reasonable to speculate that such interactions may contribute to the virus-host interplay and hence to the viral pathogenicity.The interactions between nsp10 with nsp14 and nsp16 showed bi-directionality in the mammalian two-hybrid analyses, which could be confirmed in pull-down assays and were also revealed in one of the two yeast-hybrid analyses [10]. Recently, nsp10 was shown to form a dodecameric RNA-binding protein complex [26], [27] involved in regulation of RNA synthesis and polyprotein processing [25], [59]. Nsp14 and nsp16 were proved to play an essential role in viral replication and transcription as shown by mutational analysis [37]. Considering that nsp14 has exoribonuclease activity and could play an essential role in RNA proof-reading activity [30]&#8211;[32], and that nsp16 is involved in the viral methyltransferase activity and RNA capping [34], [36], the interactions of nsp10 with nsp14 and nsp16 may indicate that nsp10 could play an important role in the formation of replication complex bound to RNA (i.e., nsp10 could mediate the binding of nsp14 and nsp16 to the RNA) and in the replication/transcription processes. Although no interactions were detected for nsp11, which was predicted to be small polypeptide with 13 amino acids, fusion of nsp11 with nsp10 strengthened the respective interactions of nsp10 with nsp14 and nsp16. These observations may imply that nsp11 could act as a cofactor for nsp10 and the predicted cleavage site between nsp10 and nsp11 may be inefficient in vivo, thus resulting in the production of nsp10/nsp11 fusion protein in cells. Indeed, no evidence for existence of the fully processed nsp11 has been presented in published researches. However, such speculation needs to be confirmed by further investigations.Several interactions detected in current work have also been observed in previous studies. For example, nsp7 and nsp8 could associate with each other and thus form a hexadecameric supercomplex with a central channel that has dimensions and positive electrostatic properties favorable for nucleic acid binding [21]. Consistent with our results, self-interactions of N protein and nsp15 have also been revealed and the results indicate that these proteins have the propensity to oligomerize [56], [57], [61]. In this study, a SARS-CoV replicon containing a sensitive luciferase reporter was constructed, with its expression under the control of M gene TRS. In this situation, the reporter activity would indicate the efficiency of both genome replication and transcription. When individual viral proteins were provided in trans to the reporter replicon, most proteins did not exert significant influence on the reporter activity, with exception of nsp3, nsp12 (polymerase), and nucleocapsid protein N. As protein nsp3 is a large multidomain protein, different domains were tested separately in this system. While nsp3.1 that covers domains predicted for adenosine diphosphate-ribose 1&#8243;-phosphatase [12] and methyltransferase [35] could enhance the reporter activity, the nsp3.2 had a negative effect, indicating the domains in nsp3.2 may play structural roles in the formation of replication/transcription complex and could behave in a dominant-negative manner when separated from other domains. In accordance with this hypothesis, nsp3.2 was found to associate with itself, nsp4 and nsp12 in this work.N protein of coronaviruses increases the rescue efficiency of coronaviruses from infectious RNA transcripts [37], [49], [62] and is required for efficient genome replication [53], [63]. In this study, we showed that N protein was not absolutely essential for starting the replication but may play important roles at the early stages of genome replication and/or transcription because the reporter replicon in absence of additional N protein provided in trans also resulted in obvious replication and transcription but in lower efficiency and delayed manner. The enhancement effects of N protein phased out at late stage probably when the N protein expressed from the genome had accumulated to certain level. Although the exact mechanisms for N-mediated stimulation activity at early stages are not known, several scenarios could be envisaged, such as protecting viral genome RNA, increasing the translation efficiency of viral RNA, stabilizing replication/transcription protein complex and inhibition of cellular innate immune responses, as is supported by the recent reports on diversified functions of N protein [64], [65].In summary, current work constructed an interaction network of SARS-CoV proteins and established a sensitive replicon for studying the functions of individual viral proteins. The intraviral protein interactions identified in this study, in combination with the data obtained by using other systems, could serve as a basis for further studies of viral protein functions and molecular mechanisms of the genome replication/transcription processes of coronaviruses.\nMaterials and Methods\n\nCells and viral cDNAs\nAfrican green monkey kidney (Vero E6) cells, baby hamster kidney (BHK21) cells and 293T cells were grown and maintained in Dulbecco's modified Eagle medium and modified Eagle medium (Gibco Invitrogen), respectively, supplemented with 10% heat-inactivated fetal bovine serum and 100 U/ml of penicillin and 100 &#181;g/ml of streptomycin (Gibco Invitrogen). Viral cDNAs encoding individual SARS-CoV ORFs were generated and described in our previous work [39], [66].\nCloning and expression of the SARS-CoV ORFs\nThe primers for individual ORFs were designed according to the genome sequence of SARS coronavirus strain WHU (GenBank accession number: AY394850) with restriction sites which were compatible for the downstream subclonings. The reagents for PCR were 0.2 &#181;M dNTPs, 0.4 &#181;M forward and reverse primers, &#8764;10 ng template DNA and 1 U of KOD DNA polymerase (TOYOBO) in 50 &#181;l reaction system. The amplification conditions were 94&#176;C for 2 min and 30 cycles of 94&#176;C for 15 sec, 52&#176;C for 15 sec and 68&#176;C for 3 min, followed by 68&#176;C for 10 min and 4&#176;C for 10 min.After the separation by agarose gel electrophoresis, PCR fragments were recovered from the gels by DNA extraction kit (OMEGA BIO-TECH) and cloned into pGEM-T vector (Promega) after A-tailing reaction according to standard protocols. The sequences of positive clones examined by restriction and PCR analyses were confirmed by DNA sequencing on ABI 3730 DNA Analyzer (Applied Biosystems) and CEQ&#8482; 8000 Sequencer (Beckman Coulter). Subsequently, the coding sequences were cut from pGEM-T constructs with appropriate restriction enzymes and cloned into the mammalian two-hybrid vectors pM and pVP16 (Clontech), Escherichia coli protein expression vectors pET30a (Novagen), pGEX-6P-1 (Amersham Biosciences) and pMAL-c2X (New England Biolabs), and mammalian expression vectors pcDNA3.0 (Invitrogen Corporation), pCMV-Tag2b (Stratagene Corporation) and pCIH that was generated by adding an ATG-HA-tag oligo at the XhoI-EcoRI sites of vector pCI-Neo (Promega). Details of the cloning processes and various constructs can be provided upon request.\nMammalian two-hybrid analysis\nThe reporter construct pG5-luc was reconstructed from pG5CAT (Promega) by replacing chloramphenicol acetyl transferase (CAT) gene with luciferase gene sequence. For protein interaction analysis, 0.3 &#181;g of DNA-binding domain fusion constructs in plasmid pM, 0.3 &#181;g of transcriptional activation domain fusion constructs in plasmid pVP16, 0.15 &#181;g of reporter construct pG5-luc and 0.1 &#181;g of vector pRL-TK (Promega) as internal control were transfected into 293T cells by calcium phosphate transfection method. One day before transfection, 293T cells were seeded in 24-well plate with &#8764;1&#215;105 cells per well. On the following day, the DNA of specified amount and 2.5 &#181;l of 2.5 M CaCl2 were diluted with deionized H2O to reach a final volume of 25 &#181;l. After an incubation for 5 minutes, 25 &#181;l of 2&#215;HBS solution (50 mM HEPES, 10 mM KCl, 12 mM Dextrose, 280 mM NaCl, 1.5 mM Na2HPO4, pH 7.05) was added and mixed. The calcium phosphate-DNA solution was added dropwise to the cell culture medium while swirling the plate gently. After incubation for 6 h at 37&#176;C with 5% CO2, the medium was replaced with fresh one. 24 h later, the luciferase assays were performed by Dual-Luciferase reporter Assay System (Promega Corporation) according to the provider's instructions. Several control constructs were adopted in the analysis, including pM-53 which encodes the p53 protein, pVP16-T which encodes the SV40 large T antigen, and the empty vectors pM and pVP16. The known interaction between p53 and T antigen was used as positive control, and the combinations of test construct and one control plasmid were assigned as negative controls. The assays on each pair of plasmid combinations were repeated at least 4 times and 3 wells were examined per assay.\nGST-pull down assays\nRecombinant glutathione S-transferase (GST)-fusion proteins and maltose-binding protein (MBP)-fusion proteins were expressed in E. coli BL21 and DH5&#945;, respectively. Bacterial cells harvested were resuspended in phosphate-buffered saline (PBS, 10 mM sodium phosphate, 150 mM NaCl, pH 7.5) and lysed by sonication in ice. The lysate was subsequently clarified by centrifugation at 12,000 g for 30 min at 4&#176;C. GST fusion proteins and GST protein immobilized on glutathione-Sepharose were coated firstly with MBP protein and were subsequently incubated with the lysate for MBP-fusion proteins in PBS containing protease inhibitor cocktail. After incubation with rotation for 2 h, the resin of glutathione-Sepharose was precipitated by brief spin and washed by PBS at least for 5 times. The resin was re-suspended in SDS-PAGE loading buffer and heated at 100&#176;C for 5 min. The samples were separated in 10% SDS-PAGE and transferred to PVDF membrane following the standard procedures of Western-blotting. Anti-MBP rabbit serum (New England Biolabs, dilution 1&#8758;5000) was used in the immunoblot analysis to identify MBP-fusion proteins.\nConstruction of a replication and transcription report system for SARS-CoV\nThe reporter system was based on the primary SARS-CoV replicon (pBAC-SARS-CoV-REP) [37]. To construct a reporter-containing replicon including a selection gene for mammalian cells, the luciferase and neomycin fusion gene cassette was amplified by PCR from a hepatitis C virus replicon provided by Dr. Ralf Bartenschlager [67]. The luciferase-neomycin gene sequence was fused with and thus controlled by the transcription regulation sequence (TRS) of SARS-CoV M gene. The fused sequence containing TRS/M-luciferase-neomycin cassette was inserted between AscI and BamHI sites in pBAC-SARS-CoV-REP, resulting in the reporter replicon construct pBAC-Rep-SCV-luc/neo. The cloning details can be provided upon request. Several transfection strategies and reagents were examined for the transfection of pBAC-Rep-SCV-luc/neo and the relatively higher transfection efficiency was obtained with FuGENE HD Transfection Reagent (Roche Applied Science). Several cell lines were also examined and BHK21 turned out to better support the replication/transcription of Rep-SCV-luc/neo and produced consistent results.",
        "pmid": "18827877",
        "title": "Genome-Wide Analysis of Protein-Protein Interactions and Involvement of Viral Proteins in SARS-CoV Replication",
        "journal_title": "PLoS ONE",
        "authors": "Ji'An Pan;Xiaoxue Peng;Yajing Gao;Zhilin Li;Xiaolu Lu;Yingzhao Chen;Musarat Ishaq;Dan Liu;Marta L. DeDiego;Luis Enjuanes;Deyin Guo",
        "doi": "10.1371/journal.pone.0003299"
    },
    {
        "keywords": "",
        "abstract": "Human coronavirus 229E (HCoV-229E) is one of the first coronavirus strains being described. It is linked to common cold symptoms in healthy adults. Younger children and the elderly are considered vulnerable to developing lower respiratory tract infections (LRTIs). In particular, immunocompromised patients have been reported with severe and life-threatening LRTIs attributed to HCoV-229E. We report for the first time a case of LRTI and acute respiratory distress syndrome developed in a healthy adult with no comorbidities and HCoV-229E strain identified as the only causative agent. A 45-year-old female with a clear medical history presented with fever, cough, and headache. Respiratory tract infection was diagnosed, and empirical antibiotics were started. Within two days, she developed bilateral pleural effusions, diffuse consolidations, and ground glass opacities involving all lung fields. She needed immediate oxygen supply, while ABGs deteriorated and chest imaging and PaO2/FiO2 indicated ARDS. Early administration of systemic corticosteroids led to gradual clinical improvement. Multiplex PCR from nasal secretions was positive only for HCoV-229E and negative for multiple other pathogens. It remains to be elucidated how an immunocompetent adult developed a life-threatening LRTI caused by a &#8220;benign considered&#8221; coronavirus strain, the HCoV-229E.",
        "body": "1. Introduction\nCoronaviruses (CoVs), a genus of the Coronaviridae family, are positive-stranded RNA viruses. The first human coronavirus (HCoV) appeared in reports in the mid-1960s and was isolated from persons with common cold. Two species were first detected: HCoV-229E and subsequently HCoV-OC43 [1, 2]. Since then, more species were described [3&#8211;5].The HCoV-229E strain was associated with common cold symptoms [6]. Younger children and the elderly were considered more vulnerable to lower respiratory tract infections. Severe lower respiratory tract infection so far has only been described in immunocompromised patients [7, 8]. To our knowledge, there is no report describing life-threatening conditions in immunocompetent adults attributed to HCoV-229E. We report a case of acute respiratory distress syndrome developed in a healthy adult with no comorbidities and HCoV-229E strain identified as the only causative agent.\n2. Case Presentation\nA 45-year-old female patient presented to the emergency department with dry cough, headache, and fever up to 39.5&#176;C lasting a few hours. Her past medical history was unremarkable, and she did not take any medication regularly. She has never smoked, worked as a teacher at a local high school, and has not recently travelled.Clinical examination revealed rales at her left lower lung fields. Chest X-ray showed diffuse opacities and consolidation at this field. The arterial blood gases (ABGs) were normal, and intravenous ceftriaxone and azithromycin were empirically administered for lower respiratory tract infection (LRTI). S. pneumoniae and L. pneumophila antigen in the patient's urine specimen was negative, and blood cultures were sterile.Over the next two days, the patient's clinical condition rapidly deteriorated, with development of tachypnea (34 respirations/minute), dyspnea, and hypoxemia. ABGs changed to PaO2 of 55.3&#8201;mmHg, PCO2 of 31.4&#8201;mmHg, and pH of 7.487. Lung auscultation revealed diffuse rhonchi symmetrically all over her chest, bronchial breathing at her right and left lower lobes, and diminished vesicular sounds. Chest CT scan displayed bibasilar pleural effusions and diffuse consolidations plus ground glass opacities involving all lung fields (Figure 1). Oxygen was supplied at 5&#8201;L/min, and antimicrobial therapy was changed to levofloxacin 500&#8201;mg/day. Systemic corticosteroids and bronchodilators were added about 40 hours after her hospitalization. Samples of the pleural fluid showed exudate with 260 cells/mm3, negative Gram stain, and sterile cultures.Nasal secretions were collected, and multiplex PCR technology was applied targeting multiple pathogens (RespiFinder&#174; 22, PathoFinder), including coronavirus 229E; coronavirus NL63, HKU1, and OC43; influenza A, B, and H1N1; parainfluenza 1, 2, 3, and 4; Mycoplasma pneumoniae; Legionella pneumophila; Bordetella pertussis; bocavirus; rhinovirus/Enterovirus; adenovirus; RSV A and B; and Chlamydophila pneumoniae. The result was positive for HCoV-229E, while negative for the other tested pathogens; PCR for SARS-CoV and MERS-CoV was also negative.Within the next few hours, the patient's clinical condition further worsened and she required increased oxygen supply. New ABGs showed PaO2&#8201;=&#8201;76&#8201;mmHg, PCO2&#8201;=&#8201;33&#8201;mmHg, and pH&#8201;=&#8201;7.45 at FiO2&#8201;=&#8201;0.50 with PaO2/FiO2&#8201;=&#8201;152, indicating ARDS. The patient was in severe respiratory distress and remained febrile and tachypneic, and a new chest X-ray showed multiple consolidations all over her lung fields (Figure 2). Intravenous linezolid was added to her regimen empirically in order to treat a possible community-acquired Staphylococcus aureus pneumonia.A repeat one-step RT-PCR in a nasal sample (Taqman, in-house protocol, Hellenic Pasteur Institute) confirmed the exclusive presence of human coronavirus 229E (HuCoV-229E). After the administration of systemic corticosteroids, the patient started to display clinical improvement within the first 24 hours. Further laboratory analyses did not reveal any immune defect. After a week, she was discharged from the hospital well and remained healthy 23 months later (Figure 3).\n3. Discussion\nThe initially described coronavirus strain 229E has been previously identified as the second most frequent cause of common cold after rhinoviruses in healthy adults. Predominant symptoms were acute rhinorrhea, nasal congestion, and/or sore throat [9, 10]. Nasal discharge was the hallmark of all symptoms after inoculation of HuCoV-229E to healthy volunteers, and further observed symptoms were malaise, headache, chills, and cough [6].HCoV-229E has been associated with bronchitis, acute exacerbations of COPD, and pneumonia in infants, children, and elderly persons with underlying illnesses [11&#8211;13]. Life-threatening infections have only been described in immunocompromised patients [7, 8], but the correlation of HCoV-229E with LRTI in healthy adult individuals is uncertain [9]. An adult patient with pneumonia tested positive for HCoV-229E has been described in a study conducted in rural Thailand, but it is not made clear if other comorbidities were present [14]. Nine Italian patients hospitalized with LRTI have also been tested positive for HCoV-229E; however, their age is not specified [15]. &#913;lthough numerous studies have tentatively linked 229E infections to severe respiratory tract illness over many years, no study controlling for age and underlying illness has demonstrated an epidemiologic association between infection with HcoV-229E in healthy adults and any illness other than the common cold. Furthermore, no case of HCoV-229E-associated ARDS has been reported in immunocompetent adults. Only a few cases of pulmonary infection and ARDS have been described in a 76-year-old woman infected with the closely related alpha coronavirus HCoV-NL63 [16] and in a 39-year-old woman with poorly controlled DM and infected with the beta coronavirus HCoV-OC43.The patient was a teacher and thus exposed to multiple pathogens from her students. She was an immunocompetent adult with no underlying disease. Her symptoms progressed rapidly, despite the immediate administration of broad-spectrum antibiotics, and clinical, laboratory, and radiologic findings were compatible with ARDS [17]. The patient came very close to intubation and mechanical ventilation, but early addition of corticosteroids in her therapeutic regimen seems to have played a decisive role towards her favorable outcome. Close monitoring and continuous recording and assessment of her vital signs warranted the borderline avoidance of her transfer to the ICU.HCoV-229E was isolated twice from the patient's nasal secretions; she was not intubated, and thus, the BAL sample was not taken. Extensive workup did not reveal any immune defect; all microbiological and serological studies remained negative for other pathogens. Rapid and reliable diagnosis of human coronavirus infections is of pronounced clinical importance. New RT-PCR methods [18] in sputum and nasal aspirates successfully have diagnosed human coronavirus infections. Multiplex RT-PCR is used increasingly to diagnose respiratory infections and has shown to be more sensitive than viral culture and antigen detection and also rapid and cost-effective [19], with greater sensitivity and similar specificity compared to real-time RT-PCR [20].\n4. Conclusion\nTo our knowledge, it is the first time that human coronavirus HCoV-229E has been detected in severe lower respiratory tract infection with ARDS of a healthy adult with no comorbidities. Although it is considered as a &#8220;benign&#8221; microorganism and linked to mild respiratory symptoms, the presence of HCoV-229E should not be underestimated and considered as a possible pathogen even in coinfections with other microorganisms and in more serious LRTIs. The reason why HuCoV-229E causes different clinical manifestations in diverse patient groups has not yet been answered. The process through which HCoV-229E may evade normal immune defense and cause life-threatening illness remains to be elucidated.",
        "pmid": "29850307",
        "title": "A Rare Case of Human Coronavirus 229E Associated with Acute Respiratory Distress Syndrome in a Healthy Adult",
        "journal_title": "Case Reports in Infectious Diseases",
        "authors": "Foula Vassilara;Aikaterini Spyridaki;George Pothitos;Athanassia Deliveliotou;Antonios Papadopoulos",
        "doi": "10.1155/2018/6796839"
    },
    {
        "keywords": "2019 Novel Coronavirus;Culture;Microscopy, Electron;Phylogeny;Korea",
        "abstract": "Novel coronavirus (SARS-CoV-2) is found to cause a large outbreak started from Wuhan since December 2019 in China and SARS-CoV-2 infections have been reported with epidemiological linkage to China in 25 countries until now. We isolated SARS-CoV-2 from the oropharyngeal sample obtained from the patient with the first laboratory-confirmed SARS-CoV-2 infection in Korea. Cytopathic effects of SARS-CoV-2 in the Vero cell cultures were confluent 3 days after the first blind passage of the sample. Coronavirus was confirmed with spherical particle having a fringe reminiscent of crown on transmission electron microscopy. Phylogenetic analyses of whole genome sequences showed that it clustered with other SARS-CoV-2 reported from Wuhan.",
        "body": "",
        "pmid": "32080990",
        "title": "Virus Isolation from the First Patient with SARS-CoV-2 in Korea",
        "journal_title": "Journal of Korean Medical Science",
        "authors": "Wan Beom Park;Nak-Jung Kwon;Su-Jin Choi;Chang Kyung Kang;Pyoeng Gyun Choe;Jin Yong Kim;Jiyoung Yun;Gir-Won Lee;Moon-Woo Seong;Nam Joong Kim;Jeong-Sun Seo;Myoung-don Oh",
        "doi": "10.3346/jkms.2020.35.e84"
    },
    {
        "keywords": "Coronavirus;severe combined immunodeficiency;pneumonia;children",
        "abstract": "Please cite this paper as: Szczawinska&#8208;Poplonyk et&#160;al. (2012) Fatal respiratory distress syndrome due to coronavirus infection in a child with severe combined immunodeficiency. Influenza and Other Respiratory Viruses DOI: 10.1111/irv.12059.\nCoronaviruses have been demonstrated to contribute substantially to respiratory tract infections among the child population. Though infected children commonly present mild upper airway symptoms, in high&#8208;risk patients with underlying conditions, particularly in immunocompromised children these pathogens may lead to severe lung infection and extrapulmonary disorders. In this paper, we provide the first report of the case of a 15&#8208;month&#8208;old child with severe combined immunodeficiency and coronavirus HKU1&#8208;related pneumonia with fatal respiratory distress syndrome.",
        "body": "Introduction\nSevere combined immunodeficiency (SCID) is a genetically and clinically heterogeneous group of the most severe primary immunodeficiencies, characterized by the absence of functional T lymphocytes resulting in profound impairment of the cellular and humoral adaptive immunity. Depending on the genetic defect, B lymphocytes and natural killer (NK) cells may be present or absent and this feature constitutes the basis for the classical division into T&#8208;B&#8195;+&#8195;SCID and T&#8208;B&#8722;SCID, with further subdivisions into NK+ and NK&#8722; disorders.\n1\n Respiratory tract infection is a common manifestation in children in question and may be present within the neonatal period or in early infancy. Opportunistic pathogens may lead to rapidly progressive, fatal interstitial pneumonitis accompanied by hyperinflation resulting from small airway obstruction or to persistent bronchiolitic presentation. Apart from pyogenic bacteria, such as Pseudomonas aeruginosa, Stenotrophomonas spp, Burkholderia spp, as well as Mycobacteria and fungi, in particular Pneumocystis jiroveci, respiratory viruses like respiratory syncytial virus (RSV), adenovirus, parainfluenza virus, human metapneumovirus (hMPV) and other viruses &#8211; cytomegalovirus (CMV), varicella&#8211;zoster virus (VZV), and Epstein&#8211;Barr virus (EBV) are associated with severe pneumonia in SCID children.\n2\n Human coronaviruses (HCoV) HCoV&#8208;229E and HCoV&#8208;OC43 and related new strains HCoV&#8208;NL63 and HCoV&#8208;HKU1, identified after the epidemic outbreak of severe acquired respiratory syndrome (SARS) coronavirus, are likely to be common respiratory viruses in otherwise healthy children and were not implicated in severe lung infections in immunocompromised patients thus far.\n3\n In this report we present the case of a child with delayed&#8208;onset SCID and fatal respiratory coronavirus infection.\nCase presentation\nA 15&#8208;month&#8208;old girl was referred to the University Hospital due to persistent fever and interstitial pneumonitis for the purposes of diagnosis and treatment.She was the first child of young, non&#8208;consanguineous parents, born from the first pregnancy which was terminated in the 39th week of gestation using Cesarian section surgery because of condylomata acuminata due to human papilloma virus (HPV) infection in the mother. In early infancy, mild eczema on the child&#8217;s face, in the perioral and periorbital areas was observed. By the end of the first year of life the child had thrived and had not suffered from any severe infections. She received live BCG vaccine, tetanus and diphtheria toxoids, inactivated pertussis, inactivated poliomyelitis, recombinant hepatitis B and conjugated pneumococcal as well as Haemophilus influenzae B vaccines without adverse reactions after the immunizations.On admission the child demonstrated paroxysmal non&#8208;productive cough and clinical signs of respiratory insufficiency along with injected oral pharyngeal mucosa and dry erythematous lips. Fine papular skin eruption affecting the face was observed. Neither peripheral lymph nodes nor internal organs of the abdominal cavity were palpable. During hospitalization, lymphopenia, anemia, and thrombocytosis (the peak platelet count was 778&#8195;&#215;&#8195;109/l) as well as increased levels of inflammatory markers and coagulopathy were found in laboratory tests. Dilation of the right coronary artery was revealed in echocardiography, giving rise to the suspicion of Kawasaki disease and the instituting of treatment with aspirin and immunoglobulins. Despite intensive pharmacotherapy with antibiotics, trimethoprime/sulfamethoxazole, acyclovir and antifungal agents, rapid deterioration of the child&#8217;s clinical state and the exacerbation of respiratory insufficiency accompanied by progression in radiological features of the respiratory distress syndrome (RDS) occured. On a chest X&#8208;ray, massive alveolar and interstitial infiltrations with bilaterally decreased aeration of the lung fields and blurred borders of the diaphragm and the heart shape (as shown on Figure&#160;1) were discernible. Differential diagnostics that included examinations of the tracheal aspirate samples aimed at infectious agents were carried out. Infections with viruses &#8211; RSV A and B, parainfluenza viruses 1,2, and 3, influenza A including H1N1 subtype and B, adenoviruses, CMV, EBV, MPV, rhinovirus, human immunodeficiency virus (HIV), with bacteria, such as Streptococcus pneumoniae, Haemophilus influenzae, Legionella, Bordetella pertussis, Mycoplasma pneumoniae, Chlamydophila pneumoniae, as well as with fungi &#8211;Candida spp. and P.&#160;jiroveci were excluded based on negative PCR examinations, whereas the human coronavirus HKU1 &#8211; RNA proved positive. Peripheral blood lymphocyte flow cytometric immunophenotyping revealed a total lack of expression of antigens characteristic for CD4 and CD8 T&#8208;cell subsets as well as NK cells along with the presence of functionally immature transitional and na&#239;ve B cells. This SCID phenotype was subsequently confirmed by bone marrow flow cytometric evaluation. However, the presence of few NK cells was revealed, indicating for T&#8208;B&#8195;+&#8195;NK&#8195;+&#8195;SCID.Intensive therapy and mechanical ventilation conducted in the Department of Pediatric Anesthesiology and Intensive Care did not contribute to either clinical or radiological improvement and the child died because of multiorgan failure.\nDiscussion\nAnalysis of the available data concerning the effects of non&#8208;SARS human coronaviruses (HCoV) in children suggests that their clinical relevance in children is substantial, particularly in the hospital settings, even though the incidence of HCoV airway infections are generally less frequent than with other viruses which have an established role in respiratory disease, such as RSV and influenza.\n4\n However, detailed epidemiological data on the prevalence of HCoV infections in children are discordant, ranging from 2&#183;5% of NL63 strain in young children with bronchiolitis reported by Ebihara et&#160;al.\n\n5\n to 18% in the study by Vabret et&#160;al.\n\n6\n in different age groups of the child population. Moreover, seroconversion with regard to HCoV&#8208;229E and above&#8208;mentioned HCoV&#8208;NL63 in young children was much higher and was estimated to 42&#183;9&#8211;50% and 75%, respectively.\n7\n The characteristics of clinical manifestation of coronavirus respiratory tract infection are predominantly reliant on case reports, and in otherwise, healthy children are comparable with bronchitis, bronchiolitis, and pneumonia due to other viral infections. The epidemiological study by Kuypers et&#160;al.\n\n8\n indicated that a considerable proportion of coronavirus&#8208;infected children had underlying chronic central nervous system, cardiovascular, pulmonary, allergic, and renal or hepatic conditions and diseases. These authors also paid attention to immunocompromised pediatric patients with acute lymphocytic leukemia and organ transplant recipients as a high&#8208;risk group for the development of severe lung disease. However, it is worth noting that coronavirus respiratory infections have not been described in children with genetically determined immunodeficiencies thus far and this is the first report of a documented HCoV&#8208;HKU1&#8208;related pneumonia with the RDS in a child with SCID. It is also interesting to note that the preliminary clinical diagnosis in this patient was Kawasaki disease, what is consistent with the hypothesis by Esper et&#160;al.\n\n9\n regarding the association between Kawasaki disease with HCoV infection, supported by identification of the &#8216;New Heaven&#8217; coronavirus (HCoV&#8208;NH) in 72&#183;7% of respiratory specimen from affected children.\nConcluding remarks\nThe identification of HCoV&#8208;HKU1 provides a novel insight into the epidemiology and clinical implications of coronavirus infections in severely immunocompromised children and indicates for consideration of this pathogen&#8208;related etiology of respiratory infection in SCID. Further, epidemiological studies are necessary to define the impact of HCoV on lung disease in children with immunodeficiencies.\nCompeting interests\nThe authors have no competing interests.",
        "pmid": "23199056",
        "title": "Fatal respiratory distress syndrome due to coronavirus infection in a child with severe combined immunodeficiency",
        "journal_title": "Influenza and Other Respiratory Viruses",
        "authors": "Aleksandra Szczawinska\u2010Poplonyk;Katarzyna Jonczyk\u2010Potoczna;Anna Breborowicz;Alicja Bartkowska\u2010Sniatkowska;Magdalena Figlerowicz",
        "doi": "10.1111/irv.12059"
    },
    {
        "keywords": "",
        "abstract": "The Middle East respiratory syndrome (MERS) coronavirus, a newly identified pathogen, causes severe pneumonia in humans. MERS is caused by a coronavirus known as MERS-CoV, which attacks the respiratory system. The recently defined receptor for MERS-CoV, dipeptidyl peptidase 4 (DPP4), is generally expressed in endothelial and epithelial cells and has been shown to be present on cultured human nonciliated bronchiolar epithelium cells. In this paper, a class of novel four-dimensional dynamic model describing the infection of MERS-CoV is given, and then global stability of the equilibria of the model is discussed. Our results show that the spread of MERS-CoV can also be controlled by decreasing the expression rate of DPP4.",
        "body": "1. Introduction\nThe Middle East respiratory syndrome (MERS) coronavirus, a newly identified pathogen, causes severe pneumonia in humans, with a mortality of nearly 44%. Human-to-human spread has been demonstrated, raising the possibility that the infection could become pandemic [1]. A colorized electron micrograph shows the coronavirus MERS-CoV acute viral respiratory illness that is characterized primarily by cough, fever, and shortness of breath and is sometimes associated with severe and potentially fatal complications such as pneumonia and kidney failure. The illness was first observed in June 2012 in Jiddah, Saudi Arabia, and soon afterward it was reported in other countries in the Middle East, including Jordan, Qatar, and the United Arab Emirates (UAE). It later was detected in Europe, including cases in France, Germany, Italy, and the United Kingdom; in the North African country of Tunisia; and in countries more distant from the Middle East, including China, Malaysia, South Korea, and the United States. The largest MERS outbreak outside Saudi Arabia occurred in 2015, when an individual who had recently traveled to the Middle East subsequently fell ill in South Korea, transmitting the disease to close contacts. The dissemination of the disease by infected travelers leaving the Middle East suggested that MERS had the potential to escalate into an international public health emergency. The possibility of a pandemic was thought to be impeded, however, by the limited ability of the disease to be passed from one person to another. MERS is caused by a coronavirus known as MERS-CoV, which attacks the respiratory system. The recently defined receptor for MERS-CoV, dipeptidyl peptidase 4 (DPP4), is generally expressed in endothelial and epithelial cells and has been shown to be present on cultured human nonciliated bronchiolar epithelium cells, providing further information on the respiratory tropism of MERS-CoV [2]. Symptoms of illness appear anytime from 2 to 14 days following infection. Cough, fever, and shortness of breath are the primary symptoms, but others such as diarrhea, nausea, vomiting, and myalgia (muscle pain) can also occur. In some persons, infection produces no symptoms or only mild cold-like symptoms, whereas in others, particularly in persons with underlying medical conditions, infection can produce severe illness [3].It is well-known that dynamic models are still playing important roles in describing the interactions among uninfected cells, free viruses, and immune responses (see, e.g., [4&#8211;7]). A three-dimensional dynamic model for viral infection is proposed by Nowak et al. (see, e.g., [5&#8211;7]).(1)T&#729;=&#955;&#8722;&#946;vtTt&#8722;dTt,I&#729;=&#946;vtTt&#8722;d1It,v&#729;=d1NIt&#8722;cvt.In model (1), T(t), I(t), and v(t) denote the concentration of uninfected cells, infected cells, and free viruses at time t, respectively. The constant &#955; > 0 is the rate at which new uninfected cells are generated (from a pool of precursor cells). The constants d > 0 and &#946; &#8805; 0 are the death rate of uninfected cells and the rate constant characterizing infection of the cells, respectively. The constant d1 > 0 is the death rate of the infected cells due to either viruses or immune responses. The infected cells produce new viruses at the rate d1N during their life, on average having the length 1/d1, where N > 0 is some integer number. The constant c > 0 is the rate at which the viruses are cleared, and the average lifetime of a free virus is 1/c.\nFigure 1 shows a interaction procedure between uninfected cells and MERS-CoV mediated by DPP4 receptors. Based on basic dynamic model (1) and Figure 1, we propose the following novel four-dimensional dynamic model which describes the spread of the MERS-CoV and the expression of DPP4:(2)T&#729;=&#955;&#8722;&#946;DtvtTt&#8722;dTt,I&#729;=&#946;DtvtTt&#8722;d1It,v&#729;=d1NIt&#8722;cvt,D&#729;=&#955;1&#8722;&#946;1&#946;DtvtTt&#8722;&#947;Dt.In model (2), D(t) represents the concentration of DPP4 on the surface of uninfected cells, which can be recognized by surface spike (S) protein of MERS-CoV (see, e.g., [8]). Infected cells are produced from uninfected cells and free viruses at the rate (&#946;D(t))v(t)T(t). It is assumed that DPP4 is produced from the surface of uninfected cells at the constant rate &#955;1 > 0. DPP4 is destroyed, when free viruses try to infect uninfected cells, at the rate &#946;1(&#946;D(t))v(t)T(t), and is hydrolyzed at the rate &#947;D(t). Here, &#946;1 &#8805; 0 and &#947; > 0 are constants. It is assumed that there is no undestroyed DPP4 on the surface of infected cells. All other parameters in model (2) have similar biological meanings to that in model (1).The initial condition of model (2) is given as T(0) &#8805; 0, I(0) &#8805; 0, v(0) &#8805; 0, and D(0) &#8805; 0. It is not difficult to show that the solution (T(t), I(t), v(t), D(t)) with the initial condition is existent, unique, bounded, and nonnegative for all t &#8805; 0 (in fact, it also has T(t) > 0 and D(t) > 0 for all t > 0). If T(0) > 0, I(0) > 0, v(0) > 0, and D(0) > 0, it is easily proven that the corresponding solution (T(t), I(t), v(t), D(t)) is positive for all t &#8805; 0.Furthermore, it can be easily shown that the set (3)&#937;=T,I,v,D&#8201;&#8739;&#8201;0&#8804;T&#8804;T0,&#8201;&#8201;I&#8805;0,&#8201;&#8201;v&#8805;0,&#8201;&#8201;0&#8804;D&#8804;D0,&#8201;&#8201;T+I+aNv&#8804;&#955;&#956;is attractive and positively invariant with respect to model (2), where 0 < a < 1, &#956; = min&#8289;{d, (1 &#8722; a)d1, c}.The purpose of the paper is to study local and global stability of the equilibria of model (2) by using Roth-Hurwitz criterion and constructing suitable Lyapunov function (see, e.g., [9&#8211;13]).\n2. Local and Global Stability of the Equilibria\nThe basic reproductive ratio of the virus for model (2) is R0 = N&#946;&#955;&#955;1/cd&#947;. Model (2) always has an infection-free equilibrium E0 = (T0, 0,0, D0) = (&#955;/d, 0,0, &#955;1/&#947;). If R0 > 1, model (2) also has unique infected equilibrium E&#8727; = (T&#8727;, I&#8727;, v&#8727;, D&#8727;), where, for &#946;1 = 0, v = v&#8727; = d&#947;(R0 &#8722; 1)/&#946;&#955;1, for &#946;1 > 0, v = v&#8727; > 0 is the positive root of the equation &#946;&#946;1c2v2 &#8722; Nc&#946;(&#955;1 + &#955;&#946;1)v + Ncd&#947;(R0 &#8722; 1) = 0, and (4)T&#8727;=&#955;d&#8722;cv&#8727;Nd,I&#8727;=cv&#8727;Nd1,D&#8727;=cN&#946;T&#8727;,v&#8727;=N&#946;&#955;1+&#955;&#946;1&#8722;N2&#946;2&#955;1&#8722;&#955;&#946;12+4N&#946;1&#946;cd&#947;2&#946;&#946;1c.First, we have the following result.\nTheorem 1 . With respect to the set &#937;1 = {(T, I, v, D)&#8739;(T, I, v, D) &#8712; &#937;, T > 0, D > 0}, the infection-free equilibrium E0 = (T0, 0,0, D0) is globally asymptotically stable when R0 < 1 and globally attractive when R0 = 1.\n\nProofAt any equilibrium (T, I, v, D), Jacobian matrix of model (2) is(5)J=&#8722;&#946;vD&#8722;d0&#8722;&#946;TD&#8722;&#946;vT&#946;vD&#8722;d1&#946;TD&#946;vT0Nd1&#8722;c0&#8722;&#946;1&#946;vD0&#8722;&#946;1&#946;TD&#8722;&#946;1&#946;vT&#8722;&#947;.By simple computations, we can get that the characteristic equation at E0 is f(&#961;) = (&#961; + d)(&#961; + &#947;)[&#961;2 + (c + d1)&#961; + cd1(1 &#8722; R0)] = 0. Clearly, if R0 < 1, all roots of f(&#961;) = 0 have negative real parts. Hence, E0 is local asymptotic stability by Routh-Hurwitz criterion. If R0 = 1, f(&#961;) = 0 has the zero root &#961; = 0 and three negative roots. Hence, E0 is linearly stable.Construct the Lyapunov function as follows: (6)UT,I,v,D=T0TT0&#8722;1&#8722;ln&#8289;TT0+1+&#946;1I+1+&#946;1Nv+D0DD0&#8722;1&#8722;ln&#8289;DD0.It is clear that U is continuous on &#937;1 and positive definite with respect to E0 and satisfies condition (ii) of Definition 1.1 in [14] or Lemma 3.1 in [15] on &#8706;&#937; = &#937;&#8726;&#937;1. Calculating the derivative of U along the solutions of model (2), we have, for t &#8805; 0,(7)dUdt&#955;&#8722;&#946;DvT&#8722;dT&#8722;T0&#955;T&#8722;&#946;Dv&#8722;d+&#946;DvT+&#946;1&#946;DvT&#8722;1+&#946;1d1I+1+&#946;1Nd1NI&#8722;cvN&#8722;&#946;1cvN+&#955;1&#8722;&#946;1&#946;DvT&#8722;&#947;D&#8722;D0&#955;1D&#8722;&#946;1&#946;vT&#8722;&#947;=&#955;+d&#955;d&#8722;dT&#8722;&#955;T0T+&#955;1+&#947;&#955;1&#947;&#8722;&#947;D&#8722;&#955;1D0D+&#946;DvT0&#8722;cvN+&#946;1&#946;D0vT&#8722;&#946;1cvN=&#955;2&#8722;TT0&#8722;T0T+&#955;12&#8722;DD0&#8722;D0D+v&#946;DT0&#8722;cN+&#946;1v&#946;D0T&#8722;cN&#8804;&#955;2&#8722;TT0&#8722;T0T+&#955;12&#8722;DD0&#8722;D0D+cNR0&#8722;11+&#946;1v.Clearly, dU/dt &#8804; 0 on &#937;1 by R0 &#8804; 1. Define Q = {dU/dt = 0&#8739;(T, I, v, D) &#8712; &#937;, U(T, I, v, D) < +&#8734;}. Let M be the largest subset in Q which is invariant with respect to the set model (2). Hence, we have that M &#8834; Q &#8834; {(T, I, v, D)&#8739;(T, I, v, D) &#8712; &#937;, T = T0, D = D0}. From the invariance of M and model (2), we can easily show that M = {E0}. Therefore, it follows from Theorem 1.2 in [14] or Lemma 3.1 in [15] that E0 is globally attractive. This completes the proof.\nFor local and global stability of the infected equilibrium E&#8727;, we have the following result.\nTheorem 2 . With respect to the set &#937;2 = {(T, I, v, D)&#8739;(T, I, v, D) &#8712; &#937;, T > 0, I > 0, v > 0, D > 0}, the infected equilibrium E&#8727; is locally asymptotically stable when R0 > 1. In addition, if (2d&#947;&#956;a)2 &#8805; &#946;1&#946;2&#955;1&#955;3N2, where 0 < a < 1, &#956; = min&#8289;{d, (1 &#8722; a)d1, c}, the infected equilibrium E&#8727; is globally asymptotically stable.\n\nProofThe characteristic equation at of model (2) at E&#8727; is (8)g&#961;=&#961;4+a3&#961;3+a2&#961;2+a1&#961;+a0=0,where (9)a0=T&#8727;&#946;cdv&#8727;&#946;1d1+D&#8727;&#946;c&#947;v&#8727;d1,a1=T&#8727;&#946;cv&#8727;&#946;1d1+T&#8727;&#946;dv&#8727;&#946;1d1+D&#8727;&#946;c&#947;v&#8727;+D&#8727;&#946;cv&#8727;d1+D&#8727;&#946;&#947;v&#8727;d1+&#946;cdv&#8727;&#946;1+cd&#947;+d&#947;d1,a2=T&#8727;&#946;cv&#8727;&#946;1+T&#8727;&#946;dv&#8727;&#946;1+T&#8727;&#946;v&#8727;&#946;1d1+D&#8727;&#946;cv&#8727;+D&#8727;&#946;&#947;v&#8727;+D&#8727;&#946;v&#8727;d1+cd+c&#947;+d&#947;+dd1+&#947;d1,a3=&#946;1&#946;v&#8727;T&#8727;+&#946;v&#8727;D&#8727;+c+d+&#947;+d1.It is obvious that ai > 0&#8201;&#8201;(i = 0,1, 2,3). Furthermore, by using Matlab program, it can been shown that &#916;3 = a1a2a3 &#8722; a12 &#8722; a32a0 has 200 items in which all items are positive. Hence, E&#8727; is local asymptotic stability by Routh-Hurwitz criterion.Construct the Lyapunov function as follows: (10)W=&#946;1T&#8727;TT&#8727;&#8722;1&#8722;ln&#8289;TT&#8727;+&#946;1I&#8727;II&#8727;&#8722;1&#8722;ln&#8289;II&#8727;+&#946;1Nv&#8727;vv&#8727;&#8722;1&#8722;ln&#8289;vv&#8727;+D&#8727;DD&#8727;&#8722;1&#8722;ln&#8289;DD&#8727;.It is clear that W is continuous on &#937;2 and positive definite with respect to E&#8727; and satisfies condition (ii) of Definition 1.1 in [14] or Lemma 3.1 in [15] on &#8706;&#937; = &#937;&#8726;&#937;2. Calculating the derivative of W along the solutions of model (2), we have, for t &#8805; 0, (11)dWdt=&#946;1&#955;&#8722;dT&#8722;&#946;1T&#8727;&#955;T&#8722;&#946;Dv&#8722;d&#8722;&#946;1I&#8727;&#946;DvTI&#8722;d1&#8722;&#946;1v&#8727;d1Iv&#8722;cN+&#955;1&#8722;&#946;1&#946;DvT&#8722;&#947;D&#8722;D&#8727;&#955;1D&#8722;&#946;1&#946;Tv&#8722;&#947;&#8722;&#946;1cvN=&#946;1&#946;D&#8727;v&#8727;T&#8727;+2&#946;1dT&#8727;&#8722;&#946;1dT&#8722;&#946;1T&#8727;T&#946;D&#8727;v&#8727;T&#8727;+dT&#8727;+&#946;1&#946;DvT&#8727;&#8722;&#946;1&#946;DvTI&#8727;I+&#946;1d1I&#8727;&#8722;&#946;1d1Iv&#8727;v+&#946;1cv&#8727;N+&#946;1&#946;D&#8727;v&#8727;T&#8727;+2&#947;D&#8727;&#8722;&#946;1&#946;DvT&#8722;&#947;D&#8722;D&#8727;D&#946;1&#946;D&#8727;v&#8727;T&#8727;+&#947;D&#8727;+&#946;1&#946;D&#8727;Tv&#8722;&#946;1cvN=2&#946;1dT&#8727;&#8722;&#946;1dT&#8722;&#946;1T&#8727;TdT&#8727;+2&#947;D&#8727;&#8722;&#947;D&#8722;D&#8727;D&#947;D&#8727;+2&#946;1&#946;D&#8727;v&#8727;T&#8727;+&#946;1cv&#8727;N+&#946;1d1I&#8727;&#8722;&#946;1&#946;D&#8727;v&#8727;T&#8727;D&#8727;D&#8722;&#946;1&#946;D&#8727;v&#8727;T&#8727;T&#8727;T&#8722;&#946;1&#946;DvTI&#8727;I&#8722;&#946;1d1Iv&#8727;v+&#946;1&#946;DvT&#8727;+&#946;1&#946;D&#8727;Tv&#8722;&#946;1cvN&#8722;&#946;1&#946;DvT=&#946;1dT&#8727;2&#8722;TT&#8727;&#8722;T&#8727;T+&#947;D&#8727;2&#8722;DD&#8727;&#8722;D&#8727;D+&#946;1d1I&#8727;4&#8722;T&#8727;T&#8722;D&#8727;D&#8722;&#946;TvDd1I&#8722;Iv&#8727;I&#8727;v+&#946;1&#946;vTD&#8727;+T&#8727;D&#8722;TD&#8722;c&#946;N=&#946;1dT&#8727;2&#8722;TT&#8727;&#8722;T&#8727;T+&#947;D&#8727;2&#8722;DD&#8727;&#8722;D&#8727;D+&#946;1d1I&#8727;4&#8722;T&#8727;T&#8722;D&#8727;D&#8722;TvDI&#8727;T&#8727;v&#8727;D&#8727;I&#8722;Iv&#8727;I&#8727;v+&#946;1&#946;vTD&#8727;+T&#8727;D&#8722;TD&#8722;T&#8727;D&#8727;=&#8722;&#946;1dTT&#8722;T&#8727;2&#8722;&#947;DD&#8722;D&#8727;2&#8722;&#946;&#946;1vD&#8722;D&#8727;T&#8722;T&#8727;+&#946;1d1I&#8727;4&#8722;T&#8727;T&#8722;D&#8727;D&#8722;TvDI&#8727;T&#8727;v&#8727;D&#8727;I&#8722;Iv&#8727;I&#8727;v.Since the arithmetical mean is greater than or equal to the geometrical mean, we have that T&#8727;/T + D&#8727;/D + TvDI&#8727;/T&#8727;v&#8727;D&#8727;I + Iv&#8727;/I&#8727;v &#8722; 4 &#8804; 0, for any T, I, v, D > 0, and that T&#8727;/T + D&#8727;/D + TvDI&#8727;/T&#8727;v&#8727;D&#8727;I + Iv&#8727;/I&#8727;v &#8722; 4 = 0 only if T&#8727;/T = D&#8727;/D = TvDI&#8727;/T&#8727;v&#8727;D&#8727;I = Iv&#8727;/I&#8727;v. Thus, we have T = T&#8727;, D = D&#8727;.On the other hand, notice the inequality in [16]: (12)&#8722;xz2+yz&#8804;&#8722;12xz2+y22xx>0,&#8201;&#8201;y&#8805;0,&#8201;&#8201;z&#8805;0.We have (13)&#8722;&#946;1dTT&#8722;T&#8727;2&#8722;12&#946;&#946;1vD&#8722;D&#8727;T&#8722;T&#8727;&#8722;&#947;DD&#8722;D&#8727;2&#8722;12&#946;&#946;1vD&#8722;D&#8727;T&#8722;T&#8727;&#8804;&#8722;&#946;1d2TT&#8722;T&#8727;2+D&#946;&#946;1v28&#947;T&#8722;T&#8727;2&#8722;&#947;2DD&#8722;D&#8727;2+T&#946;&#946;1v28&#946;1dD&#8722;D&#8727;2=&#8722;&#946;1d2T&#8722;D&#946;&#946;1v28&#947;T&#8722;T&#8727;2&#8722;&#947;2D&#8722;T&#946;&#946;1v28&#946;1dD&#8722;D&#8727;2.Note that the inequalities &#946;1d/2T &#8722; D(&#946;&#946;1v)2/8&#947; &#8805; 0 and &#947;/2D &#8722; T(&#946;&#946;1v)2/8&#946;1d &#8805; 0 are equivalent to the inequality 4d&#947; &#8805; &#946;1&#946;2TDv2. Since T(t) &#8804; T0, D &#8804; D0, and v(t) &#8804; &#955;N/&#956;a for all t &#8805; 0, we have that the inequality 4d&#947; &#8805; &#946;1&#946;2TDv2 holds, if the condition (2d&#947;&#956;a)2 &#8805; &#946;1&#946;2&#955;1&#955;3N2 in Theorem 2 is satisfied. Therefore, dW/dt &#8804; 0 on &#937;2.Define Q = {dW/dt = 0&#8739;(T, I, v, D) &#8712; &#937;, W(T, I, v, D) < +&#8734;}. Let M be the largest subset in Q which is invariant with respect to the set of model (2). Hence, we have that M &#8834; Q &#8834; {(T, I, v, D)&#8739;(T, I, v, D) &#8712; &#937;, T = T&#8727;, D = D&#8727;}. From the invariance of M and model (2), we can also show that M = {E&#8727;}. Hence, it follows from Theorem 1.2 in [14] or Lemma 3.1 in [15] that E&#8727; is globally attractive. This completes the proof.\n\n3. Simulations and Conclusions\nLet us first give some numerical simulations on the orbits of model (2). Take the following a set of parameters, &#955; = &#955;1 = 1, &#946; = 0.001, d = d1 = 0.05, N = 1, c = 0.2, &#946;1 = 1, and &#947; = 0.11. We can compute the values of the infection-free equilibrium E0 and the basic reproductive ratio, E0 = (20,0, 0,9.0909) and R0 = 0.90909 < 1.  Figure 2(a) shows the trajectory of model (2) with suitable initial condition, which shows that the infection-free equilibrium E0 is asymptotically stable.Let us take &#947; = 0.05, and all the other parameters are the same as above. We can also compute the values of the infection-free equilibrium E0, the infected equilibrium E&#8727;, and the basic reproductive ratio, E0 = (20,0, 0,20), E&#8727; = (14.142,5.8579,1.4645,14.142), and R0 = 2 > 1.  Figure 2(b) shows orbits of model (2) with suitable initial conditions, which shows that the infected equilibrium E&#8727; is asymptotically stable. We would like to point out here that, based on the numerical simulations, the condition (2d&#947;&#956;a)2 &#8805; &#946;1&#946;2&#955;1&#955;3N2 may be further weakened or even removed.Finally, by using the basic reproductive ratio R0 = N&#946;&#955;&#955;1/cd&#947;, let us give some simple discussions on the interactions between the protein DPP4 and the virus infection. Usually, in the absence of any drug treatment, all the parameters in model (2) and the corresponding basic reproductive ratio R0 can be regarded as relatively fixed constants. If some drug treatment measures are taken, the effectiveness of the treatment can be reflected in the regulation of the parameter &#947;. For example, by increasing the value of &#947;, the value of the basic reproductive ratio of R0 can be changed from greater than 1 to less than 1. In the numerical simulations in this section, Figure 2(b) shows that the virus infection will be persistent, when &#947; = 0.05 and R0 = 2 > 1. If increasing &#947; from &#947; = 0.05 to &#947; = 0.11, Figure 2(a) shows that the virus infection can be controlled, since R0 = 0.9090 < 1.",
        "pmid": "28894474",
        "title": "A Novel Dynamic Model Describing the Spread of the MERS-CoV and the Expression of Dipeptidyl Peptidase 4",
        "journal_title": "Computational and Mathematical Methods in Medicine",
        "authors": "Siming Tang;Wanbiao Ma;Peifan Bai",
        "doi": "10.1155/2017/5285810"
    },
    {
        "keywords": "Metapneumovirus;coronavirus;severe acute respiratory syndrome;SARS;virus isolation;Hong Kong;China",
        "abstract": "We used a combination approach of conventional virus isolation and molecular techniques to detect human metapneumovirus (HMPV) in patients with severe acute respiratory syndrome (SARS). Of the 48 study patients, 25 (52.1%) were infected with HMPV; 6 of these 25 patients were also infected with coronavirus, and another 5 patients (10.4%) were infected with coronavirus alone. Using this combination approach, we found that human laryngeal carcinoma (HEp-2) cells were superior to rhesus monkey kidney (LLC-MK2) cells commonly used in previous studies for isolation of HMPV. These widely available HEp-2 cells should be included in conjunction with a molecular method for cell culture followup to detect HMPV, particularly in patients with SARS.",
        "body": "Methods\n\nStudy Population\nIn early March 2003, an outbreak of SARS occurred in the Prince of Wales Hospital (the teaching hospital of The Chinese University of Hong Kong). Our study participants were patients admitted to our hospital for suspected SARS during the first week of the outbreak (12). These patients fulfilled the World Health Organization definition for probable SARS cases (13). Briefly, patients had an acute onset of fever (>38&#176;C, most with chills or rigor), dyspnea, myalgia, headache, and hypoxemia. Peripheral air-space consolidation subsequently developed in all study patients as observed on chest radiographs or thoracic computed tomographic scan; patients showed no response to antimicrobial drugs prescribed for typical and atypical pneumonia (&#946;-lactams, macrolides, and fluoroquinolones).During our study, we examined 48 patients who comprised our first group of SARS patients and had a clear history of exposure. Forty-five participants were adults (26 men, 19 women) 21&#8211;69 years of age (mean 35.4 years of age; standard deviation 11.5 years). The group included 26 healthcare workers and 7 medical students who worked in a ward (index ward) in the hospital where a few patients with SARS had stayed. The remaining 12 patients had been hospitalized or were visitors to the same ward. Three study participants were children (two boys, one girl) 2&#8211;7 years of age. All these children were living with persons who had been hospitalized or were visitors to the index ward and who had contracted SARS.\nVirus Isolation\nNasopharyngeal aspirate (NPA) samples were taken from all patients by inserting a suction catheter into the nasopharyngeal area via the nostril. A low suction force was applied to collect approximately 0.5 mL fluid, which was then transferred into 2 mL of viral transport medium. All NPAs were added onto rhesus monkey kidney (LLC-MK2), human laryngeal carcinoma (HEp-2), Mardin Darby Canine Kidney (MDCK), human embryonic lung fibroblast, Buffalo green monkey kidney (BGM), and African green monkey kidney (Vero) monolayers. All cell cultures were incubated at 37&#176;C, except for MDCK, which was incubated at 33&#176;C. All NPAs were added to an additional LLC-MK2 cell culture tube and incubated at 33&#176;C. Cell monolayers were examined daily for cytopathic effect. After 14 days of incubation, a hemadsorption test for LLC-MK2 and MDCK monolayers was performed. All cell cultures materials were kept frozen for subsequent analyses.\nHuman Metapneumovirus Reverse Transcription-Polymerase Chain Reaction (RT-PCR)\nTo detect HMPV, we used a nested RT-PCR focused on the F-gene. This RT-PCR was applied on all cell cultures, regardless of cytopathic effect. After one cycle of freeze-and-thaw, RNA was extracted from cell cultures by using the QIAamp Viral RNA Mini Kit (QIAGEN GmbH, Hilden, Germany), according to the manufacturer&#8217;s protocol. The outer primers were 5&#8242;-AGC TGT TCC ATT GGC AGC A-3&#8242; for RT and amplification and 5&#8242;-ATG CTG TTC RCC YTC AAC TTT-3&#8242; (R = A or G, Y = C or T) for amplification. These primers were designed on the basis of HMPV sequences available from GenBank. The reaction was carried out in a single-tube (Superscript One-Step RT-PCR and Platinum Taq; Invitrogen Corp., Carlsbad, CA) by using 0.2 &#956;M of each primer and thermal cycling conditions of 50&#176;C for 30 min and 94&#176;C for 3 min; followed by 40 cycles of 94&#176;C for 30 s, 52&#176;C for 30 s, 72&#176;C for 45 s, and a final extension at 72&#176;C for 7 min. For the second round of amplification, we used 0.2 &#956;M of inner primers 5&#8242;-GAG TAG GGA TCA TCA AGC A-3&#8242; and 5&#8242;-GCT TAG CTG RTA TAC AGT GTT-3&#8242;. The PCR was conducted at 95&#176;C for 15 min for denaturation of DNA templates and activation of the hot-start DNA polymerase (HotStarTaq, QIAGEN GmbH), followed by 40 cycles at 94&#176;C for 30 s, 54&#176;C for 30 s, and 72&#176;C for 45 s, and a final extension at 72&#176;C for 7 min. PCR products detected by agarose gel electrophoresis were analyzed for sequence homology with known HMPV strains. In addition to virus isolation, RNA was extracted directly from NPAs for HMPV RT-PCR by using the same protocol as for cell cultures.\nCoronavirus RT-PCR\nRNA was extracted from the supernatant of Vero cell cultures showing cytopathic effect by using the same method as for HMPV. Coronavirus was detected by RT-PCR with primers COR-1 (sense) 5&#8242; CAC CGT TTC TAC AGG TTA GCT AAC GA 3&#8242; and COR-2 (antisense) 5&#8242; AAA TGT TTA CGC AGG TAA GCG TAA AA 3&#8242;, which had been shown to be specific for the novel coronavirus detected from patients with SARS (14). The RT-PCR for coronavirus was conducted similarly to HMPV (by using 0.6 &#956;M of each primer and thermal cycling conditions of 54&#176;C for 30 min, 94&#176;C for 3 min; 45 cycles of 94&#176;C for 45 s, 60&#176;C for 45 s, 72&#176;C for 45 s; and 72&#176;C for 7 min).\nSequence Analysis\nThe nucleotide sequence of purified PCR products was determined by PCR-based cycle sequencing performed with the inner primers. Sequencing reactions were performed according to the manufacturer&#8217;s protocol (BigDye Terminator Cycle Sequencing Kit version 3.1, Applied Biosystems, Foster City, CA) and run on the ABI Prism 3100 Genetic Analyzer. All sequences were confirmed by repeated PCRs and sequencing from both directions.\nElectron Microscopy\nSelected cell cultures that showed cytopathic effect were examined by electron microscopy. Cell culture supernatants were coated on formvar-carbon grids and stained with 2% phosphotungstic acid.\nAntibody Detection\nTo ascertain the HMPV culture results, we obtained paired serum samples (first sample collected within 5 days and second sample collected >14 days after onset of illness) and tested for HMPV antibody. HMPV&#8211;infected LLC-MK2 cells were coated on 12-well glass slide and fixed in acetone. The presence of antibody in serum samples was tested for by using the direct immunofluorescence technique.\nExclusion of Cross-Contamination and Test for Reproducibility\nSpecimen processing, viral culture inoculation, RNA extraction, RT-PCR amplification, and PCR product analyses were conducted in different rooms. Special care was taken to avoid contamination with RNase, and to avoid cross-contamination between reactions. During the inoculation of cell monolayers, we placed a negative control using the same cell line injected with maintenance medium after every fifth cell culture tube. These negative control cell culture tubes were also incubated, examined for cytopathic effect, and processed for RT-PCR as for cell culture tubes injected with specimens. For RNA extraction and RT-PCR procedures, we placed negative controls using cell culture medium to replace cell supernatant injected with NPAs or double distilled water to replace NPA sample after every fifth reaction. These negative controls did not show positive results, which indicated the absence of cross-contamination. To test the reproducibility of RT-PCR results, we repeated the testing of all positive samples and 30 randomly selected negative samples; all results were reproducible. We also spiked negative NPA samples with HMPV RNA and repeated the extraction and RT-PCR procedures. The results showed no inhibitors were present in the extracted RNA preparations.\nResults\nOf the 48 NPAs studied, we observed no cytopathic effect on HEp-2, MDCK, human embryonic lung fibroblast, and BGM monolayers. Eleven (22.9%) specimens showed cytopathic effect of diffuse refractile rounding of cells on Vero cell monolayers 2&#8211;4 days after incubation, progressed rapidly, and involved the whole monolayer within 12&#8211;24 hours. The same cytopathic effect was reproducible on passage to Vero cells, and appeared 1 to 2 days after incubation. These Vero cell cultures were all positive by the coronavirus RT-PCR. The Vero cell culture supernatants showing cytopathic effect were randomly selected for electron microscopy examination, and coronavirus particles were seen.Five specimens showed cytopathic effect of focal refractile rounding of cells in LLC-MK2 monolayers. All these LLC-MK2 cell cultures had been incubated at 37&#176;C. Unlike the cytopathic effect observed in Vero cells attributable to coronavirus, this cytopathic effect developed after 10 to 12 days of incubation and progressed slowly to detachment from the cell monolayer.The HMPV RT-PCR examination of cell cultures was negative for human embryonic lung fibroblast, BGM cells, and Vero cells (including those positive for coronavirus). In contrast, HMPV RT-PCR showed a PCR product of the expected size (89 bp) from 25 (52.1%) isolation materials injected with specimens. The nucleotide sequences of the PCR products were identical to the F-gene fragment of HMPV (GenBank accession no. NC 004148) (1). We retrospectively examined the first round of PCR products of all positive samples. Those positive samples derived from direct NPAs did not show positive band, indicating a nested RT-PCR was necessary. However, most (27 [90%] of 30) of those derived from cell cultures showed a positive band of the expected size from the first round of PCR. The distribution of HMPV RT-PCR results on direct detection of NPAs and from different cell culture types is shown in the Table. Overall, the sensitivity of direct detection of NPAs using HMPV RT-PCR was 2 (8.0%) of 25 samples. In one of these two samples, we isolated the virus from three cell lines. In the other sample, we isolated virus from HEp-2 and LLC-MK2. Overall, HEp-2 was the most sensitive cell lines (22 [88.0%] of 25 HMPV positive samples); LLC-MK2 cells detected 6 (24.0%) of 25 samples, and MDCK cells detected 2 (8.0%) of 25 samples. Most (with the exception of three LLC-MK2&#8211;positive samples) showed positive results in HEp-2 cells. All six LLC-MK2 cell cultures positive for HMPV were incubated at 37&#176;C; three of these positive cultures that had had the corresponding LLC-MK2 cell cultures incubated at 33&#176;C showed positive results.To ascertain that cell cultures with HMPV RT-PCR&#8211;positive results represented the isolation of HMPV, all LLC-MK2 (incubated at 37&#176;C), HEp-2, and MDCK cell cultures, regardless of the HMPV RT-PCR findings, were passaged to LLC-MK2 cells for a prolonged incubation of 28 days. HEp-2 cells were not used for this purpose because HEp-2 cell monolayers are often difficult to maintain for >2 weeks. The results showed that all passages from HMPV RT-PCR&#8211;positive cell cultures showed cytopathic effect of focal refractile rounding of cells that occurred after 10 to 22 days of incubation (Figure 1). The cytopathic effect progressed slowly to detachment from the cell monolayer (Figure 2). The supernatants of these passages were also positive by the HMPV RT-PCR and had visible HMPV viral particles on electron microscopy examination (Figure 3). The passages from HMPV RT-PCR&#8211;negative supernatants did not show positive results by the above tests. We also passaged five Vero cell cultures that were positive for coronavirus to LLC-MK2 cells in a similar way. All of these passages did not show cytopathic effect and were negative by the HMPV RT-PCR.To reconfirm the fact that HMPV infections detected by this combination approach represented genuine infections, we coated HMPV-infected LLC-MK2 cells onto slides for antibody detection using the immunofluorescence technique. All HMPV culture&#8211;positive patients who had serologic evidence of infection had a more than fourfold rise in antibody titers, and 15 patients seroconverted.Overall, our results indicated that the combination approach of using conventional virus isolation and molecular detection could be successfully applied to the isolation of HMPV (Figure 4). With this approach, we found that among the 48 study participants, 6 (12.5%) had both HMPV and coronavirus isolated from NPAs, 19 (39.6%) had HMPV, and 5 (10.4%) had coronavirus. Eighteen (37.5%) had no virus isolated from the cell lines that we used.\nDiscussion\nOn the basis of a combination of conventional virus isolation system and molecular techniques, we found that 52.1% (25/48) of patients with SARS admitted to our hospital had HMPV infection. Isolation of HMPV is known to be difficult, which is why the virus could not be detected until recently. The first report on HMPV by van den Hoogen et al. (1) showed that the virus produced syncytia formation in tertiary monkey kidney cells, followed by rapid internal disruption of the cells and subsequent detachment from cell monolayer. The virus replicated poorly in Vero cells and human lung adenocarcinoma (A-549) cells and could not be propagated in MDCK cells or chicken embryo fibroblasts (1). In the study from Boivin et al. (7), multiple cell lines including LLC-MK2, HEp-2, MDCK, human foreskin fibroblast, Vero, Mink lung, A-549, human rhabdomyosarcoma (RD), transformed human kidney (293), and human colon adenocarcinoma (HT-29), were used for isolation of HMPV. The results showed that HMPV only grew on LLC-MK2 cells with cytopathic effect of round and refringent cells but without syncytia formation in most cases, an observation in agreement with our results. In that study, HEp-2 cell monolayers did not show cytopathic effect. Since the HEp-2 cells were not tested for HMPV RNA, we do not know whether our findings on HEp-2 cells were also observed by Boivin et al. In another study reported by Peret et al. (6), LLC-MK2, MDCK, and NCI-H292 cells were used; those researchers found that only LLC-MK2 cells produced cytopathic effect of focal rounding and without syncytia formation, which is also similar to our observation. The major difference in our approach for HMPV isolation compared to previous studies is the use of RT-PCR to enhance the detection of HMPV isolated from cell cultures. With this approach, we found that HEp-2 cells, a widely available and commonly used cell line, support the growth of HMPV. When RT-PCR was used to follow up all cell cultures, the sensitivity of HEp-2 cells was higher than LLC-MK2 cells, the cell line most commonly used in previous studies for HMPV. However, even using our approach, LLC-MK2 cells cannot be discarded, as in 12% of cases HMPV was only isolated from LLC-MK2 cells. In contrast, in the presence of HEp-2 cells, MDCK cells gave little additional value, as both specimens positive by MDCK cells had the viruses isolated from HEp-2 cells. In addition, our initial incubation of 14 days might not be optimal for isolating HMPV because Boivin et al. reported that the cytopathic effect took a mean incubation time of 17.3 days to develop (7). By prolonging the initial incubation of LLC-MK2 cells to 21 or 28 days, more HMPV infections might have been detected from our &#8220;negative&#8221; group.Because all our study samples were collected from patients related to the outbreak of SARS that occurred in our hospital, one cannot simply infer that this in vitro growth property can be applied to all HMPV strains in general. Nevertheless, our approach of including HEp-2 cells, a widely available cell line, to search for HMPV, in particular for those cases related to SARS, needs to be considered. In our study, six patients were co-infected with HMPV and coronavirus. Although the number was limited, our findings suggest that HMPV and coronavirus have different in vitro tropisms, and the isolation of one virus does not affect the recovery of the other from different cell lines.Overall, we confirmed that 25 (52.1%) of 48 patients admitted to our hospital with SARS had HMPV infections, with 6 also co-infected with coronavirus. However, the data on such high prevalence of HMPV should be interpreted cautiously. Our study population was based on persons and their family members who had been exposed in the index ward in our hospital. Thus, a co-circulation of two pathogens within our study group was possible. While the clinical presentations of all our study participants fulfilled the World Health Organization definition for a probable case of SARS (13), one should not infer, at this stage, that the prevalence of HMPV is similarly high in SARS outbreaks occurring elsewhere. On the other hand, the possibility of an important role of HMPV in the current worldwide outbreak of SARS should not be neglected. HMPV has also been detected in five of six SARS patients living in Canada (15). In that study series, coronavirus was also detected in five of six patients and four patients were co-infected with HMPV and coronavirus. A few recent studies implicate a strong association of a novel coronavirus with the worldwide outbreak of SARS (16&#8211;18). While both HMPV and coronavirus infections may result in severe respiratory tract diseases, their transmission efficiency may not be the same. This urgent question must be answered because the answer affects the priority for immediate development of control strategies.During this outbreak of SARS, we have applied this combination approach of conventional virus isolation and molecular detection to establish the viral infection status of other patients hospitalized for SARS. We are in the process of analyzing a larger cohort to elucidate their clinical conditions, treatment responses, and epidemiologic links with respect to the infection status for both HMPV and coronavirus. Similar work in other parts of the world is needed.",
        "pmid": "14519240",
        "title": "Human Metapneumovirus Detection in Patients with Severe Acute Respiratory Syndrome",
        "journal_title": "Emerging Infectious Diseases",
        "authors": "Paul K.S. Chan;John S. Tam;Ching-Wan Lam;Edward Chan;Alan Wu;Chi-Kong Li;Thomas A. Buckley;King-Cheung Ng;Gavin M. Joynt;Frankie W.T. Cheng;Ka-Fai To;Nelson Lee;David S.C. Hui;Jo L.K. Cheung;Ida Chu;Esther Liu;Sydney S.C. Chung;Joseph J.Y. Sung",
        "doi": "10.3201/eid0909.030304"
    },
    {
        "keywords": "coronaviruses;nsp4;nonstructural proteins",
        "abstract": "The structure of the cytosolic C-terminal domain of nonstructural protein 4 from feline coronavirus has been determined and analyzed.",
        "body": "Introduction\nThe Coronaviridae family, which is comprised of the genera Coronavirus and Torovirus, and the more distantly related Arteriviridae and Roniviridae families together form the order Nidovirales (Gorbalenya et al., 2006 &#9654;). Coronaviruses are positive-stranded RNA viruses that are frequently associated with enteric or respiratory diseases in humans, livestock and companion animals (Dye & Siddell, 2005 &#9654;). At present, they are formally classified into three genetic groups (1&#8211;3), with the first two groups further divided into two subgroups (1a/b and 2a/b; Gorbalenya et al., 2004 &#9654;; Lai & Holmes, 2001 &#9654;), but as our understanding of natural coronavirus diversity progresses novel subgroups continue to be recognized (Woo et al., 2009 &#9654;). Viruses that belong to different subgroups have diverged profoundly. A fraction of their proteins are subgroup-specific and the amino-acid sequences of their most conserved proteins may differ by as much as 50%. The best-known member of this family, severe acute respiratory syndrome coronavirus (SARS-CoV), belongs to subgroup 2b, whereas feline coronavirus (FCoV), characterized in this study, belongs to subgroup 1a (Gorbalenya et al., 2004 &#9654;).Feline infectious peritonitis virus (FIPV) is a pathogenic FCoV variant that emerged by mutation of the relatively benign enteric FCoV (Poland et al., 1996 &#9654;; Vennema et al., 1998 &#9654;) and causes a fatal immune-mediated disease in cats (Pedersen, 1995 &#9654;). Since the variations are minor and despite the fact that we are working with a construct derived from an FIPV strain, we will henceforth use FCoV as an abbreviation for the virus. The FCoV genome (strain FIPV WSU-79/1146) consists of 29&#8197;125 nucleotides and contains six open reading frames (ORFs; Dye & Siddell, 2005 &#9654;). The first two ORFs, namely ORF1a and ORF1b, comprising the 5&#8242;-most gene (i.e. gene 1), encode two large replicase polyproteins, pp1a and pp1ab. Proteolytic cleavage of these polyproteins by virus-encoded proteinases, i.e. the 3C-like main proteinase (Mpro in nsp5) and the papain-like accessory proteinases (PLpro 1 and 2 in nsp3), are predicted to give rise to a total of 16 mature nonstructural proteins (nsps; Ziebuhr et al., 2000 &#9654;; Dye & Siddell, 2005 &#9654;). In addition to the proteases mentioned above, associated enzymatic activities have been identified for several coronavirus nsps, including deubiquitinating and adenosine diphosphate-ribose-1&#8242;-phosphatase (ADRP) functions (nsp3), RNA-dependent RNA polymerases with low (nsp8) and high (nsp12) processivity, helicase (nsp13), RNA exonuclease and N7-methyltransferase (nsp14), RNA endoribonuclease (nsp15) and 2&#8242;-O-methyltransferase (nsp16) (Anand et al., 2003 &#9654;; Bhardwaj et al., 2004 &#9654;; Chen et al., 2009 &#9654;; Cheng et al., 2005 &#9654;; Gorbalenya et al., 1989 &#9654;; Harcourt et al., 2004 &#9654;; Imbert et al., 2008 &#9654;; Ivanov, Hertzig et al., 2004 &#9654;; Ivanov, Thiel et al., 2004 &#9654;; Ratia et al., 2006 &#9654;; Seybert et al., 2000 &#9654;; Snijder et al., 2003 &#9654;). Tertiary structures, solved using X-ray or/and NMR analyses, have been reported for a substantial number of nsps from at least one coronavirus, typically SARS-CoV (reviewed in Bartlam et al., 2005 &#9654;; Mesters et al., 2006 &#9654;). These structures represent a variety of separate domains, entire proteins and even multiprotein complexes. Despite these remarkable advances, many domains, including those residing in nsp4, remain poorly characterized.Nsp4 is an approximately 500-amino-acid replicase subunit that is released by the combined activity of the nsp3 and nsp5 proteases. It is predicted to be one of the three membrane-spanning proteins (the others are nsp3 and nsp6) among coronavirus nsps and bioinformatic analyses consistently predict four transmembrane domains in nsp4 (Clementz et al., 2008 &#9654;; Oostra et al., 2007 &#9654;). An N-terminal transmembrane region (amino acids 1&#8211;30) is presumably followed by a large lumenal domain (amino acids 30&#8211;280), three closely spaced additional transmembrane regions (amino acids 280&#8211;400) and finally a C-terminal domain of about 100 residues that is exposed at the cytoplasmic face of the membrane. Coronavirus infection induces the extensive reorganization of endoplasmic reticulum membranes into a reticulovesicular network (Knoops et al., 2008 &#9654;) that includes many unusual double-membrane vesicles (Gosert et al., 2002 &#9654;; Harcourt et al., 2004 &#9654;; Shi et al., 1999 &#9654;; Snijder et al., 2006 &#9654;; Stertz et al., 2007 &#9654;). It is currently believed that nsp4 functions in anchoring the viral replication&#8211;transcription complex (RTC) to these modified membranes and independent genetic studies have demonstrated its importance for replication (Clementz et al., 2008 &#9654;; Sparks et al., 2007 &#9654;).In this paper, we present the first X-ray structure of the C-&#173;terminal domain of the FCoV nsp4. Together with structural data, a family-wide comparative sequence analysis of the nsp4 C-terminal domain was performed in order to identify residues/regions that might be important for function rather than for structural integrity.\nExperimental procedures\n\nLibrary-based construct screening\nThe sequence encoding the FCoV nsp4 (residues 2337&#8211;2826 of the polyprotein pp1a from strain FIPV WSU-79/1146; Genebank/RefSeq accession No. NC_007025.1) was RT-PCR amplified from viral RNA and cloned into the pMM8 vector. pMM8 is a modified pET-43 (Novagen) bacterial expression vector containing restriction sites suitable for exonuclease-based construct-library generation (Cornvik et al., 2006 &#9654;) and a Gateway cassette for recombination cloning inserted downstream of the His-tag coding sequence. An N-terminally deleted construct library was generated using an exonuclease strategy and screened for a soluble construct using the colony-filtration blot (Cornvik et al., 2005 &#9654;, 2006 &#9654;). A soluble and well expressing construct containing residues 2731&#8211;2826 (here called the nsp4ct domain) was chosen for scale-up expression and purification. This construct has 14 additional N-terminal residues, including a noncleavable His6 tag.\nExpression\nThe expression of soluble nsp4ct was performed in Escherichia coli strain BL21 (DE3) (Novagen). Cultures were grown at 310&#8197;K in LB medium containing 50&#8197;&#181;g&#8197;ml&#8722;1 ampicillin until the OD600 reached 0.8. Protein synthesis was induced by the addition of 1&#8197;mM isopropyl &#946;-d-1-thio&#173;galactopyranoside (IPTG) and the culture was grown to stationary phase overnight at 288&#8197;K. Cells were harvested by centrifugation at 4000g (30&#8197;min, 277&#8197;K) and frozen at 253&#8197;K. Selenomethionine-substituted nsp4ct was expressed in the non-methionine auxotrophic E. coli strain BL21 (DE3) (Novagen). Bacteria were grown in minimal medium at 310&#8197;K until the OD600 reached 0.8. Feedback-inhibition amino-acid mix (Lys, Thr, Phe, Leu, Ile, Val and SeMet) was added and after 15&#8197;min cells were induced with 1&#8197;mM IPTG. The culture was left shaking overnight at 288&#8197;K and the cells were subsequently harvested by centrifugation at 4000g (30&#8197;min, 277&#8197;K). Cell pellets were frozen at 253&#8197;K.\nPurification\nBoth the native and the SeMet-substituted nsp4ct proteins were purified following the same protocol. A pellet from 1&#8197;l cell culture was resuspended in 20&#8197;ml buffer A (10&#8197;mM CHES pH 9.1 and 300&#8197;mM NaCl, plus 2&#8197;mM &#946;-mercaptoethanol in the case of SeMet-substituted nsp4ct). The cells were sonicated and the protein was purified from the soluble cellular fraction by Ni&#8211;NTA affinity chromatography and eluted with buffer A containing 500&#8197;mM imidazole. The eluate was buffer-exchanged into buffer A using PD10 columns (GE Healthcare Life Sciences). nsp4ct was subsequently concentrated and applied onto a Superdex 75 (16/60) gel-filtration column (GE Healthcare Life Sciences) pre-equilibrated with buffer A. The protein was concentrated to 10&#8197;mg&#8197;ml&#8722;1 and its purity was examined by SDS&#8211;PAGE.\nCrystallization\nInitial crystallization trials were carried out using the sitting-drop vapour-diffusion method in 96-well plates (Greiner) at 292&#8197;K at the EMBL Hamburg High-throughput Crystallization Facility (Mueller-Dieckmann, 2006 &#9654;). Crystals were obtained under various conditions. Further optimization of these conditions was performed manually in 24-well plates (Qiagen) using the hanging-drop vapour-diffusion method at 292&#8197;K. Crystals were obtained at a protein concentration of 7&#8197;mg&#8197;ml&#8722;1 in 0.22&#8197;M ammonium sulfate and 25%(w/v) PEG 5000.\nData collection and processing\nThe crystals were cryoprotected in a solution consisting of 0.22&#8197;M ammonium sulfate, 25%(w/v) PEG 5000 and 15%(v/v) ethylene glycol prior to data collection. Three data sets were collected: two single-wavelength native data sets (data sets 1 and 2) and a single-wavelength anomalous diffraction (SAD) data set (at peak wavelength; data set 3). Data set 1 was collected from a single crystal at 100&#8197;K on the European Synchrotron Radiation Facility (ESRF) beamline ID23-2 using a MAR 225 CCD detector. The oscillation range was 1&#176;, with a crystal-to-detector distance of 346.2&#8197;mm. 90 images were collected to a maximum resolution of 3.1&#8197;&#197;. Data set 2 was collected from a single crystal at 100&#8197;K on the EMBL beamline X12 at DESY using a MAR 225 detector. The crystal-to-detector distance was 300&#8197;mm, with an oscillation range of 0.25&#176;. A total of 670 images were collected to a maximum resolution of 2.76&#8197;&#197;. Data set 3 was also collected from a single crystal on beamline X12 (EMBL Hamburg). The crystal-to-detector distance was 280&#8197;mm and the oscillation range was 1&#176;. 200 images at the selenium absorption edge were collected to a maximum resolution of 3.3&#8197;&#197;.In all three cases the recorded images were processed with XDS (Kabsch, 1988 &#9654;) and the reflection intensities were processed with COMBAT and scaled with SCALA (Evans, 1993 &#9654;) from the CCP4 program suite (Collaborative Computational Project, Number 4, 1994 &#9654;). Data-collection statistics are shown in Table 1 &#9654;.\nStructure determination\nThe structure was solved using the SIRAS protocol of the Auto-Rickshaw automated crystal structure-determination platform (Panjikar et al., 2005 &#9654;). F\n               A values were calculated using the program SHELXC (Sheldrick, 2008 &#9654;). Based on an initial analysis of the data, the maximum resolution for substructure determination and initial phase calculation was set to 3.8&#8197;&#197;. 20 selenium positions were found using the program SHELXD (Sheldrick, 2008 &#9654;). The correct hand of the substructure was determined using the programs ABS (Hao, 2004 &#9654;) and SHELXE (Sheldrick, 2008 &#9654;). The occupancy of all substructure atoms was refined using the program BP3 (Pannu et al., 2003 &#9654;; Pannu & Read, 2004 &#9654;). The initial phases were improved using density modification, noncrystallographic symmetry (NCS) averaging and phase extension using the program RESOLVE (Terwilliger, 2000 &#9654;). A partial &#945;-helical model was produced using the program HELICAP (Morris et al., 2004 &#9654;). The partial model contained 119 of the total of 440 residues expected for four molecules. The initial phases were improved by phase combination of experimental and model phases using the program SIGMAA (Read, 1986 &#9654;). The density modification and fourfold NCS averaging were repeated again as described above. The resultant phases were used to continue model building using the program ARP/wARP (Perrakis et al., 1999 &#9654;), resulting in the placement of 242 residues. The partial models generated in the intermediate steps of ARP/wARP were then used to assemble an almost complete dimer using the graphics program Coot (Emsley & Cowtan, 2004 &#9654;). This dimer was then used to find the second dimer in the electron density using phased molecular-replacement techniques as implemented in MOLREP (Vagin & Teplyakov, 1997 &#9654;). 2F\n               o &#8722; F\n               c and F\n               o &#8722; F\n               c electron-density maps calculated at this stage showed additional electron density indicating the presence of a fifth molecule in the asymmetric unit. The phased molecular replacement was repeated again to place the fifth molecule in the electron-density map. The resultant model was then used for restrained refinement in REFMAC5 (Murshudov et al., 1997 &#9654;), including use of the translation, libration and screw method (TLS; Schomaker & Trueblood, 1968 &#9654;) for describing group motions.The structure was manually modified, followed by cycles of refinement, using the program Coot. The progress of the refinement was monitored by means of the free R factor (Br&#252;nger, 1992 &#9654;). Water molecules were included where clear peaks were present in both the 2F\n               o &#8722; F\n               c and F\n               o &#8722; F\n               c maps and where appropriate hydrogen bonds could be made to surrounding residues or to other water molecules. The stereochemistry of the model was evaluated with the program MOLPROBITY (Davis et al., 2007 &#9654;).Interfaces between molecules were analyzed with the PISA server (Krissinel & Henrick, 2007 &#9654;). Interactions between molecules were initially evaluated using the CCP4 program CONTACT with a maximum contact distance of 3.6&#8197;&#197;.\nResults and discussion\n\nStructure determination\nThe recombinant His6-tagged FCoV nsp4ct domain (residues 2731&#8211;2826 of pp1a and residues 395&#8211;490 of nsp4; here renumbered as 1&#8211;96) was expressed in E. coli. The protein was also expressed with the substitution of methionine by seleno&#173;methionine (SeMet). The incorporation of SeMet was verified by matrix-assisted laser desorption ionization (MALDI) mass spectrometry. Both native and SeMet-substituted proteins were crystallized from conditions containing ammonium sulfate and PEG 5000. The crystals belonged to space group P43, with unit-cell parameters a = b = 127.5, c&#160;=&#160;42.8&#8197;&#197; (data set 2 in Table 1 &#9654;). There are five molecules (chains A&#8211;E) in the asymmetric unit, which corresponds to a 64% solvent content (Matthews, 1968 &#9654;). The structure was refined at 2.8&#8197;&#197; resolution to a final R value of 24.0% (R\n               free = 29.9%). The final model contains 96 residues in molecule A (residues 0&#8211;95), 93 residues in molecule B (residues 0&#8211;49 and 53&#8211;95), 92 residues in molecule C (residues 0&#8211;91), 91 residues in molecule D (residues 1&#8211;91) and 84 residues in molecule E (residues 0&#8211;49 and 56&#8211;89). 88.0% of the residues are located in the preferred regions of the Ramachandran diagram and 10.9% are in allowed regions. Residues Met55 (chains A and B), Glu57 (chains C and D) and Ala58 (chains A&#8211;E) are Ramachandran outliers. Glu57 and Ala58 are located in the N-&#173;terminus of helix &#945;3. The geometry in this region may be influenced by hydrogen bonding between Glu57&#8197;O&#8714; and Arg61&#8197;N&#951;. The refined structure contains two sulfate ions and 40 solvent molecules. A detailed summary of the data-collection and structure-refinement statistics is given in Tables 1 &#9654; and 2 &#9654;.\nOverall structure\nThe FCoV nsp4ct structure contains six short &#946;-strands &#946;1&#8211;&#173;&#946;6 and four &#945;-helices &#945;1&#8211;&#945;4 (Fig. 1 &#9654;). Strands &#946;1 and &#946;2 and&#160;strands &#946;3 and &#946;5 form small two-stranded antiparallel sheets. Strands &#946;4 and &#946;6 participate in the formation of the dimer interface. Strand &#946;4 is observed in molecules C and D and strand &#946;6 is observed in molecules A&#8211;D. The characteristic feature of the structure is the 21-residue-long helix &#945;4.&#160;Analysis using EBI web tools (PDBsum/ProFunc, Catalytic&#160;site search; http://www.ebi.ac.uk), DALI (http://ekhidna.biocenter.helsinki.fi/dali_server/) and GRATH (http://protein.hbu.cn/cath/cathwww.biochem.ucl.ac.uk/cgi-bin/cath/Grath.html) for both the monomer and dimer did not result in any significant indicators of function or similarities in structure. The structure therefore represents a novel protein fold. Nsp4ct has nine conserved and two nonconserved hydrophobic residues (Fig. 2 &#9654;) which form the hydrophobic core of the structure. These residues are grouped into a mainly aliphatic group (Phe19, Ile21, Leu29, Ile38, Leu68, Leu72) and two aromatic groups (Phe11, Tyr41, Tyr65 and Tyr26, Tyr75) (Fig. 3 &#9654;).The r.m.s.d. between C&#945; atoms of molecules A&#8211;E is less than 0.1&#8197;&#197; (for the 63 common C&#945; atoms). Differences between molecules are present in the N-terminus (residues 0&#8211;4), the C-&#173;terminus (from residue 88 onwards) and the region between residues 46 and 64, which includes the C-terminal part of helix &#945;3, the flexible loop L&#945;3&#8211;&#945;4 and the N-terminus of helix &#945;4.\nDimer interface\nThe FCoV nsp4ct crystal contains five molecules in the asymmetric unit. Molecules A/C and B/D form very similar dimers (Fig. 4 &#9654;\n               a), each of approximate dimensions 60 &#215; 20 &#215; 20&#8197;&#197;. The average buried surface area per molecule is approximately 961&#8197;&#197;2. The buried surface of each dimer involves approximately 25 residues from &#945;1, &#945;3, &#945;4, loop L&#945;3&#8211;&#945;4 and the C-terminus. Interestingly, the interaction interface contains an intramolecular three-stranded antiparallel &#946;-&#173;sheet. The order of the strands in this sheet is &#946;4C-&#946;6A-&#946;6C in the case of dimer A/C and &#946;4D-&#946;6B-&#946;6D in the case of dimer B/D. Strands &#946;4C and &#946;4D include residues 51&#8211;53, strands &#946;6A and &#946;6B include residues 89&#8211;92 and strands &#946;6C and &#946;6D include residues 88&#8211;90. The major interactions at this interface are the &#946;-sheet hydrogen bonds Val89&#8197;N&#8943;Gly53&#8197;O, Val89&#8197;O&#8943;Gly53&#8197;N, Ser90&#8197;N&#8943;Ser90&#8197;O, Ser90&#8197;O&#8943;Ser90&#8197;N, Val91&#8197;N&#8943;Tyr51&#8197;O, Asn92&#8197;N&#8943;Tyr88&#8197;O and Asn92&#8197;O&#8943;Tyr88&#8197;N. Five hydrogen bonds located outside the &#946;-sheet region are formed: Met55&#8197;O&#8943;Tyr60&#8197;O&#951;, Tyr60&#8197;O&#951;&#8943;Met55&#8197;O, Tyr60&#8197;O&#951;&#8943;Met55&#8197;N, Thr94&#8197;O&#947;&#8943;Thr85&#8197;O and Thr94&#8197;O&#947;&#8943;Thr85&#8197;O&#947; (Fig. 4 &#9654;\n               b). Strong van der Waals contacts between residues in the dimer buried area are also important in defining the interface.The results obtained from analytical size-exclusion chromatography of nsp4ct demonstrated that the protein is monomeric in solution under the experimental conditions used (results not shown). Furthermore, the crystal structure con&#173;tains a monomer as well as two dimers. The buried surface area supports the likelihood of dimerization and this may have physiological significance. It is conceivable that in vivo nsp4 dimerization during membrane modification or formation of the RTC may help in bringing the components together and could therefore aid their correct spatial orientation. This would agree with the previously proposed role of nsp4 as an anchor for the assembly of the viral RTC.\nNsp4ct sequence alignment\nFig. 2 &#9654; shows the sequence alignment, produced with ClustalW2 (Larkin et al., 2007 &#9654;), of the C-terminal domain of nsp4 for the five coronaviral subgroups. These viruses are FCoV (group 1a), human coronavirus NL63 (HCoV-NL63; group 1b), murine hepatitis virus (MHV; group 2a), SARS-CoV (group 2b) and infectious bronchitis virus (IBV; group 3). The nsp4ct sequence identity between viruses belonging to the same group (but different subgroups) is higher than that for viruses belonging to different groups. The sequence identity between FCoV and HCoV-NL63 is 68% and that between MHV and SARS-CoV is 53%. IBV, on the other hand, is the most distantly related virus and its sequence identity in all possible combinations with the other viruses is around 35%. This is consistent with previously published phylogenetic analyses of coronaviruses (Gorbalenya et al., 2004 &#9654;). The sequence alignment shows a high level of conservation of the nsp4ct domain, with 17 of around 100 residues being identical between all five subgroups. Most of the aromatic amino acids of coronavirus nsp4ct are highly conserved. This includes residues Phe19, Tyr41, Tyr50, Tyr60 and Tyr84, which are fully conserved, and Phe11 (Tyr in IBV), Tyr26 (Phe in IBV), Phe45 (Tyr in four other coronaviruses) and Tyr51 and Tyr75 (both Phe in MHV and SARS-CoV). Interestingly, Phe45, Tyr50, Tyr51, Tyr60 and Tyr84 are part of the FCoV nsp4ct dimer interface and the fully conserved Tyr60 forms a side-chain (O&#951;) hydrogen-bond interaction with the main-chain carbonyl of Met55 from the second monomer. The two fully conserved C-terminal residues (Leu95 and Gln96) are part of the recognition site for the coronavirus Mpro (Hegyi & Ziebuhr, 2002 &#9654;).There are four clusters of highly conserved residues. The first is between residues 9 and 19 and includes residues in helix &#945;1 and strand &#946;3. The second comprises residues 45&#8211;53 that belong to helix &#945;3 and part of loop L&#945;3&#8211;&#945;4. Interestingly, the five independent chains of the FCoV nsp4ct structure differ most profoundly in this region, suggesting that it is highly flexible. In the cases of molecules B and E it was disordered and there was no electron density visible for residues 50&#8211;52 and 50&#8211;55, respectively. In molecules A, C and D this region could be placed into electron density and is involved in dimer formation. High sequence conservation of this cluster and its structural flexibility suggests that it may play an important role in the nsp4ct domain function. Residues 60&#8211;71 that belong to helix &#945;4 form the third highly conserved cluster and the fourth cluster consists of the C-terminal residues 81&#8211;96. This last cluster contains the highly conserved Tyr84, Pro86 and Pro87 which form the YxPP motif, which is the inverse of the consensus PPxY sequence recognized by the class I WW domains (Linn et al., 1997 &#9654;). Di Leva et al. (2006 &#9654;) showed that the class I WW domain does not require a peptide with a consensus sequence and can also bind an inverted peptide sequence. The only condition is the presence of the polyproline II (PPII) con&#173;formation, which is observed in the case of FCoV nsp4ct. This suggests that region 84&#8211;87 is a reasonable candidate for protein&#8211;protein interactions. PROSITE (http://www.expasy.org/prosite/) analysis of all FCoV nsps did not identify any possible WW domains, suggesting that the YxPP motif interacting partner is a host protein. Furthermore, localization of the Pro&#8211;Pro motif may protect the extended unstructured C-terminus from proteolytic cleavage by host enzymes (Vanhoof et al., 1995 &#9654;).\nConclusions\nThe high conservation of the C-terminal domain of nsp4 suggests not only that it plays a ubiquitous role in the coronavirus life cycle, but also that nsp4 proteins from different subgroups are structurally similar and have similar modes of operation. In this context, it is a surprising finding that deletion of the nsp4ct of MHV (using a reverse genetics system) was reported to be tolerated by the virus (Clementz et al., 2008 &#9654;; Sparks et al., 2007 &#9654;), with the resulting mutant displaying a modestly attenuated phenotype. Thus, although a similar mutant has not been generated for FCoV or any other coronavirus, the conservation of the nsp4ct domain outlined above would suggest that it is not absolutely required for coronavirus RNA synthesis and/or RTC formation per se. This opens the possibility that, like some other recently characterized coronavirus enzyme functions (Eriksson et al., 2008 &#9654;; Roth-Cross et al., 2009 &#9654;), the nsp4ct domain might play a role in specific virus&#8211;host interactions of the type that are not easily uncovered in cell culture-based systems for virus propagation. Further functional studies are required in order to better understand the detailed role of nsp4 and to identify its partners and therefore its significance in the viral life cycle.\nSupplementary Material\n",
        "pmid": "19622868",
        "title": "Structure of the C-terminal domain of nsp4 from feline coronavirus",
        "journal_title": "Acta Crystallographica Section D: Biological Crystallography",
        "authors": "Ioannis Manolaridis;Justyna A. Wojdyla;Santosh Panjikar;Eric J. Snijder;Alexander E. Gorbalenya;Hanna Berglind;P\u00e4r Nordlund;Bruno Coutard;Paul A. Tucker",
        "doi": "10.1107/S0907444909018253"
    },
    {
        "keywords": "Canine coronavirus;Canine distemper virus;Canine parvovirus;Cape verde;Molecular surveillance",
        "abstract": "Background\nInfections caused by canine parvovirus, canine distemper virus and canine coronavirus are an important cause of mortality and morbidity in dogs worldwide. Prior to this study, no information was available concerning the incidence and prevalence of these viruses in Cape Verde archipelago.Results\nTo provide information regarding the health status of the canine population in Vila do Maio, Maio Island, Cape Verde, 53 rectal swabs were collected from 53 stray dogs during 2010 and 93 rectal swabs and 88 blood samples were collected from 125 stray dogs in 2011. All rectal swabs (2010 n&#8201;=&#8201;53; 2011 n&#8201;=&#8201;93) were analysed for the presence of canine parvovirus, canine distemper virus and canine coronavirus nucleic acids by quantitative PCR methods. Specific antibodies against canine distemper virus and canine parvovirus were also assessed (2011 n&#8201;=&#8201;88).From the 2010 sampling, 43.3% (23/53) were positive for canine parvovirus DNA, 11.3% (6/53) for canine distemper virus RNA and 1.9% (1/53) for canine coronavirus RNA. In 2011, the prevalence values for canine parvovirus and canine coronavirus were quite similar to those from the previous year, respectively 44.1% (41/93), and 1.1% (1/93), but canine distemper virus was not detected in any of the samples analysed (0%, 0/93). Antibodies against canine parvovirus were detected in 71.6% (63/88) blood samples and the seroprevalence found for canine distemper virus was 51.1% (45/88).Conclusions\nThis study discloses the data obtained in a molecular and serological epidemiological surveillance carried out in urban populations of stray and domestic animals. Virus transmission and spreading occurs easily in large dog populations leading to high mortality rates particularly in unvaccinated susceptible animals. In addition, these animals can act as disease reservoirs for wild animal populations by occasional contact. Identification of susceptible wildlife of Maio Island is of upmost importance to evaluate the risk of pathogen spill over from domestic to wild animals in Cape Verde and to evaluate the associated threat to the wild susceptible species.",
        "body": "Background\nOver the past few years, efforts have been made towards a better understanding of the health status of animal populations, particularly regarding viral infections. Due to their high mutation rate and replication strategies, viruses are responsible for recently recognized emerging diseases, posing a danger not only to domestic and wild animals, but also to humans [1,2].The high density of domestic and stray animals in urban areas enables viral dissemination and maintenance in these populations. Consequently, these animals can act as reservoirs of diseases, with the possibility of transmission to wildlife populations through occasional contact.Canine parvovirus (CPV) was first identified in the late 1970s and was responsible for severe hemorrhagic gastroenteritis and myocarditis in dogs [3]. Parvoviruses are extremely stable in the environment and indirect transmission assumes a critical role in spreading and maintenance of the virus in animal populations, especially in wild carnivores, in which contact rates between animals are lower [4]. Shortly after its initial detection, CPV-2 was replaced by two antigenic variants, CPV-2a and CPV-2b and more recently a third variant was described CPV-2c [5,6].Canine distemper virus (CDV) is the etiological agent of canine distemper, a highly contagious disease, responsible for high mortality rates in dogs worldwide. Sequence analysis of CDV strains originated in different geographical areas from several animal species, showed that the hemagglutinin gene has undergone a genetic drift according to the geographic location [7,8]. Phylogenetic analysis based on this gene revealed the existence of at least nine strains in different geographical areas, namely America-1, America-2, Asia-1, Asia-2, Europe-1/South America 1, European wildlife, Arctic-like, South America 2 and Southern Africa [9,10].Canine coronavirus (CCoV) causes a mild to moderate enteritis in dogs and its infection is characterized by high morbidity and low mortality. CCoV is transmitted by faecal-oral route and spreads rapidly through a group of susceptible animals [11]. Stressful environments with large concentrations of animals and poor hygienic conditions, often seen in kennels, favour the development of this disease [12]. Although a higher mortality rate is observed in animals with multiple infections with other pathogens such as CPV-2, canine adenovirus type 1 and CDV, CCoV represents per si a major infectious agent responsible for several epidemics [13,14].Virological surveys are conducted throughout the world, allowing the detection and analysis of a large variety of viruses in different animal populations. In Cape Verde archipelago to our knowledge, no similar study had been conducted so far. In order to detect the presence of canine viruses on Maio island, samples collected from stray dogs from Vila do Maio were tested for canine parvovirus (CPV), canine distemper virus (CDV) and canine coronavirus (CCoV), to estimate the viral prevalence in this population and investigate the role of these animals in the maintenance and potential spread of common viral pathogens.\nResults\nRecords were only available for the specimens sampled in 2011. Of the 125 dogs, all them of undetermined or mixed breeds, 65 were females (52%) and 57 males (46%). For 3 dogs (2%) no data was registered regarding gender.Diaharreic feaces were described for 4 animals (3%).Only two dogs had been vaccinated, both with Tetradog&#174; vaccine and no information regarding vaccination of the rest of the animals was available (NA).The percentage of positivity for CPV-DNA was very similar in the 2010 and 2011 sampling; 23/53 (43.3%) and 41/93 (44.1%, respectively). From the 88 sera sampling collected during 2011, 63 (71.6%) tested positive for CPV antibodies, with 10 animals included in the first ELISA Unit (EU) class (100&#8211;1000 EU), 29 in the second EU class (1000&#8211;10000 EU) and 24 in the third EU class (>10000 EU) (Tables&#160;1 and 2).Antibodies against CPV were detected in 20% of the animals aged less than 6&#160;months (2/10), in 57.1% in dogs aged between 6&#160;months and 1&#160;year (8/14), in 87.5% in dogs with 1 to 2&#160;years (14/16), in 85.3% in dogs with 2 to 5&#160;years (29/34), in 75% in dogs with 5 to 7&#160;years (6/8) and in 1/1 dog older than 7&#160;years (Figure&#160;1). The proportion of seropositive animals was significantly higher in older animals (p&#8201;&#8201;0.05). From the 56 samples that were tested for virology and serology, 46.6% (26/56) were positive for CPV-DNA and 64.3% (36/56) were seropositive. Out of the 26 dogs that were excreting the virus at the time of collection, 7 were seronegative for CPV specific IgG.Regarding the 2010 samples tested for CDV-RNA (n&#8201;= 53), 6 animals were positive (11.3%), of which 2 were also co-infected with CPV. All samples from 2011 were found CDV-RNA negative. As for serology, 45 of the 88 animals sampled during 2011 were seropositive for CDV (51.1%). Two groups were identified according to the antibody (Ab) titer: 1) low Ab titer (IIF values 1/20-1/40: n&#8201;=&#8201;43 (96%)); and 2) medium Ab titer (IIF values 1/80-1/160: n&#8201;=&#8201;2 (4%)).Antibodies against CDV were detected in 30% of animals aged less than 6&#160;months (3/10), in 50% of dogs aged between 6&#160;months and 1&#160;year (7/14), in 56.3% of dogs with 1 to 2&#160;years (9/16), in 53% of dogs with 2 to 5&#160;years (18/34), in 75% in dogs with 5 to 7&#160;years (6/8) and in 1/1 dog older than 7&#160;years (2011 samples) (Figure&#160;2). Although there was a linear increase of seropositive animals with age, this association was not statistically significant (p&#8201;>&#8201;0.05). The presence of antibodies was independent of gender (p&#8201;>&#8201;0.05).Only two samples, collected in each year of the survey, tested positive for CCoV-RNA, (2010 (1/53, 1.9%) and 2011(1/93,1.1%)).\nDiscussion\nThis study describes for the first time, the shedding of three common enteric canine viruses, CPV, CDV and CCoV, in 178 stray dogs from Vila do Maio, Cape Verde and reports data on CPV and CDV seroprevalence.Samples were collected in two consecutive years, 2010 and 2011. Similar frequency of positive CPV animals was found, namely 43.3% in 2010 and 44.1% in 2011, most probably reflecting the high environmental resistance of CPV. Viral transmission and spread of CPV can occur easily, without direct contact between animals. The virus is shed at high titres in faeces and the excretion period may last longer, allowing a higher opportunity for contact between the virus and the new hosts. Although most of the positive dogs did not show signals of diarrhoea, recovered animals may also serve as asymptomatic reservoirs and shed virus periodically, contributing for the persistence and continuous circulation of CPV in the environment, as already reported for cats [15].Also, the CPV resistance to environmental conditions may explain the high seroprevalence obtained (71.6%), similar to that reported in studies conducted with non-vaccinated dog populations [16,17]. The vast majority of the positive samples presented a high antibody titer (n&#8201;= 53) (Table&#160;1 and 2), even though it could not be correlated with hemagglutination inhibition values, the gold standard assay for titration of CPV antibodies, however we did not had the possibility to perform this assay. This observation suggests the persistence of the virus in the environment, which is in accordance with the high percentage of carrier animals found (44.1%). Within the 56 animals from which both sample types were obtained, CPV-DNA was detected in 46.6% and seropositivity in 64.3%.Only 7 of the 26 dogs shedding the virus had no circulating antibodies. It is possible that these animals were at an early stage of infection. Primary IgMs were not investigated by the ELISA kit used in this study, which is specific for IgG detection.The seroprevalence for CPV was significantly higher in older animals (p&#8201;<&#8201;0.05), most probably reflecting a high likelihood of virus exposure over time. Moreover, as younger dogs are more susceptible to the virus, they can succumb to the disease and therefore be removed from the population. Progressive evolution of the CPV-2 led to the emergence of three antigenic variants, 2a, 2b and 2c, with different properties from the original strain [5]. Monitoring the prevalence of the different CPV in Cape Verde archipelago would be important, not only to assess the distribution of viral variants in this geographic location, but also to understand the evolutionary pattern of the virus in this circunscripted population. In addition, given the high nucleotide substitution rate of CPV, similar to the RNA viruses [18] and the terrain characteristics of Cape Verde as an island its possible transportation through people, goods and infected animals, should be considered.In 2010 the prevalence of CDV detected by real-time PCR was 11.3% while in 2011 all the samples were CDV-RNA negative. Interestingly, in this year, the seroprevalence obtained for this virus was 51.1%. As these animals were non-vaccinated the explanation for the seropositivity may lie with a previous contact with the virus through ill animals, which could have occurred the previous year.Several factors may explain the absence of CDV shedding in faeces of dogs sampled in 2011. Taking into account the CDV poor resistance to dry and hot environments, it is possible that the virus did not resist the Cape Verde summer high temperatures, reducing the opportunity for dogs being exposed to the virus [19,20]. In addition, the high mortality rates caused by CDV contribute to the low virus spread in canine populations since the animals that succumbed to infection stop shedding. On the other hand, as the animal density of this city is low compared to larger cities, less contact rates between animals affect the virus maintenance and spread [21]. Still, 51.1% of the animals had anti-CDV antibodies. Comparing the presence of antibodies with the age of the dogs a linear increase in seroprevalence with age was found, which may be related to a possible past CDV outbreak in the population.Although it is not statistically significant, the higher seroprevalence in older dogs has already been reported [21]. The presence of CDV antibodies was not associated with gender.All CDV seropositive animals (51.1%) showed a protective antibody titer, according to Twark and Dodds [22], by establishing a comparison between serum neutralization and IIF assays. Although 96% of these animals had low antibody titers (Table&#160;1 and 2), Schultz et al. [23] reported that in actively immunized dogs, either naturally or through vaccination, the antibody titer is not very relevant, provided that is detectable. Nevertheless, and despite the good sensitivity and specificity of ELISA, it would be advisable to test the samples using the serum neutralization test to confirm the presence and titer of neutralizing antibodies, to fully assess the immune status of these animal populations.Relating to the genetic variability of CDV, it would interesting to identify the different CDV genotypes in positive samples, in order to differentiate between vaccine and field strains, and determine which lineages circulate on this island.Regarding CCoV, we found a low prevalence of 1.9% in 2010 and 1.1% in 2011, which is in agreement with similar surveys conducted in areas with similar characteristics of Vila do Maio dog population [24,25]. The low prevalence may indicate a low viral circulation, probably due to the virus instability in normal environmental conditions and also to the reduced number viral particles in faeces. As for CPV, dogs recovered from a CCoV infection may function as asymptomatic reservoirs and shed the virus periodically, resulting in a persistent and continued circulation of CCoV in the environment. Available data concerning the epidemiological mechanisms of CCoV suggests that the environment provided by kennels can be fundamental in maintaining this infection in canine populations [25-28].Seroprevalence against CCoV was not performed due to the rapid decay of antibodies caused by natural exposure [29]. Moreover the available serological assays are based on the identification of antibodies against CCoV-II, and their efficacy is unknown for the detection of CCoV-I antibodies [30], questioning the usefulness of these methods.\nConclusion\nThe results presented in this study demonstrate that CPV, CDV and CCoV are circulating in the canine population of Vila do Maio, Cape Verde.The presence of susceptible animals, the high frequency of infections, the prolonged period of virus shedding and environmental persistence of these agents, especially of CPV, contribute to their continuous circulation in this population. Thus, information regarding the spatial distribution of circulating viruses and the risk factors associated with infections will definitely facilitate the planning of control strategies. To our knowledge no vaccination program is undertaken in this region and its implementation could contribute to increase the immunity of this population, reduce viral circulation, and consequently a decrease the population susceptibility to a future disease outbreaks. Additionally, the absence of control measures may increase the risk of pathogen spill over, either for susceptible newcomers&#8217; hosts or for resident susceptible new hosts as sympatric carnivores&#8217; species.Due to the wide range of CPV and CDV susceptible hosts, it would be important to identify Maio Island wildlife, to assess the potential risk of infection of these species.\nMethods\n\nStudy population and sampling\nSampling was conducted in Vila do Maio, located on Maio Island, Cape Verde, under a neutering and health surveillance program developed by Veterinarians without Frontiers, Portugal (VSF), in two distinct periods: 2010 with collection of rectal swabs from 53 animals; and 2011, including blood samples (n&#8201;=&#8201;88) and rectal swabs (n&#8201;=&#8201;93) from 125 animals (Table&#160;3). Biological samples were stored under refrigeration until processing.\nSample processing\nRectal swabs were homogenized in 300&#160;&#956;l of PBS. After centrifugation at 10000xg/10&#160;min, the supernatant was collected and stored at &#8722;80&#176;C. Blood samples were centrifuged at 4000xg/10&#160;min to separate plasma from blood cells. Plasma was stored at &#8722;20&#176;C until processing.\nPCR assay procedures\nFor viral nucleic acid extraction, 200&#160;&#956;l of supernatant from each rectal swab, were processed with the Qiamp Minelute kit&#174; (Qiagen, Germany) according to the manufacturer's instructions, for viral DNA and RNA co-extraction. Although using a commercial kit for co- extraction of viral nucleic acids, from a non-recommended biological matrix may imply a reduced nucleic acid yield, it is sufficient in their experience [31,32].Detection of viral nucleic acids using real-time PCR (qPCR) and real-time rt-PCR (rt qPCR) CPV DNA was amplified by qPCR using TaqMan&#174; Gene Expression 2&#215; Master Mix (Applied Biosystems); CDV and CCoV RNA were amplified by rt-qPCR using the TaqMan&#174; RNA-to-Ct(TM) 1 step kit in a 20&#160;&#956;l reaction with 50&#160;ng of template.Primers and TaqMan&#174; probes were calculated using the Primer designing tool of NCBI (http://www.ncbi.nlm.nih.gov/tools/primer-blast/). For CPV, primers were based on the nucleotide sequence of the vp1 gene, available through its access number (AN) AB437433.1. CDV primers were chosen within the nucleocapsid gene (AN JN896987.1) [7] and CCoV primers targeted the highly conserved 7b gene (AN JQ404410.1) as already described by [33] (Table&#160;4). A final concentration of 900 nM for the forward primer, 900 nM of reverse primer and 250 nM of each TaqMan&#174; probe was used (Table&#160;4).The amplification was performed in the StepOne Plus thermocycler (Applied Biosystems) and the cycling conditions comprised an initial denaturation step at 95&#176;C for 10&#160;minutes, followed by 40&#160;cycles at 95&#176;C for 15&#160;seconds and 1&#160;minute at 60&#176;C. When the template was RNA the amplification cycle included a reverse transcription step at 48&#176;C for 15&#160;minutes.For CPV tenfold dilutions of CPV-2-780 916 Cornell strain (Tetradog&#174;, Merial) were used as positive control. Regarding CDV, a 287&#160;bp fragment, including the targeted region was amplified from CDV RNA (Caniffa&#174;, Merial) [7] and cloned in pGEM&#174; Teasy vector (Promega) according to the manufactures instructions. The CDV recombinant plasmid was used as positive control. A similar approach was used for CCoV as already described [34].The assay specificity was confirmed by direct sequencing of the CPV amplicon. For CDV and CCoV sequencing was performed after plasmid cloning. No cross reactivity was detected between CDV/CCoV/CPV. The sensitivity of the rt-qPCR/qPCR for all three agents surpassed the detection of 10 target copies/&#956;l, assessed by conversion of the positive control mass (g/&#956;l) in molecules/&#956;l, based on the following formula: number of copies (molecules/&#956;l)&#8201;=&#8201;[mass (g/&#956;l)/(number of base pairs x bp (660)] x Avogadro's number (6,022 x 1023).\nAntibody detection\nAntibody (Ab) detection was only performed for 2011 serum samples for CPV and CDV using an indirect ELISA kit (Ingezim Canine Parvo 15.CPV.K1&#174; and Ingezim Moquillo 1.5.CDG.K.1&#174;-Ingenasa), for specific IgG detection, according to the manufactures instructions.This method allowed the quantification of the Ab titer for CPV, using a formula provided by the kit, although it did not specify the correspondence with the hemagglutination inhibition assay for quantification of anti-CPV antibodies. The values were organized in three classes: 1) 100&#8211;1000 ELISA Units (EU); 2) 1000&#8211;10000 EU and 3)&#8201;>&#8201;10000 EU.For CDV the OD values had correspondence with the indirect immunofluorescence (IIF) method, and the values were divided into three categories: 1) low titer (IIF values: 1/20-1/40), 2) medium titer (IIF values: 1/80-1/160) and 3) high titer (IIF values: &#8805; 1/320).\nStatistical analysis\nA possible association between serological findings and the age and gender of the animals was determined by chi-square statistical test (&#967;2) with IBM SPSS Statistics 19.0 software. A P value <0.05 was considered significant.\nCompeting interest\nThe authors declare that they have no competing interests.\nAuthors\u2019 contributions\nPC, AD and LT were responsible for the conception and the study design and actively participated in the analysis and data interpretation. PC and AD were also involved in the drafting and revision of the article. SG and CC were involved in the data analysis. MM and SV were responsible for the organization of the field work and sample collection.",
        "pmid": "24755118",
        "title": "Molecular and serological surveillance of canine enteric viruses in stray dogs from Vila do Maio, Cape Verde",
        "journal_title": "BMC Veterinary Research",
        "authors": "Pedro Castanheira;Ana Duarte;Solange Gil;Clara Cartaxeiro;Manuel Malta;Sara Vieira;Luis Tavares",
        "doi": "10.1186/1746-6148-10-91"
    },
    {
        "keywords": "",
        "abstract": "Porcine epidemic diarrhea virus (PEDV) is a coronavirus that infects pigs and can have mortality rates approaching 100% in piglets, causing serious economic impact. The 3C-like protease (3CLpro) is essential for the coronaviral life cycle and is an appealing target for the development of therapeutics. We report the expression, purification, crystallization and 2.10&#8201;&#197; X-ray structure of 3CLpro from PEDV. Analysis of the PEDV 3CLpro structure and comparison to other coronaviral 3CLpro&#8217;s from the same alpha-coronavirus phylogeny shows that the overall structures and active site architectures across 3CLpro&#8217;s are conserved, with the exception of a loop that comprises the protease S2 pocket. We found a known inhibitor of severe acute respiratory syndrome coronavirus (SARS-CoV) 3CLpro, (R)-16, to have inhibitor activity against PEDV 3CLpro, despite that SARS-3CLpro and PEDV 3CLpro share only 45.4% sequence identity. Structural comparison reveals that the majority of residues involved in (R)-16 binding to SARS-3CLpro are conserved in PEDV-3CLpro; however, the sequence variation and positional difference in the loop forming the S2 pocket may account for large observed difference in IC50 values. This work advances our understanding of the subtle, but important, differences in coronaviral 3CLpro architecture and contributes to the broader structural knowledge of coronaviral 3CLpro&#8217;s.",
        "body": "Results and Discussion\nPEDV-3CLpro crystallized in space group P212121 as a dimer in the asymmetric unit (Table 1). As reported for both human coronavirus 229E 3CLpro and SARS-3CLpro, PEDV-3CLpro is a homodimer containing three domains in each monomer (Fig. 1)1112. The active site of PEDV-3CLpro, which contains a catalytic dyad formed from residues Cys144 and His41, is located in a cleft between domains I and II (Fig. 1a). Domain III is involved in monomer dimerization, which is ultimately responsible for forming the active protease (Fig. 1a)1213. In the absence of substrate, water and solvent molecules (MPD, DMSO, and IPA) reside in the active site of PEDV-3CLpro, which is solvent exposed on one side (Fig. 1a,b). Interestingly, different solvent molecules are found in the respective active sites of the PEDV-3CLpro monomers composing the PEDV-3CLpro dimer, where the a single DMSO molecule resides in the Chain A active site and the Chain B active site houses both isopropanol (IPA) and 2-methyl-2,4-pentanediol (MPD) (Fig. 1b). The presence of different solvent molecules in each of the active sites of the dimer supports the assignment of one biological dimer in the asymmetric unit.Least-squares (LSQ) superposition of PEDV-3CLpro and the unbound form of human coronavirus 229E 3CLpro (PDB entry 1P9S)12, which are both from the same alpha-coronavirus phylogenetic group and share 69.3% sequence identity, resulted in an all-atom root-mean-square deviation (RMSD) value of 1.69&#8201;&#197; and a C-alpha RMSD value of 1.28&#8201;&#197; (Fig. 2a). The LSQ superposition of 229E-3CLpro and PEDV-3CLpro shows that the overall architectures of both the 229E-3CLpro and PEDV-3CLpro active sites in their unbound states are structurally very similar, and the active site residues of the catalytic dyad, residues Cys144 and His41 in both PEDV-3CLpro and 229E-3CLpro, are located in almost identical structural space within the active site cavity, which is solvent exposed on one side (Fig. 2b). In the absence of substrate, both water and non-water solvent molecules (dioxane in 229E-3CLpro and MPD, DMSO, and IPA in PEDV-3CLpro) are found in the active site (Fig. 2b).In order to better understand the features of the PEDV-3CLpro active site that are important in inhibitor and substrate binding, we generated an LSQ superposition of PEDV-3CLpro and an inhibitor-bound form of feline infectious peritonitis virus 3CLpro (FIPV-3CLpro, PDB entry 4ZRO), which belongs to the same alpha-coronavirus lineage as PEDV-3CLpro and has 61.9% sequence identity (Fig. 2c)14. LSQ superposition resulted in an all-atom RMSD value of 2.11&#8201;&#197; and a C-alpha RMSD value of 1.69&#8201;&#197;. We found the overall active site architectures of the unbound PEDV-3CLpro and the inhibitor-bound form of FIPV-3CLpro to be remarkably similar with the catalytic dyad residues (Cys144 and His41 in both FIPV- and PEDV-3CLpro) in nearly identical orientations despite Cys144 of FIPV-3CLpro being covalently modified by the inhibitor, compound 6 (Fig. 2d).Interestingly, in both the superimpositions of PEDV-3CLpro with 229E-3CLpro and FIPV-3CLpro, the loops comprising the protease subsites of the 3CLpro active site are in nearly identical structural locations, with the exception of the loop that comprises the S2 subsite, the S2 loop. The S2 loop forms the outer boundary of the S2 binding pocket and shows positional variability across the X-ray structures of 229E-, FIPV-, and PEDV-3CLpro, which may lead to differences in the size of the S2 subsites across 3CLpro&#8217;s (Fig. 2b,d).Our observations of the overall conserved structural features surrounding the PEDV-3CLpro catalytic dyad, but subtle differences in the overall active site architecture, made us curious as to whether one of the inhibitors we developed for SARS-3CLpro would also inhibit PEDV-3CLpro15. SARS-3CLpro belongs to a different phylogenetic lineage than PEDV-3CLpro and shares lower sequence identity (45.4%) with PEDV-3CLpro; however, we reasoned that the similar tertiary structure and conserved active site architecture of 3CLpro&#8217;s would allow for inhibition by the same molecule. We therefore tested the inhibition of PEDV-3CLpro by (R)-16, which was developed as a non-covalent inhibitor against SARS-3CLpro with potential broad-spectrum activity (Fig. 3a)15. We found (R)-16 to inhibit PEDV-3CLpro with an IC50 value of 25.4&#8201;&#177;&#8201;1.4&#8201;&#956;M, where the representative curve for (R)-16 inhibition of PEDV-3CLpro is shown in Fig. 3a and the data for (R)-16 inhibition of SARS-3CLpro has been previously published15. The previously reported IC50 of (R)-16 against SARS-3CLpro is 1.5&#8201;&#177;&#8201;0.3&#8201;&#956;M15, which indicates a ~17-fold weaker interaction of (R)-16 with PEDV-3CLpro. The inhibition of PEDV-3CLpro by (R)-16, though weak, is significant because it indicates that the development of non-covalent broad-spectrum inhibitors of 3CLpro&#8217;s may be possible.To gain structural insights into how (R)-16 may bind to PEDV-3CLpro, a structural alignment of the X-ray structure of SARS-3CLpro:(R)-16 complex with PEDV-3CLpro was generated (Fig. 3b). LSQ superposition of unbound PEDV-3CLpro and inhibitor-bound SARS-3CLpro (PDB entry 3V3M)15, where PEDV-3CLpro is from the alpha-coronavirus phylogenetic group and SARS-3CLpro is from the beta-coronavirus phylogenetic subgroup 2b, resulted in an all-atom RMSD value of 5.18&#8201;&#197; and a C-alpha RMSD value of 4.98&#8201;&#197;. The structural alignment shows that, similarly to 229E-3CLpro and FIPV-3CLpro, the overall active site architectures of SARS-3CLpro and PEDV-3CLpro are largely similar despite their lower sequence identity (Fig. 3b&#8211;d), and the residues directly involved in binding to (R)-16 via hydrogen-bonding interactions are all conserved (Gly142 and His162 in PEDV-3CLpro and Gly143 and His163 in SARS-3CLpro, Fig. 3c,d).The active sites of both SARS-3CLpro and PEDV-3CLpro are solvent exposed on one side with the residues of their catalytic dyads, His41 and Cys144 (for PEDV-3CLpro) or Cys145 (for SARS-3CLpro), located in almost identical structural space (Fig. 3c,d). The superposition shows that (R)-16 similarly occupies the S2-S1&#8242; subpockets in each of the 3CLpro active sites, where the tert-butyl amide resides in the channel leading to the active site, the tert-butylanilido group (P2) sits in the S2 pocket, the 3-pyridyl group (P1) resides in the S1 region, and the tetrahydrofuran (P1&#8242;) occupies the S1&#8242; subsite (Fig. 3c,d). As shown in the SARS-3CLpro 3V3M structure, the 3-pyridyl nitrogen of (R)-16 acts as a hydrogen-bond acceptor for SARS-3CLpro His163, with a distance of 2.8&#8201;&#197; between heteroatoms (Fig. 3d). This interaction is likely conserved in the inhibition of PEDV-3CLpro by (R)-16 as His162 of PEDV-3CLpro is in an almost identical structural location as that of SARS-3CLpro His163. Additionally, the bifurcated interaction between the furan ring oxygen and the amide carbonyl oxygen of (R)-16 and the backbone amide NH of SARS-3CLpro Gly143 is mimicked by PEDV-3CLpro Gly142 (Fig. 3d).Though (R)-16 apparently binds in the same orientation in the PEDV-3CLpro active site as it does in the SARS-3CLpro active site, and likely utilizes the same hydrogen-bonding interactions, the IC50 of (R)-16 against PEDV-3CLpro is 17-fold higher than against SARS-3CLpro. We were therefore curious if the difference in position of the S2 loop between PEDV- and SARS-3CLpro may be important for (R)-16 binding. A structural alignment of the ligand-bound structures of FIPV- and SARS-3CLpro (4ZRO and 3V3M, respectively) shows that compounds 6 and (R)-16 bind to the respective 3CLpro&#8217;s by positioning their sterically bulky, hydrophobic groups in the S2 subsite (leucine and t-butylanilido, respectively; Fig. 4a). This suggests that hydrophobic interactions between the inhibitor or substrate and residues of the S2 loop are important for their binding.We then analyzed the reported X-ray structures and sequences for other alpha-coronavirus 3CLpro&#8217;s in the PDB databank including the enzymes from the following coronaviruses: FIPV, transmissible gastroenteritis virus (TGEV), and human coronaviruses NL63 and 229E121314151617181920. A sequence comparison of the S2 loops of FIPV-, TGEV-, NL63-, 229E-, PEDV-, and SARS-3CLpro shows that the SARS-3CLpro S2 loop is one residue longer than those from the alpha-coronaviruses and shares no sequence identity with FIPV-, TGEV-, NL63-, 229E- and PEDV-3CLpro (Fig. 4b). Additionally, the S2 loop of SARS-3CLpro contains anionic amino acids, Glu and Asp, at positions 47 and 48, respectively. These variations in the S2 loop between the alpha-coronavirus 3CLpro&#8217;s and SARS-3CLpro may explain the large observed difference in the IC50 value of (R)-16 against PEDV-3CLpro as compared to SARS-3CLpro (25.4&#8201;&#177;&#8201;1.4&#8201;&#956;M vs. 1.5&#8201;&#177;&#8201;0.3&#8201;&#956;M, respectively). Furthermore, the combination of the increase in length and charge properties of the S2 loop of SARS-3CLpro relative to that of the alpha-coronavirus 3CLpro&#8217;s likely changes the size and shape of the S2 binding pocket and therefore may allow for increased variability at the P2 site of inhibitor and substrate molecules.To further investigate the variation in the S2-subsite and relationship to inhibitor and substrate binding, we examined the recognition cleavage sequences in polyprotein 1ab of both PEDV- and SARS-3CLpro (Fig. 5)2122. The S2-subsite residue preference is identical for PEDV- and SARS-3CLpro in most all cleavage sites, with the exception of nsp5/6 and nsp12/13. SARS-3CLpro can accommodate a larger P2-phenylalanine, e.g. at the nsp5/6 cleavage site, while PEDV-3CLpro recognizes residues only as large as a P2-leucine at this position in the polyprotein. At the nsp12/13 cleavage site, PEDV polyprotein 1ab has a serine at the P2 site, while SARS polyprotein 1ab has a P2-leucine. A larger size of the S2 pocket in SARS-3CLpro is consistent with the SARS polyprotein 1ab having a phenylalanine residue at the P2 position of the nsp5/6 cleavage site. This larger S2-pocket in SARS-3CLpro may allow for better binding of (R)-16, allowing the bulky t-butylanilido group to take full advantage of hydrophobic interactions within the S2 subsite and position (R)-16 to optimize hydrogen-bonding with SARS-3CLpro Gly143 and His163.\nConclusions\nIn this study, we have determined the X-ray crystal structure of an unbound form of PEDV-3CLpro to 2.10&#8201;&#197; resolution. We found the structure of PEDV-3CLpro to be similar to the X-ray structure of the unbound human coronavirus 229E 3CLpro, which belongs to the same coronaviral phylogenetic subgroup as PEDV-3CLpro. To investigate the role of inhibitor binding on the overall architecture of the catalytic site, we generated a superimposition of PEDV-3CLpro with FIPV-3CLpro (PDB entry 4zro), which belongs to the same alpha-coronavirus phylogenetic group as PEDV-3CLpro and shares 61.9% sequence identity, in complex with the covalent inhibitor 6. We observed little difference between the active site architectures of unbound PEDV-3CLpro and the inhibitor-bound form of FIPV-3CLpro, except for differences in the position and composition of a loop comprising the S2 subsite, which was also observed in comparison to the 229E-3CLpro structure.We tested PEDV-3CLpro for inhibition by a known SARS-3CLpro inhibitor, compound (R)-16 and found (R)-16 to be capable of inhibiting PEDV-3CLpro although the IC50 value was roughly 17-fold higher than the reported IC50 for (R)-16 against SARS-3CLpro. Structural comparison of SARS-3CLpro bound with (R)-16 and PEDV-3CLpro in its unbound form revealed that the residues that directly interact with (R)-16 in SARS-3CLpro are conserved in PEDV-3CLpro. Additional analysis of the S2 loop across alpha-coronavirus 3CLpro&#8217;s and SARS-3CLpro proved that the sequence identity of the S2 loop is not conserved across alpha- and beta-coronaviral 3CLpro&#8217;s and the SARS-3CLpro S2 loop is one residue longer than that of the alpha-coronaviral 3CLpro&#8217;s, which likely increases the effective volume of the SARS-3CLpro S2 protease subsite relative to the alpha-coronaviral 3CLpro S2 subsites. These findings provide a potential explanation for the roughly 17-fold weaker inhibition of PEDV-3CLpro by (R)-16 compared to SARS-3CLpro.This work advances our understanding of the subtle, but important, structural differences between 3C-like proteases from different coronaviral phylogenetic groups and contributes to the broader structural knowledge of coronaviral 3CLpro&#8217;s. Small structural changes have been shown to be essential in enzymatic catalysis and an understanding of these such differences is also very pertinent for the design of both broad-spectrum and selective coronaviral 3CLpro inhibitors for the treatment of coronaviral infection23.\nMethods\n\nProtein expression and purification\nThe gene encoding the 3CLpro of PEDV (residues 2998&#8211;3299 in the Porcine epidemic diarrhea virus polyprotein AHA38151.1)24 was codon optimized for expression E. coli and cloned into pET-11a expression vector with an N-terminal (His)6-tag followed by nsp4-/5 auto-cleavage site by BioBasic Inc. This pET-11a PEDV-3CLpro construct was used because it results in the expression of PEDV-3CLpro without an N-terminal or C-terminal extension.E. coli BL21(DE3) cells were transformed with the pET-11a PEDV-3CLpro plasmid and then grown at 25&#8201;&#176;C for 24&#8201;hours in 500&#8201;mL Super LB media (3&#8201;g potassium phosphate monobasic, 6&#8201;g sodium phosphate dibasic, 20&#8201;g tryptone, 5&#8201;g yeast extracts, 5&#8201;g sodium in 1&#8201;L water, pH 7.2 adjusted with 1&#8201;M NaOH) that was also supplemented with 1&#8201;mL 100&#8201;mg mL&#8722;1 carbenicillin, 25&#8201;mL 8% lactose, 10&#8201;mL 60% glycerol, and 5&#8201;mL of 10% glucose per 1&#8201;L of expression culture. The cells were harvested by centrifugation (8,400&#8201;g for 20&#8201;min) to yield 14.5&#8201;g L&#8722;1 of cells. The cell pellet was then re-suspended in Buffer A (50&#8201;mM Tris pH 7.5, 0.2&#8201;M ammonium sulfate, 0.05&#8201;mM EDTA, 5&#8201;mM BME) containing 1&#8201;mg mL&#8722;1 lysozyme, where 5&#8201;mL of Buffer A was used per 1&#8201;g cell pellet. After the cells were homogenized, they were lysed via sonication for a total of 10&#8201;minutes with 10&#8201;s pulses and 20&#8201;s delays at 50% amplitude using a Branson Digital Sonifier.The cell lysate was clarified by pelleting the cell debris via centrifugation (28,960&#8201;g, 4&#8201;&#176;C, 20&#8201;minutes).The resultant supernatant was loaded onto a 60&#8201;mL Phenyl Sepharose 6 Fast Flow HiSub (GE Healthcare) column equilibrated with Buffer A. Protein was eluted with a linear gradient to 100% Buffer B (20&#8201;mM Tris pH 7.5, 0.05&#8201;mM EDTA, 5&#8201;mM BME) over five column volumes (300&#8201;mL) collecting 5&#8201;mL fractions. Fractions containing PEDV-3CLpro enzymatic activity were pooled and loaded onto a 60&#8201;mL DEAE Sepharose Fast Flow (GE Healthcare) column equilibrated with Buffer B. Protein was eluted with a linear gradient to 50% Buffer C (50&#8201;mM Tris pH 7.5, 1&#8201;M sodium chloride, 0.05&#8201;mM EDTA, 5&#8201;mM BME, 10% glycerol) over five column volumes (300&#8201;mL) collecting 5&#8201;mL fractions. Fractions containing pure PEDV-3CLpro, as judged by SDS-PAGE and specific activity, were pooled, dialyzed into storage buffer (25&#8201;mM HEPES, pH 7.5, 2.5&#8201;mM DTT, 10% glycerol), and concentrated using a spin concentrator (Millipore) with a 10&#8201;kDa molecular weight cutoff membrane. The protein was flash-frozen in liquid nitrogen and then stored at &#8722;80&#8201;&#176;C in a freezer until further use (Fig. 1a). Prior to crystallization, PEDV-3CLpro was concentrated to 4.0&#8201;mg mL&#8722;1 using a spin concentrator (10&#8201;kDa cutoff membrane, Millipore) and loaded onto a 24&#8201;mL Superdex 75 (GE Healthcare) size exclusion chromatography column equilibrated with 25&#8201;mM HEPES pH 7.5 and 2.5&#8201;mM DTT. Peak fractions containing PEDV-3CLpro, as determined via visualization of SDS-PAGE and confirmed by protein activity assay, were pooled and concentrated to protein concentrations of 1.5, 2.8, and 4.0&#8201;mg mL&#8722;1 for crystallization.\nCrystallization\nThe PEGS II Suite (Qiagen) sparse-matrix screen was used to screen for initial PEDV-3CLpro crystallization conditions. Crystallization trials (150&#8201;nL protein and 150&#8201;nL of crystallization solution) were set up in a 96-well sitting drop tray (Intelli-Plate 96) using a TTP LabTech Mosquito&#174; liquid robotics system at PEDV-3CLpro concentrations of 1.5, 2.8, and 4.0&#8201;mg mL&#8722;1 in buffer containing 25&#8201;mM HEPES pH 7.5 and 2.5&#8201;mM DTT. An initial hit of clusters of needle-like crystals was obtained at 20&#8201;&#176;C from the PEGS II Suite condition No. 50 consisting of 10% (w/v) PEG-4000 and 20% (w/v) isopropanol. Crystals were also obtained at each of the other PEDV-3CLpro concentrations. Optimization of this condition was achieved by varying the concentration of both isopropanol and PEG-4000 between 15&#8211;30% and 7&#8211;22% respectively, where diffraction quality crystals grew at a variety of conditions from this round of optimization. The best crystals grew at 20&#8201;&#176;C in 25% isopropanol and 22% PEG-4000 at a protein concentration of 1.5&#8201;mg mL&#8722;1 and appeared after 24&#8211;72&#8201;hours (Fig. 4). Crystals were transferred using 0.05&#8211;0.1&#8201;&#956;m nylon loops to small drops containing the crystallization solution plus the cryo-protectant, which was 15% 2-methyl-2,4-pentanediol (MPD). After cryo-protection, the crystals were remounted into the nylon loop and rapidly flash-cooled in liquid nitrogen.\nData collection and structure refinement\nX-ray diffraction data were collected on LS-CAT beamline 21-ID-F at the Advanced Photon Source (APS) at Argonne National Laboratory, Argonne, Illinois, USA. Data were indexed, integrated and scaled using HKL-200025 and the resulting structure factor amplitudes were used for molecular replacement (MR). The program Phaser-MR (simple interface) module of PHENIX26 was used to perform MR and the X-ray structure of the human coronavirus 229E-3CLpro in complex with the peptidomimetic compound EPDTC (PDB entry 2zu213) with ligands and waters removed was used as a search model. Iterative rounds of manual building and structural refinement were completed using PHENIX, and manual inspection, rebuilding and the addition of water molecules were accomplished using the programs Coot27 and the refinement module of PHENIX. The final data collection and refinement statistics are summarized in Table 1.\nInhibitor characterization\nThe compound (R)-16 was synthesized according to Jacob et al.15. Inhibition of PEDV-3CLpro by (R)-16 at a concentration of 100&#8201;&#956;M was first tested in an enzymatic assay containing the following buffer (50&#8201;mM HEPES, 0.1&#8201;mg/mL BSA, 0.01% TritonX-100, 1&#8201;mM DTT). The assays were carried out in triplicate using Costar 3694 EIA/RIA 96-Well half-area, flat bottom, black polystyrene plates from Corning Incorporated. 1&#8201;&#956;L of 100X inhibitor stock in DMSO was added to 79&#8201;&#956;L of enzyme in assay buffer and the enzyme-inhibitor mixture was incubated for 10&#8201;minutes. The reaction was initiated by the addition of 20&#8201;&#956;L of 10&#8201;&#956;M UIVT3 substrate, a custom synthesized F&#246;rster resonance energy transfer peptide substrate with the following sequence: HilyteFluorTM488-ESARLQSGLRKAK-QXL520TM-NH2, producing final concentrations of 100&#8201;nM for the 3CLpro enzyme and 2&#8201;&#956;M UIVT3 substrate. The increase in fluorescence intensity of the reaction was then measured over a period of 10&#8201;minutes as relative fluorescence units (RFUt). An excitation wavelength of 485 (bandwidth of 20&#8201;nm) and an emission wavelength at 528 (bandwidth of 20&#8201;nm) was used to monitor the reactions using a BioTek Synergy H1 multimode microplate reader. The percent inhibition of the PEDV-3CLpro was determined using the following equation (1):The IC50 value of (R)-16 against PEDV-3CLpro was determined at ambient temperature using 100&#8201;&#956;L assays conducted in the following buffer: 50&#8201;mM HEPES, 0.1&#8201;mg/mL BSA, 0.01% TritonX-100, 1&#8201;mM DTT. Kinetic assays were carried out in triplicate and in Costar 3694 EIA/RIA 96-Well Half Area, flat bottom, black polystyrene plates from Corning Incorporated. (R)-16 was tested at concentrations of 0.313, 0.652, 1.25, 2.5, 5.0, 10, 20, 40, 60, 80, 100 and 120&#8201;&#956;M by adding 1&#8201;&#956;L of 100X inhibitor stock in DMSO to 79&#8201;&#956;L of enzyme in assay buffer and then incubating the enzyme-inhibitor mixture for 10&#8201;minutes. The reaction was initiated by the addition of 20&#8201;&#956;L of 10&#8201;&#956;M UIVT3 substrate, producing final concentrations of 100&#8201;nM for 3CLpro and 2&#8201;&#956;M for the UIVT3 substrate. The increase in fluorescence intensity over time during the reaction was then measured.The average percent inhibition of PEDV-3CLpro was then calculated from triplicate data, and the final averaged data with standard deviation were then plotted as a function of inhibitor concentration. The data were fit to the following equation (2) to determine the IC50:where %Imax is the percent maximum inhibition of 3CLpro. The errors in the IC50 and %Imax values are the errors in the fitted parameters resulting from fitting of the equations to the data.\nAdditional Information\nAccession codes: Model coordinates and structure factor amplitudes for the X-ray crystal structure of unbound PEDV-3CLpro were deposited in the Protein Data Bank (acc. #: 5HYO).How to cite this article: St. John, S. E. et al. X-Ray Structure and Inhibition of 3C-like Protease from Porcine Epidemic Diarrhea Virus. Sci. Rep.\n6, 25961; doi: 10.1038/srep25961 (2016).",
        "pmid": "27173881",
        "title": "X-Ray Structure and Inhibition of 3C-like Protease from Porcine Epidemic Diarrhea Virus",
        "journal_title": "Scientific Reports",
        "authors": "Sarah E. St. John;Brandon J. Anson;Andrew D. Mesecar",
        "doi": "10.1038/srep25961"
    },
    {
        "keywords": "",
        "abstract": "An RNA-dependent RNA polymerase (RdRp) is the central catalytic subunit of the RNA-synthesizing machinery of all positive-strand RNA viruses. Usually, RdRp domains are readily identifiable by comparative sequence analysis, but biochemical confirmation and characterization can be hampered by intrinsic protein properties and technical complications. It is presumed that replication and transcription of the &#8764;30-kb severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) RNA genome are catalyzed by an RdRp domain in the C-terminal part of nonstructural protein 12 (nsp12), one of 16 replicase subunits. However, thus far full-length nsp12 has proven refractory to expression in bacterial systems, which has hindered both the biochemical characterization of coronavirus RNA synthesis and RdRp-targeted antiviral drug design. Here, we describe a combined strategy involving bacterial expression of an nsp12 fusion protein and its in vivo cleavage to generate and purify stable SARS-CoV nsp12 (106 kDa) with a natural N-terminus and C-terminal hexahistidine tag. This recombinant protein possesses robust in vitro RdRp activity, as well as a significant DNA-dependent activity that may facilitate future inhibitor studies. The SARS-CoV nsp12 is primer dependent on both homo- and heteropolymeric templates, supporting the likeliness of a close enzymatic collaboration with the intriguing RNA primase activity that was recently proposed for coronavirus nsp8.",
        "body": "INTRODUCTION\nThe central enzymatic activity in the life cycle of positive-strand RNA (+RNA) viruses is the process of RNA-templated RNA synthesis. In mammalian cells, the replication of +RNA genomes always occurs in the cytoplasm where the viral RNA-synthesizing machinery is commonly associated with (modified) cytoplasmic membranes, which presumably provide physical support and may serve to protect the complex from host defense mechanisms (1,2). The viral enzyme complex contains an RNA-dependent RNA polymerase (RdRp) as its core catalytic subunit for the production of negative strand RNA (&#8211;RNA), new genome molecules and in many virus groups also subgenomic (sg) messenger RNAs (mRNAs) (1,3).Coronaviruses (CoVs) are a family of +RNA viruses that infect a wide range of vertebrates including humans, as exemplified by severe acute respiratory syndrome (SARS) and various respiratory infections caused by less-virulent family members (4). The group is also known for its exceptionally large polycistronic genome of &#8764;30 kb, which is 5&#8242; capped and 3&#8242; polyadenylated RNA. The 5&#8242;-proximal two-thirds of the CoV genome (open reading frames 1a and 1b) are translated into the viral replicase polyproteins pp1a and pp1ab. The 3&#8242;-proximal third encodes the viral structural proteins and several so-called accessory proteins, which can only be expressed following the production of a set of four to nine sg mRNAs (5,6). In particular, it is this CoV hallmark that has attracted attention over the years, since these sg mRNAs consist of sequences that are noncontiguous in the viral genome: a common 5&#8242; leader sequence is attached to body segments representing a variable part of the 3&#8242;-proximal genomic region. According to the now widely supported transcription model of Sawicki and Sawicki (7), the arterivirus and CoV sg RNAs derive from a discontinuous step during minus-strand RNA synthesis that is guided by specific RNA signals and resembles copy-choice RNA recombination (5,6,8). This mechanism produces a set of subgenome-length negative-strand templates from which the sg mRNAs are produced. The unusual genome size of CoVs and the complexity of their RNA synthesis hint at an RdRp that may have special properties, either by itself or in conjunction with other subunits of the CoV replication/transcription complex (RTC).CoV replication and transcription are driven by the 15 or 16 viral nonstructural proteins (nsps) encoded in the replicase gene, which include many enzyme activities that are rare or lacking in other families of +RNA viruses (9,10). These nsps are produced during co- and post-translational processing of the pp1a and pp1ab replicase polyproteins by two or three internal viral proteinases (11). In the years following the 2003 SARS outbreak, bioinformatics, structural biology, (reverse) genetics and biochemical studies have all contributed to the in-depth characterization of CoV nsps in general and those of SARS-CoV in particular (9,10,12). Currently documented enzyme activities include proteinases (11), a putative RNA primase (13), a superfamily 1 helicase (14), an exo- and an endoribonuclease (15,16), single-stranded RNA (ssRNA)-binding proteins (17,18) and two methyltransferases (19,20). Following the sequence analysis of the first CoV genomes, more than two decades ago, the putative RdRp domain of the coronavirus RTC had already been identified by comparative sequence analysis (21,22). The C-terminal, two-thirds of the 932-amino-acid subunit, now known as nsp12 was confidently aligned with the conserved motifs of well-known RdRps (22,23). In view of its unprecedented size among viral RdRp subunits, which commonly consist of 500&#8211;600 amino acids, nsp12 may well harbor other functional domains in its as-yet uncharacterized N-terminal domain. Biochemical information on the CoV RdRp has, however, remained scarce thus far, particularly since full-length nsp12 was refractory to expression in bacterial systems. Preliminary evidence for the in vitro RdRp activity of nsp12 came from a study of the SARS-CoV enzyme by Cheng et al. (24). However, their experiments, employing a glutathione S-transferase (GST)-nsp12 fusion protein, were hampered by protein instability, which resulted in the fragmentation of the protein into three parts. Primer-dependent activity on poly(A) templates was observed in filter binding assays, but the enzyme biochemistry was not further characterized in detail.Given its pivotal role in viral replication and the efficacy of polymerase inhibitors used to combat other virus infections, the SARS-CoV RdRp is widely regarded as an important and attractive target for the rational design of anti-CoV drugs (23,25). We therefore sought to solve the technical issues described above and develop reliable protocols for stable expression, purification and in vitro activity of the full-length CoV nsp12. In this article, we describe an expression strategy utilizing in vivo cleavage in a bacterial expression system by ubiquitin carboxyl-terminal hydrolase 1 (Ubp1), which releases the small N-terminal fusion partner ubiquitin (Ub) and generates a recombinant nsp12 protein containing its natural N-terminus. This was found to be a critical step on the road toward purification of a stable recombinant SARS-CoV RdRp. We subsequently established that full-length nsp12 can readily synthesize RNA in a primer-dependent fashion on both homo- and heteropolymeric RNA templates and is also efficient at incorporating deoxy-nucleotides on a DNA template. This work therefore provides a solid basis for the further biochemical characterization of the coronavirus nsp12 and the detailed dissection of its role in viral replication and transcription. Understanding the enzyme&#8217;s properties will be essential for the development of selective coronavirus RdRp inhibitors and will likely provide novel insights into the intriguing (evolutionary) relation between the RdRp of nidoviruses and other groups of +RNA viruses.\nMATERIALS AND METHODS\n\nCloning and expression of recombinant SARS-CoV nsp12\nThe amino acid sequence of SARS-CoV nsp12, residues S4370 to Q5301 of replicase pp1ab, is fully conserved among all sequenced SARS-CoV isolates. For our study, the nsp12-coding region was amplified by reverse transcription-polymerase chain reaction (RT-PCR) from the genome of SARS-CoV Frankfurt-1 (AY291315). During amplification, the ORF1a/1b &#8722;1 ribosomal frameshift near the 5&#8242;-end of the gene was corrected in a translationally silent manner. Primers used for amplification were SAV424 5&#8242;-GCGGGTACCCCGCGGTGGATCTGCGGATGCATCAA-3&#8242; and SAV425 5&#8242;-GCGCGATCGGGATCCCTGCAAGACTGTATGT-3&#8242;, corresponding to SARS-CoV genome positions 13 327&#8211;13 387 and 16 152&#8211;16 166, respectively. The PCR amplicon was digested with SacII and BamHI, and ligated into expression vector pET26-Ub-CHis6 (26). Following confirmation of the nucleotide sequence of the amplicon and using the flanking XbaI and XhoI restriction sites, the ubiquitin-nsp12-His6 fusion gene (Ub-nsp12-CHis6) was subcloned into vector pASK3 (IBA) and thereby placed under the control of a tetracycline-inducible promoter. This ruled out the known risk of potential contamination with T7 RNA polymerase, as it can occur when using a T7 promoter to drive expression of other RdRps in bacterial expression systems. Nsp12 active-site mutant D618A, carrying an Asp618Ala substitution, was engineered via site-directed mutagenesis according to the QuikChange protocol (Stratagene), using primers SAV526 5&#8242;-CCTTATGGGTTGGGCTTATCCAAAATGTG-3&#8242; and SAV527 5&#8242;-CACATTTTGGATAAGCCCAACCCATAAGGA-3&#8242;.For nsp12 expression, the pASK3-Ub-nsp12-CHis6 plasmid was transformed to Escherichia coli C2523 cells (New England Biolabs) together with the Ubp1 protease expression plasmid pCGI (26), unless indicated otherwise for particular experiments. Routinely, 250 ml of Luria Broth, containing ampicillin (50 &#956;g/ml) and chloramphenicol (34 &#956;g/ml), was inoculated 1:1000 with 3 ml o/n precultures and cells were grown at 37&#176;C to OD600 >0.7. Subsequently, the cells were slowly cooled to room temperature, tetracycline was added to a final concentration of 200 &#956;g/ml and the cells were further grown at 20&#176;C for another 16 h. Cells were harvested by centrifugation, washed once with ice-cold phosphate-buffered saline (PBS) and stored at &#8722;20&#176;C until protein purification.SARS-CoV nsp12 expression vectors pKM596-nsp12 (kindly provided by Dr. Isabelle Imbert and Dr. Bruno Canard) and pET102-Thio-nsp12 (a kind gift of Dr. Susanne Pfefferle and Dr. Christian Drosten) were used to transform E. coli BL21(DE3) cells and expression was performed using the auto-induction medium ZYM-5052 (27).\nPurification of SARS-CoV nsp12-His\nCell pellets were thawed on ice, resuspended in binding buffer (20 mM HEPES pH 7.4, 500 mM NaCl, 20 mM imidazole, 10% glycerol, 0.05% Tween-20 and 5 mM &#946;-mercaptoethanol) and lysed by sonication. Cell debris was removed by ultra-centrifugation at 20 000 g and the cleared supernatant was incubated with Talon beads (Clontech) for 2 h at 4&#176;C. The beads were washed three times with 20 vol of binding buffer, loaded onto spin columns and finally the His-tagged protein was eluted with 300 mM imidazole in binding buffer. The eluate was analyzed by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS&#8211;PAGE) and typically found to be 80&#8211;85% pure. Subsequently, gel filtration on a Superdex 200 column (GE Healthcare) in 20 mM HEPES, 500 mM NaCl, 5% glycerol, 5 mM &#946;-mercaptoethanol and 0.1% Tween-20 was used to improve purity to &#8764;95%. To estimate the molecular mass of nsp12, a standard curve for hydrodynamic elution was made using chymotrypsinagen A, conalbumin, aldolase and dextran blue (all purchased from GE Healthcare) as size markers and fitted to a linear equation (R2 = 0.99).After the gel filtration step, the nsp12-containing fractions were pooled, concentrated with Ultrafree 10-kDa filter columns (Millipore) and dialyzed overnight against 1000 sample volumes of dialysis buffer (20 mM HEPES pH 7.4, 100 mM NaCl, 0.1% Triton-X 100, 1 mM DTT and 50% glycerol) in cellulose tubing (SnakeSkin, Pierce). Protein concentrations were determined using a Bradford assay and a bovine serum albumin (BSA) standard curve, and the purified protein was stored at &#8722;20&#176;C. When using expression vectors pKM596-nsp12 and pET102-Thio-nsp12, protein purification procedures were essentially similar to those described above, except that amylose resin (New England Biolabs) was used to bind the pMK596-nsp12 product, which was eluted using a buffer containing 10 mM amylose instead of imidazole.\nSDS\u2013PAGE and western blotting\nCells were lysed in Laemmli&#8217;s sample buffer (28), separated on 8% SDS&#8211;PAGE gels and either electroblotted to polyvinylidene fluoride (PVDF) membrane (GE Healthcare) or stained directly with Coomassie G-250 according to standard protocols. Prior to incubation with a SARS-CoV nsp12-specific rabbit antiserum (a kind gift from Dr. Mark Denison) (29), blot membranes were blocked with 10% skimmed milk and 0.02% Tween-20 in PBS. Blots were washed with 0.02% Tween-20 in PBS, incubated with a swine-anti-rabbit horseradish peroxidase conjugate diluted in 5% skimmed milk, 0.02% Tween-20 and PBS, washed and subjected to ECL Plus chemiluminescence detection (GE Healthcare).\nPreparation of templates\nRNA oligonucleotides and their DNA counterparts SAV557R 5&#8242; -GCUAUGUGAGAUUAAGUUAU-3&#8242; and SAV481R 5&#8242;-UUUUUUUUUUAUAACUUAAUCUCACAUAGC-3&#8242;, SAV555R 5&#8242;- UUUUUUUUUUUUUUUUUUUUAUAACUUAAUCUCACAUAGC-3&#8242; and SAV556R 5&#8242;-UCUCUCUCUCUCUCUCUCUCAUAACUUAAUCUCACAUAGC-3&#8242; were purchased from Eurogentec, excised as a single band from 7 M urea/15% polyacrylamide gels, eluted overnight in deionized water and desalted using NAP-10 columns (GE Healthcare). To anneal RNA duplexes, oligonucleotide mixtures in annealing buffer [20 mM Tris&#8211;HCl pH 8.0, 50 mM NaCl and 5 mM ethylenediaminetetraacetic acid (EDTA)] were heat denatured, slowly cooled to room temperature and subsequently purified from 15% nondenaturing polyacrylamide gels.\nRdRp activity assays\nTo test the in vitro RdRp activity of SARS-CoV nsp12-CHis6, 0.1 &#956;M of the protein (unless otherwise indicated) was incubated at 30&#176;C in the presence of 1 &#956;M template, 0.17 &#956;M [&#945;-32P]ATP (0.5 &#956;Ci/&#956;l; Perkin-Elmer), 50 &#956;M ATP (Roche), 20 mM Tris&#8211;HCl (pH 8.0), 10 mM NaCl, 10 mM KCl, 6 mM MgCl2, 5% glycerol, 0.01% Triton-X 100, 1 mM DTT and 0.5 U of RNaseOUT (Invitrogen). Glycerol and NaCl were introduced with the addition of enzyme due to their presence in the nsp12 storage buffer. RdRp reactions were terminated by adding an equal volume of 90% formamide, 50 mM EDTA pH 8.0, 0.01% bromophenol blue and xylene cyanol. Reaction products were analyzed on 7 M Urea, 20% polyacrylamide gels buffered with 1&#215; TBE (90 mM Tris, 90 mM H3BO3 and 2 mM EDTA). Gels were run for 5 h at 2000 V and analyzed using phosphorimaging. Plates were scanned on a Typhoon variable mode scanner (GE Healthcare) and products were quantified with ImageQuant TL 7.0 (GE Healthcare). To calculate the amount of [&#945;-32P]AMP incorporated from the phosphorimager signal, a 10&#8722;2&#8211;10&#8722;5 dilution series of the [&#945;-32P]ATP stock was spotted in triplicate on Whatman filter and exposed alongside the PAGE gel of interest. Using the standard curve, phosphorimager counts were converted into the amount of label incorporation and kinetic traces were fit to a linear function with the MATLAB 2008b Curve Fitting Toolbox (Mathworks).\nElectrophoretic mobility shift assays\nA dilution series of SARS-CoV RdRp in storage buffer was incubated for 10 min at room temperature with 0.2 nM of 32P-labeled oligonucleotide SAV557R (see above), in the presence or absence of the complementary oligonucleotide SAV481R. Samples were then directly loaded on to 8% acrylamide gels containing 5% glycerol and 0.5&#215; TBE and were run at 100 V for 2 h at 4&#176;C. Gels were dried on Whatman filter paper and bands were quantified by phosphorimaging. Using the MATLAB 2008b Curve Fitting Toolbox, the percentage of RNA bound was fit to the Hill equation RNAbound = b*[nsp12]n/(Kdn+ [nsp12]n), where b is the upper binding limit, [nsp12] is the nsp12 concentration, n is the Hill coefficient and Kd is the enzyme&#8217;s dissociation constant.\nRESULTS\n\nStable expression and purification of full-length SARS-CoV nsp12\nA PCR amplicon encoding SARS-CoV nsp12 was first cloned into T7 expression vector pET26-Ub-CHis6, which provided the 932-aa viral protein with an N-terminal Ub tag (76 amino acids) and a C-terminal hexahistidine tag (Figure 1A). To avoid the known risk of T7 RNA polymerase contamination when purifying another RdRp using a T7 promoter-driven expression system [like E. coli BL21(DE3)], the Ub-nsp12-CHis6 fusion gene was subsequently transferred to plasmid pASK3, placing it under control of a tetracycline-inducible promoter. Initially, the protease that was later used to remove the Ub tag was not co-expressed and the expression of the full-length Ub-tagged nsp12-CHis6 was compared to that of variants containing either maltose binding protein (MBP) or thioredoxin (Thio) as fusion partner. As shown in Figure 1B and as also documented in the previous work of Cheng et al. (24), expression and purification of nsp12 fusion proteins resulted in the abundant and progressive fragmentation of the protein. Based on their sizes in SDS&#8211;PAGE, the cleavage products likely were the 64- and 39-kDa nsp12 degradation products as found by Cheng et al. (24) when working with their GST-nsp12 fusion construct (Figure 1B). Therefore, our results confirmed and extended the previously reported observations regarding the instability of SARS-CoV nsp12 when expressed in E. coli as part of a fusion protein.\nFigure 1.Expression and purification of the SARS-CoV nsp12. (A) Expression of nsp12 was performed in the presence of the ubiquitin (Ub) protease Ubp1, which removed the N-terminal Ub fusion partner by cleaving after the LRGG signature that formed the junction between the Ub and nsp12 moieties. This in vivo cleavage created the native nsp12 amino-terminus with the sequence SADAS. (B) Western blotting using an nsp12-specific polyclonal antiserum identified a single band of the expected mass after purification of the in vivo cleaved expression product. Isolation of fusion proteins with different N-terminal tags resulted in common patterns of degradation. Similar results were reported by Cheng et al. (24) for a GST-nsp12 fusion protein. (C) Purified nsp12-CHis6 was readily visualized by Coomassie staining as a &#8764;106-kDa protein product, in line with the expected mass. Inductions in the absence of Ubp1 expression, however, resulted in a mixture of full-length fusion protein and degradation products, likely as a result of improper folding. No RdRp activity was observed for these preparations. (D) Gel filtration analysis of purified nsp12 in assay buffer showed a single peak, corresponding to a globular molecular weight of 140 &#177; 10 kDa, suggesting that only the monomeric form of nsp12 is present under the conditions used.In studies with the poliovirus (PV) RdRp (3Dpol), it was previously shown that N-terminal foreign sequences, even the addition of a single methionine residue, can have a significant impact on enzyme activity (26,30). Structural studies subsequently revealed that a native N-terminus is required for the proper folding of the RdRp&#8217;s &#8216;fingers&#8217; subdomain and the positioning of the 3Dpol active site (30). These findings suggested to us that N-terminal additions might also hamper the proper folding of the SARS-CoV RdRp and could thus be the principal cause of its subsequent degradation. We therefore co-expressed the C-terminal Ub protease Ubp1 (26), which specifically recognizes the cleavage site in our fusion protein that is located between the Ub moiety and the N-terminus of nsp12 (LRGG&#8595;SADAS). Proteolytic cleavage of the fusion protein at this site liberated an nsp12-CHis6 product (Figure 1A), now containing its natural N-terminus. The expression of the Ubp1 protease was driven by the relatively weak constitutive ADH1 promoter (26). Hence, to ensure optimal release of nsp12-CHis6 by the protease, induction of Ub-nsp12-CHis6 expression was performed only in the middle or late log-phase of bacterial growth. Induction at lower densities or on-column cleavages performed with the cleavable MBP and Thio fusion proteins resulted in significant amounts of degraded nsp12, which was inactive in RdRp assays (data not shown).As depicted in Figure 1B and C, the in vivo cleavage product that we were able to routinely purify was of the expected 106-kDa size (9) and was found to be stable, even after multiple rounds of purification and prolonged storage at &#8722;20&#176;C. Non-denaturing gel filtration yielded a single peak corresponding to a size of about 140 &#177; 10 kDa as estimated by comparison to a standard curve (Figure 1E&#8211;F). This is different from the calculated mass of 106 kDa, but may be explained by, e.g. a very open or nonglobular structure of the protein. Moreover, unexpected behavior in gel filtration is not without precedent for RdRps, as was, e.g. reported for the monomeric &#966;6 RdRp P2 (31). Future structural studies involving electron microscopy or crystallography will hopefully allow us to address this question in more detail.\nSARS-CoV nsp12 is a primer-dependent RdRp\n+RNA virus replication may proceed through a variety of scenarios, but essentially viral RdRps employ only two fundamentally different mechanisms during the first step of RNA synthesis: initiation can occur either de novo, when the 3&#8242; hydroxyl group of the first nucleotide essentially is the &#8216;primer&#8217; to which a second nucleotide is added, or initiation can depend on an oligonucleotide or protein primer (32). Imbert et al. (13) recently showed that the CoV nsp8 subunit, a relatively small protein (22 kDa) that forms a hexadecameric complex with nsp7 (33), can synthesize short oligonucleotides using poly(C) templates or templates containing a 3&#8242; CCG/U motif. Although the exact role of nsp8 remains to be studied in much more detail, one of its proposed functions is that of an &#8216;RNA primase&#8217; (13). According to this hypothesis, and similar to what has been documented for cellular DNA polymerases (34), the oligonucleotide products of nsp8 activity would serve as primers for the viral &#8216;main RdRp&#8217;, i.e. nsp12. Some other observations were also consistent with the possibility that the SARS-CoV main RdRp belongs to the class of primer-dependent polymerases. First, modeling of the structure of SARS-CoV nsp12 residues 400&#8211;900 suggested that the central domain of the protein contains a G motif, a signature present in all primer-dependent RdRps (23). Second, when using the unstable GST-nsp12 fusion protein in combination with a poly(A) template and a poly(U) primer, Cheng et al. (24) observed a moderate RdRp activity in filter-binding assays. Convincing evidence for the extension of an RNA primer in vitro, however, has not been documented thus far.To qualitatively assess the polymerase activity of the nsp12, we monitored the incorporation of nucleotides on a well-defined synthetic RNA template, which was predicted to lack secondary structures that could affect the enzyme&#8217;s processivity. This template (Figure 2A) consisted of a 20-nt duplex and a 3&#8242; overhang of 10 uridylates, which could serve as template during extension of the 20-mer primer. As shown in Figure 2, primer extension was revealed by both the incorporation of [&#945;-32P]ATP (Figure 2B) and the extension of the 32P-labeled primer (Figure 2C). The use of single-stranded or homopolymeric [poly(U) or poly(C)] templates did not lead to significant incorporation of nucleotides (data not shown).\nFigure 2.SARS-CoV nsp12 primer extension assay and active-site mutant. (A) Schematic showing the structure of the partially double-stranded RNA template with 3&#8242; U10 stretch that served as template for primer extension in the initial nsp12-based nucleotide incorporation assay. (B) Comparison of the RdRp activity of wild-type nsp12 and a D618A active-site mutant, which displayed minimal activity after a 60-min incubation at 30&#176;C. (C) A primer extension assay using a 5&#8242; 32P-labeled primer and ATP confirmed that wild-type nsp12, in contrast to the D618A mutant, was able to elongate the 20-mer primer. The slight reduction of input primer in the D618A lane probably resulted from degradation by RNase activity. The D618A mutant showed 8 &#177; 3% residual activity (mean of three independent experiments). In the loading control (Figure 2B and C, lower panels) the two nsp12 variants were visualized by silver staining of an SDS&#8211;PAGE gel.Based on sequence alignments of SARS-CoV nsp12 with RdRps of known structure, it was previously concluded that the canonical aspartic acid residues responsible for the coordination of the divalent cation in the RdRp&#8217;s active site should be nsp12 residues 618, 760 and 761, the latter two amino acids being part of the nidovirus-specific SDD motif (23,35&#8211;37). In other viral RdRps, single or multiple substitutions of the equivalent residues with alanine or asparagine were shown to change or even abolish polymerase activity (38,39), depending on the assay conditions used. For the PV 3Dpol, for example, it was found that mutagenesis of the motif C core sequence, changing it from GDD to GDN, was detrimental in the presence of Mg2+, but not when Mn2+ or Fe2+ were provided as divalent cation (38). To obtain an active site mutant that could be used as negative control during further development of our assays, we engineered nsp12 mutant D618A. As portrayed in Figure 2B and C, the mutant protein displayed only 8 &#177; 3% residual primer extension activity in our assays (Figure 2B&#8211;D), thus confirming the nsp12-specific nature of the RdRp activity described here and supporting the prior hypothesis that this residue is a key player in the enzyme&#8217;s active site. Further or complete knockdown of activity may be obtained under slightly different assay conditions or in combination with substitution of other active site residues, which remain to be tested in future studies.\nBiochemical determinants of the \nIt is common practice to employ filter binding assays during optimization of polymerase activity assays. Although convenient and less time consuming than gel analysis of reaction products, these assays merely reveal incorporation of label, but do not provide direct information on the length and integrity of the products. Therefore, when exploring the effects of different reaction buffer compositions, we consistently quantified and analyzed [&#945;-32P]ATP incorporation using denaturing 20% acrylamide gels. The standard assay buffer consisted of 20 mM Tris&#8211;HCl (pH 8.0), 10 mM KCl, 1 mM DTT, 6 mM MgCl2, 5% glycerol, 0.1% Triton-X 100 and 50 &#956;M ATP.First, the influence of reaction temperature on RdRp activity was tested within the range of 20&#8211;37&#176;C. Activity at 37&#176;C was 40% higher than at 30&#176;C, which is more commonly used in RdRp assays (Figure 3A) (37,40&#8211;42). However, a more detailed analysis revealed that this increase largely stemmed from enhanced initiation of primer extension rather than elongation, with the latter being decreased by 35% as judged from the amount of full-length product (primer + 10, Figure 3B). Therefore, all subsequent experiments were performed at 30&#176;C. Contrary to expectations however, the major extension product never matched the length of the primer plus 10 nt, suggesting that nsp12 generally terminates before reaching the end of the primed poly(U) template. The results obtained with other templates are discussed subsequently.\nFigure 3.Biochemical determinants of SARS-CoV nsp12 activity. (A and B) Tests to determine the influence of temperature indicated that nsp12 incorporates ATP most efficiently at 37&#176;C, although this effect mainly stemmed from increased initiation. Based on these results, 30&#176;C was taken as the standard for subsequent experiments. (C and D) Titration of the Mg2+ concentration showed that activity reaches its maximum at 6 mM. (E and F) The effect of pH on RdRp activity was evident as well, with lower pHs limiting the activity of the enzyme. All reactions were incubated for 60 min at 30&#176;C, unless otherwise indicated. Error bars in A, C and E represent standard error of the mean (n = 3).It is well established that nucleic acid polymerases utilize divalent cations, mostly Mg2+, as cofactors to bind and coordinate the incoming nucleotide during the polymerization reaction (35,36). As shown in Figure 3C and D, the activity of nsp12 was positively correlated with the Mg2+ concentration and reached its optimum at 6 mM. Higher concentrations (&#8805;20 mM) ultimately abolished activity (data not shown). The SARS-CoV RdRp does not require Mn2+ for RNA synthesis, as previously reported for the nsp9-RdRp of the arterivirus equine arteritis virus (EAV), a distant relative in the order Nidovirales (37). In fact, the presence of Mn2+ in an RdRp assay utilizing a poly(U) template decreased the fidelity of the SARS-CoV nsp12 relative to Mg2+ and facilitated both transversional and transitional misincorporations (Figure 4). The specificity for the nucleotide&#8217;s sugar moiety seemed not affected, however, since deoxyadenosine triphosphate (dATP) was not incorporated when using an RNA template in the presence of Mn2+. This discriminates the SARS-CoV RdRp from, e.g. its PV equivalent, for which correct base pairing was more important than the presence of a 2&#8242; hydroxyl group (40).\nFigure 4.Nucleotide incorporation fidelity of nsp12. (A) Experimental set-up of pulse-chase experiments with different nucleotides and a primed poly(U) template (see Figure 2A). The reactions were initiated with a limiting concentration of [&#945;-32P]ATP to allow the incorporation of a first nucleotide and the formation of a stable polymerase-template complex. Subsequently, after 10 min, different unlabeled nucleotides were added to a final concentration of 50 &#956;M to allow elongation for another 30 min. (B) SARS-CoV nsp12 allows only limited transversional and transitional misincorporations. Interestingly, also in the presence of dATP no significant activity was observed, implying that the SARS-CoV RdRp is capable of discriminating between ATP and dATP. (C) Pulse-chase experiments in the presence of 6 mM Mn2+ show that manganese ions promote misincorporation of ribonucleotides (both transversions and transitions). The selection against dATP remained unaltered.To establish the importance of pH, we tested SARS-CoV RdRp activity in the pH range of 6.0&#8211;8.5. We clearly observed an increase of nsp12 overall activity with an optimum at pH 8.0 (Figure 3E), but differences between pH 7.0 and 8.5 were marginal in terms of both overall activity and amount of full-length product produced (Figure 3F). Interestingly, pH 7.5 consistently resulted in a significantly larger amount of initiation product, suggesting that higher pH values stimulate the processivity of the enzyme.In additional assays, we tested the effect of monovalent cations, bovine serum albumin and spermidine (data not shown). No negative influence of potassium or sodium ions was observed below a concentration of 50 mM. Neutral to slightly negative effects were found for both BSA (tested for 0&#8211;0.1 mg/ml), which is known to stabilize enzymes and reduce potential side effects such as absorbance to the plastic surface of the reaction tube used. Finally, addition of spermidine, which binds to the phosphate backbone of nucleic acids and is a known stimulator of various polymerases, did not improve activity in the tested concentration range up to 5 mM.\nRNA-binding properties of SARS-CoV nsp12\nOur combined observations suggested that nsp12 is only active in a primer-dependent fashion and may thus prefer binding to double-stranded RNA (dsRNA). To determine the affinity of nsp12 for dsRNA and ssRNA we performed electrophoretic mobility shift assays, in which we titrated purified nsp12 in the presence of 0.2 nM 5&#8242;-32P-labeled RNA (Figure 5). Subsequently, the percentage of bound template and the Hill equation (see &#8216;Materials and Methods&#8217; section) were used to derive the enzyme&#8217;s dissociation constants (Kd). For dsRNA the Kd was calculated at 0.13 &#177; 0.03 &#956;M, whereas the Kd for ssRNA was estimated to be 0.10 &#177; 0.02 &#956;M.\nFigure 5.RNA binding affinity of nsp12. (A) A fixed concentration (0.2 nM) of radiolabeled dsRNA or ssRNA (data not shown) was titrated with purified nsp12 and the complexes formed were separated from unbound RNA on a native 8% polyacrylamide gel. (B) Free and bound RNA were quantified and fit to the Hill equation (see &#8216;Materials and Methods&#8217; section), resulting in Kd values of 0.13 &#177; 0.3 &#956;M for dsRNA and 0.10 &#177; 0.2 &#956;M for ssRNA binding. R2 values of these Hill fits were 0.97 and 0.98 for dsRNA and ssRNA, respectively. Error bars represent standard error of the mean (n = 3).\nNsp12 nucleotide incorporation rates\nFor various viral RdRps, in vitro studies have defined biochemical properties that could not be determined or were largely obscured in in vivo assays. Furthermore, following the basic characterization of individual enzymes, the interplay with templates, cofactors and other subunits of the viral enzyme complex can be addressed. Ultimately, this should facilitate the in vitro reconstitution of the holo-enzyme complex and the analysis of its interactions at each step of viral replication and transcription. After the purification of active SARS-CoV nsp12 and exploration of the reaction conditions, our subsequent step toward such a future goal was to evaluate the rate of nucleotide incorporation by the SARS-CoV RdRp. On a primed U10 template we found relatively poor incorporation rates (1.2 &#177; 0.4 nM/min) and a major product that was 2&#8211;3 nt shorter than the expected full-length product (Figure 6A), with product sizes displaying a similar Gaussian-like distribution as depicted in the graphs in Figure 3. On the other hand, we also observed minor products that were slightly longer than full length, suggesting slippage of the polymerase at the end of the template. We did not detect any terminal transferase activity on the templates used (data not shown).\nFigure 6.Analysis of nucleotide incorporation by nsp12. (A) Reactions with 0.1 &#956;M nsp12, 1 &#956;M template and 50 &#956;M ATP were followed over time and quenched with 50 mM EDTA at the time points indicated. Gel analysis showed that, similar to the curves in Figure 3, the major accumulating product is the primer (p) extended by 7 nt (i.e. p + 7) and not the expected full-length product of n + 10. (B) To assess whether nsp12 was capable of DNA-templated incorporation of dATP, reactions were performed as in Figure 6A, but now using a DNA template. The time course illustrates robust [&#945;-32P]dAMP incorporation, implying that the SARS-CoV RdRp is able to utilize both RNA and DNA templates. The incorporation of dAMP was 20% lower than that of AMP on an RNA template. (C) Time courses were additionally performed using an RNA template containing a U20 template region. Similar to the shorter 10-nt template used in Figure 6A, the major product here was around 2 nt shorter than the expected full-length product. (D) Changing the single-stranded template from U20 to a heteromeric (CU)10 not only increased the nucleotide incorporation rate by 34%, but also the behavior of nsp12 on the template. Products of the expected full-length size (p + 20) were observed, but also products longer than the template, hinting at, e.g. template switching by nsp12. Future research will address this observation in more detail. In this reaction both 50 &#956;M ATP and GTP were present. The gel image represents the top half of a 20% denaturing PAGE gel.Previously, Cheng et al. (24) reported that their unstable GST-nsp12 fusion protein had limited reverse transcriptase activity. In contrast to these prior results, our nsp12 was not able to efficiently use dATP to extend an RNA primer (Figure 4A and B) and a similar result was obtained with a DNA primer (data not shown). To test whether nsp12 was able to incorporate [&#945;-32P] deoxyadenosine monophosphate (dAMP) when using DNA templates, we employed a primed T10 template similar to the RNA templates described above. As shown in Figure 6B, nsp12 was capable of incorporating dAMP when incubated with a DNA template. On average, a minor reduction (&#8764;20%) in dAMP incorporation was observed in comparison to the activity on an RNA template. The size distribution of products was more different, and showed significantly less initiation than on an RNA template (compare Figure 6A and B).The incorporation rate of nsp12 increased dramatically when we doubled the template length (VU20 = 27 &#177; 3 nM/min, Figures 6C and 7), but we again observed a major product that was 2 nt shorter than the expected full-length product. When the primed poly(U) template was changed to a primed heteromeric (CU)10, nsp12 activity increased further (V(CU)10 = 45 &#177; 2 nM/min) and reached a level comparable to that obtained for other viral RdRps on similar short templates (40). When the products of the latter reactions were resolved by denaturing PAGE (Figure 6D) they were found to be longer than template length. Similar products extending beyond unit length have been often observed when short templates were used for PV 3Dpol (43) and their generation can be explained by various mechanisms, such as distributive slippage, processive slippage, template switching or terminal transferase activity (43 and references therein). Future research is expected to address these mechanisms in more detail.\nFigure 7.Analysis of the nucleotide incorporation rate of nsp12. Steady-state time courses were performed with 0.1 &#956;M nsp12 as described in Figure 6. Experimental data were subsequently fit to linear regression to obtain NMP incorporation rates. These clearly illustrated an increase of the incorporation velocity with template length (VU10 = 1.2 &#177; 0.4 nM/min and VU20 = 27 &#177; 3 nM/min) and from homopolymeric template to copolymeric template (V(CU)10 = 45 &#177; 2 nM/min). R2 values were 0.97, 0.99 and 0.99, respectively, and error bars indicate standard error of the mean (n = 3).\nDISCUSSION\nDetailed kinetic and stoichiometric information plays an important role in explaining the molecular basis of RdRp-catalyzed ribonucleotide incorporation, the associated mutation rate of viral genomes and, consequently, RNA virus evolution. Furthermore, a comprehensive understanding of the differences between viral RdRps and cellular polymerases is crucial in the design of specific inhibitors of viral RNA synthesis and their subsequent mechanistic characterization and optimization. Up till now, all of the above information was particularly limited, if not completely lacking, for the CoV RdRp, due to enzyme insolubility, instability and low purification yields.This study reports the first successful expression and purification of stable, soluble SARS-CoV nsp12, thus paving the way for its further biochemical characterization. Using a two-step procedure, the protein was purified to near homogeneity, as confirmed by SDS&#8211;PAGE and non-denaturing gel filtration. We found that even a single-step purification scheme could produce an enzyme preparation that combined specific activity with reasonable purity (>80&#8211;85%). Of particular importance in our expression protocol was the co-expression of the Ubp1 protease that mediated the in vivo removal of the N-terminal Ub tag (Figure 1A) (26). Apparently, similar to PV 3Dpol (30), SARS-CoV RdRp does not tolerate additional amino acids at its N-terminus, likely because they induce an unstable and largely inactive fold. Because of the observed sensitivity to N-terminal fusions, it is tempting to speculate that the processing of the pp1ab cleavage generating the nsp12 N-terminus is of great importance for proper folding and activity in vivo as well. In line with this notion, it was reported that for the related group 2 CoV mouse hepatitis virus (MHV) this cleavage site indeed is indispensable for virus replication (44). It still remains to be tested whether there is any effect of the C-terminal His-tag on the activity of the enzyme used in this study.The SARS-CoV nsp12 is a relatively large protein (932-amino-acid residues), e.g. much larger than the well described RdRps of HCV (591 amino acids) and PV (461 amino acids), and may have, in addition to the C-terminal RdRp domain, other functional or contributing domains in its still uncharacterized N-terminal domain(s). Previously, the PV and HCV RdRps were demonstrated to bind RNA with Kd values of 2&#8211;10 &#956;M (45) and &#8764;3 &#956;M (46), respectively. Our data suggest that SARS-CoV nsp12 binds RNA with slightly higher affinity for both dsRNA and ssRNA (Kd &#8764; 0.1 &#956;M). The relatively similar affinities for dsRNA and ssRNA suggest that the lack of de novo initiation may largely be due to a relatively open structure and the absence of a conserved priming loop in the predicted thumb subdomain (23). The latter structural element, a &#946;-hairpin in, e.g. hepatitis C virus (HCV) NS5B, is present in many known de novo initiating RNA polymerases. Canonically, it functions by partly obstructing the template binding cleft and is believed to be essential for association with the single-stranded 3&#8242; end of the template RNA (23,47,48).Contrary to previous GST-pull downs performed for nsp12 (49) and data obtained for other RdRps such as the PV 3Dpol&#8212;which was reported to form multimers and even lattices (45)&#8212;we were not able to establish nsp12 multimerization using non-denaturing gel filtration. Although this suggests that SARS-CoV RdRp exists predominantly as monomer under the conditions used here, it is likely that nsp12 binds to various other viral replicase subunits (50) and cellular cofactors (51) in vivo. The CoV holo-enzyme complex formed in this manner may have more finely tuned characteristics than documented in this study. In addition, it remains to be tested whether the C-terminal hexahistidine tag interferes with multimerization. Clearly, these and other questions should be addressed in more detail in future studies.Upon binding to template and primer, we found nsp12 to have reproducible nucleotide incorporation rates on the templates tested, but incorporation depended significantly on the length of the template. Furthermore, on heteromeric templates the SARS-CoV RdRp also performed better and reached incorporation rates in the same order (>nM/min) as the model polymerases from PV and HCV, when correcting for enzyme and template concentrations (40,52). In addition, the SARS-CoV nsp12 is also active on a DNA template, which may greatly facilitate further investigations in view of the superior stability of DNA templates and the fact that various other SARS-CoV enzymes (17,33,53) have documented affinity for DNA as well. It remains to be tested whether SARS-CoV nsp12 is also active on a longer single-stranded template, e.g. RNA templates based on the terminal sequences of the viral genome, and whether it is able to traverse the RNA secondary structures present in such a molecule. Taken together, our data suggest that our present purification scheme and assay constitute a concrete starting point for further biochemical exploration of the SARS-CoV RdRp and nsp12 targeted inhibitor development.\nFUNDING\nFunding for open access charge: The Netherlands Organization for Scientific Research (NWO; Top talent grant 021.001.037); the Council for Chemical Sciences (NWO-CW; ECHO grant 700.55.002); and by grant AI45818 from NIAID/NIH (to J.A. and C.C.).Conflict of interest statement. None declared.",
        "pmid": "19875418",
        "title": "The RNA polymerase activity of SARS-coronavirus nsp12 is primer dependent",
        "journal_title": "Nucleic Acids Research",
        "authors": "Aartjan J. W. te Velthuis;Jamie J. Arnold;Craig E. Cameron;Sjoerd H. E. van den Worm;Eric J. Snijder",
        "doi": "10.1093/nar/gkp904"
    },
    {
        "keywords": "enteric pathogens;virulence;sand;feces;parks",
        "abstract": "The identification of pathogens of viral (Rotavirus, Coronavirus), parasitic (Toxocara spp.) and bacterial (Escherichia coli, Salmonella spp., Rhodococcus equi) origin shed in feces, and the virulence profile of R. equi and E. coli isolates were investigated in 200 samples of sand obtained from 40 parks, located in central region of state of Sao Paulo, Brazil, using different diagnostic methods. From 200 samples analyzed, 23 (11.5%) strains of R. equi were isolated. None of the R. equi isolates showed a virulent (vapA gene) or intermediately virulent (vapB gene) profiles. Sixty-three (31.5%) strains of E. coli were identified. The following genes encoding virulence factors were identified in E. coli: eae, bfp, saa, iucD, papGI, sfa and hly. Phylogenetic classification showed that 63 E. coli isolates belonged to groups B1 (52.4%), A (25.4%) and B2 (22.2%). No E. coli serotype O157:H7 was identified. Eggs of Toxocara sp. were found in three parks and genetic material of bovine Coronavirus was identified in one sample of one park. No Salmonella spp. and Rotavirus isolates were identified in the samples of sand. The presence of R. equi, Toxocara sp, bovine Coronavirus and virulent E. coli isolates in the environment of parks indicates that the sanitary conditions of the sand should be improved in order to reduce the risks of fecal transmission of pathogens of zoonotic potential to humans in these places.",
        "body": "Introduction\nEnteric pathogens are a major group of organisms related to infections in humans and animals. These pathogens are resistant to adverse environmental conditions and are frequently transmitted by oral route due to fecal contamination of foods, water, vegetables and fruits (Acha and Szyfres, 2003).Sandboxes are commonly found in parks all over the world, and are mainly used by children and adolescents. Access of companion animals, birds and occasionally livestock, inadequate hygiene of the sandboxes, precarious hygiene habits of children, and lack of knowledge about the risks posed by microorganisms eliminated in feces of animals favor the transmission of pathogens of animal origin to humans in these places (Matsuo and Nakashio, 2005).The present study investigated the presence of pathogens of viral (Coronavirus, Rotavirus), parasitic (Toxocara spp.) and bacterial (E. coli, Salmonella spp., R. equi) origin eliminated in feces of animals, and the virulence profile of R. equi and E. coli isolates, obtained from the sand of parks located in the central region of state of Sao Paulo, Brazil.\nMaterials and Methods\n\nCollection of samples\nTwo hundred samples of sand from 40 parks were analyzed. The strains were collected between 2008 and 2009. All parks were located in the central region of state of Sao Paulo, Brazil. After superficial dirt was removed, about 250 g of soil were collected 10&#8211;15 cm deep. Samples were placed in individual plastic bags and taken to laboratory under refrigeration (4&#8211;8 &#176;C).\nCulture, identification and storage of bacterial isolates\nAll samples were processed in the Laboratory of Microbiology and Infectious Diseases of Animals, Department of Veterinary Hygiene and Public Health, School of Veterinary Medicine, UNESP - Universidade Estadual Paulista, Botucatu, Sao Paulo, Brazil. Samples were kept under refrigeration (4&#8211;8 &#176;C) or frozen (&#8722;20 &#176;C) until they were analyzed.Samples (25 g) of feces from all parks were inoculated aseptically in 225 mL sterilized distilled water. After homogenization, 0.03 &#956;L of material was inoculated in defibrinated bovine blood agar (5%) and MacConkey agar for E. coli isolation. Plates were incubated at 37 &#176;C for three days and were assessed every day for bacterial growth. Simultaneously, 0.03 &#956;L of these samples were cultured in NANAT selective media for R. equi (Takai et al., 1996). Microorganisms were identified by colony morphology, staining methods, and biochemical tests (Quinn et al., 1994). Isolates of bacterial origin were stored in Ligni&#232;ris agar at 25 &#176;C.\nIdentification of \nBriefly, samples (25 g each) were inoculated into 250 mL of peptone water 1.0% (Oxoid) and incubated at 35 &#176;C for 24 h. Aliquots of 0.1 mL and 1 mL were inoculated each into 10 mL of Rappaport-Vassiliadis (RV) (Oxoid) and Tetrathionate (TT) (Oxoid) broth, respectively, and incubated at 42 &#176;C (RV) and 35 &#176;C (TT) for 24 h. A loopful of each broth culture was inoculated simultaneously in xylose-lysine-desoxycholate agar (XLD) (Oxoid) and bismuth sulfite agar (BS) (Oxoid), followed by incubation at 35 &#176;C for 24 h. Colonies suggestive of Salmonella were inoculated in triple sugar iron (TSI) and lysine iron (LIA) agar slants. Tubes were incubated for 35 &#176;C for 24 h. Colonies suggestive of Salmonella spp. in at least one of the culture media (TSI or LIA) were submitted to biochemical tests, agglutination test using polyvalent anti-Salmonella serum (Probac) (Quinn et al., 1994; Andrews et al., 1998), and serotype identification(Popoff and Le Minor, 1992).\nVirulence of \nIsolation of plasmid DNA was obtained by using an alkalin lysis method (Takai et al., 2003). Target DNA for PCR amplification was based on the genes encoding a 15&#8211;17 kDa antigen (vapA gene), and a 20 kDa antigen (vapB gene) sequence. Plasmid DNA was digested with restriction endonucleases (EcoRI, EcoT22I and HinddIII). Primer 1 (5&#8242;-GACTCTTCACAAGACGGT-3&#8242;) and primer 2 (5&#8242;-TAGGCGTTGTGCCAGCTA-3&#8242;) were used to detect virulent (vapA gene) strains and the 569-552 bp expected product. Primer 3 (5&#8242;-AACGTAGTCGCGGTGAG AA-3&#8242;) and primer 4 (5&#8242;-ACCGAGACTTGAGCGACTA-3&#8242;) were used for intermediately virulent (vapB gene) isolates to detect the 1066-1048 bp expected product. Samples were submitted to 30 cycles of amplification as follows: denaturation for 90 s at 94 &#176;C, annealing for 1 min at 55 &#176;C, and extension for 2 min at 72 &#176;C (Takai et al., 1996; Takai et al., 2003). Characterization of plasmid virulence was performed in Kitasato University, Japan.\nNone\nSorbitol-negative O157:H7 serotypes were submitted to agglutination test with O157 and H7 sera (Probac). Reference strains were E. coli FVL2 (sfa, pap, iucD, hly, cnf-1), FV35 (afa, iucD, cnf-1), J96 (papGII, papGIII), O157:H7 (vt1, vt2, eae), 2348/69 (eae, bfp, eaf), IANO (stb, lt1), EAEC O42 (eaec), B90 (cnf-2), FVL16 (cnf-1, hly, pap, sfa, iucD), ETEC13 (sta), EIEC (ipaH), supplied by the Laboratory of Bacterial Antigens, Campinas State University, Brazil. E. coli DH5&#945; strain was used as a negative control. First, primers for virulence factor genes were determined individually using a template DNA from appropriate positive and negative control strains. The presence of the following groups of genes were analyzed by PCR: papC and papG alleles (P fimbria), sfaC/D (S fimbria), afaB/C (afimbrial adhesin), saa (self-agglutinating adhesin), iucD (aerobactin), cnf-1 and cnf-2 (cytotoxic necrotizing factor type 1 and 2), hly (&#945;-hemolysin), vt1 and vt2 (verotoxins), sta and stb (heat stable toxins), lt1 (heat labile toxins), eaec (E. coli EAEC), ipaH (E. coli EIEC), and eae, eaf and bfp (E. coli EPEC). Appropriate primer sequences, annealing temperature, and size of amplified fragment (base pairs - bp) for these genes were determined in previous studies (Schmidt et al., 1995; Yamamoto et al., 1995; Blanco et al., 1996; Blanco et al., 1997; Karkkainen et al., 1998; Aranda et al., 2004; Villareal et al., 2006). Phylogenetic classification (chuA, yjaA, TspE4.C2) in groups A, B1, B2 and D was performed by PCR (Em&#246;dy et al., 2003).\nIdentification of \nFlotation-centrifugation with sodium nitrate (Na2NO3) 1.20 g/cm3 was used for the recovery of Toxocara spp. eggs. Centrifugation was performed at 2.500 rpm (679 g) for 5 min. After that, the supernatant of each tube was placed in microscope slides, covered with coverslips, and examined under a light microscope (10x). This process was repeated three times for each sample (Santar&#233;m et al., 1998).\nDiagnosis of bovine coronavirus (BCoV)\nSamples were tested for the presence of BCoV with a group II coronavirus specific RT-PCR assay targeted to the RNA-dependent RNA-polymerase gene (RdRp) with a 136-bp predicted product (Brand&#227;o et al., 2005). BCoV Kakegawa strain (Akashi et al., 1980) and PBS were used as positive and negative controls, respectively.\nDiagnosis of rotavirus\nSamples were analyzed for the presence of rotavirus 11-segment RNA using polyacrylamide gel electrophoresis-PAGE (Herring et al., 1982). NCDV group A rotavirus strain was used as the positive control.\nStatistical analysis\nChi-square test (Epi-Info, 6.4) was used to evaluate the differences in the presence of different pathogens in the parks, considering p <0.05 (Triola, 2005).\nResults\nThe frequency of pathogens identified in samples of sand obtained from parks is shown in Table 1. There was no statistical difference (p >0.05) between the presence of the different pathogens in the parks sampled.E. coli and R. equi strains were the most common pathogens isolated throughout the study. R. equi strains were isolated in 23 (11.5%) sand samples. None of the R. equi isolates showed virulent (vapA gene) or intermediately virulent (vapB gene) plasmid profiles. Sixty-three (31.5%) strains of E. coli were identified. The following virulence factor genes were identified in the E. coli strains: eae, bfp, saa, iucD, papGI, sfa and hly. Phylogenetic classification showed that the 63 E. coli isolates belonged to groups B1 (52.4%), A (25.4%) and B2 (22.2%). No E. coli serotype O157:H7 was identified (Table 2).Eggs of Toxocara spp. were recovered only in three of the parks.Genetic material of bovine Coronavirus was identified in one public park (Table 1), as suggested by the sequencing analysis of the 136 bp amplicon obtained in one sample (data not show). No Salmonella spp. or Rotavirus isolates were identified in the samples of sand.\nDiscussion\nRhodococcus equi is a well-recognized Gram positive intracellular bacterium associated with different clinical manifestations in humans and animals. The organism is widespread in soil, particularly in feces of foals, other herbivores and their environment (Prescott, 1991). The virulence mechanism of this pathogen is related with the presence of virulence-associated plasmids-Vap (Takai et al., 1991), and three levels of virulence are currently recognized: virulent, intermediately virulent and avirulent (Takai, 1997). Virulent R. equi strains contain a large plasmid of 85&#8211;90 kb, responsible for encoding the 15&#8211;17-kDa antigens (VapA) that are considered the major causes of suppurative pneumonia in foals (Ribeiro et al., 2005). VapB or intermediately virulent isolates present 20-kDa antigens and a 79&#8211;100 kb plasmid (Takai, 1997). They are frequently observed in swine lymphadenitis (Takai et al., 1996) and patients infected by acquired immunodeficiency syndrome-AIDS (Takai et al., 2003). In contrast, avirulent strains show no evidence of either vapA or vapB genes. These strains are found in the soil of areas where foals are raised, in soil and/or sand of human dwelling, mainly in yards and parks, and in humans with rhodococcosis co-infected by AIDS virus (Takai et al., 1996; Takai, 1997).All our R. equi strains were classified as avirulent. These results are in agreement with similar study in Japan, which also reported the presence of avirulent R. equi in the soil of parks and yards (Takai et al., 1996). Avirulent strains have been frequently identified in the environment of domestic animals, particularly foals (Takai, 1997). Currently, R. equi has emerged as a pulmonary pathogen among immunosuppressed patients, mainly those infected by AIDS virus (Acha and Szyfres, 2003). A recent survey of R. equi virulence profile in 20 humans in Brazil showed 11 patients infected with avirulent strains (Ribeiro et al., 2011). Plasmid virulence of R. equi strains isolated in Brazil was characterized in foals (Ribeiro et al., 2005) and a dog (Farias et al., 2007). The present study was the first investigation in this country about virulence profile of R. equi strains isolated from park sand. Beside the absence of virulent or intermediately virulent R. equi strains, the presence of this microorganism in the sand of parks constitutes a public health problem. This risk is particularly important to children and immunocompromised people, especially HIV-positive patients, because avirulent R. equi may cause the disease in immunosuppressed and non-immunosuppressed patients (Takai et al., 2003), including in Brazil (Ribeiro et al., 2011).E. coli is a very diverse species of bacteria found both in the intestinal tract of humans and animals, and in the environment. The microorganism is classified in six different pathotypes based on enteric manifestations, as follows: enterotoxigenic (ETEC), enteropathogenic (EPEC), enteroinvasive (EIEC), enterohemorrhagic (EHEC), enteroaggregative (EAEC), and diffusely adherent (DAEC). Pathogenic manifestations of E. coli are closely related with different virulence factors, including enterotoxins, cytotoxins, fimbriae, adhesins, and iron chelation mechanisms (Kaper et al., 2004).Gene eae encodes intimin, which mediates the intimate attachment of EPEC and EHEC to epithelial cells, and stimulates mucosal immune response and intestinal crypt hyperplasia. Gene eae and bfp were found in three strains isolated from the sand of parks in the present study, and are generally related with atypic enteropathogenic E. coli. This class of E. coli EPEC causes diarrhea in children younger one year of age, mainly in emerging countries (Kaper et al., 2004). In Brazil, there was a case of concurrent infection of a child and a dog by enteropathogenic E. coli that showed eae gene and was isolated from feces (Rodrigues et al., 2004). Atypic EPEC isolated from parks constitutes a public health risk, especially for children and immunosuppressed humans.P fimbriae are known to contribute for E. coli pathogenesis by promoting colonization of host tissues and stimulating injurious inflammatory response in the host (Kuehn et al., 1994). PapG adhesin is located on P fimbria. Three classes of PapG (PapGI, PapGII and PapGIII) are recognized: papG Class I are predominantly found in fecal strains; allele GII in strains involved in pyelonephritis and bacteremia cases; and allele GIII in isolates responsible for cystitis cases in humans and animals (Bergsten et al., 2005). S pillus is constituted of subunits: sfaS subunit mediates E. coli interaction with intestinal and other epithelial cells. sfaS gene is associated with human pyelonephritis, meningitis and sepsis (F&#233;ria et al., 2001). Haemolysin is a pore-forming cytotoxin that lyses erythrocytes, leukocytes, and endothelial and epithelial cells of mammals. The hly genes are frequently found in extraintestinal E. coli infections in humans and animals (Johnson et al., 1991). One of our isolates harbored the genes that encode papGI, sfa and hly. The identification of these virulence factors in a same isolate may be explained by the presence of a pathogenicity island (PAI), which enhances the infectivity of the microorganism. PAIs have been frequently found in E. coli responsible for human extraintestinal infections (Kurazono et al., 2000). In Brazil, genes that encode papG adhesins, as well as hly and sfa genes, were found in E. coli strains isolated from pyometra, urinary tract infections, and feces of dogs (Siqueira et al., 2009). Free access of dogs to parks increases the risk of human infection with virulent E. coli. These animals may act as reservoirs, harboring pathogenic strains with virulence factors such as papG, hly, and sfa genes.Iron is essential for bacterial metabolism. E. coli uses this ion for the transport and storage of both electrons and oxygen, and for DNA synthesis (Em&#246;dy et al., 2003). Growth of bacteria under restricted iron concentrations make them use successfully competitive mechanisms to obtain this ion from the host. Aerobactin is the most effective iron chelation system employed by E. coli for iron acquisition, mediated by iuc genes types A, B, C and D (Griffiths, 1997). In humans, this virulence factor is intimately associated with urinary infections and septicemia (Torres et al., 2001). The iucD genes were found in only one isolate of our study. Currently, iuc genes have been found in dogs with pyometra (Coogan et al., 2004), urinary tract infections, and in dog feces in Brazil (Siqueira et al., 2009). Like other virulence factors, the presence of iucD gene in E. coli strains isolated from sand represents a risk to the population visiting these parks.The presence of a self-agglutinating adhesin (saa) in E. coli has been previously described (Paton et al., 2001). Virulence of this adhesin to humans and domestic animals remains unclear. However, saa gene was found in 19.0% of E. coli strains obtained from the sand of parks in the current study. This result suggests that further studies should be carried out in order to investigate the role of this adhesin as an E. coli virulence factor.E. coli have been phylogenetically classified in four groups named A, B1, B2 and D. E. coli strains belonging to groups B2 and D are commonly pathogenic for humans and animals, whereas A and B1 are less pathogenic (Clermont et al., 2000). Based on phylogenetic systematics, E. coli isolates obtained from the sand of parks were classified in A and B1 groups. Although these groups are predominantly related with non-pathogenic E. coli strains, these results indicate fecal contamination of the environment.Toxocariasis is a cosmopolitan parasitic zoonosis. Toxocara spp. is one of the most common parasites of young dogs and cats. Eggs of the parasite are frequently shed in large amounts in the feces of companion animals. Toxocariasis in human occur by spread of the larvae, leading to different clinical forms of disease. Clinical manifestations involve serious neurological, ophthalmologic, pulmonary, and/or cutaneous signs (Acha and Szyfres, 2003). The presence of eggs of Toxocara spp. in the sand of parks have been reported in several countries (Dubn&#225; et al., 2007; Matsuo and Nakashio, 2005), including in Brazil(Santar&#233;m et al., 1998). Three parks had positive samples for eggs of this parasite. These results suggest environmental contamination by feces of companion animals and indicate risk of toxocariasis to humans that use these parks, particularly children.Coronavirus infections in animals were reviewed elsewhere (Brand&#227;o et al., 2001). In Brazil, previous studies have identified Coronavirus in feces of cattle and dogs with and without diarrhea (Brand&#227;o et al., 2005, 2007). Identification of bovine Coronavirus in the sand from parks in Brazil is uncommon, although it also represents fecal contamination of the environment.Rotavirus was detected in the feces of domestic animals (Rodriguez et al., 2004; Ruiz et al., 2009) and chickens (Villarreal et al., 2006) with and without diarrhea in Brazil. Likewise, different Salmonella spp. serotypes were detected in feces of livestock (Ribeiro et al., 2010), birds and chickens (Hofer et al., 1997) in this country. None of sand samples collected in our parks showed Rotavirus and Salmonella spp. In contrast, an epidemiological study involving human patients with salmonellosis in several European countries revealed that one the major risk factors for the disease was the access of children up to four years of age to the sand of parks (Doorduyn et al., 2006). These findings indicate that similar studies must be performed in other regions in Brazil in order to investigate the occurrence of Salmonella spp. and Rotavirus in the sand of parks. Despite the absence of Salmonella spp. and Rotavirus in the samples analyzed, these pathogens should be included in microbiological testing required to determine the sanitary conditions of the sand used in parks, as they may be shed in the feces of birds and domestic animals.The identification of R. equi, E. coli EPEC, bovine Coronavirus, and Toxocara spp. are indicators of fecal contamination of the sand of the parks sampled. Contamination may have been caused by feces from domestic animals (Takai et al., 1996), birds (Prescott, 1991), or contaminated shoes of people who visit these places. Our results suggest the need to introduce control measures to prevent contamination of the sand by pathogens eliminated in animal feces. In fact, the risks of the transmission of pathogens shed in animal feces in parks may reduce if access of domestic animals to these places is prevented, fecal material is daily removed from the sand, sand is periodically tested for sanitary quality and replaced with material of known origin, and people are continuously educated on hygiene habits before using parks.The presence of R. equi, E. coli EPEC, Toxocara spp. and bovine Coronavirus identified in parks studied indicates environmental contamination by microorganisms found in feces of domestic animals, birds, and/or contaminated shoes of people. These results represent a risk for the transmission of pathogens with zoonotic potential to humans in these places, particularly to children.",
        "pmid": "24294244",
        "title": "Identification of pathogens and virulence profile of Rhodococcus equi and Escherichia coli strains obtained from sand of parks",
        "journal_title": "Brazilian Journal of Microbiology",
        "authors": "M.C. Fernandes;S. Takai;D.S. Leite;J.P.A.N. Pinto;P.E. Brand\u00e3o;V.A. Santar\u00e9m;F.J.P. Listoni;A.V. Da Silva;M.G. Ribeiro",
        "doi": "10.1590/S1517-83822013005000044"
    },
    {
        "keywords": "",
        "abstract": "In 2012 a novel coronavirus, MERS-CoV, associated with severe respiratory disease emerged in the Arabian Peninsula. To date, 55 human cases have been reported, including 31 fatal cases. Several of the cases were likely a result of human-to-human transmission. The emergence of this novel coronavirus prompts the need for a small animal model to study the pathogenesis of this virus and to test the efficacy of potential intervention strategies. In this study we explored the use of Syrian hamsters as a small animal disease model, using intratracheal inoculation and inoculation via aerosol. Clinical signs of disease, virus replication, histological lesions, cytokine upregulation nor seroconversion were observed in any of the inoculated animals, indicating that MERS-CoV does not replicate in Syrian hamsters. ",
        "body": "Introduction\nIn June of 2012, a novel coronavirus, designated Middle East respiratory syndrome coronavirus (MERS-CoV) [1] and classified as a 2c betacoronavirus [2,3], was isolated from a patient with a fatal case of pneumonia and renal failure in Saudi Arabia [3]. To date, 55 human cases of MERS-CoV have been reported with 31 fatalities [4], including two fatal cases in Jordan in April of 2012 that were retrospectively detected. A cluster of cases in the UK in February of 2013 suggested human-to-human transmission in two cases with no travel history to the Middle East [4].Several 2c betacoronaviruses with high sequence identity to MERS-CoV have been detected in bats in Europe, Ghana and Mexico [5,6] suggesting that bats are the natural reservoir of MERS-CoV. It is currently unclear whether human cases were a result of direct zoonotic transmission from this reservoir to humans or whether an intermediate host was involved. In vitro data suggest that MERS-CoV has a broad host range [7], increasing the likelihood that an intermediate host was involved in amplifying or transmitting the virus from its natural reservoir to humans.In 2003 another betacoronavirus, SARS-CoV, caused a pandemic with approximately 8000 human cases and a case fatality rate of ~10% [8]. The emergence of MERS-CoV with a high case fatality rate and the potential to transmit between humans stresses the need for a small animal model to study the pathogenesis of this virus and to test the efficacy of potential therapeutic or prophylactic intervention strategies. In Syrian hamsters, SARS-CoV replicates extensively in the respiratory tract in the first week after intranasal inoculation with 103 TCID50 [9] and, depending on the SARS-CoV strain used, can cause mortality in a small subset of infected animals [10]. In the present study, we explored the suitability of the Syrian hamster as a small animal model for MERS-CoV isolate HCoV-EMC/2012 infection or disease.\nMaterials and Methods\nEthics Statement. All animal experiments were approved by the Institutional Animal Care and Use Committee of the Rocky Mountain Laboratories, and performed following the guidelines of the Association for Assessment and Accreditation of Laboratory Animal Care, International (AAALAC) by certified staff in an AAALAC-approved facility.The work with infectious MERS-CoV was approved under BSL3 conditions by the Institutional Biosafety Committee (IBC). Sample inactivation was performed according to standard operating procedures approved by the IBC for removal of specimens from high containment.Virus and cells. MERS-CoV isolate HCoV-EMC/2012 was propagated in VeroE6 cells in Dulbecco&#8217;s Modified Eagle Medium (DMEM; Sigma) supplemented with 2% fetal calf serum (Logan), 1 mM L-glutamine (Lonza), 50 U/ml penicillin and 50 &#181;g/ml streptomycin (Gibco). The HCoV-EMC/2012 virus stock was titrated by end-point titration in VeroE6 cells. VeroE6 cells were inoculated with tenfold serial dilutions of virus stock in DMEM supplemented with 2% fetal calf serum, 1 mM L-glutamine, 50 U/ml penicillin and 50 &#181;g/ml streptomycin. Five days after inoculation, cytopathic effect (CPE) was scored and the TCID50 was calculated from 10 replicates by the method of Spearman-Karber.Animal experiments. Three groups of 40 hamsters were inoculated with MERS-CoV isolate HCoV-EMC/2012; one group received 4x102 TCID50 via aerosols (see below), one group received 103 TCID50 intratracheally in a total volume of 300 &#181;l, and one group received 106 TCID50 intratracheally in a total volume of 300 &#181;l. Mock infected hamsters (four animals per time point) inoculated with DMEM intratracheally or via aerosols were included as controls for histopathology and cytokine analysis. Eight hamsters in each group and eight control hamsters (4 intratracheal and 4 aerosol) were injected with an IPTT-300 temperature transponder (BMDS) to monitor body temperature daily. Animals were weighed daily and observed for signs of disease. Nasal, oropharyngeal, urogenital and rectal swabs were obtained on 1, 2, 3, 4, 5, 6, 7, 9 and 11 days post inoculation (dpi) and analyzed for the presence of viral RNA. On 2, 4, 8, 14 and 21 dpi, eight hamsters in each group were euthanized and trachea, heart, lung, spleen, liver, kidney, ileum, colon, bladder, nasal turbinates and brain were collected for virological and histopathological analysis.Aerosol challenge. Hamsters were exposed to MERS-CoV isolate HCoV-EMC/2012 using a 38 liter, whole body exposure chamber within a Class II biological safety cabinet inside high containment. The animals received a single, 10 minute aerosol exposure and were contained using stainless steel, wire mesh cages (2 hamsters/cage). Anesthesia was not administered to the animals. The aerosol challenge consisted of 5 runs with each run containing 4 wire mesh cages. Viral inoculum and collection material were prepared the day of the exposure using DMEM and 1% fetal calf serum. Aerosol particles were generated by a 3-jet Collison nebulizer (BGI Inc., MA, USA) operating at 7.5 lpm (25-30 PSI) and ranged from 1&#8211;3 &#181;m in size. The contents of the aerosol exposure were collected in an All Glass Impinger (AGI, Ace Glass Inc., NJ, USA) continuously operating at 6.0 lpm. Both generator and sampler were flow checked using a frictionless bubble meter (Gilibrator, Sensidyne LP, FL, USA) to ensure the desired flow rates and pressure were achieved. In between aerosol runs a period of 5 minutes allowed the particles from the previous run to settle. The exposure chamber maintained negative pressure throughout the duration of the inoculation. An aerosol management platform (AeroMP, Biaera Technologies, USA) was used to conduct the exposures. The automated aerosol system controls, monitors, and maintains several parameters that impact aerosol studies [11]. Respiratory minute volume rates were determined using a formula derived by Alexander et al [12]. Weights were averaged for aerosol exposure group/run (n=8, n=40 total). A presented dose was calculated using the simplified formula D = R x Caero x Texp [11], where D is the presented or inhaled dose (expressed in TCID50), R is the respiratory minute volume (L/min), Caero is the aerosol concentration (TCID50/L), and Texp is duration of the exposure (min).Histopathology. Histopathology was performed on select hamster tissues. After fixation for 7 days in 10% neutral-buffered formalin and embedding in paraffin, tissue sections were stained with hematoxylin and eosin (H&E). For the histopathological analysis of the nasal turbinates whole hamster skulls were used. The skulls were decalcified using a 20% EDTA solution in sucrose (Newcomer Supply) and allowed to sit at room temperature for 3 weeks. The 20% EDTA/sucrose solution was changed once prior to mid-sagittal sectioning of the skull.Immunohistochemistry. Immunohistochemistry was performed on lungs, kidney, small intestine, urinary bladder and colon of normal hamsters using an &#945;-DPP4 antibody (&#945;-CD26; 1:800; Abcam). The tissues were processed for immunohistochemistry using the Discovery XT automated processor (Ventana Medical Systems) with a DapMap kit (Ventana Medical Systems).RNA extraction. RNA was extracted from swab samples using the NucleoSpin 96 Virus Core kit (Macherey-Nagel) and a Corbett Robotics model CAS 1820 automatic RNA extractor. RNA was eluted in 100 &#181;l. RNA was extracted from whole blood using the QiaAmp Viral RNA kit (Qiagen) according to the manufacturer&#8217;s instructions. RNA was eluted in 60 &#181;l. Tissues (30 mg) were homogenized in RLT buffer and RNA was extracted using the RNeasy kit (Qiagen). RNA was eluted in 50 &#181;l.Quantitative PCR. For detection of viral RNA, 5 &#181;l RNA was used in a one-step real-time RT-PCR upE assay [13] using the Rotor-GeneTM probe kit (Qiagen) according to instructions from the manufacturer. In each run, standard dilutions of a titered virus stock were run in parallel, to calculate TCID50 equivalents in the samples. Hamster Mx2 gene expression was determined as described previously [14]. qRT-PCR was performed as described above using Mx2 specific primers. The fold-change of each gene was calculated by normalizing the change in CT (&#916;CT) to the CT values for RPL18 as an internal reference gene for each sample and comparing this to the CT values of mock inoculated hamsters (2-&#916;&#916;CT).ELISA. Immuno-globulin G antibody responses were measured in an enzyme-linked immunosorbent assay (ELISA) using MERS-CoV isolate HCoV-EMC/2012. HCoV-EMC/2012 containing cell culture supernatant was concentrated and purified by spinning two hours at 21000 rpm over a 15% OptiPrep (Axis-Shield) cushion. The pellet was resuspended in PBS and triton X-100 was added to a final concentration of 1% and the suspension was then gamma-irradiated before removal from high containment. This suspension was then used to coat immuno 96 microwell maxisorp plates (NUNC) at 4&#176;C overnight. Subsequently, plates were blocked with 5% skim milk in PBS containing 0.05% Tween 20 (PBST) for 1.5 hours at 4&#176;C. After 3 washes with PBST, 50 &#181;l of diluted serum samples were added, and the plates were incubated for 1 hour at 37&#176;C. Bound antibodies were detected after 3 washes using an anti-hamster secondary antibody conjugated with horseradish peroxidase (HRP; KPL). Following incubation for 1 hour at 37&#176;C, bound HRP was detected using the ABTS&#174; Peroxidase Substrate System (KPL). The absorbance at 405 nm was measured using a microplate spectrophotometer. Sera were considered positive when absorbance was higher than three standard deviations above the mean of negative control sera. Sera obtained from rabbits immunized with HCoV-EMC/2012 were used as a positive control.\nResults & Discussion\nHamsters were divided into 3 groups of 40 hamsters. The first group was inoculated intratracheally with 103 TCID50 of MERS-CoV isolate HCoV-EMC/2012; the second group was inoculated intratracheally with 106 TCID50 of HCoV-EMC/2012 and the third group was inoculated with 4x102 TCID50 HCoV-EMC/2012 via aerosolization. Hamsters were observed for clinical signs of disease daily for 21 dpi and body weight and body temperature were measured. None of the hamsters in the three inoculated groups showed signs of disease, weight loss or increased body temperature (Figure 1). Nasal, oropharyngeal, urogenital and rectal swabs were obtained daily between 1 and 11 dpi and were all negative by qRT-PCR. Upon necropsy on 2, 4, 8, 14 and 21 dpi, no gross lesions were observed. Lungs, spleen and mandibular lymph nodes collected on 2, 4 and 8 dpi were analyzed for the presence of HCoV-EMC/2012 vRNA by qRT-PCR and found to be negative. Trachea, heart, lung, spleen, liver, kidney, ileum, colon, urinary bladder, nasal turbinates and brain collected on 2, 4, 8, 14 and 21 dpi were used for histopathological analysis; no lesions were observed that could be attributed to the virus in any of the tissues examined (Figure 2).Hamsters were mock inoculated (&#9679;), inoculated intratracheally with 103 TCID50 HCoV-EMC/2012 (&#9632;), inoculated intratracheally with 106 TCID50 HCoV-EMC/2012 (&#9650;) or inoculated with 4x102 TCID50 via aerosols (&#9660;) and body weight (A) and temperature (B) were measured. Average and standard error of the mean are plotted for 8 animals per time point.Hamsters were mock inoculated (A and E), inoculated intratracheally with 103 TCID50 HCoV-EMC/2012 (B and F), inoculated intratracheally with 106 TCID50 HCoV-EMC/2012 (C and G) or inoculated with 4x102 TCID50 via aerosols (D and H). On 4 days post inoculation tissue sections of the lungs (A&#8211;D) and kidneys (E&#8211;H) of these animals were collected and stained with hematoxylin-eosin. The panels shown in this figure are presentative for 4 animals analyzed per tissue per group.Lungs, spleen and lymph nodes collected from the hamsters intratracheally inoculated with 106 TCID50 and inoculated via aerosols on 2, 4 and 8 dpi were also analyzed for upregulation of the transcription of the Mx2 gene. Mx2 gene expression was analyzed as an indicator of an innate immune response to virus infection, since Mx2 is downstream of STAT1 and as such is an indicator of a type I or type III IFN response (reviewed in 15. A statistically significant upregulation of Mx2 gene expression could not be detected in any of the hamsters, independent of the inoculation route, inoculation dose or tissue analyzed (Figure 3). Taken together these data indicate that the hamsters were either not infected at all or the infection occurred below the limit of detection of our assays. To determine whether the animals were infected at all, we developed an ELISA assay to detect antibodies against MERS-CoV. Hamster sera collected on 21 dpi were analyzed for the presence of antibodies; there was no evidence for seroconversion in any of the inoculated animals, regardless of inoculation route or dose. Thus, the MERS-CoV does not replicate in Syrian hamsters and they are an unsuitable model for studies into pathogenesis or potential prophylactic or therapeutic intervention strategies. To determine whether the recently described receptor for MERS-CoV, DPP4 [16], is expressed in the Syrian hamster, we performed immunohistochemistry on lung, kidney, small intestine, urinary bladder and colon of normal hamsters using an &#945;-DPP4 antibody. DPP4 could be detected in all tested hamster tissues (Table 1) and was abundantly present in the lungs and kidneys of Syrian hamsters (Figure 4). The abundant expression of DPP4 on bronchiolar epithelium and the occasional presence on type I pneumocytes indicates that the absence of replication in the Syrian hamster is not due to a lack of the receptor, but rather to the fact that the epitope that MERS-CoV binds to is not conserved in Syrian hamster DPP4 or that other restrictions exist on the cellular level that prevent virus replication.HCoV-EMC/2012. A qRT-PCR assay to detect Mx2 was performed on RNA isolated from lungs, spleen and mandibular lymph nodes collected on 2, 4 and 8 days post inoculation (dpi) of Syrian hamsters with 106 TCID50 of HCoV-EMC/2012 (green bars) via the intratracheal route (left panels) or 4x102 TCID50 of HCoV-EMC/2012 via aerosol (right panels, green bars) and compared to mock inoculated animals (blue bars). Data are shown as the fold change of Mx2 over uninfected controls and normalized to an internal reference gene (RPL18). Means were calculated from 4 mock animals and 4 (2 dpi) or 8 (4 and 8 dpi) inoculated animals. Error bars represent standard deviation.Immunohistochemistry was performed on Syrian hamster lung and kidney tissues using an &#945;-DPP4 antibody. In the lung (A and B), DPP4 was abundantly present on bronchiolar epithelium (blue arrow) and smooth muscle cells (black arrows) and was occasionally present on type I pneumocytes (black arrowhead and inset in panel B). In the kidney, DPP4 was present on glomerular parietal epithelium (white arrowhead) and in nerve tissue (purple arrow).Although these experiments did not result in a small animal model for MERS-CoV, they do provide insight into the potential host range of this virus.Given the continuing circulation of the MERS-CoV virus and the associated high case fatality rate, the search for an animal infection/disease model is of utmost importance for our understanding of the pathogenesis of this virus and for the development of effective countermeasures.",
        "pmid": "23844250",
        "title": "The Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Does Not Replicate in Syrian Hamsters",
        "journal_title": "PLoS ONE",
        "authors": "Emmie de Wit;Joseph Prescott;Laura Baseler;Trenton Bushmaker;Tina Thomas;Matthew G. Lackemeyer;Cynthia Martellaro;Shauna Milne-Price;Elaine Haddock;Bart L. Haagmans;Heinz Feldmann;Vincent J. Munster",
        "doi": "10.1371/journal.pone.0069127"
    },
    {
        "keywords": "MERS-CoV;Coronavirus;Middle East;Animal;Dromedary;Camels;Bats;Middle East Respiratory Syndrome",
        "abstract": "Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans. Though not confirmed yet, multiple surveillance and phylogenetic studies suggest a bat origin. The disease is heavily endemic in dromedary camel populations of East Africa and the Middle East. It is unclear as to when the virus was introduced to dromedary camels, but data from studies that investigated stored dromedary camel sera and geographical distribution of involved dromedary camel populations suggested that the virus was present in dromedary camels several decades ago. Though bats and alpacas can serve as potential reservoirs for MERS-CoV, dromedary camels seem to be the only animal host responsible for the spill over human infections.",
        "body": "Background\nA novel coronavirus was isolated from a Saudi Arabian patient with severe acute respiratory syndrome in June 2012 and the virus was later named Middle East Respiratory Syndrome - Coronavirus (MERS-CoV) [1]. Since then, multiple outbreaks have been reported in or been epidemiologically linked to the Arabian Peninsula. Up to April 18, 2016, the Saudi authorities reported a total of 1386 cases of which 587 died [2]. With a high mortality rate, lack of antiviral treatment, or preventive vaccine, MERS-CoV remains a major public concern.The first human coronavirus was cultivated in 1965 on human ciliated embryonal tracheal cells [3]. Literature about human coronaviruses was limited up until the Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) outbreak in 2002. The interest in coronaviruses reemerged as a hot topic after MERS-CoV outbreaks. Coronaviruses are enveloped, positive stranded RNA viruses classified as a family within the Nidovirales order. They have a crown appearance under the electron microscopy due to the spike protein on the surface. The coronavirus subfamily is classified into genera: alpha, beta, gama, and delta. Human coronaviruses belong to alpha, and beta coronaviruses [4].MERS-CoV is a lineage C betacoronavirus with a genotype that is very closely related to bat coronaviruses from the same lineage, such as BtCoV-HKU4 and BtCoV-HKU5, though its evolutionary pathway is still unclear [5].A limited number of coronaviruses is known to cause human disease. Most of known coronaviruses infect and circulate in animals, mainly bats. So when it comes to a new novel coronavirus with limited geographic distribution, one would think of a zoonotic disease with animal reservoir. This proved to be true with SARS and seems to be the case with MERS-CoV. The origin of the virus and the extent of its involvement in both human and animal populations remain hot topics that are being explored with phylogenicity and surveillance studies. In this review we summarize the current evidence about MERS-CoV origin and animal reservoir.\nMERS-CoV origin\nIt is believed that MERS-CoV, like many other coronaviruses, originated in bats. This is based on the isolation of other lineage C beta-coronaviruses that are very closely related to MERS-CoV on phylogenetic analysis [6, 7].A large screening study for beta-coronaviruses from 5030 bats&#8217; fecal specimens was conducted between 2009 and 2011. In that study 4758 bats of ten different species from Ghana, and 272 Pipistrellus bats from four European countries were included [8]. Reverse Transcription Polymerase Chain Reaction (RT-PCR) was used to detect coronavirus RNA. Of the ten bat species tested in Ghana, only Nycteris gambiensis was found to carry 2c beta-coronavirus. A round one fourth of the 185 Nycteris bats tested positive; this accounted for 1&#160;% of whole tested bat&#8217;s population from Ghana. Of the Pipistrellus bat species tested in Europe, 40 out of 272 (14.7&#160;%) carried 2c beta-coronavirus. Both 2c beta-coronaviruses isolated are genetically very closely related to MERS-CoV. This relatedness indicates that MERS-CoV likely originated from bats [8]. Though the study had a good sample size and screened different geographical areas, none of those areas is known for MERS-CoV infection in humans or domestic animals.A report from South Africa identified a bat derived coronavirus that has a very close phylogenetic relationship to MERS-CoV [9]. The virus was isolated from a Neoromicia cf. zuluensis bat sampled in 2011 [9]. In Saudi Arabia, bat fecal samples collected in October of 2012 were tested for MERS-CoV RNA. A product obtained by PCR amplification of nucleic acid from a fecal pellet of a Taphozous perforatus bat captured in Bisha showed 100&#160;% nucleotide match to the MERS-CoV cloned from an index case patient living in the same area [10]. Though it was a single 190-nucleotide segment, it does give a clue that MERS-CoV might be circulating in bats and that Taphozous perforatus (Egyptian tomb bat) might be a source of MERS-CoV.A study of 821 bats captured in Egypt and Lebanon between February 2013 and April 2015 did not detect MERS-CoV in any of the bats sampled [11]. In that study three species: Taphozous perforatus, Pipistrellus deserti, and Rousettus aegyptiacus, were sampled from Egypt, and four species: Rhinolophus hipposideros, Miniopterus schribersii, Rhinolophus ferrumequinum, and Rousettus aegyptiacus, were sampled from Lebanon [11]. Serum samples, oral and rectal swabs were collected and tested from live bats. Homogenized lung and liver material was tested from 72 bats that died or were euthanized upon capture [11]. One limitation of this study is that most of the bats screened belong to one species, R. aegyptiacus (85&#160;%), and limited numbers of the other species were tested, especially T. perforatus (10&#160;%).To further investigate the role of bats as a potential reservoir for MERS-CoV, an experimental study was conducted on Jamaican fruit bat (Artibeus jamaicensis). Ten bats were inoculated through intranasal and intraperitoneal routes with MERS-CoV [12]. All bats showed evidence of infection as they shed the virus from their respiratory and, to a lesser extent, intestinal tract, but none of the bats showed clinical signs of disease [12]. The ability of MERS-CoV to replicate in bats without clinical signs of disease indicates that they can serve as a reservoir for MERS-CoV.Thus, MERS-CoV, or its immediate ancestor, could have likely originated in bats, and that bats can serve as an ideal reservoir for MERS-CoV. It is difficult to proof that bats are the direct source of human disease, based on the available bats&#8217; screening studies. In addition, here is no clear direct contact between bats and humans, especially in Saudi Arabia where most of the cases are being diagnosed. A summary of important studies that screened bats for beta-coronaviruses and MERS-CoV is shown in Table&#160;1.Table 1Summary of important studies that screened bats for beta-coronaviruses and MERS-CoVLocationYearSpeciesNumberSpecimenVirus% PositiveGhana [8]2009&#8211;2011\nNycteris cf. gambiensis\n185Fecal2c betacoronaviruses(closely related to MERS-CoV)24.9&#160;%Ghana [8]2009&#8211;20119 different speciesa\n4573Fecal0&#160;%Europe [8] (Germany, Netherland, Romania, Ukraine)2009&#8211;2011\nPipistrellus kuhlii, P. nathusii, P. pipistrellus, P. pygmaeus\n272Fecal14.7&#160;%South Africa [9]2011&#8211;201213 different speciesb\n62Fecal pelletsbat related-alphacoronaviruses betacoronavirus6.4 % 1.6 %Saudi Arabia [10]2012\nRhinopomahardwickii, R.microphyllum, Taphozous perforatus, P. kuhlii, Eptesicus bottae, Eidolon helvum, and Rosettus aegyptiacus\n96Throat swab, serum, urine, rectal swab or fecal pelletsMERS-CoV1&#160;%Saudi Arabia [10]2013\nR.hardwickii, T.perforates, P.kuhlii\n14Throat swabs, roost fecesMERS-CoV0&#160;%Egypt [11]2013&#8211;2015\nT. perforatus\n82Serum/rectal (alive)MERS-CoV0&#160;%\nP. deserti\n31\nR. aegyptiacus\n257Lebanon [11]2013&#8211;2015\nR. hipposideros\n4Homogenized lung and liver material (if died or euthanized upon capture)0&#160;%\nMiniopterus schribersii\n6\nR. ferrumequinm\n3\nR. aegyptiacus\n438\naColeura afra, Hipposiderosabae, H. cf. gigas, H. fuliginosus, H. jonesi, H. cf. ruber, Rhinolophus alcyone, R. landeri, Taphozous perforates\nb\nChaerephonpumilus, Mops condylurus, Tadaridaaegyptiaca, H.caffer, Miniopterus natalensis, Nycteristhebaica, R.clivosus, R. darlingi, Neoromicia capensis, N. nana, N. cf. zuluensis, Scotophilus viridis, Rousettus aegyptiacus\n\nAnimal reservoir\n\nDoes MERS-CoV infect dromedary camels? Is there a role for dromedary camels in human disease?\nThe first evidence to link MERS-CoV to dromedary camels came from a serological study that investigated different animals: dromedary camels, cattle, sheep, goats and various other camelid species. MERS-CoV specific antibodies were only found in dromedary camels [13].Another evidence to link MERS-CoV to dromedaries was found after two human cases of MERS-CoV infection, diagnosed in October of 2013, and were linked to a farm in Qatar [14]. In response, all the 14 dromedary camels on that farm were tested with RT-PCR. Eleven dromedary camels had positive nasal swabs for MERS-CoV. The nucleotide sequence of an ORF1a fragment and a 4&#8201;&#183;&#8201;2&#160;kb concatenated fragment of three dromedary camel samples were very similar to the sequence from the two human cases linked to that farm [14]. Another study from Saudi Arabia described a 43&#160;year old male who owned nine dromedary camels and was in direct contact with them up until he was diagnosed with MERS-CoV infection in November 2013 [15]. Four of his dromedary camels were sick before his symptoms started. Cell cultures from a laboratory confirmed dromedary camel and the patient grew genetically identical MERS-CoV viruses [15].In addition to providing a virological confirmation of MERS-CoV in dromedary camels, the last two studies indicated a potential cross infection between dromedary camels and humans and that the virus could be transmitted from dromedary camels to humans through close contact [14, 15]. A study that obtained the full genome of MERS-CoV from a dromedary camel in Qatar showed that MERS-CoVs from dromedary camel and humans are nearly identical [16].A nation-wide cross-sectional serological study done in Saudi Arabia between December 1st 2012 and December 1st 2013 in which serum samples from slightly over ten thousand individuals, whom age and sex distribution largely matched the general population, were tested for MERS-CoV antibodies [17]. Positive results were confirmed in 15 individuals (0.15&#160;%). Of note, seropositivity was significantly higher in dromedary camel exposed individuals compared to the rest of the study population; fifteen times higher in shepherds and twenty three times higher in slaughterhouse workers [17]. This study does not only support the theory of dromedary camel role in human infection but also indicates a higher prevalence of human seropositivity than expected, based on reported cases from Saudi Arabia.In another study, 498 serum samples obtained from Qatar and Europe between 2013 and 2014 were tested for MERS-CoV antibodies. 294 samples were obtained from persons with occupational dromedary camel contact in Qatar; the remaining 204 samples were obtained from persons with no dromedary camel contacts from both Qatar and Europe. Around 7&#160;% of those with dromedary camel contact tested positive for MERS-CoV antibodies, while none of the samples from persons with no dromedary camel contacts tested positive [18].A case control study assessed differences in environmental exposure between thirty laboratory confirmed primary MERS-CoV cases and 116 controls. The study was conducted between March and November of 2014, and the cases represented 8 out of 13 regions in Saudi Arabia. The study concluded that direct dromedary camel exposure within two weeks of symptoms onset is independently associated with MERS-CoV infection [19].So the evidence points to a close dromedary camel contact as a major risk for human infection. In order to better understand the ecology of MERS-CoV infection in dromedary camels, three adult dromedary camels were inoculated with a human isolate of MERS-CoV. Each dromedary camel was inoculated though intratracheal, intranasal and conjunctival routes. Transient, primarily upper respiratory tract infection developed in each of the three camels. Each dromedary camel shed large quantities of virus from the upper respiratory tract, evidenced by the presence of infectious viruses in nasal swab samples and, to a lesser extent, in oral samples. No infectious virus or viral RNA was detected in fecal, urine, serum or whole blood samples [20]. After authentication, the virus was recovered from respiratory tissues and some lymph nodes draining the respiratory system of one dromedary camel, but not in any other organs. These findings were consistent with the data from naturally infected dromedary camels [6, 15, 21, 22]. Some studies reported the detection of MERS-CoV virus from fecal specimen, though virus detection rates were far lower than in nasal swab specimens [22&#8211;24]. One study in particular reported detection of MERS-CoV RNA in the milk of five out of seven infected dromedary camels [23].A vaccine expressing the MERS-CoV spike protein was shown to confer mucosal immunity in dromedary camels with evidence of serum neutralizing antibodies and significant reduction in excreted infectious virus and viral RNA transcripts in vaccinated animals [25].Based on surveillance and epidemiological studies, it is obvious that MERS-CoV infects dromedary camels, which serve as a reservoir with spell over human infections through close dromedary camel contacts [13&#8211;15, 17&#8211;19, 24]. The exact routes of transmission are not very well understoodbut direct contacts with Dromedary camels or the handling of surfaces or objects contaminated with dromedary camels&#8217; respiratory or fecal material may as well pose a risk for infection.\nWhat is the extent of MERS-CoV infection in dromedary camels?\nMultiple surveillance studies explored the extent of MERS-CoV infection in dromedaries. Presence of specific MERS-CoV antibodies in dromedary camels&#8217; sera was used as an indicator of previous exposure to the virus, while the presence of MERS-CoV RNA material in nasal secretions, usually identified through RT-PCR, indicated current infection and active viral shedding.Serum samples from 303 dromedary camels from Saudi Arabia were screened in 2013 and found to have a high seropositivity of 72&#160;% to MERS-CoV [21].All serum samples from 50 dromedary camels in Oman were positive for MERS-CoV specific antibodies [13]. Similar results were reached from a larger study conducted in United Arab Emirates (UAE), where 500 dromedary camels&#8217; sera screened in 2013 revealed 96&#160;% seropositivity [26].In Africa, a study assessed the geographic distribution of MERS-CoV among dromedaries by investigating serum samples from Nigeria, Tunisia, and Ethiopia. In Nigeria, serum samples collected between 2010 and 2011 from 358 adult dromedaries distributed over 4 provinces were tested, and 94&#160;% were positive for MERS-CoV antibodies. In Tunisia, serum samples collected in 2009 and 2013 from 204 dromedaries distributed over 3 provinces were tested and 48.5&#160;% were positive for MERS-CoV. In Ethiopia, 96.3&#160;% of the serum samples collected between 2011 and 2013 from 188 dromedaries, distributed over 3 provinces, were positive for MERS-CoV antibodies [27].Another study was conducted in Kano, Nigeria in January 2015. Nasal swabs and blood samples were collected from dromedary camels shortly after slaughter at a slaughterhouse. The samples were tested for the presence of MERS-CoV RNA using RT-PCR from nasal swabs. Sera were tested for the presence of specific MERS-CoV antibodies. Of the 132 dromedary camels screened with nasal swabs, 14 were found to carry the virus RNA (11&#160;%). The overall seropositivity was 95&#160;% [28].A random group of 105 dromedary camels presented for slaughter in Qatar at two occasions in 2014 were sampled for MERS-CoV. Nasal, oral, bronchial and rectal swabs were tested for MERS-CoV. A high proportion (59&#160;%) of them was actively shedding the virus at the time of slaughter. The percentage of positive samples was the highest for nasal samples, followed by oral swabs, fecal swabs, and bronchial swabs. Co-circulation of multiple MERS-CoV variants, evidenced by five different sequence types, demonstrates multiple virus introductions likely related to the flow of new dromedary camels from different origins [24].Those findings indicate a widespread exposure to MERS-CoV in the dromedary camel populations of Africa and the Arabian Peninsula, to the extent that almost every adult dromedary camel got infected at some point in life.\nDo dromedary camels differ in their vulnerability to MERS-CoV infection based on age?\nTwo hundred and three samples from live dromedary camels in Saudi Arabia were collected in 2013 and found to have a high seropositivity (72&#160;%) to MERS-CoV [21]. Seropositivity was higher among adults dromedary camels (two years and older) compared to juvenile dromedary camels (less than two years of age), 95&#160;% vs. 55&#160;% respectively [21]. Increasing seropositivity with age might be related to increased likelihood of exposure and subsequent infection over time.In the same study, 202 dromedary camels&#8217; nasal swabs were tested for the presence of MERS-CoV RNA material using RT-PCR; 25&#160;% were positive. In other words, one fourth of the tested dromedary camel population was shedding the virus and was potentially infectious. Of those shedding the virus 71&#160;% were juvenile and 29&#160;% were adult dromedary camels older than two years [21]. The results indicate that juvenile dromedary camels might be at higher risk of contracting the virus. An explanation could be that they are naive to the virus and lack neutralizing antibodies.\nWhen was the virus introduced to dromedary camels?\nIn an attempt to investigate the time frame of MERS-CoV introduction to dromedary camel population, multiple studies screened stored serum samples. 100&#160;% seropositivity was found in 151 dromedary camel serum samples obtained in 2003 from UAE dromedary camels [26]. Archived serum samples, obtained from dromedary camels in Saudi Arabia from1992 through 2010, were found to have a high seropositivity ranging from 72&#160;% to 100&#160;% [21]. 189 stored dromedary camel serum samples from Egypt, collected in 1997, and from Sudan and Somalia, collected between 1983 and 1984, were tested, and 81&#160;% were found to have neutralizing antibodies to MERS-CoV [29]. Another study from Kenya showed similar results [30]. This implies that MERS-CoV has been heavily endemic in both the Middle East and East Africa&#8217;s dromedary camels for decades.In addition to evaluating MERS-CoV distribution and the infection burden in dromedary camel populations, screening different geographic areas might also help predict when MERS-CoV was introduced to dromedary camels. Historically and up to the early twentieth century, both dromedary and bactrian camels played a vital role in carrying passengers and goods in North Africa, the Middle East, and East Asia. For the same reason, they were imported to Australia in the nineteenth century. After the invention of automobile and the use of trains, use and travel of dromedary and bactrian camels between countries was limited.Serum samples collected from 105 dromedary camels living in the Canary Islands, a Spanish archipelago located just off the southern coast of Morocco, between 2012 and 2013 were tested, and 14&#160;% were found to have antibodies against MERS-CoV [13]. This indicates that a small proportion of Canary Island dromedary camels were exposed to the virus at some point in time. This low percentage, compared to East Africa and the Arabian Peninsula, might be related to the isolated nature of the Canary Islands, as they are off the West African coast and are governed by Spain. Those two factors might limit the interaction of the island&#8217;s dromedary camels with those from other parts of Africa. This theory was later supported by another study conducted in early 2015. In this study, a representative sample of 170 dromedary camels on the Canary Islands was investigated; only 4.1&#160;% were seropositive for MERS-CoV. All the seropositive dromedary camels were imported from Africa 20 or more years prior. This led to the conclusion that active infection and shedding didn&#8217;t take place on the Canary Islands [31].In an attempt to screen feral camels in Australia, 307 dromedary camels&#8217; sera from two different locations were sampled between December 2013 and June 2014. All tested negative for specific MERS-CoV antibodies [32].In Kazakhstan 550 camels&#8217; sera: 455 dormeday and 95 bactrian, camels with two humps (Camelus bactrianus), were screened for MERS-CoV between February and March 2015, and all tested negative [33]. Another surveillance study was conducted in southern Mongolia in November of 2014. In that study, 210 bactrian camels from 12 herds were screened. All samples tested negative, and the study concluded that MERS-CoV was not present in bactrian camels of the screened areas [34].A serological and virological surveillance study of ten bactrian camel herds in three areas of the West of Inner Mongolia Autonomous Region (IMAR) was conducted in 2015. One hundred and ninety Bactrian camels were sampled; nasal swab and serum samples were collected from each Bactrian camel. All samples turned negative and the study concluded that there was no MERS-CoV circulating among Bactrian camels in the West IMAR [35].Based on the above studies, it seems that MERS-CoV infection is limited to dromedary camel populations in Africa and the Middle East. This may indicate that dromedary camel infection could have been introduced at a point in time when the dromedary camel caravans between Asia, Middle East, and Africa were no longer active.\nAre there other domestic animal reservoirs for MERS-CoV?\nIn an attempt to investigate the possibility of other reservoirs, multiple domestic animal populations were screened. In a serological study, conducted in Saudi Arabia between 2010 and 2013, sera from 100 sheep, 45 goats, 50 cattle, and 240 chickens representing different geographical areas within the country were collected. All samples tested negative for MERS-CoV [36]. In another study, serum samples from 36 goats and 102 sheep from central Saudi Arabia tested negative for MERS-CoV antibodies as well [21]. In 2013 a similar study was conducted in Jordan. None of the 150 goats, 126 sheep and 91 cows tested was found positive for MERS-CoV antibodies [37]. Sera from European sheep, goats, and cattle had no evidence of MERS-CoV antibodies as well [13].Equids were also screened for MERS-CoV antibodies. 192 adult horse samples from UAE and 861 samples from Spain (697 horses, 82 donkeys, and 82 mules) were screened. All were seronegative for MERS-CoV [38].Dromedary camels seemed to be the only domestic animal reservoir for MERS-CoV up until a recent study conducted in Qatar in April of 2015 investigated the MERS-CoV infection status of 15 healthy alpacas (Vicugna pacos) in a herd of 20 that shared a barn complex with dromedaries. All tested alpacas were seropositive to MERS-CoV (100&#160;%). Of note, MERS-CoV is endemic in Qatar&#8217;s dromedaries, and nine out of ten dromedary camels that shared the same barn complex were seropositive for MERS-CoV [39]. This indicates the susceptibility of alpacas for natural MERS-CoV infection and the potential for a new MERS-CoV animal reservoir. Of note, a previous study found no evidence for MERS-CoV infection in alpacas from regions where MERS-CoV is not endemic [13].In another study 3 alpacas were experimentally infected through intranasal insulation of MERS-CoV viruses. All got infected and shed the virus; they also transmitted the infection to two out of three other alpacas that shared the same room. Similar to dromedary camels, infected alpacas didn&#8217;t develop fever, but unlike dromedary camels, none of the alpacas had any observable nasal discharge over the course of infection. All infected animals were able to mount neutralizing antibodies to MERS-CoV [40].Those studies indicate that alpacas, similar to dromedary camels, can be infected and potentially serve as a reservoir.\nConclusion\nOur experience with MERS-CoV is of zoonotic nature, transmitted to humans from infected dromedary camels. The origin of MERS-CoV viral infection is not very well understood. It could have originated in bats and transmitted to dromedary camels at some unknown time in the past.The virus seems to be well maintained in dromedaries, which serve as a reservoir with a spill over human infections. Sporadic human cases in areas where MERS-CoV is endemic in dromedary camels are likely to continue to happen.The awareness of the disease and the easy access to a more developed health care system could explain the higher incidence of MERS-CoV diagnosis in Saudi Arabia compared to other countries in Africa where the disease is likely to be overlooked.Larger scale serological screening of human populations in areas where MERS-CoV is endemic in dromedary camels should be considered. More extensive screening of bats in Saudi Arabia and East Africa, especially the Egyptian tomb bat, needs to be considered. Screening dromedary camel populations in Africa (Sahara desert and surrounding areas), and East Asia (Pakistan, Afghanistan, and Iran) will help better delineate the geographical distribution of dromedaries involvement.Experimental MERS-CoV inoculation of other domestic animals will help define predisposed groups and should be considered so as to guide screening efforts for other potential reservoirs.\nAbbreviations\nIMAR, Inner Mongolia Autonomous Region; MERS-CoV, Middle East Respiratory Syndrome - Coronavirus; RT-PCR, Reverse Transcription - Polymerase Chain Reaction; SARS-CoV, Severe Acute Respiratory Syndrome - Coronavirus; UAE, United Arab Emirates",
        "pmid": "27255185",
        "title": "Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir",
        "journal_title": "Virology Journal",
        "authors": "Hamzah A. Mohd;Jaffar A. Al-Tawfiq;Ziad A. Memish",
        "doi": "10.1186/s12985-016-0544-0"
    },
    {
        "keywords": "human coronavirus;virus\u2013host interactions;apoptosis;innate immunity;ER stress;MAPK;NF-\u03baB",
        "abstract": "Human coronaviruses (HCoVs) are known respiratory pathogens associated with a range of respiratory outcomes. In the past 14 years, the onset of severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) have thrust HCoVs into spotlight of the research community due to their high pathogenicity in humans. The study of HCoV-host interactions has contributed extensively to our understanding of HCoV pathogenesis. In this review, we discuss some of the recent findings of host cell factors that might be exploited by HCoVs to facilitate their own replication cycle. We also discuss various cellular processes, such as apoptosis, innate immunity, ER stress response, mitogen-activated protein kinase (MAPK) pathway and nuclear factor kappa B (NF-&#954;B) pathway that may be modulated by HCoVs.",
        "body": "1. Introduction\nHuman coronaviruses (HCoVs) represent a major group of coronaviruses (CoVs) associated with multiple respiratory diseases of varying severity, including common cold, pneumonia and bronchilitis [1]. Today, HCoVs are recognised as one of the most rapidly evolving viruses owing to its high genomic nucleotide substitution rates and recombination [2]. In recent years, evolution of HCoVs has also been expedited by factors such as urbanization and poultry farming. These have permitted the frequent mixing of species and facilitated the crossing of species barrier and genomic recombination of these viruses [3]. To date, six known HCoVs have been identified, namely HCoV-229E, HCoV-NL63, HCoV-OC43, HCoV-HKU1, severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV); of which, four HCoVs (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) are globally circulated in the human population and contribute to approximately one-third of common cold infections in humans [4]. In severe cases, these four HCoVs can cause life-threatening pneumonia and bronchiolitis especially in elderly, children and immunocompromised patients [1,5,6]. Besides respiratory illnesses, they may also cause enteric and neurological diseases [7,8,9,10,11].SARS-CoV first emerged in 2002&#8211;2003 in Guangdong, China as an atypical pneumonia marked by fever, headache and subsequent onset of respiratory symptoms such as cough and pneumonia, which may later develop into life-threatening respiratory failure and acute respiratory distress syndrome [12]. Being highly transmissible among humans, it quickly spread across 29 countries, infecting more than 8000 individuals with a mortality rate of about 10% [13,14]. Originally, palm civets were thought to be the natural reservoir for the virus [15]. However, subsequent phylogenetic studies pointed to the bat origin of SARS-CoV based on sequences of SARS-like virus found in bats [16]. The MERS-CoV epidemic surfaced in Saudi Arabia in 2012 with similar clinical symptoms as SARS-CoV but with a much higher mortality rate of about 35% [17]. Unlike SARS-CoV, which exhibits super-spreader events, transmission of MERS-CoV is geographically limited [12]. In fact, reported cases of MERS-CoV often stem from outbreaks within the Middle Eastern countries or recent travel to the region [18,19].\nTaxonomy, Genomic Structure and Morphology\nCoVs are a group of large enveloped RNA viruses under the Coronaviridae family. Together with Artierivirdae and Roniviridae, Coronaviridae is classified under the Nidovirale order [20]. As proposed by the International Committee for Taxonomy of Viruses, CoVs are further categorized into four main genera, Alpha-, Beta-, Gamma- and Deltacoronaviruses based on sequence comparisons of entire viral genomes [21,22]. These CoVs can infect a wide variety of hosts, including avian, swine and humans. HCoVs are identified to be either in the Alpha- or Betacoronavirus genera, including Alphacoronaviruses, HCoV-229E and HCoV-NL63, and Betacoronaviruses, HCoV-HKU1, SARS-CoV, MERS-CoV and HCoV-OC43 (Table 1).Under the electron microscope, the CoV virions appear to be roughly spherical or moderately pleomorphic, with distinct &#8220;club-like&#8221; projections formed by the spike (S) protein [23,24]. Within the virion interior lies a helically symmetrical nucleocapsid that encloses a single-stranded and positive sense RNA viral genome of an extraordinarily large size of about 26 to 32 kilobases [20]. The positive sense viral genomic RNA acts as a messenger RNA (mRNA), comprising a 5&#8242; terminal cap structure and a 3&#8242; poly A tail. This genomic RNA acts in three capacities during the viral life cycle: (1) as an initial RNA of the infectious cycle; (2) as a template for replication and transcription; and (3) as a substrate for packaging into the progeny virus. The replicase-transcriptase is the only protein translated from the genome, while the viral products of all downstream open reading frames are derived from subgenomic mRNAs. In all CoVs, the replicase gene makes up approximately 5&#8242; two-thirds of the genome and is comprised of two overlapping open reading frames (ORFs), ORF1a and ORF1b, which encodes 16 non-structural proteins. The final one-third of the CoV genomic RNA encodes CoV canonical set of four structural protein genes, in the order of spike (S), envelope (E), membrane (M) and nucleocapsid (N). In addition, several accessory ORFs are also interspersed along the structural protein genes and the number and location varies among CoV species [25] (Figure 1).\n2. Involvement of Host Factors in Viral Replication and Pathogenesis\nAs intracellular obligate parasites, HCoVs exploit the host cell machinery for their own replication and spread. Since virus&#8211;host interactions form the basis of diseases, knowledge about their interplay is of great research interest. Here, we describe what is currently known of the cell&#8217;s contribution in CoV infection cycle: attachment; entry into the host cell; translation of the replicase-transcriptase; replication of genome and transcription of mRNAs; and assembly and budding of newly packaged virions (Figure 2). \n2.1. Coronavirus Attachment and Entry\nCoV infection is initiated by the attachment to specific host cellular receptors via the spike (S) protein. The host receptor is a major determinant of pathogenicity, tissue tropism and host range of the virus. The S protein comprises of two domains: S1 and S2. The interaction between the S1 domain and its cognate receptor triggers a conformational change in the S protein, which then promotes membrane fusion between the viral and cell membrane through the S2 domain. Today, the main host cell receptors utilised by all HCoVs are known: aminopeptidase N by HCoV-229E [26], angiotensin-converting enzyme 2 (ACE2) by SARS-CoV [27] and HCoV-NL63 [28,29], dipeptidyl peptidase 4 (DPP4) by MERS-CoV [30] and 9-O-acetylated sialic acid by HCoV-OC43 and HCoV-HKU1 [31,32].Apart from the conventional endosomal route of entry, some CoVs may also gain entry into the cell via the non-endosomal pathway, or a combination of both. The low pH in the cellular environment and endosomal cysteine protease cathepsins may help to facilitate membrane fusion and endosomal CoV cell entry [33]. Recent evidence has supported the role of cathepsin L in SARS-CoV and MERS-CoV entry [34,35,36]. Other host proteases, such as transmembrane protease serine 2 (TMPRSS2) and airway trypsin-like protease TMPRSS11D, could also perform S1/S2 cleavage to activate the S protein for non-endosomal virus entry at the cell plasma membrane during HCoV-229E and SARS-CoV infection [37,38]. In addition, MERS-CoV is also activated by furin, a serine endopeptidase that has been implicated in the cell entry of other RNA viruses and S1/S2 cleavage during viral egress [39].Many host cells also utilise its own factors to restrict viral entry. Using cell culture system and pseudotype virus, many groups have identified a family of interferon inducible transmembrane proteins (IFITM), which could inhibit global circulating HCoV-229E and HCoV-NL63 S protein mediated entry, and also the highly pathogenic SARS-CoV and MERS-CoV [12,40]. While the IFITM mode of action remains elusive, cell-to-cell fusion assays performed by some research groups suggest that IFITM3 blocks the enveloped virus entry by preventing fusion of the viral envelope with the plasma membrane or endosomal membranes through modulating the host membrane fluidity [41].\n2.2. Coronavirus Replication\nFollowing the release and uncoating of viral nucleocapsid to the cytoplasm, CoV replication begins with the translation of ORF 1a and 1b into polyproteins pp1a (4382 amino acids) and pp1ab (7073 amino acids). Here, the downstream ORF1b is translated through ribosomal frameshifting mechanism, in which a translating ribosome shifts one nucleotide in the &#8722;1 direction, from the ORF1a reading frame into ORF1b reading frame. This repositioning is enabled by two RNA elements&#8212;a 5&#8242;-UUUAAAC-3&#8242; heptanucleotide slippery sequence and RNA pseudoknot structure. Subsequently, polyproteins pp1a and pp1ab are cleaved into at least 15 nsp, which assemble and form the replication-transcription complex. With the assembly of the replicase-polymerase, the full-length positive strand of genomic RNA is transcribed to form a full-length negative-strand template for the synthesis of new genomic RNAs and overlapping subgenomic negative-strand templates. These subgenomic mRNAs are then transcribed and translated to produce the structural and accessory proteins. Several heterologous nuclear ribonucleoprotein (hnRNA) family members (hnRNPA1, PTB, SYN-CRYP) have been found to be essential for efficient RNA replication [42]. Other RNA-binding proteins have also been suggested to play a role in CoV replication, such as m-aconitase and poly-A-binding protein (PABP), DDX1, PCBP1/2 and zinc finger CCHC-type and RNA-binding motif 1 (MADP1) [43,44,45].\n2.3. Coronavirus Assembly and Egress\nThe assembly of virions is quickly ensued with the accumulation of new genomic RNA and structural components. In this phase of the infection cycle, the helical nucleocapsid containing the genomic RNA interacts with other viral structural proteins (S, E and M proteins) to form the assembled virion. The assembly of CoV particles is completed through budding of the helical nucleocapsid through membranes early in the secretory pathway from the endoplasmic reticulum to the Golgi intermediate compartment (ERGIC). The contributions of the host in this phase of the infection cycle have rarely been explored. Currently, it is known that the M protein orchestrates the entire assembly process by selecting and organizing the viral envelope components at the assembly sites and by mediating the interactions with the nucleocapsid to allow the budding of virions [46]. The M protein interacts with different viral structural proteins, such as the E protein, to assemble into a mature virus. This interaction generates the scaffold of the virion envelope and induces the budding and release of the M protein-modified membrane and with the S protein to assemble the spikes into the viral envelope [46,47]. Following assembly and budding, the virions are transported in vesicles and eventually released by exocytosis. In a recent study, an inhibition of a Valosin-containing protein (VCP/p97) resulted in virus accumulation in early endosome in infectious bronchitis virus (IBV), suggesting a role for VCP in the maturation of virus-loaded endosomes [48].\n3. Human Coronavirus Infection and Apoptosis\nApoptosis is a process of programmed cell death that is tightly regulated and anti-inflammatory. When cells undergo apoptosis, they demonstrate specific hallmarks such as cell shrinkage, extensive plasma membrane blebbing, nuclear pykosis, DNA fragmentation and asymmetrical distribution of plasma membrane [49,50,51]. To date, two main mechanisms of apoptosis have been established&#8212;the extrinsic and intrinsic pathways. The extrinsic pathway is initiated by the binding of extracellular death ligands (such as Fas ligand (FasL) and TNF-receptor-related apoptosis-inducing ligands (TRAIL)) to death receptors from the tumour necrosis factor (TNF) super-family [52]. These death receptors then recruit various death adapter proteins, such as Fas-associated death domain protein (FADD) [53], and initiator procaspases 8 and 10 to form the death-inducing signalling complex (DISC) [54,55]. Consequently, the two initiator procapases are cleaved to their active forms and induce a signalling cascade to eventually activate effector caspases 3 and 7. On the other hand, the intrinsic pathway occurs internally in the cell and involves changes in the mitochondrial outer membrane permeability (MOMP) based on the ratio of pro-apoptotic and anti-apoptotic B-cell lymphoma 2 (Bcl2) family of proteins (Figure 3). Enhanced MOMP causes the release of pro-apoptotic factors such as cytochrome c to activate initiator caspase 9. Like the extrinsic pathway, activation of initiator caspase 9 in the intrinsic pathway results in the proteolytic cleavage effector caspases 3 and 7, which in turn process many key cellular proteins essential for apoptosis [56]. Convergence between the two pathways may also occur even before effector caspase activation, when Bid, a pro-apoptotic Bcl2 family protein, is directly cleaved by caspase 8 [57].During viral infections, apoptosis is induced as one of the host antiviral responses to limit virus replication and production. Many viruses have evolved distinct strategies to subvert apoptosis [58]. For example, some viruses encode for viral proteins that act as Bcl2 family protein homologues [59]. Alternatively, viruses might develop mechanisms to regulate Bcl2 family proteins or caspase activation either directly or indirectly through other molecular pathways such as mitogen-activated protein (MAPK) and nuclear factor kappa B (NF-&#954;B) pathways [60,61,62,63,64,65]. Interestingly, some viruses may engage the apoptotic machineries for efficient viral infection. For instance, alphaviruses and flaviviruses contain phosphatidylserine-rich viral membranes to imitate apoptotic cells to promote viral entry [66].\n3.1. Cell Tropism and Apoptosis\nAs HCoVs are respiratory pathogens known to infect tissue cultures and cell lines derived from the respiratory tract, these viruses may also infect other tissue cultures and cell lines. Infection of these tissues and cells may induce apoptosis [67,68]. However, although HCoVs mainly target the respiratory tract during infection, they have also been associated to apoptosis induction in a wide spectrum of cell types, including intestinal mucosal cells, kidney tubular cells and neuronal cells [69,70,71,72,73,74]. Autopsy studies of SARS-CoV-infected tissues revealed apoptosis induced in lung, spleen and thyroid [75]. HCoVs have also been shown to infect the immune system and induce apoptosis in immune cells such as macrophages, monocytes, T lymphocytes and dendritic cells [69,76,77,78,79]. Because these immune cells are associated with the activation of the innate and acquired immunity, it is reasonable to speculate that the massive elimination of these cells could be a viral strategy to suppress the host innate and adaptive immune responses. In a recent study, it was reported that HCoV-229E infection resulted in massive CPE and cell death in dendritic cells [80]. Since dendritic cells are prevalent throughout our bodies, it is possible that they are used as a vehicle to facilitate viral spread and impair our immune systems [80,81].\n3.2. Molecular Mechanisms in Apoptosis\nOn a molecular level, HCoV infections have been reported to trigger apoptosis through multiple mechanisms. SARS-CoV-induced apoptosis was shown to be caspase-dependent and could be inhibited by caspase inhibitor Z-VAD-FMK or overexpression of Bcl2 [82,83]. Although viral replication was required for apoptosis induction [83], apoptosis did not affect the viral replication kinetics of SARS-CoV [82]. On the other hand, infection of primary T lymphocytes by MERS-CoV was shown to induce DNA fragmentation and caspase 8 and 9 activation, suggesting that both extrinsic and intrinsic pathways were activated. Unlike SARS-CoV infection, MERS-CoV replication was not necessary to induce apoptosis in infected T lymphocytes [79]. Apoptosis can also be induced by the less pathogenic strains of HCoVs, as substantiated by microarray data showing significant changes in pro-apoptotic and anti-apoptotic gene expression of Bcl2 family members during HCoV-229E infection [84]. Infection of HCoV-OC43 was shown to promote BAX translocation to the mitochondria in human neuronal cells [74]. Although caspases 3 and 9 were activated in HCoV-OC43-infected murine and human neuronal cells [9,74], addition of pan-caspase inhibitor Z-VAD-FMK and the caspase-9 inhibitor Z-LEHD-FMK did not affect the viability of these infected neuronal cells, indicating that programmed cell death induced by HCoV-OC43 could be caspase-independent [74]. This highlights the possibility of a non-classical programmed cell death mechanism induced in HCoV infection.Apoptotic mechanisms during HCoV infection are likely to be manipulated by viral proteins (Figure 4), although this has only mostly been studied in SARS-CoV. Specifically, SARS-CoV S, N, E, M, ORF-6, 7a and 9b proteins have been shown to serve pro-apoptotic functions in their host cells [77,85,86,87,88,89,90,91]. Expression of SARS-CoV E protein and 7a protein promoted mitochondrial-mediated apoptosis by sequestering the anti-apoptotic Bcl-XL protein to the endoplasmic reticulum (ER) membranes [77,92]. SARS-CoV M protein is also highly pro-apoptotic and mediates activation of both caspases 8 and 9 [90]. Additionally, HCoV-OC43 wild type S protein has been shown to induce unfolded protein response (UPR) in human neuronal NT2-N and LA-N-5 cell lines, which may lead to apoptosis [93]. A recombinant HCoV-OC43 harbouring point mutations at its S protein induced stronger caspase 3 activation and nuclear fragmentation than the wild-type virus [93]. It is interesting to note that the localisation of SARS-CoV N and 9b protein is associated with induction of caspase-dependent apoptosis [89,94]. This finding opens up to novel perspectives of the link between subcellular localisation of viral proteins and caspase activation as a mode of apoptosis regulation by HCoVs.\n4. Human Coronavirus Infection and Innate Immunity\nWhen the cells are exposed to pathogens such as viruses, immune responses are induced as a form of host defence. The immune response is modulated during pathogen exposure in a cell-type dependent fashion. Innate immunity is the first line of defence mounted against the virus before the adaptive immune system is generated. Both the host and virus can manipulate innate immune mechanisms as a form of defence or evasion strategy [95,96].\n4.1. Pattern Recognition Receptors\nCells in the immune system detect the viral pathogens via several recognition strategies. Of which, the most well characterized is the pattern recognition receptors (PRR), which engage various microbial pathogens via evolutionarily conserved structures known as pathogen-associated molecular patterns (PAMPs). PRRs are mainly categorized into three classes, namely Toll-like receptors (TLRs), retinoic acid-inducible gene I (RIG-I)-like receptors (RLRs) and nucleotide oligomerisation domain (NOD)-like receptors (NLRs).TLR is a type I transmembrane protein localized to either the cell surface or endosomal vesicles. Their leucine-rich repeats (LRR) domain mediates the recognition of PAMPs and damage-associated molecular patterns (DAMPs) from various sources including bacteria, fungi and viruses [97]. Activation of TLRs occurs mainly in antigen-presenting cells such as dendritic cells (DCs), macrophages, monocytes and B cells. Of the 10 known TLRs in human, TLR2, 3, 4, 7 and 9 are found to be involved in viral detection [98,99]. TLR3 recognizes double-stranded RNA (dsRNA), a replicative intermediate produced during viral RNA replication [100]. TLR7 and 8 detect single-stranded RNA (ssRNA) and TLR9 recognizes unmethylated CpG DNA present in DNA viruses [101,102,103]. Besides nucleic acids, other TLRs, such as TLR2 and 4, sense viral proteins as exemplified in respiratory syncytial virus (RSV), hepatitis virus, measles virus and human immunodeficiency virus [104,105,106,107]. Upon recognition of viral components, TLRs recruit Toll/interleukin-1 receptor (TIR)-containing signalling adaptor molecules, such as MyD88 (myeloid differentiation primary response protein 88) and TIR-domain-containing adapter-inducing interferon-&#946; (TRIF) [108,109,110]. MyD88 and TRIF then stimulate the MAPK and NF-&#954;B pathways to boost IFN and pro-inflammatory cytokine production [111].Unlike TLRs, RLRs and NLRs are expressed ubiquitously. RLRs are a family of cytoplasmic receptors that comprise of three members: retinoic acid-inducible gene I (RIG-I), melanoma differentiation associated factor 5 (MDA5), and laboratory of genetics and physiology 2 (LGP2). RIG-I and MDA5 possess two caspase-recruitment domains (CARDs) at their N terminal, a DExD/H-box RNA helicase domain (where x can be any amino acid) and a repressor domain (RD) at the C terminal. On the other hand, LGP2 lacks the CARD domain [112], and it regulates RIG-I and MDA5 either positively or negatively [113,114,115]. RIG-I recognizes 5&#8242;-triphosphate moieties present in viral genomic RNA, as well as double-stranded &#8220;panhandle&#8221; structure formed by self-annealing of complementary ends of the viral genome [116,117]. In contrast, MDA5 usually detects dsRNA sequences of longer length. Binding of RIG-I and MDA5 to the viral RNA causes a conformational change to expose the CARD domain. An adaptor protein known as mitochondrial antiviral signalling adaptor, MAVS, which is localized at the mitochondria and peroxisomes, is recruited. MAVS then activates transcription factors such as interferon regulatory factor 1 (IRF1), IRF3 and NF-&#954;B to trigger the expression of interferons (IFNs) and pro-inflammatory cytokines [117] (Figure 5).NLRs are another large family of cytosolic proteins that are organized into three main domains: a CARD domain, pyrin domain (PYD), or baculovirus inhibitor repeat domain at the N terminal, a conserved NOD motif at the intermediate region and LRR motifs at the C terminal. The LRR motifs detect viral PAMPs to induce a structural rearrangement. Subsequently, a diverse range of signalling pathways, including MAPK and NF-&#954;B pathways, is activated [118], Furthermore, assembly of multimeric protein complexes known as inflammasomes is mediated by some NLR family members such as NLR family PYD-containing 1 (NLRP1), NLRP3 and NLR family CARD-containing 4 (NLRC4). These inflammasomes activate inflammatory caspase, capase-1, that induces the cleavage of pro-IL-1&#946; and pro-IL-18 into their active forms [119].\n4.2. Interferon Responses\nIFNs are classified into type I, II and III based on their preference for specific IFN receptors. In particular, type I IFN is best known for its antiviral actions. Binding of type I IFNs to the IFN-&#945;/&#946; receptor (IFNAR) induces the oligomerisation of its receptor subunits, IFN-&#945;R1 and IFN-&#945;R2, and consequently conveys downstream signalling via the Janus kinase-Signal Transducer and Activator of Transcription (JAK-STAT) pathway. Autophosphorylation of JAK domains in the IFNAR results in subsequent phosphorylation of STAT1 and STAT2 proteins at their tyrosine residues. This is followed by dimerisation and nuclear translocation of the activated STAT proteins, which recruit IFN regulatory factor 9 (IRF9) to form the IFN-stimulated gene factor 3 (ISGF3). ISFG3 is a transcription factor that binds to its cognate DNA sequence known as IFN-stimulated response elements (ISREs) to activate transcription of IFN-stimulated genes (ISGs) [120] (Figure 5). Many of these ISGs such as 2&#8242;&#8211;5&#8242; oligoadenylatesynthetase and protein kinase R (PKR) confer resistance against virus invasions [121]. Additionally, type I IFNs facilitates the maturation of dendritic cells (DCs), cytotoxicity of natural killer (NK) cells, and differentiation of T lymphocytes [98].\n4.3. Modulation of Innate Immunity\nInfection by HCoVs, especially the highly pathogenic SARS-CoV and MERS-CoV, is associated with suppression of IFN synthesis [122,123,124,125,126]. The capability of the virus to regulate type I IFN signalling is an important hallmark for virulence [127]. As compared to SARS-CoV and MERS-CoV, a huge rise in type I IFNs were detected in cells infected with HCoV strain 229E [80,124,128].Based on studies from SARS-CoV and Mouse Hepatitis Virus (MHV)-infected cells, two mechanisms have been proposed to explain the HCoV-mediated inhibition of type I IFN production [13,126]. Firstly, CoV genomic and subgenomic RNA replication takes place in double membrane vesicles to prevent detection by PRRs [13,129]. Secondly, proteins encoded by the virus could interfere with innate immune pathways [13,130]. The structural proteins, nonstructural proteins and accessory proteins of HCoVs have been shown to modify innate immune responses (Figure 5).\n4.3.1. Viral Proteins Involved in Innate Immunity\n\nStructural Proteins of HCoVs\nExpression of SARS-CoV M protein could suppress type I IFN production mediated by RIG-I, but not MDA5, in infected HEK293 cells [131], likely through its first transmembrane domain. However, this inhibition was not observed when expressing the M protein of HCoV-HKU1, suggesting that this activity is not conserved among all HCoV strains [132]. In another study, it was shown that the MERS-CoV M protein could also suppress type I IFN by inhibiting the translocation of IRF3 into the nucleus, although the exact mechanism has not yet been elucidated [133]. Additionally, SARS-CoV N protein was also shown to interfere with the function of IRF3 [134]. The N protein of SARS-CoV likely act at the initial recognition stage of viral RNA via its RNA binding activity, although it neither forms a complex with RIG-I nor MDA5 [135]. This implies that the N protein possibly acts on other viral RNA recognition strategies of the host.\nNon-Structural and Accessory Proteins of HCoVs\nBesides the structural proteins, other nonstructural proteins (nsp) and accessory proteins of HCoVs have also been implicated in the modulation of innate immunity. For instance, nsp1 of both SARS-CoV and MERS-CoV has been demonstrated to modify capped non-viral RNAs to facilitate endonucleolytic cleavage of host messenger RNA (mRNA) [136,137]. Additionally, SARS-CoV nsp1 interacted with the 40S subunit of ribosome to prevent host mRNA translation [136]. This induces the host shutoff mechanism, as transcription and translation of viral RNA are more favoured over that of host mRNA. In a recent study, several residues of SARS-CoV nsp1 were identified to affect IFN-dependent signalling [138]. In addition to nsp1, SARS-CoV and MERS-CoV nsp3 proteins, which possess papain-like protease (PLpro) domain and a PLP2 domain, also antagonize IFN production. Both SARS-CoV and MERS-CoV PLpro domains are deISGylating enzymes and they downregulated mRNA levels of pro-inflammatory cytokines including CCL5, IFN&#946;, and CXCL10 [139]. Suppression of IFN responses by SARS-CoV PLpro is not mediated by its protease activity. Rather, SARS-CoV PLpro inhibited the phosphorylation of interferon-regulatory factor 3 (IRF3) and its translocation to the nucleus to enhance IFN gene transcription [140]. Expression of MERS-CoV PLpro also antagonizes IFN production and is required for suppression on RIG-I and MDA5 [139,141]. Furthermore, it has been identified that the ADP-ribose-1-monophosphatase macrodomain encoded within nsp3 in HCoV-229E and SARS-CoV is responsible for suppressing IFN induction [142].Despite being dispensable in viral replication, HCoV accessory proteins are essential in diverse cellular signalling, such as cell proliferation, apoptosis and interferon signalling [25]. In SARS-CoV, ORF3b and -6 are shown to interfere with IFN&#946; synthesis by inhibiting the phosphorylation and nuclear translocation of IRF3. Furthermore, these accessory proteins also disrupt IFN signalling by preventing IFN&#946;-induced activation of interferon-stimulated response element (ISRE) found in the promoter region of ISG [134]. The accessory proteins of MERS-CoV, ORF4a, -4b and -5, could similarly suppress IRF3 nuclear translocation, hence significantly reducing IFN-&#946; promoter-driven luciferase activity in cells transfected with these accessory proteins [133].\n5. Human Coronavirus and ER Stress Response\nThe endoplasmic reticulum (ER) is a cellular organelle important for protein synthesis, folding, processing and post-translational modifications. In normal circumstances, the ER can be loaded with a very high concentration of proteins without perturbing its unique luminal environment [143]. However, when the protein load exceeds the ER folding and processing capacity, rapid accumulation of misfolded or unfolded proteins occurs within the ER lumen. Various signalling pathways, collectively known as ER stress response or UPR, are activated. These pathways are initiated by three ER transmembrane sensors-protein-kinase-R (PKR)-like endoplasmic reticulum kinase (PERK), inositol-requiring protein 1 (IRE1) and activating transcriptional factor 6 (ATF6) to orchestrate the restoration of ER homeostasis by enhancing protein folding, attenuating protein translation and upregulating genes related to protein folding, chaperoning and ER-assisted degradation (ERAD) (Figure 6). In cases of prolonged and irreversible ER stress, apoptosis mechanisms are triggered [144]. During viral infections, ER stress response is induced. This massive utilisation of the ER elicit immense burden, causing the host to mount UPR as its antiviral response [145].\n5.1. PERK Signalling Pathway\nActivation of PERK is initiated by its dissociation of the luminal domain from the ER chaperone, binding immunoglobulin protein (BiP). This is followed by the oligomerisation and autophosphorylation of PERK. In its active form, PERK phosphorylates Ser51 at the &#945;-subunit of eukaryotic initiation factor 2 (eIF2&#945;) to attenuate protein translation [146]. Activation of PERK plays a pro-survival role in cells, as clearly demonstrated by PERK&#8722;/&#8722; mouse embryonic fibroblasts, which exhibited higher cell death when treated with cycloheximide, an ER stress-inducing agent [147]. Phosphorylated eIF2&#945; not only triggers a shutdown of global protein synthesis, but also enhances the translation of activating transcription factor ATF4 [148]. ATF4 stimulates target gene expression such as GADD153 (also known as CHOP or C/EBP-homologous protein), to enhance transcription of pro-apoptotic genes [149]. Additionally, eIF2&#945; can be phosphorylated by other kinases such as PKR, heme-regulated inhibitor kinase (HRI), and general control non-derepressible-2 (GCN2) [144]. These kinases activate various downstream signalling pathways, which together form the integrated stress response [146,150].PKR and eIF2&#945; phosphorylation was detected in SARS-CoV-infected cells and inhibition of PKR using antisense peptide-conjugated phosphorodiamidate morpholino oligomers did not affect eIF2&#945; phosphorylation but significantly reduced SARS-CoV-induced apoptosis. SARS-CoV protein replication and virus production were not affected by PKR knockdown. Therefore, it is likely that SARS-CoV adopts a strategy to counteract against the antiviral effects of PKR, thus enabling viral mRNA translation to proceed regardless of eIF2&#945; phosphorylation. PERK was also found to be activated during SARS-CoV infection [75], possibly through its S and 3a proteins [151,152]. In another study, it was demonstrated that expression of a dominant-negative PERK mutant, that inhibited PERK kinase activity, suppressed the transcriptional activation of Grp78 and Grp94 promoters mediated by S proteins of SARS-CoV and HCoV-HKU1 [153]. However, PERK activation is unlikely to occur in all HCoV strains. In neuronal cells lines infected with HCoV-OC43, it was shown that eIF2&#945; was only transiently phosphorylated at the early stage of infection, but was subsequently suppressed and returned back to its basal level of phosphorylation, similar to the mock-infected cells [93]. On the other hand, previous studies from this group showed that PKR, PERK and eIF2&#945; were moderately induced at the early stages of IBV infection, but were subsequently suppressed at late infection stages [154,155]. Nevertheless, the moderate and transient increase in eIF2&#945; phosphorylation was sufficient to activate ATF4 protein translation and upregulate the downstream targets of ATF4, ATF3 and GADD153. Knockdown of PKR and PERK in IBV-infected cells attenuated IBV-induced GADD153 upregulation and IBV-induced apoptosis, although the viral protein replication was unaffected [155]. Upregulation of GADD153 is postulated to induce pro-apoptotic protein TRIB3 and inhibit pro-survival ERK protein [154], as well as provide a negative feedback to rapidly dephosphorylate eIF2&#945; at late stages of IBV infection [155]. Based on these findings, we speculate the HCoVs might use similar mechanism to modulate the PKR/PERK/eIF2&#945; pathway in infected cells. More studies could be done on HCoV infection to analyze the activation of the PKR/PERK/eIF2&#945; pathway at various stages of infection.\n5.2. ATF6 Signalling Pathway\nLike PERK, activation of ATF6 is initiated by dissociation from the ER chaperone, BiP, although alternative mechanisms such as deglycosylation and reduction of disulphide bonds could occur [156,157]. ATF6 then translocates into the Golgi apparatus, where it is proteolyzed by site-1 and site-2 proteases (S1P and S2P). The processed ATF6 then migrates to the nucleus where it turns on expression of genes containing an ER stress response element (ERSE) in their promoters [158]. Like ATF4, ATF6 also induces expression of ER chaperone proteins such as GRP78, GRP94 and transcription factors CHOP and X box-binding protein 1 (XBP1) [150]. XBP1 is essential for IRE signalling [159].As compared to the two other UPR branches, PERK and IRE1, the ATF6 branch is less well studied. As GRP94/78 are also target genes of ATF6, and their promoter activities were enhanced by SARS-CoV S protein, one could hypothesize that ATF6 pathway could also be induced by SARS-CoV S. Surprisingly, overexpression of SARS-CoV S protein did not affect ATF6 promoter luciferase activity [152]. Deletion of E protein in recombinant SARS-CoV also did not significantly activate ATF6 [91]. Intriguingly, 8ab protein, an accessory protein of SARS-CoV, was shown to reside in the luminal surface of the ER surface and activate ATF6 via facilitating its proteolysis and translocation of the processed ATF6 into the nucleus [160]. 8ab protein from SARS-CoV was found in civet cats and early human isolates, but was subsequently split into two accessory proteins, 8a and 8b, with a characteristic 29-nucleotide deletion [161].\n5.3. IRE1 Signalling Pathway\nIRE1 is believed to be the last UPR branch to be activated in cells undergoing ER stress [162]. It is also the most conserved among all UPR arms [163]. Although IRE1 was initially proposed to be activated in the same mechanism as PERK [162], later studies suggested that the N-terminal luminal domain (NLD) of IRE1 can directly bind unfolded proteins [164,165]. Activation of its RNase domain results in unconventional splicing of a 252-nucleotide intron from homologous to Atf/Creb1 (HAC1) mRNA in yeasts and a 26-nucleotide intron from X-box binding protein 1 (XBP1) mRNA in humans [166]. Splicing of XBP1 generates a potent transcription factor, XBP1s, that induces expression of genes related to protein entry into the ER, folding and ERAD [159]. In a negative feedback mechanism, XBP1s also promotes the transcription of E3 ubiquitin ligase synoviolin to enhance IRE1 ubiquitination [167]. The unspliced variant XBP1u contained a nuclear exclusion signal to sequester XBP1s from the nucleus, thus making XBP1u another negative feedback regulator of XBP1s. [168]. In a separate mechanism, IRE1 can cleave ER-associated mRNA species through regulated IRE1-dependent decay (RIDD) during late stages of ER stress [145]. It is believed that initial XBP1/HAC1 splicing by IRE1 promotes survival but subsequent activation of RIDD upon prolonged ER stress leads to cell death, thus allowing IRE1 to play dual role in apoptosis [169,170]. Another important enzymatic activation of IRE1 is its kinase activity. The kinase domain of phosphorylated IRE1 recruits the TNF receptor-associated factor 2 (TRAF2), which then activates other kinases to eventually activate the c-Jun N-terminal kinase (JNK) and regulates ER stress-dependent apoptosis [171].Previous studies have investigated the role of IRE1-XBP1 pathway during SARS-CoV infection. Although no increase in XBP1 splicing was observed in SARS-CoV-infected cells [172], deletion of E protein in recombinant SARS-CoV resulted in significant XBP1 splicing and higher rate of apoptosis [91]. On the other hand, infection of HCoV-OC43 caused an induction in XBP1 splicing and enhanced expression of genes regulated by XBP1s, namely Edem, Herp, Grp94 and P58-ipk. However, introduction of two point mutations (H183R and Y241H) in the S protein of HCoV-OC43 led to a higher degree of XBP1 cleavage, followed by a strong activation of caspase-3 and nuclear fragmentation [93]. Since IRE1 pathway is closely associated to JNK activation, it is possible that the JNK pathway is also implicated during HCoV-OC43 infection.Similar to HCoV infections, it has been shown that the IRE1-XBP1 pathway is activated during IBV infection. Knockdown of IRE1 using specific siRNA in IBV-infected cells augmented IBV-induced apoptosis; however, an opposite effect was observed by XBP1 knockdown in IBV-infected cells. Consistent with the knockdown experiments, transient overexpression of the full-length IRE1&#945; attenuated IBV-induced apoptosis. When both spliced and unspliced forms of XBP1 were overexpressed in IBV-infected cells, the spliced form of XBP1 was shown to be anti-apoptotic and the unspliced form was pro-apoptotic. Overexpression of a dominant-negative XBP1 enhanced IBV-induced apoptosis. Therefore, our findings showed that the anti-apoptotic function of IRE1 during IBV infection could be mediated by its splicing of XBP1, hence converting XBP1 from a pro-apoptotic to anti-apoptotic form. Finally, IRE1 induction during IBV infection was shown to mediate JNK hyperphosphorylation and Akt hypophosphorylation to potentiate the IBV-infected cells to apoptosis [173].\n6. Human Coronavirus and MAPK Pathways\nThe MAPKs are a group of evolutionally conserved serine/theronine kinases that are activated in response to environmental stresses including oxidative stress, DNA damage, cancer development and viral infections [174,175,176]. To date, multiple MAPK pathways have been identified in mammals, and they can be broadly classified into three major categories&#8212;the extracellular signal-regulated kinase (ERK), p38 MAPK and stress-activated protein kinase/c-Jun N-terminal kinase (SAPK/JNK) [177]. In all MAPK pathways, the signals are transduced downstream via a three-tier protein kinase cascade. In each tier, the kinases are activated by upstream kinases by dual phosphorylation at the Thr-X-Tyr motif (X represents any amino acid). Presence of extracellular stimuli triggers the activation of MAP kinase kinase kinases (MAPKKKs), which then activate the MAP kinase kinases (MAPKKs). These sequential phosphorylation events eventually activate MAPKs which in turn regulate a variety of fundamental cellular processes such as cell proliferation, survival, motility, differentiation, autophagy, apoptosis and regulation of cytokine production (Figure 7) [178]. \n6.1. Modulation of MAPK Pathways\nPhosphorylation of all three MAPK members has been detected in cells infected with SARS-CoV [179,180]. Additionally, the MAPK pathways are also activated during infection by other HCoVs, as discussed below (Figure 7).\n6.1.1. ERK Pathway\nActivation of ERK and its upstream kinases, MEK1/2, was detected in cells infected with SARS-CoV or overexpressed with SARS-CoV S protein [179,181]. However, ERK activation did not contribute to phosphorylation of its downstream target, p90 ribosomal S6 kinase (p90RSK) in SARS-CoV-infected Vero E6 cells [182]. This could be attributed to the higher phosphorylation levels of ERK1 compared to ERK2 in the SARS-CoV-infected cells, as ERK1 has been shown to suppress p90RSK phosphorylation and functional activity [183]. In another recent study, vimentin, a type III intermediate filament protein, was shown to be critical for entry of SARS-CoV, via direct interaction with the viral S protein [184]. Since vimentin could associate with &#946;-adrenergic receptor to regulate ERK activation, one could speculate that induction of ERK pathway by SARS-CoV S protein might be through the interaction between vimentin and the ACE2 receptor, which is required for entry of SARS-CoV [185]. Additionally, binding of SARS-CoV S protein to ACE2 receptor stimulates upregulation of chemokine (C&#8211;C motif) ligand 2 (CCL2) mediated by ERK/AP-1 activation. CCL2 is believed to be responsible for respiratory inflammatory symptoms in SARS patients [181]. SARS-CoV S protein-induced ERK phosphorylation was shown to enhance IL8 release [186]. Other viral proteins of SARS-CoV have also been shown to induce ERK activation. Expression of SARS-CoV PLpro was shown to increase ERK1 ubiquitin-mediated degradation to suppress IFN-induced responses [187]. SARS-CoV 3b protein was also involved in ERK phosphorylation to potentiate AP-1 dependent activity of pro-inflammatory cytokine monocyte chemoattractant protein-1 (MCP-1) [188]. Besides SARS-CoV, MERS-CoV infection is also associated with enhanced ERK phosphorylation profile. The use of ER pathway inhibitor inhibited MERS-CoV infection by approximately 50% [189]. Although ERK phosphorylation was not significantly affected by HCoV-229E infection, the level of phosphorylated ERK can be enhanced by the use of chloroquine, a known antiviral agent against viruses [190]. Therefore, targeting the ERK pathway might have significant antiviral potential during HCoV infection.\n6.1.2. JNK Pathway\nPhosphorylation of JNK and its upstream kinases, MKK4/7, was detected in SARS-CoV-infected cells [191]. Overexpression of SARS-CoV 3a and 7a protein increased JNK activation and augmented IL8 promoter activity [192]. SARS-CoV 3b protein was shown to induce JNK/c-Jun/AP-1 activation to mediate transcription of MCP-1 [188]. Concurrently, the S protein of SARS-CoV was also shown to induce the activation of protein kinase epsilon via JNK activation [193]. Expression of SARS-CoV N protein is associated with the downregulation of prosurvival factors and apoptosis induction in COS-1 cells, possibly mediated by JNK activation [86]. Apoptosis induced by SARS-CoV 6 and 7a protein in Vero E6 and COS-7 cells were blocked by a JNK inhibitor [88]. Therefore, these findings suggest that JNK could act as a pro-apoptotic protein during SARS-CoV infection. However, in another study, phosphorylation of JNK was required for the maintenance of Vero E6 cells persistently infected with SARS-CoV. As persistent infection is only established after apoptotic events, it was proposed that JNK might act as a pro-apoptotic during acute phase of infection, but subsequently switched to become anti-apoptotic during prolonged infection [191]. It is uncertain if these observations noted were dependent on cell-type specificity. In our recent study, our group has found out that JNK was also activated during HCoV-229E infection and serves an anti-apoptotic role via modulation of Bcl2 family proteins. Furthermore, JNK contributes to the production of IFN&#946; and IL8 in HCoV-229E-infected cells (unpublished). The anti-apoptotic role of JNK during HCoV-229E infection contradicts our observations in H1299 cells infected with animal coronaviruses such as IBV, in which JNK has been shown to be pro-apoptotic [194]. The discrepancy of the involvement of JNK in CoV infections could be attributed to the different virus strains used in the experiments.\n6.1.3. p38 MAPK Pathway\nActivation of p38 MAPK has been reported in cells infected with SARS-CoV, MERS-CoV and HCoV-229E [180,189,190]. The upstream kinases of p38 MAPK, MKK3/6 were also shown to be phosphorylated [179]. On the other hand, downstream effectors of p38, MAPK-activated protein kinase-2 (MAPKAPK-2), cAMP response element-binding protein (CREB) and activation transcription factor-1 (ATF-1), were also activated in SARS-CoV-infected cells. SARS-CoV-induced phosphorylation of eukaryotic initiation factor 4E (eIF4a) was attenuated by p38 MAPK inhibitor. Although eIF4a is involved in protein translation, SARS-CoV viral protein translation and viral replication was not attenuated by p38 MAPK inhibitor, suggesting that SARS-CoV did not ultilise p38 for viral protein synthesis [180]. Furthermore, SARS-CoV infection induced p90RSK Ser380 phosphorylation, which could be partially negated by p38 MAPK inhibitor. Analysis of individual SARS-CoV proteins demonstrated that transfection of its 7a protein induced p38 activation when fused with GFP [195]. Since SARS-CoV 7a and 3a protein co-immunoprecipitated, they are likely to co-interact in virus-infected cells [196]. Therefore, it is possible that simultaneous co-expression of 7a and 3a proteins might further enhance p38 phosphorylation. In a recent study, the PBZ-binding motif (PBM) of SARS-CoV E protein was demonstrated to bind to syntenin and redistributes it to the cytoplasm, where it acts as a major scaffolding protein for p38 signalling cascade. Inhibition of syntenin using specific siRNA in cells transfected with functional E protein led to decreased p38 MAPK signalling [197]. Consistent with cell culture studies, phosphorylated p38 MAPK level was found to be increased in leukocytes of SARS patients. Enhanced p38, but not ERK, activation likely contributes to elevated IL8 levels and abnormal cytokine profile in SARS patients [198]. Additionally, p38 might also be involved in HCoV replication, as exemplified by HCoV-229E. Inhibition of p38 by its inhibitor SB203580 suppressed the HCoV-229E-induced cytopathic effects and significantly reduced viral titres in a dose dependent manner [190]. The use of chloroquine was also shown to exert antiviral effects against HCoV-229E infection, possibly via its attenuation on p38 activation [190].\n7. Human Coronavirus and NF-\u03baB Pathway\nThe NF-&#954;B proteins are a family of transcription factors that regulate expression of genes to control a broad range of biological processes, such as cell death, inflammation, innate and adaptive immune responses. Mammalian NF-&#954;B family composes of five members, RelA (also named p65), RelB, c-Rel, NF-&#954;B1 p50, and NF-&#954;B2 p52, which form dimers in the cytoplasm. It has been established that NF-&#954;B pathway is often targeted by viral pathogens to enhance viral replication, host cell survival and host immune evasion [199,200]. There are two main pathways for NF-&#954;B signalling&#8212;the canonical and non-canonical pathways. In the canonical pathway, the latent NF-&#954;B forms a complex with its inhibitor I&#954;B protein and is sequestered in the cytoplasm. As mentioned above, presence of viral pathogens activate various membrane sensors such as RIG-I, which induces the phosphorylation I&#954;B by I&#954;B kinase (IKK) complex and its subsequent ubiquitination. IKK complex consists of trimeric subunits including two catalytic subunits, IKK&#945; and IKK&#946;, and a regulatory subunit, IKK&#947; (also named NF-&#954;B essential modulator or NEMO). NF-&#954;B is thus released from the inhibitory effects of I&#954;B and translocates to the nucleus, where it stimulates transcription of target genes, either alone or in combination with other transcription factors including AP-1, Ets, and Stat [201]. On the other hand, the non-canonical pathway is independent of I&#954;B degradation, but instead relies on inducible p100 processing. Activation of NF-&#954;B inducing kinase (NIK), a MAPKKK, induces the phosphorylation and activation of IKK&#945; dimeric complex, in turn activating p100. This results in the release of p52/RelB heterodimer from sequestration by p100. The p52/RelB heterodimer translocates to the nucleus to activate target genes related to a number of cellular functions, in particularly cell proliferation, survival and innate immunity [202].\n7.1. Modulation of NF-\u03baB Pathway\nNF-&#954;B pathway has been shown to play an important role in HCoV infections. NF-&#954;B was activated in lungs of mice infected with recombinant SARS-CoV [203]. In the same study, however, subsequent treatment of these infected lung cells with NF-&#954;B inhibitors did not affect virus titres but reduce expression of TNF, CCL2 and CXCL2, hence suggesting that NF-&#954;B is essential for SARS-CoV-mediated induction of pro-inflammatory cytokines [203]. HCoV-229E was also shown to mediate IL8 induction in peripheral blood mononuclear cells (PBMC), which could be attenuated by NF-&#954;B inhibitor [204]. Modulation of NF-&#954;B is mediated via several viral proteins of HCoVs (Figure 5).\n7.1.1. Structural Proteins\nIn SARS-CoV, the S, M, E and N structural proteins have been demonstrated to interfere with NF-&#954;B signalling. A recent study reported an enhanced nuclear NF-&#954;B activity in PBMCs treated with a purified and recombinant SARS-CoV S protein. Synthesis and secretion of IL8 in these S protein-treated cells could be suppressed by NF-&#954;B inhibitor, hence suggesting that NF-&#954;B regulates pro-inflammatory cytokine levels in these cells [204]. SARS-CoV S protein likely stimulates NF-&#954;B via the upregulation of an upstream protein kinase C (PKC) isozyme PKC&#945;, since S activated ERK and JNK were PKC dependent [192]. Although IL8 synthesis and secretion was not promoted by SARS-CoV E protein [204], deletion of E protein in recombinant SARS-CoV reduced NF-&#954;B activation [203]. Additionally, overexpression of SARS-CoV N protein significantly increased NF-&#954;B luciferase activity in a dose-dependent manner in Vero E6 cells, but not Vero, HeLa and Huh-7 cells, suggesting that this induction of NF-&#954;B might be cell-specific [205,206]. On the other hand, co-immunoprecipitation experiments showed that SARS-CoV M protein physically bind to IKKb to sequester it in the cytoplasm, hence inhibiting the activation of NF-&#954;B [207]. However, the MERS-CoV M protein did not affect the luciferase activity controlled by a promoter with NF-&#954;B binding sites, although we cannot rule out the possibility that some other structural or non-structural MERS-CoV protein might be involved [133]. For HCoV-OC43, expression of its N protein alone was unable to activate NF-&#954;B, unless under the stimulation of TNF&#945;. This potentiation of NF-&#954;B activation was through the interaction between HCoV-OC43 N and microRNA 9, which inhibits NF-&#954;B [208].\n7.1.2. Nonstructural and Accessory Proteins\nPreviously, it was demonstrated that overexpression of SARS-CoV nsp1, but not HCoV-229E nsp1, could induce NF-&#954;B activation [138]. The use of NF-&#954;B inhibitor suppressed SARS-CoV nsp1-induced chemokine expression in a dose-dependent manner [209]. The PLpro domain found within nsp3 of SARS-CoV and MERS-CoV was shown to antagonize IFN and NF-&#954;B activities [139,210]. However, in another study, SARS-CoV PLpro domain did not significantly negate the induction of expression of NF&#954;B-dependent genes by Sendai virus infection [140]. Nevertheless, it was shown that SARS-CoV PLpro repressed NF-&#954;B activation by removing K48-linked ubiquitination from I&#954;B&#945; [211].Other accessory proteins of HCoVs have also been shown to interfere with NF-&#954;B signalling. Expression of SARS-CoV 3a and 7a proteins was shown to significantly induce NF-&#954;B-dependent luciferase activity. Enhancement of IL8 promoter activity by SARS-CoV 3a and 7a protein was negated by mutating the NF-&#954;B binding site on the promoter [192]. Previously, expression of MERS-CoV ORF4a, but not ORF4b and ORF5, was shown to inhibit Sendai virus-induced firefly luciferase activity under the control of a NF&#954;B-responsive promoter [133]. However, in another study, MERS-CoV ORF4b was demonstrated to moderately attenuate NF-&#954;B-dependent luciferase activity induced upon TNF&#945; treatment [212].\n8. Conclusions\nThe relationship between a virus and its host is a complicated affair: a myriad of factors from the virus and the host are involved in viral infection and consequential pathogenesis. During viral infections, the host must respond to the virus by putting multiple lines of defence mechanisms in place. As intracellular obligate parasites, viruses have also evolved various strategies to hijack the host machineries. In this review, we first showed how viral factors could manipulate the host cell to expedite its own replication cycle and pathogenesis. We also highlighted how multiple cellular and viral factors come into play in their long-standing battle against one another. For years, HCoVs have been identified as mild respiratory pathogens that affect the human population. However, it was the emergence of SARS-CoV that thrust these human viruses into the spotlight of the research field. Therefore, most of the HCoV research today is pertained towards SARS-CoV. While the recent MERS-CoV outbreak has been mostly limited to the Middle East region, it is likely that more emerging or re-emerging HCoVs might surface to threaten the global public health, as seen from the high mortality rates in the past two outbreaks: SARS-CoV (10%) and MERS-CoV (35%). Therefore, study of the pathogenesis of all HCoVs would gain more insights for the development of antiviral therapeutics and vaccines.",
        "pmid": "28933406",
        "title": "Human Coronaviruses: A Review of Virus&#8211;Host Interactions",
        "journal_title": "Diseases",
        "authors": "Yvonne Xinyi Lim;Yan Ling Ng;James P. Tam;Ding Xiang Liu",
        "doi": "10.3390/diseases4030026"
    },
    {
        "keywords": "",
        "abstract": "Many coronaviruses are capable of interspecies transmission. Some of them have caused\nworldwide panic as emerging human pathogens in recent years, e.g., severe acute\nrespiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome\ncoronavirus (MERS-CoV). In order to assess their threat to humans, we explored to\ninfer the potential hosts of coronaviruses using a dual-model approach based on\nnineteen parameters computed from spike genes of coronaviruses. Both the support\nvector machine (SVM) model and the Mahalanobis distance (MD) discriminant model\nachieved high accuracies in leave-one-out cross-validation of training data\nconsisting of 730 representative coronaviruses (99.86% and 98.08% respectively).\nPredictions on 47 additional coronaviruses precisely conformed to conclusions or\nspeculations by other researchers. Our approach is implemented as a web server that\ncan be accessed at http://bioinfo.ihb.ac.cn/seq2hosts.",
        "body": "Results\n\nNucleotide composition analysis\nNineteen parameters, including three mononucleotide frequencies (G, C and T) and\n16 dinucleotide biases, were computed from 777 spike gene sequences (see Supplementary Table S1). All\nparameters show significant differences across the host groups (Kruskal-Wallis\ntests, p < 2.2e&#8211;16); therefore, they were subsequently\nemployed as factors in statistical models for discriminant analyses.\nEmpirically, a dinucleotide relative abundance or dinucleotide bias (e.g.,\n) is significantly high if  or extremely low if 24. Among the 16 dinucleotides in this study, the CpA and TpG show\nan average abundance that is significantly higher than the expected values\n(&#8201;=&#8201;1.29,\n&#8201;=&#8201;1.28), whereas\nthe average bias of CpG is extremely low (&#8201;=&#8201;0.44). This result indicates that the observed\nabundances of CpA and TpG are significantly higher than their expected values,\nand the observed abundance of CpG is significantly lower than the expected\nvalue. The G+C content is minimal (31&#8211;47%). This finding indicates\nthat coronaviruses exhibit a low density of nucleotide sequences and may be\nsensitive to heat or alkali. The low G+C content also indicates a preference for\ncodons ending with A or T and a higher mutability.\nTraining and validation of statistical models\nThe data matrix with 19 factors as columns and 730 samples as rows was fitted to\nSVM and MD models, all predictions in leave-one-out cross-validations were\nlisted in Supplementary Table S2 and\nsummarized in Table 1 according to host species. The\nvalidations indicate that both models achieved high accuracies on the training\ndata set: 99.86% for the SVM and 98.08% for the MD. All incorrect cases in\nunsupervised predictions are listed in Table 2. The only\nincorrect prediction by the SVM is sample NC_016996.1, which is isolated from an\navian species but was predicted to infect humans. Among all 14 incorrect\npredictions by MD, bats are the common predicted hosts. No sample was\nincorrectly predicted by both models.\nPredictions for viruses capable of interspecies transmission\nThe trained models were applied to 47 additional samples, and the predictions\nunveiled clues regarding potential interspecies transmission (See Table 3). Sequences 1&#8211;31 comprise spike genes\nof coronaviruses that were primarily isolated from palm civets from restaurants,\nanimal markets, or farms in southern China when SARS wreaked havoc in 2003. The\nsequences of these coronaviruses (civet-CoVs) are similar not only to each other\nbut also to SARS-CoV. Cross-host evolution research of SARS-CoV in palm civet\nand humans indicated that the variations in spike genes seemed to be essential\nfor the transition of coronavirus from animal-to-human transmission to\nhuman-to-human transmission25. In addition to\ncross-neutralization with SARS-CoV, these SARS-like civet-CoVs can use human\nACE2 as an entry receptor26. Bats are the reservoir hosts of a\nnumber of coronaviruses, and a recent study also suggests that bats are natural\nreservoirs of these SARS-like coronaviruses, whereas palm civets and humans are\nintermediate hosts1. All hosts predicted by the SVM are humans,\nwhich supports the previously mentioned research. The MD identified both bats\nand humans as hosts of these samples, but bats are the preferable hosts for\nsamples 1&#8211;26 and the second choice for samples 27&#8211;31.\nThis finding is also expected as bats are considered to be natural hosts of\nthese viruses.Sequences 32&#8211;40 comprise spike genes of MERS-CoVs from dromedaries\nafter the outbreak in the Middle East in 2012. MERS-CoVs are similar to the bat\ncoronaviruses HKU5 and HKU4 in their amino acid sequences27, and\nthey can use human DPP4 as an entry receptor28. MERS-CoVs was\nassumed to originate from HKU5 in pipistrelle, which is a type of Japanese\nbat3. In our study, these MERS-CoVs isolated from camels were\npredicted to be capable of infecting humans; and bats are also likely hosts next\nto humans in predictions by MD. This result is obviously consistent with above\nspeculations, and also supports the WHO advices about avoiding close contact\nwith camels (http://www.who.int/csr/don/2014_07_23_mers/en/).The 41st sample was a SARS-associated coronavirus that was transmitted from human\nto pig29, and both SVM and MD detected its threat to humans. Bat\nand avian might be potential hosts since both models suggest that they are more\nvulnerable than porcine. Samples 42&#8211;44 (RsSHC014, Rs3367 and\nSL-CoV-WIV1) consist of three SARS-like coronaviruses from bats30. Analyses based on the sequence similarities and cultures in the cell lines\nsuggest that Rs3367 and SL-CoV-WIV1 are capable of using a SARS-CoV receptor for\ncell entry and pose a threat to humans, whereas RsSHC014 cannot30. Our study provides a precise support to these conclusions. The MD correctly\npredicts bats as the natural hosts of the three viruses, and the SVM indicates\nthat Rs3367 and SL-CoV-WIV1 are harmful to humans.The 45th sample was isolated from an alpaca by Jin et al. in 2007 with a\nserotype of bovine; the phylogenetic analysis suggests that it shares the same\nancestor with bovine-coronaviruses31. Our analysis supports the\nfinding that this coronavirus is capable of infecting bovine. These analyses\nimply that this strain is capable of interspecies transmission between bovines\nand alpacas. Samples 46 and 47 are enteric coronaviruses from bovines and\nhumans; they have been identified as the same strain named &#8220;Human\nenteric coronavirus 4408&#8221; in the NCBI database due to the similarity\nbetween their spike protein sequences of 99.9%. Although they are similar to the\nhuman coronavirus OC43 and the bovine coronavirus, evidences from morphological,\nimmunological, and genomic studies indicate that they are closer to bovine\ncoronavirus than to human coronavirus (unpublished research, from personal\ncommunication). This finding is consistent with our analysis. In addition, avian\nand bat are worthy of attentions as potential hosts due to the small MD\nvalues.\nTendencies of MD and SVM in predictions\nTwo groups of two-dimensional data are plotted in Fig. 1.\nThe blue points represent a &#8220;loose&#8221; population with a\nlarger standard deviation (SD) of N(1, 1), and the red points represent a\n&#8220;tight&#8221; population with a smaller SD of N(3.5,\n0.5). The red line separates the two groups classified by the MD, and the groups\npredicted by the SVM are delimited by the blue line. In this figure, two\nindividuals (the red triangles between the two lines) from the\n&#8220;tight&#8221; population were classified into the\n&#8220;loose&#8221; group by the MD, whereas the SVM accidentally\nexcluded four points (the blue reversed triangles between the two lines) from\nthe &#8220;loose&#8221; population. This example shows that MD and\nSVM have inverse tendencies in some cases, i.e., when a\n&#8220;loose&#8221; population is close to a\n&#8220;tight&#8221; population, MD intends to classify outliers of\nthe &#8220;tight&#8221; population into the former. The opposite\nsituation is valid for the SVM.\nDiscussion\nNucleotide composition analysis revealed the overrepresentation of CpA and TpG\ndinucleotides and the suppression of CpG dinucleotides (see Supplementary Table S1), which indicates that\ncoronaviruses generally prefer motifs that contain CpAs and TpGs and avoid CpGs in\nsequences. These dinucleotide biases are common characteristics of RNA viruses in\nvertebrates11121516. As most vertebrates exhibit a very low\nCpG representation in genomes, RNA viruses may gradually adapt to the accumulation\nof host mutations and mimic the host gene&#8217;s dinucleotide patterns for\nsurvival11. For DNA viruses, the most-accepted mechanisms for the\nsuppression of CpG dinucleotides are the methylation of CpG nucleotides and the\nsubsequent deamination of 5-methylcytosine, which renders CpG a mutational\nhotspot24. For RNA viruses, a different hypothesis is that the\nRNA viruses encounter different selection pressures when they switch to a new host,\nand viral RNA genes mimic host mRNAs to avoid immune detection11.\nSimilar to other human ssRNA viruses, coronaviruses show a strong correlation\nbetween CpG pressure and C+G content (Pearson&#8217;s correlation coefficient,\nr&#8201;=&#8201;0.5443,\np&#8201;<&#8201;2.2e-16, our data). A lower C+G content\nusually indicates that the nucleotide sequence of the virus is unstable or is highly\nvariable under evolutionary selection pressure. Considering that the mutation rates\nfor RNA viruses are significantly higher than the mutation rates for DNA\nviruses32, mutational pressure may be the most important\ndeterminant of the bias in codon usage in human RNA viruses, such as\ncoronaviruses14.The capabilities to bind with receptors and to replicate in host cells are essential\nfor any virus to infect hosts. Different genes contribute to these biological\nprocesses. Variations on these genes may enable a virus to transmit cross-species.\nOne famous example would be the polymerase 2 (PB2) of influenza A virus, in which\namino acid change from E to K at its 627th position would render the virus to\nreplicate in mammalian cells333435. In coronaviruses, the spike\nprotein is functionally associated with recognition of hosts and the RNA-dependent\nRNA polymerase (RdRp) is related to proliferation of virus. However, there are two\nobstacles limiting the use of RdRp gene: (1) The similarities among nucleotide\nsequences is too high to train MD model, i.e., the variation rate of RdRp sequence\nis slower and cannot provide enough resolution to discriminate different\ncoronaviruses; (2) Even worse, available full-length CDSs in public databases are\nvery limited &#8212; only 23 or so. On the contrary, the spike gene perfectly\nsatisfied the requirements for variation rate and availability, therefore was\nadopted as markers in this study.MD and SVM show opposite tendencies in judging outliers (See Fig.\n1), which reflects the different principles of the two classification\napproaches. Unlike the Euclidian distance (ED), which measures the absolute distance\nbetween points or mass centres in space, the Mahalanobis distance considers the\nvariances within a population and the covariance between variables. In some cases,\nespecially when a population with individuals who are scattered across a wide range\nis located close to a &#8220;tight&#8221; population with smaller\ninternal variations, the MD may classify marginal individuals from the latter into\nthe &#8220;loose&#8221; population even if they are\n&#8220;close&#8221; to a &#8220;tight&#8221; population\naccording to the ED. The MD enables &#8220;loose&#8221; populations to\nhave a greater number of points. The SVM has a different philosophy. SVM separates\npopulations by finding a hyperplane that maximizes the distances between\npopulations. When a &#8220;loose&#8221; population is close to the\nboundary of a &#8220;tight&#8221; population, SVM is more likely to find\nthis hyperplane within the former. This finding explains SVM&#8217;s tendency\nto exclude outliers from a &#8220;loose&#8221; population.Bats are the reservoir hosts of a number of coronaviruses that can survive in bats\nand accumulate variations in the long evolutionary process13637.\nThus, coronaviruses in bats constitute a &#8220;loose&#8221; population\nwith larger internal gaps. We assume that some strains of viruses in bats gain\nsufficient variation to enable them to infect other organisms; these viruses form a\nnew &#8220;tight&#8221; population at the edge of the original group. In\nthis case, the MD emphasizes the connection of a virus with the original source,\nwhereas the SVM may be more sensitive to the possibility of infecting new hosts.\nTherefore, the incorporation of analyses using the MD and SVM can be especially\nhelpful for revealing the profile of interspecies transmission.According to the predictions by MD, bats are not only the hosts in all 14 incorrect\ncases from training data set (See Table 2), but also in the\nhost list of each coronaviruses for testing (See Table 3).\nFurthermore, bats were predicted to host of 64.02% training samples isolated from\nother hosts (See Table 1). These facts convincingly support\nthe notion that these viruses originated from bats and shifted to other hosts.Next to bats, avians could be infected by 36.64% samples from other hosts. If bats\nare the only reservoir hosts and coronaviruses spread from bats to avians and other\nanimals, according to the stochastic event model, the probability of co-infectivity\nto both bat and avian can be the product of the infectivity probabilities to each of\nthem, i.e., 0.3494 (0.5164&#8201;&#215;&#8201;0.6767, see Table 1), then 255\n(0.3494&#8201;&#215;&#8201;730) samples are expected to be of\nco-infectivity. However, only 173 samples were predicted to be of co-infectivity to\nbats and avians. So avians might be the second independent source of coronavirus in\nparallel to bats. If this speculation is true, people will have to maintain\nvigilance to avian coronaviruses apart from avian influenza viruses. Especially, due\nto the high accuracy of the SVM in cross-validation, we should seriously consider\nits only &#8220;wrong&#8221; prediction: perhaps it is sensible to\ninvestigate whether the NC_016996.1 virus from avian is capable of infecting\nhumans.For the viruses that are capable of spreading across a host species barrier, the\ncombination of the MD and the SVM is valuable for assessing their potential threat.\nThe origin and interspecies transmission of coronaviruses have been extensively\ndiscussed in the past ten years, and the coronaviruses of most mammals are believed\nto originate from their ancestors in bats13637. Our analysis with\ndual statistical models support the finding that SARS-CoVs and MERS-CoVs spread from\nbats to humans and other animals. In most cases, our approach provided convincing\npredictions. The dual-model approach can be expected to become a useful tool in\nfuture studies. Typically, when a novel coronavirus is isolated, the combination of\nthe MD and the SVM may provide meaningful hints regarding its origin and potential\nthreat to humans or other animals. As soon as more virus genomes are sequenced, this\napproach can be applied to investigate the interspecies transmission route of other\nthreatening viruses, including the recent Ebola outbreak in West Africa.\nMethods\n\nData preparation\nAll genome sequences and complete coding sequences (CDSs) of spike genes were\ndownloaded from the National Centre for Biotechnology Information (NCBI)\ndatabase (http://www.ncbi.nlm.nih.gov/) on July 17, 2014. Sequences of\nspike genes were extracted from the 1044 coronavirus genomes and pooled with\n1380 downloaded CDSs. Then, we removed replicate sequences and sequences that\ncontained non-standard bases or were incapable of coding complete products. The\nlength of each sequence is longer than 3,000 bases. Among all 777 valid\nnucleotide sequences that are listed in Supplementary Data S1, 730 sequences fall into six categories\naccording to different hosts: 196 for humans, 182 for porcines, 77 for bovines,\n74 for bats, 28 for murines and 173 for avians. The majority of the remaining 47\nviruses were isolated from the two epidemic diseases caused by the coronavirus\nin the past 12 years. Although we only listed the hosts from which they were\nisolated, these viruses have been verified or suspected to have the ability to\ninfect different hosts; thus, all 47 sequences were employed to explore\ninterspecies transmission of coronaviruses. Viruses from other mammals,\nincluding canines, felines, rabbits, equines, alpacas and whales, were excluded\nfrom the data set as the number of spike sequences for each host is insufficient\nfor establishing a separate group.\nNucleotide composition analysis\nThe mononucleotide frequencies and dinucleotide biases of the spike sequences\nwere computed using our original Python scripts. Dinucleotide bias is the ratio\nof the observed value to the expected frequency of each of the 16 dinucleotides:\n, where  is\nthe dinucleotide bias, fXY is the frequency of\ndinucleotide XY, fX and fY are\nthe frequencies of nucleotide X and nucleotide Y38,\nrespectively.In this study, we considered 19 factors, including three mononucleotide\nfrequencies (G, C and T) and 16 dinucleotide biases. As none of the frequencies\nhas a normal distribution, the nonparametric &#8220;Kruskal-Wallis\nTest&#8221; was employed to investigate the difference in each factor\namong six categories. As a result, significant differences across categories\nwere detected for each factor; thus, all 19 factors were employed for\nmodelling.\nModelling, validation and prediction\nAs a classifier, the SVM can efficiently perform a nonlinear classification using\na kernel technique that is rooted in structural risk minimization. In this\nstudy, the R package e1071 (Version: 1.6&#8211;3)20 was\nemployed for the SVM analysis. &#8220;C-classification&#8221; was\nadopted as the model type and &#8220;Radial&#8221; was adopted as\nthe SVM kernel in our analysis. The MD is a measure of the distance from a point\nto the centre of a distribution; the principle of this discriminant is that\nindividuals belong to the closest group in the distance. The MD is defined as\n, where X denotes the population,\nx denotes the individual, &#956; is the mean value of\nthe population, T denotes the matrix transpose, and  denotes the covariance matrix of population39. The R program\n&#8220;distinguish.distance.R&#8221;40 was\nemployed in the MD analysis. Leave-one-out cross-validation was employed for\nboth SVM and MD analyses.When the trained models are applied to a sequence for testing, each of the six\ncategories of hosts will obtain a p value from SVM and a MD value. Based\non p values and MD values, three steps will be taken to determine\ncandidate hosts. First, the host of minimal p value or MD value is\nreasonably regarded as the preferable host. Then, two adjustable empirical\nthresholds can be used for each model to pick out other potential hosts. In this\nstudy, we adopted 0.05 and 0.01 for p value, 200 and 100 for MD value;\ni.e., likely hosts were determined if\np&#8201;<=&#8201;0.05 or\nMD&#8201;<=&#8201;200, and very likely hosts were defined by\np&#8201;<=&#8201;0.01 or\nMD&#8201;<=&#8201;100. The two steps are unsupervised\nprediction. In case that the isolate source is among the six host groups for\nmodelling, a supervised prediction can be applied as the third step, i.e., all\nhost species with p values or MD values no more than those of the\nobserved host will be listed as potential hosts, which can be practical\nreferences for researchers to evaluate a virus&#8217;s threats to human or\nother animals.\nCompare the tendencies of MD and SVM in predictions\nTwo groups of two-dimensional vectors were generated in silico as two\npopulations. The number of vectors in the first population are randomly\ngenerated from the normal distribution N(1, 1), and the number of vectors\nin the second population are randomly generated from N(3.5, 0.5). As the\nfirst population has a larger standard deviation (SD), we refer to it as the\n&#8220;loose&#8221; population and refer to the second population as\nthe &#8220;tight&#8221; population. The two groups of data are\nemployed for the leave-one-out cross-validations of MD and SVM.All Python and R scripts employed in this study are available from the authors\nupon request. The prediction can be performed using the spike gene sequences of\nthe coronaviruses on our web server, which is available to the public at no cost\nat http://bioinfo.ihb.ac.cn/seq2hosts.\nAdditional Information\nHow to cite this article: Tang, Q. et al. Inferring the hosts of\ncoronavirus using dual statistical models based on nucleotide composition. Sci.\nRep.\n5, 17155; doi: 10.1038/srep17155 (2015).\nSupplementary Material\n",
        "pmid": "26607834",
        "title": "Inferring the hosts of coronavirus using dual statistical models based on nucleotide\ncomposition",
        "journal_title": "Scientific Reports",
        "authors": "Qin Tang;Yulong Song;Mijuan Shi;Yingyin Cheng;Wanting Zhang;Xiao-Qin Xia",
        "doi": "10.1038/srep17155"
    },
    {
        "keywords": "canine coronavirus;RT-PCR;serological prevalence;serotype",
        "abstract": "We collected rectal swabs from dogs in Japan during 2011 to 2014, and canine coronavirus (CCoV) nucleocapsid\ngene was detected by RT-PCR. The relationship between CCoV infection and the manifestation of diarrhea\nsymptoms was investigated, and a correlation was noted (df=1, &#967;2=8.90,\nP<0.005). The types of CCoV detected in samples from CCoV-infected dogs were CCoV-I in\n88.9% and CCoV-II in 7.4%, respectively. We retrospectively investigated the seroprevalence of CCoV-I in dogs\nin Japan during 1998 to 2006. The sera were tested with a neutralizing antibody test. In the absence of CCoV-I\nlaboratory strain, we used feline coronavirus (FCoV)-I that shares high sequence homology in the S protein\nwith CCoV-I. 77.7% of the sera were positive for neutralizing anti-FCoV-I antibodies.",
        "body": "",
        "pmid": "26460314",
        "title": "Prevalence of canine coronavirus (CCoV) in dog in Japan: detection of CCoV RNA and retrospective\nserological analysis",
        "journal_title": "The Journal of Veterinary Medical Science",
        "authors": "Tomomi TAKANO;Saya YAMASHITA;Michiko MURATA-OHKUBO;Kumi SATOH;Tomoyoshi DOKI;Tsutomu HOHDATSU",
        "doi": "10.1292/jvms.15-0347"
    }
]



var words = {}; //Obj with splitted words
var counter = {}; //Obj with all words and how many repeats
var sortable = []; //Sorted counter in format array of {word, counter}
let res = {}; //Return value


// Parse keywords
const splitWords = (str, i, id) => {
    let w = 1;
    let k = 0;
    words[i] = {};
    while (str[k]) {
        if (str[k] === ";") {
            counter[words[i][w]] = (counter[words[i][w]] || 0) + 1;
            k++;
            w++;
        }
        words[i][w] = (words[i][w] || '') + str[k];
        k++;
    }
    words[i]["id"] = id;
}
// sorting from high to low
const Sort = (counter) => {    
for (w in counter) {
    sortable.push([w, counter[w]]);
    }
    sortable.sort((a, b) => {
        return b[1] - a[1];
    });
}
// Create obj with keyword to prevent errors
const createObj = (w) => {
    if (res[w])
        return;
    else
        res[w] = {};
}

//Finding exact article in example
const findArticle = (str) => {
    const id = str["id"];
    for (k in example) {
        if (example[k]["pmid"] === id)
            return example[k];
    }
}
// Checking is article already in res obj
const checkArticle = (art) => {
    for (k in res) {
        for (j in res[k]) {
            if (res[k][j] === art)
                return true;
        }
    }
    return false;
}

const sortedRes = () => {
    let i = 0; //variable to able add multi obj to one word exmaple: {"coronavirus": {{i: "..."}, {i + 1: "..."}, {i + 2: "..."}}
    let z = 0; //variable to loop thru sortable
    res['Other'] = {};
    while (sortable[z]) {
        if (z >= 29) { 
            for (k in words) {
                for (j in words[k]) {
                    let art = findArticle(words[k] || "");
                    if (art) {
                     if (!checkArticle(art)) {
                         res['Other'][i] = art;
                        }
                    }
                    i++;
                }
            }
        }
        else
            for (k in words) {
                for (j in words[k]) {
                    if (!words[k])
                        break;
                    let w = words[k][j];
                    if (words[k][j] === sortable[z][0]) {
                        let art = findArticle(words[k]);
                        if (!checkArticle(art)) {
                            createObj(w);
                            res[w][i] = art;
                        }
                        i++;
                    }
                }
            }
        z++;
    }
}

// Begining
const generator = (example) => {
    let i = 1;
    for (k of example) {
        splitWords((k["keywords"]).toLowerCase(), i, k['pmid']) //pmid using as id for each article. variable I same as string 1070
        i++;
    }
    Sort(counter); //sorting form hight to low
    sortedRes(sortable, words); //Using sortable as  for fill up result
    // console.log(words);
    // console.log(counter)
    // console.log(sortable)
    console.log(res);
    return;
}

generator(example);
// fs.writeFile('./test8.json', JSON.stringify(res), err => {
//     if (err) {
//         console.log('Error writing file', err)
//     } else {
//         console.log('Successfully wrote file')
//     }
// })